{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#copy data from Globus to gannet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir /Volumes/web/metacarcinus/Salmo_Calig/FASTQS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "#see markdown post about how data was transfered from Sequencing core to Gannet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#copy compressed directory to mox"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#untar files \n",
    "#tar -xvf UW_Trigg_190718.tar.gz\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m16C_26psu_1_S13_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[34mReports\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[34mStats\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n"
     ]
    }
   ],
   "source": [
    "!ls /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 100% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 100% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 10% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L002_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/Shared/bioinformatics/FastQC/fastqc \\\n",
    "--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722 \\\n",
    "--threads 4 \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L002_R2_001.fastq.gz \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:44: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  configs = yaml.load(f)\n",
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:50: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  sp = yaml.load(f)\n",
      "[WARNING]         multiqc : MultiQC Version v1.7 now available!\n",
      "[INFO   ]         multiqc : This is MultiQC v1.0.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R1_001_fastqc.html'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R2_001_fastqc.zip'\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]        bargraph : cannot import name 'get_backend'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/bargraph.py\", line 21, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 40, in <module>\n",
      "    import matplotlib._constrained_layout as constrained_layout\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/_constrained_layout.py\", line 52, in <module>\n",
      "    from matplotlib.legend import Legend\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/legend.py\", line 43, in <module>\n",
      "    from matplotlib.offsetbox import HPacker, VPacker, TextArea, DrawingArea\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/offsetbox.py\", line 33, in <module>\n",
      "    from matplotlib.image import BboxImage\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/image.py\", line 19, in <module>\n",
      "    from matplotlib.backend_bases import FigureCanvasBase\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/backend_bases.py\", line 46, in <module>\n",
      "    from matplotlib import (\n",
      "ImportError: cannot import name 'get_backend'\n",
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]       linegraph : module 'matplotlib' has no attribute 'artist'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/linegraph.py\", line 20, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 28, in <module>\n",
      "    import matplotlib.artist as martist\n",
      "AttributeError: module 'matplotlib' has no attribute 'artist'\n",
      "[INFO   ]          fastqc : Found 92 reports\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/bin/multiqc\", line 4, in <module>\n",
      "    __import__('pkg_resources').run_script('multiqc==1.0.dev0', 'multiqc')\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 666, in run_script\n",
      "    self.require(requires)[0].run_script(script_name, ns)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 1453, in run_script\n",
      "    exec(code, namespace, namespace)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 581, in <module>\n",
      "    multiqc()\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 764, in __call__\n",
      "    return self.main(*args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 717, in main\n",
      "    rv = self.invoke(ctx)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 956, in invoke\n",
      "    return ctx.invoke(self.callback, **ctx.params)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 555, in invoke\n",
      "    return callback(*args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 509, in multiqc\n",
      "    report_output = j_template.render(report=report, config=config)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/asyncsupport.py\", line 76, in render\n",
      "    return original_render(self, *args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/environment.py\", line 1008, in render\n",
      "    return self.environment.handle_exception(exc_info, True)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/environment.py\", line 780, in handle_exception\n",
      "    reraise(exc_type, exc_value, tb)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/_compat.py\", line 37, in reraise\n",
      "    raise value.with_traceback(tb)\n",
      "  File \"/var/folders/_v/3qsffc0s69l8d4v9795nknd40000gt/T/tmpe5h8nmy5/base.html\", line 19, in top-level template code\n",
      "    {% include 'includes.html' %}\n",
      "  File \"/var/folders/_v/3qsffc0s69l8d4v9795nknd40000gt/T/tmpe5h8nmy5/includes.html\", line 33, in top-level template code\n",
      "    <style type=\"text/css\">{{ include_file(css_href, None) }}</style>\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 496, in include_file\n",
      "    with io.open (os.path.join(fdir, name), \"r\", encoding='utf-8') as f:\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/posixpath.py\", line 80, in join\n",
      "    a = os.fspath(a)\n",
      "TypeError: expected str, bytes or os.PathLike object, not NoneType\n"
     ]
    }
   ],
   "source": [
    "!cd /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722\n",
    "!/Users/strigg/anaconda3/bin/multiqc ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "#make new directory for trim output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt: File exists\n",
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC: File exists\n"
     ]
    }
   ],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt\n",
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "#trim adapters off reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Multicore support not enabled. Proceeding with single-core trimming.\n",
      "Path to Cutadapt set as: '/Users/strigg/anaconda3/bin/cutadapt' (user defined)\n",
      "Cutadapt seems to be working fine (tested command '/Users/strigg/anaconda3/bin/cutadapt --version')\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 921.66 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,366,785 (78.4%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              20,636,466 bp (0.6%)\n",
      "Total written (filtered):  2,599,584,885 bp (78.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17366785 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2904827\t5539983.8\t0\t2904827\n",
      "2\t423079\t1384995.9\t0\t423079\n",
      "3\t231866\t346249.0\t0\t231866\n",
      "4\t143449\t86562.2\t0\t143449\n",
      "5\t117570\t21640.6\t0\t117570\n",
      "6\t115529\t5410.1\t0\t115529\n",
      "7\t96119\t1352.5\t0\t96119\n",
      "8\t119364\t338.1\t0\t119364\n",
      "9\t118406\t84.5\t0\t116886 1520\n",
      "10\t112051\t21.1\t1\t106032 6019\n",
      "11\t122987\t5.3\t1\t113734 9253\n",
      "12\t119477\t1.3\t1\t111842 7635\n",
      "13\t120023\t0.3\t1\t112246 7777\n",
      "14\t122963\t0.3\t1\t114085 8878\n",
      "15\t127528\t0.3\t1\t119209 8319\n",
      "16\t126445\t0.3\t1\t117002 9443\n",
      "17\t152761\t0.3\t1\t140488 12273\n",
      "18\t129037\t0.3\t1\t121213 7824\n",
      "19\t112889\t0.3\t1\t105773 7116\n",
      "20\t133616\t0.3\t1\t125004 8612\n",
      "21\t146516\t0.3\t1\t136291 10225\n",
      "22\t126419\t0.3\t1\t119045 7374\n",
      "23\t145391\t0.3\t1\t135716 9675\n",
      "24\t136894\t0.3\t1\t125670 11224\n",
      "25\t145117\t0.3\t1\t135313 9804\n",
      "26\t134897\t0.3\t1\t126717 8180\n",
      "27\t148757\t0.3\t1\t137734 11023\n",
      "28\t145635\t0.3\t1\t135492 10143\n",
      "29\t153188\t0.3\t1\t140948 12240\n",
      "30\t163373\t0.3\t1\t154030 9343\n",
      "31\t130689\t0.3\t1\t122129 8560\n",
      "32\t157390\t0.3\t1\t147156 10234\n",
      "33\t145226\t0.3\t1\t135269 9957\n",
      "34\t158524\t0.3\t1\t147826 10698\n",
      "35\t161582\t0.3\t1\t148734 12848\n",
      "36\t173098\t0.3\t1\t162838 10260\n",
      "37\t141921\t0.3\t1\t133053 8868\n",
      "38\t164933\t0.3\t1\t154468 10465\n",
      "39\t167185\t0.3\t1\t155913 11272\n",
      "40\t175799\t0.3\t1\t163207 12592\n",
      "41\t197239\t0.3\t1\t176427 20812\n",
      "42\t165247\t0.3\t1\t150517 14730\n",
      "43\t270149\t0.3\t1\t254538 15611\n",
      "44\t85600\t0.3\t1\t76961 8639\n",
      "45\t197628\t0.3\t1\t175697 21931\n",
      "46\t723044\t0.3\t1\t688287 34757\n",
      "47\t254889\t0.3\t1\t240372 14517\n",
      "48\t114855\t0.3\t1\t101708 13147\n",
      "49\t409546\t0.3\t1\t390013 19533\n",
      "50\t125438\t0.3\t1\t116192 9246\n",
      "51\t110069\t0.3\t1\t99667 10402\n",
      "52\t381945\t0.3\t1\t364475 17470\n",
      "53\t196688\t0.3\t1\t188239 8449\n",
      "54\t63003\t0.3\t1\t58174 4829\n",
      "55\t106109\t0.3\t1\t101117 4992\n",
      "56\t55521\t0.3\t1\t50154 5367\n",
      "57\t169184\t0.3\t1\t161794 7390\n",
      "58\t35182\t0.3\t1\t32212 2970\n",
      "59\t44244\t0.3\t1\t39185 5059\n",
      "60\t173147\t0.3\t1\t165425 7722\n",
      "61\t86440\t0.3\t1\t82402 4038\n",
      "62\t40559\t0.3\t1\t36479 4080\n",
      "63\t130914\t0.3\t1\t123404 7510\n",
      "64\t122315\t0.3\t1\t116337 5978\n",
      "65\t65805\t0.3\t1\t60428 5377\n",
      "66\t148115\t0.3\t1\t139100 9015\n",
      "67\t175930\t0.3\t1\t163369 12561\n",
      "68\t293688\t0.3\t1\t278052 15636\n",
      "69\t189726\t0.3\t1\t179114 10612\n",
      "70\t75474\t0.3\t1\t69457 6017\n",
      "71\t27345\t0.3\t1\t23952 3393\n",
      "72\t97969\t0.3\t1\t91968 6001\n",
      "73\t138145\t0.3\t1\t130111 8034\n",
      "74\t143624\t0.3\t1\t135279 8345\n",
      "75\t142674\t0.3\t1\t134066 8608\n",
      "76\t141682\t0.3\t1\t133308 8374\n",
      "77\t141205\t0.3\t1\t132947 8258\n",
      "78\t133036\t0.3\t1\t124966 8070\n",
      "79\t131630\t0.3\t1\t123633 7997\n",
      "80\t128859\t0.3\t1\t121116 7743\n",
      "81\t126119\t0.3\t1\t118667 7452\n",
      "82\t123492\t0.3\t1\t116262 7230\n",
      "83\t118847\t0.3\t1\t111615 7232\n",
      "84\t115583\t0.3\t1\t108642 6941\n",
      "85\t111409\t0.3\t1\t104817 6592\n",
      "86\t107099\t0.3\t1\t100545 6554\n",
      "87\t102731\t0.3\t1\t96477 6254\n",
      "88\t101527\t0.3\t1\t95245 6282\n",
      "89\t95383\t0.3\t1\t89600 5783\n",
      "90\t95037\t0.3\t1\t89230 5807\n",
      "91\t89836\t0.3\t1\t84395 5441\n",
      "92\t84977\t0.3\t1\t79865 5112\n",
      "93\t79479\t0.3\t1\t74762 4717\n",
      "94\t76672\t0.3\t1\t72010 4662\n",
      "95\t70676\t0.3\t1\t66313 4363\n",
      "96\t66886\t0.3\t1\t62766 4120\n",
      "97\t63105\t0.3\t1\t59150 3955\n",
      "98\t58813\t0.3\t1\t55101 3712\n",
      "99\t56801\t0.3\t1\t52921 3880\n",
      "100\t51251\t0.3\t1\t47973 3278\n",
      "101\t48131\t0.3\t1\t44969 3162\n",
      "102\t43975\t0.3\t1\t41134 2841\n",
      "103\t40749\t0.3\t1\t38132 2617\n",
      "104\t37602\t0.3\t1\t35059 2543\n",
      "105\t33601\t0.3\t1\t31318 2283\n",
      "106\t30687\t0.3\t1\t28622 2065\n",
      "107\t27550\t0.3\t1\t25698 1852\n",
      "108\t25919\t0.3\t1\t24135 1784\n",
      "109\t23271\t0.3\t1\t21584 1687\n",
      "110\t20524\t0.3\t1\t19077 1447\n",
      "111\t18563\t0.3\t1\t17263 1300\n",
      "112\t16204\t0.3\t1\t15036 1168\n",
      "113\t14110\t0.3\t1\t13017 1093\n",
      "114\t12321\t0.3\t1\t11394 927\n",
      "115\t10629\t0.3\t1\t9884 745\n",
      "116\t9483\t0.3\t1\t8756 727\n",
      "117\t8057\t0.3\t1\t7440 617\n",
      "118\t7067\t0.3\t1\t6491 576\n",
      "119\t5777\t0.3\t1\t5264 513\n",
      "120\t4917\t0.3\t1\t4511 406\n",
      "121\t4181\t0.3\t1\t3800 381\n",
      "122\t3506\t0.3\t1\t3174 332\n",
      "123\t2912\t0.3\t1\t2640 272\n",
      "124\t2546\t0.3\t1\t2297 249\n",
      "125\t2101\t0.3\t1\t1902 199\n",
      "126\t1717\t0.3\t1\t1548 169\n",
      "127\t1536\t0.3\t1\t1361 175\n",
      "128\t1201\t0.3\t1\t1095 106\n",
      "129\t1012\t0.3\t1\t912 100\n",
      "130\t739\t0.3\t1\t668 71\n",
      "131\t638\t0.3\t1\t572 66\n",
      "132\t553\t0.3\t1\t500 53\n",
      "133\t556\t0.3\t1\t500 56\n",
      "134\t665\t0.3\t1\t607 58\n",
      "135\t362\t0.3\t1\t324 38\n",
      "136\t207\t0.3\t1\t176 31\n",
      "137\t143\t0.3\t1\t128 15\n",
      "138\t99\t0.3\t1\t86 13\n",
      "139\t84\t0.3\t1\t72 12\n",
      "140\t58\t0.3\t1\t50 8\n",
      "141\t42\t0.3\t1\t38 4\n",
      "142\t37\t0.3\t1\t29 8\n",
      "143\t24\t0.3\t1\t22 2\n",
      "144\t25\t0.3\t1\t17 8\n",
      "145\t26\t0.3\t1\t22 4\n",
      "146\t29\t0.3\t1\t21 8\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t64\t0.3\t1\t59 5\n",
      "149\t40\t0.3\t1\t35 5\n",
      "150\t355\t0.3\t1\t318 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4155 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 979.66 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,263,070 (77.9%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              16,760,179 bp (0.5%)\n",
      "Total written (filtered):  2,603,569,143 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17263070 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2980698\t5539983.8\t0\t2980698\n",
      "2\t434290\t1384995.9\t0\t434290\n",
      "3\t244115\t346249.0\t0\t244115\n",
      "4\t143736\t86562.2\t0\t143736\n",
      "5\t130207\t21640.6\t0\t130207\n",
      "6\t112454\t5410.1\t0\t112454\n",
      "7\t91383\t1352.5\t0\t91383\n",
      "8\t130413\t338.1\t0\t130413\n",
      "9\t102897\t84.5\t0\t102495 402\n",
      "10\t113241\t21.1\t1\t107719 5522\n",
      "11\t119188\t5.3\t1\t111572 7616\n",
      "12\t120663\t1.3\t1\t112735 7928\n",
      "13\t133942\t0.3\t1\t125965 7977\n",
      "14\t126547\t0.3\t1\t117452 9095\n",
      "15\t102830\t0.3\t1\t96138 6692\n",
      "16\t149163\t0.3\t1\t139761 9402\n",
      "17\t119889\t0.3\t1\t111631 8258\n",
      "18\t113552\t0.3\t1\t106594 6958\n",
      "19\t144826\t0.3\t1\t135202 9624\n",
      "20\t135016\t0.3\t1\t126722 8294\n",
      "21\t118040\t0.3\t1\t110327 7713\n",
      "22\t135122\t0.3\t1\t126795 8327\n",
      "23\t139623\t0.3\t1\t130868 8755\n",
      "24\t139327\t0.3\t1\t130312 9015\n",
      "25\t156486\t0.3\t1\t147346 9140\n",
      "26\t131367\t0.3\t1\t122873 8494\n",
      "27\t144343\t0.3\t1\t134126 10217\n",
      "28\t135732\t0.3\t1\t128445 7287\n",
      "29\t144460\t0.3\t1\t135341 9119\n",
      "30\t143460\t0.3\t1\t135475 7985\n",
      "31\t154935\t0.3\t1\t145507 9428\n",
      "32\t140211\t0.3\t1\t132501 7710\n",
      "33\t154714\t0.3\t1\t145023 9691\n",
      "34\t197197\t0.3\t1\t185506 11691\n",
      "35\t114069\t0.3\t1\t107701 6368\n",
      "36\t154713\t0.3\t1\t145095 9618\n",
      "37\t166945\t0.3\t1\t157027 9918\n",
      "38\t155394\t0.3\t1\t147358 8036\n",
      "39\t151879\t0.3\t1\t143914 7965\n",
      "40\t168764\t0.3\t1\t158797 9967\n",
      "41\t162964\t0.3\t1\t154146 8818\n",
      "42\t166211\t0.3\t1\t157158 9053\n",
      "43\t152395\t0.3\t1\t144005 8390\n",
      "44\t174656\t0.3\t1\t164812 9844\n",
      "45\t186871\t0.3\t1\t176811 10060\n",
      "46\t135036\t0.3\t1\t127666 7370\n",
      "47\t168636\t0.3\t1\t159853 8783\n",
      "48\t166288\t0.3\t1\t157802 8486\n",
      "49\t174530\t0.3\t1\t165351 9179\n",
      "50\t164769\t0.3\t1\t155907 8862\n",
      "51\t207765\t0.3\t1\t197011 10754\n",
      "52\t147151\t0.3\t1\t139795 7356\n",
      "53\t177051\t0.3\t1\t168817 8234\n",
      "54\t133625\t0.3\t1\t126900 6725\n",
      "55\t174567\t0.3\t1\t166338 8229\n",
      "56\t178950\t0.3\t1\t169886 9064\n",
      "57\t168945\t0.3\t1\t161037 7908\n",
      "58\t161931\t0.3\t1\t154618 7313\n",
      "59\t165789\t0.3\t1\t157874 7915\n",
      "60\t171736\t0.3\t1\t163328 8408\n",
      "61\t175826\t0.3\t1\t167046 8780\n",
      "62\t192123\t0.3\t1\t182534 9589\n",
      "63\t252743\t0.3\t1\t241513 11230\n",
      "64\t207112\t0.3\t1\t199124 7988\n",
      "65\t135660\t0.3\t1\t129354 6306\n",
      "66\t145866\t0.3\t1\t139110 6756\n",
      "67\t145748\t0.3\t1\t138731 7017\n",
      "68\t146161\t0.3\t1\t139522 6639\n",
      "69\t144487\t0.3\t1\t137768 6719\n",
      "70\t154292\t0.3\t1\t147160 7132\n",
      "71\t158353\t0.3\t1\t151327 7026\n",
      "72\t150227\t0.3\t1\t143181 7046\n",
      "73\t157376\t0.3\t1\t150704 6672\n",
      "74\t144388\t0.3\t1\t138042 6346\n",
      "75\t137234\t0.3\t1\t131094 6140\n",
      "76\t133825\t0.3\t1\t127901 5924\n",
      "77\t133664\t0.3\t1\t127640 6024\n",
      "78\t127632\t0.3\t1\t121889 5743\n",
      "79\t122934\t0.3\t1\t117397 5537\n",
      "80\t122791\t0.3\t1\t117317 5474\n",
      "81\t119094\t0.3\t1\t113976 5118\n",
      "82\t115048\t0.3\t1\t109953 5095\n",
      "83\t110957\t0.3\t1\t106119 4838\n",
      "84\t107760\t0.3\t1\t102959 4801\n",
      "85\t105919\t0.3\t1\t100973 4946\n",
      "86\t104575\t0.3\t1\t99936 4639\n",
      "87\t105119\t0.3\t1\t100537 4582\n",
      "88\t102473\t0.3\t1\t97892 4581\n",
      "89\t90861\t0.3\t1\t86820 4041\n",
      "90\t87725\t0.3\t1\t83791 3934\n",
      "91\t82796\t0.3\t1\t79052 3744\n",
      "92\t76768\t0.3\t1\t73386 3382\n",
      "93\t74497\t0.3\t1\t71224 3273\n",
      "94\t71670\t0.3\t1\t68502 3168\n",
      "95\t67300\t0.3\t1\t64357 2943\n",
      "96\t63751\t0.3\t1\t61088 2663\n",
      "97\t58781\t0.3\t1\t56250 2531\n",
      "98\t54871\t0.3\t1\t52462 2409\n",
      "99\t53000\t0.3\t1\t50773 2227\n",
      "100\t48407\t0.3\t1\t46381 2026\n",
      "101\t45093\t0.3\t1\t43177 1916\n",
      "102\t40752\t0.3\t1\t38996 1756\n",
      "103\t37584\t0.3\t1\t35906 1678\n",
      "104\t35749\t0.3\t1\t34226 1523\n",
      "105\t31850\t0.3\t1\t30464 1386\n",
      "106\t29282\t0.3\t1\t28064 1218\n",
      "107\t26084\t0.3\t1\t24956 1128\n",
      "108\t23726\t0.3\t1\t22669 1057\n",
      "109\t21646\t0.3\t1\t20704 942\n",
      "110\t19323\t0.3\t1\t18515 808\n",
      "111\t18235\t0.3\t1\t17436 799\n",
      "112\t15797\t0.3\t1\t15095 702\n",
      "113\t13713\t0.3\t1\t13089 624\n",
      "114\t11968\t0.3\t1\t11416 552\n",
      "115\t10124\t0.3\t1\t9656 468\n",
      "116\t8777\t0.3\t1\t8354 423\n",
      "117\t7339\t0.3\t1\t6995 344\n",
      "118\t6452\t0.3\t1\t6137 315\n",
      "119\t5422\t0.3\t1\t5101 321\n",
      "120\t4499\t0.3\t1\t4281 218\n",
      "121\t3883\t0.3\t1\t3680 203\n",
      "122\t3313\t0.3\t1\t3156 157\n",
      "123\t2807\t0.3\t1\t2669 138\n",
      "124\t2400\t0.3\t1\t2285 115\n",
      "125\t2022\t0.3\t1\t1907 115\n",
      "126\t1650\t0.3\t1\t1555 95\n",
      "127\t1434\t0.3\t1\t1346 88\n",
      "128\t1129\t0.3\t1\t1055 74\n",
      "129\t969\t0.3\t1\t902 67\n",
      "130\t682\t0.3\t1\t644 38\n",
      "131\t587\t0.3\t1\t552 35\n",
      "132\t523\t0.3\t1\t500 23\n",
      "133\t524\t0.3\t1\t494 30\n",
      "134\t633\t0.3\t1\t604 29\n",
      "135\t337\t0.3\t1\t318 19\n",
      "136\t197\t0.3\t1\t186 11\n",
      "137\t147\t0.3\t1\t133 14\n",
      "138\t92\t0.3\t1\t88 4\n",
      "139\t79\t0.3\t1\t71 8\n",
      "140\t57\t0.3\t1\t52 5\n",
      "141\t37\t0.3\t1\t35 2\n",
      "142\t37\t0.3\t1\t35 2\n",
      "143\t25\t0.3\t1\t20 5\n",
      "144\t20\t0.3\t1\t20\n",
      "145\t26\t0.3\t1\t25 1\n",
      "146\t24\t0.3\t1\t23 1\n",
      "147\t85\t0.3\t1\t85\n",
      "148\t61\t0.3\t1\t59 2\n",
      "149\t39\t0.3\t1\t36 3\n",
      "150\t325\t0.3\t1\t310 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t29940 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n"
     ]
    }
   ],
   "source": [
    "!/Users/strigg/anaconda3/bin/TrimGalore-0.6.0/trim_galore \\\n",
    "--output_dir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt \\\n",
    "--fastqc_args \\\n",
    "\"--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC\" \\\n",
    "--illumina \\\n",
    "--path_to_cutadapt /Users/strigg/anaconda3/bin/cutadapt \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc of trimmed reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cd /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC\n",
    "!/Users/strigg/anaconda3/bin/multiqc ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#trim first 5 bases from each read"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp\n",
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!/Users/strigg/anaconda3/bin/TrimGalore-0.6.0/trim_galore \\\n",
    "--output_dir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp \\\n",
    "--fastqc_args \\\n",
    "\"--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC\" \\\n",
    "--hardtrim3 5 \\\n",
    "--path_to_cutadapt /Users/strigg/anaconda3/bin/cutadapt \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L002_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L001_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L001_R2_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L002_R1_001_trimmed.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L002_R2_001_trimmed.fq.gz "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc on adapter + 1st5bp trimmed data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cd /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC\n",
    "!/Users/strigg/anaconda3/bin/multiqc ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
