{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#copy data from Globus to gannet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir /Volumes/web/metacarcinus/Salmo_Calig/FASTQS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "#see markdown post about how data was transfered from Sequencing core to Gannet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#copy compressed directory to mox"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#untar files \n",
    "#tar -xvf UW_Trigg_190718.tar.gz\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m16C_26psu_1_S13_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_1_S13_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_2_S14_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_3_S15_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_26psu_4_S16_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_1_S1_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_2_S2_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_3_S3_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m16C_32psu_4_S4_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_1_S9_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_2_S10_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_3_S11_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_26psu_4_S12_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_1_S5_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_2_S6_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_3_S7_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31m8C_32psu_4_S8_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mCTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[34mReports\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F1_S20_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mSealice_F2_S22_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[34mStats\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L001_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L001_R2_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L002_R1_001.fastq.gz\u001b[m\u001b[m\r\n",
      "\u001b[31mUndetermined_S0_L002_R2_001.fastq.gz\u001b[m\u001b[m\r\n"
     ]
    }
   ],
   "source": [
    "!ls /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Approx 100% complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Approx 100% complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 10% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L002_R1_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L002_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L002_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/Shared/bioinformatics/FastQC/fastqc \\\n",
    "--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722 \\\n",
    "--threads 4 \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Undetermined_S0_L002_R2_001.fastq.gz \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:44: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  configs = yaml.load(f)\n",
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:50: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  sp = yaml.load(f)\n",
      "[WARNING]         multiqc : MultiQC Version v1.7 now available!\n",
      "[INFO   ]         multiqc : This is MultiQC v1.0.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_1_S13_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_2_S14_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_3_S15_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_26psu_4_S16_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_1_S1_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_2_S2_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_3_S3_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/16C_32psu_4_S4_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_1_S9_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_2_S10_L002_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R1_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R1_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R2_001_fastqc.html'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L001_R2_001_fastqc.zip'\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R1_001_fastqc.html'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_3_S11_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_26psu_4_S12_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_1_S5_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_2_S6_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_3_S7_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/8C_32psu_4_S8_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_1_S19_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_16C_26psu_2_S21_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_1_S17_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/CTRL_8C_26psu_2_S18_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F1_S20_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Sealice_F2_S22_L002_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L001_R2_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R1_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R1_001_fastqc.zip'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R2_001_fastqc.html'\r\n",
      "[INFO   ]         multiqc : Searching '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/Undetermined_S0_L002_R2_001_fastqc.zip'\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]        bargraph : cannot import name 'get_backend'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/bargraph.py\", line 21, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 40, in <module>\n",
      "    import matplotlib._constrained_layout as constrained_layout\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/_constrained_layout.py\", line 52, in <module>\n",
      "    from matplotlib.legend import Legend\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/legend.py\", line 43, in <module>\n",
      "    from matplotlib.offsetbox import HPacker, VPacker, TextArea, DrawingArea\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/offsetbox.py\", line 33, in <module>\n",
      "    from matplotlib.image import BboxImage\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/image.py\", line 19, in <module>\n",
      "    from matplotlib.backend_bases import FigureCanvasBase\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/backend_bases.py\", line 46, in <module>\n",
      "    from matplotlib import (\n",
      "ImportError: cannot import name 'get_backend'\n",
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]       linegraph : module 'matplotlib' has no attribute 'artist'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/linegraph.py\", line 20, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 28, in <module>\n",
      "    import matplotlib.artist as martist\n",
      "AttributeError: module 'matplotlib' has no attribute 'artist'\n",
      "[INFO   ]          fastqc : Found 92 reports\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/bin/multiqc\", line 4, in <module>\n",
      "    __import__('pkg_resources').run_script('multiqc==1.0.dev0', 'multiqc')\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 666, in run_script\n",
      "    self.require(requires)[0].run_script(script_name, ns)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 1453, in run_script\n",
      "    exec(code, namespace, namespace)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 581, in <module>\n",
      "    multiqc()\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 764, in __call__\n",
      "    return self.main(*args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 717, in main\n",
      "    rv = self.invoke(ctx)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 956, in invoke\n",
      "    return ctx.invoke(self.callback, **ctx.params)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/click/core.py\", line 555, in invoke\n",
      "    return callback(*args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 509, in multiqc\n",
      "    report_output = j_template.render(report=report, config=config)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/asyncsupport.py\", line 76, in render\n",
      "    return original_render(self, *args, **kwargs)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/environment.py\", line 1008, in render\n",
      "    return self.environment.handle_exception(exc_info, True)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/environment.py\", line 780, in handle_exception\n",
      "    reraise(exc_type, exc_value, tb)\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/jinja2/_compat.py\", line 37, in reraise\n",
      "    raise value.with_traceback(tb)\n",
      "  File \"/var/folders/_v/3qsffc0s69l8d4v9795nknd40000gt/T/tmpe5h8nmy5/base.html\", line 19, in top-level template code\n",
      "    {% include 'includes.html' %}\n",
      "  File \"/var/folders/_v/3qsffc0s69l8d4v9795nknd40000gt/T/tmpe5h8nmy5/includes.html\", line 33, in top-level template code\n",
      "    <style type=\"text/css\">{{ include_file(css_href, None) }}</style>\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/EGG-INFO/scripts/multiqc\", line 496, in include_file\n",
      "    with io.open (os.path.join(fdir, name), \"r\", encoding='utf-8') as f:\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/posixpath.py\", line 80, in join\n",
      "    a = os.fspath(a)\n",
      "TypeError: expected str, bytes or os.PathLike object, not NoneType\n"
     ]
    }
   ],
   "source": [
    "!cd /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722\n",
    "!/Users/strigg/anaconda3/bin/multiqc ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "#make new directory for trim output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt: File exists\n",
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC: File exists\n"
     ]
    }
   ],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt\n",
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "#trim adapters off reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Multicore support not enabled. Proceeding with single-core trimming.\n",
      "Path to Cutadapt set as: '/Users/strigg/anaconda3/bin/cutadapt' (user defined)\n",
      "Cutadapt seems to be working fine (tested command '/Users/strigg/anaconda3/bin/cutadapt --version')\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 921.66 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,366,785 (78.4%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              20,636,466 bp (0.6%)\n",
      "Total written (filtered):  2,599,584,885 bp (78.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17366785 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2904827\t5539983.8\t0\t2904827\n",
      "2\t423079\t1384995.9\t0\t423079\n",
      "3\t231866\t346249.0\t0\t231866\n",
      "4\t143449\t86562.2\t0\t143449\n",
      "5\t117570\t21640.6\t0\t117570\n",
      "6\t115529\t5410.1\t0\t115529\n",
      "7\t96119\t1352.5\t0\t96119\n",
      "8\t119364\t338.1\t0\t119364\n",
      "9\t118406\t84.5\t0\t116886 1520\n",
      "10\t112051\t21.1\t1\t106032 6019\n",
      "11\t122987\t5.3\t1\t113734 9253\n",
      "12\t119477\t1.3\t1\t111842 7635\n",
      "13\t120023\t0.3\t1\t112246 7777\n",
      "14\t122963\t0.3\t1\t114085 8878\n",
      "15\t127528\t0.3\t1\t119209 8319\n",
      "16\t126445\t0.3\t1\t117002 9443\n",
      "17\t152761\t0.3\t1\t140488 12273\n",
      "18\t129037\t0.3\t1\t121213 7824\n",
      "19\t112889\t0.3\t1\t105773 7116\n",
      "20\t133616\t0.3\t1\t125004 8612\n",
      "21\t146516\t0.3\t1\t136291 10225\n",
      "22\t126419\t0.3\t1\t119045 7374\n",
      "23\t145391\t0.3\t1\t135716 9675\n",
      "24\t136894\t0.3\t1\t125670 11224\n",
      "25\t145117\t0.3\t1\t135313 9804\n",
      "26\t134897\t0.3\t1\t126717 8180\n",
      "27\t148757\t0.3\t1\t137734 11023\n",
      "28\t145635\t0.3\t1\t135492 10143\n",
      "29\t153188\t0.3\t1\t140948 12240\n",
      "30\t163373\t0.3\t1\t154030 9343\n",
      "31\t130689\t0.3\t1\t122129 8560\n",
      "32\t157390\t0.3\t1\t147156 10234\n",
      "33\t145226\t0.3\t1\t135269 9957\n",
      "34\t158524\t0.3\t1\t147826 10698\n",
      "35\t161582\t0.3\t1\t148734 12848\n",
      "36\t173098\t0.3\t1\t162838 10260\n",
      "37\t141921\t0.3\t1\t133053 8868\n",
      "38\t164933\t0.3\t1\t154468 10465\n",
      "39\t167185\t0.3\t1\t155913 11272\n",
      "40\t175799\t0.3\t1\t163207 12592\n",
      "41\t197239\t0.3\t1\t176427 20812\n",
      "42\t165247\t0.3\t1\t150517 14730\n",
      "43\t270149\t0.3\t1\t254538 15611\n",
      "44\t85600\t0.3\t1\t76961 8639\n",
      "45\t197628\t0.3\t1\t175697 21931\n",
      "46\t723044\t0.3\t1\t688287 34757\n",
      "47\t254889\t0.3\t1\t240372 14517\n",
      "48\t114855\t0.3\t1\t101708 13147\n",
      "49\t409546\t0.3\t1\t390013 19533\n",
      "50\t125438\t0.3\t1\t116192 9246\n",
      "51\t110069\t0.3\t1\t99667 10402\n",
      "52\t381945\t0.3\t1\t364475 17470\n",
      "53\t196688\t0.3\t1\t188239 8449\n",
      "54\t63003\t0.3\t1\t58174 4829\n",
      "55\t106109\t0.3\t1\t101117 4992\n",
      "56\t55521\t0.3\t1\t50154 5367\n",
      "57\t169184\t0.3\t1\t161794 7390\n",
      "58\t35182\t0.3\t1\t32212 2970\n",
      "59\t44244\t0.3\t1\t39185 5059\n",
      "60\t173147\t0.3\t1\t165425 7722\n",
      "61\t86440\t0.3\t1\t82402 4038\n",
      "62\t40559\t0.3\t1\t36479 4080\n",
      "63\t130914\t0.3\t1\t123404 7510\n",
      "64\t122315\t0.3\t1\t116337 5978\n",
      "65\t65805\t0.3\t1\t60428 5377\n",
      "66\t148115\t0.3\t1\t139100 9015\n",
      "67\t175930\t0.3\t1\t163369 12561\n",
      "68\t293688\t0.3\t1\t278052 15636\n",
      "69\t189726\t0.3\t1\t179114 10612\n",
      "70\t75474\t0.3\t1\t69457 6017\n",
      "71\t27345\t0.3\t1\t23952 3393\n",
      "72\t97969\t0.3\t1\t91968 6001\n",
      "73\t138145\t0.3\t1\t130111 8034\n",
      "74\t143624\t0.3\t1\t135279 8345\n",
      "75\t142674\t0.3\t1\t134066 8608\n",
      "76\t141682\t0.3\t1\t133308 8374\n",
      "77\t141205\t0.3\t1\t132947 8258\n",
      "78\t133036\t0.3\t1\t124966 8070\n",
      "79\t131630\t0.3\t1\t123633 7997\n",
      "80\t128859\t0.3\t1\t121116 7743\n",
      "81\t126119\t0.3\t1\t118667 7452\n",
      "82\t123492\t0.3\t1\t116262 7230\n",
      "83\t118847\t0.3\t1\t111615 7232\n",
      "84\t115583\t0.3\t1\t108642 6941\n",
      "85\t111409\t0.3\t1\t104817 6592\n",
      "86\t107099\t0.3\t1\t100545 6554\n",
      "87\t102731\t0.3\t1\t96477 6254\n",
      "88\t101527\t0.3\t1\t95245 6282\n",
      "89\t95383\t0.3\t1\t89600 5783\n",
      "90\t95037\t0.3\t1\t89230 5807\n",
      "91\t89836\t0.3\t1\t84395 5441\n",
      "92\t84977\t0.3\t1\t79865 5112\n",
      "93\t79479\t0.3\t1\t74762 4717\n",
      "94\t76672\t0.3\t1\t72010 4662\n",
      "95\t70676\t0.3\t1\t66313 4363\n",
      "96\t66886\t0.3\t1\t62766 4120\n",
      "97\t63105\t0.3\t1\t59150 3955\n",
      "98\t58813\t0.3\t1\t55101 3712\n",
      "99\t56801\t0.3\t1\t52921 3880\n",
      "100\t51251\t0.3\t1\t47973 3278\n",
      "101\t48131\t0.3\t1\t44969 3162\n",
      "102\t43975\t0.3\t1\t41134 2841\n",
      "103\t40749\t0.3\t1\t38132 2617\n",
      "104\t37602\t0.3\t1\t35059 2543\n",
      "105\t33601\t0.3\t1\t31318 2283\n",
      "106\t30687\t0.3\t1\t28622 2065\n",
      "107\t27550\t0.3\t1\t25698 1852\n",
      "108\t25919\t0.3\t1\t24135 1784\n",
      "109\t23271\t0.3\t1\t21584 1687\n",
      "110\t20524\t0.3\t1\t19077 1447\n",
      "111\t18563\t0.3\t1\t17263 1300\n",
      "112\t16204\t0.3\t1\t15036 1168\n",
      "113\t14110\t0.3\t1\t13017 1093\n",
      "114\t12321\t0.3\t1\t11394 927\n",
      "115\t10629\t0.3\t1\t9884 745\n",
      "116\t9483\t0.3\t1\t8756 727\n",
      "117\t8057\t0.3\t1\t7440 617\n",
      "118\t7067\t0.3\t1\t6491 576\n",
      "119\t5777\t0.3\t1\t5264 513\n",
      "120\t4917\t0.3\t1\t4511 406\n",
      "121\t4181\t0.3\t1\t3800 381\n",
      "122\t3506\t0.3\t1\t3174 332\n",
      "123\t2912\t0.3\t1\t2640 272\n",
      "124\t2546\t0.3\t1\t2297 249\n",
      "125\t2101\t0.3\t1\t1902 199\n",
      "126\t1717\t0.3\t1\t1548 169\n",
      "127\t1536\t0.3\t1\t1361 175\n",
      "128\t1201\t0.3\t1\t1095 106\n",
      "129\t1012\t0.3\t1\t912 100\n",
      "130\t739\t0.3\t1\t668 71\n",
      "131\t638\t0.3\t1\t572 66\n",
      "132\t553\t0.3\t1\t500 53\n",
      "133\t556\t0.3\t1\t500 56\n",
      "134\t665\t0.3\t1\t607 58\n",
      "135\t362\t0.3\t1\t324 38\n",
      "136\t207\t0.3\t1\t176 31\n",
      "137\t143\t0.3\t1\t128 15\n",
      "138\t99\t0.3\t1\t86 13\n",
      "139\t84\t0.3\t1\t72 12\n",
      "140\t58\t0.3\t1\t50 8\n",
      "141\t42\t0.3\t1\t38 4\n",
      "142\t37\t0.3\t1\t29 8\n",
      "143\t24\t0.3\t1\t22 2\n",
      "144\t25\t0.3\t1\t17 8\n",
      "145\t26\t0.3\t1\t22 4\n",
      "146\t29\t0.3\t1\t21 8\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t64\t0.3\t1\t59 5\n",
      "149\t40\t0.3\t1\t35 5\n",
      "150\t355\t0.3\t1\t318 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4155 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 979.66 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,263,070 (77.9%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              16,760,179 bp (0.5%)\n",
      "Total written (filtered):  2,603,569,143 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17263070 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2980698\t5539983.8\t0\t2980698\n",
      "2\t434290\t1384995.9\t0\t434290\n",
      "3\t244115\t346249.0\t0\t244115\n",
      "4\t143736\t86562.2\t0\t143736\n",
      "5\t130207\t21640.6\t0\t130207\n",
      "6\t112454\t5410.1\t0\t112454\n",
      "7\t91383\t1352.5\t0\t91383\n",
      "8\t130413\t338.1\t0\t130413\n",
      "9\t102897\t84.5\t0\t102495 402\n",
      "10\t113241\t21.1\t1\t107719 5522\n",
      "11\t119188\t5.3\t1\t111572 7616\n",
      "12\t120663\t1.3\t1\t112735 7928\n",
      "13\t133942\t0.3\t1\t125965 7977\n",
      "14\t126547\t0.3\t1\t117452 9095\n",
      "15\t102830\t0.3\t1\t96138 6692\n",
      "16\t149163\t0.3\t1\t139761 9402\n",
      "17\t119889\t0.3\t1\t111631 8258\n",
      "18\t113552\t0.3\t1\t106594 6958\n",
      "19\t144826\t0.3\t1\t135202 9624\n",
      "20\t135016\t0.3\t1\t126722 8294\n",
      "21\t118040\t0.3\t1\t110327 7713\n",
      "22\t135122\t0.3\t1\t126795 8327\n",
      "23\t139623\t0.3\t1\t130868 8755\n",
      "24\t139327\t0.3\t1\t130312 9015\n",
      "25\t156486\t0.3\t1\t147346 9140\n",
      "26\t131367\t0.3\t1\t122873 8494\n",
      "27\t144343\t0.3\t1\t134126 10217\n",
      "28\t135732\t0.3\t1\t128445 7287\n",
      "29\t144460\t0.3\t1\t135341 9119\n",
      "30\t143460\t0.3\t1\t135475 7985\n",
      "31\t154935\t0.3\t1\t145507 9428\n",
      "32\t140211\t0.3\t1\t132501 7710\n",
      "33\t154714\t0.3\t1\t145023 9691\n",
      "34\t197197\t0.3\t1\t185506 11691\n",
      "35\t114069\t0.3\t1\t107701 6368\n",
      "36\t154713\t0.3\t1\t145095 9618\n",
      "37\t166945\t0.3\t1\t157027 9918\n",
      "38\t155394\t0.3\t1\t147358 8036\n",
      "39\t151879\t0.3\t1\t143914 7965\n",
      "40\t168764\t0.3\t1\t158797 9967\n",
      "41\t162964\t0.3\t1\t154146 8818\n",
      "42\t166211\t0.3\t1\t157158 9053\n",
      "43\t152395\t0.3\t1\t144005 8390\n",
      "44\t174656\t0.3\t1\t164812 9844\n",
      "45\t186871\t0.3\t1\t176811 10060\n",
      "46\t135036\t0.3\t1\t127666 7370\n",
      "47\t168636\t0.3\t1\t159853 8783\n",
      "48\t166288\t0.3\t1\t157802 8486\n",
      "49\t174530\t0.3\t1\t165351 9179\n",
      "50\t164769\t0.3\t1\t155907 8862\n",
      "51\t207765\t0.3\t1\t197011 10754\n",
      "52\t147151\t0.3\t1\t139795 7356\n",
      "53\t177051\t0.3\t1\t168817 8234\n",
      "54\t133625\t0.3\t1\t126900 6725\n",
      "55\t174567\t0.3\t1\t166338 8229\n",
      "56\t178950\t0.3\t1\t169886 9064\n",
      "57\t168945\t0.3\t1\t161037 7908\n",
      "58\t161931\t0.3\t1\t154618 7313\n",
      "59\t165789\t0.3\t1\t157874 7915\n",
      "60\t171736\t0.3\t1\t163328 8408\n",
      "61\t175826\t0.3\t1\t167046 8780\n",
      "62\t192123\t0.3\t1\t182534 9589\n",
      "63\t252743\t0.3\t1\t241513 11230\n",
      "64\t207112\t0.3\t1\t199124 7988\n",
      "65\t135660\t0.3\t1\t129354 6306\n",
      "66\t145866\t0.3\t1\t139110 6756\n",
      "67\t145748\t0.3\t1\t138731 7017\n",
      "68\t146161\t0.3\t1\t139522 6639\n",
      "69\t144487\t0.3\t1\t137768 6719\n",
      "70\t154292\t0.3\t1\t147160 7132\n",
      "71\t158353\t0.3\t1\t151327 7026\n",
      "72\t150227\t0.3\t1\t143181 7046\n",
      "73\t157376\t0.3\t1\t150704 6672\n",
      "74\t144388\t0.3\t1\t138042 6346\n",
      "75\t137234\t0.3\t1\t131094 6140\n",
      "76\t133825\t0.3\t1\t127901 5924\n",
      "77\t133664\t0.3\t1\t127640 6024\n",
      "78\t127632\t0.3\t1\t121889 5743\n",
      "79\t122934\t0.3\t1\t117397 5537\n",
      "80\t122791\t0.3\t1\t117317 5474\n",
      "81\t119094\t0.3\t1\t113976 5118\n",
      "82\t115048\t0.3\t1\t109953 5095\n",
      "83\t110957\t0.3\t1\t106119 4838\n",
      "84\t107760\t0.3\t1\t102959 4801\n",
      "85\t105919\t0.3\t1\t100973 4946\n",
      "86\t104575\t0.3\t1\t99936 4639\n",
      "87\t105119\t0.3\t1\t100537 4582\n",
      "88\t102473\t0.3\t1\t97892 4581\n",
      "89\t90861\t0.3\t1\t86820 4041\n",
      "90\t87725\t0.3\t1\t83791 3934\n",
      "91\t82796\t0.3\t1\t79052 3744\n",
      "92\t76768\t0.3\t1\t73386 3382\n",
      "93\t74497\t0.3\t1\t71224 3273\n",
      "94\t71670\t0.3\t1\t68502 3168\n",
      "95\t67300\t0.3\t1\t64357 2943\n",
      "96\t63751\t0.3\t1\t61088 2663\n",
      "97\t58781\t0.3\t1\t56250 2531\n",
      "98\t54871\t0.3\t1\t52462 2409\n",
      "99\t53000\t0.3\t1\t50773 2227\n",
      "100\t48407\t0.3\t1\t46381 2026\n",
      "101\t45093\t0.3\t1\t43177 1916\n",
      "102\t40752\t0.3\t1\t38996 1756\n",
      "103\t37584\t0.3\t1\t35906 1678\n",
      "104\t35749\t0.3\t1\t34226 1523\n",
      "105\t31850\t0.3\t1\t30464 1386\n",
      "106\t29282\t0.3\t1\t28064 1218\n",
      "107\t26084\t0.3\t1\t24956 1128\n",
      "108\t23726\t0.3\t1\t22669 1057\n",
      "109\t21646\t0.3\t1\t20704 942\n",
      "110\t19323\t0.3\t1\t18515 808\n",
      "111\t18235\t0.3\t1\t17436 799\n",
      "112\t15797\t0.3\t1\t15095 702\n",
      "113\t13713\t0.3\t1\t13089 624\n",
      "114\t11968\t0.3\t1\t11416 552\n",
      "115\t10124\t0.3\t1\t9656 468\n",
      "116\t8777\t0.3\t1\t8354 423\n",
      "117\t7339\t0.3\t1\t6995 344\n",
      "118\t6452\t0.3\t1\t6137 315\n",
      "119\t5422\t0.3\t1\t5101 321\n",
      "120\t4499\t0.3\t1\t4281 218\n",
      "121\t3883\t0.3\t1\t3680 203\n",
      "122\t3313\t0.3\t1\t3156 157\n",
      "123\t2807\t0.3\t1\t2669 138\n",
      "124\t2400\t0.3\t1\t2285 115\n",
      "125\t2022\t0.3\t1\t1907 115\n",
      "126\t1650\t0.3\t1\t1555 95\n",
      "127\t1434\t0.3\t1\t1346 88\n",
      "128\t1129\t0.3\t1\t1055 74\n",
      "129\t969\t0.3\t1\t902 67\n",
      "130\t682\t0.3\t1\t644 38\n",
      "131\t587\t0.3\t1\t552 35\n",
      "132\t523\t0.3\t1\t500 23\n",
      "133\t524\t0.3\t1\t494 30\n",
      "134\t633\t0.3\t1\t604 29\n",
      "135\t337\t0.3\t1\t318 19\n",
      "136\t197\t0.3\t1\t186 11\n",
      "137\t147\t0.3\t1\t133 14\n",
      "138\t92\t0.3\t1\t88 4\n",
      "139\t79\t0.3\t1\t71 8\n",
      "140\t57\t0.3\t1\t52 5\n",
      "141\t37\t0.3\t1\t35 2\n",
      "142\t37\t0.3\t1\t35 2\n",
      "143\t25\t0.3\t1\t20 5\n",
      "144\t20\t0.3\t1\t20\n",
      "145\t26\t0.3\t1\t25 1\n",
      "146\t24\t0.3\t1\t23 1\n",
      "147\t85\t0.3\t1\t85\n",
      "148\t61\t0.3\t1\t59 2\n",
      "149\t39\t0.3\t1\t36 3\n",
      "150\t325\t0.3\t1\t310 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t29940 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 940.13 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,055,675\n",
      "Reads with adapters:                17,288,884 (78.4%)\n",
      "Reads written (passing filters):    22,055,675 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,308,351,250 bp\n",
      "Quality-trimmed:              20,473,501 bp (0.6%)\n",
      "Total written (filtered):  2,585,092,173 bp (78.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17288884 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2888739\t5513918.8\t0\t2888739\n",
      "2\t417982\t1378479.7\t0\t417982\n",
      "3\t229648\t344619.9\t0\t229648\n",
      "4\t140558\t86155.0\t0\t140558\n",
      "5\t117330\t21538.7\t0\t117330\n",
      "6\t114463\t5384.7\t0\t114463\n",
      "7\t92119\t1346.2\t0\t92119\n",
      "8\t118305\t336.5\t0\t118305\n",
      "9\t116870\t84.1\t0\t115423 1447\n",
      "10\t109085\t21.0\t1\t102190 6895\n",
      "11\t122273\t5.3\t1\t111534 10739\n",
      "12\t117728\t1.3\t1\t108915 8813\n",
      "13\t119559\t0.3\t1\t110607 8952\n",
      "14\t120718\t0.3\t1\t110803 9915\n",
      "15\t126246\t0.3\t1\t116067 10179\n",
      "16\t124980\t0.3\t1\t114018 10962\n",
      "17\t152992\t0.3\t1\t138965 14027\n",
      "18\t127404\t0.3\t1\t118125 9279\n",
      "19\t110945\t0.3\t1\t102558 8387\n",
      "20\t133534\t0.3\t1\t123405 10129\n",
      "21\t147067\t0.3\t1\t135223 11844\n",
      "22\t124353\t0.3\t1\t115585 8768\n",
      "23\t146038\t0.3\t1\t134941 11097\n",
      "24\t136315\t0.3\t1\t124052 12263\n",
      "25\t144095\t0.3\t1\t132766 11329\n",
      "26\t132325\t0.3\t1\t123014 9311\n",
      "27\t148607\t0.3\t1\t136108 12499\n",
      "28\t145176\t0.3\t1\t133796 11380\n",
      "29\t153982\t0.3\t1\t140231 13751\n",
      "30\t162931\t0.3\t1\t151904 11027\n",
      "31\t127590\t0.3\t1\t117988 9602\n",
      "32\t158563\t0.3\t1\t146202 12361\n",
      "33\t143459\t0.3\t1\t131864 11595\n",
      "34\t158293\t0.3\t1\t145587 12706\n",
      "35\t161600\t0.3\t1\t146632 14968\n",
      "36\t173713\t0.3\t1\t161385 12328\n",
      "37\t138576\t0.3\t1\t128088 10488\n",
      "38\t164914\t0.3\t1\t152362 12552\n",
      "39\t166196\t0.3\t1\t152847 13349\n",
      "40\t174913\t0.3\t1\t159838 15075\n",
      "41\t195923\t0.3\t1\t172793 23130\n",
      "42\t162924\t0.3\t1\t146209 16715\n",
      "43\t267611\t0.3\t1\t248773 18838\n",
      "44\t88981\t0.3\t1\t79036 9945\n",
      "45\t199179\t0.3\t1\t175262 23917\n",
      "46\t690085\t0.3\t1\t650909 39176\n",
      "47\t257754\t0.3\t1\t240653 17101\n",
      "48\t117426\t0.3\t1\t103023 14403\n",
      "49\t402804\t0.3\t1\t380078 22726\n",
      "50\t126965\t0.3\t1\t116321 10644\n",
      "51\t110357\t0.3\t1\t99078 11279\n",
      "52\t376565\t0.3\t1\t356163 20402\n",
      "53\t193192\t0.3\t1\t182832 10360\n",
      "54\t64424\t0.3\t1\t58616 5808\n",
      "55\t110681\t0.3\t1\t104197 6484\n",
      "56\t61759\t0.3\t1\t55308 6451\n",
      "57\t169454\t0.3\t1\t160172 9282\n",
      "58\t37968\t0.3\t1\t34229 3739\n",
      "59\t46940\t0.3\t1\t41198 5742\n",
      "60\t169988\t0.3\t1\t161028 8960\n",
      "61\t86428\t0.3\t1\t81402 5026\n",
      "62\t43888\t0.3\t1\t39006 4882\n",
      "63\t131951\t0.3\t1\t123251 8700\n",
      "64\t119378\t0.3\t1\t112438 6940\n",
      "65\t67995\t0.3\t1\t61783 6212\n",
      "66\t148705\t0.3\t1\t138293 10412\n",
      "67\t175463\t0.3\t1\t161782 13681\n",
      "68\t280115\t0.3\t1\t262817 17298\n",
      "69\t200997\t0.3\t1\t188546 12451\n",
      "70\t91468\t0.3\t1\t84121 7347\n",
      "71\t26418\t0.3\t1\t22702 3716\n",
      "72\t93747\t0.3\t1\t87405 6342\n",
      "73\t136042\t0.3\t1\t127373 8669\n",
      "74\t142888\t0.3\t1\t133128 9760\n",
      "75\t142439\t0.3\t1\t132394 10045\n",
      "76\t141347\t0.3\t1\t131130 10217\n",
      "77\t141016\t0.3\t1\t131311 9705\n",
      "78\t132511\t0.3\t1\t122849 9662\n",
      "79\t132027\t0.3\t1\t122622 9405\n",
      "80\t128769\t0.3\t1\t119387 9382\n",
      "81\t126902\t0.3\t1\t118148 8754\n",
      "82\t123681\t0.3\t1\t114826 8855\n",
      "83\t118947\t0.3\t1\t110667 8280\n",
      "84\t114559\t0.3\t1\t106314 8245\n",
      "85\t111485\t0.3\t1\t103536 7949\n",
      "86\t107436\t0.3\t1\t99695 7741\n",
      "87\t103344\t0.3\t1\t96004 7340\n",
      "88\t100991\t0.3\t1\t93808 7183\n",
      "89\t95415\t0.3\t1\t88771 6644\n",
      "90\t94998\t0.3\t1\t88363 6635\n",
      "91\t89676\t0.3\t1\t83270 6406\n",
      "92\t85299\t0.3\t1\t79247 6052\n",
      "93\t79596\t0.3\t1\t73937 5659\n",
      "94\t76506\t0.3\t1\t71043 5463\n",
      "95\t71120\t0.3\t1\t65893 5227\n",
      "96\t67197\t0.3\t1\t62368 4829\n",
      "97\t63520\t0.3\t1\t59045 4475\n",
      "98\t58336\t0.3\t1\t54045 4291\n",
      "99\t56900\t0.3\t1\t52530 4370\n",
      "100\t51732\t0.3\t1\t47977 3755\n",
      "101\t47750\t0.3\t1\t44264 3486\n",
      "102\t44471\t0.3\t1\t41149 3322\n",
      "103\t40594\t0.3\t1\t37603 2991\n",
      "104\t37603\t0.3\t1\t34872 2731\n",
      "105\t33894\t0.3\t1\t31456 2438\n",
      "106\t30780\t0.3\t1\t28387 2393\n",
      "107\t27659\t0.3\t1\t25538 2121\n",
      "108\t26065\t0.3\t1\t24009 2056\n",
      "109\t23274\t0.3\t1\t21434 1840\n",
      "110\t20689\t0.3\t1\t19111 1578\n",
      "111\t18796\t0.3\t1\t17291 1505\n",
      "112\t16460\t0.3\t1\t15232 1228\n",
      "113\t14306\t0.3\t1\t13173 1133\n",
      "114\t12558\t0.3\t1\t11518 1040\n",
      "115\t10829\t0.3\t1\t9919 910\n",
      "116\t9235\t0.3\t1\t8487 748\n",
      "117\t7908\t0.3\t1\t7205 703\n",
      "118\t6968\t0.3\t1\t6363 605\n",
      "119\t5769\t0.3\t1\t5245 524\n",
      "120\t4920\t0.3\t1\t4475 445\n",
      "121\t4219\t0.3\t1\t3831 388\n",
      "122\t3544\t0.3\t1\t3202 342\n",
      "123\t3058\t0.3\t1\t2744 314\n",
      "124\t2517\t0.3\t1\t2275 242\n",
      "125\t2174\t0.3\t1\t1950 224\n",
      "126\t1777\t0.3\t1\t1603 174\n",
      "127\t1561\t0.3\t1\t1396 165\n",
      "128\t1149\t0.3\t1\t1015 134\n",
      "129\t996\t0.3\t1\t899 97\n",
      "130\t753\t0.3\t1\t670 83\n",
      "131\t665\t0.3\t1\t585 80\n",
      "132\t569\t0.3\t1\t501 68\n",
      "133\t543\t0.3\t1\t476 67\n",
      "134\t671\t0.3\t1\t623 48\n",
      "135\t378\t0.3\t1\t347 31\n",
      "136\t222\t0.3\t1\t190 32\n",
      "137\t159\t0.3\t1\t149 10\n",
      "138\t107\t0.3\t1\t97 10\n",
      "139\t82\t0.3\t1\t68 14\n",
      "140\t62\t0.3\t1\t55 7\n",
      "141\t48\t0.3\t1\t42 6\n",
      "142\t40\t0.3\t1\t35 5\n",
      "143\t27\t0.3\t1\t21 6\n",
      "144\t22\t0.3\t1\t19 3\n",
      "145\t23\t0.3\t1\t21 2\n",
      "146\t31\t0.3\t1\t24 7\n",
      "147\t94\t0.3\t1\t87 7\n",
      "148\t48\t0.3\t1\t41 7\n",
      "149\t39\t0.3\t1\t31 8\n",
      "150\t382\t0.3\t1\t330 52\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "22055675 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4286 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_1_S13_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 984.11 s (45 us/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,055,675\n",
      "Reads with adapters:                17,164,009 (77.8%)\n",
      "Reads written (passing filters):    22,055,675 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,308,351,250 bp\n",
      "Quality-trimmed:              16,172,247 bp (0.5%)\n",
      "Total written (filtered):  2,590,121,341 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17164009 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2956348\t5513918.8\t0\t2956348\n",
      "2\t421838\t1378479.7\t0\t421838\n",
      "3\t248359\t344619.9\t0\t248359\n",
      "4\t140787\t86155.0\t0\t140787\n",
      "5\t125775\t21538.7\t0\t125775\n",
      "6\t111856\t5384.7\t0\t111856\n",
      "7\t89214\t1346.2\t0\t89214\n",
      "8\t128837\t336.5\t0\t128837\n",
      "9\t101494\t84.1\t0\t101161 333\n",
      "10\t109255\t21.0\t1\t101714 7541\n",
      "11\t118195\t5.3\t1\t108364 9831\n",
      "12\t117038\t1.3\t1\t107170 9868\n",
      "13\t134296\t0.3\t1\t123167 11129\n",
      "14\t123942\t0.3\t1\t112784 11158\n",
      "15\t100887\t0.3\t1\t92534 8353\n",
      "16\t146698\t0.3\t1\t134607 12091\n",
      "17\t124219\t0.3\t1\t113532 10687\n",
      "18\t108678\t0.3\t1\t99668 9010\n",
      "19\t144281\t0.3\t1\t132500 11781\n",
      "20\t133615\t0.3\t1\t123129 10486\n",
      "21\t116612\t0.3\t1\t107459 9153\n",
      "22\t133618\t0.3\t1\t123290 10328\n",
      "23\t141009\t0.3\t1\t130007 11002\n",
      "24\t139397\t0.3\t1\t128279 11118\n",
      "25\t156124\t0.3\t1\t144895 11229\n",
      "26\t130436\t0.3\t1\t119727 10709\n",
      "27\t145243\t0.3\t1\t132821 12422\n",
      "28\t129948\t0.3\t1\t120966 8982\n",
      "29\t147371\t0.3\t1\t136169 11202\n",
      "30\t140000\t0.3\t1\t130086 9914\n",
      "31\t156053\t0.3\t1\t144716 11337\n",
      "32\t135242\t0.3\t1\t125682 9560\n",
      "33\t156256\t0.3\t1\t144163 12093\n",
      "34\t200062\t0.3\t1\t185381 14681\n",
      "35\t108741\t0.3\t1\t100819 7922\n",
      "36\t156564\t0.3\t1\t144598 11966\n",
      "37\t165143\t0.3\t1\t153098 12045\n",
      "38\t153915\t0.3\t1\t143649 10266\n",
      "39\t147077\t0.3\t1\t137848 9229\n",
      "40\t167810\t0.3\t1\t155933 11877\n",
      "41\t163682\t0.3\t1\t152622 11060\n",
      "42\t166228\t0.3\t1\t154981 11247\n",
      "43\t150608\t0.3\t1\t140711 9897\n",
      "44\t176387\t0.3\t1\t164024 12363\n",
      "45\t185146\t0.3\t1\t172572 12574\n",
      "46\t132758\t0.3\t1\t123550 9208\n",
      "47\t167253\t0.3\t1\t156952 10301\n",
      "48\t165177\t0.3\t1\t154705 10472\n",
      "49\t173975\t0.3\t1\t162376 11599\n",
      "50\t166191\t0.3\t1\t155002 11189\n",
      "51\t203241\t0.3\t1\t190158 13083\n",
      "52\t146701\t0.3\t1\t137201 9500\n",
      "53\t181035\t0.3\t1\t170415 10620\n",
      "54\t130972\t0.3\t1\t122872 8100\n",
      "55\t173160\t0.3\t1\t163051 10109\n",
      "56\t176881\t0.3\t1\t165551 11330\n",
      "57\t169136\t0.3\t1\t159682 9454\n",
      "58\t163469\t0.3\t1\t154161 9308\n",
      "59\t163465\t0.3\t1\t153708 9757\n",
      "60\t171325\t0.3\t1\t161169 10156\n",
      "61\t174265\t0.3\t1\t164163 10102\n",
      "62\t189943\t0.3\t1\t178404 11539\n",
      "63\t247557\t0.3\t1\t234468 13089\n",
      "64\t202460\t0.3\t1\t193222 9238\n",
      "65\t135184\t0.3\t1\t127502 7682\n",
      "66\t146610\t0.3\t1\t138357 8253\n",
      "67\t145979\t0.3\t1\t137259 8720\n",
      "68\t147766\t0.3\t1\t139368 8398\n",
      "69\t145296\t0.3\t1\t136585 8711\n",
      "70\t155955\t0.3\t1\t146888 9067\n",
      "71\t158414\t0.3\t1\t149557 8857\n",
      "72\t151267\t0.3\t1\t142345 8922\n",
      "73\t157245\t0.3\t1\t148240 9005\n",
      "74\t144510\t0.3\t1\t136069 8441\n",
      "75\t137730\t0.3\t1\t129832 7898\n",
      "76\t133886\t0.3\t1\t125901 7985\n",
      "77\t133306\t0.3\t1\t125623 7683\n",
      "78\t127271\t0.3\t1\t119937 7334\n",
      "79\t123387\t0.3\t1\t116350 7037\n",
      "80\t122486\t0.3\t1\t115616 6870\n",
      "81\t120050\t0.3\t1\t113299 6751\n",
      "82\t115066\t0.3\t1\t108580 6486\n",
      "83\t111130\t0.3\t1\t104980 6150\n",
      "84\t106741\t0.3\t1\t100726 6015\n",
      "85\t105619\t0.3\t1\t99800 5819\n",
      "86\t104981\t0.3\t1\t99314 5667\n",
      "87\t105156\t0.3\t1\t99595 5561\n",
      "88\t102130\t0.3\t1\t96657 5473\n",
      "89\t91200\t0.3\t1\t86213 4987\n",
      "90\t88189\t0.3\t1\t83375 4814\n",
      "91\t82519\t0.3\t1\t77931 4588\n",
      "92\t77564\t0.3\t1\t73377 4187\n",
      "93\t74406\t0.3\t1\t70415 3991\n",
      "94\t71559\t0.3\t1\t67703 3856\n",
      "95\t67768\t0.3\t1\t64110 3658\n",
      "96\t64417\t0.3\t1\t60976 3441\n",
      "97\t59309\t0.3\t1\t56218 3091\n",
      "98\t54600\t0.3\t1\t51799 2801\n",
      "99\t53526\t0.3\t1\t50607 2919\n",
      "100\t48940\t0.3\t1\t46308 2632\n",
      "101\t44993\t0.3\t1\t42581 2412\n",
      "102\t41335\t0.3\t1\t39097 2238\n",
      "103\t37548\t0.3\t1\t35551 1997\n",
      "104\t35714\t0.3\t1\t33829 1885\n",
      "105\t32156\t0.3\t1\t30392 1764\n",
      "106\t29290\t0.3\t1\t27775 1515\n",
      "107\t26004\t0.3\t1\t24590 1414\n",
      "108\t24018\t0.3\t1\t22665 1353\n",
      "109\t21637\t0.3\t1\t20542 1095\n",
      "110\t19578\t0.3\t1\t18483 1095\n",
      "111\t18464\t0.3\t1\t17459 1005\n",
      "112\t15937\t0.3\t1\t15101 836\n",
      "113\t14018\t0.3\t1\t13272 746\n",
      "114\t12104\t0.3\t1\t11450 654\n",
      "115\t10138\t0.3\t1\t9597 541\n",
      "116\t8616\t0.3\t1\t8197 419\n",
      "117\t7239\t0.3\t1\t6863 376\n",
      "118\t6371\t0.3\t1\t6012 359\n",
      "119\t5403\t0.3\t1\t5093 310\n",
      "120\t4535\t0.3\t1\t4258 277\n",
      "121\t3911\t0.3\t1\t3687 224\n",
      "122\t3351\t0.3\t1\t3130 221\n",
      "123\t2870\t0.3\t1\t2695 175\n",
      "124\t2400\t0.3\t1\t2272 128\n",
      "125\t2105\t0.3\t1\t1957 148\n",
      "126\t1712\t0.3\t1\t1597 115\n",
      "127\t1490\t0.3\t1\t1401 89\n",
      "128\t1075\t0.3\t1\t998 77\n",
      "129\t923\t0.3\t1\t859 64\n",
      "130\t711\t0.3\t1\t666 45\n",
      "131\t624\t0.3\t1\t582 42\n",
      "132\t524\t0.3\t1\t489 35\n",
      "133\t507\t0.3\t1\t469 38\n",
      "134\t649\t0.3\t1\t623 26\n",
      "135\t367\t0.3\t1\t349 18\n",
      "136\t204\t0.3\t1\t188 16\n",
      "137\t158\t0.3\t1\t144 14\n",
      "138\t102\t0.3\t1\t92 10\n",
      "139\t75\t0.3\t1\t67 8\n",
      "140\t58\t0.3\t1\t57 1\n",
      "141\t47\t0.3\t1\t40 7\n",
      "142\t38\t0.3\t1\t34 4\n",
      "143\t23\t0.3\t1\t20 3\n",
      "144\t21\t0.3\t1\t20 1\n",
      "145\t27\t0.3\t1\t23 4\n",
      "146\t22\t0.3\t1\t20 2\n",
      "147\t86\t0.3\t1\t82 4\n",
      "148\t43\t0.3\t1\t41 2\n",
      "149\t33\t0.3\t1\t32 1\n",
      "150\t345\t0.3\t1\t321 24\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "22055675 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t27448 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 723.87 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,243,447\n",
      "Reads with adapters:                13,329,210 (77.3%)\n",
      "Reads written (passing filters):    17,243,447 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,586,517,050 bp\n",
      "Quality-trimmed:               5,286,503 bp (0.2%)\n",
      "Total written (filtered):  2,048,045,774 bp (79.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13329210 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.2%\n",
      "  G: 20.5%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2371278\t4310861.8\t0\t2371278\n",
      "2\t355428\t1077715.4\t0\t355428\n",
      "3\t196735\t269428.9\t0\t196735\n",
      "4\t114807\t67357.2\t0\t114807\n",
      "5\t92976\t16839.3\t0\t92976\n",
      "6\t90363\t4209.8\t0\t90363\n",
      "7\t74432\t1052.5\t0\t74432\n",
      "8\t93807\t263.1\t0\t93807\n",
      "9\t93533\t65.8\t0\t92435 1098\n",
      "10\t87230\t16.4\t1\t82587 4643\n",
      "11\t96624\t4.1\t1\t89271 7353\n",
      "12\t92957\t1.0\t1\t86987 5970\n",
      "13\t93747\t0.3\t1\t87764 5983\n",
      "14\t96011\t0.3\t1\t89197 6814\n",
      "15\t98978\t0.3\t1\t92592 6386\n",
      "16\t98159\t0.3\t1\t90785 7374\n",
      "17\t119944\t0.3\t1\t110196 9748\n",
      "18\t100650\t0.3\t1\t94699 5951\n",
      "19\t86734\t0.3\t1\t81180 5554\n",
      "20\t104244\t0.3\t1\t97548 6696\n",
      "21\t115327\t0.3\t1\t107159 8168\n",
      "22\t97273\t0.3\t1\t91575 5698\n",
      "23\t115900\t0.3\t1\t108235 7665\n",
      "24\t104136\t0.3\t1\t95675 8461\n",
      "25\t112997\t0.3\t1\t105393 7604\n",
      "26\t102184\t0.3\t1\t96084 6100\n",
      "27\t114471\t0.3\t1\t106298 8173\n",
      "28\t113184\t0.3\t1\t105296 7888\n",
      "29\t118291\t0.3\t1\t108737 9554\n",
      "30\t125680\t0.3\t1\t118630 7050\n",
      "31\t99517\t0.3\t1\t93132 6385\n",
      "32\t120246\t0.3\t1\t112659 7587\n",
      "33\t112299\t0.3\t1\t104687 7612\n",
      "34\t122051\t0.3\t1\t114065 7986\n",
      "35\t122132\t0.3\t1\t112527 9605\n",
      "36\t121056\t0.3\t1\t113558 7498\n",
      "37\t131245\t0.3\t1\t122809 8436\n",
      "38\t117529\t0.3\t1\t109870 7659\n",
      "39\t130463\t0.3\t1\t121802 8661\n",
      "40\t125603\t0.3\t1\t117585 8018\n",
      "41\t145134\t0.3\t1\t134544 10590\n",
      "42\t108964\t0.3\t1\t101116 7848\n",
      "43\t193675\t0.3\t1\t183736 9939\n",
      "44\t56524\t0.3\t1\t51515 5009\n",
      "45\t130295\t0.3\t1\t122170 8125\n",
      "46\t121941\t0.3\t1\t114229 7712\n",
      "47\t128745\t0.3\t1\t120966 7779\n",
      "48\t126992\t0.3\t1\t118817 8175\n",
      "49\t138745\t0.3\t1\t128994 9751\n",
      "50\t118346\t0.3\t1\t111437 6909\n",
      "51\t127998\t0.3\t1\t119617 8381\n",
      "52\t132118\t0.3\t1\t122819 9299\n",
      "53\t142995\t0.3\t1\t134373 8622\n",
      "54\t111526\t0.3\t1\t104331 7195\n",
      "55\t132598\t0.3\t1\t124212 8386\n",
      "56\t162335\t0.3\t1\t152964 9371\n",
      "57\t103809\t0.3\t1\t96508 7301\n",
      "58\t132526\t0.3\t1\t125372 7154\n",
      "59\t115914\t0.3\t1\t108457 7457\n",
      "60\t128350\t0.3\t1\t120205 8145\n",
      "61\t126189\t0.3\t1\t118700 7489\n",
      "62\t137584\t0.3\t1\t129699 7885\n",
      "63\t110035\t0.3\t1\t102791 7244\n",
      "64\t120964\t0.3\t1\t114082 6882\n",
      "65\t135744\t0.3\t1\t127467 8277\n",
      "66\t140396\t0.3\t1\t131867 8529\n",
      "67\t145532\t0.3\t1\t134835 10697\n",
      "68\t287021\t0.3\t1\t273202 13819\n",
      "69\t48096\t0.3\t1\t43999 4097\n",
      "70\t18303\t0.3\t1\t15162 3141\n",
      "71\t72881\t0.3\t1\t68623 4258\n",
      "72\t103446\t0.3\t1\t97521 5925\n",
      "73\t109833\t0.3\t1\t103450 6383\n",
      "74\t107925\t0.3\t1\t101499 6426\n",
      "75\t106292\t0.3\t1\t100152 6140\n",
      "76\t104330\t0.3\t1\t98203 6127\n",
      "77\t104241\t0.3\t1\t98017 6224\n",
      "78\t98088\t0.3\t1\t92293 5795\n",
      "79\t96948\t0.3\t1\t91096 5852\n",
      "80\t94252\t0.3\t1\t88790 5462\n",
      "81\t93362\t0.3\t1\t87783 5579\n",
      "82\t90202\t0.3\t1\t84811 5391\n",
      "83\t87147\t0.3\t1\t81902 5245\n",
      "84\t83793\t0.3\t1\t78976 4817\n",
      "85\t80030\t0.3\t1\t75251 4779\n",
      "86\t77694\t0.3\t1\t73076 4618\n",
      "87\t74005\t0.3\t1\t69629 4376\n",
      "88\t72748\t0.3\t1\t68492 4256\n",
      "89\t68708\t0.3\t1\t64648 4060\n",
      "90\t68047\t0.3\t1\t63799 4248\n",
      "91\t64435\t0.3\t1\t60450 3985\n",
      "92\t60503\t0.3\t1\t56736 3767\n",
      "93\t56921\t0.3\t1\t53460 3461\n",
      "94\t55199\t0.3\t1\t51895 3304\n",
      "95\t50827\t0.3\t1\t47651 3176\n",
      "96\t48189\t0.3\t1\t45189 3000\n",
      "97\t45155\t0.3\t1\t42374 2781\n",
      "98\t41036\t0.3\t1\t38566 2470\n",
      "99\t40350\t0.3\t1\t37693 2657\n",
      "100\t35870\t0.3\t1\t33662 2208\n",
      "101\t33328\t0.3\t1\t31103 2225\n",
      "102\t30823\t0.3\t1\t28817 2006\n",
      "103\t28586\t0.3\t1\t26749 1837\n",
      "104\t26108\t0.3\t1\t24403 1705\n",
      "105\t23329\t0.3\t1\t21857 1472\n",
      "106\t21254\t0.3\t1\t19878 1376\n",
      "107\t19359\t0.3\t1\t18082 1277\n",
      "108\t18005\t0.3\t1\t16770 1235\n",
      "109\t16024\t0.3\t1\t14870 1154\n",
      "110\t14310\t0.3\t1\t13331 979\n",
      "111\t12919\t0.3\t1\t11980 939\n",
      "112\t11039\t0.3\t1\t10251 788\n",
      "113\t9662\t0.3\t1\t8961 701\n",
      "114\t8278\t0.3\t1\t7631 647\n",
      "115\t7400\t0.3\t1\t6862 538\n",
      "116\t6380\t0.3\t1\t5871 509\n",
      "117\t5590\t0.3\t1\t5116 474\n",
      "118\t4883\t0.3\t1\t4498 385\n",
      "119\t3958\t0.3\t1\t3624 334\n",
      "120\t3465\t0.3\t1\t3177 288\n",
      "121\t3026\t0.3\t1\t2769 257\n",
      "122\t2531\t0.3\t1\t2300 231\n",
      "123\t2116\t0.3\t1\t1896 220\n",
      "124\t1699\t0.3\t1\t1542 157\n",
      "125\t1434\t0.3\t1\t1302 132\n",
      "126\t1188\t0.3\t1\t1073 115\n",
      "127\t1126\t0.3\t1\t995 131\n",
      "128\t869\t0.3\t1\t759 110\n",
      "129\t701\t0.3\t1\t622 79\n",
      "130\t549\t0.3\t1\t500 49\n",
      "131\t448\t0.3\t1\t398 50\n",
      "132\t432\t0.3\t1\t385 47\n",
      "133\t454\t0.3\t1\t410 44\n",
      "134\t600\t0.3\t1\t564 36\n",
      "135\t275\t0.3\t1\t252 23\n",
      "136\t173\t0.3\t1\t157 16\n",
      "137\t109\t0.3\t1\t101 8\n",
      "138\t92\t0.3\t1\t82 10\n",
      "139\t63\t0.3\t1\t54 9\n",
      "140\t64\t0.3\t1\t52 12\n",
      "141\t59\t0.3\t1\t51 8\n",
      "142\t40\t0.3\t1\t37 3\n",
      "143\t53\t0.3\t1\t48 5\n",
      "144\t32\t0.3\t1\t31 1\n",
      "145\t53\t0.3\t1\t50 3\n",
      "146\t166\t0.3\t1\t154 12\n",
      "147\t548\t0.3\t1\t525 23\n",
      "148\t402\t0.3\t1\t390 12\n",
      "149\t263\t0.3\t1\t250 13\n",
      "150\t2873\t0.3\t1\t2693 180\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17243447 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7271 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 753.42 s (44 us/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,243,447\n",
      "Reads with adapters:                13,270,528 (77.0%)\n",
      "Reads written (passing filters):    17,243,447 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,586,517,050 bp\n",
      "Quality-trimmed:              11,661,219 bp (0.5%)\n",
      "Total written (filtered):  2,049,890,632 bp (79.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13270528 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.8%\n",
      "  G: 23.5%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2410570\t4310861.8\t0\t2410570\n",
      "2\t362455\t1077715.4\t0\t362455\n",
      "3\t204625\t269428.9\t0\t204625\n",
      "4\t116204\t67357.2\t0\t116204\n",
      "5\t102265\t16839.3\t0\t102265\n",
      "6\t87780\t4209.8\t0\t87780\n",
      "7\t72496\t1052.5\t0\t72496\n",
      "8\t102751\t263.1\t0\t102751\n",
      "9\t81658\t65.8\t0\t81386 272\n",
      "10\t89121\t16.4\t1\t84631 4490\n",
      "11\t93864\t4.1\t1\t87983 5881\n",
      "12\t94565\t1.0\t1\t88491 6074\n",
      "13\t105522\t0.3\t1\t99372 6150\n",
      "14\t98340\t0.3\t1\t91512 6828\n",
      "15\t80980\t0.3\t1\t75936 5044\n",
      "16\t116316\t0.3\t1\t109001 7315\n",
      "17\t93497\t0.3\t1\t87166 6331\n",
      "18\t89538\t0.3\t1\t84129 5409\n",
      "19\t112544\t0.3\t1\t105239 7305\n",
      "20\t105651\t0.3\t1\t99238 6413\n",
      "21\t93625\t0.3\t1\t87782 5843\n",
      "22\t104844\t0.3\t1\t98723 6121\n",
      "23\t109552\t0.3\t1\t102808 6744\n",
      "24\t108841\t0.3\t1\t101825 7016\n",
      "25\t121233\t0.3\t1\t114032 7201\n",
      "26\t101015\t0.3\t1\t94639 6376\n",
      "27\t112490\t0.3\t1\t104790 7700\n",
      "28\t105074\t0.3\t1\t99527 5547\n",
      "29\t111755\t0.3\t1\t104926 6829\n",
      "30\t110664\t0.3\t1\t104658 6006\n",
      "31\t120328\t0.3\t1\t113206 7122\n",
      "32\t108098\t0.3\t1\t102252 5846\n",
      "33\t119317\t0.3\t1\t112198 7119\n",
      "34\t151071\t0.3\t1\t142232 8839\n",
      "35\t87186\t0.3\t1\t82458 4728\n",
      "36\t120833\t0.3\t1\t113650 7183\n",
      "37\t128643\t0.3\t1\t121059 7584\n",
      "38\t120495\t0.3\t1\t114429 6066\n",
      "39\t117189\t0.3\t1\t111171 6018\n",
      "40\t129860\t0.3\t1\t122333 7527\n",
      "41\t123836\t0.3\t1\t117366 6470\n",
      "42\t127654\t0.3\t1\t120939 6715\n",
      "43\t116428\t0.3\t1\t110085 6343\n",
      "44\t133061\t0.3\t1\t125767 7294\n",
      "45\t139911\t0.3\t1\t132604 7307\n",
      "46\t103489\t0.3\t1\t98069 5420\n",
      "47\t129515\t0.3\t1\t122821 6694\n",
      "48\t126933\t0.3\t1\t120634 6299\n",
      "49\t132432\t0.3\t1\t125535 6897\n",
      "50\t125382\t0.3\t1\t118618 6764\n",
      "51\t155910\t0.3\t1\t147950 7960\n",
      "52\t112001\t0.3\t1\t106554 5447\n",
      "53\t134464\t0.3\t1\t128383 6081\n",
      "54\t103158\t0.3\t1\t98052 5106\n",
      "55\t131983\t0.3\t1\t126024 5959\n",
      "56\t136358\t0.3\t1\t129807 6551\n",
      "57\t127296\t0.3\t1\t121594 5702\n",
      "58\t122071\t0.3\t1\t116674 5397\n",
      "59\t124219\t0.3\t1\t118380 5839\n",
      "60\t128066\t0.3\t1\t121909 6157\n",
      "61\t132129\t0.3\t1\t125705 6424\n",
      "62\t142837\t0.3\t1\t135944 6893\n",
      "63\t190582\t0.3\t1\t182484 8098\n",
      "64\t150467\t0.3\t1\t144624 5843\n",
      "65\t101740\t0.3\t1\t97088 4652\n",
      "66\t109816\t0.3\t1\t104968 4848\n",
      "67\t107749\t0.3\t1\t102613 5136\n",
      "68\t108633\t0.3\t1\t103740 4893\n",
      "69\t107596\t0.3\t1\t102648 4948\n",
      "70\t114794\t0.3\t1\t109576 5218\n",
      "71\t116610\t0.3\t1\t111540 5070\n",
      "72\t110184\t0.3\t1\t105059 5125\n",
      "73\t115749\t0.3\t1\t110692 5057\n",
      "74\t106275\t0.3\t1\t101612 4663\n",
      "75\t101750\t0.3\t1\t97259 4491\n",
      "76\t99053\t0.3\t1\t94705 4348\n",
      "77\t99069\t0.3\t1\t94745 4324\n",
      "78\t94226\t0.3\t1\t90009 4217\n",
      "79\t90763\t0.3\t1\t86805 3958\n",
      "80\t89778\t0.3\t1\t85958 3820\n",
      "81\t88669\t0.3\t1\t84684 3985\n",
      "82\t84401\t0.3\t1\t80642 3759\n",
      "83\t81824\t0.3\t1\t78236 3588\n",
      "84\t78990\t0.3\t1\t75425 3565\n",
      "85\t76455\t0.3\t1\t73024 3431\n",
      "86\t76386\t0.3\t1\t73029 3357\n",
      "87\t75988\t0.3\t1\t72761 3227\n",
      "88\t73818\t0.3\t1\t70558 3260\n",
      "89\t65970\t0.3\t1\t63085 2885\n",
      "90\t63019\t0.3\t1\t60213 2806\n",
      "91\t58914\t0.3\t1\t56366 2548\n",
      "92\t54704\t0.3\t1\t52314 2390\n",
      "93\t52692\t0.3\t1\t50404 2288\n",
      "94\t51139\t0.3\t1\t48941 2198\n",
      "95\t47428\t0.3\t1\t45341 2087\n",
      "96\t45523\t0.3\t1\t43617 1906\n",
      "97\t43074\t0.3\t1\t41264 1810\n",
      "98\t39986\t0.3\t1\t38323 1663\n",
      "99\t40439\t0.3\t1\t38720 1719\n",
      "100\t35462\t0.3\t1\t34051 1411\n",
      "101\t31343\t0.3\t1\t29998 1345\n",
      "102\t28451\t0.3\t1\t27239 1212\n",
      "103\t26443\t0.3\t1\t25307 1136\n",
      "104\t24686\t0.3\t1\t23698 988\n",
      "105\t22319\t0.3\t1\t21394 925\n",
      "106\t20298\t0.3\t1\t19438 860\n",
      "107\t18363\t0.3\t1\t17549 814\n",
      "108\t16498\t0.3\t1\t15775 723\n",
      "109\t15075\t0.3\t1\t14486 589\n",
      "110\t13470\t0.3\t1\t12883 587\n",
      "111\t12724\t0.3\t1\t12179 545\n",
      "112\t10843\t0.3\t1\t10385 458\n",
      "113\t9574\t0.3\t1\t9156 418\n",
      "114\t8065\t0.3\t1\t7698 367\n",
      "115\t6969\t0.3\t1\t6632 337\n",
      "116\t5939\t0.3\t1\t5624 315\n",
      "117\t5106\t0.3\t1\t4872 234\n",
      "118\t4425\t0.3\t1\t4206 219\n",
      "119\t3716\t0.3\t1\t3520 196\n",
      "120\t3161\t0.3\t1\t3007 154\n",
      "121\t2854\t0.3\t1\t2727 127\n",
      "122\t2418\t0.3\t1\t2305 113\n",
      "123\t2039\t0.3\t1\t1907 132\n",
      "124\t1593\t0.3\t1\t1516 77\n",
      "125\t1395\t0.3\t1\t1318 77\n",
      "126\t1136\t0.3\t1\t1075 61\n",
      "127\t1068\t0.3\t1\t1004 64\n",
      "128\t801\t0.3\t1\t764 37\n",
      "129\t660\t0.3\t1\t625 35\n",
      "130\t510\t0.3\t1\t471 39\n",
      "131\t432\t0.3\t1\t407 25\n",
      "132\t400\t0.3\t1\t378 22\n",
      "133\t460\t0.3\t1\t436 24\n",
      "134\t592\t0.3\t1\t573 19\n",
      "135\t262\t0.3\t1\t249 13\n",
      "136\t158\t0.3\t1\t150 8\n",
      "137\t105\t0.3\t1\t102 3\n",
      "138\t83\t0.3\t1\t78 5\n",
      "139\t62\t0.3\t1\t57 5\n",
      "140\t55\t0.3\t1\t49 6\n",
      "141\t58\t0.3\t1\t56 2\n",
      "142\t36\t0.3\t1\t36\n",
      "143\t49\t0.3\t1\t47 2\n",
      "144\t33\t0.3\t1\t30 3\n",
      "145\t52\t0.3\t1\t45 7\n",
      "146\t154\t0.3\t1\t152 2\n",
      "147\t543\t0.3\t1\t532 11\n",
      "148\t399\t0.3\t1\t383 16\n",
      "149\t287\t0.3\t1\t273 14\n",
      "150\t2663\t0.3\t1\t2573 90\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17243447 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t23344 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 733.43 s (43 us/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,174,050\n",
      "Reads with adapters:                13,278,050 (77.3%)\n",
      "Reads written (passing filters):    17,174,050 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,107,500 bp\n",
      "Quality-trimmed:               5,444,792 bp (0.2%)\n",
      "Total written (filtered):  2,038,071,183 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13278050 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.3%\n",
      "  G: 20.5%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2359801\t4293512.5\t0\t2359801\n",
      "2\t349363\t1073378.1\t0\t349363\n",
      "3\t195448\t268344.5\t0\t195448\n",
      "4\t112766\t67086.1\t0\t112766\n",
      "5\t92768\t16771.5\t0\t92768\n",
      "6\t90305\t4192.9\t0\t90305\n",
      "7\t71626\t1048.2\t0\t71626\n",
      "8\t92261\t262.1\t0\t92261\n",
      "9\t91976\t65.5\t0\t90926 1050\n",
      "10\t85467\t16.4\t1\t79925 5542\n",
      "11\t96691\t4.1\t1\t88355 8336\n",
      "12\t91713\t1.0\t1\t84839 6874\n",
      "13\t92754\t0.3\t1\t85809 6945\n",
      "14\t95464\t0.3\t1\t87621 7843\n",
      "15\t99277\t0.3\t1\t91369 7908\n",
      "16\t97348\t0.3\t1\t88681 8667\n",
      "17\t119420\t0.3\t1\t108369 11051\n",
      "18\t98719\t0.3\t1\t91552 7167\n",
      "19\t85310\t0.3\t1\t78884 6426\n",
      "20\t103518\t0.3\t1\t95770 7748\n",
      "21\t115588\t0.3\t1\t106188 9400\n",
      "22\t95906\t0.3\t1\t89272 6634\n",
      "23\t115386\t0.3\t1\t106435 8951\n",
      "24\t102949\t0.3\t1\t93582 9367\n",
      "25\t112836\t0.3\t1\t104100 8736\n",
      "26\t100484\t0.3\t1\t93502 6982\n",
      "27\t114811\t0.3\t1\t105307 9504\n",
      "28\t113076\t0.3\t1\t104189 8887\n",
      "29\t118710\t0.3\t1\t108207 10503\n",
      "30\t126214\t0.3\t1\t117742 8472\n",
      "31\t97121\t0.3\t1\t89907 7214\n",
      "32\t120516\t0.3\t1\t111427 9089\n",
      "33\t110848\t0.3\t1\t101943 8905\n",
      "34\t122175\t0.3\t1\t112281 9894\n",
      "35\t122064\t0.3\t1\t111107 10957\n",
      "36\t119580\t0.3\t1\t110830 8750\n",
      "37\t130636\t0.3\t1\t120649 9987\n",
      "38\t115572\t0.3\t1\t106536 9036\n",
      "39\t129905\t0.3\t1\t119211 10694\n",
      "40\t124968\t0.3\t1\t115039 9929\n",
      "41\t145398\t0.3\t1\t132929 12469\n",
      "42\t108151\t0.3\t1\t98831 9320\n",
      "43\t190551\t0.3\t1\t178361 12190\n",
      "44\t58099\t0.3\t1\t52480 5619\n",
      "45\t130794\t0.3\t1\t121367 9427\n",
      "46\t121400\t0.3\t1\t112370 9030\n",
      "47\t129048\t0.3\t1\t119631 9417\n",
      "48\t126222\t0.3\t1\t116730 9492\n",
      "49\t139750\t0.3\t1\t128710 11040\n",
      "50\t117206\t0.3\t1\t109130 8076\n",
      "51\t127012\t0.3\t1\t117729 9283\n",
      "52\t132886\t0.3\t1\t122231 10655\n",
      "53\t141918\t0.3\t1\t132027 9891\n",
      "54\t110897\t0.3\t1\t103000 7897\n",
      "55\t132094\t0.3\t1\t122452 9642\n",
      "56\t162870\t0.3\t1\t151676 11194\n",
      "57\t103170\t0.3\t1\t94887 8283\n",
      "58\t132668\t0.3\t1\t124347 8321\n",
      "59\t116116\t0.3\t1\t107462 8654\n",
      "60\t129021\t0.3\t1\t119747 9274\n",
      "61\t124026\t0.3\t1\t115604 8422\n",
      "62\t137460\t0.3\t1\t128199 9261\n",
      "63\t110698\t0.3\t1\t102173 8525\n",
      "64\t120813\t0.3\t1\t112906 7907\n",
      "65\t134938\t0.3\t1\t125626 9312\n",
      "66\t138037\t0.3\t1\t128352 9685\n",
      "67\t146504\t0.3\t1\t134535 11969\n",
      "68\t277060\t0.3\t1\t261442 15618\n",
      "69\t59870\t0.3\t1\t54647 5223\n",
      "70\t22664\t0.3\t1\t19044 3620\n",
      "71\t71972\t0.3\t1\t67240 4732\n",
      "72\t103041\t0.3\t1\t96552 6489\n",
      "73\t110046\t0.3\t1\t102755 7291\n",
      "74\t107843\t0.3\t1\t100538 7305\n",
      "75\t106162\t0.3\t1\t98772 7390\n",
      "76\t105323\t0.3\t1\t97926 7397\n",
      "77\t104408\t0.3\t1\t97267 7141\n",
      "78\t97073\t0.3\t1\t90169 6904\n",
      "79\t96839\t0.3\t1\t89879 6960\n",
      "80\t93988\t0.3\t1\t87339 6649\n",
      "81\t93546\t0.3\t1\t86918 6628\n",
      "82\t90241\t0.3\t1\t83957 6284\n",
      "83\t86944\t0.3\t1\t80903 6041\n",
      "84\t83168\t0.3\t1\t77255 5913\n",
      "85\t80437\t0.3\t1\t74815 5622\n",
      "86\t77338\t0.3\t1\t71860 5478\n",
      "87\t73439\t0.3\t1\t68315 5124\n",
      "88\t73001\t0.3\t1\t67818 5183\n",
      "89\t68544\t0.3\t1\t63628 4916\n",
      "90\t68084\t0.3\t1\t63234 4850\n",
      "91\t64534\t0.3\t1\t60016 4518\n",
      "92\t61115\t0.3\t1\t56823 4292\n",
      "93\t56566\t0.3\t1\t52542 4024\n",
      "94\t55074\t0.3\t1\t51146 3928\n",
      "95\t50909\t0.3\t1\t47255 3654\n",
      "96\t48357\t0.3\t1\t44862 3495\n",
      "97\t45610\t0.3\t1\t42541 3069\n",
      "98\t41384\t0.3\t1\t38355 3029\n",
      "99\t39655\t0.3\t1\t36624 3031\n",
      "100\t36325\t0.3\t1\t33739 2586\n",
      "101\t33659\t0.3\t1\t31212 2447\n",
      "102\t31157\t0.3\t1\t28883 2274\n",
      "103\t28746\t0.3\t1\t26792 1954\n",
      "104\t26425\t0.3\t1\t24518 1907\n",
      "105\t23705\t0.3\t1\t22021 1684\n",
      "106\t21201\t0.3\t1\t19619 1582\n",
      "107\t19613\t0.3\t1\t18232 1381\n",
      "108\t17790\t0.3\t1\t16434 1356\n",
      "109\t16006\t0.3\t1\t14830 1176\n",
      "110\t14326\t0.3\t1\t13263 1063\n",
      "111\t12771\t0.3\t1\t11779 992\n",
      "112\t11222\t0.3\t1\t10318 904\n",
      "113\t9534\t0.3\t1\t8824 710\n",
      "114\t8358\t0.3\t1\t7698 660\n",
      "115\t7285\t0.3\t1\t6692 593\n",
      "116\t6411\t0.3\t1\t5816 595\n",
      "117\t5570\t0.3\t1\t5108 462\n",
      "118\t4832\t0.3\t1\t4420 412\n",
      "119\t3906\t0.3\t1\t3549 357\n",
      "120\t3397\t0.3\t1\t3108 289\n",
      "121\t2929\t0.3\t1\t2664 265\n",
      "122\t2444\t0.3\t1\t2212 232\n",
      "123\t2198\t0.3\t1\t1992 206\n",
      "124\t1767\t0.3\t1\t1602 165\n",
      "125\t1386\t0.3\t1\t1230 156\n",
      "126\t1228\t0.3\t1\t1089 139\n",
      "127\t1105\t0.3\t1\t989 116\n",
      "128\t786\t0.3\t1\t705 81\n",
      "129\t664\t0.3\t1\t591 73\n",
      "130\t581\t0.3\t1\t508 73\n",
      "131\t479\t0.3\t1\t435 44\n",
      "132\t414\t0.3\t1\t361 53\n",
      "133\t450\t0.3\t1\t408 42\n",
      "134\t646\t0.3\t1\t602 44\n",
      "135\t289\t0.3\t1\t262 27\n",
      "136\t173\t0.3\t1\t154 19\n",
      "137\t125\t0.3\t1\t108 17\n",
      "138\t98\t0.3\t1\t88 10\n",
      "139\t90\t0.3\t1\t77 13\n",
      "140\t54\t0.3\t1\t46 8\n",
      "141\t52\t0.3\t1\t45 7\n",
      "142\t38\t0.3\t1\t35 3\n",
      "143\t50\t0.3\t1\t43 7\n",
      "144\t42\t0.3\t1\t37 5\n",
      "145\t46\t0.3\t1\t45 1\n",
      "146\t125\t0.3\t1\t116 9\n",
      "147\t651\t0.3\t1\t623 28\n",
      "148\t442\t0.3\t1\t421 21\n",
      "149\t216\t0.3\t1\t204 12\n",
      "150\t2921\t0.3\t1\t2713 208\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17174050 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7457 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_2_S14_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 765.50 s (45 us/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,174,050\n",
      "Reads with adapters:                13,208,659 (76.9%)\n",
      "Reads written (passing filters):    17,174,050 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,107,500 bp\n",
      "Quality-trimmed:              11,281,504 bp (0.4%)\n",
      "Total written (filtered):  2,040,417,795 bp (79.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13208659 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 12.8%\n",
      "  G: 23.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2393304\t4293512.5\t0\t2393304\n",
      "2\t355208\t1073378.1\t0\t355208\n",
      "3\t207534\t268344.5\t0\t207534\n",
      "4\t112818\t67086.1\t0\t112818\n",
      "5\t98722\t16771.5\t0\t98722\n",
      "6\t88016\t4192.9\t0\t88016\n",
      "7\t70973\t1048.2\t0\t70973\n",
      "8\t100906\t262.1\t0\t100906\n",
      "9\t80596\t65.5\t0\t80310 286\n",
      "10\t85921\t16.4\t1\t80103 5818\n",
      "11\t93456\t4.1\t1\t85698 7758\n",
      "12\t91973\t1.0\t1\t84465 7508\n",
      "13\t105255\t0.3\t1\t96925 8330\n",
      "14\t97607\t0.3\t1\t89028 8579\n",
      "15\t80440\t0.3\t1\t74131 6309\n",
      "16\t114723\t0.3\t1\t105648 9075\n",
      "17\t95977\t0.3\t1\t87931 8046\n",
      "18\t85383\t0.3\t1\t78648 6735\n",
      "19\t112212\t0.3\t1\t103391 8821\n",
      "20\t104516\t0.3\t1\t96596 7920\n",
      "21\t91790\t0.3\t1\t84587 7203\n",
      "22\t103890\t0.3\t1\t96083 7807\n",
      "23\t109193\t0.3\t1\t100822 8371\n",
      "24\t108259\t0.3\t1\t99753 8506\n",
      "25\t121670\t0.3\t1\t113120 8550\n",
      "26\t100521\t0.3\t1\t92626 7895\n",
      "27\t113578\t0.3\t1\t104198 9380\n",
      "28\t101690\t0.3\t1\t94983 6707\n",
      "29\t113641\t0.3\t1\t105222 8419\n",
      "30\t108610\t0.3\t1\t101210 7400\n",
      "31\t120920\t0.3\t1\t112244 8676\n",
      "32\t104000\t0.3\t1\t96851 7149\n",
      "33\t120229\t0.3\t1\t111277 8952\n",
      "34\t153685\t0.3\t1\t142847 10838\n",
      "35\t83436\t0.3\t1\t77543 5893\n",
      "36\t120970\t0.3\t1\t112035 8935\n",
      "37\t127408\t0.3\t1\t118333 9075\n",
      "38\t119755\t0.3\t1\t112084 7671\n",
      "39\t113420\t0.3\t1\t106417 7003\n",
      "40\t129274\t0.3\t1\t120419 8855\n",
      "41\t124811\t0.3\t1\t116558 8253\n",
      "42\t127899\t0.3\t1\t119534 8365\n",
      "43\t115450\t0.3\t1\t107954 7496\n",
      "44\t133939\t0.3\t1\t124786 9153\n",
      "45\t139673\t0.3\t1\t130506 9167\n",
      "46\t102301\t0.3\t1\t95697 6604\n",
      "47\t129785\t0.3\t1\t121923 7862\n",
      "48\t125953\t0.3\t1\t118248 7705\n",
      "49\t131617\t0.3\t1\t122911 8706\n",
      "50\t126447\t0.3\t1\t118299 8148\n",
      "51\t153054\t0.3\t1\t143414 9640\n",
      "52\t111902\t0.3\t1\t104877 7025\n",
      "53\t136845\t0.3\t1\t129094 7751\n",
      "54\t100542\t0.3\t1\t94550 5992\n",
      "55\t131225\t0.3\t1\t123466 7759\n",
      "56\t134910\t0.3\t1\t126386 8524\n",
      "57\t127245\t0.3\t1\t120240 7005\n",
      "58\t123733\t0.3\t1\t116761 6972\n",
      "59\t122876\t0.3\t1\t115856 7020\n",
      "60\t129560\t0.3\t1\t122153 7407\n",
      "61\t129535\t0.3\t1\t122248 7287\n",
      "62\t141327\t0.3\t1\t132778 8549\n",
      "63\t187124\t0.3\t1\t177463 9661\n",
      "64\t148865\t0.3\t1\t142160 6705\n",
      "65\t101493\t0.3\t1\t96017 5476\n",
      "66\t109966\t0.3\t1\t103977 5989\n",
      "67\t109147\t0.3\t1\t102972 6175\n",
      "68\t110252\t0.3\t1\t104088 6164\n",
      "69\t109266\t0.3\t1\t102978 6288\n",
      "70\t115917\t0.3\t1\t109437 6480\n",
      "71\t118081\t0.3\t1\t111469 6612\n",
      "72\t111629\t0.3\t1\t105360 6269\n",
      "73\t116346\t0.3\t1\t109876 6470\n",
      "74\t106757\t0.3\t1\t100604 6153\n",
      "75\t101733\t0.3\t1\t95970 5763\n",
      "76\t99872\t0.3\t1\t94223 5649\n",
      "77\t99194\t0.3\t1\t93520 5674\n",
      "78\t93229\t0.3\t1\t87953 5276\n",
      "79\t90739\t0.3\t1\t85605 5134\n",
      "80\t89347\t0.3\t1\t84416 4931\n",
      "81\t88970\t0.3\t1\t84141 4829\n",
      "82\t84536\t0.3\t1\t79929 4607\n",
      "83\t81530\t0.3\t1\t77128 4402\n",
      "84\t78480\t0.3\t1\t74228 4252\n",
      "85\t77346\t0.3\t1\t73110 4236\n",
      "86\t75912\t0.3\t1\t71909 4003\n",
      "87\t75393\t0.3\t1\t71530 3863\n",
      "88\t73839\t0.3\t1\t69922 3917\n",
      "89\t65607\t0.3\t1\t62233 3374\n",
      "90\t63315\t0.3\t1\t59995 3320\n",
      "91\t59514\t0.3\t1\t56442 3072\n",
      "92\t55456\t0.3\t1\t52465 2991\n",
      "93\t52632\t0.3\t1\t49948 2684\n",
      "94\t51163\t0.3\t1\t48481 2682\n",
      "95\t47871\t0.3\t1\t45320 2551\n",
      "96\t45902\t0.3\t1\t43474 2428\n",
      "97\t43468\t0.3\t1\t41231 2237\n",
      "98\t40225\t0.3\t1\t38177 2048\n",
      "99\t39861\t0.3\t1\t37850 2011\n",
      "100\t35922\t0.3\t1\t34182 1740\n",
      "101\t31906\t0.3\t1\t30324 1582\n",
      "102\t28881\t0.3\t1\t27260 1621\n",
      "103\t26612\t0.3\t1\t25202 1410\n",
      "104\t25092\t0.3\t1\t23747 1345\n",
      "105\t22553\t0.3\t1\t21323 1230\n",
      "106\t20249\t0.3\t1\t19195 1054\n",
      "107\t18698\t0.3\t1\t17704 994\n",
      "108\t16333\t0.3\t1\t15423 910\n",
      "109\t15105\t0.3\t1\t14342 763\n",
      "110\t13538\t0.3\t1\t12818 720\n",
      "111\t12635\t0.3\t1\t11928 707\n",
      "112\t10864\t0.3\t1\t10306 558\n",
      "113\t9370\t0.3\t1\t8881 489\n",
      "114\t8127\t0.3\t1\t7692 435\n",
      "115\t6826\t0.3\t1\t6440 386\n",
      "116\t5914\t0.3\t1\t5590 324\n",
      "117\t5167\t0.3\t1\t4841 326\n",
      "118\t4505\t0.3\t1\t4244 261\n",
      "119\t3657\t0.3\t1\t3441 216\n",
      "120\t3157\t0.3\t1\t2964 193\n",
      "121\t2768\t0.3\t1\t2620 148\n",
      "122\t2319\t0.3\t1\t2199 120\n",
      "123\t2119\t0.3\t1\t1978 141\n",
      "124\t1696\t0.3\t1\t1576 120\n",
      "125\t1359\t0.3\t1\t1259 100\n",
      "126\t1191\t0.3\t1\t1105 86\n",
      "127\t1007\t0.3\t1\t955 52\n",
      "128\t721\t0.3\t1\t673 48\n",
      "129\t632\t0.3\t1\t599 33\n",
      "130\t537\t0.3\t1\t494 43\n",
      "131\t454\t0.3\t1\t423 31\n",
      "132\t387\t0.3\t1\t367 20\n",
      "133\t461\t0.3\t1\t442 19\n",
      "134\t637\t0.3\t1\t611 26\n",
      "135\t279\t0.3\t1\t264 15\n",
      "136\t164\t0.3\t1\t153 11\n",
      "137\t121\t0.3\t1\t110 11\n",
      "138\t99\t0.3\t1\t89 10\n",
      "139\t86\t0.3\t1\t79 7\n",
      "140\t59\t0.3\t1\t52 7\n",
      "141\t49\t0.3\t1\t44 5\n",
      "142\t39\t0.3\t1\t36 3\n",
      "143\t48\t0.3\t1\t45 3\n",
      "144\t41\t0.3\t1\t36 5\n",
      "145\t47\t0.3\t1\t41 6\n",
      "146\t121\t0.3\t1\t118 3\n",
      "147\t643\t0.3\t1\t616 27\n",
      "148\t431\t0.3\t1\t416 15\n",
      "149\t244\t0.3\t1\t225 19\n",
      "150\t2704\t0.3\t1\t2570 134\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17174050 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t22162 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 608.39 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,705,097\n",
      "Reads with adapters:                11,084,660 (75.4%)\n",
      "Reads written (passing filters):    14,705,097 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,205,764,550 bp\n",
      "Quality-trimmed:               5,654,261 bp (0.3%)\n",
      "Total written (filtered):  1,768,343,252 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11084660 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 18.1%\n",
      "  G: 19.4%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2274798\t3676274.2\t0\t2274798\n",
      "2\t348366\t919068.6\t0\t348366\n",
      "3\t204139\t229767.1\t0\t204139\n",
      "4\t102009\t57441.8\t0\t102009\n",
      "5\t74565\t14360.4\t0\t74565\n",
      "6\t72782\t3590.1\t0\t72782\n",
      "7\t57000\t897.5\t0\t57000\n",
      "8\t74016\t224.4\t0\t74016\n",
      "9\t74077\t56.1\t0\t73070 1007\n",
      "10\t67767\t14.0\t1\t63550 4217\n",
      "11\t78426\t3.5\t1\t71779 6647\n",
      "12\t73178\t0.9\t1\t67904 5274\n",
      "13\t73121\t0.2\t1\t67865 5256\n",
      "14\t76375\t0.2\t1\t70227 6148\n",
      "15\t78023\t0.2\t1\t72295 5728\n",
      "16\t76757\t0.2\t1\t70359 6398\n",
      "17\t96301\t0.2\t1\t87532 8769\n",
      "18\t78109\t0.2\t1\t72884 5225\n",
      "19\t66841\t0.2\t1\t61960 4881\n",
      "20\t81976\t0.2\t1\t75979 5997\n",
      "21\t91230\t0.2\t1\t84192 7038\n",
      "22\t74923\t0.2\t1\t70026 4897\n",
      "23\t89318\t0.2\t1\t82578 6740\n",
      "24\t80595\t0.2\t1\t73239 7356\n",
      "25\t88530\t0.2\t1\t81693 6837\n",
      "26\t80024\t0.2\t1\t74706 5318\n",
      "27\t90563\t0.2\t1\t83032 7531\n",
      "28\t88117\t0.2\t1\t81366 6751\n",
      "29\t90600\t0.2\t1\t82528 8072\n",
      "30\t100870\t0.2\t1\t94452 6418\n",
      "31\t72291\t0.2\t1\t67161 5130\n",
      "32\t93105\t0.2\t1\t86376 6729\n",
      "33\t85828\t0.2\t1\t79302 6526\n",
      "34\t93973\t0.2\t1\t87113 6860\n",
      "35\t95201\t0.2\t1\t86703 8498\n",
      "36\t98814\t0.2\t1\t91143 7671\n",
      "37\t92594\t0.2\t1\t86632 5962\n",
      "38\t92492\t0.2\t1\t85700 6792\n",
      "39\t99594\t0.2\t1\t92552 7042\n",
      "40\t99241\t0.2\t1\t91728 7513\n",
      "41\t109705\t0.2\t1\t100318 9387\n",
      "42\t91534\t0.2\t1\t84273 7261\n",
      "43\t144391\t0.2\t1\t135954 8437\n",
      "44\t48359\t0.2\t1\t43755 4604\n",
      "45\t101091\t0.2\t1\t93630 7461\n",
      "46\t97319\t0.2\t1\t90173 7146\n",
      "47\t102355\t0.2\t1\t95522 6833\n",
      "48\t95140\t0.2\t1\t87635 7505\n",
      "49\t109171\t0.2\t1\t100789 8382\n",
      "50\t88275\t0.2\t1\t82648 5627\n",
      "51\t96905\t0.2\t1\t90014 6891\n",
      "52\t106355\t0.2\t1\t97463 8892\n",
      "53\t116272\t0.2\t1\t109052 7220\n",
      "54\t82504\t0.2\t1\t76626 5878\n",
      "55\t104155\t0.2\t1\t97146 7009\n",
      "56\t117878\t0.2\t1\t110588 7290\n",
      "57\t94064\t0.2\t1\t87248 6816\n",
      "58\t98547\t0.2\t1\t93098 5449\n",
      "59\t78147\t0.2\t1\t72464 5683\n",
      "60\t103609\t0.2\t1\t96779 6830\n",
      "61\t99715\t0.2\t1\t93367 6348\n",
      "62\t96712\t0.2\t1\t90590 6122\n",
      "63\t91683\t0.2\t1\t85401 6282\n",
      "64\t88404\t0.2\t1\t82993 5411\n",
      "65\t98564\t0.2\t1\t92125 6439\n",
      "66\t106366\t0.2\t1\t99301 7065\n",
      "67\t113416\t0.2\t1\t104214 9202\n",
      "68\t212253\t0.2\t1\t201102 11151\n",
      "69\t52200\t0.2\t1\t48055 4145\n",
      "70\t18616\t0.2\t1\t15750 2866\n",
      "71\t46522\t0.2\t1\t43178 3344\n",
      "72\t78090\t0.2\t1\t73258 4832\n",
      "73\t85268\t0.2\t1\t79938 5330\n",
      "74\t84510\t0.2\t1\t79145 5365\n",
      "75\t84172\t0.2\t1\t78812 5360\n",
      "76\t82514\t0.2\t1\t77266 5248\n",
      "77\t82709\t0.2\t1\t77348 5361\n",
      "78\t77578\t0.2\t1\t72635 4943\n",
      "79\t76394\t0.2\t1\t71265 5129\n",
      "80\t74740\t0.2\t1\t69853 4887\n",
      "81\t73428\t0.2\t1\t68638 4790\n",
      "82\t72487\t0.2\t1\t67836 4651\n",
      "83\t69366\t0.2\t1\t64850 4516\n",
      "84\t68485\t0.2\t1\t64014 4471\n",
      "85\t65855\t0.2\t1\t61694 4161\n",
      "86\t63892\t0.2\t1\t59720 4172\n",
      "87\t59968\t0.2\t1\t56023 3945\n",
      "88\t59481\t0.2\t1\t55667 3814\n",
      "89\t57389\t0.2\t1\t53698 3691\n",
      "90\t56546\t0.2\t1\t52783 3763\n",
      "91\t53907\t0.2\t1\t50295 3612\n",
      "92\t51001\t0.2\t1\t47631 3370\n",
      "93\t48245\t0.2\t1\t45064 3181\n",
      "94\t47040\t0.2\t1\t43981 3059\n",
      "95\t44130\t0.2\t1\t41149 2981\n",
      "96\t41844\t0.2\t1\t38970 2874\n",
      "97\t39077\t0.2\t1\t36479 2598\n",
      "98\t36411\t0.2\t1\t33904 2507\n",
      "99\t35258\t0.2\t1\t32713 2545\n",
      "100\t32027\t0.2\t1\t29780 2247\n",
      "101\t30331\t0.2\t1\t28259 2072\n",
      "102\t28208\t0.2\t1\t26138 2070\n",
      "103\t26173\t0.2\t1\t24325 1848\n",
      "104\t24471\t0.2\t1\t22816 1655\n",
      "105\t21780\t0.2\t1\t20187 1593\n",
      "106\t20446\t0.2\t1\t18956 1490\n",
      "107\t18149\t0.2\t1\t16771 1378\n",
      "108\t17180\t0.2\t1\t15945 1235\n",
      "109\t15082\t0.2\t1\t13906 1176\n",
      "110\t14103\t0.2\t1\t12998 1105\n",
      "111\t13190\t0.2\t1\t12191 999\n",
      "112\t11450\t0.2\t1\t10565 885\n",
      "113\t9717\t0.2\t1\t8858 859\n",
      "114\t8899\t0.2\t1\t8160 739\n",
      "115\t7941\t0.2\t1\t7316 625\n",
      "116\t6735\t0.2\t1\t6226 509\n",
      "117\t5866\t0.2\t1\t5379 487\n",
      "118\t5055\t0.2\t1\t4584 471\n",
      "119\t4372\t0.2\t1\t3977 395\n",
      "120\t3711\t0.2\t1\t3362 349\n",
      "121\t3540\t0.2\t1\t3243 297\n",
      "122\t2903\t0.2\t1\t2643 260\n",
      "123\t2410\t0.2\t1\t2172 238\n",
      "124\t2227\t0.2\t1\t2020 207\n",
      "125\t1832\t0.2\t1\t1640 192\n",
      "126\t1658\t0.2\t1\t1492 166\n",
      "127\t1471\t0.2\t1\t1327 144\n",
      "128\t1080\t0.2\t1\t953 127\n",
      "129\t958\t0.2\t1\t848 110\n",
      "130\t709\t0.2\t1\t628 81\n",
      "131\t795\t0.2\t1\t718 77\n",
      "132\t862\t0.2\t1\t774 88\n",
      "133\t1124\t0.2\t1\t1033 91\n",
      "134\t1984\t0.2\t1\t1883 101\n",
      "135\t812\t0.2\t1\t780 32\n",
      "136\t310\t0.2\t1\t269 41\n",
      "137\t196\t0.2\t1\t180 16\n",
      "138\t185\t0.2\t1\t168 17\n",
      "139\t138\t0.2\t1\t131 7\n",
      "140\t94\t0.2\t1\t84 10\n",
      "141\t74\t0.2\t1\t68 6\n",
      "142\t62\t0.2\t1\t50 12\n",
      "143\t92\t0.2\t1\t84 8\n",
      "144\t74\t0.2\t1\t67 7\n",
      "145\t205\t0.2\t1\t196 9\n",
      "146\t780\t0.2\t1\t755 25\n",
      "147\t4835\t0.2\t1\t4652 183\n",
      "148\t3283\t0.2\t1\t3150 133\n",
      "149\t1826\t0.2\t1\t1757 69\n",
      "150\t16714\t0.2\t1\t15810 904\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14705097 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t34522 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 625.12 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,705,097\n",
      "Reads with adapters:                10,980,117 (74.7%)\n",
      "Reads written (passing filters):    14,705,097 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,205,764,550 bp\n",
      "Quality-trimmed:              13,510,954 bp (0.6%)\n",
      "Total written (filtered):  1,769,417,806 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10980117 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 14.1%\n",
      "  G: 23.8%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2264307\t3676274.2\t0\t2264307\n",
      "2\t373214\t919068.6\t0\t373214\n",
      "3\t218063\t229767.1\t0\t218063\n",
      "4\t104965\t57441.8\t0\t104965\n",
      "5\t83066\t14360.4\t0\t83066\n",
      "6\t70406\t3590.1\t0\t70406\n",
      "7\t54672\t897.5\t0\t54672\n",
      "8\t80609\t224.4\t0\t80609\n",
      "9\t63967\t56.1\t0\t63675 292\n",
      "10\t69384\t14.0\t1\t65411 3973\n",
      "11\t74885\t3.5\t1\t69453 5432\n",
      "12\t74890\t0.9\t1\t69291 5599\n",
      "13\t82476\t0.2\t1\t76999 5477\n",
      "14\t78491\t0.2\t1\t72263 6228\n",
      "15\t61874\t0.2\t1\t57566 4308\n",
      "16\t91303\t0.2\t1\t84745 6558\n",
      "17\t72220\t0.2\t1\t66746 5474\n",
      "18\t69582\t0.2\t1\t64867 4715\n",
      "19\t88040\t0.2\t1\t81744 6296\n",
      "20\t81620\t0.2\t1\t76073 5547\n",
      "21\t72843\t0.2\t1\t67713 5130\n",
      "22\t82391\t0.2\t1\t76741 5650\n",
      "23\t83759\t0.2\t1\t77887 5872\n",
      "24\t82590\t0.2\t1\t76725 5865\n",
      "25\t94974\t0.2\t1\t88688 6286\n",
      "26\t78108\t0.2\t1\t72602 5506\n",
      "27\t87345\t0.2\t1\t80708 6637\n",
      "28\t80301\t0.2\t1\t75686 4615\n",
      "29\t86749\t0.2\t1\t80805 5944\n",
      "30\t84969\t0.2\t1\t79761 5208\n",
      "31\t91560\t0.2\t1\t85496 6064\n",
      "32\t82472\t0.2\t1\t77608 4864\n",
      "33\t91075\t0.2\t1\t85011 6064\n",
      "34\t115701\t0.2\t1\t108310 7391\n",
      "35\t65587\t0.2\t1\t61631 3956\n",
      "36\t92396\t0.2\t1\t86112 6284\n",
      "37\t97880\t0.2\t1\t91450 6430\n",
      "38\t93812\t0.2\t1\t88555 5257\n",
      "39\t89857\t0.2\t1\t84658 5199\n",
      "40\t100686\t0.2\t1\t94165 6521\n",
      "41\t95917\t0.2\t1\t90320 5597\n",
      "42\t99426\t0.2\t1\t93710 5716\n",
      "43\t89895\t0.2\t1\t84632 5263\n",
      "44\t101978\t0.2\t1\t95733 6245\n",
      "45\t107145\t0.2\t1\t100731 6414\n",
      "46\t78559\t0.2\t1\t74098 4461\n",
      "47\t99270\t0.2\t1\t93634 5636\n",
      "48\t97566\t0.2\t1\t92156 5410\n",
      "49\t101230\t0.2\t1\t95394 5836\n",
      "50\t95465\t0.2\t1\t89881 5584\n",
      "51\t119107\t0.2\t1\t112449 6658\n",
      "52\t84719\t0.2\t1\t80119 4600\n",
      "53\t101734\t0.2\t1\t96544 5190\n",
      "54\t78783\t0.2\t1\t74521 4262\n",
      "55\t102169\t0.2\t1\t97154 5015\n",
      "56\t104926\t0.2\t1\t99510 5416\n",
      "57\t97330\t0.2\t1\t92599 4731\n",
      "58\t94743\t0.2\t1\t90360 4383\n",
      "59\t96773\t0.2\t1\t91767 5006\n",
      "60\t98229\t0.2\t1\t93172 5057\n",
      "61\t101518\t0.2\t1\t96208 5310\n",
      "62\t108759\t0.2\t1\t103148 5611\n",
      "63\t145227\t0.2\t1\t138420 6807\n",
      "64\t118502\t0.2\t1\t113544 4958\n",
      "65\t77875\t0.2\t1\t74180 3695\n",
      "66\t85279\t0.2\t1\t81158 4121\n",
      "67\t82906\t0.2\t1\t78696 4210\n",
      "68\t83568\t0.2\t1\t79559 4009\n",
      "69\t83055\t0.2\t1\t79009 4046\n",
      "70\t85605\t0.2\t1\t81510 4095\n",
      "71\t91438\t0.2\t1\t87015 4423\n",
      "72\t86608\t0.2\t1\t82445 4163\n",
      "73\t91905\t0.2\t1\t87632 4273\n",
      "74\t84442\t0.2\t1\t80649 3793\n",
      "75\t80482\t0.2\t1\t76627 3855\n",
      "76\t78424\t0.2\t1\t74783 3641\n",
      "77\t78513\t0.2\t1\t74716 3797\n",
      "78\t74066\t0.2\t1\t70442 3624\n",
      "79\t70946\t0.2\t1\t67597 3349\n",
      "80\t70802\t0.2\t1\t67560 3242\n",
      "81\t69586\t0.2\t1\t66414 3172\n",
      "82\t67330\t0.2\t1\t64176 3154\n",
      "83\t64772\t0.2\t1\t61853 2919\n",
      "84\t63911\t0.2\t1\t60972 2939\n",
      "85\t62434\t0.2\t1\t59365 3069\n",
      "86\t62539\t0.2\t1\t59521 3018\n",
      "87\t61465\t0.2\t1\t58594 2871\n",
      "88\t60482\t0.2\t1\t57646 2836\n",
      "89\t54895\t0.2\t1\t52333 2562\n",
      "90\t52052\t0.2\t1\t49599 2453\n",
      "91\t49106\t0.2\t1\t46822 2284\n",
      "92\t45843\t0.2\t1\t43672 2171\n",
      "93\t44679\t0.2\t1\t42566 2113\n",
      "94\t43439\t0.2\t1\t41408 2031\n",
      "95\t41369\t0.2\t1\t39446 1923\n",
      "96\t39301\t0.2\t1\t37521 1780\n",
      "97\t36285\t0.2\t1\t34613 1672\n",
      "98\t34034\t0.2\t1\t32424 1610\n",
      "99\t33720\t0.2\t1\t32265 1455\n",
      "100\t30495\t0.2\t1\t29039 1456\n",
      "101\t27811\t0.2\t1\t26487 1324\n",
      "102\t25647\t0.2\t1\t24471 1176\n",
      "103\t23848\t0.2\t1\t22756 1092\n",
      "104\t23030\t0.2\t1\t22018 1012\n",
      "105\t20481\t0.2\t1\t19504 977\n",
      "106\t19291\t0.2\t1\t18417 874\n",
      "107\t17027\t0.2\t1\t16232 795\n",
      "108\t15744\t0.2\t1\t15015 729\n",
      "109\t13986\t0.2\t1\t13295 691\n",
      "110\t13128\t0.2\t1\t12523 605\n",
      "111\t12855\t0.2\t1\t12274 581\n",
      "112\t11119\t0.2\t1\t10542 577\n",
      "113\t9437\t0.2\t1\t8968 469\n",
      "114\t8605\t0.2\t1\t8220 385\n",
      "115\t7442\t0.2\t1\t7067 375\n",
      "116\t6235\t0.2\t1\t5927 308\n",
      "117\t5342\t0.2\t1\t5066 276\n",
      "118\t4634\t0.2\t1\t4366 268\n",
      "119\t4041\t0.2\t1\t3836 205\n",
      "120\t3372\t0.2\t1\t3167 205\n",
      "121\t3329\t0.2\t1\t3149 180\n",
      "122\t2717\t0.2\t1\t2578 139\n",
      "123\t2317\t0.2\t1\t2172 145\n",
      "124\t2087\t0.2\t1\t1969 118\n",
      "125\t1757\t0.2\t1\t1630 127\n",
      "126\t1582\t0.2\t1\t1504 78\n",
      "127\t1368\t0.2\t1\t1287 81\n",
      "128\t1019\t0.2\t1\t960 59\n",
      "129\t887\t0.2\t1\t843 44\n",
      "130\t669\t0.2\t1\t637 32\n",
      "131\t775\t0.2\t1\t731 44\n",
      "132\t814\t0.2\t1\t771 43\n",
      "133\t1245\t0.2\t1\t1202 43\n",
      "134\t1982\t0.2\t1\t1926 56\n",
      "135\t810\t0.2\t1\t778 32\n",
      "136\t281\t0.2\t1\t260 21\n",
      "137\t198\t0.2\t1\t181 17\n",
      "138\t183\t0.2\t1\t171 12\n",
      "139\t135\t0.2\t1\t124 11\n",
      "140\t85\t0.2\t1\t83 2\n",
      "141\t71\t0.2\t1\t66 5\n",
      "142\t55\t0.2\t1\t52 3\n",
      "143\t87\t0.2\t1\t84 3\n",
      "144\t82\t0.2\t1\t77 5\n",
      "145\t214\t0.2\t1\t204 10\n",
      "146\t799\t0.2\t1\t768 31\n",
      "147\t4799\t0.2\t1\t4649 150\n",
      "148\t3222\t0.2\t1\t3110 112\n",
      "149\t1903\t0.2\t1\t1826 77\n",
      "150\t15860\t0.2\t1\t15305 555\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14705097 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t55347 (0.4%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 595.20 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,622,552\n",
      "Reads with adapters:                11,026,853 (75.4%)\n",
      "Reads written (passing filters):    14,622,552 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,193,382,800 bp\n",
      "Quality-trimmed:               5,795,035 bp (0.3%)\n",
      "Total written (filtered):  1,756,312,964 bp (80.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11026853 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.2%\n",
      "  C: 18.2%\n",
      "  G: 19.4%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2255035\t3655638.0\t0\t2255035\n",
      "2\t341709\t913909.5\t0\t341709\n",
      "3\t202869\t228477.4\t0\t202869\n",
      "4\t99930\t57119.3\t0\t99930\n",
      "5\t74502\t14279.8\t0\t74502\n",
      "6\t72604\t3570.0\t0\t72604\n",
      "7\t55151\t892.5\t0\t55151\n",
      "8\t72986\t223.1\t0\t72986\n",
      "9\t72897\t55.8\t0\t71832 1065\n",
      "10\t66371\t13.9\t1\t61637 4734\n",
      "11\t77738\t3.5\t1\t70168 7570\n",
      "12\t72088\t0.9\t1\t66222 5866\n",
      "13\t72584\t0.2\t1\t66413 6171\n",
      "14\t75037\t0.2\t1\t68093 6944\n",
      "15\t77546\t0.2\t1\t70821 6725\n",
      "16\t76138\t0.2\t1\t68711 7427\n",
      "17\t96440\t0.2\t1\t86428 10012\n",
      "18\t77278\t0.2\t1\t71237 6041\n",
      "19\t65884\t0.2\t1\t60354 5530\n",
      "20\t80966\t0.2\t1\t74091 6875\n",
      "21\t91593\t0.2\t1\t83562 8031\n",
      "22\t72795\t0.2\t1\t67343 5452\n",
      "23\t89502\t0.2\t1\t81872 7630\n",
      "24\t80431\t0.2\t1\t72300 8131\n",
      "25\t87978\t0.2\t1\t80303 7675\n",
      "26\t79303\t0.2\t1\t73214 6089\n",
      "27\t90256\t0.2\t1\t81812 8444\n",
      "28\t88151\t0.2\t1\t80669 7482\n",
      "29\t90769\t0.2\t1\t81855 8914\n",
      "30\t100156\t0.2\t1\t92655 7501\n",
      "31\t70749\t0.2\t1\t65028 5721\n",
      "32\t94056\t0.2\t1\t86125 7931\n",
      "33\t85223\t0.2\t1\t77723 7500\n",
      "34\t93938\t0.2\t1\t85684 8254\n",
      "35\t95264\t0.2\t1\t85673 9591\n",
      "36\t98487\t0.2\t1\t89904 8583\n",
      "37\t91367\t0.2\t1\t84496 6871\n",
      "38\t91067\t0.2\t1\t83299 7768\n",
      "39\t98566\t0.2\t1\t90240 8326\n",
      "40\t98669\t0.2\t1\t89677 8992\n",
      "41\t109128\t0.2\t1\t98383 10745\n",
      "42\t90124\t0.2\t1\t81722 8402\n",
      "43\t141587\t0.2\t1\t131681 9906\n",
      "44\t48458\t0.2\t1\t43335 5123\n",
      "45\t101474\t0.2\t1\t92754 8720\n",
      "46\t96951\t0.2\t1\t88935 8016\n",
      "47\t101587\t0.2\t1\t93762 7825\n",
      "48\t93451\t0.2\t1\t85286 8165\n",
      "49\t109864\t0.2\t1\t100369 9495\n",
      "50\t86419\t0.2\t1\t80081 6338\n",
      "51\t97144\t0.2\t1\t89290 7854\n",
      "52\t106661\t0.2\t1\t96904 9757\n",
      "53\t115583\t0.2\t1\t107526 8057\n",
      "54\t81720\t0.2\t1\t74951 6769\n",
      "55\t103125\t0.2\t1\t94975 8150\n",
      "56\t119342\t0.2\t1\t110505 8837\n",
      "57\t94296\t0.2\t1\t86511 7785\n",
      "58\t95906\t0.2\t1\t89537 6369\n",
      "59\t78497\t0.2\t1\t71935 6562\n",
      "60\t103672\t0.2\t1\t95850 7822\n",
      "61\t99103\t0.2\t1\t91989 7114\n",
      "62\t98519\t0.2\t1\t91357 7162\n",
      "63\t92047\t0.2\t1\t85069 6978\n",
      "64\t87141\t0.2\t1\t80859 6282\n",
      "65\t98035\t0.2\t1\t90511 7524\n",
      "66\t104856\t0.2\t1\t96876 7980\n",
      "67\t112499\t0.2\t1\t102554 9945\n",
      "68\t203504\t0.2\t1\t191009 12495\n",
      "69\t62594\t0.2\t1\t57415 5179\n",
      "70\t23252\t0.2\t1\t19824 3428\n",
      "71\t45182\t0.2\t1\t41443 3739\n",
      "72\t77078\t0.2\t1\t71850 5228\n",
      "73\t85031\t0.2\t1\t79046 5985\n",
      "74\t84254\t0.2\t1\t78257 5997\n",
      "75\t83996\t0.2\t1\t77711 6285\n",
      "76\t82731\t0.2\t1\t76446 6285\n",
      "77\t82380\t0.2\t1\t76363 6017\n",
      "78\t77279\t0.2\t1\t71312 5967\n",
      "79\t77209\t0.2\t1\t71223 5986\n",
      "80\t74817\t0.2\t1\t69029 5788\n",
      "81\t73406\t0.2\t1\t67839 5567\n",
      "82\t72738\t0.2\t1\t67207 5531\n",
      "83\t69208\t0.2\t1\t64025 5183\n",
      "84\t68393\t0.2\t1\t63216 5177\n",
      "85\t66034\t0.2\t1\t60887 5147\n",
      "86\t63919\t0.2\t1\t59135 4784\n",
      "87\t60801\t0.2\t1\t56313 4488\n",
      "88\t60788\t0.2\t1\t56093 4695\n",
      "89\t57692\t0.2\t1\t53269 4423\n",
      "90\t56584\t0.2\t1\t52478 4106\n",
      "91\t53589\t0.2\t1\t49546 4043\n",
      "92\t51833\t0.2\t1\t47890 3943\n",
      "93\t48199\t0.2\t1\t44575 3624\n",
      "94\t46944\t0.2\t1\t43278 3666\n",
      "95\t43653\t0.2\t1\t40307 3346\n",
      "96\t41698\t0.2\t1\t38443 3255\n",
      "97\t38960\t0.2\t1\t36083 2877\n",
      "98\t36779\t0.2\t1\t33843 2936\n",
      "99\t34473\t0.2\t1\t31757 2716\n",
      "100\t32693\t0.2\t1\t30095 2598\n",
      "101\t30447\t0.2\t1\t28045 2402\n",
      "102\t27845\t0.2\t1\t25608 2237\n",
      "103\t26414\t0.2\t1\t24289 2125\n",
      "104\t24929\t0.2\t1\t23050 1879\n",
      "105\t21773\t0.2\t1\t20195 1578\n",
      "106\t20226\t0.2\t1\t18586 1640\n",
      "107\t18400\t0.2\t1\t16897 1503\n",
      "108\t17326\t0.2\t1\t15891 1435\n",
      "109\t15645\t0.2\t1\t14342 1303\n",
      "110\t14020\t0.2\t1\t12816 1204\n",
      "111\t12849\t0.2\t1\t11744 1105\n",
      "112\t11234\t0.2\t1\t10286 948\n",
      "113\t9985\t0.2\t1\t9120 865\n",
      "114\t8637\t0.2\t1\t7820 817\n",
      "115\t7857\t0.2\t1\t7138 719\n",
      "116\t6705\t0.2\t1\t6073 632\n",
      "117\t6037\t0.2\t1\t5491 546\n",
      "118\t5126\t0.2\t1\t4659 467\n",
      "119\t4476\t0.2\t1\t4067 409\n",
      "120\t3856\t0.2\t1\t3446 410\n",
      "121\t3506\t0.2\t1\t3100 406\n",
      "122\t2914\t0.2\t1\t2611 303\n",
      "123\t2582\t0.2\t1\t2308 274\n",
      "124\t2250\t0.2\t1\t1996 254\n",
      "125\t1869\t0.2\t1\t1644 225\n",
      "126\t1660\t0.2\t1\t1451 209\n",
      "127\t1486\t0.2\t1\t1318 168\n",
      "128\t1135\t0.2\t1\t1016 119\n",
      "129\t1048\t0.2\t1\t958 90\n",
      "130\t730\t0.2\t1\t642 88\n",
      "131\t781\t0.2\t1\t698 83\n",
      "132\t769\t0.2\t1\t708 61\n",
      "133\t1082\t0.2\t1\t1006 76\n",
      "134\t1977\t0.2\t1\t1867 110\n",
      "135\t796\t0.2\t1\t751 45\n",
      "136\t320\t0.2\t1\t290 30\n",
      "137\t204\t0.2\t1\t187 17\n",
      "138\t179\t0.2\t1\t161 18\n",
      "139\t112\t0.2\t1\t104 8\n",
      "140\t111\t0.2\t1\t94 17\n",
      "141\t71\t0.2\t1\t68 3\n",
      "142\t79\t0.2\t1\t67 12\n",
      "143\t106\t0.2\t1\t93 13\n",
      "144\t63\t0.2\t1\t61 2\n",
      "145\t233\t0.2\t1\t213 20\n",
      "146\t995\t0.2\t1\t960 35\n",
      "147\t4990\t0.2\t1\t4774 216\n",
      "148\t3265\t0.2\t1\t3135 130\n",
      "149\t1955\t0.2\t1\t1846 109\n",
      "150\t16845\t0.2\t1\t15820 1025\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14622552 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t34997 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_3_S15_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 637.92 s (44 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,622,552\n",
      "Reads with adapters:                10,913,941 (74.6%)\n",
      "Reads written (passing filters):    14,622,552 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,193,382,800 bp\n",
      "Quality-trimmed:              13,012,389 bp (0.6%)\n",
      "Total written (filtered):  1,758,102,450 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10913941 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 14.2%\n",
      "  G: 23.9%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2243893\t3655638.0\t0\t2243893\n",
      "2\t361235\t913909.5\t0\t361235\n",
      "3\t221045\t228477.4\t0\t221045\n",
      "4\t102953\t57119.3\t0\t102953\n",
      "5\t80629\t14279.8\t0\t80629\n",
      "6\t70223\t3570.0\t0\t70223\n",
      "7\t53470\t892.5\t0\t53470\n",
      "8\t79479\t223.1\t0\t79479\n",
      "9\t62911\t55.8\t0\t62617 294\n",
      "10\t66925\t13.9\t1\t61688 5237\n",
      "11\t73700\t3.5\t1\t66819 6881\n",
      "12\t72572\t0.9\t1\t65940 6632\n",
      "13\t82700\t0.2\t1\t75488 7212\n",
      "14\t76573\t0.2\t1\t68946 7627\n",
      "15\t61263\t0.2\t1\t55865 5398\n",
      "16\t90146\t0.2\t1\t82109 8037\n",
      "17\t74386\t0.2\t1\t67549 6837\n",
      "18\t67318\t0.2\t1\t61385 5933\n",
      "19\t87145\t0.2\t1\t79499 7646\n",
      "20\t80434\t0.2\t1\t73597 6837\n",
      "21\t71959\t0.2\t1\t65758 6201\n",
      "22\t80520\t0.2\t1\t73704 6816\n",
      "23\t83808\t0.2\t1\t76805 7003\n",
      "24\t83121\t0.2\t1\t75853 7268\n",
      "25\t94721\t0.2\t1\t87319 7402\n",
      "26\t78397\t0.2\t1\t71438 6959\n",
      "27\t87589\t0.2\t1\t79828 7761\n",
      "28\t77436\t0.2\t1\t71759 5677\n",
      "29\t88231\t0.2\t1\t81047 7184\n",
      "30\t81944\t0.2\t1\t75545 6399\n",
      "31\t92581\t0.2\t1\t85292 7289\n",
      "32\t80085\t0.2\t1\t74132 5953\n",
      "33\t91916\t0.2\t1\t84460 7456\n",
      "34\t117300\t0.2\t1\t108202 9098\n",
      "35\t62830\t0.2\t1\t57836 4994\n",
      "36\t92603\t0.2\t1\t85123 7480\n",
      "37\t97494\t0.2\t1\t89682 7812\n",
      "38\t93013\t0.2\t1\t86322 6691\n",
      "39\t86916\t0.2\t1\t80841 6075\n",
      "40\t100050\t0.2\t1\t92533 7517\n",
      "41\t96502\t0.2\t1\t89613 6889\n",
      "42\t98694\t0.2\t1\t91554 7140\n",
      "43\t88091\t0.2\t1\t81948 6143\n",
      "44\t102146\t0.2\t1\t94589 7557\n",
      "45\t106497\t0.2\t1\t98693 7804\n",
      "46\t77918\t0.2\t1\t72391 5527\n",
      "47\t98307\t0.2\t1\t91760 6547\n",
      "48\t96325\t0.2\t1\t89754 6571\n",
      "49\t100699\t0.2\t1\t93521 7178\n",
      "50\t95936\t0.2\t1\t89184 6752\n",
      "51\t117412\t0.2\t1\t109335 8077\n",
      "52\t84318\t0.2\t1\t78674 5644\n",
      "53\t103753\t0.2\t1\t97370 6383\n",
      "54\t76703\t0.2\t1\t71688 5015\n",
      "55\t100303\t0.2\t1\t94078 6225\n",
      "56\t104185\t0.2\t1\t97076 7109\n",
      "57\t98544\t0.2\t1\t92645 5899\n",
      "58\t94450\t0.2\t1\t88927 5523\n",
      "59\t94831\t0.2\t1\t88887 5944\n",
      "60\t99091\t0.2\t1\t92777 6314\n",
      "61\t100905\t0.2\t1\t94656 6249\n",
      "62\t109121\t0.2\t1\t102218 6903\n",
      "63\t142655\t0.2\t1\t134487 8168\n",
      "64\t117121\t0.2\t1\t111484 5637\n",
      "65\t77324\t0.2\t1\t72904 4420\n",
      "66\t84925\t0.2\t1\t79850 5075\n",
      "67\t82998\t0.2\t1\t77768 5230\n",
      "68\t84356\t0.2\t1\t79351 5005\n",
      "69\t83969\t0.2\t1\t78702 5267\n",
      "70\t86991\t0.2\t1\t81734 5257\n",
      "71\t92058\t0.2\t1\t86658 5400\n",
      "72\t87408\t0.2\t1\t82156 5252\n",
      "73\t92562\t0.2\t1\t87094 5468\n",
      "74\t84507\t0.2\t1\t79446 5061\n",
      "75\t80670\t0.2\t1\t75777 4893\n",
      "76\t78563\t0.2\t1\t73818 4745\n",
      "77\t77927\t0.2\t1\t73312 4615\n",
      "78\t73796\t0.2\t1\t69197 4599\n",
      "79\t71841\t0.2\t1\t67592 4249\n",
      "80\t70609\t0.2\t1\t66510 4099\n",
      "81\t69349\t0.2\t1\t65453 3896\n",
      "82\t67523\t0.2\t1\t63514 4009\n",
      "83\t64665\t0.2\t1\t60869 3796\n",
      "84\t63972\t0.2\t1\t60280 3692\n",
      "85\t62543\t0.2\t1\t58851 3692\n",
      "86\t62248\t0.2\t1\t58801 3447\n",
      "87\t61991\t0.2\t1\t58664 3327\n",
      "88\t61620\t0.2\t1\t58173 3447\n",
      "89\t55062\t0.2\t1\t51949 3113\n",
      "90\t52411\t0.2\t1\t49511 2900\n",
      "91\t49315\t0.2\t1\t46549 2766\n",
      "92\t47010\t0.2\t1\t44328 2682\n",
      "93\t44789\t0.2\t1\t42339 2450\n",
      "94\t43445\t0.2\t1\t41012 2433\n",
      "95\t40986\t0.2\t1\t38680 2306\n",
      "96\t39258\t0.2\t1\t37094 2164\n",
      "97\t36644\t0.2\t1\t34643 2001\n",
      "98\t34564\t0.2\t1\t32670 1894\n",
      "99\t32993\t0.2\t1\t31157 1836\n",
      "100\t31055\t0.2\t1\t29316 1739\n",
      "101\t27872\t0.2\t1\t26391 1481\n",
      "102\t25485\t0.2\t1\t24012 1473\n",
      "103\t24164\t0.2\t1\t22767 1397\n",
      "104\t23318\t0.2\t1\t22028 1290\n",
      "105\t20591\t0.2\t1\t19438 1153\n",
      "106\t19078\t0.2\t1\t17973 1105\n",
      "107\t17270\t0.2\t1\t16333 937\n",
      "108\t15786\t0.2\t1\t14877 909\n",
      "109\t14538\t0.2\t1\t13717 821\n",
      "110\t13063\t0.2\t1\t12333 730\n",
      "111\t12532\t0.2\t1\t11819 713\n",
      "112\t10933\t0.2\t1\t10317 616\n",
      "113\t9707\t0.2\t1\t9177 530\n",
      "114\t8352\t0.2\t1\t7861 491\n",
      "115\t7373\t0.2\t1\t6927 446\n",
      "116\t6174\t0.2\t1\t5812 362\n",
      "117\t5508\t0.2\t1\t5151 357\n",
      "118\t4695\t0.2\t1\t4387 308\n",
      "119\t4109\t0.2\t1\t3838 271\n",
      "120\t3479\t0.2\t1\t3278 201\n",
      "121\t3246\t0.2\t1\t3039 207\n",
      "122\t2778\t0.2\t1\t2609 169\n",
      "123\t2432\t0.2\t1\t2288 144\n",
      "124\t2091\t0.2\t1\t1964 127\n",
      "125\t1753\t0.2\t1\t1626 127\n",
      "126\t1569\t0.2\t1\t1461 108\n",
      "127\t1430\t0.2\t1\t1321 109\n",
      "128\t1051\t0.2\t1\t972 79\n",
      "129\t984\t0.2\t1\t936 48\n",
      "130\t686\t0.2\t1\t639 47\n",
      "131\t713\t0.2\t1\t668 45\n",
      "132\t719\t0.2\t1\t681 38\n",
      "133\t1234\t0.2\t1\t1177 57\n",
      "134\t2006\t0.2\t1\t1917 89\n",
      "135\t763\t0.2\t1\t728 35\n",
      "136\t321\t0.2\t1\t299 22\n",
      "137\t192\t0.2\t1\t179 13\n",
      "138\t174\t0.2\t1\t164 10\n",
      "139\t115\t0.2\t1\t111 4\n",
      "140\t101\t0.2\t1\t87 14\n",
      "141\t72\t0.2\t1\t66 6\n",
      "142\t70\t0.2\t1\t63 7\n",
      "143\t96\t0.2\t1\t90 6\n",
      "144\t66\t0.2\t1\t61 5\n",
      "145\t236\t0.2\t1\t225 11\n",
      "146\t987\t0.2\t1\t943 44\n",
      "147\t4949\t0.2\t1\t4744 205\n",
      "148\t3207\t0.2\t1\t3070 137\n",
      "149\t1976\t0.2\t1\t1884 92\n",
      "150\t15929\t0.2\t1\t15207 722\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14622552 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t54056 (0.4%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 690.05 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,274,454\n",
      "Reads with adapters:                13,728,900 (79.5%)\n",
      "Reads written (passing filters):    17,274,454 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,591,168,100 bp\n",
      "Quality-trimmed:               5,179,651 bp (0.2%)\n",
      "Total written (filtered):  1,987,019,797 bp (76.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13728900 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 17.1%\n",
      "  G: 19.6%\n",
      "  T: 33.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2136043\t4318613.5\t0\t2136043\n",
      "2\t318200\t1079653.4\t0\t318200\n",
      "3\t181191\t269913.3\t0\t181191\n",
      "4\t105391\t67478.3\t0\t105391\n",
      "5\t86281\t16869.6\t0\t86281\n",
      "6\t85402\t4217.4\t0\t85402\n",
      "7\t69532\t1054.3\t0\t69532\n",
      "8\t89673\t263.6\t0\t89673\n",
      "9\t85385\t65.9\t0\t84182 1203\n",
      "10\t81888\t16.5\t1\t77249 4639\n",
      "11\t92800\t4.1\t1\t85219 7581\n",
      "12\t87898\t1.0\t1\t82011 5887\n",
      "13\t88385\t0.3\t1\t82215 6170\n",
      "14\t91465\t0.3\t1\t84417 7048\n",
      "15\t93851\t0.3\t1\t87200 6651\n",
      "16\t93365\t0.3\t1\t85770 7595\n",
      "17\t112928\t0.3\t1\t103341 9587\n",
      "18\t97669\t0.3\t1\t91300 6369\n",
      "19\t82484\t0.3\t1\t76767 5717\n",
      "20\t99682\t0.3\t1\t92884 6798\n",
      "21\t110868\t0.3\t1\t102713 8155\n",
      "22\t94466\t0.3\t1\t88707 5759\n",
      "23\t107932\t0.3\t1\t100154 7778\n",
      "24\t103619\t0.3\t1\t94765 8854\n",
      "25\t110068\t0.3\t1\t102116 7952\n",
      "26\t100495\t0.3\t1\t94095 6400\n",
      "27\t112487\t0.3\t1\t103684 8803\n",
      "28\t109738\t0.3\t1\t101919 7819\n",
      "29\t116657\t0.3\t1\t106828 9829\n",
      "30\t126889\t0.3\t1\t119461 7428\n",
      "31\t92971\t0.3\t1\t86609 6362\n",
      "32\t117804\t0.3\t1\t109725 8079\n",
      "33\t111542\t0.3\t1\t103765 7777\n",
      "34\t119656\t0.3\t1\t110996 8660\n",
      "35\t119627\t0.3\t1\t110861 8766\n",
      "36\t135056\t0.3\t1\t123455 11601\n",
      "37\t141108\t0.3\t1\t132535 8573\n",
      "38\t98717\t0.3\t1\t92159 6558\n",
      "39\t131001\t0.3\t1\t121415 9586\n",
      "40\t131538\t0.3\t1\t121991 9547\n",
      "41\t151149\t0.3\t1\t140064 11085\n",
      "42\t103859\t0.3\t1\t95703 8156\n",
      "43\t200319\t0.3\t1\t189316 11003\n",
      "44\t53496\t0.3\t1\t48400 5096\n",
      "45\t131702\t0.3\t1\t122909 8793\n",
      "46\t127887\t0.3\t1\t119378 8509\n",
      "47\t129236\t0.3\t1\t120995 8241\n",
      "48\t131832\t0.3\t1\t122637 9195\n",
      "49\t143882\t0.3\t1\t133714 10168\n",
      "50\t119251\t0.3\t1\t111807 7444\n",
      "51\t131353\t0.3\t1\t122552 8801\n",
      "52\t136704\t0.3\t1\t126618 10086\n",
      "53\t148364\t0.3\t1\t138822 9542\n",
      "54\t114737\t0.3\t1\t107428 7309\n",
      "55\t140460\t0.3\t1\t130746 9714\n",
      "56\t175570\t0.3\t1\t164830 10740\n",
      "57\t103575\t0.3\t1\t95859 7716\n",
      "58\t143113\t0.3\t1\t134869 8244\n",
      "59\t126187\t0.3\t1\t117760 8427\n",
      "60\t135064\t0.3\t1\t125919 9145\n",
      "61\t135372\t0.3\t1\t126642 8730\n",
      "62\t149463\t0.3\t1\t140362 9101\n",
      "63\t114425\t0.3\t1\t106420 8005\n",
      "64\t130898\t0.3\t1\t122796 8102\n",
      "65\t148784\t0.3\t1\t139139 9645\n",
      "66\t153297\t0.3\t1\t143445 9852\n",
      "67\t160682\t0.3\t1\t147924 12758\n",
      "68\t313433\t0.3\t1\t297800 15633\n",
      "69\t49296\t0.3\t1\t44675 4621\n",
      "70\t20166\t0.3\t1\t16553 3613\n",
      "71\t84125\t0.3\t1\t78872 5253\n",
      "72\t115954\t0.3\t1\t109117 6837\n",
      "73\t122861\t0.3\t1\t115327 7534\n",
      "74\t121230\t0.3\t1\t113900 7330\n",
      "75\t120996\t0.3\t1\t113426 7570\n",
      "76\t120220\t0.3\t1\t112755 7465\n",
      "77\t120174\t0.3\t1\t112559 7615\n",
      "78\t113112\t0.3\t1\t105946 7166\n",
      "79\t112516\t0.3\t1\t105557 6959\n",
      "80\t110113\t0.3\t1\t103237 6876\n",
      "81\t108721\t0.3\t1\t102021 6700\n",
      "82\t106548\t0.3\t1\t99970 6578\n",
      "83\t102870\t0.3\t1\t96372 6498\n",
      "84\t99814\t0.3\t1\t93650 6164\n",
      "85\t97423\t0.3\t1\t91330 6093\n",
      "86\t93928\t0.3\t1\t88109 5819\n",
      "87\t90540\t0.3\t1\t84909 5631\n",
      "88\t88338\t0.3\t1\t82703 5635\n",
      "89\t85264\t0.3\t1\t79857 5407\n",
      "90\t84889\t0.3\t1\t79581 5308\n",
      "91\t80908\t0.3\t1\t75799 5109\n",
      "92\t77367\t0.3\t1\t72491 4876\n",
      "93\t72198\t0.3\t1\t67528 4670\n",
      "94\t71195\t0.3\t1\t66820 4375\n",
      "95\t66612\t0.3\t1\t62421 4191\n",
      "96\t62397\t0.3\t1\t58348 4049\n",
      "97\t59079\t0.3\t1\t55329 3750\n",
      "98\t55292\t0.3\t1\t51774 3518\n",
      "99\t54334\t0.3\t1\t50683 3651\n",
      "100\t49120\t0.3\t1\t45957 3163\n",
      "101\t46590\t0.3\t1\t43597 2993\n",
      "102\t43817\t0.3\t1\t40930 2887\n",
      "103\t40178\t0.3\t1\t37514 2664\n",
      "104\t37716\t0.3\t1\t35273 2443\n",
      "105\t33568\t0.3\t1\t31289 2279\n",
      "106\t31261\t0.3\t1\t29161 2100\n",
      "107\t28961\t0.3\t1\t27009 1952\n",
      "108\t26580\t0.3\t1\t24746 1834\n",
      "109\t24178\t0.3\t1\t22520 1658\n",
      "110\t22331\t0.3\t1\t20687 1644\n",
      "111\t19845\t0.3\t1\t18444 1401\n",
      "112\t17480\t0.3\t1\t16231 1249\n",
      "113\t15260\t0.3\t1\t14048 1212\n",
      "114\t13572\t0.3\t1\t12544 1028\n",
      "115\t12245\t0.3\t1\t11265 980\n",
      "116\t10672\t0.3\t1\t9863 809\n",
      "117\t9108\t0.3\t1\t8423 685\n",
      "118\t7910\t0.3\t1\t7333 577\n",
      "119\t6724\t0.3\t1\t6143 581\n",
      "120\t5760\t0.3\t1\t5265 495\n",
      "121\t5240\t0.3\t1\t4780 460\n",
      "122\t4324\t0.3\t1\t3902 422\n",
      "123\t3607\t0.3\t1\t3320 287\n",
      "124\t3039\t0.3\t1\t2741 298\n",
      "125\t2579\t0.3\t1\t2329 250\n",
      "126\t2117\t0.3\t1\t1915 202\n",
      "127\t1827\t0.3\t1\t1662 165\n",
      "128\t1426\t0.3\t1\t1277 149\n",
      "129\t1179\t0.3\t1\t1062 117\n",
      "130\t902\t0.3\t1\t808 94\n",
      "131\t765\t0.3\t1\t671 94\n",
      "132\t696\t0.3\t1\t619 77\n",
      "133\t768\t0.3\t1\t705 63\n",
      "134\t1110\t0.3\t1\t1039 71\n",
      "135\t519\t0.3\t1\t468 51\n",
      "136\t251\t0.3\t1\t213 38\n",
      "137\t140\t0.3\t1\t126 14\n",
      "138\t122\t0.3\t1\t106 16\n",
      "139\t85\t0.3\t1\t72 13\n",
      "140\t62\t0.3\t1\t56 6\n",
      "141\t41\t0.3\t1\t39 2\n",
      "142\t38\t0.3\t1\t34 4\n",
      "143\t29\t0.3\t1\t23 6\n",
      "144\t18\t0.3\t1\t9 9\n",
      "145\t46\t0.3\t1\t40 6\n",
      "146\t43\t0.3\t1\t40 3\n",
      "147\t83\t0.3\t1\t73 10\n",
      "148\t54\t0.3\t1\t46 8\n",
      "149\t57\t0.3\t1\t48 9\n",
      "150\t531\t0.3\t1\t439 92\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17274454 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5534 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 716.93 s (42 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,274,454\n",
      "Reads with adapters:                13,645,900 (79.0%)\n",
      "Reads written (passing filters):    17,274,454 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,591,168,100 bp\n",
      "Quality-trimmed:              13,876,099 bp (0.5%)\n",
      "Total written (filtered):  1,990,000,241 bp (76.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13645900 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 13.6%\n",
      "  G: 24.3%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2190701\t4318613.5\t0\t2190701\n",
      "2\t333937\t1079653.4\t0\t333937\n",
      "3\t193297\t269913.3\t0\t193297\n",
      "4\t108228\t67478.3\t0\t108228\n",
      "5\t96340\t16869.6\t0\t96340\n",
      "6\t82360\t4217.4\t0\t82360\n",
      "7\t66626\t1054.3\t0\t66626\n",
      "8\t94895\t263.6\t0\t94895\n",
      "9\t77384\t65.9\t0\t77094 290\n",
      "10\t82708\t16.5\t1\t78572 4136\n",
      "11\t89357\t4.1\t1\t83568 5789\n",
      "12\t89625\t1.0\t1\t83594 6031\n",
      "13\t99746\t0.3\t1\t93650 6096\n",
      "14\t92613\t0.3\t1\t86134 6479\n",
      "15\t76114\t0.3\t1\t71248 4866\n",
      "16\t111173\t0.3\t1\t104156 7017\n",
      "17\t86562\t0.3\t1\t80610 5952\n",
      "18\t85573\t0.3\t1\t80223 5350\n",
      "19\t106850\t0.3\t1\t99751 7099\n",
      "20\t100367\t0.3\t1\t94128 6239\n",
      "21\t90921\t0.3\t1\t85159 5762\n",
      "22\t101235\t0.3\t1\t95178 6057\n",
      "23\t104933\t0.3\t1\t98391 6542\n",
      "24\t104547\t0.3\t1\t97881 6666\n",
      "25\t118815\t0.3\t1\t111628 7187\n",
      "26\t97285\t0.3\t1\t91138 6147\n",
      "27\t109196\t0.3\t1\t101511 7685\n",
      "28\t101869\t0.3\t1\t96609 5260\n",
      "29\t109573\t0.3\t1\t102802 6771\n",
      "30\t108062\t0.3\t1\t102093 5969\n",
      "31\t117609\t0.3\t1\t110646 6963\n",
      "32\t106258\t0.3\t1\t100439 5819\n",
      "33\t117522\t0.3\t1\t110451 7071\n",
      "34\t150731\t0.3\t1\t141819 8912\n",
      "35\t85761\t0.3\t1\t81040 4721\n",
      "36\t120033\t0.3\t1\t112538 7495\n",
      "37\t128229\t0.3\t1\t120721 7508\n",
      "38\t120207\t0.3\t1\t114092 6115\n",
      "39\t117619\t0.3\t1\t111621 5998\n",
      "40\t131723\t0.3\t1\t123921 7802\n",
      "41\t124695\t0.3\t1\t117864 6831\n",
      "42\t129575\t0.3\t1\t122681 6894\n",
      "43\t116234\t0.3\t1\t109884 6350\n",
      "44\t134420\t0.3\t1\t127012 7408\n",
      "45\t140609\t0.3\t1\t133114 7495\n",
      "46\t107286\t0.3\t1\t101613 5673\n",
      "47\t132633\t0.3\t1\t125745 6888\n",
      "48\t129386\t0.3\t1\t122927 6459\n",
      "49\t135218\t0.3\t1\t128159 7059\n",
      "50\t128960\t0.3\t1\t121933 7027\n",
      "51\t158199\t0.3\t1\t150290 7909\n",
      "52\t115635\t0.3\t1\t110127 5508\n",
      "53\t138536\t0.3\t1\t132192 6344\n",
      "54\t107841\t0.3\t1\t102539 5302\n",
      "55\t138155\t0.3\t1\t131891 6264\n",
      "56\t143145\t0.3\t1\t136120 7025\n",
      "57\t134481\t0.3\t1\t128427 6054\n",
      "58\t129238\t0.3\t1\t123474 5764\n",
      "59\t132359\t0.3\t1\t126122 6237\n",
      "60\t136774\t0.3\t1\t130227 6547\n",
      "61\t140635\t0.3\t1\t133777 6858\n",
      "62\t151391\t0.3\t1\t144144 7247\n",
      "63\t203160\t0.3\t1\t194409 8751\n",
      "64\t167385\t0.3\t1\t160989 6396\n",
      "65\t109630\t0.3\t1\t104767 4863\n",
      "66\t118791\t0.3\t1\t113616 5175\n",
      "67\t117636\t0.3\t1\t112066 5570\n",
      "68\t118814\t0.3\t1\t113546 5268\n",
      "69\t118228\t0.3\t1\t112812 5416\n",
      "70\t124633\t0.3\t1\t118945 5688\n",
      "71\t132024\t0.3\t1\t126339 5685\n",
      "72\t125001\t0.3\t1\t119396 5605\n",
      "73\t132138\t0.3\t1\t126543 5595\n",
      "74\t121069\t0.3\t1\t115983 5086\n",
      "75\t115800\t0.3\t1\t110816 4984\n",
      "76\t113271\t0.3\t1\t108471 4800\n",
      "77\t113434\t0.3\t1\t108479 4955\n",
      "78\t108048\t0.3\t1\t103289 4759\n",
      "79\t104688\t0.3\t1\t100095 4593\n",
      "80\t104276\t0.3\t1\t99811 4465\n",
      "81\t103173\t0.3\t1\t98658 4515\n",
      "82\t98745\t0.3\t1\t94559 4186\n",
      "83\t96106\t0.3\t1\t91915 4191\n",
      "84\t93144\t0.3\t1\t88994 4150\n",
      "85\t92176\t0.3\t1\t88129 4047\n",
      "86\t91669\t0.3\t1\t87622 4047\n",
      "87\t92154\t0.3\t1\t88295 3859\n",
      "88\t89148\t0.3\t1\t85288 3860\n",
      "89\t81149\t0.3\t1\t77666 3483\n",
      "90\t78174\t0.3\t1\t74845 3329\n",
      "91\t73485\t0.3\t1\t70395 3090\n",
      "92\t69397\t0.3\t1\t66501 2896\n",
      "93\t66171\t0.3\t1\t63215 2956\n",
      "94\t65063\t0.3\t1\t62271 2792\n",
      "95\t61764\t0.3\t1\t59094 2670\n",
      "96\t58368\t0.3\t1\t55944 2424\n",
      "97\t55755\t0.3\t1\t53423 2332\n",
      "98\t52903\t0.3\t1\t50745 2158\n",
      "99\t53244\t0.3\t1\t51088 2156\n",
      "100\t47728\t0.3\t1\t45793 1935\n",
      "101\t43677\t0.3\t1\t41891 1786\n",
      "102\t42206\t0.3\t1\t40454 1752\n",
      "103\t39193\t0.3\t1\t37641 1552\n",
      "104\t38151\t0.3\t1\t36638 1513\n",
      "105\t33427\t0.3\t1\t32096 1331\n",
      "106\t30687\t0.3\t1\t29498 1189\n",
      "107\t27695\t0.3\t1\t26647 1048\n",
      "108\t24686\t0.3\t1\t23641 1045\n",
      "109\t22730\t0.3\t1\t21817 913\n",
      "110\t20941\t0.3\t1\t20113 828\n",
      "111\t19623\t0.3\t1\t18797 826\n",
      "112\t17030\t0.3\t1\t16327 703\n",
      "113\t14883\t0.3\t1\t14295 588\n",
      "114\t13223\t0.3\t1\t12713 510\n",
      "115\t11548\t0.3\t1\t11042 506\n",
      "116\t9826\t0.3\t1\t9358 468\n",
      "117\t8325\t0.3\t1\t7930 395\n",
      "118\t7266\t0.3\t1\t6933 333\n",
      "119\t6187\t0.3\t1\t5890 297\n",
      "120\t5282\t0.3\t1\t5029 253\n",
      "121\t4888\t0.3\t1\t4641 247\n",
      "122\t4001\t0.3\t1\t3828 173\n",
      "123\t3492\t0.3\t1\t3320 172\n",
      "124\t2873\t0.3\t1\t2729 144\n",
      "125\t2466\t0.3\t1\t2331 135\n",
      "126\t2028\t0.3\t1\t1925 103\n",
      "127\t1732\t0.3\t1\t1662 70\n",
      "128\t1274\t0.3\t1\t1214 60\n",
      "129\t1089\t0.3\t1\t1028 61\n",
      "130\t827\t0.3\t1\t782 45\n",
      "131\t699\t0.3\t1\t652 47\n",
      "132\t657\t0.3\t1\t618 39\n",
      "133\t790\t0.3\t1\t750 40\n",
      "134\t1089\t0.3\t1\t1048 41\n",
      "135\t486\t0.3\t1\t468 18\n",
      "136\t232\t0.3\t1\t220 12\n",
      "137\t137\t0.3\t1\t130 7\n",
      "138\t117\t0.3\t1\t101 16\n",
      "139\t79\t0.3\t1\t72 7\n",
      "140\t58\t0.3\t1\t53 5\n",
      "141\t42\t0.3\t1\t40 2\n",
      "142\t42\t0.3\t1\t37 5\n",
      "143\t28\t0.3\t1\t25 3\n",
      "144\t19\t0.3\t1\t18 1\n",
      "145\t43\t0.3\t1\t41 2\n",
      "146\t41\t0.3\t1\t36 5\n",
      "147\t80\t0.3\t1\t71 9\n",
      "148\t57\t0.3\t1\t57\n",
      "149\t45\t0.3\t1\t41 4\n",
      "150\t372\t0.3\t1\t355 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17274454 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t28578 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 696.49 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,208,353\n",
      "Reads with adapters:                13,674,200 (79.5%)\n",
      "Reads written (passing filters):    17,208,353 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,581,252,950 bp\n",
      "Quality-trimmed:               5,331,759 bp (0.2%)\n",
      "Total written (filtered):  1,977,555,409 bp (76.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13674200 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 17.1%\n",
      "  G: 19.6%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2124508\t4302088.2\t0\t2124508\n",
      "2\t313423\t1075522.1\t0\t313423\n",
      "3\t180126\t268880.5\t0\t180126\n",
      "4\t103470\t67220.1\t0\t103470\n",
      "5\t86090\t16805.0\t0\t86090\n",
      "6\t84599\t4201.3\t0\t84599\n",
      "7\t66757\t1050.3\t0\t66757\n",
      "8\t88962\t262.6\t0\t88962\n",
      "9\t83630\t65.6\t0\t82470 1160\n",
      "10\t80181\t16.4\t1\t74703 5478\n",
      "11\t92233\t4.1\t1\t83672 8561\n",
      "12\t86362\t1.0\t1\t79524 6838\n",
      "13\t87841\t0.3\t1\t80589 7252\n",
      "14\t90274\t0.3\t1\t82360 7914\n",
      "15\t93818\t0.3\t1\t85881 7937\n",
      "16\t92355\t0.3\t1\t83764 8591\n",
      "17\t113048\t0.3\t1\t101963 11085\n",
      "18\t96642\t0.3\t1\t89228 7414\n",
      "19\t80013\t0.3\t1\t73649 6364\n",
      "20\t99407\t0.3\t1\t91339 8068\n",
      "21\t111176\t0.3\t1\t101662 9514\n",
      "22\t92147\t0.3\t1\t85457 6690\n",
      "23\t107114\t0.3\t1\t98360 8754\n",
      "24\t102847\t0.3\t1\t93006 9841\n",
      "25\t109576\t0.3\t1\t100448 9128\n",
      "26\t99232\t0.3\t1\t91712 7520\n",
      "27\t112349\t0.3\t1\t102540 9809\n",
      "28\t109676\t0.3\t1\t100710 8966\n",
      "29\t116787\t0.3\t1\t105939 10848\n",
      "30\t125526\t0.3\t1\t116820 8706\n",
      "31\t91212\t0.3\t1\t83997 7215\n",
      "32\t117558\t0.3\t1\t108076 9482\n",
      "33\t110064\t0.3\t1\t101072 8992\n",
      "34\t118132\t0.3\t1\t107623 10509\n",
      "35\t118711\t0.3\t1\t108495 10216\n",
      "36\t135621\t0.3\t1\t122255 13366\n",
      "37\t140972\t0.3\t1\t130809 10163\n",
      "38\t95392\t0.3\t1\t87779 7613\n",
      "39\t130577\t0.3\t1\t118922 11655\n",
      "40\t131214\t0.3\t1\t119601 11613\n",
      "41\t152494\t0.3\t1\t139337 13157\n",
      "42\t102460\t0.3\t1\t93074 9386\n",
      "43\t197072\t0.3\t1\t184157 12915\n",
      "44\t54139\t0.3\t1\t48245 5894\n",
      "45\t131336\t0.3\t1\t121138 10198\n",
      "46\t127690\t0.3\t1\t117856 9834\n",
      "47\t129086\t0.3\t1\t119565 9521\n",
      "48\t131921\t0.3\t1\t121440 10481\n",
      "49\t145001\t0.3\t1\t133240 11761\n",
      "50\t116964\t0.3\t1\t108432 8532\n",
      "51\t131690\t0.3\t1\t121283 10407\n",
      "52\t137379\t0.3\t1\t125946 11433\n",
      "53\t148180\t0.3\t1\t137434 10746\n",
      "54\t113408\t0.3\t1\t104863 8545\n",
      "55\t140308\t0.3\t1\t129251 11057\n",
      "56\t175262\t0.3\t1\t162851 12411\n",
      "57\t103098\t0.3\t1\t94387 8711\n",
      "58\t143210\t0.3\t1\t133468 9742\n",
      "59\t125785\t0.3\t1\t116067 9718\n",
      "60\t133778\t0.3\t1\t123739 10039\n",
      "61\t134602\t0.3\t1\t124844 9758\n",
      "62\t150181\t0.3\t1\t139349 10832\n",
      "63\t115498\t0.3\t1\t106236 9262\n",
      "64\t129364\t0.3\t1\t120287 9077\n",
      "65\t148548\t0.3\t1\t137651 10897\n",
      "66\t150760\t0.3\t1\t139512 11248\n",
      "67\t160846\t0.3\t1\t146823 14023\n",
      "68\t302542\t0.3\t1\t285033 17509\n",
      "69\t61997\t0.3\t1\t56378 5619\n",
      "70\t24824\t0.3\t1\t20506 4318\n",
      "71\t82321\t0.3\t1\t76477 5844\n",
      "72\t114537\t0.3\t1\t107064 7473\n",
      "73\t122133\t0.3\t1\t114073 8060\n",
      "74\t121733\t0.3\t1\t113185 8548\n",
      "75\t121574\t0.3\t1\t112683 8891\n",
      "76\t119912\t0.3\t1\t110967 8945\n",
      "77\t120383\t0.3\t1\t111732 8651\n",
      "78\t113881\t0.3\t1\t105452 8429\n",
      "79\t111936\t0.3\t1\t103610 8326\n",
      "80\t109761\t0.3\t1\t101635 8126\n",
      "81\t108328\t0.3\t1\t100397 7931\n",
      "82\t105284\t0.3\t1\t97525 7759\n",
      "83\t103557\t0.3\t1\t95997 7560\n",
      "84\t99896\t0.3\t1\t92540 7356\n",
      "85\t96863\t0.3\t1\t89671 7192\n",
      "86\t94827\t0.3\t1\t87870 6957\n",
      "87\t90622\t0.3\t1\t84095 6527\n",
      "88\t89624\t0.3\t1\t82896 6728\n",
      "89\t85693\t0.3\t1\t79392 6301\n",
      "90\t85156\t0.3\t1\t79011 6145\n",
      "91\t80919\t0.3\t1\t74951 5968\n",
      "92\t77468\t0.3\t1\t71924 5544\n",
      "93\t72342\t0.3\t1\t67111 5231\n",
      "94\t71272\t0.3\t1\t66088 5184\n",
      "95\t65969\t0.3\t1\t61048 4921\n",
      "96\t63375\t0.3\t1\t58785 4590\n",
      "97\t59353\t0.3\t1\t55151 4202\n",
      "98\t55614\t0.3\t1\t51400 4214\n",
      "99\t54309\t0.3\t1\t50115 4194\n",
      "100\t49832\t0.3\t1\t46148 3684\n",
      "101\t46196\t0.3\t1\t42723 3473\n",
      "102\t43375\t0.3\t1\t40084 3291\n",
      "103\t40365\t0.3\t1\t37360 3005\n",
      "104\t37934\t0.3\t1\t35154 2780\n",
      "105\t33958\t0.3\t1\t31403 2555\n",
      "106\t31440\t0.3\t1\t29092 2348\n",
      "107\t28594\t0.3\t1\t26406 2188\n",
      "108\t27035\t0.3\t1\t24984 2051\n",
      "109\t24211\t0.3\t1\t22261 1950\n",
      "110\t22056\t0.3\t1\t20350 1706\n",
      "111\t20216\t0.3\t1\t18643 1573\n",
      "112\t17663\t0.3\t1\t16163 1500\n",
      "113\t15323\t0.3\t1\t14109 1214\n",
      "114\t13594\t0.3\t1\t12500 1094\n",
      "115\t12178\t0.3\t1\t11179 999\n",
      "116\t10756\t0.3\t1\t9835 921\n",
      "117\t9186\t0.3\t1\t8396 790\n",
      "118\t8066\t0.3\t1\t7378 688\n",
      "119\t6801\t0.3\t1\t6177 624\n",
      "120\t5605\t0.3\t1\t5122 483\n",
      "121\t5036\t0.3\t1\t4635 401\n",
      "122\t4356\t0.3\t1\t3917 439\n",
      "123\t3703\t0.3\t1\t3365 338\n",
      "124\t3136\t0.3\t1\t2845 291\n",
      "125\t2679\t0.3\t1\t2426 253\n",
      "126\t2221\t0.3\t1\t2017 204\n",
      "127\t1848\t0.3\t1\t1642 206\n",
      "128\t1384\t0.3\t1\t1237 147\n",
      "129\t1223\t0.3\t1\t1070 153\n",
      "130\t986\t0.3\t1\t889 97\n",
      "131\t829\t0.3\t1\t741 88\n",
      "132\t691\t0.3\t1\t626 65\n",
      "133\t780\t0.3\t1\t701 79\n",
      "134\t1086\t0.3\t1\t1026 60\n",
      "135\t511\t0.3\t1\t480 31\n",
      "136\t248\t0.3\t1\t207 41\n",
      "137\t178\t0.3\t1\t152 26\n",
      "138\t125\t0.3\t1\t113 12\n",
      "139\t79\t0.3\t1\t67 12\n",
      "140\t60\t0.3\t1\t49 11\n",
      "141\t36\t0.3\t1\t30 6\n",
      "142\t37\t0.3\t1\t26 11\n",
      "143\t39\t0.3\t1\t35 4\n",
      "144\t23\t0.3\t1\t20 3\n",
      "145\t36\t0.3\t1\t29 7\n",
      "146\t37\t0.3\t1\t31 6\n",
      "147\t101\t0.3\t1\t92 9\n",
      "148\t53\t0.3\t1\t44 9\n",
      "149\t35\t0.3\t1\t29 6\n",
      "150\t573\t0.3\t1\t484 89\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17208353 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5621 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_26psu_4_S16_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 728.62 s (42 us/read; 1.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,208,353\n",
      "Reads with adapters:                13,587,185 (79.0%)\n",
      "Reads written (passing filters):    17,208,353 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,581,252,950 bp\n",
      "Quality-trimmed:              13,418,385 bp (0.5%)\n",
      "Total written (filtered):  1,981,075,586 bp (76.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13587185 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 13.6%\n",
      "  G: 24.4%\n",
      "  T: 34.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2180320\t4302088.2\t0\t2180320\n",
      "2\t324825\t1075522.1\t0\t324825\n",
      "3\t196027\t268880.5\t0\t196027\n",
      "4\t105938\t67220.1\t0\t105938\n",
      "5\t93022\t16805.0\t0\t93022\n",
      "6\t82169\t4201.3\t0\t82169\n",
      "7\t65208\t1050.3\t0\t65208\n",
      "8\t94076\t262.6\t0\t94076\n",
      "9\t75651\t65.6\t0\t75373 278\n",
      "10\t79743\t16.4\t1\t74241 5502\n",
      "11\t88328\t4.1\t1\t81010 7318\n",
      "12\t86868\t1.0\t1\t79554 7314\n",
      "13\t99881\t0.3\t1\t91747 8134\n",
      "14\t90834\t0.3\t1\t82543 8291\n",
      "15\t75785\t0.3\t1\t69708 6077\n",
      "16\t109506\t0.3\t1\t100461 9045\n",
      "17\t89706\t0.3\t1\t82164 7542\n",
      "18\t82342\t0.3\t1\t75779 6563\n",
      "19\t105511\t0.3\t1\t97019 8492\n",
      "20\t99413\t0.3\t1\t91913 7500\n",
      "21\t89589\t0.3\t1\t82387 7202\n",
      "22\t99486\t0.3\t1\t91727 7759\n",
      "23\t104499\t0.3\t1\t96352 8147\n",
      "24\t104372\t0.3\t1\t95967 8405\n",
      "25\t118405\t0.3\t1\t110027 8378\n",
      "26\t97628\t0.3\t1\t89447 8181\n",
      "27\t110575\t0.3\t1\t101151 9424\n",
      "28\t97751\t0.3\t1\t91186 6565\n",
      "29\t111505\t0.3\t1\t102881 8624\n",
      "30\t104420\t0.3\t1\t97112 7308\n",
      "31\t118911\t0.3\t1\t110316 8595\n",
      "32\t102066\t0.3\t1\t95076 6990\n",
      "33\t118362\t0.3\t1\t109639 8723\n",
      "34\t152523\t0.3\t1\t141524 10999\n",
      "35\t81147\t0.3\t1\t75248 5899\n",
      "36\t120338\t0.3\t1\t111218 9120\n",
      "37\t127944\t0.3\t1\t118775 9169\n",
      "38\t118773\t0.3\t1\t111107 7666\n",
      "39\t114524\t0.3\t1\t107290 7234\n",
      "40\t130962\t0.3\t1\t121798 9164\n",
      "41\t127042\t0.3\t1\t118718 8324\n",
      "42\t130066\t0.3\t1\t121378 8688\n",
      "43\t114447\t0.3\t1\t107009 7438\n",
      "44\t134439\t0.3\t1\t125115 9324\n",
      "45\t139316\t0.3\t1\t130061 9255\n",
      "46\t105889\t0.3\t1\t98943 6946\n",
      "47\t133294\t0.3\t1\t125179 8115\n",
      "48\t129000\t0.3\t1\t120992 8008\n",
      "49\t134472\t0.3\t1\t125771 8701\n",
      "50\t129511\t0.3\t1\t120901 8610\n",
      "51\t156322\t0.3\t1\t146348 9974\n",
      "52\t115554\t0.3\t1\t108318 7236\n",
      "53\t141127\t0.3\t1\t132995 8132\n",
      "54\t105383\t0.3\t1\t99036 6347\n",
      "55\t137072\t0.3\t1\t129093 7979\n",
      "56\t141674\t0.3\t1\t132689 8985\n",
      "57\t134402\t0.3\t1\t126942 7460\n",
      "58\t130317\t0.3\t1\t123061 7256\n",
      "59\t129827\t0.3\t1\t122170 7657\n",
      "60\t136682\t0.3\t1\t128763 7919\n",
      "61\t138438\t0.3\t1\t130627 7811\n",
      "62\t151584\t0.3\t1\t142493 9091\n",
      "63\t198913\t0.3\t1\t188533 10380\n",
      "64\t165224\t0.3\t1\t157788 7436\n",
      "65\t109943\t0.3\t1\t103886 6057\n",
      "66\t119196\t0.3\t1\t112621 6575\n",
      "67\t118137\t0.3\t1\t111336 6801\n",
      "68\t119874\t0.3\t1\t113325 6549\n",
      "69\t119266\t0.3\t1\t112294 6972\n",
      "70\t126609\t0.3\t1\t119544 7065\n",
      "71\t132620\t0.3\t1\t125297 7323\n",
      "72\t125393\t0.3\t1\t118453 6940\n",
      "73\t132184\t0.3\t1\t124866 7318\n",
      "74\t122169\t0.3\t1\t115334 6835\n",
      "75\t116493\t0.3\t1\t110071 6422\n",
      "76\t113442\t0.3\t1\t106974 6468\n",
      "77\t113685\t0.3\t1\t107141 6544\n",
      "78\t108607\t0.3\t1\t102395 6212\n",
      "79\t104353\t0.3\t1\t98543 5810\n",
      "80\t104020\t0.3\t1\t98279 5741\n",
      "81\t102310\t0.3\t1\t96699 5611\n",
      "82\t98054\t0.3\t1\t92664 5390\n",
      "83\t96480\t0.3\t1\t91293 5187\n",
      "84\t93016\t0.3\t1\t87985 5031\n",
      "85\t91834\t0.3\t1\t86918 4916\n",
      "86\t92118\t0.3\t1\t87322 4796\n",
      "87\t91952\t0.3\t1\t87281 4671\n",
      "88\t89832\t0.3\t1\t85225 4607\n",
      "89\t81915\t0.3\t1\t77620 4295\n",
      "90\t78846\t0.3\t1\t74588 4258\n",
      "91\t74284\t0.3\t1\t70441 3843\n",
      "92\t69858\t0.3\t1\t66293 3565\n",
      "93\t66592\t0.3\t1\t63127 3465\n",
      "94\t65539\t0.3\t1\t62049 3490\n",
      "95\t61188\t0.3\t1\t57883 3305\n",
      "96\t59534\t0.3\t1\t56516 3018\n",
      "97\t56011\t0.3\t1\t53174 2837\n",
      "98\t53355\t0.3\t1\t50647 2708\n",
      "99\t53101\t0.3\t1\t50373 2728\n",
      "100\t48195\t0.3\t1\t45837 2358\n",
      "101\t43522\t0.3\t1\t41188 2334\n",
      "102\t41863\t0.3\t1\t39765 2098\n",
      "103\t39452\t0.3\t1\t37477 1975\n",
      "104\t38339\t0.3\t1\t36409 1930\n",
      "105\t33606\t0.3\t1\t31895 1711\n",
      "106\t30952\t0.3\t1\t29448 1504\n",
      "107\t27675\t0.3\t1\t26330 1345\n",
      "108\t24951\t0.3\t1\t23635 1316\n",
      "109\t22743\t0.3\t1\t21560 1183\n",
      "110\t20811\t0.3\t1\t19684 1127\n",
      "111\t19926\t0.3\t1\t18958 968\n",
      "112\t17187\t0.3\t1\t16283 904\n",
      "113\t15096\t0.3\t1\t14331 765\n",
      "114\t13260\t0.3\t1\t12650 610\n",
      "115\t11435\t0.3\t1\t10854 581\n",
      "116\t9981\t0.3\t1\t9425 556\n",
      "117\t8327\t0.3\t1\t7873 454\n",
      "118\t7364\t0.3\t1\t6994 370\n",
      "119\t6240\t0.3\t1\t5915 325\n",
      "120\t5208\t0.3\t1\t4920 288\n",
      "121\t4741\t0.3\t1\t4467 274\n",
      "122\t4055\t0.3\t1\t3851 204\n",
      "123\t3533\t0.3\t1\t3332 201\n",
      "124\t2985\t0.3\t1\t2813 172\n",
      "125\t2559\t0.3\t1\t2417 142\n",
      "126\t2133\t0.3\t1\t2012 121\n",
      "127\t1706\t0.3\t1\t1579 127\n",
      "128\t1327\t0.3\t1\t1241 86\n",
      "129\t1121\t0.3\t1\t1038 83\n",
      "130\t901\t0.3\t1\t842 59\n",
      "131\t784\t0.3\t1\t738 46\n",
      "132\t657\t0.3\t1\t620 37\n",
      "133\t809\t0.3\t1\t770 39\n",
      "134\t1081\t0.3\t1\t1031 50\n",
      "135\t490\t0.3\t1\t465 25\n",
      "136\t231\t0.3\t1\t213 18\n",
      "137\t165\t0.3\t1\t153 12\n",
      "138\t122\t0.3\t1\t117 5\n",
      "139\t68\t0.3\t1\t64 4\n",
      "140\t60\t0.3\t1\t53 7\n",
      "141\t31\t0.3\t1\t25 6\n",
      "142\t34\t0.3\t1\t30 4\n",
      "143\t37\t0.3\t1\t30 7\n",
      "144\t24\t0.3\t1\t20 4\n",
      "145\t49\t0.3\t1\t45 4\n",
      "146\t39\t0.3\t1\t34 5\n",
      "147\t87\t0.3\t1\t77 10\n",
      "148\t52\t0.3\t1\t48 4\n",
      "149\t25\t0.3\t1\t23 2\n",
      "150\t408\t0.3\t1\t378 30\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17208353 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t27317 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 745.76 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,624,894\n",
      "Reads with adapters:                16,461,713 (83.9%)\n",
      "Reads written (passing filters):    19,624,894 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,943,734,100 bp\n",
      "Quality-trimmed:               8,161,343 bp (0.3%)\n",
      "Total written (filtered):  2,138,119,098 bp (72.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16461713 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 14.5%\n",
      "  G: 21.1%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1876405\t4906223.5\t0\t1876405\n",
      "2\t285639\t1226555.9\t0\t285639\n",
      "3\t166471\t306639.0\t0\t166471\n",
      "4\t104694\t76659.7\t0\t104694\n",
      "5\t90631\t19164.9\t0\t90631\n",
      "6\t90221\t4791.2\t0\t90221\n",
      "7\t75553\t1197.8\t0\t75553\n",
      "8\t93443\t299.5\t0\t93443\n",
      "9\t94201\t74.9\t0\t93032 1169\n",
      "10\t87809\t18.7\t1\t83198 4611\n",
      "11\t99635\t4.7\t1\t92145 7490\n",
      "12\t94726\t1.2\t1\t88812 5914\n",
      "13\t97394\t0.3\t1\t91137 6257\n",
      "14\t100020\t0.3\t1\t93031 6989\n",
      "15\t103009\t0.3\t1\t96393 6616\n",
      "16\t102618\t0.3\t1\t94905 7713\n",
      "17\t124436\t0.3\t1\t114364 10072\n",
      "18\t107976\t0.3\t1\t101489 6487\n",
      "19\t93413\t0.3\t1\t87422 5991\n",
      "20\t112556\t0.3\t1\t105383 7173\n",
      "21\t124612\t0.3\t1\t116055 8557\n",
      "22\t104064\t0.3\t1\t98066 5998\n",
      "23\t123879\t0.3\t1\t115825 8054\n",
      "24\t114769\t0.3\t1\t105177 9592\n",
      "25\t125576\t0.3\t1\t117036 8540\n",
      "26\t114132\t0.3\t1\t107345 6787\n",
      "27\t127880\t0.3\t1\t118576 9304\n",
      "28\t126438\t0.3\t1\t117548 8890\n",
      "29\t134319\t0.3\t1\t123537 10782\n",
      "30\t144898\t0.3\t1\t136865 8033\n",
      "31\t113550\t0.3\t1\t106291 7259\n",
      "32\t136410\t0.3\t1\t127848 8562\n",
      "33\t133695\t0.3\t1\t124518 9177\n",
      "34\t143757\t0.3\t1\t134451 9306\n",
      "35\t157890\t0.3\t1\t145316 12574\n",
      "36\t132282\t0.3\t1\t123929 8353\n",
      "37\t149188\t0.3\t1\t140073 9115\n",
      "38\t146870\t0.3\t1\t136072 10798\n",
      "39\t172309\t0.3\t1\t159375 12934\n",
      "40\t136570\t0.3\t1\t128574 7996\n",
      "41\t180683\t0.3\t1\t167134 13549\n",
      "42\t139132\t0.3\t1\t127852 11280\n",
      "43\t244330\t0.3\t1\t231176 13154\n",
      "44\t73162\t0.3\t1\t66477 6685\n",
      "45\t161358\t0.3\t1\t150280 11078\n",
      "46\t191244\t0.3\t1\t177928 13316\n",
      "47\t181249\t0.3\t1\t170520 10729\n",
      "48\t141208\t0.3\t1\t129881 11327\n",
      "49\t209170\t0.3\t1\t195203 13967\n",
      "50\t130797\t0.3\t1\t122619 8178\n",
      "51\t161830\t0.3\t1\t151113 10717\n",
      "52\t228270\t0.3\t1\t212080 16190\n",
      "53\t234755\t0.3\t1\t222809 11946\n",
      "54\t127266\t0.3\t1\t118588 8678\n",
      "55\t194077\t0.3\t1\t183800 10277\n",
      "56\t176229\t0.3\t1\t165657 10572\n",
      "57\t215239\t0.3\t1\t203613 11626\n",
      "58\t135465\t0.3\t1\t128771 6694\n",
      "59\t101608\t0.3\t1\t93951 7657\n",
      "60\t209358\t0.3\t1\t197818 11540\n",
      "61\t195964\t0.3\t1\t186497 9467\n",
      "62\t138436\t0.3\t1\t130152 8284\n",
      "63\t180787\t0.3\t1\t170284 10503\n",
      "64\t159166\t0.3\t1\t150739 8427\n",
      "65\t158205\t0.3\t1\t148817 9388\n",
      "66\t190752\t0.3\t1\t179381 11371\n",
      "67\t212991\t0.3\t1\t197325 15666\n",
      "68\t398495\t0.3\t1\t379493 19002\n",
      "69\t144273\t0.3\t1\t135320 8953\n",
      "70\t44631\t0.3\t1\t39323 5308\n",
      "71\t66845\t0.3\t1\t61618 5227\n",
      "72\t147002\t0.3\t1\t138540 8462\n",
      "73\t167373\t0.3\t1\t157828 9545\n",
      "74\t168016\t0.3\t1\t158069 9947\n",
      "75\t167181\t0.3\t1\t157169 10012\n",
      "76\t166317\t0.3\t1\t156562 9755\n",
      "77\t166255\t0.3\t1\t156536 9719\n",
      "78\t159048\t0.3\t1\t149642 9406\n",
      "79\t157934\t0.3\t1\t148379 9555\n",
      "80\t154378\t0.3\t1\t145219 9159\n",
      "81\t154058\t0.3\t1\t144915 9143\n",
      "82\t150711\t0.3\t1\t141779 8932\n",
      "83\t145124\t0.3\t1\t136331 8793\n",
      "84\t142844\t0.3\t1\t134506 8338\n",
      "85\t138019\t0.3\t1\t129804 8215\n",
      "86\t134359\t0.3\t1\t126493 7866\n",
      "87\t130206\t0.3\t1\t122445 7761\n",
      "88\t128131\t0.3\t1\t120526 7605\n",
      "89\t122017\t0.3\t1\t114723 7294\n",
      "90\t121870\t0.3\t1\t114562 7308\n",
      "91\t117180\t0.3\t1\t110087 7093\n",
      "92\t111859\t0.3\t1\t105091 6768\n",
      "93\t105927\t0.3\t1\t99576 6351\n",
      "94\t102547\t0.3\t1\t96466 6081\n",
      "95\t95663\t0.3\t1\t89977 5686\n",
      "96\t91505\t0.3\t1\t85936 5569\n",
      "97\t86471\t0.3\t1\t81209 5262\n",
      "98\t81070\t0.3\t1\t76155 4915\n",
      "99\t79954\t0.3\t1\t74613 5341\n",
      "100\t72882\t0.3\t1\t68442 4440\n",
      "101\t68704\t0.3\t1\t64356 4348\n",
      "102\t63576\t0.3\t1\t59490 4086\n",
      "103\t59368\t0.3\t1\t55615 3753\n",
      "104\t55418\t0.3\t1\t51914 3504\n",
      "105\t49653\t0.3\t1\t46415 3238\n",
      "106\t45561\t0.3\t1\t42651 2910\n",
      "107\t41569\t0.3\t1\t38864 2705\n",
      "108\t39242\t0.3\t1\t36735 2507\n",
      "109\t35181\t0.3\t1\t32884 2297\n",
      "110\t32351\t0.3\t1\t30217 2134\n",
      "111\t29181\t0.3\t1\t27222 1959\n",
      "112\t25447\t0.3\t1\t23718 1729\n",
      "113\t22680\t0.3\t1\t21019 1661\n",
      "114\t19902\t0.3\t1\t18472 1430\n",
      "115\t17153\t0.3\t1\t15867 1286\n",
      "116\t15113\t0.3\t1\t14061 1052\n",
      "117\t13006\t0.3\t1\t12016 990\n",
      "118\t11539\t0.3\t1\t10694 845\n",
      "119\t9431\t0.3\t1\t8642 789\n",
      "120\t7730\t0.3\t1\t7108 622\n",
      "121\t6854\t0.3\t1\t6273 581\n",
      "122\t5610\t0.3\t1\t5167 443\n",
      "123\t4861\t0.3\t1\t4443 418\n",
      "124\t4094\t0.3\t1\t3727 367\n",
      "125\t3465\t0.3\t1\t3160 305\n",
      "126\t2986\t0.3\t1\t2695 291\n",
      "127\t2510\t0.3\t1\t2289 221\n",
      "128\t1832\t0.3\t1\t1674 158\n",
      "129\t1619\t0.3\t1\t1450 169\n",
      "130\t1319\t0.3\t1\t1201 118\n",
      "131\t1116\t0.3\t1\t999 117\n",
      "132\t1063\t0.3\t1\t961 102\n",
      "133\t1337\t0.3\t1\t1208 129\n",
      "134\t1485\t0.3\t1\t1390 95\n",
      "135\t668\t0.3\t1\t615 53\n",
      "136\t381\t0.3\t1\t342 39\n",
      "137\t211\t0.3\t1\t197 14\n",
      "138\t160\t0.3\t1\t144 16\n",
      "139\t121\t0.3\t1\t106 15\n",
      "140\t71\t0.3\t1\t59 12\n",
      "141\t68\t0.3\t1\t57 11\n",
      "142\t52\t0.3\t1\t47 5\n",
      "143\t43\t0.3\t1\t38 5\n",
      "144\t24\t0.3\t1\t22 2\n",
      "145\t53\t0.3\t1\t40 13\n",
      "146\t42\t0.3\t1\t34 8\n",
      "147\t125\t0.3\t1\t106 19\n",
      "148\t83\t0.3\t1\t69 14\n",
      "149\t78\t0.3\t1\t73 5\n",
      "150\t763\t0.3\t1\t673 90\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "19624894 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8021 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 777.07 s (40 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,624,894\n",
      "Reads with adapters:                16,339,161 (83.3%)\n",
      "Reads written (passing filters):    19,624,894 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,943,734,100 bp\n",
      "Quality-trimmed:              15,599,605 bp (0.5%)\n",
      "Total written (filtered):  2,144,342,165 bp (72.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16339161 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 12.6%\n",
      "  G: 25.0%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1942196\t4906223.5\t0\t1942196\n",
      "2\t299564\t1226555.9\t0\t299564\n",
      "3\t177742\t306639.0\t0\t177742\n",
      "4\t105333\t76659.7\t0\t105333\n",
      "5\t100538\t19164.9\t0\t100538\n",
      "6\t87955\t4791.2\t0\t87955\n",
      "7\t71440\t1197.8\t0\t71440\n",
      "8\t102524\t299.5\t0\t102524\n",
      "9\t81367\t74.9\t0\t81047 320\n",
      "10\t89669\t18.7\t1\t85117 4552\n",
      "11\t95729\t4.7\t1\t89915 5814\n",
      "12\t95768\t1.2\t1\t89687 6081\n",
      "13\t108234\t0.3\t1\t101948 6286\n",
      "14\t103335\t0.3\t1\t96341 6994\n",
      "15\t82616\t0.3\t1\t77316 5300\n",
      "16\t121595\t0.3\t1\t114016 7579\n",
      "17\t98896\t0.3\t1\t92218 6678\n",
      "18\t93415\t0.3\t1\t87589 5826\n",
      "19\t120547\t0.3\t1\t112557 7990\n",
      "20\t113077\t0.3\t1\t106404 6673\n",
      "21\t99743\t0.3\t1\t93549 6194\n",
      "22\t113380\t0.3\t1\t106586 6794\n",
      "23\t116836\t0.3\t1\t109692 7144\n",
      "24\t119139\t0.3\t1\t111646 7493\n",
      "25\t135500\t0.3\t1\t127577 7923\n",
      "26\t112154\t0.3\t1\t104938 7216\n",
      "27\t122636\t0.3\t1\t114199 8437\n",
      "28\t117975\t0.3\t1\t111836 6139\n",
      "29\t125073\t0.3\t1\t117359 7714\n",
      "30\t127096\t0.3\t1\t120402 6694\n",
      "31\t137659\t0.3\t1\t129729 7930\n",
      "32\t123762\t0.3\t1\t117342 6420\n",
      "33\t138965\t0.3\t1\t130590 8375\n",
      "34\t180950\t0.3\t1\t170447 10503\n",
      "35\t102752\t0.3\t1\t97235 5517\n",
      "36\t141525\t0.3\t1\t133105 8420\n",
      "37\t151296\t0.3\t1\t142475 8821\n",
      "38\t141804\t0.3\t1\t134802 7002\n",
      "39\t139699\t0.3\t1\t132682 7017\n",
      "40\t156245\t0.3\t1\t147123 9122\n",
      "41\t150848\t0.3\t1\t143057 7791\n",
      "42\t156523\t0.3\t1\t148491 8032\n",
      "43\t142855\t0.3\t1\t135344 7511\n",
      "44\t166066\t0.3\t1\t157102 8964\n",
      "45\t178466\t0.3\t1\t169227 9239\n",
      "46\t128203\t0.3\t1\t121604 6599\n",
      "47\t163666\t0.3\t1\t155525 8141\n",
      "48\t161229\t0.3\t1\t153502 7727\n",
      "49\t169595\t0.3\t1\t161041 8554\n",
      "50\t162191\t0.3\t1\t153908 8283\n",
      "51\t204248\t0.3\t1\t193780 10468\n",
      "52\t147164\t0.3\t1\t140471 6693\n",
      "53\t177383\t0.3\t1\t169465 7918\n",
      "54\t135474\t0.3\t1\t129206 6268\n",
      "55\t177540\t0.3\t1\t169636 7904\n",
      "56\t183270\t0.3\t1\t174617 8653\n",
      "57\t175273\t0.3\t1\t167802 7471\n",
      "58\t168130\t0.3\t1\t160997 7133\n",
      "59\t172563\t0.3\t1\t164868 7695\n",
      "60\t179506\t0.3\t1\t171470 8036\n",
      "61\t186851\t0.3\t1\t178181 8670\n",
      "62\t203675\t0.3\t1\t194173 9502\n",
      "63\t273998\t0.3\t1\t262527 11471\n",
      "64\t226808\t0.3\t1\t218491 8317\n",
      "65\t147108\t0.3\t1\t140746 6362\n",
      "66\t160644\t0.3\t1\t153693 6951\n",
      "67\t159286\t0.3\t1\t152193 7093\n",
      "68\t161933\t0.3\t1\t155050 6883\n",
      "69\t160655\t0.3\t1\t153681 6974\n",
      "70\t170416\t0.3\t1\t162996 7420\n",
      "71\t179225\t0.3\t1\t171910 7315\n",
      "72\t171231\t0.3\t1\t163925 7306\n",
      "73\t181424\t0.3\t1\t173901 7523\n",
      "74\t167534\t0.3\t1\t160668 6866\n",
      "75\t160295\t0.3\t1\t153643 6652\n",
      "76\t157715\t0.3\t1\t151296 6419\n",
      "77\t157679\t0.3\t1\t150903 6776\n",
      "78\t152483\t0.3\t1\t146130 6353\n",
      "79\t147737\t0.3\t1\t141492 6245\n",
      "80\t146667\t0.3\t1\t140666 6001\n",
      "81\t145920\t0.3\t1\t140003 5917\n",
      "82\t140689\t0.3\t1\t134853 5836\n",
      "83\t136034\t0.3\t1\t130439 5595\n",
      "84\t133908\t0.3\t1\t128250 5658\n",
      "85\t131675\t0.3\t1\t126012 5663\n",
      "86\t131672\t0.3\t1\t126197 5475\n",
      "87\t133848\t0.3\t1\t128384 5464\n",
      "88\t129850\t0.3\t1\t124580 5270\n",
      "89\t116878\t0.3\t1\t112187 4691\n",
      "90\t113165\t0.3\t1\t108451 4714\n",
      "91\t107851\t0.3\t1\t103636 4215\n",
      "92\t101212\t0.3\t1\t97187 4025\n",
      "93\t98995\t0.3\t1\t94995 4000\n",
      "94\t95394\t0.3\t1\t91542 3852\n",
      "95\t90020\t0.3\t1\t86380 3640\n",
      "96\t85913\t0.3\t1\t82550 3363\n",
      "97\t80493\t0.3\t1\t77312 3181\n",
      "98\t75879\t0.3\t1\t72925 2954\n",
      "99\t77605\t0.3\t1\t74598 3007\n",
      "100\t70637\t0.3\t1\t67912 2725\n",
      "101\t67043\t0.3\t1\t64475 2568\n",
      "102\t62076\t0.3\t1\t59759 2317\n",
      "103\t56002\t0.3\t1\t53869 2133\n",
      "104\t53213\t0.3\t1\t51172 2041\n",
      "105\t47322\t0.3\t1\t45493 1829\n",
      "106\t43803\t0.3\t1\t42161 1642\n",
      "107\t39428\t0.3\t1\t37938 1490\n",
      "108\t36024\t0.3\t1\t34621 1403\n",
      "109\t33005\t0.3\t1\t31801 1204\n",
      "110\t30413\t0.3\t1\t29223 1190\n",
      "111\t28718\t0.3\t1\t27704 1014\n",
      "112\t24826\t0.3\t1\t23909 917\n",
      "113\t22246\t0.3\t1\t21395 851\n",
      "114\t19255\t0.3\t1\t18504 751\n",
      "115\t16125\t0.3\t1\t15460 665\n",
      "116\t14013\t0.3\t1\t13452 561\n",
      "117\t11755\t0.3\t1\t11314 441\n",
      "118\t10504\t0.3\t1\t10075 429\n",
      "119\t8740\t0.3\t1\t8381 359\n",
      "120\t7099\t0.3\t1\t6789 310\n",
      "121\t6395\t0.3\t1\t6127 268\n",
      "122\t5283\t0.3\t1\t5086 197\n",
      "123\t4656\t0.3\t1\t4456 200\n",
      "124\t3824\t0.3\t1\t3672 152\n",
      "125\t3323\t0.3\t1\t3181 142\n",
      "126\t2821\t0.3\t1\t2694 127\n",
      "127\t2334\t0.3\t1\t2219 115\n",
      "128\t1727\t0.3\t1\t1654 73\n",
      "129\t1502\t0.3\t1\t1423 79\n",
      "130\t1215\t0.3\t1\t1157 58\n",
      "131\t1055\t0.3\t1\t1002 53\n",
      "132\t1009\t0.3\t1\t967 42\n",
      "133\t1312\t0.3\t1\t1260 52\n",
      "134\t1503\t0.3\t1\t1462 41\n",
      "135\t647\t0.3\t1\t618 29\n",
      "136\t354\t0.3\t1\t338 16\n",
      "137\t216\t0.3\t1\t209 7\n",
      "138\t150\t0.3\t1\t140 10\n",
      "139\t114\t0.3\t1\t110 4\n",
      "140\t64\t0.3\t1\t60 4\n",
      "141\t66\t0.3\t1\t61 5\n",
      "142\t51\t0.3\t1\t49 2\n",
      "143\t40\t0.3\t1\t37 3\n",
      "144\t27\t0.3\t1\t25 2\n",
      "145\t50\t0.3\t1\t45 5\n",
      "146\t38\t0.3\t1\t37 1\n",
      "147\t120\t0.3\t1\t112 8\n",
      "148\t74\t0.3\t1\t68 6\n",
      "149\t76\t0.3\t1\t73 3\n",
      "150\t581\t0.3\t1\t559 22\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "19624894 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t31376 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 755.54 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,571,235\n",
      "Reads with adapters:                16,407,295 (83.8%)\n",
      "Reads written (passing filters):    19,571,235 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,935,685,250 bp\n",
      "Quality-trimmed:               8,266,740 bp (0.3%)\n",
      "Total written (filtered):  2,130,672,037 bp (72.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16407295 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 14.6%\n",
      "  G: 21.1%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1871205\t4892808.8\t0\t1871205\n",
      "2\t281413\t1223202.2\t0\t281413\n",
      "3\t165620\t305800.5\t0\t165620\n",
      "4\t102750\t76450.1\t0\t102750\n",
      "5\t91621\t19112.5\t0\t91621\n",
      "6\t89878\t4778.1\t0\t89878\n",
      "7\t72653\t1194.5\t0\t72653\n",
      "8\t92540\t298.6\t0\t92540\n",
      "9\t93344\t74.7\t0\t92169 1175\n",
      "10\t86372\t18.7\t1\t80800 5572\n",
      "11\t99125\t4.7\t1\t90406 8719\n",
      "12\t93690\t1.2\t1\t86859 6831\n",
      "13\t96288\t0.3\t1\t88938 7350\n",
      "14\t98016\t0.3\t1\t89939 8077\n",
      "15\t102285\t0.3\t1\t94257 8028\n",
      "16\t101242\t0.3\t1\t92425 8817\n",
      "17\t123641\t0.3\t1\t112025 11616\n",
      "18\t107224\t0.3\t1\t99543 7681\n",
      "19\t90677\t0.3\t1\t83921 6756\n",
      "20\t111453\t0.3\t1\t102938 8515\n",
      "21\t124864\t0.3\t1\t115203 9661\n",
      "22\t102217\t0.3\t1\t95257 6960\n",
      "23\t124937\t0.3\t1\t115301 9636\n",
      "24\t113499\t0.3\t1\t102977 10522\n",
      "25\t124401\t0.3\t1\t114862 9539\n",
      "26\t112120\t0.3\t1\t104445 7675\n",
      "27\t128185\t0.3\t1\t117570 10615\n",
      "28\t125849\t0.3\t1\t115930 9919\n",
      "29\t134332\t0.3\t1\t122337 11995\n",
      "30\t144690\t0.3\t1\t135178 9512\n",
      "31\t111404\t0.3\t1\t103047 8357\n",
      "32\t135894\t0.3\t1\t125650 10244\n",
      "33\t131757\t0.3\t1\t121043 10714\n",
      "34\t143190\t0.3\t1\t131548 11642\n",
      "35\t157418\t0.3\t1\t142743 14675\n",
      "36\t129247\t0.3\t1\t119787 9460\n",
      "37\t149221\t0.3\t1\t138285 10936\n",
      "38\t145468\t0.3\t1\t133187 12281\n",
      "39\t172504\t0.3\t1\t157173 15331\n",
      "40\t134071\t0.3\t1\t124322 9749\n",
      "41\t180864\t0.3\t1\t164818 16046\n",
      "42\t137219\t0.3\t1\t124408 12811\n",
      "43\t239349\t0.3\t1\t223461 15888\n",
      "44\t74486\t0.3\t1\t66856 7630\n",
      "45\t161688\t0.3\t1\t148747 12941\n",
      "46\t191505\t0.3\t1\t176273 15232\n",
      "47\t179273\t0.3\t1\t166898 12375\n",
      "48\t140851\t0.3\t1\t128229 12622\n",
      "49\t208539\t0.3\t1\t192459 16080\n",
      "50\t127958\t0.3\t1\t118792 9166\n",
      "51\t162484\t0.3\t1\t150649 11835\n",
      "52\t227313\t0.3\t1\t208878 18435\n",
      "53\t230954\t0.3\t1\t217665 13289\n",
      "54\t127985\t0.3\t1\t117878 10107\n",
      "55\t190088\t0.3\t1\t177776 12312\n",
      "56\t183666\t0.3\t1\t171198 12468\n",
      "57\t212537\t0.3\t1\t198874 13663\n",
      "58\t133097\t0.3\t1\t125108 7989\n",
      "59\t104505\t0.3\t1\t95591 8914\n",
      "60\t206694\t0.3\t1\t193657 13037\n",
      "61\t190421\t0.3\t1\t179352 11069\n",
      "62\t144000\t0.3\t1\t134103 9897\n",
      "63\t180563\t0.3\t1\t168547 12016\n",
      "64\t155217\t0.3\t1\t145289 9928\n",
      "65\t158098\t0.3\t1\t147007 11091\n",
      "66\t188945\t0.3\t1\t175941 13004\n",
      "67\t211986\t0.3\t1\t194786 17200\n",
      "68\t378408\t0.3\t1\t357096 21312\n",
      "69\t162579\t0.3\t1\t151778 10801\n",
      "70\t55965\t0.3\t1\t49649 6316\n",
      "71\t63814\t0.3\t1\t58189 5625\n",
      "72\t145461\t0.3\t1\t136394 9067\n",
      "73\t166164\t0.3\t1\t155484 10680\n",
      "74\t168411\t0.3\t1\t157197 11214\n",
      "75\t167596\t0.3\t1\t156098 11498\n",
      "76\t166388\t0.3\t1\t154457 11931\n",
      "77\t167040\t0.3\t1\t155459 11581\n",
      "78\t159704\t0.3\t1\t148462 11242\n",
      "79\t158706\t0.3\t1\t147370 11336\n",
      "80\t154542\t0.3\t1\t143508 11034\n",
      "81\t154753\t0.3\t1\t143966 10787\n",
      "82\t150886\t0.3\t1\t140115 10771\n",
      "83\t145780\t0.3\t1\t135596 10184\n",
      "84\t142086\t0.3\t1\t132138 9948\n",
      "85\t137811\t0.3\t1\t128196 9615\n",
      "86\t134898\t0.3\t1\t125576 9322\n",
      "87\t129676\t0.3\t1\t120849 8827\n",
      "88\t128629\t0.3\t1\t119738 8891\n",
      "89\t123313\t0.3\t1\t114615 8698\n",
      "90\t123132\t0.3\t1\t114553 8579\n",
      "91\t117637\t0.3\t1\t109407 8230\n",
      "92\t112493\t0.3\t1\t104754 7739\n",
      "93\t106111\t0.3\t1\t98757 7354\n",
      "94\t103337\t0.3\t1\t96126 7211\n",
      "95\t95958\t0.3\t1\t89095 6863\n",
      "96\t91266\t0.3\t1\t84746 6520\n",
      "97\t86736\t0.3\t1\t80632 6104\n",
      "98\t81864\t0.3\t1\t75993 5871\n",
      "99\t80342\t0.3\t1\t74325 6017\n",
      "100\t73408\t0.3\t1\t68128 5280\n",
      "101\t68599\t0.3\t1\t63783 4816\n",
      "102\t64339\t0.3\t1\t59627 4712\n",
      "103\t59235\t0.3\t1\t54992 4243\n",
      "104\t55526\t0.3\t1\t51587 3939\n",
      "105\t49435\t0.3\t1\t45975 3460\n",
      "106\t45302\t0.3\t1\t42055 3247\n",
      "107\t41568\t0.3\t1\t38569 2999\n",
      "108\t39344\t0.3\t1\t36429 2915\n",
      "109\t35142\t0.3\t1\t32532 2610\n",
      "110\t32440\t0.3\t1\t30035 2405\n",
      "111\t29201\t0.3\t1\t26917 2284\n",
      "112\t25771\t0.3\t1\t23691 2080\n",
      "113\t22359\t0.3\t1\t20490 1869\n",
      "114\t19994\t0.3\t1\t18373 1621\n",
      "115\t17421\t0.3\t1\t16000 1421\n",
      "116\t15295\t0.3\t1\t14082 1213\n",
      "117\t13006\t0.3\t1\t11960 1046\n",
      "118\t11522\t0.3\t1\t10599 923\n",
      "119\t9598\t0.3\t1\t8847 751\n",
      "120\t7795\t0.3\t1\t7132 663\n",
      "121\t6634\t0.3\t1\t6018 616\n",
      "122\t5895\t0.3\t1\t5383 512\n",
      "123\t4806\t0.3\t1\t4352 454\n",
      "124\t4169\t0.3\t1\t3804 365\n",
      "125\t3459\t0.3\t1\t3117 342\n",
      "126\t3006\t0.3\t1\t2739 267\n",
      "127\t2458\t0.3\t1\t2229 229\n",
      "128\t1849\t0.3\t1\t1632 217\n",
      "129\t1614\t0.3\t1\t1454 160\n",
      "130\t1309\t0.3\t1\t1172 137\n",
      "131\t1113\t0.3\t1\t990 123\n",
      "132\t1037\t0.3\t1\t938 99\n",
      "133\t1203\t0.3\t1\t1080 123\n",
      "134\t1440\t0.3\t1\t1347 93\n",
      "135\t651\t0.3\t1\t599 52\n",
      "136\t330\t0.3\t1\t299 31\n",
      "137\t262\t0.3\t1\t226 36\n",
      "138\t169\t0.3\t1\t150 19\n",
      "139\t125\t0.3\t1\t105 20\n",
      "140\t97\t0.3\t1\t85 12\n",
      "141\t76\t0.3\t1\t66 10\n",
      "142\t39\t0.3\t1\t32 7\n",
      "143\t36\t0.3\t1\t31 5\n",
      "144\t28\t0.3\t1\t23 5\n",
      "145\t62\t0.3\t1\t55 7\n",
      "146\t39\t0.3\t1\t35 4\n",
      "147\t138\t0.3\t1\t123 15\n",
      "148\t73\t0.3\t1\t68 5\n",
      "149\t57\t0.3\t1\t50 7\n",
      "150\t745\t0.3\t1\t637 108\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "19571235 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7786 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_1_S1_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 791.00 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,571,235\n",
      "Reads with adapters:                16,271,535 (83.1%)\n",
      "Reads written (passing filters):    19,571,235 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,935,685,250 bp\n",
      "Quality-trimmed:              14,968,782 bp (0.5%)\n",
      "Total written (filtered):  2,137,866,393 bp (72.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16271535 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.4%\n",
      "  C: 12.6%\n",
      "  G: 25.1%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1934443\t4892808.8\t0\t1934443\n",
      "2\t292006\t1223202.2\t0\t292006\n",
      "3\t180981\t305800.5\t0\t180981\n",
      "4\t103045\t76450.1\t0\t103045\n",
      "5\t98608\t19112.5\t0\t98608\n",
      "6\t87562\t4778.1\t0\t87562\n",
      "7\t69739\t1194.5\t0\t69739\n",
      "8\t101138\t298.6\t0\t101138\n",
      "9\t80590\t74.7\t0\t80291 299\n",
      "10\t87083\t18.7\t1\t80903 6180\n",
      "11\t94744\t4.7\t1\t86859 7885\n",
      "12\t93509\t1.2\t1\t85833 7676\n",
      "13\t107672\t0.3\t1\t99050 8622\n",
      "14\t100767\t0.3\t1\t91771 8996\n",
      "15\t81145\t0.3\t1\t74763 6382\n",
      "16\t119727\t0.3\t1\t109867 9860\n",
      "17\t101003\t0.3\t1\t92682 8321\n",
      "18\t90163\t0.3\t1\t83122 7041\n",
      "19\t118403\t0.3\t1\t108874 9529\n",
      "20\t111750\t0.3\t1\t103085 8665\n",
      "21\t98369\t0.3\t1\t90814 7555\n",
      "22\t111808\t0.3\t1\t103242 8566\n",
      "23\t118255\t0.3\t1\t109009 9246\n",
      "24\t118284\t0.3\t1\t108899 9385\n",
      "25\t134913\t0.3\t1\t125516 9397\n",
      "26\t111710\t0.3\t1\t102736 8974\n",
      "27\t124043\t0.3\t1\t113612 10431\n",
      "28\t113048\t0.3\t1\t105567 7481\n",
      "29\t126854\t0.3\t1\t117529 9325\n",
      "30\t124172\t0.3\t1\t115653 8519\n",
      "31\t138567\t0.3\t1\t128798 9769\n",
      "32\t118532\t0.3\t1\t110581 7951\n",
      "33\t139520\t0.3\t1\t129113 10407\n",
      "34\t182115\t0.3\t1\t169415 12700\n",
      "35\t97650\t0.3\t1\t90702 6948\n",
      "36\t141232\t0.3\t1\t130648 10584\n",
      "37\t151183\t0.3\t1\t140571 10612\n",
      "38\t140794\t0.3\t1\t131792 9002\n",
      "39\t135094\t0.3\t1\t126791 8303\n",
      "40\t155470\t0.3\t1\t144881 10589\n",
      "41\t152110\t0.3\t1\t142307 9803\n",
      "42\t156034\t0.3\t1\t145893 10141\n",
      "43\t140408\t0.3\t1\t131398 9010\n",
      "44\t166488\t0.3\t1\t155279 11209\n",
      "45\t176759\t0.3\t1\t165212 11547\n",
      "46\t127616\t0.3\t1\t119430 8186\n",
      "47\t162788\t0.3\t1\t153156 9632\n",
      "48\t159821\t0.3\t1\t150024 9797\n",
      "49\t167515\t0.3\t1\t156791 10724\n",
      "50\t162854\t0.3\t1\t152416 10438\n",
      "51\t201852\t0.3\t1\t189155 12697\n",
      "52\t145803\t0.3\t1\t136773 9030\n",
      "53\t181599\t0.3\t1\t171435 10164\n",
      "54\t132563\t0.3\t1\t124771 7792\n",
      "55\t175847\t0.3\t1\t165921 9926\n",
      "56\t182238\t0.3\t1\t170952 11286\n",
      "57\t175759\t0.3\t1\t166454 9305\n",
      "58\t168899\t0.3\t1\t159768 9131\n",
      "59\t170487\t0.3\t1\t160788 9699\n",
      "60\t179591\t0.3\t1\t169442 10149\n",
      "61\t183203\t0.3\t1\t173168 10035\n",
      "62\t201509\t0.3\t1\t189650 11859\n",
      "63\t267932\t0.3\t1\t254493 13439\n",
      "64\t223354\t0.3\t1\t213922 9432\n",
      "65\t145944\t0.3\t1\t138249 7695\n",
      "66\t160488\t0.3\t1\t151671 8817\n",
      "67\t160429\t0.3\t1\t151245 9184\n",
      "68\t163626\t0.3\t1\t154620 9006\n",
      "69\t162388\t0.3\t1\t153192 9196\n",
      "70\t171183\t0.3\t1\t161721 9462\n",
      "71\t179954\t0.3\t1\t170347 9607\n",
      "72\t173057\t0.3\t1\t163544 9513\n",
      "73\t181347\t0.3\t1\t171665 9682\n",
      "74\t168687\t0.3\t1\t159362 9325\n",
      "75\t161354\t0.3\t1\t152554 8800\n",
      "76\t158086\t0.3\t1\t149538 8548\n",
      "77\t158408\t0.3\t1\t149654 8754\n",
      "78\t152524\t0.3\t1\t144195 8329\n",
      "79\t148468\t0.3\t1\t140463 8005\n",
      "80\t146791\t0.3\t1\t138965 7826\n",
      "81\t146914\t0.3\t1\t138951 7963\n",
      "82\t140738\t0.3\t1\t133245 7493\n",
      "83\t136720\t0.3\t1\t129557 7163\n",
      "84\t133340\t0.3\t1\t126399 6941\n",
      "85\t131509\t0.3\t1\t124949 6560\n",
      "86\t131725\t0.3\t1\t125074 6651\n",
      "87\t132992\t0.3\t1\t126515 6477\n",
      "88\t130544\t0.3\t1\t124131 6413\n",
      "89\t118247\t0.3\t1\t112246 6001\n",
      "90\t114880\t0.3\t1\t109042 5838\n",
      "91\t108202\t0.3\t1\t102767 5435\n",
      "92\t102465\t0.3\t1\t97298 5167\n",
      "93\t99313\t0.3\t1\t94369 4944\n",
      "94\t96247\t0.3\t1\t91557 4690\n",
      "95\t90355\t0.3\t1\t85764 4591\n",
      "96\t86105\t0.3\t1\t81859 4246\n",
      "97\t80863\t0.3\t1\t77002 3861\n",
      "98\t76806\t0.3\t1\t73205 3601\n",
      "99\t77725\t0.3\t1\t74014 3711\n",
      "100\t71514\t0.3\t1\t68090 3424\n",
      "101\t67122\t0.3\t1\t63869 3253\n",
      "102\t62547\t0.3\t1\t59630 2917\n",
      "103\t55990\t0.3\t1\t53291 2699\n",
      "104\t53130\t0.3\t1\t50496 2634\n",
      "105\t47112\t0.3\t1\t44737 2375\n",
      "106\t43436\t0.3\t1\t41396 2040\n",
      "107\t39313\t0.3\t1\t37430 1883\n",
      "108\t36083\t0.3\t1\t34294 1789\n",
      "109\t32958\t0.3\t1\t31374 1584\n",
      "110\t30549\t0.3\t1\t29023 1526\n",
      "111\t28680\t0.3\t1\t27349 1331\n",
      "112\t25012\t0.3\t1\t23754 1258\n",
      "113\t21932\t0.3\t1\t20848 1084\n",
      "114\t19318\t0.3\t1\t18424 894\n",
      "115\t16367\t0.3\t1\t15528 839\n",
      "116\t14214\t0.3\t1\t13511 703\n",
      "117\t11890\t0.3\t1\t11258 632\n",
      "118\t10558\t0.3\t1\t10008 550\n",
      "119\t8964\t0.3\t1\t8531 433\n",
      "120\t7160\t0.3\t1\t6810 350\n",
      "121\t6260\t0.3\t1\t5925 335\n",
      "122\t5526\t0.3\t1\t5261 265\n",
      "123\t4601\t0.3\t1\t4388 213\n",
      "124\t3924\t0.3\t1\t3711 213\n",
      "125\t3337\t0.3\t1\t3153 184\n",
      "126\t2851\t0.3\t1\t2683 168\n",
      "127\t2269\t0.3\t1\t2147 122\n",
      "128\t1715\t0.3\t1\t1615 100\n",
      "129\t1527\t0.3\t1\t1454 73\n",
      "130\t1240\t0.3\t1\t1181 59\n",
      "131\t1038\t0.3\t1\t985 53\n",
      "132\t992\t0.3\t1\t936 56\n",
      "133\t1237\t0.3\t1\t1180 57\n",
      "134\t1424\t0.3\t1\t1372 52\n",
      "135\t637\t0.3\t1\t603 34\n",
      "136\t325\t0.3\t1\t316 9\n",
      "137\t239\t0.3\t1\t218 21\n",
      "138\t165\t0.3\t1\t161 4\n",
      "139\t114\t0.3\t1\t104 10\n",
      "140\t92\t0.3\t1\t87 5\n",
      "141\t71\t0.3\t1\t68 3\n",
      "142\t32\t0.3\t1\t29 3\n",
      "143\t33\t0.3\t1\t28 5\n",
      "144\t26\t0.3\t1\t21 5\n",
      "145\t62\t0.3\t1\t60 2\n",
      "146\t42\t0.3\t1\t41 1\n",
      "147\t127\t0.3\t1\t113 14\n",
      "148\t65\t0.3\t1\t60 5\n",
      "149\t42\t0.3\t1\t37 5\n",
      "150\t529\t0.3\t1\t500 29\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "19571235 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t29241 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 674.03 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,734,461\n",
      "Reads with adapters:                14,777,967 (83.3%)\n",
      "Reads written (passing filters):    17,734,461 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,660,169,150 bp\n",
      "Quality-trimmed:               5,148,925 bp (0.2%)\n",
      "Total written (filtered):  1,960,839,520 bp (73.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14777967 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 12.9%\n",
      "  G: 21.9%\n",
      "  T: 35.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1738050\t4433615.2\t0\t1738050\n",
      "2\t278207\t1108403.8\t0\t278207\n",
      "3\t163757\t277101.0\t0\t163757\n",
      "4\t100314\t69275.2\t0\t100314\n",
      "5\t85611\t17318.8\t0\t85611\n",
      "6\t82934\t4329.7\t0\t82934\n",
      "7\t71901\t1082.4\t0\t71901\n",
      "8\t92752\t270.6\t0\t92752\n",
      "9\t83307\t67.7\t0\t82257 1050\n",
      "10\t83535\t16.9\t1\t78970 4565\n",
      "11\t92909\t4.2\t1\t85625 7284\n",
      "12\t89396\t1.1\t1\t83407 5989\n",
      "13\t91391\t0.3\t1\t85129 6262\n",
      "14\t92986\t0.3\t1\t86205 6781\n",
      "15\t95417\t0.3\t1\t88994 6423\n",
      "16\t96485\t0.3\t1\t89177 7308\n",
      "17\t117735\t0.3\t1\t108243 9492\n",
      "18\t99811\t0.3\t1\t93600 6211\n",
      "19\t87560\t0.3\t1\t81823 5737\n",
      "20\t104404\t0.3\t1\t97366 7038\n",
      "21\t115608\t0.3\t1\t107491 8117\n",
      "22\t99044\t0.3\t1\t93521 5523\n",
      "23\t114295\t0.3\t1\t106563 7732\n",
      "24\t108405\t0.3\t1\t99404 9001\n",
      "25\t115898\t0.3\t1\t108064 7834\n",
      "26\t106400\t0.3\t1\t100103 6297\n",
      "27\t119862\t0.3\t1\t110913 8949\n",
      "28\t117009\t0.3\t1\t108719 8290\n",
      "29\t126066\t0.3\t1\t115846 10220\n",
      "30\t135672\t0.3\t1\t127802 7870\n",
      "31\t103769\t0.3\t1\t96862 6907\n",
      "32\t134576\t0.3\t1\t125712 8864\n",
      "33\t115506\t0.3\t1\t107678 7828\n",
      "34\t131922\t0.3\t1\t122476 9446\n",
      "35\t139540\t0.3\t1\t128729 10811\n",
      "36\t129944\t0.3\t1\t121921 8023\n",
      "37\t155181\t0.3\t1\t145058 10123\n",
      "38\t125033\t0.3\t1\t115777 9256\n",
      "39\t137711\t0.3\t1\t129241 8470\n",
      "40\t145763\t0.3\t1\t136531 9232\n",
      "41\t145406\t0.3\t1\t134994 10412\n",
      "42\t140227\t0.3\t1\t129509 10718\n",
      "43\t213869\t0.3\t1\t202923 10946\n",
      "44\t71935\t0.3\t1\t65627 6308\n",
      "45\t151697\t0.3\t1\t142261 9436\n",
      "46\t139466\t0.3\t1\t130673 8793\n",
      "47\t151314\t0.3\t1\t141909 9405\n",
      "48\t149772\t0.3\t1\t140199 9573\n",
      "49\t163566\t0.3\t1\t152198 11368\n",
      "50\t142863\t0.3\t1\t134146 8717\n",
      "51\t154145\t0.3\t1\t144042 10103\n",
      "52\t159345\t0.3\t1\t148339 11006\n",
      "53\t168861\t0.3\t1\t158401 10460\n",
      "54\t140944\t0.3\t1\t132160 8784\n",
      "55\t163358\t0.3\t1\t152729 10629\n",
      "56\t198374\t0.3\t1\t186858 11516\n",
      "57\t129041\t0.3\t1\t119923 9118\n",
      "58\t168939\t0.3\t1\t159723 9216\n",
      "59\t149936\t0.3\t1\t140432 9504\n",
      "60\t161773\t0.3\t1\t151304 10469\n",
      "61\t159837\t0.3\t1\t150130 9707\n",
      "62\t178889\t0.3\t1\t168432 10457\n",
      "63\t140394\t0.3\t1\t131036 9358\n",
      "64\t159197\t0.3\t1\t149784 9413\n",
      "65\t178150\t0.3\t1\t167258 10892\n",
      "66\t185093\t0.3\t1\t173786 11307\n",
      "67\t199374\t0.3\t1\t184232 15142\n",
      "68\t376952\t0.3\t1\t359289 17663\n",
      "69\t53765\t0.3\t1\t48910 4855\n",
      "70\t23496\t0.3\t1\t19245 4251\n",
      "71\t105909\t0.3\t1\t99682 6227\n",
      "72\t143025\t0.3\t1\t134667 8358\n",
      "73\t150010\t0.3\t1\t141153 8857\n",
      "74\t148858\t0.3\t1\t139863 8995\n",
      "75\t147559\t0.3\t1\t138722 8837\n",
      "76\t146256\t0.3\t1\t137657 8599\n",
      "77\t146473\t0.3\t1\t137642 8831\n",
      "78\t139439\t0.3\t1\t130923 8516\n",
      "79\t138812\t0.3\t1\t130392 8420\n",
      "80\t134925\t0.3\t1\t126986 7939\n",
      "81\t132740\t0.3\t1\t124849 7891\n",
      "82\t129707\t0.3\t1\t121848 7859\n",
      "83\t125045\t0.3\t1\t117720 7325\n",
      "84\t120955\t0.3\t1\t113756 7199\n",
      "85\t117638\t0.3\t1\t110705 6933\n",
      "86\t114999\t0.3\t1\t108021 6978\n",
      "87\t110985\t0.3\t1\t104322 6663\n",
      "88\t108344\t0.3\t1\t101912 6432\n",
      "89\t103347\t0.3\t1\t97100 6247\n",
      "90\t102084\t0.3\t1\t95939 6145\n",
      "91\t97477\t0.3\t1\t91376 6101\n",
      "92\t92174\t0.3\t1\t86611 5563\n",
      "93\t86985\t0.3\t1\t81755 5230\n",
      "94\t84316\t0.3\t1\t79155 5161\n",
      "95\t78474\t0.3\t1\t73684 4790\n",
      "96\t73594\t0.3\t1\t68994 4600\n",
      "97\t68665\t0.3\t1\t64422 4243\n",
      "98\t65130\t0.3\t1\t60988 4142\n",
      "99\t62724\t0.3\t1\t58428 4296\n",
      "100\t57069\t0.3\t1\t53328 3741\n",
      "101\t53096\t0.3\t1\t49656 3440\n",
      "102\t48453\t0.3\t1\t45307 3146\n",
      "103\t44883\t0.3\t1\t41855 3028\n",
      "104\t41860\t0.3\t1\t39145 2715\n",
      "105\t37319\t0.3\t1\t34804 2515\n",
      "106\t33117\t0.3\t1\t30846 2271\n",
      "107\t30224\t0.3\t1\t28086 2138\n",
      "108\t28577\t0.3\t1\t26568 2009\n",
      "109\t24900\t0.3\t1\t23148 1752\n",
      "110\t22182\t0.3\t1\t20539 1643\n",
      "111\t19835\t0.3\t1\t18458 1377\n",
      "112\t17455\t0.3\t1\t16149 1306\n",
      "113\t15019\t0.3\t1\t13793 1226\n",
      "114\t13036\t0.3\t1\t12026 1010\n",
      "115\t11314\t0.3\t1\t10380 934\n",
      "116\t9567\t0.3\t1\t8809 758\n",
      "117\t8005\t0.3\t1\t7289 716\n",
      "118\t6890\t0.3\t1\t6310 580\n",
      "119\t5712\t0.3\t1\t5183 529\n",
      "120\t4607\t0.3\t1\t4200 407\n",
      "121\t4029\t0.3\t1\t3642 387\n",
      "122\t3364\t0.3\t1\t3030 334\n",
      "123\t2640\t0.3\t1\t2382 258\n",
      "124\t2162\t0.3\t1\t1941 221\n",
      "125\t1794\t0.3\t1\t1596 198\n",
      "126\t1531\t0.3\t1\t1353 178\n",
      "127\t1215\t0.3\t1\t1098 117\n",
      "128\t1005\t0.3\t1\t879 126\n",
      "129\t809\t0.3\t1\t695 114\n",
      "130\t661\t0.3\t1\t597 64\n",
      "131\t509\t0.3\t1\t455 54\n",
      "132\t443\t0.3\t1\t389 54\n",
      "133\t409\t0.3\t1\t355 54\n",
      "134\t483\t0.3\t1\t430 53\n",
      "135\t256\t0.3\t1\t227 29\n",
      "136\t159\t0.3\t1\t133 26\n",
      "137\t104\t0.3\t1\t91 13\n",
      "138\t67\t0.3\t1\t53 14\n",
      "139\t54\t0.3\t1\t47 7\n",
      "140\t47\t0.3\t1\t39 8\n",
      "141\t21\t0.3\t1\t15 6\n",
      "142\t21\t0.3\t1\t19 2\n",
      "143\t8\t0.3\t1\t4 4\n",
      "144\t21\t0.3\t1\t20 1\n",
      "145\t51\t0.3\t1\t46 5\n",
      "146\t37\t0.3\t1\t29 8\n",
      "147\t70\t0.3\t1\t63 7\n",
      "148\t65\t0.3\t1\t52 13\n",
      "149\t39\t0.3\t1\t35 4\n",
      "150\t480\t0.3\t1\t358 122\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17734461 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3391 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 700.79 s (40 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,734,461\n",
      "Reads with adapters:                14,708,042 (82.9%)\n",
      "Reads written (passing filters):    17,734,461 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,660,169,150 bp\n",
      "Quality-trimmed:              13,508,119 bp (0.5%)\n",
      "Total written (filtered):  1,965,332,665 bp (73.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14708042 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 12.0%\n",
      "  G: 24.7%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1823726\t4433615.2\t0\t1823726\n",
      "2\t284755\t1108403.8\t0\t284755\n",
      "3\t172034\t277101.0\t0\t172034\n",
      "4\t100056\t69275.2\t0\t100056\n",
      "5\t94653\t17318.8\t0\t94653\n",
      "6\t82500\t4329.7\t0\t82500\n",
      "7\t67622\t1082.4\t0\t67622\n",
      "8\t96831\t270.6\t0\t96831\n",
      "9\t76935\t67.7\t0\t76693 242\n",
      "10\t84917\t16.9\t1\t81330 3587\n",
      "11\t89902\t4.2\t1\t84914 4988\n",
      "12\t90846\t1.1\t1\t85794 5052\n",
      "13\t101679\t0.3\t1\t96540 5139\n",
      "14\t96092\t0.3\t1\t90143 5949\n",
      "15\t77319\t0.3\t1\t73208 4111\n",
      "16\t114151\t0.3\t1\t107904 6247\n",
      "17\t91752\t0.3\t1\t86386 5366\n",
      "18\t88174\t0.3\t1\t83679 4495\n",
      "19\t112277\t0.3\t1\t105925 6352\n",
      "20\t105306\t0.3\t1\t99944 5362\n",
      "21\t93914\t0.3\t1\t89017 4897\n",
      "22\t106559\t0.3\t1\t101114 5445\n",
      "23\t110793\t0.3\t1\t104912 5881\n",
      "24\t108885\t0.3\t1\t102936 5949\n",
      "25\t124131\t0.3\t1\t117742 6389\n",
      "26\t104655\t0.3\t1\t98869 5786\n",
      "27\t118794\t0.3\t1\t111576 7218\n",
      "28\t109018\t0.3\t1\t104356 4662\n",
      "29\t118362\t0.3\t1\t112044 6318\n",
      "30\t117104\t0.3\t1\t111822 5282\n",
      "31\t127211\t0.3\t1\t120779 6432\n",
      "32\t116001\t0.3\t1\t110611 5390\n",
      "33\t128546\t0.3\t1\t121947 6599\n",
      "34\t164203\t0.3\t1\t155704 8499\n",
      "35\t96279\t0.3\t1\t92057 4222\n",
      "36\t133909\t0.3\t1\t126861 7048\n",
      "37\t140147\t0.3\t1\t132941 7206\n",
      "38\t132076\t0.3\t1\t126615 5461\n",
      "39\t129476\t0.3\t1\t124081 5395\n",
      "40\t145043\t0.3\t1\t137854 7189\n",
      "41\t140152\t0.3\t1\t133933 6219\n",
      "42\t149889\t0.3\t1\t143207 6682\n",
      "43\t127335\t0.3\t1\t121774 5561\n",
      "44\t151174\t0.3\t1\t144187 6987\n",
      "45\t157069\t0.3\t1\t149770 7299\n",
      "46\t127123\t0.3\t1\t121787 5336\n",
      "47\t151889\t0.3\t1\t145363 6526\n",
      "48\t148136\t0.3\t1\t142156 5980\n",
      "49\t155657\t0.3\t1\t148850 6807\n",
      "50\t151455\t0.3\t1\t144647 6808\n",
      "51\t188717\t0.3\t1\t180540 8177\n",
      "52\t134588\t0.3\t1\t129312 5276\n",
      "53\t165157\t0.3\t1\t158982 6175\n",
      "54\t125197\t0.3\t1\t120399 4798\n",
      "55\t161733\t0.3\t1\t155845 5888\n",
      "56\t166592\t0.3\t1\t159894 6698\n",
      "57\t159442\t0.3\t1\t153722 5720\n",
      "58\t155066\t0.3\t1\t149532 5534\n",
      "59\t156693\t0.3\t1\t150794 5899\n",
      "60\t162642\t0.3\t1\t156571 6071\n",
      "61\t166749\t0.3\t1\t160204 6545\n",
      "62\t183711\t0.3\t1\t176590 7121\n",
      "63\t247983\t0.3\t1\t239379 8604\n",
      "64\t207871\t0.3\t1\t201612 6259\n",
      "65\t131895\t0.3\t1\t127166 4729\n",
      "66\t143535\t0.3\t1\t138546 4989\n",
      "67\t143350\t0.3\t1\t137946 5404\n",
      "68\t144808\t0.3\t1\t139662 5146\n",
      "69\t143872\t0.3\t1\t138552 5320\n",
      "70\t149770\t0.3\t1\t144363 5407\n",
      "71\t159740\t0.3\t1\t154143 5597\n",
      "72\t151950\t0.3\t1\t146385 5565\n",
      "73\t160797\t0.3\t1\t155248 5549\n",
      "74\t148028\t0.3\t1\t142917 5111\n",
      "75\t141398\t0.3\t1\t136490 4908\n",
      "76\t138432\t0.3\t1\t133692 4740\n",
      "77\t138578\t0.3\t1\t133680 4898\n",
      "78\t133953\t0.3\t1\t129214 4739\n",
      "79\t129618\t0.3\t1\t125018 4600\n",
      "80\t128335\t0.3\t1\t124016 4319\n",
      "81\t126134\t0.3\t1\t121892 4242\n",
      "82\t121190\t0.3\t1\t117136 4054\n",
      "83\t117273\t0.3\t1\t113373 3900\n",
      "84\t113306\t0.3\t1\t109397 3909\n",
      "85\t112548\t0.3\t1\t108604 3944\n",
      "86\t113235\t0.3\t1\t109296 3939\n",
      "87\t114063\t0.3\t1\t110237 3826\n",
      "88\t110699\t0.3\t1\t106931 3768\n",
      "89\t99019\t0.3\t1\t95692 3327\n",
      "90\t94514\t0.3\t1\t91234 3280\n",
      "91\t89156\t0.3\t1\t86055 3101\n",
      "92\t83513\t0.3\t1\t80693 2820\n",
      "93\t80899\t0.3\t1\t78065 2834\n",
      "94\t78488\t0.3\t1\t75830 2658\n",
      "95\t74185\t0.3\t1\t71733 2452\n",
      "96\t70096\t0.3\t1\t67821 2275\n",
      "97\t63309\t0.3\t1\t61201 2108\n",
      "98\t59315\t0.3\t1\t57291 2024\n",
      "99\t56316\t0.3\t1\t54360 1956\n",
      "100\t50955\t0.3\t1\t49217 1738\n",
      "101\t47839\t0.3\t1\t46277 1562\n",
      "102\t45011\t0.3\t1\t43553 1458\n",
      "103\t42598\t0.3\t1\t41263 1335\n",
      "104\t41495\t0.3\t1\t40193 1302\n",
      "105\t36516\t0.3\t1\t35305 1211\n",
      "106\t32285\t0.3\t1\t31286 999\n",
      "107\t28907\t0.3\t1\t27939 968\n",
      "108\t26521\t0.3\t1\t25551 970\n",
      "109\t23377\t0.3\t1\t22637 740\n",
      "110\t20921\t0.3\t1\t20208 713\n",
      "111\t19648\t0.3\t1\t19007 641\n",
      "112\t16880\t0.3\t1\t16303 577\n",
      "113\t14733\t0.3\t1\t14236 497\n",
      "114\t12634\t0.3\t1\t12219 415\n",
      "115\t10616\t0.3\t1\t10211 405\n",
      "116\t8792\t0.3\t1\t8474 318\n",
      "117\t7302\t0.3\t1\t7031 271\n",
      "118\t6324\t0.3\t1\t6071 253\n",
      "119\t5278\t0.3\t1\t5066 212\n",
      "120\t4245\t0.3\t1\t4065 180\n",
      "121\t3766\t0.3\t1\t3593 173\n",
      "122\t3122\t0.3\t1\t2986 136\n",
      "123\t2579\t0.3\t1\t2463 116\n",
      "124\t2054\t0.3\t1\t1977 77\n",
      "125\t1702\t0.3\t1\t1630 72\n",
      "126\t1444\t0.3\t1\t1379 65\n",
      "127\t1157\t0.3\t1\t1113 44\n",
      "128\t945\t0.3\t1\t906 39\n",
      "129\t732\t0.3\t1\t690 42\n",
      "130\t636\t0.3\t1\t596 40\n",
      "131\t486\t0.3\t1\t462 24\n",
      "132\t421\t0.3\t1\t393 28\n",
      "133\t391\t0.3\t1\t380 11\n",
      "134\t460\t0.3\t1\t437 23\n",
      "135\t244\t0.3\t1\t230 14\n",
      "136\t145\t0.3\t1\t134 11\n",
      "137\t97\t0.3\t1\t92 5\n",
      "138\t61\t0.3\t1\t60 1\n",
      "139\t51\t0.3\t1\t44 7\n",
      "140\t41\t0.3\t1\t36 5\n",
      "141\t18\t0.3\t1\t15 3\n",
      "142\t19\t0.3\t1\t18 1\n",
      "143\t6\t0.3\t1\t6\n",
      "144\t18\t0.3\t1\t16 2\n",
      "145\t49\t0.3\t1\t42 7\n",
      "146\t32\t0.3\t1\t31 1\n",
      "147\t61\t0.3\t1\t61\n",
      "148\t55\t0.3\t1\t53 2\n",
      "149\t39\t0.3\t1\t37 2\n",
      "150\t364\t0.3\t1\t342 22\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17734461 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t21505 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 682.27 s (39 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,682,252\n",
      "Reads with adapters:                14,725,601 (83.3%)\n",
      "Reads written (passing filters):    17,682,252 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,652,337,800 bp\n",
      "Quality-trimmed:               5,252,718 bp (0.2%)\n",
      "Total written (filtered):  1,953,712,678 bp (73.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14725601 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 12.9%\n",
      "  G: 21.9%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1730837\t4420563.0\t0\t1730837\n",
      "2\t274288\t1105140.8\t0\t274288\n",
      "3\t164540\t276285.2\t0\t164540\n",
      "4\t98245\t69071.3\t0\t98245\n",
      "5\t85133\t17267.8\t0\t85133\n",
      "6\t82977\t4317.0\t0\t82977\n",
      "7\t68628\t1079.2\t0\t68628\n",
      "8\t92219\t269.8\t0\t92219\n",
      "9\t82148\t67.5\t0\t81058 1090\n",
      "10\t81207\t16.9\t1\t75933 5274\n",
      "11\t91764\t4.2\t1\t83517 8247\n",
      "12\t88451\t1.1\t1\t81680 6771\n",
      "13\t90852\t0.3\t1\t83635 7217\n",
      "14\t92006\t0.3\t1\t84006 8000\n",
      "15\t95802\t0.3\t1\t88019 7783\n",
      "16\t94793\t0.3\t1\t86152 8641\n",
      "17\t117965\t0.3\t1\t106745 11220\n",
      "18\t99003\t0.3\t1\t91800 7203\n",
      "19\t85234\t0.3\t1\t78539 6695\n",
      "20\t103603\t0.3\t1\t95615 7988\n",
      "21\t115441\t0.3\t1\t106021 9420\n",
      "22\t96644\t0.3\t1\t89824 6820\n",
      "23\t112899\t0.3\t1\t103928 8971\n",
      "24\t106903\t0.3\t1\t96909 9994\n",
      "25\t114269\t0.3\t1\t105193 9076\n",
      "26\t105406\t0.3\t1\t97809 7597\n",
      "27\t119549\t0.3\t1\t109420 10129\n",
      "28\t117559\t0.3\t1\t108344 9215\n",
      "29\t126569\t0.3\t1\t115202 11367\n",
      "30\t135359\t0.3\t1\t126047 9312\n",
      "31\t102444\t0.3\t1\t94598 7846\n",
      "32\t135060\t0.3\t1\t124361 10699\n",
      "33\t113717\t0.3\t1\t104773 8944\n",
      "34\t130221\t0.3\t1\t118947 11274\n",
      "35\t139590\t0.3\t1\t127417 12173\n",
      "36\t127748\t0.3\t1\t118449 9299\n",
      "37\t156612\t0.3\t1\t144323 12289\n",
      "38\t124619\t0.3\t1\t114023 10596\n",
      "39\t136370\t0.3\t1\t126274 10096\n",
      "40\t143456\t0.3\t1\t132041 11415\n",
      "41\t145337\t0.3\t1\t132898 12439\n",
      "42\t139531\t0.3\t1\t126917 12614\n",
      "43\t211559\t0.3\t1\t197954 13605\n",
      "44\t73750\t0.3\t1\t66328 7422\n",
      "45\t151399\t0.3\t1\t140473 10926\n",
      "46\t138024\t0.3\t1\t127568 10456\n",
      "47\t150595\t0.3\t1\t139553 11042\n",
      "48\t147780\t0.3\t1\t136652 11128\n",
      "49\t162871\t0.3\t1\t149899 12972\n",
      "50\t141270\t0.3\t1\t131458 9812\n",
      "51\t154346\t0.3\t1\t142859 11487\n",
      "52\t160748\t0.3\t1\t148256 12492\n",
      "53\t167551\t0.3\t1\t155619 11932\n",
      "54\t139669\t0.3\t1\t129569 10100\n",
      "55\t162273\t0.3\t1\t150174 12099\n",
      "56\t200304\t0.3\t1\t186484 13820\n",
      "57\t127569\t0.3\t1\t117186 10383\n",
      "58\t167466\t0.3\t1\t156693 10773\n",
      "59\t150014\t0.3\t1\t138880 11134\n",
      "60\t160710\t0.3\t1\t148928 11782\n",
      "61\t158834\t0.3\t1\t147715 11119\n",
      "62\t178143\t0.3\t1\t165938 12205\n",
      "63\t141715\t0.3\t1\t130934 10781\n",
      "64\t158746\t0.3\t1\t148093 10653\n",
      "65\t176936\t0.3\t1\t164216 12720\n",
      "66\t182618\t0.3\t1\t169484 13134\n",
      "67\t198670\t0.3\t1\t182225 16445\n",
      "68\t365118\t0.3\t1\t345570 19548\n",
      "69\t68264\t0.3\t1\t62074 6190\n",
      "70\t29092\t0.3\t1\t24265 4827\n",
      "71\t103442\t0.3\t1\t96474 6968\n",
      "72\t142212\t0.3\t1\t133025 9187\n",
      "73\t149684\t0.3\t1\t139925 9759\n",
      "74\t148896\t0.3\t1\t138764 10132\n",
      "75\t148109\t0.3\t1\t137782 10327\n",
      "76\t146530\t0.3\t1\t136141 10389\n",
      "77\t146539\t0.3\t1\t136490 10049\n",
      "78\t139703\t0.3\t1\t129860 9843\n",
      "79\t137432\t0.3\t1\t127635 9797\n",
      "80\t135397\t0.3\t1\t125660 9737\n",
      "81\t133287\t0.3\t1\t123761 9526\n",
      "82\t129169\t0.3\t1\t119849 9320\n",
      "83\t125954\t0.3\t1\t117089 8865\n",
      "84\t121677\t0.3\t1\t112941 8736\n",
      "85\t117830\t0.3\t1\t109511 8319\n",
      "86\t114988\t0.3\t1\t106799 8189\n",
      "87\t110799\t0.3\t1\t103288 7511\n",
      "88\t109823\t0.3\t1\t102149 7674\n",
      "89\t103812\t0.3\t1\t96531 7281\n",
      "90\t101371\t0.3\t1\t94159 7212\n",
      "91\t96555\t0.3\t1\t89763 6792\n",
      "92\t92588\t0.3\t1\t86028 6560\n",
      "93\t86787\t0.3\t1\t80806 5981\n",
      "94\t84568\t0.3\t1\t78482 6086\n",
      "95\t78600\t0.3\t1\t72903 5697\n",
      "96\t74361\t0.3\t1\t69003 5358\n",
      "97\t69748\t0.3\t1\t64791 4957\n",
      "98\t65245\t0.3\t1\t60478 4767\n",
      "99\t62580\t0.3\t1\t57800 4780\n",
      "100\t56847\t0.3\t1\t52715 4132\n",
      "101\t53263\t0.3\t1\t49277 3986\n",
      "102\t48648\t0.3\t1\t45084 3564\n",
      "103\t45338\t0.3\t1\t42014 3324\n",
      "104\t41917\t0.3\t1\t38870 3047\n",
      "105\t37301\t0.3\t1\t34586 2715\n",
      "106\t33679\t0.3\t1\t31071 2608\n",
      "107\t30205\t0.3\t1\t27947 2258\n",
      "108\t27998\t0.3\t1\t25811 2187\n",
      "109\t24717\t0.3\t1\t22799 1918\n",
      "110\t22343\t0.3\t1\t20572 1771\n",
      "111\t20279\t0.3\t1\t18723 1556\n",
      "112\t17492\t0.3\t1\t16037 1455\n",
      "113\t14993\t0.3\t1\t13741 1252\n",
      "114\t13102\t0.3\t1\t12018 1084\n",
      "115\t11167\t0.3\t1\t10176 991\n",
      "116\t9702\t0.3\t1\t8851 851\n",
      "117\t8008\t0.3\t1\t7282 726\n",
      "118\t6784\t0.3\t1\t6194 590\n",
      "119\t5714\t0.3\t1\t5190 524\n",
      "120\t4836\t0.3\t1\t4363 473\n",
      "121\t3994\t0.3\t1\t3606 388\n",
      "122\t3193\t0.3\t1\t2889 304\n",
      "123\t2794\t0.3\t1\t2493 301\n",
      "124\t2161\t0.3\t1\t1934 227\n",
      "125\t1832\t0.3\t1\t1606 226\n",
      "126\t1531\t0.3\t1\t1356 175\n",
      "127\t1197\t0.3\t1\t1054 143\n",
      "128\t960\t0.3\t1\t854 106\n",
      "129\t853\t0.3\t1\t753 100\n",
      "130\t591\t0.3\t1\t514 77\n",
      "131\t531\t0.3\t1\t470 61\n",
      "132\t430\t0.3\t1\t380 50\n",
      "133\t462\t0.3\t1\t411 51\n",
      "134\t470\t0.3\t1\t418 52\n",
      "135\t263\t0.3\t1\t225 38\n",
      "136\t146\t0.3\t1\t127 19\n",
      "137\t106\t0.3\t1\t90 16\n",
      "138\t84\t0.3\t1\t66 18\n",
      "139\t59\t0.3\t1\t48 11\n",
      "140\t48\t0.3\t1\t41 7\n",
      "141\t28\t0.3\t1\t20 8\n",
      "142\t22\t0.3\t1\t17 5\n",
      "143\t22\t0.3\t1\t15 7\n",
      "144\t21\t0.3\t1\t19 2\n",
      "145\t41\t0.3\t1\t38 3\n",
      "146\t29\t0.3\t1\t26 3\n",
      "147\t81\t0.3\t1\t70 11\n",
      "148\t47\t0.3\t1\t39 8\n",
      "149\t50\t0.3\t1\t42 8\n",
      "150\t504\t0.3\t1\t392 112\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17682252 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3479 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_2_S2_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 713.19 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,682,252\n",
      "Reads with adapters:                14,653,341 (82.9%)\n",
      "Reads written (passing filters):    17,682,252 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,652,337,800 bp\n",
      "Quality-trimmed:              13,059,983 bp (0.5%)\n",
      "Total written (filtered):  1,958,600,443 bp (73.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14653341 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.0%\n",
      "  G: 24.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1817135\t4420563.0\t0\t1817135\n",
      "2\t279096\t1105140.8\t0\t279096\n",
      "3\t175846\t276285.2\t0\t175846\n",
      "4\t97770\t69071.3\t0\t97770\n",
      "5\t91322\t17267.8\t0\t91322\n",
      "6\t82979\t4317.0\t0\t82979\n",
      "7\t65452\t1079.2\t0\t65452\n",
      "8\t95994\t269.8\t0\t95994\n",
      "9\t76273\t67.5\t0\t76051 222\n",
      "10\t81686\t16.9\t1\t76699 4987\n",
      "11\t88683\t4.2\t1\t82075 6608\n",
      "12\t88254\t1.1\t1\t81734 6520\n",
      "13\t102186\t0.3\t1\t94885 7301\n",
      "14\t94341\t0.3\t1\t86650 7691\n",
      "15\t77377\t0.3\t1\t72050 5327\n",
      "16\t111829\t0.3\t1\t103609 8220\n",
      "17\t95099\t0.3\t1\t87974 7125\n",
      "18\t85068\t0.3\t1\t79390 5678\n",
      "19\t110655\t0.3\t1\t102928 7727\n",
      "20\t104031\t0.3\t1\t97040 6991\n",
      "21\t92000\t0.3\t1\t85864 6136\n",
      "22\t104100\t0.3\t1\t97146 6954\n",
      "23\t109645\t0.3\t1\t102128 7517\n",
      "24\t108307\t0.3\t1\t100742 7565\n",
      "25\t123264\t0.3\t1\t115703 7561\n",
      "26\t104401\t0.3\t1\t96855 7546\n",
      "27\t119574\t0.3\t1\t110821 8753\n",
      "28\t105530\t0.3\t1\t99453 6077\n",
      "29\t120646\t0.3\t1\t112687 7959\n",
      "30\t114291\t0.3\t1\t107348 6943\n",
      "31\t129583\t0.3\t1\t121518 8065\n",
      "32\t112118\t0.3\t1\t105486 6632\n",
      "33\t129638\t0.3\t1\t121053 8585\n",
      "34\t165788\t0.3\t1\t155262 10526\n",
      "35\t90769\t0.3\t1\t85358 5411\n",
      "36\t134484\t0.3\t1\t125338 9146\n",
      "37\t141040\t0.3\t1\t132135 8905\n",
      "38\t130890\t0.3\t1\t123508 7382\n",
      "39\t125992\t0.3\t1\t119419 6573\n",
      "40\t143114\t0.3\t1\t134816 8298\n",
      "41\t141161\t0.3\t1\t133004 8157\n",
      "42\t150560\t0.3\t1\t142032 8528\n",
      "43\t126059\t0.3\t1\t119274 6785\n",
      "44\t153119\t0.3\t1\t144163 8956\n",
      "45\t156350\t0.3\t1\t146944 9406\n",
      "46\t124535\t0.3\t1\t117729 6806\n",
      "47\t151106\t0.3\t1\t143277 7829\n",
      "48\t145709\t0.3\t1\t138077 7632\n",
      "49\t153645\t0.3\t1\t144717 8928\n",
      "50\t152443\t0.3\t1\t143810 8633\n",
      "51\t186444\t0.3\t1\t176105 10339\n",
      "52\t135339\t0.3\t1\t128275 7064\n",
      "53\t167309\t0.3\t1\t158978 8331\n",
      "54\t122506\t0.3\t1\t116488 6018\n",
      "55\t159881\t0.3\t1\t152187 7694\n",
      "56\t166549\t0.3\t1\t157620 8929\n",
      "57\t159405\t0.3\t1\t152142 7263\n",
      "58\t154742\t0.3\t1\t147569 7173\n",
      "59\t154815\t0.3\t1\t147039 7776\n",
      "60\t162353\t0.3\t1\t154259 8094\n",
      "61\t164751\t0.3\t1\t156984 7767\n",
      "62\t181848\t0.3\t1\t172769 9079\n",
      "63\t243575\t0.3\t1\t232900 10675\n",
      "64\t204233\t0.3\t1\t196775 7458\n",
      "65\t131464\t0.3\t1\t125558 5906\n",
      "66\t143870\t0.3\t1\t137254 6616\n",
      "67\t143305\t0.3\t1\t136348 6957\n",
      "68\t146020\t0.3\t1\t139340 6680\n",
      "69\t145720\t0.3\t1\t138556 7164\n",
      "70\t152388\t0.3\t1\t145229 7159\n",
      "71\t160483\t0.3\t1\t152923 7560\n",
      "72\t153910\t0.3\t1\t146668 7242\n",
      "73\t161080\t0.3\t1\t153734 7346\n",
      "74\t148742\t0.3\t1\t141577 7165\n",
      "75\t142283\t0.3\t1\t135459 6824\n",
      "76\t139212\t0.3\t1\t132662 6550\n",
      "77\t138672\t0.3\t1\t132174 6498\n",
      "78\t134120\t0.3\t1\t127702 6418\n",
      "79\t128493\t0.3\t1\t122569 5924\n",
      "80\t128817\t0.3\t1\t123042 5775\n",
      "81\t126667\t0.3\t1\t120859 5808\n",
      "82\t120873\t0.3\t1\t115386 5487\n",
      "83\t118191\t0.3\t1\t112948 5243\n",
      "84\t114338\t0.3\t1\t109255 5083\n",
      "85\t112679\t0.3\t1\t107691 4988\n",
      "86\t112861\t0.3\t1\t107936 4925\n",
      "87\t113253\t0.3\t1\t108504 4749\n",
      "88\t112053\t0.3\t1\t107404 4649\n",
      "89\t99896\t0.3\t1\t95509 4387\n",
      "90\t94224\t0.3\t1\t90152 4072\n",
      "91\t88708\t0.3\t1\t84909 3799\n",
      "92\t84091\t0.3\t1\t80464 3627\n",
      "93\t80972\t0.3\t1\t77474 3498\n",
      "94\t78965\t0.3\t1\t75719 3246\n",
      "95\t74676\t0.3\t1\t71434 3242\n",
      "96\t70973\t0.3\t1\t67917 3056\n",
      "97\t64791\t0.3\t1\t62029 2762\n",
      "98\t59852\t0.3\t1\t57281 2571\n",
      "99\t56272\t0.3\t1\t53840 2432\n",
      "100\t50916\t0.3\t1\t48630 2286\n",
      "101\t48192\t0.3\t1\t46113 2079\n",
      "102\t45225\t0.3\t1\t43338 1887\n",
      "103\t42981\t0.3\t1\t41068 1913\n",
      "104\t41627\t0.3\t1\t39818 1809\n",
      "105\t36524\t0.3\t1\t34997 1527\n",
      "106\t32960\t0.3\t1\t31559 1401\n",
      "107\t28968\t0.3\t1\t27743 1225\n",
      "108\t25878\t0.3\t1\t24765 1113\n",
      "109\t23275\t0.3\t1\t22256 1019\n",
      "110\t21045\t0.3\t1\t20178 867\n",
      "111\t19919\t0.3\t1\t19038 881\n",
      "112\t16987\t0.3\t1\t16274 713\n",
      "113\t14633\t0.3\t1\t14022 611\n",
      "114\t12693\t0.3\t1\t12155 538\n",
      "115\t10546\t0.3\t1\t10065 481\n",
      "116\t8985\t0.3\t1\t8625 360\n",
      "117\t7253\t0.3\t1\t6908 345\n",
      "118\t6254\t0.3\t1\t5971 283\n",
      "119\t5291\t0.3\t1\t5043 248\n",
      "120\t4411\t0.3\t1\t4200 211\n",
      "121\t3756\t0.3\t1\t3560 196\n",
      "122\t3023\t0.3\t1\t2867 156\n",
      "123\t2649\t0.3\t1\t2500 149\n",
      "124\t2061\t0.3\t1\t1951 110\n",
      "125\t1723\t0.3\t1\t1623 100\n",
      "126\t1447\t0.3\t1\t1342 105\n",
      "127\t1137\t0.3\t1\t1082 55\n",
      "128\t888\t0.3\t1\t830 58\n",
      "129\t801\t0.3\t1\t760 41\n",
      "130\t520\t0.3\t1\t487 33\n",
      "131\t493\t0.3\t1\t458 35\n",
      "132\t407\t0.3\t1\t377 30\n",
      "133\t444\t0.3\t1\t414 30\n",
      "134\t445\t0.3\t1\t433 12\n",
      "135\t251\t0.3\t1\t241 10\n",
      "136\t140\t0.3\t1\t129 11\n",
      "137\t98\t0.3\t1\t90 8\n",
      "138\t74\t0.3\t1\t66 8\n",
      "139\t51\t0.3\t1\t50 1\n",
      "140\t46\t0.3\t1\t45 1\n",
      "141\t23\t0.3\t1\t22 1\n",
      "142\t21\t0.3\t1\t19 2\n",
      "143\t18\t0.3\t1\t16 2\n",
      "144\t19\t0.3\t1\t19\n",
      "145\t44\t0.3\t1\t42 2\n",
      "146\t27\t0.3\t1\t26 1\n",
      "147\t73\t0.3\t1\t69 4\n",
      "148\t42\t0.3\t1\t38 4\n",
      "149\t58\t0.3\t1\t55 3\n",
      "150\t398\t0.3\t1\t372 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17682252 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t19996 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 501.97 s (37 us/read; 1.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,688,650\n",
      "Reads with adapters:                11,692,195 (85.4%)\n",
      "Reads written (passing filters):    13,688,650 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,053,297,500 bp\n",
      "Quality-trimmed:               4,119,462 bp (0.2%)\n",
      "Total written (filtered):  1,465,023,732 bp (71.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11692195 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.2%\n",
      "  C: 13.5%\n",
      "  G: 21.8%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1153280\t3422162.5\t0\t1153280\n",
      "2\t188115\t855540.6\t0\t188115\n",
      "3\t111422\t213885.2\t0\t111422\n",
      "4\t69723\t53471.3\t0\t69723\n",
      "5\t62088\t13367.8\t0\t62088\n",
      "6\t60662\t3342.0\t0\t60662\n",
      "7\t51512\t835.5\t0\t51512\n",
      "8\t64264\t208.9\t0\t64264\n",
      "9\t63671\t52.2\t0\t62836 835\n",
      "10\t60542\t13.1\t1\t57278 3264\n",
      "11\t67613\t3.3\t1\t62345 5268\n",
      "12\t65175\t0.8\t1\t60838 4337\n",
      "13\t67473\t0.2\t1\t62952 4521\n",
      "14\t68261\t0.2\t1\t63487 4774\n",
      "15\t72178\t0.2\t1\t67286 4892\n",
      "16\t70299\t0.2\t1\t65086 5213\n",
      "17\t86060\t0.2\t1\t78938 7122\n",
      "18\t74650\t0.2\t1\t70063 4587\n",
      "19\t65040\t0.2\t1\t60840 4200\n",
      "20\t78366\t0.2\t1\t73149 5217\n",
      "21\t84846\t0.2\t1\t78850 5996\n",
      "22\t74487\t0.2\t1\t70107 4380\n",
      "23\t86083\t0.2\t1\t80316 5767\n",
      "24\t80721\t0.2\t1\t73972 6749\n",
      "25\t87753\t0.2\t1\t81677 6076\n",
      "26\t80514\t0.2\t1\t75647 4867\n",
      "27\t90438\t0.2\t1\t83662 6776\n",
      "28\t90174\t0.2\t1\t83716 6458\n",
      "29\t93389\t0.2\t1\t85784 7605\n",
      "30\t100786\t0.2\t1\t94960 5826\n",
      "31\t81881\t0.2\t1\t76679 5202\n",
      "32\t98375\t0.2\t1\t91951 6424\n",
      "33\t93955\t0.2\t1\t87212 6743\n",
      "34\t99759\t0.2\t1\t92551 7208\n",
      "35\t105302\t0.2\t1\t98208 7094\n",
      "36\t99885\t0.2\t1\t93387 6498\n",
      "37\t106334\t0.2\t1\t98270 8064\n",
      "38\t113806\t0.2\t1\t106308 7498\n",
      "39\t98325\t0.2\t1\t92047 6278\n",
      "40\t111911\t0.2\t1\t104331 7580\n",
      "41\t123334\t0.2\t1\t114370 8964\n",
      "42\t99813\t0.2\t1\t92381 7432\n",
      "43\t174315\t0.2\t1\t165200 9115\n",
      "44\t51463\t0.2\t1\t46797 4666\n",
      "45\t117571\t0.2\t1\t110330 7241\n",
      "46\t111696\t0.2\t1\t104449 7247\n",
      "47\t119263\t0.2\t1\t111850 7413\n",
      "48\t117185\t0.2\t1\t109504 7681\n",
      "49\t129249\t0.2\t1\t119912 9337\n",
      "50\t113265\t0.2\t1\t106460 6805\n",
      "51\t123578\t0.2\t1\t115577 8001\n",
      "52\t126860\t0.2\t1\t117703 9157\n",
      "53\t136778\t0.2\t1\t128183 8595\n",
      "54\t111542\t0.2\t1\t104628 6914\n",
      "55\t130509\t0.2\t1\t122033 8476\n",
      "56\t160806\t0.2\t1\t151398 9408\n",
      "57\t105238\t0.2\t1\t97627 7611\n",
      "58\t136703\t0.2\t1\t129347 7356\n",
      "59\t118654\t0.2\t1\t110900 7754\n",
      "60\t133349\t0.2\t1\t124657 8692\n",
      "61\t131151\t0.2\t1\t123141 8010\n",
      "62\t146081\t0.2\t1\t137646 8435\n",
      "63\t116527\t0.2\t1\t108569 7958\n",
      "64\t131388\t0.2\t1\t123568 7820\n",
      "65\t146211\t0.2\t1\t137343 8868\n",
      "66\t153623\t0.2\t1\t144098 9525\n",
      "67\t163622\t0.2\t1\t150969 12653\n",
      "68\t316517\t0.2\t1\t301519 14998\n",
      "69\t50118\t0.2\t1\t45752 4366\n",
      "70\t19043\t0.2\t1\t15441 3602\n",
      "71\t86203\t0.2\t1\t80944 5259\n",
      "72\t120080\t0.2\t1\t113140 6940\n",
      "73\t127657\t0.2\t1\t119892 7765\n",
      "74\t126886\t0.2\t1\t119284 7602\n",
      "75\t126438\t0.2\t1\t118691 7747\n",
      "76\t124650\t0.2\t1\t117103 7547\n",
      "77\t126032\t0.2\t1\t118426 7606\n",
      "78\t119720\t0.2\t1\t112423 7297\n",
      "79\t118994\t0.2\t1\t111880 7114\n",
      "80\t116560\t0.2\t1\t109502 7058\n",
      "81\t114889\t0.2\t1\t107938 6951\n",
      "82\t112663\t0.2\t1\t105830 6833\n",
      "83\t109839\t0.2\t1\t103073 6766\n",
      "84\t105779\t0.2\t1\t99490 6289\n",
      "85\t104543\t0.2\t1\t98380 6163\n",
      "86\t100937\t0.2\t1\t94813 6124\n",
      "87\t97367\t0.2\t1\t91433 5934\n",
      "88\t96957\t0.2\t1\t90929 6028\n",
      "89\t91739\t0.2\t1\t86072 5667\n",
      "90\t91064\t0.2\t1\t85580 5484\n",
      "91\t86291\t0.2\t1\t80995 5296\n",
      "92\t83034\t0.2\t1\t77959 5075\n",
      "93\t78558\t0.2\t1\t73687 4871\n",
      "94\t75965\t0.2\t1\t71248 4717\n",
      "95\t70608\t0.2\t1\t66214 4394\n",
      "96\t66613\t0.2\t1\t62491 4122\n",
      "97\t63167\t0.2\t1\t59156 4011\n",
      "98\t59278\t0.2\t1\t55535 3743\n",
      "99\t58315\t0.2\t1\t54380 3935\n",
      "100\t52512\t0.2\t1\t49178 3334\n",
      "101\t49162\t0.2\t1\t45965 3197\n",
      "102\t45067\t0.2\t1\t42105 2962\n",
      "103\t41984\t0.2\t1\t39185 2799\n",
      "104\t39240\t0.2\t1\t36628 2612\n",
      "105\t34957\t0.2\t1\t32577 2380\n",
      "106\t31819\t0.2\t1\t29724 2095\n",
      "107\t29159\t0.2\t1\t27051 2108\n",
      "108\t27128\t0.2\t1\t25184 1944\n",
      "109\t24322\t0.2\t1\t22614 1708\n",
      "110\t21994\t0.2\t1\t20354 1640\n",
      "111\t19565\t0.2\t1\t18156 1409\n",
      "112\t17167\t0.2\t1\t15854 1313\n",
      "113\t14819\t0.2\t1\t13686 1133\n",
      "114\t13381\t0.2\t1\t12379 1002\n",
      "115\t11437\t0.2\t1\t10576 861\n",
      "116\t9792\t0.2\t1\t9003 789\n",
      "117\t8262\t0.2\t1\t7551 711\n",
      "118\t7359\t0.2\t1\t6738 621\n",
      "119\t6022\t0.2\t1\t5480 542\n",
      "120\t5103\t0.2\t1\t4650 453\n",
      "121\t4422\t0.2\t1\t4023 399\n",
      "122\t3603\t0.2\t1\t3261 342\n",
      "123\t3138\t0.2\t1\t2830 308\n",
      "124\t2526\t0.2\t1\t2281 245\n",
      "125\t2064\t0.2\t1\t1886 178\n",
      "126\t1770\t0.2\t1\t1574 196\n",
      "127\t1476\t0.2\t1\t1324 152\n",
      "128\t1074\t0.2\t1\t953 121\n",
      "129\t984\t0.2\t1\t870 114\n",
      "130\t708\t0.2\t1\t638 70\n",
      "131\t577\t0.2\t1\t522 55\n",
      "132\t547\t0.2\t1\t477 70\n",
      "133\t519\t0.2\t1\t468 51\n",
      "134\t567\t0.2\t1\t518 49\n",
      "135\t307\t0.2\t1\t275 32\n",
      "136\t217\t0.2\t1\t195 22\n",
      "137\t172\t0.2\t1\t148 24\n",
      "138\t111\t0.2\t1\t95 16\n",
      "139\t77\t0.2\t1\t67 10\n",
      "140\t52\t0.2\t1\t47 5\n",
      "141\t43\t0.2\t1\t38 5\n",
      "142\t44\t0.2\t1\t33 11\n",
      "143\t50\t0.2\t1\t44 6\n",
      "144\t49\t0.2\t1\t46 3\n",
      "145\t55\t0.2\t1\t44 11\n",
      "146\t89\t0.2\t1\t78 11\n",
      "147\t261\t0.2\t1\t233 28\n",
      "148\t216\t0.2\t1\t199 17\n",
      "149\t67\t0.2\t1\t64 3\n",
      "150\t763\t0.2\t1\t633 130\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13688650 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4825 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 535.60 s (39 us/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,688,650\n",
      "Reads with adapters:                11,592,048 (84.7%)\n",
      "Reads written (passing filters):    13,688,650 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,053,297,500 bp\n",
      "Quality-trimmed:              12,031,923 bp (0.6%)\n",
      "Total written (filtered):  1,469,819,891 bp (71.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11592048 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.3%\n",
      "  G: 24.9%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1214265\t3422162.5\t0\t1214265\n",
      "2\t193972\t855540.6\t0\t193972\n",
      "3\t118499\t213885.2\t0\t118499\n",
      "4\t69716\t53471.3\t0\t69716\n",
      "5\t68874\t13367.8\t0\t68874\n",
      "6\t59538\t3342.0\t0\t59538\n",
      "7\t48194\t835.5\t0\t48194\n",
      "8\t70200\t208.9\t0\t70200\n",
      "9\t54828\t52.2\t0\t54657 171\n",
      "10\t61679\t13.1\t1\t58439 3240\n",
      "11\t64699\t3.3\t1\t60413 4286\n",
      "12\t66050\t0.8\t1\t61625 4425\n",
      "13\t73701\t0.2\t1\t69161 4540\n",
      "14\t72487\t0.2\t1\t67153 5334\n",
      "15\t56503\t0.2\t1\t52652 3851\n",
      "16\t83460\t0.2\t1\t77990 5470\n",
      "17\t68575\t0.2\t1\t63675 4900\n",
      "18\t63646\t0.2\t1\t59572 4074\n",
      "19\t84575\t0.2\t1\t78822 5753\n",
      "20\t78107\t0.2\t1\t73276 4831\n",
      "21\t67477\t0.2\t1\t63200 4277\n",
      "22\t79410\t0.2\t1\t74414 4996\n",
      "23\t81981\t0.2\t1\t76795 5186\n",
      "24\t83330\t0.2\t1\t77735 5595\n",
      "25\t95526\t0.2\t1\t89714 5812\n",
      "26\t78503\t0.2\t1\t73254 5249\n",
      "27\t85319\t0.2\t1\t79062 6257\n",
      "28\t83102\t0.2\t1\t78541 4561\n",
      "29\t88168\t0.2\t1\t82573 5595\n",
      "30\t89538\t0.2\t1\t84298 5240\n",
      "31\t96658\t0.2\t1\t90668 5990\n",
      "32\t88293\t0.2\t1\t83312 4981\n",
      "33\t97915\t0.2\t1\t91746 6169\n",
      "34\t124838\t0.2\t1\t117069 7769\n",
      "35\t74005\t0.2\t1\t69745 4260\n",
      "36\t101096\t0.2\t1\t94692 6404\n",
      "37\t106262\t0.2\t1\t99713 6549\n",
      "38\t101787\t0.2\t1\t96374 5413\n",
      "39\t98728\t0.2\t1\t93495 5233\n",
      "40\t111334\t0.2\t1\t104562 6772\n",
      "41\t108250\t0.2\t1\t102276 5974\n",
      "42\t114250\t0.2\t1\t107889 6361\n",
      "43\t101869\t0.2\t1\t96202 5667\n",
      "44\t119468\t0.2\t1\t112651 6817\n",
      "45\t134184\t0.2\t1\t126701 7483\n",
      "46\t88069\t0.2\t1\t83261 4808\n",
      "47\t117053\t0.2\t1\t110868 6185\n",
      "48\t117366\t0.2\t1\t111327 6039\n",
      "49\t124438\t0.2\t1\t117817 6621\n",
      "50\t117444\t0.2\t1\t111059 6385\n",
      "51\t154247\t0.2\t1\t146044 8203\n",
      "52\t106291\t0.2\t1\t100997 5294\n",
      "53\t129680\t0.2\t1\t123785 5895\n",
      "54\t97816\t0.2\t1\t92887 4929\n",
      "55\t130824\t0.2\t1\t124577 6247\n",
      "56\t135217\t0.2\t1\t128490 6727\n",
      "57\t127931\t0.2\t1\t122024 5907\n",
      "58\t124239\t0.2\t1\t118784 5455\n",
      "59\t127293\t0.2\t1\t121311 5982\n",
      "60\t131981\t0.2\t1\t125772 6209\n",
      "61\t137697\t0.2\t1\t130983 6714\n",
      "62\t152253\t0.2\t1\t144774 7479\n",
      "63\t201898\t0.2\t1\t193113 8785\n",
      "64\t172675\t0.2\t1\t166389 6286\n",
      "65\t108874\t0.2\t1\t104021 4853\n",
      "66\t119610\t0.2\t1\t114446 5164\n",
      "67\t118964\t0.2\t1\t113321 5643\n",
      "68\t119680\t0.2\t1\t114293 5387\n",
      "69\t119889\t0.2\t1\t114293 5596\n",
      "70\t125777\t0.2\t1\t120013 5764\n",
      "71\t132876\t0.2\t1\t127109 5767\n",
      "72\t127787\t0.2\t1\t121904 5883\n",
      "73\t134940\t0.2\t1\t129055 5885\n",
      "74\t125685\t0.2\t1\t120224 5461\n",
      "75\t120939\t0.2\t1\t115502 5437\n",
      "76\t116878\t0.2\t1\t111806 5072\n",
      "77\t119317\t0.2\t1\t114031 5286\n",
      "78\t114912\t0.2\t1\t109837 5075\n",
      "79\t111961\t0.2\t1\t107098 4863\n",
      "80\t111439\t0.2\t1\t106611 4828\n",
      "81\t110252\t0.2\t1\t105525 4727\n",
      "82\t104612\t0.2\t1\t100154 4458\n",
      "83\t103450\t0.2\t1\t99126 4324\n",
      "84\t97944\t0.2\t1\t93515 4429\n",
      "85\t99039\t0.2\t1\t94552 4487\n",
      "86\t98012\t0.2\t1\t93716 4296\n",
      "87\t99549\t0.2\t1\t95300 4249\n",
      "88\t97562\t0.2\t1\t93249 4313\n",
      "89\t87175\t0.2\t1\t83280 3895\n",
      "90\t83950\t0.2\t1\t80287 3663\n",
      "91\t78836\t0.2\t1\t75281 3555\n",
      "92\t74886\t0.2\t1\t71664 3222\n",
      "93\t72687\t0.2\t1\t69632 3055\n",
      "94\t70208\t0.2\t1\t67258 2950\n",
      "95\t66190\t0.2\t1\t63387 2803\n",
      "96\t63977\t0.2\t1\t61304 2673\n",
      "97\t60139\t0.2\t1\t57625 2514\n",
      "98\t58153\t0.2\t1\t55728 2425\n",
      "99\t56999\t0.2\t1\t54663 2336\n",
      "100\t49226\t0.2\t1\t47327 1899\n",
      "101\t45428\t0.2\t1\t43567 1861\n",
      "102\t41140\t0.2\t1\t39441 1699\n",
      "103\t38223\t0.2\t1\t36634 1589\n",
      "104\t36626\t0.2\t1\t35088 1538\n",
      "105\t32982\t0.2\t1\t31676 1306\n",
      "106\t30175\t0.2\t1\t28923 1252\n",
      "107\t27378\t0.2\t1\t26210 1168\n",
      "108\t24859\t0.2\t1\t23806 1053\n",
      "109\t22755\t0.2\t1\t21778 977\n",
      "110\t20637\t0.2\t1\t19770 867\n",
      "111\t19236\t0.2\t1\t18439 797\n",
      "112\t16601\t0.2\t1\t15906 695\n",
      "113\t14600\t0.2\t1\t13964 636\n",
      "114\t12946\t0.2\t1\t12440 506\n",
      "115\t10705\t0.2\t1\t10194 511\n",
      "116\t9020\t0.2\t1\t8602 418\n",
      "117\t7442\t0.2\t1\t7107 335\n",
      "118\t6690\t0.2\t1\t6379 311\n",
      "119\t5516\t0.2\t1\t5293 223\n",
      "120\t4632\t0.2\t1\t4427 205\n",
      "121\t4146\t0.2\t1\t3939 207\n",
      "122\t3352\t0.2\t1\t3190 162\n",
      "123\t2944\t0.2\t1\t2813 131\n",
      "124\t2397\t0.2\t1\t2279 118\n",
      "125\t1984\t0.2\t1\t1898 86\n",
      "126\t1679\t0.2\t1\t1593 86\n",
      "127\t1365\t0.2\t1\t1295 70\n",
      "128\t964\t0.2\t1\t919 45\n",
      "129\t925\t0.2\t1\t886 39\n",
      "130\t667\t0.2\t1\t644 23\n",
      "131\t533\t0.2\t1\t504 29\n",
      "132\t511\t0.2\t1\t485 26\n",
      "133\t492\t0.2\t1\t476 16\n",
      "134\t547\t0.2\t1\t526 21\n",
      "135\t276\t0.2\t1\t266 10\n",
      "136\t213\t0.2\t1\t198 15\n",
      "137\t138\t0.2\t1\t130 8\n",
      "138\t104\t0.2\t1\t100 4\n",
      "139\t66\t0.2\t1\t61 5\n",
      "140\t52\t0.2\t1\t48 4\n",
      "141\t37\t0.2\t1\t34 3\n",
      "142\t40\t0.2\t1\t40\n",
      "143\t44\t0.2\t1\t40 4\n",
      "144\t45\t0.2\t1\t42 3\n",
      "145\t50\t0.2\t1\t48 2\n",
      "146\t86\t0.2\t1\t82 4\n",
      "147\t237\t0.2\t1\t232 5\n",
      "148\t198\t0.2\t1\t192 6\n",
      "149\t62\t0.2\t1\t55 7\n",
      "150\t658\t0.2\t1\t640 18\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13688650 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t23444 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 510.86 s (37 us/read; 1.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,670,957\n",
      "Reads with adapters:                11,663,120 (85.3%)\n",
      "Reads written (passing filters):    13,670,957 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,050,643,550 bp\n",
      "Quality-trimmed:               4,231,959 bp (0.2%)\n",
      "Total written (filtered):  1,462,745,020 bp (71.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11663120 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.2%\n",
      "  C: 13.5%\n",
      "  G: 21.8%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1152864\t3417739.2\t0\t1152864\n",
      "2\t186575\t854434.8\t0\t186575\n",
      "3\t110892\t213608.7\t0\t110892\n",
      "4\t68686\t53402.2\t0\t68686\n",
      "5\t61523\t13350.5\t0\t61523\n",
      "6\t61066\t3337.6\t0\t61066\n",
      "7\t49563\t834.4\t0\t49563\n",
      "8\t63725\t208.6\t0\t63725\n",
      "9\t62710\t52.2\t0\t61945 765\n",
      "10\t59136\t13.0\t1\t55273 3863\n",
      "11\t67741\t3.3\t1\t61750 5991\n",
      "12\t63923\t0.8\t1\t59045 4878\n",
      "13\t66568\t0.2\t1\t61427 5141\n",
      "14\t66808\t0.2\t1\t61285 5523\n",
      "15\t71031\t0.2\t1\t65247 5784\n",
      "16\t70289\t0.2\t1\t63984 6305\n",
      "17\t86030\t0.2\t1\t77755 8275\n",
      "18\t73739\t0.2\t1\t68290 5449\n",
      "19\t63306\t0.2\t1\t58569 4737\n",
      "20\t77355\t0.2\t1\t71402 5953\n",
      "21\t85573\t0.2\t1\t78654 6919\n",
      "22\t72710\t0.2\t1\t67584 5126\n",
      "23\t86414\t0.2\t1\t79515 6899\n",
      "24\t79027\t0.2\t1\t71591 7436\n",
      "25\t86879\t0.2\t1\t79866 7013\n",
      "26\t79473\t0.2\t1\t73820 5653\n",
      "27\t90733\t0.2\t1\t83035 7698\n",
      "28\t90102\t0.2\t1\t83123 6979\n",
      "29\t92865\t0.2\t1\t84237 8628\n",
      "30\t100457\t0.2\t1\t93462 6995\n",
      "31\t79802\t0.2\t1\t73917 5885\n",
      "32\t97719\t0.2\t1\t90137 7582\n",
      "33\t93085\t0.2\t1\t85177 7908\n",
      "34\t99262\t0.2\t1\t90786 8476\n",
      "35\t104947\t0.2\t1\t96795 8152\n",
      "36\t99380\t0.2\t1\t91895 7485\n",
      "37\t106441\t0.2\t1\t96879 9562\n",
      "38\t114379\t0.2\t1\t105498 8881\n",
      "39\t96631\t0.2\t1\t89100 7531\n",
      "40\t112490\t0.2\t1\t103151 9339\n",
      "41\t123125\t0.2\t1\t112471 10654\n",
      "42\t98677\t0.2\t1\t90013 8664\n",
      "43\t172897\t0.2\t1\t162100 10797\n",
      "44\t52704\t0.2\t1\t47226 5478\n",
      "45\t117558\t0.2\t1\t108914 8644\n",
      "46\t110418\t0.2\t1\t102198 8220\n",
      "47\t119359\t0.2\t1\t110565 8794\n",
      "48\t116841\t0.2\t1\t108072 8769\n",
      "49\t129611\t0.2\t1\t119238 10373\n",
      "50\t111895\t0.2\t1\t104242 7653\n",
      "51\t123193\t0.2\t1\t114083 9110\n",
      "52\t127566\t0.2\t1\t117108 10458\n",
      "53\t137342\t0.2\t1\t127459 9883\n",
      "54\t110167\t0.2\t1\t102021 8146\n",
      "55\t130673\t0.2\t1\t120967 9706\n",
      "56\t160643\t0.2\t1\t149587 11056\n",
      "57\t104913\t0.2\t1\t96311 8602\n",
      "58\t136727\t0.2\t1\t128029 8698\n",
      "59\t118611\t0.2\t1\t109706 8905\n",
      "60\t133859\t0.2\t1\t123864 9995\n",
      "61\t131656\t0.2\t1\t122381 9275\n",
      "62\t147218\t0.2\t1\t137094 10124\n",
      "63\t117137\t0.2\t1\t107812 9325\n",
      "64\t130610\t0.2\t1\t121719 8891\n",
      "65\t145884\t0.2\t1\t135248 10636\n",
      "66\t150729\t0.2\t1\t139945 10784\n",
      "67\t163505\t0.2\t1\t149584 13921\n",
      "68\t306709\t0.2\t1\t289681 17028\n",
      "69\t62043\t0.2\t1\t56562 5481\n",
      "70\t23965\t0.2\t1\t19784 4181\n",
      "71\t84291\t0.2\t1\t78568 5723\n",
      "72\t119692\t0.2\t1\t111910 7782\n",
      "73\t127200\t0.2\t1\t118724 8476\n",
      "74\t126943\t0.2\t1\t118057 8886\n",
      "75\t127010\t0.2\t1\t118252 8758\n",
      "76\t125373\t0.2\t1\t116194 9179\n",
      "77\t125254\t0.2\t1\t116388 8866\n",
      "78\t119533\t0.2\t1\t110788 8745\n",
      "79\t118816\t0.2\t1\t110233 8583\n",
      "80\t116241\t0.2\t1\t107681 8560\n",
      "81\t114798\t0.2\t1\t106595 8203\n",
      "82\t112468\t0.2\t1\t104279 8189\n",
      "83\t109667\t0.2\t1\t101888 7779\n",
      "84\t105890\t0.2\t1\t98240 7650\n",
      "85\t103302\t0.2\t1\t95804 7498\n",
      "86\t101064\t0.2\t1\t93722 7342\n",
      "87\t97500\t0.2\t1\t90687 6813\n",
      "88\t96898\t0.2\t1\t89881 7017\n",
      "89\t91450\t0.2\t1\t84968 6482\n",
      "90\t91495\t0.2\t1\t84920 6575\n",
      "91\t86388\t0.2\t1\t80232 6156\n",
      "92\t82291\t0.2\t1\t76556 5735\n",
      "93\t78431\t0.2\t1\t72788 5643\n",
      "94\t75874\t0.2\t1\t70339 5535\n",
      "95\t70751\t0.2\t1\t65675 5076\n",
      "96\t67352\t0.2\t1\t62368 4984\n",
      "97\t63954\t0.2\t1\t59443 4511\n",
      "98\t58850\t0.2\t1\t54439 4411\n",
      "99\t58771\t0.2\t1\t54248 4523\n",
      "100\t52467\t0.2\t1\t48655 3812\n",
      "101\t49569\t0.2\t1\t45811 3758\n",
      "102\t45464\t0.2\t1\t42074 3390\n",
      "103\t41939\t0.2\t1\t38904 3035\n",
      "104\t39055\t0.2\t1\t36189 2866\n",
      "105\t34942\t0.2\t1\t32296 2646\n",
      "106\t31969\t0.2\t1\t29559 2410\n",
      "107\t29286\t0.2\t1\t27065 2221\n",
      "108\t27282\t0.2\t1\t25136 2146\n",
      "109\t24507\t0.2\t1\t22544 1963\n",
      "110\t21811\t0.2\t1\t20137 1674\n",
      "111\t19775\t0.2\t1\t18232 1543\n",
      "112\t17157\t0.2\t1\t15740 1417\n",
      "113\t14815\t0.2\t1\t13543 1272\n",
      "114\t13327\t0.2\t1\t12234 1093\n",
      "115\t11366\t0.2\t1\t10408 958\n",
      "116\t9845\t0.2\t1\t9004 841\n",
      "117\t8405\t0.2\t1\t7663 742\n",
      "118\t7384\t0.2\t1\t6703 681\n",
      "119\t6035\t0.2\t1\t5477 558\n",
      "120\t4910\t0.2\t1\t4438 472\n",
      "121\t4090\t0.2\t1\t3702 388\n",
      "122\t3634\t0.2\t1\t3283 351\n",
      "123\t3044\t0.2\t1\t2741 303\n",
      "124\t2552\t0.2\t1\t2281 271\n",
      "125\t2030\t0.2\t1\t1827 203\n",
      "126\t1757\t0.2\t1\t1559 198\n",
      "127\t1362\t0.2\t1\t1204 158\n",
      "128\t1097\t0.2\t1\t981 116\n",
      "129\t1002\t0.2\t1\t874 128\n",
      "130\t785\t0.2\t1\t685 100\n",
      "131\t638\t0.2\t1\t562 76\n",
      "132\t540\t0.2\t1\t480 60\n",
      "133\t462\t0.2\t1\t415 47\n",
      "134\t556\t0.2\t1\t494 62\n",
      "135\t352\t0.2\t1\t314 38\n",
      "136\t204\t0.2\t1\t176 28\n",
      "137\t149\t0.2\t1\t133 16\n",
      "138\t114\t0.2\t1\t99 15\n",
      "139\t71\t0.2\t1\t62 9\n",
      "140\t64\t0.2\t1\t52 12\n",
      "141\t32\t0.2\t1\t25 7\n",
      "142\t45\t0.2\t1\t42 3\n",
      "143\t51\t0.2\t1\t43 8\n",
      "144\t31\t0.2\t1\t25 6\n",
      "145\t45\t0.2\t1\t31 14\n",
      "146\t90\t0.2\t1\t77 13\n",
      "147\t226\t0.2\t1\t201 25\n",
      "148\t203\t0.2\t1\t185 18\n",
      "149\t84\t0.2\t1\t77 7\n",
      "150\t819\t0.2\t1\t651 168\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13670957 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4822 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_3_S3_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 549.57 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,670,957\n",
      "Reads with adapters:                11,550,197 (84.5%)\n",
      "Reads written (passing filters):    13,670,957 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,050,643,550 bp\n",
      "Quality-trimmed:              11,580,867 bp (0.6%)\n",
      "Total written (filtered):  1,468,475,211 bp (71.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11550197 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.3%\n",
      "  G: 24.9%\n",
      "  T: 34.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1212421\t3417739.2\t0\t1212421\n",
      "2\t191566\t854434.8\t0\t191566\n",
      "3\t121338\t213608.7\t0\t121338\n",
      "4\t68246\t53402.2\t0\t68246\n",
      "5\t66256\t13350.5\t0\t66256\n",
      "6\t60146\t3337.6\t0\t60146\n",
      "7\t46826\t834.4\t0\t46826\n",
      "8\t69191\t208.6\t0\t69191\n",
      "9\t54203\t52.2\t0\t54035 168\n",
      "10\t59001\t13.0\t1\t54514 4487\n",
      "11\t64399\t3.3\t1\t58784 5615\n",
      "12\t63560\t0.8\t1\t57831 5729\n",
      "13\t73714\t0.2\t1\t67412 6302\n",
      "14\t69969\t0.2\t1\t63352 6617\n",
      "15\t54986\t0.2\t1\t50399 4587\n",
      "16\t83002\t0.2\t1\t75869 7133\n",
      "17\t70639\t0.2\t1\t64421 6218\n",
      "18\t61266\t0.2\t1\t56207 5059\n",
      "19\t82751\t0.2\t1\t75847 6904\n",
      "20\t76799\t0.2\t1\t70584 6215\n",
      "21\t67039\t0.2\t1\t61694 5345\n",
      "22\t77648\t0.2\t1\t71294 6354\n",
      "23\t82616\t0.2\t1\t75896 6720\n",
      "24\t82091\t0.2\t1\t75051 7040\n",
      "25\t94761\t0.2\t1\t87866 6895\n",
      "26\t78379\t0.2\t1\t71576 6803\n",
      "27\t86685\t0.2\t1\t79069 7616\n",
      "28\t79766\t0.2\t1\t74088 5678\n",
      "29\t88958\t0.2\t1\t82020 6938\n",
      "30\t87347\t0.2\t1\t80721 6626\n",
      "31\t96436\t0.2\t1\t89116 7320\n",
      "32\t84184\t0.2\t1\t78013 6171\n",
      "33\t99064\t0.2\t1\t91212 7852\n",
      "34\t126286\t0.2\t1\t116784 9502\n",
      "35\t70478\t0.2\t1\t65263 5215\n",
      "36\t102056\t0.2\t1\t93957 8099\n",
      "37\t106427\t0.2\t1\t98476 7951\n",
      "38\t101342\t0.2\t1\t94365 6977\n",
      "39\t95863\t0.2\t1\t89455 6408\n",
      "40\t111287\t0.2\t1\t103069 8218\n",
      "41\t109099\t0.2\t1\t101498 7601\n",
      "42\t114117\t0.2\t1\t106148 7969\n",
      "43\t100632\t0.2\t1\t93666 6966\n",
      "44\t121370\t0.2\t1\t112650 8720\n",
      "45\t132112\t0.2\t1\t122836 9276\n",
      "46\t87481\t0.2\t1\t81280 6201\n",
      "47\t117137\t0.2\t1\t109733 7404\n",
      "48\t116770\t0.2\t1\t109144 7626\n",
      "49\t122930\t0.2\t1\t114503 8427\n",
      "50\t118230\t0.2\t1\t110078 8152\n",
      "51\t151502\t0.2\t1\t141327 10175\n",
      "52\t106392\t0.2\t1\t99321 7071\n",
      "53\t132940\t0.2\t1\t125179 7761\n",
      "54\t95060\t0.2\t1\t89153 5907\n",
      "55\t129755\t0.2\t1\t121823 7932\n",
      "56\t133247\t0.2\t1\t124572 8675\n",
      "57\t128656\t0.2\t1\t121414 7242\n",
      "58\t125177\t0.2\t1\t117960 7217\n",
      "59\t125701\t0.2\t1\t118073 7628\n",
      "60\t133212\t0.2\t1\t125145 8067\n",
      "61\t136943\t0.2\t1\t128912 8031\n",
      "62\t152161\t0.2\t1\t142895 9266\n",
      "63\t197608\t0.2\t1\t187217 10391\n",
      "64\t170164\t0.2\t1\t162758 7406\n",
      "65\t108925\t0.2\t1\t102718 6207\n",
      "66\t119060\t0.2\t1\t112190 6870\n",
      "67\t118638\t0.2\t1\t111538 7100\n",
      "68\t122066\t0.2\t1\t114957 7109\n",
      "69\t121361\t0.2\t1\t113998 7363\n",
      "70\t127330\t0.2\t1\t119753 7577\n",
      "71\t132672\t0.2\t1\t124891 7781\n",
      "72\t129459\t0.2\t1\t121850 7609\n",
      "73\t134714\t0.2\t1\t127048 7666\n",
      "74\t126025\t0.2\t1\t118830 7195\n",
      "75\t121497\t0.2\t1\t114268 7229\n",
      "76\t117925\t0.2\t1\t110905 7020\n",
      "77\t118215\t0.2\t1\t111150 7065\n",
      "78\t114702\t0.2\t1\t108059 6643\n",
      "79\t111977\t0.2\t1\t105512 6465\n",
      "80\t111238\t0.2\t1\t104912 6326\n",
      "81\t110074\t0.2\t1\t103870 6204\n",
      "82\t104747\t0.2\t1\t98712 6035\n",
      "83\t102994\t0.2\t1\t97190 5804\n",
      "84\t97939\t0.2\t1\t92413 5526\n",
      "85\t98027\t0.2\t1\t92612 5415\n",
      "86\t97975\t0.2\t1\t92622 5353\n",
      "87\t99113\t0.2\t1\t93888 5225\n",
      "88\t97377\t0.2\t1\t92204 5173\n",
      "89\t87121\t0.2\t1\t82286 4835\n",
      "90\t84514\t0.2\t1\t79963 4551\n",
      "91\t79415\t0.2\t1\t75190 4225\n",
      "92\t74774\t0.2\t1\t70849 3925\n",
      "93\t72752\t0.2\t1\t68819 3933\n",
      "94\t70292\t0.2\t1\t66493 3799\n",
      "95\t66513\t0.2\t1\t62910 3603\n",
      "96\t64556\t0.2\t1\t61082 3474\n",
      "97\t60915\t0.2\t1\t57800 3115\n",
      "98\t57473\t0.2\t1\t54587 2886\n",
      "99\t57713\t0.2\t1\t54715 2998\n",
      "100\t49481\t0.2\t1\t46774 2707\n",
      "101\t45740\t0.2\t1\t43384 2356\n",
      "102\t41586\t0.2\t1\t39421 2165\n",
      "103\t38173\t0.2\t1\t36099 2074\n",
      "104\t36449\t0.2\t1\t34460 1989\n",
      "105\t32790\t0.2\t1\t31067 1723\n",
      "106\t30381\t0.2\t1\t28796 1585\n",
      "107\t27663\t0.2\t1\t26191 1472\n",
      "108\t25091\t0.2\t1\t23805 1286\n",
      "109\t22776\t0.2\t1\t21667 1109\n",
      "110\t20493\t0.2\t1\t19406 1087\n",
      "111\t19435\t0.2\t1\t18473 962\n",
      "112\t16596\t0.2\t1\t15727 869\n",
      "113\t14532\t0.2\t1\t13752 780\n",
      "114\t12824\t0.2\t1\t12122 702\n",
      "115\t10619\t0.2\t1\t10098 521\n",
      "116\t9188\t0.2\t1\t8686 502\n",
      "117\t7612\t0.2\t1\t7172 440\n",
      "118\t6805\t0.2\t1\t6447 358\n",
      "119\t5613\t0.2\t1\t5287 326\n",
      "120\t4455\t0.2\t1\t4215 240\n",
      "121\t3795\t0.2\t1\t3594 201\n",
      "122\t3385\t0.2\t1\t3227 158\n",
      "123\t2919\t0.2\t1\t2743 176\n",
      "124\t2401\t0.2\t1\t2252 149\n",
      "125\t1949\t0.2\t1\t1858 91\n",
      "126\t1662\t0.2\t1\t1556 106\n",
      "127\t1275\t0.2\t1\t1189 86\n",
      "128\t1017\t0.2\t1\t960 57\n",
      "129\t911\t0.2\t1\t861 50\n",
      "130\t712\t0.2\t1\t666 46\n",
      "131\t583\t0.2\t1\t553 30\n",
      "132\t498\t0.2\t1\t473 25\n",
      "133\t459\t0.2\t1\t441 18\n",
      "134\t540\t0.2\t1\t495 45\n",
      "135\t343\t0.2\t1\t318 25\n",
      "136\t185\t0.2\t1\t166 19\n",
      "137\t142\t0.2\t1\t134 8\n",
      "138\t102\t0.2\t1\t96 6\n",
      "139\t72\t0.2\t1\t71 1\n",
      "140\t56\t0.2\t1\t51 5\n",
      "141\t30\t0.2\t1\t27 3\n",
      "142\t45\t0.2\t1\t39 6\n",
      "143\t48\t0.2\t1\t43 5\n",
      "144\t33\t0.2\t1\t32 1\n",
      "145\t35\t0.2\t1\t32 3\n",
      "146\t81\t0.2\t1\t79 2\n",
      "147\t221\t0.2\t1\t208 13\n",
      "148\t193\t0.2\t1\t178 15\n",
      "149\t78\t0.2\t1\t71 7\n",
      "150\t649\t0.2\t1\t611 38\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13670957 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t21942 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 672.52 s (43 us/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,706,000\n",
      "Reads with adapters:                11,318,033 (72.1%)\n",
      "Reads written (passing filters):    15,706,000 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,355,900,000 bp\n",
      "Quality-trimmed:               7,618,669 bp (0.3%)\n",
      "Total written (filtered):  1,966,841,132 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11318033 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 13.1%\n",
      "  G: 19.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2758722\t3926500.0\t0\t2758722\n",
      "2\t358574\t981625.0\t0\t358574\n",
      "3\t190965\t245406.2\t0\t190965\n",
      "4\t113114\t61351.6\t0\t113114\n",
      "5\t86888\t15337.9\t0\t86888\n",
      "6\t85123\t3834.5\t0\t85123\n",
      "7\t70336\t958.6\t0\t70336\n",
      "8\t85928\t239.7\t0\t85928\n",
      "9\t87066\t59.9\t0\t86195 871\n",
      "10\t79602\t15.0\t1\t75862 3740\n",
      "11\t88089\t3.7\t1\t81938 6151\n",
      "12\t85321\t0.9\t1\t80325 4996\n",
      "13\t84763\t0.2\t1\t79667 5096\n",
      "14\t88305\t0.2\t1\t82326 5979\n",
      "15\t88981\t0.2\t1\t83685 5296\n",
      "16\t88786\t0.2\t1\t82621 6165\n",
      "17\t106554\t0.2\t1\t98505 8049\n",
      "18\t89122\t0.2\t1\t84209 4913\n",
      "19\t77746\t0.2\t1\t73215 4531\n",
      "20\t92038\t0.2\t1\t86658 5380\n",
      "21\t99609\t0.2\t1\t93006 6603\n",
      "22\t85867\t0.2\t1\t81408 4459\n",
      "23\t100095\t0.2\t1\t93955 6140\n",
      "24\t90779\t0.2\t1\t83874 6905\n",
      "25\t97845\t0.2\t1\t91593 6252\n",
      "26\t88800\t0.2\t1\t83917 4883\n",
      "27\t97906\t0.2\t1\t90971 6935\n",
      "28\t105510\t0.2\t1\t98167 7343\n",
      "29\t95263\t0.2\t1\t88170 7093\n",
      "30\t108904\t0.2\t1\t103370 5534\n",
      "31\t79085\t0.2\t1\t74523 4562\n",
      "32\t103105\t0.2\t1\t96799 6306\n",
      "33\t92937\t0.2\t1\t86945 5992\n",
      "34\t100117\t0.2\t1\t93522 6595\n",
      "35\t106537\t0.2\t1\t100062 6475\n",
      "36\t91203\t0.2\t1\t86381 4822\n",
      "37\t99865\t0.2\t1\t93587 6278\n",
      "38\t105531\t0.2\t1\t98508 7023\n",
      "39\t105906\t0.2\t1\t99469 6437\n",
      "40\t104183\t0.2\t1\t97213 6970\n",
      "41\t116528\t0.2\t1\t106559 9969\n",
      "42\t93606\t0.2\t1\t87149 6457\n",
      "43\t153884\t0.2\t1\t145422 8462\n",
      "44\t48528\t0.2\t1\t44473 4055\n",
      "45\t107110\t0.2\t1\t99279 7831\n",
      "46\t172499\t0.2\t1\t161736 10763\n",
      "47\t116179\t0.2\t1\t109492 6687\n",
      "48\t80709\t0.2\t1\t73983 6726\n",
      "49\t159482\t0.2\t1\t150516 8966\n",
      "50\t64228\t0.2\t1\t59915 4313\n",
      "51\t90888\t0.2\t1\t84987 5901\n",
      "52\t193279\t0.2\t1\t183371 9908\n",
      "53\t141743\t0.2\t1\t135721 6022\n",
      "54\t59956\t0.2\t1\t56077 3879\n",
      "55\t102925\t0.2\t1\t98274 4651\n",
      "56\t66784\t0.2\t1\t62498 4286\n",
      "57\t123279\t0.2\t1\t118108 5171\n",
      "58\t43777\t0.2\t1\t41446 2331\n",
      "59\t39827\t0.2\t1\t36676 3151\n",
      "60\t113430\t0.2\t1\t108052 5378\n",
      "61\t89200\t0.2\t1\t85501 3699\n",
      "62\t44869\t0.2\t1\t41907 2962\n",
      "63\t85974\t0.2\t1\t81524 4450\n",
      "64\t74880\t0.2\t1\t71341 3539\n",
      "65\t59793\t0.2\t1\t56240 3553\n",
      "66\t87755\t0.2\t1\t82874 4881\n",
      "67\t100481\t0.2\t1\t94117 6364\n",
      "68\t167488\t0.2\t1\t159452 8036\n",
      "69\t79463\t0.2\t1\t74811 4652\n",
      "70\t27878\t0.2\t1\t25310 2568\n",
      "71\t23051\t0.2\t1\t21160 1891\n",
      "72\t60528\t0.2\t1\t57268 3260\n",
      "73\t72149\t0.2\t1\t68285 3864\n",
      "74\t73360\t0.2\t1\t69333 4027\n",
      "75\t71612\t0.2\t1\t67692 3920\n",
      "76\t70649\t0.2\t1\t66797 3852\n",
      "77\t70150\t0.2\t1\t66254 3896\n",
      "78\t65804\t0.2\t1\t62003 3801\n",
      "79\t64610\t0.2\t1\t60969 3641\n",
      "80\t63497\t0.2\t1\t60035 3462\n",
      "81\t61317\t0.2\t1\t58042 3275\n",
      "82\t58860\t0.2\t1\t55659 3201\n",
      "83\t57047\t0.2\t1\t53828 3219\n",
      "84\t54423\t0.2\t1\t51371 3052\n",
      "85\t52481\t0.2\t1\t49574 2907\n",
      "86\t50571\t0.2\t1\t47688 2883\n",
      "87\t48288\t0.2\t1\t45567 2721\n",
      "88\t45981\t0.2\t1\t43442 2539\n",
      "89\t43761\t0.2\t1\t41217 2544\n",
      "90\t42661\t0.2\t1\t40245 2416\n",
      "91\t40737\t0.2\t1\t38345 2392\n",
      "92\t38688\t0.2\t1\t36453 2235\n",
      "93\t34947\t0.2\t1\t32894 2053\n",
      "94\t34299\t0.2\t1\t32291 2008\n",
      "95\t31760\t0.2\t1\t29854 1906\n",
      "96\t29525\t0.2\t1\t27743 1782\n",
      "97\t27419\t0.2\t1\t25829 1590\n",
      "98\t24938\t0.2\t1\t23465 1473\n",
      "99\t24209\t0.2\t1\t22665 1544\n",
      "100\t21792\t0.2\t1\t20438 1354\n",
      "101\t20393\t0.2\t1\t19119 1274\n",
      "102\t18597\t0.2\t1\t17433 1164\n",
      "103\t16893\t0.2\t1\t15821 1072\n",
      "104\t15643\t0.2\t1\t14642 1001\n",
      "105\t14032\t0.2\t1\t13104 928\n",
      "106\t12725\t0.2\t1\t11896 829\n",
      "107\t11164\t0.2\t1\t10388 776\n",
      "108\t10327\t0.2\t1\t9611 716\n",
      "109\t9052\t0.2\t1\t8399 653\n",
      "110\t8194\t0.2\t1\t7576 618\n",
      "111\t7205\t0.2\t1\t6690 515\n",
      "112\t6374\t0.2\t1\t5898 476\n",
      "113\t5432\t0.2\t1\t5054 378\n",
      "114\t4742\t0.2\t1\t4349 393\n",
      "115\t4031\t0.2\t1\t3736 295\n",
      "116\t3440\t0.2\t1\t3126 314\n",
      "117\t3047\t0.2\t1\t2793 254\n",
      "118\t2620\t0.2\t1\t2385 235\n",
      "119\t2131\t0.2\t1\t1953 178\n",
      "120\t1732\t0.2\t1\t1592 140\n",
      "121\t1562\t0.2\t1\t1390 172\n",
      "122\t1210\t0.2\t1\t1094 116\n",
      "123\t1050\t0.2\t1\t947 103\n",
      "124\t965\t0.2\t1\t849 116\n",
      "125\t722\t0.2\t1\t635 87\n",
      "126\t663\t0.2\t1\t591 72\n",
      "127\t539\t0.2\t1\t460 79\n",
      "128\t435\t0.2\t1\t362 73\n",
      "129\t406\t0.2\t1\t358 48\n",
      "130\t274\t0.2\t1\t240 34\n",
      "131\t291\t0.2\t1\t261 30\n",
      "132\t213\t0.2\t1\t190 23\n",
      "133\t259\t0.2\t1\t233 26\n",
      "134\t361\t0.2\t1\t342 19\n",
      "135\t151\t0.2\t1\t128 23\n",
      "136\t85\t0.2\t1\t68 17\n",
      "137\t53\t0.2\t1\t45 8\n",
      "138\t43\t0.2\t1\t37 6\n",
      "139\t17\t0.2\t1\t15 2\n",
      "140\t21\t0.2\t1\t18 3\n",
      "141\t17\t0.2\t1\t15 2\n",
      "142\t9\t0.2\t1\t8 1\n",
      "143\t9\t0.2\t1\t7 2\n",
      "144\t15\t0.2\t1\t10 5\n",
      "145\t12\t0.2\t1\t10 2\n",
      "146\t12\t0.2\t1\t10 2\n",
      "147\t49\t0.2\t1\t45 4\n",
      "148\t35\t0.2\t1\t30 5\n",
      "149\t12\t0.2\t1\t11 1\n",
      "150\t163\t0.2\t1\t152 11\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "15706000 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t1881 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 713.54 s (45 us/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,706,000\n",
      "Reads with adapters:                11,248,615 (71.6%)\n",
      "Reads written (passing filters):    15,706,000 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,355,900,000 bp\n",
      "Quality-trimmed:               9,159,976 bp (0.4%)\n",
      "Total written (filtered):  1,968,243,578 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11248615 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 12.2%\n",
      "  G: 21.7%\n",
      "  T: 36.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2742334\t3926500.0\t0\t2742334\n",
      "2\t376488\t981625.0\t0\t376488\n",
      "3\t202415\t245406.2\t0\t202415\n",
      "4\t115660\t61351.6\t0\t115660\n",
      "5\t96934\t15337.9\t0\t96934\n",
      "6\t82183\t3834.5\t0\t82183\n",
      "7\t66579\t958.6\t0\t66579\n",
      "8\t95001\t239.7\t0\t95001\n",
      "9\t75363\t59.9\t0\t75125 238\n",
      "10\t81209\t15.0\t1\t77351 3858\n",
      "11\t85598\t3.7\t1\t80376 5222\n",
      "12\t86515\t0.9\t1\t81221 5294\n",
      "13\t96527\t0.2\t1\t91069 5458\n",
      "14\t89686\t0.2\t1\t83554 6132\n",
      "15\t72806\t0.2\t1\t68412 4394\n",
      "16\t105343\t0.2\t1\t99018 6325\n",
      "17\t82599\t0.2\t1\t77199 5400\n",
      "18\t79503\t0.2\t1\t74873 4630\n",
      "19\t100299\t0.2\t1\t94044 6255\n",
      "20\t93401\t0.2\t1\t87924 5477\n",
      "21\t81559\t0.2\t1\t76639 4920\n",
      "22\t92216\t0.2\t1\t86910 5306\n",
      "23\t94734\t0.2\t1\t89009 5725\n",
      "24\t95149\t0.2\t1\t89373 5776\n",
      "25\t105782\t0.2\t1\t99776 6006\n",
      "26\t87770\t0.2\t1\t82449 5321\n",
      "27\t95338\t0.2\t1\t88893 6445\n",
      "28\t90298\t0.2\t1\t85892 4406\n",
      "29\t95418\t0.2\t1\t89790 5628\n",
      "30\t93873\t0.2\t1\t89134 4739\n",
      "31\t100963\t0.2\t1\t95205 5758\n",
      "32\t90423\t0.2\t1\t85945 4478\n",
      "33\t99633\t0.2\t1\t93789 5844\n",
      "34\t126170\t0.2\t1\t118899 7271\n",
      "35\t71917\t0.2\t1\t68415 3502\n",
      "36\t98752\t0.2\t1\t93147 5605\n",
      "37\t104121\t0.2\t1\t98203 5918\n",
      "38\t97206\t0.2\t1\t92554 4652\n",
      "39\t94225\t0.2\t1\t89765 4460\n",
      "40\t104186\t0.2\t1\t98424 5762\n",
      "41\t99925\t0.2\t1\t94861 5064\n",
      "42\t102480\t0.2\t1\t97343 5137\n",
      "43\t92586\t0.2\t1\t87878 4708\n",
      "44\t105897\t0.2\t1\t100306 5591\n",
      "45\t112588\t0.2\t1\t106743 5845\n",
      "46\t80544\t0.2\t1\t76510 4034\n",
      "47\t101141\t0.2\t1\t96374 4767\n",
      "48\t98066\t0.2\t1\t93549 4517\n",
      "49\t101791\t0.2\t1\t96776 5015\n",
      "50\t97034\t0.2\t1\t92147 4887\n",
      "51\t120282\t0.2\t1\t114381 5901\n",
      "52\t84235\t0.2\t1\t80401 3834\n",
      "53\t102171\t0.2\t1\t97747 4424\n",
      "54\t76841\t0.2\t1\t73430 3411\n",
      "55\t99264\t0.2\t1\t95032 4232\n",
      "56\t101316\t0.2\t1\t96570 4746\n",
      "57\t94879\t0.2\t1\t90678 4201\n",
      "58\t90444\t0.2\t1\t86700 3744\n",
      "59\t92322\t0.2\t1\t88235 4087\n",
      "60\t94688\t0.2\t1\t90370 4318\n",
      "61\t97150\t0.2\t1\t92685 4465\n",
      "62\t103718\t0.2\t1\t98963 4755\n",
      "63\t136885\t0.2\t1\t131195 5690\n",
      "64\t109397\t0.2\t1\t105405 3992\n",
      "65\t71767\t0.2\t1\t68765 3002\n",
      "66\t78028\t0.2\t1\t74793 3235\n",
      "67\t77090\t0.2\t1\t73810 3280\n",
      "68\t76844\t0.2\t1\t73617 3227\n",
      "69\t75212\t0.2\t1\t71969 3243\n",
      "70\t76863\t0.2\t1\t73510 3353\n",
      "71\t80211\t0.2\t1\t76869 3342\n",
      "72\t75920\t0.2\t1\t72620 3300\n",
      "73\t78093\t0.2\t1\t74948 3145\n",
      "74\t72443\t0.2\t1\t69570 2873\n",
      "75\t68538\t0.2\t1\t65762 2776\n",
      "76\t66619\t0.2\t1\t63921 2698\n",
      "77\t66151\t0.2\t1\t63438 2713\n",
      "78\t62817\t0.2\t1\t60225 2592\n",
      "79\t60312\t0.2\t1\t57778 2534\n",
      "80\t60163\t0.2\t1\t57704 2459\n",
      "81\t57937\t0.2\t1\t55639 2298\n",
      "82\t54845\t0.2\t1\t52637 2208\n",
      "83\t53627\t0.2\t1\t51399 2228\n",
      "84\t50792\t0.2\t1\t48633 2159\n",
      "85\t49633\t0.2\t1\t47523 2110\n",
      "86\t49264\t0.2\t1\t47185 2079\n",
      "87\t49251\t0.2\t1\t47314 1937\n",
      "88\t46501\t0.2\t1\t44723 1778\n",
      "89\t41655\t0.2\t1\t39945 1710\n",
      "90\t39420\t0.2\t1\t37741 1679\n",
      "91\t37399\t0.2\t1\t35859 1540\n",
      "92\t35077\t0.2\t1\t33634 1443\n",
      "93\t32838\t0.2\t1\t31397 1441\n",
      "94\t31873\t0.2\t1\t30582 1291\n",
      "95\t29856\t0.2\t1\t28661 1195\n",
      "96\t27831\t0.2\t1\t26723 1108\n",
      "97\t25609\t0.2\t1\t24584 1025\n",
      "98\t23598\t0.2\t1\t22615 983\n",
      "99\t23296\t0.2\t1\t22392 904\n",
      "100\t21689\t0.2\t1\t20900 789\n",
      "101\t19855\t0.2\t1\t19105 750\n",
      "102\t17867\t0.2\t1\t17193 674\n",
      "103\t15691\t0.2\t1\t15073 618\n",
      "104\t15000\t0.2\t1\t14394 606\n",
      "105\t13326\t0.2\t1\t12799 527\n",
      "106\t12111\t0.2\t1\t11648 463\n",
      "107\t10556\t0.2\t1\t10155 401\n",
      "108\t9460\t0.2\t1\t9085 375\n",
      "109\t8474\t0.2\t1\t8135 339\n",
      "110\t7655\t0.2\t1\t7315 340\n",
      "111\t7041\t0.2\t1\t6747 294\n",
      "112\t6163\t0.2\t1\t5902 261\n",
      "113\t5337\t0.2\t1\t5089 248\n",
      "114\t4565\t0.2\t1\t4367 198\n",
      "115\t3763\t0.2\t1\t3606 157\n",
      "116\t3164\t0.2\t1\t2993 171\n",
      "117\t2789\t0.2\t1\t2653 136\n",
      "118\t2380\t0.2\t1\t2277 103\n",
      "119\t1999\t0.2\t1\t1886 113\n",
      "120\t1562\t0.2\t1\t1470 92\n",
      "121\t1457\t0.2\t1\t1367 90\n",
      "122\t1103\t0.2\t1\t1044 59\n",
      "123\t1029\t0.2\t1\t972 57\n",
      "124\t896\t0.2\t1\t845 51\n",
      "125\t678\t0.2\t1\t637 41\n",
      "126\t611\t0.2\t1\t570 41\n",
      "127\t481\t0.2\t1\t451 30\n",
      "128\t387\t0.2\t1\t354 33\n",
      "129\t379\t0.2\t1\t356 23\n",
      "130\t249\t0.2\t1\t234 15\n",
      "131\t280\t0.2\t1\t263 17\n",
      "132\t196\t0.2\t1\t186 10\n",
      "133\t279\t0.2\t1\t264 15\n",
      "134\t375\t0.2\t1\t361 14\n",
      "135\t138\t0.2\t1\t127 11\n",
      "136\t76\t0.2\t1\t71 5\n",
      "137\t48\t0.2\t1\t43 5\n",
      "138\t41\t0.2\t1\t37 4\n",
      "139\t18\t0.2\t1\t17 1\n",
      "140\t14\t0.2\t1\t12 2\n",
      "141\t8\t0.2\t1\t8\n",
      "142\t12\t0.2\t1\t8 4\n",
      "143\t10\t0.2\t1\t6 4\n",
      "144\t13\t0.2\t1\t11 2\n",
      "145\t15\t0.2\t1\t13 2\n",
      "146\t12\t0.2\t1\t10 2\n",
      "147\t37\t0.2\t1\t35 2\n",
      "148\t25\t0.2\t1\t24 1\n",
      "149\t9\t0.2\t1\t6 3\n",
      "150\t99\t0.2\t1\t92 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "15706000 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t14180 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 679.80 s (44 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,591,852\n",
      "Reads with adapters:                11,241,936 (72.1%)\n",
      "Reads written (passing filters):    15,591,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,338,777,800 bp\n",
      "Quality-trimmed:               7,762,195 bp (0.3%)\n",
      "Total written (filtered):  1,950,418,217 bp (83.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11241936 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 13.2%\n",
      "  G: 19.5%\n",
      "  T: 35.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2728314\t3897963.0\t0\t2728314\n",
      "2\t352949\t974490.8\t0\t352949\n",
      "3\t189042\t243622.7\t0\t189042\n",
      "4\t110295\t60905.7\t0\t110295\n",
      "5\t86754\t15226.4\t0\t86754\n",
      "6\t84318\t3806.6\t0\t84318\n",
      "7\t67898\t951.7\t0\t67898\n",
      "8\t84774\t237.9\t0\t84774\n",
      "9\t86162\t59.5\t0\t85323 839\n",
      "10\t77737\t14.9\t1\t73221 4516\n",
      "11\t88459\t3.7\t1\t81353 7106\n",
      "12\t84258\t0.9\t1\t78502 5756\n",
      "13\t83829\t0.2\t1\t77646 6183\n",
      "14\t87427\t0.2\t1\t80554 6873\n",
      "15\t88669\t0.2\t1\t82188 6481\n",
      "16\t87155\t0.2\t1\t79865 7290\n",
      "17\t108105\t0.2\t1\t98581 9524\n",
      "18\t88189\t0.2\t1\t82109 6080\n",
      "19\t75841\t0.2\t1\t70534 5307\n",
      "20\t90690\t0.2\t1\t84354 6336\n",
      "21\t100155\t0.2\t1\t92424 7731\n",
      "22\t84109\t0.2\t1\t78568 5541\n",
      "23\t100437\t0.2\t1\t93211 7226\n",
      "24\t89008\t0.2\t1\t81375 7633\n",
      "25\t97433\t0.2\t1\t90437 6996\n",
      "26\t86900\t0.2\t1\t81328 5572\n",
      "27\t98051\t0.2\t1\t90237 7814\n",
      "28\t106220\t0.2\t1\t97819 8401\n",
      "29\t94302\t0.2\t1\t86322 7980\n",
      "30\t107823\t0.2\t1\t101345 6478\n",
      "31\t77079\t0.2\t1\t71662 5417\n",
      "32\t102558\t0.2\t1\t95070 7488\n",
      "33\t91366\t0.2\t1\t84409 6957\n",
      "34\t99532\t0.2\t1\t91471 8061\n",
      "35\t107666\t0.2\t1\t99772 7894\n",
      "36\t89079\t0.2\t1\t83528 5551\n",
      "37\t100059\t0.2\t1\t92531 7528\n",
      "38\t104683\t0.2\t1\t96200 8483\n",
      "39\t105342\t0.2\t1\t97340 8002\n",
      "40\t103603\t0.2\t1\t95197 8406\n",
      "41\t116472\t0.2\t1\t105109 11363\n",
      "42\t91799\t0.2\t1\t84082 7717\n",
      "43\t151482\t0.2\t1\t141550 9932\n",
      "44\t48893\t0.2\t1\t44323 4570\n",
      "45\t107330\t0.2\t1\t98158 9172\n",
      "46\t169588\t0.2\t1\t157239 12349\n",
      "47\t115292\t0.2\t1\t107496 7796\n",
      "48\t81237\t0.2\t1\t73490 7747\n",
      "49\t158364\t0.2\t1\t148089 10275\n",
      "50\t62839\t0.2\t1\t58050 4789\n",
      "51\t90568\t0.2\t1\t83874 6694\n",
      "52\t192734\t0.2\t1\t181187 11547\n",
      "53\t137640\t0.2\t1\t130713 6927\n",
      "54\t59884\t0.2\t1\t55438 4446\n",
      "55\t102380\t0.2\t1\t96895 5485\n",
      "56\t71739\t0.2\t1\t66587 5152\n",
      "57\t121813\t0.2\t1\t115448 6365\n",
      "58\t43981\t0.2\t1\t41008 2973\n",
      "59\t41261\t0.2\t1\t37490 3771\n",
      "60\t111430\t0.2\t1\t105252 6178\n",
      "61\t87063\t0.2\t1\t82619 4444\n",
      "62\t47900\t0.2\t1\t44309 3591\n",
      "63\t85477\t0.2\t1\t80258 5219\n",
      "64\t72824\t0.2\t1\t68588 4236\n",
      "65\t60641\t0.2\t1\t56399 4242\n",
      "66\t87315\t0.2\t1\t81607 5708\n",
      "67\t100034\t0.2\t1\t92897 7137\n",
      "68\t160033\t0.2\t1\t150907 9126\n",
      "69\t87754\t0.2\t1\t82192 5562\n",
      "70\t34259\t0.2\t1\t30999 3260\n",
      "71\t22148\t0.2\t1\t20058 2090\n",
      "72\t59270\t0.2\t1\t55764 3506\n",
      "73\t72359\t0.2\t1\t68127 4232\n",
      "74\t73545\t0.2\t1\t68961 4584\n",
      "75\t72172\t0.2\t1\t67396 4776\n",
      "76\t70342\t0.2\t1\t65617 4725\n",
      "77\t69830\t0.2\t1\t65084 4746\n",
      "78\t66115\t0.2\t1\t61597 4518\n",
      "79\t64360\t0.2\t1\t60010 4350\n",
      "80\t63230\t0.2\t1\t58997 4233\n",
      "81\t61274\t0.2\t1\t57172 4102\n",
      "82\t59014\t0.2\t1\t55084 3930\n",
      "83\t57125\t0.2\t1\t53358 3767\n",
      "84\t54574\t0.2\t1\t50808 3766\n",
      "85\t52527\t0.2\t1\t48940 3587\n",
      "86\t50257\t0.2\t1\t46911 3346\n",
      "87\t48208\t0.2\t1\t45026 3182\n",
      "88\t45866\t0.2\t1\t42781 3085\n",
      "89\t44045\t0.2\t1\t41050 2995\n",
      "90\t43438\t0.2\t1\t40512 2926\n",
      "91\t40328\t0.2\t1\t37640 2688\n",
      "92\t38164\t0.2\t1\t35615 2549\n",
      "93\t35352\t0.2\t1\t32983 2369\n",
      "94\t33785\t0.2\t1\t31458 2327\n",
      "95\t31871\t0.2\t1\t29668 2203\n",
      "96\t29463\t0.2\t1\t27470 1993\n",
      "97\t27711\t0.2\t1\t25891 1820\n",
      "98\t25451\t0.2\t1\t23610 1841\n",
      "99\t24415\t0.2\t1\t22590 1825\n",
      "100\t21725\t0.2\t1\t20167 1558\n",
      "101\t20433\t0.2\t1\t19008 1425\n",
      "102\t18660\t0.2\t1\t17274 1386\n",
      "103\t16794\t0.2\t1\t15616 1178\n",
      "104\t15844\t0.2\t1\t14677 1167\n",
      "105\t14021\t0.2\t1\t13036 985\n",
      "106\t12448\t0.2\t1\t11562 886\n",
      "107\t11573\t0.2\t1\t10706 867\n",
      "108\t10412\t0.2\t1\t9602 810\n",
      "109\t8989\t0.2\t1\t8297 692\n",
      "110\t8150\t0.2\t1\t7523 627\n",
      "111\t7254\t0.2\t1\t6680 574\n",
      "112\t6183\t0.2\t1\t5709 474\n",
      "113\t5353\t0.2\t1\t4937 416\n",
      "114\t4766\t0.2\t1\t4364 402\n",
      "115\t4132\t0.2\t1\t3707 425\n",
      "116\t3477\t0.2\t1\t3165 312\n",
      "117\t3036\t0.2\t1\t2738 298\n",
      "118\t2618\t0.2\t1\t2398 220\n",
      "119\t2115\t0.2\t1\t1918 197\n",
      "120\t1769\t0.2\t1\t1599 170\n",
      "121\t1495\t0.2\t1\t1327 168\n",
      "122\t1258\t0.2\t1\t1133 125\n",
      "123\t1079\t0.2\t1\t967 112\n",
      "124\t920\t0.2\t1\t808 112\n",
      "125\t777\t0.2\t1\t674 103\n",
      "126\t681\t0.2\t1\t592 89\n",
      "127\t553\t0.2\t1\t494 59\n",
      "128\t387\t0.2\t1\t336 51\n",
      "129\t401\t0.2\t1\t353 48\n",
      "130\t291\t0.2\t1\t256 35\n",
      "131\t270\t0.2\t1\t235 35\n",
      "132\t242\t0.2\t1\t222 20\n",
      "133\t268\t0.2\t1\t225 43\n",
      "134\t327\t0.2\t1\t297 30\n",
      "135\t148\t0.2\t1\t131 17\n",
      "136\t79\t0.2\t1\t65 14\n",
      "137\t63\t0.2\t1\t55 8\n",
      "138\t35\t0.2\t1\t28 7\n",
      "139\t28\t0.2\t1\t22 6\n",
      "140\t22\t0.2\t1\t20 2\n",
      "141\t16\t0.2\t1\t14 2\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t16\t0.2\t1\t11 5\n",
      "144\t10\t0.2\t1\t5 5\n",
      "145\t13\t0.2\t1\t8 5\n",
      "146\t16\t0.2\t1\t12 4\n",
      "147\t29\t0.2\t1\t25 4\n",
      "148\t26\t0.2\t1\t23 3\n",
      "149\t19\t0.2\t1\t18 1\n",
      "150\t167\t0.2\t1\t150 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "15591852 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t1857 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/16C_32psu_4_S4_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 724.45 s (46 us/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,591,852\n",
      "Reads with adapters:                11,164,940 (71.6%)\n",
      "Reads written (passing filters):    15,591,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,338,777,800 bp\n",
      "Quality-trimmed:               8,987,489 bp (0.4%)\n",
      "Total written (filtered):  1,952,377,119 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11164940 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 12.2%\n",
      "  G: 21.8%\n",
      "  T: 35.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2716063\t3897963.0\t0\t2716063\n",
      "2\t365801\t974490.8\t0\t365801\n",
      "3\t204110\t243622.7\t0\t204110\n",
      "4\t111706\t60905.7\t0\t111706\n",
      "5\t93686\t15226.4\t0\t93686\n",
      "6\t82310\t3806.6\t0\t82310\n",
      "7\t64821\t951.7\t0\t64821\n",
      "8\t93347\t237.9\t0\t93347\n",
      "9\t74551\t59.5\t0\t74263 288\n",
      "10\t78346\t14.9\t1\t72991 5355\n",
      "11\t85380\t3.7\t1\t78328 7052\n",
      "12\t84150\t0.9\t1\t77541 6609\n",
      "13\t96178\t0.2\t1\t88615 7563\n",
      "14\t88433\t0.2\t1\t80671 7762\n",
      "15\t72019\t0.2\t1\t66503 5516\n",
      "16\t102952\t0.2\t1\t94755 8197\n",
      "17\t86769\t0.2\t1\t79687 7082\n",
      "18\t76619\t0.2\t1\t70853 5766\n",
      "19\t98606\t0.2\t1\t91042 7564\n",
      "20\t91841\t0.2\t1\t84883 6958\n",
      "21\t80822\t0.2\t1\t74556 6266\n",
      "22\t90084\t0.2\t1\t83382 6702\n",
      "23\t95275\t0.2\t1\t87967 7308\n",
      "24\t93761\t0.2\t1\t86637 7124\n",
      "25\t105154\t0.2\t1\t98108 7046\n",
      "26\t87685\t0.2\t1\t80687 6998\n",
      "27\t96238\t0.2\t1\t88248 7990\n",
      "28\t86982\t0.2\t1\t81506 5476\n",
      "29\t96532\t0.2\t1\t89458 7074\n",
      "30\t91145\t0.2\t1\t85081 6064\n",
      "31\t100852\t0.2\t1\t93803 7049\n",
      "32\t86615\t0.2\t1\t80885 5730\n",
      "33\t99917\t0.2\t1\t92622 7295\n",
      "34\t128321\t0.2\t1\t119309 9012\n",
      "35\t68466\t0.2\t1\t63767 4699\n",
      "36\t99171\t0.2\t1\t91848 7323\n",
      "37\t104228\t0.2\t1\t97074 7154\n",
      "38\t95840\t0.2\t1\t89901 5939\n",
      "39\t91223\t0.2\t1\t85899 5324\n",
      "40\t103901\t0.2\t1\t97025 6876\n",
      "41\t100699\t0.2\t1\t94360 6339\n",
      "42\t101648\t0.2\t1\t95335 6313\n",
      "43\t91398\t0.2\t1\t85693 5705\n",
      "44\t105931\t0.2\t1\t98930 7001\n",
      "45\t111589\t0.2\t1\t104455 7134\n",
      "46\t79183\t0.2\t1\t74169 5014\n",
      "47\t99997\t0.2\t1\t94251 5746\n",
      "48\t97219\t0.2\t1\t91470 5749\n",
      "49\t102240\t0.2\t1\t95693 6547\n",
      "50\t96699\t0.2\t1\t90628 6071\n",
      "51\t118228\t0.2\t1\t110940 7288\n",
      "52\t84607\t0.2\t1\t79439 5168\n",
      "53\t104742\t0.2\t1\t98960 5782\n",
      "54\t75371\t0.2\t1\t71116 4255\n",
      "55\t98114\t0.2\t1\t92755 5359\n",
      "56\t100972\t0.2\t1\t94974 5998\n",
      "57\t94787\t0.2\t1\t89866 4921\n",
      "58\t91176\t0.2\t1\t86320 4856\n",
      "59\t91225\t0.2\t1\t86196 5029\n",
      "60\t94303\t0.2\t1\t88940 5363\n",
      "61\t95802\t0.2\t1\t90516 5286\n",
      "62\t104062\t0.2\t1\t98063 5999\n",
      "63\t134017\t0.2\t1\t127276 6741\n",
      "64\t107807\t0.2\t1\t103277 4530\n",
      "65\t71585\t0.2\t1\t67908 3677\n",
      "66\t78177\t0.2\t1\t74035 4142\n",
      "67\t77189\t0.2\t1\t72913 4276\n",
      "68\t77503\t0.2\t1\t73378 4125\n",
      "69\t76227\t0.2\t1\t72009 4218\n",
      "70\t77805\t0.2\t1\t73707 4098\n",
      "71\t80348\t0.2\t1\t76129 4219\n",
      "72\t76686\t0.2\t1\t72546 4140\n",
      "73\t78718\t0.2\t1\t74553 4165\n",
      "74\t72998\t0.2\t1\t69110 3888\n",
      "75\t69291\t0.2\t1\t65521 3770\n",
      "76\t66128\t0.2\t1\t62546 3582\n",
      "77\t65756\t0.2\t1\t62222 3534\n",
      "78\t62918\t0.2\t1\t59420 3498\n",
      "79\t59982\t0.2\t1\t56797 3185\n",
      "80\t59872\t0.2\t1\t56761 3111\n",
      "81\t57826\t0.2\t1\t54814 3012\n",
      "82\t55215\t0.2\t1\t52336 2879\n",
      "83\t53442\t0.2\t1\t50600 2842\n",
      "84\t50951\t0.2\t1\t48209 2742\n",
      "85\t49724\t0.2\t1\t47191 2533\n",
      "86\t49032\t0.2\t1\t46473 2559\n",
      "87\t49028\t0.2\t1\t46704 2324\n",
      "88\t46498\t0.2\t1\t44150 2348\n",
      "89\t42159\t0.2\t1\t40004 2155\n",
      "90\t40296\t0.2\t1\t38283 2013\n",
      "91\t37292\t0.2\t1\t35403 1889\n",
      "92\t34841\t0.2\t1\t33039 1802\n",
      "93\t33152\t0.2\t1\t31479 1673\n",
      "94\t31763\t0.2\t1\t30149 1614\n",
      "95\t30241\t0.2\t1\t28665 1576\n",
      "96\t27836\t0.2\t1\t26423 1413\n",
      "97\t25827\t0.2\t1\t24589 1238\n",
      "98\t24001\t0.2\t1\t22767 1234\n",
      "99\t23498\t0.2\t1\t22301 1197\n",
      "100\t21732\t0.2\t1\t20644 1088\n",
      "101\t19788\t0.2\t1\t18836 952\n",
      "102\t17924\t0.2\t1\t16990 934\n",
      "103\t15850\t0.2\t1\t15000 850\n",
      "104\t14943\t0.2\t1\t14170 773\n",
      "105\t13279\t0.2\t1\t12646 633\n",
      "106\t11897\t0.2\t1\t11297 600\n",
      "107\t10895\t0.2\t1\t10336 559\n",
      "108\t9562\t0.2\t1\t9057 505\n",
      "109\t8353\t0.2\t1\t7873 480\n",
      "110\t7641\t0.2\t1\t7229 412\n",
      "111\t7111\t0.2\t1\t6763 348\n",
      "112\t6029\t0.2\t1\t5694 335\n",
      "113\t5217\t0.2\t1\t4974 243\n",
      "114\t4567\t0.2\t1\t4342 225\n",
      "115\t3814\t0.2\t1\t3593 221\n",
      "116\t3236\t0.2\t1\t3069 167\n",
      "117\t2708\t0.2\t1\t2550 158\n",
      "118\t2402\t0.2\t1\t2269 133\n",
      "119\t1976\t0.2\t1\t1850 126\n",
      "120\t1573\t0.2\t1\t1480 93\n",
      "121\t1360\t0.2\t1\t1290 70\n",
      "122\t1161\t0.2\t1\t1085 76\n",
      "123\t1044\t0.2\t1\t961 83\n",
      "124\t817\t0.2\t1\t759 58\n",
      "125\t740\t0.2\t1\t678 62\n",
      "126\t632\t0.2\t1\t586 46\n",
      "127\t507\t0.2\t1\t481 26\n",
      "128\t371\t0.2\t1\t328 43\n",
      "129\t368\t0.2\t1\t334 34\n",
      "130\t272\t0.2\t1\t248 24\n",
      "131\t250\t0.2\t1\t232 18\n",
      "132\t226\t0.2\t1\t220 6\n",
      "133\t268\t0.2\t1\t248 20\n",
      "134\t319\t0.2\t1\t304 15\n",
      "135\t142\t0.2\t1\t138 4\n",
      "136\t65\t0.2\t1\t62 3\n",
      "137\t61\t0.2\t1\t60 1\n",
      "138\t31\t0.2\t1\t25 6\n",
      "139\t26\t0.2\t1\t21 5\n",
      "140\t13\t0.2\t1\t11 2\n",
      "141\t16\t0.2\t1\t13 3\n",
      "142\t11\t0.2\t1\t8 3\n",
      "143\t13\t0.2\t1\t11 2\n",
      "144\t7\t0.2\t1\t5 2\n",
      "145\t9\t0.2\t1\t6 3\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t30\t0.2\t1\t26 4\n",
      "148\t17\t0.2\t1\t16 1\n",
      "149\t16\t0.2\t1\t14 2\n",
      "150\t117\t0.2\t1\t107 10\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "15591852 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t13005 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 734.35 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,172,174\n",
      "Reads with adapters:                14,009,565 (77.1%)\n",
      "Reads written (passing filters):    18,172,174 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,725,826,100 bp\n",
      "Quality-trimmed:               5,576,029 bp (0.2%)\n",
      "Total written (filtered):  2,153,538,533 bp (79.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14009565 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 14.7%\n",
      "  G: 20.2%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2553382\t4543043.5\t0\t2553382\n",
      "2\t361304\t1135760.9\t0\t361304\n",
      "3\t203451\t283940.2\t0\t203451\n",
      "4\t119346\t70985.1\t0\t119346\n",
      "5\t96383\t17746.3\t0\t96383\n",
      "6\t92443\t4436.6\t0\t92443\n",
      "7\t75722\t1109.1\t0\t75722\n",
      "8\t95119\t277.3\t0\t95119\n",
      "9\t94842\t69.3\t0\t93616 1226\n",
      "10\t89178\t17.3\t1\t84587 4591\n",
      "11\t98839\t4.3\t1\t91164 7675\n",
      "12\t94690\t1.1\t1\t88668 6022\n",
      "13\t96487\t0.3\t1\t90169 6318\n",
      "14\t97956\t0.3\t1\t91158 6798\n",
      "15\t101508\t0.3\t1\t94950 6558\n",
      "16\t99917\t0.3\t1\t92571 7346\n",
      "17\t122129\t0.3\t1\t112362 9767\n",
      "18\t103253\t0.3\t1\t97095 6158\n",
      "19\t88992\t0.3\t1\t83121 5871\n",
      "20\t106240\t0.3\t1\t99305 6935\n",
      "21\t117020\t0.3\t1\t108991 8029\n",
      "22\t99969\t0.3\t1\t94314 5655\n",
      "23\t116939\t0.3\t1\t109185 7754\n",
      "24\t106769\t0.3\t1\t98031 8738\n",
      "25\t115163\t0.3\t1\t107352 7811\n",
      "26\t105376\t0.3\t1\t99072 6304\n",
      "27\t117162\t0.3\t1\t108493 8669\n",
      "28\t115218\t0.3\t1\t107267 7951\n",
      "29\t121831\t0.3\t1\t112202 9629\n",
      "30\t130309\t0.3\t1\t123069 7240\n",
      "31\t100032\t0.3\t1\t93538 6494\n",
      "32\t125002\t0.3\t1\t117224 7778\n",
      "33\t114973\t0.3\t1\t107024 7949\n",
      "34\t140462\t0.3\t1\t130516 9946\n",
      "35\t105315\t0.3\t1\t99186 6129\n",
      "36\t133473\t0.3\t1\t123640 9833\n",
      "37\t118492\t0.3\t1\t111283 7209\n",
      "38\t133856\t0.3\t1\t125329 8527\n",
      "39\t129659\t0.3\t1\t120554 9105\n",
      "40\t124082\t0.3\t1\t117052 7030\n",
      "41\t145782\t0.3\t1\t135364 10418\n",
      "42\t118797\t0.3\t1\t109967 8830\n",
      "43\t194297\t0.3\t1\t184362 9935\n",
      "44\t63920\t0.3\t1\t58531 5389\n",
      "45\t132756\t0.3\t1\t124789 7967\n",
      "46\t128569\t0.3\t1\t120475 8094\n",
      "47\t134969\t0.3\t1\t126850 8119\n",
      "48\t127793\t0.3\t1\t119492 8301\n",
      "49\t141939\t0.3\t1\t132171 9768\n",
      "50\t122730\t0.3\t1\t115608 7122\n",
      "51\t132062\t0.3\t1\t123880 8182\n",
      "52\t137197\t0.3\t1\t127456 9741\n",
      "53\t151575\t0.3\t1\t142793 8782\n",
      "54\t114349\t0.3\t1\t107125 7224\n",
      "55\t137708\t0.3\t1\t129156 8552\n",
      "56\t161649\t0.3\t1\t152383 9266\n",
      "57\t114410\t0.3\t1\t106500 7910\n",
      "58\t137398\t0.3\t1\t130375 7023\n",
      "59\t114545\t0.3\t1\t107545 7000\n",
      "60\t136347\t0.3\t1\t128034 8313\n",
      "61\t131449\t0.3\t1\t123861 7588\n",
      "62\t139001\t0.3\t1\t131253 7748\n",
      "63\t117841\t0.3\t1\t110235 7606\n",
      "64\t124937\t0.3\t1\t117975 6962\n",
      "65\t137827\t0.3\t1\t129558 8269\n",
      "66\t145197\t0.3\t1\t136356 8841\n",
      "67\t156037\t0.3\t1\t144343 11694\n",
      "68\t293703\t0.3\t1\t279898 13805\n",
      "69\t56423\t0.3\t1\t51953 4470\n",
      "70\t20870\t0.3\t1\t17669 3201\n",
      "71\t73391\t0.3\t1\t68872 4519\n",
      "72\t109127\t0.3\t1\t102948 6179\n",
      "73\t115790\t0.3\t1\t109200 6590\n",
      "74\t114108\t0.3\t1\t107376 6732\n",
      "75\t112468\t0.3\t1\t105902 6566\n",
      "76\t111908\t0.3\t1\t105400 6508\n",
      "77\t110508\t0.3\t1\t103945 6563\n",
      "78\t104515\t0.3\t1\t98282 6233\n",
      "79\t103447\t0.3\t1\t97321 6126\n",
      "80\t100314\t0.3\t1\t94349 5965\n",
      "81\t99307\t0.3\t1\t93451 5856\n",
      "82\t96030\t0.3\t1\t90418 5612\n",
      "83\t93920\t0.3\t1\t88302 5618\n",
      "84\t90786\t0.3\t1\t85374 5412\n",
      "85\t87976\t0.3\t1\t82940 5036\n",
      "86\t84575\t0.3\t1\t79489 5086\n",
      "87\t80800\t0.3\t1\t75885 4915\n",
      "88\t79827\t0.3\t1\t75057 4770\n",
      "89\t76306\t0.3\t1\t71788 4518\n",
      "90\t75447\t0.3\t1\t70879 4568\n",
      "91\t70739\t0.3\t1\t66457 4282\n",
      "92\t67827\t0.3\t1\t63701 4126\n",
      "93\t63802\t0.3\t1\t60033 3769\n",
      "94\t61718\t0.3\t1\t57911 3807\n",
      "95\t57339\t0.3\t1\t53808 3531\n",
      "96\t54465\t0.3\t1\t51099 3366\n",
      "97\t51194\t0.3\t1\t48085 3109\n",
      "98\t47800\t0.3\t1\t44777 3023\n",
      "99\t45494\t0.3\t1\t42548 2946\n",
      "100\t41995\t0.3\t1\t39278 2717\n",
      "101\t38874\t0.3\t1\t36431 2443\n",
      "102\t36656\t0.3\t1\t34295 2361\n",
      "103\t33608\t0.3\t1\t31418 2190\n",
      "104\t31440\t0.3\t1\t29409 2031\n",
      "105\t28173\t0.3\t1\t26291 1882\n",
      "106\t25331\t0.3\t1\t23628 1703\n",
      "107\t23358\t0.3\t1\t21803 1555\n",
      "108\t21762\t0.3\t1\t20261 1501\n",
      "109\t19593\t0.3\t1\t18199 1394\n",
      "110\t17697\t0.3\t1\t16476 1221\n",
      "111\t15926\t0.3\t1\t14765 1161\n",
      "112\t13846\t0.3\t1\t12871 975\n",
      "113\t11827\t0.3\t1\t10954 873\n",
      "114\t10902\t0.3\t1\t10102 800\n",
      "115\t9531\t0.3\t1\t8821 710\n",
      "116\t8041\t0.3\t1\t7361 680\n",
      "117\t6932\t0.3\t1\t6371 561\n",
      "118\t5995\t0.3\t1\t5469 526\n",
      "119\t4903\t0.3\t1\t4506 397\n",
      "120\t4372\t0.3\t1\t3973 399\n",
      "121\t3600\t0.3\t1\t3268 332\n",
      "122\t2879\t0.3\t1\t2586 293\n",
      "123\t2449\t0.3\t1\t2225 224\n",
      "124\t2193\t0.3\t1\t1959 234\n",
      "125\t1829\t0.3\t1\t1616 213\n",
      "126\t1642\t0.3\t1\t1488 154\n",
      "127\t1294\t0.3\t1\t1160 134\n",
      "128\t1024\t0.3\t1\t902 122\n",
      "129\t830\t0.3\t1\t734 96\n",
      "130\t651\t0.3\t1\t579 72\n",
      "131\t623\t0.3\t1\t568 55\n",
      "132\t611\t0.3\t1\t554 57\n",
      "133\t770\t0.3\t1\t705 65\n",
      "134\t1146\t0.3\t1\t1081 65\n",
      "135\t448\t0.3\t1\t402 46\n",
      "136\t201\t0.3\t1\t171 30\n",
      "137\t163\t0.3\t1\t138 25\n",
      "138\t82\t0.3\t1\t67 15\n",
      "139\t72\t0.3\t1\t55 17\n",
      "140\t56\t0.3\t1\t42 14\n",
      "141\t36\t0.3\t1\t35 1\n",
      "142\t25\t0.3\t1\t15 10\n",
      "143\t23\t0.3\t1\t18 5\n",
      "144\t8\t0.3\t1\t7 1\n",
      "145\t24\t0.3\t1\t19 5\n",
      "146\t21\t0.3\t1\t20 1\n",
      "147\t76\t0.3\t1\t66 10\n",
      "148\t65\t0.3\t1\t57 8\n",
      "149\t66\t0.3\t1\t57 9\n",
      "150\t608\t0.3\t1\t538 70\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18172174 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5180 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 767.43 s (42 us/read; 1.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,172,174\n",
      "Reads with adapters:                13,925,142 (76.6%)\n",
      "Reads written (passing filters):    18,172,174 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,725,826,100 bp\n",
      "Quality-trimmed:              12,772,261 bp (0.5%)\n",
      "Total written (filtered):  2,156,656,577 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13925142 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.4%\n",
      "  G: 23.9%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2593588\t4543043.5\t0\t2593588\n",
      "2\t376254\t1135760.9\t0\t376254\n",
      "3\t210205\t283940.2\t0\t210205\n",
      "4\t119037\t70985.1\t0\t119037\n",
      "5\t104651\t17746.3\t0\t104651\n",
      "6\t89236\t4436.6\t0\t89236\n",
      "7\t73097\t1109.1\t0\t73097\n",
      "8\t103442\t277.3\t0\t103442\n",
      "9\t83266\t69.3\t0\t82975 291\n",
      "10\t90702\t17.3\t1\t86533 4169\n",
      "11\t96126\t4.3\t1\t90174 5952\n",
      "12\t96465\t1.1\t1\t90519 5946\n",
      "13\t107004\t0.3\t1\t100993 6011\n",
      "14\t100898\t0.3\t1\t93952 6946\n",
      "15\t82733\t0.3\t1\t77655 5078\n",
      "16\t119602\t0.3\t1\t112441 7161\n",
      "17\t94839\t0.3\t1\t88603 6236\n",
      "18\t92099\t0.3\t1\t86695 5404\n",
      "19\t114214\t0.3\t1\t107016 7198\n",
      "20\t107713\t0.3\t1\t101500 6213\n",
      "21\t95562\t0.3\t1\t89784 5778\n",
      "22\t107588\t0.3\t1\t101222 6366\n",
      "23\t111031\t0.3\t1\t104295 6736\n",
      "24\t110341\t0.3\t1\t103677 6664\n",
      "25\t124129\t0.3\t1\t117008 7121\n",
      "26\t103231\t0.3\t1\t96892 6339\n",
      "27\t114476\t0.3\t1\t106707 7769\n",
      "28\t107750\t0.3\t1\t102430 5320\n",
      "29\t114219\t0.3\t1\t107523 6696\n",
      "30\t113089\t0.3\t1\t107416 5673\n",
      "31\t123120\t0.3\t1\t116178 6942\n",
      "32\t110386\t0.3\t1\t104842 5544\n",
      "33\t123147\t0.3\t1\t116091 7056\n",
      "34\t156685\t0.3\t1\t148041 8644\n",
      "35\t89283\t0.3\t1\t84701 4582\n",
      "36\t120443\t0.3\t1\t113382 7061\n",
      "37\t129458\t0.3\t1\t122208 7250\n",
      "38\t122546\t0.3\t1\t116642 5904\n",
      "39\t119280\t0.3\t1\t113530 5750\n",
      "40\t132471\t0.3\t1\t125158 7313\n",
      "41\t126758\t0.3\t1\t120385 6373\n",
      "42\t130992\t0.3\t1\t124472 6520\n",
      "43\t120675\t0.3\t1\t114532 6143\n",
      "44\t136598\t0.3\t1\t129517 7081\n",
      "45\t144732\t0.3\t1\t137448 7284\n",
      "46\t106581\t0.3\t1\t101258 5323\n",
      "47\t132125\t0.3\t1\t125807 6318\n",
      "48\t130489\t0.3\t1\t124465 6024\n",
      "49\t135590\t0.3\t1\t129070 6520\n",
      "50\t128632\t0.3\t1\t122091 6541\n",
      "51\t159357\t0.3\t1\t151626 7731\n",
      "52\t116120\t0.3\t1\t110786 5334\n",
      "53\t139419\t0.3\t1\t133495 5924\n",
      "54\t106851\t0.3\t1\t101979 4872\n",
      "55\t135687\t0.3\t1\t129810 5877\n",
      "56\t138774\t0.3\t1\t132432 6342\n",
      "57\t131848\t0.3\t1\t126296 5552\n",
      "58\t127061\t0.3\t1\t121713 5348\n",
      "59\t129536\t0.3\t1\t124051 5485\n",
      "60\t133143\t0.3\t1\t127312 5831\n",
      "61\t136216\t0.3\t1\t129973 6243\n",
      "62\t147811\t0.3\t1\t140964 6847\n",
      "63\t197994\t0.3\t1\t190073 7921\n",
      "64\t157927\t0.3\t1\t152075 5852\n",
      "65\t105906\t0.3\t1\t101567 4339\n",
      "66\t114677\t0.3\t1\t109835 4842\n",
      "67\t114010\t0.3\t1\t108938 5072\n",
      "68\t114564\t0.3\t1\t109670 4894\n",
      "69\t112505\t0.3\t1\t107761 4744\n",
      "70\t118303\t0.3\t1\t113227 5076\n",
      "71\t123531\t0.3\t1\t118496 5035\n",
      "72\t117539\t0.3\t1\t112405 5134\n",
      "73\t122781\t0.3\t1\t117929 4852\n",
      "74\t113055\t0.3\t1\t108472 4583\n",
      "75\t107896\t0.3\t1\t103502 4394\n",
      "76\t106239\t0.3\t1\t101975 4264\n",
      "77\t105105\t0.3\t1\t100663 4442\n",
      "78\t99929\t0.3\t1\t95794 4135\n",
      "79\t96680\t0.3\t1\t92707 3973\n",
      "80\t95740\t0.3\t1\t91762 3978\n",
      "81\t93945\t0.3\t1\t90030 3915\n",
      "82\t89626\t0.3\t1\t86049 3577\n",
      "83\t87905\t0.3\t1\t84339 3566\n",
      "84\t84781\t0.3\t1\t81246 3535\n",
      "85\t83386\t0.3\t1\t79899 3487\n",
      "86\t83197\t0.3\t1\t79728 3469\n",
      "87\t82660\t0.3\t1\t79311 3349\n",
      "88\t81067\t0.3\t1\t77676 3391\n",
      "89\t73343\t0.3\t1\t70291 3052\n",
      "90\t69973\t0.3\t1\t67072 2901\n",
      "91\t65096\t0.3\t1\t62445 2651\n",
      "92\t61646\t0.3\t1\t59119 2527\n",
      "93\t59393\t0.3\t1\t56935 2458\n",
      "94\t57203\t0.3\t1\t54848 2355\n",
      "95\t53756\t0.3\t1\t51532 2224\n",
      "96\t50956\t0.3\t1\t49026 1930\n",
      "97\t47276\t0.3\t1\t45371 1905\n",
      "98\t44746\t0.3\t1\t43017 1729\n",
      "99\t42641\t0.3\t1\t41013 1628\n",
      "100\t39899\t0.3\t1\t38401 1498\n",
      "101\t36809\t0.3\t1\t35396 1413\n",
      "102\t34264\t0.3\t1\t32958 1306\n",
      "103\t31058\t0.3\t1\t29842 1216\n",
      "104\t30002\t0.3\t1\t28912 1090\n",
      "105\t26784\t0.3\t1\t25668 1116\n",
      "106\t24257\t0.3\t1\t23303 954\n",
      "107\t21941\t0.3\t1\t21141 800\n",
      "108\t20010\t0.3\t1\t19251 759\n",
      "109\t18313\t0.3\t1\t17584 729\n",
      "110\t16702\t0.3\t1\t16007 695\n",
      "111\t15559\t0.3\t1\t14968 591\n",
      "112\t13419\t0.3\t1\t12890 529\n",
      "113\t11631\t0.3\t1\t11113 518\n",
      "114\t10507\t0.3\t1\t10110 397\n",
      "115\t8927\t0.3\t1\t8561 366\n",
      "116\t7354\t0.3\t1\t7008 346\n",
      "117\t6258\t0.3\t1\t5993 265\n",
      "118\t5422\t0.3\t1\t5194 228\n",
      "119\t4575\t0.3\t1\t4365 210\n",
      "120\t3972\t0.3\t1\t3799 173\n",
      "121\t3338\t0.3\t1\t3179 159\n",
      "122\t2723\t0.3\t1\t2578 145\n",
      "123\t2312\t0.3\t1\t2208 104\n",
      "124\t2044\t0.3\t1\t1948 96\n",
      "125\t1725\t0.3\t1\t1623 102\n",
      "126\t1567\t0.3\t1\t1498 69\n",
      "127\t1207\t0.3\t1\t1135 72\n",
      "128\t957\t0.3\t1\t886 71\n",
      "129\t782\t0.3\t1\t739 43\n",
      "130\t614\t0.3\t1\t575 39\n",
      "131\t601\t0.3\t1\t565 36\n",
      "132\t573\t0.3\t1\t543 30\n",
      "133\t862\t0.3\t1\t829 33\n",
      "134\t1185\t0.3\t1\t1148 37\n",
      "135\t434\t0.3\t1\t412 22\n",
      "136\t191\t0.3\t1\t179 12\n",
      "137\t150\t0.3\t1\t137 13\n",
      "138\t67\t0.3\t1\t59 8\n",
      "139\t66\t0.3\t1\t55 11\n",
      "140\t41\t0.3\t1\t35 6\n",
      "141\t28\t0.3\t1\t25 3\n",
      "142\t18\t0.3\t1\t17 1\n",
      "143\t17\t0.3\t1\t17\n",
      "144\t10\t0.3\t1\t7 3\n",
      "145\t31\t0.3\t1\t30 1\n",
      "146\t24\t0.3\t1\t21 3\n",
      "147\t69\t0.3\t1\t63 6\n",
      "148\t53\t0.3\t1\t49 4\n",
      "149\t41\t0.3\t1\t39 2\n",
      "150\t371\t0.3\t1\t357 14\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18172174 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t23837 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 742.92 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,056,494\n",
      "Reads with adapters:                13,928,110 (77.1%)\n",
      "Reads written (passing filters):    18,056,494 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,708,474,100 bp\n",
      "Quality-trimmed:               5,741,117 bp (0.2%)\n",
      "Total written (filtered):  2,137,545,368 bp (78.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13928110 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 14.8%\n",
      "  G: 20.2%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2529563\t4514123.5\t0\t2529563\n",
      "2\t355142\t1128530.9\t0\t355142\n",
      "3\t202264\t282132.7\t0\t202264\n",
      "4\t116958\t70533.2\t0\t116958\n",
      "5\t97699\t17633.3\t0\t97699\n",
      "6\t92485\t4408.3\t0\t92485\n",
      "7\t72663\t1102.1\t0\t72663\n",
      "8\t94383\t275.5\t0\t94383\n",
      "9\t92644\t68.9\t0\t91510 1134\n",
      "10\t86522\t17.2\t1\t80894 5628\n",
      "11\t97786\t4.3\t1\t89346 8440\n",
      "12\t93298\t1.1\t1\t86415 6883\n",
      "13\t95771\t0.3\t1\t88325 7446\n",
      "14\t95946\t0.3\t1\t88058 7888\n",
      "15\t101399\t0.3\t1\t93509 7890\n",
      "16\t98158\t0.3\t1\t89606 8552\n",
      "17\t122597\t0.3\t1\t111303 11294\n",
      "18\t101750\t0.3\t1\t94436 7314\n",
      "19\t86831\t0.3\t1\t80395 6436\n",
      "20\t105075\t0.3\t1\t97137 7938\n",
      "21\t118601\t0.3\t1\t108999 9602\n",
      "22\t97228\t0.3\t1\t90470 6758\n",
      "23\t116927\t0.3\t1\t107741 9186\n",
      "24\t105532\t0.3\t1\t95756 9776\n",
      "25\t113828\t0.3\t1\t105061 8767\n",
      "26\t103778\t0.3\t1\t96608 7170\n",
      "27\t117560\t0.3\t1\t107874 9686\n",
      "28\t114336\t0.3\t1\t105504 8832\n",
      "29\t122580\t0.3\t1\t111680 10900\n",
      "30\t130099\t0.3\t1\t121451 8648\n",
      "31\t97669\t0.3\t1\t90483 7186\n",
      "32\t124295\t0.3\t1\t114657 9638\n",
      "33\t112476\t0.3\t1\t103286 9190\n",
      "34\t140784\t0.3\t1\t128812 11972\n",
      "35\t103370\t0.3\t1\t96154 7216\n",
      "36\t134329\t0.3\t1\t122737 11592\n",
      "37\t117550\t0.3\t1\t108946 8604\n",
      "38\t133588\t0.3\t1\t123334 10254\n",
      "39\t130482\t0.3\t1\t119347 11135\n",
      "40\t122686\t0.3\t1\t113961 8725\n",
      "41\t145432\t0.3\t1\t132894 12538\n",
      "42\t118079\t0.3\t1\t107701 10378\n",
      "43\t189764\t0.3\t1\t177939 11825\n",
      "44\t65407\t0.3\t1\t59230 6177\n",
      "45\t132188\t0.3\t1\t122729 9459\n",
      "46\t127290\t0.3\t1\t117848 9442\n",
      "47\t135499\t0.3\t1\t126055 9444\n",
      "48\t126038\t0.3\t1\t116373 9665\n",
      "49\t142407\t0.3\t1\t130956 11451\n",
      "50\t121520\t0.3\t1\t113382 8138\n",
      "51\t130837\t0.3\t1\t121331 9506\n",
      "52\t138254\t0.3\t1\t127263 10991\n",
      "53\t150154\t0.3\t1\t140134 10020\n",
      "54\t113941\t0.3\t1\t105808 8133\n",
      "55\t137202\t0.3\t1\t127382 9820\n",
      "56\t163618\t0.3\t1\t152691 10927\n",
      "57\t113941\t0.3\t1\t104904 9037\n",
      "58\t135543\t0.3\t1\t127222 8321\n",
      "59\t114833\t0.3\t1\t106375 8458\n",
      "60\t134934\t0.3\t1\t125495 9439\n",
      "61\t131069\t0.3\t1\t122212 8857\n",
      "62\t140049\t0.3\t1\t130678 9371\n",
      "63\t118477\t0.3\t1\t109768 8709\n",
      "64\t123661\t0.3\t1\t115672 7989\n",
      "65\t136910\t0.3\t1\t127445 9465\n",
      "66\t142766\t0.3\t1\t132932 9834\n",
      "67\t155161\t0.3\t1\t142407 12754\n",
      "68\t281796\t0.3\t1\t266464 15332\n",
      "69\t68541\t0.3\t1\t62954 5587\n",
      "70\t25774\t0.3\t1\t21904 3870\n",
      "71\t70954\t0.3\t1\t66131 4823\n",
      "72\t107448\t0.3\t1\t100862 6586\n",
      "73\t114621\t0.3\t1\t107221 7400\n",
      "74\t114007\t0.3\t1\t106477 7530\n",
      "75\t113035\t0.3\t1\t105200 7835\n",
      "76\t110785\t0.3\t1\t102896 7889\n",
      "77\t110971\t0.3\t1\t103289 7682\n",
      "78\t105110\t0.3\t1\t97785 7325\n",
      "79\t103881\t0.3\t1\t96578 7303\n",
      "80\t100380\t0.3\t1\t93184 7196\n",
      "81\t99197\t0.3\t1\t92210 6987\n",
      "82\t96750\t0.3\t1\t90051 6699\n",
      "83\t92818\t0.3\t1\t86379 6439\n",
      "84\t91340\t0.3\t1\t84942 6398\n",
      "85\t87669\t0.3\t1\t81553 6116\n",
      "86\t83869\t0.3\t1\t77984 5885\n",
      "87\t80883\t0.3\t1\t75364 5519\n",
      "88\t79450\t0.3\t1\t73820 5630\n",
      "89\t76461\t0.3\t1\t71121 5340\n",
      "90\t75182\t0.3\t1\t69973 5209\n",
      "91\t71175\t0.3\t1\t66391 4784\n",
      "92\t67480\t0.3\t1\t62890 4590\n",
      "93\t64001\t0.3\t1\t59615 4386\n",
      "94\t61685\t0.3\t1\t57457 4228\n",
      "95\t57220\t0.3\t1\t53201 4019\n",
      "96\t54819\t0.3\t1\t51015 3804\n",
      "97\t51023\t0.3\t1\t47551 3472\n",
      "98\t47615\t0.3\t1\t44173 3442\n",
      "99\t45999\t0.3\t1\t42533 3466\n",
      "100\t42439\t0.3\t1\t39454 2985\n",
      "101\t39032\t0.3\t1\t36135 2897\n",
      "102\t36673\t0.3\t1\t33901 2772\n",
      "103\t33569\t0.3\t1\t31176 2393\n",
      "104\t31500\t0.3\t1\t29209 2291\n",
      "105\t28075\t0.3\t1\t26011 2064\n",
      "106\t25509\t0.3\t1\t23615 1894\n",
      "107\t23441\t0.3\t1\t21761 1680\n",
      "108\t21666\t0.3\t1\t19921 1745\n",
      "109\t19354\t0.3\t1\t17847 1507\n",
      "110\t17443\t0.3\t1\t16060 1383\n",
      "111\t15864\t0.3\t1\t14569 1295\n",
      "112\t13943\t0.3\t1\t12833 1110\n",
      "113\t12164\t0.3\t1\t11191 973\n",
      "114\t10656\t0.3\t1\t9783 873\n",
      "115\t9192\t0.3\t1\t8432 760\n",
      "116\t8137\t0.3\t1\t7437 700\n",
      "117\t6646\t0.3\t1\t6079 567\n",
      "118\t6028\t0.3\t1\t5475 553\n",
      "119\t4941\t0.3\t1\t4488 453\n",
      "120\t4276\t0.3\t1\t3914 362\n",
      "121\t3804\t0.3\t1\t3440 364\n",
      "122\t2994\t0.3\t1\t2656 338\n",
      "123\t2568\t0.3\t1\t2298 270\n",
      "124\t2204\t0.3\t1\t2017 187\n",
      "125\t1837\t0.3\t1\t1657 180\n",
      "126\t1524\t0.3\t1\t1357 167\n",
      "127\t1388\t0.3\t1\t1226 162\n",
      "128\t1017\t0.3\t1\t890 127\n",
      "129\t886\t0.3\t1\t773 113\n",
      "130\t713\t0.3\t1\t646 67\n",
      "131\t614\t0.3\t1\t547 67\n",
      "132\t610\t0.3\t1\t554 56\n",
      "133\t738\t0.3\t1\t662 76\n",
      "134\t1198\t0.3\t1\t1126 72\n",
      "135\t479\t0.3\t1\t433 46\n",
      "136\t180\t0.3\t1\t151 29\n",
      "137\t120\t0.3\t1\t102 18\n",
      "138\t100\t0.3\t1\t79 21\n",
      "139\t54\t0.3\t1\t43 11\n",
      "140\t63\t0.3\t1\t49 14\n",
      "141\t34\t0.3\t1\t27 7\n",
      "142\t36\t0.3\t1\t27 9\n",
      "143\t19\t0.3\t1\t17 2\n",
      "144\t13\t0.3\t1\t11 2\n",
      "145\t29\t0.3\t1\t22 7\n",
      "146\t21\t0.3\t1\t18 3\n",
      "147\t72\t0.3\t1\t61 11\n",
      "148\t53\t0.3\t1\t46 7\n",
      "149\t48\t0.3\t1\t42 6\n",
      "150\t601\t0.3\t1\t530 71\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18056494 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5143 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_1_S9_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 779.38 s (43 us/read; 1.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,056,494\n",
      "Reads with adapters:                13,833,491 (76.6%)\n",
      "Reads written (passing filters):    18,056,494 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,708,474,100 bp\n",
      "Quality-trimmed:              12,349,451 bp (0.5%)\n",
      "Total written (filtered):  2,141,212,582 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13833491 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.5%\n",
      "  G: 24.0%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2568781\t4514123.5\t0\t2568781\n",
      "2\t366057\t1128530.9\t0\t366057\n",
      "3\t212077\t282132.7\t0\t212077\n",
      "4\t115719\t70533.2\t0\t115719\n",
      "5\t101735\t17633.3\t0\t101735\n",
      "6\t89696\t4408.3\t0\t89696\n",
      "7\t71264\t1102.1\t0\t71264\n",
      "8\t102329\t275.5\t0\t102329\n",
      "9\t81047\t68.9\t0\t80732 315\n",
      "10\t86697\t17.2\t1\t80878 5819\n",
      "11\t94697\t4.3\t1\t87025 7672\n",
      "12\t93974\t1.1\t1\t86609 7365\n",
      "13\t107111\t0.3\t1\t98790 8321\n",
      "14\t98830\t0.3\t1\t90363 8467\n",
      "15\t81863\t0.3\t1\t75568 6295\n",
      "16\t116617\t0.3\t1\t107560 9057\n",
      "17\t98434\t0.3\t1\t90554 7880\n",
      "18\t88102\t0.3\t1\t81367 6735\n",
      "19\t112551\t0.3\t1\t103902 8649\n",
      "20\t106499\t0.3\t1\t98559 7940\n",
      "21\t94529\t0.3\t1\t87451 7078\n",
      "22\t105525\t0.3\t1\t97752 7773\n",
      "23\t111250\t0.3\t1\t102948 8302\n",
      "24\t110289\t0.3\t1\t101801 8488\n",
      "25\t123454\t0.3\t1\t115174 8280\n",
      "26\t102700\t0.3\t1\t94614 8086\n",
      "27\t115670\t0.3\t1\t106461 9209\n",
      "28\t103454\t0.3\t1\t96996 6458\n",
      "29\t116676\t0.3\t1\t108241 8435\n",
      "30\t110369\t0.3\t1\t103124 7245\n",
      "31\t123948\t0.3\t1\t115651 8297\n",
      "32\t105488\t0.3\t1\t98700 6788\n",
      "33\t122972\t0.3\t1\t114148 8824\n",
      "34\t158602\t0.3\t1\t147860 10742\n",
      "35\t85176\t0.3\t1\t79388 5788\n",
      "36\t121188\t0.3\t1\t112686 8502\n",
      "37\t129603\t0.3\t1\t120852 8751\n",
      "38\t121650\t0.3\t1\t114412 7238\n",
      "39\t117499\t0.3\t1\t110622 6877\n",
      "40\t132022\t0.3\t1\t123436 8586\n",
      "41\t127439\t0.3\t1\t119665 7774\n",
      "42\t130897\t0.3\t1\t122691 8206\n",
      "43\t118466\t0.3\t1\t111262 7204\n",
      "44\t136805\t0.3\t1\t128115 8690\n",
      "45\t143267\t0.3\t1\t134250 9017\n",
      "46\t105232\t0.3\t1\t98689 6543\n",
      "47\t132632\t0.3\t1\t125125 7507\n",
      "48\t128580\t0.3\t1\t121138 7442\n",
      "49\t134590\t0.3\t1\t126196 8394\n",
      "50\t129680\t0.3\t1\t121594 8086\n",
      "51\t156936\t0.3\t1\t147378 9558\n",
      "52\t115575\t0.3\t1\t108787 6788\n",
      "53\t141220\t0.3\t1\t133568 7652\n",
      "54\t104058\t0.3\t1\t98240 5818\n",
      "55\t134625\t0.3\t1\t127372 7253\n",
      "56\t138566\t0.3\t1\t130360 8206\n",
      "57\t132925\t0.3\t1\t126016 6909\n",
      "58\t127424\t0.3\t1\t120849 6575\n",
      "59\t127371\t0.3\t1\t120396 6975\n",
      "60\t132296\t0.3\t1\t124969 7327\n",
      "61\t135056\t0.3\t1\t127808 7248\n",
      "62\t146657\t0.3\t1\t138388 8269\n",
      "63\t193947\t0.3\t1\t184650 9297\n",
      "64\t155657\t0.3\t1\t148867 6790\n",
      "65\t105922\t0.3\t1\t100517 5405\n",
      "66\t114816\t0.3\t1\t108826 5990\n",
      "67\t114246\t0.3\t1\t108023 6223\n",
      "68\t115039\t0.3\t1\t108957 6082\n",
      "69\t113573\t0.3\t1\t107431 6142\n",
      "70\t118689\t0.3\t1\t112276 6413\n",
      "71\t124010\t0.3\t1\t117553 6457\n",
      "72\t118164\t0.3\t1\t111917 6247\n",
      "73\t122662\t0.3\t1\t116340 6322\n",
      "74\t113578\t0.3\t1\t107524 6054\n",
      "75\t108812\t0.3\t1\t102905 5907\n",
      "76\t105084\t0.3\t1\t99284 5800\n",
      "77\t105545\t0.3\t1\t99708 5837\n",
      "78\t100872\t0.3\t1\t95568 5304\n",
      "79\t97026\t0.3\t1\t91844 5182\n",
      "80\t95305\t0.3\t1\t90356 4949\n",
      "81\t93857\t0.3\t1\t89014 4843\n",
      "82\t90295\t0.3\t1\t85582 4713\n",
      "83\t87301\t0.3\t1\t82821 4480\n",
      "84\t85471\t0.3\t1\t81218 4253\n",
      "85\t82978\t0.3\t1\t78704 4274\n",
      "86\t82245\t0.3\t1\t78140 4105\n",
      "87\t82525\t0.3\t1\t78515 4010\n",
      "88\t80619\t0.3\t1\t76785 3834\n",
      "89\t73370\t0.3\t1\t69724 3646\n",
      "90\t70068\t0.3\t1\t66584 3484\n",
      "91\t65835\t0.3\t1\t62601 3234\n",
      "92\t61556\t0.3\t1\t58566 2990\n",
      "93\t59482\t0.3\t1\t56574 2908\n",
      "94\t57109\t0.3\t1\t54269 2840\n",
      "95\t53973\t0.3\t1\t51267 2706\n",
      "96\t51596\t0.3\t1\t49048 2548\n",
      "97\t47434\t0.3\t1\t45017 2417\n",
      "98\t44668\t0.3\t1\t42540 2128\n",
      "99\t43184\t0.3\t1\t41061 2123\n",
      "100\t40483\t0.3\t1\t38511 1972\n",
      "101\t37015\t0.3\t1\t35171 1844\n",
      "102\t34179\t0.3\t1\t32525 1654\n",
      "103\t31254\t0.3\t1\t29799 1455\n",
      "104\t30003\t0.3\t1\t28488 1515\n",
      "105\t26714\t0.3\t1\t25377 1337\n",
      "106\t24481\t0.3\t1\t23332 1149\n",
      "107\t22254\t0.3\t1\t21123 1131\n",
      "108\t20003\t0.3\t1\t19023 980\n",
      "109\t18094\t0.3\t1\t17150 944\n",
      "110\t16326\t0.3\t1\t15479 847\n",
      "111\t15507\t0.3\t1\t14783 724\n",
      "112\t13504\t0.3\t1\t12807 697\n",
      "113\t11779\t0.3\t1\t11204 575\n",
      "114\t10207\t0.3\t1\t9684 523\n",
      "115\t8697\t0.3\t1\t8286 411\n",
      "116\t7460\t0.3\t1\t7104 356\n",
      "117\t6031\t0.3\t1\t5703 328\n",
      "118\t5441\t0.3\t1\t5182 259\n",
      "119\t4540\t0.3\t1\t4315 225\n",
      "120\t3964\t0.3\t1\t3742 222\n",
      "121\t3581\t0.3\t1\t3393 188\n",
      "122\t2805\t0.3\t1\t2657 148\n",
      "123\t2429\t0.3\t1\t2278 151\n",
      "124\t2051\t0.3\t1\t1954 97\n",
      "125\t1777\t0.3\t1\t1683 94\n",
      "126\t1416\t0.3\t1\t1338 78\n",
      "127\t1308\t0.3\t1\t1214 94\n",
      "128\t919\t0.3\t1\t855 64\n",
      "129\t823\t0.3\t1\t781 42\n",
      "130\t653\t0.3\t1\t614 39\n",
      "131\t581\t0.3\t1\t548 33\n",
      "132\t573\t0.3\t1\t541 32\n",
      "133\t789\t0.3\t1\t754 35\n",
      "134\t1231\t0.3\t1\t1179 52\n",
      "135\t458\t0.3\t1\t447 11\n",
      "136\t175\t0.3\t1\t164 11\n",
      "137\t112\t0.3\t1\t103 9\n",
      "138\t94\t0.3\t1\t84 10\n",
      "139\t57\t0.3\t1\t53 4\n",
      "140\t65\t0.3\t1\t58 7\n",
      "141\t32\t0.3\t1\t29 3\n",
      "142\t35\t0.3\t1\t31 4\n",
      "143\t17\t0.3\t1\t15 2\n",
      "144\t11\t0.3\t1\t9 2\n",
      "145\t30\t0.3\t1\t28 2\n",
      "146\t19\t0.3\t1\t19\n",
      "147\t64\t0.3\t1\t61 3\n",
      "148\t47\t0.3\t1\t44 3\n",
      "149\t34\t0.3\t1\t32 2\n",
      "150\t415\t0.3\t1\t377 38\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18056494 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t22077 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 725.23 s (39 us/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,482,577\n",
      "Reads with adapters:                14,846,196 (80.3%)\n",
      "Reads written (passing filters):    18,482,577 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,772,386,550 bp\n",
      "Quality-trimmed:               5,236,502 bp (0.2%)\n",
      "Total written (filtered):  2,109,664,469 bp (76.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14846196 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 14.7%\n",
      "  G: 20.4%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2201281\t4620644.2\t0\t2201281\n",
      "2\t324820\t1155161.1\t0\t324820\n",
      "3\t181187\t288790.3\t0\t181187\n",
      "4\t111024\t72197.6\t0\t111024\n",
      "5\t91444\t18049.4\t0\t91444\n",
      "6\t89339\t4512.3\t0\t89339\n",
      "7\t75499\t1128.1\t0\t75499\n",
      "8\t92079\t282.0\t0\t92079\n",
      "9\t94613\t70.5\t0\t93305 1308\n",
      "10\t87393\t17.6\t1\t82726 4667\n",
      "11\t97272\t4.4\t1\t89821 7451\n",
      "12\t94618\t1.1\t1\t88577 6041\n",
      "13\t94580\t0.3\t1\t88291 6289\n",
      "14\t98608\t0.3\t1\t91401 7207\n",
      "15\t100703\t0.3\t1\t94036 6667\n",
      "16\t100759\t0.3\t1\t93186 7573\n",
      "17\t122345\t0.3\t1\t112475 9870\n",
      "18\t103137\t0.3\t1\t96868 6269\n",
      "19\t89996\t0.3\t1\t84224 5772\n",
      "20\t107642\t0.3\t1\t100714 6928\n",
      "21\t118859\t0.3\t1\t110501 8358\n",
      "22\t101938\t0.3\t1\t96023 5915\n",
      "23\t117604\t0.3\t1\t109853 7751\n",
      "24\t110878\t0.3\t1\t101488 9390\n",
      "25\t118235\t0.3\t1\t110039 8196\n",
      "26\t108371\t0.3\t1\t101787 6584\n",
      "27\t121571\t0.3\t1\t112662 8909\n",
      "28\t119395\t0.3\t1\t110884 8511\n",
      "29\t129888\t0.3\t1\t119161 10727\n",
      "30\t136399\t0.3\t1\t128723 7676\n",
      "31\t101982\t0.3\t1\t95349 6633\n",
      "32\t129923\t0.3\t1\t121526 8397\n",
      "33\t121306\t0.3\t1\t112461 8845\n",
      "34\t134618\t0.3\t1\t125125 9493\n",
      "35\t135522\t0.3\t1\t127399 8123\n",
      "36\t126532\t0.3\t1\t116217 10315\n",
      "37\t132624\t0.3\t1\t124414 8210\n",
      "38\t135436\t0.3\t1\t126589 8847\n",
      "39\t144343\t0.3\t1\t135079 9264\n",
      "40\t131706\t0.3\t1\t123113 8593\n",
      "41\t147103\t0.3\t1\t136472 10631\n",
      "42\t135693\t0.3\t1\t124813 10880\n",
      "43\t208641\t0.3\t1\t197928 10713\n",
      "44\t70084\t0.3\t1\t63748 6336\n",
      "45\t146631\t0.3\t1\t137638 8993\n",
      "46\t134243\t0.3\t1\t125635 8608\n",
      "47\t148092\t0.3\t1\t138684 9408\n",
      "48\t143207\t0.3\t1\t133950 9257\n",
      "49\t157250\t0.3\t1\t146216 11034\n",
      "50\t135654\t0.3\t1\t127338 8316\n",
      "51\t145480\t0.3\t1\t136047 9433\n",
      "52\t151730\t0.3\t1\t141076 10654\n",
      "53\t160594\t0.3\t1\t150536 10058\n",
      "54\t131608\t0.3\t1\t123193 8415\n",
      "55\t153566\t0.3\t1\t143551 10015\n",
      "56\t188903\t0.3\t1\t177823 11080\n",
      "57\t122969\t0.3\t1\t114039 8930\n",
      "58\t155982\t0.3\t1\t147471 8511\n",
      "59\t136864\t0.3\t1\t128026 8838\n",
      "60\t152804\t0.3\t1\t142938 9866\n",
      "61\t149972\t0.3\t1\t141027 8945\n",
      "62\t164460\t0.3\t1\t154790 9670\n",
      "63\t131600\t0.3\t1\t122512 9088\n",
      "64\t147097\t0.3\t1\t138434 8663\n",
      "65\t163813\t0.3\t1\t153627 10186\n",
      "66\t170976\t0.3\t1\t160133 10843\n",
      "67\t181725\t0.3\t1\t167572 14153\n",
      "68\t347424\t0.3\t1\t330644 16780\n",
      "69\t53801\t0.3\t1\t49133 4668\n",
      "70\t21119\t0.3\t1\t17296 3823\n",
      "71\t93829\t0.3\t1\t88082 5747\n",
      "72\t129343\t0.3\t1\t121746 7597\n",
      "73\t137109\t0.3\t1\t128961 8148\n",
      "74\t135639\t0.3\t1\t127288 8351\n",
      "75\t134319\t0.3\t1\t126050 8269\n",
      "76\t133485\t0.3\t1\t125306 8179\n",
      "77\t133818\t0.3\t1\t125641 8177\n",
      "78\t125826\t0.3\t1\t118003 7823\n",
      "79\t124061\t0.3\t1\t116366 7695\n",
      "80\t122381\t0.3\t1\t114907 7474\n",
      "81\t121340\t0.3\t1\t113873 7467\n",
      "82\t118564\t0.3\t1\t111221 7343\n",
      "83\t114400\t0.3\t1\t107282 7118\n",
      "84\t110060\t0.3\t1\t103458 6602\n",
      "85\t107144\t0.3\t1\t100729 6415\n",
      "86\t104597\t0.3\t1\t98129 6468\n",
      "87\t99965\t0.3\t1\t93811 6154\n",
      "88\t99634\t0.3\t1\t93629 6005\n",
      "89\t94213\t0.3\t1\t88311 5902\n",
      "90\t93784\t0.3\t1\t87912 5872\n",
      "91\t88966\t0.3\t1\t83468 5498\n",
      "92\t84640\t0.3\t1\t79309 5331\n",
      "93\t79324\t0.3\t1\t74386 4938\n",
      "94\t77199\t0.3\t1\t72415 4784\n",
      "95\t72155\t0.3\t1\t67634 4521\n",
      "96\t68840\t0.3\t1\t64535 4305\n",
      "97\t64425\t0.3\t1\t60326 4099\n",
      "98\t60153\t0.3\t1\t56199 3954\n",
      "99\t58789\t0.3\t1\t54783 4006\n",
      "100\t53231\t0.3\t1\t49872 3359\n",
      "101\t49747\t0.3\t1\t46474 3273\n",
      "102\t46514\t0.3\t1\t43441 3073\n",
      "103\t42759\t0.3\t1\t39961 2798\n",
      "104\t40408\t0.3\t1\t37725 2683\n",
      "105\t35915\t0.3\t1\t33458 2457\n",
      "106\t33343\t0.3\t1\t31066 2277\n",
      "107\t30056\t0.3\t1\t28019 2037\n",
      "108\t28759\t0.3\t1\t26783 1976\n",
      "109\t25234\t0.3\t1\t23454 1780\n",
      "110\t22936\t0.3\t1\t21306 1630\n",
      "111\t20750\t0.3\t1\t19240 1510\n",
      "112\t17996\t0.3\t1\t16644 1352\n",
      "113\t15908\t0.3\t1\t14749 1159\n",
      "114\t14075\t0.3\t1\t12991 1084\n",
      "115\t12424\t0.3\t1\t11459 965\n",
      "116\t10747\t0.3\t1\t9910 837\n",
      "117\t9113\t0.3\t1\t8333 780\n",
      "118\t8109\t0.3\t1\t7445 664\n",
      "119\t6883\t0.3\t1\t6316 567\n",
      "120\t5713\t0.3\t1\t5209 504\n",
      "121\t5168\t0.3\t1\t4711 457\n",
      "122\t4185\t0.3\t1\t3804 381\n",
      "123\t3552\t0.3\t1\t3218 334\n",
      "124\t3100\t0.3\t1\t2796 304\n",
      "125\t2576\t0.3\t1\t2315 261\n",
      "126\t2168\t0.3\t1\t1937 231\n",
      "127\t1692\t0.3\t1\t1526 166\n",
      "128\t1332\t0.3\t1\t1165 167\n",
      "129\t1229\t0.3\t1\t1107 122\n",
      "130\t923\t0.3\t1\t838 85\n",
      "131\t751\t0.3\t1\t663 88\n",
      "132\t654\t0.3\t1\t579 75\n",
      "133\t679\t0.3\t1\t607 72\n",
      "134\t1033\t0.3\t1\t947 86\n",
      "135\t428\t0.3\t1\t390 38\n",
      "136\t228\t0.3\t1\t207 21\n",
      "137\t170\t0.3\t1\t154 16\n",
      "138\t136\t0.3\t1\t118 18\n",
      "139\t81\t0.3\t1\t70 11\n",
      "140\t68\t0.3\t1\t55 13\n",
      "141\t49\t0.3\t1\t43 6\n",
      "142\t55\t0.3\t1\t47 8\n",
      "143\t45\t0.3\t1\t36 9\n",
      "144\t21\t0.3\t1\t19 2\n",
      "145\t58\t0.3\t1\t54 4\n",
      "146\t122\t0.3\t1\t110 12\n",
      "147\t390\t0.3\t1\t355 35\n",
      "148\t248\t0.3\t1\t225 23\n",
      "149\t170\t0.3\t1\t149 21\n",
      "150\t1709\t0.3\t1\t1353 356\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18482577 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7148 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 760.84 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,482,577\n",
      "Reads with adapters:                14,753,598 (79.8%)\n",
      "Reads written (passing filters):    18,482,577 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,772,386,550 bp\n",
      "Quality-trimmed:              14,899,580 bp (0.5%)\n",
      "Total written (filtered):  2,113,888,686 bp (76.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14753598 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 12.5%\n",
      "  G: 24.2%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2286581\t4620644.2\t0\t2286581\n",
      "2\t330310\t1155161.1\t0\t330310\n",
      "3\t190033\t288790.3\t0\t190033\n",
      "4\t111417\t72197.6\t0\t111417\n",
      "5\t101556\t18049.4\t0\t101556\n",
      "6\t87150\t4512.3\t0\t87150\n",
      "7\t71559\t1128.1\t0\t71559\n",
      "8\t101450\t282.0\t0\t101450\n",
      "9\t81706\t70.5\t0\t81446 260\n",
      "10\t88967\t17.6\t1\t84555 4412\n",
      "11\t93801\t4.4\t1\t88117 5684\n",
      "12\t95662\t1.1\t1\t89704 5958\n",
      "13\t105924\t0.3\t1\t99758 6166\n",
      "14\t100634\t0.3\t1\t93826 6808\n",
      "15\t81502\t0.3\t1\t76286 5216\n",
      "16\t118419\t0.3\t1\t111030 7389\n",
      "17\t95042\t0.3\t1\t88712 6330\n",
      "18\t90615\t0.3\t1\t85225 5390\n",
      "19\t115837\t0.3\t1\t108486 7351\n",
      "20\t108535\t0.3\t1\t102151 6384\n",
      "21\t96158\t0.3\t1\t90241 5917\n",
      "22\t109681\t0.3\t1\t102998 6683\n",
      "23\t113064\t0.3\t1\t106255 6809\n",
      "24\t112402\t0.3\t1\t105369 7033\n",
      "25\t127609\t0.3\t1\t120247 7362\n",
      "26\t106198\t0.3\t1\t99534 6664\n",
      "27\t117379\t0.3\t1\t109394 7985\n",
      "28\t111310\t0.3\t1\t105607 5703\n",
      "29\t119257\t0.3\t1\t111876 7381\n",
      "30\t117422\t0.3\t1\t111230 6192\n",
      "31\t127799\t0.3\t1\t120363 7436\n",
      "32\t115547\t0.3\t1\t109515 6032\n",
      "33\t127479\t0.3\t1\t119672 7807\n",
      "34\t163653\t0.3\t1\t154089 9564\n",
      "35\t93410\t0.3\t1\t88452 4958\n",
      "36\t129603\t0.3\t1\t121811 7792\n",
      "37\t138172\t0.3\t1\t130055 8117\n",
      "38\t129571\t0.3\t1\t123089 6482\n",
      "39\t126899\t0.3\t1\t120584 6315\n",
      "40\t141131\t0.3\t1\t133007 8124\n",
      "41\t136856\t0.3\t1\t129602 7254\n",
      "42\t140475\t0.3\t1\t133264 7211\n",
      "43\t128682\t0.3\t1\t121911 6771\n",
      "44\t148231\t0.3\t1\t140352 7879\n",
      "45\t152393\t0.3\t1\t144497 7896\n",
      "46\t120446\t0.3\t1\t114128 6318\n",
      "47\t146135\t0.3\t1\t138949 7186\n",
      "48\t143271\t0.3\t1\t136268 7003\n",
      "49\t150045\t0.3\t1\t142543 7502\n",
      "50\t141618\t0.3\t1\t134295 7323\n",
      "51\t177184\t0.3\t1\t168273 8911\n",
      "52\t127868\t0.3\t1\t121986 5882\n",
      "53\t154234\t0.3\t1\t147269 6965\n",
      "54\t117590\t0.3\t1\t112052 5538\n",
      "55\t152743\t0.3\t1\t146054 6689\n",
      "56\t157680\t0.3\t1\t150094 7586\n",
      "57\t149551\t0.3\t1\t143020 6531\n",
      "58\t143483\t0.3\t1\t137470 6013\n",
      "59\t145998\t0.3\t1\t139352 6646\n",
      "60\t151232\t0.3\t1\t144310 6922\n",
      "61\t157550\t0.3\t1\t150162 7388\n",
      "62\t169489\t0.3\t1\t161516 7973\n",
      "63\t228128\t0.3\t1\t218668 9460\n",
      "64\t191978\t0.3\t1\t184960 7018\n",
      "65\t121739\t0.3\t1\t116395 5344\n",
      "66\t132352\t0.3\t1\t126524 5828\n",
      "67\t131028\t0.3\t1\t125146 5882\n",
      "68\t132829\t0.3\t1\t127004 5825\n",
      "69\t131726\t0.3\t1\t125866 5860\n",
      "70\t137789\t0.3\t1\t131635 6154\n",
      "71\t147946\t0.3\t1\t141769 6177\n",
      "72\t139415\t0.3\t1\t133242 6173\n",
      "73\t148725\t0.3\t1\t142498 6227\n",
      "74\t135207\t0.3\t1\t129643 5564\n",
      "75\t128520\t0.3\t1\t123034 5486\n",
      "76\t126065\t0.3\t1\t120765 5300\n",
      "77\t126228\t0.3\t1\t120717 5511\n",
      "78\t120120\t0.3\t1\t114978 5142\n",
      "79\t115495\t0.3\t1\t110595 4900\n",
      "80\t116213\t0.3\t1\t111348 4865\n",
      "81\t114795\t0.3\t1\t109987 4808\n",
      "82\t110451\t0.3\t1\t105814 4637\n",
      "83\t106907\t0.3\t1\t102515 4392\n",
      "84\t102431\t0.3\t1\t98006 4425\n",
      "85\t101678\t0.3\t1\t97283 4395\n",
      "86\t102018\t0.3\t1\t97662 4356\n",
      "87\t102379\t0.3\t1\t98265 4114\n",
      "88\t100269\t0.3\t1\t95923 4346\n",
      "89\t89766\t0.3\t1\t85935 3831\n",
      "90\t86353\t0.3\t1\t82825 3528\n",
      "91\t80857\t0.3\t1\t77356 3501\n",
      "92\t76306\t0.3\t1\t73107 3199\n",
      "93\t73595\t0.3\t1\t70492 3103\n",
      "94\t71587\t0.3\t1\t68546 3041\n",
      "95\t68110\t0.3\t1\t65320 2790\n",
      "96\t64867\t0.3\t1\t62264 2603\n",
      "97\t59245\t0.3\t1\t56835 2410\n",
      "98\t55305\t0.3\t1\t52942 2363\n",
      "99\t53891\t0.3\t1\t51732 2159\n",
      "100\t50005\t0.3\t1\t47952 2053\n",
      "101\t45894\t0.3\t1\t44011 1883\n",
      "102\t43298\t0.3\t1\t41576 1722\n",
      "103\t39322\t0.3\t1\t37781 1541\n",
      "104\t38059\t0.3\t1\t36523 1536\n",
      "105\t33918\t0.3\t1\t32538 1380\n",
      "106\t31438\t0.3\t1\t30241 1197\n",
      "107\t28204\t0.3\t1\t27113 1091\n",
      "108\t26258\t0.3\t1\t25170 1088\n",
      "109\t23534\t0.3\t1\t22556 978\n",
      "110\t21525\t0.3\t1\t20673 852\n",
      "111\t20335\t0.3\t1\t19508 827\n",
      "112\t17540\t0.3\t1\t16818 722\n",
      "113\t15580\t0.3\t1\t14966 614\n",
      "114\t13626\t0.3\t1\t13076 550\n",
      "115\t11545\t0.3\t1\t11044 501\n",
      "116\t9893\t0.3\t1\t9487 406\n",
      "117\t8220\t0.3\t1\t7857 363\n",
      "118\t7416\t0.3\t1\t7106 310\n",
      "119\t6364\t0.3\t1\t6067 297\n",
      "120\t5214\t0.3\t1\t4965 249\n",
      "121\t4800\t0.3\t1\t4601 199\n",
      "122\t3856\t0.3\t1\t3683 173\n",
      "123\t3424\t0.3\t1\t3271 153\n",
      "124\t2951\t0.3\t1\t2795 156\n",
      "125\t2462\t0.3\t1\t2355 107\n",
      "126\t2026\t0.3\t1\t1922 104\n",
      "127\t1586\t0.3\t1\t1509 77\n",
      "128\t1233\t0.3\t1\t1162 71\n",
      "129\t1142\t0.3\t1\t1091 51\n",
      "130\t841\t0.3\t1\t788 53\n",
      "131\t705\t0.3\t1\t671 34\n",
      "132\t609\t0.3\t1\t572 37\n",
      "133\t673\t0.3\t1\t638 35\n",
      "134\t1003\t0.3\t1\t960 43\n",
      "135\t407\t0.3\t1\t396 11\n",
      "136\t221\t0.3\t1\t202 19\n",
      "137\t162\t0.3\t1\t156 6\n",
      "138\t117\t0.3\t1\t109 8\n",
      "139\t71\t0.3\t1\t63 8\n",
      "140\t67\t0.3\t1\t63 4\n",
      "141\t48\t0.3\t1\t47 1\n",
      "142\t50\t0.3\t1\t49 1\n",
      "143\t41\t0.3\t1\t37 4\n",
      "144\t23\t0.3\t1\t20 3\n",
      "145\t62\t0.3\t1\t59 3\n",
      "146\t126\t0.3\t1\t123 3\n",
      "147\t381\t0.3\t1\t366 15\n",
      "148\t234\t0.3\t1\t226 8\n",
      "149\t166\t0.3\t1\t155 11\n",
      "150\t1412\t0.3\t1\t1344 68\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18482577 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t29885 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 730.98 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,388,474\n",
      "Reads with adapters:                14,774,548 (80.3%)\n",
      "Reads written (passing filters):    18,388,474 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,758,271,100 bp\n",
      "Quality-trimmed:               5,422,014 bp (0.2%)\n",
      "Total written (filtered):  2,096,179,695 bp (76.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14774548 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 14.7%\n",
      "  G: 20.4%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2179283\t4597118.5\t0\t2179283\n",
      "2\t319389\t1149279.6\t0\t319389\n",
      "3\t179573\t287319.9\t0\t179573\n",
      "4\t108126\t71830.0\t0\t108126\n",
      "5\t90944\t17957.5\t0\t90944\n",
      "6\t89410\t4489.4\t0\t89410\n",
      "7\t72797\t1122.3\t0\t72797\n",
      "8\t91422\t280.6\t0\t91422\n",
      "9\t93678\t70.1\t0\t92415 1263\n",
      "10\t84673\t17.5\t1\t79267 5406\n",
      "11\t97249\t4.4\t1\t88769 8480\n",
      "12\t93054\t1.1\t1\t86160 6894\n",
      "13\t93570\t0.3\t1\t86447 7123\n",
      "14\t97334\t0.3\t1\t89025 8309\n",
      "15\t99065\t0.3\t1\t91404 7661\n",
      "16\t99365\t0.3\t1\t90487 8878\n",
      "17\t122566\t0.3\t1\t111250 11316\n",
      "18\t102597\t0.3\t1\t95121 7476\n",
      "19\t88093\t0.3\t1\t81291 6802\n",
      "20\t107045\t0.3\t1\t98814 8231\n",
      "21\t119128\t0.3\t1\t109388 9740\n",
      "22\t98363\t0.3\t1\t91503 6860\n",
      "23\t118120\t0.3\t1\t108869 9251\n",
      "24\t109512\t0.3\t1\t99530 9982\n",
      "25\t117620\t0.3\t1\t108476 9144\n",
      "26\t107233\t0.3\t1\t99745 7488\n",
      "27\t121679\t0.3\t1\t111586 10093\n",
      "28\t118815\t0.3\t1\t109364 9451\n",
      "29\t129909\t0.3\t1\t118061 11848\n",
      "30\t136643\t0.3\t1\t127560 9083\n",
      "31\t99248\t0.3\t1\t91716 7532\n",
      "32\t129461\t0.3\t1\t119254 10207\n",
      "33\t120389\t0.3\t1\t110237 10152\n",
      "34\t133674\t0.3\t1\t122112 11562\n",
      "35\t135326\t0.3\t1\t125969 9357\n",
      "36\t125558\t0.3\t1\t113748 11810\n",
      "37\t131811\t0.3\t1\t122131 9680\n",
      "38\t134596\t0.3\t1\t124192 10404\n",
      "39\t144370\t0.3\t1\t133180 11190\n",
      "40\t130508\t0.3\t1\t120297 10211\n",
      "41\t146230\t0.3\t1\t133678 12552\n",
      "42\t134185\t0.3\t1\t121889 12296\n",
      "43\t205437\t0.3\t1\t192857 12580\n",
      "44\t71406\t0.3\t1\t64241 7165\n",
      "45\t146811\t0.3\t1\t136113 10698\n",
      "46\t133431\t0.3\t1\t123482 9949\n",
      "47\t146963\t0.3\t1\t136319 10644\n",
      "48\t142772\t0.3\t1\t132090 10682\n",
      "49\t156871\t0.3\t1\t144261 12610\n",
      "50\t134522\t0.3\t1\t125136 9386\n",
      "51\t145234\t0.3\t1\t134471 10763\n",
      "52\t151741\t0.3\t1\t139780 11961\n",
      "53\t160323\t0.3\t1\t148993 11330\n",
      "54\t129391\t0.3\t1\t119763 9628\n",
      "55\t153504\t0.3\t1\t142065 11439\n",
      "56\t189438\t0.3\t1\t176473 12965\n",
      "57\t121885\t0.3\t1\t111816 10069\n",
      "58\t156277\t0.3\t1\t146133 10144\n",
      "59\t138166\t0.3\t1\t127798 10368\n",
      "60\t151809\t0.3\t1\t140696 11113\n",
      "61\t148378\t0.3\t1\t138229 10149\n",
      "62\t164350\t0.3\t1\t153017 11333\n",
      "63\t132176\t0.3\t1\t121776 10400\n",
      "64\t146283\t0.3\t1\t136343 9940\n",
      "65\t162567\t0.3\t1\t150970 11597\n",
      "66\t169475\t0.3\t1\t157430 12045\n",
      "67\t180215\t0.3\t1\t164843 15372\n",
      "68\t335801\t0.3\t1\t317089 18712\n",
      "69\t66723\t0.3\t1\t60845 5878\n",
      "70\t26090\t0.3\t1\t21811 4279\n",
      "71\t91663\t0.3\t1\t85518 6145\n",
      "72\t129140\t0.3\t1\t120846 8294\n",
      "73\t135992\t0.3\t1\t126800 9192\n",
      "74\t136581\t0.3\t1\t127154 9427\n",
      "75\t134341\t0.3\t1\t124926 9415\n",
      "76\t133080\t0.3\t1\t123316 9764\n",
      "77\t133409\t0.3\t1\t123959 9450\n",
      "78\t126607\t0.3\t1\t117538 9069\n",
      "79\t125070\t0.3\t1\t116034 9036\n",
      "80\t121818\t0.3\t1\t113223 8595\n",
      "81\t121825\t0.3\t1\t113140 8685\n",
      "82\t118244\t0.3\t1\t109604 8640\n",
      "83\t114640\t0.3\t1\t106502 8138\n",
      "84\t109963\t0.3\t1\t101985 7978\n",
      "85\t107204\t0.3\t1\t99478 7726\n",
      "86\t104741\t0.3\t1\t97192 7549\n",
      "87\t100621\t0.3\t1\t93543 7078\n",
      "88\t98776\t0.3\t1\t91675 7101\n",
      "89\t94677\t0.3\t1\t87957 6720\n",
      "90\t94332\t0.3\t1\t87620 6712\n",
      "91\t89612\t0.3\t1\t83296 6316\n",
      "92\t85598\t0.3\t1\t79627 5971\n",
      "93\t79579\t0.3\t1\t73824 5755\n",
      "94\t77934\t0.3\t1\t72425 5509\n",
      "95\t72110\t0.3\t1\t66875 5235\n",
      "96\t68840\t0.3\t1\t63845 4995\n",
      "97\t64799\t0.3\t1\t60100 4699\n",
      "98\t60773\t0.3\t1\t56257 4516\n",
      "99\t58381\t0.3\t1\t53874 4507\n",
      "100\t53848\t0.3\t1\t49904 3944\n",
      "101\t49943\t0.3\t1\t46245 3698\n",
      "102\t46241\t0.3\t1\t42732 3509\n",
      "103\t43196\t0.3\t1\t39990 3206\n",
      "104\t40167\t0.3\t1\t37244 2923\n",
      "105\t35834\t0.3\t1\t33163 2671\n",
      "106\t33023\t0.3\t1\t30489 2534\n",
      "107\t30207\t0.3\t1\t27948 2259\n",
      "108\t28472\t0.3\t1\t26245 2227\n",
      "109\t25094\t0.3\t1\t23102 1992\n",
      "110\t22988\t0.3\t1\t21194 1794\n",
      "111\t20538\t0.3\t1\t18949 1589\n",
      "112\t18521\t0.3\t1\t17080 1441\n",
      "113\t15947\t0.3\t1\t14645 1302\n",
      "114\t14330\t0.3\t1\t13057 1273\n",
      "115\t12384\t0.3\t1\t11329 1055\n",
      "116\t11096\t0.3\t1\t10163 933\n",
      "117\t9114\t0.3\t1\t8363 751\n",
      "118\t8187\t0.3\t1\t7390 797\n",
      "119\t6803\t0.3\t1\t6183 620\n",
      "120\t5832\t0.3\t1\t5301 531\n",
      "121\t5072\t0.3\t1\t4620 452\n",
      "122\t4206\t0.3\t1\t3803 403\n",
      "123\t3440\t0.3\t1\t3104 336\n",
      "124\t3042\t0.3\t1\t2726 316\n",
      "125\t2501\t0.3\t1\t2242 259\n",
      "126\t2111\t0.3\t1\t1904 207\n",
      "127\t1791\t0.3\t1\t1610 181\n",
      "128\t1444\t0.3\t1\t1292 152\n",
      "129\t1148\t0.3\t1\t1034 114\n",
      "130\t877\t0.3\t1\t785 92\n",
      "131\t693\t0.3\t1\t607 86\n",
      "132\t646\t0.3\t1\t559 87\n",
      "133\t689\t0.3\t1\t596 93\n",
      "134\t1090\t0.3\t1\t1005 85\n",
      "135\t492\t0.3\t1\t440 52\n",
      "136\t216\t0.3\t1\t190 26\n",
      "137\t184\t0.3\t1\t157 27\n",
      "138\t124\t0.3\t1\t103 21\n",
      "139\t70\t0.3\t1\t60 10\n",
      "140\t61\t0.3\t1\t49 12\n",
      "141\t35\t0.3\t1\t30 5\n",
      "142\t23\t0.3\t1\t21 2\n",
      "143\t27\t0.3\t1\t25 2\n",
      "144\t40\t0.3\t1\t34 6\n",
      "145\t86\t0.3\t1\t71 15\n",
      "146\t101\t0.3\t1\t97 4\n",
      "147\t403\t0.3\t1\t365 38\n",
      "148\t275\t0.3\t1\t252 23\n",
      "149\t182\t0.3\t1\t169 13\n",
      "150\t1714\t0.3\t1\t1347 367\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18388474 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7210 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_2_S10_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 772.54 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,388,474\n",
      "Reads with adapters:                14,677,085 (79.8%)\n",
      "Reads written (passing filters):    18,388,474 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,758,271,100 bp\n",
      "Quality-trimmed:              14,282,591 bp (0.5%)\n",
      "Total written (filtered):  2,101,274,280 bp (76.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14677085 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 12.6%\n",
      "  G: 24.2%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2268778\t4597118.5\t0\t2268778\n",
      "2\t320662\t1149279.6\t0\t320662\n",
      "3\t193123\t287319.9\t0\t193123\n",
      "4\t107505\t71830.0\t0\t107505\n",
      "5\t98136\t17957.5\t0\t98136\n",
      "6\t87605\t4489.4\t0\t87605\n",
      "7\t69984\t1122.3\t0\t69984\n",
      "8\t100736\t280.6\t0\t100736\n",
      "9\t80882\t70.1\t0\t80636 246\n",
      "10\t85130\t17.5\t1\t79348 5782\n",
      "11\t93238\t4.4\t1\t85689 7549\n",
      "12\t92545\t1.1\t1\t85062 7483\n",
      "13\t106121\t0.3\t1\t97648 8473\n",
      "14\t98586\t0.3\t1\t90015 8571\n",
      "15\t79603\t0.3\t1\t73456 6147\n",
      "16\t116674\t0.3\t1\t107221 9453\n",
      "17\t98395\t0.3\t1\t90346 8049\n",
      "18\t87513\t0.3\t1\t80658 6855\n",
      "19\t114688\t0.3\t1\t105582 9106\n",
      "20\t107373\t0.3\t1\t99285 8088\n",
      "21\t94108\t0.3\t1\t86828 7280\n",
      "22\t106536\t0.3\t1\t98497 8039\n",
      "23\t113709\t0.3\t1\t104976 8733\n",
      "24\t111940\t0.3\t1\t103269 8671\n",
      "25\t127403\t0.3\t1\t118648 8755\n",
      "26\t106762\t0.3\t1\t98223 8539\n",
      "27\t118132\t0.3\t1\t108645 9487\n",
      "28\t106930\t0.3\t1\t99955 6975\n",
      "29\t120647\t0.3\t1\t111606 9041\n",
      "30\t115537\t0.3\t1\t107678 7859\n",
      "31\t127962\t0.3\t1\t118733 9229\n",
      "32\t110371\t0.3\t1\t102883 7488\n",
      "33\t128535\t0.3\t1\t119135 9400\n",
      "34\t166074\t0.3\t1\t154379 11695\n",
      "35\t88153\t0.3\t1\t82039 6114\n",
      "36\t130243\t0.3\t1\t120604 9639\n",
      "37\t137437\t0.3\t1\t127611 9826\n",
      "38\t129360\t0.3\t1\t121250 8110\n",
      "39\t122871\t0.3\t1\t115373 7498\n",
      "40\t140913\t0.3\t1\t131295 9618\n",
      "41\t137375\t0.3\t1\t128487 8888\n",
      "42\t139853\t0.3\t1\t130785 9068\n",
      "43\t126879\t0.3\t1\t118647 8232\n",
      "44\t149301\t0.3\t1\t139495 9806\n",
      "45\t151738\t0.3\t1\t141722 10016\n",
      "46\t118782\t0.3\t1\t111205 7577\n",
      "47\t144918\t0.3\t1\t136263 8655\n",
      "48\t142402\t0.3\t1\t133845 8557\n",
      "49\t148370\t0.3\t1\t138956 9414\n",
      "50\t143101\t0.3\t1\t133924 9177\n",
      "51\t175097\t0.3\t1\t164092 11005\n",
      "52\t127562\t0.3\t1\t119653 7909\n",
      "53\t156552\t0.3\t1\t147706 8846\n",
      "54\t114039\t0.3\t1\t107283 6756\n",
      "55\t151265\t0.3\t1\t142708 8557\n",
      "56\t156754\t0.3\t1\t146983 9771\n",
      "57\t149767\t0.3\t1\t141656 8111\n",
      "58\t144481\t0.3\t1\t136475 8006\n",
      "59\t145577\t0.3\t1\t137588 7989\n",
      "60\t151210\t0.3\t1\t142573 8637\n",
      "61\t154000\t0.3\t1\t145457 8543\n",
      "62\t169208\t0.3\t1\t159141 10067\n",
      "63\t223876\t0.3\t1\t212453 11423\n",
      "64\t187828\t0.3\t1\t179799 8029\n",
      "65\t121115\t0.3\t1\t114588 6527\n",
      "66\t133231\t0.3\t1\t126151 7080\n",
      "67\t130874\t0.3\t1\t123245 7629\n",
      "68\t134376\t0.3\t1\t127002 7374\n",
      "69\t132637\t0.3\t1\t125114 7523\n",
      "70\t138657\t0.3\t1\t130979 7678\n",
      "71\t148179\t0.3\t1\t140033 8146\n",
      "72\t141424\t0.3\t1\t133547 7877\n",
      "73\t147578\t0.3\t1\t139587 7991\n",
      "74\t136704\t0.3\t1\t129204 7500\n",
      "75\t129081\t0.3\t1\t122024 7057\n",
      "76\t126400\t0.3\t1\t119414 6986\n",
      "77\t125876\t0.3\t1\t118871 7005\n",
      "78\t121079\t0.3\t1\t114302 6777\n",
      "79\t116625\t0.3\t1\t110208 6417\n",
      "80\t115718\t0.3\t1\t109559 6159\n",
      "81\t115429\t0.3\t1\t109090 6339\n",
      "82\t109791\t0.3\t1\t103782 6009\n",
      "83\t107050\t0.3\t1\t101360 5690\n",
      "84\t102872\t0.3\t1\t97327 5545\n",
      "85\t102015\t0.3\t1\t96553 5462\n",
      "86\t101949\t0.3\t1\t96606 5343\n",
      "87\t102660\t0.3\t1\t97419 5241\n",
      "88\t99584\t0.3\t1\t94677 4907\n",
      "89\t90213\t0.3\t1\t85644 4569\n",
      "90\t87335\t0.3\t1\t82845 4490\n",
      "91\t81890\t0.3\t1\t77745 4145\n",
      "92\t77363\t0.3\t1\t73520 3843\n",
      "93\t73690\t0.3\t1\t70013 3677\n",
      "94\t72610\t0.3\t1\t68880 3730\n",
      "95\t68219\t0.3\t1\t64684 3535\n",
      "96\t65235\t0.3\t1\t61847 3388\n",
      "97\t59727\t0.3\t1\t56753 2974\n",
      "98\t56047\t0.3\t1\t53250 2797\n",
      "99\t53654\t0.3\t1\t50884 2770\n",
      "100\t50544\t0.3\t1\t48047 2497\n",
      "101\t46267\t0.3\t1\t43991 2276\n",
      "102\t43012\t0.3\t1\t40866 2146\n",
      "103\t39825\t0.3\t1\t37849 1976\n",
      "104\t37894\t0.3\t1\t35883 2011\n",
      "105\t33884\t0.3\t1\t32215 1669\n",
      "106\t31153\t0.3\t1\t29674 1479\n",
      "107\t28444\t0.3\t1\t27053 1391\n",
      "108\t25994\t0.3\t1\t24692 1302\n",
      "109\t23338\t0.3\t1\t22212 1126\n",
      "110\t21606\t0.3\t1\t20453 1153\n",
      "111\t20243\t0.3\t1\t19234 1009\n",
      "112\t17922\t0.3\t1\t16980 942\n",
      "113\t15507\t0.3\t1\t14731 776\n",
      "114\t13757\t0.3\t1\t13111 646\n",
      "115\t11617\t0.3\t1\t11010 607\n",
      "116\t10230\t0.3\t1\t9713 517\n",
      "117\t8353\t0.3\t1\t7888 465\n",
      "118\t7422\t0.3\t1\t7053 369\n",
      "119\t6284\t0.3\t1\t5939 345\n",
      "120\t5303\t0.3\t1\t5008 295\n",
      "121\t4751\t0.3\t1\t4456 295\n",
      "122\t3933\t0.3\t1\t3679 254\n",
      "123\t3312\t0.3\t1\t3148 164\n",
      "124\t2831\t0.3\t1\t2672 159\n",
      "125\t2374\t0.3\t1\t2211 163\n",
      "126\t2009\t0.3\t1\t1881 128\n",
      "127\t1689\t0.3\t1\t1578 111\n",
      "128\t1328\t0.3\t1\t1253 75\n",
      "129\t1073\t0.3\t1\t1017 56\n",
      "130\t798\t0.3\t1\t754 44\n",
      "131\t632\t0.3\t1\t599 33\n",
      "132\t607\t0.3\t1\t572 35\n",
      "133\t679\t0.3\t1\t639 40\n",
      "134\t1080\t0.3\t1\t1037 43\n",
      "135\t476\t0.3\t1\t455 21\n",
      "136\t209\t0.3\t1\t191 18\n",
      "137\t180\t0.3\t1\t172 8\n",
      "138\t119\t0.3\t1\t105 14\n",
      "139\t64\t0.3\t1\t58 6\n",
      "140\t54\t0.3\t1\t50 4\n",
      "141\t33\t0.3\t1\t30 3\n",
      "142\t22\t0.3\t1\t22\n",
      "143\t32\t0.3\t1\t26 6\n",
      "144\t37\t0.3\t1\t35 2\n",
      "145\t89\t0.3\t1\t87 2\n",
      "146\t113\t0.3\t1\t108 5\n",
      "147\t383\t0.3\t1\t362 21\n",
      "148\t262\t0.3\t1\t256 6\n",
      "149\t189\t0.3\t1\t171 18\n",
      "150\t1406\t0.3\t1\t1326 80\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18388474 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t27591 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 629.90 s (37 us/read; 1.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,221,011\n",
      "Reads with adapters:                14,907,348 (86.6%)\n",
      "Reads written (passing filters):    17,221,011 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,583,151,650 bp\n",
      "Quality-trimmed:               4,970,961 bp (0.2%)\n",
      "Total written (filtered):  1,823,787,303 bp (70.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14907348 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 14.0%\n",
      "  G: 22.2%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1339401\t4305252.8\t0\t1339401\n",
      "2\t221513\t1076313.2\t0\t221513\n",
      "3\t135887\t269078.3\t0\t135887\n",
      "4\t86877\t67269.6\t0\t86877\n",
      "5\t77129\t16817.4\t0\t77129\n",
      "6\t76711\t4204.3\t0\t76711\n",
      "7\t64192\t1051.1\t0\t64192\n",
      "8\t80722\t262.8\t0\t80722\n",
      "9\t81072\t65.7\t0\t80149 923\n",
      "10\t77051\t16.4\t1\t73224 3827\n",
      "11\t85961\t4.1\t1\t79699 6262\n",
      "12\t83198\t1.0\t1\t78237 4961\n",
      "13\t86825\t0.3\t1\t81365 5460\n",
      "14\t84806\t0.3\t1\t79388 5418\n",
      "15\t92317\t0.3\t1\t86654 5663\n",
      "16\t89335\t0.3\t1\t83153 6182\n",
      "17\t108017\t0.3\t1\t99755 8262\n",
      "18\t99925\t0.3\t1\t94377 5548\n",
      "19\t81101\t0.3\t1\t76294 4807\n",
      "20\t100416\t0.3\t1\t94250 6166\n",
      "21\t108388\t0.3\t1\t101446 6942\n",
      "22\t94203\t0.3\t1\t89329 4874\n",
      "23\t109171\t0.3\t1\t102371 6800\n",
      "24\t104496\t0.3\t1\t96618 7878\n",
      "25\t111703\t0.3\t1\t104639 7064\n",
      "26\t103882\t0.3\t1\t98042 5840\n",
      "27\t115801\t0.3\t1\t107736 8065\n",
      "28\t113315\t0.3\t1\t105981 7334\n",
      "29\t122732\t0.3\t1\t113578 9154\n",
      "30\t129896\t0.3\t1\t123034 6862\n",
      "31\t104938\t0.3\t1\t98897 6041\n",
      "32\t124822\t0.3\t1\t117479 7343\n",
      "33\t120235\t0.3\t1\t112658 7577\n",
      "34\t135691\t0.3\t1\t125410 10281\n",
      "35\t143924\t0.3\t1\t135732 8192\n",
      "36\t114731\t0.3\t1\t107772 6959\n",
      "37\t138113\t0.3\t1\t128706 9407\n",
      "38\t183834\t0.3\t1\t174741 9093\n",
      "39\t91269\t0.3\t1\t83934 7335\n",
      "40\t144200\t0.3\t1\t135345 8855\n",
      "41\t167039\t0.3\t1\t155843 11196\n",
      "42\t124510\t0.3\t1\t116014 8496\n",
      "43\t229902\t0.3\t1\t218995 10907\n",
      "44\t61751\t0.3\t1\t56507 5244\n",
      "45\t153093\t0.3\t1\t144067 9026\n",
      "46\t145804\t0.3\t1\t137245 8559\n",
      "47\t154404\t0.3\t1\t145632 8772\n",
      "48\t154492\t0.3\t1\t145156 9336\n",
      "49\t169124\t0.3\t1\t158402 10722\n",
      "50\t144085\t0.3\t1\t136047 8038\n",
      "51\t159010\t0.3\t1\t149344 9666\n",
      "52\t165606\t0.3\t1\t154687 10919\n",
      "53\t178026\t0.3\t1\t167943 10083\n",
      "54\t145073\t0.3\t1\t136817 8256\n",
      "55\t172012\t0.3\t1\t161558 10454\n",
      "56\t210944\t0.3\t1\t199588 11356\n",
      "57\t131803\t0.3\t1\t123033 8770\n",
      "58\t178194\t0.3\t1\t169188 9006\n",
      "59\t157761\t0.3\t1\t148537 9224\n",
      "60\t170653\t0.3\t1\t160454 10199\n",
      "61\t170629\t0.3\t1\t161244 9385\n",
      "62\t190264\t0.3\t1\t180168 10096\n",
      "63\t148311\t0.3\t1\t139056 9255\n",
      "64\t170495\t0.3\t1\t161109 9386\n",
      "65\t191349\t0.3\t1\t180352 10997\n",
      "66\t199623\t0.3\t1\t188220 11403\n",
      "67\t212270\t0.3\t1\t197036 15234\n",
      "68\t412894\t0.3\t1\t394953 17941\n",
      "69\t59319\t0.3\t1\t54157 5162\n",
      "70\t23953\t0.3\t1\t19703 4250\n",
      "71\t116809\t0.3\t1\t110410 6399\n",
      "72\t157277\t0.3\t1\t148914 8363\n",
      "73\t165912\t0.3\t1\t156892 9020\n",
      "74\t164518\t0.3\t1\t155310 9208\n",
      "75\t163178\t0.3\t1\t154082 9096\n",
      "76\t162323\t0.3\t1\t153580 8743\n",
      "77\t162174\t0.3\t1\t153149 9025\n",
      "78\t154081\t0.3\t1\t145262 8819\n",
      "79\t153448\t0.3\t1\t144767 8681\n",
      "80\t149998\t0.3\t1\t141770 8228\n",
      "81\t150056\t0.3\t1\t141671 8385\n",
      "82\t147165\t0.3\t1\t139005 8160\n",
      "83\t142399\t0.3\t1\t134518 7881\n",
      "84\t137532\t0.3\t1\t129880 7652\n",
      "85\t134287\t0.3\t1\t126887 7400\n",
      "86\t129443\t0.3\t1\t122227 7216\n",
      "87\t125510\t0.3\t1\t118421 7089\n",
      "88\t124525\t0.3\t1\t117462 7063\n",
      "89\t118345\t0.3\t1\t111665 6680\n",
      "90\t118308\t0.3\t1\t111670 6638\n",
      "91\t111912\t0.3\t1\t105689 6223\n",
      "92\t106445\t0.3\t1\t100472 5973\n",
      "93\t100557\t0.3\t1\t94827 5730\n",
      "94\t97750\t0.3\t1\t92088 5662\n",
      "95\t91346\t0.3\t1\t86041 5305\n",
      "96\t86885\t0.3\t1\t81735 5150\n",
      "97\t81399\t0.3\t1\t76644 4755\n",
      "98\t75784\t0.3\t1\t71395 4389\n",
      "99\t72904\t0.3\t1\t68310 4594\n",
      "100\t67617\t0.3\t1\t63610 4007\n",
      "101\t63261\t0.3\t1\t59615 3646\n",
      "102\t58086\t0.3\t1\t54536 3550\n",
      "103\t54397\t0.3\t1\t51127 3270\n",
      "104\t50493\t0.3\t1\t47351 3142\n",
      "105\t44481\t0.3\t1\t41579 2902\n",
      "106\t41013\t0.3\t1\t38506 2507\n",
      "107\t36987\t0.3\t1\t34603 2384\n",
      "108\t34766\t0.3\t1\t32511 2255\n",
      "109\t30744\t0.3\t1\t28764 1980\n",
      "110\t27509\t0.3\t1\t25598 1911\n",
      "111\t24797\t0.3\t1\t23127 1670\n",
      "112\t21804\t0.3\t1\t20294 1510\n",
      "113\t18744\t0.3\t1\t17435 1309\n",
      "114\t16804\t0.3\t1\t15658 1146\n",
      "115\t14315\t0.3\t1\t13251 1064\n",
      "116\t12612\t0.3\t1\t11627 985\n",
      "117\t10473\t0.3\t1\t9684 789\n",
      "118\t8921\t0.3\t1\t8216 705\n",
      "119\t7563\t0.3\t1\t6922 641\n",
      "120\t6332\t0.3\t1\t5781 551\n",
      "121\t5619\t0.3\t1\t5122 497\n",
      "122\t4576\t0.3\t1\t4198 378\n",
      "123\t3737\t0.3\t1\t3370 367\n",
      "124\t3104\t0.3\t1\t2823 281\n",
      "125\t2623\t0.3\t1\t2320 303\n",
      "126\t2119\t0.3\t1\t1927 192\n",
      "127\t1819\t0.3\t1\t1640 179\n",
      "128\t1359\t0.3\t1\t1208 151\n",
      "129\t1165\t0.3\t1\t1033 132\n",
      "130\t852\t0.3\t1\t749 103\n",
      "131\t856\t0.3\t1\t765 91\n",
      "132\t726\t0.3\t1\t633 93\n",
      "133\t874\t0.3\t1\t793 81\n",
      "134\t1491\t0.3\t1\t1401 90\n",
      "135\t611\t0.3\t1\t548 63\n",
      "136\t207\t0.3\t1\t187 20\n",
      "137\t166\t0.3\t1\t155 11\n",
      "138\t132\t0.3\t1\t107 25\n",
      "139\t75\t0.3\t1\t67 8\n",
      "140\t69\t0.3\t1\t60 9\n",
      "141\t42\t0.3\t1\t32 10\n",
      "142\t26\t0.3\t1\t19 7\n",
      "143\t26\t0.3\t1\t15 11\n",
      "144\t18\t0.3\t1\t13 5\n",
      "145\t37\t0.3\t1\t33 4\n",
      "146\t26\t0.3\t1\t20 6\n",
      "147\t73\t0.3\t1\t60 13\n",
      "148\t47\t0.3\t1\t37 10\n",
      "149\t55\t0.3\t1\t48 7\n",
      "150\t636\t0.3\t1\t548 88\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17221011 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6252 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 667.73 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,221,011\n",
      "Reads with adapters:                14,791,093 (85.9%)\n",
      "Reads written (passing filters):    17,221,011 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,583,151,650 bp\n",
      "Quality-trimmed:              13,446,962 bp (0.5%)\n",
      "Total written (filtered):  1,830,093,875 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14791093 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 12.3%\n",
      "  G: 25.7%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1396647\t4305252.8\t0\t1396647\n",
      "2\t233877\t1076313.2\t0\t233877\n",
      "3\t144492\t269078.3\t0\t144492\n",
      "4\t87098\t67269.6\t0\t87098\n",
      "5\t86055\t16817.4\t0\t86055\n",
      "6\t74999\t4204.3\t0\t74999\n",
      "7\t60602\t1051.1\t0\t60602\n",
      "8\t88349\t262.8\t0\t88349\n",
      "9\t70295\t65.7\t0\t70075 220\n",
      "10\t78524\t16.4\t1\t74626 3898\n",
      "11\t82895\t4.1\t1\t77738 5157\n",
      "12\t84476\t1.0\t1\t79233 5243\n",
      "13\t94326\t0.3\t1\t88847 5479\n",
      "14\t89984\t0.3\t1\t83845 6139\n",
      "15\t72879\t0.3\t1\t68520 4359\n",
      "16\t107100\t0.3\t1\t100533 6567\n",
      "17\t87871\t0.3\t1\t82033 5838\n",
      "18\t83139\t0.3\t1\t78397 4742\n",
      "19\t107298\t0.3\t1\t100402 6896\n",
      "20\t100171\t0.3\t1\t94390 5781\n",
      "21\t88114\t0.3\t1\t82713 5401\n",
      "22\t101076\t0.3\t1\t95306 5770\n",
      "23\t105877\t0.3\t1\t99537 6340\n",
      "24\t106645\t0.3\t1\t100049 6596\n",
      "25\t121932\t0.3\t1\t115168 6764\n",
      "26\t100476\t0.3\t1\t94344 6132\n",
      "27\t110166\t0.3\t1\t102758 7408\n",
      "28\t107176\t0.3\t1\t101924 5252\n",
      "29\t114111\t0.3\t1\t107493 6618\n",
      "30\t115359\t0.3\t1\t109345 6014\n",
      "31\t124714\t0.3\t1\t117673 7041\n",
      "32\t113114\t0.3\t1\t107249 5865\n",
      "33\t126653\t0.3\t1\t119283 7370\n",
      "34\t161416\t0.3\t1\t152249 9167\n",
      "35\t95165\t0.3\t1\t90366 4799\n",
      "36\t130060\t0.3\t1\t122395 7665\n",
      "37\t137916\t0.3\t1\t130202 7714\n",
      "38\t132706\t0.3\t1\t126385 6321\n",
      "39\t129385\t0.3\t1\t123068 6317\n",
      "40\t145600\t0.3\t1\t137496 8104\n",
      "41\t141184\t0.3\t1\t134070 7114\n",
      "42\t147655\t0.3\t1\t140213 7442\n",
      "43\t133334\t0.3\t1\t126611 6723\n",
      "44\t155705\t0.3\t1\t147696 8009\n",
      "45\t171369\t0.3\t1\t162810 8559\n",
      "46\t117297\t0.3\t1\t111483 5814\n",
      "47\t154795\t0.3\t1\t147535 7260\n",
      "48\t153124\t0.3\t1\t145909 7215\n",
      "49\t161825\t0.3\t1\t153802 8023\n",
      "50\t152605\t0.3\t1\t145077 7528\n",
      "51\t196303\t0.3\t1\t186610 9693\n",
      "52\t140348\t0.3\t1\t134075 6273\n",
      "53\t169943\t0.3\t1\t162909 7034\n",
      "54\t129681\t0.3\t1\t123901 5780\n",
      "55\t171452\t0.3\t1\t164350 7102\n",
      "56\t175614\t0.3\t1\t167693 7921\n",
      "57\t166121\t0.3\t1\t159314 6807\n",
      "58\t161387\t0.3\t1\t155069 6318\n",
      "59\t166619\t0.3\t1\t159538 7081\n",
      "60\t171867\t0.3\t1\t164363 7504\n",
      "61\t178391\t0.3\t1\t170609 7782\n",
      "62\t196679\t0.3\t1\t188081 8598\n",
      "63\t261274\t0.3\t1\t251029 10245\n",
      "64\t223191\t0.3\t1\t215765 7426\n",
      "65\t140786\t0.3\t1\t135022 5764\n",
      "66\t154538\t0.3\t1\t148311 6227\n",
      "67\t154027\t0.3\t1\t147524 6503\n",
      "68\t155618\t0.3\t1\t149295 6323\n",
      "69\t156069\t0.3\t1\t149599 6470\n",
      "70\t165853\t0.3\t1\t158998 6855\n",
      "71\t174385\t0.3\t1\t167574 6811\n",
      "72\t166867\t0.3\t1\t160011 6856\n",
      "73\t177560\t0.3\t1\t170679 6881\n",
      "74\t163422\t0.3\t1\t157175 6247\n",
      "75\t155704\t0.3\t1\t149734 5970\n",
      "76\t153010\t0.3\t1\t147031 5979\n",
      "77\t153177\t0.3\t1\t146921 6256\n",
      "78\t148366\t0.3\t1\t142418 5948\n",
      "79\t143302\t0.3\t1\t137662 5640\n",
      "80\t143393\t0.3\t1\t137924 5469\n",
      "81\t142595\t0.3\t1\t137094 5501\n",
      "82\t136572\t0.3\t1\t131469 5103\n",
      "83\t132737\t0.3\t1\t127619 5118\n",
      "84\t127991\t0.3\t1\t122783 5208\n",
      "85\t126965\t0.3\t1\t121917 5048\n",
      "86\t125977\t0.3\t1\t120895 5082\n",
      "87\t128246\t0.3\t1\t123231 5015\n",
      "88\t125283\t0.3\t1\t120412 4871\n",
      "89\t112372\t0.3\t1\t108049 4323\n",
      "90\t108791\t0.3\t1\t104559 4232\n",
      "91\t102461\t0.3\t1\t98288 4173\n",
      "92\t96447\t0.3\t1\t92722 3725\n",
      "93\t93869\t0.3\t1\t90286 3583\n",
      "94\t91252\t0.3\t1\t87743 3509\n",
      "95\t86357\t0.3\t1\t83052 3305\n",
      "96\t82971\t0.3\t1\t79818 3153\n",
      "97\t76434\t0.3\t1\t73514 2920\n",
      "98\t72048\t0.3\t1\t69275 2773\n",
      "99\t70150\t0.3\t1\t67632 2518\n",
      "100\t63666\t0.3\t1\t61347 2319\n",
      "101\t59861\t0.3\t1\t57732 2129\n",
      "102\t55598\t0.3\t1\t53622 1976\n",
      "103\t51026\t0.3\t1\t49092 1934\n",
      "104\t48386\t0.3\t1\t46675 1711\n",
      "105\t42639\t0.3\t1\t41080 1559\n",
      "106\t39613\t0.3\t1\t38200 1413\n",
      "107\t35228\t0.3\t1\t33949 1279\n",
      "108\t31897\t0.3\t1\t30649 1248\n",
      "109\t28672\t0.3\t1\t27578 1094\n",
      "110\t25914\t0.3\t1\t24951 963\n",
      "111\t24295\t0.3\t1\t23481 814\n",
      "112\t21304\t0.3\t1\t20453 851\n",
      "113\t18350\t0.3\t1\t17653 697\n",
      "114\t16256\t0.3\t1\t15627 629\n",
      "115\t13357\t0.3\t1\t12793 564\n",
      "116\t11612\t0.3\t1\t11150 462\n",
      "117\t9484\t0.3\t1\t9116 368\n",
      "118\t8205\t0.3\t1\t7826 379\n",
      "119\t6936\t0.3\t1\t6629 307\n",
      "120\t5765\t0.3\t1\t5525 240\n",
      "121\t5231\t0.3\t1\t5015 216\n",
      "122\t4247\t0.3\t1\t4080 167\n",
      "123\t3547\t0.3\t1\t3416 131\n",
      "124\t2936\t0.3\t1\t2808 128\n",
      "125\t2425\t0.3\t1\t2314 111\n",
      "126\t2011\t0.3\t1\t1915 96\n",
      "127\t1702\t0.3\t1\t1617 85\n",
      "128\t1242\t0.3\t1\t1178 64\n",
      "129\t1065\t0.3\t1\t1006 59\n",
      "130\t766\t0.3\t1\t721 45\n",
      "131\t781\t0.3\t1\t739 42\n",
      "132\t668\t0.3\t1\t634 34\n",
      "133\t887\t0.3\t1\t866 21\n",
      "134\t1489\t0.3\t1\t1430 59\n",
      "135\t583\t0.3\t1\t560 23\n",
      "136\t206\t0.3\t1\t191 15\n",
      "137\t151\t0.3\t1\t146 5\n",
      "138\t116\t0.3\t1\t111 5\n",
      "139\t72\t0.3\t1\t65 7\n",
      "140\t56\t0.3\t1\t54 2\n",
      "141\t35\t0.3\t1\t27 8\n",
      "142\t30\t0.3\t1\t26 4\n",
      "143\t21\t0.3\t1\t18 3\n",
      "144\t19\t0.3\t1\t17 2\n",
      "145\t37\t0.3\t1\t35 2\n",
      "146\t25\t0.3\t1\t22 3\n",
      "147\t50\t0.3\t1\t46 4\n",
      "148\t39\t0.3\t1\t37 2\n",
      "149\t45\t0.3\t1\t42 3\n",
      "150\t451\t0.3\t1\t418 33\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17221011 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t26089 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 637.91 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,175,482\n",
      "Reads with adapters:                14,848,929 (86.5%)\n",
      "Reads written (passing filters):    17,175,482 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,322,300 bp\n",
      "Quality-trimmed:               5,047,974 bp (0.2%)\n",
      "Total written (filtered):  1,818,633,191 bp (70.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14848929 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 14.0%\n",
      "  G: 22.1%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1338924\t4293870.5\t0\t1338924\n",
      "2\t219387\t1073467.6\t0\t219387\n",
      "3\t135027\t268366.9\t0\t135027\n",
      "4\t85370\t67091.7\t0\t85370\n",
      "5\t77474\t16772.9\t0\t77474\n",
      "6\t77037\t4193.2\t0\t77037\n",
      "7\t61988\t1048.3\t0\t61988\n",
      "8\t79862\t262.1\t0\t79862\n",
      "9\t79304\t65.5\t0\t78415 889\n",
      "10\t74894\t16.4\t1\t70380 4514\n",
      "11\t85385\t4.1\t1\t78459 6926\n",
      "12\t81941\t1.0\t1\t76091 5850\n",
      "13\t85904\t0.3\t1\t79720 6184\n",
      "14\t83830\t0.3\t1\t77435 6395\n",
      "15\t91251\t0.3\t1\t84448 6803\n",
      "16\t87362\t0.3\t1\t80132 7230\n",
      "17\t108027\t0.3\t1\t98268 9759\n",
      "18\t97765\t0.3\t1\t91194 6571\n",
      "19\t79668\t0.3\t1\t74013 5655\n",
      "20\t98415\t0.3\t1\t91325 7090\n",
      "21\t108861\t0.3\t1\t100754 8107\n",
      "22\t92432\t0.3\t1\t86497 5935\n",
      "23\t109093\t0.3\t1\t101170 7923\n",
      "24\t102961\t0.3\t1\t94241 8720\n",
      "25\t110529\t0.3\t1\t102334 8195\n",
      "26\t102489\t0.3\t1\t95783 6706\n",
      "27\t115669\t0.3\t1\t106602 9067\n",
      "28\t113524\t0.3\t1\t105254 8270\n",
      "29\t122345\t0.3\t1\t111712 10633\n",
      "30\t129585\t0.3\t1\t121424 8161\n",
      "31\t102733\t0.3\t1\t95540 7193\n",
      "32\t124071\t0.3\t1\t115107 8964\n",
      "33\t119235\t0.3\t1\t110316 8919\n",
      "34\t135312\t0.3\t1\t123053 12259\n",
      "35\t144848\t0.3\t1\t135072 9776\n",
      "36\t111252\t0.3\t1\t103350 7902\n",
      "37\t137690\t0.3\t1\t126540 11150\n",
      "38\t183318\t0.3\t1\t172054 11264\n",
      "39\t89425\t0.3\t1\t80789 8636\n",
      "40\t143455\t0.3\t1\t132529 10926\n",
      "41\t167032\t0.3\t1\t153468 13564\n",
      "42\t121530\t0.3\t1\t111758 9772\n",
      "43\t226339\t0.3\t1\t213169 13170\n",
      "44\t63952\t0.3\t1\t57726 6226\n",
      "45\t153236\t0.3\t1\t142659 10577\n",
      "46\t145389\t0.3\t1\t135235 10154\n",
      "47\t153266\t0.3\t1\t143025 10241\n",
      "48\t152876\t0.3\t1\t141831 11045\n",
      "49\t168461\t0.3\t1\t156017 12444\n",
      "50\t143668\t0.3\t1\t134304 9364\n",
      "51\t158306\t0.3\t1\t147182 11124\n",
      "52\t165500\t0.3\t1\t153119 12381\n",
      "53\t177823\t0.3\t1\t165831 11992\n",
      "54\t142891\t0.3\t1\t133186 9705\n",
      "55\t170342\t0.3\t1\t158433 11909\n",
      "56\t212493\t0.3\t1\t198685 13808\n",
      "57\t130099\t0.3\t1\t120181 9918\n",
      "58\t177795\t0.3\t1\t167072 10723\n",
      "59\t156878\t0.3\t1\t145935 10943\n",
      "60\t169887\t0.3\t1\t158157 11730\n",
      "61\t170084\t0.3\t1\t159037 11047\n",
      "62\t189821\t0.3\t1\t177611 12210\n",
      "63\t149110\t0.3\t1\t138396 10714\n",
      "64\t169980\t0.3\t1\t159400 10580\n",
      "65\t190750\t0.3\t1\t178067 12683\n",
      "66\t196470\t0.3\t1\t183447 13023\n",
      "67\t212313\t0.3\t1\t195505 16808\n",
      "68\t400251\t0.3\t1\t379902 20349\n",
      "69\t75488\t0.3\t1\t68954 6534\n",
      "70\t29970\t0.3\t1\t25043 4927\n",
      "71\t114346\t0.3\t1\t107216 7130\n",
      "72\t155823\t0.3\t1\t146728 9095\n",
      "73\t164934\t0.3\t1\t154662 10272\n",
      "74\t163683\t0.3\t1\t153217 10466\n",
      "75\t161985\t0.3\t1\t151549 10436\n",
      "76\t161854\t0.3\t1\t151132 10722\n",
      "77\t161503\t0.3\t1\t151032 10471\n",
      "78\t154856\t0.3\t1\t144180 10676\n",
      "79\t152863\t0.3\t1\t142631 10232\n",
      "80\t150353\t0.3\t1\t140171 10182\n",
      "81\t150454\t0.3\t1\t140509 9945\n",
      "82\t145676\t0.3\t1\t135805 9871\n",
      "83\t141824\t0.3\t1\t132558 9266\n",
      "84\t137190\t0.3\t1\t128021 9169\n",
      "85\t134663\t0.3\t1\t125634 9029\n",
      "86\t130352\t0.3\t1\t121592 8760\n",
      "87\t125876\t0.3\t1\t117794 8082\n",
      "88\t125030\t0.3\t1\t116803 8227\n",
      "89\t117792\t0.3\t1\t109945 7847\n",
      "90\t117830\t0.3\t1\t110051 7779\n",
      "91\t112910\t0.3\t1\t105546 7364\n",
      "92\t106310\t0.3\t1\t99476 6834\n",
      "93\t100338\t0.3\t1\t93657 6681\n",
      "94\t97374\t0.3\t1\t90887 6487\n",
      "95\t91711\t0.3\t1\t85495 6216\n",
      "96\t86651\t0.3\t1\t80705 5946\n",
      "97\t81512\t0.3\t1\t76160 5352\n",
      "98\t75813\t0.3\t1\t70557 5256\n",
      "99\t73615\t0.3\t1\t68470 5145\n",
      "100\t67314\t0.3\t1\t62734 4580\n",
      "101\t63335\t0.3\t1\t58922 4413\n",
      "102\t58250\t0.3\t1\t54144 4106\n",
      "103\t54120\t0.3\t1\t50374 3746\n",
      "104\t50265\t0.3\t1\t46857 3408\n",
      "105\t44626\t0.3\t1\t41570 3056\n",
      "106\t40793\t0.3\t1\t37936 2857\n",
      "107\t37321\t0.3\t1\t34670 2651\n",
      "108\t34705\t0.3\t1\t32091 2614\n",
      "109\t31002\t0.3\t1\t28677 2325\n",
      "110\t27578\t0.3\t1\t25574 2004\n",
      "111\t24603\t0.3\t1\t22785 1818\n",
      "112\t21972\t0.3\t1\t20247 1725\n",
      "113\t18720\t0.3\t1\t17311 1409\n",
      "114\t16688\t0.3\t1\t15435 1253\n",
      "115\t14432\t0.3\t1\t13232 1200\n",
      "116\t12245\t0.3\t1\t11232 1013\n",
      "117\t10552\t0.3\t1\t9672 880\n",
      "118\t8984\t0.3\t1\t8244 740\n",
      "119\t7381\t0.3\t1\t6703 678\n",
      "120\t6463\t0.3\t1\t5860 603\n",
      "121\t5515\t0.3\t1\t4963 552\n",
      "122\t4409\t0.3\t1\t4006 403\n",
      "123\t3731\t0.3\t1\t3377 354\n",
      "124\t3097\t0.3\t1\t2824 273\n",
      "125\t2558\t0.3\t1\t2286 272\n",
      "126\t2102\t0.3\t1\t1889 213\n",
      "127\t1729\t0.3\t1\t1530 199\n",
      "128\t1388\t0.3\t1\t1248 140\n",
      "129\t1146\t0.3\t1\t1007 139\n",
      "130\t884\t0.3\t1\t785 99\n",
      "131\t740\t0.3\t1\t651 89\n",
      "132\t734\t0.3\t1\t647 87\n",
      "133\t973\t0.3\t1\t886 87\n",
      "134\t1537\t0.3\t1\t1449 88\n",
      "135\t609\t0.3\t1\t551 58\n",
      "136\t234\t0.3\t1\t208 26\n",
      "137\t157\t0.3\t1\t138 19\n",
      "138\t104\t0.3\t1\t87 17\n",
      "139\t80\t0.3\t1\t60 20\n",
      "140\t68\t0.3\t1\t51 17\n",
      "141\t33\t0.3\t1\t25 8\n",
      "142\t34\t0.3\t1\t26 8\n",
      "143\t16\t0.3\t1\t16\n",
      "144\t16\t0.3\t1\t14 2\n",
      "145\t39\t0.3\t1\t27 12\n",
      "146\t32\t0.3\t1\t26 6\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t60\t0.3\t1\t51 9\n",
      "149\t55\t0.3\t1\t49 6\n",
      "150\t652\t0.3\t1\t552 100\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17175482 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6308 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_3_S11_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 678.15 s (39 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,175,482\n",
      "Reads with adapters:                14,720,565 (85.7%)\n",
      "Reads written (passing filters):    17,175,482 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,322,300 bp\n",
      "Quality-trimmed:              12,927,719 bp (0.5%)\n",
      "Total written (filtered):  1,825,809,518 bp (70.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14720565 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 12.3%\n",
      "  G: 25.7%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1394679\t4293870.5\t0\t1394679\n",
      "2\t231063\t1073467.6\t0\t231063\n",
      "3\t146836\t268366.9\t0\t146836\n",
      "4\t85160\t67091.7\t0\t85160\n",
      "5\t83607\t16772.9\t0\t83607\n",
      "6\t75270\t4193.2\t0\t75270\n",
      "7\t59401\t1048.3\t0\t59401\n",
      "8\t87249\t262.1\t0\t87249\n",
      "9\t68581\t65.5\t0\t68364 217\n",
      "10\t75306\t16.4\t1\t69907 5399\n",
      "11\t81844\t4.1\t1\t75220 6624\n",
      "12\t81567\t1.0\t1\t74921 6646\n",
      "13\t94550\t0.3\t1\t87043 7507\n",
      "14\t88612\t0.3\t1\t80763 7849\n",
      "15\t71080\t0.3\t1\t65748 5332\n",
      "16\t104508\t0.3\t1\t96265 8243\n",
      "17\t90450\t0.3\t1\t83158 7292\n",
      "18\t79309\t0.3\t1\t73193 6116\n",
      "19\t106248\t0.3\t1\t97745 8503\n",
      "20\t97943\t0.3\t1\t90478 7465\n",
      "21\t87195\t0.3\t1\t80598 6597\n",
      "22\t99625\t0.3\t1\t92109 7516\n",
      "23\t105941\t0.3\t1\t98004 7937\n",
      "24\t105924\t0.3\t1\t97524 8400\n",
      "25\t120959\t0.3\t1\t112658 8301\n",
      "26\t100855\t0.3\t1\t92739 8116\n",
      "27\t111349\t0.3\t1\t102191 9158\n",
      "28\t103794\t0.3\t1\t97081 6713\n",
      "29\t115018\t0.3\t1\t106647 8371\n",
      "30\t112098\t0.3\t1\t104443 7655\n",
      "31\t125293\t0.3\t1\t116600 8693\n",
      "32\t108268\t0.3\t1\t101076 7192\n",
      "33\t127668\t0.3\t1\t118305 9363\n",
      "34\t163116\t0.3\t1\t151740 11376\n",
      "35\t90695\t0.3\t1\t84564 6131\n",
      "36\t130252\t0.3\t1\t120794 9458\n",
      "37\t137508\t0.3\t1\t128167 9341\n",
      "38\t131385\t0.3\t1\t123153 8232\n",
      "39\t125429\t0.3\t1\t117971 7458\n",
      "40\t144394\t0.3\t1\t134617 9777\n",
      "41\t142244\t0.3\t1\t133129 9115\n",
      "42\t146211\t0.3\t1\t136886 9325\n",
      "43\t130656\t0.3\t1\t122570 8086\n",
      "44\t158062\t0.3\t1\t147788 10274\n",
      "45\t169410\t0.3\t1\t158451 10959\n",
      "46\t116471\t0.3\t1\t109140 7331\n",
      "47\t153771\t0.3\t1\t145129 8642\n",
      "48\t151126\t0.3\t1\t142022 9104\n",
      "49\t158775\t0.3\t1\t148794 9981\n",
      "50\t155083\t0.3\t1\t145500 9583\n",
      "51\t193098\t0.3\t1\t181154 11944\n",
      "52\t138930\t0.3\t1\t130761 8169\n",
      "53\t172888\t0.3\t1\t163380 9508\n",
      "54\t126226\t0.3\t1\t118962 7264\n",
      "55\t168553\t0.3\t1\t159300 9253\n",
      "56\t174469\t0.3\t1\t163938 10531\n",
      "57\t166262\t0.3\t1\t157805 8457\n",
      "58\t161940\t0.3\t1\t153447 8493\n",
      "59\t163516\t0.3\t1\t154551 8965\n",
      "60\t171750\t0.3\t1\t162391 9359\n",
      "61\t176379\t0.3\t1\t166983 9396\n",
      "62\t194831\t0.3\t1\t183858 10973\n",
      "63\t257381\t0.3\t1\t244702 12679\n",
      "64\t218615\t0.3\t1\t209942 8673\n",
      "65\t141285\t0.3\t1\t134011 7274\n",
      "66\t154574\t0.3\t1\t146600 7974\n",
      "67\t155286\t0.3\t1\t146846 8440\n",
      "68\t158519\t0.3\t1\t150042 8477\n",
      "69\t156924\t0.3\t1\t148115 8809\n",
      "70\t167343\t0.3\t1\t158462 8881\n",
      "71\t174948\t0.3\t1\t165808 9140\n",
      "72\t168193\t0.3\t1\t159291 8902\n",
      "73\t175743\t0.3\t1\t166626 9117\n",
      "74\t163090\t0.3\t1\t154296 8794\n",
      "75\t155077\t0.3\t1\t146999 8078\n",
      "76\t152837\t0.3\t1\t144700 8137\n",
      "77\t152701\t0.3\t1\t144644 8057\n",
      "78\t148868\t0.3\t1\t140946 7922\n",
      "79\t143248\t0.3\t1\t135788 7460\n",
      "80\t143729\t0.3\t1\t136194 7535\n",
      "81\t142811\t0.3\t1\t135360 7451\n",
      "82\t135758\t0.3\t1\t128640 7118\n",
      "83\t132076\t0.3\t1\t125333 6743\n",
      "84\t127993\t0.3\t1\t121417 6576\n",
      "85\t127663\t0.3\t1\t121292 6371\n",
      "86\t126712\t0.3\t1\t120456 6256\n",
      "87\t127809\t0.3\t1\t121761 6048\n",
      "88\t125518\t0.3\t1\t119426 6092\n",
      "89\t112440\t0.3\t1\t106917 5523\n",
      "90\t109295\t0.3\t1\t103850 5445\n",
      "91\t103669\t0.3\t1\t98616 5053\n",
      "92\t96386\t0.3\t1\t91748 4638\n",
      "93\t94174\t0.3\t1\t89557 4617\n",
      "94\t91197\t0.3\t1\t86759 4438\n",
      "95\t86770\t0.3\t1\t82553 4217\n",
      "96\t83121\t0.3\t1\t79129 3992\n",
      "97\t76650\t0.3\t1\t72963 3687\n",
      "98\t72211\t0.3\t1\t68803 3408\n",
      "99\t70867\t0.3\t1\t67509 3358\n",
      "100\t63540\t0.3\t1\t60596 2944\n",
      "101\t60139\t0.3\t1\t57240 2899\n",
      "102\t55801\t0.3\t1\t53158 2643\n",
      "103\t50487\t0.3\t1\t47969 2518\n",
      "104\t48235\t0.3\t1\t45902 2333\n",
      "105\t42853\t0.3\t1\t40742 2111\n",
      "106\t39140\t0.3\t1\t37246 1894\n",
      "107\t35386\t0.3\t1\t33624 1762\n",
      "108\t31865\t0.3\t1\t30331 1534\n",
      "109\t29026\t0.3\t1\t27576 1450\n",
      "110\t25978\t0.3\t1\t24750 1228\n",
      "111\t24137\t0.3\t1\t23015 1122\n",
      "112\t21305\t0.3\t1\t20312 993\n",
      "113\t18391\t0.3\t1\t17493 898\n",
      "114\t16197\t0.3\t1\t15347 850\n",
      "115\t13468\t0.3\t1\t12821 647\n",
      "116\t11291\t0.3\t1\t10728 563\n",
      "117\t9599\t0.3\t1\t9121 478\n",
      "118\t8211\t0.3\t1\t7795 416\n",
      "119\t6772\t0.3\t1\t6417 355\n",
      "120\t5885\t0.3\t1\t5586 299\n",
      "121\t5082\t0.3\t1\t4799 283\n",
      "122\t4085\t0.3\t1\t3891 194\n",
      "123\t3540\t0.3\t1\t3373 167\n",
      "124\t2933\t0.3\t1\t2771 162\n",
      "125\t2424\t0.3\t1\t2286 138\n",
      "126\t1979\t0.3\t1\t1877 102\n",
      "127\t1585\t0.3\t1\t1504 81\n",
      "128\t1307\t0.3\t1\t1233 74\n",
      "129\t1065\t0.3\t1\t999 66\n",
      "130\t818\t0.3\t1\t765 53\n",
      "131\t695\t0.3\t1\t661 34\n",
      "132\t692\t0.3\t1\t651 41\n",
      "133\t1029\t0.3\t1\t991 38\n",
      "134\t1555\t0.3\t1\t1499 56\n",
      "135\t581\t0.3\t1\t564 17\n",
      "136\t217\t0.3\t1\t210 7\n",
      "137\t148\t0.3\t1\t133 15\n",
      "138\t96\t0.3\t1\t87 9\n",
      "139\t66\t0.3\t1\t63 3\n",
      "140\t62\t0.3\t1\t58 4\n",
      "141\t31\t0.3\t1\t28 3\n",
      "142\t29\t0.3\t1\t25 4\n",
      "143\t15\t0.3\t1\t15\n",
      "144\t16\t0.3\t1\t16\n",
      "145\t37\t0.3\t1\t37\n",
      "146\t31\t0.3\t1\t30 1\n",
      "147\t76\t0.3\t1\t72 4\n",
      "148\t44\t0.3\t1\t40 4\n",
      "149\t38\t0.3\t1\t34 4\n",
      "150\t422\t0.3\t1\t392 30\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17175482 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t24770 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 507.33 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,592,704\n",
      "Reads with adapters:                 8,958,992 (71.1%)\n",
      "Reads written (passing filters):    12,592,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,888,905,600 bp\n",
      "Quality-trimmed:               4,951,530 bp (0.3%)\n",
      "Total written (filtered):  1,571,128,982 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8958992 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.4%\n",
      "  C: 15.5%\n",
      "  G: 17.3%\n",
      "  T: 35.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2360389\t3148176.0\t0\t2360389\n",
      "2\t294874\t787044.0\t0\t294874\n",
      "3\t161693\t196761.0\t0\t161693\n",
      "4\t87416\t49190.2\t0\t87416\n",
      "5\t59722\t12297.6\t0\t59722\n",
      "6\t57196\t3074.4\t0\t57196\n",
      "7\t45727\t768.6\t0\t45727\n",
      "8\t58159\t192.1\t0\t58159\n",
      "9\t58762\t48.0\t0\t57759 1003\n",
      "10\t54824\t12.0\t1\t51199 3625\n",
      "11\t61143\t3.0\t1\t55667 5476\n",
      "12\t58318\t0.8\t1\t53717 4601\n",
      "13\t56920\t0.2\t1\t52502 4418\n",
      "14\t60297\t0.2\t1\t55050 5247\n",
      "15\t60573\t0.2\t1\t55708 4865\n",
      "16\t61479\t0.2\t1\t55954 5525\n",
      "17\t74409\t0.2\t1\t67438 6971\n",
      "18\t61009\t0.2\t1\t56681 4328\n",
      "19\t53089\t0.2\t1\t48849 4240\n",
      "20\t63972\t0.2\t1\t58870 5102\n",
      "21\t70043\t0.2\t1\t64287 5756\n",
      "22\t58590\t0.2\t1\t54441 4149\n",
      "23\t68150\t0.2\t1\t62728 5422\n",
      "24\t64229\t0.2\t1\t58023 6206\n",
      "25\t68130\t0.2\t1\t62567 5563\n",
      "26\t62420\t0.2\t1\t57820 4600\n",
      "27\t69993\t0.2\t1\t63874 6119\n",
      "28\t67875\t0.2\t1\t62387 5488\n",
      "29\t71196\t0.2\t1\t64376 6820\n",
      "30\t75799\t0.2\t1\t70433 5366\n",
      "31\t59056\t0.2\t1\t54472 4584\n",
      "32\t73671\t0.2\t1\t68075 5596\n",
      "33\t63947\t0.2\t1\t58700 5247\n",
      "34\t74583\t0.2\t1\t67925 6658\n",
      "35\t70104\t0.2\t1\t64152 5952\n",
      "36\t77250\t0.2\t1\t71393 5857\n",
      "37\t68309\t0.2\t1\t62839 5470\n",
      "38\t72505\t0.2\t1\t67371 5134\n",
      "39\t75409\t0.2\t1\t69866 5543\n",
      "40\t72478\t0.2\t1\t66543 5935\n",
      "41\t78235\t0.2\t1\t70876 7359\n",
      "42\t73557\t0.2\t1\t66866 6691\n",
      "43\t105788\t0.2\t1\t99220 6568\n",
      "44\t39203\t0.2\t1\t34910 4293\n",
      "45\t75980\t0.2\t1\t70161 5819\n",
      "46\t74377\t0.2\t1\t68346 6031\n",
      "47\t78776\t0.2\t1\t72951 5825\n",
      "48\t70677\t0.2\t1\t64756 5921\n",
      "49\t81336\t0.2\t1\t74410 6926\n",
      "50\t66909\t0.2\t1\t62092 4817\n",
      "51\t72737\t0.2\t1\t67212 5525\n",
      "52\t78934\t0.2\t1\t71865 7069\n",
      "53\t86579\t0.2\t1\t80550 6029\n",
      "54\t62022\t0.2\t1\t57173 4849\n",
      "55\t75974\t0.2\t1\t70348 5626\n",
      "56\t85877\t0.2\t1\t79807 6070\n",
      "57\t71505\t0.2\t1\t65936 5569\n",
      "58\t69856\t0.2\t1\t65606 4250\n",
      "59\t57167\t0.2\t1\t52447 4720\n",
      "60\t78076\t0.2\t1\t72356 5720\n",
      "61\t73472\t0.2\t1\t68201 5271\n",
      "62\t71844\t0.2\t1\t66745 5099\n",
      "63\t67131\t0.2\t1\t62070 5061\n",
      "64\t65598\t0.2\t1\t60901 4697\n",
      "65\t70779\t0.2\t1\t65435 5344\n",
      "66\t77405\t0.2\t1\t71580 5825\n",
      "67\t84608\t0.2\t1\t76761 7847\n",
      "68\t150617\t0.2\t1\t142058 8559\n",
      "69\t37132\t0.2\t1\t33982 3150\n",
      "70\t13103\t0.2\t1\t10914 2189\n",
      "71\t35026\t0.2\t1\t32099 2927\n",
      "72\t57779\t0.2\t1\t53691 4088\n",
      "73\t61230\t0.2\t1\t56931 4299\n",
      "74\t61166\t0.2\t1\t56777 4389\n",
      "75\t60302\t0.2\t1\t55958 4344\n",
      "76\t59928\t0.2\t1\t55484 4444\n",
      "77\t59440\t0.2\t1\t54970 4470\n",
      "78\t56149\t0.2\t1\t51928 4221\n",
      "79\t55286\t0.2\t1\t51002 4284\n",
      "80\t53792\t0.2\t1\t49846 3946\n",
      "81\t52637\t0.2\t1\t48756 3881\n",
      "82\t51226\t0.2\t1\t47353 3873\n",
      "83\t49579\t0.2\t1\t45690 3889\n",
      "84\t48468\t0.2\t1\t44896 3572\n",
      "85\t45867\t0.2\t1\t42408 3459\n",
      "86\t44887\t0.2\t1\t41555 3332\n",
      "87\t42251\t0.2\t1\t38964 3287\n",
      "88\t42216\t0.2\t1\t39029 3187\n",
      "89\t40149\t0.2\t1\t37066 3083\n",
      "90\t40387\t0.2\t1\t37102 3285\n",
      "91\t37214\t0.2\t1\t34343 2871\n",
      "92\t35614\t0.2\t1\t32860 2754\n",
      "93\t33363\t0.2\t1\t30676 2687\n",
      "94\t32052\t0.2\t1\t29598 2454\n",
      "95\t29858\t0.2\t1\t27450 2408\n",
      "96\t28594\t0.2\t1\t26301 2293\n",
      "97\t26730\t0.2\t1\t24584 2146\n",
      "98\t25261\t0.2\t1\t23132 2129\n",
      "99\t24232\t0.2\t1\t22120 2112\n",
      "100\t22159\t0.2\t1\t20358 1801\n",
      "101\t20488\t0.2\t1\t18713 1775\n",
      "102\t19117\t0.2\t1\t17450 1667\n",
      "103\t17468\t0.2\t1\t15984 1484\n",
      "104\t16181\t0.2\t1\t14728 1453\n",
      "105\t14521\t0.2\t1\t13231 1290\n",
      "106\t13349\t0.2\t1\t12105 1244\n",
      "107\t12159\t0.2\t1\t11005 1154\n",
      "108\t11376\t0.2\t1\t10302 1074\n",
      "109\t10099\t0.2\t1\t9151 948\n",
      "110\t9379\t0.2\t1\t8450 929\n",
      "111\t8577\t0.2\t1\t7720 857\n",
      "112\t7623\t0.2\t1\t6797 826\n",
      "113\t6486\t0.2\t1\t5771 715\n",
      "114\t5676\t0.2\t1\t5112 564\n",
      "115\t5089\t0.2\t1\t4501 588\n",
      "116\t4254\t0.2\t1\t3781 473\n",
      "117\t3599\t0.2\t1\t3183 416\n",
      "118\t3110\t0.2\t1\t2758 352\n",
      "119\t2723\t0.2\t1\t2363 360\n",
      "120\t2307\t0.2\t1\t1995 312\n",
      "121\t2051\t0.2\t1\t1780 271\n",
      "122\t1835\t0.2\t1\t1588 247\n",
      "123\t1508\t0.2\t1\t1291 217\n",
      "124\t1263\t0.2\t1\t1083 180\n",
      "125\t1119\t0.2\t1\t966 153\n",
      "126\t1055\t0.2\t1\t916 139\n",
      "127\t900\t0.2\t1\t780 120\n",
      "128\t681\t0.2\t1\t568 113\n",
      "129\t550\t0.2\t1\t468 82\n",
      "130\t520\t0.2\t1\t423 97\n",
      "131\t523\t0.2\t1\t467 56\n",
      "132\t590\t0.2\t1\t526 64\n",
      "133\t974\t0.2\t1\t878 96\n",
      "134\t2631\t0.2\t1\t2466 165\n",
      "135\t822\t0.2\t1\t768 54\n",
      "136\t169\t0.2\t1\t154 15\n",
      "137\t111\t0.2\t1\t90 21\n",
      "138\t83\t0.2\t1\t75 8\n",
      "139\t53\t0.2\t1\t41 12\n",
      "140\t56\t0.2\t1\t43 13\n",
      "141\t26\t0.2\t1\t20 6\n",
      "142\t15\t0.2\t1\t12 3\n",
      "143\t20\t0.2\t1\t16 4\n",
      "144\t56\t0.2\t1\t47 9\n",
      "145\t231\t0.2\t1\t197 34\n",
      "146\t229\t0.2\t1\t199 30\n",
      "147\t1212\t0.2\t1\t1078 134\n",
      "148\t767\t0.2\t1\t701 66\n",
      "149\t472\t0.2\t1\t430 42\n",
      "150\t3085\t0.2\t1\t2595 490\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "12592704 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t12180 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 547.67 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,592,704\n",
      "Reads with adapters:                 8,852,263 (70.3%)\n",
      "Reads written (passing filters):    12,592,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,888,905,600 bp\n",
      "Quality-trimmed:              12,948,585 bp (0.7%)\n",
      "Total written (filtered):  1,572,160,709 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8852263 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.4%\n",
      "  C: 12.6%\n",
      "  G: 20.6%\n",
      "  T: 36.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2390954\t3148176.0\t0\t2390954\n",
      "2\t298677\t787044.0\t0\t298677\n",
      "3\t170286\t196761.0\t0\t170286\n",
      "4\t88634\t49190.2\t0\t88634\n",
      "5\t67201\t12297.6\t0\t67201\n",
      "6\t54871\t3074.4\t0\t54871\n",
      "7\t42570\t768.6\t0\t42570\n",
      "8\t63033\t192.1\t0\t63033\n",
      "9\t50827\t48.0\t0\t50608 219\n",
      "10\t54798\t12.0\t1\t51719 3079\n",
      "11\t58410\t3.0\t1\t54143 4267\n",
      "12\t59146\t0.8\t1\t54631 4515\n",
      "13\t64024\t0.2\t1\t59760 4264\n",
      "14\t60295\t0.2\t1\t55485 4810\n",
      "15\t48131\t0.2\t1\t44661 3470\n",
      "16\t72030\t0.2\t1\t66991 5039\n",
      "17\t55432\t0.2\t1\t51368 4064\n",
      "18\t54012\t0.2\t1\t50286 3726\n",
      "19\t67964\t0.2\t1\t63040 4924\n",
      "20\t63154\t0.2\t1\t58908 4246\n",
      "21\t56080\t0.2\t1\t52250 3830\n",
      "22\t62812\t0.2\t1\t58598 4214\n",
      "23\t64652\t0.2\t1\t60263 4389\n",
      "24\t64121\t0.2\t1\t59590 4531\n",
      "25\t72378\t0.2\t1\t67726 4652\n",
      "26\t60189\t0.2\t1\t55991 4198\n",
      "27\t67272\t0.2\t1\t62190 5082\n",
      "28\t62061\t0.2\t1\t58469 3592\n",
      "29\t66706\t0.2\t1\t62082 4624\n",
      "30\t64680\t0.2\t1\t60760 3920\n",
      "31\t70875\t0.2\t1\t66272 4603\n",
      "32\t63295\t0.2\t1\t59659 3636\n",
      "33\t68948\t0.2\t1\t64437 4511\n",
      "34\t89723\t0.2\t1\t83808 5915\n",
      "35\t49481\t0.2\t1\t46615 2866\n",
      "36\t69552\t0.2\t1\t64903 4649\n",
      "37\t74271\t0.2\t1\t69458 4813\n",
      "38\t69359\t0.2\t1\t65691 3668\n",
      "39\t67616\t0.2\t1\t63803 3813\n",
      "40\t75243\t0.2\t1\t70372 4871\n",
      "41\t71025\t0.2\t1\t67013 4012\n",
      "42\t73877\t0.2\t1\t69581 4296\n",
      "43\t66995\t0.2\t1\t63092 3903\n",
      "44\t76573\t0.2\t1\t71945 4628\n",
      "45\t80934\t0.2\t1\t76127 4807\n",
      "46\t59833\t0.2\t1\t56439 3394\n",
      "47\t73828\t0.2\t1\t69722 4106\n",
      "48\t73312\t0.2\t1\t69349 3963\n",
      "49\t75078\t0.2\t1\t70776 4302\n",
      "50\t70693\t0.2\t1\t66714 3979\n",
      "51\t88709\t0.2\t1\t83803 4906\n",
      "52\t62743\t0.2\t1\t59451 3292\n",
      "53\t75598\t0.2\t1\t71882 3716\n",
      "54\t57634\t0.2\t1\t54617 3017\n",
      "55\t73802\t0.2\t1\t70193 3609\n",
      "56\t76463\t0.2\t1\t72320 4143\n",
      "57\t71883\t0.2\t1\t68382 3501\n",
      "58\t68729\t0.2\t1\t65597 3132\n",
      "59\t69792\t0.2\t1\t66296 3496\n",
      "60\t72493\t0.2\t1\t68898 3595\n",
      "61\t74181\t0.2\t1\t70431 3750\n",
      "62\t80362\t0.2\t1\t76257 4105\n",
      "63\t105716\t0.2\t1\t100889 4827\n",
      "64\t86565\t0.2\t1\t82909 3656\n",
      "65\t55528\t0.2\t1\t52821 2707\n",
      "66\t61319\t0.2\t1\t58488 2831\n",
      "67\t60096\t0.2\t1\t57177 2919\n",
      "68\t60017\t0.2\t1\t57079 2938\n",
      "69\t59676\t0.2\t1\t56862 2814\n",
      "70\t62200\t0.2\t1\t59177 3023\n",
      "71\t65595\t0.2\t1\t62486 3109\n",
      "72\t62243\t0.2\t1\t59355 2888\n",
      "73\t65321\t0.2\t1\t62407 2914\n",
      "74\t60107\t0.2\t1\t57408 2699\n",
      "75\t56927\t0.2\t1\t54342 2585\n",
      "76\t56197\t0.2\t1\t53642 2555\n",
      "77\t55543\t0.2\t1\t53010 2533\n",
      "78\t53264\t0.2\t1\t50829 2435\n",
      "79\t50917\t0.2\t1\t48532 2385\n",
      "80\t50251\t0.2\t1\t48020 2231\n",
      "81\t49279\t0.2\t1\t47040 2239\n",
      "82\t47377\t0.2\t1\t45259 2118\n",
      "83\t45689\t0.2\t1\t43632 2057\n",
      "84\t44996\t0.2\t1\t42952 2044\n",
      "85\t43197\t0.2\t1\t41186 2011\n",
      "86\t43336\t0.2\t1\t41365 1971\n",
      "87\t42821\t0.2\t1\t40993 1828\n",
      "88\t42339\t0.2\t1\t40455 1884\n",
      "89\t37924\t0.2\t1\t36191 1733\n",
      "90\t36727\t0.2\t1\t35019 1708\n",
      "91\t33632\t0.2\t1\t32108 1524\n",
      "92\t32065\t0.2\t1\t30559 1506\n",
      "93\t30818\t0.2\t1\t29434 1384\n",
      "94\t30025\t0.2\t1\t28706 1319\n",
      "95\t28433\t0.2\t1\t27167 1266\n",
      "96\t27472\t0.2\t1\t26315 1157\n",
      "97\t25080\t0.2\t1\t23956 1124\n",
      "98\t23110\t0.2\t1\t22057 1053\n",
      "99\t21635\t0.2\t1\t20659 976\n",
      "100\t19482\t0.2\t1\t18587 895\n",
      "101\t17939\t0.2\t1\t17090 849\n",
      "102\t17260\t0.2\t1\t16487 773\n",
      "103\t15967\t0.2\t1\t15237 730\n",
      "104\t15618\t0.2\t1\t14883 735\n",
      "105\t13954\t0.2\t1\t13334 620\n",
      "106\t12766\t0.2\t1\t12218 548\n",
      "107\t11338\t0.2\t1\t10826 512\n",
      "108\t10218\t0.2\t1\t9719 499\n",
      "109\t9371\t0.2\t1\t8900 471\n",
      "110\t8643\t0.2\t1\t8250 393\n",
      "111\t8343\t0.2\t1\t7959 384\n",
      "112\t7194\t0.2\t1\t6837 357\n",
      "113\t6183\t0.2\t1\t5873 310\n",
      "114\t5369\t0.2\t1\t5131 238\n",
      "115\t4668\t0.2\t1\t4438 230\n",
      "116\t3833\t0.2\t1\t3644 189\n",
      "117\t3219\t0.2\t1\t3007 212\n",
      "118\t2818\t0.2\t1\t2669 149\n",
      "119\t2453\t0.2\t1\t2312 141\n",
      "120\t2060\t0.2\t1\t1936 124\n",
      "121\t1912\t0.2\t1\t1802 110\n",
      "122\t1619\t0.2\t1\t1536 83\n",
      "123\t1396\t0.2\t1\t1315 81\n",
      "124\t1132\t0.2\t1\t1057 75\n",
      "125\t1047\t0.2\t1\t976 71\n",
      "126\t972\t0.2\t1\t915 57\n",
      "127\t819\t0.2\t1\t758 61\n",
      "128\t624\t0.2\t1\t581 43\n",
      "129\t486\t0.2\t1\t450 36\n",
      "130\t458\t0.2\t1\t430 28\n",
      "131\t510\t0.2\t1\t482 28\n",
      "132\t584\t0.2\t1\t562 22\n",
      "133\t1105\t0.2\t1\t1055 50\n",
      "134\t2683\t0.2\t1\t2615 68\n",
      "135\t802\t0.2\t1\t778 24\n",
      "136\t171\t0.2\t1\t159 12\n",
      "137\t100\t0.2\t1\t93 7\n",
      "138\t80\t0.2\t1\t73 7\n",
      "139\t45\t0.2\t1\t40 5\n",
      "140\t54\t0.2\t1\t48 6\n",
      "141\t22\t0.2\t1\t17 5\n",
      "142\t19\t0.2\t1\t18 1\n",
      "143\t19\t0.2\t1\t16 3\n",
      "144\t60\t0.2\t1\t57 3\n",
      "145\t202\t0.2\t1\t195 7\n",
      "146\t219\t0.2\t1\t210 9\n",
      "147\t1124\t0.2\t1\t1091 33\n",
      "148\t721\t0.2\t1\t692 29\n",
      "149\t428\t0.2\t1\t410 18\n",
      "150\t2417\t0.2\t1\t2310 107\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "12592704 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t31314 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 514.49 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,486,748\n",
      "Reads with adapters:                 8,893,888 (71.2%)\n",
      "Reads written (passing filters):    12,486,748 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,873,012,200 bp\n",
      "Quality-trimmed:               5,043,033 bp (0.3%)\n",
      "Total written (filtered):  1,556,339,778 bp (83.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8893888 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 15.6%\n",
      "  G: 17.3%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2335523\t3121687.0\t0\t2335523\n",
      "2\t288272\t780421.8\t0\t288272\n",
      "3\t160403\t195105.4\t0\t160403\n",
      "4\t85608\t48776.4\t0\t85608\n",
      "5\t58915\t12194.1\t0\t58915\n",
      "6\t57362\t3048.5\t0\t57362\n",
      "7\t44207\t762.1\t0\t44207\n",
      "8\t57380\t190.5\t0\t57380\n",
      "9\t58488\t47.6\t0\t57517 971\n",
      "10\t52992\t11.9\t1\t48842 4150\n",
      "11\t61046\t3.0\t1\t54898 6148\n",
      "12\t57274\t0.7\t1\t52182 5092\n",
      "13\t56700\t0.2\t1\t51661 5039\n",
      "14\t59620\t0.2\t1\t53923 5697\n",
      "15\t59755\t0.2\t1\t54283 5472\n",
      "16\t59818\t0.2\t1\t53514 6304\n",
      "17\t74695\t0.2\t1\t66802 7893\n",
      "18\t60082\t0.2\t1\t55008 5074\n",
      "19\t51866\t0.2\t1\t47284 4582\n",
      "20\t63201\t0.2\t1\t57371 5830\n",
      "21\t69970\t0.2\t1\t63395 6575\n",
      "22\t57953\t0.2\t1\t53164 4789\n",
      "23\t68365\t0.2\t1\t61918 6447\n",
      "24\t63806\t0.2\t1\t56950 6856\n",
      "25\t67673\t0.2\t1\t61556 6117\n",
      "26\t61661\t0.2\t1\t56449 5212\n",
      "27\t70143\t0.2\t1\t63185 6958\n",
      "28\t67771\t0.2\t1\t61544 6227\n",
      "29\t71226\t0.2\t1\t63790 7436\n",
      "30\t75482\t0.2\t1\t69371 6111\n",
      "31\t58014\t0.2\t1\t52857 5157\n",
      "32\t74386\t0.2\t1\t67729 6657\n",
      "33\t63663\t0.2\t1\t57495 6168\n",
      "34\t74108\t0.2\t1\t66532 7576\n",
      "35\t70037\t0.2\t1\t63230 6807\n",
      "36\t76753\t0.2\t1\t69831 6922\n",
      "37\t67916\t0.2\t1\t61720 6196\n",
      "38\t71617\t0.2\t1\t65621 5996\n",
      "39\t75012\t0.2\t1\t68484 6528\n",
      "40\t71594\t0.2\t1\t64730 6864\n",
      "41\t78317\t0.2\t1\t70095 8222\n",
      "42\t72352\t0.2\t1\t64854 7498\n",
      "43\t105118\t0.2\t1\t97302 7816\n",
      "44\t39192\t0.2\t1\t34640 4552\n",
      "45\t76811\t0.2\t1\t69875 6936\n",
      "46\t73857\t0.2\t1\t67212 6645\n",
      "47\t78745\t0.2\t1\t71969 6776\n",
      "48\t69568\t0.2\t1\t62853 6715\n",
      "49\t81769\t0.2\t1\t74077 7692\n",
      "50\t65947\t0.2\t1\t60677 5270\n",
      "51\t71662\t0.2\t1\t65506 6156\n",
      "52\t80128\t0.2\t1\t72415 7713\n",
      "53\t85941\t0.2\t1\t79178 6763\n",
      "54\t61047\t0.2\t1\t55540 5507\n",
      "55\t75533\t0.2\t1\t69160 6373\n",
      "56\t87285\t0.2\t1\t80207 7078\n",
      "57\t70909\t0.2\t1\t64686 6223\n",
      "58\t68890\t0.2\t1\t64023 4867\n",
      "59\t56798\t0.2\t1\t51598 5200\n",
      "60\t76540\t0.2\t1\t70247 6293\n",
      "61\t72774\t0.2\t1\t66897 5877\n",
      "62\t72381\t0.2\t1\t66643 5738\n",
      "63\t67661\t0.2\t1\t61998 5663\n",
      "64\t65324\t0.2\t1\t60081 5243\n",
      "65\t70599\t0.2\t1\t64735 5864\n",
      "66\t77128\t0.2\t1\t70592 6536\n",
      "67\t84260\t0.2\t1\t76015 8245\n",
      "68\t144203\t0.2\t1\t134904 9299\n",
      "69\t43407\t0.2\t1\t39642 3765\n",
      "70\t15848\t0.2\t1\t13317 2531\n",
      "71\t33644\t0.2\t1\t30540 3104\n",
      "72\t56928\t0.2\t1\t52537 4391\n",
      "73\t61080\t0.2\t1\t56221 4859\n",
      "74\t61529\t0.2\t1\t56438 5091\n",
      "75\t60533\t0.2\t1\t55505 5028\n",
      "76\t58948\t0.2\t1\t53903 5045\n",
      "77\t59043\t0.2\t1\t53976 5067\n",
      "78\t55894\t0.2\t1\t51078 4816\n",
      "79\t55224\t0.2\t1\t50589 4635\n",
      "80\t53749\t0.2\t1\t49095 4654\n",
      "81\t52818\t0.2\t1\t48513 4305\n",
      "82\t51098\t0.2\t1\t46773 4325\n",
      "83\t49182\t0.2\t1\t45013 4169\n",
      "84\t47970\t0.2\t1\t43859 4111\n",
      "85\t46307\t0.2\t1\t42293 4014\n",
      "86\t45032\t0.2\t1\t41160 3872\n",
      "87\t42066\t0.2\t1\t38573 3493\n",
      "88\t42344\t0.2\t1\t38592 3752\n",
      "89\t39930\t0.2\t1\t36464 3466\n",
      "90\t40509\t0.2\t1\t36927 3582\n",
      "91\t36941\t0.2\t1\t33684 3257\n",
      "92\t35322\t0.2\t1\t32274 3048\n",
      "93\t32946\t0.2\t1\t30001 2945\n",
      "94\t32705\t0.2\t1\t29772 2933\n",
      "95\t29601\t0.2\t1\t27010 2591\n",
      "96\t28129\t0.2\t1\t25553 2576\n",
      "97\t26467\t0.2\t1\t24046 2421\n",
      "98\t25094\t0.2\t1\t22854 2240\n",
      "99\t24292\t0.2\t1\t21966 2326\n",
      "100\t22228\t0.2\t1\t20140 2088\n",
      "101\t20127\t0.2\t1\t18275 1852\n",
      "102\t18734\t0.2\t1\t17008 1726\n",
      "103\t17312\t0.2\t1\t15695 1617\n",
      "104\t16349\t0.2\t1\t14760 1589\n",
      "105\t14697\t0.2\t1\t13311 1386\n",
      "106\t13248\t0.2\t1\t11852 1396\n",
      "107\t12309\t0.2\t1\t11095 1214\n",
      "108\t11118\t0.2\t1\t10010 1108\n",
      "109\t9883\t0.2\t1\t8876 1007\n",
      "110\t9429\t0.2\t1\t8493 936\n",
      "111\t8334\t0.2\t1\t7393 941\n",
      "112\t7520\t0.2\t1\t6729 791\n",
      "113\t6506\t0.2\t1\t5771 735\n",
      "114\t5675\t0.2\t1\t5040 635\n",
      "115\t4923\t0.2\t1\t4354 569\n",
      "116\t4197\t0.2\t1\t3676 521\n",
      "117\t3761\t0.2\t1\t3289 472\n",
      "118\t3198\t0.2\t1\t2835 363\n",
      "119\t2789\t0.2\t1\t2461 328\n",
      "120\t2403\t0.2\t1\t2050 353\n",
      "121\t1996\t0.2\t1\t1771 225\n",
      "122\t1692\t0.2\t1\t1462 230\n",
      "123\t1495\t0.2\t1\t1306 189\n",
      "124\t1325\t0.2\t1\t1129 196\n",
      "125\t1099\t0.2\t1\t932 167\n",
      "126\t1059\t0.2\t1\t919 140\n",
      "127\t931\t0.2\t1\t822 109\n",
      "128\t654\t0.2\t1\t548 106\n",
      "129\t550\t0.2\t1\t477 73\n",
      "130\t491\t0.2\t1\t418 73\n",
      "131\t529\t0.2\t1\t438 91\n",
      "132\t576\t0.2\t1\t513 63\n",
      "133\t972\t0.2\t1\t858 114\n",
      "134\t2641\t0.2\t1\t2458 183\n",
      "135\t835\t0.2\t1\t788 47\n",
      "136\t183\t0.2\t1\t162 21\n",
      "137\t124\t0.2\t1\t105 19\n",
      "138\t78\t0.2\t1\t70 8\n",
      "139\t41\t0.2\t1\t32 9\n",
      "140\t43\t0.2\t1\t37 6\n",
      "141\t24\t0.2\t1\t20 4\n",
      "142\t18\t0.2\t1\t15 3\n",
      "143\t16\t0.2\t1\t12 4\n",
      "144\t54\t0.2\t1\t49 5\n",
      "145\t231\t0.2\t1\t211 20\n",
      "146\t231\t0.2\t1\t210 21\n",
      "147\t1327\t0.2\t1\t1201 126\n",
      "148\t804\t0.2\t1\t738 66\n",
      "149\t437\t0.2\t1\t393 44\n",
      "150\t3016\t0.2\t1\t2525 491\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "12486748 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t12233 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_26psu_4_S12_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 557.67 s (45 us/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,486,748\n",
      "Reads with adapters:                 8,772,809 (70.3%)\n",
      "Reads written (passing filters):    12,486,748 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,873,012,200 bp\n",
      "Quality-trimmed:              12,376,106 bp (0.7%)\n",
      "Total written (filtered):  1,558,147,987 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8772809 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.4%\n",
      "  C: 12.6%\n",
      "  G: 20.8%\n",
      "  T: 36.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2358875\t3121687.0\t0\t2358875\n",
      "2\t288344\t780421.8\t0\t288344\n",
      "3\t173054\t195105.4\t0\t173054\n",
      "4\t87112\t48776.4\t0\t87112\n",
      "5\t64247\t12194.1\t0\t64247\n",
      "6\t55212\t3048.5\t0\t55212\n",
      "7\t41727\t762.1\t0\t41727\n",
      "8\t62151\t190.5\t0\t62151\n",
      "9\t50342\t47.6\t0\t50157 185\n",
      "10\t52158\t11.9\t1\t48148 4010\n",
      "11\t57855\t3.0\t1\t52397 5458\n",
      "12\t56897\t0.7\t1\t51566 5331\n",
      "13\t64575\t0.2\t1\t58979 5596\n",
      "14\t59154\t0.2\t1\t53315 5839\n",
      "15\t47002\t0.2\t1\t42866 4136\n",
      "16\t69658\t0.2\t1\t63405 6253\n",
      "17\t57386\t0.2\t1\t52045 5341\n",
      "18\t51929\t0.2\t1\t47327 4602\n",
      "19\t66863\t0.2\t1\t60907 5956\n",
      "20\t62288\t0.2\t1\t57071 5217\n",
      "21\t55272\t0.2\t1\t50547 4725\n",
      "22\t61984\t0.2\t1\t56766 5218\n",
      "23\t65026\t0.2\t1\t59587 5439\n",
      "24\t63838\t0.2\t1\t58214 5624\n",
      "25\t72434\t0.2\t1\t66848 5586\n",
      "26\t60587\t0.2\t1\t55112 5475\n",
      "27\t67754\t0.2\t1\t61588 6166\n",
      "28\t59366\t0.2\t1\t54979 4387\n",
      "29\t67701\t0.2\t1\t62118 5583\n",
      "30\t62432\t0.2\t1\t57688 4744\n",
      "31\t72162\t0.2\t1\t66403 5759\n",
      "32\t60976\t0.2\t1\t56497 4479\n",
      "33\t69840\t0.2\t1\t64116 5724\n",
      "34\t91102\t0.2\t1\t84079 7023\n",
      "35\t47382\t0.2\t1\t43590 3792\n",
      "36\t69435\t0.2\t1\t63655 5780\n",
      "37\t74003\t0.2\t1\t67997 6006\n",
      "38\t68560\t0.2\t1\t63793 4767\n",
      "39\t65708\t0.2\t1\t61286 4422\n",
      "40\t74240\t0.2\t1\t68710 5530\n",
      "41\t72214\t0.2\t1\t67124 5090\n",
      "42\t72980\t0.2\t1\t67853 5127\n",
      "43\t66699\t0.2\t1\t62062 4637\n",
      "44\t76923\t0.2\t1\t71373 5550\n",
      "45\t80740\t0.2\t1\t74874 5866\n",
      "46\t58943\t0.2\t1\t54854 4089\n",
      "47\t73645\t0.2\t1\t68927 4718\n",
      "48\t72329\t0.2\t1\t67389 4940\n",
      "49\t74646\t0.2\t1\t69490 5156\n",
      "50\t70816\t0.2\t1\t65987 4829\n",
      "51\t86746\t0.2\t1\t80793 5953\n",
      "52\t63282\t0.2\t1\t59102 4180\n",
      "53\t76728\t0.2\t1\t71964 4764\n",
      "54\t55670\t0.2\t1\t52124 3546\n",
      "55\t73350\t0.2\t1\t68943 4407\n",
      "56\t76024\t0.2\t1\t70858 5166\n",
      "57\t71982\t0.2\t1\t67849 4133\n",
      "58\t68868\t0.2\t1\t64881 3987\n",
      "59\t68360\t0.2\t1\t64151 4209\n",
      "60\t71518\t0.2\t1\t67054 4464\n",
      "61\t73466\t0.2\t1\t69155 4311\n",
      "62\t79595\t0.2\t1\t74642 4953\n",
      "63\t103748\t0.2\t1\t97894 5854\n",
      "64\t85376\t0.2\t1\t81260 4116\n",
      "65\t55857\t0.2\t1\t52588 3269\n",
      "66\t61954\t0.2\t1\t58405 3549\n",
      "67\t59930\t0.2\t1\t56159 3771\n",
      "68\t60490\t0.2\t1\t56878 3612\n",
      "69\t60452\t0.2\t1\t56708 3744\n",
      "70\t61904\t0.2\t1\t58197 3707\n",
      "71\t65599\t0.2\t1\t61887 3712\n",
      "72\t62591\t0.2\t1\t58936 3655\n",
      "73\t64906\t0.2\t1\t61091 3815\n",
      "74\t60780\t0.2\t1\t57167 3613\n",
      "75\t57385\t0.2\t1\t53955 3430\n",
      "76\t55285\t0.2\t1\t51973 3312\n",
      "77\t55087\t0.2\t1\t51740 3347\n",
      "78\t52962\t0.2\t1\t49812 3150\n",
      "79\t51023\t0.2\t1\t48174 2849\n",
      "80\t50358\t0.2\t1\t47516 2842\n",
      "81\t49364\t0.2\t1\t46484 2880\n",
      "82\t47242\t0.2\t1\t44529 2713\n",
      "83\t45686\t0.2\t1\t43106 2580\n",
      "84\t44410\t0.2\t1\t41863 2547\n",
      "85\t43592\t0.2\t1\t41103 2489\n",
      "86\t43331\t0.2\t1\t40952 2379\n",
      "87\t42675\t0.2\t1\t40428 2247\n",
      "88\t42372\t0.2\t1\t40023 2349\n",
      "89\t37747\t0.2\t1\t35602 2145\n",
      "90\t37018\t0.2\t1\t34943 2075\n",
      "91\t33562\t0.2\t1\t31712 1850\n",
      "92\t31899\t0.2\t1\t30215 1684\n",
      "93\t30547\t0.2\t1\t28874 1673\n",
      "94\t30566\t0.2\t1\t28886 1680\n",
      "95\t28277\t0.2\t1\t26704 1573\n",
      "96\t27180\t0.2\t1\t25709 1471\n",
      "97\t24703\t0.2\t1\t23381 1322\n",
      "98\t23174\t0.2\t1\t21893 1281\n",
      "99\t21741\t0.2\t1\t20527 1214\n",
      "100\t19732\t0.2\t1\t18608 1124\n",
      "101\t17665\t0.2\t1\t16698 967\n",
      "102\t16908\t0.2\t1\t15954 954\n",
      "103\t15998\t0.2\t1\t15059 939\n",
      "104\t15634\t0.2\t1\t14747 887\n",
      "105\t13889\t0.2\t1\t13156 733\n",
      "106\t12626\t0.2\t1\t11933 693\n",
      "107\t11469\t0.2\t1\t10791 678\n",
      "108\t10122\t0.2\t1\t9581 541\n",
      "109\t9139\t0.2\t1\t8610 529\n",
      "110\t8759\t0.2\t1\t8236 523\n",
      "111\t8023\t0.2\t1\t7545 478\n",
      "112\t7113\t0.2\t1\t6705 408\n",
      "113\t6235\t0.2\t1\t5883 352\n",
      "114\t5409\t0.2\t1\t5057 352\n",
      "115\t4485\t0.2\t1\t4203 282\n",
      "116\t3795\t0.2\t1\t3556 239\n",
      "117\t3367\t0.2\t1\t3120 247\n",
      "118\t2891\t0.2\t1\t2702 189\n",
      "119\t2519\t0.2\t1\t2367 152\n",
      "120\t2120\t0.2\t1\t1963 157\n",
      "121\t1866\t0.2\t1\t1743 123\n",
      "122\t1561\t0.2\t1\t1440 121\n",
      "123\t1426\t0.2\t1\t1322 104\n",
      "124\t1196\t0.2\t1\t1106 90\n",
      "125\t1010\t0.2\t1\t943 67\n",
      "126\t988\t0.2\t1\t920 68\n",
      "127\t903\t0.2\t1\t840 63\n",
      "128\t611\t0.2\t1\t560 51\n",
      "129\t510\t0.2\t1\t470 40\n",
      "130\t453\t0.2\t1\t423 30\n",
      "131\t514\t0.2\t1\t467 47\n",
      "132\t553\t0.2\t1\t529 24\n",
      "133\t1090\t0.2\t1\t1032 58\n",
      "134\t2641\t0.2\t1\t2566 75\n",
      "135\t828\t0.2\t1\t794 34\n",
      "136\t185\t0.2\t1\t169 16\n",
      "137\t110\t0.2\t1\t95 15\n",
      "138\t79\t0.2\t1\t73 6\n",
      "139\t44\t0.2\t1\t40 4\n",
      "140\t39\t0.2\t1\t36 3\n",
      "141\t22\t0.2\t1\t17 5\n",
      "142\t20\t0.2\t1\t14 6\n",
      "143\t16\t0.2\t1\t15 1\n",
      "144\t54\t0.2\t1\t47 7\n",
      "145\t219\t0.2\t1\t204 15\n",
      "146\t229\t0.2\t1\t213 16\n",
      "147\t1254\t0.2\t1\t1201 53\n",
      "148\t756\t0.2\t1\t716 40\n",
      "149\t417\t0.2\t1\t400 17\n",
      "150\t2389\t0.2\t1\t2234 155\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "12486748 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t29715 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 479.07 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,000,021\n",
      "Reads with adapters:                12,585,652 (89.9%)\n",
      "Reads written (passing filters):    14,000,021 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,100,003,150 bp\n",
      "Quality-trimmed:               3,933,676 bp (0.2%)\n",
      "Total written (filtered):  1,383,592,146 bp (65.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12585652 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 15.4%\n",
      "  G: 21.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t794832\t3500005.2\t0\t794832\n",
      "2\t137499\t875001.3\t0\t137499\n",
      "3\t88574\t218750.3\t0\t88574\n",
      "4\t55463\t54687.6\t0\t55463\n",
      "5\t52467\t13671.9\t0\t52467\n",
      "6\t51009\t3418.0\t0\t51009\n",
      "7\t43440\t854.5\t0\t43440\n",
      "8\t53882\t213.6\t0\t53882\n",
      "9\t54563\t53.4\t0\t53971 592\n",
      "10\t51819\t13.4\t1\t49209 2610\n",
      "11\t57911\t3.3\t1\t53768 4143\n",
      "12\t56442\t0.8\t1\t53088 3354\n",
      "13\t57081\t0.2\t1\t53583 3498\n",
      "14\t60408\t0.2\t1\t56320 4088\n",
      "15\t62034\t0.2\t1\t58378 3656\n",
      "16\t63250\t0.2\t1\t58624 4626\n",
      "17\t76887\t0.2\t1\t70864 6023\n",
      "18\t66288\t0.2\t1\t62429 3859\n",
      "19\t56979\t0.2\t1\t53412 3567\n",
      "20\t69709\t0.2\t1\t65411 4298\n",
      "21\t77411\t0.2\t1\t72341 5070\n",
      "22\t66479\t0.2\t1\t62829 3650\n",
      "23\t78177\t0.2\t1\t73134 5043\n",
      "24\t74482\t0.2\t1\t68678 5804\n",
      "25\t80818\t0.2\t1\t75690 5128\n",
      "26\t74228\t0.2\t1\t70119 4109\n",
      "27\t83394\t0.2\t1\t77679 5715\n",
      "28\t82403\t0.2\t1\t76936 5467\n",
      "29\t90670\t0.2\t1\t83444 7226\n",
      "30\t98605\t0.2\t1\t93586 5019\n",
      "31\t72678\t0.2\t1\t68276 4402\n",
      "32\t92171\t0.2\t1\t86613 5558\n",
      "33\t90475\t0.2\t1\t84682 5793\n",
      "34\t99774\t0.2\t1\t93593 6181\n",
      "35\t93327\t0.2\t1\t87313 6014\n",
      "36\t107093\t0.2\t1\t100026 7067\n",
      "37\t103597\t0.2\t1\t96127 7470\n",
      "38\t104511\t0.2\t1\t98484 6027\n",
      "39\t106907\t0.2\t1\t100401 6506\n",
      "40\t106662\t0.2\t1\t100220 6442\n",
      "41\t122253\t0.2\t1\t114028 8225\n",
      "42\t104346\t0.2\t1\t96706 7640\n",
      "43\t169940\t0.2\t1\t161783 8157\n",
      "44\t58164\t0.2\t1\t53449 4715\n",
      "45\t118320\t0.2\t1\t111612 6708\n",
      "46\t116293\t0.2\t1\t109496 6797\n",
      "47\t124205\t0.2\t1\t117076 7129\n",
      "48\t123424\t0.2\t1\t116139 7285\n",
      "49\t135448\t0.2\t1\t126575 8873\n",
      "50\t119092\t0.2\t1\t112535 6557\n",
      "51\t128971\t0.2\t1\t121071 7900\n",
      "52\t136415\t0.2\t1\t127168 9247\n",
      "53\t146369\t0.2\t1\t138214 8155\n",
      "54\t119815\t0.2\t1\t112779 7036\n",
      "55\t143900\t0.2\t1\t135212 8688\n",
      "56\t174050\t0.2\t1\t164777 9273\n",
      "57\t117416\t0.2\t1\t109615 7801\n",
      "58\t150456\t0.2\t1\t142969 7487\n",
      "59\t132324\t0.2\t1\t124478 7846\n",
      "60\t149629\t0.2\t1\t140649 8980\n",
      "61\t147926\t0.2\t1\t139679 8247\n",
      "62\t163513\t0.2\t1\t154547 8966\n",
      "63\t134239\t0.2\t1\t125798 8441\n",
      "64\t148604\t0.2\t1\t140497 8107\n",
      "65\t167908\t0.2\t1\t158298 9610\n",
      "66\t177387\t0.2\t1\t167350 10037\n",
      "67\t194374\t0.2\t1\t180373 14001\n",
      "68\t370239\t0.2\t1\t354139 16100\n",
      "69\t62415\t0.2\t1\t57251 5164\n",
      "70\t26365\t0.2\t1\t22180 4185\n",
      "71\t101894\t0.2\t1\t96350 5544\n",
      "72\t143779\t0.2\t1\t135850 7929\n",
      "73\t154356\t0.2\t1\t145962 8394\n",
      "74\t154436\t0.2\t1\t145829 8607\n",
      "75\t155208\t0.2\t1\t146440 8768\n",
      "76\t154614\t0.2\t1\t146122 8492\n",
      "77\t156184\t0.2\t1\t147502 8682\n",
      "78\t149615\t0.2\t1\t141324 8291\n",
      "79\t149903\t0.2\t1\t141513 8390\n",
      "80\t146761\t0.2\t1\t138705 8056\n",
      "81\t147739\t0.2\t1\t139495 8244\n",
      "82\t146032\t0.2\t1\t137919 8113\n",
      "83\t142429\t0.2\t1\t134446 7983\n",
      "84\t139641\t0.2\t1\t132001 7640\n",
      "85\t137362\t0.2\t1\t129737 7625\n",
      "86\t134899\t0.2\t1\t127375 7524\n",
      "87\t130083\t0.2\t1\t122670 7413\n",
      "88\t129596\t0.2\t1\t122229 7367\n",
      "89\t124621\t0.2\t1\t117492 7129\n",
      "90\t125612\t0.2\t1\t118454 7158\n",
      "91\t119798\t0.2\t1\t113079 6719\n",
      "92\t115717\t0.2\t1\t109150 6567\n",
      "93\t109774\t0.2\t1\t103564 6210\n",
      "94\t107743\t0.2\t1\t101578 6165\n",
      "95\t101804\t0.2\t1\t95988 5816\n",
      "96\t96724\t0.2\t1\t91079 5645\n",
      "97\t92176\t0.2\t1\t86874 5302\n",
      "98\t87424\t0.2\t1\t82235 5189\n",
      "99\t86913\t0.2\t1\t81280 5633\n",
      "100\t78985\t0.2\t1\t74394 4591\n",
      "101\t75410\t0.2\t1\t70955 4455\n",
      "102\t69075\t0.2\t1\t64932 4143\n",
      "103\t64987\t0.2\t1\t61075 3912\n",
      "104\t61751\t0.2\t1\t58031 3720\n",
      "105\t55260\t0.2\t1\t51879 3381\n",
      "106\t50545\t0.2\t1\t47309 3236\n",
      "107\t46617\t0.2\t1\t43701 2916\n",
      "108\t44206\t0.2\t1\t41370 2836\n",
      "109\t39556\t0.2\t1\t36926 2630\n",
      "110\t36531\t0.2\t1\t34163 2368\n",
      "111\t32673\t0.2\t1\t30562 2111\n",
      "112\t28896\t0.2\t1\t26907 1989\n",
      "113\t25292\t0.2\t1\t23480 1812\n",
      "114\t22657\t0.2\t1\t21036 1621\n",
      "115\t19857\t0.2\t1\t18470 1387\n",
      "116\t17394\t0.2\t1\t16192 1202\n",
      "117\t14695\t0.2\t1\t13605 1090\n",
      "118\t12814\t0.2\t1\t11815 999\n",
      "119\t11036\t0.2\t1\t10138 898\n",
      "120\t9387\t0.2\t1\t8609 778\n",
      "121\t8006\t0.2\t1\t7333 673\n",
      "122\t6428\t0.2\t1\t5956 472\n",
      "123\t5508\t0.2\t1\t5035 473\n",
      "124\t4687\t0.2\t1\t4248 439\n",
      "125\t3879\t0.2\t1\t3508 371\n",
      "126\t3097\t0.2\t1\t2779 318\n",
      "127\t2720\t0.2\t1\t2457 263\n",
      "128\t2100\t0.2\t1\t1898 202\n",
      "129\t1767\t0.2\t1\t1595 172\n",
      "130\t1354\t0.2\t1\t1200 154\n",
      "131\t1174\t0.2\t1\t1057 117\n",
      "132\t1133\t0.2\t1\t1029 104\n",
      "133\t1115\t0.2\t1\t1004 111\n",
      "134\t1260\t0.2\t1\t1149 111\n",
      "135\t611\t0.2\t1\t547 64\n",
      "136\t316\t0.2\t1\t271 45\n",
      "137\t241\t0.2\t1\t219 22\n",
      "138\t161\t0.2\t1\t140 21\n",
      "139\t107\t0.2\t1\t93 14\n",
      "140\t137\t0.2\t1\t117 20\n",
      "141\t53\t0.2\t1\t47 6\n",
      "142\t47\t0.2\t1\t41 6\n",
      "143\t37\t0.2\t1\t32 5\n",
      "144\t37\t0.2\t1\t26 11\n",
      "145\t44\t0.2\t1\t34 10\n",
      "146\t50\t0.2\t1\t46 4\n",
      "147\t126\t0.2\t1\t111 15\n",
      "148\t84\t0.2\t1\t69 15\n",
      "149\t71\t0.2\t1\t62 9\n",
      "150\t642\t0.2\t1\t568 74\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14000021 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7494 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 499.63 s (36 us/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,000,021\n",
      "Reads with adapters:                12,482,400 (89.2%)\n",
      "Reads written (passing filters):    14,000,021 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,100,003,150 bp\n",
      "Quality-trimmed:              11,507,684 bp (0.5%)\n",
      "Total written (filtered):  1,389,836,957 bp (66.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12482400 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 13.5%\n",
      "  G: 25.9%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t842057\t3500005.2\t0\t842057\n",
      "2\t142966\t875001.3\t0\t142966\n",
      "3\t90255\t218750.3\t0\t90255\n",
      "4\t55571\t54687.6\t0\t55571\n",
      "5\t57545\t13671.9\t0\t57545\n",
      "6\t49671\t3418.0\t0\t49671\n",
      "7\t41298\t854.5\t0\t41298\n",
      "8\t59001\t213.6\t0\t59001\n",
      "9\t47476\t53.4\t0\t47315 161\n",
      "10\t53026\t13.4\t1\t50683 2343\n",
      "11\t55985\t3.3\t1\t52789 3196\n",
      "12\t57070\t0.8\t1\t53792 3278\n",
      "13\t64601\t0.2\t1\t61294 3307\n",
      "14\t62201\t0.2\t1\t58283 3918\n",
      "15\t50727\t0.2\t1\t47889 2838\n",
      "16\t74265\t0.2\t1\t70212 4053\n",
      "17\t60248\t0.2\t1\t56657 3591\n",
      "18\t57873\t0.2\t1\t54797 3076\n",
      "19\t74512\t0.2\t1\t70251 4261\n",
      "20\t70260\t0.2\t1\t66610 3650\n",
      "21\t62230\t0.2\t1\t58933 3297\n",
      "22\t72026\t0.2\t1\t68190 3836\n",
      "23\t75580\t0.2\t1\t71483 4097\n",
      "24\t76103\t0.2\t1\t71906 4197\n",
      "25\t87020\t0.2\t1\t82705 4315\n",
      "26\t72653\t0.2\t1\t68640 4013\n",
      "27\t81722\t0.2\t1\t76658 5064\n",
      "28\t76893\t0.2\t1\t73629 3264\n",
      "29\t84223\t0.2\t1\t79742 4481\n",
      "30\t83568\t0.2\t1\t79746 3822\n",
      "31\t92428\t0.2\t1\t87765 4663\n",
      "32\t83785\t0.2\t1\t79986 3799\n",
      "33\t94083\t0.2\t1\t89299 4784\n",
      "34\t124234\t0.2\t1\t117883 6351\n",
      "35\t70662\t0.2\t1\t67495 3167\n",
      "36\t97571\t0.2\t1\t92509 5062\n",
      "37\t105963\t0.2\t1\t100553 5410\n",
      "38\t100112\t0.2\t1\t95880 4232\n",
      "39\t98883\t0.2\t1\t94704 4179\n",
      "40\t111592\t0.2\t1\t106078 5514\n",
      "41\t108482\t0.2\t1\t103543 4939\n",
      "42\t113048\t0.2\t1\t108097 4951\n",
      "43\t104564\t0.2\t1\t100168 4396\n",
      "44\t122044\t0.2\t1\t116433 5611\n",
      "45\t132901\t0.2\t1\t126882 6019\n",
      "46\t96495\t0.2\t1\t92302 4193\n",
      "47\t123777\t0.2\t1\t118653 5124\n",
      "48\t123419\t0.2\t1\t118253 5166\n",
      "49\t130090\t0.2\t1\t124494 5596\n",
      "50\t124769\t0.2\t1\t119306 5463\n",
      "51\t159790\t0.2\t1\t152747 7043\n",
      "52\t114726\t0.2\t1\t110175 4551\n",
      "53\t140573\t0.2\t1\t135310 5263\n",
      "54\t107631\t0.2\t1\t103512 4119\n",
      "55\t142110\t0.2\t1\t137013 5097\n",
      "56\t148511\t0.2\t1\t142711 5800\n",
      "57\t141596\t0.2\t1\t136449 5147\n",
      "58\t138299\t0.2\t1\t133566 4733\n",
      "59\t142121\t0.2\t1\t137002 5119\n",
      "60\t148830\t0.2\t1\t143312 5518\n",
      "61\t154735\t0.2\t1\t148878 5857\n",
      "62\t170128\t0.2\t1\t163868 6260\n",
      "63\t234733\t0.2\t1\t226789 7944\n",
      "64\t212296\t0.2\t1\t206143 6153\n",
      "65\t126771\t0.2\t1\t122389 4382\n",
      "66\t138120\t0.2\t1\t133306 4814\n",
      "67\t139286\t0.2\t1\t134193 5093\n",
      "68\t141731\t0.2\t1\t136845 4886\n",
      "69\t141571\t0.2\t1\t136513 5058\n",
      "70\t151101\t0.2\t1\t145752 5349\n",
      "71\t165000\t0.2\t1\t159543 5457\n",
      "72\t156137\t0.2\t1\t150635 5502\n",
      "73\t168957\t0.2\t1\t163425 5532\n",
      "74\t155346\t0.2\t1\t150282 5064\n",
      "75\t149711\t0.2\t1\t144684 5027\n",
      "76\t146688\t0.2\t1\t141765 4923\n",
      "77\t148035\t0.2\t1\t142983 5052\n",
      "78\t144141\t0.2\t1\t139293 4848\n",
      "79\t140116\t0.2\t1\t135357 4759\n",
      "80\t139815\t0.2\t1\t135255 4560\n",
      "81\t140275\t0.2\t1\t135701 4574\n",
      "82\t136237\t0.2\t1\t131702 4535\n",
      "83\t133248\t0.2\t1\t128960 4288\n",
      "84\t130802\t0.2\t1\t126388 4414\n",
      "85\t131118\t0.2\t1\t126640 4478\n",
      "86\t132538\t0.2\t1\t128154 4384\n",
      "87\t134528\t0.2\t1\t130166 4362\n",
      "88\t132145\t0.2\t1\t127913 4232\n",
      "89\t119589\t0.2\t1\t115716 3873\n",
      "90\t116381\t0.2\t1\t112493 3888\n",
      "91\t109945\t0.2\t1\t106186 3759\n",
      "92\t104899\t0.2\t1\t101501 3398\n",
      "93\t102752\t0.2\t1\t99467 3285\n",
      "94\t100398\t0.2\t1\t97258 3140\n",
      "95\t96752\t0.2\t1\t93687 3065\n",
      "96\t92474\t0.2\t1\t89641 2833\n",
      "97\t85160\t0.2\t1\t82565 2595\n",
      "98\t81057\t0.2\t1\t78431 2626\n",
      "99\t79148\t0.2\t1\t76636 2512\n",
      "100\t72036\t0.2\t1\t69758 2278\n",
      "101\t69452\t0.2\t1\t67404 2048\n",
      "102\t66047\t0.2\t1\t64053 1994\n",
      "103\t62024\t0.2\t1\t60217 1807\n",
      "104\t59709\t0.2\t1\t57936 1773\n",
      "105\t53244\t0.2\t1\t51636 1608\n",
      "106\t48557\t0.2\t1\t47145 1412\n",
      "107\t44301\t0.2\t1\t42956 1345\n",
      "108\t40672\t0.2\t1\t39383 1289\n",
      "109\t36835\t0.2\t1\t35717 1118\n",
      "110\t34356\t0.2\t1\t33352 1004\n",
      "111\t32280\t0.2\t1\t31328 952\n",
      "112\t28083\t0.2\t1\t27306 777\n",
      "113\t24882\t0.2\t1\t24139 743\n",
      "114\t22038\t0.2\t1\t21386 652\n",
      "115\t18554\t0.2\t1\t17942 612\n",
      "116\t16085\t0.2\t1\t15523 562\n",
      "117\t13238\t0.2\t1\t12820 418\n",
      "118\t11657\t0.2\t1\t11269 388\n",
      "119\t10199\t0.2\t1\t9853 346\n",
      "120\t8562\t0.2\t1\t8250 312\n",
      "121\t7440\t0.2\t1\t7187 253\n",
      "122\t5970\t0.2\t1\t5758 212\n",
      "123\t5287\t0.2\t1\t5099 188\n",
      "124\t4439\t0.2\t1\t4276 163\n",
      "125\t3716\t0.2\t1\t3583 133\n",
      "126\t2906\t0.2\t1\t2772 134\n",
      "127\t2521\t0.2\t1\t2432 89\n",
      "128\t1926\t0.2\t1\t1855 71\n",
      "129\t1631\t0.2\t1\t1564 67\n",
      "130\t1264\t0.2\t1\t1221 43\n",
      "131\t1080\t0.2\t1\t1044 36\n",
      "132\t1051\t0.2\t1\t1017 34\n",
      "133\t1128\t0.2\t1\t1088 40\n",
      "134\t1229\t0.2\t1\t1204 25\n",
      "135\t567\t0.2\t1\t545 22\n",
      "136\t312\t0.2\t1\t296 16\n",
      "137\t233\t0.2\t1\t225 8\n",
      "138\t145\t0.2\t1\t135 10\n",
      "139\t105\t0.2\t1\t99 6\n",
      "140\t127\t0.2\t1\t124 3\n",
      "141\t54\t0.2\t1\t50 4\n",
      "142\t42\t0.2\t1\t36 6\n",
      "143\t34\t0.2\t1\t31 3\n",
      "144\t35\t0.2\t1\t32 3\n",
      "145\t45\t0.2\t1\t43 2\n",
      "146\t40\t0.2\t1\t38 2\n",
      "147\t105\t0.2\t1\t96 9\n",
      "148\t62\t0.2\t1\t59 3\n",
      "149\t54\t0.2\t1\t50 4\n",
      "150\t462\t0.2\t1\t437 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14000021 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t21943 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 485.81 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,980,098\n",
      "Reads with adapters:                12,547,273 (89.8%)\n",
      "Reads written (passing filters):    13,980,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,097,014,700 bp\n",
      "Quality-trimmed:               3,995,329 bp (0.2%)\n",
      "Total written (filtered):  1,381,322,734 bp (65.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12547273 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 15.4%\n",
      "  G: 21.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t797770\t3495024.5\t0\t797770\n",
      "2\t136013\t873756.1\t0\t136013\n",
      "3\t88578\t218439.0\t0\t88578\n",
      "4\t54507\t54609.8\t0\t54507\n",
      "5\t51433\t13652.4\t0\t51433\n",
      "6\t50368\t3413.1\t0\t50368\n",
      "7\t41781\t853.3\t0\t41781\n",
      "8\t52923\t213.3\t0\t52923\n",
      "9\t54051\t53.3\t0\t53431 620\n",
      "10\t50288\t13.3\t1\t47174 3114\n",
      "11\t58137\t3.3\t1\t53328 4809\n",
      "12\t55530\t0.8\t1\t51625 3905\n",
      "13\t56413\t0.2\t1\t52378 4035\n",
      "14\t59004\t0.2\t1\t54278 4726\n",
      "15\t60957\t0.2\t1\t56359 4598\n",
      "16\t62617\t0.2\t1\t57287 5330\n",
      "17\t77034\t0.2\t1\t70041 6993\n",
      "18\t64851\t0.2\t1\t60415 4436\n",
      "19\t55636\t0.2\t1\t51675 3961\n",
      "20\t69068\t0.2\t1\t64118 4950\n",
      "21\t78331\t0.2\t1\t72354 5977\n",
      "22\t65036\t0.2\t1\t60845 4191\n",
      "23\t77471\t0.2\t1\t71757 5714\n",
      "24\t73017\t0.2\t1\t66655 6362\n",
      "25\t80097\t0.2\t1\t74072 6025\n",
      "26\t72725\t0.2\t1\t67966 4759\n",
      "27\t83104\t0.2\t1\t76594 6510\n",
      "28\t81931\t0.2\t1\t75845 6086\n",
      "29\t90548\t0.2\t1\t82404 8144\n",
      "30\t97688\t0.2\t1\t91724 5964\n",
      "31\t71240\t0.2\t1\t66196 5044\n",
      "32\t92668\t0.2\t1\t85724 6944\n",
      "33\t88978\t0.2\t1\t82032 6946\n",
      "34\t98712\t0.2\t1\t91302 7410\n",
      "35\t92968\t0.2\t1\t85787 7181\n",
      "36\t107083\t0.2\t1\t98683 8400\n",
      "37\t102524\t0.2\t1\t93933 8591\n",
      "38\t103053\t0.2\t1\t96018 7035\n",
      "39\t106015\t0.2\t1\t98258 7757\n",
      "40\t105210\t0.2\t1\t97541 7669\n",
      "41\t121909\t0.2\t1\t112224 9685\n",
      "42\t102833\t0.2\t1\t93869 8964\n",
      "43\t167631\t0.2\t1\t157548 10083\n",
      "44\t59347\t0.2\t1\t53867 5480\n",
      "45\t119124\t0.2\t1\t111193 7931\n",
      "46\t115006\t0.2\t1\t106994 8012\n",
      "47\t123238\t0.2\t1\t114804 8434\n",
      "48\t121882\t0.2\t1\t113175 8707\n",
      "49\t136657\t0.2\t1\t126388 10269\n",
      "50\t117352\t0.2\t1\t109816 7536\n",
      "51\t128981\t0.2\t1\t119944 9037\n",
      "52\t136441\t0.2\t1\t126118 10323\n",
      "53\t146650\t0.2\t1\t137000 9650\n",
      "54\t118855\t0.2\t1\t110740 8115\n",
      "55\t142441\t0.2\t1\t132669 9772\n",
      "56\t173419\t0.2\t1\t162260 11159\n",
      "57\t115395\t0.2\t1\t106550 8845\n",
      "58\t148737\t0.2\t1\t140083 8654\n",
      "59\t131828\t0.2\t1\t122332 9496\n",
      "60\t148604\t0.2\t1\t138387 10217\n",
      "61\t147891\t0.2\t1\t138429 9462\n",
      "62\t164090\t0.2\t1\t153607 10483\n",
      "63\t134153\t0.2\t1\t124429 9724\n",
      "64\t146788\t0.2\t1\t137613 9175\n",
      "65\t167405\t0.2\t1\t156219 11186\n",
      "66\t175152\t0.2\t1\t163306 11846\n",
      "67\t193973\t0.2\t1\t178444 15529\n",
      "68\t355893\t0.2\t1\t337623 18270\n",
      "69\t77628\t0.2\t1\t71288 6340\n",
      "70\t32194\t0.2\t1\t27237 4957\n",
      "71\t98744\t0.2\t1\t92367 6377\n",
      "72\t142722\t0.2\t1\t134109 8613\n",
      "73\t153911\t0.2\t1\t144323 9588\n",
      "74\t152982\t0.2\t1\t143267 9715\n",
      "75\t155312\t0.2\t1\t145139 10173\n",
      "76\t154296\t0.2\t1\t143821 10475\n",
      "77\t156257\t0.2\t1\t146141 10116\n",
      "78\t149276\t0.2\t1\t139442 9834\n",
      "79\t150468\t0.2\t1\t140290 10178\n",
      "80\t147626\t0.2\t1\t137573 10053\n",
      "81\t147629\t0.2\t1\t137952 9677\n",
      "82\t146000\t0.2\t1\t136195 9805\n",
      "83\t142550\t0.2\t1\t133188 9362\n",
      "84\t139878\t0.2\t1\t130510 9368\n",
      "85\t136886\t0.2\t1\t127696 9190\n",
      "86\t134793\t0.2\t1\t125685 9108\n",
      "87\t130207\t0.2\t1\t121771 8436\n",
      "88\t129730\t0.2\t1\t121003 8727\n",
      "89\t125338\t0.2\t1\t117059 8279\n",
      "90\t125920\t0.2\t1\t117611 8309\n",
      "91\t120727\t0.2\t1\t112765 7962\n",
      "92\t116230\t0.2\t1\t108426 7804\n",
      "93\t110261\t0.2\t1\t102836 7425\n",
      "94\t107731\t0.2\t1\t100461 7270\n",
      "95\t101765\t0.2\t1\t94845 6920\n",
      "96\t97730\t0.2\t1\t91189 6541\n",
      "97\t92862\t0.2\t1\t86658 6204\n",
      "98\t87553\t0.2\t1\t81547 6006\n",
      "99\t87158\t0.2\t1\t80825 6333\n",
      "100\t78638\t0.2\t1\t73170 5468\n",
      "101\t74653\t0.2\t1\t69508 5145\n",
      "102\t70275\t0.2\t1\t65468 4807\n",
      "103\t65373\t0.2\t1\t60911 4462\n",
      "104\t61766\t0.2\t1\t57502 4264\n",
      "105\t55492\t0.2\t1\t51811 3681\n",
      "106\t51179\t0.2\t1\t47624 3555\n",
      "107\t46618\t0.2\t1\t43293 3325\n",
      "108\t44553\t0.2\t1\t41440 3113\n",
      "109\t39712\t0.2\t1\t36840 2872\n",
      "110\t36630\t0.2\t1\t33977 2653\n",
      "111\t32740\t0.2\t1\t30268 2472\n",
      "112\t29144\t0.2\t1\t26890 2254\n",
      "113\t25615\t0.2\t1\t23550 2065\n",
      "114\t22855\t0.2\t1\t21063 1792\n",
      "115\t19838\t0.2\t1\t18299 1539\n",
      "116\t17636\t0.2\t1\t16221 1415\n",
      "117\t14547\t0.2\t1\t13334 1213\n",
      "118\t13235\t0.2\t1\t12137 1098\n",
      "119\t10988\t0.2\t1\t10016 972\n",
      "120\t9462\t0.2\t1\t8648 814\n",
      "121\t7752\t0.2\t1\t7051 701\n",
      "122\t6591\t0.2\t1\t5988 603\n",
      "123\t5413\t0.2\t1\t4917 496\n",
      "124\t4773\t0.2\t1\t4290 483\n",
      "125\t3899\t0.2\t1\t3520 379\n",
      "126\t3211\t0.2\t1\t2920 291\n",
      "127\t2698\t0.2\t1\t2418 280\n",
      "128\t2171\t0.2\t1\t1932 239\n",
      "129\t1648\t0.2\t1\t1461 187\n",
      "130\t1400\t0.2\t1\t1257 143\n",
      "131\t1218\t0.2\t1\t1094 124\n",
      "132\t1037\t0.2\t1\t929 108\n",
      "133\t1169\t0.2\t1\t1044 125\n",
      "134\t1204\t0.2\t1\t1109 95\n",
      "135\t602\t0.2\t1\t535 67\n",
      "136\t395\t0.2\t1\t343 52\n",
      "137\t232\t0.2\t1\t196 36\n",
      "138\t155\t0.2\t1\t142 13\n",
      "139\t118\t0.2\t1\t95 23\n",
      "140\t87\t0.2\t1\t71 16\n",
      "141\t57\t0.2\t1\t45 12\n",
      "142\t68\t0.2\t1\t58 10\n",
      "143\t37\t0.2\t1\t32 5\n",
      "144\t24\t0.2\t1\t22 2\n",
      "145\t36\t0.2\t1\t30 6\n",
      "146\t26\t0.2\t1\t23 3\n",
      "147\t105\t0.2\t1\t86 19\n",
      "148\t85\t0.2\t1\t66 19\n",
      "149\t76\t0.2\t1\t64 12\n",
      "150\t640\t0.2\t1\t546 94\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13980098 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7427 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_1_S5_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 508.63 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,980,098\n",
      "Reads with adapters:                12,441,322 (89.0%)\n",
      "Reads written (passing filters):    13,980,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,097,014,700 bp\n",
      "Quality-trimmed:              11,100,984 bp (0.5%)\n",
      "Total written (filtered):  1,388,068,565 bp (66.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12441322 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 13.5%\n",
      "  G: 25.9%\n",
      "  T: 33.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t845065\t3495024.5\t0\t845065\n",
      "2\t141219\t873756.1\t0\t141219\n",
      "3\t93432\t218439.0\t0\t93432\n",
      "4\t54397\t54609.8\t0\t54397\n",
      "5\t54738\t13652.4\t0\t54738\n",
      "6\t49515\t3413.1\t0\t49515\n",
      "7\t40356\t853.3\t0\t40356\n",
      "8\t58207\t213.3\t0\t58207\n",
      "9\t46723\t53.3\t0\t46571 152\n",
      "10\t50819\t13.3\t1\t47536 3283\n",
      "11\t55858\t3.3\t1\t51611 4247\n",
      "12\t55535\t0.8\t1\t51267 4268\n",
      "13\t64627\t0.2\t1\t59860 4767\n",
      "14\t60188\t0.2\t1\t55222 4966\n",
      "15\t49387\t0.2\t1\t45866 3521\n",
      "16\t72997\t0.2\t1\t67649 5348\n",
      "17\t62490\t0.2\t1\t57882 4608\n",
      "18\t55351\t0.2\t1\t51495 3856\n",
      "19\t73045\t0.2\t1\t67571 5474\n",
      "20\t69542\t0.2\t1\t64858 4684\n",
      "21\t61947\t0.2\t1\t57754 4193\n",
      "22\t70571\t0.2\t1\t65755 4816\n",
      "23\t75040\t0.2\t1\t69830 5210\n",
      "24\t75107\t0.2\t1\t69631 5476\n",
      "25\t86702\t0.2\t1\t81241 5461\n",
      "26\t72229\t0.2\t1\t66945 5284\n",
      "27\t82253\t0.2\t1\t76106 6147\n",
      "28\t73797\t0.2\t1\t69537 4260\n",
      "29\t84769\t0.2\t1\t79179 5590\n",
      "30\t80847\t0.2\t1\t75988 4859\n",
      "31\t93818\t0.2\t1\t87884 5934\n",
      "32\t80840\t0.2\t1\t75981 4859\n",
      "33\t94296\t0.2\t1\t88030 6266\n",
      "34\t125592\t0.2\t1\t117608 7984\n",
      "35\t67216\t0.2\t1\t63145 4071\n",
      "36\t97900\t0.2\t1\t91347 6553\n",
      "37\t105322\t0.2\t1\t98633 6689\n",
      "38\t99129\t0.2\t1\t93524 5605\n",
      "39\t95232\t0.2\t1\t90264 4968\n",
      "40\t111486\t0.2\t1\t104598 6888\n",
      "41\t108655\t0.2\t1\t102291 6364\n",
      "42\t112712\t0.2\t1\t106221 6491\n",
      "43\t103445\t0.2\t1\t97749 5696\n",
      "44\t122874\t0.2\t1\t115506 7368\n",
      "45\t132722\t0.2\t1\t124931 7791\n",
      "46\t94746\t0.2\t1\t89536 5210\n",
      "47\t122933\t0.2\t1\t116693 6240\n",
      "48\t121112\t0.2\t1\t114755 6357\n",
      "49\t129508\t0.2\t1\t122295 7213\n",
      "50\t125201\t0.2\t1\t118349 6852\n",
      "51\t157565\t0.2\t1\t148874 8691\n",
      "52\t113806\t0.2\t1\t107741 6065\n",
      "53\t143800\t0.2\t1\t136726 7074\n",
      "54\t105294\t0.2\t1\t100142 5152\n",
      "55\t140198\t0.2\t1\t133598 6600\n",
      "56\t145988\t0.2\t1\t138211 7777\n",
      "57\t140653\t0.2\t1\t134358 6295\n",
      "58\t138055\t0.2\t1\t131586 6469\n",
      "59\t139154\t0.2\t1\t132424 6730\n",
      "60\t148619\t0.2\t1\t141526 7093\n",
      "61\t153296\t0.2\t1\t146146 7150\n",
      "62\t170542\t0.2\t1\t162169 8373\n",
      "63\t228765\t0.2\t1\t219044 9721\n",
      "64\t207127\t0.2\t1\t199944 7183\n",
      "65\t126856\t0.2\t1\t121054 5802\n",
      "66\t139494\t0.2\t1\t133042 6452\n",
      "67\t139853\t0.2\t1\t133076 6777\n",
      "68\t143593\t0.2\t1\t136916 6677\n",
      "69\t142441\t0.2\t1\t135436 7005\n",
      "70\t151914\t0.2\t1\t144843 7071\n",
      "71\t164443\t0.2\t1\t156971 7472\n",
      "72\t157523\t0.2\t1\t150188 7335\n",
      "73\t168905\t0.2\t1\t161158 7747\n",
      "74\t154929\t0.2\t1\t147615 7314\n",
      "75\t149956\t0.2\t1\t142906 7050\n",
      "76\t146942\t0.2\t1\t140083 6859\n",
      "77\t148479\t0.2\t1\t141471 7008\n",
      "78\t143688\t0.2\t1\t136908 6780\n",
      "79\t140442\t0.2\t1\t133903 6539\n",
      "80\t140871\t0.2\t1\t134402 6469\n",
      "81\t139994\t0.2\t1\t133731 6263\n",
      "82\t136315\t0.2\t1\t130107 6208\n",
      "83\t133480\t0.2\t1\t127541 5939\n",
      "84\t130953\t0.2\t1\t125135 5818\n",
      "85\t130441\t0.2\t1\t124710 5731\n",
      "86\t132446\t0.2\t1\t126752 5694\n",
      "87\t133712\t0.2\t1\t128137 5575\n",
      "88\t132133\t0.2\t1\t126501 5632\n",
      "89\t120737\t0.2\t1\t115669 5068\n",
      "90\t117385\t0.2\t1\t112365 5020\n",
      "91\t111344\t0.2\t1\t106559 4785\n",
      "92\t105918\t0.2\t1\t101512 4406\n",
      "93\t103040\t0.2\t1\t98784 4256\n",
      "94\t100992\t0.2\t1\t96771 4221\n",
      "95\t96578\t0.2\t1\t92479 4099\n",
      "96\t93826\t0.2\t1\t89971 3855\n",
      "97\t86326\t0.2\t1\t82928 3398\n",
      "98\t81391\t0.2\t1\t78062 3329\n",
      "99\t79676\t0.2\t1\t76340 3336\n",
      "100\t71969\t0.2\t1\t68956 3013\n",
      "101\t69033\t0.2\t1\t66191 2842\n",
      "102\t66979\t0.2\t1\t64266 2713\n",
      "103\t62488\t0.2\t1\t59972 2516\n",
      "104\t59740\t0.2\t1\t57324 2416\n",
      "105\t53204\t0.2\t1\t51066 2138\n",
      "106\t49286\t0.2\t1\t47321 1965\n",
      "107\t44161\t0.2\t1\t42327 1834\n",
      "108\t41055\t0.2\t1\t39500 1555\n",
      "109\t37107\t0.2\t1\t35641 1466\n",
      "110\t34404\t0.2\t1\t32996 1408\n",
      "111\t32112\t0.2\t1\t30868 1244\n",
      "112\t28362\t0.2\t1\t27169 1193\n",
      "113\t24976\t0.2\t1\t23918 1058\n",
      "114\t22140\t0.2\t1\t21281 859\n",
      "115\t18628\t0.2\t1\t17866 762\n",
      "116\t16277\t0.2\t1\t15608 669\n",
      "117\t13249\t0.2\t1\t12647 602\n",
      "118\t12071\t0.2\t1\t11527 544\n",
      "119\t10131\t0.2\t1\t9673 458\n",
      "120\t8604\t0.2\t1\t8215 389\n",
      "121\t7264\t0.2\t1\t6947 317\n",
      "122\t6137\t0.2\t1\t5844 293\n",
      "123\t5194\t0.2\t1\t4979 215\n",
      "124\t4472\t0.2\t1\t4261 211\n",
      "125\t3689\t0.2\t1\t3519 170\n",
      "126\t3057\t0.2\t1\t2936 121\n",
      "127\t2513\t0.2\t1\t2380 133\n",
      "128\t1994\t0.2\t1\t1883 111\n",
      "129\t1536\t0.2\t1\t1459 77\n",
      "130\t1296\t0.2\t1\t1243 53\n",
      "131\t1130\t0.2\t1\t1079 51\n",
      "132\t980\t0.2\t1\t938 42\n",
      "133\t1174\t0.2\t1\t1133 41\n",
      "134\t1195\t0.2\t1\t1157 38\n",
      "135\t568\t0.2\t1\t538 30\n",
      "136\t352\t0.2\t1\t335 17\n",
      "137\t213\t0.2\t1\t202 11\n",
      "138\t143\t0.2\t1\t132 11\n",
      "139\t111\t0.2\t1\t103 8\n",
      "140\t71\t0.2\t1\t64 7\n",
      "141\t52\t0.2\t1\t46 6\n",
      "142\t65\t0.2\t1\t60 5\n",
      "143\t34\t0.2\t1\t32 2\n",
      "144\t27\t0.2\t1\t25 2\n",
      "145\t36\t0.2\t1\t30 6\n",
      "146\t27\t0.2\t1\t25 2\n",
      "147\t78\t0.2\t1\t72 6\n",
      "148\t68\t0.2\t1\t62 6\n",
      "149\t57\t0.2\t1\t55 2\n",
      "150\t443\t0.2\t1\t415 28\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13980098 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t20569 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 511.85 s (40 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,939,154\n",
      "Reads with adapters:                10,340,662 (79.9%)\n",
      "Reads written (passing filters):    12,939,154 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,940,873,100 bp\n",
      "Quality-trimmed:               3,580,813 bp (0.2%)\n",
      "Total written (filtered):  1,492,993,480 bp (76.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10340662 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.7%\n",
      "  G: 20.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1552261\t3234788.5\t0\t1552261\n",
      "2\t232662\t808697.1\t0\t232662\n",
      "3\t131614\t202174.3\t0\t131614\n",
      "4\t79987\t50543.6\t0\t79987\n",
      "5\t67280\t12635.9\t0\t67280\n",
      "6\t66472\t3159.0\t0\t66472\n",
      "7\t55108\t789.7\t0\t55108\n",
      "8\t68928\t197.4\t0\t68928\n",
      "9\t68982\t49.4\t0\t68109 873\n",
      "10\t64214\t12.3\t1\t60894 3320\n",
      "11\t72301\t3.1\t1\t66989 5312\n",
      "12\t69430\t0.8\t1\t65113 4317\n",
      "13\t71580\t0.2\t1\t67020 4560\n",
      "14\t70477\t0.2\t1\t65870 4607\n",
      "15\t75676\t0.2\t1\t70785 4891\n",
      "16\t72936\t0.2\t1\t67871 5065\n",
      "17\t87298\t0.2\t1\t80460 6838\n",
      "18\t79611\t0.2\t1\t74935 4676\n",
      "19\t65534\t0.2\t1\t61501 4033\n",
      "20\t79604\t0.2\t1\t74644 4960\n",
      "21\t85256\t0.2\t1\t79605 5651\n",
      "22\t73838\t0.2\t1\t69798 4040\n",
      "23\t84971\t0.2\t1\t79628 5343\n",
      "24\t80676\t0.2\t1\t74456 6220\n",
      "25\t85904\t0.2\t1\t80404 5500\n",
      "26\t79805\t0.2\t1\t75204 4601\n",
      "27\t88623\t0.2\t1\t82274 6349\n",
      "28\t86173\t0.2\t1\t80506 5667\n",
      "29\t91469\t0.2\t1\t84355 7114\n",
      "30\t95793\t0.2\t1\t90568 5225\n",
      "31\t77688\t0.2\t1\t72851 4837\n",
      "32\t92600\t0.2\t1\t86938 5662\n",
      "33\t87750\t0.2\t1\t82121 5629\n",
      "34\t99316\t0.2\t1\t92291 7025\n",
      "35\t87021\t0.2\t1\t82303 4718\n",
      "36\t99065\t0.2\t1\t92449 6616\n",
      "37\t101447\t0.2\t1\t95128 6319\n",
      "38\t97080\t0.2\t1\t90884 6196\n",
      "39\t103846\t0.2\t1\t96530 7316\n",
      "40\t94829\t0.2\t1\t89475 5354\n",
      "41\t105310\t0.2\t1\t97483 7827\n",
      "42\t95966\t0.2\t1\t89196 6770\n",
      "43\t151686\t0.2\t1\t144208 7478\n",
      "44\t46892\t0.2\t1\t42892 4000\n",
      "45\t105867\t0.2\t1\t99520 6347\n",
      "46\t96566\t0.2\t1\t90512 6054\n",
      "47\t103096\t0.2\t1\t97153 5943\n",
      "48\t102489\t0.2\t1\t96145 6344\n",
      "49\t112243\t0.2\t1\t104909 7334\n",
      "50\t94609\t0.2\t1\t89107 5502\n",
      "51\t103165\t0.2\t1\t96761 6404\n",
      "52\t107310\t0.2\t1\t100073 7237\n",
      "53\t113172\t0.2\t1\t106618 6554\n",
      "54\t92162\t0.2\t1\t86748 5414\n",
      "55\t108541\t0.2\t1\t101655 6886\n",
      "56\t133677\t0.2\t1\t126253 7424\n",
      "57\t81980\t0.2\t1\t76375 5605\n",
      "58\t110163\t0.2\t1\t104385 5778\n",
      "59\t98316\t0.2\t1\t92340 5976\n",
      "60\t104988\t0.2\t1\t98449 6539\n",
      "61\t103910\t0.2\t1\t97861 6049\n",
      "62\t115389\t0.2\t1\t108890 6499\n",
      "63\t89259\t0.2\t1\t83482 5777\n",
      "64\t102584\t0.2\t1\t96877 5707\n",
      "65\t114383\t0.2\t1\t107700 6683\n",
      "66\t118785\t0.2\t1\t111674 7111\n",
      "67\t124927\t0.2\t1\t115790 9137\n",
      "68\t236831\t0.2\t1\t226263 10568\n",
      "69\t32490\t0.2\t1\t29503 2987\n",
      "70\t14743\t0.2\t1\t12126 2617\n",
      "71\t68006\t0.2\t1\t64211 3795\n",
      "72\t89001\t0.2\t1\t84078 4923\n",
      "73\t92459\t0.2\t1\t87195 5264\n",
      "74\t91411\t0.2\t1\t86036 5375\n",
      "75\t90970\t0.2\t1\t85629 5341\n",
      "76\t89709\t0.2\t1\t84538 5171\n",
      "77\t89140\t0.2\t1\t84081 5059\n",
      "78\t83843\t0.2\t1\t78811 5032\n",
      "79\t83141\t0.2\t1\t78204 4937\n",
      "80\t80837\t0.2\t1\t76149 4688\n",
      "81\t79618\t0.2\t1\t74844 4774\n",
      "82\t77902\t0.2\t1\t73376 4526\n",
      "83\t75605\t0.2\t1\t71189 4416\n",
      "84\t73149\t0.2\t1\t68920 4229\n",
      "85\t70442\t0.2\t1\t66393 4049\n",
      "86\t67311\t0.2\t1\t63325 3986\n",
      "87\t65584\t0.2\t1\t61688 3896\n",
      "88\t63773\t0.2\t1\t60070 3703\n",
      "89\t60201\t0.2\t1\t56655 3546\n",
      "90\t59708\t0.2\t1\t56173 3535\n",
      "91\t56748\t0.2\t1\t53363 3385\n",
      "92\t53671\t0.2\t1\t50489 3182\n",
      "93\t50670\t0.2\t1\t47624 3046\n",
      "94\t48704\t0.2\t1\t45782 2922\n",
      "95\t45200\t0.2\t1\t42460 2740\n",
      "96\t42497\t0.2\t1\t39903 2594\n",
      "97\t39947\t0.2\t1\t37451 2496\n",
      "98\t36683\t0.2\t1\t34398 2285\n",
      "99\t36692\t0.2\t1\t34265 2427\n",
      "100\t32868\t0.2\t1\t30853 2015\n",
      "101\t30512\t0.2\t1\t28489 2023\n",
      "102\t27906\t0.2\t1\t26115 1791\n",
      "103\t26175\t0.2\t1\t24518 1657\n",
      "104\t23983\t0.2\t1\t22398 1585\n",
      "105\t21704\t0.2\t1\t20274 1430\n",
      "106\t19667\t0.2\t1\t18358 1309\n",
      "107\t17851\t0.2\t1\t16610 1241\n",
      "108\t16494\t0.2\t1\t15292 1202\n",
      "109\t14962\t0.2\t1\t13856 1106\n",
      "110\t13424\t0.2\t1\t12418 1006\n",
      "111\t12110\t0.2\t1\t11218 892\n",
      "112\t10668\t0.2\t1\t9941 727\n",
      "113\t8879\t0.2\t1\t8200 679\n",
      "114\t7950\t0.2\t1\t7388 562\n",
      "115\t7058\t0.2\t1\t6505 553\n",
      "116\t6198\t0.2\t1\t5702 496\n",
      "117\t5071\t0.2\t1\t4638 433\n",
      "118\t4567\t0.2\t1\t4226 341\n",
      "119\t3829\t0.2\t1\t3494 335\n",
      "120\t3187\t0.2\t1\t2901 286\n",
      "121\t2891\t0.2\t1\t2612 279\n",
      "122\t2303\t0.2\t1\t2110 193\n",
      "123\t1890\t0.2\t1\t1705 185\n",
      "124\t1701\t0.2\t1\t1504 197\n",
      "125\t1363\t0.2\t1\t1208 155\n",
      "126\t1199\t0.2\t1\t1086 113\n",
      "127\t954\t0.2\t1\t859 95\n",
      "128\t812\t0.2\t1\t702 110\n",
      "129\t610\t0.2\t1\t539 71\n",
      "130\t529\t0.2\t1\t472 57\n",
      "131\t405\t0.2\t1\t354 51\n",
      "132\t324\t0.2\t1\t279 45\n",
      "133\t284\t0.2\t1\t248 36\n",
      "134\t306\t0.2\t1\t272 34\n",
      "135\t179\t0.2\t1\t161 18\n",
      "136\t123\t0.2\t1\t97 26\n",
      "137\t92\t0.2\t1\t81 11\n",
      "138\t74\t0.2\t1\t60 14\n",
      "139\t42\t0.2\t1\t38 4\n",
      "140\t25\t0.2\t1\t20 5\n",
      "141\t21\t0.2\t1\t16 5\n",
      "142\t15\t0.2\t1\t14 1\n",
      "143\t11\t0.2\t1\t8 3\n",
      "144\t7\t0.2\t1\t5 2\n",
      "145\t18\t0.2\t1\t14 4\n",
      "146\t10\t0.2\t1\t8 2\n",
      "147\t32\t0.2\t1\t32\n",
      "148\t16\t0.2\t1\t16\n",
      "149\t9\t0.2\t1\t5 4\n",
      "150\t128\t0.2\t1\t105 23\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "12939154 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2164 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 536.72 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,939,154\n",
      "Reads with adapters:                10,288,624 (79.5%)\n",
      "Reads written (passing filters):    12,939,154 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,940,873,100 bp\n",
      "Quality-trimmed:               9,202,420 bp (0.5%)\n",
      "Total written (filtered):  1,495,378,762 bp (77.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10288624 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.2%\n",
      "  C: 13.0%\n",
      "  G: 23.8%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1595771\t3234788.5\t0\t1595771\n",
      "2\t240375\t808697.1\t0\t240375\n",
      "3\t138443\t202174.3\t0\t138443\n",
      "4\t81428\t50543.6\t0\t81428\n",
      "5\t75038\t12635.9\t0\t75038\n",
      "6\t64775\t3159.0\t0\t64775\n",
      "7\t52085\t789.7\t0\t52085\n",
      "8\t75127\t197.4\t0\t75127\n",
      "9\t60527\t49.4\t0\t60355 172\n",
      "10\t65326\t12.3\t1\t62475 2851\n",
      "11\t69712\t3.1\t1\t65682 4030\n",
      "12\t70487\t0.8\t1\t66516 3971\n",
      "13\t78320\t0.2\t1\t74147 4173\n",
      "14\t73989\t0.2\t1\t69268 4721\n",
      "15\t60342\t0.2\t1\t57080 3262\n",
      "16\t88180\t0.2\t1\t83244 4936\n",
      "17\t69750\t0.2\t1\t65392 4358\n",
      "18\t67540\t0.2\t1\t63841 3699\n",
      "19\t85313\t0.2\t1\t80354 4959\n",
      "20\t79769\t0.2\t1\t75478 4291\n",
      "21\t70275\t0.2\t1\t66324 3951\n",
      "22\t79177\t0.2\t1\t74939 4238\n",
      "23\t82659\t0.2\t1\t78081 4578\n",
      "24\t81915\t0.2\t1\t77232 4683\n",
      "25\t93464\t0.2\t1\t88557 4907\n",
      "26\t77147\t0.2\t1\t72835 4312\n",
      "27\t85811\t0.2\t1\t80374 5437\n",
      "28\t80396\t0.2\t1\t76862 3534\n",
      "29\t85946\t0.2\t1\t81104 4842\n",
      "30\t85726\t0.2\t1\t81647 4079\n",
      "31\t92050\t0.2\t1\t87288 4762\n",
      "32\t83804\t0.2\t1\t79934 3870\n",
      "33\t92087\t0.2\t1\t87200 4887\n",
      "34\t119466\t0.2\t1\t113056 6410\n",
      "35\t67982\t0.2\t1\t64809 3173\n",
      "36\t94466\t0.2\t1\t89392 5074\n",
      "37\t100368\t0.2\t1\t95090 5278\n",
      "38\t94027\t0.2\t1\t89878 4149\n",
      "39\t91028\t0.2\t1\t87016 4012\n",
      "40\t101155\t0.2\t1\t95949 5206\n",
      "41\t98098\t0.2\t1\t93343 4755\n",
      "42\t100747\t0.2\t1\t96031 4716\n",
      "43\t92371\t0.2\t1\t88129 4242\n",
      "44\t106061\t0.2\t1\t101051 5010\n",
      "45\t109648\t0.2\t1\t104507 5141\n",
      "46\t85668\t0.2\t1\t81729 3939\n",
      "47\t104356\t0.2\t1\t99808 4548\n",
      "48\t101281\t0.2\t1\t96981 4300\n",
      "49\t106782\t0.2\t1\t101997 4785\n",
      "50\t100514\t0.2\t1\t95958 4556\n",
      "51\t125782\t0.2\t1\t120079 5703\n",
      "52\t90772\t0.2\t1\t86983 3789\n",
      "53\t109598\t0.2\t1\t105306 4292\n",
      "54\t83322\t0.2\t1\t79849 3473\n",
      "55\t107016\t0.2\t1\t102862 4154\n",
      "56\t110967\t0.2\t1\t106233 4734\n",
      "57\t104357\t0.2\t1\t100229 4128\n",
      "58\t100691\t0.2\t1\t96894 3797\n",
      "59\t102578\t0.2\t1\t98489 4089\n",
      "60\t106303\t0.2\t1\t102044 4259\n",
      "61\t108531\t0.2\t1\t104028 4503\n",
      "62\t118161\t0.2\t1\t113451 4710\n",
      "63\t158201\t0.2\t1\t152213 5988\n",
      "64\t129612\t0.2\t1\t125287 4325\n",
      "65\t83984\t0.2\t1\t80809 3175\n",
      "66\t91901\t0.2\t1\t88385 3516\n",
      "67\t89742\t0.2\t1\t86128 3614\n",
      "68\t91530\t0.2\t1\t88057 3473\n",
      "69\t90402\t0.2\t1\t86917 3485\n",
      "70\t96053\t0.2\t1\t92305 3748\n",
      "71\t99803\t0.2\t1\t96104 3699\n",
      "72\t94159\t0.2\t1\t90619 3540\n",
      "73\t98787\t0.2\t1\t95125 3662\n",
      "74\t90680\t0.2\t1\t87379 3301\n",
      "75\t86705\t0.2\t1\t83434 3271\n",
      "76\t84928\t0.2\t1\t81792 3136\n",
      "77\t84096\t0.2\t1\t80874 3222\n",
      "78\t80178\t0.2\t1\t77109 3069\n",
      "79\t77254\t0.2\t1\t74397 2857\n",
      "80\t77028\t0.2\t1\t74180 2848\n",
      "81\t75249\t0.2\t1\t72550 2699\n",
      "82\t72339\t0.2\t1\t69794 2545\n",
      "83\t70398\t0.2\t1\t67790 2608\n",
      "84\t68189\t0.2\t1\t65635 2554\n",
      "85\t66784\t0.2\t1\t64244 2540\n",
      "86\t65482\t0.2\t1\t63096 2386\n",
      "87\t67016\t0.2\t1\t64553 2463\n",
      "88\t64151\t0.2\t1\t61741 2410\n",
      "89\t57363\t0.2\t1\t55205 2158\n",
      "90\t54985\t0.2\t1\t52902 2083\n",
      "91\t52080\t0.2\t1\t50216 1864\n",
      "92\t48422\t0.2\t1\t46687 1735\n",
      "93\t47310\t0.2\t1\t45600 1710\n",
      "94\t45459\t0.2\t1\t43795 1664\n",
      "95\t43150\t0.2\t1\t41608 1542\n",
      "96\t40380\t0.2\t1\t38964 1416\n",
      "97\t36752\t0.2\t1\t35440 1312\n",
      "98\t33446\t0.2\t1\t32251 1195\n",
      "99\t33482\t0.2\t1\t32255 1227\n",
      "100\t30657\t0.2\t1\t29566 1091\n",
      "101\t28785\t0.2\t1\t27832 953\n",
      "102\t27470\t0.2\t1\t26566 904\n",
      "103\t25360\t0.2\t1\t24494 866\n",
      "104\t23640\t0.2\t1\t22887 753\n",
      "105\t21192\t0.2\t1\t20482 710\n",
      "106\t19098\t0.2\t1\t18446 652\n",
      "107\t16996\t0.2\t1\t16368 628\n",
      "108\t15172\t0.2\t1\t14616 556\n",
      "109\t14001\t0.2\t1\t13446 555\n",
      "110\t12575\t0.2\t1\t12154 421\n",
      "111\t11902\t0.2\t1\t11531 371\n",
      "112\t10341\t0.2\t1\t9986 355\n",
      "113\t8669\t0.2\t1\t8372 297\n",
      "114\t7690\t0.2\t1\t7405 285\n",
      "115\t6634\t0.2\t1\t6377 257\n",
      "116\t5717\t0.2\t1\t5480 237\n",
      "117\t4541\t0.2\t1\t4344 197\n",
      "118\t4158\t0.2\t1\t3977 181\n",
      "119\t3510\t0.2\t1\t3351 159\n",
      "120\t2896\t0.2\t1\t2771 125\n",
      "121\t2680\t0.2\t1\t2560 120\n",
      "122\t2179\t0.2\t1\t2075 104\n",
      "123\t1805\t0.2\t1\t1731 74\n",
      "124\t1586\t0.2\t1\t1495 91\n",
      "125\t1276\t0.2\t1\t1206 70\n",
      "126\t1153\t0.2\t1\t1112 41\n",
      "127\t891\t0.2\t1\t852 39\n",
      "128\t721\t0.2\t1\t684 37\n",
      "129\t559\t0.2\t1\t529 30\n",
      "130\t484\t0.2\t1\t453 31\n",
      "131\t369\t0.2\t1\t355 14\n",
      "132\t299\t0.2\t1\t280 19\n",
      "133\t268\t0.2\t1\t245 23\n",
      "134\t297\t0.2\t1\t280 17\n",
      "135\t162\t0.2\t1\t155 7\n",
      "136\t110\t0.2\t1\t102 8\n",
      "137\t81\t0.2\t1\t76 5\n",
      "138\t67\t0.2\t1\t59 8\n",
      "139\t34\t0.2\t1\t30 4\n",
      "140\t26\t0.2\t1\t23 3\n",
      "141\t15\t0.2\t1\t15\n",
      "142\t15\t0.2\t1\t15\n",
      "143\t8\t0.2\t1\t8\n",
      "144\t6\t0.2\t1\t4 2\n",
      "145\t17\t0.2\t1\t15 2\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t33\t0.2\t1\t27 6\n",
      "148\t14\t0.2\t1\t14\n",
      "149\t8\t0.2\t1\t6 2\n",
      "150\t114\t0.2\t1\t107 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "12939154 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t15624 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 519.98 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,868,767\n",
      "Reads with adapters:                10,286,843 (79.9%)\n",
      "Reads written (passing filters):    12,868,767 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,930,315,050 bp\n",
      "Quality-trimmed:               3,674,826 bp (0.2%)\n",
      "Total written (filtered):  1,483,181,496 bp (76.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10286843 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.7%\n",
      "  G: 20.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1541222\t3217191.8\t0\t1541222\n",
      "2\t228030\t804297.9\t0\t228030\n",
      "3\t129528\t201074.5\t0\t129528\n",
      "4\t78688\t50268.6\t0\t78688\n",
      "5\t67124\t12567.2\t0\t67124\n",
      "6\t66330\t3141.8\t0\t66330\n",
      "7\t52508\t785.4\t0\t52508\n",
      "8\t67652\t196.4\t0\t67652\n",
      "9\t67615\t49.1\t0\t66819 796\n",
      "10\t63063\t12.3\t1\t59213 3850\n",
      "11\t71894\t3.1\t1\t65954 5940\n",
      "12\t67897\t0.8\t1\t62829 5068\n",
      "13\t71151\t0.2\t1\t65849 5302\n",
      "14\t69055\t0.2\t1\t63709 5346\n",
      "15\t75069\t0.2\t1\t69302 5767\n",
      "16\t71424\t0.2\t1\t65420 6004\n",
      "17\t86810\t0.2\t1\t79021 7789\n",
      "18\t77955\t0.2\t1\t72600 5355\n",
      "19\t63383\t0.2\t1\t58809 4574\n",
      "20\t78965\t0.2\t1\t73063 5902\n",
      "21\t85485\t0.2\t1\t79014 6471\n",
      "22\t72855\t0.2\t1\t68021 4834\n",
      "23\t85346\t0.2\t1\t78944 6402\n",
      "24\t80318\t0.2\t1\t73301 7017\n",
      "25\t85430\t0.2\t1\t78969 6461\n",
      "26\t78740\t0.2\t1\t73485 5255\n",
      "27\t88451\t0.2\t1\t81474 6977\n",
      "28\t85790\t0.2\t1\t79338 6452\n",
      "29\t91791\t0.2\t1\t83845 7946\n",
      "30\t95449\t0.2\t1\t89482 5967\n",
      "31\t75838\t0.2\t1\t70423 5415\n",
      "32\t92731\t0.2\t1\t85851 6880\n",
      "33\t86762\t0.2\t1\t80055 6707\n",
      "34\t99437\t0.2\t1\t91057 8380\n",
      "35\t85081\t0.2\t1\t79384 5697\n",
      "36\t99297\t0.2\t1\t91300 7997\n",
      "37\t101918\t0.2\t1\t94163 7755\n",
      "38\t96379\t0.2\t1\t88997 7382\n",
      "39\t103865\t0.2\t1\t95226 8639\n",
      "40\t94376\t0.2\t1\t87615 6761\n",
      "41\t104329\t0.2\t1\t95123 9206\n",
      "42\t94892\t0.2\t1\t86892 8000\n",
      "43\t148535\t0.2\t1\t139624 8911\n",
      "44\t48085\t0.2\t1\t43529 4556\n",
      "45\t105311\t0.2\t1\t97887 7424\n",
      "46\t95734\t0.2\t1\t88889 6845\n",
      "47\t101958\t0.2\t1\t94804 7154\n",
      "48\t102091\t0.2\t1\t94808 7283\n",
      "49\t112404\t0.2\t1\t104035 8369\n",
      "50\t93295\t0.2\t1\t87104 6191\n",
      "51\t104120\t0.2\t1\t96595 7525\n",
      "52\t107257\t0.2\t1\t99149 8108\n",
      "53\t113672\t0.2\t1\t105961 7711\n",
      "54\t91213\t0.2\t1\t85172 6041\n",
      "55\t107994\t0.2\t1\t100085 7909\n",
      "56\t133513\t0.2\t1\t124815 8698\n",
      "57\t81884\t0.2\t1\t75254 6630\n",
      "58\t110562\t0.2\t1\t103755 6807\n",
      "59\t98179\t0.2\t1\t91032 7147\n",
      "60\t104453\t0.2\t1\t97116 7337\n",
      "61\t104191\t0.2\t1\t97239 6952\n",
      "62\t114950\t0.2\t1\t107431 7519\n",
      "63\t89541\t0.2\t1\t82900 6641\n",
      "64\t101345\t0.2\t1\t94868 6477\n",
      "65\t113295\t0.2\t1\t105436 7859\n",
      "66\t116379\t0.2\t1\t108386 7993\n",
      "67\t124994\t0.2\t1\t114943 10051\n",
      "68\t229157\t0.2\t1\t217120 12037\n",
      "69\t41354\t0.2\t1\t37561 3793\n",
      "70\t17861\t0.2\t1\t14972 2889\n",
      "71\t66015\t0.2\t1\t61819 4196\n",
      "72\t88699\t0.2\t1\t83225 5474\n",
      "73\t92274\t0.2\t1\t86536 5738\n",
      "74\t90496\t0.2\t1\t84396 6100\n",
      "75\t91072\t0.2\t1\t84946 6126\n",
      "76\t89676\t0.2\t1\t83357 6319\n",
      "77\t89231\t0.2\t1\t83068 6163\n",
      "78\t83699\t0.2\t1\t77980 5719\n",
      "79\t83174\t0.2\t1\t77470 5704\n",
      "80\t80873\t0.2\t1\t75140 5733\n",
      "81\t79905\t0.2\t1\t74411 5494\n",
      "82\t77850\t0.2\t1\t72450 5400\n",
      "83\t75463\t0.2\t1\t70383 5080\n",
      "84\t72909\t0.2\t1\t67760 5149\n",
      "85\t70237\t0.2\t1\t65392 4845\n",
      "86\t67846\t0.2\t1\t63270 4576\n",
      "87\t65212\t0.2\t1\t60837 4375\n",
      "88\t63724\t0.2\t1\t59434 4290\n",
      "89\t60287\t0.2\t1\t56154 4133\n",
      "90\t60221\t0.2\t1\t55941 4280\n",
      "91\t56630\t0.2\t1\t52736 3894\n",
      "92\t53891\t0.2\t1\t50132 3759\n",
      "93\t50029\t0.2\t1\t46571 3458\n",
      "94\t48812\t0.2\t1\t45399 3413\n",
      "95\t45157\t0.2\t1\t41906 3251\n",
      "96\t42860\t0.2\t1\t39861 2999\n",
      "97\t40400\t0.2\t1\t37713 2687\n",
      "98\t37633\t0.2\t1\t34970 2663\n",
      "99\t36386\t0.2\t1\t33717 2669\n",
      "100\t32744\t0.2\t1\t30375 2369\n",
      "101\t30684\t0.2\t1\t28451 2233\n",
      "102\t28336\t0.2\t1\t26275 2061\n",
      "103\t26125\t0.2\t1\t24257 1868\n",
      "104\t24496\t0.2\t1\t22774 1722\n",
      "105\t21603\t0.2\t1\t20114 1489\n",
      "106\t19703\t0.2\t1\t18252 1451\n",
      "107\t18114\t0.2\t1\t16740 1374\n",
      "108\t16491\t0.2\t1\t15275 1216\n",
      "109\t15050\t0.2\t1\t13866 1184\n",
      "110\t13423\t0.2\t1\t12430 993\n",
      "111\t12182\t0.2\t1\t11225 957\n",
      "112\t10547\t0.2\t1\t9689 858\n",
      "113\t9161\t0.2\t1\t8400 761\n",
      "114\t7859\t0.2\t1\t7209 650\n",
      "115\t6948\t0.2\t1\t6344 604\n",
      "116\t6269\t0.2\t1\t5743 526\n",
      "117\t5202\t0.2\t1\t4768 434\n",
      "118\t4673\t0.2\t1\t4242 431\n",
      "119\t3797\t0.2\t1\t3484 313\n",
      "120\t3288\t0.2\t1\t2990 298\n",
      "121\t2786\t0.2\t1\t2513 273\n",
      "122\t2451\t0.2\t1\t2220 231\n",
      "123\t1988\t0.2\t1\t1766 222\n",
      "124\t1684\t0.2\t1\t1506 178\n",
      "125\t1280\t0.2\t1\t1137 143\n",
      "126\t1164\t0.2\t1\t1040 124\n",
      "127\t900\t0.2\t1\t789 111\n",
      "128\t728\t0.2\t1\t654 74\n",
      "129\t628\t0.2\t1\t547 81\n",
      "130\t442\t0.2\t1\t393 49\n",
      "131\t450\t0.2\t1\t380 70\n",
      "132\t321\t0.2\t1\t277 44\n",
      "133\t283\t0.2\t1\t243 40\n",
      "134\t293\t0.2\t1\t262 31\n",
      "135\t181\t0.2\t1\t154 27\n",
      "136\t135\t0.2\t1\t102 33\n",
      "137\t78\t0.2\t1\t61 17\n",
      "138\t63\t0.2\t1\t56 7\n",
      "139\t49\t0.2\t1\t46 3\n",
      "140\t36\t0.2\t1\t28 8\n",
      "141\t27\t0.2\t1\t20 7\n",
      "142\t11\t0.2\t1\t8 3\n",
      "143\t11\t0.2\t1\t10 1\n",
      "144\t9\t0.2\t1\t4 5\n",
      "145\t13\t0.2\t1\t10 3\n",
      "146\t15\t0.2\t1\t11 4\n",
      "147\t27\t0.2\t1\t25 2\n",
      "148\t26\t0.2\t1\t23 3\n",
      "149\t15\t0.2\t1\t15\n",
      "150\t143\t0.2\t1\t117 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "12868767 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2224 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_2_S6_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 547.25 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,868,767\n",
      "Reads with adapters:                10,225,148 (79.5%)\n",
      "Reads written (passing filters):    12,868,767 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,930,315,050 bp\n",
      "Quality-trimmed:               8,911,212 bp (0.5%)\n",
      "Total written (filtered):  1,486,049,208 bp (77.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10225148 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 13.1%\n",
      "  G: 23.9%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1581903\t3217191.8\t0\t1581903\n",
      "2\t233784\t804297.9\t0\t233784\n",
      "3\t139412\t201074.5\t0\t139412\n",
      "4\t79292\t50268.6\t0\t79292\n",
      "5\t72507\t12567.2\t0\t72507\n",
      "6\t64695\t3141.8\t0\t64695\n",
      "7\t50784\t785.4\t0\t50784\n",
      "8\t73943\t196.4\t0\t73943\n",
      "9\t59253\t49.1\t0\t59073 180\n",
      "10\t62906\t12.3\t1\t58824 4082\n",
      "11\t69220\t3.1\t1\t63844 5376\n",
      "12\t67633\t0.8\t1\t62443 5190\n",
      "13\t78337\t0.2\t1\t72455 5882\n",
      "14\t72684\t0.2\t1\t66542 6142\n",
      "15\t59253\t0.2\t1\t54996 4257\n",
      "16\t85878\t0.2\t1\t79407 6471\n",
      "17\t71616\t0.2\t1\t66058 5558\n",
      "18\t64381\t0.2\t1\t59775 4606\n",
      "19\t83620\t0.2\t1\t77611 6009\n",
      "20\t78383\t0.2\t1\t72906 5477\n",
      "21\t69471\t0.2\t1\t64687 4784\n",
      "22\t78479\t0.2\t1\t73016 5463\n",
      "23\t83210\t0.2\t1\t77230 5980\n",
      "24\t82011\t0.2\t1\t76117 5894\n",
      "25\t93094\t0.2\t1\t87025 6069\n",
      "26\t77459\t0.2\t1\t71634 5825\n",
      "27\t86527\t0.2\t1\t79790 6737\n",
      "28\t77466\t0.2\t1\t72862 4604\n",
      "29\t87078\t0.2\t1\t81023 6055\n",
      "30\t83340\t0.2\t1\t78135 5205\n",
      "31\t92587\t0.2\t1\t86504 6083\n",
      "32\t80491\t0.2\t1\t75656 4835\n",
      "33\t92967\t0.2\t1\t86621 6346\n",
      "34\t120697\t0.2\t1\t112803 7894\n",
      "35\t64048\t0.2\t1\t59979 4069\n",
      "36\t95138\t0.2\t1\t88610 6528\n",
      "37\t100610\t0.2\t1\t93904 6706\n",
      "38\t93157\t0.2\t1\t87717 5440\n",
      "39\t88669\t0.2\t1\t83789 4880\n",
      "40\t101334\t0.2\t1\t95182 6152\n",
      "41\t98089\t0.2\t1\t92269 5820\n",
      "42\t100106\t0.2\t1\t94212 5894\n",
      "43\t90866\t0.2\t1\t85570 5296\n",
      "44\t106583\t0.2\t1\t100029 6554\n",
      "45\t108549\t0.2\t1\t101877 6672\n",
      "46\t84237\t0.2\t1\t79394 4843\n",
      "47\t103605\t0.2\t1\t98067 5538\n",
      "48\t100222\t0.2\t1\t94797 5425\n",
      "49\t105528\t0.2\t1\t99377 6151\n",
      "50\t101252\t0.2\t1\t95306 5946\n",
      "51\t124662\t0.2\t1\t117482 7180\n",
      "52\t89980\t0.2\t1\t85108 4872\n",
      "53\t112122\t0.2\t1\t106274 5848\n",
      "54\t81922\t0.2\t1\t77760 4162\n",
      "55\t105894\t0.2\t1\t100558 5336\n",
      "56\t109567\t0.2\t1\t103471 6096\n",
      "57\t105084\t0.2\t1\t100060 5024\n",
      "58\t101375\t0.2\t1\t96453 4922\n",
      "59\t101412\t0.2\t1\t96219 5193\n",
      "60\t106019\t0.2\t1\t100584 5435\n",
      "61\t107866\t0.2\t1\t102516 5350\n",
      "62\t117046\t0.2\t1\t111012 6034\n",
      "63\t155210\t0.2\t1\t148115 7095\n",
      "64\t127345\t0.2\t1\t122328 5017\n",
      "65\t83662\t0.2\t1\t79683 3979\n",
      "66\t91095\t0.2\t1\t86676 4419\n",
      "67\t90432\t0.2\t1\t85682 4750\n",
      "68\t91566\t0.2\t1\t87021 4545\n",
      "69\t91038\t0.2\t1\t86455 4583\n",
      "70\t97071\t0.2\t1\t92328 4743\n",
      "71\t100160\t0.2\t1\t95222 4938\n",
      "72\t95370\t0.2\t1\t90793 4577\n",
      "73\t99146\t0.2\t1\t94377 4769\n",
      "74\t90186\t0.2\t1\t85699 4487\n",
      "75\t87292\t0.2\t1\t82908 4384\n",
      "76\t84846\t0.2\t1\t80550 4296\n",
      "77\t84110\t0.2\t1\t79782 4328\n",
      "78\t80083\t0.2\t1\t76044 4039\n",
      "79\t77641\t0.2\t1\t73764 3877\n",
      "80\t76649\t0.2\t1\t72933 3716\n",
      "81\t75643\t0.2\t1\t71998 3645\n",
      "82\t72511\t0.2\t1\t69001 3510\n",
      "83\t70456\t0.2\t1\t67059 3397\n",
      "84\t67826\t0.2\t1\t64473 3353\n",
      "85\t66652\t0.2\t1\t63633 3019\n",
      "86\t65922\t0.2\t1\t62797 3125\n",
      "87\t65939\t0.2\t1\t62926 3013\n",
      "88\t64115\t0.2\t1\t61226 2889\n",
      "89\t57559\t0.2\t1\t54978 2581\n",
      "90\t55692\t0.2\t1\t53148 2544\n",
      "91\t51986\t0.2\t1\t49669 2317\n",
      "92\t49131\t0.2\t1\t46914 2217\n",
      "93\t46886\t0.2\t1\t44668 2218\n",
      "94\t45747\t0.2\t1\t43801 1946\n",
      "95\t42950\t0.2\t1\t40887 2063\n",
      "96\t40716\t0.2\t1\t38860 1856\n",
      "97\t37404\t0.2\t1\t35730 1674\n",
      "98\t34443\t0.2\t1\t32843 1600\n",
      "99\t33264\t0.2\t1\t31734 1530\n",
      "100\t30705\t0.2\t1\t29262 1443\n",
      "101\t28943\t0.2\t1\t27671 1272\n",
      "102\t27964\t0.2\t1\t26691 1273\n",
      "103\t25366\t0.2\t1\t24255 1111\n",
      "104\t24064\t0.2\t1\t22990 1074\n",
      "105\t21070\t0.2\t1\t20080 990\n",
      "106\t19078\t0.2\t1\t18244 834\n",
      "107\t17343\t0.2\t1\t16582 761\n",
      "108\t15238\t0.2\t1\t14525 713\n",
      "109\t14062\t0.2\t1\t13403 659\n",
      "110\t12670\t0.2\t1\t12101 569\n",
      "111\t11892\t0.2\t1\t11362 530\n",
      "112\t10243\t0.2\t1\t9780 463\n",
      "113\t8834\t0.2\t1\t8449 385\n",
      "114\t7653\t0.2\t1\t7300 353\n",
      "115\t6468\t0.2\t1\t6167 301\n",
      "116\t5767\t0.2\t1\t5513 254\n",
      "117\t4744\t0.2\t1\t4519 225\n",
      "118\t4267\t0.2\t1\t4031 236\n",
      "119\t3534\t0.2\t1\t3331 203\n",
      "120\t2971\t0.2\t1\t2813 158\n",
      "121\t2616\t0.2\t1\t2484 132\n",
      "122\t2300\t0.2\t1\t2180 120\n",
      "123\t1889\t0.2\t1\t1769 120\n",
      "124\t1591\t0.2\t1\t1513 78\n",
      "125\t1198\t0.2\t1\t1143 55\n",
      "126\t1107\t0.2\t1\t1053 54\n",
      "127\t843\t0.2\t1\t791 52\n",
      "128\t694\t0.2\t1\t650 44\n",
      "129\t581\t0.2\t1\t539 42\n",
      "130\t407\t0.2\t1\t381 26\n",
      "131\t412\t0.2\t1\t395 17\n",
      "132\t294\t0.2\t1\t275 19\n",
      "133\t274\t0.2\t1\t263 11\n",
      "134\t295\t0.2\t1\t282 13\n",
      "135\t162\t0.2\t1\t152 10\n",
      "136\t127\t0.2\t1\t120 7\n",
      "137\t72\t0.2\t1\t67 5\n",
      "138\t53\t0.2\t1\t46 7\n",
      "139\t48\t0.2\t1\t43 5\n",
      "140\t33\t0.2\t1\t30 3\n",
      "141\t23\t0.2\t1\t20 3\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t12\t0.2\t1\t11 1\n",
      "144\t5\t0.2\t1\t3 2\n",
      "145\t11\t0.2\t1\t11\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t30\t0.2\t1\t28 2\n",
      "148\t16\t0.2\t1\t14 2\n",
      "149\t14\t0.2\t1\t13 1\n",
      "150\t92\t0.2\t1\t87 5\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "12868767 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t14322 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 537.88 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,166,205\n",
      "Reads with adapters:                11,846,180 (83.6%)\n",
      "Reads written (passing filters):    14,166,205 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,124,930,750 bp\n",
      "Quality-trimmed:               4,298,957 bp (0.2%)\n",
      "Total written (filtered):  1,559,729,002 bp (73.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11846180 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 15.1%\n",
      "  G: 21.0%\n",
      "  T: 34.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1373757\t3541551.2\t0\t1373757\n",
      "2\t216270\t885387.8\t0\t216270\n",
      "3\t129588\t221347.0\t0\t129588\n",
      "4\t80077\t55336.7\t0\t80077\n",
      "5\t69163\t13834.2\t0\t69163\n",
      "6\t67518\t3458.5\t0\t67518\n",
      "7\t56035\t864.6\t0\t56035\n",
      "8\t70267\t216.2\t0\t70267\n",
      "9\t70629\t54.0\t0\t69702 927\n",
      "10\t66297\t13.5\t1\t62862 3435\n",
      "11\t73903\t3.4\t1\t68283 5620\n",
      "12\t71104\t0.8\t1\t66580 4524\n",
      "13\t72370\t0.2\t1\t67647 4723\n",
      "14\t73869\t0.2\t1\t68608 5261\n",
      "15\t77649\t0.2\t1\t72603 5046\n",
      "16\t76835\t0.2\t1\t71293 5542\n",
      "17\t93103\t0.2\t1\t85654 7449\n",
      "18\t80962\t0.2\t1\t76131 4831\n",
      "19\t69251\t0.2\t1\t64739 4512\n",
      "20\t83140\t0.2\t1\t77746 5394\n",
      "21\t91983\t0.2\t1\t85635 6348\n",
      "22\t78972\t0.2\t1\t74340 4632\n",
      "23\t91741\t0.2\t1\t85612 6129\n",
      "24\t86730\t0.2\t1\t79637 7093\n",
      "25\t93905\t0.2\t1\t87575 6330\n",
      "26\t84785\t0.2\t1\t79850 4935\n",
      "27\t95659\t0.2\t1\t88641 7018\n",
      "28\t93289\t0.2\t1\t86777 6512\n",
      "29\t102324\t0.2\t1\t94020 8304\n",
      "30\t106573\t0.2\t1\t100735 5838\n",
      "31\t82302\t0.2\t1\t76920 5382\n",
      "32\t102146\t0.2\t1\t95503 6643\n",
      "33\t96707\t0.2\t1\t90380 6327\n",
      "34\t106974\t0.2\t1\t99423 7551\n",
      "35\t107836\t0.2\t1\t100264 7572\n",
      "36\t113175\t0.2\t1\t105186 7989\n",
      "37\t100704\t0.2\t1\t95019 5685\n",
      "38\t121468\t0.2\t1\t112012 9456\n",
      "39\t103157\t0.2\t1\t96822 6335\n",
      "40\t115108\t0.2\t1\t108010 7098\n",
      "41\t129750\t0.2\t1\t120171 9579\n",
      "42\t102179\t0.2\t1\t94823 7356\n",
      "43\t171532\t0.2\t1\t162826 8706\n",
      "44\t55606\t0.2\t1\t50892 4714\n",
      "45\t120800\t0.2\t1\t113289 7511\n",
      "46\t113431\t0.2\t1\t106413 7018\n",
      "47\t120541\t0.2\t1\t113256 7285\n",
      "48\t119747\t0.2\t1\t111882 7865\n",
      "49\t130730\t0.2\t1\t121756 8974\n",
      "50\t111508\t0.2\t1\t104914 6594\n",
      "51\t122095\t0.2\t1\t114233 7862\n",
      "52\t127642\t0.2\t1\t118613 9029\n",
      "53\t137409\t0.2\t1\t129309 8100\n",
      "54\t108827\t0.2\t1\t102141 6686\n",
      "55\t130904\t0.2\t1\t122687 8217\n",
      "56\t157783\t0.2\t1\t148768 9015\n",
      "57\t104771\t0.2\t1\t97389 7382\n",
      "58\t132433\t0.2\t1\t125333 7100\n",
      "59\t115612\t0.2\t1\t108308 7304\n",
      "60\t130650\t0.2\t1\t122277 8373\n",
      "61\t128910\t0.2\t1\t121303 7607\n",
      "62\t139690\t0.2\t1\t131489 8201\n",
      "63\t114068\t0.2\t1\t106664 7404\n",
      "64\t125901\t0.2\t1\t118809 7092\n",
      "65\t141358\t0.2\t1\t132756 8602\n",
      "66\t146823\t0.2\t1\t137956 8867\n",
      "67\t159981\t0.2\t1\t148074 11907\n",
      "68\t301471\t0.2\t1\t287602 13869\n",
      "69\t51550\t0.2\t1\t47181 4369\n",
      "70\t21339\t0.2\t1\t17862 3477\n",
      "71\t81265\t0.2\t1\t76296 4969\n",
      "72\t113463\t0.2\t1\t106973 6490\n",
      "73\t120176\t0.2\t1\t113199 6977\n",
      "74\t119128\t0.2\t1\t112036 7092\n",
      "75\t118829\t0.2\t1\t111689 7140\n",
      "76\t117859\t0.2\t1\t110976 6883\n",
      "77\t119047\t0.2\t1\t112165 6882\n",
      "78\t111406\t0.2\t1\t104659 6747\n",
      "79\t110628\t0.2\t1\t103829 6799\n",
      "80\t108577\t0.2\t1\t102197 6380\n",
      "81\t107611\t0.2\t1\t101254 6357\n",
      "82\t104685\t0.2\t1\t98322 6363\n",
      "83\t101689\t0.2\t1\t95553 6136\n",
      "84\t99042\t0.2\t1\t93220 5822\n",
      "85\t96588\t0.2\t1\t90890 5698\n",
      "86\t93102\t0.2\t1\t87585 5517\n",
      "87\t89497\t0.2\t1\t84148 5349\n",
      "88\t88681\t0.2\t1\t83310 5371\n",
      "89\t83884\t0.2\t1\t78862 5022\n",
      "90\t83820\t0.2\t1\t78644 5176\n",
      "91\t79519\t0.2\t1\t74630 4889\n",
      "92\t75751\t0.2\t1\t71312 4439\n",
      "93\t71060\t0.2\t1\t66734 4326\n",
      "94\t68805\t0.2\t1\t64395 4410\n",
      "95\t63794\t0.2\t1\t59908 3886\n",
      "96\t60732\t0.2\t1\t57025 3707\n",
      "97\t57614\t0.2\t1\t54075 3539\n",
      "98\t53272\t0.2\t1\t49912 3360\n",
      "99\t52704\t0.2\t1\t49079 3625\n",
      "100\t47213\t0.2\t1\t44245 2968\n",
      "101\t43905\t0.2\t1\t41069 2836\n",
      "102\t40757\t0.2\t1\t38090 2667\n",
      "103\t36990\t0.2\t1\t34581 2409\n",
      "104\t34678\t0.2\t1\t32449 2229\n",
      "105\t30517\t0.2\t1\t28453 2064\n",
      "106\t28231\t0.2\t1\t26371 1860\n",
      "107\t25422\t0.2\t1\t23676 1746\n",
      "108\t24129\t0.2\t1\t22480 1649\n",
      "109\t20909\t0.2\t1\t19298 1611\n",
      "110\t19145\t0.2\t1\t17778 1367\n",
      "111\t17228\t0.2\t1\t15946 1282\n",
      "112\t14959\t0.2\t1\t13818 1141\n",
      "113\t12878\t0.2\t1\t11894 984\n",
      "114\t11383\t0.2\t1\t10467 916\n",
      "115\t9899\t0.2\t1\t9096 803\n",
      "116\t8618\t0.2\t1\t7872 746\n",
      "117\t7168\t0.2\t1\t6561 607\n",
      "118\t6256\t0.2\t1\t5711 545\n",
      "119\t5200\t0.2\t1\t4764 436\n",
      "120\t4401\t0.2\t1\t4012 389\n",
      "121\t3754\t0.2\t1\t3388 366\n",
      "122\t3023\t0.2\t1\t2711 312\n",
      "123\t2648\t0.2\t1\t2374 274\n",
      "124\t2210\t0.2\t1\t1991 219\n",
      "125\t1851\t0.2\t1\t1661 190\n",
      "126\t1538\t0.2\t1\t1379 159\n",
      "127\t1281\t0.2\t1\t1146 135\n",
      "128\t1028\t0.2\t1\t906 122\n",
      "129\t844\t0.2\t1\t750 94\n",
      "130\t578\t0.2\t1\t506 72\n",
      "131\t477\t0.2\t1\t408 69\n",
      "132\t403\t0.2\t1\t349 54\n",
      "133\t425\t0.2\t1\t364 61\n",
      "134\t500\t0.2\t1\t457 43\n",
      "135\t253\t0.2\t1\t229 24\n",
      "136\t154\t0.2\t1\t133 21\n",
      "137\t116\t0.2\t1\t99 17\n",
      "138\t91\t0.2\t1\t74 17\n",
      "139\t63\t0.2\t1\t48 15\n",
      "140\t40\t0.2\t1\t34 6\n",
      "141\t25\t0.2\t1\t20 5\n",
      "142\t20\t0.2\t1\t12 8\n",
      "143\t19\t0.2\t1\t15 4\n",
      "144\t21\t0.2\t1\t17 4\n",
      "145\t20\t0.2\t1\t16 4\n",
      "146\t14\t0.2\t1\t12 2\n",
      "147\t31\t0.2\t1\t27 4\n",
      "148\t15\t0.2\t1\t13 2\n",
      "149\t18\t0.2\t1\t14 4\n",
      "150\t240\t0.2\t1\t195 45\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14166205 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2985 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 35% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 557.64 s (39 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,166,205\n",
      "Reads with adapters:                11,771,953 (83.1%)\n",
      "Reads written (passing filters):    14,166,205 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,124,930,750 bp\n",
      "Quality-trimmed:              10,797,724 bp (0.5%)\n",
      "Total written (filtered):  1,563,454,753 bp (73.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11771953 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.6%\n",
      "  C: 13.3%\n",
      "  G: 24.5%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1427757\t3541551.2\t0\t1427757\n",
      "2\t222493\t885387.8\t0\t222493\n",
      "3\t133332\t221347.0\t0\t133332\n",
      "4\t79137\t55336.7\t0\t79137\n",
      "5\t75786\t13834.2\t0\t75786\n",
      "6\t65915\t3458.5\t0\t65915\n",
      "7\t53576\t864.6\t0\t53576\n",
      "8\t76744\t216.2\t0\t76744\n",
      "9\t61976\t54.0\t0\t61779 197\n",
      "10\t67080\t13.5\t1\t64106 2974\n",
      "11\t71681\t3.4\t1\t67501 4180\n",
      "12\t72222\t0.8\t1\t68053 4169\n",
      "13\t80561\t0.2\t1\t76346 4215\n",
      "14\t76520\t0.2\t1\t71606 4914\n",
      "15\t62607\t0.2\t1\t59130 3477\n",
      "16\t91383\t0.2\t1\t86350 5033\n",
      "17\t72745\t0.2\t1\t68212 4533\n",
      "18\t70883\t0.2\t1\t67020 3863\n",
      "19\t89801\t0.2\t1\t84529 5272\n",
      "20\t83841\t0.2\t1\t79367 4474\n",
      "21\t75027\t0.2\t1\t70918 4109\n",
      "22\t84425\t0.2\t1\t79923 4502\n",
      "23\t88024\t0.2\t1\t83097 4927\n",
      "24\t88855\t0.2\t1\t83891 4964\n",
      "25\t101226\t0.2\t1\t95974 5252\n",
      "26\t83308\t0.2\t1\t78591 4717\n",
      "27\t92909\t0.2\t1\t87066 5843\n",
      "28\t87156\t0.2\t1\t83145 4011\n",
      "29\t93947\t0.2\t1\t88848 5099\n",
      "30\t93821\t0.2\t1\t89436 4385\n",
      "31\t101908\t0.2\t1\t96520 5388\n",
      "32\t92154\t0.2\t1\t87903 4251\n",
      "33\t101850\t0.2\t1\t96541 5309\n",
      "34\t132968\t0.2\t1\t125972 6996\n",
      "35\t75102\t0.2\t1\t71554 3548\n",
      "36\t106526\t0.2\t1\t100872 5654\n",
      "37\t112114\t0.2\t1\t106255 5859\n",
      "38\t106187\t0.2\t1\t101516 4671\n",
      "39\t103250\t0.2\t1\t98795 4455\n",
      "40\t116350\t0.2\t1\t110242 6108\n",
      "41\t111543\t0.2\t1\t106565 4978\n",
      "42\t115913\t0.2\t1\t110317 5596\n",
      "43\t105510\t0.2\t1\t100613 4897\n",
      "44\t121504\t0.2\t1\t115737 5767\n",
      "45\t131840\t0.2\t1\t125697 6143\n",
      "46\t94875\t0.2\t1\t90604 4271\n",
      "47\t121018\t0.2\t1\t115877 5141\n",
      "48\t118842\t0.2\t1\t113797 5045\n",
      "49\t124446\t0.2\t1\t118832 5614\n",
      "50\t118268\t0.2\t1\t112777 5491\n",
      "51\t147285\t0.2\t1\t140570 6715\n",
      "52\t109448\t0.2\t1\t105060 4388\n",
      "53\t130385\t0.2\t1\t125319 5066\n",
      "54\t99487\t0.2\t1\t95470 4017\n",
      "55\t129239\t0.2\t1\t124277 4962\n",
      "56\t133599\t0.2\t1\t127967 5632\n",
      "57\t126390\t0.2\t1\t121587 4803\n",
      "58\t121811\t0.2\t1\t117475 4336\n",
      "59\t124529\t0.2\t1\t119540 4989\n",
      "60\t130401\t0.2\t1\t125258 5143\n",
      "61\t134015\t0.2\t1\t128535 5480\n",
      "62\t145629\t0.2\t1\t139763 5866\n",
      "63\t197423\t0.2\t1\t190204 7219\n",
      "64\t168240\t0.2\t1\t162928 5312\n",
      "65\t106543\t0.2\t1\t102625 3918\n",
      "66\t114942\t0.2\t1\t110706 4236\n",
      "67\t114164\t0.2\t1\t109736 4428\n",
      "68\t116350\t0.2\t1\t112096 4254\n",
      "69\t114565\t0.2\t1\t110115 4450\n",
      "70\t124406\t0.2\t1\t119762 4644\n",
      "71\t128031\t0.2\t1\t123514 4517\n",
      "72\t120687\t0.2\t1\t116066 4621\n",
      "73\t126907\t0.2\t1\t122307 4600\n",
      "74\t117912\t0.2\t1\t113666 4246\n",
      "75\t113448\t0.2\t1\t109343 4105\n",
      "76\t111278\t0.2\t1\t107269 4009\n",
      "77\t112780\t0.2\t1\t108679 4101\n",
      "78\t106864\t0.2\t1\t102856 4008\n",
      "79\t103353\t0.2\t1\t99664 3689\n",
      "80\t103266\t0.2\t1\t99595 3671\n",
      "81\t101983\t0.2\t1\t98265 3718\n",
      "82\t97716\t0.2\t1\t94285 3431\n",
      "83\t95425\t0.2\t1\t92073 3352\n",
      "84\t92931\t0.2\t1\t89513 3418\n",
      "85\t92256\t0.2\t1\t88832 3424\n",
      "86\t91168\t0.2\t1\t87863 3305\n",
      "87\t91423\t0.2\t1\t88169 3254\n",
      "88\t89550\t0.2\t1\t86420 3130\n",
      "89\t80366\t0.2\t1\t77566 2800\n",
      "90\t77351\t0.2\t1\t74500 2851\n",
      "91\t72940\t0.2\t1\t70306 2634\n",
      "92\t68488\t0.2\t1\t66133 2355\n",
      "93\t65935\t0.2\t1\t63542 2393\n",
      "94\t63561\t0.2\t1\t61328 2233\n",
      "95\t59923\t0.2\t1\t57811 2112\n",
      "96\t58074\t0.2\t1\t56092 1982\n",
      "97\t54057\t0.2\t1\t52194 1863\n",
      "98\t50980\t0.2\t1\t49204 1776\n",
      "99\t50084\t0.2\t1\t48441 1643\n",
      "100\t43642\t0.2\t1\t42153 1489\n",
      "101\t40618\t0.2\t1\t39288 1330\n",
      "102\t38299\t0.2\t1\t37001 1298\n",
      "103\t34998\t0.2\t1\t33809 1189\n",
      "104\t34454\t0.2\t1\t33375 1079\n",
      "105\t29814\t0.2\t1\t28817 997\n",
      "106\t27473\t0.2\t1\t26595 878\n",
      "107\t24384\t0.2\t1\t23578 806\n",
      "108\t22338\t0.2\t1\t21561 777\n",
      "109\t19549\t0.2\t1\t18854 695\n",
      "110\t18085\t0.2\t1\t17505 580\n",
      "111\t16893\t0.2\t1\t16326 567\n",
      "112\t14541\t0.2\t1\t14060 481\n",
      "113\t12649\t0.2\t1\t12207 442\n",
      "114\t11108\t0.2\t1\t10661 447\n",
      "115\t9325\t0.2\t1\t9006 319\n",
      "116\t7950\t0.2\t1\t7620 330\n",
      "117\t6499\t0.2\t1\t6272 227\n",
      "118\t5695\t0.2\t1\t5463 232\n",
      "119\t4844\t0.2\t1\t4637 207\n",
      "120\t4029\t0.2\t1\t3868 161\n",
      "121\t3484\t0.2\t1\t3355 129\n",
      "122\t2831\t0.2\t1\t2700 131\n",
      "123\t2522\t0.2\t1\t2436 86\n",
      "124\t2113\t0.2\t1\t2005 108\n",
      "125\t1739\t0.2\t1\t1651 88\n",
      "126\t1465\t0.2\t1\t1407 58\n",
      "127\t1198\t0.2\t1\t1145 53\n",
      "128\t931\t0.2\t1\t888 43\n",
      "129\t754\t0.2\t1\t713 41\n",
      "130\t539\t0.2\t1\t514 25\n",
      "131\t443\t0.2\t1\t421 22\n",
      "132\t375\t0.2\t1\t351 24\n",
      "133\t401\t0.2\t1\t386 15\n",
      "134\t477\t0.2\t1\t456 21\n",
      "135\t242\t0.2\t1\t234 8\n",
      "136\t136\t0.2\t1\t132 4\n",
      "137\t103\t0.2\t1\t99 4\n",
      "138\t81\t0.2\t1\t73 8\n",
      "139\t54\t0.2\t1\t51 3\n",
      "140\t37\t0.2\t1\t32 5\n",
      "141\t20\t0.2\t1\t19 1\n",
      "142\t21\t0.2\t1\t20 1\n",
      "143\t18\t0.2\t1\t16 2\n",
      "144\t17\t0.2\t1\t16 1\n",
      "145\t18\t0.2\t1\t16 2\n",
      "146\t15\t0.2\t1\t14 1\n",
      "147\t32\t0.2\t1\t29 3\n",
      "148\t19\t0.2\t1\t17 2\n",
      "149\t11\t0.2\t1\t10 1\n",
      "150\t176\t0.2\t1\t165 11\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14166205 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t18363 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 544.25 s (39 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,120,008\n",
      "Reads with adapters:                11,800,625 (83.6%)\n",
      "Reads written (passing filters):    14,120,008 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,118,001,200 bp\n",
      "Quality-trimmed:               4,373,755 bp (0.2%)\n",
      "Total written (filtered):  1,553,898,462 bp (73.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11800625 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 15.1%\n",
      "  G: 21.0%\n",
      "  T: 34.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1368833\t3530002.0\t0\t1368833\n",
      "2\t213297\t882500.5\t0\t213297\n",
      "3\t129961\t220625.1\t0\t129961\n",
      "4\t79035\t55156.3\t0\t79035\n",
      "5\t69573\t13789.1\t0\t69573\n",
      "6\t67663\t3447.3\t0\t67663\n",
      "7\t54249\t861.8\t0\t54249\n",
      "8\t69306\t215.5\t0\t69306\n",
      "9\t69302\t53.9\t0\t68512 790\n",
      "10\t64667\t13.5\t1\t60704 3963\n",
      "11\t73696\t3.4\t1\t67523 6173\n",
      "12\t69675\t0.8\t1\t64547 5128\n",
      "13\t71952\t0.2\t1\t66511 5441\n",
      "14\t73593\t0.2\t1\t67472 6121\n",
      "15\t76996\t0.2\t1\t70887 6109\n",
      "16\t75526\t0.2\t1\t69104 6422\n",
      "17\t93355\t0.2\t1\t84545 8810\n",
      "18\t79593\t0.2\t1\t73933 5660\n",
      "19\t68124\t0.2\t1\t63039 5085\n",
      "20\t82749\t0.2\t1\t76505 6244\n",
      "21\t92275\t0.2\t1\t85139 7136\n",
      "22\t77618\t0.2\t1\t72438 5180\n",
      "23\t92025\t0.2\t1\t84966 7059\n",
      "24\t85340\t0.2\t1\t77863 7477\n",
      "25\t92374\t0.2\t1\t85186 7188\n",
      "26\t84405\t0.2\t1\t78632 5773\n",
      "27\t95395\t0.2\t1\t87567 7828\n",
      "28\t93111\t0.2\t1\t85941 7170\n",
      "29\t103004\t0.2\t1\t93769 9235\n",
      "30\t106080\t0.2\t1\t99144 6936\n",
      "31\t80431\t0.2\t1\t74401 6030\n",
      "32\t102254\t0.2\t1\t94262 7992\n",
      "33\t95806\t0.2\t1\t88463 7343\n",
      "34\t106075\t0.2\t1\t96928 9147\n",
      "35\t108269\t0.2\t1\t99389 8880\n",
      "36\t112253\t0.2\t1\t102783 9470\n",
      "37\t99044\t0.2\t1\t92235 6809\n",
      "38\t121329\t0.2\t1\t110516 10813\n",
      "39\t100947\t0.2\t1\t93251 7696\n",
      "40\t114342\t0.2\t1\t105610 8732\n",
      "41\t128592\t0.2\t1\t117446 11146\n",
      "42\t100668\t0.2\t1\t92131 8537\n",
      "43\t169999\t0.2\t1\t159204 10795\n",
      "44\t56249\t0.2\t1\t50770 5479\n",
      "45\t119775\t0.2\t1\t111031 8744\n",
      "46\t112063\t0.2\t1\t103816 8247\n",
      "47\t119122\t0.2\t1\t110508 8614\n",
      "48\t118788\t0.2\t1\t109597 9191\n",
      "49\t131198\t0.2\t1\t120814 10384\n",
      "50\t109980\t0.2\t1\t102619 7361\n",
      "51\t122763\t0.2\t1\t113852 8911\n",
      "52\t127644\t0.2\t1\t117521 10123\n",
      "53\t136952\t0.2\t1\t127680 9272\n",
      "54\t107784\t0.2\t1\t99974 7810\n",
      "55\t130694\t0.2\t1\t121255 9439\n",
      "56\t158527\t0.2\t1\t147787 10740\n",
      "57\t103844\t0.2\t1\t95331 8513\n",
      "58\t132382\t0.2\t1\t124247 8135\n",
      "59\t116544\t0.2\t1\t107902 8642\n",
      "60\t129622\t0.2\t1\t120349 9273\n",
      "61\t128569\t0.2\t1\t119774 8795\n",
      "62\t140522\t0.2\t1\t130790 9732\n",
      "63\t114632\t0.2\t1\t105994 8638\n",
      "64\t124792\t0.2\t1\t116366 8426\n",
      "65\t140416\t0.2\t1\t130644 9772\n",
      "66\t145932\t0.2\t1\t135671 10261\n",
      "67\t159891\t0.2\t1\t146767 13124\n",
      "68\t288595\t0.2\t1\t272910 15685\n",
      "69\t63057\t0.2\t1\t57575 5482\n",
      "70\t26024\t0.2\t1\t21992 4032\n",
      "71\t79266\t0.2\t1\t73983 5283\n",
      "72\t112824\t0.2\t1\t105728 7096\n",
      "73\t119835\t0.2\t1\t111892 7943\n",
      "74\t119611\t0.2\t1\t111533 8078\n",
      "75\t117923\t0.2\t1\t109788 8135\n",
      "76\t117731\t0.2\t1\t109296 8435\n",
      "77\t118271\t0.2\t1\t109919 8352\n",
      "78\t111566\t0.2\t1\t103610 7956\n",
      "79\t111519\t0.2\t1\t103611 7908\n",
      "80\t109197\t0.2\t1\t101453 7744\n",
      "81\t107570\t0.2\t1\t99856 7714\n",
      "82\t104569\t0.2\t1\t97118 7451\n",
      "83\t102383\t0.2\t1\t95228 7155\n",
      "84\t99102\t0.2\t1\t92021 7081\n",
      "85\t96000\t0.2\t1\t89098 6902\n",
      "86\t93524\t0.2\t1\t86838 6686\n",
      "87\t89373\t0.2\t1\t83285 6088\n",
      "88\t88917\t0.2\t1\t82644 6273\n",
      "89\t84001\t0.2\t1\t77961 6040\n",
      "90\t83693\t0.2\t1\t77869 5824\n",
      "91\t80101\t0.2\t1\t74574 5527\n",
      "92\t76032\t0.2\t1\t70730 5302\n",
      "93\t71132\t0.2\t1\t66158 4974\n",
      "94\t68649\t0.2\t1\t63740 4909\n",
      "95\t64070\t0.2\t1\t59484 4586\n",
      "96\t60750\t0.2\t1\t56444 4306\n",
      "97\t57341\t0.2\t1\t53268 4073\n",
      "98\t53260\t0.2\t1\t49347 3913\n",
      "99\t52636\t0.2\t1\t48599 4037\n",
      "100\t46791\t0.2\t1\t43393 3398\n",
      "101\t44342\t0.2\t1\t41093 3249\n",
      "102\t40590\t0.2\t1\t37587 3003\n",
      "103\t37417\t0.2\t1\t34694 2723\n",
      "104\t34470\t0.2\t1\t31943 2527\n",
      "105\t30930\t0.2\t1\t28658 2272\n",
      "106\t28279\t0.2\t1\t26119 2160\n",
      "107\t25736\t0.2\t1\t23845 1891\n",
      "108\t23849\t0.2\t1\t22042 1807\n",
      "109\t21192\t0.2\t1\t19537 1655\n",
      "110\t19116\t0.2\t1\t17675 1441\n",
      "111\t17231\t0.2\t1\t15808 1423\n",
      "112\t14798\t0.2\t1\t13558 1240\n",
      "113\t12618\t0.2\t1\t11582 1036\n",
      "114\t11317\t0.2\t1\t10327 990\n",
      "115\t9619\t0.2\t1\t8804 815\n",
      "116\t8405\t0.2\t1\t7688 717\n",
      "117\t7048\t0.2\t1\t6428 620\n",
      "118\t6316\t0.2\t1\t5766 550\n",
      "119\t5076\t0.2\t1\t4600 476\n",
      "120\t4250\t0.2\t1\t3802 448\n",
      "121\t3670\t0.2\t1\t3303 367\n",
      "122\t3096\t0.2\t1\t2757 339\n",
      "123\t2589\t0.2\t1\t2312 277\n",
      "124\t2201\t0.2\t1\t1979 222\n",
      "125\t1708\t0.2\t1\t1510 198\n",
      "126\t1570\t0.2\t1\t1383 187\n",
      "127\t1266\t0.2\t1\t1112 154\n",
      "128\t926\t0.2\t1\t821 105\n",
      "129\t844\t0.2\t1\t748 96\n",
      "130\t568\t0.2\t1\t488 80\n",
      "131\t507\t0.2\t1\t451 56\n",
      "132\t422\t0.2\t1\t368 54\n",
      "133\t416\t0.2\t1\t358 58\n",
      "134\t449\t0.2\t1\t401 48\n",
      "135\t219\t0.2\t1\t188 31\n",
      "136\t167\t0.2\t1\t147 20\n",
      "137\t91\t0.2\t1\t79 12\n",
      "138\t80\t0.2\t1\t64 16\n",
      "139\t47\t0.2\t1\t42 5\n",
      "140\t27\t0.2\t1\t21 6\n",
      "141\t27\t0.2\t1\t23 4\n",
      "142\t18\t0.2\t1\t10 8\n",
      "143\t22\t0.2\t1\t11 11\n",
      "144\t14\t0.2\t1\t10 4\n",
      "145\t18\t0.2\t1\t16 2\n",
      "146\t18\t0.2\t1\t14 4\n",
      "147\t32\t0.2\t1\t28 4\n",
      "148\t19\t0.2\t1\t17 2\n",
      "149\t14\t0.2\t1\t10 4\n",
      "150\t202\t0.2\t1\t177 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14120008 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2849 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_3_S7_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 567.54 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,120,008\n",
      "Reads with adapters:                11,722,164 (83.0%)\n",
      "Reads written (passing filters):    14,120,008 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,118,001,200 bp\n",
      "Quality-trimmed:              10,398,441 bp (0.5%)\n",
      "Total written (filtered):  1,557,917,839 bp (73.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11722164 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.6%\n",
      "  C: 13.3%\n",
      "  G: 24.5%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1423484\t3530002.0\t0\t1423484\n",
      "2\t217404\t882500.5\t0\t217404\n",
      "3\t135747\t220625.1\t0\t135747\n",
      "4\t77416\t55156.3\t0\t77416\n",
      "5\t73163\t13789.1\t0\t73163\n",
      "6\t65957\t3447.3\t0\t65957\n",
      "7\t52727\t861.8\t0\t52727\n",
      "8\t75617\t215.5\t0\t75617\n",
      "9\t60824\t53.9\t0\t60611 213\n",
      "10\t64613\t13.5\t1\t60561 4052\n",
      "11\t71071\t3.4\t1\t65663 5408\n",
      "12\t69904\t0.8\t1\t64695 5209\n",
      "13\t80991\t0.2\t1\t75018 5973\n",
      "14\t75627\t0.2\t1\t69287 6340\n",
      "15\t61630\t0.2\t1\t57275 4355\n",
      "16\t89541\t0.2\t1\t82804 6737\n",
      "17\t75362\t0.2\t1\t69553 5809\n",
      "18\t67936\t0.2\t1\t63097 4839\n",
      "19\t88766\t0.2\t1\t82324 6442\n",
      "20\t83180\t0.2\t1\t77328 5852\n",
      "21\t74299\t0.2\t1\t69124 5175\n",
      "22\t83499\t0.2\t1\t77755 5744\n",
      "23\t88403\t0.2\t1\t82296 6107\n",
      "24\t88389\t0.2\t1\t82109 6280\n",
      "25\t100119\t0.2\t1\t93881 6238\n",
      "26\t84031\t0.2\t1\t77871 6160\n",
      "27\t93205\t0.2\t1\t86091 7114\n",
      "28\t83882\t0.2\t1\t78923 4959\n",
      "29\t95544\t0.2\t1\t89031 6513\n",
      "30\t91423\t0.2\t1\t85830 5593\n",
      "31\t102885\t0.2\t1\t96159 6726\n",
      "32\t88346\t0.2\t1\t83048 5298\n",
      "33\t103020\t0.2\t1\t96084 6936\n",
      "34\t134691\t0.2\t1\t125875 8816\n",
      "35\t71511\t0.2\t1\t67043 4468\n",
      "36\t106372\t0.2\t1\t99204 7168\n",
      "37\t111667\t0.2\t1\t104414 7253\n",
      "38\t105292\t0.2\t1\t99304 5988\n",
      "39\t99654\t0.2\t1\t94273 5381\n",
      "40\t115518\t0.2\t1\t108179 7339\n",
      "41\t111214\t0.2\t1\t104737 6477\n",
      "42\t115443\t0.2\t1\t108784 6659\n",
      "43\t104707\t0.2\t1\t98628 6079\n",
      "44\t122428\t0.2\t1\t115040 7388\n",
      "45\t129896\t0.2\t1\t122003 7893\n",
      "46\t92988\t0.2\t1\t87594 5394\n",
      "47\t119950\t0.2\t1\t113634 6316\n",
      "48\t117307\t0.2\t1\t110893 6414\n",
      "49\t123614\t0.2\t1\t116211 7403\n",
      "50\t118595\t0.2\t1\t111825 6770\n",
      "51\t146157\t0.2\t1\t137800 8357\n",
      "52\t108528\t0.2\t1\t102556 5972\n",
      "53\t133051\t0.2\t1\t126144 6907\n",
      "54\t97258\t0.2\t1\t92286 4972\n",
      "55\t128012\t0.2\t1\t121714 6298\n",
      "56\t133239\t0.2\t1\t125868 7371\n",
      "57\t126482\t0.2\t1\t120493 5989\n",
      "58\t123495\t0.2\t1\t117409 6086\n",
      "59\t123171\t0.2\t1\t116902 6269\n",
      "60\t129991\t0.2\t1\t123447 6544\n",
      "61\t132791\t0.2\t1\t126344 6447\n",
      "62\t145203\t0.2\t1\t137595 7608\n",
      "63\t193193\t0.2\t1\t184570 8623\n",
      "64\t164505\t0.2\t1\t158250 6255\n",
      "65\t106154\t0.2\t1\t101191 4963\n",
      "66\t116219\t0.2\t1\t110707 5512\n",
      "67\t115499\t0.2\t1\t109857 5642\n",
      "68\t116592\t0.2\t1\t111069 5523\n",
      "69\t115665\t0.2\t1\t109783 5882\n",
      "70\t124608\t0.2\t1\t118610 5998\n",
      "71\t128211\t0.2\t1\t122084 6127\n",
      "72\t122112\t0.2\t1\t116125 5987\n",
      "73\t126921\t0.2\t1\t120760 6161\n",
      "74\t118790\t0.2\t1\t112963 5827\n",
      "75\t113173\t0.2\t1\t107712 5461\n",
      "76\t111463\t0.2\t1\t105998 5465\n",
      "77\t111662\t0.2\t1\t106140 5522\n",
      "78\t107207\t0.2\t1\t101895 5312\n",
      "79\t104292\t0.2\t1\t99234 5058\n",
      "80\t103845\t0.2\t1\t98916 4929\n",
      "81\t101938\t0.2\t1\t97104 4834\n",
      "82\t97773\t0.2\t1\t93124 4649\n",
      "83\t95971\t0.2\t1\t91487 4484\n",
      "84\t92983\t0.2\t1\t88672 4311\n",
      "85\t91731\t0.2\t1\t87597 4134\n",
      "86\t91538\t0.2\t1\t87240 4298\n",
      "87\t90919\t0.2\t1\t86818 4101\n",
      "88\t89775\t0.2\t1\t85803 3972\n",
      "89\t80478\t0.2\t1\t76988 3490\n",
      "90\t77967\t0.2\t1\t74477 3490\n",
      "91\t73781\t0.2\t1\t70569 3212\n",
      "92\t69447\t0.2\t1\t66340 3107\n",
      "93\t66394\t0.2\t1\t63362 3032\n",
      "94\t63823\t0.2\t1\t61051 2772\n",
      "95\t60418\t0.2\t1\t57617 2801\n",
      "96\t57934\t0.2\t1\t55484 2450\n",
      "97\t54242\t0.2\t1\t51870 2372\n",
      "98\t51153\t0.2\t1\t49011 2142\n",
      "99\t50193\t0.2\t1\t47988 2205\n",
      "100\t43401\t0.2\t1\t41462 1939\n",
      "101\t41188\t0.2\t1\t39459 1729\n",
      "102\t38238\t0.2\t1\t36636 1602\n",
      "103\t35697\t0.2\t1\t34094 1603\n",
      "104\t34260\t0.2\t1\t32771 1489\n",
      "105\t30330\t0.2\t1\t29031 1299\n",
      "106\t27539\t0.2\t1\t26325 1214\n",
      "107\t24680\t0.2\t1\t23635 1045\n",
      "108\t22009\t0.2\t1\t21062 947\n",
      "109\t19974\t0.2\t1\t19117 857\n",
      "110\t18015\t0.2\t1\t17209 806\n",
      "111\t16915\t0.2\t1\t16205 710\n",
      "112\t14332\t0.2\t1\t13689 643\n",
      "113\t12446\t0.2\t1\t11895 551\n",
      "114\t10968\t0.2\t1\t10478 490\n",
      "115\t9016\t0.2\t1\t8590 426\n",
      "116\t7818\t0.2\t1\t7478 340\n",
      "117\t6323\t0.2\t1\t6027 296\n",
      "118\t5766\t0.2\t1\t5518 248\n",
      "119\t4704\t0.2\t1\t4500 204\n",
      "120\t3901\t0.2\t1\t3708 193\n",
      "121\t3388\t0.2\t1\t3195 193\n",
      "122\t2862\t0.2\t1\t2731 131\n",
      "123\t2458\t0.2\t1\t2315 143\n",
      "124\t2060\t0.2\t1\t1986 74\n",
      "125\t1642\t0.2\t1\t1547 95\n",
      "126\t1477\t0.2\t1\t1395 82\n",
      "127\t1193\t0.2\t1\t1127 66\n",
      "128\t861\t0.2\t1\t803 58\n",
      "129\t803\t0.2\t1\t750 53\n",
      "130\t513\t0.2\t1\t479 34\n",
      "131\t475\t0.2\t1\t451 24\n",
      "132\t384\t0.2\t1\t349 35\n",
      "133\t402\t0.2\t1\t380 22\n",
      "134\t430\t0.2\t1\t416 14\n",
      "135\t203\t0.2\t1\t185 18\n",
      "136\t160\t0.2\t1\t156 4\n",
      "137\t84\t0.2\t1\t74 10\n",
      "138\t68\t0.2\t1\t64 4\n",
      "139\t47\t0.2\t1\t44 3\n",
      "140\t21\t0.2\t1\t20 1\n",
      "141\t24\t0.2\t1\t21 3\n",
      "142\t16\t0.2\t1\t13 3\n",
      "143\t16\t0.2\t1\t13 3\n",
      "144\t10\t0.2\t1\t10\n",
      "145\t20\t0.2\t1\t19 1\n",
      "146\t18\t0.2\t1\t18\n",
      "147\t25\t0.2\t1\t23 2\n",
      "148\t19\t0.2\t1\t15 4\n",
      "149\t13\t0.2\t1\t12 1\n",
      "150\t153\t0.2\t1\t145 8\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14120008 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t16796 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 614.41 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,686,084\n",
      "Reads with adapters:                11,112,819 (75.7%)\n",
      "Reads written (passing filters):    14,686,084 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,202,912,600 bp\n",
      "Quality-trimmed:               9,079,514 bp (0.4%)\n",
      "Total written (filtered):  1,783,990,340 bp (81.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11112819 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 11.7%\n",
      "  G: 21.0%\n",
      "  T: 36.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2196567\t3671521.0\t0\t2196567\n",
      "2\t292186\t917880.2\t0\t292186\n",
      "3\t157941\t229470.1\t0\t157941\n",
      "4\t100753\t57367.5\t0\t100753\n",
      "5\t83202\t14341.9\t0\t83202\n",
      "6\t81280\t3585.5\t0\t81280\n",
      "7\t68619\t896.4\t0\t68619\n",
      "8\t84673\t224.1\t0\t84673\n",
      "9\t83412\t56.0\t0\t82605 807\n",
      "10\t77900\t14.0\t1\t74163 3737\n",
      "11\t86076\t3.5\t1\t80161 5915\n",
      "12\t82933\t0.9\t1\t78087 4846\n",
      "13\t83856\t0.2\t1\t78977 4879\n",
      "14\t84709\t0.2\t1\t79208 5501\n",
      "15\t88097\t0.2\t1\t82938 5159\n",
      "16\t88174\t0.2\t1\t82008 6166\n",
      "17\t105389\t0.2\t1\t97630 7759\n",
      "18\t89620\t0.2\t1\t84611 5009\n",
      "19\t77413\t0.2\t1\t72875 4538\n",
      "20\t91427\t0.2\t1\t86037 5390\n",
      "21\t101267\t0.2\t1\t94618 6649\n",
      "22\t86534\t0.2\t1\t82143 4391\n",
      "23\t99603\t0.2\t1\t93638 5965\n",
      "24\t92844\t0.2\t1\t85594 7250\n",
      "25\t98567\t0.2\t1\t92532 6035\n",
      "26\t89487\t0.2\t1\t84719 4768\n",
      "27\t100312\t0.2\t1\t93524 6788\n",
      "28\t97676\t0.2\t1\t91379 6297\n",
      "29\t103869\t0.2\t1\t95980 7889\n",
      "30\t115073\t0.2\t1\t109144 5929\n",
      "31\t81679\t0.2\t1\t76971 4708\n",
      "32\t104832\t0.2\t1\t98556 6276\n",
      "33\t97135\t0.2\t1\t91030 6105\n",
      "34\t104303\t0.2\t1\t98156 6147\n",
      "35\t102079\t0.2\t1\t95269 6810\n",
      "36\t105105\t0.2\t1\t98314 6791\n",
      "37\t106489\t0.2\t1\t100050 6439\n",
      "38\t111587\t0.2\t1\t104304 7283\n",
      "39\t98240\t0.2\t1\t93162 5078\n",
      "40\t110224\t0.2\t1\t103236 6988\n",
      "41\t119138\t0.2\t1\t109010 10128\n",
      "42\t107751\t0.2\t1\t99258 8493\n",
      "43\t167048\t0.2\t1\t157990 9058\n",
      "44\t54792\t0.2\t1\t50043 4749\n",
      "45\t116148\t0.2\t1\t107107 9041\n",
      "46\t253343\t0.2\t1\t239166 14177\n",
      "47\t141220\t0.2\t1\t133245 7975\n",
      "48\t70788\t0.2\t1\t63989 6799\n",
      "49\t218187\t0.2\t1\t207304 10883\n",
      "50\t67793\t0.2\t1\t62869 4924\n",
      "51\t84957\t0.2\t1\t78961 5996\n",
      "52\t231141\t0.2\t1\t220209 10932\n",
      "53\t159552\t0.2\t1\t153244 6308\n",
      "54\t57820\t0.2\t1\t54077 3743\n",
      "55\t96525\t0.2\t1\t92498 4027\n",
      "56\t56201\t0.2\t1\t52298 3903\n",
      "57\t134736\t0.2\t1\t129198 5538\n",
      "58\t33322\t0.2\t1\t31241 2081\n",
      "59\t35361\t0.2\t1\t32301 3060\n",
      "60\t121909\t0.2\t1\t116692 5217\n",
      "61\t80368\t0.2\t1\t77075 3293\n",
      "62\t37240\t0.2\t1\t34469 2771\n",
      "63\t91660\t0.2\t1\t86780 4880\n",
      "64\t81468\t0.2\t1\t77877 3591\n",
      "65\t53211\t0.2\t1\t49893 3318\n",
      "66\t93831\t0.2\t1\t88616 5215\n",
      "67\t107034\t0.2\t1\t100335 6699\n",
      "68\t187851\t0.2\t1\t179197 8654\n",
      "69\t100255\t0.2\t1\t94892 5363\n",
      "70\t35399\t0.2\t1\t32342 3057\n",
      "71\t19897\t0.2\t1\t17997 1900\n",
      "72\t63849\t0.2\t1\t60613 3236\n",
      "73\t81267\t0.2\t1\t77051 4216\n",
      "74\t82339\t0.2\t1\t77944 4395\n",
      "75\t80643\t0.2\t1\t76099 4544\n",
      "76\t79016\t0.2\t1\t74790 4226\n",
      "77\t78401\t0.2\t1\t74108 4293\n",
      "78\t73966\t0.2\t1\t69851 4115\n",
      "79\t71627\t0.2\t1\t67584 4043\n",
      "80\t70426\t0.2\t1\t66563 3863\n",
      "81\t69818\t0.2\t1\t65954 3864\n",
      "82\t67366\t0.2\t1\t63575 3791\n",
      "83\t63649\t0.2\t1\t60193 3456\n",
      "84\t61172\t0.2\t1\t57824 3348\n",
      "85\t59276\t0.2\t1\t56040 3236\n",
      "86\t56575\t0.2\t1\t53454 3121\n",
      "87\t53078\t0.2\t1\t50236 2842\n",
      "88\t51788\t0.2\t1\t48829 2959\n",
      "89\t48314\t0.2\t1\t45638 2676\n",
      "90\t48321\t0.2\t1\t45629 2692\n",
      "91\t45543\t0.2\t1\t43073 2470\n",
      "92\t42075\t0.2\t1\t39779 2296\n",
      "93\t38814\t0.2\t1\t36658 2156\n",
      "94\t37722\t0.2\t1\t35616 2106\n",
      "95\t34423\t0.2\t1\t32423 2000\n",
      "96\t32157\t0.2\t1\t30284 1873\n",
      "97\t29965\t0.2\t1\t28206 1759\n",
      "98\t27347\t0.2\t1\t25797 1550\n",
      "99\t26166\t0.2\t1\t24583 1583\n",
      "100\t23354\t0.2\t1\t21969 1385\n",
      "101\t21467\t0.2\t1\t20157 1310\n",
      "102\t19539\t0.2\t1\t18344 1195\n",
      "103\t17855\t0.2\t1\t16735 1120\n",
      "104\t16360\t0.2\t1\t15329 1031\n",
      "105\t14296\t0.2\t1\t13369 927\n",
      "106\t12633\t0.2\t1\t11870 763\n",
      "107\t11358\t0.2\t1\t10612 746\n",
      "108\t10534\t0.2\t1\t9836 698\n",
      "109\t9078\t0.2\t1\t8461 617\n",
      "110\t8053\t0.2\t1\t7464 589\n",
      "111\t6944\t0.2\t1\t6454 490\n",
      "112\t6153\t0.2\t1\t5691 462\n",
      "113\t5306\t0.2\t1\t4906 400\n",
      "114\t4518\t0.2\t1\t4189 329\n",
      "115\t3910\t0.2\t1\t3603 307\n",
      "116\t3193\t0.2\t1\t2951 242\n",
      "117\t2651\t0.2\t1\t2439 212\n",
      "118\t2314\t0.2\t1\t2103 211\n",
      "119\t1875\t0.2\t1\t1673 202\n",
      "120\t1577\t0.2\t1\t1427 150\n",
      "121\t1320\t0.2\t1\t1198 122\n",
      "122\t1105\t0.2\t1\t995 110\n",
      "123\t920\t0.2\t1\t816 104\n",
      "124\t791\t0.2\t1\t709 82\n",
      "125\t642\t0.2\t1\t583 59\n",
      "126\t507\t0.2\t1\t450 57\n",
      "127\t422\t0.2\t1\t367 55\n",
      "128\t333\t0.2\t1\t289 44\n",
      "129\t238\t0.2\t1\t206 32\n",
      "130\t186\t0.2\t1\t162 24\n",
      "131\t174\t0.2\t1\t151 23\n",
      "132\t134\t0.2\t1\t118 16\n",
      "133\t148\t0.2\t1\t129 19\n",
      "134\t197\t0.2\t1\t182 15\n",
      "135\t113\t0.2\t1\t97 16\n",
      "136\t58\t0.2\t1\t50 8\n",
      "137\t58\t0.2\t1\t48 10\n",
      "138\t42\t0.2\t1\t35 7\n",
      "139\t30\t0.2\t1\t28 2\n",
      "140\t22\t0.2\t1\t20 2\n",
      "141\t15\t0.2\t1\t9 6\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t11\t0.2\t1\t6 5\n",
      "144\t2\t0.2\t1\t1 1\n",
      "145\t7\t0.2\t1\t6 1\n",
      "146\t5\t0.2\t1\t4 1\n",
      "147\t21\t0.2\t1\t20 1\n",
      "148\t23\t0.2\t1\t19 4\n",
      "149\t14\t0.2\t1\t12 2\n",
      "150\t119\t0.2\t1\t95 24\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14686084 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t1248 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 649.11 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,686,084\n",
      "Reads with adapters:                11,052,394 (75.3%)\n",
      "Reads written (passing filters):    14,686,084 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,202,912,600 bp\n",
      "Quality-trimmed:               8,170,892 bp (0.4%)\n",
      "Total written (filtered):  1,785,498,326 bp (81.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11052394 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 11.2%\n",
      "  G: 23.2%\n",
      "  T: 36.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2172306\t3671521.0\t0\t2172306\n",
      "2\t315945\t917880.2\t0\t315945\n",
      "3\t169586\t229470.1\t0\t169586\n",
      "4\t102198\t57367.5\t0\t102198\n",
      "5\t92121\t14341.9\t0\t92121\n",
      "6\t78915\t3585.5\t0\t78915\n",
      "7\t65916\t896.4\t0\t65916\n",
      "8\t91966\t224.1\t0\t91966\n",
      "9\t73557\t56.0\t0\t73312 245\n",
      "10\t79999\t14.0\t1\t76173 3826\n",
      "11\t83777\t3.5\t1\t78744 5033\n",
      "12\t84445\t0.9\t1\t79272 5173\n",
      "13\t92865\t0.2\t1\t87504 5361\n",
      "14\t88444\t0.2\t1\t82453 5991\n",
      "15\t72972\t0.2\t1\t68400 4572\n",
      "16\t103628\t0.2\t1\t97383 6245\n",
      "17\t83622\t0.2\t1\t78153 5469\n",
      "18\t78826\t0.2\t1\t74157 4669\n",
      "19\t100647\t0.2\t1\t94372 6275\n",
      "20\t93537\t0.2\t1\t88067 5470\n",
      "21\t82005\t0.2\t1\t77188 4817\n",
      "22\t92897\t0.2\t1\t87543 5354\n",
      "23\t95873\t0.2\t1\t90242 5631\n",
      "24\t96154\t0.2\t1\t90197 5957\n",
      "25\t107344\t0.2\t1\t101210 6134\n",
      "26\t88692\t0.2\t1\t83193 5499\n",
      "27\t96620\t0.2\t1\t90107 6513\n",
      "28\t92786\t0.2\t1\t88163 4623\n",
      "29\t97544\t0.2\t1\t91712 5832\n",
      "30\t97471\t0.2\t1\t92326 5145\n",
      "31\t103994\t0.2\t1\t98068 5926\n",
      "32\t93992\t0.2\t1\t89009 4983\n",
      "33\t103947\t0.2\t1\t97692 6255\n",
      "34\t130356\t0.2\t1\t123003 7353\n",
      "35\t75888\t0.2\t1\t71960 3928\n",
      "36\t101074\t0.2\t1\t95184 5890\n",
      "37\t109064\t0.2\t1\t102746 6318\n",
      "38\t100719\t0.2\t1\t95809 4910\n",
      "39\t99097\t0.2\t1\t94226 4871\n",
      "40\t108626\t0.2\t1\t102518 6108\n",
      "41\t105593\t0.2\t1\t100195 5398\n",
      "42\t108346\t0.2\t1\t102605 5741\n",
      "43\t99135\t0.2\t1\t93859 5276\n",
      "44\t112964\t0.2\t1\t106912 6052\n",
      "45\t120273\t0.2\t1\t114139 6134\n",
      "46\t86822\t0.2\t1\t82356 4466\n",
      "47\t107964\t0.2\t1\t102668 5296\n",
      "48\t105861\t0.2\t1\t100664 5197\n",
      "49\t110040\t0.2\t1\t104589 5451\n",
      "50\t103888\t0.2\t1\t98336 5552\n",
      "51\t130541\t0.2\t1\t123943 6598\n",
      "52\t92963\t0.2\t1\t88623 4340\n",
      "53\t111922\t0.2\t1\t106898 5024\n",
      "54\t84092\t0.2\t1\t80032 4060\n",
      "55\t108967\t0.2\t1\t104198 4769\n",
      "56\t110137\t0.2\t1\t104842 5295\n",
      "57\t104380\t0.2\t1\t99728 4652\n",
      "58\t100352\t0.2\t1\t95962 4390\n",
      "59\t101272\t0.2\t1\t96515 4757\n",
      "60\t103619\t0.2\t1\t98712 4907\n",
      "61\t107020\t0.2\t1\t101830 5190\n",
      "62\t116477\t0.2\t1\t111010 5467\n",
      "63\t153560\t0.2\t1\t147083 6477\n",
      "64\t118136\t0.2\t1\t113593 4543\n",
      "65\t80776\t0.2\t1\t77236 3540\n",
      "66\t87168\t0.2\t1\t83301 3867\n",
      "67\t85836\t0.2\t1\t81942 3894\n",
      "68\t86253\t0.2\t1\t82443 3810\n",
      "69\t84506\t0.2\t1\t80622 3884\n",
      "70\t92306\t0.2\t1\t88038 4268\n",
      "71\t90825\t0.2\t1\t86927 3898\n",
      "72\t86185\t0.2\t1\t82424 3761\n",
      "73\t89325\t0.2\t1\t85551 3774\n",
      "74\t82222\t0.2\t1\t78765 3457\n",
      "75\t77552\t0.2\t1\t74138 3414\n",
      "76\t75204\t0.2\t1\t71865 3339\n",
      "77\t74674\t0.2\t1\t71463 3211\n",
      "78\t71366\t0.2\t1\t68191 3175\n",
      "79\t67543\t0.2\t1\t64632 2911\n",
      "80\t67004\t0.2\t1\t64137 2867\n",
      "81\t66579\t0.2\t1\t63792 2787\n",
      "82\t63260\t0.2\t1\t60623 2637\n",
      "83\t59885\t0.2\t1\t57259 2626\n",
      "84\t57771\t0.2\t1\t55217 2554\n",
      "85\t56672\t0.2\t1\t54103 2569\n",
      "86\t55528\t0.2\t1\t53139 2389\n",
      "87\t54690\t0.2\t1\t52283 2407\n",
      "88\t52598\t0.2\t1\t50360 2238\n",
      "89\t46378\t0.2\t1\t44318 2060\n",
      "90\t44805\t0.2\t1\t42852 1953\n",
      "91\t41839\t0.2\t1\t39962 1877\n",
      "92\t38539\t0.2\t1\t36849 1690\n",
      "93\t36522\t0.2\t1\t35046 1476\n",
      "94\t35648\t0.2\t1\t34150 1498\n",
      "95\t33020\t0.2\t1\t31626 1394\n",
      "96\t31039\t0.2\t1\t29760 1279\n",
      "97\t28162\t0.2\t1\t26958 1204\n",
      "98\t25327\t0.2\t1\t24248 1079\n",
      "99\t23909\t0.2\t1\t22911 998\n",
      "100\t21397\t0.2\t1\t20519 878\n",
      "101\t19468\t0.2\t1\t18700 768\n",
      "102\t18321\t0.2\t1\t17542 779\n",
      "103\t16795\t0.2\t1\t16092 703\n",
      "104\t15607\t0.2\t1\t14948 659\n",
      "105\t13615\t0.2\t1\t13013 602\n",
      "106\t12134\t0.2\t1\t11629 505\n",
      "107\t10718\t0.2\t1\t10267 451\n",
      "108\t9677\t0.2\t1\t9284 393\n",
      "109\t8561\t0.2\t1\t8182 379\n",
      "110\t7592\t0.2\t1\t7242 350\n",
      "111\t6810\t0.2\t1\t6531 279\n",
      "112\t6017\t0.2\t1\t5741 276\n",
      "113\t5139\t0.2\t1\t4918 221\n",
      "114\t4338\t0.2\t1\t4134 204\n",
      "115\t3645\t0.2\t1\t3478 167\n",
      "116\t2960\t0.2\t1\t2807 153\n",
      "117\t2433\t0.2\t1\t2312 121\n",
      "118\t2180\t0.2\t1\t2058 122\n",
      "119\t1702\t0.2\t1\t1624 78\n",
      "120\t1461\t0.2\t1\t1377 84\n",
      "121\t1230\t0.2\t1\t1157 73\n",
      "122\t1031\t0.2\t1\t968 63\n",
      "123\t857\t0.2\t1\t808 49\n",
      "124\t756\t0.2\t1\t706 50\n",
      "125\t617\t0.2\t1\t580 37\n",
      "126\t477\t0.2\t1\t448 29\n",
      "127\t390\t0.2\t1\t359 31\n",
      "128\t303\t0.2\t1\t291 12\n",
      "129\t226\t0.2\t1\t207 19\n",
      "130\t179\t0.2\t1\t161 18\n",
      "131\t150\t0.2\t1\t142 8\n",
      "132\t131\t0.2\t1\t123 8\n",
      "133\t129\t0.2\t1\t118 11\n",
      "134\t199\t0.2\t1\t187 12\n",
      "135\t112\t0.2\t1\t107 5\n",
      "136\t50\t0.2\t1\t48 2\n",
      "137\t54\t0.2\t1\t49 5\n",
      "138\t38\t0.2\t1\t37 1\n",
      "139\t28\t0.2\t1\t24 4\n",
      "140\t20\t0.2\t1\t18 2\n",
      "141\t13\t0.2\t1\t12 1\n",
      "142\t11\t0.2\t1\t10 1\n",
      "143\t8\t0.2\t1\t7 1\n",
      "144\t5\t0.2\t1\t3 2\n",
      "145\t4\t0.2\t1\t3 1\n",
      "146\t7\t0.2\t1\t3 4\n",
      "147\t20\t0.2\t1\t18 2\n",
      "148\t18\t0.2\t1\t17 1\n",
      "149\t13\t0.2\t1\t11 2\n",
      "150\t93\t0.2\t1\t85 8\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14686084 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t11190 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 621.57 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,607,942\n",
      "Reads with adapters:                11,056,021 (75.7%)\n",
      "Reads written (passing filters):    14,607,942 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,191,191,300 bp\n",
      "Quality-trimmed:               9,138,103 bp (0.4%)\n",
      "Total written (filtered):  1,772,690,067 bp (80.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11056021 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 11.8%\n",
      "  G: 21.0%\n",
      "  T: 36.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2179670\t3651985.5\t0\t2179670\n",
      "2\t288357\t912996.4\t0\t288357\n",
      "3\t156435\t228249.1\t0\t156435\n",
      "4\t98004\t57062.3\t0\t98004\n",
      "5\t83153\t14265.6\t0\t83153\n",
      "6\t80849\t3566.4\t0\t80849\n",
      "7\t66215\t891.6\t0\t66215\n",
      "8\t83652\t222.9\t0\t83652\n",
      "9\t81421\t55.7\t0\t80631 790\n",
      "10\t76420\t13.9\t1\t72069 4351\n",
      "11\t85164\t3.5\t1\t78240 6924\n",
      "12\t81348\t0.9\t1\t75711 5637\n",
      "13\t83598\t0.2\t1\t77773 5825\n",
      "14\t83352\t0.2\t1\t76916 6436\n",
      "15\t87371\t0.2\t1\t80774 6597\n",
      "16\t86829\t0.2\t1\t79522 7307\n",
      "17\t105942\t0.2\t1\t96774 9168\n",
      "18\t88487\t0.2\t1\t82567 5920\n",
      "19\t76290\t0.2\t1\t70996 5294\n",
      "20\t90707\t0.2\t1\t84433 6274\n",
      "21\t101596\t0.2\t1\t93808 7788\n",
      "22\t84911\t0.2\t1\t79523 5388\n",
      "23\t99744\t0.2\t1\t92683 7061\n",
      "24\t90626\t0.2\t1\t82850 7776\n",
      "25\t98474\t0.2\t1\t91331 7143\n",
      "26\t89046\t0.2\t1\t83387 5659\n",
      "27\t100470\t0.2\t1\t92717 7753\n",
      "28\t97480\t0.2\t1\t90362 7118\n",
      "29\t103196\t0.2\t1\t94309 8887\n",
      "30\t114370\t0.2\t1\t107320 7050\n",
      "31\t79392\t0.2\t1\t73939 5453\n",
      "32\t105208\t0.2\t1\t97664 7544\n",
      "33\t95289\t0.2\t1\t88120 7169\n",
      "34\t103964\t0.2\t1\t96266 7698\n",
      "35\t100819\t0.2\t1\t93035 7784\n",
      "36\t104164\t0.2\t1\t96092 8072\n",
      "37\t105927\t0.2\t1\t98240 7687\n",
      "38\t112135\t0.2\t1\t103582 8553\n",
      "39\t96387\t0.2\t1\t90041 6346\n",
      "40\t110619\t0.2\t1\t102036 8583\n",
      "41\t118329\t0.2\t1\t106686 11643\n",
      "42\t106988\t0.2\t1\t97227 9761\n",
      "43\t165097\t0.2\t1\t154382 10715\n",
      "44\t56454\t0.2\t1\t50914 5540\n",
      "45\t117022\t0.2\t1\t106731 10291\n",
      "46\t248165\t0.2\t1\t231776 16389\n",
      "47\t141744\t0.2\t1\t132412 9332\n",
      "48\t71383\t0.2\t1\t63944 7439\n",
      "49\t214966\t0.2\t1\t202098 12868\n",
      "50\t67123\t0.2\t1\t61729 5394\n",
      "51\t84537\t0.2\t1\t77868 6669\n",
      "52\t227448\t0.2\t1\t214969 12479\n",
      "53\t153876\t0.2\t1\t146571 7305\n",
      "54\t58063\t0.2\t1\t53681 4382\n",
      "55\t96470\t0.2\t1\t91390 5080\n",
      "56\t61719\t0.2\t1\t56899 4820\n",
      "57\t135082\t0.2\t1\t128247 6835\n",
      "58\t34302\t0.2\t1\t31774 2528\n",
      "59\t37018\t0.2\t1\t33361 3657\n",
      "60\t120449\t0.2\t1\t114261 6188\n",
      "61\t79440\t0.2\t1\t75334 4106\n",
      "62\t40303\t0.2\t1\t37043 3260\n",
      "63\t92254\t0.2\t1\t86676 5578\n",
      "64\t78944\t0.2\t1\t74628 4316\n",
      "65\t54545\t0.2\t1\t50567 3978\n",
      "66\t92915\t0.2\t1\t86970 5945\n",
      "67\t106337\t0.2\t1\t98786 7551\n",
      "68\t178710\t0.2\t1\t168688 10022\n",
      "69\t108665\t0.2\t1\t102241 6424\n",
      "70\t43742\t0.2\t1\t39918 3824\n",
      "71\t19154\t0.2\t1\t17119 2035\n",
      "72\t62321\t0.2\t1\t58726 3595\n",
      "73\t80997\t0.2\t1\t76319 4678\n",
      "74\t82292\t0.2\t1\t77235 5057\n",
      "75\t80453\t0.2\t1\t75230 5223\n",
      "76\t78946\t0.2\t1\t73647 5299\n",
      "77\t78496\t0.2\t1\t73454 5042\n",
      "78\t73891\t0.2\t1\t68991 4900\n",
      "79\t72359\t0.2\t1\t67790 4569\n",
      "80\t70385\t0.2\t1\t65644 4741\n",
      "81\t70687\t0.2\t1\t66072 4615\n",
      "82\t67765\t0.2\t1\t63360 4405\n",
      "83\t64119\t0.2\t1\t59858 4261\n",
      "84\t61128\t0.2\t1\t56996 4132\n",
      "85\t58834\t0.2\t1\t54971 3863\n",
      "86\t56645\t0.2\t1\t52920 3725\n",
      "87\t53397\t0.2\t1\t50007 3390\n",
      "88\t52020\t0.2\t1\t48563 3457\n",
      "89\t48843\t0.2\t1\t45677 3166\n",
      "90\t48488\t0.2\t1\t45284 3204\n",
      "91\t45775\t0.2\t1\t42701 3074\n",
      "92\t42633\t0.2\t1\t39741 2892\n",
      "93\t39061\t0.2\t1\t36500 2561\n",
      "94\t37084\t0.2\t1\t34707 2377\n",
      "95\t34306\t0.2\t1\t31956 2350\n",
      "96\t31990\t0.2\t1\t29804 2186\n",
      "97\t29916\t0.2\t1\t28016 1900\n",
      "98\t27386\t0.2\t1\t25513 1873\n",
      "99\t26105\t0.2\t1\t24176 1929\n",
      "100\t23728\t0.2\t1\t22014 1714\n",
      "101\t21360\t0.2\t1\t19893 1467\n",
      "102\t19677\t0.2\t1\t18296 1381\n",
      "103\t17691\t0.2\t1\t16474 1217\n",
      "104\t16298\t0.2\t1\t15194 1104\n",
      "105\t14499\t0.2\t1\t13498 1001\n",
      "106\t13016\t0.2\t1\t12109 907\n",
      "107\t11299\t0.2\t1\t10529 770\n",
      "108\t10657\t0.2\t1\t9906 751\n",
      "109\t9041\t0.2\t1\t8358 683\n",
      "110\t8101\t0.2\t1\t7484 617\n",
      "111\t7203\t0.2\t1\t6631 572\n",
      "112\t6115\t0.2\t1\t5677 438\n",
      "113\t5247\t0.2\t1\t4830 417\n",
      "114\t4477\t0.2\t1\t4092 385\n",
      "115\t3837\t0.2\t1\t3499 338\n",
      "116\t3178\t0.2\t1\t2885 293\n",
      "117\t2717\t0.2\t1\t2476 241\n",
      "118\t2308\t0.2\t1\t2079 229\n",
      "119\t1913\t0.2\t1\t1734 179\n",
      "120\t1562\t0.2\t1\t1431 131\n",
      "121\t1324\t0.2\t1\t1204 120\n",
      "122\t1075\t0.2\t1\t965 110\n",
      "123\t859\t0.2\t1\t768 91\n",
      "124\t724\t0.2\t1\t640 84\n",
      "125\t591\t0.2\t1\t519 72\n",
      "126\t475\t0.2\t1\t409 66\n",
      "127\t455\t0.2\t1\t404 51\n",
      "128\t340\t0.2\t1\t294 46\n",
      "129\t243\t0.2\t1\t193 50\n",
      "130\t216\t0.2\t1\t182 34\n",
      "131\t170\t0.2\t1\t146 24\n",
      "132\t150\t0.2\t1\t133 17\n",
      "133\t129\t0.2\t1\t114 15\n",
      "134\t222\t0.2\t1\t199 23\n",
      "135\t80\t0.2\t1\t75 5\n",
      "136\t46\t0.2\t1\t42 4\n",
      "137\t44\t0.2\t1\t37 7\n",
      "138\t28\t0.2\t1\t28\n",
      "139\t21\t0.2\t1\t18 3\n",
      "140\t10\t0.2\t1\t6 4\n",
      "141\t12\t0.2\t1\t8 4\n",
      "142\t16\t0.2\t1\t13 3\n",
      "143\t7\t0.2\t1\t3 4\n",
      "144\t10\t0.2\t1\t10\n",
      "145\t6\t0.2\t1\t4 2\n",
      "146\t6\t0.2\t1\t4 2\n",
      "147\t26\t0.2\t1\t24 2\n",
      "148\t13\t0.2\t1\t11 2\n",
      "149\t11\t0.2\t1\t11\n",
      "150\t141\t0.2\t1\t126 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14607942 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t1190 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/8C_32psu_4_S8_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 658.84 s (45 us/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,607,942\n",
      "Reads with adapters:                10,984,887 (75.2%)\n",
      "Reads written (passing filters):    14,607,942 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,191,191,300 bp\n",
      "Quality-trimmed:               8,040,292 bp (0.4%)\n",
      "Total written (filtered):  1,774,637,805 bp (81.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10984887 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 11.2%\n",
      "  G: 23.3%\n",
      "  T: 36.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2154515\t3651985.5\t0\t2154515\n",
      "2\t307769\t912996.4\t0\t307769\n",
      "3\t171355\t228249.1\t0\t171355\n",
      "4\t100114\t57062.3\t0\t100114\n",
      "5\t89281\t14265.6\t0\t89281\n",
      "6\t78941\t3566.4\t0\t78941\n",
      "7\t64482\t891.6\t0\t64482\n",
      "8\t90895\t222.9\t0\t90895\n",
      "9\t71519\t55.7\t0\t71259 260\n",
      "10\t77769\t13.9\t1\t72602 5167\n",
      "11\t81991\t3.5\t1\t75629 6362\n",
      "12\t81945\t0.9\t1\t75457 6488\n",
      "13\t93276\t0.2\t1\t85901 7375\n",
      "14\t86548\t0.2\t1\t79069 7479\n",
      "15\t71575\t0.2\t1\t66062 5513\n",
      "16\t101873\t0.2\t1\t93937 7936\n",
      "17\t86756\t0.2\t1\t79766 6990\n",
      "18\t75151\t0.2\t1\t69487 5664\n",
      "19\t100031\t0.2\t1\t92361 7670\n",
      "20\t92540\t0.2\t1\t85573 6967\n",
      "21\t80563\t0.2\t1\t74553 6010\n",
      "22\t91001\t0.2\t1\t84166 6835\n",
      "23\t96186\t0.2\t1\t88873 7313\n",
      "24\t95097\t0.2\t1\t87831 7266\n",
      "25\t107490\t0.2\t1\t100206 7284\n",
      "26\t89500\t0.2\t1\t82469 7031\n",
      "27\t97464\t0.2\t1\t89433 8031\n",
      "28\t89257\t0.2\t1\t83469 5788\n",
      "29\t97776\t0.2\t1\t90646 7130\n",
      "30\t94678\t0.2\t1\t88320 6358\n",
      "31\t104562\t0.2\t1\t97122 7440\n",
      "32\t90375\t0.2\t1\t84247 6128\n",
      "33\t104255\t0.2\t1\t96395 7860\n",
      "34\t131765\t0.2\t1\t122731 9034\n",
      "35\t71827\t0.2\t1\t67071 4756\n",
      "36\t101000\t0.2\t1\t93594 7406\n",
      "37\t108384\t0.2\t1\t100841 7543\n",
      "38\t100305\t0.2\t1\t93893 6412\n",
      "39\t96265\t0.2\t1\t90338 5927\n",
      "40\t108609\t0.2\t1\t101413 7196\n",
      "41\t105469\t0.2\t1\t98528 6941\n",
      "42\t108491\t0.2\t1\t101465 7026\n",
      "43\t97504\t0.2\t1\t91262 6242\n",
      "44\t114735\t0.2\t1\t107039 7696\n",
      "45\t119691\t0.2\t1\t111821 7870\n",
      "46\t85524\t0.2\t1\t79914 5610\n",
      "47\t107169\t0.2\t1\t100947 6222\n",
      "48\t104472\t0.2\t1\t98077 6395\n",
      "49\t109888\t0.2\t1\t102993 6895\n",
      "50\t104895\t0.2\t1\t98149 6746\n",
      "51\t128024\t0.2\t1\t120005 8019\n",
      "52\t92440\t0.2\t1\t86601 5839\n",
      "53\t114005\t0.2\t1\t107513 6492\n",
      "54\t82023\t0.2\t1\t77215 4808\n",
      "55\t107798\t0.2\t1\t101620 6178\n",
      "56\t110208\t0.2\t1\t103344 6864\n",
      "57\t105022\t0.2\t1\t99470 5552\n",
      "58\t100686\t0.2\t1\t95258 5428\n",
      "59\t100045\t0.2\t1\t94333 5712\n",
      "60\t103616\t0.2\t1\t97757 5859\n",
      "61\t106446\t0.2\t1\t100410 6036\n",
      "62\t115794\t0.2\t1\t108819 6975\n",
      "63\t149726\t0.2\t1\t142046 7680\n",
      "64\t116619\t0.2\t1\t111405 5214\n",
      "65\t80564\t0.2\t1\t76264 4300\n",
      "66\t87049\t0.2\t1\t82185 4864\n",
      "67\t86779\t0.2\t1\t81921 4858\n",
      "68\t87371\t0.2\t1\t82619 4752\n",
      "69\t85219\t0.2\t1\t80357 4862\n",
      "70\t93639\t0.2\t1\t88366 5273\n",
      "71\t91437\t0.2\t1\t86289 5148\n",
      "72\t86910\t0.2\t1\t81943 4967\n",
      "73\t89655\t0.2\t1\t84810 4845\n",
      "74\t82770\t0.2\t1\t78094 4676\n",
      "75\t77878\t0.2\t1\t73550 4328\n",
      "76\t75167\t0.2\t1\t70959 4208\n",
      "77\t74827\t0.2\t1\t70567 4260\n",
      "78\t71263\t0.2\t1\t67257 4006\n",
      "79\t68082\t0.2\t1\t64314 3768\n",
      "80\t66978\t0.2\t1\t63430 3548\n",
      "81\t67480\t0.2\t1\t63757 3723\n",
      "82\t63500\t0.2\t1\t60093 3407\n",
      "83\t60414\t0.2\t1\t57170 3244\n",
      "84\t57646\t0.2\t1\t54576 3070\n",
      "85\t56216\t0.2\t1\t53261 2955\n",
      "86\t55705\t0.2\t1\t52806 2899\n",
      "87\t54555\t0.2\t1\t51737 2818\n",
      "88\t52657\t0.2\t1\t49839 2818\n",
      "89\t47083\t0.2\t1\t44512 2571\n",
      "90\t45017\t0.2\t1\t42539 2478\n",
      "91\t42292\t0.2\t1\t40068 2224\n",
      "92\t39093\t0.2\t1\t37073 2020\n",
      "93\t36824\t0.2\t1\t34979 1845\n",
      "94\t35114\t0.2\t1\t33278 1836\n",
      "95\t33006\t0.2\t1\t31281 1725\n",
      "96\t30878\t0.2\t1\t29246 1632\n",
      "97\t28202\t0.2\t1\t26749 1453\n",
      "98\t25663\t0.2\t1\t24339 1324\n",
      "99\t24079\t0.2\t1\t22771 1308\n",
      "100\t21840\t0.2\t1\t20690 1150\n",
      "101\t19586\t0.2\t1\t18574 1012\n",
      "102\t18375\t0.2\t1\t17447 928\n",
      "103\t16706\t0.2\t1\t15871 835\n",
      "104\t15675\t0.2\t1\t14815 860\n",
      "105\t13756\t0.2\t1\t13047 709\n",
      "106\t12568\t0.2\t1\t11946 622\n",
      "107\t10619\t0.2\t1\t10045 574\n",
      "108\t9793\t0.2\t1\t9277 516\n",
      "109\t8520\t0.2\t1\t8082 438\n",
      "110\t7642\t0.2\t1\t7254 388\n",
      "111\t7055\t0.2\t1\t6693 362\n",
      "112\t5993\t0.2\t1\t5692 301\n",
      "113\t5089\t0.2\t1\t4789 300\n",
      "114\t4335\t0.2\t1\t4087 248\n",
      "115\t3576\t0.2\t1\t3383 193\n",
      "116\t2943\t0.2\t1\t2759 184\n",
      "117\t2497\t0.2\t1\t2342 155\n",
      "118\t2119\t0.2\t1\t1989 130\n",
      "119\t1740\t0.2\t1\t1643 97\n",
      "120\t1465\t0.2\t1\t1354 111\n",
      "121\t1235\t0.2\t1\t1152 83\n",
      "122\t1011\t0.2\t1\t954 57\n",
      "123\t812\t0.2\t1\t749 63\n",
      "124\t706\t0.2\t1\t655 51\n",
      "125\t559\t0.2\t1\t512 47\n",
      "126\t438\t0.2\t1\t415 23\n",
      "127\t439\t0.2\t1\t406 33\n",
      "128\t306\t0.2\t1\t286 20\n",
      "129\t222\t0.2\t1\t208 14\n",
      "130\t202\t0.2\t1\t190 12\n",
      "131\t161\t0.2\t1\t148 13\n",
      "132\t146\t0.2\t1\t132 14\n",
      "133\t137\t0.2\t1\t126 11\n",
      "134\t210\t0.2\t1\t203 7\n",
      "135\t81\t0.2\t1\t72 9\n",
      "136\t43\t0.2\t1\t38 5\n",
      "137\t44\t0.2\t1\t36 8\n",
      "138\t25\t0.2\t1\t22 3\n",
      "139\t23\t0.2\t1\t23\n",
      "140\t5\t0.2\t1\t5\n",
      "141\t9\t0.2\t1\t9\n",
      "142\t12\t0.2\t1\t11 1\n",
      "143\t2\t0.2\t1\t2\n",
      "144\t10\t0.2\t1\t7 3\n",
      "145\t5\t0.2\t1\t5\n",
      "146\t6\t0.2\t1\t6\n",
      "147\t23\t0.2\t1\t22 1\n",
      "148\t13\t0.2\t1\t13\n",
      "149\t11\t0.2\t1\t10 1\n",
      "150\t112\t0.2\t1\t105 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14607942 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t10571 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 755.53 s (41 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,576,202\n",
      "Reads with adapters:                14,608,493 (78.6%)\n",
      "Reads written (passing filters):    18,576,202 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,786,430,300 bp\n",
      "Quality-trimmed:               5,505,335 bp (0.2%)\n",
      "Total written (filtered):  2,183,212,736 bp (78.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14608493 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.1%\n",
      "  G: 20.7%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2367687\t4644050.5\t0\t2367687\n",
      "2\t362390\t1161012.6\t0\t362390\n",
      "3\t199560\t290253.2\t0\t199560\n",
      "4\t122411\t72563.3\t0\t122411\n",
      "5\t101824\t18140.8\t0\t101824\n",
      "6\t98694\t4535.2\t0\t98694\n",
      "7\t85930\t1133.8\t0\t85930\n",
      "8\t108616\t283.5\t0\t108616\n",
      "9\t98214\t70.9\t0\t96968 1246\n",
      "10\t98262\t17.7\t1\t93000 5262\n",
      "11\t106716\t4.4\t1\t98716 8000\n",
      "12\t102761\t1.1\t1\t96030 6731\n",
      "13\t106394\t0.3\t1\t99361 7033\n",
      "14\t105192\t0.3\t1\t97880 7312\n",
      "15\t111384\t0.3\t1\t104056 7328\n",
      "16\t109489\t0.3\t1\t101402 8087\n",
      "17\t132410\t0.3\t1\t121978 10432\n",
      "18\t113576\t0.3\t1\t106813 6763\n",
      "19\t98216\t0.3\t1\t92113 6103\n",
      "20\t116366\t0.3\t1\t109085 7281\n",
      "21\t128770\t0.3\t1\t119833 8937\n",
      "22\t109971\t0.3\t1\t103795 6176\n",
      "23\t129883\t0.3\t1\t121543 8340\n",
      "24\t115035\t0.3\t1\t105774 9261\n",
      "25\t127548\t0.3\t1\t119043 8505\n",
      "26\t115805\t0.3\t1\t109020 6785\n",
      "27\t128062\t0.3\t1\t118816 9246\n",
      "28\t133464\t0.3\t1\t123994 9470\n",
      "29\t128355\t0.3\t1\t118623 9732\n",
      "30\t145787\t0.3\t1\t137741 8046\n",
      "31\t107347\t0.3\t1\t100584 6763\n",
      "32\t136432\t0.3\t1\t127704 8728\n",
      "33\t126911\t0.3\t1\t118459 8452\n",
      "34\t150928\t0.3\t1\t140208 10720\n",
      "35\t123580\t0.3\t1\t115931 7649\n",
      "36\t142531\t0.3\t1\t133504 9027\n",
      "37\t133143\t0.3\t1\t125509 7634\n",
      "38\t143413\t0.3\t1\t133925 9488\n",
      "39\t142038\t0.3\t1\t132863 9175\n",
      "40\t145926\t0.3\t1\t136263 9663\n",
      "41\t151709\t0.3\t1\t139050 12659\n",
      "42\t137644\t0.3\t1\t127697 9947\n",
      "43\t208996\t0.3\t1\t198310 10686\n",
      "44\t73738\t0.3\t1\t67589 6149\n",
      "45\t147037\t0.3\t1\t138219 8818\n",
      "46\t140744\t0.3\t1\t131869 8875\n",
      "47\t152223\t0.3\t1\t143116 9107\n",
      "48\t139803\t0.3\t1\t130633 9170\n",
      "49\t156282\t0.3\t1\t145534 10748\n",
      "50\t133834\t0.3\t1\t126283 7551\n",
      "51\t143388\t0.3\t1\t134512 8876\n",
      "52\t151003\t0.3\t1\t140213 10790\n",
      "53\t166892\t0.3\t1\t157230 9662\n",
      "54\t123800\t0.3\t1\t116115 7685\n",
      "55\t151573\t0.3\t1\t142303 9270\n",
      "56\t178039\t0.3\t1\t168271 9768\n",
      "57\t131467\t0.3\t1\t122895 8572\n",
      "58\t147115\t0.3\t1\t139737 7378\n",
      "59\t118925\t0.3\t1\t111394 7531\n",
      "60\t151799\t0.3\t1\t142575 9224\n",
      "61\t144120\t0.3\t1\t136068 8052\n",
      "62\t147428\t0.3\t1\t139233 8195\n",
      "63\t128075\t0.3\t1\t119968 8107\n",
      "64\t135764\t0.3\t1\t128209 7555\n",
      "65\t146614\t0.3\t1\t138085 8529\n",
      "66\t155870\t0.3\t1\t146365 9505\n",
      "67\t168571\t0.3\t1\t156591 11980\n",
      "68\t310336\t0.3\t1\t295745 14591\n",
      "69\t61543\t0.3\t1\t56640 4903\n",
      "70\t21841\t0.3\t1\t18433 3408\n",
      "71\t77515\t0.3\t1\t72834 4681\n",
      "72\t115950\t0.3\t1\t109389 6561\n",
      "73\t122667\t0.3\t1\t115604 7063\n",
      "74\t120840\t0.3\t1\t113936 6904\n",
      "75\t118343\t0.3\t1\t111514 6829\n",
      "76\t117750\t0.3\t1\t110989 6761\n",
      "77\t117217\t0.3\t1\t110486 6731\n",
      "78\t109143\t0.3\t1\t102786 6357\n",
      "79\t107883\t0.3\t1\t101668 6215\n",
      "80\t105101\t0.3\t1\t99015 6086\n",
      "81\t104048\t0.3\t1\t98069 5979\n",
      "82\t100791\t0.3\t1\t95040 5751\n",
      "83\t96002\t0.3\t1\t90436 5566\n",
      "84\t92813\t0.3\t1\t87473 5340\n",
      "85\t89718\t0.3\t1\t84646 5072\n",
      "86\t86520\t0.3\t1\t81561 4959\n",
      "87\t81906\t0.3\t1\t77243 4663\n",
      "88\t81347\t0.3\t1\t76501 4846\n",
      "89\t76428\t0.3\t1\t72015 4413\n",
      "90\t76790\t0.3\t1\t72233 4557\n",
      "91\t72129\t0.3\t1\t67780 4349\n",
      "92\t67716\t0.3\t1\t63638 4078\n",
      "93\t63437\t0.3\t1\t59587 3850\n",
      "94\t61274\t0.3\t1\t57720 3554\n",
      "95\t56931\t0.3\t1\t53504 3427\n",
      "96\t53326\t0.3\t1\t50061 3265\n",
      "97\t50390\t0.3\t1\t47429 2961\n",
      "98\t46074\t0.3\t1\t43201 2873\n",
      "99\t45190\t0.3\t1\t42231 2959\n",
      "100\t40830\t0.3\t1\t38265 2565\n",
      "101\t37913\t0.3\t1\t35541 2372\n",
      "102\t35110\t0.3\t1\t32897 2213\n",
      "103\t31900\t0.3\t1\t29985 1915\n",
      "104\t30069\t0.3\t1\t28092 1977\n",
      "105\t26590\t0.3\t1\t24861 1729\n",
      "106\t23847\t0.3\t1\t22283 1564\n",
      "107\t21922\t0.3\t1\t20517 1405\n",
      "108\t20575\t0.3\t1\t19261 1314\n",
      "109\t18220\t0.3\t1\t17014 1206\n",
      "110\t16359\t0.3\t1\t15225 1134\n",
      "111\t14698\t0.3\t1\t13709 989\n",
      "112\t12826\t0.3\t1\t11919 907\n",
      "113\t11006\t0.3\t1\t10256 750\n",
      "114\t9719\t0.3\t1\t9039 680\n",
      "115\t8407\t0.3\t1\t7839 568\n",
      "116\t7289\t0.3\t1\t6725 564\n",
      "117\t6390\t0.3\t1\t5903 487\n",
      "118\t5317\t0.3\t1\t4911 406\n",
      "119\t4366\t0.3\t1\t4006 360\n",
      "120\t3632\t0.3\t1\t3354 278\n",
      "121\t3334\t0.3\t1\t3055 279\n",
      "122\t2775\t0.3\t1\t2532 243\n",
      "123\t2262\t0.3\t1\t2067 195\n",
      "124\t1853\t0.3\t1\t1718 135\n",
      "125\t1573\t0.3\t1\t1418 155\n",
      "126\t1249\t0.3\t1\t1114 135\n",
      "127\t1068\t0.3\t1\t970 98\n",
      "128\t811\t0.3\t1\t720 91\n",
      "129\t753\t0.3\t1\t679 74\n",
      "130\t521\t0.3\t1\t476 45\n",
      "131\t504\t0.3\t1\t454 50\n",
      "132\t357\t0.3\t1\t326 31\n",
      "133\t348\t0.3\t1\t306 42\n",
      "134\t413\t0.3\t1\t388 25\n",
      "135\t242\t0.3\t1\t222 20\n",
      "136\t135\t0.3\t1\t122 13\n",
      "137\t135\t0.3\t1\t121 14\n",
      "138\t66\t0.3\t1\t62 4\n",
      "139\t41\t0.3\t1\t36 5\n",
      "140\t32\t0.3\t1\t27 5\n",
      "141\t40\t0.3\t1\t34 6\n",
      "142\t26\t0.3\t1\t24 2\n",
      "143\t30\t0.3\t1\t25 5\n",
      "144\t18\t0.3\t1\t15 3\n",
      "145\t14\t0.3\t1\t12 2\n",
      "146\t21\t0.3\t1\t19 2\n",
      "147\t69\t0.3\t1\t65 4\n",
      "148\t44\t0.3\t1\t40 4\n",
      "149\t37\t0.3\t1\t35 2\n",
      "150\t304\t0.3\t1\t264 40\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18576202 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2936 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 791.42 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,576,202\n",
      "Reads with adapters:                14,527,088 (78.2%)\n",
      "Reads written (passing filters):    18,576,202 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,786,430,300 bp\n",
      "Quality-trimmed:              12,297,720 bp (0.4%)\n",
      "Total written (filtered):  2,186,026,388 bp (78.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14527088 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 12.4%\n",
      "  G: 24.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2397075\t4644050.5\t0\t2397075\n",
      "2\t377772\t1161012.6\t0\t377772\n",
      "3\t209931\t290253.2\t0\t209931\n",
      "4\t123717\t72563.3\t0\t123717\n",
      "5\t112624\t18140.8\t0\t112624\n",
      "6\t97477\t4535.2\t0\t97477\n",
      "7\t81404\t1133.8\t0\t81404\n",
      "8\t114796\t283.5\t0\t114796\n",
      "9\t89736\t70.9\t0\t89386 350\n",
      "10\t100021\t17.7\t1\t95112 4909\n",
      "11\t103726\t4.4\t1\t97357 6369\n",
      "12\t104603\t1.1\t1\t97938 6665\n",
      "13\t116572\t0.3\t1\t109738 6834\n",
      "14\t110440\t0.3\t1\t102780 7660\n",
      "15\t91038\t0.3\t1\t85192 5846\n",
      "16\t129548\t0.3\t1\t121431 8117\n",
      "17\t105093\t0.3\t1\t97969 7124\n",
      "18\t99076\t0.3\t1\t93067 6009\n",
      "19\t126890\t0.3\t1\t118635 8255\n",
      "20\t118457\t0.3\t1\t111491 6966\n",
      "21\t103736\t0.3\t1\t97365 6371\n",
      "22\t118151\t0.3\t1\t110881 7270\n",
      "23\t121946\t0.3\t1\t114464 7482\n",
      "24\t121962\t0.3\t1\t114186 7776\n",
      "25\t136854\t0.3\t1\t128708 8146\n",
      "26\t113321\t0.3\t1\t106068 7253\n",
      "27\t126381\t0.3\t1\t117798 8583\n",
      "28\t118846\t0.3\t1\t112727 6119\n",
      "29\t125505\t0.3\t1\t117793 7712\n",
      "30\t125215\t0.3\t1\t118459 6756\n",
      "31\t135187\t0.3\t1\t127291 7896\n",
      "32\t121491\t0.3\t1\t114900 6591\n",
      "33\t135254\t0.3\t1\t127041 8213\n",
      "34\t169655\t0.3\t1\t159717 9938\n",
      "35\t101342\t0.3\t1\t95776 5566\n",
      "36\t136505\t0.3\t1\t128363 8142\n",
      "37\t143293\t0.3\t1\t134875 8418\n",
      "38\t134051\t0.3\t1\t127357 6694\n",
      "39\t131563\t0.3\t1\t124738 6825\n",
      "40\t145300\t0.3\t1\t136735 8565\n",
      "41\t138947\t0.3\t1\t131652 7295\n",
      "42\t144334\t0.3\t1\t136660 7674\n",
      "43\t131871\t0.3\t1\t124782 7089\n",
      "44\t149402\t0.3\t1\t141199 8203\n",
      "45\t161283\t0.3\t1\t152880 8403\n",
      "46\t115740\t0.3\t1\t109551 6189\n",
      "47\t146650\t0.3\t1\t139277 7373\n",
      "48\t144979\t0.3\t1\t137763 7216\n",
      "49\t149370\t0.3\t1\t141741 7629\n",
      "50\t140556\t0.3\t1\t132993 7563\n",
      "51\t176763\t0.3\t1\t167628 9135\n",
      "52\t125327\t0.3\t1\t119270 6057\n",
      "53\t150468\t0.3\t1\t143751 6717\n",
      "54\t114488\t0.3\t1\t108938 5550\n",
      "55\t150264\t0.3\t1\t143436 6828\n",
      "56\t154642\t0.3\t1\t147075 7567\n",
      "57\t145698\t0.3\t1\t138895 6803\n",
      "58\t139003\t0.3\t1\t132772 6231\n",
      "59\t141639\t0.3\t1\t135020 6619\n",
      "60\t145216\t0.3\t1\t138147 7069\n",
      "61\t149173\t0.3\t1\t141978 7195\n",
      "62\t161580\t0.3\t1\t153777 7803\n",
      "63\t211777\t0.3\t1\t202721 9056\n",
      "64\t166013\t0.3\t1\t159545 6468\n",
      "65\t115072\t0.3\t1\t109927 5145\n",
      "66\t124032\t0.3\t1\t118392 5640\n",
      "67\t121997\t0.3\t1\t116256 5741\n",
      "68\t123305\t0.3\t1\t117689 5616\n",
      "69\t119979\t0.3\t1\t114424 5555\n",
      "70\t126522\t0.3\t1\t120645 5877\n",
      "71\t131428\t0.3\t1\t125503 5925\n",
      "72\t124583\t0.3\t1\t118750 5833\n",
      "73\t129753\t0.3\t1\t124066 5687\n",
      "74\t119016\t0.3\t1\t113726 5290\n",
      "75\t113191\t0.3\t1\t108419 4772\n",
      "76\t111393\t0.3\t1\t106547 4846\n",
      "77\t111470\t0.3\t1\t106513 4957\n",
      "78\t104799\t0.3\t1\t100049 4750\n",
      "79\t100883\t0.3\t1\t96421 4462\n",
      "80\t100236\t0.3\t1\t95706 4530\n",
      "81\t98928\t0.3\t1\t94651 4277\n",
      "82\t94279\t0.3\t1\t90155 4124\n",
      "83\t90115\t0.3\t1\t86173 3942\n",
      "84\t87249\t0.3\t1\t83373 3876\n",
      "85\t85689\t0.3\t1\t81793 3896\n",
      "86\t84888\t0.3\t1\t81207 3681\n",
      "87\t84171\t0.3\t1\t80558 3613\n",
      "88\t82226\t0.3\t1\t78630 3596\n",
      "89\t73279\t0.3\t1\t70084 3195\n",
      "90\t71368\t0.3\t1\t68189 3179\n",
      "91\t66468\t0.3\t1\t63502 2966\n",
      "92\t61869\t0.3\t1\t59204 2665\n",
      "93\t59677\t0.3\t1\t57086 2591\n",
      "94\t57840\t0.3\t1\t55318 2522\n",
      "95\t54875\t0.3\t1\t52595 2280\n",
      "96\t51698\t0.3\t1\t49449 2249\n",
      "97\t47519\t0.3\t1\t45519 2000\n",
      "98\t43527\t0.3\t1\t41676 1851\n",
      "99\t42094\t0.3\t1\t40263 1831\n",
      "100\t37940\t0.3\t1\t36384 1556\n",
      "101\t34646\t0.3\t1\t33175 1471\n",
      "102\t32213\t0.3\t1\t30921 1292\n",
      "103\t29202\t0.3\t1\t27960 1242\n",
      "104\t28501\t0.3\t1\t27313 1188\n",
      "105\t25308\t0.3\t1\t24224 1084\n",
      "106\t22773\t0.3\t1\t21858 915\n",
      "107\t20894\t0.3\t1\t20051 843\n",
      "108\t18912\t0.3\t1\t18081 831\n",
      "109\t17072\t0.3\t1\t16336 736\n",
      "110\t15374\t0.3\t1\t14700 674\n",
      "111\t14518\t0.3\t1\t13895 623\n",
      "112\t12456\t0.3\t1\t11891 565\n",
      "113\t10771\t0.3\t1\t10329 442\n",
      "114\t9374\t0.3\t1\t8976 398\n",
      "115\t7920\t0.3\t1\t7555 365\n",
      "116\t6787\t0.3\t1\t6458 329\n",
      "117\t5803\t0.3\t1\t5538 265\n",
      "118\t4857\t0.3\t1\t4636 221\n",
      "119\t4060\t0.3\t1\t3861 199\n",
      "120\t3400\t0.3\t1\t3228 172\n",
      "121\t3132\t0.3\t1\t2976 156\n",
      "122\t2609\t0.3\t1\t2493 116\n",
      "123\t2165\t0.3\t1\t2072 93\n",
      "124\t1770\t0.3\t1\t1676 94\n",
      "125\t1516\t0.3\t1\t1432 84\n",
      "126\t1213\t0.3\t1\t1153 60\n",
      "127\t1001\t0.3\t1\t941 60\n",
      "128\t776\t0.3\t1\t728 48\n",
      "129\t744\t0.3\t1\t699 45\n",
      "130\t486\t0.3\t1\t463 23\n",
      "131\t460\t0.3\t1\t438 22\n",
      "132\t343\t0.3\t1\t315 28\n",
      "133\t355\t0.3\t1\t338 17\n",
      "134\t428\t0.3\t1\t403 25\n",
      "135\t226\t0.3\t1\t216 10\n",
      "136\t133\t0.3\t1\t120 13\n",
      "137\t125\t0.3\t1\t117 8\n",
      "138\t70\t0.3\t1\t62 8\n",
      "139\t38\t0.3\t1\t35 3\n",
      "140\t38\t0.3\t1\t31 7\n",
      "141\t35\t0.3\t1\t33 2\n",
      "142\t27\t0.3\t1\t26 1\n",
      "143\t32\t0.3\t1\t30 2\n",
      "144\t14\t0.3\t1\t13 1\n",
      "145\t17\t0.3\t1\t16 1\n",
      "146\t23\t0.3\t1\t23\n",
      "147\t69\t0.3\t1\t65 4\n",
      "148\t40\t0.3\t1\t38 2\n",
      "149\t38\t0.3\t1\t34 4\n",
      "150\t229\t0.3\t1\t219 10\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18576202 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t20489 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 763.05 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,490,322\n",
      "Reads with adapters:                14,542,227 (78.6%)\n",
      "Reads written (passing filters):    18,490,322 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,773,548,300 bp\n",
      "Quality-trimmed:               5,682,539 bp (0.2%)\n",
      "Total written (filtered):  2,170,540,982 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14542227 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.1%\n",
      "  G: 20.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2352565\t4622580.5\t0\t2352565\n",
      "2\t355744\t1155645.1\t0\t355744\n",
      "3\t197435\t288911.3\t0\t197435\n",
      "4\t119803\t72227.8\t0\t119803\n",
      "5\t101491\t18057.0\t0\t101491\n",
      "6\t98683\t4514.2\t0\t98683\n",
      "7\t82227\t1128.6\t0\t82227\n",
      "8\t107087\t282.1\t0\t107087\n",
      "9\t96108\t70.5\t0\t94984 1124\n",
      "10\t95916\t17.6\t1\t89656 6260\n",
      "11\t105518\t4.4\t1\t96331 9187\n",
      "12\t101326\t1.1\t1\t93731 7595\n",
      "13\t105529\t0.3\t1\t97295 8234\n",
      "14\t104207\t0.3\t1\t95707 8500\n",
      "15\t110166\t0.3\t1\t101496 8670\n",
      "16\t108369\t0.3\t1\t98900 9469\n",
      "17\t132137\t0.3\t1\t119869 12268\n",
      "18\t111785\t0.3\t1\t103776 8009\n",
      "19\t94978\t0.3\t1\t88024 6954\n",
      "20\t115852\t0.3\t1\t107171 8681\n",
      "21\t128602\t0.3\t1\t118248 10354\n",
      "22\t106928\t0.3\t1\t99662 7266\n",
      "23\t130246\t0.3\t1\t120393 9853\n",
      "24\t113598\t0.3\t1\t103348 10250\n",
      "25\t127529\t0.3\t1\t117557 9972\n",
      "26\t114329\t0.3\t1\t106365 7964\n",
      "27\t127671\t0.3\t1\t117117 10554\n",
      "28\t133545\t0.3\t1\t122886 10659\n",
      "29\t127218\t0.3\t1\t116368 10850\n",
      "30\t145807\t0.3\t1\t136218 9589\n",
      "31\t104710\t0.3\t1\t97327 7383\n",
      "32\t136712\t0.3\t1\t126029 10683\n",
      "33\t125051\t0.3\t1\t114983 10068\n",
      "34\t151172\t0.3\t1\t138255 12917\n",
      "35\t121506\t0.3\t1\t112294 9212\n",
      "36\t142450\t0.3\t1\t131732 10718\n",
      "37\t131704\t0.3\t1\t122359 9345\n",
      "38\t144055\t0.3\t1\t132700 11355\n",
      "39\t140259\t0.3\t1\t129402 10857\n",
      "40\t145119\t0.3\t1\t132998 12121\n",
      "41\t151317\t0.3\t1\t136837 14480\n",
      "42\t135755\t0.3\t1\t124185 11570\n",
      "43\t206150\t0.3\t1\t193096 13054\n",
      "44\t75981\t0.3\t1\t68776 7205\n",
      "45\t147101\t0.3\t1\t136563 10538\n",
      "46\t140010\t0.3\t1\t129569 10441\n",
      "47\t150202\t0.3\t1\t139726 10476\n",
      "48\t137985\t0.3\t1\t127406 10579\n",
      "49\t157361\t0.3\t1\t144848 12513\n",
      "50\t132126\t0.3\t1\t123324 8802\n",
      "51\t143254\t0.3\t1\t133169 10085\n",
      "52\t152534\t0.3\t1\t140225 12309\n",
      "53\t166653\t0.3\t1\t155543 11110\n",
      "54\t122896\t0.3\t1\t113877 9019\n",
      "55\t150505\t0.3\t1\t139851 10654\n",
      "56\t178712\t0.3\t1\t166946 11766\n",
      "57\t130999\t0.3\t1\t120759 10240\n",
      "58\t145047\t0.3\t1\t136344 8703\n",
      "59\t119684\t0.3\t1\t110878 8806\n",
      "60\t151206\t0.3\t1\t140682 10524\n",
      "61\t142715\t0.3\t1\t133251 9464\n",
      "62\t148943\t0.3\t1\t139114 9829\n",
      "63\t129912\t0.3\t1\t120416 9496\n",
      "64\t134038\t0.3\t1\t125474 8564\n",
      "65\t146595\t0.3\t1\t136655 9940\n",
      "66\t153978\t0.3\t1\t143441 10537\n",
      "67\t168100\t0.3\t1\t155171 12929\n",
      "68\t298170\t0.3\t1\t281868 16302\n",
      "69\t74393\t0.3\t1\t68405 5988\n",
      "70\t26909\t0.3\t1\t22880 4029\n",
      "71\t75109\t0.3\t1\t70102 5007\n",
      "72\t114931\t0.3\t1\t107946 6985\n",
      "73\t122734\t0.3\t1\t114818 7916\n",
      "74\t120820\t0.3\t1\t112962 7858\n",
      "75\t119099\t0.3\t1\t111010 8089\n",
      "76\t117029\t0.3\t1\t108855 8174\n",
      "77\t117800\t0.3\t1\t109726 8074\n",
      "78\t109388\t0.3\t1\t101815 7573\n",
      "79\t108107\t0.3\t1\t100589 7518\n",
      "80\t105715\t0.3\t1\t98229 7486\n",
      "81\t104556\t0.3\t1\t97305 7251\n",
      "82\t101011\t0.3\t1\t93996 7015\n",
      "83\t97159\t0.3\t1\t90577 6582\n",
      "84\t92283\t0.3\t1\t85890 6393\n",
      "85\t90035\t0.3\t1\t83799 6236\n",
      "86\t86319\t0.3\t1\t80378 5941\n",
      "87\t82376\t0.3\t1\t76893 5483\n",
      "88\t81375\t0.3\t1\t75753 5622\n",
      "89\t77211\t0.3\t1\t71720 5491\n",
      "90\t76508\t0.3\t1\t71173 5335\n",
      "91\t72845\t0.3\t1\t67788 5057\n",
      "92\t67776\t0.3\t1\t63180 4596\n",
      "93\t63675\t0.3\t1\t59371 4304\n",
      "94\t61289\t0.3\t1\t57047 4242\n",
      "95\t57075\t0.3\t1\t53036 4039\n",
      "96\t54087\t0.3\t1\t50327 3760\n",
      "97\t51019\t0.3\t1\t47507 3512\n",
      "98\t46423\t0.3\t1\t43108 3315\n",
      "99\t45164\t0.3\t1\t41742 3422\n",
      "100\t40910\t0.3\t1\t38024 2886\n",
      "101\t38601\t0.3\t1\t35925 2676\n",
      "102\t35185\t0.3\t1\t32582 2603\n",
      "103\t32331\t0.3\t1\t30034 2297\n",
      "104\t30306\t0.3\t1\t28179 2127\n",
      "105\t26497\t0.3\t1\t24651 1846\n",
      "106\t24125\t0.3\t1\t22458 1667\n",
      "107\t22000\t0.3\t1\t20417 1583\n",
      "108\t20719\t0.3\t1\t19189 1530\n",
      "109\t18051\t0.3\t1\t16712 1339\n",
      "110\t16439\t0.3\t1\t15194 1245\n",
      "111\t14466\t0.3\t1\t13407 1059\n",
      "112\t12793\t0.3\t1\t11768 1025\n",
      "113\t11344\t0.3\t1\t10481 863\n",
      "114\t9637\t0.3\t1\t8889 748\n",
      "115\t8596\t0.3\t1\t7840 756\n",
      "116\t7497\t0.3\t1\t6926 571\n",
      "117\t6216\t0.3\t1\t5692 524\n",
      "118\t5413\t0.3\t1\t4943 470\n",
      "119\t4488\t0.3\t1\t4121 367\n",
      "120\t3934\t0.3\t1\t3598 336\n",
      "121\t3277\t0.3\t1\t2991 286\n",
      "122\t2701\t0.3\t1\t2461 240\n",
      "123\t2240\t0.3\t1\t2028 212\n",
      "124\t1944\t0.3\t1\t1730 214\n",
      "125\t1578\t0.3\t1\t1422 156\n",
      "126\t1320\t0.3\t1\t1199 121\n",
      "127\t1172\t0.3\t1\t1056 116\n",
      "128\t861\t0.3\t1\t773 88\n",
      "129\t759\t0.3\t1\t683 76\n",
      "130\t595\t0.3\t1\t520 75\n",
      "131\t487\t0.3\t1\t426 61\n",
      "132\t384\t0.3\t1\t352 32\n",
      "133\t353\t0.3\t1\t318 35\n",
      "134\t434\t0.3\t1\t399 35\n",
      "135\t273\t0.3\t1\t245 28\n",
      "136\t156\t0.3\t1\t137 19\n",
      "137\t94\t0.3\t1\t83 11\n",
      "138\t77\t0.3\t1\t69 8\n",
      "139\t51\t0.3\t1\t44 7\n",
      "140\t30\t0.3\t1\t25 5\n",
      "141\t33\t0.3\t1\t24 9\n",
      "142\t30\t0.3\t1\t24 6\n",
      "143\t35\t0.3\t1\t27 8\n",
      "144\t18\t0.3\t1\t17 1\n",
      "145\t11\t0.3\t1\t9 2\n",
      "146\t25\t0.3\t1\t21 4\n",
      "147\t75\t0.3\t1\t73 2\n",
      "148\t42\t0.3\t1\t37 5\n",
      "149\t44\t0.3\t1\t41 3\n",
      "150\t294\t0.3\t1\t257 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18490322 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3001 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 804.94 s (44 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,490,322\n",
      "Reads with adapters:                14,450,091 (78.1%)\n",
      "Reads written (passing filters):    18,490,322 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,773,548,300 bp\n",
      "Quality-trimmed:              11,878,254 bp (0.4%)\n",
      "Total written (filtered):  2,174,052,769 bp (78.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14450091 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 12.4%\n",
      "  G: 24.6%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2377683\t4622580.5\t0\t2377683\n",
      "2\t369953\t1155645.1\t0\t369953\n",
      "3\t212079\t288911.3\t0\t212079\n",
      "4\t120512\t72227.8\t0\t120512\n",
      "5\t108962\t18057.0\t0\t108962\n",
      "6\t97421\t4514.2\t0\t97421\n",
      "7\t79386\t1128.6\t0\t79386\n",
      "8\t113054\t282.1\t0\t113054\n",
      "9\t87971\t70.5\t0\t87624 347\n",
      "10\t96473\t17.6\t1\t89879 6594\n",
      "11\t101734\t4.4\t1\t93536 8198\n",
      "12\t102089\t1.1\t1\t93747 8342\n",
      "13\t117081\t0.3\t1\t107549 9532\n",
      "14\t108685\t0.3\t1\t98929 9756\n",
      "15\t89117\t0.3\t1\t82133 6984\n",
      "16\t127440\t0.3\t1\t117058 10382\n",
      "17\t108227\t0.3\t1\t99362 8865\n",
      "18\t94672\t0.3\t1\t87210 7462\n",
      "19\t124816\t0.3\t1\t114994 9822\n",
      "20\t116877\t0.3\t1\t108056 8821\n",
      "21\t101808\t0.3\t1\t94028 7780\n",
      "22\t115329\t0.3\t1\t106403 8926\n",
      "23\t122296\t0.3\t1\t112903 9393\n",
      "24\t121720\t0.3\t1\t111954 9766\n",
      "25\t136356\t0.3\t1\t126910 9446\n",
      "26\t113979\t0.3\t1\t104759 9220\n",
      "27\t127054\t0.3\t1\t116622 10432\n",
      "28\t113808\t0.3\t1\t106074 7734\n",
      "29\t126916\t0.3\t1\t117431 9485\n",
      "30\t122281\t0.3\t1\t113729 8552\n",
      "31\t135760\t0.3\t1\t126137 9623\n",
      "32\t117176\t0.3\t1\t109066 8110\n",
      "33\t136110\t0.3\t1\t125744 10366\n",
      "34\t171537\t0.3\t1\t159519 12018\n",
      "35\t95945\t0.3\t1\t89044 6901\n",
      "36\t137046\t0.3\t1\t126917 10129\n",
      "37\t143037\t0.3\t1\t132698 10339\n",
      "38\t133329\t0.3\t1\t124497 8832\n",
      "39\t127534\t0.3\t1\t119434 8100\n",
      "40\t144687\t0.3\t1\t134651 10036\n",
      "41\t140384\t0.3\t1\t131070 9314\n",
      "42\t144299\t0.3\t1\t134780 9519\n",
      "43\t129916\t0.3\t1\t121294 8622\n",
      "44\t151641\t0.3\t1\t141122 10519\n",
      "45\t160400\t0.3\t1\t149699 10701\n",
      "46\t114273\t0.3\t1\t106605 7668\n",
      "47\t144997\t0.3\t1\t136130 8867\n",
      "48\t143166\t0.3\t1\t134124 9042\n",
      "49\t148482\t0.3\t1\t138822 9660\n",
      "50\t140955\t0.3\t1\t131828 9127\n",
      "51\t174978\t0.3\t1\t163816 11162\n",
      "52\t126038\t0.3\t1\t118003 8035\n",
      "53\t153652\t0.3\t1\t144809 8843\n",
      "54\t111499\t0.3\t1\t104710 6789\n",
      "55\t148731\t0.3\t1\t140120 8611\n",
      "56\t152852\t0.3\t1\t143418 9434\n",
      "57\t145621\t0.3\t1\t137708 7913\n",
      "58\t139629\t0.3\t1\t131703 7926\n",
      "59\t139468\t0.3\t1\t131381 8087\n",
      "60\t145979\t0.3\t1\t137374 8605\n",
      "61\t147098\t0.3\t1\t138678 8420\n",
      "62\t160909\t0.3\t1\t151040 9869\n",
      "63\t208484\t0.3\t1\t197875 10609\n",
      "64\t163339\t0.3\t1\t155950 7389\n",
      "65\t115211\t0.3\t1\t108911 6300\n",
      "66\t124628\t0.3\t1\t117681 6947\n",
      "67\t122836\t0.3\t1\t115645 7191\n",
      "68\t124659\t0.3\t1\t117627 7032\n",
      "69\t121178\t0.3\t1\t114114 7064\n",
      "70\t126813\t0.3\t1\t119514 7299\n",
      "71\t132039\t0.3\t1\t124540 7499\n",
      "72\t125242\t0.3\t1\t118005 7237\n",
      "73\t130262\t0.3\t1\t123006 7256\n",
      "74\t119815\t0.3\t1\t112862 6953\n",
      "75\t114051\t0.3\t1\t107533 6518\n",
      "76\t111158\t0.3\t1\t104747 6411\n",
      "77\t111769\t0.3\t1\t105274 6495\n",
      "78\t104990\t0.3\t1\t98882 6108\n",
      "79\t101071\t0.3\t1\t95399 5672\n",
      "80\t100953\t0.3\t1\t95281 5672\n",
      "81\t99176\t0.3\t1\t93590 5586\n",
      "82\t94350\t0.3\t1\t89142 5208\n",
      "83\t91413\t0.3\t1\t86366 5047\n",
      "84\t87258\t0.3\t1\t82554 4704\n",
      "85\t85916\t0.3\t1\t81265 4651\n",
      "86\t84604\t0.3\t1\t80107 4497\n",
      "87\t84396\t0.3\t1\t79885 4511\n",
      "88\t82425\t0.3\t1\t78207 4218\n",
      "89\t73764\t0.3\t1\t69903 3861\n",
      "90\t71339\t0.3\t1\t67438 3901\n",
      "91\t67256\t0.3\t1\t63682 3574\n",
      "92\t62204\t0.3\t1\t58897 3307\n",
      "93\t60201\t0.3\t1\t57003 3198\n",
      "94\t58001\t0.3\t1\t54921 3080\n",
      "95\t54924\t0.3\t1\t52030 2894\n",
      "96\t52451\t0.3\t1\t49666 2785\n",
      "97\t48389\t0.3\t1\t45913 2476\n",
      "98\t44117\t0.3\t1\t41843 2274\n",
      "99\t42086\t0.3\t1\t39795 2291\n",
      "100\t38235\t0.3\t1\t36240 1995\n",
      "101\t35522\t0.3\t1\t33621 1901\n",
      "102\t32226\t0.3\t1\t30512 1714\n",
      "103\t29643\t0.3\t1\t27969 1674\n",
      "104\t28709\t0.3\t1\t27141 1568\n",
      "105\t25222\t0.3\t1\t23921 1301\n",
      "106\t23132\t0.3\t1\t21958 1174\n",
      "107\t20950\t0.3\t1\t19852 1098\n",
      "108\t19130\t0.3\t1\t18127 1003\n",
      "109\t16951\t0.3\t1\t16010 941\n",
      "110\t15532\t0.3\t1\t14697 835\n",
      "111\t14263\t0.3\t1\t13491 772\n",
      "112\t12448\t0.3\t1\t11737 711\n",
      "113\t11105\t0.3\t1\t10532 573\n",
      "114\t9378\t0.3\t1\t8878 500\n",
      "115\t8004\t0.3\t1\t7541 463\n",
      "116\t7028\t0.3\t1\t6643 385\n",
      "117\t5705\t0.3\t1\t5388 317\n",
      "118\t5009\t0.3\t1\t4724 285\n",
      "119\t4196\t0.3\t1\t3948 248\n",
      "120\t3592\t0.3\t1\t3360 232\n",
      "121\t3098\t0.3\t1\t2900 198\n",
      "122\t2572\t0.3\t1\t2409 163\n",
      "123\t2164\t0.3\t1\t2037 127\n",
      "124\t1827\t0.3\t1\t1705 122\n",
      "125\t1543\t0.3\t1\t1435 108\n",
      "126\t1244\t0.3\t1\t1160 84\n",
      "127\t1099\t0.3\t1\t1018 81\n",
      "128\t829\t0.3\t1\t778 51\n",
      "129\t718\t0.3\t1\t677 41\n",
      "130\t539\t0.3\t1\t503 36\n",
      "131\t460\t0.3\t1\t425 35\n",
      "132\t375\t0.3\t1\t343 32\n",
      "133\t337\t0.3\t1\t318 19\n",
      "134\t432\t0.3\t1\t410 22\n",
      "135\t267\t0.3\t1\t250 17\n",
      "136\t145\t0.3\t1\t136 9\n",
      "137\t88\t0.3\t1\t77 11\n",
      "138\t77\t0.3\t1\t69 8\n",
      "139\t43\t0.3\t1\t40 3\n",
      "140\t29\t0.3\t1\t26 3\n",
      "141\t28\t0.3\t1\t23 5\n",
      "142\t25\t0.3\t1\t23 2\n",
      "143\t30\t0.3\t1\t30\n",
      "144\t16\t0.3\t1\t15 1\n",
      "145\t18\t0.3\t1\t15 3\n",
      "146\t29\t0.3\t1\t26 3\n",
      "147\t70\t0.3\t1\t66 4\n",
      "148\t40\t0.3\t1\t37 3\n",
      "149\t46\t0.3\t1\t43 3\n",
      "150\t248\t0.3\t1\t232 16\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18490322 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t19007 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 497.62 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,433,110\n",
      "Reads with adapters:                11,700,096 (87.1%)\n",
      "Reads written (passing filters):    13,433,110 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,014,966,500 bp\n",
      "Quality-trimmed:               3,988,642 bp (0.2%)\n",
      "Total written (filtered):  1,426,352,975 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11700096 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 12.5%\n",
      "  G: 23.6%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t952009\t3358277.5\t0\t952009\n",
      "2\t175701\t839569.4\t0\t175701\n",
      "3\t106315\t209892.3\t0\t106315\n",
      "4\t68924\t52473.1\t0\t68924\n",
      "5\t62282\t13118.3\t0\t62282\n",
      "6\t61633\t3279.6\t0\t61633\n",
      "7\t52554\t819.9\t0\t52554\n",
      "8\t65132\t205.0\t0\t65132\n",
      "9\t65306\t51.2\t0\t64521 785\n",
      "10\t61900\t12.8\t1\t58796 3104\n",
      "11\t68989\t3.2\t1\t63880 5109\n",
      "12\t67245\t0.8\t1\t63248 3997\n",
      "13\t69213\t0.2\t1\t64859 4354\n",
      "14\t71768\t0.2\t1\t66846 4922\n",
      "15\t75062\t0.2\t1\t70553 4509\n",
      "16\t74514\t0.2\t1\t69133 5381\n",
      "17\t90491\t0.2\t1\t83695 6796\n",
      "18\t77417\t0.2\t1\t72980 4437\n",
      "19\t68949\t0.2\t1\t64670 4279\n",
      "20\t81015\t0.2\t1\t76032 4983\n",
      "21\t89626\t0.2\t1\t83646 5980\n",
      "22\t77941\t0.2\t1\t73613 4328\n",
      "23\t90688\t0.2\t1\t84959 5729\n",
      "24\t86332\t0.2\t1\t79509 6823\n",
      "25\t91755\t0.2\t1\t85888 5867\n",
      "26\t86085\t0.2\t1\t81297 4788\n",
      "27\t95329\t0.2\t1\t88851 6478\n",
      "28\t95528\t0.2\t1\t89551 5977\n",
      "29\t99349\t0.2\t1\t92034 7315\n",
      "30\t106567\t0.2\t1\t101023 5544\n",
      "31\t85587\t0.2\t1\t80666 4921\n",
      "32\t103697\t0.2\t1\t97691 6006\n",
      "33\t99669\t0.2\t1\t93324 6345\n",
      "34\t119451\t0.2\t1\t111551 7900\n",
      "35\t94360\t0.2\t1\t88955 5405\n",
      "36\t108596\t0.2\t1\t102097 6499\n",
      "37\t113594\t0.2\t1\t106631 6963\n",
      "38\t108790\t0.2\t1\t101383 7407\n",
      "39\t126706\t0.2\t1\t119375 7331\n",
      "40\t103711\t0.2\t1\t97817 5894\n",
      "41\t130432\t0.2\t1\t122153 8279\n",
      "42\t110139\t0.2\t1\t101991 8148\n",
      "43\t183261\t0.2\t1\t174857 8404\n",
      "44\t57830\t0.2\t1\t53254 4576\n",
      "45\t123796\t0.2\t1\t116975 6821\n",
      "46\t119988\t0.2\t1\t113024 6964\n",
      "47\t130141\t0.2\t1\t122976 7165\n",
      "48\t120988\t0.2\t1\t113912 7076\n",
      "49\t138463\t0.2\t1\t129394 9069\n",
      "50\t122247\t0.2\t1\t115770 6477\n",
      "51\t132587\t0.2\t1\t124814 7773\n",
      "52\t139338\t0.2\t1\t129873 9465\n",
      "53\t153954\t0.2\t1\t145822 8132\n",
      "54\t118275\t0.2\t1\t111358 6917\n",
      "55\t139368\t0.2\t1\t131390 7978\n",
      "56\t164199\t0.2\t1\t155690 8509\n",
      "57\t126552\t0.2\t1\t118427 8125\n",
      "58\t144214\t0.2\t1\t137457 6757\n",
      "59\t116278\t0.2\t1\t109533 6745\n",
      "60\t144971\t0.2\t1\t136747 8224\n",
      "61\t139800\t0.2\t1\t132274 7526\n",
      "62\t147932\t0.2\t1\t140286 7646\n",
      "63\t129438\t0.2\t1\t121520 7918\n",
      "64\t136148\t0.2\t1\t129121 7027\n",
      "65\t148136\t0.2\t1\t139981 8155\n",
      "66\t158867\t0.2\t1\t150035 8832\n",
      "67\t172296\t0.2\t1\t160373 11923\n",
      "68\t325502\t0.2\t1\t311428 14074\n",
      "69\t62909\t0.2\t1\t58208 4701\n",
      "70\t23526\t0.2\t1\t20186 3340\n",
      "71\t82740\t0.2\t1\t78158 4582\n",
      "72\t124178\t0.2\t1\t117785 6393\n",
      "73\t132921\t0.2\t1\t125940 6981\n",
      "74\t131071\t0.2\t1\t124071 7000\n",
      "75\t128974\t0.2\t1\t122028 6946\n",
      "76\t128069\t0.2\t1\t121139 6930\n",
      "77\t126522\t0.2\t1\t119822 6700\n",
      "78\t120502\t0.2\t1\t113788 6714\n",
      "79\t118821\t0.2\t1\t112166 6655\n",
      "80\t115866\t0.2\t1\t109547 6319\n",
      "81\t115264\t0.2\t1\t108988 6276\n",
      "82\t110755\t0.2\t1\t104576 6179\n",
      "83\t107566\t0.2\t1\t101669 5897\n",
      "84\t105597\t0.2\t1\t99846 5751\n",
      "85\t102751\t0.2\t1\t97220 5531\n",
      "86\t97615\t0.2\t1\t92219 5396\n",
      "87\t92989\t0.2\t1\t87786 5203\n",
      "88\t92437\t0.2\t1\t87288 5149\n",
      "89\t88099\t0.2\t1\t83258 4841\n",
      "90\t86764\t0.2\t1\t81847 4917\n",
      "91\t81829\t0.2\t1\t77279 4550\n",
      "92\t77590\t0.2\t1\t73265 4325\n",
      "93\t73175\t0.2\t1\t69095 4080\n",
      "94\t69476\t0.2\t1\t65477 3999\n",
      "95\t65431\t0.2\t1\t61622 3809\n",
      "96\t61527\t0.2\t1\t57946 3581\n",
      "97\t57358\t0.2\t1\t54065 3293\n",
      "98\t53633\t0.2\t1\t50498 3135\n",
      "99\t51351\t0.2\t1\t48285 3066\n",
      "100\t46130\t0.2\t1\t43457 2673\n",
      "101\t43563\t0.2\t1\t40904 2659\n",
      "102\t39390\t0.2\t1\t37016 2374\n",
      "103\t35867\t0.2\t1\t33682 2185\n",
      "104\t32606\t0.2\t1\t30634 1972\n",
      "105\t28646\t0.2\t1\t26808 1838\n",
      "106\t26227\t0.2\t1\t24640 1587\n",
      "107\t23172\t0.2\t1\t21743 1429\n",
      "108\t21298\t0.2\t1\t19891 1407\n",
      "109\t18913\t0.2\t1\t17580 1333\n",
      "110\t16998\t0.2\t1\t15871 1127\n",
      "111\t15038\t0.2\t1\t14044 994\n",
      "112\t12635\t0.2\t1\t11815 820\n",
      "113\t11352\t0.2\t1\t10604 748\n",
      "114\t9838\t0.2\t1\t9171 667\n",
      "115\t8151\t0.2\t1\t7529 622\n",
      "116\t7025\t0.2\t1\t6508 517\n",
      "117\t5831\t0.2\t1\t5406 425\n",
      "118\t5154\t0.2\t1\t4783 371\n",
      "119\t4064\t0.2\t1\t3743 321\n",
      "120\t3640\t0.2\t1\t3365 275\n",
      "121\t2769\t0.2\t1\t2540 229\n",
      "122\t2341\t0.2\t1\t2118 223\n",
      "123\t2056\t0.2\t1\t1883 173\n",
      "124\t1687\t0.2\t1\t1543 144\n",
      "125\t1340\t0.2\t1\t1233 107\n",
      "126\t1317\t0.2\t1\t1201 116\n",
      "127\t1059\t0.2\t1\t975 84\n",
      "128\t810\t0.2\t1\t740 70\n",
      "129\t681\t0.2\t1\t600 81\n",
      "130\t748\t0.2\t1\t679 69\n",
      "131\t837\t0.2\t1\t795 42\n",
      "132\t954\t0.2\t1\t903 51\n",
      "133\t1339\t0.2\t1\t1266 73\n",
      "134\t1750\t0.2\t1\t1667 83\n",
      "135\t759\t0.2\t1\t718 41\n",
      "136\t229\t0.2\t1\t209 20\n",
      "137\t121\t0.2\t1\t104 17\n",
      "138\t84\t0.2\t1\t73 11\n",
      "139\t75\t0.2\t1\t63 12\n",
      "140\t73\t0.2\t1\t58 15\n",
      "141\t35\t0.2\t1\t32 3\n",
      "142\t38\t0.2\t1\t35 3\n",
      "143\t19\t0.2\t1\t15 4\n",
      "144\t19\t0.2\t1\t11 8\n",
      "145\t22\t0.2\t1\t21 1\n",
      "146\t64\t0.2\t1\t59 5\n",
      "147\t278\t0.2\t1\t250 28\n",
      "148\t168\t0.2\t1\t142 26\n",
      "149\t83\t0.2\t1\t70 13\n",
      "150\t577\t0.2\t1\t506 71\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13433110 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7588 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 516.35 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,433,110\n",
      "Reads with adapters:                11,642,556 (86.7%)\n",
      "Reads written (passing filters):    13,433,110 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,014,966,500 bp\n",
      "Quality-trimmed:               8,709,221 bp (0.4%)\n",
      "Total written (filtered):  1,432,856,177 bp (71.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11642556 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 11.7%\n",
      "  G: 26.2%\n",
      "  T: 34.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1015848\t3358277.5\t0\t1015848\n",
      "2\t179924\t839569.4\t0\t179924\n",
      "3\t110740\t209892.3\t0\t110740\n",
      "4\t68288\t52473.1\t0\t68288\n",
      "5\t68265\t13118.3\t0\t68265\n",
      "6\t61219\t3279.6\t0\t61219\n",
      "7\t51190\t819.9\t0\t51190\n",
      "8\t71859\t205.0\t0\t71859\n",
      "9\t56699\t51.2\t0\t56454 245\n",
      "10\t63956\t12.8\t1\t61051 2905\n",
      "11\t67245\t3.2\t1\t63190 4055\n",
      "12\t68588\t0.8\t1\t64563 4025\n",
      "13\t77409\t0.2\t1\t73173 4236\n",
      "14\t74943\t0.2\t1\t70185 4758\n",
      "15\t61575\t0.2\t1\t58112 3463\n",
      "16\t87672\t0.2\t1\t82660 5012\n",
      "17\t73552\t0.2\t1\t68954 4598\n",
      "18\t68802\t0.2\t1\t65063 3739\n",
      "19\t88259\t0.2\t1\t82939 5320\n",
      "20\t82556\t0.2\t1\t78141 4415\n",
      "21\t73225\t0.2\t1\t69045 4180\n",
      "22\t83863\t0.2\t1\t79245 4618\n",
      "23\t87811\t0.2\t1\t82813 4998\n",
      "24\t89014\t0.2\t1\t83968 5046\n",
      "25\t100238\t0.2\t1\t94973 5265\n",
      "26\t83574\t0.2\t1\t78786 4788\n",
      "27\t93033\t0.2\t1\t87412 5621\n",
      "28\t89387\t0.2\t1\t85387 4000\n",
      "29\t95696\t0.2\t1\t90729 4967\n",
      "30\t94787\t0.2\t1\t90441 4346\n",
      "31\t102169\t0.2\t1\t97061 5108\n",
      "32\t94007\t0.2\t1\t89720 4287\n",
      "33\t105833\t0.2\t1\t100418 5415\n",
      "34\t134739\t0.2\t1\t127847 6892\n",
      "35\t80075\t0.2\t1\t76383 3692\n",
      "36\t104516\t0.2\t1\t98895 5621\n",
      "37\t113410\t0.2\t1\t107671 5739\n",
      "38\t107520\t0.2\t1\t102889 4631\n",
      "39\t105637\t0.2\t1\t101025 4612\n",
      "40\t118785\t0.2\t1\t112992 5793\n",
      "41\t115631\t0.2\t1\t110394 5237\n",
      "42\t122110\t0.2\t1\t116550 5560\n",
      "43\t111230\t0.2\t1\t106271 4959\n",
      "44\t128541\t0.2\t1\t122788 5753\n",
      "45\t137963\t0.2\t1\t131813 6150\n",
      "46\t99340\t0.2\t1\t94897 4443\n",
      "47\t125922\t0.2\t1\t120367 5555\n",
      "48\t125595\t0.2\t1\t120423 5172\n",
      "49\t132842\t0.2\t1\t126935 5907\n",
      "50\t127676\t0.2\t1\t122004 5672\n",
      "51\t163515\t0.2\t1\t156244 7271\n",
      "52\t117884\t0.2\t1\t113079 4805\n",
      "53\t142736\t0.2\t1\t137315 5421\n",
      "54\t107101\t0.2\t1\t102800 4301\n",
      "55\t139222\t0.2\t1\t133903 5319\n",
      "56\t143656\t0.2\t1\t137859 5797\n",
      "57\t140073\t0.2\t1\t134871 5202\n",
      "58\t134625\t0.2\t1\t129700 4925\n",
      "59\t134950\t0.2\t1\t129802 5148\n",
      "60\t140295\t0.2\t1\t134781 5514\n",
      "61\t144854\t0.2\t1\t139165 5689\n",
      "62\t159948\t0.2\t1\t153492 6456\n",
      "63\t215119\t0.2\t1\t207339 7780\n",
      "64\t166765\t0.2\t1\t161464 5301\n",
      "65\t114876\t0.2\t1\t110601 4275\n",
      "66\t125878\t0.2\t1\t121273 4605\n",
      "67\t126455\t0.2\t1\t121514 4941\n",
      "68\t128071\t0.2\t1\t123304 4767\n",
      "69\t126908\t0.2\t1\t122039 4869\n",
      "70\t132340\t0.2\t1\t127378 4962\n",
      "71\t137419\t0.2\t1\t132563 4856\n",
      "72\t133254\t0.2\t1\t128394 4860\n",
      "73\t140768\t0.2\t1\t135639 5129\n",
      "74\t130186\t0.2\t1\t125634 4552\n",
      "75\t123476\t0.2\t1\t119176 4300\n",
      "76\t121581\t0.2\t1\t117266 4315\n",
      "77\t120182\t0.2\t1\t115605 4577\n",
      "78\t115742\t0.2\t1\t111521 4221\n",
      "79\t111421\t0.2\t1\t107330 4091\n",
      "80\t110726\t0.2\t1\t106773 3953\n",
      "81\t109599\t0.2\t1\t105677 3922\n",
      "82\t103036\t0.2\t1\t99365 3671\n",
      "83\t100716\t0.2\t1\t97173 3543\n",
      "84\t98439\t0.2\t1\t94811 3628\n",
      "85\t96847\t0.2\t1\t93310 3537\n",
      "86\t96817\t0.2\t1\t93338 3479\n",
      "87\t95905\t0.2\t1\t92582 3323\n",
      "88\t93526\t0.2\t1\t90163 3363\n",
      "89\t83831\t0.2\t1\t80785 3046\n",
      "90\t80158\t0.2\t1\t77319 2839\n",
      "91\t74964\t0.2\t1\t72203 2761\n",
      "92\t70024\t0.2\t1\t67553 2471\n",
      "93\t68027\t0.2\t1\t65623 2404\n",
      "94\t64372\t0.2\t1\t62143 2229\n",
      "95\t61456\t0.2\t1\t59284 2172\n",
      "96\t57585\t0.2\t1\t55560 2025\n",
      "97\t53463\t0.2\t1\t51605 1858\n",
      "98\t50229\t0.2\t1\t48488 1741\n",
      "99\t48358\t0.2\t1\t46710 1648\n",
      "100\t43743\t0.2\t1\t42301 1442\n",
      "101\t39334\t0.2\t1\t37976 1358\n",
      "102\t35681\t0.2\t1\t34444 1237\n",
      "103\t32169\t0.2\t1\t31042 1127\n",
      "104\t30344\t0.2\t1\t29354 990\n",
      "105\t26673\t0.2\t1\t25747 926\n",
      "106\t24610\t0.2\t1\t23753 857\n",
      "107\t21493\t0.2\t1\t20733 760\n",
      "108\t19129\t0.2\t1\t18449 680\n",
      "109\t17216\t0.2\t1\t16604 612\n",
      "110\t15680\t0.2\t1\t15121 559\n",
      "111\t14422\t0.2\t1\t13916 506\n",
      "112\t12048\t0.2\t1\t11652 396\n",
      "113\t10682\t0.2\t1\t10293 389\n",
      "114\t9191\t0.2\t1\t8830 361\n",
      "115\t7355\t0.2\t1\t7077 278\n",
      "116\t6245\t0.2\t1\t6016 229\n",
      "117\t5167\t0.2\t1\t4936 231\n",
      "118\t4511\t0.2\t1\t4330 181\n",
      "119\t3552\t0.2\t1\t3385 167\n",
      "120\t3201\t0.2\t1\t3062 139\n",
      "121\t2450\t0.2\t1\t2343 107\n",
      "122\t2029\t0.2\t1\t1935 94\n",
      "123\t1814\t0.2\t1\t1715 99\n",
      "124\t1524\t0.2\t1\t1436 88\n",
      "125\t1206\t0.2\t1\t1139 67\n",
      "126\t1198\t0.2\t1\t1135 63\n",
      "127\t925\t0.2\t1\t875 50\n",
      "128\t706\t0.2\t1\t663 43\n",
      "129\t633\t0.2\t1\t590 43\n",
      "130\t698\t0.2\t1\t666 32\n",
      "131\t844\t0.2\t1\t815 29\n",
      "132\t910\t0.2\t1\t876 34\n",
      "133\t1412\t0.2\t1\t1368 44\n",
      "134\t1698\t0.2\t1\t1641 57\n",
      "135\t718\t0.2\t1\t696 22\n",
      "136\t190\t0.2\t1\t177 13\n",
      "137\t107\t0.2\t1\t98 9\n",
      "138\t70\t0.2\t1\t67 3\n",
      "139\t53\t0.2\t1\t50 3\n",
      "140\t53\t0.2\t1\t52 1\n",
      "141\t26\t0.2\t1\t26\n",
      "142\t26\t0.2\t1\t24 2\n",
      "143\t21\t0.2\t1\t20 1\n",
      "144\t18\t0.2\t1\t16 2\n",
      "145\t35\t0.2\t1\t32 3\n",
      "146\t39\t0.2\t1\t35 4\n",
      "147\t177\t0.2\t1\t169 8\n",
      "148\t82\t0.2\t1\t76 6\n",
      "149\t45\t0.2\t1\t43 2\n",
      "150\t267\t0.2\t1\t241 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13433110 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t18598 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 505.76 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,408,232\n",
      "Reads with adapters:                11,662,676 (87.0%)\n",
      "Reads written (passing filters):    13,408,232 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,011,234,800 bp\n",
      "Quality-trimmed:               4,075,788 bp (0.2%)\n",
      "Total written (filtered):  1,423,488,059 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11662676 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 12.5%\n",
      "  G: 23.6%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t950454\t3352058.0\t0\t950454\n",
      "2\t175107\t838014.5\t0\t175107\n",
      "3\t106043\t209503.6\t0\t106043\n",
      "4\t68134\t52375.9\t0\t68134\n",
      "5\t61886\t13094.0\t0\t61886\n",
      "6\t61175\t3273.5\t0\t61175\n",
      "7\t51159\t818.4\t0\t51159\n",
      "8\t64511\t204.6\t0\t64511\n",
      "9\t64557\t51.1\t0\t63785 772\n",
      "10\t60792\t12.8\t1\t57082 3710\n",
      "11\t68970\t3.2\t1\t63184 5786\n",
      "12\t66553\t0.8\t1\t61883 4670\n",
      "13\t68274\t0.2\t1\t63204 5070\n",
      "14\t71364\t0.2\t1\t65643 5721\n",
      "15\t74174\t0.2\t1\t68692 5482\n",
      "16\t73515\t0.2\t1\t67346 6169\n",
      "17\t90659\t0.2\t1\t82653 8006\n",
      "18\t76471\t0.2\t1\t71237 5234\n",
      "19\t67430\t0.2\t1\t62447 4983\n",
      "20\t80487\t0.2\t1\t74742 5745\n",
      "21\t89676\t0.2\t1\t82775 6901\n",
      "22\t76355\t0.2\t1\t71205 5150\n",
      "23\t90468\t0.2\t1\t83777 6691\n",
      "24\t85067\t0.2\t1\t77644 7423\n",
      "25\t91076\t0.2\t1\t84134 6942\n",
      "26\t84890\t0.2\t1\t79317 5573\n",
      "27\t95709\t0.2\t1\t88308 7401\n",
      "28\t95026\t0.2\t1\t88099 6927\n",
      "29\t98118\t0.2\t1\t89948 8170\n",
      "30\t107848\t0.2\t1\t101119 6729\n",
      "31\t84062\t0.2\t1\t78313 5749\n",
      "32\t103994\t0.2\t1\t96660 7334\n",
      "33\t98133\t0.2\t1\t90574 7559\n",
      "34\t120505\t0.2\t1\t111057 9448\n",
      "35\t94055\t0.2\t1\t87482 6573\n",
      "36\t107881\t0.2\t1\t100206 7675\n",
      "37\t113227\t0.2\t1\t104836 8391\n",
      "38\t107162\t0.2\t1\t98615 8547\n",
      "39\t126650\t0.2\t1\t117366 9284\n",
      "40\t101784\t0.2\t1\t94417 7367\n",
      "41\t130084\t0.2\t1\t120107 9977\n",
      "42\t108652\t0.2\t1\t99173 9479\n",
      "43\t179621\t0.2\t1\t169283 10338\n",
      "44\t59500\t0.2\t1\t54067 5433\n",
      "45\t124819\t0.2\t1\t116443 8376\n",
      "46\t119134\t0.2\t1\t110810 8324\n",
      "47\t129268\t0.2\t1\t120606 8662\n",
      "48\t119694\t0.2\t1\t111395 8299\n",
      "49\t138243\t0.2\t1\t127784 10459\n",
      "50\t121057\t0.2\t1\t113597 7460\n",
      "51\t131350\t0.2\t1\t122521 8829\n",
      "52\t138351\t0.2\t1\t127802 10549\n",
      "53\t154024\t0.2\t1\t144429 9595\n",
      "54\t116951\t0.2\t1\t108870 8081\n",
      "55\t139783\t0.2\t1\t130545 9238\n",
      "56\t165267\t0.2\t1\t155186 10081\n",
      "57\t126758\t0.2\t1\t117494 9264\n",
      "58\t142781\t0.2\t1\t134977 7804\n",
      "59\t117589\t0.2\t1\t109427 8162\n",
      "60\t144831\t0.2\t1\t135423 9408\n",
      "61\t138318\t0.2\t1\t129552 8766\n",
      "62\t148945\t0.2\t1\t139636 9309\n",
      "63\t129691\t0.2\t1\t120718 8973\n",
      "64\t134879\t0.2\t1\t126688 8191\n",
      "65\t146410\t0.2\t1\t137112 9298\n",
      "66\t156960\t0.2\t1\t146744 10216\n",
      "67\t171283\t0.2\t1\t158133 13150\n",
      "68\t312723\t0.2\t1\t296738 15985\n",
      "69\t76525\t0.2\t1\t70767 5758\n",
      "70\t28798\t0.2\t1\t24712 4086\n",
      "71\t79564\t0.2\t1\t74491 5073\n",
      "72\t122325\t0.2\t1\t115172 7153\n",
      "73\t132393\t0.2\t1\t124378 8015\n",
      "74\t131213\t0.2\t1\t123087 8126\n",
      "75\t128592\t0.2\t1\t120236 8356\n",
      "76\t128119\t0.2\t1\t119430 8689\n",
      "77\t126752\t0.2\t1\t118441 8311\n",
      "78\t121397\t0.2\t1\t113322 8075\n",
      "79\t118367\t0.2\t1\t110488 7879\n",
      "80\t116333\t0.2\t1\t108557 7776\n",
      "81\t115083\t0.2\t1\t107691 7392\n",
      "82\t111285\t0.2\t1\t103893 7392\n",
      "83\t107858\t0.2\t1\t100817 7041\n",
      "84\t105350\t0.2\t1\t98390 6960\n",
      "85\t102602\t0.2\t1\t95870 6732\n",
      "86\t97359\t0.2\t1\t90922 6437\n",
      "87\t94015\t0.2\t1\t87784 6231\n",
      "88\t92275\t0.2\t1\t86366 5909\n",
      "89\t87740\t0.2\t1\t82108 5632\n",
      "90\t86642\t0.2\t1\t80943 5699\n",
      "91\t81958\t0.2\t1\t76615 5343\n",
      "92\t78763\t0.2\t1\t73589 5174\n",
      "93\t72932\t0.2\t1\t68198 4734\n",
      "94\t70403\t0.2\t1\t65851 4552\n",
      "95\t65162\t0.2\t1\t60795 4367\n",
      "96\t61147\t0.2\t1\t56990 4157\n",
      "97\t57845\t0.2\t1\t54051 3794\n",
      "98\t54088\t0.2\t1\t50353 3735\n",
      "99\t50912\t0.2\t1\t47388 3524\n",
      "100\t46451\t0.2\t1\t43306 3145\n",
      "101\t43581\t0.2\t1\t40541 3040\n",
      "102\t39156\t0.2\t1\t36546 2610\n",
      "103\t35574\t0.2\t1\t33122 2452\n",
      "104\t32789\t0.2\t1\t30678 2111\n",
      "105\t29034\t0.2\t1\t27055 1979\n",
      "106\t26298\t0.2\t1\t24343 1955\n",
      "107\t23552\t0.2\t1\t21942 1610\n",
      "108\t21227\t0.2\t1\t19666 1561\n",
      "109\t18723\t0.2\t1\t17348 1375\n",
      "110\t16933\t0.2\t1\t15728 1205\n",
      "111\t14813\t0.2\t1\t13715 1098\n",
      "112\t12788\t0.2\t1\t11763 1025\n",
      "113\t11310\t0.2\t1\t10437 873\n",
      "114\t9920\t0.2\t1\t9142 778\n",
      "115\t8131\t0.2\t1\t7451 680\n",
      "116\t6933\t0.2\t1\t6415 518\n",
      "117\t5884\t0.2\t1\t5388 496\n",
      "118\t5136\t0.2\t1\t4702 434\n",
      "119\t4097\t0.2\t1\t3751 346\n",
      "120\t3546\t0.2\t1\t3238 308\n",
      "121\t2947\t0.2\t1\t2707 240\n",
      "122\t2422\t0.2\t1\t2210 212\n",
      "123\t1924\t0.2\t1\t1761 163\n",
      "124\t1733\t0.2\t1\t1576 157\n",
      "125\t1418\t0.2\t1\t1276 142\n",
      "126\t1305\t0.2\t1\t1178 127\n",
      "127\t1104\t0.2\t1\t1004 100\n",
      "128\t823\t0.2\t1\t732 91\n",
      "129\t659\t0.2\t1\t593 66\n",
      "130\t710\t0.2\t1\t634 76\n",
      "131\t875\t0.2\t1\t812 63\n",
      "132\t926\t0.2\t1\t862 64\n",
      "133\t1378\t0.2\t1\t1297 81\n",
      "134\t1693\t0.2\t1\t1594 99\n",
      "135\t721\t0.2\t1\t691 30\n",
      "136\t229\t0.2\t1\t205 24\n",
      "137\t172\t0.2\t1\t157 15\n",
      "138\t80\t0.2\t1\t66 14\n",
      "139\t66\t0.2\t1\t55 11\n",
      "140\t62\t0.2\t1\t53 9\n",
      "141\t37\t0.2\t1\t35 2\n",
      "142\t33\t0.2\t1\t27 6\n",
      "143\t21\t0.2\t1\t19 2\n",
      "144\t18\t0.2\t1\t17 1\n",
      "145\t32\t0.2\t1\t28 4\n",
      "146\t69\t0.2\t1\t67 2\n",
      "147\t283\t0.2\t1\t250 33\n",
      "148\t183\t0.2\t1\t166 17\n",
      "149\t88\t0.2\t1\t79 9\n",
      "150\t625\t0.2\t1\t537 88\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13408232 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7650 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 526.48 s (39 us/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,408,232\n",
      "Reads with adapters:                11,603,960 (86.5%)\n",
      "Reads written (passing filters):    13,408,232 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,011,234,800 bp\n",
      "Quality-trimmed:               8,433,724 bp (0.4%)\n",
      "Total written (filtered):  1,430,263,611 bp (71.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11603960 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 11.7%\n",
      "  G: 26.2%\n",
      "  T: 34.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1017131\t3352058.0\t0\t1017131\n",
      "2\t177058\t838014.5\t0\t177058\n",
      "3\t112787\t209503.6\t0\t112787\n",
      "4\t67003\t52375.9\t0\t67003\n",
      "5\t66504\t13094.0\t0\t66504\n",
      "6\t60966\t3273.5\t0\t60966\n",
      "7\t50684\t818.4\t0\t50684\n",
      "8\t71309\t204.6\t0\t71309\n",
      "9\t56145\t51.1\t0\t55917 228\n",
      "10\t61633\t12.8\t1\t57525 4108\n",
      "11\t66766\t3.2\t1\t61633 5133\n",
      "12\t66970\t0.8\t1\t61817 5153\n",
      "13\t77136\t0.2\t1\t71259 5877\n",
      "14\t74020\t0.2\t1\t67910 6110\n",
      "15\t60571\t0.2\t1\t56229 4342\n",
      "16\t86270\t0.2\t1\t79871 6399\n",
      "17\t75982\t0.2\t1\t70248 5734\n",
      "18\t65820\t0.2\t1\t61090 4730\n",
      "19\t87582\t0.2\t1\t81169 6413\n",
      "20\t81347\t0.2\t1\t75534 5813\n",
      "21\t71738\t0.2\t1\t66607 5131\n",
      "22\t82571\t0.2\t1\t76726 5845\n",
      "23\t87855\t0.2\t1\t81586 6269\n",
      "24\t88796\t0.2\t1\t82344 6452\n",
      "25\t99857\t0.2\t1\t93510 6347\n",
      "26\t83385\t0.2\t1\t77300 6085\n",
      "27\t93625\t0.2\t1\t86769 6856\n",
      "28\t85817\t0.2\t1\t80874 4943\n",
      "29\t96248\t0.2\t1\t90002 6246\n",
      "30\t93727\t0.2\t1\t88168 5559\n",
      "31\t103646\t0.2\t1\t97099 6547\n",
      "32\t90867\t0.2\t1\t85391 5476\n",
      "33\t105892\t0.2\t1\t98959 6933\n",
      "34\t137038\t0.2\t1\t128489 8549\n",
      "35\t77044\t0.2\t1\t72414 4630\n",
      "36\t105552\t0.2\t1\t98517 7035\n",
      "37\t112682\t0.2\t1\t105690 6992\n",
      "38\t106873\t0.2\t1\t100867 6006\n",
      "39\t102355\t0.2\t1\t96889 5466\n",
      "40\t117602\t0.2\t1\t110487 7115\n",
      "41\t116162\t0.2\t1\t109429 6733\n",
      "42\t121108\t0.2\t1\t114095 7013\n",
      "43\t109639\t0.2\t1\t103525 6114\n",
      "44\t129653\t0.2\t1\t122002 7651\n",
      "45\t138376\t0.2\t1\t130424 7952\n",
      "46\t97190\t0.2\t1\t91779 5411\n",
      "47\t124803\t0.2\t1\t118327 6476\n",
      "48\t124244\t0.2\t1\t117667 6577\n",
      "49\t131343\t0.2\t1\t123780 7563\n",
      "50\t128654\t0.2\t1\t121411 7243\n",
      "51\t160151\t0.2\t1\t151381 8770\n",
      "52\t116727\t0.2\t1\t110469 6258\n",
      "53\t144929\t0.2\t1\t137769 7160\n",
      "54\t104645\t0.2\t1\t99281 5364\n",
      "55\t138649\t0.2\t1\t131848 6801\n",
      "56\t142696\t0.2\t1\t134983 7713\n",
      "57\t141410\t0.2\t1\t134925 6485\n",
      "58\t134726\t0.2\t1\t128298 6428\n",
      "59\t134797\t0.2\t1\t128190 6607\n",
      "60\t140631\t0.2\t1\t133581 7050\n",
      "61\t142568\t0.2\t1\t135579 6989\n",
      "62\t160172\t0.2\t1\t152064 8108\n",
      "63\t209874\t0.2\t1\t200586 9288\n",
      "64\t163821\t0.2\t1\t157603 6218\n",
      "65\t114086\t0.2\t1\t108681 5405\n",
      "66\t126793\t0.2\t1\t120700 6093\n",
      "67\t126587\t0.2\t1\t120373 6214\n",
      "68\t128999\t0.2\t1\t122868 6131\n",
      "69\t127101\t0.2\t1\t121101 6000\n",
      "70\t133367\t0.2\t1\t126848 6519\n",
      "71\t137940\t0.2\t1\t131348 6592\n",
      "72\t133744\t0.2\t1\t127253 6491\n",
      "73\t140847\t0.2\t1\t134101 6746\n",
      "74\t131039\t0.2\t1\t124610 6429\n",
      "75\t123648\t0.2\t1\t117677 5971\n",
      "76\t121797\t0.2\t1\t115805 5992\n",
      "77\t120668\t0.2\t1\t114606 6062\n",
      "78\t116805\t0.2\t1\t111027 5778\n",
      "79\t111250\t0.2\t1\t105828 5422\n",
      "80\t110955\t0.2\t1\t105751 5204\n",
      "81\t109348\t0.2\t1\t104246 5102\n",
      "82\t103784\t0.2\t1\t98831 4953\n",
      "83\t101052\t0.2\t1\t96319 4733\n",
      "84\t98311\t0.2\t1\t93628 4683\n",
      "85\t96878\t0.2\t1\t92456 4422\n",
      "86\t95996\t0.2\t1\t91571 4425\n",
      "87\t96605\t0.2\t1\t92256 4349\n",
      "88\t93536\t0.2\t1\t89501 4035\n",
      "89\t83632\t0.2\t1\t79805 3827\n",
      "90\t80377\t0.2\t1\t76751 3626\n",
      "91\t75608\t0.2\t1\t72292 3316\n",
      "92\t71253\t0.2\t1\t68176 3077\n",
      "93\t68064\t0.2\t1\t65127 2937\n",
      "94\t65349\t0.2\t1\t62356 2993\n",
      "95\t61296\t0.2\t1\t58482 2814\n",
      "96\t57547\t0.2\t1\t54961 2586\n",
      "97\t53666\t0.2\t1\t51329 2337\n",
      "98\t50699\t0.2\t1\t48576 2123\n",
      "99\t48177\t0.2\t1\t46098 2079\n",
      "100\t43932\t0.2\t1\t42005 1927\n",
      "101\t39615\t0.2\t1\t37850 1765\n",
      "102\t35478\t0.2\t1\t33828 1650\n",
      "103\t32345\t0.2\t1\t30762 1583\n",
      "104\t30580\t0.2\t1\t29207 1373\n",
      "105\t27062\t0.2\t1\t25870 1192\n",
      "106\t24593\t0.2\t1\t23539 1054\n",
      "107\t21633\t0.2\t1\t20641 992\n",
      "108\t19150\t0.2\t1\t18297 853\n",
      "109\t17101\t0.2\t1\t16306 795\n",
      "110\t15550\t0.2\t1\t14882 668\n",
      "111\t14159\t0.2\t1\t13496 663\n",
      "112\t12046\t0.2\t1\t11497 549\n",
      "113\t10553\t0.2\t1\t10087 466\n",
      "114\t9330\t0.2\t1\t8879 451\n",
      "115\t7323\t0.2\t1\t7020 303\n",
      "116\t6173\t0.2\t1\t5878 295\n",
      "117\t5159\t0.2\t1\t4924 235\n",
      "118\t4546\t0.2\t1\t4310 236\n",
      "119\t3676\t0.2\t1\t3456 220\n",
      "120\t3075\t0.2\t1\t2912 163\n",
      "121\t2608\t0.2\t1\t2451 157\n",
      "122\t2115\t0.2\t1\t1990 125\n",
      "123\t1712\t0.2\t1\t1620 92\n",
      "124\t1534\t0.2\t1\t1451 83\n",
      "125\t1259\t0.2\t1\t1192 67\n",
      "126\t1123\t0.2\t1\t1061 62\n",
      "127\t965\t0.2\t1\t910 55\n",
      "128\t708\t0.2\t1\t665 43\n",
      "129\t619\t0.2\t1\t586 33\n",
      "130\t674\t0.2\t1\t630 44\n",
      "131\t895\t0.2\t1\t850 45\n",
      "132\t923\t0.2\t1\t873 50\n",
      "133\t1446\t0.2\t1\t1396 50\n",
      "134\t1648\t0.2\t1\t1601 47\n",
      "135\t705\t0.2\t1\t677 28\n",
      "136\t204\t0.2\t1\t192 12\n",
      "137\t151\t0.2\t1\t142 9\n",
      "138\t69\t0.2\t1\t59 10\n",
      "139\t62\t0.2\t1\t53 9\n",
      "140\t50\t0.2\t1\t43 7\n",
      "141\t28\t0.2\t1\t22 6\n",
      "142\t25\t0.2\t1\t22 3\n",
      "143\t20\t0.2\t1\t20\n",
      "144\t13\t0.2\t1\t12 1\n",
      "145\t39\t0.2\t1\t35 4\n",
      "146\t44\t0.2\t1\t41 3\n",
      "147\t178\t0.2\t1\t162 16\n",
      "148\t92\t0.2\t1\t79 13\n",
      "149\t44\t0.2\t1\t42 2\n",
      "150\t302\t0.2\t1\t273 29\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13408232 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t17606 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 564.66 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,781,290\n",
      "Reads with adapters:                10,464,679 (75.9%)\n",
      "Reads written (passing filters):    13,781,290 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,067,193,500 bp\n",
      "Quality-trimmed:               4,066,824 bp (0.2%)\n",
      "Total written (filtered):  1,647,743,968 bp (79.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10464679 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 15.6%\n",
      "  G: 18.6%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2115194\t3445322.5\t0\t2115194\n",
      "2\t272894\t861330.6\t0\t272894\n",
      "3\t153504\t215332.7\t0\t153504\n",
      "4\t91420\t53833.2\t0\t91420\n",
      "5\t70039\t13458.3\t0\t70039\n",
      "6\t67409\t3364.6\t0\t67409\n",
      "7\t55787\t841.1\t0\t55787\n",
      "8\t69144\t210.3\t0\t69144\n",
      "9\t70298\t52.6\t0\t69211 1087\n",
      "10\t65479\t13.1\t1\t61710 3769\n",
      "11\t71864\t3.3\t1\t66237 5627\n",
      "12\t69709\t0.8\t1\t65113 4596\n",
      "13\t68649\t0.2\t1\t64099 4550\n",
      "14\t71371\t0.2\t1\t66030 5341\n",
      "15\t72335\t0.2\t1\t67349 4986\n",
      "16\t73096\t0.2\t1\t67373 5723\n",
      "17\t87823\t0.2\t1\t80650 7173\n",
      "18\t73513\t0.2\t1\t68795 4718\n",
      "19\t64905\t0.2\t1\t60375 4530\n",
      "20\t76453\t0.2\t1\t71101 5352\n",
      "21\t83756\t0.2\t1\t77496 6260\n",
      "22\t71860\t0.2\t1\t67380 4480\n",
      "23\t82380\t0.2\t1\t76793 5587\n",
      "24\t76931\t0.2\t1\t70531 6400\n",
      "25\t82900\t0.2\t1\t77013 5887\n",
      "26\t75729\t0.2\t1\t70800 4929\n",
      "27\t84549\t0.2\t1\t78103 6446\n",
      "28\t82744\t0.2\t1\t76781 5963\n",
      "29\t87966\t0.2\t1\t80505 7461\n",
      "30\t92743\t0.2\t1\t87256 5487\n",
      "31\t72956\t0.2\t1\t68033 4923\n",
      "32\t86797\t0.2\t1\t81028 5769\n",
      "33\t84425\t0.2\t1\t77963 6462\n",
      "34\t100255\t0.2\t1\t92044 8211\n",
      "35\t94925\t0.2\t1\t87424 7501\n",
      "36\t151403\t0.2\t1\t144008 7395\n",
      "37\t34431\t0.2\t1\t30234 4197\n",
      "38\t81160\t0.2\t1\t76071 5089\n",
      "39\t98494\t0.2\t1\t91430 7064\n",
      "40\t94229\t0.2\t1\t88176 6053\n",
      "41\t105491\t0.2\t1\t97539 7952\n",
      "42\t89350\t0.2\t1\t81822 7528\n",
      "43\t144554\t0.2\t1\t136557 7997\n",
      "44\t52667\t0.2\t1\t48059 4608\n",
      "45\t80312\t0.2\t1\t74870 5442\n",
      "46\t96347\t0.2\t1\t89778 6569\n",
      "47\t100619\t0.2\t1\t94323 6296\n",
      "48\t87690\t0.2\t1\t81510 6180\n",
      "49\t103414\t0.2\t1\t95838 7576\n",
      "50\t86689\t0.2\t1\t81209 5480\n",
      "51\t92177\t0.2\t1\t85927 6250\n",
      "52\t98128\t0.2\t1\t90537 7591\n",
      "53\t111470\t0.2\t1\t104728 6742\n",
      "54\t79643\t0.2\t1\t74106 5537\n",
      "55\t98690\t0.2\t1\t92247 6443\n",
      "56\t114543\t0.2\t1\t107715 6828\n",
      "57\t91001\t0.2\t1\t84632 6369\n",
      "58\t94668\t0.2\t1\t89637 5031\n",
      "59\t73433\t0.2\t1\t68107 5326\n",
      "60\t100893\t0.2\t1\t94469 6424\n",
      "61\t94258\t0.2\t1\t88482 5776\n",
      "62\t93437\t0.2\t1\t87751 5686\n",
      "63\t86189\t0.2\t1\t80341 5848\n",
      "64\t87860\t0.2\t1\t82577 5283\n",
      "65\t95579\t0.2\t1\t89442 6137\n",
      "66\t102630\t0.2\t1\t95991 6639\n",
      "67\t110068\t0.2\t1\t101200 8868\n",
      "68\t211731\t0.2\t1\t201178 10553\n",
      "69\t44241\t0.2\t1\t40907 3334\n",
      "70\t13519\t0.2\t1\t11187 2332\n",
      "71\t49103\t0.2\t1\t45715 3388\n",
      "72\t77796\t0.2\t1\t73169 4627\n",
      "73\t81869\t0.2\t1\t76719 5150\n",
      "74\t82069\t0.2\t1\t76896 5173\n",
      "75\t80967\t0.2\t1\t75833 5134\n",
      "76\t79680\t0.2\t1\t74662 5018\n",
      "77\t80336\t0.2\t1\t75241 5095\n",
      "78\t75811\t0.2\t1\t70973 4838\n",
      "79\t74367\t0.2\t1\t69524 4843\n",
      "80\t73534\t0.2\t1\t69012 4522\n",
      "81\t72214\t0.2\t1\t67540 4674\n",
      "82\t70474\t0.2\t1\t66112 4362\n",
      "83\t68507\t0.2\t1\t64201 4306\n",
      "84\t66184\t0.2\t1\t61986 4198\n",
      "85\t64533\t0.2\t1\t60445 4088\n",
      "86\t62422\t0.2\t1\t58352 4070\n",
      "87\t59642\t0.2\t1\t55842 3800\n",
      "88\t59336\t0.2\t1\t55541 3795\n",
      "89\t56406\t0.2\t1\t52805 3601\n",
      "90\t56172\t0.2\t1\t52495 3677\n",
      "91\t53832\t0.2\t1\t50312 3520\n",
      "92\t51101\t0.2\t1\t47868 3233\n",
      "93\t48168\t0.2\t1\t45095 3073\n",
      "94\t47770\t0.2\t1\t44644 3126\n",
      "95\t44085\t0.2\t1\t41250 2835\n",
      "96\t41623\t0.2\t1\t38858 2765\n",
      "97\t39066\t0.2\t1\t36394 2672\n",
      "98\t35943\t0.2\t1\t33581 2362\n",
      "99\t35291\t0.2\t1\t32748 2543\n",
      "100\t32499\t0.2\t1\t30308 2191\n",
      "101\t30357\t0.2\t1\t28386 1971\n",
      "102\t28530\t0.2\t1\t26552 1978\n",
      "103\t26074\t0.2\t1\t24341 1733\n",
      "104\t24528\t0.2\t1\t22773 1755\n",
      "105\t21898\t0.2\t1\t20353 1545\n",
      "106\t20241\t0.2\t1\t18793 1448\n",
      "107\t18328\t0.2\t1\t16963 1365\n",
      "108\t17608\t0.2\t1\t16339 1269\n",
      "109\t15247\t0.2\t1\t14123 1124\n",
      "110\t14054\t0.2\t1\t12990 1064\n",
      "111\t12816\t0.2\t1\t11851 965\n",
      "112\t11319\t0.2\t1\t10448 871\n",
      "113\t9601\t0.2\t1\t8829 772\n",
      "114\t8785\t0.2\t1\t8065 720\n",
      "115\t7727\t0.2\t1\t7075 652\n",
      "116\t6667\t0.2\t1\t6137 530\n",
      "117\t5842\t0.2\t1\t5341 501\n",
      "118\t5116\t0.2\t1\t4650 466\n",
      "119\t4257\t0.2\t1\t3896 361\n",
      "120\t3744\t0.2\t1\t3400 344\n",
      "121\t3378\t0.2\t1\t3078 300\n",
      "122\t2708\t0.2\t1\t2469 239\n",
      "123\t2529\t0.2\t1\t2246 283\n",
      "124\t2015\t0.2\t1\t1814 201\n",
      "125\t1810\t0.2\t1\t1626 184\n",
      "126\t1647\t0.2\t1\t1490 157\n",
      "127\t1294\t0.2\t1\t1150 144\n",
      "128\t1010\t0.2\t1\t909 101\n",
      "129\t964\t0.2\t1\t860 104\n",
      "130\t736\t0.2\t1\t664 72\n",
      "131\t614\t0.2\t1\t553 61\n",
      "132\t536\t0.2\t1\t487 49\n",
      "133\t628\t0.2\t1\t563 65\n",
      "134\t1415\t0.2\t1\t1342 73\n",
      "135\t608\t0.2\t1\t567 41\n",
      "136\t207\t0.2\t1\t188 19\n",
      "137\t132\t0.2\t1\t118 14\n",
      "138\t84\t0.2\t1\t75 9\n",
      "139\t56\t0.2\t1\t44 12\n",
      "140\t46\t0.2\t1\t36 10\n",
      "141\t40\t0.2\t1\t35 5\n",
      "142\t28\t0.2\t1\t23 5\n",
      "143\t39\t0.2\t1\t32 7\n",
      "144\t49\t0.2\t1\t44 5\n",
      "145\t178\t0.2\t1\t162 16\n",
      "146\t250\t0.2\t1\t233 17\n",
      "147\t1272\t0.2\t1\t1190 82\n",
      "148\t812\t0.2\t1\t771 41\n",
      "149\t417\t0.2\t1\t393 24\n",
      "150\t2828\t0.2\t1\t2398 430\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13781290 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t10281 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Approx 100% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 595.40 s (43 us/read; 1.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,781,290\n",
      "Reads with adapters:                10,393,178 (75.4%)\n",
      "Reads written (passing filters):    13,781,290 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,067,193,500 bp\n",
      "Quality-trimmed:              10,332,398 bp (0.5%)\n",
      "Total written (filtered):  1,650,693,153 bp (79.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10393178 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 12.4%\n",
      "  G: 22.9%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2177342\t3445322.5\t0\t2177342\n",
      "2\t279599\t861330.6\t0\t279599\n",
      "3\t156926\t215332.7\t0\t156926\n",
      "4\t90665\t53833.2\t0\t90665\n",
      "5\t76306\t13458.3\t0\t76306\n",
      "6\t64993\t3364.6\t0\t64993\n",
      "7\t53352\t841.1\t0\t53352\n",
      "8\t75967\t210.3\t0\t75967\n",
      "9\t61280\t52.6\t0\t61074 206\n",
      "10\t66110\t13.1\t1\t62970 3140\n",
      "11\t69828\t3.3\t1\t65500 4328\n",
      "12\t69328\t0.8\t1\t64877 4451\n",
      "13\t77454\t0.2\t1\t72977 4477\n",
      "14\t71203\t0.2\t1\t66291 4912\n",
      "15\t59758\t0.2\t1\t55879 3879\n",
      "16\t86601\t0.2\t1\t81117 5484\n",
      "17\t65384\t0.2\t1\t60933 4451\n",
      "18\t67208\t0.2\t1\t63111 4097\n",
      "19\t81178\t0.2\t1\t76029 5149\n",
      "20\t77499\t0.2\t1\t72833 4666\n",
      "21\t68546\t0.2\t1\t64113 4433\n",
      "22\t76276\t0.2\t1\t71754 4522\n",
      "23\t78702\t0.2\t1\t73839 4863\n",
      "24\t78191\t0.2\t1\t73447 4744\n",
      "25\t88316\t0.2\t1\t83190 5126\n",
      "26\t73679\t0.2\t1\t68983 4696\n",
      "27\t82923\t0.2\t1\t77281 5642\n",
      "28\t76296\t0.2\t1\t72446 3850\n",
      "29\t82111\t0.2\t1\t77123 4988\n",
      "30\t79614\t0.2\t1\t75308 4306\n",
      "31\t89456\t0.2\t1\t84223 5233\n",
      "32\t78118\t0.2\t1\t73947 4171\n",
      "33\t86787\t0.2\t1\t81520 5267\n",
      "34\t111454\t0.2\t1\t105036 6418\n",
      "35\t64200\t0.2\t1\t60716 3484\n",
      "36\t89863\t0.2\t1\t84483 5380\n",
      "37\t97027\t0.2\t1\t91271 5756\n",
      "38\t90403\t0.2\t1\t85738 4665\n",
      "39\t89856\t0.2\t1\t85089 4767\n",
      "40\t98244\t0.2\t1\t92516 5728\n",
      "41\t92652\t0.2\t1\t87737 4915\n",
      "42\t94536\t0.2\t1\t89519 5017\n",
      "43\t85154\t0.2\t1\t80486 4668\n",
      "44\t96817\t0.2\t1\t91511 5306\n",
      "45\t102115\t0.2\t1\t96823 5292\n",
      "46\t77381\t0.2\t1\t73350 4031\n",
      "47\t94700\t0.2\t1\t89756 4944\n",
      "48\t92948\t0.2\t1\t88160 4788\n",
      "49\t96945\t0.2\t1\t91985 4960\n",
      "50\t91133\t0.2\t1\t86206 4927\n",
      "51\t111309\t0.2\t1\t105778 5531\n",
      "52\t80724\t0.2\t1\t76802 3922\n",
      "53\t97109\t0.2\t1\t92582 4527\n",
      "54\t75460\t0.2\t1\t71575 3885\n",
      "55\t96939\t0.2\t1\t92436 4503\n",
      "56\t100452\t0.2\t1\t95531 4921\n",
      "57\t93931\t0.2\t1\t89647 4284\n",
      "58\t91630\t0.2\t1\t87584 4046\n",
      "59\t91476\t0.2\t1\t87193 4283\n",
      "60\t94295\t0.2\t1\t89743 4552\n",
      "61\t96075\t0.2\t1\t91458 4617\n",
      "62\t103947\t0.2\t1\t98742 5205\n",
      "63\t139881\t0.2\t1\t133644 6237\n",
      "64\t115738\t0.2\t1\t111196 4542\n",
      "65\t76600\t0.2\t1\t73143 3457\n",
      "66\t81388\t0.2\t1\t77826 3562\n",
      "67\t79784\t0.2\t1\t76063 3721\n",
      "68\t81137\t0.2\t1\t77492 3645\n",
      "69\t80187\t0.2\t1\t76468 3719\n",
      "70\t85090\t0.2\t1\t81074 4016\n",
      "71\t89717\t0.2\t1\t85778 3939\n",
      "72\t84263\t0.2\t1\t80409 3854\n",
      "73\t87216\t0.2\t1\t83465 3751\n",
      "74\t81093\t0.2\t1\t77528 3565\n",
      "75\t77500\t0.2\t1\t74052 3448\n",
      "76\t75266\t0.2\t1\t71946 3320\n",
      "77\t75853\t0.2\t1\t72329 3524\n",
      "78\t72260\t0.2\t1\t68950 3310\n",
      "79\t68908\t0.2\t1\t65817 3091\n",
      "80\t69469\t0.2\t1\t66410 3059\n",
      "81\t68119\t0.2\t1\t65042 3077\n",
      "82\t65495\t0.2\t1\t62627 2868\n",
      "83\t64033\t0.2\t1\t61248 2785\n",
      "84\t61976\t0.2\t1\t59216 2760\n",
      "85\t61317\t0.2\t1\t58515 2802\n",
      "86\t60982\t0.2\t1\t58300 2682\n",
      "87\t61128\t0.2\t1\t58498 2630\n",
      "88\t60002\t0.2\t1\t57306 2696\n",
      "89\t53699\t0.2\t1\t51332 2367\n",
      "90\t51966\t0.2\t1\t49630 2336\n",
      "91\t49026\t0.2\t1\t46908 2118\n",
      "92\t46213\t0.2\t1\t44244 1969\n",
      "93\t44796\t0.2\t1\t42840 1956\n",
      "94\t44237\t0.2\t1\t42309 1928\n",
      "95\t41556\t0.2\t1\t39752 1804\n",
      "96\t39335\t0.2\t1\t37550 1785\n",
      "97\t36382\t0.2\t1\t34875 1507\n",
      "98\t33450\t0.2\t1\t32005 1445\n",
      "99\t32975\t0.2\t1\t31610 1365\n",
      "100\t29764\t0.2\t1\t28510 1254\n",
      "101\t27741\t0.2\t1\t26626 1115\n",
      "102\t25948\t0.2\t1\t24908 1040\n",
      "103\t23781\t0.2\t1\t22799 982\n",
      "104\t22951\t0.2\t1\t22001 950\n",
      "105\t20726\t0.2\t1\t19855 871\n",
      "106\t19269\t0.2\t1\t18460 809\n",
      "107\t17116\t0.2\t1\t16344 772\n",
      "108\t16142\t0.2\t1\t15422 720\n",
      "109\t14186\t0.2\t1\t13581 605\n",
      "110\t13235\t0.2\t1\t12682 553\n",
      "111\t12589\t0.2\t1\t12054 535\n",
      "112\t10912\t0.2\t1\t10466 446\n",
      "113\t9393\t0.2\t1\t8942 451\n",
      "114\t8457\t0.2\t1\t8091 366\n",
      "115\t7197\t0.2\t1\t6874 323\n",
      "116\t6211\t0.2\t1\t5908 303\n",
      "117\t5216\t0.2\t1\t4981 235\n",
      "118\t4612\t0.2\t1\t4406 206\n",
      "119\t3969\t0.2\t1\t3800 169\n",
      "120\t3391\t0.2\t1\t3233 158\n",
      "121\t3136\t0.2\t1\t3000 136\n",
      "122\t2549\t0.2\t1\t2419 130\n",
      "123\t2350\t0.2\t1\t2231 119\n",
      "124\t1891\t0.2\t1\t1802 89\n",
      "125\t1725\t0.2\t1\t1617 108\n",
      "126\t1554\t0.2\t1\t1484 70\n",
      "127\t1178\t0.2\t1\t1121 57\n",
      "128\t930\t0.2\t1\t883 47\n",
      "129\t904\t0.2\t1\t851 53\n",
      "130\t653\t0.2\t1\t624 29\n",
      "131\t560\t0.2\t1\t529 31\n",
      "132\t495\t0.2\t1\t468 27\n",
      "133\t660\t0.2\t1\t626 34\n",
      "134\t1423\t0.2\t1\t1385 38\n",
      "135\t575\t0.2\t1\t556 19\n",
      "136\t211\t0.2\t1\t198 13\n",
      "137\t131\t0.2\t1\t127 4\n",
      "138\t83\t0.2\t1\t76 7\n",
      "139\t59\t0.2\t1\t52 7\n",
      "140\t46\t0.2\t1\t40 6\n",
      "141\t37\t0.2\t1\t35 2\n",
      "142\t28\t0.2\t1\t27 1\n",
      "143\t41\t0.2\t1\t38 3\n",
      "144\t59\t0.2\t1\t58 1\n",
      "145\t179\t0.2\t1\t178 1\n",
      "146\t263\t0.2\t1\t249 14\n",
      "147\t1229\t0.2\t1\t1166 63\n",
      "148\t791\t0.2\t1\t756 35\n",
      "149\t435\t0.2\t1\t415 20\n",
      "150\t2389\t0.2\t1\t2255 134\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13781290 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t25904 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 567.75 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,627,098\n",
      "Reads with adapters:                10,357,003 (76.0%)\n",
      "Reads written (passing filters):    13,627,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,044,064,700 bp\n",
      "Quality-trimmed:               4,208,565 bp (0.2%)\n",
      "Total written (filtered):  1,627,255,744 bp (79.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10357003 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.3%\n",
      "  C: 15.7%\n",
      "  G: 18.6%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2086122\t3406774.5\t0\t2086122\n",
      "2\t266568\t851693.6\t0\t266568\n",
      "3\t152542\t212923.4\t0\t152542\n",
      "4\t89399\t53230.9\t0\t89399\n",
      "5\t70209\t13307.7\t0\t70209\n",
      "6\t66787\t3326.9\t0\t66787\n",
      "7\t53852\t831.7\t0\t53852\n",
      "8\t67948\t207.9\t0\t67948\n",
      "9\t69323\t52.0\t0\t68200 1123\n",
      "10\t63939\t13.0\t1\t59604 4335\n",
      "11\t71265\t3.2\t1\t64858 6407\n",
      "12\t68721\t0.8\t1\t63494 5227\n",
      "13\t67596\t0.2\t1\t62302 5294\n",
      "14\t70216\t0.2\t1\t64057 6159\n",
      "15\t71638\t0.2\t1\t65898 5740\n",
      "16\t71622\t0.2\t1\t65051 6571\n",
      "17\t87796\t0.2\t1\t79687 8109\n",
      "18\t71999\t0.2\t1\t66604 5395\n",
      "19\t63344\t0.2\t1\t58390 4954\n",
      "20\t75324\t0.2\t1\t69407 5917\n",
      "21\t84178\t0.2\t1\t77194 6984\n",
      "22\t70154\t0.2\t1\t65038 5116\n",
      "23\t81908\t0.2\t1\t75394 6514\n",
      "24\t75905\t0.2\t1\t68761 7144\n",
      "25\t81978\t0.2\t1\t75339 6639\n",
      "26\t74054\t0.2\t1\t68713 5341\n",
      "27\t83694\t0.2\t1\t76279 7415\n",
      "28\t82366\t0.2\t1\t75736 6630\n",
      "29\t86579\t0.2\t1\t78687 7892\n",
      "30\t91938\t0.2\t1\t85372 6566\n",
      "31\t70649\t0.2\t1\t65160 5489\n",
      "32\t86162\t0.2\t1\t79280 6882\n",
      "33\t82680\t0.2\t1\t75408 7272\n",
      "34\t100154\t0.2\t1\t90695 9459\n",
      "35\t94358\t0.2\t1\t85860 8498\n",
      "36\t148458\t0.2\t1\t139524 8934\n",
      "37\t35889\t0.2\t1\t31244 4645\n",
      "38\t78400\t0.2\t1\t72661 5739\n",
      "39\t96881\t0.2\t1\t88590 8291\n",
      "40\t92469\t0.2\t1\t85322 7147\n",
      "41\t105619\t0.2\t1\t96453 9166\n",
      "42\t87195\t0.2\t1\t78548 8647\n",
      "43\t141743\t0.2\t1\t132343 9400\n",
      "44\t53443\t0.2\t1\t48101 5342\n",
      "45\t80210\t0.2\t1\t73892 6318\n",
      "46\t94587\t0.2\t1\t87548 7039\n",
      "47\t98921\t0.2\t1\t91675 7246\n",
      "48\t85740\t0.2\t1\t78755 6985\n",
      "49\t103381\t0.2\t1\t94837 8544\n",
      "50\t84900\t0.2\t1\t78738 6162\n",
      "51\t91291\t0.2\t1\t84483 6808\n",
      "52\t97726\t0.2\t1\t89264 8462\n",
      "53\t111539\t0.2\t1\t103945 7594\n",
      "54\t78508\t0.2\t1\t72278 6230\n",
      "55\t97750\t0.2\t1\t90509 7241\n",
      "56\t115622\t0.2\t1\t107685 7937\n",
      "57\t89129\t0.2\t1\t81822 7307\n",
      "58\t92584\t0.2\t1\t86772 5812\n",
      "59\t73435\t0.2\t1\t67515 5920\n",
      "60\t100185\t0.2\t1\t92834 7351\n",
      "61\t93396\t0.2\t1\t86891 6505\n",
      "62\t93505\t0.2\t1\t86821 6684\n",
      "63\t85463\t0.2\t1\t78881 6582\n",
      "64\t86705\t0.2\t1\t80704 6001\n",
      "65\t94221\t0.2\t1\t87522 6699\n",
      "66\t100565\t0.2\t1\t93273 7292\n",
      "67\t109840\t0.2\t1\t100139 9701\n",
      "68\t202049\t0.2\t1\t190563 11486\n",
      "69\t52634\t0.2\t1\t48583 4051\n",
      "70\t17146\t0.2\t1\t14429 2717\n",
      "71\t47197\t0.2\t1\t43653 3544\n",
      "72\t76160\t0.2\t1\t71164 4996\n",
      "73\t81740\t0.2\t1\t76081 5659\n",
      "74\t82050\t0.2\t1\t76332 5718\n",
      "75\t80287\t0.2\t1\t74463 5824\n",
      "76\t79839\t0.2\t1\t73807 6032\n",
      "77\t80016\t0.2\t1\t74278 5738\n",
      "78\t74795\t0.2\t1\t69265 5530\n",
      "79\t74038\t0.2\t1\t68576 5462\n",
      "80\t73576\t0.2\t1\t68130 5446\n",
      "81\t72243\t0.2\t1\t66987 5256\n",
      "82\t69871\t0.2\t1\t64751 5120\n",
      "83\t67762\t0.2\t1\t62929 4833\n",
      "84\t65749\t0.2\t1\t60930 4819\n",
      "85\t64210\t0.2\t1\t59436 4774\n",
      "86\t62228\t0.2\t1\t57683 4545\n",
      "87\t59511\t0.2\t1\t55197 4314\n",
      "88\t59103\t0.2\t1\t54779 4324\n",
      "89\t57029\t0.2\t1\t52911 4118\n",
      "90\t56111\t0.2\t1\t51929 4182\n",
      "91\t53588\t0.2\t1\t49754 3834\n",
      "92\t50869\t0.2\t1\t47115 3754\n",
      "93\t47789\t0.2\t1\t44363 3426\n",
      "94\t47204\t0.2\t1\t43712 3492\n",
      "95\t44549\t0.2\t1\t41263 3286\n",
      "96\t41699\t0.2\t1\t38638 3061\n",
      "97\t39043\t0.2\t1\t36086 2957\n",
      "98\t35735\t0.2\t1\t32882 2853\n",
      "99\t35123\t0.2\t1\t32338 2785\n",
      "100\t32434\t0.2\t1\t30000 2434\n",
      "101\t30205\t0.2\t1\t27895 2310\n",
      "102\t28238\t0.2\t1\t26075 2163\n",
      "103\t26099\t0.2\t1\t24128 1971\n",
      "104\t24785\t0.2\t1\t22897 1888\n",
      "105\t21850\t0.2\t1\t20176 1674\n",
      "106\t20336\t0.2\t1\t18803 1533\n",
      "107\t18435\t0.2\t1\t16991 1444\n",
      "108\t17298\t0.2\t1\t15900 1398\n",
      "109\t15693\t0.2\t1\t14418 1275\n",
      "110\t13857\t0.2\t1\t12707 1150\n",
      "111\t12767\t0.2\t1\t11710 1057\n",
      "112\t11323\t0.2\t1\t10411 912\n",
      "113\t9639\t0.2\t1\t8850 789\n",
      "114\t8830\t0.2\t1\t8052 778\n",
      "115\t7781\t0.2\t1\t7105 676\n",
      "116\t6721\t0.2\t1\t6151 570\n",
      "117\t5849\t0.2\t1\t5270 579\n",
      "118\t5202\t0.2\t1\t4706 496\n",
      "119\t4310\t0.2\t1\t3919 391\n",
      "120\t3867\t0.2\t1\t3492 375\n",
      "121\t3291\t0.2\t1\t2960 331\n",
      "122\t2743\t0.2\t1\t2485 258\n",
      "123\t2310\t0.2\t1\t2091 219\n",
      "124\t2101\t0.2\t1\t1881 220\n",
      "125\t1771\t0.2\t1\t1559 212\n",
      "126\t1539\t0.2\t1\t1390 149\n",
      "127\t1290\t0.2\t1\t1141 149\n",
      "128\t1070\t0.2\t1\t950 120\n",
      "129\t924\t0.2\t1\t833 91\n",
      "130\t685\t0.2\t1\t617 68\n",
      "131\t612\t0.2\t1\t545 67\n",
      "132\t539\t0.2\t1\t487 52\n",
      "133\t666\t0.2\t1\t596 70\n",
      "134\t1493\t0.2\t1\t1414 79\n",
      "135\t574\t0.2\t1\t548 26\n",
      "136\t201\t0.2\t1\t183 18\n",
      "137\t159\t0.2\t1\t145 14\n",
      "138\t108\t0.2\t1\t98 10\n",
      "139\t49\t0.2\t1\t44 5\n",
      "140\t42\t0.2\t1\t38 4\n",
      "141\t43\t0.2\t1\t37 6\n",
      "142\t35\t0.2\t1\t31 4\n",
      "143\t45\t0.2\t1\t39 6\n",
      "144\t56\t0.2\t1\t49 7\n",
      "145\t155\t0.2\t1\t139 16\n",
      "146\t240\t0.2\t1\t219 21\n",
      "147\t1341\t0.2\t1\t1228 113\n",
      "148\t773\t0.2\t1\t707 66\n",
      "149\t433\t0.2\t1\t399 34\n",
      "150\t2986\t0.2\t1\t2547 439\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13627098 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t10583 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 600.51 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,627,098\n",
      "Reads with adapters:                10,276,263 (75.4%)\n",
      "Reads written (passing filters):    13,627,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,044,064,700 bp\n",
      "Quality-trimmed:               9,937,551 bp (0.5%)\n",
      "Total written (filtered):  1,630,739,955 bp (79.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10276263 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 12.4%\n",
      "  G: 23.0%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2146357\t3406774.5\t0\t2146357\n",
      "2\t269360\t851693.6\t0\t269360\n",
      "3\t157893\t212923.4\t0\t157893\n",
      "4\t87930\t53230.9\t0\t87930\n",
      "5\t73543\t13307.7\t0\t73543\n",
      "6\t64791\t3326.9\t0\t64791\n",
      "7\t52189\t831.7\t0\t52189\n",
      "8\t74756\t207.9\t0\t74756\n",
      "9\t60599\t52.0\t0\t60366 233\n",
      "10\t63543\t13.0\t1\t59315 4228\n",
      "11\t68667\t3.2\t1\t63268 5399\n",
      "12\t67618\t0.8\t1\t62224 5394\n",
      "13\t77226\t0.2\t1\t71131 6095\n",
      "14\t69709\t0.2\t1\t63900 5809\n",
      "15\t58686\t0.2\t1\t54055 4631\n",
      "16\t84262\t0.2\t1\t77680 6582\n",
      "17\t67924\t0.2\t1\t62515 5409\n",
      "18\t64114\t0.2\t1\t59144 4970\n",
      "19\t80499\t0.2\t1\t74211 6288\n",
      "20\t76034\t0.2\t1\t70274 5760\n",
      "21\t67847\t0.2\t1\t62554 5293\n",
      "22\t74609\t0.2\t1\t68953 5656\n",
      "23\t78584\t0.2\t1\t72661 5923\n",
      "24\t77507\t0.2\t1\t71592 5915\n",
      "25\t87777\t0.2\t1\t81767 6010\n",
      "26\t72611\t0.2\t1\t66753 5858\n",
      "27\t83123\t0.2\t1\t76411 6712\n",
      "28\t73079\t0.2\t1\t68272 4807\n",
      "29\t82291\t0.2\t1\t76291 6000\n",
      "30\t76541\t0.2\t1\t71237 5304\n",
      "31\t89563\t0.2\t1\t83077 6486\n",
      "32\t74346\t0.2\t1\t69371 4975\n",
      "33\t86840\t0.2\t1\t80487 6353\n",
      "34\t112360\t0.2\t1\t104461 7899\n",
      "35\t60619\t0.2\t1\t56337 4282\n",
      "36\t90028\t0.2\t1\t83414 6614\n",
      "37\t96370\t0.2\t1\t89511 6859\n",
      "38\t89130\t0.2\t1\t83444 5686\n",
      "39\t86200\t0.2\t1\t80889 5311\n",
      "40\t97029\t0.2\t1\t90271 6758\n",
      "41\t93202\t0.2\t1\t87114 6088\n",
      "42\t93307\t0.2\t1\t87268 6039\n",
      "43\t83469\t0.2\t1\t77943 5526\n",
      "44\t97590\t0.2\t1\t91014 6576\n",
      "45\t100276\t0.2\t1\t93792 6484\n",
      "46\t75444\t0.2\t1\t70447 4997\n",
      "47\t93307\t0.2\t1\t87603 5704\n",
      "48\t90968\t0.2\t1\t85370 5598\n",
      "49\t95545\t0.2\t1\t89305 6240\n",
      "50\t91141\t0.2\t1\t85291 5850\n",
      "51\t108887\t0.2\t1\t102107 6780\n",
      "52\t80355\t0.2\t1\t75335 5020\n",
      "53\t99048\t0.2\t1\t93311 5737\n",
      "54\t72741\t0.2\t1\t68334 4407\n",
      "55\t95947\t0.2\t1\t90438 5509\n",
      "56\t99513\t0.2\t1\t93368 6145\n",
      "57\t93556\t0.2\t1\t88452 5104\n",
      "58\t90539\t0.2\t1\t85415 5124\n",
      "59\t90467\t0.2\t1\t85091 5376\n",
      "60\t93608\t0.2\t1\t88124 5484\n",
      "61\t95101\t0.2\t1\t89698 5403\n",
      "62\t102226\t0.2\t1\t96046 6180\n",
      "63\t136011\t0.2\t1\t129004 7007\n",
      "64\t112722\t0.2\t1\t107622 5100\n",
      "65\t75623\t0.2\t1\t71479 4144\n",
      "66\t81004\t0.2\t1\t76564 4440\n",
      "67\t79874\t0.2\t1\t75268 4606\n",
      "68\t81531\t0.2\t1\t76952 4579\n",
      "69\t80094\t0.2\t1\t75476 4618\n",
      "70\t85599\t0.2\t1\t80759 4840\n",
      "71\t89528\t0.2\t1\t84657 4871\n",
      "72\t84362\t0.2\t1\t79637 4725\n",
      "73\t87554\t0.2\t1\t82710 4844\n",
      "74\t81739\t0.2\t1\t77139 4600\n",
      "75\t76846\t0.2\t1\t72538 4308\n",
      "76\t75616\t0.2\t1\t71298 4318\n",
      "77\t75671\t0.2\t1\t71375 4296\n",
      "78\t71297\t0.2\t1\t67330 3967\n",
      "79\t68959\t0.2\t1\t65062 3897\n",
      "80\t69385\t0.2\t1\t65444 3941\n",
      "81\t68113\t0.2\t1\t64379 3734\n",
      "82\t65128\t0.2\t1\t61578 3550\n",
      "83\t63398\t0.2\t1\t59890 3508\n",
      "84\t61642\t0.2\t1\t58327 3315\n",
      "85\t61077\t0.2\t1\t57799 3278\n",
      "86\t60708\t0.2\t1\t57467 3241\n",
      "87\t60697\t0.2\t1\t57594 3103\n",
      "88\t59562\t0.2\t1\t56463 3099\n",
      "89\t54239\t0.2\t1\t51398 2841\n",
      "90\t51773\t0.2\t1\t48980 2793\n",
      "91\t49319\t0.2\t1\t46756 2563\n",
      "92\t46223\t0.2\t1\t43736 2487\n",
      "93\t44798\t0.2\t1\t42455 2343\n",
      "94\t43852\t0.2\t1\t41584 2268\n",
      "95\t41908\t0.2\t1\t39647 2261\n",
      "96\t39622\t0.2\t1\t37536 2086\n",
      "97\t36420\t0.2\t1\t34621 1799\n",
      "98\t33185\t0.2\t1\t31489 1696\n",
      "99\t32771\t0.2\t1\t31125 1646\n",
      "100\t29865\t0.2\t1\t28321 1544\n",
      "101\t27753\t0.2\t1\t26308 1445\n",
      "102\t25712\t0.2\t1\t24349 1363\n",
      "103\t23843\t0.2\t1\t22528 1315\n",
      "104\t23330\t0.2\t1\t22084 1246\n",
      "105\t20720\t0.2\t1\t19658 1062\n",
      "106\t19342\t0.2\t1\t18364 978\n",
      "107\t17248\t0.2\t1\t16375 873\n",
      "108\t15847\t0.2\t1\t15052 795\n",
      "109\t14536\t0.2\t1\t13807 729\n",
      "110\t12976\t0.2\t1\t12292 684\n",
      "111\t12529\t0.2\t1\t11885 644\n",
      "112\t10963\t0.2\t1\t10373 590\n",
      "113\t9349\t0.2\t1\t8908 441\n",
      "114\t8548\t0.2\t1\t8120 428\n",
      "115\t7299\t0.2\t1\t6916 383\n",
      "116\t6237\t0.2\t1\t5941 296\n",
      "117\t5243\t0.2\t1\t4920 323\n",
      "118\t4696\t0.2\t1\t4465 231\n",
      "119\t3975\t0.2\t1\t3763 212\n",
      "120\t3508\t0.2\t1\t3309 199\n",
      "121\t3040\t0.2\t1\t2875 165\n",
      "122\t2575\t0.2\t1\t2428 147\n",
      "123\t2186\t0.2\t1\t2073 113\n",
      "124\t1945\t0.2\t1\t1832 113\n",
      "125\t1652\t0.2\t1\t1551 101\n",
      "126\t1464\t0.2\t1\t1388 76\n",
      "127\t1181\t0.2\t1\t1120 61\n",
      "128\t990\t0.2\t1\t926 64\n",
      "129\t867\t0.2\t1\t802 65\n",
      "130\t639\t0.2\t1\t610 29\n",
      "131\t564\t0.2\t1\t532 32\n",
      "132\t517\t0.2\t1\t490 27\n",
      "133\t691\t0.2\t1\t653 38\n",
      "134\t1520\t0.2\t1\t1458 62\n",
      "135\t553\t0.2\t1\t527 26\n",
      "136\t201\t0.2\t1\t194 7\n",
      "137\t155\t0.2\t1\t146 9\n",
      "138\t95\t0.2\t1\t92 3\n",
      "139\t52\t0.2\t1\t49 3\n",
      "140\t44\t0.2\t1\t40 4\n",
      "141\t42\t0.2\t1\t38 4\n",
      "142\t33\t0.2\t1\t31 2\n",
      "143\t49\t0.2\t1\t44 5\n",
      "144\t63\t0.2\t1\t60 3\n",
      "145\t162\t0.2\t1\t155 7\n",
      "146\t237\t0.2\t1\t226 11\n",
      "147\t1276\t0.2\t1\t1212 64\n",
      "148\t739\t0.2\t1\t700 39\n",
      "149\t423\t0.2\t1\t396 27\n",
      "150\t2548\t0.2\t1\t2400 148\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13627098 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t24804 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 579.26 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,982,389\n",
      "Reads with adapters:                15,316,406 (90.2%)\n",
      "Reads written (passing filters):    16,982,389 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,547,358,350 bp\n",
      "Quality-trimmed:               5,122,052 bp (0.2%)\n",
      "Total written (filtered):  1,661,718,543 bp (65.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15316406 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 17.9%\n",
      "  G: 21.3%\n",
      "  T: 32.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t934198\t4245597.2\t0\t934198\n",
      "2\t161909\t1061399.3\t0\t161909\n",
      "3\t105071\t265349.8\t0\t105071\n",
      "4\t65361\t66337.5\t0\t65361\n",
      "5\t61151\t16584.4\t0\t61151\n",
      "6\t60130\t4146.1\t0\t60130\n",
      "7\t51265\t1036.5\t0\t51265\n",
      "8\t64755\t259.1\t0\t64755\n",
      "9\t64351\t64.8\t0\t63580 771\n",
      "10\t61337\t16.2\t1\t58020 3317\n",
      "11\t70221\t4.0\t1\t64705 5516\n",
      "12\t66216\t1.0\t1\t61889 4327\n",
      "13\t69137\t0.3\t1\t64630 4507\n",
      "14\t70780\t0.3\t1\t65742 5038\n",
      "15\t75204\t0.3\t1\t70097 5107\n",
      "16\t73779\t0.3\t1\t68019 5760\n",
      "17\t90043\t0.3\t1\t82540 7503\n",
      "18\t81214\t0.3\t1\t76392 4822\n",
      "19\t66550\t0.3\t1\t62390 4160\n",
      "20\t83867\t0.3\t1\t78220 5647\n",
      "21\t92423\t0.3\t1\t86031 6392\n",
      "22\t79989\t0.3\t1\t75306 4683\n",
      "23\t93784\t0.3\t1\t87424 6360\n",
      "24\t89031\t0.3\t1\t81718 7313\n",
      "25\t96197\t0.3\t1\t89537 6660\n",
      "26\t88926\t0.3\t1\t83710 5216\n",
      "27\t100077\t0.3\t1\t92668 7409\n",
      "28\t100613\t0.3\t1\t93553 7060\n",
      "29\t104938\t0.3\t1\t96491 8447\n",
      "30\t117488\t0.3\t1\t110880 6608\n",
      "31\t89018\t0.3\t1\t83492 5526\n",
      "32\t107667\t0.3\t1\t100716 6951\n",
      "33\t110564\t0.3\t1\t102547 8017\n",
      "34\t137673\t0.3\t1\t127216 10457\n",
      "35\t119092\t0.3\t1\t112273 6819\n",
      "36\t94075\t0.3\t1\t88092 5983\n",
      "37\t125638\t0.3\t1\t116775 8863\n",
      "38\t138305\t0.3\t1\t128145 10160\n",
      "39\t120373\t0.3\t1\t113314 7059\n",
      "40\t138522\t0.3\t1\t128397 10125\n",
      "41\t158012\t0.3\t1\t146728 11284\n",
      "42\t111371\t0.3\t1\t103118 8253\n",
      "43\t217278\t0.3\t1\t206218 11060\n",
      "44\t57407\t0.3\t1\t51985 5422\n",
      "45\t143497\t0.3\t1\t134258 9239\n",
      "46\t142849\t0.3\t1\t133725 9124\n",
      "47\t147972\t0.3\t1\t139076 8896\n",
      "48\t148858\t0.3\t1\t138933 9925\n",
      "49\t163896\t0.3\t1\t152740 11156\n",
      "50\t140197\t0.3\t1\t131951 8246\n",
      "51\t154425\t0.3\t1\t144692 9733\n",
      "52\t163399\t0.3\t1\t151525 11874\n",
      "53\t181558\t0.3\t1\t170911 10647\n",
      "54\t136765\t0.3\t1\t128031 8734\n",
      "55\t172905\t0.3\t1\t161877 11028\n",
      "56\t211987\t0.3\t1\t200164 11823\n",
      "57\t138899\t0.3\t1\t129027 9872\n",
      "58\t181048\t0.3\t1\t171675 9373\n",
      "59\t152882\t0.3\t1\t143208 9674\n",
      "60\t179132\t0.3\t1\t167976 11156\n",
      "61\t179957\t0.3\t1\t169292 10665\n",
      "62\t195431\t0.3\t1\t184537 10894\n",
      "63\t158140\t0.3\t1\t148011 10129\n",
      "64\t177885\t0.3\t1\t167763 10122\n",
      "65\t200563\t0.3\t1\t188662 11901\n",
      "66\t211859\t0.3\t1\t199050 12809\n",
      "67\t228942\t0.3\t1\t211658 17284\n",
      "68\t447452\t0.3\t1\t426829 20623\n",
      "69\t83220\t0.3\t1\t76361 6859\n",
      "70\t30543\t0.3\t1\t25376 5167\n",
      "71\t119720\t0.3\t1\t112578 7142\n",
      "72\t172151\t0.3\t1\t162467 9684\n",
      "73\t185670\t0.3\t1\t174903 10767\n",
      "74\t186107\t0.3\t1\t175086 11021\n",
      "75\t187142\t0.3\t1\t176040 11102\n",
      "76\t187583\t0.3\t1\t176804 10779\n",
      "77\t190166\t0.3\t1\t178927 11239\n",
      "78\t180337\t0.3\t1\t169693 10644\n",
      "79\t182061\t0.3\t1\t171352 10709\n",
      "80\t179279\t0.3\t1\t168912 10367\n",
      "81\t180871\t0.3\t1\t170399 10472\n",
      "82\t177227\t0.3\t1\t166777 10450\n",
      "83\t175671\t0.3\t1\t165379 10292\n",
      "84\t171899\t0.3\t1\t161834 10065\n",
      "85\t167562\t0.3\t1\t157917 9645\n",
      "86\t164690\t0.3\t1\t155117 9573\n",
      "87\t159808\t0.3\t1\t150364 9444\n",
      "88\t160575\t0.3\t1\t151109 9466\n",
      "89\t154675\t0.3\t1\t145660 9015\n",
      "90\t155657\t0.3\t1\t146266 9391\n",
      "91\t151129\t0.3\t1\t142165 8964\n",
      "92\t144841\t0.3\t1\t136295 8546\n",
      "93\t137285\t0.3\t1\t129298 7987\n",
      "94\t135380\t0.3\t1\t127295 8085\n",
      "95\t127081\t0.3\t1\t119343 7738\n",
      "96\t122579\t0.3\t1\t115210 7369\n",
      "97\t117441\t0.3\t1\t110330 7111\n",
      "98\t110426\t0.3\t1\t103591 6835\n",
      "99\t109189\t0.3\t1\t102159 7030\n",
      "100\t100853\t0.3\t1\t94763 6090\n",
      "101\t96345\t0.3\t1\t90423 5922\n",
      "102\t89845\t0.3\t1\t84149 5696\n",
      "103\t85390\t0.3\t1\t80150 5240\n",
      "104\t79941\t0.3\t1\t74924 5017\n",
      "105\t72924\t0.3\t1\t68266 4658\n",
      "106\t67356\t0.3\t1\t63014 4342\n",
      "107\t62697\t0.3\t1\t58614 4083\n",
      "108\t60572\t0.3\t1\t56580 3992\n",
      "109\t54023\t0.3\t1\t50378 3645\n",
      "110\t49550\t0.3\t1\t46112 3438\n",
      "111\t45365\t0.3\t1\t42247 3118\n",
      "112\t40518\t0.3\t1\t37628 2890\n",
      "113\t35484\t0.3\t1\t32846 2638\n",
      "114\t32042\t0.3\t1\t29706 2336\n",
      "115\t28111\t0.3\t1\t26004 2107\n",
      "116\t25248\t0.3\t1\t23329 1919\n",
      "117\t21991\t0.3\t1\t20192 1799\n",
      "118\t18981\t0.3\t1\t17374 1607\n",
      "119\t16315\t0.3\t1\t14943 1372\n",
      "120\t14190\t0.3\t1\t13063 1127\n",
      "121\t12626\t0.3\t1\t11644 982\n",
      "122\t10330\t0.3\t1\t9443 887\n",
      "123\t9025\t0.3\t1\t8218 807\n",
      "124\t7634\t0.3\t1\t6995 639\n",
      "125\t6434\t0.3\t1\t5834 600\n",
      "126\t5520\t0.3\t1\t5002 518\n",
      "127\t4681\t0.3\t1\t4249 432\n",
      "128\t3676\t0.3\t1\t3257 419\n",
      "129\t3146\t0.3\t1\t2839 307\n",
      "130\t2431\t0.3\t1\t2189 242\n",
      "131\t2160\t0.3\t1\t1904 256\n",
      "132\t1916\t0.3\t1\t1719 197\n",
      "133\t2331\t0.3\t1\t2120 211\n",
      "134\t4696\t0.3\t1\t4441 255\n",
      "135\t1717\t0.3\t1\t1602 115\n",
      "136\t624\t0.3\t1\t562 62\n",
      "137\t434\t0.3\t1\t390 44\n",
      "138\t279\t0.3\t1\t239 40\n",
      "139\t193\t0.3\t1\t171 22\n",
      "140\t139\t0.3\t1\t121 18\n",
      "141\t116\t0.3\t1\t101 15\n",
      "142\t99\t0.3\t1\t87 12\n",
      "143\t71\t0.3\t1\t56 15\n",
      "144\t68\t0.3\t1\t49 19\n",
      "145\t142\t0.3\t1\t121 21\n",
      "146\t149\t0.3\t1\t136 13\n",
      "147\t732\t0.3\t1\t658 74\n",
      "148\t489\t0.3\t1\t451 38\n",
      "149\t276\t0.3\t1\t251 25\n",
      "150\t3068\t0.3\t1\t2637 431\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "16982389 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t19765 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 602.01 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,982,389\n",
      "Reads with adapters:                15,175,955 (89.4%)\n",
      "Reads written (passing filters):    16,982,389 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,547,358,350 bp\n",
      "Quality-trimmed:              14,637,859 bp (0.6%)\n",
      "Total written (filtered):  1,670,081,107 bp (65.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15175955 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.2%\n",
      "  C: 14.2%\n",
      "  G: 26.5%\n",
      "  T: 33.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t988355\t4245597.2\t0\t988355\n",
      "2\t174327\t1061399.3\t0\t174327\n",
      "3\t112326\t265349.8\t0\t112326\n",
      "4\t66944\t66337.5\t0\t66944\n",
      "5\t68049\t16584.4\t0\t68049\n",
      "6\t59622\t4146.1\t0\t59622\n",
      "7\t48077\t1036.5\t0\t48077\n",
      "8\t70632\t259.1\t0\t70632\n",
      "9\t56419\t64.8\t0\t56220 199\n",
      "10\t62892\t16.2\t1\t59906 2986\n",
      "11\t67246\t4.0\t1\t63200 4046\n",
      "12\t67488\t1.0\t1\t63307 4181\n",
      "13\t76798\t0.3\t1\t72553 4245\n",
      "14\t73938\t0.3\t1\t68873 5065\n",
      "15\t59370\t0.3\t1\t55860 3510\n",
      "16\t87943\t0.3\t1\t82783 5160\n",
      "17\t71376\t0.3\t1\t66695 4681\n",
      "18\t68792\t0.3\t1\t64885 3907\n",
      "19\t88616\t0.3\t1\t83090 5526\n",
      "20\t83852\t0.3\t1\t79094 4758\n",
      "21\t75082\t0.3\t1\t70636 4446\n",
      "22\t86387\t0.3\t1\t81609 4778\n",
      "23\t90257\t0.3\t1\t85055 5202\n",
      "24\t90691\t0.3\t1\t85295 5396\n",
      "25\t104055\t0.3\t1\t98264 5791\n",
      "26\t86753\t0.3\t1\t81712 5041\n",
      "27\t96821\t0.3\t1\t90516 6305\n",
      "28\t92589\t0.3\t1\t88280 4309\n",
      "29\t100006\t0.3\t1\t94283 5723\n",
      "30\t100693\t0.3\t1\t95722 4971\n",
      "31\t110238\t0.3\t1\t104335 5903\n",
      "32\t99509\t0.3\t1\t94661 4848\n",
      "33\t112333\t0.3\t1\t106128 6205\n",
      "34\t145441\t0.3\t1\t137472 7969\n",
      "35\t84427\t0.3\t1\t80249 4178\n",
      "36\t119286\t0.3\t1\t112593 6693\n",
      "37\t127293\t0.3\t1\t120329 6964\n",
      "38\t123047\t0.3\t1\t117261 5786\n",
      "39\t119775\t0.3\t1\t114281 5494\n",
      "40\t135696\t0.3\t1\t128262 7434\n",
      "41\t130689\t0.3\t1\t124366 6323\n",
      "42\t135604\t0.3\t1\t129108 6496\n",
      "43\t126835\t0.3\t1\t120741 6094\n",
      "44\t146922\t0.3\t1\t139741 7181\n",
      "45\t154339\t0.3\t1\t146909 7430\n",
      "46\t118308\t0.3\t1\t112832 5476\n",
      "47\t149295\t0.3\t1\t142485 6810\n",
      "48\t148606\t0.3\t1\t142089 6517\n",
      "49\t156890\t0.3\t1\t149693 7197\n",
      "50\t148375\t0.3\t1\t141320 7055\n",
      "51\t190171\t0.3\t1\t181254 8917\n",
      "52\t136804\t0.3\t1\t130968 5836\n",
      "53\t165906\t0.3\t1\t159264 6642\n",
      "54\t128157\t0.3\t1\t122871 5286\n",
      "55\t169343\t0.3\t1\t162704 6639\n",
      "56\t177473\t0.3\t1\t169804 7669\n",
      "57\t169211\t0.3\t1\t162646 6565\n",
      "58\t164197\t0.3\t1\t158052 6145\n",
      "59\t169249\t0.3\t1\t162429 6820\n",
      "60\t177455\t0.3\t1\t170133 7322\n",
      "61\t185762\t0.3\t1\t177933 7829\n",
      "62\t204728\t0.3\t1\t196280 8448\n",
      "63\t277535\t0.3\t1\t267196 10339\n",
      "64\t238674\t0.3\t1\t230905 7769\n",
      "65\t152399\t0.3\t1\t146631 5768\n",
      "66\t166909\t0.3\t1\t160634 6275\n",
      "67\t166804\t0.3\t1\t160075 6729\n",
      "68\t169255\t0.3\t1\t162913 6342\n",
      "69\t169940\t0.3\t1\t163142 6798\n",
      "70\t182641\t0.3\t1\t175571 7070\n",
      "71\t198941\t0.3\t1\t191661 7280\n",
      "72\t188343\t0.3\t1\t181180 7163\n",
      "73\t204850\t0.3\t1\t197436 7414\n",
      "74\t188381\t0.3\t1\t181668 6713\n",
      "75\t180633\t0.3\t1\t173941 6692\n",
      "76\t178500\t0.3\t1\t172078 6422\n",
      "77\t180330\t0.3\t1\t173546 6784\n",
      "78\t173641\t0.3\t1\t167180 6461\n",
      "79\t170287\t0.3\t1\t164090 6197\n",
      "80\t170838\t0.3\t1\t164653 6185\n",
      "81\t172175\t0.3\t1\t165934 6241\n",
      "82\t165553\t0.3\t1\t159555 5998\n",
      "83\t164742\t0.3\t1\t158792 5950\n",
      "84\t160682\t0.3\t1\t154778 5904\n",
      "85\t159281\t0.3\t1\t153367 5914\n",
      "86\t160523\t0.3\t1\t154638 5885\n",
      "87\t163301\t0.3\t1\t157413 5888\n",
      "88\t161676\t0.3\t1\t155871 5805\n",
      "89\t147453\t0.3\t1\t142105 5348\n",
      "90\t143818\t0.3\t1\t138625 5193\n",
      "91\t138225\t0.3\t1\t133103 5122\n",
      "92\t131088\t0.3\t1\t126317 4771\n",
      "93\t128053\t0.3\t1\t123615 4438\n",
      "94\t125505\t0.3\t1\t121074 4431\n",
      "95\t119294\t0.3\t1\t114964 4330\n",
      "96\t116155\t0.3\t1\t112159 3996\n",
      "97\t108812\t0.3\t1\t105094 3718\n",
      "98\t104976\t0.3\t1\t101373 3603\n",
      "99\t105895\t0.3\t1\t102281 3614\n",
      "100\t97778\t0.3\t1\t94611 3167\n",
      "101\t95386\t0.3\t1\t92351 3035\n",
      "102\t88691\t0.3\t1\t85785 2906\n",
      "103\t80552\t0.3\t1\t77853 2699\n",
      "104\t77505\t0.3\t1\t74941 2564\n",
      "105\t70382\t0.3\t1\t68000 2382\n",
      "106\t65184\t0.3\t1\t63051 2133\n",
      "107\t60057\t0.3\t1\t58146 1911\n",
      "108\t55681\t0.3\t1\t53827 1854\n",
      "109\t50629\t0.3\t1\t48896 1733\n",
      "110\t46572\t0.3\t1\t45046 1526\n",
      "111\t44577\t0.3\t1\t43131 1446\n",
      "112\t39344\t0.3\t1\t38114 1230\n",
      "113\t34530\t0.3\t1\t33382 1148\n",
      "114\t30993\t0.3\t1\t30031 962\n",
      "115\t26355\t0.3\t1\t25472 883\n",
      "116\t23180\t0.3\t1\t22328 852\n",
      "117\t19825\t0.3\t1\t19141 684\n",
      "118\t17184\t0.3\t1\t16546 638\n",
      "119\t14938\t0.3\t1\t14436 502\n",
      "120\t12856\t0.3\t1\t12387 469\n",
      "121\t11667\t0.3\t1\t11246 421\n",
      "122\t9613\t0.3\t1\t9258 355\n",
      "123\t8582\t0.3\t1\t8274 308\n",
      "124\t7156\t0.3\t1\t6897 259\n",
      "125\t6130\t0.3\t1\t5897 233\n",
      "126\t5195\t0.3\t1\t5039 156\n",
      "127\t4372\t0.3\t1\t4226 146\n",
      "128\t3397\t0.3\t1\t3245 152\n",
      "129\t2899\t0.3\t1\t2785 114\n",
      "130\t2231\t0.3\t1\t2140 91\n",
      "131\t2005\t0.3\t1\t1925 80\n",
      "132\t1815\t0.3\t1\t1740 75\n",
      "133\t2411\t0.3\t1\t2321 90\n",
      "134\t4708\t0.3\t1\t4577 131\n",
      "135\t1657\t0.3\t1\t1601 56\n",
      "136\t589\t0.3\t1\t574 15\n",
      "137\t406\t0.3\t1\t391 15\n",
      "138\t265\t0.3\t1\t252 13\n",
      "139\t182\t0.3\t1\t175 7\n",
      "140\t132\t0.3\t1\t124 8\n",
      "141\t111\t0.3\t1\t106 5\n",
      "142\t101\t0.3\t1\t96 5\n",
      "143\t63\t0.3\t1\t60 3\n",
      "144\t61\t0.3\t1\t57 4\n",
      "145\t160\t0.3\t1\t154 6\n",
      "146\t146\t0.3\t1\t139 7\n",
      "147\t687\t0.3\t1\t666 21\n",
      "148\t439\t0.3\t1\t426 13\n",
      "149\t260\t0.3\t1\t251 9\n",
      "150\t2448\t0.3\t1\t2325 123\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "16982389 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t41433 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 587.68 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,972,486\n",
      "Reads with adapters:                15,279,419 (90.0%)\n",
      "Reads written (passing filters):    16,972,486 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,545,872,900 bp\n",
      "Quality-trimmed:               5,170,443 bp (0.2%)\n",
      "Total written (filtered):  1,660,911,595 bp (65.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15279419 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 17.9%\n",
      "  G: 21.2%\n",
      "  T: 32.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t937714\t4243121.5\t0\t937714\n",
      "2\t159877\t1060780.4\t0\t159877\n",
      "3\t104645\t265195.1\t0\t104645\n",
      "4\t64394\t66298.8\t0\t64394\n",
      "5\t60206\t16574.7\t0\t60206\n",
      "6\t60143\t4143.7\t0\t60143\n",
      "7\t49779\t1035.9\t0\t49779\n",
      "8\t63607\t259.0\t0\t63607\n",
      "9\t63633\t64.7\t0\t62858 775\n",
      "10\t59492\t16.2\t1\t55609 3883\n",
      "11\t70292\t4.0\t1\t63860 6432\n",
      "12\t65718\t1.0\t1\t60580 5138\n",
      "13\t68404\t0.3\t1\t63070 5334\n",
      "14\t69496\t0.3\t1\t63637 5859\n",
      "15\t74672\t0.3\t1\t68604 6068\n",
      "16\t72665\t0.3\t1\t66207 6458\n",
      "17\t90074\t0.3\t1\t81360 8714\n",
      "18\t80053\t0.3\t1\t74306 5747\n",
      "19\t65594\t0.3\t1\t60638 4956\n",
      "20\t83351\t0.3\t1\t76880 6471\n",
      "21\t92562\t0.3\t1\t85115 7447\n",
      "22\t78449\t0.3\t1\t72991 5458\n",
      "23\t93136\t0.3\t1\t85970 7166\n",
      "24\t87973\t0.3\t1\t80003 7970\n",
      "25\t95468\t0.3\t1\t87873 7595\n",
      "26\t87079\t0.3\t1\t80742 6337\n",
      "27\t100093\t0.3\t1\t91614 8479\n",
      "28\t99879\t0.3\t1\t92166 7713\n",
      "29\t105478\t0.3\t1\t95962 9516\n",
      "30\t116678\t0.3\t1\t108856 7822\n",
      "31\t86678\t0.3\t1\t80098 6580\n",
      "32\t107759\t0.3\t1\t99613 8146\n",
      "33\t109432\t0.3\t1\t99874 9558\n",
      "34\t138284\t0.3\t1\t125790 12494\n",
      "35\t117450\t0.3\t1\t109376 8074\n",
      "36\t91614\t0.3\t1\t84735 6879\n",
      "37\t124777\t0.3\t1\t114386 10391\n",
      "38\t138573\t0.3\t1\t126348 12225\n",
      "39\t118413\t0.3\t1\t109918 8495\n",
      "40\t138324\t0.3\t1\t126104 12220\n",
      "41\t158476\t0.3\t1\t144761 13715\n",
      "42\t109596\t0.3\t1\t99995 9601\n",
      "43\t214148\t0.3\t1\t200447 13701\n",
      "44\t58836\t0.3\t1\t52468 6368\n",
      "45\t143215\t0.3\t1\t132338 10877\n",
      "46\t142842\t0.3\t1\t132071 10771\n",
      "47\t146328\t0.3\t1\t136022 10306\n",
      "48\t147735\t0.3\t1\t136182 11553\n",
      "49\t164993\t0.3\t1\t151819 13174\n",
      "50\t138531\t0.3\t1\t129039 9492\n",
      "51\t154359\t0.3\t1\t142921 11438\n",
      "52\t163104\t0.3\t1\t149477 13627\n",
      "53\t181208\t0.3\t1\t168727 12481\n",
      "54\t135823\t0.3\t1\t125879 9944\n",
      "55\t172933\t0.3\t1\t159987 12946\n",
      "56\t212770\t0.3\t1\t198564 14206\n",
      "57\t137905\t0.3\t1\t126707 11198\n",
      "58\t179533\t0.3\t1\t168232 11301\n",
      "59\t154787\t0.3\t1\t143238 11549\n",
      "60\t178048\t0.3\t1\t165291 12757\n",
      "61\t178798\t0.3\t1\t166319 12479\n",
      "62\t194586\t0.3\t1\t181612 12974\n",
      "63\t159546\t0.3\t1\t147478 12068\n",
      "64\t175838\t0.3\t1\t164219 11619\n",
      "65\t199150\t0.3\t1\t185246 13904\n",
      "66\t207681\t0.3\t1\t193071 14610\n",
      "67\t227820\t0.3\t1\t208839 18981\n",
      "68\t430170\t0.3\t1\t406697 23473\n",
      "69\t101761\t0.3\t1\t93311 8450\n",
      "70\t37741\t0.3\t1\t31764 5977\n",
      "71\t116561\t0.3\t1\t108649 7912\n",
      "72\t170878\t0.3\t1\t160245 10633\n",
      "73\t185808\t0.3\t1\t173840 11968\n",
      "74\t185713\t0.3\t1\t173236 12477\n",
      "75\t186685\t0.3\t1\t173898 12787\n",
      "76\t187774\t0.3\t1\t174524 13250\n",
      "77\t190253\t0.3\t1\t177207 13046\n",
      "78\t180228\t0.3\t1\t167656 12572\n",
      "79\t181970\t0.3\t1\t169235 12735\n",
      "80\t180269\t0.3\t1\t167504 12765\n",
      "81\t180129\t0.3\t1\t167834 12295\n",
      "82\t176778\t0.3\t1\t164262 12516\n",
      "83\t174648\t0.3\t1\t162652 11996\n",
      "84\t170683\t0.3\t1\t158618 12065\n",
      "85\t167207\t0.3\t1\t155648 11559\n",
      "86\t165775\t0.3\t1\t154402 11373\n",
      "87\t159710\t0.3\t1\t148769 10941\n",
      "88\t161141\t0.3\t1\t150001 11140\n",
      "89\t154725\t0.3\t1\t143979 10746\n",
      "90\t156314\t0.3\t1\t145746 10568\n",
      "91\t151370\t0.3\t1\t140729 10641\n",
      "92\t145834\t0.3\t1\t135746 10088\n",
      "93\t137534\t0.3\t1\t127986 9548\n",
      "94\t136265\t0.3\t1\t126825 9440\n",
      "95\t127646\t0.3\t1\t118675 8971\n",
      "96\t123166\t0.3\t1\t114494 8672\n",
      "97\t117318\t0.3\t1\t109181 8137\n",
      "98\t111017\t0.3\t1\t103130 7887\n",
      "99\t109081\t0.3\t1\t101041 8040\n",
      "100\t101164\t0.3\t1\t94080 7084\n",
      "101\t96193\t0.3\t1\t89369 6824\n",
      "102\t90471\t0.3\t1\t84087 6384\n",
      "103\t85738\t0.3\t1\t79491 6247\n",
      "104\t80436\t0.3\t1\t74816 5620\n",
      "105\t72717\t0.3\t1\t67687 5030\n",
      "106\t67684\t0.3\t1\t62759 4925\n",
      "107\t62704\t0.3\t1\t58171 4533\n",
      "108\t59951\t0.3\t1\t55617 4334\n",
      "109\t54357\t0.3\t1\t50236 4121\n",
      "110\t50115\t0.3\t1\t46382 3733\n",
      "111\t45371\t0.3\t1\t41955 3416\n",
      "112\t40800\t0.3\t1\t37574 3226\n",
      "113\t35447\t0.3\t1\t32646 2801\n",
      "114\t32325\t0.3\t1\t29802 2523\n",
      "115\t28709\t0.3\t1\t26438 2271\n",
      "116\t25389\t0.3\t1\t23337 2052\n",
      "117\t21950\t0.3\t1\t20122 1828\n",
      "118\t19200\t0.3\t1\t17593 1607\n",
      "119\t16278\t0.3\t1\t14896 1382\n",
      "120\t14121\t0.3\t1\t12882 1239\n",
      "121\t12963\t0.3\t1\t11779 1184\n",
      "122\t10628\t0.3\t1\t9652 976\n",
      "123\t9232\t0.3\t1\t8435 797\n",
      "124\t7617\t0.3\t1\t6893 724\n",
      "125\t6474\t0.3\t1\t5870 604\n",
      "126\t5569\t0.3\t1\t5024 545\n",
      "127\t4721\t0.3\t1\t4238 483\n",
      "128\t3597\t0.3\t1\t3214 383\n",
      "129\t3083\t0.3\t1\t2780 303\n",
      "130\t2334\t0.3\t1\t2063 271\n",
      "131\t2035\t0.3\t1\t1816 219\n",
      "132\t1850\t0.3\t1\t1686 164\n",
      "133\t2327\t0.3\t1\t2083 244\n",
      "134\t4554\t0.3\t1\t4279 275\n",
      "135\t1805\t0.3\t1\t1652 153\n",
      "136\t606\t0.3\t1\t541 65\n",
      "137\t448\t0.3\t1\t385 63\n",
      "138\t325\t0.3\t1\t262 63\n",
      "139\t199\t0.3\t1\t170 29\n",
      "140\t136\t0.3\t1\t115 21\n",
      "141\t92\t0.3\t1\t70 22\n",
      "142\t61\t0.3\t1\t44 17\n",
      "143\t68\t0.3\t1\t60 8\n",
      "144\t63\t0.3\t1\t59 4\n",
      "145\t123\t0.3\t1\t108 15\n",
      "146\t160\t0.3\t1\t140 20\n",
      "147\t706\t0.3\t1\t621 85\n",
      "148\t490\t0.3\t1\t450 40\n",
      "149\t296\t0.3\t1\t261 35\n",
      "150\t3088\t0.3\t1\t2616 472\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "16972486 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t19505 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 616.33 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,972,486\n",
      "Reads with adapters:                15,133,409 (89.2%)\n",
      "Reads written (passing filters):    16,972,486 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,545,872,900 bp\n",
      "Quality-trimmed:              14,118,915 bp (0.6%)\n",
      "Total written (filtered):  1,669,911,692 bp (65.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15133409 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.2%\n",
      "  C: 14.2%\n",
      "  G: 26.5%\n",
      "  T: 33.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t991330\t4243121.5\t0\t991330\n",
      "2\t172347\t1060780.4\t0\t172347\n",
      "3\t115382\t265195.1\t0\t115382\n",
      "4\t65389\t66298.8\t0\t65389\n",
      "5\t65284\t16574.7\t0\t65284\n",
      "6\t59675\t4143.7\t0\t59675\n",
      "7\t47228\t1035.9\t0\t47228\n",
      "8\t69329\t259.0\t0\t69329\n",
      "9\t55587\t64.7\t0\t55396 191\n",
      "10\t60044\t16.2\t1\t55911 4133\n",
      "11\t66669\t4.0\t1\t61266 5403\n",
      "12\t66198\t1.0\t1\t60904 5294\n",
      "13\t76711\t0.3\t1\t70697 6014\n",
      "14\t72459\t0.3\t1\t66111 6348\n",
      "15\t58334\t0.3\t1\t54019 4315\n",
      "16\t86573\t0.3\t1\t79901 6672\n",
      "17\t73457\t0.3\t1\t67527 5930\n",
      "18\t65883\t0.3\t1\t60982 4901\n",
      "19\t88155\t0.3\t1\t81417 6738\n",
      "20\t82480\t0.3\t1\t76391 6089\n",
      "21\t74222\t0.3\t1\t68897 5325\n",
      "22\t85010\t0.3\t1\t78866 6144\n",
      "23\t89940\t0.3\t1\t83333 6607\n",
      "24\t90783\t0.3\t1\t83898 6885\n",
      "25\t103545\t0.3\t1\t96553 6992\n",
      "26\t86049\t0.3\t1\t79414 6635\n",
      "27\t97578\t0.3\t1\t89843 7735\n",
      "28\t88487\t0.3\t1\t82899 5588\n",
      "29\t101443\t0.3\t1\t94374 7069\n",
      "30\t98065\t0.3\t1\t91540 6525\n",
      "31\t110927\t0.3\t1\t103494 7433\n",
      "32\t96042\t0.3\t1\t90040 6002\n",
      "33\t113007\t0.3\t1\t105053 7954\n",
      "34\t146678\t0.3\t1\t136868 9810\n",
      "35\t80284\t0.3\t1\t75023 5261\n",
      "36\t119636\t0.3\t1\t111148 8488\n",
      "37\t126797\t0.3\t1\t118353 8444\n",
      "38\t121741\t0.3\t1\t114473 7268\n",
      "39\t116427\t0.3\t1\t109798 6629\n",
      "40\t134566\t0.3\t1\t125810 8756\n",
      "41\t132132\t0.3\t1\t124200 7932\n",
      "42\t135345\t0.3\t1\t127009 8336\n",
      "43\t125343\t0.3\t1\t118018 7325\n",
      "44\t147397\t0.3\t1\t138044 9353\n",
      "45\t153169\t0.3\t1\t143585 9584\n",
      "46\t116858\t0.3\t1\t110003 6855\n",
      "47\t148662\t0.3\t1\t140422 8240\n",
      "48\t146456\t0.3\t1\t138141 8315\n",
      "49\t155663\t0.3\t1\t146241 9422\n",
      "50\t150144\t0.3\t1\t140998 9146\n",
      "51\t186605\t0.3\t1\t175426 11179\n",
      "52\t135487\t0.3\t1\t127796 7691\n",
      "53\t169781\t0.3\t1\t161047 8734\n",
      "54\t125345\t0.3\t1\t118710 6635\n",
      "55\t168692\t0.3\t1\t160003 8689\n",
      "56\t176257\t0.3\t1\t166258 9999\n",
      "57\t169891\t0.3\t1\t161571 8320\n",
      "58\t164276\t0.3\t1\t156000 8276\n",
      "59\t168462\t0.3\t1\t159572 8890\n",
      "60\t177599\t0.3\t1\t168506 9093\n",
      "61\t183440\t0.3\t1\t174203 9237\n",
      "62\t202372\t0.3\t1\t191386 10986\n",
      "63\t270936\t0.3\t1\t258181 12755\n",
      "64\t234996\t0.3\t1\t225774 9222\n",
      "65\t151631\t0.3\t1\t144050 7581\n",
      "66\t166302\t0.3\t1\t157972 8330\n",
      "67\t167833\t0.3\t1\t159201 8632\n",
      "68\t170330\t0.3\t1\t161782 8548\n",
      "69\t171332\t0.3\t1\t162512 8820\n",
      "70\t183649\t0.3\t1\t174535 9114\n",
      "71\t199117\t0.3\t1\t189420 9697\n",
      "72\t190292\t0.3\t1\t180708 9584\n",
      "73\t205025\t0.3\t1\t195024 10001\n",
      "74\t188852\t0.3\t1\t179445 9407\n",
      "75\t181257\t0.3\t1\t172280 8977\n",
      "76\t178734\t0.3\t1\t169881 8853\n",
      "77\t180452\t0.3\t1\t171277 9175\n",
      "78\t172679\t0.3\t1\t164051 8628\n",
      "79\t170532\t0.3\t1\t162087 8445\n",
      "80\t171521\t0.3\t1\t163340 8181\n",
      "81\t171140\t0.3\t1\t162630 8510\n",
      "82\t165207\t0.3\t1\t157233 7974\n",
      "83\t163608\t0.3\t1\t155825 7783\n",
      "84\t160013\t0.3\t1\t152625 7388\n",
      "85\t159454\t0.3\t1\t151925 7529\n",
      "86\t161481\t0.3\t1\t153907 7574\n",
      "87\t162581\t0.3\t1\t155126 7455\n",
      "88\t162054\t0.3\t1\t154673 7381\n",
      "89\t147788\t0.3\t1\t141132 6656\n",
      "90\t145172\t0.3\t1\t138698 6474\n",
      "91\t139015\t0.3\t1\t132745 6270\n",
      "92\t132299\t0.3\t1\t126316 5983\n",
      "93\t128552\t0.3\t1\t122735 5817\n",
      "94\t126711\t0.3\t1\t120900 5811\n",
      "95\t119947\t0.3\t1\t114454 5493\n",
      "96\t117162\t0.3\t1\t111939 5223\n",
      "97\t108989\t0.3\t1\t104182 4807\n",
      "98\t105848\t0.3\t1\t101282 4566\n",
      "99\t105989\t0.3\t1\t101349 4640\n",
      "100\t98144\t0.3\t1\t93908 4236\n",
      "101\t94920\t0.3\t1\t90864 4056\n",
      "102\t89211\t0.3\t1\t85259 3952\n",
      "103\t81188\t0.3\t1\t77654 3534\n",
      "104\t77565\t0.3\t1\t74081 3484\n",
      "105\t70248\t0.3\t1\t67249 2999\n",
      "106\t65288\t0.3\t1\t62604 2684\n",
      "107\t60022\t0.3\t1\t57428 2594\n",
      "108\t55515\t0.3\t1\t53019 2496\n",
      "109\t50990\t0.3\t1\t48728 2262\n",
      "110\t46869\t0.3\t1\t44831 2038\n",
      "111\t44489\t0.3\t1\t42630 1859\n",
      "112\t39619\t0.3\t1\t37888 1731\n",
      "113\t34543\t0.3\t1\t33015 1528\n",
      "114\t31368\t0.3\t1\t30023 1345\n",
      "115\t26935\t0.3\t1\t25774 1161\n",
      "116\t23611\t0.3\t1\t22614 997\n",
      "117\t19912\t0.3\t1\t18993 919\n",
      "118\t17572\t0.3\t1\t16789 783\n",
      "119\t14871\t0.3\t1\t14194 677\n",
      "120\t12844\t0.3\t1\t12244 600\n",
      "121\t11858\t0.3\t1\t11313 545\n",
      "122\t9930\t0.3\t1\t9463 467\n",
      "123\t8753\t0.3\t1\t8342 411\n",
      "124\t7126\t0.3\t1\t6812 314\n",
      "125\t6158\t0.3\t1\t5890 268\n",
      "126\t5177\t0.3\t1\t4954 223\n",
      "127\t4434\t0.3\t1\t4233 201\n",
      "128\t3338\t0.3\t1\t3185 153\n",
      "129\t2850\t0.3\t1\t2742 108\n",
      "130\t2111\t0.3\t1\t2001 110\n",
      "131\t1887\t0.3\t1\t1807 80\n",
      "132\t1727\t0.3\t1\t1658 69\n",
      "133\t2467\t0.3\t1\t2378 89\n",
      "134\t4553\t0.3\t1\t4401 152\n",
      "135\t1770\t0.3\t1\t1714 56\n",
      "136\t598\t0.3\t1\t559 39\n",
      "137\t389\t0.3\t1\t374 15\n",
      "138\t283\t0.3\t1\t271 12\n",
      "139\t188\t0.3\t1\t181 7\n",
      "140\t133\t0.3\t1\t124 9\n",
      "141\t75\t0.3\t1\t73 2\n",
      "142\t64\t0.3\t1\t61 3\n",
      "143\t64\t0.3\t1\t59 5\n",
      "144\t68\t0.3\t1\t66 2\n",
      "145\t139\t0.3\t1\t132 7\n",
      "146\t159\t0.3\t1\t151 8\n",
      "147\t678\t0.3\t1\t647 31\n",
      "148\t467\t0.3\t1\t450 17\n",
      "149\t278\t0.3\t1\t263 15\n",
      "150\t2366\t0.3\t1\t2224 142\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "16972486 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t39693 (0.2%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 5053.89 s (45 us/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             113,474,961\n",
      "Reads with adapters:                78,847,369 (69.5%)\n",
      "Reads written (passing filters):   113,474,961 (100.0%)\n",
      "\n",
      "Total basepairs processed: 17,021,244,150 bp\n",
      "Quality-trimmed:              22,715,789 bp (0.1%)\n",
      "Total written (filtered):  14,426,732,983 bp (84.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78847369 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.1%\n",
      "  C: 9.7%\n",
      "  G: 19.7%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t22838934\t28368740.2\t0\t22838934\n",
      "2\t2342589\t7092185.1\t0\t2342589\n",
      "3\t1202279\t1773046.3\t0\t1202279\n",
      "4\t750593\t443261.6\t0\t750593\n",
      "5\t559528\t110815.4\t0\t559528\n",
      "6\t524826\t27703.8\t0\t524826\n",
      "7\t449332\t6926.0\t0\t449332\n",
      "8\t548899\t1731.5\t0\t548899\n",
      "9\t548856\t432.9\t0\t541308 7548\n",
      "10\t508456\t108.2\t1\t478983 29473\n",
      "11\t562956\t27.1\t1\t517668 45288\n",
      "12\t559587\t6.8\t1\t520090 39497\n",
      "13\t547304\t1.7\t1\t508519 38785\n",
      "14\t570426\t1.7\t1\t525734 44692\n",
      "15\t575858\t1.7\t1\t534867 40991\n",
      "16\t569860\t1.7\t1\t523134 46726\n",
      "17\t678914\t1.7\t1\t621846 57068\n",
      "18\t565739\t1.7\t1\t526451 39288\n",
      "19\t504906\t1.7\t1\t468246 36660\n",
      "20\t580103\t1.7\t1\t539012 41091\n",
      "21\t633303\t1.7\t1\t584154 49149\n",
      "22\t551495\t1.7\t1\t516023 35472\n",
      "23\t623244\t1.7\t1\t577696 45548\n",
      "24\t597040\t1.7\t1\t545080 51960\n",
      "25\t610942\t1.7\t1\t565949 44993\n",
      "26\t578327\t1.7\t1\t539305 39022\n",
      "27\t631060\t1.7\t1\t581266 49794\n",
      "28\t620380\t1.7\t1\t573987 46393\n",
      "29\t630417\t1.7\t1\t577901 52516\n",
      "30\t665372\t1.7\t1\t624016 41356\n",
      "31\t521479\t1.7\t1\t485824 35655\n",
      "32\t637552\t1.7\t1\t593972 43580\n",
      "33\t605503\t1.7\t1\t560522 44981\n",
      "34\t753213\t1.7\t1\t699064 54149\n",
      "35\t512573\t1.7\t1\t477265 35308\n",
      "36\t633868\t1.7\t1\t587959 45909\n",
      "37\t659372\t1.7\t1\t612861 46511\n",
      "38\t603868\t1.7\t1\t563307 40561\n",
      "39\t656560\t1.7\t1\t609454 47106\n",
      "40\t670911\t1.7\t1\t620635 50276\n",
      "41\t733444\t1.7\t1\t678908 54536\n",
      "42\t554355\t1.7\t1\t510902 43453\n",
      "43\t1004589\t1.7\t1\t949034 55555\n",
      "44\t281497\t1.7\t1\t254795 26702\n",
      "45\t650424\t1.7\t1\t607820 42604\n",
      "46\t622059\t1.7\t1\t580617 41442\n",
      "47\t641812\t1.7\t1\t599651 42161\n",
      "48\t616364\t1.7\t1\t574406 41958\n",
      "49\t673245\t1.7\t1\t623804 49441\n",
      "50\t604428\t1.7\t1\t565417 39011\n",
      "51\t645021\t1.7\t1\t600411 44610\n",
      "52\t648661\t1.7\t1\t600394 48267\n",
      "53\t693327\t1.7\t1\t647436 45891\n",
      "54\t550715\t1.7\t1\t514274 36441\n",
      "55\t631316\t1.7\t1\t588270 43046\n",
      "56\t767345\t1.7\t1\t719044 48301\n",
      "57\t493710\t1.7\t1\t456524 37186\n",
      "58\t647241\t1.7\t1\t610352 36889\n",
      "59\t560612\t1.7\t1\t522206 38406\n",
      "60\t620281\t1.7\t1\t578774 41507\n",
      "61\t603855\t1.7\t1\t565232 38623\n",
      "62\t664795\t1.7\t1\t623425 41370\n",
      "63\t508452\t1.7\t1\t471688 36764\n",
      "64\t573553\t1.7\t1\t538426 35127\n",
      "65\t625525\t1.7\t1\t584793 40732\n",
      "66\t654791\t1.7\t1\t611542 43249\n",
      "67\t693655\t1.7\t1\t637398 56257\n",
      "68\t1295847\t1.7\t1\t1228868 66979\n",
      "69\t207198\t1.7\t1\t188952 18246\n",
      "70\t83872\t1.7\t1\t69823 14049\n",
      "71\t351217\t1.7\t1\t328576 22641\n",
      "72\t486564\t1.7\t1\t456539 30025\n",
      "73\t511552\t1.7\t1\t479835 31717\n",
      "74\t500225\t1.7\t1\t468801 31424\n",
      "75\t493734\t1.7\t1\t462637 31097\n",
      "76\t485479\t1.7\t1\t455415 30064\n",
      "77\t472130\t1.7\t1\t442425 29705\n",
      "78\t457249\t1.7\t1\t427856 29393\n",
      "79\t452598\t1.7\t1\t423758 28840\n",
      "80\t436888\t1.7\t1\t409466 27422\n",
      "81\t424029\t1.7\t1\t397325 26704\n",
      "82\t413331\t1.7\t1\t387262 26069\n",
      "83\t395648\t1.7\t1\t370721 24927\n",
      "84\t386472\t1.7\t1\t362163 24309\n",
      "85\t373564\t1.7\t1\t350510 23054\n",
      "86\t350177\t1.7\t1\t327542 22635\n",
      "87\t338817\t1.7\t1\t317459 21358\n",
      "88\t329564\t1.7\t1\t308892 20672\n",
      "89\t310326\t1.7\t1\t290522 19804\n",
      "90\t302165\t1.7\t1\t282977 19188\n",
      "91\t286619\t1.7\t1\t268441 18178\n",
      "92\t275091\t1.7\t1\t257788 17303\n",
      "93\t261850\t1.7\t1\t245237 16613\n",
      "94\t246459\t1.7\t1\t230780 15679\n",
      "95\t229270\t1.7\t1\t214573 14697\n",
      "96\t213682\t1.7\t1\t200015 13667\n",
      "97\t200167\t1.7\t1\t187352 12815\n",
      "98\t186673\t1.7\t1\t174782 11891\n",
      "99\t177389\t1.7\t1\t165973 11416\n",
      "100\t164170\t1.7\t1\t153549 10621\n",
      "101\t152020\t1.7\t1\t142083 9937\n",
      "102\t138827\t1.7\t1\t129592 9235\n",
      "103\t126885\t1.7\t1\t118589 8296\n",
      "104\t114180\t1.7\t1\t106471 7709\n",
      "105\t103407\t1.7\t1\t96323 7084\n",
      "106\t93053\t1.7\t1\t86648 6405\n",
      "107\t82305\t1.7\t1\t76807 5498\n",
      "108\t73960\t1.7\t1\t68774 5186\n",
      "109\t66073\t1.7\t1\t61323 4750\n",
      "110\t58316\t1.7\t1\t54124 4192\n",
      "111\t51210\t1.7\t1\t47563 3647\n",
      "112\t46457\t1.7\t1\t43051 3406\n",
      "113\t39166\t1.7\t1\t36179 2987\n",
      "114\t35653\t1.7\t1\t32939 2714\n",
      "115\t29363\t1.7\t1\t27078 2285\n",
      "116\t25352\t1.7\t1\t23356 1996\n",
      "117\t20912\t1.7\t1\t19235 1677\n",
      "118\t17421\t1.7\t1\t15974 1447\n",
      "119\t14864\t1.7\t1\t13576 1288\n",
      "120\t12650\t1.7\t1\t11578 1072\n",
      "121\t10471\t1.7\t1\t9498 973\n",
      "122\t8802\t1.7\t1\t7972 830\n",
      "123\t7467\t1.7\t1\t6769 698\n",
      "124\t6267\t1.7\t1\t5666 601\n",
      "125\t5296\t1.7\t1\t4740 556\n",
      "126\t5081\t1.7\t1\t4533 548\n",
      "127\t4350\t1.7\t1\t3958 392\n",
      "128\t3255\t1.7\t1\t2949 306\n",
      "129\t2684\t1.7\t1\t2428 256\n",
      "130\t3108\t1.7\t1\t2815 293\n",
      "131\t4344\t1.7\t1\t4012 332\n",
      "132\t4969\t1.7\t1\t4639 330\n",
      "133\t7178\t1.7\t1\t6729 449\n",
      "134\t9066\t1.7\t1\t8514 552\n",
      "135\t3718\t1.7\t1\t3456 262\n",
      "136\t1190\t1.7\t1\t1067 123\n",
      "137\t629\t1.7\t1\t542 87\n",
      "138\t481\t1.7\t1\t425 56\n",
      "139\t351\t1.7\t1\t310 41\n",
      "140\t237\t1.7\t1\t196 41\n",
      "141\t164\t1.7\t1\t133 31\n",
      "142\t89\t1.7\t1\t63 26\n",
      "143\t91\t1.7\t1\t63 28\n",
      "144\t96\t1.7\t1\t72 24\n",
      "145\t237\t1.7\t1\t190 47\n",
      "146\t222\t1.7\t1\t196 26\n",
      "147\t852\t1.7\t1\t739 113\n",
      "148\t478\t1.7\t1\t374 104\n",
      "149\t260\t1.7\t1\t214 46\n",
      "150\t2350\t1.7\t1\t1965 385\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "113474961 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t37240 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 5% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 5296.87 s (47 us/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             113,474,961\n",
      "Reads with adapters:                79,235,271 (69.8%)\n",
      "Reads written (passing filters):   113,474,961 (100.0%)\n",
      "\n",
      "Total basepairs processed: 17,021,244,150 bp\n",
      "Quality-trimmed:              73,052,751 bp (0.4%)\n",
      "Total written (filtered):  14,433,737,028 bp (84.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 79235271 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 9.2%\n",
      "  G: 21.3%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t24292490\t28368740.2\t0\t24292490\n",
      "2\t2172289\t7092185.1\t0\t2172289\n",
      "3\t1212765\t1773046.3\t0\t1212765\n",
      "4\t728934\t443261.6\t0\t728934\n",
      "5\t615953\t110815.4\t0\t615953\n",
      "6\t518496\t27703.8\t0\t518496\n",
      "7\t424176\t6926.0\t0\t424176\n",
      "8\t595932\t1731.5\t0\t595932\n",
      "9\t493164\t432.9\t0\t491188 1976\n",
      "10\t515557\t108.2\t1\t490642 24915\n",
      "11\t544480\t27.1\t1\t510284 34196\n",
      "12\t557302\t6.8\t1\t522559 34743\n",
      "13\t624852\t1.7\t1\t586293 38559\n",
      "14\t566086\t1.7\t1\t527536 38550\n",
      "15\t479671\t1.7\t1\t447634 32037\n",
      "16\t677197\t1.7\t1\t634485 42712\n",
      "17\t518178\t1.7\t1\t482968 35210\n",
      "18\t513709\t1.7\t1\t481055 32654\n",
      "19\t623697\t1.7\t1\t583105 40592\n",
      "20\t590778\t1.7\t1\t553837 36941\n",
      "21\t523012\t1.7\t1\t489156 33856\n",
      "22\t582166\t1.7\t1\t546634 35532\n",
      "23\t607194\t1.7\t1\t567433 39761\n",
      "24\t596018\t1.7\t1\t558835 37183\n",
      "25\t658603\t1.7\t1\t618497 40106\n",
      "26\t563795\t1.7\t1\t525767 38028\n",
      "27\t625280\t1.7\t1\t580031 45249\n",
      "28\t565755\t1.7\t1\t537011 28744\n",
      "29\t604057\t1.7\t1\t566572 37485\n",
      "30\t575342\t1.7\t1\t545309 30033\n",
      "31\t637117\t1.7\t1\t598679 38438\n",
      "32\t567491\t1.7\t1\t538376 29115\n",
      "33\t633139\t1.7\t1\t594872 38267\n",
      "34\t807925\t1.7\t1\t759853 48072\n",
      "35\t450122\t1.7\t1\t426055 24067\n",
      "36\t611610\t1.7\t1\t574682 36928\n",
      "37\t655880\t1.7\t1\t616910 38970\n",
      "38\t600986\t1.7\t1\t571449 29537\n",
      "39\t595629\t1.7\t1\t565252 30377\n",
      "40\t651655\t1.7\t1\t614241 37414\n",
      "41\t634336\t1.7\t1\t600514 33822\n",
      "42\t641088\t1.7\t1\t607660 33428\n",
      "43\t603840\t1.7\t1\t570768 33072\n",
      "44\t681587\t1.7\t1\t644069 37518\n",
      "45\t707281\t1.7\t1\t670903 36378\n",
      "46\t517703\t1.7\t1\t489820 27883\n",
      "47\t633055\t1.7\t1\t601375 31680\n",
      "48\t613751\t1.7\t1\t584361 29390\n",
      "49\t644336\t1.7\t1\t611009 33327\n",
      "50\t626710\t1.7\t1\t592951 33759\n",
      "51\t770564\t1.7\t1\t731245 39319\n",
      "52\t552184\t1.7\t1\t525737 26447\n",
      "53\t655644\t1.7\t1\t624724 30920\n",
      "54\t497055\t1.7\t1\t473150 23905\n",
      "55\t622404\t1.7\t1\t594084 28320\n",
      "56\t630771\t1.7\t1\t599636 31135\n",
      "57\t608401\t1.7\t1\t579359 29042\n",
      "58\t589791\t1.7\t1\t563588 26203\n",
      "59\t602895\t1.7\t1\t574430 28465\n",
      "60\t610621\t1.7\t1\t581075 29546\n",
      "61\t631221\t1.7\t1\t600250 30971\n",
      "62\t678983\t1.7\t1\t645436 33547\n",
      "63\t884166\t1.7\t1\t845237 38929\n",
      "64\t697534\t1.7\t1\t670069 27465\n",
      "65\t461327\t1.7\t1\t440187 21140\n",
      "66\t505461\t1.7\t1\t482444 23017\n",
      "67\t504336\t1.7\t1\t480520 23816\n",
      "68\t506997\t1.7\t1\t484099 22898\n",
      "69\t501262\t1.7\t1\t477906 23356\n",
      "70\t506318\t1.7\t1\t482633 23685\n",
      "71\t530040\t1.7\t1\t506417 23623\n",
      "72\t512724\t1.7\t1\t488940 23784\n",
      "73\t535324\t1.7\t1\t511542 23782\n",
      "74\t491314\t1.7\t1\t470184 21130\n",
      "75\t468392\t1.7\t1\t447958 20434\n",
      "76\t453588\t1.7\t1\t433629 19959\n",
      "77\t443997\t1.7\t1\t423454 20543\n",
      "78\t435469\t1.7\t1\t416134 19335\n",
      "79\t419163\t1.7\t1\t400603 18560\n",
      "80\t414024\t1.7\t1\t395828 18196\n",
      "81\t398372\t1.7\t1\t381040 17332\n",
      "82\t382299\t1.7\t1\t365451 16848\n",
      "83\t366779\t1.7\t1\t350673 16106\n",
      "84\t355643\t1.7\t1\t339567 16076\n",
      "85\t348592\t1.7\t1\t332671 15921\n",
      "86\t340981\t1.7\t1\t325646 15335\n",
      "87\t343724\t1.7\t1\t328719 15005\n",
      "88\t328419\t1.7\t1\t314072 14347\n",
      "89\t293572\t1.7\t1\t280847 12725\n",
      "90\t277907\t1.7\t1\t265475 12432\n",
      "91\t261801\t1.7\t1\t250188 11613\n",
      "92\t247933\t1.7\t1\t237089 10844\n",
      "93\t241002\t1.7\t1\t230436 10566\n",
      "94\t225907\t1.7\t1\t216109 9798\n",
      "95\t212543\t1.7\t1\t203112 9431\n",
      "96\t199922\t1.7\t1\t191511 8411\n",
      "97\t189942\t1.7\t1\t181742 8200\n",
      "98\t181068\t1.7\t1\t173356 7712\n",
      "99\t177800\t1.7\t1\t170721 7079\n",
      "100\t161532\t1.7\t1\t154936 6596\n",
      "101\t142590\t1.7\t1\t136815 5775\n",
      "102\t127109\t1.7\t1\t121824 5285\n",
      "103\t114820\t1.7\t1\t109872 4948\n",
      "104\t106755\t1.7\t1\t102353 4402\n",
      "105\t97032\t1.7\t1\t93102 3930\n",
      "106\t87939\t1.7\t1\t84311 3628\n",
      "107\t77043\t1.7\t1\t73799 3244\n",
      "108\t66878\t1.7\t1\t63922 2956\n",
      "109\t60697\t1.7\t1\t58149 2548\n",
      "110\t53817\t1.7\t1\t51530 2287\n",
      "111\t49656\t1.7\t1\t47564 2092\n",
      "112\t44212\t1.7\t1\t42328 1884\n",
      "113\t37568\t1.7\t1\t35930 1638\n",
      "114\t33504\t1.7\t1\t32034 1470\n",
      "115\t26873\t1.7\t1\t25557 1316\n",
      "116\t23021\t1.7\t1\t21918 1103\n",
      "117\t18702\t1.7\t1\t17804 898\n",
      "118\t15710\t1.7\t1\t14954 756\n",
      "119\t13467\t1.7\t1\t12827 640\n",
      "120\t11166\t1.7\t1\t10610 556\n",
      "121\t9578\t1.7\t1\t9044 534\n",
      "122\t7970\t1.7\t1\t7515 455\n",
      "123\t6749\t1.7\t1\t6389 360\n",
      "124\t5790\t1.7\t1\t5438 352\n",
      "125\t4982\t1.7\t1\t4674 308\n",
      "126\t4691\t1.7\t1\t4415 276\n",
      "127\t3926\t1.7\t1\t3689 237\n",
      "128\t2937\t1.7\t1\t2717 220\n",
      "129\t2589\t1.7\t1\t2397 192\n",
      "130\t2951\t1.7\t1\t2762 189\n",
      "131\t4241\t1.7\t1\t4050 191\n",
      "132\t4816\t1.7\t1\t4604 212\n",
      "133\t7287\t1.7\t1\t6984 303\n",
      "134\t8892\t1.7\t1\t8589 303\n",
      "135\t3540\t1.7\t1\t3386 154\n",
      "136\t1144\t1.7\t1\t1071 73\n",
      "137\t598\t1.7\t1\t565 33\n",
      "138\t406\t1.7\t1\t369 37\n",
      "139\t302\t1.7\t1\t277 25\n",
      "140\t227\t1.7\t1\t197 30\n",
      "141\t147\t1.7\t1\t128 19\n",
      "142\t76\t1.7\t1\t62 14\n",
      "143\t79\t1.7\t1\t62 17\n",
      "144\t94\t1.7\t1\t81 13\n",
      "145\t260\t1.7\t1\t229 31\n",
      "146\t164\t1.7\t1\t145 19\n",
      "147\t597\t1.7\t1\t528 69\n",
      "148\t305\t1.7\t1\t255 50\n",
      "149\t176\t1.7\t1\t150 26\n",
      "150\t1321\t1.7\t1\t1192 129\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "113474961 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t164893 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 5100.56 s (45 us/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             112,706,276\n",
      "Reads with adapters:                78,369,527 (69.5%)\n",
      "Reads written (passing filters):   112,706,276 (100.0%)\n",
      "\n",
      "Total basepairs processed: 16,905,941,400 bp\n",
      "Quality-trimmed:              25,158,265 bp (0.1%)\n",
      "Total written (filtered):  14,312,572,664 bp (84.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78369527 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.1%\n",
      "  C: 9.8%\n",
      "  G: 19.7%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t22613071\t28176569.0\t0\t22613071\n",
      "2\t2300394\t7044142.2\t0\t2300394\n",
      "3\t1191481\t1761035.6\t0\t1191481\n",
      "4\t736265\t440258.9\t0\t736265\n",
      "5\t556744\t110064.7\t0\t556744\n",
      "6\t521852\t27516.2\t0\t521852\n",
      "7\t432500\t6879.0\t0\t432500\n",
      "8\t542896\t1719.8\t0\t542896\n",
      "9\t541192\t429.9\t0\t533921 7271\n",
      "10\t495976\t107.5\t1\t461928 34048\n",
      "11\t558120\t26.9\t1\t506660 51460\n",
      "12\t551298\t6.7\t1\t505922 45376\n",
      "13\t542939\t1.7\t1\t497854 45085\n",
      "14\t564136\t1.7\t1\t512614 51522\n",
      "15\t569498\t1.7\t1\t521423 48075\n",
      "16\t563578\t1.7\t1\t509820 53758\n",
      "17\t682231\t1.7\t1\t615144 67087\n",
      "18\t558002\t1.7\t1\t512187 45815\n",
      "19\t496544\t1.7\t1\t454420 42124\n",
      "20\t576298\t1.7\t1\t529271 47027\n",
      "21\t637299\t1.7\t1\t580654 56645\n",
      "22\t542521\t1.7\t1\t500784 41737\n",
      "23\t624224\t1.7\t1\t570946 53278\n",
      "24\t590519\t1.7\t1\t532912 57607\n",
      "25\t607047\t1.7\t1\t556132 50915\n",
      "26\t572302\t1.7\t1\t527531 44771\n",
      "27\t630635\t1.7\t1\t573588 57047\n",
      "28\t620404\t1.7\t1\t568899 51505\n",
      "29\t625305\t1.7\t1\t567099 58206\n",
      "30\t664393\t1.7\t1\t616071 48322\n",
      "31\t510018\t1.7\t1\t469355 40663\n",
      "32\t637215\t1.7\t1\t584852 52363\n",
      "33\t597607\t1.7\t1\t545528 52079\n",
      "34\t753926\t1.7\t1\t688559 65367\n",
      "35\t503645\t1.7\t1\t462577 41068\n",
      "36\t630961\t1.7\t1\t577612 53349\n",
      "37\t659262\t1.7\t1\t604319 54943\n",
      "38\t595913\t1.7\t1\t547877 48036\n",
      "39\t653746\t1.7\t1\t597162 56584\n",
      "40\t670529\t1.7\t1\t610664 59865\n",
      "41\t732564\t1.7\t1\t667967 64597\n",
      "42\t545711\t1.7\t1\t495424 50287\n",
      "43\t991735\t1.7\t1\t924678 67057\n",
      "44\t286953\t1.7\t1\t256328 30625\n",
      "45\t652219\t1.7\t1\t601722 50497\n",
      "46\t619313\t1.7\t1\t571113 48200\n",
      "47\t635341\t1.7\t1\t585973 49368\n",
      "48\t610799\t1.7\t1\t562441 48358\n",
      "49\t674495\t1.7\t1\t617955 56540\n",
      "50\t598712\t1.7\t1\t554202 44510\n",
      "51\t649220\t1.7\t1\t598179 51041\n",
      "52\t650919\t1.7\t1\t595866 55053\n",
      "53\t695670\t1.7\t1\t642854 52816\n",
      "54\t545982\t1.7\t1\t504022 41960\n",
      "55\t632591\t1.7\t1\t583378 49213\n",
      "56\t771796\t1.7\t1\t714466 57330\n",
      "57\t493419\t1.7\t1\t450919 42500\n",
      "58\t643644\t1.7\t1\t599707 43937\n",
      "59\t562738\t1.7\t1\t517495 45243\n",
      "60\t620089\t1.7\t1\t571985 48104\n",
      "61\t599005\t1.7\t1\t554664 44341\n",
      "62\t665940\t1.7\t1\t617471 48469\n",
      "63\t511736\t1.7\t1\t469216 42520\n",
      "64\t570922\t1.7\t1\t530032 40890\n",
      "65\t624134\t1.7\t1\t576865 47269\n",
      "66\t644685\t1.7\t1\t596010 48675\n",
      "67\t692204\t1.7\t1\t630686 61518\n",
      "68\t1251198\t1.7\t1\t1176122 75076\n",
      "69\t255502\t1.7\t1\t232815 22687\n",
      "70\t102840\t1.7\t1\t86533 16307\n",
      "71\t343135\t1.7\t1\t318441 24694\n",
      "72\t484356\t1.7\t1\t451690 32666\n",
      "73\t511359\t1.7\t1\t476313 35046\n",
      "74\t501885\t1.7\t1\t465645 36240\n",
      "75\t495613\t1.7\t1\t458847 36766\n",
      "76\t482971\t1.7\t1\t446569 36402\n",
      "77\t471725\t1.7\t1\t437257 34468\n",
      "78\t457894\t1.7\t1\t423781 34113\n",
      "79\t452070\t1.7\t1\t418062 34008\n",
      "80\t438484\t1.7\t1\t405088 33396\n",
      "81\t425410\t1.7\t1\t393919 31491\n",
      "82\t414132\t1.7\t1\t383041 31091\n",
      "83\t396751\t1.7\t1\t367831 28920\n",
      "84\t386912\t1.7\t1\t357963 28949\n",
      "85\t373056\t1.7\t1\t345296 27760\n",
      "86\t351920\t1.7\t1\t325697 26223\n",
      "87\t339788\t1.7\t1\t315178 24610\n",
      "88\t331230\t1.7\t1\t306931 24299\n",
      "89\t311047\t1.7\t1\t288166 22881\n",
      "90\t302757\t1.7\t1\t280581 22176\n",
      "91\t287473\t1.7\t1\t266368 21105\n",
      "92\t274821\t1.7\t1\t254675 20146\n",
      "93\t261527\t1.7\t1\t242454 19073\n",
      "94\t247083\t1.7\t1\t228486 18597\n",
      "95\t229569\t1.7\t1\t212524 17045\n",
      "96\t215425\t1.7\t1\t199345 16080\n",
      "97\t200580\t1.7\t1\t186258 14322\n",
      "98\t186104\t1.7\t1\t172508 13596\n",
      "99\t176938\t1.7\t1\t163984 12954\n",
      "100\t163824\t1.7\t1\t151757 12067\n",
      "101\t152410\t1.7\t1\t141174 11236\n",
      "102\t139508\t1.7\t1\t128964 10544\n",
      "103\t126668\t1.7\t1\t117083 9585\n",
      "104\t114659\t1.7\t1\t106240 8419\n",
      "105\t103515\t1.7\t1\t95951 7564\n",
      "106\t93675\t1.7\t1\t86344 7331\n",
      "107\t82801\t1.7\t1\t76358 6443\n",
      "108\t74274\t1.7\t1\t68589 5685\n",
      "109\t65750\t1.7\t1\t60455 5295\n",
      "110\t58361\t1.7\t1\t53718 4643\n",
      "111\t51503\t1.7\t1\t47445 4058\n",
      "112\t46515\t1.7\t1\t42704 3811\n",
      "113\t39788\t1.7\t1\t36495 3293\n",
      "114\t35516\t1.7\t1\t32502 3014\n",
      "115\t29286\t1.7\t1\t26808 2478\n",
      "116\t25103\t1.7\t1\t22972 2131\n",
      "117\t21040\t1.7\t1\t19128 1912\n",
      "118\t17330\t1.7\t1\t15763 1567\n",
      "119\t14917\t1.7\t1\t13608 1309\n",
      "120\t12642\t1.7\t1\t11544 1098\n",
      "121\t10638\t1.7\t1\t9581 1057\n",
      "122\t8906\t1.7\t1\t8053 853\n",
      "123\t7247\t1.7\t1\t6562 685\n",
      "124\t6246\t1.7\t1\t5598 648\n",
      "125\t5323\t1.7\t1\t4772 551\n",
      "126\t5106\t1.7\t1\t4581 525\n",
      "127\t4365\t1.7\t1\t3924 441\n",
      "128\t3325\t1.7\t1\t2922 403\n",
      "129\t2686\t1.7\t1\t2381 305\n",
      "130\t3038\t1.7\t1\t2747 291\n",
      "131\t4431\t1.7\t1\t4079 352\n",
      "132\t5138\t1.7\t1\t4758 380\n",
      "133\t6888\t1.7\t1\t6396 492\n",
      "134\t9151\t1.7\t1\t8614 537\n",
      "135\t3733\t1.7\t1\t3470 263\n",
      "136\t1176\t1.7\t1\t1019 157\n",
      "137\t621\t1.7\t1\t528 93\n",
      "138\t473\t1.7\t1\t413 60\n",
      "139\t304\t1.7\t1\t257 47\n",
      "140\t204\t1.7\t1\t168 36\n",
      "141\t175\t1.7\t1\t143 32\n",
      "142\t95\t1.7\t1\t74 21\n",
      "143\t82\t1.7\t1\t66 16\n",
      "144\t113\t1.7\t1\t89 24\n",
      "145\t229\t1.7\t1\t203 26\n",
      "146\t212\t1.7\t1\t174 38\n",
      "147\t863\t1.7\t1\t726 137\n",
      "148\t473\t1.7\t1\t383 90\n",
      "149\t260\t1.7\t1\t220 40\n",
      "150\t2364\t1.7\t1\t1963 401\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "112706276 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t37209 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F1_S20_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 5357.16 s (48 us/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             112,706,276\n",
      "Reads with adapters:                78,702,417 (69.8%)\n",
      "Reads written (passing filters):   112,706,276 (100.0%)\n",
      "\n",
      "Total basepairs processed: 16,905,941,400 bp\n",
      "Quality-trimmed:              71,979,281 bp (0.4%)\n",
      "Total written (filtered):  14,323,556,948 bp (84.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78702417 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 9.3%\n",
      "  G: 21.4%\n",
      "  T: 32.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t24049259\t28176569.0\t0\t24049259\n",
      "2\t2103822\t7044142.2\t0\t2103822\n",
      "3\t1234623\t1761035.6\t0\t1234623\n",
      "4\t713926\t440258.9\t0\t713926\n",
      "5\t593548\t110064.7\t0\t593548\n",
      "6\t520489\t27516.2\t0\t520489\n",
      "7\t412592\t6879.0\t0\t412592\n",
      "8\t591207\t1719.8\t0\t591207\n",
      "9\t484607\t429.9\t0\t482791 1816\n",
      "10\t495729\t107.5\t1\t462070 33659\n",
      "11\t538210\t26.9\t1\t494011 44199\n",
      "12\t541780\t6.7\t1\t498135 43645\n",
      "13\t624482\t1.7\t1\t573035 51447\n",
      "14\t557547\t1.7\t1\t507941 49606\n",
      "15\t471226\t1.7\t1\t432359 38867\n",
      "16\t663767\t1.7\t1\t609668 54099\n",
      "17\t541007\t1.7\t1\t495593 45414\n",
      "18\t492493\t1.7\t1\t452345 40148\n",
      "19\t620390\t1.7\t1\t571312 49078\n",
      "20\t584573\t1.7\t1\t538235 46338\n",
      "21\t515948\t1.7\t1\t474451 41497\n",
      "22\t574265\t1.7\t1\t529265 45000\n",
      "23\t609172\t1.7\t1\t559895 49277\n",
      "24\t593165\t1.7\t1\t546488 46677\n",
      "25\t658890\t1.7\t1\t611466 47424\n",
      "26\t563159\t1.7\t1\t515387 47772\n",
      "27\t631617\t1.7\t1\t576879 54738\n",
      "28\t542598\t1.7\t1\t506943 35655\n",
      "29\t609830\t1.7\t1\t564437 45393\n",
      "30\t560210\t1.7\t1\t522268 37942\n",
      "31\t641241\t1.7\t1\t594510 46731\n",
      "32\t544093\t1.7\t1\t508074 36019\n",
      "33\t636374\t1.7\t1\t588179 48195\n",
      "34\t815281\t1.7\t1\t756328 58953\n",
      "35\t430454\t1.7\t1\t399829 30625\n",
      "36\t615273\t1.7\t1\t568703 46570\n",
      "37\t652049\t1.7\t1\t605036 47013\n",
      "38\t597268\t1.7\t1\t559355 37913\n",
      "39\t577372\t1.7\t1\t541971 35401\n",
      "40\t649729\t1.7\t1\t605597 44132\n",
      "41\t640639\t1.7\t1\t597788 42851\n",
      "42\t636924\t1.7\t1\t595731 41193\n",
      "43\t595156\t1.7\t1\t555850 39306\n",
      "44\t689081\t1.7\t1\t641836 47245\n",
      "45\t705414\t1.7\t1\t658767 46647\n",
      "46\t511671\t1.7\t1\t477756 33915\n",
      "47\t627959\t1.7\t1\t589895 38064\n",
      "48\t606191\t1.7\t1\t569490 36701\n",
      "49\t638780\t1.7\t1\t597375 41405\n",
      "50\t632872\t1.7\t1\t591113 41759\n",
      "51\t767529\t1.7\t1\t718431 49098\n",
      "52\t550691\t1.7\t1\t515808 34883\n",
      "53\t670844\t1.7\t1\t631272 39572\n",
      "54\t486646\t1.7\t1\t457582 29064\n",
      "55\t619044\t1.7\t1\t583828 35216\n",
      "56\t629194\t1.7\t1\t589212 39982\n",
      "57\t612138\t1.7\t1\t577341 34797\n",
      "58\t592465\t1.7\t1\t558875 33590\n",
      "59\t595468\t1.7\t1\t560221 35247\n",
      "60\t612569\t1.7\t1\t576229 36340\n",
      "61\t622280\t1.7\t1\t586602 35678\n",
      "62\t676352\t1.7\t1\t634848 41504\n",
      "63\t865976\t1.7\t1\t819740 46236\n",
      "64\t686055\t1.7\t1\t654087 31968\n",
      "65\t461983\t1.7\t1\t436125 25858\n",
      "66\t506619\t1.7\t1\t477735 28884\n",
      "67\t507018\t1.7\t1\t477302 29716\n",
      "68\t510556\t1.7\t1\t481168 29388\n",
      "69\t507332\t1.7\t1\t477239 30093\n",
      "70\t512308\t1.7\t1\t482423 29885\n",
      "71\t532127\t1.7\t1\t501527 30600\n",
      "72\t518080\t1.7\t1\t487908 30172\n",
      "73\t537157\t1.7\t1\t506806 30351\n",
      "74\t495539\t1.7\t1\t466890 28649\n",
      "75\t470503\t1.7\t1\t443438 27065\n",
      "76\t452980\t1.7\t1\t426860 26120\n",
      "77\t443451\t1.7\t1\t417656 25795\n",
      "78\t435086\t1.7\t1\t409995 25091\n",
      "79\t419597\t1.7\t1\t395575 24022\n",
      "80\t414921\t1.7\t1\t392020 22901\n",
      "81\t399796\t1.7\t1\t377201 22595\n",
      "82\t383327\t1.7\t1\t361982 21345\n",
      "83\t367682\t1.7\t1\t347411 20271\n",
      "84\t357304\t1.7\t1\t337566 19738\n",
      "85\t348720\t1.7\t1\t329480 19240\n",
      "86\t342051\t1.7\t1\t323486 18565\n",
      "87\t342842\t1.7\t1\t324860 17982\n",
      "88\t330713\t1.7\t1\t313318 17395\n",
      "89\t294825\t1.7\t1\t279096 15729\n",
      "90\t280371\t1.7\t1\t265521 14850\n",
      "91\t263612\t1.7\t1\t249496 14116\n",
      "92\t248322\t1.7\t1\t235160 13162\n",
      "93\t241085\t1.7\t1\t228414 12671\n",
      "94\t227365\t1.7\t1\t215260 12105\n",
      "95\t212715\t1.7\t1\t201138 11577\n",
      "96\t201763\t1.7\t1\t191081 10682\n",
      "97\t190347\t1.7\t1\t180592 9755\n",
      "98\t180927\t1.7\t1\t171879 9048\n",
      "99\t177126\t1.7\t1\t168081 9045\n",
      "100\t161967\t1.7\t1\t153628 8339\n",
      "101\t143338\t1.7\t1\t136043 7295\n",
      "102\t127435\t1.7\t1\t120719 6716\n",
      "103\t114642\t1.7\t1\t108603 6039\n",
      "104\t106928\t1.7\t1\t101222 5706\n",
      "105\t97130\t1.7\t1\t92087 5043\n",
      "106\t88346\t1.7\t1\t83845 4501\n",
      "107\t77435\t1.7\t1\t73389 4046\n",
      "108\t67272\t1.7\t1\t63761 3511\n",
      "109\t60831\t1.7\t1\t57633 3198\n",
      "110\t53937\t1.7\t1\t51056 2881\n",
      "111\t49603\t1.7\t1\t47000 2603\n",
      "112\t44226\t1.7\t1\t41868 2358\n",
      "113\t38028\t1.7\t1\t35980 2048\n",
      "114\t33519\t1.7\t1\t31680 1839\n",
      "115\t26723\t1.7\t1\t25283 1440\n",
      "116\t22936\t1.7\t1\t21626 1310\n",
      "117\t19039\t1.7\t1\t17966 1073\n",
      "118\t15805\t1.7\t1\t14850 955\n",
      "119\t13430\t1.7\t1\t12634 796\n",
      "120\t11390\t1.7\t1\t10687 703\n",
      "121\t9723\t1.7\t1\t9101 622\n",
      "122\t8226\t1.7\t1\t7693 533\n",
      "123\t6741\t1.7\t1\t6334 407\n",
      "124\t5699\t1.7\t1\t5276 423\n",
      "125\t5010\t1.7\t1\t4637 373\n",
      "126\t4661\t1.7\t1\t4329 332\n",
      "127\t3987\t1.7\t1\t3713 274\n",
      "128\t2901\t1.7\t1\t2665 236\n",
      "129\t2466\t1.7\t1\t2254 212\n",
      "130\t2939\t1.7\t1\t2734 205\n",
      "131\t4275\t1.7\t1\t4050 225\n",
      "132\t4899\t1.7\t1\t4690 209\n",
      "133\t7071\t1.7\t1\t6772 299\n",
      "134\t8958\t1.7\t1\t8601 357\n",
      "135\t3598\t1.7\t1\t3428 170\n",
      "136\t1085\t1.7\t1\t1002 83\n",
      "137\t586\t1.7\t1\t522 64\n",
      "138\t435\t1.7\t1\t381 54\n",
      "139\t265\t1.7\t1\t235 30\n",
      "140\t195\t1.7\t1\t157 38\n",
      "141\t157\t1.7\t1\t136 21\n",
      "142\t96\t1.7\t1\t75 21\n",
      "143\t89\t1.7\t1\t70 19\n",
      "144\t116\t1.7\t1\t91 25\n",
      "145\t262\t1.7\t1\t239 23\n",
      "146\t193\t1.7\t1\t171 22\n",
      "147\t646\t1.7\t1\t552 94\n",
      "148\t333\t1.7\t1\t271 62\n",
      "149\t178\t1.7\t1\t147 31\n",
      "150\t1314\t1.7\t1\t1144 170\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "112706276 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t159281 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3621.61 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              82,177,525\n",
      "Reads with adapters:                58,714,265 (71.4%)\n",
      "Reads written (passing filters):    82,177,525 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,326,628,750 bp\n",
      "Quality-trimmed:              17,863,332 bp (0.1%)\n",
      "Total written (filtered):  10,356,260,345 bp (84.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58714265 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.4%\n",
      "  C: 11.4%\n",
      "  G: 19.3%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15334703\t20544381.2\t0\t15334703\n",
      "2\t1608191\t5136095.3\t0\t1608191\n",
      "3\t847327\t1284023.8\t0\t847327\n",
      "4\t558993\t321006.0\t0\t558993\n",
      "5\t451836\t80251.5\t0\t451836\n",
      "6\t436528\t20062.9\t0\t436528\n",
      "7\t362332\t5015.7\t0\t362332\n",
      "8\t445624\t1253.9\t0\t445624\n",
      "9\t448853\t313.5\t0\t444071 4782\n",
      "10\t414974\t78.4\t1\t395643 19331\n",
      "11\t451608\t19.6\t1\t421964 29644\n",
      "12\t446668\t4.9\t1\t421394 25274\n",
      "13\t448758\t1.2\t1\t423173 25585\n",
      "14\t458328\t1.2\t1\t429437 28891\n",
      "15\t470816\t1.2\t1\t443807 27009\n",
      "16\t459713\t1.2\t1\t429546 30167\n",
      "17\t546940\t1.2\t1\t508999 37941\n",
      "18\t471227\t1.2\t1\t445754 25473\n",
      "19\t402058\t1.2\t1\t378920 23138\n",
      "20\t471685\t1.2\t1\t445002 26683\n",
      "21\t519729\t1.2\t1\t487230 32499\n",
      "22\t441834\t1.2\t1\t419209 22625\n",
      "23\t505202\t1.2\t1\t475325 29877\n",
      "24\t480556\t1.2\t1\t445808 34748\n",
      "25\t495463\t1.2\t1\t465784 29679\n",
      "26\t468036\t1.2\t1\t442448 25588\n",
      "27\t512154\t1.2\t1\t478482 33672\n",
      "28\t492492\t1.2\t1\t462349 30143\n",
      "29\t513596\t1.2\t1\t476912 36684\n",
      "30\t549763\t1.2\t1\t521613 28150\n",
      "31\t416541\t1.2\t1\t392371 24170\n",
      "32\t537990\t1.2\t1\t507240 30750\n",
      "33\t461062\t1.2\t1\t434172 26890\n",
      "34\t514898\t1.2\t1\t482060 32838\n",
      "35\t565682\t1.2\t1\t527137 38545\n",
      "36\t520514\t1.2\t1\t491038 29476\n",
      "37\t563159\t1.2\t1\t535989 27170\n",
      "38\t400017\t1.2\t1\t369694 30323\n",
      "39\t571304\t1.2\t1\t535130 36174\n",
      "40\t474621\t1.2\t1\t448623 25998\n",
      "41\t600561\t1.2\t1\t562019 38542\n",
      "42\t458766\t1.2\t1\t426402 32364\n",
      "43\t800955\t1.2\t1\t763696 37259\n",
      "44\t235002\t1.2\t1\t215582 19420\n",
      "45\t521706\t1.2\t1\t491650 30056\n",
      "46\t537461\t1.2\t1\t505916 31545\n",
      "47\t540258\t1.2\t1\t510434 29824\n",
      "48\t449883\t1.2\t1\t420809 29074\n",
      "49\t566867\t1.2\t1\t529971 36896\n",
      "50\t457025\t1.2\t1\t432856 24169\n",
      "51\t503689\t1.2\t1\t474572 29117\n",
      "52\t554879\t1.2\t1\t516852 38027\n",
      "53\t630277\t1.2\t1\t598162 32115\n",
      "54\t395812\t1.2\t1\t371351 24461\n",
      "55\t516387\t1.2\t1\t488413 27974\n",
      "56\t554072\t1.2\t1\t525098 28974\n",
      "57\t493826\t1.2\t1\t465005 28821\n",
      "58\t455432\t1.2\t1\t435241 20191\n",
      "59\t348004\t1.2\t1\t326224 21780\n",
      "60\t524024\t1.2\t1\t495485 28539\n",
      "61\t494677\t1.2\t1\t470013 24664\n",
      "62\t445581\t1.2\t1\t422101 23480\n",
      "63\t440868\t1.2\t1\t415824 25044\n",
      "64\t419014\t1.2\t1\t397741 21273\n",
      "65\t430484\t1.2\t1\t407046 23438\n",
      "66\t478836\t1.2\t1\t451552 27284\n",
      "67\t526917\t1.2\t1\t490973 35944\n",
      "68\t974235\t1.2\t1\t930569 43666\n",
      "69\t263384\t1.2\t1\t246743 16641\n",
      "70\t78060\t1.2\t1\t68230 9830\n",
      "71\t188944\t1.2\t1\t177000 11944\n",
      "72\t350249\t1.2\t1\t331713 18536\n",
      "73\t385004\t1.2\t1\t364669 20335\n",
      "74\t373731\t1.2\t1\t353304 20427\n",
      "75\t372636\t1.2\t1\t352683 19953\n",
      "76\t364052\t1.2\t1\t344608 19444\n",
      "77\t349658\t1.2\t1\t330711 18947\n",
      "78\t336268\t1.2\t1\t317764 18504\n",
      "79\t333907\t1.2\t1\t315525 18382\n",
      "80\t320027\t1.2\t1\t302554 17473\n",
      "81\t308343\t1.2\t1\t291381 16962\n",
      "82\t303079\t1.2\t1\t286638 16441\n",
      "83\t285198\t1.2\t1\t269542 15656\n",
      "84\t281618\t1.2\t1\t266189 15429\n",
      "85\t270078\t1.2\t1\t255167 14911\n",
      "86\t256741\t1.2\t1\t242477 14264\n",
      "87\t246300\t1.2\t1\t232699 13601\n",
      "88\t239359\t1.2\t1\t226087 13272\n",
      "89\t221157\t1.2\t1\t208813 12344\n",
      "90\t214943\t1.2\t1\t202774 12169\n",
      "91\t203469\t1.2\t1\t192053 11416\n",
      "92\t193081\t1.2\t1\t182042 11039\n",
      "93\t185749\t1.2\t1\t175077 10672\n",
      "94\t173524\t1.2\t1\t163677 9847\n",
      "95\t160182\t1.2\t1\t150706 9476\n",
      "96\t148958\t1.2\t1\t140034 8924\n",
      "97\t139705\t1.2\t1\t131505 8200\n",
      "98\t129444\t1.2\t1\t121874 7570\n",
      "99\t121244\t1.2\t1\t114040 7204\n",
      "100\t113165\t1.2\t1\t106483 6682\n",
      "101\t102845\t1.2\t1\t96512 6333\n",
      "102\t94108\t1.2\t1\t88374 5734\n",
      "103\t85449\t1.2\t1\t80229 5220\n",
      "104\t76917\t1.2\t1\t72056 4861\n",
      "105\t69454\t1.2\t1\t64912 4542\n",
      "106\t62657\t1.2\t1\t58630 4027\n",
      "107\t54683\t1.2\t1\t51109 3574\n",
      "108\t49110\t1.2\t1\t45908 3202\n",
      "109\t44137\t1.2\t1\t41058 3079\n",
      "110\t38372\t1.2\t1\t35617 2755\n",
      "111\t33619\t1.2\t1\t31136 2483\n",
      "112\t30434\t1.2\t1\t28239 2195\n",
      "113\t25366\t1.2\t1\t23490 1876\n",
      "114\t23496\t1.2\t1\t21717 1779\n",
      "115\t19033\t1.2\t1\t17590 1443\n",
      "116\t16344\t1.2\t1\t14972 1372\n",
      "117\t13454\t1.2\t1\t12326 1128\n",
      "118\t11466\t1.2\t1\t10507 959\n",
      "119\t8924\t1.2\t1\t8136 788\n",
      "120\t8008\t1.2\t1\t7309 699\n",
      "121\t6748\t1.2\t1\t6146 602\n",
      "122\t5464\t1.2\t1\t4939 525\n",
      "123\t4525\t1.2\t1\t4074 451\n",
      "124\t3864\t1.2\t1\t3437 427\n",
      "125\t3136\t1.2\t1\t2820 316\n",
      "126\t2478\t1.2\t1\t2171 307\n",
      "127\t2123\t1.2\t1\t1865 258\n",
      "128\t1659\t1.2\t1\t1463 196\n",
      "129\t1306\t1.2\t1\t1161 145\n",
      "130\t1169\t1.2\t1\t1027 142\n",
      "131\t997\t1.2\t1\t875 122\n",
      "132\t756\t1.2\t1\t652 104\n",
      "133\t674\t1.2\t1\t579 95\n",
      "134\t665\t1.2\t1\t580 85\n",
      "135\t442\t1.2\t1\t374 68\n",
      "136\t281\t1.2\t1\t246 35\n",
      "137\t223\t1.2\t1\t178 45\n",
      "138\t175\t1.2\t1\t145 30\n",
      "139\t132\t1.2\t1\t108 24\n",
      "140\t105\t1.2\t1\t88 17\n",
      "141\t84\t1.2\t1\t61 23\n",
      "142\t69\t1.2\t1\t59 10\n",
      "143\t49\t1.2\t1\t38 11\n",
      "144\t68\t1.2\t1\t56 12\n",
      "145\t42\t1.2\t1\t31 11\n",
      "146\t49\t1.2\t1\t40 9\n",
      "147\t75\t1.2\t1\t67 8\n",
      "148\t68\t1.2\t1\t58 10\n",
      "149\t58\t1.2\t1\t50 8\n",
      "150\t375\t1.2\t1\t309 66\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "82177525 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5465 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3850.13 s (47 us/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              82,177,525\n",
      "Reads with adapters:                58,336,382 (71.0%)\n",
      "Reads written (passing filters):    82,177,525 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,326,628,750 bp\n",
      "Quality-trimmed:              41,419,626 bp (0.3%)\n",
      "Total written (filtered):  10,364,431,817 bp (84.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58336382 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.7%\n",
      "  C: 9.9%\n",
      "  G: 22.0%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15350057\t20544381.2\t0\t15350057\n",
      "2\t1685474\t5136095.3\t0\t1685474\n",
      "3\t903882\t1284023.8\t0\t903882\n",
      "4\t571222\t321006.0\t0\t571222\n",
      "5\t501936\t80251.5\t0\t501936\n",
      "6\t415827\t20062.9\t0\t415827\n",
      "7\t347542\t5015.7\t0\t347542\n",
      "8\t483968\t1253.9\t0\t483968\n",
      "9\t396796\t313.5\t0\t395692 1104\n",
      "10\t420274\t78.4\t1\t402160 18114\n",
      "11\t438864\t19.6\t1\t414608 24256\n",
      "12\t449536\t4.9\t1\t424165 25371\n",
      "13\t510213\t1.2\t1\t482515 27698\n",
      "14\t455618\t1.2\t1\t427602 28016\n",
      "15\t389210\t1.2\t1\t366432 22778\n",
      "16\t555171\t1.2\t1\t523342 31829\n",
      "17\t419400\t1.2\t1\t394087 25313\n",
      "18\t418646\t1.2\t1\t395171 23475\n",
      "19\t509488\t1.2\t1\t480141 29347\n",
      "20\t480908\t1.2\t1\t453798 27110\n",
      "21\t427296\t1.2\t1\t402705 24591\n",
      "22\t474401\t1.2\t1\t448452 25949\n",
      "23\t490252\t1.2\t1\t461826 28426\n",
      "24\t484904\t1.2\t1\t457572 27332\n",
      "25\t539104\t1.2\t1\t510128 28976\n",
      "26\t453073\t1.2\t1\t425411 27662\n",
      "27\t501857\t1.2\t1\t468630 33227\n",
      "28\t457877\t1.2\t1\t436189 21688\n",
      "29\t491156\t1.2\t1\t462299 28857\n",
      "30\t462417\t1.2\t1\t439850 22567\n",
      "31\t522905\t1.2\t1\t492898 30007\n",
      "32\t460424\t1.2\t1\t437863 22561\n",
      "33\t512386\t1.2\t1\t483241 29145\n",
      "34\t657426\t1.2\t1\t620019 37407\n",
      "35\t363385\t1.2\t1\t345024 18361\n",
      "36\t487371\t1.2\t1\t459211 28160\n",
      "37\t533683\t1.2\t1\t504069 29614\n",
      "38\t481104\t1.2\t1\t458652 22452\n",
      "39\t485376\t1.2\t1\t462217 23159\n",
      "40\t532302\t1.2\t1\t503234 29068\n",
      "41\t511389\t1.2\t1\t485806 25583\n",
      "42\t521546\t1.2\t1\t495601 25945\n",
      "43\t480650\t1.2\t1\t455868 24782\n",
      "44\t546125\t1.2\t1\t517755 28370\n",
      "45\t570120\t1.2\t1\t542069 28051\n",
      "46\t418884\t1.2\t1\t397786 21098\n",
      "47\t505326\t1.2\t1\t481272 24054\n",
      "48\t488484\t1.2\t1\t466397 22087\n",
      "49\t514214\t1.2\t1\t489094 25120\n",
      "50\t501739\t1.2\t1\t476010 25729\n",
      "51\t610289\t1.2\t1\t580951 29338\n",
      "52\t441047\t1.2\t1\t421068 19979\n",
      "53\t520314\t1.2\t1\t497213 23101\n",
      "54\t398195\t1.2\t1\t379916 18279\n",
      "55\t496805\t1.2\t1\t475323 21482\n",
      "56\t496057\t1.2\t1\t473000 23057\n",
      "57\t480165\t1.2\t1\t458650 21515\n",
      "58\t465002\t1.2\t1\t445044 19958\n",
      "59\t473368\t1.2\t1\t452197 21171\n",
      "60\t477456\t1.2\t1\t455816 21640\n",
      "61\t491054\t1.2\t1\t468379 22675\n",
      "62\t525084\t1.2\t1\t500148 24936\n",
      "63\t688105\t1.2\t1\t659460 28645\n",
      "64\t513363\t1.2\t1\t493656 19707\n",
      "65\t351437\t1.2\t1\t336316 15121\n",
      "66\t388237\t1.2\t1\t371824 16413\n",
      "67\t391608\t1.2\t1\t373999 17609\n",
      "68\t389520\t1.2\t1\t372639 16881\n",
      "69\t387449\t1.2\t1\t370434 17015\n",
      "70\t386662\t1.2\t1\t369736 16926\n",
      "71\t401493\t1.2\t1\t384632 16861\n",
      "72\t387884\t1.2\t1\t370945 16939\n",
      "73\t410793\t1.2\t1\t393742 17051\n",
      "74\t371310\t1.2\t1\t356033 15277\n",
      "75\t357117\t1.2\t1\t342286 14831\n",
      "76\t343694\t1.2\t1\t329251 14443\n",
      "77\t331451\t1.2\t1\t317224 14227\n",
      "78\t322073\t1.2\t1\t308399 13674\n",
      "79\t311753\t1.2\t1\t298772 12981\n",
      "80\t303996\t1.2\t1\t291594 12402\n",
      "81\t292352\t1.2\t1\t280150 12202\n",
      "82\t283173\t1.2\t1\t271407 11766\n",
      "83\t268006\t1.2\t1\t256925 11081\n",
      "84\t264798\t1.2\t1\t253456 11342\n",
      "85\t258296\t1.2\t1\t247093 11203\n",
      "86\t251541\t1.2\t1\t240826 10715\n",
      "87\t251948\t1.2\t1\t241533 10415\n",
      "88\t239999\t1.2\t1\t230066 9933\n",
      "89\t210279\t1.2\t1\t201522 8757\n",
      "90\t198432\t1.2\t1\t189963 8469\n",
      "91\t186654\t1.2\t1\t178634 8020\n",
      "92\t175644\t1.2\t1\t168171 7473\n",
      "93\t173446\t1.2\t1\t166287 7159\n",
      "94\t161637\t1.2\t1\t154843 6794\n",
      "95\t151052\t1.2\t1\t144859 6193\n",
      "96\t140237\t1.2\t1\t134508 5729\n",
      "97\t130886\t1.2\t1\t125542 5344\n",
      "98\t122501\t1.2\t1\t117472 5029\n",
      "99\t115054\t1.2\t1\t110586 4468\n",
      "100\t107801\t1.2\t1\t103547 4254\n",
      "101\t95638\t1.2\t1\t91917 3721\n",
      "102\t87834\t1.2\t1\t84365 3469\n",
      "103\t80762\t1.2\t1\t77514 3248\n",
      "104\t75976\t1.2\t1\t73089 2887\n",
      "105\t67619\t1.2\t1\t64884 2735\n",
      "106\t60836\t1.2\t1\t58481 2355\n",
      "107\t52108\t1.2\t1\t50074 2034\n",
      "108\t45224\t1.2\t1\t43372 1852\n",
      "109\t41239\t1.2\t1\t39535 1704\n",
      "110\t35967\t1.2\t1\t34472 1495\n",
      "111\t33060\t1.2\t1\t31766 1294\n",
      "112\t29404\t1.2\t1\t28175 1229\n",
      "113\t24708\t1.2\t1\t23673 1035\n",
      "114\t22438\t1.2\t1\t21537 901\n",
      "115\t17752\t1.2\t1\t17001 751\n",
      "116\t15010\t1.2\t1\t14377 633\n",
      "117\t12079\t1.2\t1\t11520 559\n",
      "118\t10437\t1.2\t1\t9967 470\n",
      "119\t8208\t1.2\t1\t7826 382\n",
      "120\t7212\t1.2\t1\t6891 321\n",
      "121\t6273\t1.2\t1\t5988 285\n",
      "122\t5055\t1.2\t1\t4818 237\n",
      "123\t4309\t1.2\t1\t4110 199\n",
      "124\t3571\t1.2\t1\t3389 182\n",
      "125\t2948\t1.2\t1\t2833 115\n",
      "126\t2296\t1.2\t1\t2182 114\n",
      "127\t1959\t1.2\t1\t1851 108\n",
      "128\t1542\t1.2\t1\t1473 69\n",
      "129\t1221\t1.2\t1\t1160 61\n",
      "130\t1047\t1.2\t1\t984 63\n",
      "131\t945\t1.2\t1\t892 53\n",
      "132\t678\t1.2\t1\t641 37\n",
      "133\t631\t1.2\t1\t593 38\n",
      "134\t648\t1.2\t1\t616 32\n",
      "135\t387\t1.2\t1\t364 23\n",
      "136\t258\t1.2\t1\t239 19\n",
      "137\t195\t1.2\t1\t184 11\n",
      "138\t163\t1.2\t1\t151 12\n",
      "139\t116\t1.2\t1\t108 8\n",
      "140\t92\t1.2\t1\t83 9\n",
      "141\t73\t1.2\t1\t63 10\n",
      "142\t63\t1.2\t1\t54 9\n",
      "143\t46\t1.2\t1\t37 9\n",
      "144\t54\t1.2\t1\t48 6\n",
      "145\t44\t1.2\t1\t35 9\n",
      "146\t46\t1.2\t1\t42 4\n",
      "147\t78\t1.2\t1\t75 3\n",
      "148\t63\t1.2\t1\t57 6\n",
      "149\t48\t1.2\t1\t46 2\n",
      "150\t297\t1.2\t1\t280 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "82177525 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t73881 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3673.76 s (45 us/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              81,699,034\n",
      "Reads with adapters:                58,422,606 (71.5%)\n",
      "Reads written (passing filters):    81,699,034 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,254,855,100 bp\n",
      "Quality-trimmed:              19,487,828 bp (0.2%)\n",
      "Total written (filtered):  10,283,682,296 bp (83.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58422606 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.4%\n",
      "  C: 11.4%\n",
      "  G: 19.4%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15199129\t20424758.5\t0\t15199129\n",
      "2\t1580922\t5106189.6\t0\t1580922\n",
      "3\t842043\t1276547.4\t0\t842043\n",
      "4\t550022\t319136.9\t0\t550022\n",
      "5\t449997\t79784.2\t0\t449997\n",
      "6\t434481\t19946.1\t0\t434481\n",
      "7\t347898\t4986.5\t0\t347898\n",
      "8\t441084\t1246.6\t0\t441084\n",
      "9\t443045\t311.7\t0\t438482 4563\n",
      "10\t405413\t77.9\t1\t382098 23315\n",
      "11\t450388\t19.5\t1\t415672 34716\n",
      "12\t442770\t4.9\t1\t412586 30184\n",
      "13\t446024\t1.2\t1\t415108 30916\n",
      "14\t453376\t1.2\t1\t419388 33988\n",
      "15\t468278\t1.2\t1\t435263 33015\n",
      "16\t452666\t1.2\t1\t416929 35737\n",
      "17\t549238\t1.2\t1\t503600 45638\n",
      "18\t466886\t1.2\t1\t435945 30941\n",
      "19\t393788\t1.2\t1\t366567 27221\n",
      "20\t467902\t1.2\t1\t436166 31736\n",
      "21\t521792\t1.2\t1\t483620 38172\n",
      "22\t433680\t1.2\t1\t406255 27425\n",
      "23\t504819\t1.2\t1\t468719 36100\n",
      "24\t475517\t1.2\t1\t437053 38464\n",
      "25\t491820\t1.2\t1\t457067 34753\n",
      "26\t461178\t1.2\t1\t431235 29943\n",
      "27\t512653\t1.2\t1\t473769 38884\n",
      "28\t491651\t1.2\t1\t457681 33970\n",
      "29\t512510\t1.2\t1\t470963 41547\n",
      "30\t548141\t1.2\t1\t514010 34131\n",
      "31\t409025\t1.2\t1\t380832 28193\n",
      "32\t540631\t1.2\t1\t502491 38140\n",
      "33\t452892\t1.2\t1\t421055 31837\n",
      "34\t512380\t1.2\t1\t471655 40725\n",
      "35\t569764\t1.2\t1\t523370 46394\n",
      "36\t519856\t1.2\t1\t484148 35708\n",
      "37\t553142\t1.2\t1\t519802 33340\n",
      "38\t400544\t1.2\t1\t364810 35734\n",
      "39\t573002\t1.2\t1\t528499 44503\n",
      "40\t466836\t1.2\t1\t435154 31682\n",
      "41\t600726\t1.2\t1\t554364 46362\n",
      "42\t453939\t1.2\t1\t416218 37721\n",
      "43\t790864\t1.2\t1\t744421 46443\n",
      "44\t241028\t1.2\t1\t218439 22589\n",
      "45\t522905\t1.2\t1\t486598 36307\n",
      "46\t536313\t1.2\t1\t498381 37932\n",
      "47\t536772\t1.2\t1\t501509 35263\n",
      "48\t445960\t1.2\t1\t412336 33624\n",
      "49\t569976\t1.2\t1\t527084 42892\n",
      "50\t451713\t1.2\t1\t423569 28144\n",
      "51\t503554\t1.2\t1\t469351 34203\n",
      "52\t559241\t1.2\t1\t515373 43868\n",
      "53\t626961\t1.2\t1\t588950 38011\n",
      "54\t397202\t1.2\t1\t368180 29022\n",
      "55\t510273\t1.2\t1\t477059 33214\n",
      "56\t566018\t1.2\t1\t530313 35705\n",
      "57\t490065\t1.2\t1\t455879 34186\n",
      "58\t448852\t1.2\t1\t423488 25364\n",
      "59\t352650\t1.2\t1\t326693 25957\n",
      "60\t521343\t1.2\t1\t488341 33002\n",
      "61\t489445\t1.2\t1\t459888 29557\n",
      "62\t454351\t1.2\t1\t425498 28853\n",
      "63\t442859\t1.2\t1\t412719 30140\n",
      "64\t413454\t1.2\t1\t388538 24916\n",
      "65\t428487\t1.2\t1\t400331 28156\n",
      "66\t475641\t1.2\t1\t444595 31046\n",
      "67\t527546\t1.2\t1\t487484 40062\n",
      "68\t932793\t1.2\t1\t883259 49534\n",
      "69\t306211\t1.2\t1\t285490 20721\n",
      "70\t98330\t1.2\t1\t86452 11878\n",
      "71\t183213\t1.2\t1\t170097 13116\n",
      "72\t347828\t1.2\t1\t327310 20518\n",
      "73\t384493\t1.2\t1\t361546 22947\n",
      "74\t375434\t1.2\t1\t351755 23679\n",
      "75\t372810\t1.2\t1\t349151 23659\n",
      "76\t366345\t1.2\t1\t341732 24613\n",
      "77\t349631\t1.2\t1\t326987 22644\n",
      "78\t336085\t1.2\t1\t313727 22358\n",
      "79\t334791\t1.2\t1\t312735 22056\n",
      "80\t320766\t1.2\t1\t299322 21444\n",
      "81\t308813\t1.2\t1\t288662 20151\n",
      "82\t303376\t1.2\t1\t282871 20505\n",
      "83\t287613\t1.2\t1\t268884 18729\n",
      "84\t281524\t1.2\t1\t262646 18878\n",
      "85\t269523\t1.2\t1\t251601 17922\n",
      "86\t255982\t1.2\t1\t238963 17019\n",
      "87\t246467\t1.2\t1\t230371 16096\n",
      "88\t240096\t1.2\t1\t224172 15924\n",
      "89\t222064\t1.2\t1\t207380 14684\n",
      "90\t214626\t1.2\t1\t200391 14235\n",
      "91\t204511\t1.2\t1\t191212 13299\n",
      "92\t193574\t1.2\t1\t180679 12895\n",
      "93\t185417\t1.2\t1\t173008 12409\n",
      "94\t174914\t1.2\t1\t163234 11680\n",
      "95\t159670\t1.2\t1\t148732 10938\n",
      "96\t150695\t1.2\t1\t140365 10330\n",
      "97\t140751\t1.2\t1\t131592 9159\n",
      "98\t129553\t1.2\t1\t120450 9103\n",
      "99\t122439\t1.2\t1\t114105 8334\n",
      "100\t113910\t1.2\t1\t106001 7909\n",
      "101\t103551\t1.2\t1\t96319 7232\n",
      "102\t95288\t1.2\t1\t88666 6622\n",
      "103\t86137\t1.2\t1\t80145 5992\n",
      "104\t77327\t1.2\t1\t71858 5469\n",
      "105\t69993\t1.2\t1\t65045 4948\n",
      "106\t63171\t1.2\t1\t58489 4682\n",
      "107\t54679\t1.2\t1\t50722 3957\n",
      "108\t49406\t1.2\t1\t45775 3631\n",
      "109\t44550\t1.2\t1\t41176 3374\n",
      "110\t38825\t1.2\t1\t35771 3054\n",
      "111\t33501\t1.2\t1\t30848 2653\n",
      "112\t30928\t1.2\t1\t28428 2500\n",
      "113\t25660\t1.2\t1\t23574 2086\n",
      "114\t23733\t1.2\t1\t21729 2004\n",
      "115\t19374\t1.2\t1\t17711 1663\n",
      "116\t16667\t1.2\t1\t15185 1482\n",
      "117\t13526\t1.2\t1\t12278 1248\n",
      "118\t11411\t1.2\t1\t10398 1013\n",
      "119\t9123\t1.2\t1\t8243 880\n",
      "120\t8085\t1.2\t1\t7320 765\n",
      "121\t6821\t1.2\t1\t6153 668\n",
      "122\t5630\t1.2\t1\t5011 619\n",
      "123\t4598\t1.2\t1\t4108 490\n",
      "124\t3796\t1.2\t1\t3390 406\n",
      "125\t3140\t1.2\t1\t2802 338\n",
      "126\t2509\t1.2\t1\t2205 304\n",
      "127\t2125\t1.2\t1\t1865 260\n",
      "128\t1671\t1.2\t1\t1452 219\n",
      "129\t1329\t1.2\t1\t1142 187\n",
      "130\t1120\t1.2\t1\t991 129\n",
      "131\t940\t1.2\t1\t808 132\n",
      "132\t761\t1.2\t1\t666 95\n",
      "133\t675\t1.2\t1\t585 90\n",
      "134\t682\t1.2\t1\t604 78\n",
      "135\t447\t1.2\t1\t388 59\n",
      "136\t330\t1.2\t1\t286 44\n",
      "137\t223\t1.2\t1\t175 48\n",
      "138\t187\t1.2\t1\t160 27\n",
      "139\t149\t1.2\t1\t117 32\n",
      "140\t100\t1.2\t1\t80 20\n",
      "141\t64\t1.2\t1\t50 14\n",
      "142\t63\t1.2\t1\t51 12\n",
      "143\t43\t1.2\t1\t29 14\n",
      "144\t41\t1.2\t1\t32 9\n",
      "145\t44\t1.2\t1\t36 8\n",
      "146\t35\t1.2\t1\t24 11\n",
      "147\t75\t1.2\t1\t69 6\n",
      "148\t65\t1.2\t1\t52 13\n",
      "149\t34\t1.2\t1\t28 6\n",
      "150\t405\t1.2\t1\t345 60\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "81699034 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5455 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "Writing report to '/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/Sealice_F2_S22_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; user defined)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3884.21 s (48 us/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              81,699,034\n",
      "Reads with adapters:                57,979,113 (71.0%)\n",
      "Reads written (passing filters):    81,699,034 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,254,855,100 bp\n",
      "Quality-trimmed:              41,067,962 bp (0.3%)\n",
      "Total written (filtered):  10,294,963,583 bp (84.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 57979113 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.7%\n",
      "  C: 9.9%\n",
      "  G: 22.0%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15214651\t20424758.5\t0\t15214651\n",
      "2\t1629144\t5106189.6\t0\t1629144\n",
      "3\t914799\t1276547.4\t0\t914799\n",
      "4\t558975\t319136.9\t0\t558975\n",
      "5\t482109\t79784.2\t0\t482109\n",
      "6\t418136\t19946.1\t0\t418136\n",
      "7\t337570\t4986.5\t0\t337570\n",
      "8\t479586\t1246.6\t0\t479586\n",
      "9\t390650\t311.7\t0\t389615 1035\n",
      "10\t405041\t77.9\t1\t379605 25436\n",
      "11\t435020\t19.5\t1\t401922 33098\n",
      "12\t439217\t4.9\t1\t406727 32490\n",
      "13\t510754\t1.2\t1\t472376 38378\n",
      "14\t448612\t1.2\t1\t412140 36472\n",
      "15\t384027\t1.2\t1\t355388 28639\n",
      "16\t544298\t1.2\t1\t502865 41433\n",
      "17\t437427\t1.2\t1\t403719 33708\n",
      "18\t402979\t1.2\t1\t373023 29956\n",
      "19\t503067\t1.2\t1\t466488 36579\n",
      "20\t474971\t1.2\t1\t440326 34645\n",
      "21\t421874\t1.2\t1\t390910 30964\n",
      "22\t467522\t1.2\t1\t433687 33835\n",
      "23\t490952\t1.2\t1\t454530 36422\n",
      "24\t482710\t1.2\t1\t447678 35032\n",
      "25\t536922\t1.2\t1\t501311 35611\n",
      "26\t451785\t1.2\t1\t415820 35965\n",
      "27\t506974\t1.2\t1\t466123 40851\n",
      "28\t440254\t1.2\t1\t412940 27314\n",
      "29\t498335\t1.2\t1\t462544 35791\n",
      "30\t447616\t1.2\t1\t418293 29323\n",
      "31\t531574\t1.2\t1\t494290 37284\n",
      "32\t441801\t1.2\t1\t413285 28516\n",
      "33\t515665\t1.2\t1\t478090 37575\n",
      "34\t666212\t1.2\t1\t619804 46408\n",
      "35\t346551\t1.2\t1\t323116 23435\n",
      "36\t489499\t1.2\t1\t454133 35366\n",
      "37\t532182\t1.2\t1\t495525 36657\n",
      "38\t478707\t1.2\t1\t449260 29447\n",
      "39\t471359\t1.2\t1\t443454 27905\n",
      "40\t529670\t1.2\t1\t494697 34973\n",
      "41\t516177\t1.2\t1\t483272 32905\n",
      "42\t520597\t1.2\t1\t487926 32671\n",
      "43\t474470\t1.2\t1\t444318 30152\n",
      "44\t553215\t1.2\t1\t517017 36198\n",
      "45\t566191\t1.2\t1\t530084 36107\n",
      "46\t413041\t1.2\t1\t386808 26233\n",
      "47\t502147\t1.2\t1\t473043 29104\n",
      "48\t484308\t1.2\t1\t456231 28077\n",
      "49\t510392\t1.2\t1\t478462 31930\n",
      "50\t506166\t1.2\t1\t473768 32398\n",
      "51\t605207\t1.2\t1\t567914 37293\n",
      "52\t440187\t1.2\t1\t413511 26676\n",
      "53\t531819\t1.2\t1\t501544 30275\n",
      "54\t389426\t1.2\t1\t367011 22415\n",
      "55\t491938\t1.2\t1\t464562 27376\n",
      "56\t495033\t1.2\t1\t464600 30433\n",
      "57\t481452\t1.2\t1\t455340 26112\n",
      "58\t468033\t1.2\t1\t442245 25788\n",
      "59\t468551\t1.2\t1\t441634 26917\n",
      "60\t478522\t1.2\t1\t450900 27622\n",
      "61\t487423\t1.2\t1\t460617 26806\n",
      "62\t522057\t1.2\t1\t491364 30693\n",
      "63\t673446\t1.2\t1\t638844 34602\n",
      "64\t507915\t1.2\t1\t484583 23332\n",
      "65\t350472\t1.2\t1\t331804 18668\n",
      "66\t389451\t1.2\t1\t368357 21094\n",
      "67\t396849\t1.2\t1\t374262 22587\n",
      "68\t394541\t1.2\t1\t372684 21857\n",
      "69\t391045\t1.2\t1\t368794 22251\n",
      "70\t390887\t1.2\t1\t368825 22062\n",
      "71\t401886\t1.2\t1\t379624 22262\n",
      "72\t392080\t1.2\t1\t370490 21590\n",
      "73\t411065\t1.2\t1\t388688 22377\n",
      "74\t374532\t1.2\t1\t353760 20772\n",
      "75\t357961\t1.2\t1\t338145 19816\n",
      "76\t346375\t1.2\t1\t327002 19373\n",
      "77\t331945\t1.2\t1\t313179 18766\n",
      "78\t321538\t1.2\t1\t303798 17740\n",
      "79\t313177\t1.2\t1\t295979 17198\n",
      "80\t304441\t1.2\t1\t287925 16516\n",
      "81\t293724\t1.2\t1\t277950 15774\n",
      "82\t282766\t1.2\t1\t267333 15433\n",
      "83\t270058\t1.2\t1\t255507 14551\n",
      "84\t265196\t1.2\t1\t251091 14105\n",
      "85\t257861\t1.2\t1\t244467 13394\n",
      "86\t250545\t1.2\t1\t237466 13079\n",
      "87\t251526\t1.2\t1\t238890 12636\n",
      "88\t240557\t1.2\t1\t228280 12277\n",
      "89\t211890\t1.2\t1\t200619 11271\n",
      "90\t199074\t1.2\t1\t188762 10312\n",
      "91\t188118\t1.2\t1\t178454 9664\n",
      "92\t176434\t1.2\t1\t167450 8984\n",
      "93\t173462\t1.2\t1\t164594 8868\n",
      "94\t163597\t1.2\t1\t155286 8311\n",
      "95\t151070\t1.2\t1\t143169 7901\n",
      "96\t142096\t1.2\t1\t134776 7320\n",
      "97\t132469\t1.2\t1\t125885 6584\n",
      "98\t123111\t1.2\t1\t116966 6145\n",
      "99\t116382\t1.2\t1\t110385 5997\n",
      "100\t108683\t1.2\t1\t103116 5567\n",
      "101\t96750\t1.2\t1\t91809 4941\n",
      "102\t88826\t1.2\t1\t84306 4520\n",
      "103\t81399\t1.2\t1\t77208 4191\n",
      "104\t76198\t1.2\t1\t72290 3908\n",
      "105\t68158\t1.2\t1\t64724 3434\n",
      "106\t61130\t1.2\t1\t58100 3030\n",
      "107\t52011\t1.2\t1\t49328 2683\n",
      "108\t45529\t1.2\t1\t43239 2290\n",
      "109\t41683\t1.2\t1\t39576 2107\n",
      "110\t36218\t1.2\t1\t34303 1915\n",
      "111\t32883\t1.2\t1\t31247 1636\n",
      "112\t29793\t1.2\t1\t28318 1475\n",
      "113\t24999\t1.2\t1\t23719 1280\n",
      "114\t22662\t1.2\t1\t21525 1137\n",
      "115\t18096\t1.2\t1\t17176 920\n",
      "116\t15270\t1.2\t1\t14496 774\n",
      "117\t12170\t1.2\t1\t11514 656\n",
      "118\t10414\t1.2\t1\t9832 582\n",
      "119\t8403\t1.2\t1\t7959 444\n",
      "120\t7343\t1.2\t1\t6952 391\n",
      "121\t6317\t1.2\t1\t5964 353\n",
      "122\t5204\t1.2\t1\t4941 263\n",
      "123\t4340\t1.2\t1\t4122 218\n",
      "124\t3564\t1.2\t1\t3376 188\n",
      "125\t2965\t1.2\t1\t2783 182\n",
      "126\t2377\t1.2\t1\t2237 140\n",
      "127\t1922\t1.2\t1\t1812 110\n",
      "128\t1530\t1.2\t1\t1436 94\n",
      "129\t1224\t1.2\t1\t1157 67\n",
      "130\t1037\t1.2\t1\t967 70\n",
      "131\t861\t1.2\t1\t806 55\n",
      "132\t686\t1.2\t1\t633 53\n",
      "133\t661\t1.2\t1\t620 41\n",
      "134\t651\t1.2\t1\t603 48\n",
      "135\t432\t1.2\t1\t408 24\n",
      "136\t309\t1.2\t1\t286 23\n",
      "137\t192\t1.2\t1\t170 22\n",
      "138\t168\t1.2\t1\t155 13\n",
      "139\t128\t1.2\t1\t109 19\n",
      "140\t85\t1.2\t1\t78 7\n",
      "141\t53\t1.2\t1\t46 7\n",
      "142\t58\t1.2\t1\t51 7\n",
      "143\t31\t1.2\t1\t26 5\n",
      "144\t40\t1.2\t1\t32 8\n",
      "145\t40\t1.2\t1\t33 7\n",
      "146\t40\t1.2\t1\t29 11\n",
      "147\t75\t1.2\t1\t67 8\n",
      "148\t58\t1.2\t1\t52 6\n",
      "149\t37\t1.2\t1\t34 3\n",
      "150\t319\t1.2\t1\t294 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "81699034 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t70001 (0.1%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/strigg/anaconda3/bin/TrimGalore-0.6.0/trim_galore \\\n",
    "--output_dir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt \\\n",
    "--fastqc_args \\\n",
    "\"--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC\" \\\n",
    "--illumina \\\n",
    "--path_to_cutadapt /Users/strigg/anaconda3/bin/cutadapt \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_1_S13_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_2_S14_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_3_S15_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_26psu_4_S16_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_1_S1_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_2_S2_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_3_S3_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/16C_32psu_4_S4_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_1_S9_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_2_S10_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_3_S11_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_26psu_4_S12_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_1_S5_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_2_S6_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_3_S7_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/8C_32psu_4_S8_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F1_S20_L002_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L001_R2_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R1_001.fastq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/FASTQS/UW_Trigg_190718_done/Sealice_F2_S22_L002_R2_001.fastq.gz "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc of trimmed reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:44: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  configs = yaml.load(f)\n",
      "/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/utils/config.py:50: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n",
      "  sp = yaml.load(f)\n",
      "[WARNING]         multiqc : MultiQC Version v1.7 now available!\n",
      "[INFO   ]         multiqc : This is MultiQC v1.0.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '.'\n",
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]        bargraph : cannot import name 'get_backend'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/bargraph.py\", line 21, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 40, in <module>\n",
      "    import matplotlib._constrained_layout as constrained_layout\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/_constrained_layout.py\", line 52, in <module>\n",
      "    from matplotlib.legend import Legend\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/legend.py\", line 43, in <module>\n",
      "    from matplotlib.offsetbox import HPacker, VPacker, TextArea, DrawingArea\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/offsetbox.py\", line 33, in <module>\n",
      "    from matplotlib.image import BboxImage\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/image.py\", line 19, in <module>\n",
      "    from matplotlib.backend_bases import FigureCanvasBase\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/backend_bases.py\", line 46, in <module>\n",
      "    from matplotlib import (\n",
      "ImportError: cannot import name 'get_backend'\n",
      "##### ERROR! MatPlotLib library could not be loaded!    #####\n",
      "##### Flat plots will instead be plotted as interactive #####\n",
      "[ERROR  ]       linegraph : module 'matplotlib' has no attribute 'artist'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/multiqc-1.0.dev0-py3.6.egg/multiqc/plots/linegraph.py\", line 20, in <module>\n",
      "    import matplotlib\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 1111, in <module>\n",
      "    rcParamsOrig = RcParams(rcParams.copy())\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py\", line 891, in __getitem__\n",
      "    from matplotlib import pyplot as plt\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/pyplot.py\", line 32, in <module>\n",
      "    import matplotlib.colorbar\n",
      "  File \"/Users/strigg/anaconda3/lib/python3.6/site-packages/matplotlib/colorbar.py\", line 28, in <module>\n",
      "    import matplotlib.artist as martist\n",
      "AttributeError: module 'matplotlib' has no attribute 'artist'\n",
      "[WARNING]         multiqc : No analysis results found. Cleaning up..\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "!cd /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/FASTQC\n",
    "!/Users/strigg/anaconda3/bin/multiqc ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "#trim first 5 bases from each read ended up runnin"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp: File exists\n",
      "mkdir: /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC: File exists\n"
     ]
    }
   ],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp\n",
    "!mkdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#ended up running this on Mox with this code: /gscratch/srlab/strigg/jobs/20190725_Trim1st5bp.sh\n",
    "# also I couldn't figure out how to get this loop to work because xargs sh -c expects \n",
    "## commands in '' and awk introduces another set of ''. I found a work around here:\n",
    "## https://unix.stackexchange.com/questions/282503/right-syntax-for-awk-usage-in-combination-with-other-command-inside-xargs-sh-c\n",
    "## but had a problem when including the redirect\n",
    "#maybe some day I'll learn to code better!\n",
    "####  !find /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/*R1_001_trimmed.fq.gz \\\n",
    "#### | xargs basename -s _L001_R1_001_trimmed.fq.gz | xargs -I{} sh -c \\\n",
    "#### 'gzcat /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_adapt/{}_L001_R1_001_trimmed.fq.gz;awk '\\''{if(($1 !~ /^@A00177\\:/) && ($1 !~ /^\\+$/))print substr($1,6,length($1));else print $1}'\\'';gzip > /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/{}_L001_R1_001_trimmed.5bp_3prime.fq.gz'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run fastqc on trimmed 1st 5bp + adapter trimmed reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 55% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 10% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 100% complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 10% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 20% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Started analysis of Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/Shared/bioinformatics/FastQC/fastqc \\\n",
    "--outdir /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC \\\n",
    "--threads 4 \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_1_S13_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_1_S13_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_1_S13_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_1_S13_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_2_S14_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_2_S14_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_2_S14_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_2_S14_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_3_S15_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_3_S15_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_3_S15_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_3_S15_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_4_S16_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_4_S16_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_4_S16_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_26psu_4_S16_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_1_S1_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_1_S1_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_1_S1_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_1_S1_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_2_S2_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_2_S2_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_2_S2_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_2_S2_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_3_S3_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_3_S3_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_3_S3_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_3_S3_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_4_S4_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_4_S4_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_4_S4_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/16C_32psu_4_S4_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_1_S9_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_1_S9_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_1_S9_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_1_S9_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_2_S10_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_2_S10_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_2_S10_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_2_S10_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_3_S11_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_3_S11_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_3_S11_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_3_S11_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_4_S12_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_4_S12_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_4_S12_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_26psu_4_S12_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_1_S5_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_1_S5_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_1_S5_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_1_S5_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_2_S6_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_2_S6_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_2_S6_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_2_S6_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_3_S7_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_3_S7_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_3_S7_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_3_S7_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_4_S8_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_4_S8_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_4_S8_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/8C_32psu_4_S8_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F1_S20_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F1_S20_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F1_S20_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F1_S20_L002_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F2_S22_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F2_S22_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F2_S22_L002_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/Sealice_F2_S22_L002_R2_001_trimmed.5bp_3prime.fq.gz "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run multiqc on adapter + 1st5bp trimmed data; did this on emu and copied back over to Gannet:\n",
    "# /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/FASTQC/multiqc_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#concatenate L001 and L002 fastqs; cat just appends files so no unzipping necessary\n",
    "# for concatenating fastqs, see here:\n",
    "#for xargs command use, see here: https://stackoverflow.com/questions/845863/how-to-use-in-an-xargs-command\n",
    "# ran this job on mox (https://gannet.fish.washington.edu/metacarcinus/mox_jobs/20190805_Concat_L1_L2_SalmoCalig.sh)\n",
    "\n",
    "##%%bash \n",
    "\n",
    "##find /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/*_L001_R1_001_trimmed.5bp_3prime.fq.gz \\\n",
    "##| xargs basename -s _L001_R1_001_trimmed.5bp_3prime.fq.gz | xargs -I {} sh -c 'cat \\\n",
    "##\"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/$1_L001_R1_001_trimmed.5bp_3prime.fq.gz\" \\\n",
    "##\"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/$1_L002_R1_001_trimmed.5bp_3prime.fq.gz\" \\\n",
    "##> \"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_cat/$1_R1_001_trimmed.5bp_3prime.fq.gz\"' -- {}\n",
    "\n",
    "##%%bash \n",
    "\n",
    "##find /Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/*_L001_R2_001_trimmed.5bp_3prime.fq.gz \\\n",
    "##| xargs basename -s _L001_R2_001_trimmed.5bp_3prime.fq.gz | xargs -I {} sh -c 'cat \\\n",
    "##\"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/$1_L001_R2_001_trimmed.5bp_3prime.fq.gz\" \\\n",
    "###\"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_1st5bp/$1_L002_R2_001_trimmed.5bp_3prime.fq.gz\" \\\n",
    "##> \"/Volumes/web/metacarcinus/Salmo_Calig/analyses/20190722/TRIM_cat/$1_R2_001_trimmed.5bp_3prime.fq.gz\"' -- {}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "## ran test alignment on mox using adapter trimmed and adapter trimmed + 5bp trimmed reads, but had very poor alignment\n",
    "## attempting to follow these notes: https://sequencing.qcfail.com/articles/mispriming-in-pbat-libraries-causes-methylation-bias-and-poor-mapping-efficiencies/\n",
    "## to improve mapping"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Multicore support not enabled. Proceeding with single-core trimming.\n",
      "Path to Cutadapt set as: '/Users/strigg/anaconda3/bin/cutadapt' (user defined)\n",
      "Cutadapt seems to be working fine (tested command '/Users/strigg/anaconda3/bin/cutadapt --version')\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /Users/strigg/Desktop/Salmo_Calig/TRIM/\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t538094\tAGATCGGAAGAGC\t1000000\t53.81\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 538094). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_1_S13_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 835.50 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,366,785 (78.4%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              20,636,466 bp (0.6%)\n",
      "Total written (filtered):  2,599,584,885 bp (78.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17366785 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2904827\t5539983.8\t0\t2904827\n",
      "2\t423079\t1384995.9\t0\t423079\n",
      "3\t231866\t346249.0\t0\t231866\n",
      "4\t143449\t86562.2\t0\t143449\n",
      "5\t117570\t21640.6\t0\t117570\n",
      "6\t115529\t5410.1\t0\t115529\n",
      "7\t96119\t1352.5\t0\t96119\n",
      "8\t119364\t338.1\t0\t119364\n",
      "9\t118406\t84.5\t0\t116886 1520\n",
      "10\t112051\t21.1\t1\t106032 6019\n",
      "11\t122987\t5.3\t1\t113734 9253\n",
      "12\t119477\t1.3\t1\t111842 7635\n",
      "13\t120023\t0.3\t1\t112246 7777\n",
      "14\t122963\t0.3\t1\t114085 8878\n",
      "15\t127528\t0.3\t1\t119209 8319\n",
      "16\t126445\t0.3\t1\t117002 9443\n",
      "17\t152761\t0.3\t1\t140488 12273\n",
      "18\t129037\t0.3\t1\t121213 7824\n",
      "19\t112889\t0.3\t1\t105773 7116\n",
      "20\t133616\t0.3\t1\t125004 8612\n",
      "21\t146516\t0.3\t1\t136291 10225\n",
      "22\t126419\t0.3\t1\t119045 7374\n",
      "23\t145391\t0.3\t1\t135716 9675\n",
      "24\t136894\t0.3\t1\t125670 11224\n",
      "25\t145117\t0.3\t1\t135313 9804\n",
      "26\t134897\t0.3\t1\t126717 8180\n",
      "27\t148757\t0.3\t1\t137734 11023\n",
      "28\t145635\t0.3\t1\t135492 10143\n",
      "29\t153188\t0.3\t1\t140948 12240\n",
      "30\t163373\t0.3\t1\t154030 9343\n",
      "31\t130689\t0.3\t1\t122129 8560\n",
      "32\t157390\t0.3\t1\t147156 10234\n",
      "33\t145226\t0.3\t1\t135269 9957\n",
      "34\t158524\t0.3\t1\t147826 10698\n",
      "35\t161582\t0.3\t1\t148734 12848\n",
      "36\t173098\t0.3\t1\t162838 10260\n",
      "37\t141921\t0.3\t1\t133053 8868\n",
      "38\t164933\t0.3\t1\t154468 10465\n",
      "39\t167185\t0.3\t1\t155913 11272\n",
      "40\t175799\t0.3\t1\t163207 12592\n",
      "41\t197239\t0.3\t1\t176427 20812\n",
      "42\t165247\t0.3\t1\t150517 14730\n",
      "43\t270149\t0.3\t1\t254538 15611\n",
      "44\t85600\t0.3\t1\t76961 8639\n",
      "45\t197628\t0.3\t1\t175697 21931\n",
      "46\t723044\t0.3\t1\t688287 34757\n",
      "47\t254889\t0.3\t1\t240372 14517\n",
      "48\t114855\t0.3\t1\t101708 13147\n",
      "49\t409546\t0.3\t1\t390013 19533\n",
      "50\t125438\t0.3\t1\t116192 9246\n",
      "51\t110069\t0.3\t1\t99667 10402\n",
      "52\t381945\t0.3\t1\t364475 17470\n",
      "53\t196688\t0.3\t1\t188239 8449\n",
      "54\t63003\t0.3\t1\t58174 4829\n",
      "55\t106109\t0.3\t1\t101117 4992\n",
      "56\t55521\t0.3\t1\t50154 5367\n",
      "57\t169184\t0.3\t1\t161794 7390\n",
      "58\t35182\t0.3\t1\t32212 2970\n",
      "59\t44244\t0.3\t1\t39185 5059\n",
      "60\t173147\t0.3\t1\t165425 7722\n",
      "61\t86440\t0.3\t1\t82402 4038\n",
      "62\t40559\t0.3\t1\t36479 4080\n",
      "63\t130914\t0.3\t1\t123404 7510\n",
      "64\t122315\t0.3\t1\t116337 5978\n",
      "65\t65805\t0.3\t1\t60428 5377\n",
      "66\t148115\t0.3\t1\t139100 9015\n",
      "67\t175930\t0.3\t1\t163369 12561\n",
      "68\t293688\t0.3\t1\t278052 15636\n",
      "69\t189726\t0.3\t1\t179114 10612\n",
      "70\t75474\t0.3\t1\t69457 6017\n",
      "71\t27345\t0.3\t1\t23952 3393\n",
      "72\t97969\t0.3\t1\t91968 6001\n",
      "73\t138145\t0.3\t1\t130111 8034\n",
      "74\t143624\t0.3\t1\t135279 8345\n",
      "75\t142674\t0.3\t1\t134066 8608\n",
      "76\t141682\t0.3\t1\t133308 8374\n",
      "77\t141205\t0.3\t1\t132947 8258\n",
      "78\t133036\t0.3\t1\t124966 8070\n",
      "79\t131630\t0.3\t1\t123633 7997\n",
      "80\t128859\t0.3\t1\t121116 7743\n",
      "81\t126119\t0.3\t1\t118667 7452\n",
      "82\t123492\t0.3\t1\t116262 7230\n",
      "83\t118847\t0.3\t1\t111615 7232\n",
      "84\t115583\t0.3\t1\t108642 6941\n",
      "85\t111409\t0.3\t1\t104817 6592\n",
      "86\t107099\t0.3\t1\t100545 6554\n",
      "87\t102731\t0.3\t1\t96477 6254\n",
      "88\t101527\t0.3\t1\t95245 6282\n",
      "89\t95383\t0.3\t1\t89600 5783\n",
      "90\t95037\t0.3\t1\t89230 5807\n",
      "91\t89836\t0.3\t1\t84395 5441\n",
      "92\t84977\t0.3\t1\t79865 5112\n",
      "93\t79479\t0.3\t1\t74762 4717\n",
      "94\t76672\t0.3\t1\t72010 4662\n",
      "95\t70676\t0.3\t1\t66313 4363\n",
      "96\t66886\t0.3\t1\t62766 4120\n",
      "97\t63105\t0.3\t1\t59150 3955\n",
      "98\t58813\t0.3\t1\t55101 3712\n",
      "99\t56801\t0.3\t1\t52921 3880\n",
      "100\t51251\t0.3\t1\t47973 3278\n",
      "101\t48131\t0.3\t1\t44969 3162\n",
      "102\t43975\t0.3\t1\t41134 2841\n",
      "103\t40749\t0.3\t1\t38132 2617\n",
      "104\t37602\t0.3\t1\t35059 2543\n",
      "105\t33601\t0.3\t1\t31318 2283\n",
      "106\t30687\t0.3\t1\t28622 2065\n",
      "107\t27550\t0.3\t1\t25698 1852\n",
      "108\t25919\t0.3\t1\t24135 1784\n",
      "109\t23271\t0.3\t1\t21584 1687\n",
      "110\t20524\t0.3\t1\t19077 1447\n",
      "111\t18563\t0.3\t1\t17263 1300\n",
      "112\t16204\t0.3\t1\t15036 1168\n",
      "113\t14110\t0.3\t1\t13017 1093\n",
      "114\t12321\t0.3\t1\t11394 927\n",
      "115\t10629\t0.3\t1\t9884 745\n",
      "116\t9483\t0.3\t1\t8756 727\n",
      "117\t8057\t0.3\t1\t7440 617\n",
      "118\t7067\t0.3\t1\t6491 576\n",
      "119\t5777\t0.3\t1\t5264 513\n",
      "120\t4917\t0.3\t1\t4511 406\n",
      "121\t4181\t0.3\t1\t3800 381\n",
      "122\t3506\t0.3\t1\t3174 332\n",
      "123\t2912\t0.3\t1\t2640 272\n",
      "124\t2546\t0.3\t1\t2297 249\n",
      "125\t2101\t0.3\t1\t1902 199\n",
      "126\t1717\t0.3\t1\t1548 169\n",
      "127\t1536\t0.3\t1\t1361 175\n",
      "128\t1201\t0.3\t1\t1095 106\n",
      "129\t1012\t0.3\t1\t912 100\n",
      "130\t739\t0.3\t1\t668 71\n",
      "131\t638\t0.3\t1\t572 66\n",
      "132\t553\t0.3\t1\t500 53\n",
      "133\t556\t0.3\t1\t500 56\n",
      "134\t665\t0.3\t1\t607 58\n",
      "135\t362\t0.3\t1\t324 38\n",
      "136\t207\t0.3\t1\t176 31\n",
      "137\t143\t0.3\t1\t128 15\n",
      "138\t99\t0.3\t1\t86 13\n",
      "139\t84\t0.3\t1\t72 12\n",
      "140\t58\t0.3\t1\t50 8\n",
      "141\t42\t0.3\t1\t38 4\n",
      "142\t37\t0.3\t1\t29 8\n",
      "143\t24\t0.3\t1\t22 2\n",
      "144\t25\t0.3\t1\t17 8\n",
      "145\t26\t0.3\t1\t22 4\n",
      "146\t29\t0.3\t1\t21 8\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t64\t0.3\t1\t59 5\n",
      "149\t40\t0.3\t1\t35 5\n",
      "150\t355\t0.3\t1\t318 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_1_S13_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 882.58 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,263,070 (77.9%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              16,760,179 bp (0.5%)\n",
      "Total written (filtered):  2,603,569,143 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17263070 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2980698\t5539983.8\t0\t2980698\n",
      "2\t434290\t1384995.9\t0\t434290\n",
      "3\t244115\t346249.0\t0\t244115\n",
      "4\t143736\t86562.2\t0\t143736\n",
      "5\t130207\t21640.6\t0\t130207\n",
      "6\t112454\t5410.1\t0\t112454\n",
      "7\t91383\t1352.5\t0\t91383\n",
      "8\t130413\t338.1\t0\t130413\n",
      "9\t102897\t84.5\t0\t102495 402\n",
      "10\t113241\t21.1\t1\t107719 5522\n",
      "11\t119188\t5.3\t1\t111572 7616\n",
      "12\t120663\t1.3\t1\t112735 7928\n",
      "13\t133942\t0.3\t1\t125965 7977\n",
      "14\t126547\t0.3\t1\t117452 9095\n",
      "15\t102830\t0.3\t1\t96138 6692\n",
      "16\t149163\t0.3\t1\t139761 9402\n",
      "17\t119889\t0.3\t1\t111631 8258\n",
      "18\t113552\t0.3\t1\t106594 6958\n",
      "19\t144826\t0.3\t1\t135202 9624\n",
      "20\t135016\t0.3\t1\t126722 8294\n",
      "21\t118040\t0.3\t1\t110327 7713\n",
      "22\t135122\t0.3\t1\t126795 8327\n",
      "23\t139623\t0.3\t1\t130868 8755\n",
      "24\t139327\t0.3\t1\t130312 9015\n",
      "25\t156486\t0.3\t1\t147346 9140\n",
      "26\t131367\t0.3\t1\t122873 8494\n",
      "27\t144343\t0.3\t1\t134126 10217\n",
      "28\t135732\t0.3\t1\t128445 7287\n",
      "29\t144460\t0.3\t1\t135341 9119\n",
      "30\t143460\t0.3\t1\t135475 7985\n",
      "31\t154935\t0.3\t1\t145507 9428\n",
      "32\t140211\t0.3\t1\t132501 7710\n",
      "33\t154714\t0.3\t1\t145023 9691\n",
      "34\t197197\t0.3\t1\t185506 11691\n",
      "35\t114069\t0.3\t1\t107701 6368\n",
      "36\t154713\t0.3\t1\t145095 9618\n",
      "37\t166945\t0.3\t1\t157027 9918\n",
      "38\t155394\t0.3\t1\t147358 8036\n",
      "39\t151879\t0.3\t1\t143914 7965\n",
      "40\t168764\t0.3\t1\t158797 9967\n",
      "41\t162964\t0.3\t1\t154146 8818\n",
      "42\t166211\t0.3\t1\t157158 9053\n",
      "43\t152395\t0.3\t1\t144005 8390\n",
      "44\t174656\t0.3\t1\t164812 9844\n",
      "45\t186871\t0.3\t1\t176811 10060\n",
      "46\t135036\t0.3\t1\t127666 7370\n",
      "47\t168636\t0.3\t1\t159853 8783\n",
      "48\t166288\t0.3\t1\t157802 8486\n",
      "49\t174530\t0.3\t1\t165351 9179\n",
      "50\t164769\t0.3\t1\t155907 8862\n",
      "51\t207765\t0.3\t1\t197011 10754\n",
      "52\t147151\t0.3\t1\t139795 7356\n",
      "53\t177051\t0.3\t1\t168817 8234\n",
      "54\t133625\t0.3\t1\t126900 6725\n",
      "55\t174567\t0.3\t1\t166338 8229\n",
      "56\t178950\t0.3\t1\t169886 9064\n",
      "57\t168945\t0.3\t1\t161037 7908\n",
      "58\t161931\t0.3\t1\t154618 7313\n",
      "59\t165789\t0.3\t1\t157874 7915\n",
      "60\t171736\t0.3\t1\t163328 8408\n",
      "61\t175826\t0.3\t1\t167046 8780\n",
      "62\t192123\t0.3\t1\t182534 9589\n",
      "63\t252743\t0.3\t1\t241513 11230\n",
      "64\t207112\t0.3\t1\t199124 7988\n",
      "65\t135660\t0.3\t1\t129354 6306\n",
      "66\t145866\t0.3\t1\t139110 6756\n",
      "67\t145748\t0.3\t1\t138731 7017\n",
      "68\t146161\t0.3\t1\t139522 6639\n",
      "69\t144487\t0.3\t1\t137768 6719\n",
      "70\t154292\t0.3\t1\t147160 7132\n",
      "71\t158353\t0.3\t1\t151327 7026\n",
      "72\t150227\t0.3\t1\t143181 7046\n",
      "73\t157376\t0.3\t1\t150704 6672\n",
      "74\t144388\t0.3\t1\t138042 6346\n",
      "75\t137234\t0.3\t1\t131094 6140\n",
      "76\t133825\t0.3\t1\t127901 5924\n",
      "77\t133664\t0.3\t1\t127640 6024\n",
      "78\t127632\t0.3\t1\t121889 5743\n",
      "79\t122934\t0.3\t1\t117397 5537\n",
      "80\t122791\t0.3\t1\t117317 5474\n",
      "81\t119094\t0.3\t1\t113976 5118\n",
      "82\t115048\t0.3\t1\t109953 5095\n",
      "83\t110957\t0.3\t1\t106119 4838\n",
      "84\t107760\t0.3\t1\t102959 4801\n",
      "85\t105919\t0.3\t1\t100973 4946\n",
      "86\t104575\t0.3\t1\t99936 4639\n",
      "87\t105119\t0.3\t1\t100537 4582\n",
      "88\t102473\t0.3\t1\t97892 4581\n",
      "89\t90861\t0.3\t1\t86820 4041\n",
      "90\t87725\t0.3\t1\t83791 3934\n",
      "91\t82796\t0.3\t1\t79052 3744\n",
      "92\t76768\t0.3\t1\t73386 3382\n",
      "93\t74497\t0.3\t1\t71224 3273\n",
      "94\t71670\t0.3\t1\t68502 3168\n",
      "95\t67300\t0.3\t1\t64357 2943\n",
      "96\t63751\t0.3\t1\t61088 2663\n",
      "97\t58781\t0.3\t1\t56250 2531\n",
      "98\t54871\t0.3\t1\t52462 2409\n",
      "99\t53000\t0.3\t1\t50773 2227\n",
      "100\t48407\t0.3\t1\t46381 2026\n",
      "101\t45093\t0.3\t1\t43177 1916\n",
      "102\t40752\t0.3\t1\t38996 1756\n",
      "103\t37584\t0.3\t1\t35906 1678\n",
      "104\t35749\t0.3\t1\t34226 1523\n",
      "105\t31850\t0.3\t1\t30464 1386\n",
      "106\t29282\t0.3\t1\t28064 1218\n",
      "107\t26084\t0.3\t1\t24956 1128\n",
      "108\t23726\t0.3\t1\t22669 1057\n",
      "109\t21646\t0.3\t1\t20704 942\n",
      "110\t19323\t0.3\t1\t18515 808\n",
      "111\t18235\t0.3\t1\t17436 799\n",
      "112\t15797\t0.3\t1\t15095 702\n",
      "113\t13713\t0.3\t1\t13089 624\n",
      "114\t11968\t0.3\t1\t11416 552\n",
      "115\t10124\t0.3\t1\t9656 468\n",
      "116\t8777\t0.3\t1\t8354 423\n",
      "117\t7339\t0.3\t1\t6995 344\n",
      "118\t6452\t0.3\t1\t6137 315\n",
      "119\t5422\t0.3\t1\t5101 321\n",
      "120\t4499\t0.3\t1\t4281 218\n",
      "121\t3883\t0.3\t1\t3680 203\n",
      "122\t3313\t0.3\t1\t3156 157\n",
      "123\t2807\t0.3\t1\t2669 138\n",
      "124\t2400\t0.3\t1\t2285 115\n",
      "125\t2022\t0.3\t1\t1907 115\n",
      "126\t1650\t0.3\t1\t1555 95\n",
      "127\t1434\t0.3\t1\t1346 88\n",
      "128\t1129\t0.3\t1\t1055 74\n",
      "129\t969\t0.3\t1\t902 67\n",
      "130\t682\t0.3\t1\t644 38\n",
      "131\t587\t0.3\t1\t552 35\n",
      "132\t523\t0.3\t1\t500 23\n",
      "133\t524\t0.3\t1\t494 30\n",
      "134\t633\t0.3\t1\t604 29\n",
      "135\t337\t0.3\t1\t318 19\n",
      "136\t197\t0.3\t1\t186 11\n",
      "137\t147\t0.3\t1\t133 14\n",
      "138\t92\t0.3\t1\t88 4\n",
      "139\t79\t0.3\t1\t71 8\n",
      "140\t57\t0.3\t1\t52 5\n",
      "141\t37\t0.3\t1\t35 2\n",
      "142\t37\t0.3\t1\t35 2\n",
      "143\t25\t0.3\t1\t20 5\n",
      "144\t20\t0.3\t1\t20\n",
      "145\t26\t0.3\t1\t25 1\n",
      "146\t24\t0.3\t1\t23 1\n",
      "147\t85\t0.3\t1\t85\n",
      "148\t61\t0.3\t1\t59 2\n",
      "149\t39\t0.3\t1\t36 3\n",
      "150\t325\t0.3\t1\t310 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz, file_2: 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 22159935\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 42248 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_1_S13_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 847.75 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,055,675\n",
      "Reads with adapters:                17,288,884 (78.4%)\n",
      "Reads written (passing filters):    22,055,675 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,308,351,250 bp\n",
      "Quality-trimmed:              20,473,501 bp (0.6%)\n",
      "Total written (filtered):  2,585,092,173 bp (78.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17288884 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2888739\t5513918.8\t0\t2888739\n",
      "2\t417982\t1378479.7\t0\t417982\n",
      "3\t229648\t344619.9\t0\t229648\n",
      "4\t140558\t86155.0\t0\t140558\n",
      "5\t117330\t21538.7\t0\t117330\n",
      "6\t114463\t5384.7\t0\t114463\n",
      "7\t92119\t1346.2\t0\t92119\n",
      "8\t118305\t336.5\t0\t118305\n",
      "9\t116870\t84.1\t0\t115423 1447\n",
      "10\t109085\t21.0\t1\t102190 6895\n",
      "11\t122273\t5.3\t1\t111534 10739\n",
      "12\t117728\t1.3\t1\t108915 8813\n",
      "13\t119559\t0.3\t1\t110607 8952\n",
      "14\t120718\t0.3\t1\t110803 9915\n",
      "15\t126246\t0.3\t1\t116067 10179\n",
      "16\t124980\t0.3\t1\t114018 10962\n",
      "17\t152992\t0.3\t1\t138965 14027\n",
      "18\t127404\t0.3\t1\t118125 9279\n",
      "19\t110945\t0.3\t1\t102558 8387\n",
      "20\t133534\t0.3\t1\t123405 10129\n",
      "21\t147067\t0.3\t1\t135223 11844\n",
      "22\t124353\t0.3\t1\t115585 8768\n",
      "23\t146038\t0.3\t1\t134941 11097\n",
      "24\t136315\t0.3\t1\t124052 12263\n",
      "25\t144095\t0.3\t1\t132766 11329\n",
      "26\t132325\t0.3\t1\t123014 9311\n",
      "27\t148607\t0.3\t1\t136108 12499\n",
      "28\t145176\t0.3\t1\t133796 11380\n",
      "29\t153982\t0.3\t1\t140231 13751\n",
      "30\t162931\t0.3\t1\t151904 11027\n",
      "31\t127590\t0.3\t1\t117988 9602\n",
      "32\t158563\t0.3\t1\t146202 12361\n",
      "33\t143459\t0.3\t1\t131864 11595\n",
      "34\t158293\t0.3\t1\t145587 12706\n",
      "35\t161600\t0.3\t1\t146632 14968\n",
      "36\t173713\t0.3\t1\t161385 12328\n",
      "37\t138576\t0.3\t1\t128088 10488\n",
      "38\t164914\t0.3\t1\t152362 12552\n",
      "39\t166196\t0.3\t1\t152847 13349\n",
      "40\t174913\t0.3\t1\t159838 15075\n",
      "41\t195923\t0.3\t1\t172793 23130\n",
      "42\t162924\t0.3\t1\t146209 16715\n",
      "43\t267611\t0.3\t1\t248773 18838\n",
      "44\t88981\t0.3\t1\t79036 9945\n",
      "45\t199179\t0.3\t1\t175262 23917\n",
      "46\t690085\t0.3\t1\t650909 39176\n",
      "47\t257754\t0.3\t1\t240653 17101\n",
      "48\t117426\t0.3\t1\t103023 14403\n",
      "49\t402804\t0.3\t1\t380078 22726\n",
      "50\t126965\t0.3\t1\t116321 10644\n",
      "51\t110357\t0.3\t1\t99078 11279\n",
      "52\t376565\t0.3\t1\t356163 20402\n",
      "53\t193192\t0.3\t1\t182832 10360\n",
      "54\t64424\t0.3\t1\t58616 5808\n",
      "55\t110681\t0.3\t1\t104197 6484\n",
      "56\t61759\t0.3\t1\t55308 6451\n",
      "57\t169454\t0.3\t1\t160172 9282\n",
      "58\t37968\t0.3\t1\t34229 3739\n",
      "59\t46940\t0.3\t1\t41198 5742\n",
      "60\t169988\t0.3\t1\t161028 8960\n",
      "61\t86428\t0.3\t1\t81402 5026\n",
      "62\t43888\t0.3\t1\t39006 4882\n",
      "63\t131951\t0.3\t1\t123251 8700\n",
      "64\t119378\t0.3\t1\t112438 6940\n",
      "65\t67995\t0.3\t1\t61783 6212\n",
      "66\t148705\t0.3\t1\t138293 10412\n",
      "67\t175463\t0.3\t1\t161782 13681\n",
      "68\t280115\t0.3\t1\t262817 17298\n",
      "69\t200997\t0.3\t1\t188546 12451\n",
      "70\t91468\t0.3\t1\t84121 7347\n",
      "71\t26418\t0.3\t1\t22702 3716\n",
      "72\t93747\t0.3\t1\t87405 6342\n",
      "73\t136042\t0.3\t1\t127373 8669\n",
      "74\t142888\t0.3\t1\t133128 9760\n",
      "75\t142439\t0.3\t1\t132394 10045\n",
      "76\t141347\t0.3\t1\t131130 10217\n",
      "77\t141016\t0.3\t1\t131311 9705\n",
      "78\t132511\t0.3\t1\t122849 9662\n",
      "79\t132027\t0.3\t1\t122622 9405\n",
      "80\t128769\t0.3\t1\t119387 9382\n",
      "81\t126902\t0.3\t1\t118148 8754\n",
      "82\t123681\t0.3\t1\t114826 8855\n",
      "83\t118947\t0.3\t1\t110667 8280\n",
      "84\t114559\t0.3\t1\t106314 8245\n",
      "85\t111485\t0.3\t1\t103536 7949\n",
      "86\t107436\t0.3\t1\t99695 7741\n",
      "87\t103344\t0.3\t1\t96004 7340\n",
      "88\t100991\t0.3\t1\t93808 7183\n",
      "89\t95415\t0.3\t1\t88771 6644\n",
      "90\t94998\t0.3\t1\t88363 6635\n",
      "91\t89676\t0.3\t1\t83270 6406\n",
      "92\t85299\t0.3\t1\t79247 6052\n",
      "93\t79596\t0.3\t1\t73937 5659\n",
      "94\t76506\t0.3\t1\t71043 5463\n",
      "95\t71120\t0.3\t1\t65893 5227\n",
      "96\t67197\t0.3\t1\t62368 4829\n",
      "97\t63520\t0.3\t1\t59045 4475\n",
      "98\t58336\t0.3\t1\t54045 4291\n",
      "99\t56900\t0.3\t1\t52530 4370\n",
      "100\t51732\t0.3\t1\t47977 3755\n",
      "101\t47750\t0.3\t1\t44264 3486\n",
      "102\t44471\t0.3\t1\t41149 3322\n",
      "103\t40594\t0.3\t1\t37603 2991\n",
      "104\t37603\t0.3\t1\t34872 2731\n",
      "105\t33894\t0.3\t1\t31456 2438\n",
      "106\t30780\t0.3\t1\t28387 2393\n",
      "107\t27659\t0.3\t1\t25538 2121\n",
      "108\t26065\t0.3\t1\t24009 2056\n",
      "109\t23274\t0.3\t1\t21434 1840\n",
      "110\t20689\t0.3\t1\t19111 1578\n",
      "111\t18796\t0.3\t1\t17291 1505\n",
      "112\t16460\t0.3\t1\t15232 1228\n",
      "113\t14306\t0.3\t1\t13173 1133\n",
      "114\t12558\t0.3\t1\t11518 1040\n",
      "115\t10829\t0.3\t1\t9919 910\n",
      "116\t9235\t0.3\t1\t8487 748\n",
      "117\t7908\t0.3\t1\t7205 703\n",
      "118\t6968\t0.3\t1\t6363 605\n",
      "119\t5769\t0.3\t1\t5245 524\n",
      "120\t4920\t0.3\t1\t4475 445\n",
      "121\t4219\t0.3\t1\t3831 388\n",
      "122\t3544\t0.3\t1\t3202 342\n",
      "123\t3058\t0.3\t1\t2744 314\n",
      "124\t2517\t0.3\t1\t2275 242\n",
      "125\t2174\t0.3\t1\t1950 224\n",
      "126\t1777\t0.3\t1\t1603 174\n",
      "127\t1561\t0.3\t1\t1396 165\n",
      "128\t1149\t0.3\t1\t1015 134\n",
      "129\t996\t0.3\t1\t899 97\n",
      "130\t753\t0.3\t1\t670 83\n",
      "131\t665\t0.3\t1\t585 80\n",
      "132\t569\t0.3\t1\t501 68\n",
      "133\t543\t0.3\t1\t476 67\n",
      "134\t671\t0.3\t1\t623 48\n",
      "135\t378\t0.3\t1\t347 31\n",
      "136\t222\t0.3\t1\t190 32\n",
      "137\t159\t0.3\t1\t149 10\n",
      "138\t107\t0.3\t1\t97 10\n",
      "139\t82\t0.3\t1\t68 14\n",
      "140\t62\t0.3\t1\t55 7\n",
      "141\t48\t0.3\t1\t42 6\n",
      "142\t40\t0.3\t1\t35 5\n",
      "143\t27\t0.3\t1\t21 6\n",
      "144\t22\t0.3\t1\t19 3\n",
      "145\t23\t0.3\t1\t21 2\n",
      "146\t31\t0.3\t1\t24 7\n",
      "147\t94\t0.3\t1\t87 7\n",
      "148\t48\t0.3\t1\t41 7\n",
      "149\t39\t0.3\t1\t31 8\n",
      "150\t382\t0.3\t1\t330 52\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "22055675 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_1_S13_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 898.69 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,055,675\n",
      "Reads with adapters:                17,164,009 (77.8%)\n",
      "Reads written (passing filters):    22,055,675 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,308,351,250 bp\n",
      "Quality-trimmed:              16,172,247 bp (0.5%)\n",
      "Total written (filtered):  2,590,121,341 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17164009 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2956348\t5513918.8\t0\t2956348\n",
      "2\t421838\t1378479.7\t0\t421838\n",
      "3\t248359\t344619.9\t0\t248359\n",
      "4\t140787\t86155.0\t0\t140787\n",
      "5\t125775\t21538.7\t0\t125775\n",
      "6\t111856\t5384.7\t0\t111856\n",
      "7\t89214\t1346.2\t0\t89214\n",
      "8\t128837\t336.5\t0\t128837\n",
      "9\t101494\t84.1\t0\t101161 333\n",
      "10\t109255\t21.0\t1\t101714 7541\n",
      "11\t118195\t5.3\t1\t108364 9831\n",
      "12\t117038\t1.3\t1\t107170 9868\n",
      "13\t134296\t0.3\t1\t123167 11129\n",
      "14\t123942\t0.3\t1\t112784 11158\n",
      "15\t100887\t0.3\t1\t92534 8353\n",
      "16\t146698\t0.3\t1\t134607 12091\n",
      "17\t124219\t0.3\t1\t113532 10687\n",
      "18\t108678\t0.3\t1\t99668 9010\n",
      "19\t144281\t0.3\t1\t132500 11781\n",
      "20\t133615\t0.3\t1\t123129 10486\n",
      "21\t116612\t0.3\t1\t107459 9153\n",
      "22\t133618\t0.3\t1\t123290 10328\n",
      "23\t141009\t0.3\t1\t130007 11002\n",
      "24\t139397\t0.3\t1\t128279 11118\n",
      "25\t156124\t0.3\t1\t144895 11229\n",
      "26\t130436\t0.3\t1\t119727 10709\n",
      "27\t145243\t0.3\t1\t132821 12422\n",
      "28\t129948\t0.3\t1\t120966 8982\n",
      "29\t147371\t0.3\t1\t136169 11202\n",
      "30\t140000\t0.3\t1\t130086 9914\n",
      "31\t156053\t0.3\t1\t144716 11337\n",
      "32\t135242\t0.3\t1\t125682 9560\n",
      "33\t156256\t0.3\t1\t144163 12093\n",
      "34\t200062\t0.3\t1\t185381 14681\n",
      "35\t108741\t0.3\t1\t100819 7922\n",
      "36\t156564\t0.3\t1\t144598 11966\n",
      "37\t165143\t0.3\t1\t153098 12045\n",
      "38\t153915\t0.3\t1\t143649 10266\n",
      "39\t147077\t0.3\t1\t137848 9229\n",
      "40\t167810\t0.3\t1\t155933 11877\n",
      "41\t163682\t0.3\t1\t152622 11060\n",
      "42\t166228\t0.3\t1\t154981 11247\n",
      "43\t150608\t0.3\t1\t140711 9897\n",
      "44\t176387\t0.3\t1\t164024 12363\n",
      "45\t185146\t0.3\t1\t172572 12574\n",
      "46\t132758\t0.3\t1\t123550 9208\n",
      "47\t167253\t0.3\t1\t156952 10301\n",
      "48\t165177\t0.3\t1\t154705 10472\n",
      "49\t173975\t0.3\t1\t162376 11599\n",
      "50\t166191\t0.3\t1\t155002 11189\n",
      "51\t203241\t0.3\t1\t190158 13083\n",
      "52\t146701\t0.3\t1\t137201 9500\n",
      "53\t181035\t0.3\t1\t170415 10620\n",
      "54\t130972\t0.3\t1\t122872 8100\n",
      "55\t173160\t0.3\t1\t163051 10109\n",
      "56\t176881\t0.3\t1\t165551 11330\n",
      "57\t169136\t0.3\t1\t159682 9454\n",
      "58\t163469\t0.3\t1\t154161 9308\n",
      "59\t163465\t0.3\t1\t153708 9757\n",
      "60\t171325\t0.3\t1\t161169 10156\n",
      "61\t174265\t0.3\t1\t164163 10102\n",
      "62\t189943\t0.3\t1\t178404 11539\n",
      "63\t247557\t0.3\t1\t234468 13089\n",
      "64\t202460\t0.3\t1\t193222 9238\n",
      "65\t135184\t0.3\t1\t127502 7682\n",
      "66\t146610\t0.3\t1\t138357 8253\n",
      "67\t145979\t0.3\t1\t137259 8720\n",
      "68\t147766\t0.3\t1\t139368 8398\n",
      "69\t145296\t0.3\t1\t136585 8711\n",
      "70\t155955\t0.3\t1\t146888 9067\n",
      "71\t158414\t0.3\t1\t149557 8857\n",
      "72\t151267\t0.3\t1\t142345 8922\n",
      "73\t157245\t0.3\t1\t148240 9005\n",
      "74\t144510\t0.3\t1\t136069 8441\n",
      "75\t137730\t0.3\t1\t129832 7898\n",
      "76\t133886\t0.3\t1\t125901 7985\n",
      "77\t133306\t0.3\t1\t125623 7683\n",
      "78\t127271\t0.3\t1\t119937 7334\n",
      "79\t123387\t0.3\t1\t116350 7037\n",
      "80\t122486\t0.3\t1\t115616 6870\n",
      "81\t120050\t0.3\t1\t113299 6751\n",
      "82\t115066\t0.3\t1\t108580 6486\n",
      "83\t111130\t0.3\t1\t104980 6150\n",
      "84\t106741\t0.3\t1\t100726 6015\n",
      "85\t105619\t0.3\t1\t99800 5819\n",
      "86\t104981\t0.3\t1\t99314 5667\n",
      "87\t105156\t0.3\t1\t99595 5561\n",
      "88\t102130\t0.3\t1\t96657 5473\n",
      "89\t91200\t0.3\t1\t86213 4987\n",
      "90\t88189\t0.3\t1\t83375 4814\n",
      "91\t82519\t0.3\t1\t77931 4588\n",
      "92\t77564\t0.3\t1\t73377 4187\n",
      "93\t74406\t0.3\t1\t70415 3991\n",
      "94\t71559\t0.3\t1\t67703 3856\n",
      "95\t67768\t0.3\t1\t64110 3658\n",
      "96\t64417\t0.3\t1\t60976 3441\n",
      "97\t59309\t0.3\t1\t56218 3091\n",
      "98\t54600\t0.3\t1\t51799 2801\n",
      "99\t53526\t0.3\t1\t50607 2919\n",
      "100\t48940\t0.3\t1\t46308 2632\n",
      "101\t44993\t0.3\t1\t42581 2412\n",
      "102\t41335\t0.3\t1\t39097 2238\n",
      "103\t37548\t0.3\t1\t35551 1997\n",
      "104\t35714\t0.3\t1\t33829 1885\n",
      "105\t32156\t0.3\t1\t30392 1764\n",
      "106\t29290\t0.3\t1\t27775 1515\n",
      "107\t26004\t0.3\t1\t24590 1414\n",
      "108\t24018\t0.3\t1\t22665 1353\n",
      "109\t21637\t0.3\t1\t20542 1095\n",
      "110\t19578\t0.3\t1\t18483 1095\n",
      "111\t18464\t0.3\t1\t17459 1005\n",
      "112\t15937\t0.3\t1\t15101 836\n",
      "113\t14018\t0.3\t1\t13272 746\n",
      "114\t12104\t0.3\t1\t11450 654\n",
      "115\t10138\t0.3\t1\t9597 541\n",
      "116\t8616\t0.3\t1\t8197 419\n",
      "117\t7239\t0.3\t1\t6863 376\n",
      "118\t6371\t0.3\t1\t6012 359\n",
      "119\t5403\t0.3\t1\t5093 310\n",
      "120\t4535\t0.3\t1\t4258 277\n",
      "121\t3911\t0.3\t1\t3687 224\n",
      "122\t3351\t0.3\t1\t3130 221\n",
      "123\t2870\t0.3\t1\t2695 175\n",
      "124\t2400\t0.3\t1\t2272 128\n",
      "125\t2105\t0.3\t1\t1957 148\n",
      "126\t1712\t0.3\t1\t1597 115\n",
      "127\t1490\t0.3\t1\t1401 89\n",
      "128\t1075\t0.3\t1\t998 77\n",
      "129\t923\t0.3\t1\t859 64\n",
      "130\t711\t0.3\t1\t666 45\n",
      "131\t624\t0.3\t1\t582 42\n",
      "132\t524\t0.3\t1\t489 35\n",
      "133\t507\t0.3\t1\t469 38\n",
      "134\t649\t0.3\t1\t623 26\n",
      "135\t367\t0.3\t1\t349 18\n",
      "136\t204\t0.3\t1\t188 16\n",
      "137\t158\t0.3\t1\t144 14\n",
      "138\t102\t0.3\t1\t92 10\n",
      "139\t75\t0.3\t1\t67 8\n",
      "140\t58\t0.3\t1\t57 1\n",
      "141\t47\t0.3\t1\t40 7\n",
      "142\t38\t0.3\t1\t34 4\n",
      "143\t23\t0.3\t1\t20 3\n",
      "144\t21\t0.3\t1\t20 1\n",
      "145\t27\t0.3\t1\t23 4\n",
      "146\t22\t0.3\t1\t20 2\n",
      "147\t86\t0.3\t1\t82 4\n",
      "148\t43\t0.3\t1\t41 2\n",
      "149\t33\t0.3\t1\t32 1\n",
      "150\t345\t0.3\t1\t321 24\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "22055675 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz, file_2: 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 22055675\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 39525 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_1_S13_L002_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_2_S14_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 656.89 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,243,447\n",
      "Reads with adapters:                13,329,210 (77.3%)\n",
      "Reads written (passing filters):    17,243,447 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,586,517,050 bp\n",
      "Quality-trimmed:               5,286,503 bp (0.2%)\n",
      "Total written (filtered):  2,048,045,774 bp (79.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13329210 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.2%\n",
      "  G: 20.5%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2371278\t4310861.8\t0\t2371278\n",
      "2\t355428\t1077715.4\t0\t355428\n",
      "3\t196735\t269428.9\t0\t196735\n",
      "4\t114807\t67357.2\t0\t114807\n",
      "5\t92976\t16839.3\t0\t92976\n",
      "6\t90363\t4209.8\t0\t90363\n",
      "7\t74432\t1052.5\t0\t74432\n",
      "8\t93807\t263.1\t0\t93807\n",
      "9\t93533\t65.8\t0\t92435 1098\n",
      "10\t87230\t16.4\t1\t82587 4643\n",
      "11\t96624\t4.1\t1\t89271 7353\n",
      "12\t92957\t1.0\t1\t86987 5970\n",
      "13\t93747\t0.3\t1\t87764 5983\n",
      "14\t96011\t0.3\t1\t89197 6814\n",
      "15\t98978\t0.3\t1\t92592 6386\n",
      "16\t98159\t0.3\t1\t90785 7374\n",
      "17\t119944\t0.3\t1\t110196 9748\n",
      "18\t100650\t0.3\t1\t94699 5951\n",
      "19\t86734\t0.3\t1\t81180 5554\n",
      "20\t104244\t0.3\t1\t97548 6696\n",
      "21\t115327\t0.3\t1\t107159 8168\n",
      "22\t97273\t0.3\t1\t91575 5698\n",
      "23\t115900\t0.3\t1\t108235 7665\n",
      "24\t104136\t0.3\t1\t95675 8461\n",
      "25\t112997\t0.3\t1\t105393 7604\n",
      "26\t102184\t0.3\t1\t96084 6100\n",
      "27\t114471\t0.3\t1\t106298 8173\n",
      "28\t113184\t0.3\t1\t105296 7888\n",
      "29\t118291\t0.3\t1\t108737 9554\n",
      "30\t125680\t0.3\t1\t118630 7050\n",
      "31\t99517\t0.3\t1\t93132 6385\n",
      "32\t120246\t0.3\t1\t112659 7587\n",
      "33\t112299\t0.3\t1\t104687 7612\n",
      "34\t122051\t0.3\t1\t114065 7986\n",
      "35\t122132\t0.3\t1\t112527 9605\n",
      "36\t121056\t0.3\t1\t113558 7498\n",
      "37\t131245\t0.3\t1\t122809 8436\n",
      "38\t117529\t0.3\t1\t109870 7659\n",
      "39\t130463\t0.3\t1\t121802 8661\n",
      "40\t125603\t0.3\t1\t117585 8018\n",
      "41\t145134\t0.3\t1\t134544 10590\n",
      "42\t108964\t0.3\t1\t101116 7848\n",
      "43\t193675\t0.3\t1\t183736 9939\n",
      "44\t56524\t0.3\t1\t51515 5009\n",
      "45\t130295\t0.3\t1\t122170 8125\n",
      "46\t121941\t0.3\t1\t114229 7712\n",
      "47\t128745\t0.3\t1\t120966 7779\n",
      "48\t126992\t0.3\t1\t118817 8175\n",
      "49\t138745\t0.3\t1\t128994 9751\n",
      "50\t118346\t0.3\t1\t111437 6909\n",
      "51\t127998\t0.3\t1\t119617 8381\n",
      "52\t132118\t0.3\t1\t122819 9299\n",
      "53\t142995\t0.3\t1\t134373 8622\n",
      "54\t111526\t0.3\t1\t104331 7195\n",
      "55\t132598\t0.3\t1\t124212 8386\n",
      "56\t162335\t0.3\t1\t152964 9371\n",
      "57\t103809\t0.3\t1\t96508 7301\n",
      "58\t132526\t0.3\t1\t125372 7154\n",
      "59\t115914\t0.3\t1\t108457 7457\n",
      "60\t128350\t0.3\t1\t120205 8145\n",
      "61\t126189\t0.3\t1\t118700 7489\n",
      "62\t137584\t0.3\t1\t129699 7885\n",
      "63\t110035\t0.3\t1\t102791 7244\n",
      "64\t120964\t0.3\t1\t114082 6882\n",
      "65\t135744\t0.3\t1\t127467 8277\n",
      "66\t140396\t0.3\t1\t131867 8529\n",
      "67\t145532\t0.3\t1\t134835 10697\n",
      "68\t287021\t0.3\t1\t273202 13819\n",
      "69\t48096\t0.3\t1\t43999 4097\n",
      "70\t18303\t0.3\t1\t15162 3141\n",
      "71\t72881\t0.3\t1\t68623 4258\n",
      "72\t103446\t0.3\t1\t97521 5925\n",
      "73\t109833\t0.3\t1\t103450 6383\n",
      "74\t107925\t0.3\t1\t101499 6426\n",
      "75\t106292\t0.3\t1\t100152 6140\n",
      "76\t104330\t0.3\t1\t98203 6127\n",
      "77\t104241\t0.3\t1\t98017 6224\n",
      "78\t98088\t0.3\t1\t92293 5795\n",
      "79\t96948\t0.3\t1\t91096 5852\n",
      "80\t94252\t0.3\t1\t88790 5462\n",
      "81\t93362\t0.3\t1\t87783 5579\n",
      "82\t90202\t0.3\t1\t84811 5391\n",
      "83\t87147\t0.3\t1\t81902 5245\n",
      "84\t83793\t0.3\t1\t78976 4817\n",
      "85\t80030\t0.3\t1\t75251 4779\n",
      "86\t77694\t0.3\t1\t73076 4618\n",
      "87\t74005\t0.3\t1\t69629 4376\n",
      "88\t72748\t0.3\t1\t68492 4256\n",
      "89\t68708\t0.3\t1\t64648 4060\n",
      "90\t68047\t0.3\t1\t63799 4248\n",
      "91\t64435\t0.3\t1\t60450 3985\n",
      "92\t60503\t0.3\t1\t56736 3767\n",
      "93\t56921\t0.3\t1\t53460 3461\n",
      "94\t55199\t0.3\t1\t51895 3304\n",
      "95\t50827\t0.3\t1\t47651 3176\n",
      "96\t48189\t0.3\t1\t45189 3000\n",
      "97\t45155\t0.3\t1\t42374 2781\n",
      "98\t41036\t0.3\t1\t38566 2470\n",
      "99\t40350\t0.3\t1\t37693 2657\n",
      "100\t35870\t0.3\t1\t33662 2208\n",
      "101\t33328\t0.3\t1\t31103 2225\n",
      "102\t30823\t0.3\t1\t28817 2006\n",
      "103\t28586\t0.3\t1\t26749 1837\n",
      "104\t26108\t0.3\t1\t24403 1705\n",
      "105\t23329\t0.3\t1\t21857 1472\n",
      "106\t21254\t0.3\t1\t19878 1376\n",
      "107\t19359\t0.3\t1\t18082 1277\n",
      "108\t18005\t0.3\t1\t16770 1235\n",
      "109\t16024\t0.3\t1\t14870 1154\n",
      "110\t14310\t0.3\t1\t13331 979\n",
      "111\t12919\t0.3\t1\t11980 939\n",
      "112\t11039\t0.3\t1\t10251 788\n",
      "113\t9662\t0.3\t1\t8961 701\n",
      "114\t8278\t0.3\t1\t7631 647\n",
      "115\t7400\t0.3\t1\t6862 538\n",
      "116\t6380\t0.3\t1\t5871 509\n",
      "117\t5590\t0.3\t1\t5116 474\n",
      "118\t4883\t0.3\t1\t4498 385\n",
      "119\t3958\t0.3\t1\t3624 334\n",
      "120\t3465\t0.3\t1\t3177 288\n",
      "121\t3026\t0.3\t1\t2769 257\n",
      "122\t2531\t0.3\t1\t2300 231\n",
      "123\t2116\t0.3\t1\t1896 220\n",
      "124\t1699\t0.3\t1\t1542 157\n",
      "125\t1434\t0.3\t1\t1302 132\n",
      "126\t1188\t0.3\t1\t1073 115\n",
      "127\t1126\t0.3\t1\t995 131\n",
      "128\t869\t0.3\t1\t759 110\n",
      "129\t701\t0.3\t1\t622 79\n",
      "130\t549\t0.3\t1\t500 49\n",
      "131\t448\t0.3\t1\t398 50\n",
      "132\t432\t0.3\t1\t385 47\n",
      "133\t454\t0.3\t1\t410 44\n",
      "134\t600\t0.3\t1\t564 36\n",
      "135\t275\t0.3\t1\t252 23\n",
      "136\t173\t0.3\t1\t157 16\n",
      "137\t109\t0.3\t1\t101 8\n",
      "138\t92\t0.3\t1\t82 10\n",
      "139\t63\t0.3\t1\t54 9\n",
      "140\t64\t0.3\t1\t52 12\n",
      "141\t59\t0.3\t1\t51 8\n",
      "142\t40\t0.3\t1\t37 3\n",
      "143\t53\t0.3\t1\t48 5\n",
      "144\t32\t0.3\t1\t31 1\n",
      "145\t53\t0.3\t1\t50 3\n",
      "146\t166\t0.3\t1\t154 12\n",
      "147\t548\t0.3\t1\t525 23\n",
      "148\t402\t0.3\t1\t390 12\n",
      "149\t263\t0.3\t1\t250 13\n",
      "150\t2873\t0.3\t1\t2693 180\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17243447 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_2_S14_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 684.72 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,243,447\n",
      "Reads with adapters:                13,270,528 (77.0%)\n",
      "Reads written (passing filters):    17,243,447 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,586,517,050 bp\n",
      "Quality-trimmed:              11,661,219 bp (0.5%)\n",
      "Total written (filtered):  2,049,890,632 bp (79.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13270528 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.8%\n",
      "  G: 23.5%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2410570\t4310861.8\t0\t2410570\n",
      "2\t362455\t1077715.4\t0\t362455\n",
      "3\t204625\t269428.9\t0\t204625\n",
      "4\t116204\t67357.2\t0\t116204\n",
      "5\t102265\t16839.3\t0\t102265\n",
      "6\t87780\t4209.8\t0\t87780\n",
      "7\t72496\t1052.5\t0\t72496\n",
      "8\t102751\t263.1\t0\t102751\n",
      "9\t81658\t65.8\t0\t81386 272\n",
      "10\t89121\t16.4\t1\t84631 4490\n",
      "11\t93864\t4.1\t1\t87983 5881\n",
      "12\t94565\t1.0\t1\t88491 6074\n",
      "13\t105522\t0.3\t1\t99372 6150\n",
      "14\t98340\t0.3\t1\t91512 6828\n",
      "15\t80980\t0.3\t1\t75936 5044\n",
      "16\t116316\t0.3\t1\t109001 7315\n",
      "17\t93497\t0.3\t1\t87166 6331\n",
      "18\t89538\t0.3\t1\t84129 5409\n",
      "19\t112544\t0.3\t1\t105239 7305\n",
      "20\t105651\t0.3\t1\t99238 6413\n",
      "21\t93625\t0.3\t1\t87782 5843\n",
      "22\t104844\t0.3\t1\t98723 6121\n",
      "23\t109552\t0.3\t1\t102808 6744\n",
      "24\t108841\t0.3\t1\t101825 7016\n",
      "25\t121233\t0.3\t1\t114032 7201\n",
      "26\t101015\t0.3\t1\t94639 6376\n",
      "27\t112490\t0.3\t1\t104790 7700\n",
      "28\t105074\t0.3\t1\t99527 5547\n",
      "29\t111755\t0.3\t1\t104926 6829\n",
      "30\t110664\t0.3\t1\t104658 6006\n",
      "31\t120328\t0.3\t1\t113206 7122\n",
      "32\t108098\t0.3\t1\t102252 5846\n",
      "33\t119317\t0.3\t1\t112198 7119\n",
      "34\t151071\t0.3\t1\t142232 8839\n",
      "35\t87186\t0.3\t1\t82458 4728\n",
      "36\t120833\t0.3\t1\t113650 7183\n",
      "37\t128643\t0.3\t1\t121059 7584\n",
      "38\t120495\t0.3\t1\t114429 6066\n",
      "39\t117189\t0.3\t1\t111171 6018\n",
      "40\t129860\t0.3\t1\t122333 7527\n",
      "41\t123836\t0.3\t1\t117366 6470\n",
      "42\t127654\t0.3\t1\t120939 6715\n",
      "43\t116428\t0.3\t1\t110085 6343\n",
      "44\t133061\t0.3\t1\t125767 7294\n",
      "45\t139911\t0.3\t1\t132604 7307\n",
      "46\t103489\t0.3\t1\t98069 5420\n",
      "47\t129515\t0.3\t1\t122821 6694\n",
      "48\t126933\t0.3\t1\t120634 6299\n",
      "49\t132432\t0.3\t1\t125535 6897\n",
      "50\t125382\t0.3\t1\t118618 6764\n",
      "51\t155910\t0.3\t1\t147950 7960\n",
      "52\t112001\t0.3\t1\t106554 5447\n",
      "53\t134464\t0.3\t1\t128383 6081\n",
      "54\t103158\t0.3\t1\t98052 5106\n",
      "55\t131983\t0.3\t1\t126024 5959\n",
      "56\t136358\t0.3\t1\t129807 6551\n",
      "57\t127296\t0.3\t1\t121594 5702\n",
      "58\t122071\t0.3\t1\t116674 5397\n",
      "59\t124219\t0.3\t1\t118380 5839\n",
      "60\t128066\t0.3\t1\t121909 6157\n",
      "61\t132129\t0.3\t1\t125705 6424\n",
      "62\t142837\t0.3\t1\t135944 6893\n",
      "63\t190582\t0.3\t1\t182484 8098\n",
      "64\t150467\t0.3\t1\t144624 5843\n",
      "65\t101740\t0.3\t1\t97088 4652\n",
      "66\t109816\t0.3\t1\t104968 4848\n",
      "67\t107749\t0.3\t1\t102613 5136\n",
      "68\t108633\t0.3\t1\t103740 4893\n",
      "69\t107596\t0.3\t1\t102648 4948\n",
      "70\t114794\t0.3\t1\t109576 5218\n",
      "71\t116610\t0.3\t1\t111540 5070\n",
      "72\t110184\t0.3\t1\t105059 5125\n",
      "73\t115749\t0.3\t1\t110692 5057\n",
      "74\t106275\t0.3\t1\t101612 4663\n",
      "75\t101750\t0.3\t1\t97259 4491\n",
      "76\t99053\t0.3\t1\t94705 4348\n",
      "77\t99069\t0.3\t1\t94745 4324\n",
      "78\t94226\t0.3\t1\t90009 4217\n",
      "79\t90763\t0.3\t1\t86805 3958\n",
      "80\t89778\t0.3\t1\t85958 3820\n",
      "81\t88669\t0.3\t1\t84684 3985\n",
      "82\t84401\t0.3\t1\t80642 3759\n",
      "83\t81824\t0.3\t1\t78236 3588\n",
      "84\t78990\t0.3\t1\t75425 3565\n",
      "85\t76455\t0.3\t1\t73024 3431\n",
      "86\t76386\t0.3\t1\t73029 3357\n",
      "87\t75988\t0.3\t1\t72761 3227\n",
      "88\t73818\t0.3\t1\t70558 3260\n",
      "89\t65970\t0.3\t1\t63085 2885\n",
      "90\t63019\t0.3\t1\t60213 2806\n",
      "91\t58914\t0.3\t1\t56366 2548\n",
      "92\t54704\t0.3\t1\t52314 2390\n",
      "93\t52692\t0.3\t1\t50404 2288\n",
      "94\t51139\t0.3\t1\t48941 2198\n",
      "95\t47428\t0.3\t1\t45341 2087\n",
      "96\t45523\t0.3\t1\t43617 1906\n",
      "97\t43074\t0.3\t1\t41264 1810\n",
      "98\t39986\t0.3\t1\t38323 1663\n",
      "99\t40439\t0.3\t1\t38720 1719\n",
      "100\t35462\t0.3\t1\t34051 1411\n",
      "101\t31343\t0.3\t1\t29998 1345\n",
      "102\t28451\t0.3\t1\t27239 1212\n",
      "103\t26443\t0.3\t1\t25307 1136\n",
      "104\t24686\t0.3\t1\t23698 988\n",
      "105\t22319\t0.3\t1\t21394 925\n",
      "106\t20298\t0.3\t1\t19438 860\n",
      "107\t18363\t0.3\t1\t17549 814\n",
      "108\t16498\t0.3\t1\t15775 723\n",
      "109\t15075\t0.3\t1\t14486 589\n",
      "110\t13470\t0.3\t1\t12883 587\n",
      "111\t12724\t0.3\t1\t12179 545\n",
      "112\t10843\t0.3\t1\t10385 458\n",
      "113\t9574\t0.3\t1\t9156 418\n",
      "114\t8065\t0.3\t1\t7698 367\n",
      "115\t6969\t0.3\t1\t6632 337\n",
      "116\t5939\t0.3\t1\t5624 315\n",
      "117\t5106\t0.3\t1\t4872 234\n",
      "118\t4425\t0.3\t1\t4206 219\n",
      "119\t3716\t0.3\t1\t3520 196\n",
      "120\t3161\t0.3\t1\t3007 154\n",
      "121\t2854\t0.3\t1\t2727 127\n",
      "122\t2418\t0.3\t1\t2305 113\n",
      "123\t2039\t0.3\t1\t1907 132\n",
      "124\t1593\t0.3\t1\t1516 77\n",
      "125\t1395\t0.3\t1\t1318 77\n",
      "126\t1136\t0.3\t1\t1075 61\n",
      "127\t1068\t0.3\t1\t1004 64\n",
      "128\t801\t0.3\t1\t764 37\n",
      "129\t660\t0.3\t1\t625 35\n",
      "130\t510\t0.3\t1\t471 39\n",
      "131\t432\t0.3\t1\t407 25\n",
      "132\t400\t0.3\t1\t378 22\n",
      "133\t460\t0.3\t1\t436 24\n",
      "134\t592\t0.3\t1\t573 19\n",
      "135\t262\t0.3\t1\t249 13\n",
      "136\t158\t0.3\t1\t150 8\n",
      "137\t105\t0.3\t1\t102 3\n",
      "138\t83\t0.3\t1\t78 5\n",
      "139\t62\t0.3\t1\t57 5\n",
      "140\t55\t0.3\t1\t49 6\n",
      "141\t58\t0.3\t1\t56 2\n",
      "142\t36\t0.3\t1\t36\n",
      "143\t49\t0.3\t1\t47 2\n",
      "144\t33\t0.3\t1\t30 3\n",
      "145\t52\t0.3\t1\t45 7\n",
      "146\t154\t0.3\t1\t152 2\n",
      "147\t543\t0.3\t1\t532 11\n",
      "148\t399\t0.3\t1\t383 16\n",
      "149\t287\t0.3\t1\t273 14\n",
      "150\t2663\t0.3\t1\t2573 90\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17243447 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz, file_2: 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17243447\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 31983 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_2_S14_L001_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_2_S14_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 666.38 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,174,050\n",
      "Reads with adapters:                13,278,050 (77.3%)\n",
      "Reads written (passing filters):    17,174,050 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,107,500 bp\n",
      "Quality-trimmed:               5,444,792 bp (0.2%)\n",
      "Total written (filtered):  2,038,071,183 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13278050 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.3%\n",
      "  G: 20.5%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2359801\t4293512.5\t0\t2359801\n",
      "2\t349363\t1073378.1\t0\t349363\n",
      "3\t195448\t268344.5\t0\t195448\n",
      "4\t112766\t67086.1\t0\t112766\n",
      "5\t92768\t16771.5\t0\t92768\n",
      "6\t90305\t4192.9\t0\t90305\n",
      "7\t71626\t1048.2\t0\t71626\n",
      "8\t92261\t262.1\t0\t92261\n",
      "9\t91976\t65.5\t0\t90926 1050\n",
      "10\t85467\t16.4\t1\t79925 5542\n",
      "11\t96691\t4.1\t1\t88355 8336\n",
      "12\t91713\t1.0\t1\t84839 6874\n",
      "13\t92754\t0.3\t1\t85809 6945\n",
      "14\t95464\t0.3\t1\t87621 7843\n",
      "15\t99277\t0.3\t1\t91369 7908\n",
      "16\t97348\t0.3\t1\t88681 8667\n",
      "17\t119420\t0.3\t1\t108369 11051\n",
      "18\t98719\t0.3\t1\t91552 7167\n",
      "19\t85310\t0.3\t1\t78884 6426\n",
      "20\t103518\t0.3\t1\t95770 7748\n",
      "21\t115588\t0.3\t1\t106188 9400\n",
      "22\t95906\t0.3\t1\t89272 6634\n",
      "23\t115386\t0.3\t1\t106435 8951\n",
      "24\t102949\t0.3\t1\t93582 9367\n",
      "25\t112836\t0.3\t1\t104100 8736\n",
      "26\t100484\t0.3\t1\t93502 6982\n",
      "27\t114811\t0.3\t1\t105307 9504\n",
      "28\t113076\t0.3\t1\t104189 8887\n",
      "29\t118710\t0.3\t1\t108207 10503\n",
      "30\t126214\t0.3\t1\t117742 8472\n",
      "31\t97121\t0.3\t1\t89907 7214\n",
      "32\t120516\t0.3\t1\t111427 9089\n",
      "33\t110848\t0.3\t1\t101943 8905\n",
      "34\t122175\t0.3\t1\t112281 9894\n",
      "35\t122064\t0.3\t1\t111107 10957\n",
      "36\t119580\t0.3\t1\t110830 8750\n",
      "37\t130636\t0.3\t1\t120649 9987\n",
      "38\t115572\t0.3\t1\t106536 9036\n",
      "39\t129905\t0.3\t1\t119211 10694\n",
      "40\t124968\t0.3\t1\t115039 9929\n",
      "41\t145398\t0.3\t1\t132929 12469\n",
      "42\t108151\t0.3\t1\t98831 9320\n",
      "43\t190551\t0.3\t1\t178361 12190\n",
      "44\t58099\t0.3\t1\t52480 5619\n",
      "45\t130794\t0.3\t1\t121367 9427\n",
      "46\t121400\t0.3\t1\t112370 9030\n",
      "47\t129048\t0.3\t1\t119631 9417\n",
      "48\t126222\t0.3\t1\t116730 9492\n",
      "49\t139750\t0.3\t1\t128710 11040\n",
      "50\t117206\t0.3\t1\t109130 8076\n",
      "51\t127012\t0.3\t1\t117729 9283\n",
      "52\t132886\t0.3\t1\t122231 10655\n",
      "53\t141918\t0.3\t1\t132027 9891\n",
      "54\t110897\t0.3\t1\t103000 7897\n",
      "55\t132094\t0.3\t1\t122452 9642\n",
      "56\t162870\t0.3\t1\t151676 11194\n",
      "57\t103170\t0.3\t1\t94887 8283\n",
      "58\t132668\t0.3\t1\t124347 8321\n",
      "59\t116116\t0.3\t1\t107462 8654\n",
      "60\t129021\t0.3\t1\t119747 9274\n",
      "61\t124026\t0.3\t1\t115604 8422\n",
      "62\t137460\t0.3\t1\t128199 9261\n",
      "63\t110698\t0.3\t1\t102173 8525\n",
      "64\t120813\t0.3\t1\t112906 7907\n",
      "65\t134938\t0.3\t1\t125626 9312\n",
      "66\t138037\t0.3\t1\t128352 9685\n",
      "67\t146504\t0.3\t1\t134535 11969\n",
      "68\t277060\t0.3\t1\t261442 15618\n",
      "69\t59870\t0.3\t1\t54647 5223\n",
      "70\t22664\t0.3\t1\t19044 3620\n",
      "71\t71972\t0.3\t1\t67240 4732\n",
      "72\t103041\t0.3\t1\t96552 6489\n",
      "73\t110046\t0.3\t1\t102755 7291\n",
      "74\t107843\t0.3\t1\t100538 7305\n",
      "75\t106162\t0.3\t1\t98772 7390\n",
      "76\t105323\t0.3\t1\t97926 7397\n",
      "77\t104408\t0.3\t1\t97267 7141\n",
      "78\t97073\t0.3\t1\t90169 6904\n",
      "79\t96839\t0.3\t1\t89879 6960\n",
      "80\t93988\t0.3\t1\t87339 6649\n",
      "81\t93546\t0.3\t1\t86918 6628\n",
      "82\t90241\t0.3\t1\t83957 6284\n",
      "83\t86944\t0.3\t1\t80903 6041\n",
      "84\t83168\t0.3\t1\t77255 5913\n",
      "85\t80437\t0.3\t1\t74815 5622\n",
      "86\t77338\t0.3\t1\t71860 5478\n",
      "87\t73439\t0.3\t1\t68315 5124\n",
      "88\t73001\t0.3\t1\t67818 5183\n",
      "89\t68544\t0.3\t1\t63628 4916\n",
      "90\t68084\t0.3\t1\t63234 4850\n",
      "91\t64534\t0.3\t1\t60016 4518\n",
      "92\t61115\t0.3\t1\t56823 4292\n",
      "93\t56566\t0.3\t1\t52542 4024\n",
      "94\t55074\t0.3\t1\t51146 3928\n",
      "95\t50909\t0.3\t1\t47255 3654\n",
      "96\t48357\t0.3\t1\t44862 3495\n",
      "97\t45610\t0.3\t1\t42541 3069\n",
      "98\t41384\t0.3\t1\t38355 3029\n",
      "99\t39655\t0.3\t1\t36624 3031\n",
      "100\t36325\t0.3\t1\t33739 2586\n",
      "101\t33659\t0.3\t1\t31212 2447\n",
      "102\t31157\t0.3\t1\t28883 2274\n",
      "103\t28746\t0.3\t1\t26792 1954\n",
      "104\t26425\t0.3\t1\t24518 1907\n",
      "105\t23705\t0.3\t1\t22021 1684\n",
      "106\t21201\t0.3\t1\t19619 1582\n",
      "107\t19613\t0.3\t1\t18232 1381\n",
      "108\t17790\t0.3\t1\t16434 1356\n",
      "109\t16006\t0.3\t1\t14830 1176\n",
      "110\t14326\t0.3\t1\t13263 1063\n",
      "111\t12771\t0.3\t1\t11779 992\n",
      "112\t11222\t0.3\t1\t10318 904\n",
      "113\t9534\t0.3\t1\t8824 710\n",
      "114\t8358\t0.3\t1\t7698 660\n",
      "115\t7285\t0.3\t1\t6692 593\n",
      "116\t6411\t0.3\t1\t5816 595\n",
      "117\t5570\t0.3\t1\t5108 462\n",
      "118\t4832\t0.3\t1\t4420 412\n",
      "119\t3906\t0.3\t1\t3549 357\n",
      "120\t3397\t0.3\t1\t3108 289\n",
      "121\t2929\t0.3\t1\t2664 265\n",
      "122\t2444\t0.3\t1\t2212 232\n",
      "123\t2198\t0.3\t1\t1992 206\n",
      "124\t1767\t0.3\t1\t1602 165\n",
      "125\t1386\t0.3\t1\t1230 156\n",
      "126\t1228\t0.3\t1\t1089 139\n",
      "127\t1105\t0.3\t1\t989 116\n",
      "128\t786\t0.3\t1\t705 81\n",
      "129\t664\t0.3\t1\t591 73\n",
      "130\t581\t0.3\t1\t508 73\n",
      "131\t479\t0.3\t1\t435 44\n",
      "132\t414\t0.3\t1\t361 53\n",
      "133\t450\t0.3\t1\t408 42\n",
      "134\t646\t0.3\t1\t602 44\n",
      "135\t289\t0.3\t1\t262 27\n",
      "136\t173\t0.3\t1\t154 19\n",
      "137\t125\t0.3\t1\t108 17\n",
      "138\t98\t0.3\t1\t88 10\n",
      "139\t90\t0.3\t1\t77 13\n",
      "140\t54\t0.3\t1\t46 8\n",
      "141\t52\t0.3\t1\t45 7\n",
      "142\t38\t0.3\t1\t35 3\n",
      "143\t50\t0.3\t1\t43 7\n",
      "144\t42\t0.3\t1\t37 5\n",
      "145\t46\t0.3\t1\t45 1\n",
      "146\t125\t0.3\t1\t116 9\n",
      "147\t651\t0.3\t1\t623 28\n",
      "148\t442\t0.3\t1\t421 21\n",
      "149\t216\t0.3\t1\t204 12\n",
      "150\t2921\t0.3\t1\t2713 208\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17174050 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_2_S14_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 698.12 s (41 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,174,050\n",
      "Reads with adapters:                13,208,659 (76.9%)\n",
      "Reads written (passing filters):    17,174,050 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,107,500 bp\n",
      "Quality-trimmed:              11,281,504 bp (0.4%)\n",
      "Total written (filtered):  2,040,417,795 bp (79.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13208659 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 12.8%\n",
      "  G: 23.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2393304\t4293512.5\t0\t2393304\n",
      "2\t355208\t1073378.1\t0\t355208\n",
      "3\t207534\t268344.5\t0\t207534\n",
      "4\t112818\t67086.1\t0\t112818\n",
      "5\t98722\t16771.5\t0\t98722\n",
      "6\t88016\t4192.9\t0\t88016\n",
      "7\t70973\t1048.2\t0\t70973\n",
      "8\t100906\t262.1\t0\t100906\n",
      "9\t80596\t65.5\t0\t80310 286\n",
      "10\t85921\t16.4\t1\t80103 5818\n",
      "11\t93456\t4.1\t1\t85698 7758\n",
      "12\t91973\t1.0\t1\t84465 7508\n",
      "13\t105255\t0.3\t1\t96925 8330\n",
      "14\t97607\t0.3\t1\t89028 8579\n",
      "15\t80440\t0.3\t1\t74131 6309\n",
      "16\t114723\t0.3\t1\t105648 9075\n",
      "17\t95977\t0.3\t1\t87931 8046\n",
      "18\t85383\t0.3\t1\t78648 6735\n",
      "19\t112212\t0.3\t1\t103391 8821\n",
      "20\t104516\t0.3\t1\t96596 7920\n",
      "21\t91790\t0.3\t1\t84587 7203\n",
      "22\t103890\t0.3\t1\t96083 7807\n",
      "23\t109193\t0.3\t1\t100822 8371\n",
      "24\t108259\t0.3\t1\t99753 8506\n",
      "25\t121670\t0.3\t1\t113120 8550\n",
      "26\t100521\t0.3\t1\t92626 7895\n",
      "27\t113578\t0.3\t1\t104198 9380\n",
      "28\t101690\t0.3\t1\t94983 6707\n",
      "29\t113641\t0.3\t1\t105222 8419\n",
      "30\t108610\t0.3\t1\t101210 7400\n",
      "31\t120920\t0.3\t1\t112244 8676\n",
      "32\t104000\t0.3\t1\t96851 7149\n",
      "33\t120229\t0.3\t1\t111277 8952\n",
      "34\t153685\t0.3\t1\t142847 10838\n",
      "35\t83436\t0.3\t1\t77543 5893\n",
      "36\t120970\t0.3\t1\t112035 8935\n",
      "37\t127408\t0.3\t1\t118333 9075\n",
      "38\t119755\t0.3\t1\t112084 7671\n",
      "39\t113420\t0.3\t1\t106417 7003\n",
      "40\t129274\t0.3\t1\t120419 8855\n",
      "41\t124811\t0.3\t1\t116558 8253\n",
      "42\t127899\t0.3\t1\t119534 8365\n",
      "43\t115450\t0.3\t1\t107954 7496\n",
      "44\t133939\t0.3\t1\t124786 9153\n",
      "45\t139673\t0.3\t1\t130506 9167\n",
      "46\t102301\t0.3\t1\t95697 6604\n",
      "47\t129785\t0.3\t1\t121923 7862\n",
      "48\t125953\t0.3\t1\t118248 7705\n",
      "49\t131617\t0.3\t1\t122911 8706\n",
      "50\t126447\t0.3\t1\t118299 8148\n",
      "51\t153054\t0.3\t1\t143414 9640\n",
      "52\t111902\t0.3\t1\t104877 7025\n",
      "53\t136845\t0.3\t1\t129094 7751\n",
      "54\t100542\t0.3\t1\t94550 5992\n",
      "55\t131225\t0.3\t1\t123466 7759\n",
      "56\t134910\t0.3\t1\t126386 8524\n",
      "57\t127245\t0.3\t1\t120240 7005\n",
      "58\t123733\t0.3\t1\t116761 6972\n",
      "59\t122876\t0.3\t1\t115856 7020\n",
      "60\t129560\t0.3\t1\t122153 7407\n",
      "61\t129535\t0.3\t1\t122248 7287\n",
      "62\t141327\t0.3\t1\t132778 8549\n",
      "63\t187124\t0.3\t1\t177463 9661\n",
      "64\t148865\t0.3\t1\t142160 6705\n",
      "65\t101493\t0.3\t1\t96017 5476\n",
      "66\t109966\t0.3\t1\t103977 5989\n",
      "67\t109147\t0.3\t1\t102972 6175\n",
      "68\t110252\t0.3\t1\t104088 6164\n",
      "69\t109266\t0.3\t1\t102978 6288\n",
      "70\t115917\t0.3\t1\t109437 6480\n",
      "71\t118081\t0.3\t1\t111469 6612\n",
      "72\t111629\t0.3\t1\t105360 6269\n",
      "73\t116346\t0.3\t1\t109876 6470\n",
      "74\t106757\t0.3\t1\t100604 6153\n",
      "75\t101733\t0.3\t1\t95970 5763\n",
      "76\t99872\t0.3\t1\t94223 5649\n",
      "77\t99194\t0.3\t1\t93520 5674\n",
      "78\t93229\t0.3\t1\t87953 5276\n",
      "79\t90739\t0.3\t1\t85605 5134\n",
      "80\t89347\t0.3\t1\t84416 4931\n",
      "81\t88970\t0.3\t1\t84141 4829\n",
      "82\t84536\t0.3\t1\t79929 4607\n",
      "83\t81530\t0.3\t1\t77128 4402\n",
      "84\t78480\t0.3\t1\t74228 4252\n",
      "85\t77346\t0.3\t1\t73110 4236\n",
      "86\t75912\t0.3\t1\t71909 4003\n",
      "87\t75393\t0.3\t1\t71530 3863\n",
      "88\t73839\t0.3\t1\t69922 3917\n",
      "89\t65607\t0.3\t1\t62233 3374\n",
      "90\t63315\t0.3\t1\t59995 3320\n",
      "91\t59514\t0.3\t1\t56442 3072\n",
      "92\t55456\t0.3\t1\t52465 2991\n",
      "93\t52632\t0.3\t1\t49948 2684\n",
      "94\t51163\t0.3\t1\t48481 2682\n",
      "95\t47871\t0.3\t1\t45320 2551\n",
      "96\t45902\t0.3\t1\t43474 2428\n",
      "97\t43468\t0.3\t1\t41231 2237\n",
      "98\t40225\t0.3\t1\t38177 2048\n",
      "99\t39861\t0.3\t1\t37850 2011\n",
      "100\t35922\t0.3\t1\t34182 1740\n",
      "101\t31906\t0.3\t1\t30324 1582\n",
      "102\t28881\t0.3\t1\t27260 1621\n",
      "103\t26612\t0.3\t1\t25202 1410\n",
      "104\t25092\t0.3\t1\t23747 1345\n",
      "105\t22553\t0.3\t1\t21323 1230\n",
      "106\t20249\t0.3\t1\t19195 1054\n",
      "107\t18698\t0.3\t1\t17704 994\n",
      "108\t16333\t0.3\t1\t15423 910\n",
      "109\t15105\t0.3\t1\t14342 763\n",
      "110\t13538\t0.3\t1\t12818 720\n",
      "111\t12635\t0.3\t1\t11928 707\n",
      "112\t10864\t0.3\t1\t10306 558\n",
      "113\t9370\t0.3\t1\t8881 489\n",
      "114\t8127\t0.3\t1\t7692 435\n",
      "115\t6826\t0.3\t1\t6440 386\n",
      "116\t5914\t0.3\t1\t5590 324\n",
      "117\t5167\t0.3\t1\t4841 326\n",
      "118\t4505\t0.3\t1\t4244 261\n",
      "119\t3657\t0.3\t1\t3441 216\n",
      "120\t3157\t0.3\t1\t2964 193\n",
      "121\t2768\t0.3\t1\t2620 148\n",
      "122\t2319\t0.3\t1\t2199 120\n",
      "123\t2119\t0.3\t1\t1978 141\n",
      "124\t1696\t0.3\t1\t1576 120\n",
      "125\t1359\t0.3\t1\t1259 100\n",
      "126\t1191\t0.3\t1\t1105 86\n",
      "127\t1007\t0.3\t1\t955 52\n",
      "128\t721\t0.3\t1\t673 48\n",
      "129\t632\t0.3\t1\t599 33\n",
      "130\t537\t0.3\t1\t494 43\n",
      "131\t454\t0.3\t1\t423 31\n",
      "132\t387\t0.3\t1\t367 20\n",
      "133\t461\t0.3\t1\t442 19\n",
      "134\t637\t0.3\t1\t611 26\n",
      "135\t279\t0.3\t1\t264 15\n",
      "136\t164\t0.3\t1\t153 11\n",
      "137\t121\t0.3\t1\t110 11\n",
      "138\t99\t0.3\t1\t89 10\n",
      "139\t86\t0.3\t1\t79 7\n",
      "140\t59\t0.3\t1\t52 7\n",
      "141\t49\t0.3\t1\t44 5\n",
      "142\t39\t0.3\t1\t36 3\n",
      "143\t48\t0.3\t1\t45 3\n",
      "144\t41\t0.3\t1\t36 5\n",
      "145\t47\t0.3\t1\t41 6\n",
      "146\t121\t0.3\t1\t118 3\n",
      "147\t643\t0.3\t1\t616 27\n",
      "148\t431\t0.3\t1\t416 15\n",
      "149\t244\t0.3\t1\t225 19\n",
      "150\t2704\t0.3\t1\t2570 134\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17174050 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz, file_2: 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17174050\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 30556 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_2_S14_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_2_S14_L002_R1_001_trimmed.fq.gz and 16C_26psu_2_S14_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_3_S15_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 547.99 s (37 us/read; 1.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,705,097\n",
      "Reads with adapters:                11,084,660 (75.4%)\n",
      "Reads written (passing filters):    14,705,097 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,205,764,550 bp\n",
      "Quality-trimmed:               5,654,261 bp (0.3%)\n",
      "Total written (filtered):  1,768,343,252 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11084660 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 18.1%\n",
      "  G: 19.4%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2274798\t3676274.2\t0\t2274798\n",
      "2\t348366\t919068.6\t0\t348366\n",
      "3\t204139\t229767.1\t0\t204139\n",
      "4\t102009\t57441.8\t0\t102009\n",
      "5\t74565\t14360.4\t0\t74565\n",
      "6\t72782\t3590.1\t0\t72782\n",
      "7\t57000\t897.5\t0\t57000\n",
      "8\t74016\t224.4\t0\t74016\n",
      "9\t74077\t56.1\t0\t73070 1007\n",
      "10\t67767\t14.0\t1\t63550 4217\n",
      "11\t78426\t3.5\t1\t71779 6647\n",
      "12\t73178\t0.9\t1\t67904 5274\n",
      "13\t73121\t0.2\t1\t67865 5256\n",
      "14\t76375\t0.2\t1\t70227 6148\n",
      "15\t78023\t0.2\t1\t72295 5728\n",
      "16\t76757\t0.2\t1\t70359 6398\n",
      "17\t96301\t0.2\t1\t87532 8769\n",
      "18\t78109\t0.2\t1\t72884 5225\n",
      "19\t66841\t0.2\t1\t61960 4881\n",
      "20\t81976\t0.2\t1\t75979 5997\n",
      "21\t91230\t0.2\t1\t84192 7038\n",
      "22\t74923\t0.2\t1\t70026 4897\n",
      "23\t89318\t0.2\t1\t82578 6740\n",
      "24\t80595\t0.2\t1\t73239 7356\n",
      "25\t88530\t0.2\t1\t81693 6837\n",
      "26\t80024\t0.2\t1\t74706 5318\n",
      "27\t90563\t0.2\t1\t83032 7531\n",
      "28\t88117\t0.2\t1\t81366 6751\n",
      "29\t90600\t0.2\t1\t82528 8072\n",
      "30\t100870\t0.2\t1\t94452 6418\n",
      "31\t72291\t0.2\t1\t67161 5130\n",
      "32\t93105\t0.2\t1\t86376 6729\n",
      "33\t85828\t0.2\t1\t79302 6526\n",
      "34\t93973\t0.2\t1\t87113 6860\n",
      "35\t95201\t0.2\t1\t86703 8498\n",
      "36\t98814\t0.2\t1\t91143 7671\n",
      "37\t92594\t0.2\t1\t86632 5962\n",
      "38\t92492\t0.2\t1\t85700 6792\n",
      "39\t99594\t0.2\t1\t92552 7042\n",
      "40\t99241\t0.2\t1\t91728 7513\n",
      "41\t109705\t0.2\t1\t100318 9387\n",
      "42\t91534\t0.2\t1\t84273 7261\n",
      "43\t144391\t0.2\t1\t135954 8437\n",
      "44\t48359\t0.2\t1\t43755 4604\n",
      "45\t101091\t0.2\t1\t93630 7461\n",
      "46\t97319\t0.2\t1\t90173 7146\n",
      "47\t102355\t0.2\t1\t95522 6833\n",
      "48\t95140\t0.2\t1\t87635 7505\n",
      "49\t109171\t0.2\t1\t100789 8382\n",
      "50\t88275\t0.2\t1\t82648 5627\n",
      "51\t96905\t0.2\t1\t90014 6891\n",
      "52\t106355\t0.2\t1\t97463 8892\n",
      "53\t116272\t0.2\t1\t109052 7220\n",
      "54\t82504\t0.2\t1\t76626 5878\n",
      "55\t104155\t0.2\t1\t97146 7009\n",
      "56\t117878\t0.2\t1\t110588 7290\n",
      "57\t94064\t0.2\t1\t87248 6816\n",
      "58\t98547\t0.2\t1\t93098 5449\n",
      "59\t78147\t0.2\t1\t72464 5683\n",
      "60\t103609\t0.2\t1\t96779 6830\n",
      "61\t99715\t0.2\t1\t93367 6348\n",
      "62\t96712\t0.2\t1\t90590 6122\n",
      "63\t91683\t0.2\t1\t85401 6282\n",
      "64\t88404\t0.2\t1\t82993 5411\n",
      "65\t98564\t0.2\t1\t92125 6439\n",
      "66\t106366\t0.2\t1\t99301 7065\n",
      "67\t113416\t0.2\t1\t104214 9202\n",
      "68\t212253\t0.2\t1\t201102 11151\n",
      "69\t52200\t0.2\t1\t48055 4145\n",
      "70\t18616\t0.2\t1\t15750 2866\n",
      "71\t46522\t0.2\t1\t43178 3344\n",
      "72\t78090\t0.2\t1\t73258 4832\n",
      "73\t85268\t0.2\t1\t79938 5330\n",
      "74\t84510\t0.2\t1\t79145 5365\n",
      "75\t84172\t0.2\t1\t78812 5360\n",
      "76\t82514\t0.2\t1\t77266 5248\n",
      "77\t82709\t0.2\t1\t77348 5361\n",
      "78\t77578\t0.2\t1\t72635 4943\n",
      "79\t76394\t0.2\t1\t71265 5129\n",
      "80\t74740\t0.2\t1\t69853 4887\n",
      "81\t73428\t0.2\t1\t68638 4790\n",
      "82\t72487\t0.2\t1\t67836 4651\n",
      "83\t69366\t0.2\t1\t64850 4516\n",
      "84\t68485\t0.2\t1\t64014 4471\n",
      "85\t65855\t0.2\t1\t61694 4161\n",
      "86\t63892\t0.2\t1\t59720 4172\n",
      "87\t59968\t0.2\t1\t56023 3945\n",
      "88\t59481\t0.2\t1\t55667 3814\n",
      "89\t57389\t0.2\t1\t53698 3691\n",
      "90\t56546\t0.2\t1\t52783 3763\n",
      "91\t53907\t0.2\t1\t50295 3612\n",
      "92\t51001\t0.2\t1\t47631 3370\n",
      "93\t48245\t0.2\t1\t45064 3181\n",
      "94\t47040\t0.2\t1\t43981 3059\n",
      "95\t44130\t0.2\t1\t41149 2981\n",
      "96\t41844\t0.2\t1\t38970 2874\n",
      "97\t39077\t0.2\t1\t36479 2598\n",
      "98\t36411\t0.2\t1\t33904 2507\n",
      "99\t35258\t0.2\t1\t32713 2545\n",
      "100\t32027\t0.2\t1\t29780 2247\n",
      "101\t30331\t0.2\t1\t28259 2072\n",
      "102\t28208\t0.2\t1\t26138 2070\n",
      "103\t26173\t0.2\t1\t24325 1848\n",
      "104\t24471\t0.2\t1\t22816 1655\n",
      "105\t21780\t0.2\t1\t20187 1593\n",
      "106\t20446\t0.2\t1\t18956 1490\n",
      "107\t18149\t0.2\t1\t16771 1378\n",
      "108\t17180\t0.2\t1\t15945 1235\n",
      "109\t15082\t0.2\t1\t13906 1176\n",
      "110\t14103\t0.2\t1\t12998 1105\n",
      "111\t13190\t0.2\t1\t12191 999\n",
      "112\t11450\t0.2\t1\t10565 885\n",
      "113\t9717\t0.2\t1\t8858 859\n",
      "114\t8899\t0.2\t1\t8160 739\n",
      "115\t7941\t0.2\t1\t7316 625\n",
      "116\t6735\t0.2\t1\t6226 509\n",
      "117\t5866\t0.2\t1\t5379 487\n",
      "118\t5055\t0.2\t1\t4584 471\n",
      "119\t4372\t0.2\t1\t3977 395\n",
      "120\t3711\t0.2\t1\t3362 349\n",
      "121\t3540\t0.2\t1\t3243 297\n",
      "122\t2903\t0.2\t1\t2643 260\n",
      "123\t2410\t0.2\t1\t2172 238\n",
      "124\t2227\t0.2\t1\t2020 207\n",
      "125\t1832\t0.2\t1\t1640 192\n",
      "126\t1658\t0.2\t1\t1492 166\n",
      "127\t1471\t0.2\t1\t1327 144\n",
      "128\t1080\t0.2\t1\t953 127\n",
      "129\t958\t0.2\t1\t848 110\n",
      "130\t709\t0.2\t1\t628 81\n",
      "131\t795\t0.2\t1\t718 77\n",
      "132\t862\t0.2\t1\t774 88\n",
      "133\t1124\t0.2\t1\t1033 91\n",
      "134\t1984\t0.2\t1\t1883 101\n",
      "135\t812\t0.2\t1\t780 32\n",
      "136\t310\t0.2\t1\t269 41\n",
      "137\t196\t0.2\t1\t180 16\n",
      "138\t185\t0.2\t1\t168 17\n",
      "139\t138\t0.2\t1\t131 7\n",
      "140\t94\t0.2\t1\t84 10\n",
      "141\t74\t0.2\t1\t68 6\n",
      "142\t62\t0.2\t1\t50 12\n",
      "143\t92\t0.2\t1\t84 8\n",
      "144\t74\t0.2\t1\t67 7\n",
      "145\t205\t0.2\t1\t196 9\n",
      "146\t780\t0.2\t1\t755 25\n",
      "147\t4835\t0.2\t1\t4652 183\n",
      "148\t3283\t0.2\t1\t3150 133\n",
      "149\t1826\t0.2\t1\t1757 69\n",
      "150\t16714\t0.2\t1\t15810 904\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14705097 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_3_S15_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 583.17 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,705,097\n",
      "Reads with adapters:                10,980,117 (74.7%)\n",
      "Reads written (passing filters):    14,705,097 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,205,764,550 bp\n",
      "Quality-trimmed:              13,510,954 bp (0.6%)\n",
      "Total written (filtered):  1,769,417,806 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10980117 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 14.1%\n",
      "  G: 23.8%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2264307\t3676274.2\t0\t2264307\n",
      "2\t373214\t919068.6\t0\t373214\n",
      "3\t218063\t229767.1\t0\t218063\n",
      "4\t104965\t57441.8\t0\t104965\n",
      "5\t83066\t14360.4\t0\t83066\n",
      "6\t70406\t3590.1\t0\t70406\n",
      "7\t54672\t897.5\t0\t54672\n",
      "8\t80609\t224.4\t0\t80609\n",
      "9\t63967\t56.1\t0\t63675 292\n",
      "10\t69384\t14.0\t1\t65411 3973\n",
      "11\t74885\t3.5\t1\t69453 5432\n",
      "12\t74890\t0.9\t1\t69291 5599\n",
      "13\t82476\t0.2\t1\t76999 5477\n",
      "14\t78491\t0.2\t1\t72263 6228\n",
      "15\t61874\t0.2\t1\t57566 4308\n",
      "16\t91303\t0.2\t1\t84745 6558\n",
      "17\t72220\t0.2\t1\t66746 5474\n",
      "18\t69582\t0.2\t1\t64867 4715\n",
      "19\t88040\t0.2\t1\t81744 6296\n",
      "20\t81620\t0.2\t1\t76073 5547\n",
      "21\t72843\t0.2\t1\t67713 5130\n",
      "22\t82391\t0.2\t1\t76741 5650\n",
      "23\t83759\t0.2\t1\t77887 5872\n",
      "24\t82590\t0.2\t1\t76725 5865\n",
      "25\t94974\t0.2\t1\t88688 6286\n",
      "26\t78108\t0.2\t1\t72602 5506\n",
      "27\t87345\t0.2\t1\t80708 6637\n",
      "28\t80301\t0.2\t1\t75686 4615\n",
      "29\t86749\t0.2\t1\t80805 5944\n",
      "30\t84969\t0.2\t1\t79761 5208\n",
      "31\t91560\t0.2\t1\t85496 6064\n",
      "32\t82472\t0.2\t1\t77608 4864\n",
      "33\t91075\t0.2\t1\t85011 6064\n",
      "34\t115701\t0.2\t1\t108310 7391\n",
      "35\t65587\t0.2\t1\t61631 3956\n",
      "36\t92396\t0.2\t1\t86112 6284\n",
      "37\t97880\t0.2\t1\t91450 6430\n",
      "38\t93812\t0.2\t1\t88555 5257\n",
      "39\t89857\t0.2\t1\t84658 5199\n",
      "40\t100686\t0.2\t1\t94165 6521\n",
      "41\t95917\t0.2\t1\t90320 5597\n",
      "42\t99426\t0.2\t1\t93710 5716\n",
      "43\t89895\t0.2\t1\t84632 5263\n",
      "44\t101978\t0.2\t1\t95733 6245\n",
      "45\t107145\t0.2\t1\t100731 6414\n",
      "46\t78559\t0.2\t1\t74098 4461\n",
      "47\t99270\t0.2\t1\t93634 5636\n",
      "48\t97566\t0.2\t1\t92156 5410\n",
      "49\t101230\t0.2\t1\t95394 5836\n",
      "50\t95465\t0.2\t1\t89881 5584\n",
      "51\t119107\t0.2\t1\t112449 6658\n",
      "52\t84719\t0.2\t1\t80119 4600\n",
      "53\t101734\t0.2\t1\t96544 5190\n",
      "54\t78783\t0.2\t1\t74521 4262\n",
      "55\t102169\t0.2\t1\t97154 5015\n",
      "56\t104926\t0.2\t1\t99510 5416\n",
      "57\t97330\t0.2\t1\t92599 4731\n",
      "58\t94743\t0.2\t1\t90360 4383\n",
      "59\t96773\t0.2\t1\t91767 5006\n",
      "60\t98229\t0.2\t1\t93172 5057\n",
      "61\t101518\t0.2\t1\t96208 5310\n",
      "62\t108759\t0.2\t1\t103148 5611\n",
      "63\t145227\t0.2\t1\t138420 6807\n",
      "64\t118502\t0.2\t1\t113544 4958\n",
      "65\t77875\t0.2\t1\t74180 3695\n",
      "66\t85279\t0.2\t1\t81158 4121\n",
      "67\t82906\t0.2\t1\t78696 4210\n",
      "68\t83568\t0.2\t1\t79559 4009\n",
      "69\t83055\t0.2\t1\t79009 4046\n",
      "70\t85605\t0.2\t1\t81510 4095\n",
      "71\t91438\t0.2\t1\t87015 4423\n",
      "72\t86608\t0.2\t1\t82445 4163\n",
      "73\t91905\t0.2\t1\t87632 4273\n",
      "74\t84442\t0.2\t1\t80649 3793\n",
      "75\t80482\t0.2\t1\t76627 3855\n",
      "76\t78424\t0.2\t1\t74783 3641\n",
      "77\t78513\t0.2\t1\t74716 3797\n",
      "78\t74066\t0.2\t1\t70442 3624\n",
      "79\t70946\t0.2\t1\t67597 3349\n",
      "80\t70802\t0.2\t1\t67560 3242\n",
      "81\t69586\t0.2\t1\t66414 3172\n",
      "82\t67330\t0.2\t1\t64176 3154\n",
      "83\t64772\t0.2\t1\t61853 2919\n",
      "84\t63911\t0.2\t1\t60972 2939\n",
      "85\t62434\t0.2\t1\t59365 3069\n",
      "86\t62539\t0.2\t1\t59521 3018\n",
      "87\t61465\t0.2\t1\t58594 2871\n",
      "88\t60482\t0.2\t1\t57646 2836\n",
      "89\t54895\t0.2\t1\t52333 2562\n",
      "90\t52052\t0.2\t1\t49599 2453\n",
      "91\t49106\t0.2\t1\t46822 2284\n",
      "92\t45843\t0.2\t1\t43672 2171\n",
      "93\t44679\t0.2\t1\t42566 2113\n",
      "94\t43439\t0.2\t1\t41408 2031\n",
      "95\t41369\t0.2\t1\t39446 1923\n",
      "96\t39301\t0.2\t1\t37521 1780\n",
      "97\t36285\t0.2\t1\t34613 1672\n",
      "98\t34034\t0.2\t1\t32424 1610\n",
      "99\t33720\t0.2\t1\t32265 1455\n",
      "100\t30495\t0.2\t1\t29039 1456\n",
      "101\t27811\t0.2\t1\t26487 1324\n",
      "102\t25647\t0.2\t1\t24471 1176\n",
      "103\t23848\t0.2\t1\t22756 1092\n",
      "104\t23030\t0.2\t1\t22018 1012\n",
      "105\t20481\t0.2\t1\t19504 977\n",
      "106\t19291\t0.2\t1\t18417 874\n",
      "107\t17027\t0.2\t1\t16232 795\n",
      "108\t15744\t0.2\t1\t15015 729\n",
      "109\t13986\t0.2\t1\t13295 691\n",
      "110\t13128\t0.2\t1\t12523 605\n",
      "111\t12855\t0.2\t1\t12274 581\n",
      "112\t11119\t0.2\t1\t10542 577\n",
      "113\t9437\t0.2\t1\t8968 469\n",
      "114\t8605\t0.2\t1\t8220 385\n",
      "115\t7442\t0.2\t1\t7067 375\n",
      "116\t6235\t0.2\t1\t5927 308\n",
      "117\t5342\t0.2\t1\t5066 276\n",
      "118\t4634\t0.2\t1\t4366 268\n",
      "119\t4041\t0.2\t1\t3836 205\n",
      "120\t3372\t0.2\t1\t3167 205\n",
      "121\t3329\t0.2\t1\t3149 180\n",
      "122\t2717\t0.2\t1\t2578 139\n",
      "123\t2317\t0.2\t1\t2172 145\n",
      "124\t2087\t0.2\t1\t1969 118\n",
      "125\t1757\t0.2\t1\t1630 127\n",
      "126\t1582\t0.2\t1\t1504 78\n",
      "127\t1368\t0.2\t1\t1287 81\n",
      "128\t1019\t0.2\t1\t960 59\n",
      "129\t887\t0.2\t1\t843 44\n",
      "130\t669\t0.2\t1\t637 32\n",
      "131\t775\t0.2\t1\t731 44\n",
      "132\t814\t0.2\t1\t771 43\n",
      "133\t1245\t0.2\t1\t1202 43\n",
      "134\t1982\t0.2\t1\t1926 56\n",
      "135\t810\t0.2\t1\t778 32\n",
      "136\t281\t0.2\t1\t260 21\n",
      "137\t198\t0.2\t1\t181 17\n",
      "138\t183\t0.2\t1\t171 12\n",
      "139\t135\t0.2\t1\t124 11\n",
      "140\t85\t0.2\t1\t83 2\n",
      "141\t71\t0.2\t1\t66 5\n",
      "142\t55\t0.2\t1\t52 3\n",
      "143\t87\t0.2\t1\t84 3\n",
      "144\t82\t0.2\t1\t77 5\n",
      "145\t214\t0.2\t1\t204 10\n",
      "146\t799\t0.2\t1\t768 31\n",
      "147\t4799\t0.2\t1\t4649 150\n",
      "148\t3222\t0.2\t1\t3110 112\n",
      "149\t1903\t0.2\t1\t1826 77\n",
      "150\t15860\t0.2\t1\t15305 555\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14705097 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz, file_2: 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14705097\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 67024 (0.46%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_3_S15_L001_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_3_S15_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 554.96 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,622,552\n",
      "Reads with adapters:                11,026,853 (75.4%)\n",
      "Reads written (passing filters):    14,622,552 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,193,382,800 bp\n",
      "Quality-trimmed:               5,795,035 bp (0.3%)\n",
      "Total written (filtered):  1,756,312,964 bp (80.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11026853 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.2%\n",
      "  C: 18.2%\n",
      "  G: 19.4%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2255035\t3655638.0\t0\t2255035\n",
      "2\t341709\t913909.5\t0\t341709\n",
      "3\t202869\t228477.4\t0\t202869\n",
      "4\t99930\t57119.3\t0\t99930\n",
      "5\t74502\t14279.8\t0\t74502\n",
      "6\t72604\t3570.0\t0\t72604\n",
      "7\t55151\t892.5\t0\t55151\n",
      "8\t72986\t223.1\t0\t72986\n",
      "9\t72897\t55.8\t0\t71832 1065\n",
      "10\t66371\t13.9\t1\t61637 4734\n",
      "11\t77738\t3.5\t1\t70168 7570\n",
      "12\t72088\t0.9\t1\t66222 5866\n",
      "13\t72584\t0.2\t1\t66413 6171\n",
      "14\t75037\t0.2\t1\t68093 6944\n",
      "15\t77546\t0.2\t1\t70821 6725\n",
      "16\t76138\t0.2\t1\t68711 7427\n",
      "17\t96440\t0.2\t1\t86428 10012\n",
      "18\t77278\t0.2\t1\t71237 6041\n",
      "19\t65884\t0.2\t1\t60354 5530\n",
      "20\t80966\t0.2\t1\t74091 6875\n",
      "21\t91593\t0.2\t1\t83562 8031\n",
      "22\t72795\t0.2\t1\t67343 5452\n",
      "23\t89502\t0.2\t1\t81872 7630\n",
      "24\t80431\t0.2\t1\t72300 8131\n",
      "25\t87978\t0.2\t1\t80303 7675\n",
      "26\t79303\t0.2\t1\t73214 6089\n",
      "27\t90256\t0.2\t1\t81812 8444\n",
      "28\t88151\t0.2\t1\t80669 7482\n",
      "29\t90769\t0.2\t1\t81855 8914\n",
      "30\t100156\t0.2\t1\t92655 7501\n",
      "31\t70749\t0.2\t1\t65028 5721\n",
      "32\t94056\t0.2\t1\t86125 7931\n",
      "33\t85223\t0.2\t1\t77723 7500\n",
      "34\t93938\t0.2\t1\t85684 8254\n",
      "35\t95264\t0.2\t1\t85673 9591\n",
      "36\t98487\t0.2\t1\t89904 8583\n",
      "37\t91367\t0.2\t1\t84496 6871\n",
      "38\t91067\t0.2\t1\t83299 7768\n",
      "39\t98566\t0.2\t1\t90240 8326\n",
      "40\t98669\t0.2\t1\t89677 8992\n",
      "41\t109128\t0.2\t1\t98383 10745\n",
      "42\t90124\t0.2\t1\t81722 8402\n",
      "43\t141587\t0.2\t1\t131681 9906\n",
      "44\t48458\t0.2\t1\t43335 5123\n",
      "45\t101474\t0.2\t1\t92754 8720\n",
      "46\t96951\t0.2\t1\t88935 8016\n",
      "47\t101587\t0.2\t1\t93762 7825\n",
      "48\t93451\t0.2\t1\t85286 8165\n",
      "49\t109864\t0.2\t1\t100369 9495\n",
      "50\t86419\t0.2\t1\t80081 6338\n",
      "51\t97144\t0.2\t1\t89290 7854\n",
      "52\t106661\t0.2\t1\t96904 9757\n",
      "53\t115583\t0.2\t1\t107526 8057\n",
      "54\t81720\t0.2\t1\t74951 6769\n",
      "55\t103125\t0.2\t1\t94975 8150\n",
      "56\t119342\t0.2\t1\t110505 8837\n",
      "57\t94296\t0.2\t1\t86511 7785\n",
      "58\t95906\t0.2\t1\t89537 6369\n",
      "59\t78497\t0.2\t1\t71935 6562\n",
      "60\t103672\t0.2\t1\t95850 7822\n",
      "61\t99103\t0.2\t1\t91989 7114\n",
      "62\t98519\t0.2\t1\t91357 7162\n",
      "63\t92047\t0.2\t1\t85069 6978\n",
      "64\t87141\t0.2\t1\t80859 6282\n",
      "65\t98035\t0.2\t1\t90511 7524\n",
      "66\t104856\t0.2\t1\t96876 7980\n",
      "67\t112499\t0.2\t1\t102554 9945\n",
      "68\t203504\t0.2\t1\t191009 12495\n",
      "69\t62594\t0.2\t1\t57415 5179\n",
      "70\t23252\t0.2\t1\t19824 3428\n",
      "71\t45182\t0.2\t1\t41443 3739\n",
      "72\t77078\t0.2\t1\t71850 5228\n",
      "73\t85031\t0.2\t1\t79046 5985\n",
      "74\t84254\t0.2\t1\t78257 5997\n",
      "75\t83996\t0.2\t1\t77711 6285\n",
      "76\t82731\t0.2\t1\t76446 6285\n",
      "77\t82380\t0.2\t1\t76363 6017\n",
      "78\t77279\t0.2\t1\t71312 5967\n",
      "79\t77209\t0.2\t1\t71223 5986\n",
      "80\t74817\t0.2\t1\t69029 5788\n",
      "81\t73406\t0.2\t1\t67839 5567\n",
      "82\t72738\t0.2\t1\t67207 5531\n",
      "83\t69208\t0.2\t1\t64025 5183\n",
      "84\t68393\t0.2\t1\t63216 5177\n",
      "85\t66034\t0.2\t1\t60887 5147\n",
      "86\t63919\t0.2\t1\t59135 4784\n",
      "87\t60801\t0.2\t1\t56313 4488\n",
      "88\t60788\t0.2\t1\t56093 4695\n",
      "89\t57692\t0.2\t1\t53269 4423\n",
      "90\t56584\t0.2\t1\t52478 4106\n",
      "91\t53589\t0.2\t1\t49546 4043\n",
      "92\t51833\t0.2\t1\t47890 3943\n",
      "93\t48199\t0.2\t1\t44575 3624\n",
      "94\t46944\t0.2\t1\t43278 3666\n",
      "95\t43653\t0.2\t1\t40307 3346\n",
      "96\t41698\t0.2\t1\t38443 3255\n",
      "97\t38960\t0.2\t1\t36083 2877\n",
      "98\t36779\t0.2\t1\t33843 2936\n",
      "99\t34473\t0.2\t1\t31757 2716\n",
      "100\t32693\t0.2\t1\t30095 2598\n",
      "101\t30447\t0.2\t1\t28045 2402\n",
      "102\t27845\t0.2\t1\t25608 2237\n",
      "103\t26414\t0.2\t1\t24289 2125\n",
      "104\t24929\t0.2\t1\t23050 1879\n",
      "105\t21773\t0.2\t1\t20195 1578\n",
      "106\t20226\t0.2\t1\t18586 1640\n",
      "107\t18400\t0.2\t1\t16897 1503\n",
      "108\t17326\t0.2\t1\t15891 1435\n",
      "109\t15645\t0.2\t1\t14342 1303\n",
      "110\t14020\t0.2\t1\t12816 1204\n",
      "111\t12849\t0.2\t1\t11744 1105\n",
      "112\t11234\t0.2\t1\t10286 948\n",
      "113\t9985\t0.2\t1\t9120 865\n",
      "114\t8637\t0.2\t1\t7820 817\n",
      "115\t7857\t0.2\t1\t7138 719\n",
      "116\t6705\t0.2\t1\t6073 632\n",
      "117\t6037\t0.2\t1\t5491 546\n",
      "118\t5126\t0.2\t1\t4659 467\n",
      "119\t4476\t0.2\t1\t4067 409\n",
      "120\t3856\t0.2\t1\t3446 410\n",
      "121\t3506\t0.2\t1\t3100 406\n",
      "122\t2914\t0.2\t1\t2611 303\n",
      "123\t2582\t0.2\t1\t2308 274\n",
      "124\t2250\t0.2\t1\t1996 254\n",
      "125\t1869\t0.2\t1\t1644 225\n",
      "126\t1660\t0.2\t1\t1451 209\n",
      "127\t1486\t0.2\t1\t1318 168\n",
      "128\t1135\t0.2\t1\t1016 119\n",
      "129\t1048\t0.2\t1\t958 90\n",
      "130\t730\t0.2\t1\t642 88\n",
      "131\t781\t0.2\t1\t698 83\n",
      "132\t769\t0.2\t1\t708 61\n",
      "133\t1082\t0.2\t1\t1006 76\n",
      "134\t1977\t0.2\t1\t1867 110\n",
      "135\t796\t0.2\t1\t751 45\n",
      "136\t320\t0.2\t1\t290 30\n",
      "137\t204\t0.2\t1\t187 17\n",
      "138\t179\t0.2\t1\t161 18\n",
      "139\t112\t0.2\t1\t104 8\n",
      "140\t111\t0.2\t1\t94 17\n",
      "141\t71\t0.2\t1\t68 3\n",
      "142\t79\t0.2\t1\t67 12\n",
      "143\t106\t0.2\t1\t93 13\n",
      "144\t63\t0.2\t1\t61 2\n",
      "145\t233\t0.2\t1\t213 20\n",
      "146\t995\t0.2\t1\t960 35\n",
      "147\t4990\t0.2\t1\t4774 216\n",
      "148\t3265\t0.2\t1\t3135 130\n",
      "149\t1955\t0.2\t1\t1846 109\n",
      "150\t16845\t0.2\t1\t15820 1025\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14622552 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_3_S15_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 593.54 s (41 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,622,552\n",
      "Reads with adapters:                10,913,941 (74.6%)\n",
      "Reads written (passing filters):    14,622,552 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,193,382,800 bp\n",
      "Quality-trimmed:              13,012,389 bp (0.6%)\n",
      "Total written (filtered):  1,758,102,450 bp (80.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10913941 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 14.2%\n",
      "  G: 23.9%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2243893\t3655638.0\t0\t2243893\n",
      "2\t361235\t913909.5\t0\t361235\n",
      "3\t221045\t228477.4\t0\t221045\n",
      "4\t102953\t57119.3\t0\t102953\n",
      "5\t80629\t14279.8\t0\t80629\n",
      "6\t70223\t3570.0\t0\t70223\n",
      "7\t53470\t892.5\t0\t53470\n",
      "8\t79479\t223.1\t0\t79479\n",
      "9\t62911\t55.8\t0\t62617 294\n",
      "10\t66925\t13.9\t1\t61688 5237\n",
      "11\t73700\t3.5\t1\t66819 6881\n",
      "12\t72572\t0.9\t1\t65940 6632\n",
      "13\t82700\t0.2\t1\t75488 7212\n",
      "14\t76573\t0.2\t1\t68946 7627\n",
      "15\t61263\t0.2\t1\t55865 5398\n",
      "16\t90146\t0.2\t1\t82109 8037\n",
      "17\t74386\t0.2\t1\t67549 6837\n",
      "18\t67318\t0.2\t1\t61385 5933\n",
      "19\t87145\t0.2\t1\t79499 7646\n",
      "20\t80434\t0.2\t1\t73597 6837\n",
      "21\t71959\t0.2\t1\t65758 6201\n",
      "22\t80520\t0.2\t1\t73704 6816\n",
      "23\t83808\t0.2\t1\t76805 7003\n",
      "24\t83121\t0.2\t1\t75853 7268\n",
      "25\t94721\t0.2\t1\t87319 7402\n",
      "26\t78397\t0.2\t1\t71438 6959\n",
      "27\t87589\t0.2\t1\t79828 7761\n",
      "28\t77436\t0.2\t1\t71759 5677\n",
      "29\t88231\t0.2\t1\t81047 7184\n",
      "30\t81944\t0.2\t1\t75545 6399\n",
      "31\t92581\t0.2\t1\t85292 7289\n",
      "32\t80085\t0.2\t1\t74132 5953\n",
      "33\t91916\t0.2\t1\t84460 7456\n",
      "34\t117300\t0.2\t1\t108202 9098\n",
      "35\t62830\t0.2\t1\t57836 4994\n",
      "36\t92603\t0.2\t1\t85123 7480\n",
      "37\t97494\t0.2\t1\t89682 7812\n",
      "38\t93013\t0.2\t1\t86322 6691\n",
      "39\t86916\t0.2\t1\t80841 6075\n",
      "40\t100050\t0.2\t1\t92533 7517\n",
      "41\t96502\t0.2\t1\t89613 6889\n",
      "42\t98694\t0.2\t1\t91554 7140\n",
      "43\t88091\t0.2\t1\t81948 6143\n",
      "44\t102146\t0.2\t1\t94589 7557\n",
      "45\t106497\t0.2\t1\t98693 7804\n",
      "46\t77918\t0.2\t1\t72391 5527\n",
      "47\t98307\t0.2\t1\t91760 6547\n",
      "48\t96325\t0.2\t1\t89754 6571\n",
      "49\t100699\t0.2\t1\t93521 7178\n",
      "50\t95936\t0.2\t1\t89184 6752\n",
      "51\t117412\t0.2\t1\t109335 8077\n",
      "52\t84318\t0.2\t1\t78674 5644\n",
      "53\t103753\t0.2\t1\t97370 6383\n",
      "54\t76703\t0.2\t1\t71688 5015\n",
      "55\t100303\t0.2\t1\t94078 6225\n",
      "56\t104185\t0.2\t1\t97076 7109\n",
      "57\t98544\t0.2\t1\t92645 5899\n",
      "58\t94450\t0.2\t1\t88927 5523\n",
      "59\t94831\t0.2\t1\t88887 5944\n",
      "60\t99091\t0.2\t1\t92777 6314\n",
      "61\t100905\t0.2\t1\t94656 6249\n",
      "62\t109121\t0.2\t1\t102218 6903\n",
      "63\t142655\t0.2\t1\t134487 8168\n",
      "64\t117121\t0.2\t1\t111484 5637\n",
      "65\t77324\t0.2\t1\t72904 4420\n",
      "66\t84925\t0.2\t1\t79850 5075\n",
      "67\t82998\t0.2\t1\t77768 5230\n",
      "68\t84356\t0.2\t1\t79351 5005\n",
      "69\t83969\t0.2\t1\t78702 5267\n",
      "70\t86991\t0.2\t1\t81734 5257\n",
      "71\t92058\t0.2\t1\t86658 5400\n",
      "72\t87408\t0.2\t1\t82156 5252\n",
      "73\t92562\t0.2\t1\t87094 5468\n",
      "74\t84507\t0.2\t1\t79446 5061\n",
      "75\t80670\t0.2\t1\t75777 4893\n",
      "76\t78563\t0.2\t1\t73818 4745\n",
      "77\t77927\t0.2\t1\t73312 4615\n",
      "78\t73796\t0.2\t1\t69197 4599\n",
      "79\t71841\t0.2\t1\t67592 4249\n",
      "80\t70609\t0.2\t1\t66510 4099\n",
      "81\t69349\t0.2\t1\t65453 3896\n",
      "82\t67523\t0.2\t1\t63514 4009\n",
      "83\t64665\t0.2\t1\t60869 3796\n",
      "84\t63972\t0.2\t1\t60280 3692\n",
      "85\t62543\t0.2\t1\t58851 3692\n",
      "86\t62248\t0.2\t1\t58801 3447\n",
      "87\t61991\t0.2\t1\t58664 3327\n",
      "88\t61620\t0.2\t1\t58173 3447\n",
      "89\t55062\t0.2\t1\t51949 3113\n",
      "90\t52411\t0.2\t1\t49511 2900\n",
      "91\t49315\t0.2\t1\t46549 2766\n",
      "92\t47010\t0.2\t1\t44328 2682\n",
      "93\t44789\t0.2\t1\t42339 2450\n",
      "94\t43445\t0.2\t1\t41012 2433\n",
      "95\t40986\t0.2\t1\t38680 2306\n",
      "96\t39258\t0.2\t1\t37094 2164\n",
      "97\t36644\t0.2\t1\t34643 2001\n",
      "98\t34564\t0.2\t1\t32670 1894\n",
      "99\t32993\t0.2\t1\t31157 1836\n",
      "100\t31055\t0.2\t1\t29316 1739\n",
      "101\t27872\t0.2\t1\t26391 1481\n",
      "102\t25485\t0.2\t1\t24012 1473\n",
      "103\t24164\t0.2\t1\t22767 1397\n",
      "104\t23318\t0.2\t1\t22028 1290\n",
      "105\t20591\t0.2\t1\t19438 1153\n",
      "106\t19078\t0.2\t1\t17973 1105\n",
      "107\t17270\t0.2\t1\t16333 937\n",
      "108\t15786\t0.2\t1\t14877 909\n",
      "109\t14538\t0.2\t1\t13717 821\n",
      "110\t13063\t0.2\t1\t12333 730\n",
      "111\t12532\t0.2\t1\t11819 713\n",
      "112\t10933\t0.2\t1\t10317 616\n",
      "113\t9707\t0.2\t1\t9177 530\n",
      "114\t8352\t0.2\t1\t7861 491\n",
      "115\t7373\t0.2\t1\t6927 446\n",
      "116\t6174\t0.2\t1\t5812 362\n",
      "117\t5508\t0.2\t1\t5151 357\n",
      "118\t4695\t0.2\t1\t4387 308\n",
      "119\t4109\t0.2\t1\t3838 271\n",
      "120\t3479\t0.2\t1\t3278 201\n",
      "121\t3246\t0.2\t1\t3039 207\n",
      "122\t2778\t0.2\t1\t2609 169\n",
      "123\t2432\t0.2\t1\t2288 144\n",
      "124\t2091\t0.2\t1\t1964 127\n",
      "125\t1753\t0.2\t1\t1626 127\n",
      "126\t1569\t0.2\t1\t1461 108\n",
      "127\t1430\t0.2\t1\t1321 109\n",
      "128\t1051\t0.2\t1\t972 79\n",
      "129\t984\t0.2\t1\t936 48\n",
      "130\t686\t0.2\t1\t639 47\n",
      "131\t713\t0.2\t1\t668 45\n",
      "132\t719\t0.2\t1\t681 38\n",
      "133\t1234\t0.2\t1\t1177 57\n",
      "134\t2006\t0.2\t1\t1917 89\n",
      "135\t763\t0.2\t1\t728 35\n",
      "136\t321\t0.2\t1\t299 22\n",
      "137\t192\t0.2\t1\t179 13\n",
      "138\t174\t0.2\t1\t164 10\n",
      "139\t115\t0.2\t1\t111 4\n",
      "140\t101\t0.2\t1\t87 14\n",
      "141\t72\t0.2\t1\t66 6\n",
      "142\t70\t0.2\t1\t63 7\n",
      "143\t96\t0.2\t1\t90 6\n",
      "144\t66\t0.2\t1\t61 5\n",
      "145\t236\t0.2\t1\t225 11\n",
      "146\t987\t0.2\t1\t943 44\n",
      "147\t4949\t0.2\t1\t4744 205\n",
      "148\t3207\t0.2\t1\t3070 137\n",
      "149\t1976\t0.2\t1\t1884 92\n",
      "150\t15929\t0.2\t1\t15207 722\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14622552 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz, file_2: 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14622552\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 65653 (0.45%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_3_S15_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_3_S15_L002_R1_001_trimmed.fq.gz and 16C_26psu_3_S15_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_4_S16_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 637.40 s (37 us/read; 1.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,274,454\n",
      "Reads with adapters:                13,728,900 (79.5%)\n",
      "Reads written (passing filters):    17,274,454 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,591,168,100 bp\n",
      "Quality-trimmed:               5,179,651 bp (0.2%)\n",
      "Total written (filtered):  1,987,019,797 bp (76.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13728900 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 17.1%\n",
      "  G: 19.6%\n",
      "  T: 33.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2136043\t4318613.5\t0\t2136043\n",
      "2\t318200\t1079653.4\t0\t318200\n",
      "3\t181191\t269913.3\t0\t181191\n",
      "4\t105391\t67478.3\t0\t105391\n",
      "5\t86281\t16869.6\t0\t86281\n",
      "6\t85402\t4217.4\t0\t85402\n",
      "7\t69532\t1054.3\t0\t69532\n",
      "8\t89673\t263.6\t0\t89673\n",
      "9\t85385\t65.9\t0\t84182 1203\n",
      "10\t81888\t16.5\t1\t77249 4639\n",
      "11\t92800\t4.1\t1\t85219 7581\n",
      "12\t87898\t1.0\t1\t82011 5887\n",
      "13\t88385\t0.3\t1\t82215 6170\n",
      "14\t91465\t0.3\t1\t84417 7048\n",
      "15\t93851\t0.3\t1\t87200 6651\n",
      "16\t93365\t0.3\t1\t85770 7595\n",
      "17\t112928\t0.3\t1\t103341 9587\n",
      "18\t97669\t0.3\t1\t91300 6369\n",
      "19\t82484\t0.3\t1\t76767 5717\n",
      "20\t99682\t0.3\t1\t92884 6798\n",
      "21\t110868\t0.3\t1\t102713 8155\n",
      "22\t94466\t0.3\t1\t88707 5759\n",
      "23\t107932\t0.3\t1\t100154 7778\n",
      "24\t103619\t0.3\t1\t94765 8854\n",
      "25\t110068\t0.3\t1\t102116 7952\n",
      "26\t100495\t0.3\t1\t94095 6400\n",
      "27\t112487\t0.3\t1\t103684 8803\n",
      "28\t109738\t0.3\t1\t101919 7819\n",
      "29\t116657\t0.3\t1\t106828 9829\n",
      "30\t126889\t0.3\t1\t119461 7428\n",
      "31\t92971\t0.3\t1\t86609 6362\n",
      "32\t117804\t0.3\t1\t109725 8079\n",
      "33\t111542\t0.3\t1\t103765 7777\n",
      "34\t119656\t0.3\t1\t110996 8660\n",
      "35\t119627\t0.3\t1\t110861 8766\n",
      "36\t135056\t0.3\t1\t123455 11601\n",
      "37\t141108\t0.3\t1\t132535 8573\n",
      "38\t98717\t0.3\t1\t92159 6558\n",
      "39\t131001\t0.3\t1\t121415 9586\n",
      "40\t131538\t0.3\t1\t121991 9547\n",
      "41\t151149\t0.3\t1\t140064 11085\n",
      "42\t103859\t0.3\t1\t95703 8156\n",
      "43\t200319\t0.3\t1\t189316 11003\n",
      "44\t53496\t0.3\t1\t48400 5096\n",
      "45\t131702\t0.3\t1\t122909 8793\n",
      "46\t127887\t0.3\t1\t119378 8509\n",
      "47\t129236\t0.3\t1\t120995 8241\n",
      "48\t131832\t0.3\t1\t122637 9195\n",
      "49\t143882\t0.3\t1\t133714 10168\n",
      "50\t119251\t0.3\t1\t111807 7444\n",
      "51\t131353\t0.3\t1\t122552 8801\n",
      "52\t136704\t0.3\t1\t126618 10086\n",
      "53\t148364\t0.3\t1\t138822 9542\n",
      "54\t114737\t0.3\t1\t107428 7309\n",
      "55\t140460\t0.3\t1\t130746 9714\n",
      "56\t175570\t0.3\t1\t164830 10740\n",
      "57\t103575\t0.3\t1\t95859 7716\n",
      "58\t143113\t0.3\t1\t134869 8244\n",
      "59\t126187\t0.3\t1\t117760 8427\n",
      "60\t135064\t0.3\t1\t125919 9145\n",
      "61\t135372\t0.3\t1\t126642 8730\n",
      "62\t149463\t0.3\t1\t140362 9101\n",
      "63\t114425\t0.3\t1\t106420 8005\n",
      "64\t130898\t0.3\t1\t122796 8102\n",
      "65\t148784\t0.3\t1\t139139 9645\n",
      "66\t153297\t0.3\t1\t143445 9852\n",
      "67\t160682\t0.3\t1\t147924 12758\n",
      "68\t313433\t0.3\t1\t297800 15633\n",
      "69\t49296\t0.3\t1\t44675 4621\n",
      "70\t20166\t0.3\t1\t16553 3613\n",
      "71\t84125\t0.3\t1\t78872 5253\n",
      "72\t115954\t0.3\t1\t109117 6837\n",
      "73\t122861\t0.3\t1\t115327 7534\n",
      "74\t121230\t0.3\t1\t113900 7330\n",
      "75\t120996\t0.3\t1\t113426 7570\n",
      "76\t120220\t0.3\t1\t112755 7465\n",
      "77\t120174\t0.3\t1\t112559 7615\n",
      "78\t113112\t0.3\t1\t105946 7166\n",
      "79\t112516\t0.3\t1\t105557 6959\n",
      "80\t110113\t0.3\t1\t103237 6876\n",
      "81\t108721\t0.3\t1\t102021 6700\n",
      "82\t106548\t0.3\t1\t99970 6578\n",
      "83\t102870\t0.3\t1\t96372 6498\n",
      "84\t99814\t0.3\t1\t93650 6164\n",
      "85\t97423\t0.3\t1\t91330 6093\n",
      "86\t93928\t0.3\t1\t88109 5819\n",
      "87\t90540\t0.3\t1\t84909 5631\n",
      "88\t88338\t0.3\t1\t82703 5635\n",
      "89\t85264\t0.3\t1\t79857 5407\n",
      "90\t84889\t0.3\t1\t79581 5308\n",
      "91\t80908\t0.3\t1\t75799 5109\n",
      "92\t77367\t0.3\t1\t72491 4876\n",
      "93\t72198\t0.3\t1\t67528 4670\n",
      "94\t71195\t0.3\t1\t66820 4375\n",
      "95\t66612\t0.3\t1\t62421 4191\n",
      "96\t62397\t0.3\t1\t58348 4049\n",
      "97\t59079\t0.3\t1\t55329 3750\n",
      "98\t55292\t0.3\t1\t51774 3518\n",
      "99\t54334\t0.3\t1\t50683 3651\n",
      "100\t49120\t0.3\t1\t45957 3163\n",
      "101\t46590\t0.3\t1\t43597 2993\n",
      "102\t43817\t0.3\t1\t40930 2887\n",
      "103\t40178\t0.3\t1\t37514 2664\n",
      "104\t37716\t0.3\t1\t35273 2443\n",
      "105\t33568\t0.3\t1\t31289 2279\n",
      "106\t31261\t0.3\t1\t29161 2100\n",
      "107\t28961\t0.3\t1\t27009 1952\n",
      "108\t26580\t0.3\t1\t24746 1834\n",
      "109\t24178\t0.3\t1\t22520 1658\n",
      "110\t22331\t0.3\t1\t20687 1644\n",
      "111\t19845\t0.3\t1\t18444 1401\n",
      "112\t17480\t0.3\t1\t16231 1249\n",
      "113\t15260\t0.3\t1\t14048 1212\n",
      "114\t13572\t0.3\t1\t12544 1028\n",
      "115\t12245\t0.3\t1\t11265 980\n",
      "116\t10672\t0.3\t1\t9863 809\n",
      "117\t9108\t0.3\t1\t8423 685\n",
      "118\t7910\t0.3\t1\t7333 577\n",
      "119\t6724\t0.3\t1\t6143 581\n",
      "120\t5760\t0.3\t1\t5265 495\n",
      "121\t5240\t0.3\t1\t4780 460\n",
      "122\t4324\t0.3\t1\t3902 422\n",
      "123\t3607\t0.3\t1\t3320 287\n",
      "124\t3039\t0.3\t1\t2741 298\n",
      "125\t2579\t0.3\t1\t2329 250\n",
      "126\t2117\t0.3\t1\t1915 202\n",
      "127\t1827\t0.3\t1\t1662 165\n",
      "128\t1426\t0.3\t1\t1277 149\n",
      "129\t1179\t0.3\t1\t1062 117\n",
      "130\t902\t0.3\t1\t808 94\n",
      "131\t765\t0.3\t1\t671 94\n",
      "132\t696\t0.3\t1\t619 77\n",
      "133\t768\t0.3\t1\t705 63\n",
      "134\t1110\t0.3\t1\t1039 71\n",
      "135\t519\t0.3\t1\t468 51\n",
      "136\t251\t0.3\t1\t213 38\n",
      "137\t140\t0.3\t1\t126 14\n",
      "138\t122\t0.3\t1\t106 16\n",
      "139\t85\t0.3\t1\t72 13\n",
      "140\t62\t0.3\t1\t56 6\n",
      "141\t41\t0.3\t1\t39 2\n",
      "142\t38\t0.3\t1\t34 4\n",
      "143\t29\t0.3\t1\t23 6\n",
      "144\t18\t0.3\t1\t9 9\n",
      "145\t46\t0.3\t1\t40 6\n",
      "146\t43\t0.3\t1\t40 3\n",
      "147\t83\t0.3\t1\t73 10\n",
      "148\t54\t0.3\t1\t46 8\n",
      "149\t57\t0.3\t1\t48 9\n",
      "150\t531\t0.3\t1\t439 92\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17274454 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_4_S16_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 667.08 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,274,454\n",
      "Reads with adapters:                13,645,900 (79.0%)\n",
      "Reads written (passing filters):    17,274,454 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,591,168,100 bp\n",
      "Quality-trimmed:              13,876,099 bp (0.5%)\n",
      "Total written (filtered):  1,990,000,241 bp (76.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13645900 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 13.6%\n",
      "  G: 24.3%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2190701\t4318613.5\t0\t2190701\n",
      "2\t333937\t1079653.4\t0\t333937\n",
      "3\t193297\t269913.3\t0\t193297\n",
      "4\t108228\t67478.3\t0\t108228\n",
      "5\t96340\t16869.6\t0\t96340\n",
      "6\t82360\t4217.4\t0\t82360\n",
      "7\t66626\t1054.3\t0\t66626\n",
      "8\t94895\t263.6\t0\t94895\n",
      "9\t77384\t65.9\t0\t77094 290\n",
      "10\t82708\t16.5\t1\t78572 4136\n",
      "11\t89357\t4.1\t1\t83568 5789\n",
      "12\t89625\t1.0\t1\t83594 6031\n",
      "13\t99746\t0.3\t1\t93650 6096\n",
      "14\t92613\t0.3\t1\t86134 6479\n",
      "15\t76114\t0.3\t1\t71248 4866\n",
      "16\t111173\t0.3\t1\t104156 7017\n",
      "17\t86562\t0.3\t1\t80610 5952\n",
      "18\t85573\t0.3\t1\t80223 5350\n",
      "19\t106850\t0.3\t1\t99751 7099\n",
      "20\t100367\t0.3\t1\t94128 6239\n",
      "21\t90921\t0.3\t1\t85159 5762\n",
      "22\t101235\t0.3\t1\t95178 6057\n",
      "23\t104933\t0.3\t1\t98391 6542\n",
      "24\t104547\t0.3\t1\t97881 6666\n",
      "25\t118815\t0.3\t1\t111628 7187\n",
      "26\t97285\t0.3\t1\t91138 6147\n",
      "27\t109196\t0.3\t1\t101511 7685\n",
      "28\t101869\t0.3\t1\t96609 5260\n",
      "29\t109573\t0.3\t1\t102802 6771\n",
      "30\t108062\t0.3\t1\t102093 5969\n",
      "31\t117609\t0.3\t1\t110646 6963\n",
      "32\t106258\t0.3\t1\t100439 5819\n",
      "33\t117522\t0.3\t1\t110451 7071\n",
      "34\t150731\t0.3\t1\t141819 8912\n",
      "35\t85761\t0.3\t1\t81040 4721\n",
      "36\t120033\t0.3\t1\t112538 7495\n",
      "37\t128229\t0.3\t1\t120721 7508\n",
      "38\t120207\t0.3\t1\t114092 6115\n",
      "39\t117619\t0.3\t1\t111621 5998\n",
      "40\t131723\t0.3\t1\t123921 7802\n",
      "41\t124695\t0.3\t1\t117864 6831\n",
      "42\t129575\t0.3\t1\t122681 6894\n",
      "43\t116234\t0.3\t1\t109884 6350\n",
      "44\t134420\t0.3\t1\t127012 7408\n",
      "45\t140609\t0.3\t1\t133114 7495\n",
      "46\t107286\t0.3\t1\t101613 5673\n",
      "47\t132633\t0.3\t1\t125745 6888\n",
      "48\t129386\t0.3\t1\t122927 6459\n",
      "49\t135218\t0.3\t1\t128159 7059\n",
      "50\t128960\t0.3\t1\t121933 7027\n",
      "51\t158199\t0.3\t1\t150290 7909\n",
      "52\t115635\t0.3\t1\t110127 5508\n",
      "53\t138536\t0.3\t1\t132192 6344\n",
      "54\t107841\t0.3\t1\t102539 5302\n",
      "55\t138155\t0.3\t1\t131891 6264\n",
      "56\t143145\t0.3\t1\t136120 7025\n",
      "57\t134481\t0.3\t1\t128427 6054\n",
      "58\t129238\t0.3\t1\t123474 5764\n",
      "59\t132359\t0.3\t1\t126122 6237\n",
      "60\t136774\t0.3\t1\t130227 6547\n",
      "61\t140635\t0.3\t1\t133777 6858\n",
      "62\t151391\t0.3\t1\t144144 7247\n",
      "63\t203160\t0.3\t1\t194409 8751\n",
      "64\t167385\t0.3\t1\t160989 6396\n",
      "65\t109630\t0.3\t1\t104767 4863\n",
      "66\t118791\t0.3\t1\t113616 5175\n",
      "67\t117636\t0.3\t1\t112066 5570\n",
      "68\t118814\t0.3\t1\t113546 5268\n",
      "69\t118228\t0.3\t1\t112812 5416\n",
      "70\t124633\t0.3\t1\t118945 5688\n",
      "71\t132024\t0.3\t1\t126339 5685\n",
      "72\t125001\t0.3\t1\t119396 5605\n",
      "73\t132138\t0.3\t1\t126543 5595\n",
      "74\t121069\t0.3\t1\t115983 5086\n",
      "75\t115800\t0.3\t1\t110816 4984\n",
      "76\t113271\t0.3\t1\t108471 4800\n",
      "77\t113434\t0.3\t1\t108479 4955\n",
      "78\t108048\t0.3\t1\t103289 4759\n",
      "79\t104688\t0.3\t1\t100095 4593\n",
      "80\t104276\t0.3\t1\t99811 4465\n",
      "81\t103173\t0.3\t1\t98658 4515\n",
      "82\t98745\t0.3\t1\t94559 4186\n",
      "83\t96106\t0.3\t1\t91915 4191\n",
      "84\t93144\t0.3\t1\t88994 4150\n",
      "85\t92176\t0.3\t1\t88129 4047\n",
      "86\t91669\t0.3\t1\t87622 4047\n",
      "87\t92154\t0.3\t1\t88295 3859\n",
      "88\t89148\t0.3\t1\t85288 3860\n",
      "89\t81149\t0.3\t1\t77666 3483\n",
      "90\t78174\t0.3\t1\t74845 3329\n",
      "91\t73485\t0.3\t1\t70395 3090\n",
      "92\t69397\t0.3\t1\t66501 2896\n",
      "93\t66171\t0.3\t1\t63215 2956\n",
      "94\t65063\t0.3\t1\t62271 2792\n",
      "95\t61764\t0.3\t1\t59094 2670\n",
      "96\t58368\t0.3\t1\t55944 2424\n",
      "97\t55755\t0.3\t1\t53423 2332\n",
      "98\t52903\t0.3\t1\t50745 2158\n",
      "99\t53244\t0.3\t1\t51088 2156\n",
      "100\t47728\t0.3\t1\t45793 1935\n",
      "101\t43677\t0.3\t1\t41891 1786\n",
      "102\t42206\t0.3\t1\t40454 1752\n",
      "103\t39193\t0.3\t1\t37641 1552\n",
      "104\t38151\t0.3\t1\t36638 1513\n",
      "105\t33427\t0.3\t1\t32096 1331\n",
      "106\t30687\t0.3\t1\t29498 1189\n",
      "107\t27695\t0.3\t1\t26647 1048\n",
      "108\t24686\t0.3\t1\t23641 1045\n",
      "109\t22730\t0.3\t1\t21817 913\n",
      "110\t20941\t0.3\t1\t20113 828\n",
      "111\t19623\t0.3\t1\t18797 826\n",
      "112\t17030\t0.3\t1\t16327 703\n",
      "113\t14883\t0.3\t1\t14295 588\n",
      "114\t13223\t0.3\t1\t12713 510\n",
      "115\t11548\t0.3\t1\t11042 506\n",
      "116\t9826\t0.3\t1\t9358 468\n",
      "117\t8325\t0.3\t1\t7930 395\n",
      "118\t7266\t0.3\t1\t6933 333\n",
      "119\t6187\t0.3\t1\t5890 297\n",
      "120\t5282\t0.3\t1\t5029 253\n",
      "121\t4888\t0.3\t1\t4641 247\n",
      "122\t4001\t0.3\t1\t3828 173\n",
      "123\t3492\t0.3\t1\t3320 172\n",
      "124\t2873\t0.3\t1\t2729 144\n",
      "125\t2466\t0.3\t1\t2331 135\n",
      "126\t2028\t0.3\t1\t1925 103\n",
      "127\t1732\t0.3\t1\t1662 70\n",
      "128\t1274\t0.3\t1\t1214 60\n",
      "129\t1089\t0.3\t1\t1028 61\n",
      "130\t827\t0.3\t1\t782 45\n",
      "131\t699\t0.3\t1\t652 47\n",
      "132\t657\t0.3\t1\t618 39\n",
      "133\t790\t0.3\t1\t750 40\n",
      "134\t1089\t0.3\t1\t1048 41\n",
      "135\t486\t0.3\t1\t468 18\n",
      "136\t232\t0.3\t1\t220 12\n",
      "137\t137\t0.3\t1\t130 7\n",
      "138\t117\t0.3\t1\t101 16\n",
      "139\t79\t0.3\t1\t72 7\n",
      "140\t58\t0.3\t1\t53 5\n",
      "141\t42\t0.3\t1\t40 2\n",
      "142\t42\t0.3\t1\t37 5\n",
      "143\t28\t0.3\t1\t25 3\n",
      "144\t19\t0.3\t1\t18 1\n",
      "145\t43\t0.3\t1\t41 2\n",
      "146\t41\t0.3\t1\t36 5\n",
      "147\t80\t0.3\t1\t71 9\n",
      "148\t57\t0.3\t1\t57\n",
      "149\t45\t0.3\t1\t41 4\n",
      "150\t372\t0.3\t1\t355 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17274454 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz, file_2: 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17274454\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 42048 (0.24%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_4_S16_L001_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_4_S16_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 646.86 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,208,353\n",
      "Reads with adapters:                13,674,200 (79.5%)\n",
      "Reads written (passing filters):    17,208,353 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,581,252,950 bp\n",
      "Quality-trimmed:               5,331,759 bp (0.2%)\n",
      "Total written (filtered):  1,977,555,409 bp (76.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13674200 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 17.1%\n",
      "  G: 19.6%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2124508\t4302088.2\t0\t2124508\n",
      "2\t313423\t1075522.1\t0\t313423\n",
      "3\t180126\t268880.5\t0\t180126\n",
      "4\t103470\t67220.1\t0\t103470\n",
      "5\t86090\t16805.0\t0\t86090\n",
      "6\t84599\t4201.3\t0\t84599\n",
      "7\t66757\t1050.3\t0\t66757\n",
      "8\t88962\t262.6\t0\t88962\n",
      "9\t83630\t65.6\t0\t82470 1160\n",
      "10\t80181\t16.4\t1\t74703 5478\n",
      "11\t92233\t4.1\t1\t83672 8561\n",
      "12\t86362\t1.0\t1\t79524 6838\n",
      "13\t87841\t0.3\t1\t80589 7252\n",
      "14\t90274\t0.3\t1\t82360 7914\n",
      "15\t93818\t0.3\t1\t85881 7937\n",
      "16\t92355\t0.3\t1\t83764 8591\n",
      "17\t113048\t0.3\t1\t101963 11085\n",
      "18\t96642\t0.3\t1\t89228 7414\n",
      "19\t80013\t0.3\t1\t73649 6364\n",
      "20\t99407\t0.3\t1\t91339 8068\n",
      "21\t111176\t0.3\t1\t101662 9514\n",
      "22\t92147\t0.3\t1\t85457 6690\n",
      "23\t107114\t0.3\t1\t98360 8754\n",
      "24\t102847\t0.3\t1\t93006 9841\n",
      "25\t109576\t0.3\t1\t100448 9128\n",
      "26\t99232\t0.3\t1\t91712 7520\n",
      "27\t112349\t0.3\t1\t102540 9809\n",
      "28\t109676\t0.3\t1\t100710 8966\n",
      "29\t116787\t0.3\t1\t105939 10848\n",
      "30\t125526\t0.3\t1\t116820 8706\n",
      "31\t91212\t0.3\t1\t83997 7215\n",
      "32\t117558\t0.3\t1\t108076 9482\n",
      "33\t110064\t0.3\t1\t101072 8992\n",
      "34\t118132\t0.3\t1\t107623 10509\n",
      "35\t118711\t0.3\t1\t108495 10216\n",
      "36\t135621\t0.3\t1\t122255 13366\n",
      "37\t140972\t0.3\t1\t130809 10163\n",
      "38\t95392\t0.3\t1\t87779 7613\n",
      "39\t130577\t0.3\t1\t118922 11655\n",
      "40\t131214\t0.3\t1\t119601 11613\n",
      "41\t152494\t0.3\t1\t139337 13157\n",
      "42\t102460\t0.3\t1\t93074 9386\n",
      "43\t197072\t0.3\t1\t184157 12915\n",
      "44\t54139\t0.3\t1\t48245 5894\n",
      "45\t131336\t0.3\t1\t121138 10198\n",
      "46\t127690\t0.3\t1\t117856 9834\n",
      "47\t129086\t0.3\t1\t119565 9521\n",
      "48\t131921\t0.3\t1\t121440 10481\n",
      "49\t145001\t0.3\t1\t133240 11761\n",
      "50\t116964\t0.3\t1\t108432 8532\n",
      "51\t131690\t0.3\t1\t121283 10407\n",
      "52\t137379\t0.3\t1\t125946 11433\n",
      "53\t148180\t0.3\t1\t137434 10746\n",
      "54\t113408\t0.3\t1\t104863 8545\n",
      "55\t140308\t0.3\t1\t129251 11057\n",
      "56\t175262\t0.3\t1\t162851 12411\n",
      "57\t103098\t0.3\t1\t94387 8711\n",
      "58\t143210\t0.3\t1\t133468 9742\n",
      "59\t125785\t0.3\t1\t116067 9718\n",
      "60\t133778\t0.3\t1\t123739 10039\n",
      "61\t134602\t0.3\t1\t124844 9758\n",
      "62\t150181\t0.3\t1\t139349 10832\n",
      "63\t115498\t0.3\t1\t106236 9262\n",
      "64\t129364\t0.3\t1\t120287 9077\n",
      "65\t148548\t0.3\t1\t137651 10897\n",
      "66\t150760\t0.3\t1\t139512 11248\n",
      "67\t160846\t0.3\t1\t146823 14023\n",
      "68\t302542\t0.3\t1\t285033 17509\n",
      "69\t61997\t0.3\t1\t56378 5619\n",
      "70\t24824\t0.3\t1\t20506 4318\n",
      "71\t82321\t0.3\t1\t76477 5844\n",
      "72\t114537\t0.3\t1\t107064 7473\n",
      "73\t122133\t0.3\t1\t114073 8060\n",
      "74\t121733\t0.3\t1\t113185 8548\n",
      "75\t121574\t0.3\t1\t112683 8891\n",
      "76\t119912\t0.3\t1\t110967 8945\n",
      "77\t120383\t0.3\t1\t111732 8651\n",
      "78\t113881\t0.3\t1\t105452 8429\n",
      "79\t111936\t0.3\t1\t103610 8326\n",
      "80\t109761\t0.3\t1\t101635 8126\n",
      "81\t108328\t0.3\t1\t100397 7931\n",
      "82\t105284\t0.3\t1\t97525 7759\n",
      "83\t103557\t0.3\t1\t95997 7560\n",
      "84\t99896\t0.3\t1\t92540 7356\n",
      "85\t96863\t0.3\t1\t89671 7192\n",
      "86\t94827\t0.3\t1\t87870 6957\n",
      "87\t90622\t0.3\t1\t84095 6527\n",
      "88\t89624\t0.3\t1\t82896 6728\n",
      "89\t85693\t0.3\t1\t79392 6301\n",
      "90\t85156\t0.3\t1\t79011 6145\n",
      "91\t80919\t0.3\t1\t74951 5968\n",
      "92\t77468\t0.3\t1\t71924 5544\n",
      "93\t72342\t0.3\t1\t67111 5231\n",
      "94\t71272\t0.3\t1\t66088 5184\n",
      "95\t65969\t0.3\t1\t61048 4921\n",
      "96\t63375\t0.3\t1\t58785 4590\n",
      "97\t59353\t0.3\t1\t55151 4202\n",
      "98\t55614\t0.3\t1\t51400 4214\n",
      "99\t54309\t0.3\t1\t50115 4194\n",
      "100\t49832\t0.3\t1\t46148 3684\n",
      "101\t46196\t0.3\t1\t42723 3473\n",
      "102\t43375\t0.3\t1\t40084 3291\n",
      "103\t40365\t0.3\t1\t37360 3005\n",
      "104\t37934\t0.3\t1\t35154 2780\n",
      "105\t33958\t0.3\t1\t31403 2555\n",
      "106\t31440\t0.3\t1\t29092 2348\n",
      "107\t28594\t0.3\t1\t26406 2188\n",
      "108\t27035\t0.3\t1\t24984 2051\n",
      "109\t24211\t0.3\t1\t22261 1950\n",
      "110\t22056\t0.3\t1\t20350 1706\n",
      "111\t20216\t0.3\t1\t18643 1573\n",
      "112\t17663\t0.3\t1\t16163 1500\n",
      "113\t15323\t0.3\t1\t14109 1214\n",
      "114\t13594\t0.3\t1\t12500 1094\n",
      "115\t12178\t0.3\t1\t11179 999\n",
      "116\t10756\t0.3\t1\t9835 921\n",
      "117\t9186\t0.3\t1\t8396 790\n",
      "118\t8066\t0.3\t1\t7378 688\n",
      "119\t6801\t0.3\t1\t6177 624\n",
      "120\t5605\t0.3\t1\t5122 483\n",
      "121\t5036\t0.3\t1\t4635 401\n",
      "122\t4356\t0.3\t1\t3917 439\n",
      "123\t3703\t0.3\t1\t3365 338\n",
      "124\t3136\t0.3\t1\t2845 291\n",
      "125\t2679\t0.3\t1\t2426 253\n",
      "126\t2221\t0.3\t1\t2017 204\n",
      "127\t1848\t0.3\t1\t1642 206\n",
      "128\t1384\t0.3\t1\t1237 147\n",
      "129\t1223\t0.3\t1\t1070 153\n",
      "130\t986\t0.3\t1\t889 97\n",
      "131\t829\t0.3\t1\t741 88\n",
      "132\t691\t0.3\t1\t626 65\n",
      "133\t780\t0.3\t1\t701 79\n",
      "134\t1086\t0.3\t1\t1026 60\n",
      "135\t511\t0.3\t1\t480 31\n",
      "136\t248\t0.3\t1\t207 41\n",
      "137\t178\t0.3\t1\t152 26\n",
      "138\t125\t0.3\t1\t113 12\n",
      "139\t79\t0.3\t1\t67 12\n",
      "140\t60\t0.3\t1\t49 11\n",
      "141\t36\t0.3\t1\t30 6\n",
      "142\t37\t0.3\t1\t26 11\n",
      "143\t39\t0.3\t1\t35 4\n",
      "144\t23\t0.3\t1\t20 3\n",
      "145\t36\t0.3\t1\t29 7\n",
      "146\t37\t0.3\t1\t31 6\n",
      "147\t101\t0.3\t1\t92 9\n",
      "148\t53\t0.3\t1\t44 9\n",
      "149\t35\t0.3\t1\t29 6\n",
      "150\t573\t0.3\t1\t484 89\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17208353 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_26psu_4_S16_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 691.41 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,208,353\n",
      "Reads with adapters:                13,587,185 (79.0%)\n",
      "Reads written (passing filters):    17,208,353 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,581,252,950 bp\n",
      "Quality-trimmed:              13,418,385 bp (0.5%)\n",
      "Total written (filtered):  1,981,075,586 bp (76.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13587185 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 13.6%\n",
      "  G: 24.4%\n",
      "  T: 34.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2180320\t4302088.2\t0\t2180320\n",
      "2\t324825\t1075522.1\t0\t324825\n",
      "3\t196027\t268880.5\t0\t196027\n",
      "4\t105938\t67220.1\t0\t105938\n",
      "5\t93022\t16805.0\t0\t93022\n",
      "6\t82169\t4201.3\t0\t82169\n",
      "7\t65208\t1050.3\t0\t65208\n",
      "8\t94076\t262.6\t0\t94076\n",
      "9\t75651\t65.6\t0\t75373 278\n",
      "10\t79743\t16.4\t1\t74241 5502\n",
      "11\t88328\t4.1\t1\t81010 7318\n",
      "12\t86868\t1.0\t1\t79554 7314\n",
      "13\t99881\t0.3\t1\t91747 8134\n",
      "14\t90834\t0.3\t1\t82543 8291\n",
      "15\t75785\t0.3\t1\t69708 6077\n",
      "16\t109506\t0.3\t1\t100461 9045\n",
      "17\t89706\t0.3\t1\t82164 7542\n",
      "18\t82342\t0.3\t1\t75779 6563\n",
      "19\t105511\t0.3\t1\t97019 8492\n",
      "20\t99413\t0.3\t1\t91913 7500\n",
      "21\t89589\t0.3\t1\t82387 7202\n",
      "22\t99486\t0.3\t1\t91727 7759\n",
      "23\t104499\t0.3\t1\t96352 8147\n",
      "24\t104372\t0.3\t1\t95967 8405\n",
      "25\t118405\t0.3\t1\t110027 8378\n",
      "26\t97628\t0.3\t1\t89447 8181\n",
      "27\t110575\t0.3\t1\t101151 9424\n",
      "28\t97751\t0.3\t1\t91186 6565\n",
      "29\t111505\t0.3\t1\t102881 8624\n",
      "30\t104420\t0.3\t1\t97112 7308\n",
      "31\t118911\t0.3\t1\t110316 8595\n",
      "32\t102066\t0.3\t1\t95076 6990\n",
      "33\t118362\t0.3\t1\t109639 8723\n",
      "34\t152523\t0.3\t1\t141524 10999\n",
      "35\t81147\t0.3\t1\t75248 5899\n",
      "36\t120338\t0.3\t1\t111218 9120\n",
      "37\t127944\t0.3\t1\t118775 9169\n",
      "38\t118773\t0.3\t1\t111107 7666\n",
      "39\t114524\t0.3\t1\t107290 7234\n",
      "40\t130962\t0.3\t1\t121798 9164\n",
      "41\t127042\t0.3\t1\t118718 8324\n",
      "42\t130066\t0.3\t1\t121378 8688\n",
      "43\t114447\t0.3\t1\t107009 7438\n",
      "44\t134439\t0.3\t1\t125115 9324\n",
      "45\t139316\t0.3\t1\t130061 9255\n",
      "46\t105889\t0.3\t1\t98943 6946\n",
      "47\t133294\t0.3\t1\t125179 8115\n",
      "48\t129000\t0.3\t1\t120992 8008\n",
      "49\t134472\t0.3\t1\t125771 8701\n",
      "50\t129511\t0.3\t1\t120901 8610\n",
      "51\t156322\t0.3\t1\t146348 9974\n",
      "52\t115554\t0.3\t1\t108318 7236\n",
      "53\t141127\t0.3\t1\t132995 8132\n",
      "54\t105383\t0.3\t1\t99036 6347\n",
      "55\t137072\t0.3\t1\t129093 7979\n",
      "56\t141674\t0.3\t1\t132689 8985\n",
      "57\t134402\t0.3\t1\t126942 7460\n",
      "58\t130317\t0.3\t1\t123061 7256\n",
      "59\t129827\t0.3\t1\t122170 7657\n",
      "60\t136682\t0.3\t1\t128763 7919\n",
      "61\t138438\t0.3\t1\t130627 7811\n",
      "62\t151584\t0.3\t1\t142493 9091\n",
      "63\t198913\t0.3\t1\t188533 10380\n",
      "64\t165224\t0.3\t1\t157788 7436\n",
      "65\t109943\t0.3\t1\t103886 6057\n",
      "66\t119196\t0.3\t1\t112621 6575\n",
      "67\t118137\t0.3\t1\t111336 6801\n",
      "68\t119874\t0.3\t1\t113325 6549\n",
      "69\t119266\t0.3\t1\t112294 6972\n",
      "70\t126609\t0.3\t1\t119544 7065\n",
      "71\t132620\t0.3\t1\t125297 7323\n",
      "72\t125393\t0.3\t1\t118453 6940\n",
      "73\t132184\t0.3\t1\t124866 7318\n",
      "74\t122169\t0.3\t1\t115334 6835\n",
      "75\t116493\t0.3\t1\t110071 6422\n",
      "76\t113442\t0.3\t1\t106974 6468\n",
      "77\t113685\t0.3\t1\t107141 6544\n",
      "78\t108607\t0.3\t1\t102395 6212\n",
      "79\t104353\t0.3\t1\t98543 5810\n",
      "80\t104020\t0.3\t1\t98279 5741\n",
      "81\t102310\t0.3\t1\t96699 5611\n",
      "82\t98054\t0.3\t1\t92664 5390\n",
      "83\t96480\t0.3\t1\t91293 5187\n",
      "84\t93016\t0.3\t1\t87985 5031\n",
      "85\t91834\t0.3\t1\t86918 4916\n",
      "86\t92118\t0.3\t1\t87322 4796\n",
      "87\t91952\t0.3\t1\t87281 4671\n",
      "88\t89832\t0.3\t1\t85225 4607\n",
      "89\t81915\t0.3\t1\t77620 4295\n",
      "90\t78846\t0.3\t1\t74588 4258\n",
      "91\t74284\t0.3\t1\t70441 3843\n",
      "92\t69858\t0.3\t1\t66293 3565\n",
      "93\t66592\t0.3\t1\t63127 3465\n",
      "94\t65539\t0.3\t1\t62049 3490\n",
      "95\t61188\t0.3\t1\t57883 3305\n",
      "96\t59534\t0.3\t1\t56516 3018\n",
      "97\t56011\t0.3\t1\t53174 2837\n",
      "98\t53355\t0.3\t1\t50647 2708\n",
      "99\t53101\t0.3\t1\t50373 2728\n",
      "100\t48195\t0.3\t1\t45837 2358\n",
      "101\t43522\t0.3\t1\t41188 2334\n",
      "102\t41863\t0.3\t1\t39765 2098\n",
      "103\t39452\t0.3\t1\t37477 1975\n",
      "104\t38339\t0.3\t1\t36409 1930\n",
      "105\t33606\t0.3\t1\t31895 1711\n",
      "106\t30952\t0.3\t1\t29448 1504\n",
      "107\t27675\t0.3\t1\t26330 1345\n",
      "108\t24951\t0.3\t1\t23635 1316\n",
      "109\t22743\t0.3\t1\t21560 1183\n",
      "110\t20811\t0.3\t1\t19684 1127\n",
      "111\t19926\t0.3\t1\t18958 968\n",
      "112\t17187\t0.3\t1\t16283 904\n",
      "113\t15096\t0.3\t1\t14331 765\n",
      "114\t13260\t0.3\t1\t12650 610\n",
      "115\t11435\t0.3\t1\t10854 581\n",
      "116\t9981\t0.3\t1\t9425 556\n",
      "117\t8327\t0.3\t1\t7873 454\n",
      "118\t7364\t0.3\t1\t6994 370\n",
      "119\t6240\t0.3\t1\t5915 325\n",
      "120\t5208\t0.3\t1\t4920 288\n",
      "121\t4741\t0.3\t1\t4467 274\n",
      "122\t4055\t0.3\t1\t3851 204\n",
      "123\t3533\t0.3\t1\t3332 201\n",
      "124\t2985\t0.3\t1\t2813 172\n",
      "125\t2559\t0.3\t1\t2417 142\n",
      "126\t2133\t0.3\t1\t2012 121\n",
      "127\t1706\t0.3\t1\t1579 127\n",
      "128\t1327\t0.3\t1\t1241 86\n",
      "129\t1121\t0.3\t1\t1038 83\n",
      "130\t901\t0.3\t1\t842 59\n",
      "131\t784\t0.3\t1\t738 46\n",
      "132\t657\t0.3\t1\t620 37\n",
      "133\t809\t0.3\t1\t770 39\n",
      "134\t1081\t0.3\t1\t1031 50\n",
      "135\t490\t0.3\t1\t465 25\n",
      "136\t231\t0.3\t1\t213 18\n",
      "137\t165\t0.3\t1\t153 12\n",
      "138\t122\t0.3\t1\t117 5\n",
      "139\t68\t0.3\t1\t64 4\n",
      "140\t60\t0.3\t1\t53 7\n",
      "141\t31\t0.3\t1\t25 6\n",
      "142\t34\t0.3\t1\t30 4\n",
      "143\t37\t0.3\t1\t30 7\n",
      "144\t24\t0.3\t1\t20 4\n",
      "145\t49\t0.3\t1\t45 4\n",
      "146\t39\t0.3\t1\t34 5\n",
      "147\t87\t0.3\t1\t77 10\n",
      "148\t52\t0.3\t1\t48 4\n",
      "149\t25\t0.3\t1\t23 2\n",
      "150\t408\t0.3\t1\t378 30\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17208353 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz, file_2: 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17208353\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 40674 (0.24%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_4_S16_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_4_S16_L002_R1_001_trimmed.fq.gz and 16C_26psu_4_S16_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_1_S1_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 693.30 s (35 us/read; 1.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,624,894\n",
      "Reads with adapters:                16,461,713 (83.9%)\n",
      "Reads written (passing filters):    19,624,894 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,943,734,100 bp\n",
      "Quality-trimmed:               8,161,343 bp (0.3%)\n",
      "Total written (filtered):  2,138,119,098 bp (72.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16461713 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 14.5%\n",
      "  G: 21.1%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1876405\t4906223.5\t0\t1876405\n",
      "2\t285639\t1226555.9\t0\t285639\n",
      "3\t166471\t306639.0\t0\t166471\n",
      "4\t104694\t76659.7\t0\t104694\n",
      "5\t90631\t19164.9\t0\t90631\n",
      "6\t90221\t4791.2\t0\t90221\n",
      "7\t75553\t1197.8\t0\t75553\n",
      "8\t93443\t299.5\t0\t93443\n",
      "9\t94201\t74.9\t0\t93032 1169\n",
      "10\t87809\t18.7\t1\t83198 4611\n",
      "11\t99635\t4.7\t1\t92145 7490\n",
      "12\t94726\t1.2\t1\t88812 5914\n",
      "13\t97394\t0.3\t1\t91137 6257\n",
      "14\t100020\t0.3\t1\t93031 6989\n",
      "15\t103009\t0.3\t1\t96393 6616\n",
      "16\t102618\t0.3\t1\t94905 7713\n",
      "17\t124436\t0.3\t1\t114364 10072\n",
      "18\t107976\t0.3\t1\t101489 6487\n",
      "19\t93413\t0.3\t1\t87422 5991\n",
      "20\t112556\t0.3\t1\t105383 7173\n",
      "21\t124612\t0.3\t1\t116055 8557\n",
      "22\t104064\t0.3\t1\t98066 5998\n",
      "23\t123879\t0.3\t1\t115825 8054\n",
      "24\t114769\t0.3\t1\t105177 9592\n",
      "25\t125576\t0.3\t1\t117036 8540\n",
      "26\t114132\t0.3\t1\t107345 6787\n",
      "27\t127880\t0.3\t1\t118576 9304\n",
      "28\t126438\t0.3\t1\t117548 8890\n",
      "29\t134319\t0.3\t1\t123537 10782\n",
      "30\t144898\t0.3\t1\t136865 8033\n",
      "31\t113550\t0.3\t1\t106291 7259\n",
      "32\t136410\t0.3\t1\t127848 8562\n",
      "33\t133695\t0.3\t1\t124518 9177\n",
      "34\t143757\t0.3\t1\t134451 9306\n",
      "35\t157890\t0.3\t1\t145316 12574\n",
      "36\t132282\t0.3\t1\t123929 8353\n",
      "37\t149188\t0.3\t1\t140073 9115\n",
      "38\t146870\t0.3\t1\t136072 10798\n",
      "39\t172309\t0.3\t1\t159375 12934\n",
      "40\t136570\t0.3\t1\t128574 7996\n",
      "41\t180683\t0.3\t1\t167134 13549\n",
      "42\t139132\t0.3\t1\t127852 11280\n",
      "43\t244330\t0.3\t1\t231176 13154\n",
      "44\t73162\t0.3\t1\t66477 6685\n",
      "45\t161358\t0.3\t1\t150280 11078\n",
      "46\t191244\t0.3\t1\t177928 13316\n",
      "47\t181249\t0.3\t1\t170520 10729\n",
      "48\t141208\t0.3\t1\t129881 11327\n",
      "49\t209170\t0.3\t1\t195203 13967\n",
      "50\t130797\t0.3\t1\t122619 8178\n",
      "51\t161830\t0.3\t1\t151113 10717\n",
      "52\t228270\t0.3\t1\t212080 16190\n",
      "53\t234755\t0.3\t1\t222809 11946\n",
      "54\t127266\t0.3\t1\t118588 8678\n",
      "55\t194077\t0.3\t1\t183800 10277\n",
      "56\t176229\t0.3\t1\t165657 10572\n",
      "57\t215239\t0.3\t1\t203613 11626\n",
      "58\t135465\t0.3\t1\t128771 6694\n",
      "59\t101608\t0.3\t1\t93951 7657\n",
      "60\t209358\t0.3\t1\t197818 11540\n",
      "61\t195964\t0.3\t1\t186497 9467\n",
      "62\t138436\t0.3\t1\t130152 8284\n",
      "63\t180787\t0.3\t1\t170284 10503\n",
      "64\t159166\t0.3\t1\t150739 8427\n",
      "65\t158205\t0.3\t1\t148817 9388\n",
      "66\t190752\t0.3\t1\t179381 11371\n",
      "67\t212991\t0.3\t1\t197325 15666\n",
      "68\t398495\t0.3\t1\t379493 19002\n",
      "69\t144273\t0.3\t1\t135320 8953\n",
      "70\t44631\t0.3\t1\t39323 5308\n",
      "71\t66845\t0.3\t1\t61618 5227\n",
      "72\t147002\t0.3\t1\t138540 8462\n",
      "73\t167373\t0.3\t1\t157828 9545\n",
      "74\t168016\t0.3\t1\t158069 9947\n",
      "75\t167181\t0.3\t1\t157169 10012\n",
      "76\t166317\t0.3\t1\t156562 9755\n",
      "77\t166255\t0.3\t1\t156536 9719\n",
      "78\t159048\t0.3\t1\t149642 9406\n",
      "79\t157934\t0.3\t1\t148379 9555\n",
      "80\t154378\t0.3\t1\t145219 9159\n",
      "81\t154058\t0.3\t1\t144915 9143\n",
      "82\t150711\t0.3\t1\t141779 8932\n",
      "83\t145124\t0.3\t1\t136331 8793\n",
      "84\t142844\t0.3\t1\t134506 8338\n",
      "85\t138019\t0.3\t1\t129804 8215\n",
      "86\t134359\t0.3\t1\t126493 7866\n",
      "87\t130206\t0.3\t1\t122445 7761\n",
      "88\t128131\t0.3\t1\t120526 7605\n",
      "89\t122017\t0.3\t1\t114723 7294\n",
      "90\t121870\t0.3\t1\t114562 7308\n",
      "91\t117180\t0.3\t1\t110087 7093\n",
      "92\t111859\t0.3\t1\t105091 6768\n",
      "93\t105927\t0.3\t1\t99576 6351\n",
      "94\t102547\t0.3\t1\t96466 6081\n",
      "95\t95663\t0.3\t1\t89977 5686\n",
      "96\t91505\t0.3\t1\t85936 5569\n",
      "97\t86471\t0.3\t1\t81209 5262\n",
      "98\t81070\t0.3\t1\t76155 4915\n",
      "99\t79954\t0.3\t1\t74613 5341\n",
      "100\t72882\t0.3\t1\t68442 4440\n",
      "101\t68704\t0.3\t1\t64356 4348\n",
      "102\t63576\t0.3\t1\t59490 4086\n",
      "103\t59368\t0.3\t1\t55615 3753\n",
      "104\t55418\t0.3\t1\t51914 3504\n",
      "105\t49653\t0.3\t1\t46415 3238\n",
      "106\t45561\t0.3\t1\t42651 2910\n",
      "107\t41569\t0.3\t1\t38864 2705\n",
      "108\t39242\t0.3\t1\t36735 2507\n",
      "109\t35181\t0.3\t1\t32884 2297\n",
      "110\t32351\t0.3\t1\t30217 2134\n",
      "111\t29181\t0.3\t1\t27222 1959\n",
      "112\t25447\t0.3\t1\t23718 1729\n",
      "113\t22680\t0.3\t1\t21019 1661\n",
      "114\t19902\t0.3\t1\t18472 1430\n",
      "115\t17153\t0.3\t1\t15867 1286\n",
      "116\t15113\t0.3\t1\t14061 1052\n",
      "117\t13006\t0.3\t1\t12016 990\n",
      "118\t11539\t0.3\t1\t10694 845\n",
      "119\t9431\t0.3\t1\t8642 789\n",
      "120\t7730\t0.3\t1\t7108 622\n",
      "121\t6854\t0.3\t1\t6273 581\n",
      "122\t5610\t0.3\t1\t5167 443\n",
      "123\t4861\t0.3\t1\t4443 418\n",
      "124\t4094\t0.3\t1\t3727 367\n",
      "125\t3465\t0.3\t1\t3160 305\n",
      "126\t2986\t0.3\t1\t2695 291\n",
      "127\t2510\t0.3\t1\t2289 221\n",
      "128\t1832\t0.3\t1\t1674 158\n",
      "129\t1619\t0.3\t1\t1450 169\n",
      "130\t1319\t0.3\t1\t1201 118\n",
      "131\t1116\t0.3\t1\t999 117\n",
      "132\t1063\t0.3\t1\t961 102\n",
      "133\t1337\t0.3\t1\t1208 129\n",
      "134\t1485\t0.3\t1\t1390 95\n",
      "135\t668\t0.3\t1\t615 53\n",
      "136\t381\t0.3\t1\t342 39\n",
      "137\t211\t0.3\t1\t197 14\n",
      "138\t160\t0.3\t1\t144 16\n",
      "139\t121\t0.3\t1\t106 15\n",
      "140\t71\t0.3\t1\t59 12\n",
      "141\t68\t0.3\t1\t57 11\n",
      "142\t52\t0.3\t1\t47 5\n",
      "143\t43\t0.3\t1\t38 5\n",
      "144\t24\t0.3\t1\t22 2\n",
      "145\t53\t0.3\t1\t40 13\n",
      "146\t42\t0.3\t1\t34 8\n",
      "147\t125\t0.3\t1\t106 19\n",
      "148\t83\t0.3\t1\t69 14\n",
      "149\t78\t0.3\t1\t73 5\n",
      "150\t763\t0.3\t1\t673 90\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "19624894 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_1_S1_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 725.83 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,624,894\n",
      "Reads with adapters:                16,339,161 (83.3%)\n",
      "Reads written (passing filters):    19,624,894 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,943,734,100 bp\n",
      "Quality-trimmed:              15,599,605 bp (0.5%)\n",
      "Total written (filtered):  2,144,342,165 bp (72.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16339161 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 12.6%\n",
      "  G: 25.0%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1942196\t4906223.5\t0\t1942196\n",
      "2\t299564\t1226555.9\t0\t299564\n",
      "3\t177742\t306639.0\t0\t177742\n",
      "4\t105333\t76659.7\t0\t105333\n",
      "5\t100538\t19164.9\t0\t100538\n",
      "6\t87955\t4791.2\t0\t87955\n",
      "7\t71440\t1197.8\t0\t71440\n",
      "8\t102524\t299.5\t0\t102524\n",
      "9\t81367\t74.9\t0\t81047 320\n",
      "10\t89669\t18.7\t1\t85117 4552\n",
      "11\t95729\t4.7\t1\t89915 5814\n",
      "12\t95768\t1.2\t1\t89687 6081\n",
      "13\t108234\t0.3\t1\t101948 6286\n",
      "14\t103335\t0.3\t1\t96341 6994\n",
      "15\t82616\t0.3\t1\t77316 5300\n",
      "16\t121595\t0.3\t1\t114016 7579\n",
      "17\t98896\t0.3\t1\t92218 6678\n",
      "18\t93415\t0.3\t1\t87589 5826\n",
      "19\t120547\t0.3\t1\t112557 7990\n",
      "20\t113077\t0.3\t1\t106404 6673\n",
      "21\t99743\t0.3\t1\t93549 6194\n",
      "22\t113380\t0.3\t1\t106586 6794\n",
      "23\t116836\t0.3\t1\t109692 7144\n",
      "24\t119139\t0.3\t1\t111646 7493\n",
      "25\t135500\t0.3\t1\t127577 7923\n",
      "26\t112154\t0.3\t1\t104938 7216\n",
      "27\t122636\t0.3\t1\t114199 8437\n",
      "28\t117975\t0.3\t1\t111836 6139\n",
      "29\t125073\t0.3\t1\t117359 7714\n",
      "30\t127096\t0.3\t1\t120402 6694\n",
      "31\t137659\t0.3\t1\t129729 7930\n",
      "32\t123762\t0.3\t1\t117342 6420\n",
      "33\t138965\t0.3\t1\t130590 8375\n",
      "34\t180950\t0.3\t1\t170447 10503\n",
      "35\t102752\t0.3\t1\t97235 5517\n",
      "36\t141525\t0.3\t1\t133105 8420\n",
      "37\t151296\t0.3\t1\t142475 8821\n",
      "38\t141804\t0.3\t1\t134802 7002\n",
      "39\t139699\t0.3\t1\t132682 7017\n",
      "40\t156245\t0.3\t1\t147123 9122\n",
      "41\t150848\t0.3\t1\t143057 7791\n",
      "42\t156523\t0.3\t1\t148491 8032\n",
      "43\t142855\t0.3\t1\t135344 7511\n",
      "44\t166066\t0.3\t1\t157102 8964\n",
      "45\t178466\t0.3\t1\t169227 9239\n",
      "46\t128203\t0.3\t1\t121604 6599\n",
      "47\t163666\t0.3\t1\t155525 8141\n",
      "48\t161229\t0.3\t1\t153502 7727\n",
      "49\t169595\t0.3\t1\t161041 8554\n",
      "50\t162191\t0.3\t1\t153908 8283\n",
      "51\t204248\t0.3\t1\t193780 10468\n",
      "52\t147164\t0.3\t1\t140471 6693\n",
      "53\t177383\t0.3\t1\t169465 7918\n",
      "54\t135474\t0.3\t1\t129206 6268\n",
      "55\t177540\t0.3\t1\t169636 7904\n",
      "56\t183270\t0.3\t1\t174617 8653\n",
      "57\t175273\t0.3\t1\t167802 7471\n",
      "58\t168130\t0.3\t1\t160997 7133\n",
      "59\t172563\t0.3\t1\t164868 7695\n",
      "60\t179506\t0.3\t1\t171470 8036\n",
      "61\t186851\t0.3\t1\t178181 8670\n",
      "62\t203675\t0.3\t1\t194173 9502\n",
      "63\t273998\t0.3\t1\t262527 11471\n",
      "64\t226808\t0.3\t1\t218491 8317\n",
      "65\t147108\t0.3\t1\t140746 6362\n",
      "66\t160644\t0.3\t1\t153693 6951\n",
      "67\t159286\t0.3\t1\t152193 7093\n",
      "68\t161933\t0.3\t1\t155050 6883\n",
      "69\t160655\t0.3\t1\t153681 6974\n",
      "70\t170416\t0.3\t1\t162996 7420\n",
      "71\t179225\t0.3\t1\t171910 7315\n",
      "72\t171231\t0.3\t1\t163925 7306\n",
      "73\t181424\t0.3\t1\t173901 7523\n",
      "74\t167534\t0.3\t1\t160668 6866\n",
      "75\t160295\t0.3\t1\t153643 6652\n",
      "76\t157715\t0.3\t1\t151296 6419\n",
      "77\t157679\t0.3\t1\t150903 6776\n",
      "78\t152483\t0.3\t1\t146130 6353\n",
      "79\t147737\t0.3\t1\t141492 6245\n",
      "80\t146667\t0.3\t1\t140666 6001\n",
      "81\t145920\t0.3\t1\t140003 5917\n",
      "82\t140689\t0.3\t1\t134853 5836\n",
      "83\t136034\t0.3\t1\t130439 5595\n",
      "84\t133908\t0.3\t1\t128250 5658\n",
      "85\t131675\t0.3\t1\t126012 5663\n",
      "86\t131672\t0.3\t1\t126197 5475\n",
      "87\t133848\t0.3\t1\t128384 5464\n",
      "88\t129850\t0.3\t1\t124580 5270\n",
      "89\t116878\t0.3\t1\t112187 4691\n",
      "90\t113165\t0.3\t1\t108451 4714\n",
      "91\t107851\t0.3\t1\t103636 4215\n",
      "92\t101212\t0.3\t1\t97187 4025\n",
      "93\t98995\t0.3\t1\t94995 4000\n",
      "94\t95394\t0.3\t1\t91542 3852\n",
      "95\t90020\t0.3\t1\t86380 3640\n",
      "96\t85913\t0.3\t1\t82550 3363\n",
      "97\t80493\t0.3\t1\t77312 3181\n",
      "98\t75879\t0.3\t1\t72925 2954\n",
      "99\t77605\t0.3\t1\t74598 3007\n",
      "100\t70637\t0.3\t1\t67912 2725\n",
      "101\t67043\t0.3\t1\t64475 2568\n",
      "102\t62076\t0.3\t1\t59759 2317\n",
      "103\t56002\t0.3\t1\t53869 2133\n",
      "104\t53213\t0.3\t1\t51172 2041\n",
      "105\t47322\t0.3\t1\t45493 1829\n",
      "106\t43803\t0.3\t1\t42161 1642\n",
      "107\t39428\t0.3\t1\t37938 1490\n",
      "108\t36024\t0.3\t1\t34621 1403\n",
      "109\t33005\t0.3\t1\t31801 1204\n",
      "110\t30413\t0.3\t1\t29223 1190\n",
      "111\t28718\t0.3\t1\t27704 1014\n",
      "112\t24826\t0.3\t1\t23909 917\n",
      "113\t22246\t0.3\t1\t21395 851\n",
      "114\t19255\t0.3\t1\t18504 751\n",
      "115\t16125\t0.3\t1\t15460 665\n",
      "116\t14013\t0.3\t1\t13452 561\n",
      "117\t11755\t0.3\t1\t11314 441\n",
      "118\t10504\t0.3\t1\t10075 429\n",
      "119\t8740\t0.3\t1\t8381 359\n",
      "120\t7099\t0.3\t1\t6789 310\n",
      "121\t6395\t0.3\t1\t6127 268\n",
      "122\t5283\t0.3\t1\t5086 197\n",
      "123\t4656\t0.3\t1\t4456 200\n",
      "124\t3824\t0.3\t1\t3672 152\n",
      "125\t3323\t0.3\t1\t3181 142\n",
      "126\t2821\t0.3\t1\t2694 127\n",
      "127\t2334\t0.3\t1\t2219 115\n",
      "128\t1727\t0.3\t1\t1654 73\n",
      "129\t1502\t0.3\t1\t1423 79\n",
      "130\t1215\t0.3\t1\t1157 58\n",
      "131\t1055\t0.3\t1\t1002 53\n",
      "132\t1009\t0.3\t1\t967 42\n",
      "133\t1312\t0.3\t1\t1260 52\n",
      "134\t1503\t0.3\t1\t1462 41\n",
      "135\t647\t0.3\t1\t618 29\n",
      "136\t354\t0.3\t1\t338 16\n",
      "137\t216\t0.3\t1\t209 7\n",
      "138\t150\t0.3\t1\t140 10\n",
      "139\t114\t0.3\t1\t110 4\n",
      "140\t64\t0.3\t1\t60 4\n",
      "141\t66\t0.3\t1\t61 5\n",
      "142\t51\t0.3\t1\t49 2\n",
      "143\t40\t0.3\t1\t37 3\n",
      "144\t27\t0.3\t1\t25 2\n",
      "145\t50\t0.3\t1\t45 5\n",
      "146\t38\t0.3\t1\t37 1\n",
      "147\t120\t0.3\t1\t112 8\n",
      "148\t74\t0.3\t1\t68 6\n",
      "149\t76\t0.3\t1\t73 3\n",
      "150\t581\t0.3\t1\t559 22\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "19624894 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz, file_2: 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 19624894\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 47973 (0.24%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_1_S1_L001_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_1_S1_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 703.51 s (36 us/read; 1.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,571,235\n",
      "Reads with adapters:                16,407,295 (83.8%)\n",
      "Reads written (passing filters):    19,571,235 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,935,685,250 bp\n",
      "Quality-trimmed:               8,266,740 bp (0.3%)\n",
      "Total written (filtered):  2,130,672,037 bp (72.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16407295 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 14.6%\n",
      "  G: 21.1%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1871205\t4892808.8\t0\t1871205\n",
      "2\t281413\t1223202.2\t0\t281413\n",
      "3\t165620\t305800.5\t0\t165620\n",
      "4\t102750\t76450.1\t0\t102750\n",
      "5\t91621\t19112.5\t0\t91621\n",
      "6\t89878\t4778.1\t0\t89878\n",
      "7\t72653\t1194.5\t0\t72653\n",
      "8\t92540\t298.6\t0\t92540\n",
      "9\t93344\t74.7\t0\t92169 1175\n",
      "10\t86372\t18.7\t1\t80800 5572\n",
      "11\t99125\t4.7\t1\t90406 8719\n",
      "12\t93690\t1.2\t1\t86859 6831\n",
      "13\t96288\t0.3\t1\t88938 7350\n",
      "14\t98016\t0.3\t1\t89939 8077\n",
      "15\t102285\t0.3\t1\t94257 8028\n",
      "16\t101242\t0.3\t1\t92425 8817\n",
      "17\t123641\t0.3\t1\t112025 11616\n",
      "18\t107224\t0.3\t1\t99543 7681\n",
      "19\t90677\t0.3\t1\t83921 6756\n",
      "20\t111453\t0.3\t1\t102938 8515\n",
      "21\t124864\t0.3\t1\t115203 9661\n",
      "22\t102217\t0.3\t1\t95257 6960\n",
      "23\t124937\t0.3\t1\t115301 9636\n",
      "24\t113499\t0.3\t1\t102977 10522\n",
      "25\t124401\t0.3\t1\t114862 9539\n",
      "26\t112120\t0.3\t1\t104445 7675\n",
      "27\t128185\t0.3\t1\t117570 10615\n",
      "28\t125849\t0.3\t1\t115930 9919\n",
      "29\t134332\t0.3\t1\t122337 11995\n",
      "30\t144690\t0.3\t1\t135178 9512\n",
      "31\t111404\t0.3\t1\t103047 8357\n",
      "32\t135894\t0.3\t1\t125650 10244\n",
      "33\t131757\t0.3\t1\t121043 10714\n",
      "34\t143190\t0.3\t1\t131548 11642\n",
      "35\t157418\t0.3\t1\t142743 14675\n",
      "36\t129247\t0.3\t1\t119787 9460\n",
      "37\t149221\t0.3\t1\t138285 10936\n",
      "38\t145468\t0.3\t1\t133187 12281\n",
      "39\t172504\t0.3\t1\t157173 15331\n",
      "40\t134071\t0.3\t1\t124322 9749\n",
      "41\t180864\t0.3\t1\t164818 16046\n",
      "42\t137219\t0.3\t1\t124408 12811\n",
      "43\t239349\t0.3\t1\t223461 15888\n",
      "44\t74486\t0.3\t1\t66856 7630\n",
      "45\t161688\t0.3\t1\t148747 12941\n",
      "46\t191505\t0.3\t1\t176273 15232\n",
      "47\t179273\t0.3\t1\t166898 12375\n",
      "48\t140851\t0.3\t1\t128229 12622\n",
      "49\t208539\t0.3\t1\t192459 16080\n",
      "50\t127958\t0.3\t1\t118792 9166\n",
      "51\t162484\t0.3\t1\t150649 11835\n",
      "52\t227313\t0.3\t1\t208878 18435\n",
      "53\t230954\t0.3\t1\t217665 13289\n",
      "54\t127985\t0.3\t1\t117878 10107\n",
      "55\t190088\t0.3\t1\t177776 12312\n",
      "56\t183666\t0.3\t1\t171198 12468\n",
      "57\t212537\t0.3\t1\t198874 13663\n",
      "58\t133097\t0.3\t1\t125108 7989\n",
      "59\t104505\t0.3\t1\t95591 8914\n",
      "60\t206694\t0.3\t1\t193657 13037\n",
      "61\t190421\t0.3\t1\t179352 11069\n",
      "62\t144000\t0.3\t1\t134103 9897\n",
      "63\t180563\t0.3\t1\t168547 12016\n",
      "64\t155217\t0.3\t1\t145289 9928\n",
      "65\t158098\t0.3\t1\t147007 11091\n",
      "66\t188945\t0.3\t1\t175941 13004\n",
      "67\t211986\t0.3\t1\t194786 17200\n",
      "68\t378408\t0.3\t1\t357096 21312\n",
      "69\t162579\t0.3\t1\t151778 10801\n",
      "70\t55965\t0.3\t1\t49649 6316\n",
      "71\t63814\t0.3\t1\t58189 5625\n",
      "72\t145461\t0.3\t1\t136394 9067\n",
      "73\t166164\t0.3\t1\t155484 10680\n",
      "74\t168411\t0.3\t1\t157197 11214\n",
      "75\t167596\t0.3\t1\t156098 11498\n",
      "76\t166388\t0.3\t1\t154457 11931\n",
      "77\t167040\t0.3\t1\t155459 11581\n",
      "78\t159704\t0.3\t1\t148462 11242\n",
      "79\t158706\t0.3\t1\t147370 11336\n",
      "80\t154542\t0.3\t1\t143508 11034\n",
      "81\t154753\t0.3\t1\t143966 10787\n",
      "82\t150886\t0.3\t1\t140115 10771\n",
      "83\t145780\t0.3\t1\t135596 10184\n",
      "84\t142086\t0.3\t1\t132138 9948\n",
      "85\t137811\t0.3\t1\t128196 9615\n",
      "86\t134898\t0.3\t1\t125576 9322\n",
      "87\t129676\t0.3\t1\t120849 8827\n",
      "88\t128629\t0.3\t1\t119738 8891\n",
      "89\t123313\t0.3\t1\t114615 8698\n",
      "90\t123132\t0.3\t1\t114553 8579\n",
      "91\t117637\t0.3\t1\t109407 8230\n",
      "92\t112493\t0.3\t1\t104754 7739\n",
      "93\t106111\t0.3\t1\t98757 7354\n",
      "94\t103337\t0.3\t1\t96126 7211\n",
      "95\t95958\t0.3\t1\t89095 6863\n",
      "96\t91266\t0.3\t1\t84746 6520\n",
      "97\t86736\t0.3\t1\t80632 6104\n",
      "98\t81864\t0.3\t1\t75993 5871\n",
      "99\t80342\t0.3\t1\t74325 6017\n",
      "100\t73408\t0.3\t1\t68128 5280\n",
      "101\t68599\t0.3\t1\t63783 4816\n",
      "102\t64339\t0.3\t1\t59627 4712\n",
      "103\t59235\t0.3\t1\t54992 4243\n",
      "104\t55526\t0.3\t1\t51587 3939\n",
      "105\t49435\t0.3\t1\t45975 3460\n",
      "106\t45302\t0.3\t1\t42055 3247\n",
      "107\t41568\t0.3\t1\t38569 2999\n",
      "108\t39344\t0.3\t1\t36429 2915\n",
      "109\t35142\t0.3\t1\t32532 2610\n",
      "110\t32440\t0.3\t1\t30035 2405\n",
      "111\t29201\t0.3\t1\t26917 2284\n",
      "112\t25771\t0.3\t1\t23691 2080\n",
      "113\t22359\t0.3\t1\t20490 1869\n",
      "114\t19994\t0.3\t1\t18373 1621\n",
      "115\t17421\t0.3\t1\t16000 1421\n",
      "116\t15295\t0.3\t1\t14082 1213\n",
      "117\t13006\t0.3\t1\t11960 1046\n",
      "118\t11522\t0.3\t1\t10599 923\n",
      "119\t9598\t0.3\t1\t8847 751\n",
      "120\t7795\t0.3\t1\t7132 663\n",
      "121\t6634\t0.3\t1\t6018 616\n",
      "122\t5895\t0.3\t1\t5383 512\n",
      "123\t4806\t0.3\t1\t4352 454\n",
      "124\t4169\t0.3\t1\t3804 365\n",
      "125\t3459\t0.3\t1\t3117 342\n",
      "126\t3006\t0.3\t1\t2739 267\n",
      "127\t2458\t0.3\t1\t2229 229\n",
      "128\t1849\t0.3\t1\t1632 217\n",
      "129\t1614\t0.3\t1\t1454 160\n",
      "130\t1309\t0.3\t1\t1172 137\n",
      "131\t1113\t0.3\t1\t990 123\n",
      "132\t1037\t0.3\t1\t938 99\n",
      "133\t1203\t0.3\t1\t1080 123\n",
      "134\t1440\t0.3\t1\t1347 93\n",
      "135\t651\t0.3\t1\t599 52\n",
      "136\t330\t0.3\t1\t299 31\n",
      "137\t262\t0.3\t1\t226 36\n",
      "138\t169\t0.3\t1\t150 19\n",
      "139\t125\t0.3\t1\t105 20\n",
      "140\t97\t0.3\t1\t85 12\n",
      "141\t76\t0.3\t1\t66 10\n",
      "142\t39\t0.3\t1\t32 7\n",
      "143\t36\t0.3\t1\t31 5\n",
      "144\t28\t0.3\t1\t23 5\n",
      "145\t62\t0.3\t1\t55 7\n",
      "146\t39\t0.3\t1\t35 4\n",
      "147\t138\t0.3\t1\t123 15\n",
      "148\t73\t0.3\t1\t68 5\n",
      "149\t57\t0.3\t1\t50 7\n",
      "150\t745\t0.3\t1\t637 108\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "19571235 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_1_S1_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 738.09 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              19,571,235\n",
      "Reads with adapters:                16,271,535 (83.1%)\n",
      "Reads written (passing filters):    19,571,235 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,935,685,250 bp\n",
      "Quality-trimmed:              14,968,782 bp (0.5%)\n",
      "Total written (filtered):  2,137,866,393 bp (72.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16271535 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.4%\n",
      "  C: 12.6%\n",
      "  G: 25.1%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1934443\t4892808.8\t0\t1934443\n",
      "2\t292006\t1223202.2\t0\t292006\n",
      "3\t180981\t305800.5\t0\t180981\n",
      "4\t103045\t76450.1\t0\t103045\n",
      "5\t98608\t19112.5\t0\t98608\n",
      "6\t87562\t4778.1\t0\t87562\n",
      "7\t69739\t1194.5\t0\t69739\n",
      "8\t101138\t298.6\t0\t101138\n",
      "9\t80590\t74.7\t0\t80291 299\n",
      "10\t87083\t18.7\t1\t80903 6180\n",
      "11\t94744\t4.7\t1\t86859 7885\n",
      "12\t93509\t1.2\t1\t85833 7676\n",
      "13\t107672\t0.3\t1\t99050 8622\n",
      "14\t100767\t0.3\t1\t91771 8996\n",
      "15\t81145\t0.3\t1\t74763 6382\n",
      "16\t119727\t0.3\t1\t109867 9860\n",
      "17\t101003\t0.3\t1\t92682 8321\n",
      "18\t90163\t0.3\t1\t83122 7041\n",
      "19\t118403\t0.3\t1\t108874 9529\n",
      "20\t111750\t0.3\t1\t103085 8665\n",
      "21\t98369\t0.3\t1\t90814 7555\n",
      "22\t111808\t0.3\t1\t103242 8566\n",
      "23\t118255\t0.3\t1\t109009 9246\n",
      "24\t118284\t0.3\t1\t108899 9385\n",
      "25\t134913\t0.3\t1\t125516 9397\n",
      "26\t111710\t0.3\t1\t102736 8974\n",
      "27\t124043\t0.3\t1\t113612 10431\n",
      "28\t113048\t0.3\t1\t105567 7481\n",
      "29\t126854\t0.3\t1\t117529 9325\n",
      "30\t124172\t0.3\t1\t115653 8519\n",
      "31\t138567\t0.3\t1\t128798 9769\n",
      "32\t118532\t0.3\t1\t110581 7951\n",
      "33\t139520\t0.3\t1\t129113 10407\n",
      "34\t182115\t0.3\t1\t169415 12700\n",
      "35\t97650\t0.3\t1\t90702 6948\n",
      "36\t141232\t0.3\t1\t130648 10584\n",
      "37\t151183\t0.3\t1\t140571 10612\n",
      "38\t140794\t0.3\t1\t131792 9002\n",
      "39\t135094\t0.3\t1\t126791 8303\n",
      "40\t155470\t0.3\t1\t144881 10589\n",
      "41\t152110\t0.3\t1\t142307 9803\n",
      "42\t156034\t0.3\t1\t145893 10141\n",
      "43\t140408\t0.3\t1\t131398 9010\n",
      "44\t166488\t0.3\t1\t155279 11209\n",
      "45\t176759\t0.3\t1\t165212 11547\n",
      "46\t127616\t0.3\t1\t119430 8186\n",
      "47\t162788\t0.3\t1\t153156 9632\n",
      "48\t159821\t0.3\t1\t150024 9797\n",
      "49\t167515\t0.3\t1\t156791 10724\n",
      "50\t162854\t0.3\t1\t152416 10438\n",
      "51\t201852\t0.3\t1\t189155 12697\n",
      "52\t145803\t0.3\t1\t136773 9030\n",
      "53\t181599\t0.3\t1\t171435 10164\n",
      "54\t132563\t0.3\t1\t124771 7792\n",
      "55\t175847\t0.3\t1\t165921 9926\n",
      "56\t182238\t0.3\t1\t170952 11286\n",
      "57\t175759\t0.3\t1\t166454 9305\n",
      "58\t168899\t0.3\t1\t159768 9131\n",
      "59\t170487\t0.3\t1\t160788 9699\n",
      "60\t179591\t0.3\t1\t169442 10149\n",
      "61\t183203\t0.3\t1\t173168 10035\n",
      "62\t201509\t0.3\t1\t189650 11859\n",
      "63\t267932\t0.3\t1\t254493 13439\n",
      "64\t223354\t0.3\t1\t213922 9432\n",
      "65\t145944\t0.3\t1\t138249 7695\n",
      "66\t160488\t0.3\t1\t151671 8817\n",
      "67\t160429\t0.3\t1\t151245 9184\n",
      "68\t163626\t0.3\t1\t154620 9006\n",
      "69\t162388\t0.3\t1\t153192 9196\n",
      "70\t171183\t0.3\t1\t161721 9462\n",
      "71\t179954\t0.3\t1\t170347 9607\n",
      "72\t173057\t0.3\t1\t163544 9513\n",
      "73\t181347\t0.3\t1\t171665 9682\n",
      "74\t168687\t0.3\t1\t159362 9325\n",
      "75\t161354\t0.3\t1\t152554 8800\n",
      "76\t158086\t0.3\t1\t149538 8548\n",
      "77\t158408\t0.3\t1\t149654 8754\n",
      "78\t152524\t0.3\t1\t144195 8329\n",
      "79\t148468\t0.3\t1\t140463 8005\n",
      "80\t146791\t0.3\t1\t138965 7826\n",
      "81\t146914\t0.3\t1\t138951 7963\n",
      "82\t140738\t0.3\t1\t133245 7493\n",
      "83\t136720\t0.3\t1\t129557 7163\n",
      "84\t133340\t0.3\t1\t126399 6941\n",
      "85\t131509\t0.3\t1\t124949 6560\n",
      "86\t131725\t0.3\t1\t125074 6651\n",
      "87\t132992\t0.3\t1\t126515 6477\n",
      "88\t130544\t0.3\t1\t124131 6413\n",
      "89\t118247\t0.3\t1\t112246 6001\n",
      "90\t114880\t0.3\t1\t109042 5838\n",
      "91\t108202\t0.3\t1\t102767 5435\n",
      "92\t102465\t0.3\t1\t97298 5167\n",
      "93\t99313\t0.3\t1\t94369 4944\n",
      "94\t96247\t0.3\t1\t91557 4690\n",
      "95\t90355\t0.3\t1\t85764 4591\n",
      "96\t86105\t0.3\t1\t81859 4246\n",
      "97\t80863\t0.3\t1\t77002 3861\n",
      "98\t76806\t0.3\t1\t73205 3601\n",
      "99\t77725\t0.3\t1\t74014 3711\n",
      "100\t71514\t0.3\t1\t68090 3424\n",
      "101\t67122\t0.3\t1\t63869 3253\n",
      "102\t62547\t0.3\t1\t59630 2917\n",
      "103\t55990\t0.3\t1\t53291 2699\n",
      "104\t53130\t0.3\t1\t50496 2634\n",
      "105\t47112\t0.3\t1\t44737 2375\n",
      "106\t43436\t0.3\t1\t41396 2040\n",
      "107\t39313\t0.3\t1\t37430 1883\n",
      "108\t36083\t0.3\t1\t34294 1789\n",
      "109\t32958\t0.3\t1\t31374 1584\n",
      "110\t30549\t0.3\t1\t29023 1526\n",
      "111\t28680\t0.3\t1\t27349 1331\n",
      "112\t25012\t0.3\t1\t23754 1258\n",
      "113\t21932\t0.3\t1\t20848 1084\n",
      "114\t19318\t0.3\t1\t18424 894\n",
      "115\t16367\t0.3\t1\t15528 839\n",
      "116\t14214\t0.3\t1\t13511 703\n",
      "117\t11890\t0.3\t1\t11258 632\n",
      "118\t10558\t0.3\t1\t10008 550\n",
      "119\t8964\t0.3\t1\t8531 433\n",
      "120\t7160\t0.3\t1\t6810 350\n",
      "121\t6260\t0.3\t1\t5925 335\n",
      "122\t5526\t0.3\t1\t5261 265\n",
      "123\t4601\t0.3\t1\t4388 213\n",
      "124\t3924\t0.3\t1\t3711 213\n",
      "125\t3337\t0.3\t1\t3153 184\n",
      "126\t2851\t0.3\t1\t2683 168\n",
      "127\t2269\t0.3\t1\t2147 122\n",
      "128\t1715\t0.3\t1\t1615 100\n",
      "129\t1527\t0.3\t1\t1454 73\n",
      "130\t1240\t0.3\t1\t1181 59\n",
      "131\t1038\t0.3\t1\t985 53\n",
      "132\t992\t0.3\t1\t936 56\n",
      "133\t1237\t0.3\t1\t1180 57\n",
      "134\t1424\t0.3\t1\t1372 52\n",
      "135\t637\t0.3\t1\t603 34\n",
      "136\t325\t0.3\t1\t316 9\n",
      "137\t239\t0.3\t1\t218 21\n",
      "138\t165\t0.3\t1\t161 4\n",
      "139\t114\t0.3\t1\t104 10\n",
      "140\t92\t0.3\t1\t87 5\n",
      "141\t71\t0.3\t1\t68 3\n",
      "142\t32\t0.3\t1\t29 3\n",
      "143\t33\t0.3\t1\t28 5\n",
      "144\t26\t0.3\t1\t21 5\n",
      "145\t62\t0.3\t1\t60 2\n",
      "146\t42\t0.3\t1\t41 1\n",
      "147\t127\t0.3\t1\t113 14\n",
      "148\t65\t0.3\t1\t60 5\n",
      "149\t42\t0.3\t1\t37 5\n",
      "150\t529\t0.3\t1\t500 29\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "19571235 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz, file_2: 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 19571235\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 45454 (0.23%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_1_S1_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_1_S1_L002_R1_001_trimmed.fq.gz and 16C_32psu_1_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_2_S2_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 628.59 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,734,461\n",
      "Reads with adapters:                14,777,967 (83.3%)\n",
      "Reads written (passing filters):    17,734,461 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,660,169,150 bp\n",
      "Quality-trimmed:               5,148,925 bp (0.2%)\n",
      "Total written (filtered):  1,960,839,520 bp (73.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14777967 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 12.9%\n",
      "  G: 21.9%\n",
      "  T: 35.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1738050\t4433615.2\t0\t1738050\n",
      "2\t278207\t1108403.8\t0\t278207\n",
      "3\t163757\t277101.0\t0\t163757\n",
      "4\t100314\t69275.2\t0\t100314\n",
      "5\t85611\t17318.8\t0\t85611\n",
      "6\t82934\t4329.7\t0\t82934\n",
      "7\t71901\t1082.4\t0\t71901\n",
      "8\t92752\t270.6\t0\t92752\n",
      "9\t83307\t67.7\t0\t82257 1050\n",
      "10\t83535\t16.9\t1\t78970 4565\n",
      "11\t92909\t4.2\t1\t85625 7284\n",
      "12\t89396\t1.1\t1\t83407 5989\n",
      "13\t91391\t0.3\t1\t85129 6262\n",
      "14\t92986\t0.3\t1\t86205 6781\n",
      "15\t95417\t0.3\t1\t88994 6423\n",
      "16\t96485\t0.3\t1\t89177 7308\n",
      "17\t117735\t0.3\t1\t108243 9492\n",
      "18\t99811\t0.3\t1\t93600 6211\n",
      "19\t87560\t0.3\t1\t81823 5737\n",
      "20\t104404\t0.3\t1\t97366 7038\n",
      "21\t115608\t0.3\t1\t107491 8117\n",
      "22\t99044\t0.3\t1\t93521 5523\n",
      "23\t114295\t0.3\t1\t106563 7732\n",
      "24\t108405\t0.3\t1\t99404 9001\n",
      "25\t115898\t0.3\t1\t108064 7834\n",
      "26\t106400\t0.3\t1\t100103 6297\n",
      "27\t119862\t0.3\t1\t110913 8949\n",
      "28\t117009\t0.3\t1\t108719 8290\n",
      "29\t126066\t0.3\t1\t115846 10220\n",
      "30\t135672\t0.3\t1\t127802 7870\n",
      "31\t103769\t0.3\t1\t96862 6907\n",
      "32\t134576\t0.3\t1\t125712 8864\n",
      "33\t115506\t0.3\t1\t107678 7828\n",
      "34\t131922\t0.3\t1\t122476 9446\n",
      "35\t139540\t0.3\t1\t128729 10811\n",
      "36\t129944\t0.3\t1\t121921 8023\n",
      "37\t155181\t0.3\t1\t145058 10123\n",
      "38\t125033\t0.3\t1\t115777 9256\n",
      "39\t137711\t0.3\t1\t129241 8470\n",
      "40\t145763\t0.3\t1\t136531 9232\n",
      "41\t145406\t0.3\t1\t134994 10412\n",
      "42\t140227\t0.3\t1\t129509 10718\n",
      "43\t213869\t0.3\t1\t202923 10946\n",
      "44\t71935\t0.3\t1\t65627 6308\n",
      "45\t151697\t0.3\t1\t142261 9436\n",
      "46\t139466\t0.3\t1\t130673 8793\n",
      "47\t151314\t0.3\t1\t141909 9405\n",
      "48\t149772\t0.3\t1\t140199 9573\n",
      "49\t163566\t0.3\t1\t152198 11368\n",
      "50\t142863\t0.3\t1\t134146 8717\n",
      "51\t154145\t0.3\t1\t144042 10103\n",
      "52\t159345\t0.3\t1\t148339 11006\n",
      "53\t168861\t0.3\t1\t158401 10460\n",
      "54\t140944\t0.3\t1\t132160 8784\n",
      "55\t163358\t0.3\t1\t152729 10629\n",
      "56\t198374\t0.3\t1\t186858 11516\n",
      "57\t129041\t0.3\t1\t119923 9118\n",
      "58\t168939\t0.3\t1\t159723 9216\n",
      "59\t149936\t0.3\t1\t140432 9504\n",
      "60\t161773\t0.3\t1\t151304 10469\n",
      "61\t159837\t0.3\t1\t150130 9707\n",
      "62\t178889\t0.3\t1\t168432 10457\n",
      "63\t140394\t0.3\t1\t131036 9358\n",
      "64\t159197\t0.3\t1\t149784 9413\n",
      "65\t178150\t0.3\t1\t167258 10892\n",
      "66\t185093\t0.3\t1\t173786 11307\n",
      "67\t199374\t0.3\t1\t184232 15142\n",
      "68\t376952\t0.3\t1\t359289 17663\n",
      "69\t53765\t0.3\t1\t48910 4855\n",
      "70\t23496\t0.3\t1\t19245 4251\n",
      "71\t105909\t0.3\t1\t99682 6227\n",
      "72\t143025\t0.3\t1\t134667 8358\n",
      "73\t150010\t0.3\t1\t141153 8857\n",
      "74\t148858\t0.3\t1\t139863 8995\n",
      "75\t147559\t0.3\t1\t138722 8837\n",
      "76\t146256\t0.3\t1\t137657 8599\n",
      "77\t146473\t0.3\t1\t137642 8831\n",
      "78\t139439\t0.3\t1\t130923 8516\n",
      "79\t138812\t0.3\t1\t130392 8420\n",
      "80\t134925\t0.3\t1\t126986 7939\n",
      "81\t132740\t0.3\t1\t124849 7891\n",
      "82\t129707\t0.3\t1\t121848 7859\n",
      "83\t125045\t0.3\t1\t117720 7325\n",
      "84\t120955\t0.3\t1\t113756 7199\n",
      "85\t117638\t0.3\t1\t110705 6933\n",
      "86\t114999\t0.3\t1\t108021 6978\n",
      "87\t110985\t0.3\t1\t104322 6663\n",
      "88\t108344\t0.3\t1\t101912 6432\n",
      "89\t103347\t0.3\t1\t97100 6247\n",
      "90\t102084\t0.3\t1\t95939 6145\n",
      "91\t97477\t0.3\t1\t91376 6101\n",
      "92\t92174\t0.3\t1\t86611 5563\n",
      "93\t86985\t0.3\t1\t81755 5230\n",
      "94\t84316\t0.3\t1\t79155 5161\n",
      "95\t78474\t0.3\t1\t73684 4790\n",
      "96\t73594\t0.3\t1\t68994 4600\n",
      "97\t68665\t0.3\t1\t64422 4243\n",
      "98\t65130\t0.3\t1\t60988 4142\n",
      "99\t62724\t0.3\t1\t58428 4296\n",
      "100\t57069\t0.3\t1\t53328 3741\n",
      "101\t53096\t0.3\t1\t49656 3440\n",
      "102\t48453\t0.3\t1\t45307 3146\n",
      "103\t44883\t0.3\t1\t41855 3028\n",
      "104\t41860\t0.3\t1\t39145 2715\n",
      "105\t37319\t0.3\t1\t34804 2515\n",
      "106\t33117\t0.3\t1\t30846 2271\n",
      "107\t30224\t0.3\t1\t28086 2138\n",
      "108\t28577\t0.3\t1\t26568 2009\n",
      "109\t24900\t0.3\t1\t23148 1752\n",
      "110\t22182\t0.3\t1\t20539 1643\n",
      "111\t19835\t0.3\t1\t18458 1377\n",
      "112\t17455\t0.3\t1\t16149 1306\n",
      "113\t15019\t0.3\t1\t13793 1226\n",
      "114\t13036\t0.3\t1\t12026 1010\n",
      "115\t11314\t0.3\t1\t10380 934\n",
      "116\t9567\t0.3\t1\t8809 758\n",
      "117\t8005\t0.3\t1\t7289 716\n",
      "118\t6890\t0.3\t1\t6310 580\n",
      "119\t5712\t0.3\t1\t5183 529\n",
      "120\t4607\t0.3\t1\t4200 407\n",
      "121\t4029\t0.3\t1\t3642 387\n",
      "122\t3364\t0.3\t1\t3030 334\n",
      "123\t2640\t0.3\t1\t2382 258\n",
      "124\t2162\t0.3\t1\t1941 221\n",
      "125\t1794\t0.3\t1\t1596 198\n",
      "126\t1531\t0.3\t1\t1353 178\n",
      "127\t1215\t0.3\t1\t1098 117\n",
      "128\t1005\t0.3\t1\t879 126\n",
      "129\t809\t0.3\t1\t695 114\n",
      "130\t661\t0.3\t1\t597 64\n",
      "131\t509\t0.3\t1\t455 54\n",
      "132\t443\t0.3\t1\t389 54\n",
      "133\t409\t0.3\t1\t355 54\n",
      "134\t483\t0.3\t1\t430 53\n",
      "135\t256\t0.3\t1\t227 29\n",
      "136\t159\t0.3\t1\t133 26\n",
      "137\t104\t0.3\t1\t91 13\n",
      "138\t67\t0.3\t1\t53 14\n",
      "139\t54\t0.3\t1\t47 7\n",
      "140\t47\t0.3\t1\t39 8\n",
      "141\t21\t0.3\t1\t15 6\n",
      "142\t21\t0.3\t1\t19 2\n",
      "143\t8\t0.3\t1\t4 4\n",
      "144\t21\t0.3\t1\t20 1\n",
      "145\t51\t0.3\t1\t46 5\n",
      "146\t37\t0.3\t1\t29 8\n",
      "147\t70\t0.3\t1\t63 7\n",
      "148\t65\t0.3\t1\t52 13\n",
      "149\t39\t0.3\t1\t35 4\n",
      "150\t480\t0.3\t1\t358 122\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17734461 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_2_S2_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 655.62 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,734,461\n",
      "Reads with adapters:                14,708,042 (82.9%)\n",
      "Reads written (passing filters):    17,734,461 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,660,169,150 bp\n",
      "Quality-trimmed:              13,508,119 bp (0.5%)\n",
      "Total written (filtered):  1,965,332,665 bp (73.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14708042 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 12.0%\n",
      "  G: 24.7%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1823726\t4433615.2\t0\t1823726\n",
      "2\t284755\t1108403.8\t0\t284755\n",
      "3\t172034\t277101.0\t0\t172034\n",
      "4\t100056\t69275.2\t0\t100056\n",
      "5\t94653\t17318.8\t0\t94653\n",
      "6\t82500\t4329.7\t0\t82500\n",
      "7\t67622\t1082.4\t0\t67622\n",
      "8\t96831\t270.6\t0\t96831\n",
      "9\t76935\t67.7\t0\t76693 242\n",
      "10\t84917\t16.9\t1\t81330 3587\n",
      "11\t89902\t4.2\t1\t84914 4988\n",
      "12\t90846\t1.1\t1\t85794 5052\n",
      "13\t101679\t0.3\t1\t96540 5139\n",
      "14\t96092\t0.3\t1\t90143 5949\n",
      "15\t77319\t0.3\t1\t73208 4111\n",
      "16\t114151\t0.3\t1\t107904 6247\n",
      "17\t91752\t0.3\t1\t86386 5366\n",
      "18\t88174\t0.3\t1\t83679 4495\n",
      "19\t112277\t0.3\t1\t105925 6352\n",
      "20\t105306\t0.3\t1\t99944 5362\n",
      "21\t93914\t0.3\t1\t89017 4897\n",
      "22\t106559\t0.3\t1\t101114 5445\n",
      "23\t110793\t0.3\t1\t104912 5881\n",
      "24\t108885\t0.3\t1\t102936 5949\n",
      "25\t124131\t0.3\t1\t117742 6389\n",
      "26\t104655\t0.3\t1\t98869 5786\n",
      "27\t118794\t0.3\t1\t111576 7218\n",
      "28\t109018\t0.3\t1\t104356 4662\n",
      "29\t118362\t0.3\t1\t112044 6318\n",
      "30\t117104\t0.3\t1\t111822 5282\n",
      "31\t127211\t0.3\t1\t120779 6432\n",
      "32\t116001\t0.3\t1\t110611 5390\n",
      "33\t128546\t0.3\t1\t121947 6599\n",
      "34\t164203\t0.3\t1\t155704 8499\n",
      "35\t96279\t0.3\t1\t92057 4222\n",
      "36\t133909\t0.3\t1\t126861 7048\n",
      "37\t140147\t0.3\t1\t132941 7206\n",
      "38\t132076\t0.3\t1\t126615 5461\n",
      "39\t129476\t0.3\t1\t124081 5395\n",
      "40\t145043\t0.3\t1\t137854 7189\n",
      "41\t140152\t0.3\t1\t133933 6219\n",
      "42\t149889\t0.3\t1\t143207 6682\n",
      "43\t127335\t0.3\t1\t121774 5561\n",
      "44\t151174\t0.3\t1\t144187 6987\n",
      "45\t157069\t0.3\t1\t149770 7299\n",
      "46\t127123\t0.3\t1\t121787 5336\n",
      "47\t151889\t0.3\t1\t145363 6526\n",
      "48\t148136\t0.3\t1\t142156 5980\n",
      "49\t155657\t0.3\t1\t148850 6807\n",
      "50\t151455\t0.3\t1\t144647 6808\n",
      "51\t188717\t0.3\t1\t180540 8177\n",
      "52\t134588\t0.3\t1\t129312 5276\n",
      "53\t165157\t0.3\t1\t158982 6175\n",
      "54\t125197\t0.3\t1\t120399 4798\n",
      "55\t161733\t0.3\t1\t155845 5888\n",
      "56\t166592\t0.3\t1\t159894 6698\n",
      "57\t159442\t0.3\t1\t153722 5720\n",
      "58\t155066\t0.3\t1\t149532 5534\n",
      "59\t156693\t0.3\t1\t150794 5899\n",
      "60\t162642\t0.3\t1\t156571 6071\n",
      "61\t166749\t0.3\t1\t160204 6545\n",
      "62\t183711\t0.3\t1\t176590 7121\n",
      "63\t247983\t0.3\t1\t239379 8604\n",
      "64\t207871\t0.3\t1\t201612 6259\n",
      "65\t131895\t0.3\t1\t127166 4729\n",
      "66\t143535\t0.3\t1\t138546 4989\n",
      "67\t143350\t0.3\t1\t137946 5404\n",
      "68\t144808\t0.3\t1\t139662 5146\n",
      "69\t143872\t0.3\t1\t138552 5320\n",
      "70\t149770\t0.3\t1\t144363 5407\n",
      "71\t159740\t0.3\t1\t154143 5597\n",
      "72\t151950\t0.3\t1\t146385 5565\n",
      "73\t160797\t0.3\t1\t155248 5549\n",
      "74\t148028\t0.3\t1\t142917 5111\n",
      "75\t141398\t0.3\t1\t136490 4908\n",
      "76\t138432\t0.3\t1\t133692 4740\n",
      "77\t138578\t0.3\t1\t133680 4898\n",
      "78\t133953\t0.3\t1\t129214 4739\n",
      "79\t129618\t0.3\t1\t125018 4600\n",
      "80\t128335\t0.3\t1\t124016 4319\n",
      "81\t126134\t0.3\t1\t121892 4242\n",
      "82\t121190\t0.3\t1\t117136 4054\n",
      "83\t117273\t0.3\t1\t113373 3900\n",
      "84\t113306\t0.3\t1\t109397 3909\n",
      "85\t112548\t0.3\t1\t108604 3944\n",
      "86\t113235\t0.3\t1\t109296 3939\n",
      "87\t114063\t0.3\t1\t110237 3826\n",
      "88\t110699\t0.3\t1\t106931 3768\n",
      "89\t99019\t0.3\t1\t95692 3327\n",
      "90\t94514\t0.3\t1\t91234 3280\n",
      "91\t89156\t0.3\t1\t86055 3101\n",
      "92\t83513\t0.3\t1\t80693 2820\n",
      "93\t80899\t0.3\t1\t78065 2834\n",
      "94\t78488\t0.3\t1\t75830 2658\n",
      "95\t74185\t0.3\t1\t71733 2452\n",
      "96\t70096\t0.3\t1\t67821 2275\n",
      "97\t63309\t0.3\t1\t61201 2108\n",
      "98\t59315\t0.3\t1\t57291 2024\n",
      "99\t56316\t0.3\t1\t54360 1956\n",
      "100\t50955\t0.3\t1\t49217 1738\n",
      "101\t47839\t0.3\t1\t46277 1562\n",
      "102\t45011\t0.3\t1\t43553 1458\n",
      "103\t42598\t0.3\t1\t41263 1335\n",
      "104\t41495\t0.3\t1\t40193 1302\n",
      "105\t36516\t0.3\t1\t35305 1211\n",
      "106\t32285\t0.3\t1\t31286 999\n",
      "107\t28907\t0.3\t1\t27939 968\n",
      "108\t26521\t0.3\t1\t25551 970\n",
      "109\t23377\t0.3\t1\t22637 740\n",
      "110\t20921\t0.3\t1\t20208 713\n",
      "111\t19648\t0.3\t1\t19007 641\n",
      "112\t16880\t0.3\t1\t16303 577\n",
      "113\t14733\t0.3\t1\t14236 497\n",
      "114\t12634\t0.3\t1\t12219 415\n",
      "115\t10616\t0.3\t1\t10211 405\n",
      "116\t8792\t0.3\t1\t8474 318\n",
      "117\t7302\t0.3\t1\t7031 271\n",
      "118\t6324\t0.3\t1\t6071 253\n",
      "119\t5278\t0.3\t1\t5066 212\n",
      "120\t4245\t0.3\t1\t4065 180\n",
      "121\t3766\t0.3\t1\t3593 173\n",
      "122\t3122\t0.3\t1\t2986 136\n",
      "123\t2579\t0.3\t1\t2463 116\n",
      "124\t2054\t0.3\t1\t1977 77\n",
      "125\t1702\t0.3\t1\t1630 72\n",
      "126\t1444\t0.3\t1\t1379 65\n",
      "127\t1157\t0.3\t1\t1113 44\n",
      "128\t945\t0.3\t1\t906 39\n",
      "129\t732\t0.3\t1\t690 42\n",
      "130\t636\t0.3\t1\t596 40\n",
      "131\t486\t0.3\t1\t462 24\n",
      "132\t421\t0.3\t1\t393 28\n",
      "133\t391\t0.3\t1\t380 11\n",
      "134\t460\t0.3\t1\t437 23\n",
      "135\t244\t0.3\t1\t230 14\n",
      "136\t145\t0.3\t1\t134 11\n",
      "137\t97\t0.3\t1\t92 5\n",
      "138\t61\t0.3\t1\t60 1\n",
      "139\t51\t0.3\t1\t44 7\n",
      "140\t41\t0.3\t1\t36 5\n",
      "141\t18\t0.3\t1\t15 3\n",
      "142\t19\t0.3\t1\t18 1\n",
      "143\t6\t0.3\t1\t6\n",
      "144\t18\t0.3\t1\t16 2\n",
      "145\t49\t0.3\t1\t42 7\n",
      "146\t32\t0.3\t1\t31 1\n",
      "147\t61\t0.3\t1\t61\n",
      "148\t55\t0.3\t1\t53 2\n",
      "149\t39\t0.3\t1\t37 2\n",
      "150\t364\t0.3\t1\t342 22\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17734461 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz, file_2: 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17734461\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 31473 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_2_S2_L001_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_2_S2_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 635.56 s (36 us/read; 1.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,682,252\n",
      "Reads with adapters:                14,725,601 (83.3%)\n",
      "Reads written (passing filters):    17,682,252 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,652,337,800 bp\n",
      "Quality-trimmed:               5,252,718 bp (0.2%)\n",
      "Total written (filtered):  1,953,712,678 bp (73.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14725601 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 12.9%\n",
      "  G: 21.9%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1730837\t4420563.0\t0\t1730837\n",
      "2\t274288\t1105140.8\t0\t274288\n",
      "3\t164540\t276285.2\t0\t164540\n",
      "4\t98245\t69071.3\t0\t98245\n",
      "5\t85133\t17267.8\t0\t85133\n",
      "6\t82977\t4317.0\t0\t82977\n",
      "7\t68628\t1079.2\t0\t68628\n",
      "8\t92219\t269.8\t0\t92219\n",
      "9\t82148\t67.5\t0\t81058 1090\n",
      "10\t81207\t16.9\t1\t75933 5274\n",
      "11\t91764\t4.2\t1\t83517 8247\n",
      "12\t88451\t1.1\t1\t81680 6771\n",
      "13\t90852\t0.3\t1\t83635 7217\n",
      "14\t92006\t0.3\t1\t84006 8000\n",
      "15\t95802\t0.3\t1\t88019 7783\n",
      "16\t94793\t0.3\t1\t86152 8641\n",
      "17\t117965\t0.3\t1\t106745 11220\n",
      "18\t99003\t0.3\t1\t91800 7203\n",
      "19\t85234\t0.3\t1\t78539 6695\n",
      "20\t103603\t0.3\t1\t95615 7988\n",
      "21\t115441\t0.3\t1\t106021 9420\n",
      "22\t96644\t0.3\t1\t89824 6820\n",
      "23\t112899\t0.3\t1\t103928 8971\n",
      "24\t106903\t0.3\t1\t96909 9994\n",
      "25\t114269\t0.3\t1\t105193 9076\n",
      "26\t105406\t0.3\t1\t97809 7597\n",
      "27\t119549\t0.3\t1\t109420 10129\n",
      "28\t117559\t0.3\t1\t108344 9215\n",
      "29\t126569\t0.3\t1\t115202 11367\n",
      "30\t135359\t0.3\t1\t126047 9312\n",
      "31\t102444\t0.3\t1\t94598 7846\n",
      "32\t135060\t0.3\t1\t124361 10699\n",
      "33\t113717\t0.3\t1\t104773 8944\n",
      "34\t130221\t0.3\t1\t118947 11274\n",
      "35\t139590\t0.3\t1\t127417 12173\n",
      "36\t127748\t0.3\t1\t118449 9299\n",
      "37\t156612\t0.3\t1\t144323 12289\n",
      "38\t124619\t0.3\t1\t114023 10596\n",
      "39\t136370\t0.3\t1\t126274 10096\n",
      "40\t143456\t0.3\t1\t132041 11415\n",
      "41\t145337\t0.3\t1\t132898 12439\n",
      "42\t139531\t0.3\t1\t126917 12614\n",
      "43\t211559\t0.3\t1\t197954 13605\n",
      "44\t73750\t0.3\t1\t66328 7422\n",
      "45\t151399\t0.3\t1\t140473 10926\n",
      "46\t138024\t0.3\t1\t127568 10456\n",
      "47\t150595\t0.3\t1\t139553 11042\n",
      "48\t147780\t0.3\t1\t136652 11128\n",
      "49\t162871\t0.3\t1\t149899 12972\n",
      "50\t141270\t0.3\t1\t131458 9812\n",
      "51\t154346\t0.3\t1\t142859 11487\n",
      "52\t160748\t0.3\t1\t148256 12492\n",
      "53\t167551\t0.3\t1\t155619 11932\n",
      "54\t139669\t0.3\t1\t129569 10100\n",
      "55\t162273\t0.3\t1\t150174 12099\n",
      "56\t200304\t0.3\t1\t186484 13820\n",
      "57\t127569\t0.3\t1\t117186 10383\n",
      "58\t167466\t0.3\t1\t156693 10773\n",
      "59\t150014\t0.3\t1\t138880 11134\n",
      "60\t160710\t0.3\t1\t148928 11782\n",
      "61\t158834\t0.3\t1\t147715 11119\n",
      "62\t178143\t0.3\t1\t165938 12205\n",
      "63\t141715\t0.3\t1\t130934 10781\n",
      "64\t158746\t0.3\t1\t148093 10653\n",
      "65\t176936\t0.3\t1\t164216 12720\n",
      "66\t182618\t0.3\t1\t169484 13134\n",
      "67\t198670\t0.3\t1\t182225 16445\n",
      "68\t365118\t0.3\t1\t345570 19548\n",
      "69\t68264\t0.3\t1\t62074 6190\n",
      "70\t29092\t0.3\t1\t24265 4827\n",
      "71\t103442\t0.3\t1\t96474 6968\n",
      "72\t142212\t0.3\t1\t133025 9187\n",
      "73\t149684\t0.3\t1\t139925 9759\n",
      "74\t148896\t0.3\t1\t138764 10132\n",
      "75\t148109\t0.3\t1\t137782 10327\n",
      "76\t146530\t0.3\t1\t136141 10389\n",
      "77\t146539\t0.3\t1\t136490 10049\n",
      "78\t139703\t0.3\t1\t129860 9843\n",
      "79\t137432\t0.3\t1\t127635 9797\n",
      "80\t135397\t0.3\t1\t125660 9737\n",
      "81\t133287\t0.3\t1\t123761 9526\n",
      "82\t129169\t0.3\t1\t119849 9320\n",
      "83\t125954\t0.3\t1\t117089 8865\n",
      "84\t121677\t0.3\t1\t112941 8736\n",
      "85\t117830\t0.3\t1\t109511 8319\n",
      "86\t114988\t0.3\t1\t106799 8189\n",
      "87\t110799\t0.3\t1\t103288 7511\n",
      "88\t109823\t0.3\t1\t102149 7674\n",
      "89\t103812\t0.3\t1\t96531 7281\n",
      "90\t101371\t0.3\t1\t94159 7212\n",
      "91\t96555\t0.3\t1\t89763 6792\n",
      "92\t92588\t0.3\t1\t86028 6560\n",
      "93\t86787\t0.3\t1\t80806 5981\n",
      "94\t84568\t0.3\t1\t78482 6086\n",
      "95\t78600\t0.3\t1\t72903 5697\n",
      "96\t74361\t0.3\t1\t69003 5358\n",
      "97\t69748\t0.3\t1\t64791 4957\n",
      "98\t65245\t0.3\t1\t60478 4767\n",
      "99\t62580\t0.3\t1\t57800 4780\n",
      "100\t56847\t0.3\t1\t52715 4132\n",
      "101\t53263\t0.3\t1\t49277 3986\n",
      "102\t48648\t0.3\t1\t45084 3564\n",
      "103\t45338\t0.3\t1\t42014 3324\n",
      "104\t41917\t0.3\t1\t38870 3047\n",
      "105\t37301\t0.3\t1\t34586 2715\n",
      "106\t33679\t0.3\t1\t31071 2608\n",
      "107\t30205\t0.3\t1\t27947 2258\n",
      "108\t27998\t0.3\t1\t25811 2187\n",
      "109\t24717\t0.3\t1\t22799 1918\n",
      "110\t22343\t0.3\t1\t20572 1771\n",
      "111\t20279\t0.3\t1\t18723 1556\n",
      "112\t17492\t0.3\t1\t16037 1455\n",
      "113\t14993\t0.3\t1\t13741 1252\n",
      "114\t13102\t0.3\t1\t12018 1084\n",
      "115\t11167\t0.3\t1\t10176 991\n",
      "116\t9702\t0.3\t1\t8851 851\n",
      "117\t8008\t0.3\t1\t7282 726\n",
      "118\t6784\t0.3\t1\t6194 590\n",
      "119\t5714\t0.3\t1\t5190 524\n",
      "120\t4836\t0.3\t1\t4363 473\n",
      "121\t3994\t0.3\t1\t3606 388\n",
      "122\t3193\t0.3\t1\t2889 304\n",
      "123\t2794\t0.3\t1\t2493 301\n",
      "124\t2161\t0.3\t1\t1934 227\n",
      "125\t1832\t0.3\t1\t1606 226\n",
      "126\t1531\t0.3\t1\t1356 175\n",
      "127\t1197\t0.3\t1\t1054 143\n",
      "128\t960\t0.3\t1\t854 106\n",
      "129\t853\t0.3\t1\t753 100\n",
      "130\t591\t0.3\t1\t514 77\n",
      "131\t531\t0.3\t1\t470 61\n",
      "132\t430\t0.3\t1\t380 50\n",
      "133\t462\t0.3\t1\t411 51\n",
      "134\t470\t0.3\t1\t418 52\n",
      "135\t263\t0.3\t1\t225 38\n",
      "136\t146\t0.3\t1\t127 19\n",
      "137\t106\t0.3\t1\t90 16\n",
      "138\t84\t0.3\t1\t66 18\n",
      "139\t59\t0.3\t1\t48 11\n",
      "140\t48\t0.3\t1\t41 7\n",
      "141\t28\t0.3\t1\t20 8\n",
      "142\t22\t0.3\t1\t17 5\n",
      "143\t22\t0.3\t1\t15 7\n",
      "144\t21\t0.3\t1\t19 2\n",
      "145\t41\t0.3\t1\t38 3\n",
      "146\t29\t0.3\t1\t26 3\n",
      "147\t81\t0.3\t1\t70 11\n",
      "148\t47\t0.3\t1\t39 8\n",
      "149\t50\t0.3\t1\t42 8\n",
      "150\t504\t0.3\t1\t392 112\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17682252 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_2_S2_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 664.74 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,682,252\n",
      "Reads with adapters:                14,653,341 (82.9%)\n",
      "Reads written (passing filters):    17,682,252 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,652,337,800 bp\n",
      "Quality-trimmed:              13,059,983 bp (0.5%)\n",
      "Total written (filtered):  1,958,600,443 bp (73.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14653341 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.0%\n",
      "  G: 24.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1817135\t4420563.0\t0\t1817135\n",
      "2\t279096\t1105140.8\t0\t279096\n",
      "3\t175846\t276285.2\t0\t175846\n",
      "4\t97770\t69071.3\t0\t97770\n",
      "5\t91322\t17267.8\t0\t91322\n",
      "6\t82979\t4317.0\t0\t82979\n",
      "7\t65452\t1079.2\t0\t65452\n",
      "8\t95994\t269.8\t0\t95994\n",
      "9\t76273\t67.5\t0\t76051 222\n",
      "10\t81686\t16.9\t1\t76699 4987\n",
      "11\t88683\t4.2\t1\t82075 6608\n",
      "12\t88254\t1.1\t1\t81734 6520\n",
      "13\t102186\t0.3\t1\t94885 7301\n",
      "14\t94341\t0.3\t1\t86650 7691\n",
      "15\t77377\t0.3\t1\t72050 5327\n",
      "16\t111829\t0.3\t1\t103609 8220\n",
      "17\t95099\t0.3\t1\t87974 7125\n",
      "18\t85068\t0.3\t1\t79390 5678\n",
      "19\t110655\t0.3\t1\t102928 7727\n",
      "20\t104031\t0.3\t1\t97040 6991\n",
      "21\t92000\t0.3\t1\t85864 6136\n",
      "22\t104100\t0.3\t1\t97146 6954\n",
      "23\t109645\t0.3\t1\t102128 7517\n",
      "24\t108307\t0.3\t1\t100742 7565\n",
      "25\t123264\t0.3\t1\t115703 7561\n",
      "26\t104401\t0.3\t1\t96855 7546\n",
      "27\t119574\t0.3\t1\t110821 8753\n",
      "28\t105530\t0.3\t1\t99453 6077\n",
      "29\t120646\t0.3\t1\t112687 7959\n",
      "30\t114291\t0.3\t1\t107348 6943\n",
      "31\t129583\t0.3\t1\t121518 8065\n",
      "32\t112118\t0.3\t1\t105486 6632\n",
      "33\t129638\t0.3\t1\t121053 8585\n",
      "34\t165788\t0.3\t1\t155262 10526\n",
      "35\t90769\t0.3\t1\t85358 5411\n",
      "36\t134484\t0.3\t1\t125338 9146\n",
      "37\t141040\t0.3\t1\t132135 8905\n",
      "38\t130890\t0.3\t1\t123508 7382\n",
      "39\t125992\t0.3\t1\t119419 6573\n",
      "40\t143114\t0.3\t1\t134816 8298\n",
      "41\t141161\t0.3\t1\t133004 8157\n",
      "42\t150560\t0.3\t1\t142032 8528\n",
      "43\t126059\t0.3\t1\t119274 6785\n",
      "44\t153119\t0.3\t1\t144163 8956\n",
      "45\t156350\t0.3\t1\t146944 9406\n",
      "46\t124535\t0.3\t1\t117729 6806\n",
      "47\t151106\t0.3\t1\t143277 7829\n",
      "48\t145709\t0.3\t1\t138077 7632\n",
      "49\t153645\t0.3\t1\t144717 8928\n",
      "50\t152443\t0.3\t1\t143810 8633\n",
      "51\t186444\t0.3\t1\t176105 10339\n",
      "52\t135339\t0.3\t1\t128275 7064\n",
      "53\t167309\t0.3\t1\t158978 8331\n",
      "54\t122506\t0.3\t1\t116488 6018\n",
      "55\t159881\t0.3\t1\t152187 7694\n",
      "56\t166549\t0.3\t1\t157620 8929\n",
      "57\t159405\t0.3\t1\t152142 7263\n",
      "58\t154742\t0.3\t1\t147569 7173\n",
      "59\t154815\t0.3\t1\t147039 7776\n",
      "60\t162353\t0.3\t1\t154259 8094\n",
      "61\t164751\t0.3\t1\t156984 7767\n",
      "62\t181848\t0.3\t1\t172769 9079\n",
      "63\t243575\t0.3\t1\t232900 10675\n",
      "64\t204233\t0.3\t1\t196775 7458\n",
      "65\t131464\t0.3\t1\t125558 5906\n",
      "66\t143870\t0.3\t1\t137254 6616\n",
      "67\t143305\t0.3\t1\t136348 6957\n",
      "68\t146020\t0.3\t1\t139340 6680\n",
      "69\t145720\t0.3\t1\t138556 7164\n",
      "70\t152388\t0.3\t1\t145229 7159\n",
      "71\t160483\t0.3\t1\t152923 7560\n",
      "72\t153910\t0.3\t1\t146668 7242\n",
      "73\t161080\t0.3\t1\t153734 7346\n",
      "74\t148742\t0.3\t1\t141577 7165\n",
      "75\t142283\t0.3\t1\t135459 6824\n",
      "76\t139212\t0.3\t1\t132662 6550\n",
      "77\t138672\t0.3\t1\t132174 6498\n",
      "78\t134120\t0.3\t1\t127702 6418\n",
      "79\t128493\t0.3\t1\t122569 5924\n",
      "80\t128817\t0.3\t1\t123042 5775\n",
      "81\t126667\t0.3\t1\t120859 5808\n",
      "82\t120873\t0.3\t1\t115386 5487\n",
      "83\t118191\t0.3\t1\t112948 5243\n",
      "84\t114338\t0.3\t1\t109255 5083\n",
      "85\t112679\t0.3\t1\t107691 4988\n",
      "86\t112861\t0.3\t1\t107936 4925\n",
      "87\t113253\t0.3\t1\t108504 4749\n",
      "88\t112053\t0.3\t1\t107404 4649\n",
      "89\t99896\t0.3\t1\t95509 4387\n",
      "90\t94224\t0.3\t1\t90152 4072\n",
      "91\t88708\t0.3\t1\t84909 3799\n",
      "92\t84091\t0.3\t1\t80464 3627\n",
      "93\t80972\t0.3\t1\t77474 3498\n",
      "94\t78965\t0.3\t1\t75719 3246\n",
      "95\t74676\t0.3\t1\t71434 3242\n",
      "96\t70973\t0.3\t1\t67917 3056\n",
      "97\t64791\t0.3\t1\t62029 2762\n",
      "98\t59852\t0.3\t1\t57281 2571\n",
      "99\t56272\t0.3\t1\t53840 2432\n",
      "100\t50916\t0.3\t1\t48630 2286\n",
      "101\t48192\t0.3\t1\t46113 2079\n",
      "102\t45225\t0.3\t1\t43338 1887\n",
      "103\t42981\t0.3\t1\t41068 1913\n",
      "104\t41627\t0.3\t1\t39818 1809\n",
      "105\t36524\t0.3\t1\t34997 1527\n",
      "106\t32960\t0.3\t1\t31559 1401\n",
      "107\t28968\t0.3\t1\t27743 1225\n",
      "108\t25878\t0.3\t1\t24765 1113\n",
      "109\t23275\t0.3\t1\t22256 1019\n",
      "110\t21045\t0.3\t1\t20178 867\n",
      "111\t19919\t0.3\t1\t19038 881\n",
      "112\t16987\t0.3\t1\t16274 713\n",
      "113\t14633\t0.3\t1\t14022 611\n",
      "114\t12693\t0.3\t1\t12155 538\n",
      "115\t10546\t0.3\t1\t10065 481\n",
      "116\t8985\t0.3\t1\t8625 360\n",
      "117\t7253\t0.3\t1\t6908 345\n",
      "118\t6254\t0.3\t1\t5971 283\n",
      "119\t5291\t0.3\t1\t5043 248\n",
      "120\t4411\t0.3\t1\t4200 211\n",
      "121\t3756\t0.3\t1\t3560 196\n",
      "122\t3023\t0.3\t1\t2867 156\n",
      "123\t2649\t0.3\t1\t2500 149\n",
      "124\t2061\t0.3\t1\t1951 110\n",
      "125\t1723\t0.3\t1\t1623 100\n",
      "126\t1447\t0.3\t1\t1342 105\n",
      "127\t1137\t0.3\t1\t1082 55\n",
      "128\t888\t0.3\t1\t830 58\n",
      "129\t801\t0.3\t1\t760 41\n",
      "130\t520\t0.3\t1\t487 33\n",
      "131\t493\t0.3\t1\t458 35\n",
      "132\t407\t0.3\t1\t377 30\n",
      "133\t444\t0.3\t1\t414 30\n",
      "134\t445\t0.3\t1\t433 12\n",
      "135\t251\t0.3\t1\t241 10\n",
      "136\t140\t0.3\t1\t129 11\n",
      "137\t98\t0.3\t1\t90 8\n",
      "138\t74\t0.3\t1\t66 8\n",
      "139\t51\t0.3\t1\t50 1\n",
      "140\t46\t0.3\t1\t45 1\n",
      "141\t23\t0.3\t1\t22 1\n",
      "142\t21\t0.3\t1\t19 2\n",
      "143\t18\t0.3\t1\t16 2\n",
      "144\t19\t0.3\t1\t19\n",
      "145\t44\t0.3\t1\t42 2\n",
      "146\t27\t0.3\t1\t26 1\n",
      "147\t73\t0.3\t1\t69 4\n",
      "148\t42\t0.3\t1\t38 4\n",
      "149\t58\t0.3\t1\t55 3\n",
      "150\t398\t0.3\t1\t372 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17682252 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz, file_2: 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17682252\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 29703 (0.17%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_2_S2_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_2_S2_L002_R1_001_trimmed.fq.gz and 16C_32psu_2_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_3_S3_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 468.61 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,688,650\n",
      "Reads with adapters:                11,692,195 (85.4%)\n",
      "Reads written (passing filters):    13,688,650 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,053,297,500 bp\n",
      "Quality-trimmed:               4,119,462 bp (0.2%)\n",
      "Total written (filtered):  1,465,023,732 bp (71.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11692195 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.2%\n",
      "  C: 13.5%\n",
      "  G: 21.8%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1153280\t3422162.5\t0\t1153280\n",
      "2\t188115\t855540.6\t0\t188115\n",
      "3\t111422\t213885.2\t0\t111422\n",
      "4\t69723\t53471.3\t0\t69723\n",
      "5\t62088\t13367.8\t0\t62088\n",
      "6\t60662\t3342.0\t0\t60662\n",
      "7\t51512\t835.5\t0\t51512\n",
      "8\t64264\t208.9\t0\t64264\n",
      "9\t63671\t52.2\t0\t62836 835\n",
      "10\t60542\t13.1\t1\t57278 3264\n",
      "11\t67613\t3.3\t1\t62345 5268\n",
      "12\t65175\t0.8\t1\t60838 4337\n",
      "13\t67473\t0.2\t1\t62952 4521\n",
      "14\t68261\t0.2\t1\t63487 4774\n",
      "15\t72178\t0.2\t1\t67286 4892\n",
      "16\t70299\t0.2\t1\t65086 5213\n",
      "17\t86060\t0.2\t1\t78938 7122\n",
      "18\t74650\t0.2\t1\t70063 4587\n",
      "19\t65040\t0.2\t1\t60840 4200\n",
      "20\t78366\t0.2\t1\t73149 5217\n",
      "21\t84846\t0.2\t1\t78850 5996\n",
      "22\t74487\t0.2\t1\t70107 4380\n",
      "23\t86083\t0.2\t1\t80316 5767\n",
      "24\t80721\t0.2\t1\t73972 6749\n",
      "25\t87753\t0.2\t1\t81677 6076\n",
      "26\t80514\t0.2\t1\t75647 4867\n",
      "27\t90438\t0.2\t1\t83662 6776\n",
      "28\t90174\t0.2\t1\t83716 6458\n",
      "29\t93389\t0.2\t1\t85784 7605\n",
      "30\t100786\t0.2\t1\t94960 5826\n",
      "31\t81881\t0.2\t1\t76679 5202\n",
      "32\t98375\t0.2\t1\t91951 6424\n",
      "33\t93955\t0.2\t1\t87212 6743\n",
      "34\t99759\t0.2\t1\t92551 7208\n",
      "35\t105302\t0.2\t1\t98208 7094\n",
      "36\t99885\t0.2\t1\t93387 6498\n",
      "37\t106334\t0.2\t1\t98270 8064\n",
      "38\t113806\t0.2\t1\t106308 7498\n",
      "39\t98325\t0.2\t1\t92047 6278\n",
      "40\t111911\t0.2\t1\t104331 7580\n",
      "41\t123334\t0.2\t1\t114370 8964\n",
      "42\t99813\t0.2\t1\t92381 7432\n",
      "43\t174315\t0.2\t1\t165200 9115\n",
      "44\t51463\t0.2\t1\t46797 4666\n",
      "45\t117571\t0.2\t1\t110330 7241\n",
      "46\t111696\t0.2\t1\t104449 7247\n",
      "47\t119263\t0.2\t1\t111850 7413\n",
      "48\t117185\t0.2\t1\t109504 7681\n",
      "49\t129249\t0.2\t1\t119912 9337\n",
      "50\t113265\t0.2\t1\t106460 6805\n",
      "51\t123578\t0.2\t1\t115577 8001\n",
      "52\t126860\t0.2\t1\t117703 9157\n",
      "53\t136778\t0.2\t1\t128183 8595\n",
      "54\t111542\t0.2\t1\t104628 6914\n",
      "55\t130509\t0.2\t1\t122033 8476\n",
      "56\t160806\t0.2\t1\t151398 9408\n",
      "57\t105238\t0.2\t1\t97627 7611\n",
      "58\t136703\t0.2\t1\t129347 7356\n",
      "59\t118654\t0.2\t1\t110900 7754\n",
      "60\t133349\t0.2\t1\t124657 8692\n",
      "61\t131151\t0.2\t1\t123141 8010\n",
      "62\t146081\t0.2\t1\t137646 8435\n",
      "63\t116527\t0.2\t1\t108569 7958\n",
      "64\t131388\t0.2\t1\t123568 7820\n",
      "65\t146211\t0.2\t1\t137343 8868\n",
      "66\t153623\t0.2\t1\t144098 9525\n",
      "67\t163622\t0.2\t1\t150969 12653\n",
      "68\t316517\t0.2\t1\t301519 14998\n",
      "69\t50118\t0.2\t1\t45752 4366\n",
      "70\t19043\t0.2\t1\t15441 3602\n",
      "71\t86203\t0.2\t1\t80944 5259\n",
      "72\t120080\t0.2\t1\t113140 6940\n",
      "73\t127657\t0.2\t1\t119892 7765\n",
      "74\t126886\t0.2\t1\t119284 7602\n",
      "75\t126438\t0.2\t1\t118691 7747\n",
      "76\t124650\t0.2\t1\t117103 7547\n",
      "77\t126032\t0.2\t1\t118426 7606\n",
      "78\t119720\t0.2\t1\t112423 7297\n",
      "79\t118994\t0.2\t1\t111880 7114\n",
      "80\t116560\t0.2\t1\t109502 7058\n",
      "81\t114889\t0.2\t1\t107938 6951\n",
      "82\t112663\t0.2\t1\t105830 6833\n",
      "83\t109839\t0.2\t1\t103073 6766\n",
      "84\t105779\t0.2\t1\t99490 6289\n",
      "85\t104543\t0.2\t1\t98380 6163\n",
      "86\t100937\t0.2\t1\t94813 6124\n",
      "87\t97367\t0.2\t1\t91433 5934\n",
      "88\t96957\t0.2\t1\t90929 6028\n",
      "89\t91739\t0.2\t1\t86072 5667\n",
      "90\t91064\t0.2\t1\t85580 5484\n",
      "91\t86291\t0.2\t1\t80995 5296\n",
      "92\t83034\t0.2\t1\t77959 5075\n",
      "93\t78558\t0.2\t1\t73687 4871\n",
      "94\t75965\t0.2\t1\t71248 4717\n",
      "95\t70608\t0.2\t1\t66214 4394\n",
      "96\t66613\t0.2\t1\t62491 4122\n",
      "97\t63167\t0.2\t1\t59156 4011\n",
      "98\t59278\t0.2\t1\t55535 3743\n",
      "99\t58315\t0.2\t1\t54380 3935\n",
      "100\t52512\t0.2\t1\t49178 3334\n",
      "101\t49162\t0.2\t1\t45965 3197\n",
      "102\t45067\t0.2\t1\t42105 2962\n",
      "103\t41984\t0.2\t1\t39185 2799\n",
      "104\t39240\t0.2\t1\t36628 2612\n",
      "105\t34957\t0.2\t1\t32577 2380\n",
      "106\t31819\t0.2\t1\t29724 2095\n",
      "107\t29159\t0.2\t1\t27051 2108\n",
      "108\t27128\t0.2\t1\t25184 1944\n",
      "109\t24322\t0.2\t1\t22614 1708\n",
      "110\t21994\t0.2\t1\t20354 1640\n",
      "111\t19565\t0.2\t1\t18156 1409\n",
      "112\t17167\t0.2\t1\t15854 1313\n",
      "113\t14819\t0.2\t1\t13686 1133\n",
      "114\t13381\t0.2\t1\t12379 1002\n",
      "115\t11437\t0.2\t1\t10576 861\n",
      "116\t9792\t0.2\t1\t9003 789\n",
      "117\t8262\t0.2\t1\t7551 711\n",
      "118\t7359\t0.2\t1\t6738 621\n",
      "119\t6022\t0.2\t1\t5480 542\n",
      "120\t5103\t0.2\t1\t4650 453\n",
      "121\t4422\t0.2\t1\t4023 399\n",
      "122\t3603\t0.2\t1\t3261 342\n",
      "123\t3138\t0.2\t1\t2830 308\n",
      "124\t2526\t0.2\t1\t2281 245\n",
      "125\t2064\t0.2\t1\t1886 178\n",
      "126\t1770\t0.2\t1\t1574 196\n",
      "127\t1476\t0.2\t1\t1324 152\n",
      "128\t1074\t0.2\t1\t953 121\n",
      "129\t984\t0.2\t1\t870 114\n",
      "130\t708\t0.2\t1\t638 70\n",
      "131\t577\t0.2\t1\t522 55\n",
      "132\t547\t0.2\t1\t477 70\n",
      "133\t519\t0.2\t1\t468 51\n",
      "134\t567\t0.2\t1\t518 49\n",
      "135\t307\t0.2\t1\t275 32\n",
      "136\t217\t0.2\t1\t195 22\n",
      "137\t172\t0.2\t1\t148 24\n",
      "138\t111\t0.2\t1\t95 16\n",
      "139\t77\t0.2\t1\t67 10\n",
      "140\t52\t0.2\t1\t47 5\n",
      "141\t43\t0.2\t1\t38 5\n",
      "142\t44\t0.2\t1\t33 11\n",
      "143\t50\t0.2\t1\t44 6\n",
      "144\t49\t0.2\t1\t46 3\n",
      "145\t55\t0.2\t1\t44 11\n",
      "146\t89\t0.2\t1\t78 11\n",
      "147\t261\t0.2\t1\t233 28\n",
      "148\t216\t0.2\t1\t199 17\n",
      "149\t67\t0.2\t1\t64 3\n",
      "150\t763\t0.2\t1\t633 130\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13688650 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_3_S3_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 501.39 s (37 us/read; 1.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,688,650\n",
      "Reads with adapters:                11,592,048 (84.7%)\n",
      "Reads written (passing filters):    13,688,650 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,053,297,500 bp\n",
      "Quality-trimmed:              12,031,923 bp (0.6%)\n",
      "Total written (filtered):  1,469,819,891 bp (71.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11592048 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.3%\n",
      "  G: 24.9%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1214265\t3422162.5\t0\t1214265\n",
      "2\t193972\t855540.6\t0\t193972\n",
      "3\t118499\t213885.2\t0\t118499\n",
      "4\t69716\t53471.3\t0\t69716\n",
      "5\t68874\t13367.8\t0\t68874\n",
      "6\t59538\t3342.0\t0\t59538\n",
      "7\t48194\t835.5\t0\t48194\n",
      "8\t70200\t208.9\t0\t70200\n",
      "9\t54828\t52.2\t0\t54657 171\n",
      "10\t61679\t13.1\t1\t58439 3240\n",
      "11\t64699\t3.3\t1\t60413 4286\n",
      "12\t66050\t0.8\t1\t61625 4425\n",
      "13\t73701\t0.2\t1\t69161 4540\n",
      "14\t72487\t0.2\t1\t67153 5334\n",
      "15\t56503\t0.2\t1\t52652 3851\n",
      "16\t83460\t0.2\t1\t77990 5470\n",
      "17\t68575\t0.2\t1\t63675 4900\n",
      "18\t63646\t0.2\t1\t59572 4074\n",
      "19\t84575\t0.2\t1\t78822 5753\n",
      "20\t78107\t0.2\t1\t73276 4831\n",
      "21\t67477\t0.2\t1\t63200 4277\n",
      "22\t79410\t0.2\t1\t74414 4996\n",
      "23\t81981\t0.2\t1\t76795 5186\n",
      "24\t83330\t0.2\t1\t77735 5595\n",
      "25\t95526\t0.2\t1\t89714 5812\n",
      "26\t78503\t0.2\t1\t73254 5249\n",
      "27\t85319\t0.2\t1\t79062 6257\n",
      "28\t83102\t0.2\t1\t78541 4561\n",
      "29\t88168\t0.2\t1\t82573 5595\n",
      "30\t89538\t0.2\t1\t84298 5240\n",
      "31\t96658\t0.2\t1\t90668 5990\n",
      "32\t88293\t0.2\t1\t83312 4981\n",
      "33\t97915\t0.2\t1\t91746 6169\n",
      "34\t124838\t0.2\t1\t117069 7769\n",
      "35\t74005\t0.2\t1\t69745 4260\n",
      "36\t101096\t0.2\t1\t94692 6404\n",
      "37\t106262\t0.2\t1\t99713 6549\n",
      "38\t101787\t0.2\t1\t96374 5413\n",
      "39\t98728\t0.2\t1\t93495 5233\n",
      "40\t111334\t0.2\t1\t104562 6772\n",
      "41\t108250\t0.2\t1\t102276 5974\n",
      "42\t114250\t0.2\t1\t107889 6361\n",
      "43\t101869\t0.2\t1\t96202 5667\n",
      "44\t119468\t0.2\t1\t112651 6817\n",
      "45\t134184\t0.2\t1\t126701 7483\n",
      "46\t88069\t0.2\t1\t83261 4808\n",
      "47\t117053\t0.2\t1\t110868 6185\n",
      "48\t117366\t0.2\t1\t111327 6039\n",
      "49\t124438\t0.2\t1\t117817 6621\n",
      "50\t117444\t0.2\t1\t111059 6385\n",
      "51\t154247\t0.2\t1\t146044 8203\n",
      "52\t106291\t0.2\t1\t100997 5294\n",
      "53\t129680\t0.2\t1\t123785 5895\n",
      "54\t97816\t0.2\t1\t92887 4929\n",
      "55\t130824\t0.2\t1\t124577 6247\n",
      "56\t135217\t0.2\t1\t128490 6727\n",
      "57\t127931\t0.2\t1\t122024 5907\n",
      "58\t124239\t0.2\t1\t118784 5455\n",
      "59\t127293\t0.2\t1\t121311 5982\n",
      "60\t131981\t0.2\t1\t125772 6209\n",
      "61\t137697\t0.2\t1\t130983 6714\n",
      "62\t152253\t0.2\t1\t144774 7479\n",
      "63\t201898\t0.2\t1\t193113 8785\n",
      "64\t172675\t0.2\t1\t166389 6286\n",
      "65\t108874\t0.2\t1\t104021 4853\n",
      "66\t119610\t0.2\t1\t114446 5164\n",
      "67\t118964\t0.2\t1\t113321 5643\n",
      "68\t119680\t0.2\t1\t114293 5387\n",
      "69\t119889\t0.2\t1\t114293 5596\n",
      "70\t125777\t0.2\t1\t120013 5764\n",
      "71\t132876\t0.2\t1\t127109 5767\n",
      "72\t127787\t0.2\t1\t121904 5883\n",
      "73\t134940\t0.2\t1\t129055 5885\n",
      "74\t125685\t0.2\t1\t120224 5461\n",
      "75\t120939\t0.2\t1\t115502 5437\n",
      "76\t116878\t0.2\t1\t111806 5072\n",
      "77\t119317\t0.2\t1\t114031 5286\n",
      "78\t114912\t0.2\t1\t109837 5075\n",
      "79\t111961\t0.2\t1\t107098 4863\n",
      "80\t111439\t0.2\t1\t106611 4828\n",
      "81\t110252\t0.2\t1\t105525 4727\n",
      "82\t104612\t0.2\t1\t100154 4458\n",
      "83\t103450\t0.2\t1\t99126 4324\n",
      "84\t97944\t0.2\t1\t93515 4429\n",
      "85\t99039\t0.2\t1\t94552 4487\n",
      "86\t98012\t0.2\t1\t93716 4296\n",
      "87\t99549\t0.2\t1\t95300 4249\n",
      "88\t97562\t0.2\t1\t93249 4313\n",
      "89\t87175\t0.2\t1\t83280 3895\n",
      "90\t83950\t0.2\t1\t80287 3663\n",
      "91\t78836\t0.2\t1\t75281 3555\n",
      "92\t74886\t0.2\t1\t71664 3222\n",
      "93\t72687\t0.2\t1\t69632 3055\n",
      "94\t70208\t0.2\t1\t67258 2950\n",
      "95\t66190\t0.2\t1\t63387 2803\n",
      "96\t63977\t0.2\t1\t61304 2673\n",
      "97\t60139\t0.2\t1\t57625 2514\n",
      "98\t58153\t0.2\t1\t55728 2425\n",
      "99\t56999\t0.2\t1\t54663 2336\n",
      "100\t49226\t0.2\t1\t47327 1899\n",
      "101\t45428\t0.2\t1\t43567 1861\n",
      "102\t41140\t0.2\t1\t39441 1699\n",
      "103\t38223\t0.2\t1\t36634 1589\n",
      "104\t36626\t0.2\t1\t35088 1538\n",
      "105\t32982\t0.2\t1\t31676 1306\n",
      "106\t30175\t0.2\t1\t28923 1252\n",
      "107\t27378\t0.2\t1\t26210 1168\n",
      "108\t24859\t0.2\t1\t23806 1053\n",
      "109\t22755\t0.2\t1\t21778 977\n",
      "110\t20637\t0.2\t1\t19770 867\n",
      "111\t19236\t0.2\t1\t18439 797\n",
      "112\t16601\t0.2\t1\t15906 695\n",
      "113\t14600\t0.2\t1\t13964 636\n",
      "114\t12946\t0.2\t1\t12440 506\n",
      "115\t10705\t0.2\t1\t10194 511\n",
      "116\t9020\t0.2\t1\t8602 418\n",
      "117\t7442\t0.2\t1\t7107 335\n",
      "118\t6690\t0.2\t1\t6379 311\n",
      "119\t5516\t0.2\t1\t5293 223\n",
      "120\t4632\t0.2\t1\t4427 205\n",
      "121\t4146\t0.2\t1\t3939 207\n",
      "122\t3352\t0.2\t1\t3190 162\n",
      "123\t2944\t0.2\t1\t2813 131\n",
      "124\t2397\t0.2\t1\t2279 118\n",
      "125\t1984\t0.2\t1\t1898 86\n",
      "126\t1679\t0.2\t1\t1593 86\n",
      "127\t1365\t0.2\t1\t1295 70\n",
      "128\t964\t0.2\t1\t919 45\n",
      "129\t925\t0.2\t1\t886 39\n",
      "130\t667\t0.2\t1\t644 23\n",
      "131\t533\t0.2\t1\t504 29\n",
      "132\t511\t0.2\t1\t485 26\n",
      "133\t492\t0.2\t1\t476 16\n",
      "134\t547\t0.2\t1\t526 21\n",
      "135\t276\t0.2\t1\t266 10\n",
      "136\t213\t0.2\t1\t198 15\n",
      "137\t138\t0.2\t1\t130 8\n",
      "138\t104\t0.2\t1\t100 4\n",
      "139\t66\t0.2\t1\t61 5\n",
      "140\t52\t0.2\t1\t48 4\n",
      "141\t37\t0.2\t1\t34 3\n",
      "142\t40\t0.2\t1\t40\n",
      "143\t44\t0.2\t1\t40 4\n",
      "144\t45\t0.2\t1\t42 3\n",
      "145\t50\t0.2\t1\t48 2\n",
      "146\t86\t0.2\t1\t82 4\n",
      "147\t237\t0.2\t1\t232 5\n",
      "148\t198\t0.2\t1\t192 6\n",
      "149\t62\t0.2\t1\t55 7\n",
      "150\t658\t0.2\t1\t640 18\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13688650 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz, file_2: 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13688650\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 33910 (0.25%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_3_S3_L001_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_3_S3_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 475.53 s (35 us/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,670,957\n",
      "Reads with adapters:                11,663,120 (85.3%)\n",
      "Reads written (passing filters):    13,670,957 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,050,643,550 bp\n",
      "Quality-trimmed:               4,231,959 bp (0.2%)\n",
      "Total written (filtered):  1,462,745,020 bp (71.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11663120 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.2%\n",
      "  C: 13.5%\n",
      "  G: 21.8%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1152864\t3417739.2\t0\t1152864\n",
      "2\t186575\t854434.8\t0\t186575\n",
      "3\t110892\t213608.7\t0\t110892\n",
      "4\t68686\t53402.2\t0\t68686\n",
      "5\t61523\t13350.5\t0\t61523\n",
      "6\t61066\t3337.6\t0\t61066\n",
      "7\t49563\t834.4\t0\t49563\n",
      "8\t63725\t208.6\t0\t63725\n",
      "9\t62710\t52.2\t0\t61945 765\n",
      "10\t59136\t13.0\t1\t55273 3863\n",
      "11\t67741\t3.3\t1\t61750 5991\n",
      "12\t63923\t0.8\t1\t59045 4878\n",
      "13\t66568\t0.2\t1\t61427 5141\n",
      "14\t66808\t0.2\t1\t61285 5523\n",
      "15\t71031\t0.2\t1\t65247 5784\n",
      "16\t70289\t0.2\t1\t63984 6305\n",
      "17\t86030\t0.2\t1\t77755 8275\n",
      "18\t73739\t0.2\t1\t68290 5449\n",
      "19\t63306\t0.2\t1\t58569 4737\n",
      "20\t77355\t0.2\t1\t71402 5953\n",
      "21\t85573\t0.2\t1\t78654 6919\n",
      "22\t72710\t0.2\t1\t67584 5126\n",
      "23\t86414\t0.2\t1\t79515 6899\n",
      "24\t79027\t0.2\t1\t71591 7436\n",
      "25\t86879\t0.2\t1\t79866 7013\n",
      "26\t79473\t0.2\t1\t73820 5653\n",
      "27\t90733\t0.2\t1\t83035 7698\n",
      "28\t90102\t0.2\t1\t83123 6979\n",
      "29\t92865\t0.2\t1\t84237 8628\n",
      "30\t100457\t0.2\t1\t93462 6995\n",
      "31\t79802\t0.2\t1\t73917 5885\n",
      "32\t97719\t0.2\t1\t90137 7582\n",
      "33\t93085\t0.2\t1\t85177 7908\n",
      "34\t99262\t0.2\t1\t90786 8476\n",
      "35\t104947\t0.2\t1\t96795 8152\n",
      "36\t99380\t0.2\t1\t91895 7485\n",
      "37\t106441\t0.2\t1\t96879 9562\n",
      "38\t114379\t0.2\t1\t105498 8881\n",
      "39\t96631\t0.2\t1\t89100 7531\n",
      "40\t112490\t0.2\t1\t103151 9339\n",
      "41\t123125\t0.2\t1\t112471 10654\n",
      "42\t98677\t0.2\t1\t90013 8664\n",
      "43\t172897\t0.2\t1\t162100 10797\n",
      "44\t52704\t0.2\t1\t47226 5478\n",
      "45\t117558\t0.2\t1\t108914 8644\n",
      "46\t110418\t0.2\t1\t102198 8220\n",
      "47\t119359\t0.2\t1\t110565 8794\n",
      "48\t116841\t0.2\t1\t108072 8769\n",
      "49\t129611\t0.2\t1\t119238 10373\n",
      "50\t111895\t0.2\t1\t104242 7653\n",
      "51\t123193\t0.2\t1\t114083 9110\n",
      "52\t127566\t0.2\t1\t117108 10458\n",
      "53\t137342\t0.2\t1\t127459 9883\n",
      "54\t110167\t0.2\t1\t102021 8146\n",
      "55\t130673\t0.2\t1\t120967 9706\n",
      "56\t160643\t0.2\t1\t149587 11056\n",
      "57\t104913\t0.2\t1\t96311 8602\n",
      "58\t136727\t0.2\t1\t128029 8698\n",
      "59\t118611\t0.2\t1\t109706 8905\n",
      "60\t133859\t0.2\t1\t123864 9995\n",
      "61\t131656\t0.2\t1\t122381 9275\n",
      "62\t147218\t0.2\t1\t137094 10124\n",
      "63\t117137\t0.2\t1\t107812 9325\n",
      "64\t130610\t0.2\t1\t121719 8891\n",
      "65\t145884\t0.2\t1\t135248 10636\n",
      "66\t150729\t0.2\t1\t139945 10784\n",
      "67\t163505\t0.2\t1\t149584 13921\n",
      "68\t306709\t0.2\t1\t289681 17028\n",
      "69\t62043\t0.2\t1\t56562 5481\n",
      "70\t23965\t0.2\t1\t19784 4181\n",
      "71\t84291\t0.2\t1\t78568 5723\n",
      "72\t119692\t0.2\t1\t111910 7782\n",
      "73\t127200\t0.2\t1\t118724 8476\n",
      "74\t126943\t0.2\t1\t118057 8886\n",
      "75\t127010\t0.2\t1\t118252 8758\n",
      "76\t125373\t0.2\t1\t116194 9179\n",
      "77\t125254\t0.2\t1\t116388 8866\n",
      "78\t119533\t0.2\t1\t110788 8745\n",
      "79\t118816\t0.2\t1\t110233 8583\n",
      "80\t116241\t0.2\t1\t107681 8560\n",
      "81\t114798\t0.2\t1\t106595 8203\n",
      "82\t112468\t0.2\t1\t104279 8189\n",
      "83\t109667\t0.2\t1\t101888 7779\n",
      "84\t105890\t0.2\t1\t98240 7650\n",
      "85\t103302\t0.2\t1\t95804 7498\n",
      "86\t101064\t0.2\t1\t93722 7342\n",
      "87\t97500\t0.2\t1\t90687 6813\n",
      "88\t96898\t0.2\t1\t89881 7017\n",
      "89\t91450\t0.2\t1\t84968 6482\n",
      "90\t91495\t0.2\t1\t84920 6575\n",
      "91\t86388\t0.2\t1\t80232 6156\n",
      "92\t82291\t0.2\t1\t76556 5735\n",
      "93\t78431\t0.2\t1\t72788 5643\n",
      "94\t75874\t0.2\t1\t70339 5535\n",
      "95\t70751\t0.2\t1\t65675 5076\n",
      "96\t67352\t0.2\t1\t62368 4984\n",
      "97\t63954\t0.2\t1\t59443 4511\n",
      "98\t58850\t0.2\t1\t54439 4411\n",
      "99\t58771\t0.2\t1\t54248 4523\n",
      "100\t52467\t0.2\t1\t48655 3812\n",
      "101\t49569\t0.2\t1\t45811 3758\n",
      "102\t45464\t0.2\t1\t42074 3390\n",
      "103\t41939\t0.2\t1\t38904 3035\n",
      "104\t39055\t0.2\t1\t36189 2866\n",
      "105\t34942\t0.2\t1\t32296 2646\n",
      "106\t31969\t0.2\t1\t29559 2410\n",
      "107\t29286\t0.2\t1\t27065 2221\n",
      "108\t27282\t0.2\t1\t25136 2146\n",
      "109\t24507\t0.2\t1\t22544 1963\n",
      "110\t21811\t0.2\t1\t20137 1674\n",
      "111\t19775\t0.2\t1\t18232 1543\n",
      "112\t17157\t0.2\t1\t15740 1417\n",
      "113\t14815\t0.2\t1\t13543 1272\n",
      "114\t13327\t0.2\t1\t12234 1093\n",
      "115\t11366\t0.2\t1\t10408 958\n",
      "116\t9845\t0.2\t1\t9004 841\n",
      "117\t8405\t0.2\t1\t7663 742\n",
      "118\t7384\t0.2\t1\t6703 681\n",
      "119\t6035\t0.2\t1\t5477 558\n",
      "120\t4910\t0.2\t1\t4438 472\n",
      "121\t4090\t0.2\t1\t3702 388\n",
      "122\t3634\t0.2\t1\t3283 351\n",
      "123\t3044\t0.2\t1\t2741 303\n",
      "124\t2552\t0.2\t1\t2281 271\n",
      "125\t2030\t0.2\t1\t1827 203\n",
      "126\t1757\t0.2\t1\t1559 198\n",
      "127\t1362\t0.2\t1\t1204 158\n",
      "128\t1097\t0.2\t1\t981 116\n",
      "129\t1002\t0.2\t1\t874 128\n",
      "130\t785\t0.2\t1\t685 100\n",
      "131\t638\t0.2\t1\t562 76\n",
      "132\t540\t0.2\t1\t480 60\n",
      "133\t462\t0.2\t1\t415 47\n",
      "134\t556\t0.2\t1\t494 62\n",
      "135\t352\t0.2\t1\t314 38\n",
      "136\t204\t0.2\t1\t176 28\n",
      "137\t149\t0.2\t1\t133 16\n",
      "138\t114\t0.2\t1\t99 15\n",
      "139\t71\t0.2\t1\t62 9\n",
      "140\t64\t0.2\t1\t52 12\n",
      "141\t32\t0.2\t1\t25 7\n",
      "142\t45\t0.2\t1\t42 3\n",
      "143\t51\t0.2\t1\t43 8\n",
      "144\t31\t0.2\t1\t25 6\n",
      "145\t45\t0.2\t1\t31 14\n",
      "146\t90\t0.2\t1\t77 13\n",
      "147\t226\t0.2\t1\t201 25\n",
      "148\t203\t0.2\t1\t185 18\n",
      "149\t84\t0.2\t1\t77 7\n",
      "150\t819\t0.2\t1\t651 168\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13670957 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_3_S3_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 511.55 s (37 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,670,957\n",
      "Reads with adapters:                11,550,197 (84.5%)\n",
      "Reads written (passing filters):    13,670,957 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,050,643,550 bp\n",
      "Quality-trimmed:              11,580,867 bp (0.6%)\n",
      "Total written (filtered):  1,468,475,211 bp (71.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11550197 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.3%\n",
      "  G: 24.9%\n",
      "  T: 34.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1212421\t3417739.2\t0\t1212421\n",
      "2\t191566\t854434.8\t0\t191566\n",
      "3\t121338\t213608.7\t0\t121338\n",
      "4\t68246\t53402.2\t0\t68246\n",
      "5\t66256\t13350.5\t0\t66256\n",
      "6\t60146\t3337.6\t0\t60146\n",
      "7\t46826\t834.4\t0\t46826\n",
      "8\t69191\t208.6\t0\t69191\n",
      "9\t54203\t52.2\t0\t54035 168\n",
      "10\t59001\t13.0\t1\t54514 4487\n",
      "11\t64399\t3.3\t1\t58784 5615\n",
      "12\t63560\t0.8\t1\t57831 5729\n",
      "13\t73714\t0.2\t1\t67412 6302\n",
      "14\t69969\t0.2\t1\t63352 6617\n",
      "15\t54986\t0.2\t1\t50399 4587\n",
      "16\t83002\t0.2\t1\t75869 7133\n",
      "17\t70639\t0.2\t1\t64421 6218\n",
      "18\t61266\t0.2\t1\t56207 5059\n",
      "19\t82751\t0.2\t1\t75847 6904\n",
      "20\t76799\t0.2\t1\t70584 6215\n",
      "21\t67039\t0.2\t1\t61694 5345\n",
      "22\t77648\t0.2\t1\t71294 6354\n",
      "23\t82616\t0.2\t1\t75896 6720\n",
      "24\t82091\t0.2\t1\t75051 7040\n",
      "25\t94761\t0.2\t1\t87866 6895\n",
      "26\t78379\t0.2\t1\t71576 6803\n",
      "27\t86685\t0.2\t1\t79069 7616\n",
      "28\t79766\t0.2\t1\t74088 5678\n",
      "29\t88958\t0.2\t1\t82020 6938\n",
      "30\t87347\t0.2\t1\t80721 6626\n",
      "31\t96436\t0.2\t1\t89116 7320\n",
      "32\t84184\t0.2\t1\t78013 6171\n",
      "33\t99064\t0.2\t1\t91212 7852\n",
      "34\t126286\t0.2\t1\t116784 9502\n",
      "35\t70478\t0.2\t1\t65263 5215\n",
      "36\t102056\t0.2\t1\t93957 8099\n",
      "37\t106427\t0.2\t1\t98476 7951\n",
      "38\t101342\t0.2\t1\t94365 6977\n",
      "39\t95863\t0.2\t1\t89455 6408\n",
      "40\t111287\t0.2\t1\t103069 8218\n",
      "41\t109099\t0.2\t1\t101498 7601\n",
      "42\t114117\t0.2\t1\t106148 7969\n",
      "43\t100632\t0.2\t1\t93666 6966\n",
      "44\t121370\t0.2\t1\t112650 8720\n",
      "45\t132112\t0.2\t1\t122836 9276\n",
      "46\t87481\t0.2\t1\t81280 6201\n",
      "47\t117137\t0.2\t1\t109733 7404\n",
      "48\t116770\t0.2\t1\t109144 7626\n",
      "49\t122930\t0.2\t1\t114503 8427\n",
      "50\t118230\t0.2\t1\t110078 8152\n",
      "51\t151502\t0.2\t1\t141327 10175\n",
      "52\t106392\t0.2\t1\t99321 7071\n",
      "53\t132940\t0.2\t1\t125179 7761\n",
      "54\t95060\t0.2\t1\t89153 5907\n",
      "55\t129755\t0.2\t1\t121823 7932\n",
      "56\t133247\t0.2\t1\t124572 8675\n",
      "57\t128656\t0.2\t1\t121414 7242\n",
      "58\t125177\t0.2\t1\t117960 7217\n",
      "59\t125701\t0.2\t1\t118073 7628\n",
      "60\t133212\t0.2\t1\t125145 8067\n",
      "61\t136943\t0.2\t1\t128912 8031\n",
      "62\t152161\t0.2\t1\t142895 9266\n",
      "63\t197608\t0.2\t1\t187217 10391\n",
      "64\t170164\t0.2\t1\t162758 7406\n",
      "65\t108925\t0.2\t1\t102718 6207\n",
      "66\t119060\t0.2\t1\t112190 6870\n",
      "67\t118638\t0.2\t1\t111538 7100\n",
      "68\t122066\t0.2\t1\t114957 7109\n",
      "69\t121361\t0.2\t1\t113998 7363\n",
      "70\t127330\t0.2\t1\t119753 7577\n",
      "71\t132672\t0.2\t1\t124891 7781\n",
      "72\t129459\t0.2\t1\t121850 7609\n",
      "73\t134714\t0.2\t1\t127048 7666\n",
      "74\t126025\t0.2\t1\t118830 7195\n",
      "75\t121497\t0.2\t1\t114268 7229\n",
      "76\t117925\t0.2\t1\t110905 7020\n",
      "77\t118215\t0.2\t1\t111150 7065\n",
      "78\t114702\t0.2\t1\t108059 6643\n",
      "79\t111977\t0.2\t1\t105512 6465\n",
      "80\t111238\t0.2\t1\t104912 6326\n",
      "81\t110074\t0.2\t1\t103870 6204\n",
      "82\t104747\t0.2\t1\t98712 6035\n",
      "83\t102994\t0.2\t1\t97190 5804\n",
      "84\t97939\t0.2\t1\t92413 5526\n",
      "85\t98027\t0.2\t1\t92612 5415\n",
      "86\t97975\t0.2\t1\t92622 5353\n",
      "87\t99113\t0.2\t1\t93888 5225\n",
      "88\t97377\t0.2\t1\t92204 5173\n",
      "89\t87121\t0.2\t1\t82286 4835\n",
      "90\t84514\t0.2\t1\t79963 4551\n",
      "91\t79415\t0.2\t1\t75190 4225\n",
      "92\t74774\t0.2\t1\t70849 3925\n",
      "93\t72752\t0.2\t1\t68819 3933\n",
      "94\t70292\t0.2\t1\t66493 3799\n",
      "95\t66513\t0.2\t1\t62910 3603\n",
      "96\t64556\t0.2\t1\t61082 3474\n",
      "97\t60915\t0.2\t1\t57800 3115\n",
      "98\t57473\t0.2\t1\t54587 2886\n",
      "99\t57713\t0.2\t1\t54715 2998\n",
      "100\t49481\t0.2\t1\t46774 2707\n",
      "101\t45740\t0.2\t1\t43384 2356\n",
      "102\t41586\t0.2\t1\t39421 2165\n",
      "103\t38173\t0.2\t1\t36099 2074\n",
      "104\t36449\t0.2\t1\t34460 1989\n",
      "105\t32790\t0.2\t1\t31067 1723\n",
      "106\t30381\t0.2\t1\t28796 1585\n",
      "107\t27663\t0.2\t1\t26191 1472\n",
      "108\t25091\t0.2\t1\t23805 1286\n",
      "109\t22776\t0.2\t1\t21667 1109\n",
      "110\t20493\t0.2\t1\t19406 1087\n",
      "111\t19435\t0.2\t1\t18473 962\n",
      "112\t16596\t0.2\t1\t15727 869\n",
      "113\t14532\t0.2\t1\t13752 780\n",
      "114\t12824\t0.2\t1\t12122 702\n",
      "115\t10619\t0.2\t1\t10098 521\n",
      "116\t9188\t0.2\t1\t8686 502\n",
      "117\t7612\t0.2\t1\t7172 440\n",
      "118\t6805\t0.2\t1\t6447 358\n",
      "119\t5613\t0.2\t1\t5287 326\n",
      "120\t4455\t0.2\t1\t4215 240\n",
      "121\t3795\t0.2\t1\t3594 201\n",
      "122\t3385\t0.2\t1\t3227 158\n",
      "123\t2919\t0.2\t1\t2743 176\n",
      "124\t2401\t0.2\t1\t2252 149\n",
      "125\t1949\t0.2\t1\t1858 91\n",
      "126\t1662\t0.2\t1\t1556 106\n",
      "127\t1275\t0.2\t1\t1189 86\n",
      "128\t1017\t0.2\t1\t960 57\n",
      "129\t911\t0.2\t1\t861 50\n",
      "130\t712\t0.2\t1\t666 46\n",
      "131\t583\t0.2\t1\t553 30\n",
      "132\t498\t0.2\t1\t473 25\n",
      "133\t459\t0.2\t1\t441 18\n",
      "134\t540\t0.2\t1\t495 45\n",
      "135\t343\t0.2\t1\t318 25\n",
      "136\t185\t0.2\t1\t166 19\n",
      "137\t142\t0.2\t1\t134 8\n",
      "138\t102\t0.2\t1\t96 6\n",
      "139\t72\t0.2\t1\t71 1\n",
      "140\t56\t0.2\t1\t51 5\n",
      "141\t30\t0.2\t1\t27 3\n",
      "142\t45\t0.2\t1\t39 6\n",
      "143\t48\t0.2\t1\t43 5\n",
      "144\t33\t0.2\t1\t32 1\n",
      "145\t35\t0.2\t1\t32 3\n",
      "146\t81\t0.2\t1\t79 2\n",
      "147\t221\t0.2\t1\t208 13\n",
      "148\t193\t0.2\t1\t178 15\n",
      "149\t78\t0.2\t1\t71 7\n",
      "150\t649\t0.2\t1\t611 38\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13670957 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz, file_2: 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13670957\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 32418 (0.24%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_3_S3_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_3_S3_L002_R1_001_trimmed.fq.gz and 16C_32psu_3_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_4_S4_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 627.90 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,706,000\n",
      "Reads with adapters:                11,318,033 (72.1%)\n",
      "Reads written (passing filters):    15,706,000 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,355,900,000 bp\n",
      "Quality-trimmed:               7,618,669 bp (0.3%)\n",
      "Total written (filtered):  1,966,841,132 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11318033 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 13.1%\n",
      "  G: 19.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2758722\t3926500.0\t0\t2758722\n",
      "2\t358574\t981625.0\t0\t358574\n",
      "3\t190965\t245406.2\t0\t190965\n",
      "4\t113114\t61351.6\t0\t113114\n",
      "5\t86888\t15337.9\t0\t86888\n",
      "6\t85123\t3834.5\t0\t85123\n",
      "7\t70336\t958.6\t0\t70336\n",
      "8\t85928\t239.7\t0\t85928\n",
      "9\t87066\t59.9\t0\t86195 871\n",
      "10\t79602\t15.0\t1\t75862 3740\n",
      "11\t88089\t3.7\t1\t81938 6151\n",
      "12\t85321\t0.9\t1\t80325 4996\n",
      "13\t84763\t0.2\t1\t79667 5096\n",
      "14\t88305\t0.2\t1\t82326 5979\n",
      "15\t88981\t0.2\t1\t83685 5296\n",
      "16\t88786\t0.2\t1\t82621 6165\n",
      "17\t106554\t0.2\t1\t98505 8049\n",
      "18\t89122\t0.2\t1\t84209 4913\n",
      "19\t77746\t0.2\t1\t73215 4531\n",
      "20\t92038\t0.2\t1\t86658 5380\n",
      "21\t99609\t0.2\t1\t93006 6603\n",
      "22\t85867\t0.2\t1\t81408 4459\n",
      "23\t100095\t0.2\t1\t93955 6140\n",
      "24\t90779\t0.2\t1\t83874 6905\n",
      "25\t97845\t0.2\t1\t91593 6252\n",
      "26\t88800\t0.2\t1\t83917 4883\n",
      "27\t97906\t0.2\t1\t90971 6935\n",
      "28\t105510\t0.2\t1\t98167 7343\n",
      "29\t95263\t0.2\t1\t88170 7093\n",
      "30\t108904\t0.2\t1\t103370 5534\n",
      "31\t79085\t0.2\t1\t74523 4562\n",
      "32\t103105\t0.2\t1\t96799 6306\n",
      "33\t92937\t0.2\t1\t86945 5992\n",
      "34\t100117\t0.2\t1\t93522 6595\n",
      "35\t106537\t0.2\t1\t100062 6475\n",
      "36\t91203\t0.2\t1\t86381 4822\n",
      "37\t99865\t0.2\t1\t93587 6278\n",
      "38\t105531\t0.2\t1\t98508 7023\n",
      "39\t105906\t0.2\t1\t99469 6437\n",
      "40\t104183\t0.2\t1\t97213 6970\n",
      "41\t116528\t0.2\t1\t106559 9969\n",
      "42\t93606\t0.2\t1\t87149 6457\n",
      "43\t153884\t0.2\t1\t145422 8462\n",
      "44\t48528\t0.2\t1\t44473 4055\n",
      "45\t107110\t0.2\t1\t99279 7831\n",
      "46\t172499\t0.2\t1\t161736 10763\n",
      "47\t116179\t0.2\t1\t109492 6687\n",
      "48\t80709\t0.2\t1\t73983 6726\n",
      "49\t159482\t0.2\t1\t150516 8966\n",
      "50\t64228\t0.2\t1\t59915 4313\n",
      "51\t90888\t0.2\t1\t84987 5901\n",
      "52\t193279\t0.2\t1\t183371 9908\n",
      "53\t141743\t0.2\t1\t135721 6022\n",
      "54\t59956\t0.2\t1\t56077 3879\n",
      "55\t102925\t0.2\t1\t98274 4651\n",
      "56\t66784\t0.2\t1\t62498 4286\n",
      "57\t123279\t0.2\t1\t118108 5171\n",
      "58\t43777\t0.2\t1\t41446 2331\n",
      "59\t39827\t0.2\t1\t36676 3151\n",
      "60\t113430\t0.2\t1\t108052 5378\n",
      "61\t89200\t0.2\t1\t85501 3699\n",
      "62\t44869\t0.2\t1\t41907 2962\n",
      "63\t85974\t0.2\t1\t81524 4450\n",
      "64\t74880\t0.2\t1\t71341 3539\n",
      "65\t59793\t0.2\t1\t56240 3553\n",
      "66\t87755\t0.2\t1\t82874 4881\n",
      "67\t100481\t0.2\t1\t94117 6364\n",
      "68\t167488\t0.2\t1\t159452 8036\n",
      "69\t79463\t0.2\t1\t74811 4652\n",
      "70\t27878\t0.2\t1\t25310 2568\n",
      "71\t23051\t0.2\t1\t21160 1891\n",
      "72\t60528\t0.2\t1\t57268 3260\n",
      "73\t72149\t0.2\t1\t68285 3864\n",
      "74\t73360\t0.2\t1\t69333 4027\n",
      "75\t71612\t0.2\t1\t67692 3920\n",
      "76\t70649\t0.2\t1\t66797 3852\n",
      "77\t70150\t0.2\t1\t66254 3896\n",
      "78\t65804\t0.2\t1\t62003 3801\n",
      "79\t64610\t0.2\t1\t60969 3641\n",
      "80\t63497\t0.2\t1\t60035 3462\n",
      "81\t61317\t0.2\t1\t58042 3275\n",
      "82\t58860\t0.2\t1\t55659 3201\n",
      "83\t57047\t0.2\t1\t53828 3219\n",
      "84\t54423\t0.2\t1\t51371 3052\n",
      "85\t52481\t0.2\t1\t49574 2907\n",
      "86\t50571\t0.2\t1\t47688 2883\n",
      "87\t48288\t0.2\t1\t45567 2721\n",
      "88\t45981\t0.2\t1\t43442 2539\n",
      "89\t43761\t0.2\t1\t41217 2544\n",
      "90\t42661\t0.2\t1\t40245 2416\n",
      "91\t40737\t0.2\t1\t38345 2392\n",
      "92\t38688\t0.2\t1\t36453 2235\n",
      "93\t34947\t0.2\t1\t32894 2053\n",
      "94\t34299\t0.2\t1\t32291 2008\n",
      "95\t31760\t0.2\t1\t29854 1906\n",
      "96\t29525\t0.2\t1\t27743 1782\n",
      "97\t27419\t0.2\t1\t25829 1590\n",
      "98\t24938\t0.2\t1\t23465 1473\n",
      "99\t24209\t0.2\t1\t22665 1544\n",
      "100\t21792\t0.2\t1\t20438 1354\n",
      "101\t20393\t0.2\t1\t19119 1274\n",
      "102\t18597\t0.2\t1\t17433 1164\n",
      "103\t16893\t0.2\t1\t15821 1072\n",
      "104\t15643\t0.2\t1\t14642 1001\n",
      "105\t14032\t0.2\t1\t13104 928\n",
      "106\t12725\t0.2\t1\t11896 829\n",
      "107\t11164\t0.2\t1\t10388 776\n",
      "108\t10327\t0.2\t1\t9611 716\n",
      "109\t9052\t0.2\t1\t8399 653\n",
      "110\t8194\t0.2\t1\t7576 618\n",
      "111\t7205\t0.2\t1\t6690 515\n",
      "112\t6374\t0.2\t1\t5898 476\n",
      "113\t5432\t0.2\t1\t5054 378\n",
      "114\t4742\t0.2\t1\t4349 393\n",
      "115\t4031\t0.2\t1\t3736 295\n",
      "116\t3440\t0.2\t1\t3126 314\n",
      "117\t3047\t0.2\t1\t2793 254\n",
      "118\t2620\t0.2\t1\t2385 235\n",
      "119\t2131\t0.2\t1\t1953 178\n",
      "120\t1732\t0.2\t1\t1592 140\n",
      "121\t1562\t0.2\t1\t1390 172\n",
      "122\t1210\t0.2\t1\t1094 116\n",
      "123\t1050\t0.2\t1\t947 103\n",
      "124\t965\t0.2\t1\t849 116\n",
      "125\t722\t0.2\t1\t635 87\n",
      "126\t663\t0.2\t1\t591 72\n",
      "127\t539\t0.2\t1\t460 79\n",
      "128\t435\t0.2\t1\t362 73\n",
      "129\t406\t0.2\t1\t358 48\n",
      "130\t274\t0.2\t1\t240 34\n",
      "131\t291\t0.2\t1\t261 30\n",
      "132\t213\t0.2\t1\t190 23\n",
      "133\t259\t0.2\t1\t233 26\n",
      "134\t361\t0.2\t1\t342 19\n",
      "135\t151\t0.2\t1\t128 23\n",
      "136\t85\t0.2\t1\t68 17\n",
      "137\t53\t0.2\t1\t45 8\n",
      "138\t43\t0.2\t1\t37 6\n",
      "139\t17\t0.2\t1\t15 2\n",
      "140\t21\t0.2\t1\t18 3\n",
      "141\t17\t0.2\t1\t15 2\n",
      "142\t9\t0.2\t1\t8 1\n",
      "143\t9\t0.2\t1\t7 2\n",
      "144\t15\t0.2\t1\t10 5\n",
      "145\t12\t0.2\t1\t10 2\n",
      "146\t12\t0.2\t1\t10 2\n",
      "147\t49\t0.2\t1\t45 4\n",
      "148\t35\t0.2\t1\t30 5\n",
      "149\t12\t0.2\t1\t11 1\n",
      "150\t163\t0.2\t1\t152 11\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "15706000 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_4_S4_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 666.33 s (42 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,706,000\n",
      "Reads with adapters:                11,248,615 (71.6%)\n",
      "Reads written (passing filters):    15,706,000 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,355,900,000 bp\n",
      "Quality-trimmed:               9,159,976 bp (0.4%)\n",
      "Total written (filtered):  1,968,243,578 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11248615 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 12.2%\n",
      "  G: 21.7%\n",
      "  T: 36.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2742334\t3926500.0\t0\t2742334\n",
      "2\t376488\t981625.0\t0\t376488\n",
      "3\t202415\t245406.2\t0\t202415\n",
      "4\t115660\t61351.6\t0\t115660\n",
      "5\t96934\t15337.9\t0\t96934\n",
      "6\t82183\t3834.5\t0\t82183\n",
      "7\t66579\t958.6\t0\t66579\n",
      "8\t95001\t239.7\t0\t95001\n",
      "9\t75363\t59.9\t0\t75125 238\n",
      "10\t81209\t15.0\t1\t77351 3858\n",
      "11\t85598\t3.7\t1\t80376 5222\n",
      "12\t86515\t0.9\t1\t81221 5294\n",
      "13\t96527\t0.2\t1\t91069 5458\n",
      "14\t89686\t0.2\t1\t83554 6132\n",
      "15\t72806\t0.2\t1\t68412 4394\n",
      "16\t105343\t0.2\t1\t99018 6325\n",
      "17\t82599\t0.2\t1\t77199 5400\n",
      "18\t79503\t0.2\t1\t74873 4630\n",
      "19\t100299\t0.2\t1\t94044 6255\n",
      "20\t93401\t0.2\t1\t87924 5477\n",
      "21\t81559\t0.2\t1\t76639 4920\n",
      "22\t92216\t0.2\t1\t86910 5306\n",
      "23\t94734\t0.2\t1\t89009 5725\n",
      "24\t95149\t0.2\t1\t89373 5776\n",
      "25\t105782\t0.2\t1\t99776 6006\n",
      "26\t87770\t0.2\t1\t82449 5321\n",
      "27\t95338\t0.2\t1\t88893 6445\n",
      "28\t90298\t0.2\t1\t85892 4406\n",
      "29\t95418\t0.2\t1\t89790 5628\n",
      "30\t93873\t0.2\t1\t89134 4739\n",
      "31\t100963\t0.2\t1\t95205 5758\n",
      "32\t90423\t0.2\t1\t85945 4478\n",
      "33\t99633\t0.2\t1\t93789 5844\n",
      "34\t126170\t0.2\t1\t118899 7271\n",
      "35\t71917\t0.2\t1\t68415 3502\n",
      "36\t98752\t0.2\t1\t93147 5605\n",
      "37\t104121\t0.2\t1\t98203 5918\n",
      "38\t97206\t0.2\t1\t92554 4652\n",
      "39\t94225\t0.2\t1\t89765 4460\n",
      "40\t104186\t0.2\t1\t98424 5762\n",
      "41\t99925\t0.2\t1\t94861 5064\n",
      "42\t102480\t0.2\t1\t97343 5137\n",
      "43\t92586\t0.2\t1\t87878 4708\n",
      "44\t105897\t0.2\t1\t100306 5591\n",
      "45\t112588\t0.2\t1\t106743 5845\n",
      "46\t80544\t0.2\t1\t76510 4034\n",
      "47\t101141\t0.2\t1\t96374 4767\n",
      "48\t98066\t0.2\t1\t93549 4517\n",
      "49\t101791\t0.2\t1\t96776 5015\n",
      "50\t97034\t0.2\t1\t92147 4887\n",
      "51\t120282\t0.2\t1\t114381 5901\n",
      "52\t84235\t0.2\t1\t80401 3834\n",
      "53\t102171\t0.2\t1\t97747 4424\n",
      "54\t76841\t0.2\t1\t73430 3411\n",
      "55\t99264\t0.2\t1\t95032 4232\n",
      "56\t101316\t0.2\t1\t96570 4746\n",
      "57\t94879\t0.2\t1\t90678 4201\n",
      "58\t90444\t0.2\t1\t86700 3744\n",
      "59\t92322\t0.2\t1\t88235 4087\n",
      "60\t94688\t0.2\t1\t90370 4318\n",
      "61\t97150\t0.2\t1\t92685 4465\n",
      "62\t103718\t0.2\t1\t98963 4755\n",
      "63\t136885\t0.2\t1\t131195 5690\n",
      "64\t109397\t0.2\t1\t105405 3992\n",
      "65\t71767\t0.2\t1\t68765 3002\n",
      "66\t78028\t0.2\t1\t74793 3235\n",
      "67\t77090\t0.2\t1\t73810 3280\n",
      "68\t76844\t0.2\t1\t73617 3227\n",
      "69\t75212\t0.2\t1\t71969 3243\n",
      "70\t76863\t0.2\t1\t73510 3353\n",
      "71\t80211\t0.2\t1\t76869 3342\n",
      "72\t75920\t0.2\t1\t72620 3300\n",
      "73\t78093\t0.2\t1\t74948 3145\n",
      "74\t72443\t0.2\t1\t69570 2873\n",
      "75\t68538\t0.2\t1\t65762 2776\n",
      "76\t66619\t0.2\t1\t63921 2698\n",
      "77\t66151\t0.2\t1\t63438 2713\n",
      "78\t62817\t0.2\t1\t60225 2592\n",
      "79\t60312\t0.2\t1\t57778 2534\n",
      "80\t60163\t0.2\t1\t57704 2459\n",
      "81\t57937\t0.2\t1\t55639 2298\n",
      "82\t54845\t0.2\t1\t52637 2208\n",
      "83\t53627\t0.2\t1\t51399 2228\n",
      "84\t50792\t0.2\t1\t48633 2159\n",
      "85\t49633\t0.2\t1\t47523 2110\n",
      "86\t49264\t0.2\t1\t47185 2079\n",
      "87\t49251\t0.2\t1\t47314 1937\n",
      "88\t46501\t0.2\t1\t44723 1778\n",
      "89\t41655\t0.2\t1\t39945 1710\n",
      "90\t39420\t0.2\t1\t37741 1679\n",
      "91\t37399\t0.2\t1\t35859 1540\n",
      "92\t35077\t0.2\t1\t33634 1443\n",
      "93\t32838\t0.2\t1\t31397 1441\n",
      "94\t31873\t0.2\t1\t30582 1291\n",
      "95\t29856\t0.2\t1\t28661 1195\n",
      "96\t27831\t0.2\t1\t26723 1108\n",
      "97\t25609\t0.2\t1\t24584 1025\n",
      "98\t23598\t0.2\t1\t22615 983\n",
      "99\t23296\t0.2\t1\t22392 904\n",
      "100\t21689\t0.2\t1\t20900 789\n",
      "101\t19855\t0.2\t1\t19105 750\n",
      "102\t17867\t0.2\t1\t17193 674\n",
      "103\t15691\t0.2\t1\t15073 618\n",
      "104\t15000\t0.2\t1\t14394 606\n",
      "105\t13326\t0.2\t1\t12799 527\n",
      "106\t12111\t0.2\t1\t11648 463\n",
      "107\t10556\t0.2\t1\t10155 401\n",
      "108\t9460\t0.2\t1\t9085 375\n",
      "109\t8474\t0.2\t1\t8135 339\n",
      "110\t7655\t0.2\t1\t7315 340\n",
      "111\t7041\t0.2\t1\t6747 294\n",
      "112\t6163\t0.2\t1\t5902 261\n",
      "113\t5337\t0.2\t1\t5089 248\n",
      "114\t4565\t0.2\t1\t4367 198\n",
      "115\t3763\t0.2\t1\t3606 157\n",
      "116\t3164\t0.2\t1\t2993 171\n",
      "117\t2789\t0.2\t1\t2653 136\n",
      "118\t2380\t0.2\t1\t2277 103\n",
      "119\t1999\t0.2\t1\t1886 113\n",
      "120\t1562\t0.2\t1\t1470 92\n",
      "121\t1457\t0.2\t1\t1367 90\n",
      "122\t1103\t0.2\t1\t1044 59\n",
      "123\t1029\t0.2\t1\t972 57\n",
      "124\t896\t0.2\t1\t845 51\n",
      "125\t678\t0.2\t1\t637 41\n",
      "126\t611\t0.2\t1\t570 41\n",
      "127\t481\t0.2\t1\t451 30\n",
      "128\t387\t0.2\t1\t354 33\n",
      "129\t379\t0.2\t1\t356 23\n",
      "130\t249\t0.2\t1\t234 15\n",
      "131\t280\t0.2\t1\t263 17\n",
      "132\t196\t0.2\t1\t186 10\n",
      "133\t279\t0.2\t1\t264 15\n",
      "134\t375\t0.2\t1\t361 14\n",
      "135\t138\t0.2\t1\t127 11\n",
      "136\t76\t0.2\t1\t71 5\n",
      "137\t48\t0.2\t1\t43 5\n",
      "138\t41\t0.2\t1\t37 4\n",
      "139\t18\t0.2\t1\t17 1\n",
      "140\t14\t0.2\t1\t12 2\n",
      "141\t8\t0.2\t1\t8\n",
      "142\t12\t0.2\t1\t8 4\n",
      "143\t10\t0.2\t1\t6 4\n",
      "144\t13\t0.2\t1\t11 2\n",
      "145\t15\t0.2\t1\t13 2\n",
      "146\t12\t0.2\t1\t10 2\n",
      "147\t37\t0.2\t1\t35 2\n",
      "148\t25\t0.2\t1\t24 1\n",
      "149\t9\t0.2\t1\t6 3\n",
      "150\t99\t0.2\t1\t92 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "15706000 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz, file_2: 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 15706000\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 19399 (0.12%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_4_S4_L001_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_4_S4_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 634.36 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,591,852\n",
      "Reads with adapters:                11,241,936 (72.1%)\n",
      "Reads written (passing filters):    15,591,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,338,777,800 bp\n",
      "Quality-trimmed:               7,762,195 bp (0.3%)\n",
      "Total written (filtered):  1,950,418,217 bp (83.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11241936 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 13.2%\n",
      "  G: 19.5%\n",
      "  T: 35.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2728314\t3897963.0\t0\t2728314\n",
      "2\t352949\t974490.8\t0\t352949\n",
      "3\t189042\t243622.7\t0\t189042\n",
      "4\t110295\t60905.7\t0\t110295\n",
      "5\t86754\t15226.4\t0\t86754\n",
      "6\t84318\t3806.6\t0\t84318\n",
      "7\t67898\t951.7\t0\t67898\n",
      "8\t84774\t237.9\t0\t84774\n",
      "9\t86162\t59.5\t0\t85323 839\n",
      "10\t77737\t14.9\t1\t73221 4516\n",
      "11\t88459\t3.7\t1\t81353 7106\n",
      "12\t84258\t0.9\t1\t78502 5756\n",
      "13\t83829\t0.2\t1\t77646 6183\n",
      "14\t87427\t0.2\t1\t80554 6873\n",
      "15\t88669\t0.2\t1\t82188 6481\n",
      "16\t87155\t0.2\t1\t79865 7290\n",
      "17\t108105\t0.2\t1\t98581 9524\n",
      "18\t88189\t0.2\t1\t82109 6080\n",
      "19\t75841\t0.2\t1\t70534 5307\n",
      "20\t90690\t0.2\t1\t84354 6336\n",
      "21\t100155\t0.2\t1\t92424 7731\n",
      "22\t84109\t0.2\t1\t78568 5541\n",
      "23\t100437\t0.2\t1\t93211 7226\n",
      "24\t89008\t0.2\t1\t81375 7633\n",
      "25\t97433\t0.2\t1\t90437 6996\n",
      "26\t86900\t0.2\t1\t81328 5572\n",
      "27\t98051\t0.2\t1\t90237 7814\n",
      "28\t106220\t0.2\t1\t97819 8401\n",
      "29\t94302\t0.2\t1\t86322 7980\n",
      "30\t107823\t0.2\t1\t101345 6478\n",
      "31\t77079\t0.2\t1\t71662 5417\n",
      "32\t102558\t0.2\t1\t95070 7488\n",
      "33\t91366\t0.2\t1\t84409 6957\n",
      "34\t99532\t0.2\t1\t91471 8061\n",
      "35\t107666\t0.2\t1\t99772 7894\n",
      "36\t89079\t0.2\t1\t83528 5551\n",
      "37\t100059\t0.2\t1\t92531 7528\n",
      "38\t104683\t0.2\t1\t96200 8483\n",
      "39\t105342\t0.2\t1\t97340 8002\n",
      "40\t103603\t0.2\t1\t95197 8406\n",
      "41\t116472\t0.2\t1\t105109 11363\n",
      "42\t91799\t0.2\t1\t84082 7717\n",
      "43\t151482\t0.2\t1\t141550 9932\n",
      "44\t48893\t0.2\t1\t44323 4570\n",
      "45\t107330\t0.2\t1\t98158 9172\n",
      "46\t169588\t0.2\t1\t157239 12349\n",
      "47\t115292\t0.2\t1\t107496 7796\n",
      "48\t81237\t0.2\t1\t73490 7747\n",
      "49\t158364\t0.2\t1\t148089 10275\n",
      "50\t62839\t0.2\t1\t58050 4789\n",
      "51\t90568\t0.2\t1\t83874 6694\n",
      "52\t192734\t0.2\t1\t181187 11547\n",
      "53\t137640\t0.2\t1\t130713 6927\n",
      "54\t59884\t0.2\t1\t55438 4446\n",
      "55\t102380\t0.2\t1\t96895 5485\n",
      "56\t71739\t0.2\t1\t66587 5152\n",
      "57\t121813\t0.2\t1\t115448 6365\n",
      "58\t43981\t0.2\t1\t41008 2973\n",
      "59\t41261\t0.2\t1\t37490 3771\n",
      "60\t111430\t0.2\t1\t105252 6178\n",
      "61\t87063\t0.2\t1\t82619 4444\n",
      "62\t47900\t0.2\t1\t44309 3591\n",
      "63\t85477\t0.2\t1\t80258 5219\n",
      "64\t72824\t0.2\t1\t68588 4236\n",
      "65\t60641\t0.2\t1\t56399 4242\n",
      "66\t87315\t0.2\t1\t81607 5708\n",
      "67\t100034\t0.2\t1\t92897 7137\n",
      "68\t160033\t0.2\t1\t150907 9126\n",
      "69\t87754\t0.2\t1\t82192 5562\n",
      "70\t34259\t0.2\t1\t30999 3260\n",
      "71\t22148\t0.2\t1\t20058 2090\n",
      "72\t59270\t0.2\t1\t55764 3506\n",
      "73\t72359\t0.2\t1\t68127 4232\n",
      "74\t73545\t0.2\t1\t68961 4584\n",
      "75\t72172\t0.2\t1\t67396 4776\n",
      "76\t70342\t0.2\t1\t65617 4725\n",
      "77\t69830\t0.2\t1\t65084 4746\n",
      "78\t66115\t0.2\t1\t61597 4518\n",
      "79\t64360\t0.2\t1\t60010 4350\n",
      "80\t63230\t0.2\t1\t58997 4233\n",
      "81\t61274\t0.2\t1\t57172 4102\n",
      "82\t59014\t0.2\t1\t55084 3930\n",
      "83\t57125\t0.2\t1\t53358 3767\n",
      "84\t54574\t0.2\t1\t50808 3766\n",
      "85\t52527\t0.2\t1\t48940 3587\n",
      "86\t50257\t0.2\t1\t46911 3346\n",
      "87\t48208\t0.2\t1\t45026 3182\n",
      "88\t45866\t0.2\t1\t42781 3085\n",
      "89\t44045\t0.2\t1\t41050 2995\n",
      "90\t43438\t0.2\t1\t40512 2926\n",
      "91\t40328\t0.2\t1\t37640 2688\n",
      "92\t38164\t0.2\t1\t35615 2549\n",
      "93\t35352\t0.2\t1\t32983 2369\n",
      "94\t33785\t0.2\t1\t31458 2327\n",
      "95\t31871\t0.2\t1\t29668 2203\n",
      "96\t29463\t0.2\t1\t27470 1993\n",
      "97\t27711\t0.2\t1\t25891 1820\n",
      "98\t25451\t0.2\t1\t23610 1841\n",
      "99\t24415\t0.2\t1\t22590 1825\n",
      "100\t21725\t0.2\t1\t20167 1558\n",
      "101\t20433\t0.2\t1\t19008 1425\n",
      "102\t18660\t0.2\t1\t17274 1386\n",
      "103\t16794\t0.2\t1\t15616 1178\n",
      "104\t15844\t0.2\t1\t14677 1167\n",
      "105\t14021\t0.2\t1\t13036 985\n",
      "106\t12448\t0.2\t1\t11562 886\n",
      "107\t11573\t0.2\t1\t10706 867\n",
      "108\t10412\t0.2\t1\t9602 810\n",
      "109\t8989\t0.2\t1\t8297 692\n",
      "110\t8150\t0.2\t1\t7523 627\n",
      "111\t7254\t0.2\t1\t6680 574\n",
      "112\t6183\t0.2\t1\t5709 474\n",
      "113\t5353\t0.2\t1\t4937 416\n",
      "114\t4766\t0.2\t1\t4364 402\n",
      "115\t4132\t0.2\t1\t3707 425\n",
      "116\t3477\t0.2\t1\t3165 312\n",
      "117\t3036\t0.2\t1\t2738 298\n",
      "118\t2618\t0.2\t1\t2398 220\n",
      "119\t2115\t0.2\t1\t1918 197\n",
      "120\t1769\t0.2\t1\t1599 170\n",
      "121\t1495\t0.2\t1\t1327 168\n",
      "122\t1258\t0.2\t1\t1133 125\n",
      "123\t1079\t0.2\t1\t967 112\n",
      "124\t920\t0.2\t1\t808 112\n",
      "125\t777\t0.2\t1\t674 103\n",
      "126\t681\t0.2\t1\t592 89\n",
      "127\t553\t0.2\t1\t494 59\n",
      "128\t387\t0.2\t1\t336 51\n",
      "129\t401\t0.2\t1\t353 48\n",
      "130\t291\t0.2\t1\t256 35\n",
      "131\t270\t0.2\t1\t235 35\n",
      "132\t242\t0.2\t1\t222 20\n",
      "133\t268\t0.2\t1\t225 43\n",
      "134\t327\t0.2\t1\t297 30\n",
      "135\t148\t0.2\t1\t131 17\n",
      "136\t79\t0.2\t1\t65 14\n",
      "137\t63\t0.2\t1\t55 8\n",
      "138\t35\t0.2\t1\t28 7\n",
      "139\t28\t0.2\t1\t22 6\n",
      "140\t22\t0.2\t1\t20 2\n",
      "141\t16\t0.2\t1\t14 2\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t16\t0.2\t1\t11 5\n",
      "144\t10\t0.2\t1\t5 5\n",
      "145\t13\t0.2\t1\t8 5\n",
      "146\t16\t0.2\t1\t12 4\n",
      "147\t29\t0.2\t1\t25 4\n",
      "148\t26\t0.2\t1\t23 3\n",
      "149\t19\t0.2\t1\t18 1\n",
      "150\t167\t0.2\t1\t150 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "15591852 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/16C_32psu_4_S4_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 676.29 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              15,591,852\n",
      "Reads with adapters:                11,164,940 (71.6%)\n",
      "Reads written (passing filters):    15,591,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,338,777,800 bp\n",
      "Quality-trimmed:               8,987,489 bp (0.4%)\n",
      "Total written (filtered):  1,952,377,119 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11164940 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 12.2%\n",
      "  G: 21.8%\n",
      "  T: 35.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2716063\t3897963.0\t0\t2716063\n",
      "2\t365801\t974490.8\t0\t365801\n",
      "3\t204110\t243622.7\t0\t204110\n",
      "4\t111706\t60905.7\t0\t111706\n",
      "5\t93686\t15226.4\t0\t93686\n",
      "6\t82310\t3806.6\t0\t82310\n",
      "7\t64821\t951.7\t0\t64821\n",
      "8\t93347\t237.9\t0\t93347\n",
      "9\t74551\t59.5\t0\t74263 288\n",
      "10\t78346\t14.9\t1\t72991 5355\n",
      "11\t85380\t3.7\t1\t78328 7052\n",
      "12\t84150\t0.9\t1\t77541 6609\n",
      "13\t96178\t0.2\t1\t88615 7563\n",
      "14\t88433\t0.2\t1\t80671 7762\n",
      "15\t72019\t0.2\t1\t66503 5516\n",
      "16\t102952\t0.2\t1\t94755 8197\n",
      "17\t86769\t0.2\t1\t79687 7082\n",
      "18\t76619\t0.2\t1\t70853 5766\n",
      "19\t98606\t0.2\t1\t91042 7564\n",
      "20\t91841\t0.2\t1\t84883 6958\n",
      "21\t80822\t0.2\t1\t74556 6266\n",
      "22\t90084\t0.2\t1\t83382 6702\n",
      "23\t95275\t0.2\t1\t87967 7308\n",
      "24\t93761\t0.2\t1\t86637 7124\n",
      "25\t105154\t0.2\t1\t98108 7046\n",
      "26\t87685\t0.2\t1\t80687 6998\n",
      "27\t96238\t0.2\t1\t88248 7990\n",
      "28\t86982\t0.2\t1\t81506 5476\n",
      "29\t96532\t0.2\t1\t89458 7074\n",
      "30\t91145\t0.2\t1\t85081 6064\n",
      "31\t100852\t0.2\t1\t93803 7049\n",
      "32\t86615\t0.2\t1\t80885 5730\n",
      "33\t99917\t0.2\t1\t92622 7295\n",
      "34\t128321\t0.2\t1\t119309 9012\n",
      "35\t68466\t0.2\t1\t63767 4699\n",
      "36\t99171\t0.2\t1\t91848 7323\n",
      "37\t104228\t0.2\t1\t97074 7154\n",
      "38\t95840\t0.2\t1\t89901 5939\n",
      "39\t91223\t0.2\t1\t85899 5324\n",
      "40\t103901\t0.2\t1\t97025 6876\n",
      "41\t100699\t0.2\t1\t94360 6339\n",
      "42\t101648\t0.2\t1\t95335 6313\n",
      "43\t91398\t0.2\t1\t85693 5705\n",
      "44\t105931\t0.2\t1\t98930 7001\n",
      "45\t111589\t0.2\t1\t104455 7134\n",
      "46\t79183\t0.2\t1\t74169 5014\n",
      "47\t99997\t0.2\t1\t94251 5746\n",
      "48\t97219\t0.2\t1\t91470 5749\n",
      "49\t102240\t0.2\t1\t95693 6547\n",
      "50\t96699\t0.2\t1\t90628 6071\n",
      "51\t118228\t0.2\t1\t110940 7288\n",
      "52\t84607\t0.2\t1\t79439 5168\n",
      "53\t104742\t0.2\t1\t98960 5782\n",
      "54\t75371\t0.2\t1\t71116 4255\n",
      "55\t98114\t0.2\t1\t92755 5359\n",
      "56\t100972\t0.2\t1\t94974 5998\n",
      "57\t94787\t0.2\t1\t89866 4921\n",
      "58\t91176\t0.2\t1\t86320 4856\n",
      "59\t91225\t0.2\t1\t86196 5029\n",
      "60\t94303\t0.2\t1\t88940 5363\n",
      "61\t95802\t0.2\t1\t90516 5286\n",
      "62\t104062\t0.2\t1\t98063 5999\n",
      "63\t134017\t0.2\t1\t127276 6741\n",
      "64\t107807\t0.2\t1\t103277 4530\n",
      "65\t71585\t0.2\t1\t67908 3677\n",
      "66\t78177\t0.2\t1\t74035 4142\n",
      "67\t77189\t0.2\t1\t72913 4276\n",
      "68\t77503\t0.2\t1\t73378 4125\n",
      "69\t76227\t0.2\t1\t72009 4218\n",
      "70\t77805\t0.2\t1\t73707 4098\n",
      "71\t80348\t0.2\t1\t76129 4219\n",
      "72\t76686\t0.2\t1\t72546 4140\n",
      "73\t78718\t0.2\t1\t74553 4165\n",
      "74\t72998\t0.2\t1\t69110 3888\n",
      "75\t69291\t0.2\t1\t65521 3770\n",
      "76\t66128\t0.2\t1\t62546 3582\n",
      "77\t65756\t0.2\t1\t62222 3534\n",
      "78\t62918\t0.2\t1\t59420 3498\n",
      "79\t59982\t0.2\t1\t56797 3185\n",
      "80\t59872\t0.2\t1\t56761 3111\n",
      "81\t57826\t0.2\t1\t54814 3012\n",
      "82\t55215\t0.2\t1\t52336 2879\n",
      "83\t53442\t0.2\t1\t50600 2842\n",
      "84\t50951\t0.2\t1\t48209 2742\n",
      "85\t49724\t0.2\t1\t47191 2533\n",
      "86\t49032\t0.2\t1\t46473 2559\n",
      "87\t49028\t0.2\t1\t46704 2324\n",
      "88\t46498\t0.2\t1\t44150 2348\n",
      "89\t42159\t0.2\t1\t40004 2155\n",
      "90\t40296\t0.2\t1\t38283 2013\n",
      "91\t37292\t0.2\t1\t35403 1889\n",
      "92\t34841\t0.2\t1\t33039 1802\n",
      "93\t33152\t0.2\t1\t31479 1673\n",
      "94\t31763\t0.2\t1\t30149 1614\n",
      "95\t30241\t0.2\t1\t28665 1576\n",
      "96\t27836\t0.2\t1\t26423 1413\n",
      "97\t25827\t0.2\t1\t24589 1238\n",
      "98\t24001\t0.2\t1\t22767 1234\n",
      "99\t23498\t0.2\t1\t22301 1197\n",
      "100\t21732\t0.2\t1\t20644 1088\n",
      "101\t19788\t0.2\t1\t18836 952\n",
      "102\t17924\t0.2\t1\t16990 934\n",
      "103\t15850\t0.2\t1\t15000 850\n",
      "104\t14943\t0.2\t1\t14170 773\n",
      "105\t13279\t0.2\t1\t12646 633\n",
      "106\t11897\t0.2\t1\t11297 600\n",
      "107\t10895\t0.2\t1\t10336 559\n",
      "108\t9562\t0.2\t1\t9057 505\n",
      "109\t8353\t0.2\t1\t7873 480\n",
      "110\t7641\t0.2\t1\t7229 412\n",
      "111\t7111\t0.2\t1\t6763 348\n",
      "112\t6029\t0.2\t1\t5694 335\n",
      "113\t5217\t0.2\t1\t4974 243\n",
      "114\t4567\t0.2\t1\t4342 225\n",
      "115\t3814\t0.2\t1\t3593 221\n",
      "116\t3236\t0.2\t1\t3069 167\n",
      "117\t2708\t0.2\t1\t2550 158\n",
      "118\t2402\t0.2\t1\t2269 133\n",
      "119\t1976\t0.2\t1\t1850 126\n",
      "120\t1573\t0.2\t1\t1480 93\n",
      "121\t1360\t0.2\t1\t1290 70\n",
      "122\t1161\t0.2\t1\t1085 76\n",
      "123\t1044\t0.2\t1\t961 83\n",
      "124\t817\t0.2\t1\t759 58\n",
      "125\t740\t0.2\t1\t678 62\n",
      "126\t632\t0.2\t1\t586 46\n",
      "127\t507\t0.2\t1\t481 26\n",
      "128\t371\t0.2\t1\t328 43\n",
      "129\t368\t0.2\t1\t334 34\n",
      "130\t272\t0.2\t1\t248 24\n",
      "131\t250\t0.2\t1\t232 18\n",
      "132\t226\t0.2\t1\t220 6\n",
      "133\t268\t0.2\t1\t248 20\n",
      "134\t319\t0.2\t1\t304 15\n",
      "135\t142\t0.2\t1\t138 4\n",
      "136\t65\t0.2\t1\t62 3\n",
      "137\t61\t0.2\t1\t60 1\n",
      "138\t31\t0.2\t1\t25 6\n",
      "139\t26\t0.2\t1\t21 5\n",
      "140\t13\t0.2\t1\t11 2\n",
      "141\t16\t0.2\t1\t13 3\n",
      "142\t11\t0.2\t1\t8 3\n",
      "143\t13\t0.2\t1\t11 2\n",
      "144\t7\t0.2\t1\t5 2\n",
      "145\t9\t0.2\t1\t6 3\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t30\t0.2\t1\t26 4\n",
      "148\t17\t0.2\t1\t16 1\n",
      "149\t16\t0.2\t1\t14 2\n",
      "150\t117\t0.2\t1\t107 10\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "15591852 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz, file_2: 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 15591852\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 18174 (0.12%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_32psu_4_S4_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_32psu_4_S4_L002_R1_001_trimmed.fq.gz and 16C_32psu_4_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_1_S9_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 685.54 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,172,174\n",
      "Reads with adapters:                14,009,565 (77.1%)\n",
      "Reads written (passing filters):    18,172,174 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,725,826,100 bp\n",
      "Quality-trimmed:               5,576,029 bp (0.2%)\n",
      "Total written (filtered):  2,153,538,533 bp (79.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14009565 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 14.7%\n",
      "  G: 20.2%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2553382\t4543043.5\t0\t2553382\n",
      "2\t361304\t1135760.9\t0\t361304\n",
      "3\t203451\t283940.2\t0\t203451\n",
      "4\t119346\t70985.1\t0\t119346\n",
      "5\t96383\t17746.3\t0\t96383\n",
      "6\t92443\t4436.6\t0\t92443\n",
      "7\t75722\t1109.1\t0\t75722\n",
      "8\t95119\t277.3\t0\t95119\n",
      "9\t94842\t69.3\t0\t93616 1226\n",
      "10\t89178\t17.3\t1\t84587 4591\n",
      "11\t98839\t4.3\t1\t91164 7675\n",
      "12\t94690\t1.1\t1\t88668 6022\n",
      "13\t96487\t0.3\t1\t90169 6318\n",
      "14\t97956\t0.3\t1\t91158 6798\n",
      "15\t101508\t0.3\t1\t94950 6558\n",
      "16\t99917\t0.3\t1\t92571 7346\n",
      "17\t122129\t0.3\t1\t112362 9767\n",
      "18\t103253\t0.3\t1\t97095 6158\n",
      "19\t88992\t0.3\t1\t83121 5871\n",
      "20\t106240\t0.3\t1\t99305 6935\n",
      "21\t117020\t0.3\t1\t108991 8029\n",
      "22\t99969\t0.3\t1\t94314 5655\n",
      "23\t116939\t0.3\t1\t109185 7754\n",
      "24\t106769\t0.3\t1\t98031 8738\n",
      "25\t115163\t0.3\t1\t107352 7811\n",
      "26\t105376\t0.3\t1\t99072 6304\n",
      "27\t117162\t0.3\t1\t108493 8669\n",
      "28\t115218\t0.3\t1\t107267 7951\n",
      "29\t121831\t0.3\t1\t112202 9629\n",
      "30\t130309\t0.3\t1\t123069 7240\n",
      "31\t100032\t0.3\t1\t93538 6494\n",
      "32\t125002\t0.3\t1\t117224 7778\n",
      "33\t114973\t0.3\t1\t107024 7949\n",
      "34\t140462\t0.3\t1\t130516 9946\n",
      "35\t105315\t0.3\t1\t99186 6129\n",
      "36\t133473\t0.3\t1\t123640 9833\n",
      "37\t118492\t0.3\t1\t111283 7209\n",
      "38\t133856\t0.3\t1\t125329 8527\n",
      "39\t129659\t0.3\t1\t120554 9105\n",
      "40\t124082\t0.3\t1\t117052 7030\n",
      "41\t145782\t0.3\t1\t135364 10418\n",
      "42\t118797\t0.3\t1\t109967 8830\n",
      "43\t194297\t0.3\t1\t184362 9935\n",
      "44\t63920\t0.3\t1\t58531 5389\n",
      "45\t132756\t0.3\t1\t124789 7967\n",
      "46\t128569\t0.3\t1\t120475 8094\n",
      "47\t134969\t0.3\t1\t126850 8119\n",
      "48\t127793\t0.3\t1\t119492 8301\n",
      "49\t141939\t0.3\t1\t132171 9768\n",
      "50\t122730\t0.3\t1\t115608 7122\n",
      "51\t132062\t0.3\t1\t123880 8182\n",
      "52\t137197\t0.3\t1\t127456 9741\n",
      "53\t151575\t0.3\t1\t142793 8782\n",
      "54\t114349\t0.3\t1\t107125 7224\n",
      "55\t137708\t0.3\t1\t129156 8552\n",
      "56\t161649\t0.3\t1\t152383 9266\n",
      "57\t114410\t0.3\t1\t106500 7910\n",
      "58\t137398\t0.3\t1\t130375 7023\n",
      "59\t114545\t0.3\t1\t107545 7000\n",
      "60\t136347\t0.3\t1\t128034 8313\n",
      "61\t131449\t0.3\t1\t123861 7588\n",
      "62\t139001\t0.3\t1\t131253 7748\n",
      "63\t117841\t0.3\t1\t110235 7606\n",
      "64\t124937\t0.3\t1\t117975 6962\n",
      "65\t137827\t0.3\t1\t129558 8269\n",
      "66\t145197\t0.3\t1\t136356 8841\n",
      "67\t156037\t0.3\t1\t144343 11694\n",
      "68\t293703\t0.3\t1\t279898 13805\n",
      "69\t56423\t0.3\t1\t51953 4470\n",
      "70\t20870\t0.3\t1\t17669 3201\n",
      "71\t73391\t0.3\t1\t68872 4519\n",
      "72\t109127\t0.3\t1\t102948 6179\n",
      "73\t115790\t0.3\t1\t109200 6590\n",
      "74\t114108\t0.3\t1\t107376 6732\n",
      "75\t112468\t0.3\t1\t105902 6566\n",
      "76\t111908\t0.3\t1\t105400 6508\n",
      "77\t110508\t0.3\t1\t103945 6563\n",
      "78\t104515\t0.3\t1\t98282 6233\n",
      "79\t103447\t0.3\t1\t97321 6126\n",
      "80\t100314\t0.3\t1\t94349 5965\n",
      "81\t99307\t0.3\t1\t93451 5856\n",
      "82\t96030\t0.3\t1\t90418 5612\n",
      "83\t93920\t0.3\t1\t88302 5618\n",
      "84\t90786\t0.3\t1\t85374 5412\n",
      "85\t87976\t0.3\t1\t82940 5036\n",
      "86\t84575\t0.3\t1\t79489 5086\n",
      "87\t80800\t0.3\t1\t75885 4915\n",
      "88\t79827\t0.3\t1\t75057 4770\n",
      "89\t76306\t0.3\t1\t71788 4518\n",
      "90\t75447\t0.3\t1\t70879 4568\n",
      "91\t70739\t0.3\t1\t66457 4282\n",
      "92\t67827\t0.3\t1\t63701 4126\n",
      "93\t63802\t0.3\t1\t60033 3769\n",
      "94\t61718\t0.3\t1\t57911 3807\n",
      "95\t57339\t0.3\t1\t53808 3531\n",
      "96\t54465\t0.3\t1\t51099 3366\n",
      "97\t51194\t0.3\t1\t48085 3109\n",
      "98\t47800\t0.3\t1\t44777 3023\n",
      "99\t45494\t0.3\t1\t42548 2946\n",
      "100\t41995\t0.3\t1\t39278 2717\n",
      "101\t38874\t0.3\t1\t36431 2443\n",
      "102\t36656\t0.3\t1\t34295 2361\n",
      "103\t33608\t0.3\t1\t31418 2190\n",
      "104\t31440\t0.3\t1\t29409 2031\n",
      "105\t28173\t0.3\t1\t26291 1882\n",
      "106\t25331\t0.3\t1\t23628 1703\n",
      "107\t23358\t0.3\t1\t21803 1555\n",
      "108\t21762\t0.3\t1\t20261 1501\n",
      "109\t19593\t0.3\t1\t18199 1394\n",
      "110\t17697\t0.3\t1\t16476 1221\n",
      "111\t15926\t0.3\t1\t14765 1161\n",
      "112\t13846\t0.3\t1\t12871 975\n",
      "113\t11827\t0.3\t1\t10954 873\n",
      "114\t10902\t0.3\t1\t10102 800\n",
      "115\t9531\t0.3\t1\t8821 710\n",
      "116\t8041\t0.3\t1\t7361 680\n",
      "117\t6932\t0.3\t1\t6371 561\n",
      "118\t5995\t0.3\t1\t5469 526\n",
      "119\t4903\t0.3\t1\t4506 397\n",
      "120\t4372\t0.3\t1\t3973 399\n",
      "121\t3600\t0.3\t1\t3268 332\n",
      "122\t2879\t0.3\t1\t2586 293\n",
      "123\t2449\t0.3\t1\t2225 224\n",
      "124\t2193\t0.3\t1\t1959 234\n",
      "125\t1829\t0.3\t1\t1616 213\n",
      "126\t1642\t0.3\t1\t1488 154\n",
      "127\t1294\t0.3\t1\t1160 134\n",
      "128\t1024\t0.3\t1\t902 122\n",
      "129\t830\t0.3\t1\t734 96\n",
      "130\t651\t0.3\t1\t579 72\n",
      "131\t623\t0.3\t1\t568 55\n",
      "132\t611\t0.3\t1\t554 57\n",
      "133\t770\t0.3\t1\t705 65\n",
      "134\t1146\t0.3\t1\t1081 65\n",
      "135\t448\t0.3\t1\t402 46\n",
      "136\t201\t0.3\t1\t171 30\n",
      "137\t163\t0.3\t1\t138 25\n",
      "138\t82\t0.3\t1\t67 15\n",
      "139\t72\t0.3\t1\t55 17\n",
      "140\t56\t0.3\t1\t42 14\n",
      "141\t36\t0.3\t1\t35 1\n",
      "142\t25\t0.3\t1\t15 10\n",
      "143\t23\t0.3\t1\t18 5\n",
      "144\t8\t0.3\t1\t7 1\n",
      "145\t24\t0.3\t1\t19 5\n",
      "146\t21\t0.3\t1\t20 1\n",
      "147\t76\t0.3\t1\t66 10\n",
      "148\t65\t0.3\t1\t57 8\n",
      "149\t66\t0.3\t1\t57 9\n",
      "150\t608\t0.3\t1\t538 70\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18172174 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_1_S9_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 716.84 s (39 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,172,174\n",
      "Reads with adapters:                13,925,142 (76.6%)\n",
      "Reads written (passing filters):    18,172,174 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,725,826,100 bp\n",
      "Quality-trimmed:              12,772,261 bp (0.5%)\n",
      "Total written (filtered):  2,156,656,577 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13925142 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.4%\n",
      "  G: 23.9%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2593588\t4543043.5\t0\t2593588\n",
      "2\t376254\t1135760.9\t0\t376254\n",
      "3\t210205\t283940.2\t0\t210205\n",
      "4\t119037\t70985.1\t0\t119037\n",
      "5\t104651\t17746.3\t0\t104651\n",
      "6\t89236\t4436.6\t0\t89236\n",
      "7\t73097\t1109.1\t0\t73097\n",
      "8\t103442\t277.3\t0\t103442\n",
      "9\t83266\t69.3\t0\t82975 291\n",
      "10\t90702\t17.3\t1\t86533 4169\n",
      "11\t96126\t4.3\t1\t90174 5952\n",
      "12\t96465\t1.1\t1\t90519 5946\n",
      "13\t107004\t0.3\t1\t100993 6011\n",
      "14\t100898\t0.3\t1\t93952 6946\n",
      "15\t82733\t0.3\t1\t77655 5078\n",
      "16\t119602\t0.3\t1\t112441 7161\n",
      "17\t94839\t0.3\t1\t88603 6236\n",
      "18\t92099\t0.3\t1\t86695 5404\n",
      "19\t114214\t0.3\t1\t107016 7198\n",
      "20\t107713\t0.3\t1\t101500 6213\n",
      "21\t95562\t0.3\t1\t89784 5778\n",
      "22\t107588\t0.3\t1\t101222 6366\n",
      "23\t111031\t0.3\t1\t104295 6736\n",
      "24\t110341\t0.3\t1\t103677 6664\n",
      "25\t124129\t0.3\t1\t117008 7121\n",
      "26\t103231\t0.3\t1\t96892 6339\n",
      "27\t114476\t0.3\t1\t106707 7769\n",
      "28\t107750\t0.3\t1\t102430 5320\n",
      "29\t114219\t0.3\t1\t107523 6696\n",
      "30\t113089\t0.3\t1\t107416 5673\n",
      "31\t123120\t0.3\t1\t116178 6942\n",
      "32\t110386\t0.3\t1\t104842 5544\n",
      "33\t123147\t0.3\t1\t116091 7056\n",
      "34\t156685\t0.3\t1\t148041 8644\n",
      "35\t89283\t0.3\t1\t84701 4582\n",
      "36\t120443\t0.3\t1\t113382 7061\n",
      "37\t129458\t0.3\t1\t122208 7250\n",
      "38\t122546\t0.3\t1\t116642 5904\n",
      "39\t119280\t0.3\t1\t113530 5750\n",
      "40\t132471\t0.3\t1\t125158 7313\n",
      "41\t126758\t0.3\t1\t120385 6373\n",
      "42\t130992\t0.3\t1\t124472 6520\n",
      "43\t120675\t0.3\t1\t114532 6143\n",
      "44\t136598\t0.3\t1\t129517 7081\n",
      "45\t144732\t0.3\t1\t137448 7284\n",
      "46\t106581\t0.3\t1\t101258 5323\n",
      "47\t132125\t0.3\t1\t125807 6318\n",
      "48\t130489\t0.3\t1\t124465 6024\n",
      "49\t135590\t0.3\t1\t129070 6520\n",
      "50\t128632\t0.3\t1\t122091 6541\n",
      "51\t159357\t0.3\t1\t151626 7731\n",
      "52\t116120\t0.3\t1\t110786 5334\n",
      "53\t139419\t0.3\t1\t133495 5924\n",
      "54\t106851\t0.3\t1\t101979 4872\n",
      "55\t135687\t0.3\t1\t129810 5877\n",
      "56\t138774\t0.3\t1\t132432 6342\n",
      "57\t131848\t0.3\t1\t126296 5552\n",
      "58\t127061\t0.3\t1\t121713 5348\n",
      "59\t129536\t0.3\t1\t124051 5485\n",
      "60\t133143\t0.3\t1\t127312 5831\n",
      "61\t136216\t0.3\t1\t129973 6243\n",
      "62\t147811\t0.3\t1\t140964 6847\n",
      "63\t197994\t0.3\t1\t190073 7921\n",
      "64\t157927\t0.3\t1\t152075 5852\n",
      "65\t105906\t0.3\t1\t101567 4339\n",
      "66\t114677\t0.3\t1\t109835 4842\n",
      "67\t114010\t0.3\t1\t108938 5072\n",
      "68\t114564\t0.3\t1\t109670 4894\n",
      "69\t112505\t0.3\t1\t107761 4744\n",
      "70\t118303\t0.3\t1\t113227 5076\n",
      "71\t123531\t0.3\t1\t118496 5035\n",
      "72\t117539\t0.3\t1\t112405 5134\n",
      "73\t122781\t0.3\t1\t117929 4852\n",
      "74\t113055\t0.3\t1\t108472 4583\n",
      "75\t107896\t0.3\t1\t103502 4394\n",
      "76\t106239\t0.3\t1\t101975 4264\n",
      "77\t105105\t0.3\t1\t100663 4442\n",
      "78\t99929\t0.3\t1\t95794 4135\n",
      "79\t96680\t0.3\t1\t92707 3973\n",
      "80\t95740\t0.3\t1\t91762 3978\n",
      "81\t93945\t0.3\t1\t90030 3915\n",
      "82\t89626\t0.3\t1\t86049 3577\n",
      "83\t87905\t0.3\t1\t84339 3566\n",
      "84\t84781\t0.3\t1\t81246 3535\n",
      "85\t83386\t0.3\t1\t79899 3487\n",
      "86\t83197\t0.3\t1\t79728 3469\n",
      "87\t82660\t0.3\t1\t79311 3349\n",
      "88\t81067\t0.3\t1\t77676 3391\n",
      "89\t73343\t0.3\t1\t70291 3052\n",
      "90\t69973\t0.3\t1\t67072 2901\n",
      "91\t65096\t0.3\t1\t62445 2651\n",
      "92\t61646\t0.3\t1\t59119 2527\n",
      "93\t59393\t0.3\t1\t56935 2458\n",
      "94\t57203\t0.3\t1\t54848 2355\n",
      "95\t53756\t0.3\t1\t51532 2224\n",
      "96\t50956\t0.3\t1\t49026 1930\n",
      "97\t47276\t0.3\t1\t45371 1905\n",
      "98\t44746\t0.3\t1\t43017 1729\n",
      "99\t42641\t0.3\t1\t41013 1628\n",
      "100\t39899\t0.3\t1\t38401 1498\n",
      "101\t36809\t0.3\t1\t35396 1413\n",
      "102\t34264\t0.3\t1\t32958 1306\n",
      "103\t31058\t0.3\t1\t29842 1216\n",
      "104\t30002\t0.3\t1\t28912 1090\n",
      "105\t26784\t0.3\t1\t25668 1116\n",
      "106\t24257\t0.3\t1\t23303 954\n",
      "107\t21941\t0.3\t1\t21141 800\n",
      "108\t20010\t0.3\t1\t19251 759\n",
      "109\t18313\t0.3\t1\t17584 729\n",
      "110\t16702\t0.3\t1\t16007 695\n",
      "111\t15559\t0.3\t1\t14968 591\n",
      "112\t13419\t0.3\t1\t12890 529\n",
      "113\t11631\t0.3\t1\t11113 518\n",
      "114\t10507\t0.3\t1\t10110 397\n",
      "115\t8927\t0.3\t1\t8561 366\n",
      "116\t7354\t0.3\t1\t7008 346\n",
      "117\t6258\t0.3\t1\t5993 265\n",
      "118\t5422\t0.3\t1\t5194 228\n",
      "119\t4575\t0.3\t1\t4365 210\n",
      "120\t3972\t0.3\t1\t3799 173\n",
      "121\t3338\t0.3\t1\t3179 159\n",
      "122\t2723\t0.3\t1\t2578 145\n",
      "123\t2312\t0.3\t1\t2208 104\n",
      "124\t2044\t0.3\t1\t1948 96\n",
      "125\t1725\t0.3\t1\t1623 102\n",
      "126\t1567\t0.3\t1\t1498 69\n",
      "127\t1207\t0.3\t1\t1135 72\n",
      "128\t957\t0.3\t1\t886 71\n",
      "129\t782\t0.3\t1\t739 43\n",
      "130\t614\t0.3\t1\t575 39\n",
      "131\t601\t0.3\t1\t565 36\n",
      "132\t573\t0.3\t1\t543 30\n",
      "133\t862\t0.3\t1\t829 33\n",
      "134\t1185\t0.3\t1\t1148 37\n",
      "135\t434\t0.3\t1\t412 22\n",
      "136\t191\t0.3\t1\t179 12\n",
      "137\t150\t0.3\t1\t137 13\n",
      "138\t67\t0.3\t1\t59 8\n",
      "139\t66\t0.3\t1\t55 11\n",
      "140\t41\t0.3\t1\t35 6\n",
      "141\t28\t0.3\t1\t25 3\n",
      "142\t18\t0.3\t1\t17 1\n",
      "143\t17\t0.3\t1\t17\n",
      "144\t10\t0.3\t1\t7 3\n",
      "145\t31\t0.3\t1\t30 1\n",
      "146\t24\t0.3\t1\t21 3\n",
      "147\t69\t0.3\t1\t63 6\n",
      "148\t53\t0.3\t1\t49 4\n",
      "149\t41\t0.3\t1\t39 2\n",
      "150\t371\t0.3\t1\t357 14\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18172174 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz, file_2: 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18172174\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 34268 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_1_S9_L001_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_1_S9_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 692.87 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,056,494\n",
      "Reads with adapters:                13,928,110 (77.1%)\n",
      "Reads written (passing filters):    18,056,494 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,708,474,100 bp\n",
      "Quality-trimmed:               5,741,117 bp (0.2%)\n",
      "Total written (filtered):  2,137,545,368 bp (78.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13928110 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 14.8%\n",
      "  G: 20.2%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2529563\t4514123.5\t0\t2529563\n",
      "2\t355142\t1128530.9\t0\t355142\n",
      "3\t202264\t282132.7\t0\t202264\n",
      "4\t116958\t70533.2\t0\t116958\n",
      "5\t97699\t17633.3\t0\t97699\n",
      "6\t92485\t4408.3\t0\t92485\n",
      "7\t72663\t1102.1\t0\t72663\n",
      "8\t94383\t275.5\t0\t94383\n",
      "9\t92644\t68.9\t0\t91510 1134\n",
      "10\t86522\t17.2\t1\t80894 5628\n",
      "11\t97786\t4.3\t1\t89346 8440\n",
      "12\t93298\t1.1\t1\t86415 6883\n",
      "13\t95771\t0.3\t1\t88325 7446\n",
      "14\t95946\t0.3\t1\t88058 7888\n",
      "15\t101399\t0.3\t1\t93509 7890\n",
      "16\t98158\t0.3\t1\t89606 8552\n",
      "17\t122597\t0.3\t1\t111303 11294\n",
      "18\t101750\t0.3\t1\t94436 7314\n",
      "19\t86831\t0.3\t1\t80395 6436\n",
      "20\t105075\t0.3\t1\t97137 7938\n",
      "21\t118601\t0.3\t1\t108999 9602\n",
      "22\t97228\t0.3\t1\t90470 6758\n",
      "23\t116927\t0.3\t1\t107741 9186\n",
      "24\t105532\t0.3\t1\t95756 9776\n",
      "25\t113828\t0.3\t1\t105061 8767\n",
      "26\t103778\t0.3\t1\t96608 7170\n",
      "27\t117560\t0.3\t1\t107874 9686\n",
      "28\t114336\t0.3\t1\t105504 8832\n",
      "29\t122580\t0.3\t1\t111680 10900\n",
      "30\t130099\t0.3\t1\t121451 8648\n",
      "31\t97669\t0.3\t1\t90483 7186\n",
      "32\t124295\t0.3\t1\t114657 9638\n",
      "33\t112476\t0.3\t1\t103286 9190\n",
      "34\t140784\t0.3\t1\t128812 11972\n",
      "35\t103370\t0.3\t1\t96154 7216\n",
      "36\t134329\t0.3\t1\t122737 11592\n",
      "37\t117550\t0.3\t1\t108946 8604\n",
      "38\t133588\t0.3\t1\t123334 10254\n",
      "39\t130482\t0.3\t1\t119347 11135\n",
      "40\t122686\t0.3\t1\t113961 8725\n",
      "41\t145432\t0.3\t1\t132894 12538\n",
      "42\t118079\t0.3\t1\t107701 10378\n",
      "43\t189764\t0.3\t1\t177939 11825\n",
      "44\t65407\t0.3\t1\t59230 6177\n",
      "45\t132188\t0.3\t1\t122729 9459\n",
      "46\t127290\t0.3\t1\t117848 9442\n",
      "47\t135499\t0.3\t1\t126055 9444\n",
      "48\t126038\t0.3\t1\t116373 9665\n",
      "49\t142407\t0.3\t1\t130956 11451\n",
      "50\t121520\t0.3\t1\t113382 8138\n",
      "51\t130837\t0.3\t1\t121331 9506\n",
      "52\t138254\t0.3\t1\t127263 10991\n",
      "53\t150154\t0.3\t1\t140134 10020\n",
      "54\t113941\t0.3\t1\t105808 8133\n",
      "55\t137202\t0.3\t1\t127382 9820\n",
      "56\t163618\t0.3\t1\t152691 10927\n",
      "57\t113941\t0.3\t1\t104904 9037\n",
      "58\t135543\t0.3\t1\t127222 8321\n",
      "59\t114833\t0.3\t1\t106375 8458\n",
      "60\t134934\t0.3\t1\t125495 9439\n",
      "61\t131069\t0.3\t1\t122212 8857\n",
      "62\t140049\t0.3\t1\t130678 9371\n",
      "63\t118477\t0.3\t1\t109768 8709\n",
      "64\t123661\t0.3\t1\t115672 7989\n",
      "65\t136910\t0.3\t1\t127445 9465\n",
      "66\t142766\t0.3\t1\t132932 9834\n",
      "67\t155161\t0.3\t1\t142407 12754\n",
      "68\t281796\t0.3\t1\t266464 15332\n",
      "69\t68541\t0.3\t1\t62954 5587\n",
      "70\t25774\t0.3\t1\t21904 3870\n",
      "71\t70954\t0.3\t1\t66131 4823\n",
      "72\t107448\t0.3\t1\t100862 6586\n",
      "73\t114621\t0.3\t1\t107221 7400\n",
      "74\t114007\t0.3\t1\t106477 7530\n",
      "75\t113035\t0.3\t1\t105200 7835\n",
      "76\t110785\t0.3\t1\t102896 7889\n",
      "77\t110971\t0.3\t1\t103289 7682\n",
      "78\t105110\t0.3\t1\t97785 7325\n",
      "79\t103881\t0.3\t1\t96578 7303\n",
      "80\t100380\t0.3\t1\t93184 7196\n",
      "81\t99197\t0.3\t1\t92210 6987\n",
      "82\t96750\t0.3\t1\t90051 6699\n",
      "83\t92818\t0.3\t1\t86379 6439\n",
      "84\t91340\t0.3\t1\t84942 6398\n",
      "85\t87669\t0.3\t1\t81553 6116\n",
      "86\t83869\t0.3\t1\t77984 5885\n",
      "87\t80883\t0.3\t1\t75364 5519\n",
      "88\t79450\t0.3\t1\t73820 5630\n",
      "89\t76461\t0.3\t1\t71121 5340\n",
      "90\t75182\t0.3\t1\t69973 5209\n",
      "91\t71175\t0.3\t1\t66391 4784\n",
      "92\t67480\t0.3\t1\t62890 4590\n",
      "93\t64001\t0.3\t1\t59615 4386\n",
      "94\t61685\t0.3\t1\t57457 4228\n",
      "95\t57220\t0.3\t1\t53201 4019\n",
      "96\t54819\t0.3\t1\t51015 3804\n",
      "97\t51023\t0.3\t1\t47551 3472\n",
      "98\t47615\t0.3\t1\t44173 3442\n",
      "99\t45999\t0.3\t1\t42533 3466\n",
      "100\t42439\t0.3\t1\t39454 2985\n",
      "101\t39032\t0.3\t1\t36135 2897\n",
      "102\t36673\t0.3\t1\t33901 2772\n",
      "103\t33569\t0.3\t1\t31176 2393\n",
      "104\t31500\t0.3\t1\t29209 2291\n",
      "105\t28075\t0.3\t1\t26011 2064\n",
      "106\t25509\t0.3\t1\t23615 1894\n",
      "107\t23441\t0.3\t1\t21761 1680\n",
      "108\t21666\t0.3\t1\t19921 1745\n",
      "109\t19354\t0.3\t1\t17847 1507\n",
      "110\t17443\t0.3\t1\t16060 1383\n",
      "111\t15864\t0.3\t1\t14569 1295\n",
      "112\t13943\t0.3\t1\t12833 1110\n",
      "113\t12164\t0.3\t1\t11191 973\n",
      "114\t10656\t0.3\t1\t9783 873\n",
      "115\t9192\t0.3\t1\t8432 760\n",
      "116\t8137\t0.3\t1\t7437 700\n",
      "117\t6646\t0.3\t1\t6079 567\n",
      "118\t6028\t0.3\t1\t5475 553\n",
      "119\t4941\t0.3\t1\t4488 453\n",
      "120\t4276\t0.3\t1\t3914 362\n",
      "121\t3804\t0.3\t1\t3440 364\n",
      "122\t2994\t0.3\t1\t2656 338\n",
      "123\t2568\t0.3\t1\t2298 270\n",
      "124\t2204\t0.3\t1\t2017 187\n",
      "125\t1837\t0.3\t1\t1657 180\n",
      "126\t1524\t0.3\t1\t1357 167\n",
      "127\t1388\t0.3\t1\t1226 162\n",
      "128\t1017\t0.3\t1\t890 127\n",
      "129\t886\t0.3\t1\t773 113\n",
      "130\t713\t0.3\t1\t646 67\n",
      "131\t614\t0.3\t1\t547 67\n",
      "132\t610\t0.3\t1\t554 56\n",
      "133\t738\t0.3\t1\t662 76\n",
      "134\t1198\t0.3\t1\t1126 72\n",
      "135\t479\t0.3\t1\t433 46\n",
      "136\t180\t0.3\t1\t151 29\n",
      "137\t120\t0.3\t1\t102 18\n",
      "138\t100\t0.3\t1\t79 21\n",
      "139\t54\t0.3\t1\t43 11\n",
      "140\t63\t0.3\t1\t49 14\n",
      "141\t34\t0.3\t1\t27 7\n",
      "142\t36\t0.3\t1\t27 9\n",
      "143\t19\t0.3\t1\t17 2\n",
      "144\t13\t0.3\t1\t11 2\n",
      "145\t29\t0.3\t1\t22 7\n",
      "146\t21\t0.3\t1\t18 3\n",
      "147\t72\t0.3\t1\t61 11\n",
      "148\t53\t0.3\t1\t46 7\n",
      "149\t48\t0.3\t1\t42 6\n",
      "150\t601\t0.3\t1\t530 71\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18056494 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_1_S9_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 727.12 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,056,494\n",
      "Reads with adapters:                13,833,491 (76.6%)\n",
      "Reads written (passing filters):    18,056,494 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,708,474,100 bp\n",
      "Quality-trimmed:              12,349,451 bp (0.5%)\n",
      "Total written (filtered):  2,141,212,582 bp (79.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13833491 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 12.5%\n",
      "  G: 24.0%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2568781\t4514123.5\t0\t2568781\n",
      "2\t366057\t1128530.9\t0\t366057\n",
      "3\t212077\t282132.7\t0\t212077\n",
      "4\t115719\t70533.2\t0\t115719\n",
      "5\t101735\t17633.3\t0\t101735\n",
      "6\t89696\t4408.3\t0\t89696\n",
      "7\t71264\t1102.1\t0\t71264\n",
      "8\t102329\t275.5\t0\t102329\n",
      "9\t81047\t68.9\t0\t80732 315\n",
      "10\t86697\t17.2\t1\t80878 5819\n",
      "11\t94697\t4.3\t1\t87025 7672\n",
      "12\t93974\t1.1\t1\t86609 7365\n",
      "13\t107111\t0.3\t1\t98790 8321\n",
      "14\t98830\t0.3\t1\t90363 8467\n",
      "15\t81863\t0.3\t1\t75568 6295\n",
      "16\t116617\t0.3\t1\t107560 9057\n",
      "17\t98434\t0.3\t1\t90554 7880\n",
      "18\t88102\t0.3\t1\t81367 6735\n",
      "19\t112551\t0.3\t1\t103902 8649\n",
      "20\t106499\t0.3\t1\t98559 7940\n",
      "21\t94529\t0.3\t1\t87451 7078\n",
      "22\t105525\t0.3\t1\t97752 7773\n",
      "23\t111250\t0.3\t1\t102948 8302\n",
      "24\t110289\t0.3\t1\t101801 8488\n",
      "25\t123454\t0.3\t1\t115174 8280\n",
      "26\t102700\t0.3\t1\t94614 8086\n",
      "27\t115670\t0.3\t1\t106461 9209\n",
      "28\t103454\t0.3\t1\t96996 6458\n",
      "29\t116676\t0.3\t1\t108241 8435\n",
      "30\t110369\t0.3\t1\t103124 7245\n",
      "31\t123948\t0.3\t1\t115651 8297\n",
      "32\t105488\t0.3\t1\t98700 6788\n",
      "33\t122972\t0.3\t1\t114148 8824\n",
      "34\t158602\t0.3\t1\t147860 10742\n",
      "35\t85176\t0.3\t1\t79388 5788\n",
      "36\t121188\t0.3\t1\t112686 8502\n",
      "37\t129603\t0.3\t1\t120852 8751\n",
      "38\t121650\t0.3\t1\t114412 7238\n",
      "39\t117499\t0.3\t1\t110622 6877\n",
      "40\t132022\t0.3\t1\t123436 8586\n",
      "41\t127439\t0.3\t1\t119665 7774\n",
      "42\t130897\t0.3\t1\t122691 8206\n",
      "43\t118466\t0.3\t1\t111262 7204\n",
      "44\t136805\t0.3\t1\t128115 8690\n",
      "45\t143267\t0.3\t1\t134250 9017\n",
      "46\t105232\t0.3\t1\t98689 6543\n",
      "47\t132632\t0.3\t1\t125125 7507\n",
      "48\t128580\t0.3\t1\t121138 7442\n",
      "49\t134590\t0.3\t1\t126196 8394\n",
      "50\t129680\t0.3\t1\t121594 8086\n",
      "51\t156936\t0.3\t1\t147378 9558\n",
      "52\t115575\t0.3\t1\t108787 6788\n",
      "53\t141220\t0.3\t1\t133568 7652\n",
      "54\t104058\t0.3\t1\t98240 5818\n",
      "55\t134625\t0.3\t1\t127372 7253\n",
      "56\t138566\t0.3\t1\t130360 8206\n",
      "57\t132925\t0.3\t1\t126016 6909\n",
      "58\t127424\t0.3\t1\t120849 6575\n",
      "59\t127371\t0.3\t1\t120396 6975\n",
      "60\t132296\t0.3\t1\t124969 7327\n",
      "61\t135056\t0.3\t1\t127808 7248\n",
      "62\t146657\t0.3\t1\t138388 8269\n",
      "63\t193947\t0.3\t1\t184650 9297\n",
      "64\t155657\t0.3\t1\t148867 6790\n",
      "65\t105922\t0.3\t1\t100517 5405\n",
      "66\t114816\t0.3\t1\t108826 5990\n",
      "67\t114246\t0.3\t1\t108023 6223\n",
      "68\t115039\t0.3\t1\t108957 6082\n",
      "69\t113573\t0.3\t1\t107431 6142\n",
      "70\t118689\t0.3\t1\t112276 6413\n",
      "71\t124010\t0.3\t1\t117553 6457\n",
      "72\t118164\t0.3\t1\t111917 6247\n",
      "73\t122662\t0.3\t1\t116340 6322\n",
      "74\t113578\t0.3\t1\t107524 6054\n",
      "75\t108812\t0.3\t1\t102905 5907\n",
      "76\t105084\t0.3\t1\t99284 5800\n",
      "77\t105545\t0.3\t1\t99708 5837\n",
      "78\t100872\t0.3\t1\t95568 5304\n",
      "79\t97026\t0.3\t1\t91844 5182\n",
      "80\t95305\t0.3\t1\t90356 4949\n",
      "81\t93857\t0.3\t1\t89014 4843\n",
      "82\t90295\t0.3\t1\t85582 4713\n",
      "83\t87301\t0.3\t1\t82821 4480\n",
      "84\t85471\t0.3\t1\t81218 4253\n",
      "85\t82978\t0.3\t1\t78704 4274\n",
      "86\t82245\t0.3\t1\t78140 4105\n",
      "87\t82525\t0.3\t1\t78515 4010\n",
      "88\t80619\t0.3\t1\t76785 3834\n",
      "89\t73370\t0.3\t1\t69724 3646\n",
      "90\t70068\t0.3\t1\t66584 3484\n",
      "91\t65835\t0.3\t1\t62601 3234\n",
      "92\t61556\t0.3\t1\t58566 2990\n",
      "93\t59482\t0.3\t1\t56574 2908\n",
      "94\t57109\t0.3\t1\t54269 2840\n",
      "95\t53973\t0.3\t1\t51267 2706\n",
      "96\t51596\t0.3\t1\t49048 2548\n",
      "97\t47434\t0.3\t1\t45017 2417\n",
      "98\t44668\t0.3\t1\t42540 2128\n",
      "99\t43184\t0.3\t1\t41061 2123\n",
      "100\t40483\t0.3\t1\t38511 1972\n",
      "101\t37015\t0.3\t1\t35171 1844\n",
      "102\t34179\t0.3\t1\t32525 1654\n",
      "103\t31254\t0.3\t1\t29799 1455\n",
      "104\t30003\t0.3\t1\t28488 1515\n",
      "105\t26714\t0.3\t1\t25377 1337\n",
      "106\t24481\t0.3\t1\t23332 1149\n",
      "107\t22254\t0.3\t1\t21123 1131\n",
      "108\t20003\t0.3\t1\t19023 980\n",
      "109\t18094\t0.3\t1\t17150 944\n",
      "110\t16326\t0.3\t1\t15479 847\n",
      "111\t15507\t0.3\t1\t14783 724\n",
      "112\t13504\t0.3\t1\t12807 697\n",
      "113\t11779\t0.3\t1\t11204 575\n",
      "114\t10207\t0.3\t1\t9684 523\n",
      "115\t8697\t0.3\t1\t8286 411\n",
      "116\t7460\t0.3\t1\t7104 356\n",
      "117\t6031\t0.3\t1\t5703 328\n",
      "118\t5441\t0.3\t1\t5182 259\n",
      "119\t4540\t0.3\t1\t4315 225\n",
      "120\t3964\t0.3\t1\t3742 222\n",
      "121\t3581\t0.3\t1\t3393 188\n",
      "122\t2805\t0.3\t1\t2657 148\n",
      "123\t2429\t0.3\t1\t2278 151\n",
      "124\t2051\t0.3\t1\t1954 97\n",
      "125\t1777\t0.3\t1\t1683 94\n",
      "126\t1416\t0.3\t1\t1338 78\n",
      "127\t1308\t0.3\t1\t1214 94\n",
      "128\t919\t0.3\t1\t855 64\n",
      "129\t823\t0.3\t1\t781 42\n",
      "130\t653\t0.3\t1\t614 39\n",
      "131\t581\t0.3\t1\t548 33\n",
      "132\t573\t0.3\t1\t541 32\n",
      "133\t789\t0.3\t1\t754 35\n",
      "134\t1231\t0.3\t1\t1179 52\n",
      "135\t458\t0.3\t1\t447 11\n",
      "136\t175\t0.3\t1\t164 11\n",
      "137\t112\t0.3\t1\t103 9\n",
      "138\t94\t0.3\t1\t84 10\n",
      "139\t57\t0.3\t1\t53 4\n",
      "140\t65\t0.3\t1\t58 7\n",
      "141\t32\t0.3\t1\t29 3\n",
      "142\t35\t0.3\t1\t31 4\n",
      "143\t17\t0.3\t1\t15 2\n",
      "144\t11\t0.3\t1\t9 2\n",
      "145\t30\t0.3\t1\t28 2\n",
      "146\t19\t0.3\t1\t19\n",
      "147\t64\t0.3\t1\t61 3\n",
      "148\t47\t0.3\t1\t44 3\n",
      "149\t34\t0.3\t1\t32 2\n",
      "150\t415\t0.3\t1\t377 38\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18056494 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz, file_2: 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18056494\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 32256 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_1_S9_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_1_S9_L002_R1_001_trimmed.fq.gz and 8C_26psu_1_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_2_S10_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 672.97 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,482,577\n",
      "Reads with adapters:                14,846,196 (80.3%)\n",
      "Reads written (passing filters):    18,482,577 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,772,386,550 bp\n",
      "Quality-trimmed:               5,236,502 bp (0.2%)\n",
      "Total written (filtered):  2,109,664,469 bp (76.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14846196 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 14.7%\n",
      "  G: 20.4%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2201281\t4620644.2\t0\t2201281\n",
      "2\t324820\t1155161.1\t0\t324820\n",
      "3\t181187\t288790.3\t0\t181187\n",
      "4\t111024\t72197.6\t0\t111024\n",
      "5\t91444\t18049.4\t0\t91444\n",
      "6\t89339\t4512.3\t0\t89339\n",
      "7\t75499\t1128.1\t0\t75499\n",
      "8\t92079\t282.0\t0\t92079\n",
      "9\t94613\t70.5\t0\t93305 1308\n",
      "10\t87393\t17.6\t1\t82726 4667\n",
      "11\t97272\t4.4\t1\t89821 7451\n",
      "12\t94618\t1.1\t1\t88577 6041\n",
      "13\t94580\t0.3\t1\t88291 6289\n",
      "14\t98608\t0.3\t1\t91401 7207\n",
      "15\t100703\t0.3\t1\t94036 6667\n",
      "16\t100759\t0.3\t1\t93186 7573\n",
      "17\t122345\t0.3\t1\t112475 9870\n",
      "18\t103137\t0.3\t1\t96868 6269\n",
      "19\t89996\t0.3\t1\t84224 5772\n",
      "20\t107642\t0.3\t1\t100714 6928\n",
      "21\t118859\t0.3\t1\t110501 8358\n",
      "22\t101938\t0.3\t1\t96023 5915\n",
      "23\t117604\t0.3\t1\t109853 7751\n",
      "24\t110878\t0.3\t1\t101488 9390\n",
      "25\t118235\t0.3\t1\t110039 8196\n",
      "26\t108371\t0.3\t1\t101787 6584\n",
      "27\t121571\t0.3\t1\t112662 8909\n",
      "28\t119395\t0.3\t1\t110884 8511\n",
      "29\t129888\t0.3\t1\t119161 10727\n",
      "30\t136399\t0.3\t1\t128723 7676\n",
      "31\t101982\t0.3\t1\t95349 6633\n",
      "32\t129923\t0.3\t1\t121526 8397\n",
      "33\t121306\t0.3\t1\t112461 8845\n",
      "34\t134618\t0.3\t1\t125125 9493\n",
      "35\t135522\t0.3\t1\t127399 8123\n",
      "36\t126532\t0.3\t1\t116217 10315\n",
      "37\t132624\t0.3\t1\t124414 8210\n",
      "38\t135436\t0.3\t1\t126589 8847\n",
      "39\t144343\t0.3\t1\t135079 9264\n",
      "40\t131706\t0.3\t1\t123113 8593\n",
      "41\t147103\t0.3\t1\t136472 10631\n",
      "42\t135693\t0.3\t1\t124813 10880\n",
      "43\t208641\t0.3\t1\t197928 10713\n",
      "44\t70084\t0.3\t1\t63748 6336\n",
      "45\t146631\t0.3\t1\t137638 8993\n",
      "46\t134243\t0.3\t1\t125635 8608\n",
      "47\t148092\t0.3\t1\t138684 9408\n",
      "48\t143207\t0.3\t1\t133950 9257\n",
      "49\t157250\t0.3\t1\t146216 11034\n",
      "50\t135654\t0.3\t1\t127338 8316\n",
      "51\t145480\t0.3\t1\t136047 9433\n",
      "52\t151730\t0.3\t1\t141076 10654\n",
      "53\t160594\t0.3\t1\t150536 10058\n",
      "54\t131608\t0.3\t1\t123193 8415\n",
      "55\t153566\t0.3\t1\t143551 10015\n",
      "56\t188903\t0.3\t1\t177823 11080\n",
      "57\t122969\t0.3\t1\t114039 8930\n",
      "58\t155982\t0.3\t1\t147471 8511\n",
      "59\t136864\t0.3\t1\t128026 8838\n",
      "60\t152804\t0.3\t1\t142938 9866\n",
      "61\t149972\t0.3\t1\t141027 8945\n",
      "62\t164460\t0.3\t1\t154790 9670\n",
      "63\t131600\t0.3\t1\t122512 9088\n",
      "64\t147097\t0.3\t1\t138434 8663\n",
      "65\t163813\t0.3\t1\t153627 10186\n",
      "66\t170976\t0.3\t1\t160133 10843\n",
      "67\t181725\t0.3\t1\t167572 14153\n",
      "68\t347424\t0.3\t1\t330644 16780\n",
      "69\t53801\t0.3\t1\t49133 4668\n",
      "70\t21119\t0.3\t1\t17296 3823\n",
      "71\t93829\t0.3\t1\t88082 5747\n",
      "72\t129343\t0.3\t1\t121746 7597\n",
      "73\t137109\t0.3\t1\t128961 8148\n",
      "74\t135639\t0.3\t1\t127288 8351\n",
      "75\t134319\t0.3\t1\t126050 8269\n",
      "76\t133485\t0.3\t1\t125306 8179\n",
      "77\t133818\t0.3\t1\t125641 8177\n",
      "78\t125826\t0.3\t1\t118003 7823\n",
      "79\t124061\t0.3\t1\t116366 7695\n",
      "80\t122381\t0.3\t1\t114907 7474\n",
      "81\t121340\t0.3\t1\t113873 7467\n",
      "82\t118564\t0.3\t1\t111221 7343\n",
      "83\t114400\t0.3\t1\t107282 7118\n",
      "84\t110060\t0.3\t1\t103458 6602\n",
      "85\t107144\t0.3\t1\t100729 6415\n",
      "86\t104597\t0.3\t1\t98129 6468\n",
      "87\t99965\t0.3\t1\t93811 6154\n",
      "88\t99634\t0.3\t1\t93629 6005\n",
      "89\t94213\t0.3\t1\t88311 5902\n",
      "90\t93784\t0.3\t1\t87912 5872\n",
      "91\t88966\t0.3\t1\t83468 5498\n",
      "92\t84640\t0.3\t1\t79309 5331\n",
      "93\t79324\t0.3\t1\t74386 4938\n",
      "94\t77199\t0.3\t1\t72415 4784\n",
      "95\t72155\t0.3\t1\t67634 4521\n",
      "96\t68840\t0.3\t1\t64535 4305\n",
      "97\t64425\t0.3\t1\t60326 4099\n",
      "98\t60153\t0.3\t1\t56199 3954\n",
      "99\t58789\t0.3\t1\t54783 4006\n",
      "100\t53231\t0.3\t1\t49872 3359\n",
      "101\t49747\t0.3\t1\t46474 3273\n",
      "102\t46514\t0.3\t1\t43441 3073\n",
      "103\t42759\t0.3\t1\t39961 2798\n",
      "104\t40408\t0.3\t1\t37725 2683\n",
      "105\t35915\t0.3\t1\t33458 2457\n",
      "106\t33343\t0.3\t1\t31066 2277\n",
      "107\t30056\t0.3\t1\t28019 2037\n",
      "108\t28759\t0.3\t1\t26783 1976\n",
      "109\t25234\t0.3\t1\t23454 1780\n",
      "110\t22936\t0.3\t1\t21306 1630\n",
      "111\t20750\t0.3\t1\t19240 1510\n",
      "112\t17996\t0.3\t1\t16644 1352\n",
      "113\t15908\t0.3\t1\t14749 1159\n",
      "114\t14075\t0.3\t1\t12991 1084\n",
      "115\t12424\t0.3\t1\t11459 965\n",
      "116\t10747\t0.3\t1\t9910 837\n",
      "117\t9113\t0.3\t1\t8333 780\n",
      "118\t8109\t0.3\t1\t7445 664\n",
      "119\t6883\t0.3\t1\t6316 567\n",
      "120\t5713\t0.3\t1\t5209 504\n",
      "121\t5168\t0.3\t1\t4711 457\n",
      "122\t4185\t0.3\t1\t3804 381\n",
      "123\t3552\t0.3\t1\t3218 334\n",
      "124\t3100\t0.3\t1\t2796 304\n",
      "125\t2576\t0.3\t1\t2315 261\n",
      "126\t2168\t0.3\t1\t1937 231\n",
      "127\t1692\t0.3\t1\t1526 166\n",
      "128\t1332\t0.3\t1\t1165 167\n",
      "129\t1229\t0.3\t1\t1107 122\n",
      "130\t923\t0.3\t1\t838 85\n",
      "131\t751\t0.3\t1\t663 88\n",
      "132\t654\t0.3\t1\t579 75\n",
      "133\t679\t0.3\t1\t607 72\n",
      "134\t1033\t0.3\t1\t947 86\n",
      "135\t428\t0.3\t1\t390 38\n",
      "136\t228\t0.3\t1\t207 21\n",
      "137\t170\t0.3\t1\t154 16\n",
      "138\t136\t0.3\t1\t118 18\n",
      "139\t81\t0.3\t1\t70 11\n",
      "140\t68\t0.3\t1\t55 13\n",
      "141\t49\t0.3\t1\t43 6\n",
      "142\t55\t0.3\t1\t47 8\n",
      "143\t45\t0.3\t1\t36 9\n",
      "144\t21\t0.3\t1\t19 2\n",
      "145\t58\t0.3\t1\t54 4\n",
      "146\t122\t0.3\t1\t110 12\n",
      "147\t390\t0.3\t1\t355 35\n",
      "148\t248\t0.3\t1\t225 23\n",
      "149\t170\t0.3\t1\t149 21\n",
      "150\t1709\t0.3\t1\t1353 356\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18482577 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_2_S10_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 708.77 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,482,577\n",
      "Reads with adapters:                14,753,598 (79.8%)\n",
      "Reads written (passing filters):    18,482,577 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,772,386,550 bp\n",
      "Quality-trimmed:              14,899,580 bp (0.5%)\n",
      "Total written (filtered):  2,113,888,686 bp (76.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14753598 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 12.5%\n",
      "  G: 24.2%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2286581\t4620644.2\t0\t2286581\n",
      "2\t330310\t1155161.1\t0\t330310\n",
      "3\t190033\t288790.3\t0\t190033\n",
      "4\t111417\t72197.6\t0\t111417\n",
      "5\t101556\t18049.4\t0\t101556\n",
      "6\t87150\t4512.3\t0\t87150\n",
      "7\t71559\t1128.1\t0\t71559\n",
      "8\t101450\t282.0\t0\t101450\n",
      "9\t81706\t70.5\t0\t81446 260\n",
      "10\t88967\t17.6\t1\t84555 4412\n",
      "11\t93801\t4.4\t1\t88117 5684\n",
      "12\t95662\t1.1\t1\t89704 5958\n",
      "13\t105924\t0.3\t1\t99758 6166\n",
      "14\t100634\t0.3\t1\t93826 6808\n",
      "15\t81502\t0.3\t1\t76286 5216\n",
      "16\t118419\t0.3\t1\t111030 7389\n",
      "17\t95042\t0.3\t1\t88712 6330\n",
      "18\t90615\t0.3\t1\t85225 5390\n",
      "19\t115837\t0.3\t1\t108486 7351\n",
      "20\t108535\t0.3\t1\t102151 6384\n",
      "21\t96158\t0.3\t1\t90241 5917\n",
      "22\t109681\t0.3\t1\t102998 6683\n",
      "23\t113064\t0.3\t1\t106255 6809\n",
      "24\t112402\t0.3\t1\t105369 7033\n",
      "25\t127609\t0.3\t1\t120247 7362\n",
      "26\t106198\t0.3\t1\t99534 6664\n",
      "27\t117379\t0.3\t1\t109394 7985\n",
      "28\t111310\t0.3\t1\t105607 5703\n",
      "29\t119257\t0.3\t1\t111876 7381\n",
      "30\t117422\t0.3\t1\t111230 6192\n",
      "31\t127799\t0.3\t1\t120363 7436\n",
      "32\t115547\t0.3\t1\t109515 6032\n",
      "33\t127479\t0.3\t1\t119672 7807\n",
      "34\t163653\t0.3\t1\t154089 9564\n",
      "35\t93410\t0.3\t1\t88452 4958\n",
      "36\t129603\t0.3\t1\t121811 7792\n",
      "37\t138172\t0.3\t1\t130055 8117\n",
      "38\t129571\t0.3\t1\t123089 6482\n",
      "39\t126899\t0.3\t1\t120584 6315\n",
      "40\t141131\t0.3\t1\t133007 8124\n",
      "41\t136856\t0.3\t1\t129602 7254\n",
      "42\t140475\t0.3\t1\t133264 7211\n",
      "43\t128682\t0.3\t1\t121911 6771\n",
      "44\t148231\t0.3\t1\t140352 7879\n",
      "45\t152393\t0.3\t1\t144497 7896\n",
      "46\t120446\t0.3\t1\t114128 6318\n",
      "47\t146135\t0.3\t1\t138949 7186\n",
      "48\t143271\t0.3\t1\t136268 7003\n",
      "49\t150045\t0.3\t1\t142543 7502\n",
      "50\t141618\t0.3\t1\t134295 7323\n",
      "51\t177184\t0.3\t1\t168273 8911\n",
      "52\t127868\t0.3\t1\t121986 5882\n",
      "53\t154234\t0.3\t1\t147269 6965\n",
      "54\t117590\t0.3\t1\t112052 5538\n",
      "55\t152743\t0.3\t1\t146054 6689\n",
      "56\t157680\t0.3\t1\t150094 7586\n",
      "57\t149551\t0.3\t1\t143020 6531\n",
      "58\t143483\t0.3\t1\t137470 6013\n",
      "59\t145998\t0.3\t1\t139352 6646\n",
      "60\t151232\t0.3\t1\t144310 6922\n",
      "61\t157550\t0.3\t1\t150162 7388\n",
      "62\t169489\t0.3\t1\t161516 7973\n",
      "63\t228128\t0.3\t1\t218668 9460\n",
      "64\t191978\t0.3\t1\t184960 7018\n",
      "65\t121739\t0.3\t1\t116395 5344\n",
      "66\t132352\t0.3\t1\t126524 5828\n",
      "67\t131028\t0.3\t1\t125146 5882\n",
      "68\t132829\t0.3\t1\t127004 5825\n",
      "69\t131726\t0.3\t1\t125866 5860\n",
      "70\t137789\t0.3\t1\t131635 6154\n",
      "71\t147946\t0.3\t1\t141769 6177\n",
      "72\t139415\t0.3\t1\t133242 6173\n",
      "73\t148725\t0.3\t1\t142498 6227\n",
      "74\t135207\t0.3\t1\t129643 5564\n",
      "75\t128520\t0.3\t1\t123034 5486\n",
      "76\t126065\t0.3\t1\t120765 5300\n",
      "77\t126228\t0.3\t1\t120717 5511\n",
      "78\t120120\t0.3\t1\t114978 5142\n",
      "79\t115495\t0.3\t1\t110595 4900\n",
      "80\t116213\t0.3\t1\t111348 4865\n",
      "81\t114795\t0.3\t1\t109987 4808\n",
      "82\t110451\t0.3\t1\t105814 4637\n",
      "83\t106907\t0.3\t1\t102515 4392\n",
      "84\t102431\t0.3\t1\t98006 4425\n",
      "85\t101678\t0.3\t1\t97283 4395\n",
      "86\t102018\t0.3\t1\t97662 4356\n",
      "87\t102379\t0.3\t1\t98265 4114\n",
      "88\t100269\t0.3\t1\t95923 4346\n",
      "89\t89766\t0.3\t1\t85935 3831\n",
      "90\t86353\t0.3\t1\t82825 3528\n",
      "91\t80857\t0.3\t1\t77356 3501\n",
      "92\t76306\t0.3\t1\t73107 3199\n",
      "93\t73595\t0.3\t1\t70492 3103\n",
      "94\t71587\t0.3\t1\t68546 3041\n",
      "95\t68110\t0.3\t1\t65320 2790\n",
      "96\t64867\t0.3\t1\t62264 2603\n",
      "97\t59245\t0.3\t1\t56835 2410\n",
      "98\t55305\t0.3\t1\t52942 2363\n",
      "99\t53891\t0.3\t1\t51732 2159\n",
      "100\t50005\t0.3\t1\t47952 2053\n",
      "101\t45894\t0.3\t1\t44011 1883\n",
      "102\t43298\t0.3\t1\t41576 1722\n",
      "103\t39322\t0.3\t1\t37781 1541\n",
      "104\t38059\t0.3\t1\t36523 1536\n",
      "105\t33918\t0.3\t1\t32538 1380\n",
      "106\t31438\t0.3\t1\t30241 1197\n",
      "107\t28204\t0.3\t1\t27113 1091\n",
      "108\t26258\t0.3\t1\t25170 1088\n",
      "109\t23534\t0.3\t1\t22556 978\n",
      "110\t21525\t0.3\t1\t20673 852\n",
      "111\t20335\t0.3\t1\t19508 827\n",
      "112\t17540\t0.3\t1\t16818 722\n",
      "113\t15580\t0.3\t1\t14966 614\n",
      "114\t13626\t0.3\t1\t13076 550\n",
      "115\t11545\t0.3\t1\t11044 501\n",
      "116\t9893\t0.3\t1\t9487 406\n",
      "117\t8220\t0.3\t1\t7857 363\n",
      "118\t7416\t0.3\t1\t7106 310\n",
      "119\t6364\t0.3\t1\t6067 297\n",
      "120\t5214\t0.3\t1\t4965 249\n",
      "121\t4800\t0.3\t1\t4601 199\n",
      "122\t3856\t0.3\t1\t3683 173\n",
      "123\t3424\t0.3\t1\t3271 153\n",
      "124\t2951\t0.3\t1\t2795 156\n",
      "125\t2462\t0.3\t1\t2355 107\n",
      "126\t2026\t0.3\t1\t1922 104\n",
      "127\t1586\t0.3\t1\t1509 77\n",
      "128\t1233\t0.3\t1\t1162 71\n",
      "129\t1142\t0.3\t1\t1091 51\n",
      "130\t841\t0.3\t1\t788 53\n",
      "131\t705\t0.3\t1\t671 34\n",
      "132\t609\t0.3\t1\t572 37\n",
      "133\t673\t0.3\t1\t638 35\n",
      "134\t1003\t0.3\t1\t960 43\n",
      "135\t407\t0.3\t1\t396 11\n",
      "136\t221\t0.3\t1\t202 19\n",
      "137\t162\t0.3\t1\t156 6\n",
      "138\t117\t0.3\t1\t109 8\n",
      "139\t71\t0.3\t1\t63 8\n",
      "140\t67\t0.3\t1\t63 4\n",
      "141\t48\t0.3\t1\t47 1\n",
      "142\t50\t0.3\t1\t49 1\n",
      "143\t41\t0.3\t1\t37 4\n",
      "144\t23\t0.3\t1\t20 3\n",
      "145\t62\t0.3\t1\t59 3\n",
      "146\t126\t0.3\t1\t123 3\n",
      "147\t381\t0.3\t1\t366 15\n",
      "148\t234\t0.3\t1\t226 8\n",
      "149\t166\t0.3\t1\t155 11\n",
      "150\t1412\t0.3\t1\t1344 68\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18482577 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz, file_2: 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18482577\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 43255 (0.23%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_2_S10_L001_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_2_S10_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 680.28 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,388,474\n",
      "Reads with adapters:                14,774,548 (80.3%)\n",
      "Reads written (passing filters):    18,388,474 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,758,271,100 bp\n",
      "Quality-trimmed:               5,422,014 bp (0.2%)\n",
      "Total written (filtered):  2,096,179,695 bp (76.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14774548 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 14.7%\n",
      "  G: 20.4%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2179283\t4597118.5\t0\t2179283\n",
      "2\t319389\t1149279.6\t0\t319389\n",
      "3\t179573\t287319.9\t0\t179573\n",
      "4\t108126\t71830.0\t0\t108126\n",
      "5\t90944\t17957.5\t0\t90944\n",
      "6\t89410\t4489.4\t0\t89410\n",
      "7\t72797\t1122.3\t0\t72797\n",
      "8\t91422\t280.6\t0\t91422\n",
      "9\t93678\t70.1\t0\t92415 1263\n",
      "10\t84673\t17.5\t1\t79267 5406\n",
      "11\t97249\t4.4\t1\t88769 8480\n",
      "12\t93054\t1.1\t1\t86160 6894\n",
      "13\t93570\t0.3\t1\t86447 7123\n",
      "14\t97334\t0.3\t1\t89025 8309\n",
      "15\t99065\t0.3\t1\t91404 7661\n",
      "16\t99365\t0.3\t1\t90487 8878\n",
      "17\t122566\t0.3\t1\t111250 11316\n",
      "18\t102597\t0.3\t1\t95121 7476\n",
      "19\t88093\t0.3\t1\t81291 6802\n",
      "20\t107045\t0.3\t1\t98814 8231\n",
      "21\t119128\t0.3\t1\t109388 9740\n",
      "22\t98363\t0.3\t1\t91503 6860\n",
      "23\t118120\t0.3\t1\t108869 9251\n",
      "24\t109512\t0.3\t1\t99530 9982\n",
      "25\t117620\t0.3\t1\t108476 9144\n",
      "26\t107233\t0.3\t1\t99745 7488\n",
      "27\t121679\t0.3\t1\t111586 10093\n",
      "28\t118815\t0.3\t1\t109364 9451\n",
      "29\t129909\t0.3\t1\t118061 11848\n",
      "30\t136643\t0.3\t1\t127560 9083\n",
      "31\t99248\t0.3\t1\t91716 7532\n",
      "32\t129461\t0.3\t1\t119254 10207\n",
      "33\t120389\t0.3\t1\t110237 10152\n",
      "34\t133674\t0.3\t1\t122112 11562\n",
      "35\t135326\t0.3\t1\t125969 9357\n",
      "36\t125558\t0.3\t1\t113748 11810\n",
      "37\t131811\t0.3\t1\t122131 9680\n",
      "38\t134596\t0.3\t1\t124192 10404\n",
      "39\t144370\t0.3\t1\t133180 11190\n",
      "40\t130508\t0.3\t1\t120297 10211\n",
      "41\t146230\t0.3\t1\t133678 12552\n",
      "42\t134185\t0.3\t1\t121889 12296\n",
      "43\t205437\t0.3\t1\t192857 12580\n",
      "44\t71406\t0.3\t1\t64241 7165\n",
      "45\t146811\t0.3\t1\t136113 10698\n",
      "46\t133431\t0.3\t1\t123482 9949\n",
      "47\t146963\t0.3\t1\t136319 10644\n",
      "48\t142772\t0.3\t1\t132090 10682\n",
      "49\t156871\t0.3\t1\t144261 12610\n",
      "50\t134522\t0.3\t1\t125136 9386\n",
      "51\t145234\t0.3\t1\t134471 10763\n",
      "52\t151741\t0.3\t1\t139780 11961\n",
      "53\t160323\t0.3\t1\t148993 11330\n",
      "54\t129391\t0.3\t1\t119763 9628\n",
      "55\t153504\t0.3\t1\t142065 11439\n",
      "56\t189438\t0.3\t1\t176473 12965\n",
      "57\t121885\t0.3\t1\t111816 10069\n",
      "58\t156277\t0.3\t1\t146133 10144\n",
      "59\t138166\t0.3\t1\t127798 10368\n",
      "60\t151809\t0.3\t1\t140696 11113\n",
      "61\t148378\t0.3\t1\t138229 10149\n",
      "62\t164350\t0.3\t1\t153017 11333\n",
      "63\t132176\t0.3\t1\t121776 10400\n",
      "64\t146283\t0.3\t1\t136343 9940\n",
      "65\t162567\t0.3\t1\t150970 11597\n",
      "66\t169475\t0.3\t1\t157430 12045\n",
      "67\t180215\t0.3\t1\t164843 15372\n",
      "68\t335801\t0.3\t1\t317089 18712\n",
      "69\t66723\t0.3\t1\t60845 5878\n",
      "70\t26090\t0.3\t1\t21811 4279\n",
      "71\t91663\t0.3\t1\t85518 6145\n",
      "72\t129140\t0.3\t1\t120846 8294\n",
      "73\t135992\t0.3\t1\t126800 9192\n",
      "74\t136581\t0.3\t1\t127154 9427\n",
      "75\t134341\t0.3\t1\t124926 9415\n",
      "76\t133080\t0.3\t1\t123316 9764\n",
      "77\t133409\t0.3\t1\t123959 9450\n",
      "78\t126607\t0.3\t1\t117538 9069\n",
      "79\t125070\t0.3\t1\t116034 9036\n",
      "80\t121818\t0.3\t1\t113223 8595\n",
      "81\t121825\t0.3\t1\t113140 8685\n",
      "82\t118244\t0.3\t1\t109604 8640\n",
      "83\t114640\t0.3\t1\t106502 8138\n",
      "84\t109963\t0.3\t1\t101985 7978\n",
      "85\t107204\t0.3\t1\t99478 7726\n",
      "86\t104741\t0.3\t1\t97192 7549\n",
      "87\t100621\t0.3\t1\t93543 7078\n",
      "88\t98776\t0.3\t1\t91675 7101\n",
      "89\t94677\t0.3\t1\t87957 6720\n",
      "90\t94332\t0.3\t1\t87620 6712\n",
      "91\t89612\t0.3\t1\t83296 6316\n",
      "92\t85598\t0.3\t1\t79627 5971\n",
      "93\t79579\t0.3\t1\t73824 5755\n",
      "94\t77934\t0.3\t1\t72425 5509\n",
      "95\t72110\t0.3\t1\t66875 5235\n",
      "96\t68840\t0.3\t1\t63845 4995\n",
      "97\t64799\t0.3\t1\t60100 4699\n",
      "98\t60773\t0.3\t1\t56257 4516\n",
      "99\t58381\t0.3\t1\t53874 4507\n",
      "100\t53848\t0.3\t1\t49904 3944\n",
      "101\t49943\t0.3\t1\t46245 3698\n",
      "102\t46241\t0.3\t1\t42732 3509\n",
      "103\t43196\t0.3\t1\t39990 3206\n",
      "104\t40167\t0.3\t1\t37244 2923\n",
      "105\t35834\t0.3\t1\t33163 2671\n",
      "106\t33023\t0.3\t1\t30489 2534\n",
      "107\t30207\t0.3\t1\t27948 2259\n",
      "108\t28472\t0.3\t1\t26245 2227\n",
      "109\t25094\t0.3\t1\t23102 1992\n",
      "110\t22988\t0.3\t1\t21194 1794\n",
      "111\t20538\t0.3\t1\t18949 1589\n",
      "112\t18521\t0.3\t1\t17080 1441\n",
      "113\t15947\t0.3\t1\t14645 1302\n",
      "114\t14330\t0.3\t1\t13057 1273\n",
      "115\t12384\t0.3\t1\t11329 1055\n",
      "116\t11096\t0.3\t1\t10163 933\n",
      "117\t9114\t0.3\t1\t8363 751\n",
      "118\t8187\t0.3\t1\t7390 797\n",
      "119\t6803\t0.3\t1\t6183 620\n",
      "120\t5832\t0.3\t1\t5301 531\n",
      "121\t5072\t0.3\t1\t4620 452\n",
      "122\t4206\t0.3\t1\t3803 403\n",
      "123\t3440\t0.3\t1\t3104 336\n",
      "124\t3042\t0.3\t1\t2726 316\n",
      "125\t2501\t0.3\t1\t2242 259\n",
      "126\t2111\t0.3\t1\t1904 207\n",
      "127\t1791\t0.3\t1\t1610 181\n",
      "128\t1444\t0.3\t1\t1292 152\n",
      "129\t1148\t0.3\t1\t1034 114\n",
      "130\t877\t0.3\t1\t785 92\n",
      "131\t693\t0.3\t1\t607 86\n",
      "132\t646\t0.3\t1\t559 87\n",
      "133\t689\t0.3\t1\t596 93\n",
      "134\t1090\t0.3\t1\t1005 85\n",
      "135\t492\t0.3\t1\t440 52\n",
      "136\t216\t0.3\t1\t190 26\n",
      "137\t184\t0.3\t1\t157 27\n",
      "138\t124\t0.3\t1\t103 21\n",
      "139\t70\t0.3\t1\t60 10\n",
      "140\t61\t0.3\t1\t49 12\n",
      "141\t35\t0.3\t1\t30 5\n",
      "142\t23\t0.3\t1\t21 2\n",
      "143\t27\t0.3\t1\t25 2\n",
      "144\t40\t0.3\t1\t34 6\n",
      "145\t86\t0.3\t1\t71 15\n",
      "146\t101\t0.3\t1\t97 4\n",
      "147\t403\t0.3\t1\t365 38\n",
      "148\t275\t0.3\t1\t252 23\n",
      "149\t182\t0.3\t1\t169 13\n",
      "150\t1714\t0.3\t1\t1347 367\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18388474 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_2_S10_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 719.84 s (39 us/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,388,474\n",
      "Reads with adapters:                14,677,085 (79.8%)\n",
      "Reads written (passing filters):    18,388,474 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,758,271,100 bp\n",
      "Quality-trimmed:              14,282,591 bp (0.5%)\n",
      "Total written (filtered):  2,101,274,280 bp (76.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14677085 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 12.6%\n",
      "  G: 24.2%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2268778\t4597118.5\t0\t2268778\n",
      "2\t320662\t1149279.6\t0\t320662\n",
      "3\t193123\t287319.9\t0\t193123\n",
      "4\t107505\t71830.0\t0\t107505\n",
      "5\t98136\t17957.5\t0\t98136\n",
      "6\t87605\t4489.4\t0\t87605\n",
      "7\t69984\t1122.3\t0\t69984\n",
      "8\t100736\t280.6\t0\t100736\n",
      "9\t80882\t70.1\t0\t80636 246\n",
      "10\t85130\t17.5\t1\t79348 5782\n",
      "11\t93238\t4.4\t1\t85689 7549\n",
      "12\t92545\t1.1\t1\t85062 7483\n",
      "13\t106121\t0.3\t1\t97648 8473\n",
      "14\t98586\t0.3\t1\t90015 8571\n",
      "15\t79603\t0.3\t1\t73456 6147\n",
      "16\t116674\t0.3\t1\t107221 9453\n",
      "17\t98395\t0.3\t1\t90346 8049\n",
      "18\t87513\t0.3\t1\t80658 6855\n",
      "19\t114688\t0.3\t1\t105582 9106\n",
      "20\t107373\t0.3\t1\t99285 8088\n",
      "21\t94108\t0.3\t1\t86828 7280\n",
      "22\t106536\t0.3\t1\t98497 8039\n",
      "23\t113709\t0.3\t1\t104976 8733\n",
      "24\t111940\t0.3\t1\t103269 8671\n",
      "25\t127403\t0.3\t1\t118648 8755\n",
      "26\t106762\t0.3\t1\t98223 8539\n",
      "27\t118132\t0.3\t1\t108645 9487\n",
      "28\t106930\t0.3\t1\t99955 6975\n",
      "29\t120647\t0.3\t1\t111606 9041\n",
      "30\t115537\t0.3\t1\t107678 7859\n",
      "31\t127962\t0.3\t1\t118733 9229\n",
      "32\t110371\t0.3\t1\t102883 7488\n",
      "33\t128535\t0.3\t1\t119135 9400\n",
      "34\t166074\t0.3\t1\t154379 11695\n",
      "35\t88153\t0.3\t1\t82039 6114\n",
      "36\t130243\t0.3\t1\t120604 9639\n",
      "37\t137437\t0.3\t1\t127611 9826\n",
      "38\t129360\t0.3\t1\t121250 8110\n",
      "39\t122871\t0.3\t1\t115373 7498\n",
      "40\t140913\t0.3\t1\t131295 9618\n",
      "41\t137375\t0.3\t1\t128487 8888\n",
      "42\t139853\t0.3\t1\t130785 9068\n",
      "43\t126879\t0.3\t1\t118647 8232\n",
      "44\t149301\t0.3\t1\t139495 9806\n",
      "45\t151738\t0.3\t1\t141722 10016\n",
      "46\t118782\t0.3\t1\t111205 7577\n",
      "47\t144918\t0.3\t1\t136263 8655\n",
      "48\t142402\t0.3\t1\t133845 8557\n",
      "49\t148370\t0.3\t1\t138956 9414\n",
      "50\t143101\t0.3\t1\t133924 9177\n",
      "51\t175097\t0.3\t1\t164092 11005\n",
      "52\t127562\t0.3\t1\t119653 7909\n",
      "53\t156552\t0.3\t1\t147706 8846\n",
      "54\t114039\t0.3\t1\t107283 6756\n",
      "55\t151265\t0.3\t1\t142708 8557\n",
      "56\t156754\t0.3\t1\t146983 9771\n",
      "57\t149767\t0.3\t1\t141656 8111\n",
      "58\t144481\t0.3\t1\t136475 8006\n",
      "59\t145577\t0.3\t1\t137588 7989\n",
      "60\t151210\t0.3\t1\t142573 8637\n",
      "61\t154000\t0.3\t1\t145457 8543\n",
      "62\t169208\t0.3\t1\t159141 10067\n",
      "63\t223876\t0.3\t1\t212453 11423\n",
      "64\t187828\t0.3\t1\t179799 8029\n",
      "65\t121115\t0.3\t1\t114588 6527\n",
      "66\t133231\t0.3\t1\t126151 7080\n",
      "67\t130874\t0.3\t1\t123245 7629\n",
      "68\t134376\t0.3\t1\t127002 7374\n",
      "69\t132637\t0.3\t1\t125114 7523\n",
      "70\t138657\t0.3\t1\t130979 7678\n",
      "71\t148179\t0.3\t1\t140033 8146\n",
      "72\t141424\t0.3\t1\t133547 7877\n",
      "73\t147578\t0.3\t1\t139587 7991\n",
      "74\t136704\t0.3\t1\t129204 7500\n",
      "75\t129081\t0.3\t1\t122024 7057\n",
      "76\t126400\t0.3\t1\t119414 6986\n",
      "77\t125876\t0.3\t1\t118871 7005\n",
      "78\t121079\t0.3\t1\t114302 6777\n",
      "79\t116625\t0.3\t1\t110208 6417\n",
      "80\t115718\t0.3\t1\t109559 6159\n",
      "81\t115429\t0.3\t1\t109090 6339\n",
      "82\t109791\t0.3\t1\t103782 6009\n",
      "83\t107050\t0.3\t1\t101360 5690\n",
      "84\t102872\t0.3\t1\t97327 5545\n",
      "85\t102015\t0.3\t1\t96553 5462\n",
      "86\t101949\t0.3\t1\t96606 5343\n",
      "87\t102660\t0.3\t1\t97419 5241\n",
      "88\t99584\t0.3\t1\t94677 4907\n",
      "89\t90213\t0.3\t1\t85644 4569\n",
      "90\t87335\t0.3\t1\t82845 4490\n",
      "91\t81890\t0.3\t1\t77745 4145\n",
      "92\t77363\t0.3\t1\t73520 3843\n",
      "93\t73690\t0.3\t1\t70013 3677\n",
      "94\t72610\t0.3\t1\t68880 3730\n",
      "95\t68219\t0.3\t1\t64684 3535\n",
      "96\t65235\t0.3\t1\t61847 3388\n",
      "97\t59727\t0.3\t1\t56753 2974\n",
      "98\t56047\t0.3\t1\t53250 2797\n",
      "99\t53654\t0.3\t1\t50884 2770\n",
      "100\t50544\t0.3\t1\t48047 2497\n",
      "101\t46267\t0.3\t1\t43991 2276\n",
      "102\t43012\t0.3\t1\t40866 2146\n",
      "103\t39825\t0.3\t1\t37849 1976\n",
      "104\t37894\t0.3\t1\t35883 2011\n",
      "105\t33884\t0.3\t1\t32215 1669\n",
      "106\t31153\t0.3\t1\t29674 1479\n",
      "107\t28444\t0.3\t1\t27053 1391\n",
      "108\t25994\t0.3\t1\t24692 1302\n",
      "109\t23338\t0.3\t1\t22212 1126\n",
      "110\t21606\t0.3\t1\t20453 1153\n",
      "111\t20243\t0.3\t1\t19234 1009\n",
      "112\t17922\t0.3\t1\t16980 942\n",
      "113\t15507\t0.3\t1\t14731 776\n",
      "114\t13757\t0.3\t1\t13111 646\n",
      "115\t11617\t0.3\t1\t11010 607\n",
      "116\t10230\t0.3\t1\t9713 517\n",
      "117\t8353\t0.3\t1\t7888 465\n",
      "118\t7422\t0.3\t1\t7053 369\n",
      "119\t6284\t0.3\t1\t5939 345\n",
      "120\t5303\t0.3\t1\t5008 295\n",
      "121\t4751\t0.3\t1\t4456 295\n",
      "122\t3933\t0.3\t1\t3679 254\n",
      "123\t3312\t0.3\t1\t3148 164\n",
      "124\t2831\t0.3\t1\t2672 159\n",
      "125\t2374\t0.3\t1\t2211 163\n",
      "126\t2009\t0.3\t1\t1881 128\n",
      "127\t1689\t0.3\t1\t1578 111\n",
      "128\t1328\t0.3\t1\t1253 75\n",
      "129\t1073\t0.3\t1\t1017 56\n",
      "130\t798\t0.3\t1\t754 44\n",
      "131\t632\t0.3\t1\t599 33\n",
      "132\t607\t0.3\t1\t572 35\n",
      "133\t679\t0.3\t1\t639 40\n",
      "134\t1080\t0.3\t1\t1037 43\n",
      "135\t476\t0.3\t1\t455 21\n",
      "136\t209\t0.3\t1\t191 18\n",
      "137\t180\t0.3\t1\t172 8\n",
      "138\t119\t0.3\t1\t105 14\n",
      "139\t64\t0.3\t1\t58 6\n",
      "140\t54\t0.3\t1\t50 4\n",
      "141\t33\t0.3\t1\t30 3\n",
      "142\t22\t0.3\t1\t22\n",
      "143\t32\t0.3\t1\t26 6\n",
      "144\t37\t0.3\t1\t35 2\n",
      "145\t89\t0.3\t1\t87 2\n",
      "146\t113\t0.3\t1\t108 5\n",
      "147\t383\t0.3\t1\t362 21\n",
      "148\t262\t0.3\t1\t256 6\n",
      "149\t189\t0.3\t1\t171 18\n",
      "150\t1406\t0.3\t1\t1326 80\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18388474 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz, file_2: 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18388474\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 40702 (0.22%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_2_S10_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_2_S10_L002_R1_001_trimmed.fq.gz and 8C_26psu_2_S10_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_3_S11_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 586.37 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,221,011\n",
      "Reads with adapters:                14,907,348 (86.6%)\n",
      "Reads written (passing filters):    17,221,011 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,583,151,650 bp\n",
      "Quality-trimmed:               4,970,961 bp (0.2%)\n",
      "Total written (filtered):  1,823,787,303 bp (70.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14907348 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 14.0%\n",
      "  G: 22.2%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1339401\t4305252.8\t0\t1339401\n",
      "2\t221513\t1076313.2\t0\t221513\n",
      "3\t135887\t269078.3\t0\t135887\n",
      "4\t86877\t67269.6\t0\t86877\n",
      "5\t77129\t16817.4\t0\t77129\n",
      "6\t76711\t4204.3\t0\t76711\n",
      "7\t64192\t1051.1\t0\t64192\n",
      "8\t80722\t262.8\t0\t80722\n",
      "9\t81072\t65.7\t0\t80149 923\n",
      "10\t77051\t16.4\t1\t73224 3827\n",
      "11\t85961\t4.1\t1\t79699 6262\n",
      "12\t83198\t1.0\t1\t78237 4961\n",
      "13\t86825\t0.3\t1\t81365 5460\n",
      "14\t84806\t0.3\t1\t79388 5418\n",
      "15\t92317\t0.3\t1\t86654 5663\n",
      "16\t89335\t0.3\t1\t83153 6182\n",
      "17\t108017\t0.3\t1\t99755 8262\n",
      "18\t99925\t0.3\t1\t94377 5548\n",
      "19\t81101\t0.3\t1\t76294 4807\n",
      "20\t100416\t0.3\t1\t94250 6166\n",
      "21\t108388\t0.3\t1\t101446 6942\n",
      "22\t94203\t0.3\t1\t89329 4874\n",
      "23\t109171\t0.3\t1\t102371 6800\n",
      "24\t104496\t0.3\t1\t96618 7878\n",
      "25\t111703\t0.3\t1\t104639 7064\n",
      "26\t103882\t0.3\t1\t98042 5840\n",
      "27\t115801\t0.3\t1\t107736 8065\n",
      "28\t113315\t0.3\t1\t105981 7334\n",
      "29\t122732\t0.3\t1\t113578 9154\n",
      "30\t129896\t0.3\t1\t123034 6862\n",
      "31\t104938\t0.3\t1\t98897 6041\n",
      "32\t124822\t0.3\t1\t117479 7343\n",
      "33\t120235\t0.3\t1\t112658 7577\n",
      "34\t135691\t0.3\t1\t125410 10281\n",
      "35\t143924\t0.3\t1\t135732 8192\n",
      "36\t114731\t0.3\t1\t107772 6959\n",
      "37\t138113\t0.3\t1\t128706 9407\n",
      "38\t183834\t0.3\t1\t174741 9093\n",
      "39\t91269\t0.3\t1\t83934 7335\n",
      "40\t144200\t0.3\t1\t135345 8855\n",
      "41\t167039\t0.3\t1\t155843 11196\n",
      "42\t124510\t0.3\t1\t116014 8496\n",
      "43\t229902\t0.3\t1\t218995 10907\n",
      "44\t61751\t0.3\t1\t56507 5244\n",
      "45\t153093\t0.3\t1\t144067 9026\n",
      "46\t145804\t0.3\t1\t137245 8559\n",
      "47\t154404\t0.3\t1\t145632 8772\n",
      "48\t154492\t0.3\t1\t145156 9336\n",
      "49\t169124\t0.3\t1\t158402 10722\n",
      "50\t144085\t0.3\t1\t136047 8038\n",
      "51\t159010\t0.3\t1\t149344 9666\n",
      "52\t165606\t0.3\t1\t154687 10919\n",
      "53\t178026\t0.3\t1\t167943 10083\n",
      "54\t145073\t0.3\t1\t136817 8256\n",
      "55\t172012\t0.3\t1\t161558 10454\n",
      "56\t210944\t0.3\t1\t199588 11356\n",
      "57\t131803\t0.3\t1\t123033 8770\n",
      "58\t178194\t0.3\t1\t169188 9006\n",
      "59\t157761\t0.3\t1\t148537 9224\n",
      "60\t170653\t0.3\t1\t160454 10199\n",
      "61\t170629\t0.3\t1\t161244 9385\n",
      "62\t190264\t0.3\t1\t180168 10096\n",
      "63\t148311\t0.3\t1\t139056 9255\n",
      "64\t170495\t0.3\t1\t161109 9386\n",
      "65\t191349\t0.3\t1\t180352 10997\n",
      "66\t199623\t0.3\t1\t188220 11403\n",
      "67\t212270\t0.3\t1\t197036 15234\n",
      "68\t412894\t0.3\t1\t394953 17941\n",
      "69\t59319\t0.3\t1\t54157 5162\n",
      "70\t23953\t0.3\t1\t19703 4250\n",
      "71\t116809\t0.3\t1\t110410 6399\n",
      "72\t157277\t0.3\t1\t148914 8363\n",
      "73\t165912\t0.3\t1\t156892 9020\n",
      "74\t164518\t0.3\t1\t155310 9208\n",
      "75\t163178\t0.3\t1\t154082 9096\n",
      "76\t162323\t0.3\t1\t153580 8743\n",
      "77\t162174\t0.3\t1\t153149 9025\n",
      "78\t154081\t0.3\t1\t145262 8819\n",
      "79\t153448\t0.3\t1\t144767 8681\n",
      "80\t149998\t0.3\t1\t141770 8228\n",
      "81\t150056\t0.3\t1\t141671 8385\n",
      "82\t147165\t0.3\t1\t139005 8160\n",
      "83\t142399\t0.3\t1\t134518 7881\n",
      "84\t137532\t0.3\t1\t129880 7652\n",
      "85\t134287\t0.3\t1\t126887 7400\n",
      "86\t129443\t0.3\t1\t122227 7216\n",
      "87\t125510\t0.3\t1\t118421 7089\n",
      "88\t124525\t0.3\t1\t117462 7063\n",
      "89\t118345\t0.3\t1\t111665 6680\n",
      "90\t118308\t0.3\t1\t111670 6638\n",
      "91\t111912\t0.3\t1\t105689 6223\n",
      "92\t106445\t0.3\t1\t100472 5973\n",
      "93\t100557\t0.3\t1\t94827 5730\n",
      "94\t97750\t0.3\t1\t92088 5662\n",
      "95\t91346\t0.3\t1\t86041 5305\n",
      "96\t86885\t0.3\t1\t81735 5150\n",
      "97\t81399\t0.3\t1\t76644 4755\n",
      "98\t75784\t0.3\t1\t71395 4389\n",
      "99\t72904\t0.3\t1\t68310 4594\n",
      "100\t67617\t0.3\t1\t63610 4007\n",
      "101\t63261\t0.3\t1\t59615 3646\n",
      "102\t58086\t0.3\t1\t54536 3550\n",
      "103\t54397\t0.3\t1\t51127 3270\n",
      "104\t50493\t0.3\t1\t47351 3142\n",
      "105\t44481\t0.3\t1\t41579 2902\n",
      "106\t41013\t0.3\t1\t38506 2507\n",
      "107\t36987\t0.3\t1\t34603 2384\n",
      "108\t34766\t0.3\t1\t32511 2255\n",
      "109\t30744\t0.3\t1\t28764 1980\n",
      "110\t27509\t0.3\t1\t25598 1911\n",
      "111\t24797\t0.3\t1\t23127 1670\n",
      "112\t21804\t0.3\t1\t20294 1510\n",
      "113\t18744\t0.3\t1\t17435 1309\n",
      "114\t16804\t0.3\t1\t15658 1146\n",
      "115\t14315\t0.3\t1\t13251 1064\n",
      "116\t12612\t0.3\t1\t11627 985\n",
      "117\t10473\t0.3\t1\t9684 789\n",
      "118\t8921\t0.3\t1\t8216 705\n",
      "119\t7563\t0.3\t1\t6922 641\n",
      "120\t6332\t0.3\t1\t5781 551\n",
      "121\t5619\t0.3\t1\t5122 497\n",
      "122\t4576\t0.3\t1\t4198 378\n",
      "123\t3737\t0.3\t1\t3370 367\n",
      "124\t3104\t0.3\t1\t2823 281\n",
      "125\t2623\t0.3\t1\t2320 303\n",
      "126\t2119\t0.3\t1\t1927 192\n",
      "127\t1819\t0.3\t1\t1640 179\n",
      "128\t1359\t0.3\t1\t1208 151\n",
      "129\t1165\t0.3\t1\t1033 132\n",
      "130\t852\t0.3\t1\t749 103\n",
      "131\t856\t0.3\t1\t765 91\n",
      "132\t726\t0.3\t1\t633 93\n",
      "133\t874\t0.3\t1\t793 81\n",
      "134\t1491\t0.3\t1\t1401 90\n",
      "135\t611\t0.3\t1\t548 63\n",
      "136\t207\t0.3\t1\t187 20\n",
      "137\t166\t0.3\t1\t155 11\n",
      "138\t132\t0.3\t1\t107 25\n",
      "139\t75\t0.3\t1\t67 8\n",
      "140\t69\t0.3\t1\t60 9\n",
      "141\t42\t0.3\t1\t32 10\n",
      "142\t26\t0.3\t1\t19 7\n",
      "143\t26\t0.3\t1\t15 11\n",
      "144\t18\t0.3\t1\t13 5\n",
      "145\t37\t0.3\t1\t33 4\n",
      "146\t26\t0.3\t1\t20 6\n",
      "147\t73\t0.3\t1\t60 13\n",
      "148\t47\t0.3\t1\t37 10\n",
      "149\t55\t0.3\t1\t48 7\n",
      "150\t636\t0.3\t1\t548 88\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "17221011 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_3_S11_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 621.77 s (36 us/read; 1.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,221,011\n",
      "Reads with adapters:                14,791,093 (85.9%)\n",
      "Reads written (passing filters):    17,221,011 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,583,151,650 bp\n",
      "Quality-trimmed:              13,446,962 bp (0.5%)\n",
      "Total written (filtered):  1,830,093,875 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14791093 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 12.3%\n",
      "  G: 25.7%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1396647\t4305252.8\t0\t1396647\n",
      "2\t233877\t1076313.2\t0\t233877\n",
      "3\t144492\t269078.3\t0\t144492\n",
      "4\t87098\t67269.6\t0\t87098\n",
      "5\t86055\t16817.4\t0\t86055\n",
      "6\t74999\t4204.3\t0\t74999\n",
      "7\t60602\t1051.1\t0\t60602\n",
      "8\t88349\t262.8\t0\t88349\n",
      "9\t70295\t65.7\t0\t70075 220\n",
      "10\t78524\t16.4\t1\t74626 3898\n",
      "11\t82895\t4.1\t1\t77738 5157\n",
      "12\t84476\t1.0\t1\t79233 5243\n",
      "13\t94326\t0.3\t1\t88847 5479\n",
      "14\t89984\t0.3\t1\t83845 6139\n",
      "15\t72879\t0.3\t1\t68520 4359\n",
      "16\t107100\t0.3\t1\t100533 6567\n",
      "17\t87871\t0.3\t1\t82033 5838\n",
      "18\t83139\t0.3\t1\t78397 4742\n",
      "19\t107298\t0.3\t1\t100402 6896\n",
      "20\t100171\t0.3\t1\t94390 5781\n",
      "21\t88114\t0.3\t1\t82713 5401\n",
      "22\t101076\t0.3\t1\t95306 5770\n",
      "23\t105877\t0.3\t1\t99537 6340\n",
      "24\t106645\t0.3\t1\t100049 6596\n",
      "25\t121932\t0.3\t1\t115168 6764\n",
      "26\t100476\t0.3\t1\t94344 6132\n",
      "27\t110166\t0.3\t1\t102758 7408\n",
      "28\t107176\t0.3\t1\t101924 5252\n",
      "29\t114111\t0.3\t1\t107493 6618\n",
      "30\t115359\t0.3\t1\t109345 6014\n",
      "31\t124714\t0.3\t1\t117673 7041\n",
      "32\t113114\t0.3\t1\t107249 5865\n",
      "33\t126653\t0.3\t1\t119283 7370\n",
      "34\t161416\t0.3\t1\t152249 9167\n",
      "35\t95165\t0.3\t1\t90366 4799\n",
      "36\t130060\t0.3\t1\t122395 7665\n",
      "37\t137916\t0.3\t1\t130202 7714\n",
      "38\t132706\t0.3\t1\t126385 6321\n",
      "39\t129385\t0.3\t1\t123068 6317\n",
      "40\t145600\t0.3\t1\t137496 8104\n",
      "41\t141184\t0.3\t1\t134070 7114\n",
      "42\t147655\t0.3\t1\t140213 7442\n",
      "43\t133334\t0.3\t1\t126611 6723\n",
      "44\t155705\t0.3\t1\t147696 8009\n",
      "45\t171369\t0.3\t1\t162810 8559\n",
      "46\t117297\t0.3\t1\t111483 5814\n",
      "47\t154795\t0.3\t1\t147535 7260\n",
      "48\t153124\t0.3\t1\t145909 7215\n",
      "49\t161825\t0.3\t1\t153802 8023\n",
      "50\t152605\t0.3\t1\t145077 7528\n",
      "51\t196303\t0.3\t1\t186610 9693\n",
      "52\t140348\t0.3\t1\t134075 6273\n",
      "53\t169943\t0.3\t1\t162909 7034\n",
      "54\t129681\t0.3\t1\t123901 5780\n",
      "55\t171452\t0.3\t1\t164350 7102\n",
      "56\t175614\t0.3\t1\t167693 7921\n",
      "57\t166121\t0.3\t1\t159314 6807\n",
      "58\t161387\t0.3\t1\t155069 6318\n",
      "59\t166619\t0.3\t1\t159538 7081\n",
      "60\t171867\t0.3\t1\t164363 7504\n",
      "61\t178391\t0.3\t1\t170609 7782\n",
      "62\t196679\t0.3\t1\t188081 8598\n",
      "63\t261274\t0.3\t1\t251029 10245\n",
      "64\t223191\t0.3\t1\t215765 7426\n",
      "65\t140786\t0.3\t1\t135022 5764\n",
      "66\t154538\t0.3\t1\t148311 6227\n",
      "67\t154027\t0.3\t1\t147524 6503\n",
      "68\t155618\t0.3\t1\t149295 6323\n",
      "69\t156069\t0.3\t1\t149599 6470\n",
      "70\t165853\t0.3\t1\t158998 6855\n",
      "71\t174385\t0.3\t1\t167574 6811\n",
      "72\t166867\t0.3\t1\t160011 6856\n",
      "73\t177560\t0.3\t1\t170679 6881\n",
      "74\t163422\t0.3\t1\t157175 6247\n",
      "75\t155704\t0.3\t1\t149734 5970\n",
      "76\t153010\t0.3\t1\t147031 5979\n",
      "77\t153177\t0.3\t1\t146921 6256\n",
      "78\t148366\t0.3\t1\t142418 5948\n",
      "79\t143302\t0.3\t1\t137662 5640\n",
      "80\t143393\t0.3\t1\t137924 5469\n",
      "81\t142595\t0.3\t1\t137094 5501\n",
      "82\t136572\t0.3\t1\t131469 5103\n",
      "83\t132737\t0.3\t1\t127619 5118\n",
      "84\t127991\t0.3\t1\t122783 5208\n",
      "85\t126965\t0.3\t1\t121917 5048\n",
      "86\t125977\t0.3\t1\t120895 5082\n",
      "87\t128246\t0.3\t1\t123231 5015\n",
      "88\t125283\t0.3\t1\t120412 4871\n",
      "89\t112372\t0.3\t1\t108049 4323\n",
      "90\t108791\t0.3\t1\t104559 4232\n",
      "91\t102461\t0.3\t1\t98288 4173\n",
      "92\t96447\t0.3\t1\t92722 3725\n",
      "93\t93869\t0.3\t1\t90286 3583\n",
      "94\t91252\t0.3\t1\t87743 3509\n",
      "95\t86357\t0.3\t1\t83052 3305\n",
      "96\t82971\t0.3\t1\t79818 3153\n",
      "97\t76434\t0.3\t1\t73514 2920\n",
      "98\t72048\t0.3\t1\t69275 2773\n",
      "99\t70150\t0.3\t1\t67632 2518\n",
      "100\t63666\t0.3\t1\t61347 2319\n",
      "101\t59861\t0.3\t1\t57732 2129\n",
      "102\t55598\t0.3\t1\t53622 1976\n",
      "103\t51026\t0.3\t1\t49092 1934\n",
      "104\t48386\t0.3\t1\t46675 1711\n",
      "105\t42639\t0.3\t1\t41080 1559\n",
      "106\t39613\t0.3\t1\t38200 1413\n",
      "107\t35228\t0.3\t1\t33949 1279\n",
      "108\t31897\t0.3\t1\t30649 1248\n",
      "109\t28672\t0.3\t1\t27578 1094\n",
      "110\t25914\t0.3\t1\t24951 963\n",
      "111\t24295\t0.3\t1\t23481 814\n",
      "112\t21304\t0.3\t1\t20453 851\n",
      "113\t18350\t0.3\t1\t17653 697\n",
      "114\t16256\t0.3\t1\t15627 629\n",
      "115\t13357\t0.3\t1\t12793 564\n",
      "116\t11612\t0.3\t1\t11150 462\n",
      "117\t9484\t0.3\t1\t9116 368\n",
      "118\t8205\t0.3\t1\t7826 379\n",
      "119\t6936\t0.3\t1\t6629 307\n",
      "120\t5765\t0.3\t1\t5525 240\n",
      "121\t5231\t0.3\t1\t5015 216\n",
      "122\t4247\t0.3\t1\t4080 167\n",
      "123\t3547\t0.3\t1\t3416 131\n",
      "124\t2936\t0.3\t1\t2808 128\n",
      "125\t2425\t0.3\t1\t2314 111\n",
      "126\t2011\t0.3\t1\t1915 96\n",
      "127\t1702\t0.3\t1\t1617 85\n",
      "128\t1242\t0.3\t1\t1178 64\n",
      "129\t1065\t0.3\t1\t1006 59\n",
      "130\t766\t0.3\t1\t721 45\n",
      "131\t781\t0.3\t1\t739 42\n",
      "132\t668\t0.3\t1\t634 34\n",
      "133\t887\t0.3\t1\t866 21\n",
      "134\t1489\t0.3\t1\t1430 59\n",
      "135\t583\t0.3\t1\t560 23\n",
      "136\t206\t0.3\t1\t191 15\n",
      "137\t151\t0.3\t1\t146 5\n",
      "138\t116\t0.3\t1\t111 5\n",
      "139\t72\t0.3\t1\t65 7\n",
      "140\t56\t0.3\t1\t54 2\n",
      "141\t35\t0.3\t1\t27 8\n",
      "142\t30\t0.3\t1\t26 4\n",
      "143\t21\t0.3\t1\t18 3\n",
      "144\t19\t0.3\t1\t17 2\n",
      "145\t37\t0.3\t1\t35 2\n",
      "146\t25\t0.3\t1\t22 3\n",
      "147\t50\t0.3\t1\t46 4\n",
      "148\t39\t0.3\t1\t37 2\n",
      "149\t45\t0.3\t1\t42 3\n",
      "150\t451\t0.3\t1\t418 33\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "17221011 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz, file_2: 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17221011\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 38442 (0.22%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_3_S11_L001_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_3_S11_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 596.76 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,175,482\n",
      "Reads with adapters:                14,848,929 (86.5%)\n",
      "Reads written (passing filters):    17,175,482 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,322,300 bp\n",
      "Quality-trimmed:               5,047,974 bp (0.2%)\n",
      "Total written (filtered):  1,818,633,191 bp (70.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14848929 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 14.0%\n",
      "  G: 22.1%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1338924\t4293870.5\t0\t1338924\n",
      "2\t219387\t1073467.6\t0\t219387\n",
      "3\t135027\t268366.9\t0\t135027\n",
      "4\t85370\t67091.7\t0\t85370\n",
      "5\t77474\t16772.9\t0\t77474\n",
      "6\t77037\t4193.2\t0\t77037\n",
      "7\t61988\t1048.3\t0\t61988\n",
      "8\t79862\t262.1\t0\t79862\n",
      "9\t79304\t65.5\t0\t78415 889\n",
      "10\t74894\t16.4\t1\t70380 4514\n",
      "11\t85385\t4.1\t1\t78459 6926\n",
      "12\t81941\t1.0\t1\t76091 5850\n",
      "13\t85904\t0.3\t1\t79720 6184\n",
      "14\t83830\t0.3\t1\t77435 6395\n",
      "15\t91251\t0.3\t1\t84448 6803\n",
      "16\t87362\t0.3\t1\t80132 7230\n",
      "17\t108027\t0.3\t1\t98268 9759\n",
      "18\t97765\t0.3\t1\t91194 6571\n",
      "19\t79668\t0.3\t1\t74013 5655\n",
      "20\t98415\t0.3\t1\t91325 7090\n",
      "21\t108861\t0.3\t1\t100754 8107\n",
      "22\t92432\t0.3\t1\t86497 5935\n",
      "23\t109093\t0.3\t1\t101170 7923\n",
      "24\t102961\t0.3\t1\t94241 8720\n",
      "25\t110529\t0.3\t1\t102334 8195\n",
      "26\t102489\t0.3\t1\t95783 6706\n",
      "27\t115669\t0.3\t1\t106602 9067\n",
      "28\t113524\t0.3\t1\t105254 8270\n",
      "29\t122345\t0.3\t1\t111712 10633\n",
      "30\t129585\t0.3\t1\t121424 8161\n",
      "31\t102733\t0.3\t1\t95540 7193\n",
      "32\t124071\t0.3\t1\t115107 8964\n",
      "33\t119235\t0.3\t1\t110316 8919\n",
      "34\t135312\t0.3\t1\t123053 12259\n",
      "35\t144848\t0.3\t1\t135072 9776\n",
      "36\t111252\t0.3\t1\t103350 7902\n",
      "37\t137690\t0.3\t1\t126540 11150\n",
      "38\t183318\t0.3\t1\t172054 11264\n",
      "39\t89425\t0.3\t1\t80789 8636\n",
      "40\t143455\t0.3\t1\t132529 10926\n",
      "41\t167032\t0.3\t1\t153468 13564\n",
      "42\t121530\t0.3\t1\t111758 9772\n",
      "43\t226339\t0.3\t1\t213169 13170\n",
      "44\t63952\t0.3\t1\t57726 6226\n",
      "45\t153236\t0.3\t1\t142659 10577\n",
      "46\t145389\t0.3\t1\t135235 10154\n",
      "47\t153266\t0.3\t1\t143025 10241\n",
      "48\t152876\t0.3\t1\t141831 11045\n",
      "49\t168461\t0.3\t1\t156017 12444\n",
      "50\t143668\t0.3\t1\t134304 9364\n",
      "51\t158306\t0.3\t1\t147182 11124\n",
      "52\t165500\t0.3\t1\t153119 12381\n",
      "53\t177823\t0.3\t1\t165831 11992\n",
      "54\t142891\t0.3\t1\t133186 9705\n",
      "55\t170342\t0.3\t1\t158433 11909\n",
      "56\t212493\t0.3\t1\t198685 13808\n",
      "57\t130099\t0.3\t1\t120181 9918\n",
      "58\t177795\t0.3\t1\t167072 10723\n",
      "59\t156878\t0.3\t1\t145935 10943\n",
      "60\t169887\t0.3\t1\t158157 11730\n",
      "61\t170084\t0.3\t1\t159037 11047\n",
      "62\t189821\t0.3\t1\t177611 12210\n",
      "63\t149110\t0.3\t1\t138396 10714\n",
      "64\t169980\t0.3\t1\t159400 10580\n",
      "65\t190750\t0.3\t1\t178067 12683\n",
      "66\t196470\t0.3\t1\t183447 13023\n",
      "67\t212313\t0.3\t1\t195505 16808\n",
      "68\t400251\t0.3\t1\t379902 20349\n",
      "69\t75488\t0.3\t1\t68954 6534\n",
      "70\t29970\t0.3\t1\t25043 4927\n",
      "71\t114346\t0.3\t1\t107216 7130\n",
      "72\t155823\t0.3\t1\t146728 9095\n",
      "73\t164934\t0.3\t1\t154662 10272\n",
      "74\t163683\t0.3\t1\t153217 10466\n",
      "75\t161985\t0.3\t1\t151549 10436\n",
      "76\t161854\t0.3\t1\t151132 10722\n",
      "77\t161503\t0.3\t1\t151032 10471\n",
      "78\t154856\t0.3\t1\t144180 10676\n",
      "79\t152863\t0.3\t1\t142631 10232\n",
      "80\t150353\t0.3\t1\t140171 10182\n",
      "81\t150454\t0.3\t1\t140509 9945\n",
      "82\t145676\t0.3\t1\t135805 9871\n",
      "83\t141824\t0.3\t1\t132558 9266\n",
      "84\t137190\t0.3\t1\t128021 9169\n",
      "85\t134663\t0.3\t1\t125634 9029\n",
      "86\t130352\t0.3\t1\t121592 8760\n",
      "87\t125876\t0.3\t1\t117794 8082\n",
      "88\t125030\t0.3\t1\t116803 8227\n",
      "89\t117792\t0.3\t1\t109945 7847\n",
      "90\t117830\t0.3\t1\t110051 7779\n",
      "91\t112910\t0.3\t1\t105546 7364\n",
      "92\t106310\t0.3\t1\t99476 6834\n",
      "93\t100338\t0.3\t1\t93657 6681\n",
      "94\t97374\t0.3\t1\t90887 6487\n",
      "95\t91711\t0.3\t1\t85495 6216\n",
      "96\t86651\t0.3\t1\t80705 5946\n",
      "97\t81512\t0.3\t1\t76160 5352\n",
      "98\t75813\t0.3\t1\t70557 5256\n",
      "99\t73615\t0.3\t1\t68470 5145\n",
      "100\t67314\t0.3\t1\t62734 4580\n",
      "101\t63335\t0.3\t1\t58922 4413\n",
      "102\t58250\t0.3\t1\t54144 4106\n",
      "103\t54120\t0.3\t1\t50374 3746\n",
      "104\t50265\t0.3\t1\t46857 3408\n",
      "105\t44626\t0.3\t1\t41570 3056\n",
      "106\t40793\t0.3\t1\t37936 2857\n",
      "107\t37321\t0.3\t1\t34670 2651\n",
      "108\t34705\t0.3\t1\t32091 2614\n",
      "109\t31002\t0.3\t1\t28677 2325\n",
      "110\t27578\t0.3\t1\t25574 2004\n",
      "111\t24603\t0.3\t1\t22785 1818\n",
      "112\t21972\t0.3\t1\t20247 1725\n",
      "113\t18720\t0.3\t1\t17311 1409\n",
      "114\t16688\t0.3\t1\t15435 1253\n",
      "115\t14432\t0.3\t1\t13232 1200\n",
      "116\t12245\t0.3\t1\t11232 1013\n",
      "117\t10552\t0.3\t1\t9672 880\n",
      "118\t8984\t0.3\t1\t8244 740\n",
      "119\t7381\t0.3\t1\t6703 678\n",
      "120\t6463\t0.3\t1\t5860 603\n",
      "121\t5515\t0.3\t1\t4963 552\n",
      "122\t4409\t0.3\t1\t4006 403\n",
      "123\t3731\t0.3\t1\t3377 354\n",
      "124\t3097\t0.3\t1\t2824 273\n",
      "125\t2558\t0.3\t1\t2286 272\n",
      "126\t2102\t0.3\t1\t1889 213\n",
      "127\t1729\t0.3\t1\t1530 199\n",
      "128\t1388\t0.3\t1\t1248 140\n",
      "129\t1146\t0.3\t1\t1007 139\n",
      "130\t884\t0.3\t1\t785 99\n",
      "131\t740\t0.3\t1\t651 89\n",
      "132\t734\t0.3\t1\t647 87\n",
      "133\t973\t0.3\t1\t886 87\n",
      "134\t1537\t0.3\t1\t1449 88\n",
      "135\t609\t0.3\t1\t551 58\n",
      "136\t234\t0.3\t1\t208 26\n",
      "137\t157\t0.3\t1\t138 19\n",
      "138\t104\t0.3\t1\t87 17\n",
      "139\t80\t0.3\t1\t60 20\n",
      "140\t68\t0.3\t1\t51 17\n",
      "141\t33\t0.3\t1\t25 8\n",
      "142\t34\t0.3\t1\t26 8\n",
      "143\t16\t0.3\t1\t16\n",
      "144\t16\t0.3\t1\t14 2\n",
      "145\t39\t0.3\t1\t27 12\n",
      "146\t32\t0.3\t1\t26 6\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t60\t0.3\t1\t51 9\n",
      "149\t55\t0.3\t1\t49 6\n",
      "150\t652\t0.3\t1\t552 100\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "17175482 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_3_S11_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 633.40 s (37 us/read; 1.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              17,175,482\n",
      "Reads with adapters:                14,720,565 (85.7%)\n",
      "Reads written (passing filters):    17,175,482 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,576,322,300 bp\n",
      "Quality-trimmed:              12,927,719 bp (0.5%)\n",
      "Total written (filtered):  1,825,809,518 bp (70.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14720565 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 12.3%\n",
      "  G: 25.7%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1394679\t4293870.5\t0\t1394679\n",
      "2\t231063\t1073467.6\t0\t231063\n",
      "3\t146836\t268366.9\t0\t146836\n",
      "4\t85160\t67091.7\t0\t85160\n",
      "5\t83607\t16772.9\t0\t83607\n",
      "6\t75270\t4193.2\t0\t75270\n",
      "7\t59401\t1048.3\t0\t59401\n",
      "8\t87249\t262.1\t0\t87249\n",
      "9\t68581\t65.5\t0\t68364 217\n",
      "10\t75306\t16.4\t1\t69907 5399\n",
      "11\t81844\t4.1\t1\t75220 6624\n",
      "12\t81567\t1.0\t1\t74921 6646\n",
      "13\t94550\t0.3\t1\t87043 7507\n",
      "14\t88612\t0.3\t1\t80763 7849\n",
      "15\t71080\t0.3\t1\t65748 5332\n",
      "16\t104508\t0.3\t1\t96265 8243\n",
      "17\t90450\t0.3\t1\t83158 7292\n",
      "18\t79309\t0.3\t1\t73193 6116\n",
      "19\t106248\t0.3\t1\t97745 8503\n",
      "20\t97943\t0.3\t1\t90478 7465\n",
      "21\t87195\t0.3\t1\t80598 6597\n",
      "22\t99625\t0.3\t1\t92109 7516\n",
      "23\t105941\t0.3\t1\t98004 7937\n",
      "24\t105924\t0.3\t1\t97524 8400\n",
      "25\t120959\t0.3\t1\t112658 8301\n",
      "26\t100855\t0.3\t1\t92739 8116\n",
      "27\t111349\t0.3\t1\t102191 9158\n",
      "28\t103794\t0.3\t1\t97081 6713\n",
      "29\t115018\t0.3\t1\t106647 8371\n",
      "30\t112098\t0.3\t1\t104443 7655\n",
      "31\t125293\t0.3\t1\t116600 8693\n",
      "32\t108268\t0.3\t1\t101076 7192\n",
      "33\t127668\t0.3\t1\t118305 9363\n",
      "34\t163116\t0.3\t1\t151740 11376\n",
      "35\t90695\t0.3\t1\t84564 6131\n",
      "36\t130252\t0.3\t1\t120794 9458\n",
      "37\t137508\t0.3\t1\t128167 9341\n",
      "38\t131385\t0.3\t1\t123153 8232\n",
      "39\t125429\t0.3\t1\t117971 7458\n",
      "40\t144394\t0.3\t1\t134617 9777\n",
      "41\t142244\t0.3\t1\t133129 9115\n",
      "42\t146211\t0.3\t1\t136886 9325\n",
      "43\t130656\t0.3\t1\t122570 8086\n",
      "44\t158062\t0.3\t1\t147788 10274\n",
      "45\t169410\t0.3\t1\t158451 10959\n",
      "46\t116471\t0.3\t1\t109140 7331\n",
      "47\t153771\t0.3\t1\t145129 8642\n",
      "48\t151126\t0.3\t1\t142022 9104\n",
      "49\t158775\t0.3\t1\t148794 9981\n",
      "50\t155083\t0.3\t1\t145500 9583\n",
      "51\t193098\t0.3\t1\t181154 11944\n",
      "52\t138930\t0.3\t1\t130761 8169\n",
      "53\t172888\t0.3\t1\t163380 9508\n",
      "54\t126226\t0.3\t1\t118962 7264\n",
      "55\t168553\t0.3\t1\t159300 9253\n",
      "56\t174469\t0.3\t1\t163938 10531\n",
      "57\t166262\t0.3\t1\t157805 8457\n",
      "58\t161940\t0.3\t1\t153447 8493\n",
      "59\t163516\t0.3\t1\t154551 8965\n",
      "60\t171750\t0.3\t1\t162391 9359\n",
      "61\t176379\t0.3\t1\t166983 9396\n",
      "62\t194831\t0.3\t1\t183858 10973\n",
      "63\t257381\t0.3\t1\t244702 12679\n",
      "64\t218615\t0.3\t1\t209942 8673\n",
      "65\t141285\t0.3\t1\t134011 7274\n",
      "66\t154574\t0.3\t1\t146600 7974\n",
      "67\t155286\t0.3\t1\t146846 8440\n",
      "68\t158519\t0.3\t1\t150042 8477\n",
      "69\t156924\t0.3\t1\t148115 8809\n",
      "70\t167343\t0.3\t1\t158462 8881\n",
      "71\t174948\t0.3\t1\t165808 9140\n",
      "72\t168193\t0.3\t1\t159291 8902\n",
      "73\t175743\t0.3\t1\t166626 9117\n",
      "74\t163090\t0.3\t1\t154296 8794\n",
      "75\t155077\t0.3\t1\t146999 8078\n",
      "76\t152837\t0.3\t1\t144700 8137\n",
      "77\t152701\t0.3\t1\t144644 8057\n",
      "78\t148868\t0.3\t1\t140946 7922\n",
      "79\t143248\t0.3\t1\t135788 7460\n",
      "80\t143729\t0.3\t1\t136194 7535\n",
      "81\t142811\t0.3\t1\t135360 7451\n",
      "82\t135758\t0.3\t1\t128640 7118\n",
      "83\t132076\t0.3\t1\t125333 6743\n",
      "84\t127993\t0.3\t1\t121417 6576\n",
      "85\t127663\t0.3\t1\t121292 6371\n",
      "86\t126712\t0.3\t1\t120456 6256\n",
      "87\t127809\t0.3\t1\t121761 6048\n",
      "88\t125518\t0.3\t1\t119426 6092\n",
      "89\t112440\t0.3\t1\t106917 5523\n",
      "90\t109295\t0.3\t1\t103850 5445\n",
      "91\t103669\t0.3\t1\t98616 5053\n",
      "92\t96386\t0.3\t1\t91748 4638\n",
      "93\t94174\t0.3\t1\t89557 4617\n",
      "94\t91197\t0.3\t1\t86759 4438\n",
      "95\t86770\t0.3\t1\t82553 4217\n",
      "96\t83121\t0.3\t1\t79129 3992\n",
      "97\t76650\t0.3\t1\t72963 3687\n",
      "98\t72211\t0.3\t1\t68803 3408\n",
      "99\t70867\t0.3\t1\t67509 3358\n",
      "100\t63540\t0.3\t1\t60596 2944\n",
      "101\t60139\t0.3\t1\t57240 2899\n",
      "102\t55801\t0.3\t1\t53158 2643\n",
      "103\t50487\t0.3\t1\t47969 2518\n",
      "104\t48235\t0.3\t1\t45902 2333\n",
      "105\t42853\t0.3\t1\t40742 2111\n",
      "106\t39140\t0.3\t1\t37246 1894\n",
      "107\t35386\t0.3\t1\t33624 1762\n",
      "108\t31865\t0.3\t1\t30331 1534\n",
      "109\t29026\t0.3\t1\t27576 1450\n",
      "110\t25978\t0.3\t1\t24750 1228\n",
      "111\t24137\t0.3\t1\t23015 1122\n",
      "112\t21305\t0.3\t1\t20312 993\n",
      "113\t18391\t0.3\t1\t17493 898\n",
      "114\t16197\t0.3\t1\t15347 850\n",
      "115\t13468\t0.3\t1\t12821 647\n",
      "116\t11291\t0.3\t1\t10728 563\n",
      "117\t9599\t0.3\t1\t9121 478\n",
      "118\t8211\t0.3\t1\t7795 416\n",
      "119\t6772\t0.3\t1\t6417 355\n",
      "120\t5885\t0.3\t1\t5586 299\n",
      "121\t5082\t0.3\t1\t4799 283\n",
      "122\t4085\t0.3\t1\t3891 194\n",
      "123\t3540\t0.3\t1\t3373 167\n",
      "124\t2933\t0.3\t1\t2771 162\n",
      "125\t2424\t0.3\t1\t2286 138\n",
      "126\t1979\t0.3\t1\t1877 102\n",
      "127\t1585\t0.3\t1\t1504 81\n",
      "128\t1307\t0.3\t1\t1233 74\n",
      "129\t1065\t0.3\t1\t999 66\n",
      "130\t818\t0.3\t1\t765 53\n",
      "131\t695\t0.3\t1\t661 34\n",
      "132\t692\t0.3\t1\t651 41\n",
      "133\t1029\t0.3\t1\t991 38\n",
      "134\t1555\t0.3\t1\t1499 56\n",
      "135\t581\t0.3\t1\t564 17\n",
      "136\t217\t0.3\t1\t210 7\n",
      "137\t148\t0.3\t1\t133 15\n",
      "138\t96\t0.3\t1\t87 9\n",
      "139\t66\t0.3\t1\t63 3\n",
      "140\t62\t0.3\t1\t58 4\n",
      "141\t31\t0.3\t1\t28 3\n",
      "142\t29\t0.3\t1\t25 4\n",
      "143\t15\t0.3\t1\t15\n",
      "144\t16\t0.3\t1\t16\n",
      "145\t37\t0.3\t1\t37\n",
      "146\t31\t0.3\t1\t30 1\n",
      "147\t76\t0.3\t1\t72 4\n",
      "148\t44\t0.3\t1\t40 4\n",
      "149\t38\t0.3\t1\t34 4\n",
      "150\t422\t0.3\t1\t392 30\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "17175482 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz, file_2: 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 17175482\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 36710 (0.21%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 75% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_3_S11_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_3_S11_L002_R1_001_trimmed.fq.gz and 8C_26psu_3_S11_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_4_S12_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 476.37 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,592,704\n",
      "Reads with adapters:                 8,958,992 (71.1%)\n",
      "Reads written (passing filters):    12,592,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,888,905,600 bp\n",
      "Quality-trimmed:               4,951,530 bp (0.3%)\n",
      "Total written (filtered):  1,571,128,982 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8958992 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.4%\n",
      "  C: 15.5%\n",
      "  G: 17.3%\n",
      "  T: 35.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2360389\t3148176.0\t0\t2360389\n",
      "2\t294874\t787044.0\t0\t294874\n",
      "3\t161693\t196761.0\t0\t161693\n",
      "4\t87416\t49190.2\t0\t87416\n",
      "5\t59722\t12297.6\t0\t59722\n",
      "6\t57196\t3074.4\t0\t57196\n",
      "7\t45727\t768.6\t0\t45727\n",
      "8\t58159\t192.1\t0\t58159\n",
      "9\t58762\t48.0\t0\t57759 1003\n",
      "10\t54824\t12.0\t1\t51199 3625\n",
      "11\t61143\t3.0\t1\t55667 5476\n",
      "12\t58318\t0.8\t1\t53717 4601\n",
      "13\t56920\t0.2\t1\t52502 4418\n",
      "14\t60297\t0.2\t1\t55050 5247\n",
      "15\t60573\t0.2\t1\t55708 4865\n",
      "16\t61479\t0.2\t1\t55954 5525\n",
      "17\t74409\t0.2\t1\t67438 6971\n",
      "18\t61009\t0.2\t1\t56681 4328\n",
      "19\t53089\t0.2\t1\t48849 4240\n",
      "20\t63972\t0.2\t1\t58870 5102\n",
      "21\t70043\t0.2\t1\t64287 5756\n",
      "22\t58590\t0.2\t1\t54441 4149\n",
      "23\t68150\t0.2\t1\t62728 5422\n",
      "24\t64229\t0.2\t1\t58023 6206\n",
      "25\t68130\t0.2\t1\t62567 5563\n",
      "26\t62420\t0.2\t1\t57820 4600\n",
      "27\t69993\t0.2\t1\t63874 6119\n",
      "28\t67875\t0.2\t1\t62387 5488\n",
      "29\t71196\t0.2\t1\t64376 6820\n",
      "30\t75799\t0.2\t1\t70433 5366\n",
      "31\t59056\t0.2\t1\t54472 4584\n",
      "32\t73671\t0.2\t1\t68075 5596\n",
      "33\t63947\t0.2\t1\t58700 5247\n",
      "34\t74583\t0.2\t1\t67925 6658\n",
      "35\t70104\t0.2\t1\t64152 5952\n",
      "36\t77250\t0.2\t1\t71393 5857\n",
      "37\t68309\t0.2\t1\t62839 5470\n",
      "38\t72505\t0.2\t1\t67371 5134\n",
      "39\t75409\t0.2\t1\t69866 5543\n",
      "40\t72478\t0.2\t1\t66543 5935\n",
      "41\t78235\t0.2\t1\t70876 7359\n",
      "42\t73557\t0.2\t1\t66866 6691\n",
      "43\t105788\t0.2\t1\t99220 6568\n",
      "44\t39203\t0.2\t1\t34910 4293\n",
      "45\t75980\t0.2\t1\t70161 5819\n",
      "46\t74377\t0.2\t1\t68346 6031\n",
      "47\t78776\t0.2\t1\t72951 5825\n",
      "48\t70677\t0.2\t1\t64756 5921\n",
      "49\t81336\t0.2\t1\t74410 6926\n",
      "50\t66909\t0.2\t1\t62092 4817\n",
      "51\t72737\t0.2\t1\t67212 5525\n",
      "52\t78934\t0.2\t1\t71865 7069\n",
      "53\t86579\t0.2\t1\t80550 6029\n",
      "54\t62022\t0.2\t1\t57173 4849\n",
      "55\t75974\t0.2\t1\t70348 5626\n",
      "56\t85877\t0.2\t1\t79807 6070\n",
      "57\t71505\t0.2\t1\t65936 5569\n",
      "58\t69856\t0.2\t1\t65606 4250\n",
      "59\t57167\t0.2\t1\t52447 4720\n",
      "60\t78076\t0.2\t1\t72356 5720\n",
      "61\t73472\t0.2\t1\t68201 5271\n",
      "62\t71844\t0.2\t1\t66745 5099\n",
      "63\t67131\t0.2\t1\t62070 5061\n",
      "64\t65598\t0.2\t1\t60901 4697\n",
      "65\t70779\t0.2\t1\t65435 5344\n",
      "66\t77405\t0.2\t1\t71580 5825\n",
      "67\t84608\t0.2\t1\t76761 7847\n",
      "68\t150617\t0.2\t1\t142058 8559\n",
      "69\t37132\t0.2\t1\t33982 3150\n",
      "70\t13103\t0.2\t1\t10914 2189\n",
      "71\t35026\t0.2\t1\t32099 2927\n",
      "72\t57779\t0.2\t1\t53691 4088\n",
      "73\t61230\t0.2\t1\t56931 4299\n",
      "74\t61166\t0.2\t1\t56777 4389\n",
      "75\t60302\t0.2\t1\t55958 4344\n",
      "76\t59928\t0.2\t1\t55484 4444\n",
      "77\t59440\t0.2\t1\t54970 4470\n",
      "78\t56149\t0.2\t1\t51928 4221\n",
      "79\t55286\t0.2\t1\t51002 4284\n",
      "80\t53792\t0.2\t1\t49846 3946\n",
      "81\t52637\t0.2\t1\t48756 3881\n",
      "82\t51226\t0.2\t1\t47353 3873\n",
      "83\t49579\t0.2\t1\t45690 3889\n",
      "84\t48468\t0.2\t1\t44896 3572\n",
      "85\t45867\t0.2\t1\t42408 3459\n",
      "86\t44887\t0.2\t1\t41555 3332\n",
      "87\t42251\t0.2\t1\t38964 3287\n",
      "88\t42216\t0.2\t1\t39029 3187\n",
      "89\t40149\t0.2\t1\t37066 3083\n",
      "90\t40387\t0.2\t1\t37102 3285\n",
      "91\t37214\t0.2\t1\t34343 2871\n",
      "92\t35614\t0.2\t1\t32860 2754\n",
      "93\t33363\t0.2\t1\t30676 2687\n",
      "94\t32052\t0.2\t1\t29598 2454\n",
      "95\t29858\t0.2\t1\t27450 2408\n",
      "96\t28594\t0.2\t1\t26301 2293\n",
      "97\t26730\t0.2\t1\t24584 2146\n",
      "98\t25261\t0.2\t1\t23132 2129\n",
      "99\t24232\t0.2\t1\t22120 2112\n",
      "100\t22159\t0.2\t1\t20358 1801\n",
      "101\t20488\t0.2\t1\t18713 1775\n",
      "102\t19117\t0.2\t1\t17450 1667\n",
      "103\t17468\t0.2\t1\t15984 1484\n",
      "104\t16181\t0.2\t1\t14728 1453\n",
      "105\t14521\t0.2\t1\t13231 1290\n",
      "106\t13349\t0.2\t1\t12105 1244\n",
      "107\t12159\t0.2\t1\t11005 1154\n",
      "108\t11376\t0.2\t1\t10302 1074\n",
      "109\t10099\t0.2\t1\t9151 948\n",
      "110\t9379\t0.2\t1\t8450 929\n",
      "111\t8577\t0.2\t1\t7720 857\n",
      "112\t7623\t0.2\t1\t6797 826\n",
      "113\t6486\t0.2\t1\t5771 715\n",
      "114\t5676\t0.2\t1\t5112 564\n",
      "115\t5089\t0.2\t1\t4501 588\n",
      "116\t4254\t0.2\t1\t3781 473\n",
      "117\t3599\t0.2\t1\t3183 416\n",
      "118\t3110\t0.2\t1\t2758 352\n",
      "119\t2723\t0.2\t1\t2363 360\n",
      "120\t2307\t0.2\t1\t1995 312\n",
      "121\t2051\t0.2\t1\t1780 271\n",
      "122\t1835\t0.2\t1\t1588 247\n",
      "123\t1508\t0.2\t1\t1291 217\n",
      "124\t1263\t0.2\t1\t1083 180\n",
      "125\t1119\t0.2\t1\t966 153\n",
      "126\t1055\t0.2\t1\t916 139\n",
      "127\t900\t0.2\t1\t780 120\n",
      "128\t681\t0.2\t1\t568 113\n",
      "129\t550\t0.2\t1\t468 82\n",
      "130\t520\t0.2\t1\t423 97\n",
      "131\t523\t0.2\t1\t467 56\n",
      "132\t590\t0.2\t1\t526 64\n",
      "133\t974\t0.2\t1\t878 96\n",
      "134\t2631\t0.2\t1\t2466 165\n",
      "135\t822\t0.2\t1\t768 54\n",
      "136\t169\t0.2\t1\t154 15\n",
      "137\t111\t0.2\t1\t90 21\n",
      "138\t83\t0.2\t1\t75 8\n",
      "139\t53\t0.2\t1\t41 12\n",
      "140\t56\t0.2\t1\t43 13\n",
      "141\t26\t0.2\t1\t20 6\n",
      "142\t15\t0.2\t1\t12 3\n",
      "143\t20\t0.2\t1\t16 4\n",
      "144\t56\t0.2\t1\t47 9\n",
      "145\t231\t0.2\t1\t197 34\n",
      "146\t229\t0.2\t1\t199 30\n",
      "147\t1212\t0.2\t1\t1078 134\n",
      "148\t767\t0.2\t1\t701 66\n",
      "149\t472\t0.2\t1\t430 42\n",
      "150\t3085\t0.2\t1\t2595 490\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "12592704 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_4_S12_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 513.46 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,592,704\n",
      "Reads with adapters:                 8,852,263 (70.3%)\n",
      "Reads written (passing filters):    12,592,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,888,905,600 bp\n",
      "Quality-trimmed:              12,948,585 bp (0.7%)\n",
      "Total written (filtered):  1,572,160,709 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8852263 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.4%\n",
      "  C: 12.6%\n",
      "  G: 20.6%\n",
      "  T: 36.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2390954\t3148176.0\t0\t2390954\n",
      "2\t298677\t787044.0\t0\t298677\n",
      "3\t170286\t196761.0\t0\t170286\n",
      "4\t88634\t49190.2\t0\t88634\n",
      "5\t67201\t12297.6\t0\t67201\n",
      "6\t54871\t3074.4\t0\t54871\n",
      "7\t42570\t768.6\t0\t42570\n",
      "8\t63033\t192.1\t0\t63033\n",
      "9\t50827\t48.0\t0\t50608 219\n",
      "10\t54798\t12.0\t1\t51719 3079\n",
      "11\t58410\t3.0\t1\t54143 4267\n",
      "12\t59146\t0.8\t1\t54631 4515\n",
      "13\t64024\t0.2\t1\t59760 4264\n",
      "14\t60295\t0.2\t1\t55485 4810\n",
      "15\t48131\t0.2\t1\t44661 3470\n",
      "16\t72030\t0.2\t1\t66991 5039\n",
      "17\t55432\t0.2\t1\t51368 4064\n",
      "18\t54012\t0.2\t1\t50286 3726\n",
      "19\t67964\t0.2\t1\t63040 4924\n",
      "20\t63154\t0.2\t1\t58908 4246\n",
      "21\t56080\t0.2\t1\t52250 3830\n",
      "22\t62812\t0.2\t1\t58598 4214\n",
      "23\t64652\t0.2\t1\t60263 4389\n",
      "24\t64121\t0.2\t1\t59590 4531\n",
      "25\t72378\t0.2\t1\t67726 4652\n",
      "26\t60189\t0.2\t1\t55991 4198\n",
      "27\t67272\t0.2\t1\t62190 5082\n",
      "28\t62061\t0.2\t1\t58469 3592\n",
      "29\t66706\t0.2\t1\t62082 4624\n",
      "30\t64680\t0.2\t1\t60760 3920\n",
      "31\t70875\t0.2\t1\t66272 4603\n",
      "32\t63295\t0.2\t1\t59659 3636\n",
      "33\t68948\t0.2\t1\t64437 4511\n",
      "34\t89723\t0.2\t1\t83808 5915\n",
      "35\t49481\t0.2\t1\t46615 2866\n",
      "36\t69552\t0.2\t1\t64903 4649\n",
      "37\t74271\t0.2\t1\t69458 4813\n",
      "38\t69359\t0.2\t1\t65691 3668\n",
      "39\t67616\t0.2\t1\t63803 3813\n",
      "40\t75243\t0.2\t1\t70372 4871\n",
      "41\t71025\t0.2\t1\t67013 4012\n",
      "42\t73877\t0.2\t1\t69581 4296\n",
      "43\t66995\t0.2\t1\t63092 3903\n",
      "44\t76573\t0.2\t1\t71945 4628\n",
      "45\t80934\t0.2\t1\t76127 4807\n",
      "46\t59833\t0.2\t1\t56439 3394\n",
      "47\t73828\t0.2\t1\t69722 4106\n",
      "48\t73312\t0.2\t1\t69349 3963\n",
      "49\t75078\t0.2\t1\t70776 4302\n",
      "50\t70693\t0.2\t1\t66714 3979\n",
      "51\t88709\t0.2\t1\t83803 4906\n",
      "52\t62743\t0.2\t1\t59451 3292\n",
      "53\t75598\t0.2\t1\t71882 3716\n",
      "54\t57634\t0.2\t1\t54617 3017\n",
      "55\t73802\t0.2\t1\t70193 3609\n",
      "56\t76463\t0.2\t1\t72320 4143\n",
      "57\t71883\t0.2\t1\t68382 3501\n",
      "58\t68729\t0.2\t1\t65597 3132\n",
      "59\t69792\t0.2\t1\t66296 3496\n",
      "60\t72493\t0.2\t1\t68898 3595\n",
      "61\t74181\t0.2\t1\t70431 3750\n",
      "62\t80362\t0.2\t1\t76257 4105\n",
      "63\t105716\t0.2\t1\t100889 4827\n",
      "64\t86565\t0.2\t1\t82909 3656\n",
      "65\t55528\t0.2\t1\t52821 2707\n",
      "66\t61319\t0.2\t1\t58488 2831\n",
      "67\t60096\t0.2\t1\t57177 2919\n",
      "68\t60017\t0.2\t1\t57079 2938\n",
      "69\t59676\t0.2\t1\t56862 2814\n",
      "70\t62200\t0.2\t1\t59177 3023\n",
      "71\t65595\t0.2\t1\t62486 3109\n",
      "72\t62243\t0.2\t1\t59355 2888\n",
      "73\t65321\t0.2\t1\t62407 2914\n",
      "74\t60107\t0.2\t1\t57408 2699\n",
      "75\t56927\t0.2\t1\t54342 2585\n",
      "76\t56197\t0.2\t1\t53642 2555\n",
      "77\t55543\t0.2\t1\t53010 2533\n",
      "78\t53264\t0.2\t1\t50829 2435\n",
      "79\t50917\t0.2\t1\t48532 2385\n",
      "80\t50251\t0.2\t1\t48020 2231\n",
      "81\t49279\t0.2\t1\t47040 2239\n",
      "82\t47377\t0.2\t1\t45259 2118\n",
      "83\t45689\t0.2\t1\t43632 2057\n",
      "84\t44996\t0.2\t1\t42952 2044\n",
      "85\t43197\t0.2\t1\t41186 2011\n",
      "86\t43336\t0.2\t1\t41365 1971\n",
      "87\t42821\t0.2\t1\t40993 1828\n",
      "88\t42339\t0.2\t1\t40455 1884\n",
      "89\t37924\t0.2\t1\t36191 1733\n",
      "90\t36727\t0.2\t1\t35019 1708\n",
      "91\t33632\t0.2\t1\t32108 1524\n",
      "92\t32065\t0.2\t1\t30559 1506\n",
      "93\t30818\t0.2\t1\t29434 1384\n",
      "94\t30025\t0.2\t1\t28706 1319\n",
      "95\t28433\t0.2\t1\t27167 1266\n",
      "96\t27472\t0.2\t1\t26315 1157\n",
      "97\t25080\t0.2\t1\t23956 1124\n",
      "98\t23110\t0.2\t1\t22057 1053\n",
      "99\t21635\t0.2\t1\t20659 976\n",
      "100\t19482\t0.2\t1\t18587 895\n",
      "101\t17939\t0.2\t1\t17090 849\n",
      "102\t17260\t0.2\t1\t16487 773\n",
      "103\t15967\t0.2\t1\t15237 730\n",
      "104\t15618\t0.2\t1\t14883 735\n",
      "105\t13954\t0.2\t1\t13334 620\n",
      "106\t12766\t0.2\t1\t12218 548\n",
      "107\t11338\t0.2\t1\t10826 512\n",
      "108\t10218\t0.2\t1\t9719 499\n",
      "109\t9371\t0.2\t1\t8900 471\n",
      "110\t8643\t0.2\t1\t8250 393\n",
      "111\t8343\t0.2\t1\t7959 384\n",
      "112\t7194\t0.2\t1\t6837 357\n",
      "113\t6183\t0.2\t1\t5873 310\n",
      "114\t5369\t0.2\t1\t5131 238\n",
      "115\t4668\t0.2\t1\t4438 230\n",
      "116\t3833\t0.2\t1\t3644 189\n",
      "117\t3219\t0.2\t1\t3007 212\n",
      "118\t2818\t0.2\t1\t2669 149\n",
      "119\t2453\t0.2\t1\t2312 141\n",
      "120\t2060\t0.2\t1\t1936 124\n",
      "121\t1912\t0.2\t1\t1802 110\n",
      "122\t1619\t0.2\t1\t1536 83\n",
      "123\t1396\t0.2\t1\t1315 81\n",
      "124\t1132\t0.2\t1\t1057 75\n",
      "125\t1047\t0.2\t1\t976 71\n",
      "126\t972\t0.2\t1\t915 57\n",
      "127\t819\t0.2\t1\t758 61\n",
      "128\t624\t0.2\t1\t581 43\n",
      "129\t486\t0.2\t1\t450 36\n",
      "130\t458\t0.2\t1\t430 28\n",
      "131\t510\t0.2\t1\t482 28\n",
      "132\t584\t0.2\t1\t562 22\n",
      "133\t1105\t0.2\t1\t1055 50\n",
      "134\t2683\t0.2\t1\t2615 68\n",
      "135\t802\t0.2\t1\t778 24\n",
      "136\t171\t0.2\t1\t159 12\n",
      "137\t100\t0.2\t1\t93 7\n",
      "138\t80\t0.2\t1\t73 7\n",
      "139\t45\t0.2\t1\t40 5\n",
      "140\t54\t0.2\t1\t48 6\n",
      "141\t22\t0.2\t1\t17 5\n",
      "142\t19\t0.2\t1\t18 1\n",
      "143\t19\t0.2\t1\t16 3\n",
      "144\t60\t0.2\t1\t57 3\n",
      "145\t202\t0.2\t1\t195 7\n",
      "146\t219\t0.2\t1\t210 9\n",
      "147\t1124\t0.2\t1\t1091 33\n",
      "148\t721\t0.2\t1\t692 29\n",
      "149\t428\t0.2\t1\t410 18\n",
      "150\t2417\t0.2\t1\t2310 107\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "12592704 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz, file_2: 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 12592704\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 40242 (0.32%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_4_S12_L001_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_4_S12_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 480.47 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,486,748\n",
      "Reads with adapters:                 8,893,888 (71.2%)\n",
      "Reads written (passing filters):    12,486,748 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,873,012,200 bp\n",
      "Quality-trimmed:               5,043,033 bp (0.3%)\n",
      "Total written (filtered):  1,556,339,778 bp (83.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8893888 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 15.6%\n",
      "  G: 17.3%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2335523\t3121687.0\t0\t2335523\n",
      "2\t288272\t780421.8\t0\t288272\n",
      "3\t160403\t195105.4\t0\t160403\n",
      "4\t85608\t48776.4\t0\t85608\n",
      "5\t58915\t12194.1\t0\t58915\n",
      "6\t57362\t3048.5\t0\t57362\n",
      "7\t44207\t762.1\t0\t44207\n",
      "8\t57380\t190.5\t0\t57380\n",
      "9\t58488\t47.6\t0\t57517 971\n",
      "10\t52992\t11.9\t1\t48842 4150\n",
      "11\t61046\t3.0\t1\t54898 6148\n",
      "12\t57274\t0.7\t1\t52182 5092\n",
      "13\t56700\t0.2\t1\t51661 5039\n",
      "14\t59620\t0.2\t1\t53923 5697\n",
      "15\t59755\t0.2\t1\t54283 5472\n",
      "16\t59818\t0.2\t1\t53514 6304\n",
      "17\t74695\t0.2\t1\t66802 7893\n",
      "18\t60082\t0.2\t1\t55008 5074\n",
      "19\t51866\t0.2\t1\t47284 4582\n",
      "20\t63201\t0.2\t1\t57371 5830\n",
      "21\t69970\t0.2\t1\t63395 6575\n",
      "22\t57953\t0.2\t1\t53164 4789\n",
      "23\t68365\t0.2\t1\t61918 6447\n",
      "24\t63806\t0.2\t1\t56950 6856\n",
      "25\t67673\t0.2\t1\t61556 6117\n",
      "26\t61661\t0.2\t1\t56449 5212\n",
      "27\t70143\t0.2\t1\t63185 6958\n",
      "28\t67771\t0.2\t1\t61544 6227\n",
      "29\t71226\t0.2\t1\t63790 7436\n",
      "30\t75482\t0.2\t1\t69371 6111\n",
      "31\t58014\t0.2\t1\t52857 5157\n",
      "32\t74386\t0.2\t1\t67729 6657\n",
      "33\t63663\t0.2\t1\t57495 6168\n",
      "34\t74108\t0.2\t1\t66532 7576\n",
      "35\t70037\t0.2\t1\t63230 6807\n",
      "36\t76753\t0.2\t1\t69831 6922\n",
      "37\t67916\t0.2\t1\t61720 6196\n",
      "38\t71617\t0.2\t1\t65621 5996\n",
      "39\t75012\t0.2\t1\t68484 6528\n",
      "40\t71594\t0.2\t1\t64730 6864\n",
      "41\t78317\t0.2\t1\t70095 8222\n",
      "42\t72352\t0.2\t1\t64854 7498\n",
      "43\t105118\t0.2\t1\t97302 7816\n",
      "44\t39192\t0.2\t1\t34640 4552\n",
      "45\t76811\t0.2\t1\t69875 6936\n",
      "46\t73857\t0.2\t1\t67212 6645\n",
      "47\t78745\t0.2\t1\t71969 6776\n",
      "48\t69568\t0.2\t1\t62853 6715\n",
      "49\t81769\t0.2\t1\t74077 7692\n",
      "50\t65947\t0.2\t1\t60677 5270\n",
      "51\t71662\t0.2\t1\t65506 6156\n",
      "52\t80128\t0.2\t1\t72415 7713\n",
      "53\t85941\t0.2\t1\t79178 6763\n",
      "54\t61047\t0.2\t1\t55540 5507\n",
      "55\t75533\t0.2\t1\t69160 6373\n",
      "56\t87285\t0.2\t1\t80207 7078\n",
      "57\t70909\t0.2\t1\t64686 6223\n",
      "58\t68890\t0.2\t1\t64023 4867\n",
      "59\t56798\t0.2\t1\t51598 5200\n",
      "60\t76540\t0.2\t1\t70247 6293\n",
      "61\t72774\t0.2\t1\t66897 5877\n",
      "62\t72381\t0.2\t1\t66643 5738\n",
      "63\t67661\t0.2\t1\t61998 5663\n",
      "64\t65324\t0.2\t1\t60081 5243\n",
      "65\t70599\t0.2\t1\t64735 5864\n",
      "66\t77128\t0.2\t1\t70592 6536\n",
      "67\t84260\t0.2\t1\t76015 8245\n",
      "68\t144203\t0.2\t1\t134904 9299\n",
      "69\t43407\t0.2\t1\t39642 3765\n",
      "70\t15848\t0.2\t1\t13317 2531\n",
      "71\t33644\t0.2\t1\t30540 3104\n",
      "72\t56928\t0.2\t1\t52537 4391\n",
      "73\t61080\t0.2\t1\t56221 4859\n",
      "74\t61529\t0.2\t1\t56438 5091\n",
      "75\t60533\t0.2\t1\t55505 5028\n",
      "76\t58948\t0.2\t1\t53903 5045\n",
      "77\t59043\t0.2\t1\t53976 5067\n",
      "78\t55894\t0.2\t1\t51078 4816\n",
      "79\t55224\t0.2\t1\t50589 4635\n",
      "80\t53749\t0.2\t1\t49095 4654\n",
      "81\t52818\t0.2\t1\t48513 4305\n",
      "82\t51098\t0.2\t1\t46773 4325\n",
      "83\t49182\t0.2\t1\t45013 4169\n",
      "84\t47970\t0.2\t1\t43859 4111\n",
      "85\t46307\t0.2\t1\t42293 4014\n",
      "86\t45032\t0.2\t1\t41160 3872\n",
      "87\t42066\t0.2\t1\t38573 3493\n",
      "88\t42344\t0.2\t1\t38592 3752\n",
      "89\t39930\t0.2\t1\t36464 3466\n",
      "90\t40509\t0.2\t1\t36927 3582\n",
      "91\t36941\t0.2\t1\t33684 3257\n",
      "92\t35322\t0.2\t1\t32274 3048\n",
      "93\t32946\t0.2\t1\t30001 2945\n",
      "94\t32705\t0.2\t1\t29772 2933\n",
      "95\t29601\t0.2\t1\t27010 2591\n",
      "96\t28129\t0.2\t1\t25553 2576\n",
      "97\t26467\t0.2\t1\t24046 2421\n",
      "98\t25094\t0.2\t1\t22854 2240\n",
      "99\t24292\t0.2\t1\t21966 2326\n",
      "100\t22228\t0.2\t1\t20140 2088\n",
      "101\t20127\t0.2\t1\t18275 1852\n",
      "102\t18734\t0.2\t1\t17008 1726\n",
      "103\t17312\t0.2\t1\t15695 1617\n",
      "104\t16349\t0.2\t1\t14760 1589\n",
      "105\t14697\t0.2\t1\t13311 1386\n",
      "106\t13248\t0.2\t1\t11852 1396\n",
      "107\t12309\t0.2\t1\t11095 1214\n",
      "108\t11118\t0.2\t1\t10010 1108\n",
      "109\t9883\t0.2\t1\t8876 1007\n",
      "110\t9429\t0.2\t1\t8493 936\n",
      "111\t8334\t0.2\t1\t7393 941\n",
      "112\t7520\t0.2\t1\t6729 791\n",
      "113\t6506\t0.2\t1\t5771 735\n",
      "114\t5675\t0.2\t1\t5040 635\n",
      "115\t4923\t0.2\t1\t4354 569\n",
      "116\t4197\t0.2\t1\t3676 521\n",
      "117\t3761\t0.2\t1\t3289 472\n",
      "118\t3198\t0.2\t1\t2835 363\n",
      "119\t2789\t0.2\t1\t2461 328\n",
      "120\t2403\t0.2\t1\t2050 353\n",
      "121\t1996\t0.2\t1\t1771 225\n",
      "122\t1692\t0.2\t1\t1462 230\n",
      "123\t1495\t0.2\t1\t1306 189\n",
      "124\t1325\t0.2\t1\t1129 196\n",
      "125\t1099\t0.2\t1\t932 167\n",
      "126\t1059\t0.2\t1\t919 140\n",
      "127\t931\t0.2\t1\t822 109\n",
      "128\t654\t0.2\t1\t548 106\n",
      "129\t550\t0.2\t1\t477 73\n",
      "130\t491\t0.2\t1\t418 73\n",
      "131\t529\t0.2\t1\t438 91\n",
      "132\t576\t0.2\t1\t513 63\n",
      "133\t972\t0.2\t1\t858 114\n",
      "134\t2641\t0.2\t1\t2458 183\n",
      "135\t835\t0.2\t1\t788 47\n",
      "136\t183\t0.2\t1\t162 21\n",
      "137\t124\t0.2\t1\t105 19\n",
      "138\t78\t0.2\t1\t70 8\n",
      "139\t41\t0.2\t1\t32 9\n",
      "140\t43\t0.2\t1\t37 6\n",
      "141\t24\t0.2\t1\t20 4\n",
      "142\t18\t0.2\t1\t15 3\n",
      "143\t16\t0.2\t1\t12 4\n",
      "144\t54\t0.2\t1\t49 5\n",
      "145\t231\t0.2\t1\t211 20\n",
      "146\t231\t0.2\t1\t210 21\n",
      "147\t1327\t0.2\t1\t1201 126\n",
      "148\t804\t0.2\t1\t738 66\n",
      "149\t437\t0.2\t1\t393 44\n",
      "150\t3016\t0.2\t1\t2525 491\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "12486748 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_26psu_4_S12_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 523.49 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,486,748\n",
      "Reads with adapters:                 8,772,809 (70.3%)\n",
      "Reads written (passing filters):    12,486,748 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,873,012,200 bp\n",
      "Quality-trimmed:              12,376,106 bp (0.7%)\n",
      "Total written (filtered):  1,558,147,987 bp (83.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 8772809 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.4%\n",
      "  C: 12.6%\n",
      "  G: 20.8%\n",
      "  T: 36.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2358875\t3121687.0\t0\t2358875\n",
      "2\t288344\t780421.8\t0\t288344\n",
      "3\t173054\t195105.4\t0\t173054\n",
      "4\t87112\t48776.4\t0\t87112\n",
      "5\t64247\t12194.1\t0\t64247\n",
      "6\t55212\t3048.5\t0\t55212\n",
      "7\t41727\t762.1\t0\t41727\n",
      "8\t62151\t190.5\t0\t62151\n",
      "9\t50342\t47.6\t0\t50157 185\n",
      "10\t52158\t11.9\t1\t48148 4010\n",
      "11\t57855\t3.0\t1\t52397 5458\n",
      "12\t56897\t0.7\t1\t51566 5331\n",
      "13\t64575\t0.2\t1\t58979 5596\n",
      "14\t59154\t0.2\t1\t53315 5839\n",
      "15\t47002\t0.2\t1\t42866 4136\n",
      "16\t69658\t0.2\t1\t63405 6253\n",
      "17\t57386\t0.2\t1\t52045 5341\n",
      "18\t51929\t0.2\t1\t47327 4602\n",
      "19\t66863\t0.2\t1\t60907 5956\n",
      "20\t62288\t0.2\t1\t57071 5217\n",
      "21\t55272\t0.2\t1\t50547 4725\n",
      "22\t61984\t0.2\t1\t56766 5218\n",
      "23\t65026\t0.2\t1\t59587 5439\n",
      "24\t63838\t0.2\t1\t58214 5624\n",
      "25\t72434\t0.2\t1\t66848 5586\n",
      "26\t60587\t0.2\t1\t55112 5475\n",
      "27\t67754\t0.2\t1\t61588 6166\n",
      "28\t59366\t0.2\t1\t54979 4387\n",
      "29\t67701\t0.2\t1\t62118 5583\n",
      "30\t62432\t0.2\t1\t57688 4744\n",
      "31\t72162\t0.2\t1\t66403 5759\n",
      "32\t60976\t0.2\t1\t56497 4479\n",
      "33\t69840\t0.2\t1\t64116 5724\n",
      "34\t91102\t0.2\t1\t84079 7023\n",
      "35\t47382\t0.2\t1\t43590 3792\n",
      "36\t69435\t0.2\t1\t63655 5780\n",
      "37\t74003\t0.2\t1\t67997 6006\n",
      "38\t68560\t0.2\t1\t63793 4767\n",
      "39\t65708\t0.2\t1\t61286 4422\n",
      "40\t74240\t0.2\t1\t68710 5530\n",
      "41\t72214\t0.2\t1\t67124 5090\n",
      "42\t72980\t0.2\t1\t67853 5127\n",
      "43\t66699\t0.2\t1\t62062 4637\n",
      "44\t76923\t0.2\t1\t71373 5550\n",
      "45\t80740\t0.2\t1\t74874 5866\n",
      "46\t58943\t0.2\t1\t54854 4089\n",
      "47\t73645\t0.2\t1\t68927 4718\n",
      "48\t72329\t0.2\t1\t67389 4940\n",
      "49\t74646\t0.2\t1\t69490 5156\n",
      "50\t70816\t0.2\t1\t65987 4829\n",
      "51\t86746\t0.2\t1\t80793 5953\n",
      "52\t63282\t0.2\t1\t59102 4180\n",
      "53\t76728\t0.2\t1\t71964 4764\n",
      "54\t55670\t0.2\t1\t52124 3546\n",
      "55\t73350\t0.2\t1\t68943 4407\n",
      "56\t76024\t0.2\t1\t70858 5166\n",
      "57\t71982\t0.2\t1\t67849 4133\n",
      "58\t68868\t0.2\t1\t64881 3987\n",
      "59\t68360\t0.2\t1\t64151 4209\n",
      "60\t71518\t0.2\t1\t67054 4464\n",
      "61\t73466\t0.2\t1\t69155 4311\n",
      "62\t79595\t0.2\t1\t74642 4953\n",
      "63\t103748\t0.2\t1\t97894 5854\n",
      "64\t85376\t0.2\t1\t81260 4116\n",
      "65\t55857\t0.2\t1\t52588 3269\n",
      "66\t61954\t0.2\t1\t58405 3549\n",
      "67\t59930\t0.2\t1\t56159 3771\n",
      "68\t60490\t0.2\t1\t56878 3612\n",
      "69\t60452\t0.2\t1\t56708 3744\n",
      "70\t61904\t0.2\t1\t58197 3707\n",
      "71\t65599\t0.2\t1\t61887 3712\n",
      "72\t62591\t0.2\t1\t58936 3655\n",
      "73\t64906\t0.2\t1\t61091 3815\n",
      "74\t60780\t0.2\t1\t57167 3613\n",
      "75\t57385\t0.2\t1\t53955 3430\n",
      "76\t55285\t0.2\t1\t51973 3312\n",
      "77\t55087\t0.2\t1\t51740 3347\n",
      "78\t52962\t0.2\t1\t49812 3150\n",
      "79\t51023\t0.2\t1\t48174 2849\n",
      "80\t50358\t0.2\t1\t47516 2842\n",
      "81\t49364\t0.2\t1\t46484 2880\n",
      "82\t47242\t0.2\t1\t44529 2713\n",
      "83\t45686\t0.2\t1\t43106 2580\n",
      "84\t44410\t0.2\t1\t41863 2547\n",
      "85\t43592\t0.2\t1\t41103 2489\n",
      "86\t43331\t0.2\t1\t40952 2379\n",
      "87\t42675\t0.2\t1\t40428 2247\n",
      "88\t42372\t0.2\t1\t40023 2349\n",
      "89\t37747\t0.2\t1\t35602 2145\n",
      "90\t37018\t0.2\t1\t34943 2075\n",
      "91\t33562\t0.2\t1\t31712 1850\n",
      "92\t31899\t0.2\t1\t30215 1684\n",
      "93\t30547\t0.2\t1\t28874 1673\n",
      "94\t30566\t0.2\t1\t28886 1680\n",
      "95\t28277\t0.2\t1\t26704 1573\n",
      "96\t27180\t0.2\t1\t25709 1471\n",
      "97\t24703\t0.2\t1\t23381 1322\n",
      "98\t23174\t0.2\t1\t21893 1281\n",
      "99\t21741\t0.2\t1\t20527 1214\n",
      "100\t19732\t0.2\t1\t18608 1124\n",
      "101\t17665\t0.2\t1\t16698 967\n",
      "102\t16908\t0.2\t1\t15954 954\n",
      "103\t15998\t0.2\t1\t15059 939\n",
      "104\t15634\t0.2\t1\t14747 887\n",
      "105\t13889\t0.2\t1\t13156 733\n",
      "106\t12626\t0.2\t1\t11933 693\n",
      "107\t11469\t0.2\t1\t10791 678\n",
      "108\t10122\t0.2\t1\t9581 541\n",
      "109\t9139\t0.2\t1\t8610 529\n",
      "110\t8759\t0.2\t1\t8236 523\n",
      "111\t8023\t0.2\t1\t7545 478\n",
      "112\t7113\t0.2\t1\t6705 408\n",
      "113\t6235\t0.2\t1\t5883 352\n",
      "114\t5409\t0.2\t1\t5057 352\n",
      "115\t4485\t0.2\t1\t4203 282\n",
      "116\t3795\t0.2\t1\t3556 239\n",
      "117\t3367\t0.2\t1\t3120 247\n",
      "118\t2891\t0.2\t1\t2702 189\n",
      "119\t2519\t0.2\t1\t2367 152\n",
      "120\t2120\t0.2\t1\t1963 157\n",
      "121\t1866\t0.2\t1\t1743 123\n",
      "122\t1561\t0.2\t1\t1440 121\n",
      "123\t1426\t0.2\t1\t1322 104\n",
      "124\t1196\t0.2\t1\t1106 90\n",
      "125\t1010\t0.2\t1\t943 67\n",
      "126\t988\t0.2\t1\t920 68\n",
      "127\t903\t0.2\t1\t840 63\n",
      "128\t611\t0.2\t1\t560 51\n",
      "129\t510\t0.2\t1\t470 40\n",
      "130\t453\t0.2\t1\t423 30\n",
      "131\t514\t0.2\t1\t467 47\n",
      "132\t553\t0.2\t1\t529 24\n",
      "133\t1090\t0.2\t1\t1032 58\n",
      "134\t2641\t0.2\t1\t2566 75\n",
      "135\t828\t0.2\t1\t794 34\n",
      "136\t185\t0.2\t1\t169 16\n",
      "137\t110\t0.2\t1\t95 15\n",
      "138\t79\t0.2\t1\t73 6\n",
      "139\t44\t0.2\t1\t40 4\n",
      "140\t39\t0.2\t1\t36 3\n",
      "141\t22\t0.2\t1\t17 5\n",
      "142\t20\t0.2\t1\t14 6\n",
      "143\t16\t0.2\t1\t15 1\n",
      "144\t54\t0.2\t1\t47 7\n",
      "145\t219\t0.2\t1\t204 15\n",
      "146\t229\t0.2\t1\t213 16\n",
      "147\t1254\t0.2\t1\t1201 53\n",
      "148\t756\t0.2\t1\t716 40\n",
      "149\t417\t0.2\t1\t400 17\n",
      "150\t2389\t0.2\t1\t2234 155\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "12486748 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz, file_2: 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 12486748\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 38356 (0.31%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_26psu_4_S12_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_26psu_4_S12_L002_R1_001_trimmed.fq.gz and 8C_26psu_4_S12_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_1_S5_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 446.69 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,000,021\n",
      "Reads with adapters:                12,585,652 (89.9%)\n",
      "Reads written (passing filters):    14,000,021 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,100,003,150 bp\n",
      "Quality-trimmed:               3,933,676 bp (0.2%)\n",
      "Total written (filtered):  1,383,592,146 bp (65.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12585652 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 15.4%\n",
      "  G: 21.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t794832\t3500005.2\t0\t794832\n",
      "2\t137499\t875001.3\t0\t137499\n",
      "3\t88574\t218750.3\t0\t88574\n",
      "4\t55463\t54687.6\t0\t55463\n",
      "5\t52467\t13671.9\t0\t52467\n",
      "6\t51009\t3418.0\t0\t51009\n",
      "7\t43440\t854.5\t0\t43440\n",
      "8\t53882\t213.6\t0\t53882\n",
      "9\t54563\t53.4\t0\t53971 592\n",
      "10\t51819\t13.4\t1\t49209 2610\n",
      "11\t57911\t3.3\t1\t53768 4143\n",
      "12\t56442\t0.8\t1\t53088 3354\n",
      "13\t57081\t0.2\t1\t53583 3498\n",
      "14\t60408\t0.2\t1\t56320 4088\n",
      "15\t62034\t0.2\t1\t58378 3656\n",
      "16\t63250\t0.2\t1\t58624 4626\n",
      "17\t76887\t0.2\t1\t70864 6023\n",
      "18\t66288\t0.2\t1\t62429 3859\n",
      "19\t56979\t0.2\t1\t53412 3567\n",
      "20\t69709\t0.2\t1\t65411 4298\n",
      "21\t77411\t0.2\t1\t72341 5070\n",
      "22\t66479\t0.2\t1\t62829 3650\n",
      "23\t78177\t0.2\t1\t73134 5043\n",
      "24\t74482\t0.2\t1\t68678 5804\n",
      "25\t80818\t0.2\t1\t75690 5128\n",
      "26\t74228\t0.2\t1\t70119 4109\n",
      "27\t83394\t0.2\t1\t77679 5715\n",
      "28\t82403\t0.2\t1\t76936 5467\n",
      "29\t90670\t0.2\t1\t83444 7226\n",
      "30\t98605\t0.2\t1\t93586 5019\n",
      "31\t72678\t0.2\t1\t68276 4402\n",
      "32\t92171\t0.2\t1\t86613 5558\n",
      "33\t90475\t0.2\t1\t84682 5793\n",
      "34\t99774\t0.2\t1\t93593 6181\n",
      "35\t93327\t0.2\t1\t87313 6014\n",
      "36\t107093\t0.2\t1\t100026 7067\n",
      "37\t103597\t0.2\t1\t96127 7470\n",
      "38\t104511\t0.2\t1\t98484 6027\n",
      "39\t106907\t0.2\t1\t100401 6506\n",
      "40\t106662\t0.2\t1\t100220 6442\n",
      "41\t122253\t0.2\t1\t114028 8225\n",
      "42\t104346\t0.2\t1\t96706 7640\n",
      "43\t169940\t0.2\t1\t161783 8157\n",
      "44\t58164\t0.2\t1\t53449 4715\n",
      "45\t118320\t0.2\t1\t111612 6708\n",
      "46\t116293\t0.2\t1\t109496 6797\n",
      "47\t124205\t0.2\t1\t117076 7129\n",
      "48\t123424\t0.2\t1\t116139 7285\n",
      "49\t135448\t0.2\t1\t126575 8873\n",
      "50\t119092\t0.2\t1\t112535 6557\n",
      "51\t128971\t0.2\t1\t121071 7900\n",
      "52\t136415\t0.2\t1\t127168 9247\n",
      "53\t146369\t0.2\t1\t138214 8155\n",
      "54\t119815\t0.2\t1\t112779 7036\n",
      "55\t143900\t0.2\t1\t135212 8688\n",
      "56\t174050\t0.2\t1\t164777 9273\n",
      "57\t117416\t0.2\t1\t109615 7801\n",
      "58\t150456\t0.2\t1\t142969 7487\n",
      "59\t132324\t0.2\t1\t124478 7846\n",
      "60\t149629\t0.2\t1\t140649 8980\n",
      "61\t147926\t0.2\t1\t139679 8247\n",
      "62\t163513\t0.2\t1\t154547 8966\n",
      "63\t134239\t0.2\t1\t125798 8441\n",
      "64\t148604\t0.2\t1\t140497 8107\n",
      "65\t167908\t0.2\t1\t158298 9610\n",
      "66\t177387\t0.2\t1\t167350 10037\n",
      "67\t194374\t0.2\t1\t180373 14001\n",
      "68\t370239\t0.2\t1\t354139 16100\n",
      "69\t62415\t0.2\t1\t57251 5164\n",
      "70\t26365\t0.2\t1\t22180 4185\n",
      "71\t101894\t0.2\t1\t96350 5544\n",
      "72\t143779\t0.2\t1\t135850 7929\n",
      "73\t154356\t0.2\t1\t145962 8394\n",
      "74\t154436\t0.2\t1\t145829 8607\n",
      "75\t155208\t0.2\t1\t146440 8768\n",
      "76\t154614\t0.2\t1\t146122 8492\n",
      "77\t156184\t0.2\t1\t147502 8682\n",
      "78\t149615\t0.2\t1\t141324 8291\n",
      "79\t149903\t0.2\t1\t141513 8390\n",
      "80\t146761\t0.2\t1\t138705 8056\n",
      "81\t147739\t0.2\t1\t139495 8244\n",
      "82\t146032\t0.2\t1\t137919 8113\n",
      "83\t142429\t0.2\t1\t134446 7983\n",
      "84\t139641\t0.2\t1\t132001 7640\n",
      "85\t137362\t0.2\t1\t129737 7625\n",
      "86\t134899\t0.2\t1\t127375 7524\n",
      "87\t130083\t0.2\t1\t122670 7413\n",
      "88\t129596\t0.2\t1\t122229 7367\n",
      "89\t124621\t0.2\t1\t117492 7129\n",
      "90\t125612\t0.2\t1\t118454 7158\n",
      "91\t119798\t0.2\t1\t113079 6719\n",
      "92\t115717\t0.2\t1\t109150 6567\n",
      "93\t109774\t0.2\t1\t103564 6210\n",
      "94\t107743\t0.2\t1\t101578 6165\n",
      "95\t101804\t0.2\t1\t95988 5816\n",
      "96\t96724\t0.2\t1\t91079 5645\n",
      "97\t92176\t0.2\t1\t86874 5302\n",
      "98\t87424\t0.2\t1\t82235 5189\n",
      "99\t86913\t0.2\t1\t81280 5633\n",
      "100\t78985\t0.2\t1\t74394 4591\n",
      "101\t75410\t0.2\t1\t70955 4455\n",
      "102\t69075\t0.2\t1\t64932 4143\n",
      "103\t64987\t0.2\t1\t61075 3912\n",
      "104\t61751\t0.2\t1\t58031 3720\n",
      "105\t55260\t0.2\t1\t51879 3381\n",
      "106\t50545\t0.2\t1\t47309 3236\n",
      "107\t46617\t0.2\t1\t43701 2916\n",
      "108\t44206\t0.2\t1\t41370 2836\n",
      "109\t39556\t0.2\t1\t36926 2630\n",
      "110\t36531\t0.2\t1\t34163 2368\n",
      "111\t32673\t0.2\t1\t30562 2111\n",
      "112\t28896\t0.2\t1\t26907 1989\n",
      "113\t25292\t0.2\t1\t23480 1812\n",
      "114\t22657\t0.2\t1\t21036 1621\n",
      "115\t19857\t0.2\t1\t18470 1387\n",
      "116\t17394\t0.2\t1\t16192 1202\n",
      "117\t14695\t0.2\t1\t13605 1090\n",
      "118\t12814\t0.2\t1\t11815 999\n",
      "119\t11036\t0.2\t1\t10138 898\n",
      "120\t9387\t0.2\t1\t8609 778\n",
      "121\t8006\t0.2\t1\t7333 673\n",
      "122\t6428\t0.2\t1\t5956 472\n",
      "123\t5508\t0.2\t1\t5035 473\n",
      "124\t4687\t0.2\t1\t4248 439\n",
      "125\t3879\t0.2\t1\t3508 371\n",
      "126\t3097\t0.2\t1\t2779 318\n",
      "127\t2720\t0.2\t1\t2457 263\n",
      "128\t2100\t0.2\t1\t1898 202\n",
      "129\t1767\t0.2\t1\t1595 172\n",
      "130\t1354\t0.2\t1\t1200 154\n",
      "131\t1174\t0.2\t1\t1057 117\n",
      "132\t1133\t0.2\t1\t1029 104\n",
      "133\t1115\t0.2\t1\t1004 111\n",
      "134\t1260\t0.2\t1\t1149 111\n",
      "135\t611\t0.2\t1\t547 64\n",
      "136\t316\t0.2\t1\t271 45\n",
      "137\t241\t0.2\t1\t219 22\n",
      "138\t161\t0.2\t1\t140 21\n",
      "139\t107\t0.2\t1\t93 14\n",
      "140\t137\t0.2\t1\t117 20\n",
      "141\t53\t0.2\t1\t47 6\n",
      "142\t47\t0.2\t1\t41 6\n",
      "143\t37\t0.2\t1\t32 5\n",
      "144\t37\t0.2\t1\t26 11\n",
      "145\t44\t0.2\t1\t34 10\n",
      "146\t50\t0.2\t1\t46 4\n",
      "147\t126\t0.2\t1\t111 15\n",
      "148\t84\t0.2\t1\t69 15\n",
      "149\t71\t0.2\t1\t62 9\n",
      "150\t642\t0.2\t1\t568 74\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14000021 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_1_S5_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 467.66 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,000,021\n",
      "Reads with adapters:                12,482,400 (89.2%)\n",
      "Reads written (passing filters):    14,000,021 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,100,003,150 bp\n",
      "Quality-trimmed:              11,507,684 bp (0.5%)\n",
      "Total written (filtered):  1,389,836,957 bp (66.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12482400 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 13.5%\n",
      "  G: 25.9%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t842057\t3500005.2\t0\t842057\n",
      "2\t142966\t875001.3\t0\t142966\n",
      "3\t90255\t218750.3\t0\t90255\n",
      "4\t55571\t54687.6\t0\t55571\n",
      "5\t57545\t13671.9\t0\t57545\n",
      "6\t49671\t3418.0\t0\t49671\n",
      "7\t41298\t854.5\t0\t41298\n",
      "8\t59001\t213.6\t0\t59001\n",
      "9\t47476\t53.4\t0\t47315 161\n",
      "10\t53026\t13.4\t1\t50683 2343\n",
      "11\t55985\t3.3\t1\t52789 3196\n",
      "12\t57070\t0.8\t1\t53792 3278\n",
      "13\t64601\t0.2\t1\t61294 3307\n",
      "14\t62201\t0.2\t1\t58283 3918\n",
      "15\t50727\t0.2\t1\t47889 2838\n",
      "16\t74265\t0.2\t1\t70212 4053\n",
      "17\t60248\t0.2\t1\t56657 3591\n",
      "18\t57873\t0.2\t1\t54797 3076\n",
      "19\t74512\t0.2\t1\t70251 4261\n",
      "20\t70260\t0.2\t1\t66610 3650\n",
      "21\t62230\t0.2\t1\t58933 3297\n",
      "22\t72026\t0.2\t1\t68190 3836\n",
      "23\t75580\t0.2\t1\t71483 4097\n",
      "24\t76103\t0.2\t1\t71906 4197\n",
      "25\t87020\t0.2\t1\t82705 4315\n",
      "26\t72653\t0.2\t1\t68640 4013\n",
      "27\t81722\t0.2\t1\t76658 5064\n",
      "28\t76893\t0.2\t1\t73629 3264\n",
      "29\t84223\t0.2\t1\t79742 4481\n",
      "30\t83568\t0.2\t1\t79746 3822\n",
      "31\t92428\t0.2\t1\t87765 4663\n",
      "32\t83785\t0.2\t1\t79986 3799\n",
      "33\t94083\t0.2\t1\t89299 4784\n",
      "34\t124234\t0.2\t1\t117883 6351\n",
      "35\t70662\t0.2\t1\t67495 3167\n",
      "36\t97571\t0.2\t1\t92509 5062\n",
      "37\t105963\t0.2\t1\t100553 5410\n",
      "38\t100112\t0.2\t1\t95880 4232\n",
      "39\t98883\t0.2\t1\t94704 4179\n",
      "40\t111592\t0.2\t1\t106078 5514\n",
      "41\t108482\t0.2\t1\t103543 4939\n",
      "42\t113048\t0.2\t1\t108097 4951\n",
      "43\t104564\t0.2\t1\t100168 4396\n",
      "44\t122044\t0.2\t1\t116433 5611\n",
      "45\t132901\t0.2\t1\t126882 6019\n",
      "46\t96495\t0.2\t1\t92302 4193\n",
      "47\t123777\t0.2\t1\t118653 5124\n",
      "48\t123419\t0.2\t1\t118253 5166\n",
      "49\t130090\t0.2\t1\t124494 5596\n",
      "50\t124769\t0.2\t1\t119306 5463\n",
      "51\t159790\t0.2\t1\t152747 7043\n",
      "52\t114726\t0.2\t1\t110175 4551\n",
      "53\t140573\t0.2\t1\t135310 5263\n",
      "54\t107631\t0.2\t1\t103512 4119\n",
      "55\t142110\t0.2\t1\t137013 5097\n",
      "56\t148511\t0.2\t1\t142711 5800\n",
      "57\t141596\t0.2\t1\t136449 5147\n",
      "58\t138299\t0.2\t1\t133566 4733\n",
      "59\t142121\t0.2\t1\t137002 5119\n",
      "60\t148830\t0.2\t1\t143312 5518\n",
      "61\t154735\t0.2\t1\t148878 5857\n",
      "62\t170128\t0.2\t1\t163868 6260\n",
      "63\t234733\t0.2\t1\t226789 7944\n",
      "64\t212296\t0.2\t1\t206143 6153\n",
      "65\t126771\t0.2\t1\t122389 4382\n",
      "66\t138120\t0.2\t1\t133306 4814\n",
      "67\t139286\t0.2\t1\t134193 5093\n",
      "68\t141731\t0.2\t1\t136845 4886\n",
      "69\t141571\t0.2\t1\t136513 5058\n",
      "70\t151101\t0.2\t1\t145752 5349\n",
      "71\t165000\t0.2\t1\t159543 5457\n",
      "72\t156137\t0.2\t1\t150635 5502\n",
      "73\t168957\t0.2\t1\t163425 5532\n",
      "74\t155346\t0.2\t1\t150282 5064\n",
      "75\t149711\t0.2\t1\t144684 5027\n",
      "76\t146688\t0.2\t1\t141765 4923\n",
      "77\t148035\t0.2\t1\t142983 5052\n",
      "78\t144141\t0.2\t1\t139293 4848\n",
      "79\t140116\t0.2\t1\t135357 4759\n",
      "80\t139815\t0.2\t1\t135255 4560\n",
      "81\t140275\t0.2\t1\t135701 4574\n",
      "82\t136237\t0.2\t1\t131702 4535\n",
      "83\t133248\t0.2\t1\t128960 4288\n",
      "84\t130802\t0.2\t1\t126388 4414\n",
      "85\t131118\t0.2\t1\t126640 4478\n",
      "86\t132538\t0.2\t1\t128154 4384\n",
      "87\t134528\t0.2\t1\t130166 4362\n",
      "88\t132145\t0.2\t1\t127913 4232\n",
      "89\t119589\t0.2\t1\t115716 3873\n",
      "90\t116381\t0.2\t1\t112493 3888\n",
      "91\t109945\t0.2\t1\t106186 3759\n",
      "92\t104899\t0.2\t1\t101501 3398\n",
      "93\t102752\t0.2\t1\t99467 3285\n",
      "94\t100398\t0.2\t1\t97258 3140\n",
      "95\t96752\t0.2\t1\t93687 3065\n",
      "96\t92474\t0.2\t1\t89641 2833\n",
      "97\t85160\t0.2\t1\t82565 2595\n",
      "98\t81057\t0.2\t1\t78431 2626\n",
      "99\t79148\t0.2\t1\t76636 2512\n",
      "100\t72036\t0.2\t1\t69758 2278\n",
      "101\t69452\t0.2\t1\t67404 2048\n",
      "102\t66047\t0.2\t1\t64053 1994\n",
      "103\t62024\t0.2\t1\t60217 1807\n",
      "104\t59709\t0.2\t1\t57936 1773\n",
      "105\t53244\t0.2\t1\t51636 1608\n",
      "106\t48557\t0.2\t1\t47145 1412\n",
      "107\t44301\t0.2\t1\t42956 1345\n",
      "108\t40672\t0.2\t1\t39383 1289\n",
      "109\t36835\t0.2\t1\t35717 1118\n",
      "110\t34356\t0.2\t1\t33352 1004\n",
      "111\t32280\t0.2\t1\t31328 952\n",
      "112\t28083\t0.2\t1\t27306 777\n",
      "113\t24882\t0.2\t1\t24139 743\n",
      "114\t22038\t0.2\t1\t21386 652\n",
      "115\t18554\t0.2\t1\t17942 612\n",
      "116\t16085\t0.2\t1\t15523 562\n",
      "117\t13238\t0.2\t1\t12820 418\n",
      "118\t11657\t0.2\t1\t11269 388\n",
      "119\t10199\t0.2\t1\t9853 346\n",
      "120\t8562\t0.2\t1\t8250 312\n",
      "121\t7440\t0.2\t1\t7187 253\n",
      "122\t5970\t0.2\t1\t5758 212\n",
      "123\t5287\t0.2\t1\t5099 188\n",
      "124\t4439\t0.2\t1\t4276 163\n",
      "125\t3716\t0.2\t1\t3583 133\n",
      "126\t2906\t0.2\t1\t2772 134\n",
      "127\t2521\t0.2\t1\t2432 89\n",
      "128\t1926\t0.2\t1\t1855 71\n",
      "129\t1631\t0.2\t1\t1564 67\n",
      "130\t1264\t0.2\t1\t1221 43\n",
      "131\t1080\t0.2\t1\t1044 36\n",
      "132\t1051\t0.2\t1\t1017 34\n",
      "133\t1128\t0.2\t1\t1088 40\n",
      "134\t1229\t0.2\t1\t1204 25\n",
      "135\t567\t0.2\t1\t545 22\n",
      "136\t312\t0.2\t1\t296 16\n",
      "137\t233\t0.2\t1\t225 8\n",
      "138\t145\t0.2\t1\t135 10\n",
      "139\t105\t0.2\t1\t99 6\n",
      "140\t127\t0.2\t1\t124 3\n",
      "141\t54\t0.2\t1\t50 4\n",
      "142\t42\t0.2\t1\t36 6\n",
      "143\t34\t0.2\t1\t31 3\n",
      "144\t35\t0.2\t1\t32 3\n",
      "145\t45\t0.2\t1\t43 2\n",
      "146\t40\t0.2\t1\t38 2\n",
      "147\t105\t0.2\t1\t96 9\n",
      "148\t62\t0.2\t1\t59 3\n",
      "149\t54\t0.2\t1\t50 4\n",
      "150\t462\t0.2\t1\t437 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14000021 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz, file_2: 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14000021\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 37412 (0.27%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_1_S5_L001_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_1_S5_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 452.93 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,980,098\n",
      "Reads with adapters:                12,547,273 (89.8%)\n",
      "Reads written (passing filters):    13,980,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,097,014,700 bp\n",
      "Quality-trimmed:               3,995,329 bp (0.2%)\n",
      "Total written (filtered):  1,381,322,734 bp (65.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12547273 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 15.4%\n",
      "  G: 21.7%\n",
      "  T: 34.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t797770\t3495024.5\t0\t797770\n",
      "2\t136013\t873756.1\t0\t136013\n",
      "3\t88578\t218439.0\t0\t88578\n",
      "4\t54507\t54609.8\t0\t54507\n",
      "5\t51433\t13652.4\t0\t51433\n",
      "6\t50368\t3413.1\t0\t50368\n",
      "7\t41781\t853.3\t0\t41781\n",
      "8\t52923\t213.3\t0\t52923\n",
      "9\t54051\t53.3\t0\t53431 620\n",
      "10\t50288\t13.3\t1\t47174 3114\n",
      "11\t58137\t3.3\t1\t53328 4809\n",
      "12\t55530\t0.8\t1\t51625 3905\n",
      "13\t56413\t0.2\t1\t52378 4035\n",
      "14\t59004\t0.2\t1\t54278 4726\n",
      "15\t60957\t0.2\t1\t56359 4598\n",
      "16\t62617\t0.2\t1\t57287 5330\n",
      "17\t77034\t0.2\t1\t70041 6993\n",
      "18\t64851\t0.2\t1\t60415 4436\n",
      "19\t55636\t0.2\t1\t51675 3961\n",
      "20\t69068\t0.2\t1\t64118 4950\n",
      "21\t78331\t0.2\t1\t72354 5977\n",
      "22\t65036\t0.2\t1\t60845 4191\n",
      "23\t77471\t0.2\t1\t71757 5714\n",
      "24\t73017\t0.2\t1\t66655 6362\n",
      "25\t80097\t0.2\t1\t74072 6025\n",
      "26\t72725\t0.2\t1\t67966 4759\n",
      "27\t83104\t0.2\t1\t76594 6510\n",
      "28\t81931\t0.2\t1\t75845 6086\n",
      "29\t90548\t0.2\t1\t82404 8144\n",
      "30\t97688\t0.2\t1\t91724 5964\n",
      "31\t71240\t0.2\t1\t66196 5044\n",
      "32\t92668\t0.2\t1\t85724 6944\n",
      "33\t88978\t0.2\t1\t82032 6946\n",
      "34\t98712\t0.2\t1\t91302 7410\n",
      "35\t92968\t0.2\t1\t85787 7181\n",
      "36\t107083\t0.2\t1\t98683 8400\n",
      "37\t102524\t0.2\t1\t93933 8591\n",
      "38\t103053\t0.2\t1\t96018 7035\n",
      "39\t106015\t0.2\t1\t98258 7757\n",
      "40\t105210\t0.2\t1\t97541 7669\n",
      "41\t121909\t0.2\t1\t112224 9685\n",
      "42\t102833\t0.2\t1\t93869 8964\n",
      "43\t167631\t0.2\t1\t157548 10083\n",
      "44\t59347\t0.2\t1\t53867 5480\n",
      "45\t119124\t0.2\t1\t111193 7931\n",
      "46\t115006\t0.2\t1\t106994 8012\n",
      "47\t123238\t0.2\t1\t114804 8434\n",
      "48\t121882\t0.2\t1\t113175 8707\n",
      "49\t136657\t0.2\t1\t126388 10269\n",
      "50\t117352\t0.2\t1\t109816 7536\n",
      "51\t128981\t0.2\t1\t119944 9037\n",
      "52\t136441\t0.2\t1\t126118 10323\n",
      "53\t146650\t0.2\t1\t137000 9650\n",
      "54\t118855\t0.2\t1\t110740 8115\n",
      "55\t142441\t0.2\t1\t132669 9772\n",
      "56\t173419\t0.2\t1\t162260 11159\n",
      "57\t115395\t0.2\t1\t106550 8845\n",
      "58\t148737\t0.2\t1\t140083 8654\n",
      "59\t131828\t0.2\t1\t122332 9496\n",
      "60\t148604\t0.2\t1\t138387 10217\n",
      "61\t147891\t0.2\t1\t138429 9462\n",
      "62\t164090\t0.2\t1\t153607 10483\n",
      "63\t134153\t0.2\t1\t124429 9724\n",
      "64\t146788\t0.2\t1\t137613 9175\n",
      "65\t167405\t0.2\t1\t156219 11186\n",
      "66\t175152\t0.2\t1\t163306 11846\n",
      "67\t193973\t0.2\t1\t178444 15529\n",
      "68\t355893\t0.2\t1\t337623 18270\n",
      "69\t77628\t0.2\t1\t71288 6340\n",
      "70\t32194\t0.2\t1\t27237 4957\n",
      "71\t98744\t0.2\t1\t92367 6377\n",
      "72\t142722\t0.2\t1\t134109 8613\n",
      "73\t153911\t0.2\t1\t144323 9588\n",
      "74\t152982\t0.2\t1\t143267 9715\n",
      "75\t155312\t0.2\t1\t145139 10173\n",
      "76\t154296\t0.2\t1\t143821 10475\n",
      "77\t156257\t0.2\t1\t146141 10116\n",
      "78\t149276\t0.2\t1\t139442 9834\n",
      "79\t150468\t0.2\t1\t140290 10178\n",
      "80\t147626\t0.2\t1\t137573 10053\n",
      "81\t147629\t0.2\t1\t137952 9677\n",
      "82\t146000\t0.2\t1\t136195 9805\n",
      "83\t142550\t0.2\t1\t133188 9362\n",
      "84\t139878\t0.2\t1\t130510 9368\n",
      "85\t136886\t0.2\t1\t127696 9190\n",
      "86\t134793\t0.2\t1\t125685 9108\n",
      "87\t130207\t0.2\t1\t121771 8436\n",
      "88\t129730\t0.2\t1\t121003 8727\n",
      "89\t125338\t0.2\t1\t117059 8279\n",
      "90\t125920\t0.2\t1\t117611 8309\n",
      "91\t120727\t0.2\t1\t112765 7962\n",
      "92\t116230\t0.2\t1\t108426 7804\n",
      "93\t110261\t0.2\t1\t102836 7425\n",
      "94\t107731\t0.2\t1\t100461 7270\n",
      "95\t101765\t0.2\t1\t94845 6920\n",
      "96\t97730\t0.2\t1\t91189 6541\n",
      "97\t92862\t0.2\t1\t86658 6204\n",
      "98\t87553\t0.2\t1\t81547 6006\n",
      "99\t87158\t0.2\t1\t80825 6333\n",
      "100\t78638\t0.2\t1\t73170 5468\n",
      "101\t74653\t0.2\t1\t69508 5145\n",
      "102\t70275\t0.2\t1\t65468 4807\n",
      "103\t65373\t0.2\t1\t60911 4462\n",
      "104\t61766\t0.2\t1\t57502 4264\n",
      "105\t55492\t0.2\t1\t51811 3681\n",
      "106\t51179\t0.2\t1\t47624 3555\n",
      "107\t46618\t0.2\t1\t43293 3325\n",
      "108\t44553\t0.2\t1\t41440 3113\n",
      "109\t39712\t0.2\t1\t36840 2872\n",
      "110\t36630\t0.2\t1\t33977 2653\n",
      "111\t32740\t0.2\t1\t30268 2472\n",
      "112\t29144\t0.2\t1\t26890 2254\n",
      "113\t25615\t0.2\t1\t23550 2065\n",
      "114\t22855\t0.2\t1\t21063 1792\n",
      "115\t19838\t0.2\t1\t18299 1539\n",
      "116\t17636\t0.2\t1\t16221 1415\n",
      "117\t14547\t0.2\t1\t13334 1213\n",
      "118\t13235\t0.2\t1\t12137 1098\n",
      "119\t10988\t0.2\t1\t10016 972\n",
      "120\t9462\t0.2\t1\t8648 814\n",
      "121\t7752\t0.2\t1\t7051 701\n",
      "122\t6591\t0.2\t1\t5988 603\n",
      "123\t5413\t0.2\t1\t4917 496\n",
      "124\t4773\t0.2\t1\t4290 483\n",
      "125\t3899\t0.2\t1\t3520 379\n",
      "126\t3211\t0.2\t1\t2920 291\n",
      "127\t2698\t0.2\t1\t2418 280\n",
      "128\t2171\t0.2\t1\t1932 239\n",
      "129\t1648\t0.2\t1\t1461 187\n",
      "130\t1400\t0.2\t1\t1257 143\n",
      "131\t1218\t0.2\t1\t1094 124\n",
      "132\t1037\t0.2\t1\t929 108\n",
      "133\t1169\t0.2\t1\t1044 125\n",
      "134\t1204\t0.2\t1\t1109 95\n",
      "135\t602\t0.2\t1\t535 67\n",
      "136\t395\t0.2\t1\t343 52\n",
      "137\t232\t0.2\t1\t196 36\n",
      "138\t155\t0.2\t1\t142 13\n",
      "139\t118\t0.2\t1\t95 23\n",
      "140\t87\t0.2\t1\t71 16\n",
      "141\t57\t0.2\t1\t45 12\n",
      "142\t68\t0.2\t1\t58 10\n",
      "143\t37\t0.2\t1\t32 5\n",
      "144\t24\t0.2\t1\t22 2\n",
      "145\t36\t0.2\t1\t30 6\n",
      "146\t26\t0.2\t1\t23 3\n",
      "147\t105\t0.2\t1\t86 19\n",
      "148\t85\t0.2\t1\t66 19\n",
      "149\t76\t0.2\t1\t64 12\n",
      "150\t640\t0.2\t1\t546 94\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13980098 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_1_S5_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 474.16 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,980,098\n",
      "Reads with adapters:                12,441,322 (89.0%)\n",
      "Reads written (passing filters):    13,980,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,097,014,700 bp\n",
      "Quality-trimmed:              11,100,984 bp (0.5%)\n",
      "Total written (filtered):  1,388,068,565 bp (66.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12441322 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 13.5%\n",
      "  G: 25.9%\n",
      "  T: 33.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t845065\t3495024.5\t0\t845065\n",
      "2\t141219\t873756.1\t0\t141219\n",
      "3\t93432\t218439.0\t0\t93432\n",
      "4\t54397\t54609.8\t0\t54397\n",
      "5\t54738\t13652.4\t0\t54738\n",
      "6\t49515\t3413.1\t0\t49515\n",
      "7\t40356\t853.3\t0\t40356\n",
      "8\t58207\t213.3\t0\t58207\n",
      "9\t46723\t53.3\t0\t46571 152\n",
      "10\t50819\t13.3\t1\t47536 3283\n",
      "11\t55858\t3.3\t1\t51611 4247\n",
      "12\t55535\t0.8\t1\t51267 4268\n",
      "13\t64627\t0.2\t1\t59860 4767\n",
      "14\t60188\t0.2\t1\t55222 4966\n",
      "15\t49387\t0.2\t1\t45866 3521\n",
      "16\t72997\t0.2\t1\t67649 5348\n",
      "17\t62490\t0.2\t1\t57882 4608\n",
      "18\t55351\t0.2\t1\t51495 3856\n",
      "19\t73045\t0.2\t1\t67571 5474\n",
      "20\t69542\t0.2\t1\t64858 4684\n",
      "21\t61947\t0.2\t1\t57754 4193\n",
      "22\t70571\t0.2\t1\t65755 4816\n",
      "23\t75040\t0.2\t1\t69830 5210\n",
      "24\t75107\t0.2\t1\t69631 5476\n",
      "25\t86702\t0.2\t1\t81241 5461\n",
      "26\t72229\t0.2\t1\t66945 5284\n",
      "27\t82253\t0.2\t1\t76106 6147\n",
      "28\t73797\t0.2\t1\t69537 4260\n",
      "29\t84769\t0.2\t1\t79179 5590\n",
      "30\t80847\t0.2\t1\t75988 4859\n",
      "31\t93818\t0.2\t1\t87884 5934\n",
      "32\t80840\t0.2\t1\t75981 4859\n",
      "33\t94296\t0.2\t1\t88030 6266\n",
      "34\t125592\t0.2\t1\t117608 7984\n",
      "35\t67216\t0.2\t1\t63145 4071\n",
      "36\t97900\t0.2\t1\t91347 6553\n",
      "37\t105322\t0.2\t1\t98633 6689\n",
      "38\t99129\t0.2\t1\t93524 5605\n",
      "39\t95232\t0.2\t1\t90264 4968\n",
      "40\t111486\t0.2\t1\t104598 6888\n",
      "41\t108655\t0.2\t1\t102291 6364\n",
      "42\t112712\t0.2\t1\t106221 6491\n",
      "43\t103445\t0.2\t1\t97749 5696\n",
      "44\t122874\t0.2\t1\t115506 7368\n",
      "45\t132722\t0.2\t1\t124931 7791\n",
      "46\t94746\t0.2\t1\t89536 5210\n",
      "47\t122933\t0.2\t1\t116693 6240\n",
      "48\t121112\t0.2\t1\t114755 6357\n",
      "49\t129508\t0.2\t1\t122295 7213\n",
      "50\t125201\t0.2\t1\t118349 6852\n",
      "51\t157565\t0.2\t1\t148874 8691\n",
      "52\t113806\t0.2\t1\t107741 6065\n",
      "53\t143800\t0.2\t1\t136726 7074\n",
      "54\t105294\t0.2\t1\t100142 5152\n",
      "55\t140198\t0.2\t1\t133598 6600\n",
      "56\t145988\t0.2\t1\t138211 7777\n",
      "57\t140653\t0.2\t1\t134358 6295\n",
      "58\t138055\t0.2\t1\t131586 6469\n",
      "59\t139154\t0.2\t1\t132424 6730\n",
      "60\t148619\t0.2\t1\t141526 7093\n",
      "61\t153296\t0.2\t1\t146146 7150\n",
      "62\t170542\t0.2\t1\t162169 8373\n",
      "63\t228765\t0.2\t1\t219044 9721\n",
      "64\t207127\t0.2\t1\t199944 7183\n",
      "65\t126856\t0.2\t1\t121054 5802\n",
      "66\t139494\t0.2\t1\t133042 6452\n",
      "67\t139853\t0.2\t1\t133076 6777\n",
      "68\t143593\t0.2\t1\t136916 6677\n",
      "69\t142441\t0.2\t1\t135436 7005\n",
      "70\t151914\t0.2\t1\t144843 7071\n",
      "71\t164443\t0.2\t1\t156971 7472\n",
      "72\t157523\t0.2\t1\t150188 7335\n",
      "73\t168905\t0.2\t1\t161158 7747\n",
      "74\t154929\t0.2\t1\t147615 7314\n",
      "75\t149956\t0.2\t1\t142906 7050\n",
      "76\t146942\t0.2\t1\t140083 6859\n",
      "77\t148479\t0.2\t1\t141471 7008\n",
      "78\t143688\t0.2\t1\t136908 6780\n",
      "79\t140442\t0.2\t1\t133903 6539\n",
      "80\t140871\t0.2\t1\t134402 6469\n",
      "81\t139994\t0.2\t1\t133731 6263\n",
      "82\t136315\t0.2\t1\t130107 6208\n",
      "83\t133480\t0.2\t1\t127541 5939\n",
      "84\t130953\t0.2\t1\t125135 5818\n",
      "85\t130441\t0.2\t1\t124710 5731\n",
      "86\t132446\t0.2\t1\t126752 5694\n",
      "87\t133712\t0.2\t1\t128137 5575\n",
      "88\t132133\t0.2\t1\t126501 5632\n",
      "89\t120737\t0.2\t1\t115669 5068\n",
      "90\t117385\t0.2\t1\t112365 5020\n",
      "91\t111344\t0.2\t1\t106559 4785\n",
      "92\t105918\t0.2\t1\t101512 4406\n",
      "93\t103040\t0.2\t1\t98784 4256\n",
      "94\t100992\t0.2\t1\t96771 4221\n",
      "95\t96578\t0.2\t1\t92479 4099\n",
      "96\t93826\t0.2\t1\t89971 3855\n",
      "97\t86326\t0.2\t1\t82928 3398\n",
      "98\t81391\t0.2\t1\t78062 3329\n",
      "99\t79676\t0.2\t1\t76340 3336\n",
      "100\t71969\t0.2\t1\t68956 3013\n",
      "101\t69033\t0.2\t1\t66191 2842\n",
      "102\t66979\t0.2\t1\t64266 2713\n",
      "103\t62488\t0.2\t1\t59972 2516\n",
      "104\t59740\t0.2\t1\t57324 2416\n",
      "105\t53204\t0.2\t1\t51066 2138\n",
      "106\t49286\t0.2\t1\t47321 1965\n",
      "107\t44161\t0.2\t1\t42327 1834\n",
      "108\t41055\t0.2\t1\t39500 1555\n",
      "109\t37107\t0.2\t1\t35641 1466\n",
      "110\t34404\t0.2\t1\t32996 1408\n",
      "111\t32112\t0.2\t1\t30868 1244\n",
      "112\t28362\t0.2\t1\t27169 1193\n",
      "113\t24976\t0.2\t1\t23918 1058\n",
      "114\t22140\t0.2\t1\t21281 859\n",
      "115\t18628\t0.2\t1\t17866 762\n",
      "116\t16277\t0.2\t1\t15608 669\n",
      "117\t13249\t0.2\t1\t12647 602\n",
      "118\t12071\t0.2\t1\t11527 544\n",
      "119\t10131\t0.2\t1\t9673 458\n",
      "120\t8604\t0.2\t1\t8215 389\n",
      "121\t7264\t0.2\t1\t6947 317\n",
      "122\t6137\t0.2\t1\t5844 293\n",
      "123\t5194\t0.2\t1\t4979 215\n",
      "124\t4472\t0.2\t1\t4261 211\n",
      "125\t3689\t0.2\t1\t3519 170\n",
      "126\t3057\t0.2\t1\t2936 121\n",
      "127\t2513\t0.2\t1\t2380 133\n",
      "128\t1994\t0.2\t1\t1883 111\n",
      "129\t1536\t0.2\t1\t1459 77\n",
      "130\t1296\t0.2\t1\t1243 53\n",
      "131\t1130\t0.2\t1\t1079 51\n",
      "132\t980\t0.2\t1\t938 42\n",
      "133\t1174\t0.2\t1\t1133 41\n",
      "134\t1195\t0.2\t1\t1157 38\n",
      "135\t568\t0.2\t1\t538 30\n",
      "136\t352\t0.2\t1\t335 17\n",
      "137\t213\t0.2\t1\t202 11\n",
      "138\t143\t0.2\t1\t132 11\n",
      "139\t111\t0.2\t1\t103 8\n",
      "140\t71\t0.2\t1\t64 7\n",
      "141\t52\t0.2\t1\t46 6\n",
      "142\t65\t0.2\t1\t60 5\n",
      "143\t34\t0.2\t1\t32 2\n",
      "144\t27\t0.2\t1\t25 2\n",
      "145\t36\t0.2\t1\t30 6\n",
      "146\t27\t0.2\t1\t25 2\n",
      "147\t78\t0.2\t1\t72 6\n",
      "148\t68\t0.2\t1\t62 6\n",
      "149\t57\t0.2\t1\t55 2\n",
      "150\t443\t0.2\t1\t415 28\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13980098 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz, file_2: 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13980098\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 35976 (0.26%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_1_S5_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_1_S5_L002_R1_001_trimmed.fq.gz and 8C_32psu_1_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_2_S6_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 477.90 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,939,154\n",
      "Reads with adapters:                10,340,662 (79.9%)\n",
      "Reads written (passing filters):    12,939,154 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,940,873,100 bp\n",
      "Quality-trimmed:               3,580,813 bp (0.2%)\n",
      "Total written (filtered):  1,492,993,480 bp (76.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10340662 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.7%\n",
      "  G: 20.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1552261\t3234788.5\t0\t1552261\n",
      "2\t232662\t808697.1\t0\t232662\n",
      "3\t131614\t202174.3\t0\t131614\n",
      "4\t79987\t50543.6\t0\t79987\n",
      "5\t67280\t12635.9\t0\t67280\n",
      "6\t66472\t3159.0\t0\t66472\n",
      "7\t55108\t789.7\t0\t55108\n",
      "8\t68928\t197.4\t0\t68928\n",
      "9\t68982\t49.4\t0\t68109 873\n",
      "10\t64214\t12.3\t1\t60894 3320\n",
      "11\t72301\t3.1\t1\t66989 5312\n",
      "12\t69430\t0.8\t1\t65113 4317\n",
      "13\t71580\t0.2\t1\t67020 4560\n",
      "14\t70477\t0.2\t1\t65870 4607\n",
      "15\t75676\t0.2\t1\t70785 4891\n",
      "16\t72936\t0.2\t1\t67871 5065\n",
      "17\t87298\t0.2\t1\t80460 6838\n",
      "18\t79611\t0.2\t1\t74935 4676\n",
      "19\t65534\t0.2\t1\t61501 4033\n",
      "20\t79604\t0.2\t1\t74644 4960\n",
      "21\t85256\t0.2\t1\t79605 5651\n",
      "22\t73838\t0.2\t1\t69798 4040\n",
      "23\t84971\t0.2\t1\t79628 5343\n",
      "24\t80676\t0.2\t1\t74456 6220\n",
      "25\t85904\t0.2\t1\t80404 5500\n",
      "26\t79805\t0.2\t1\t75204 4601\n",
      "27\t88623\t0.2\t1\t82274 6349\n",
      "28\t86173\t0.2\t1\t80506 5667\n",
      "29\t91469\t0.2\t1\t84355 7114\n",
      "30\t95793\t0.2\t1\t90568 5225\n",
      "31\t77688\t0.2\t1\t72851 4837\n",
      "32\t92600\t0.2\t1\t86938 5662\n",
      "33\t87750\t0.2\t1\t82121 5629\n",
      "34\t99316\t0.2\t1\t92291 7025\n",
      "35\t87021\t0.2\t1\t82303 4718\n",
      "36\t99065\t0.2\t1\t92449 6616\n",
      "37\t101447\t0.2\t1\t95128 6319\n",
      "38\t97080\t0.2\t1\t90884 6196\n",
      "39\t103846\t0.2\t1\t96530 7316\n",
      "40\t94829\t0.2\t1\t89475 5354\n",
      "41\t105310\t0.2\t1\t97483 7827\n",
      "42\t95966\t0.2\t1\t89196 6770\n",
      "43\t151686\t0.2\t1\t144208 7478\n",
      "44\t46892\t0.2\t1\t42892 4000\n",
      "45\t105867\t0.2\t1\t99520 6347\n",
      "46\t96566\t0.2\t1\t90512 6054\n",
      "47\t103096\t0.2\t1\t97153 5943\n",
      "48\t102489\t0.2\t1\t96145 6344\n",
      "49\t112243\t0.2\t1\t104909 7334\n",
      "50\t94609\t0.2\t1\t89107 5502\n",
      "51\t103165\t0.2\t1\t96761 6404\n",
      "52\t107310\t0.2\t1\t100073 7237\n",
      "53\t113172\t0.2\t1\t106618 6554\n",
      "54\t92162\t0.2\t1\t86748 5414\n",
      "55\t108541\t0.2\t1\t101655 6886\n",
      "56\t133677\t0.2\t1\t126253 7424\n",
      "57\t81980\t0.2\t1\t76375 5605\n",
      "58\t110163\t0.2\t1\t104385 5778\n",
      "59\t98316\t0.2\t1\t92340 5976\n",
      "60\t104988\t0.2\t1\t98449 6539\n",
      "61\t103910\t0.2\t1\t97861 6049\n",
      "62\t115389\t0.2\t1\t108890 6499\n",
      "63\t89259\t0.2\t1\t83482 5777\n",
      "64\t102584\t0.2\t1\t96877 5707\n",
      "65\t114383\t0.2\t1\t107700 6683\n",
      "66\t118785\t0.2\t1\t111674 7111\n",
      "67\t124927\t0.2\t1\t115790 9137\n",
      "68\t236831\t0.2\t1\t226263 10568\n",
      "69\t32490\t0.2\t1\t29503 2987\n",
      "70\t14743\t0.2\t1\t12126 2617\n",
      "71\t68006\t0.2\t1\t64211 3795\n",
      "72\t89001\t0.2\t1\t84078 4923\n",
      "73\t92459\t0.2\t1\t87195 5264\n",
      "74\t91411\t0.2\t1\t86036 5375\n",
      "75\t90970\t0.2\t1\t85629 5341\n",
      "76\t89709\t0.2\t1\t84538 5171\n",
      "77\t89140\t0.2\t1\t84081 5059\n",
      "78\t83843\t0.2\t1\t78811 5032\n",
      "79\t83141\t0.2\t1\t78204 4937\n",
      "80\t80837\t0.2\t1\t76149 4688\n",
      "81\t79618\t0.2\t1\t74844 4774\n",
      "82\t77902\t0.2\t1\t73376 4526\n",
      "83\t75605\t0.2\t1\t71189 4416\n",
      "84\t73149\t0.2\t1\t68920 4229\n",
      "85\t70442\t0.2\t1\t66393 4049\n",
      "86\t67311\t0.2\t1\t63325 3986\n",
      "87\t65584\t0.2\t1\t61688 3896\n",
      "88\t63773\t0.2\t1\t60070 3703\n",
      "89\t60201\t0.2\t1\t56655 3546\n",
      "90\t59708\t0.2\t1\t56173 3535\n",
      "91\t56748\t0.2\t1\t53363 3385\n",
      "92\t53671\t0.2\t1\t50489 3182\n",
      "93\t50670\t0.2\t1\t47624 3046\n",
      "94\t48704\t0.2\t1\t45782 2922\n",
      "95\t45200\t0.2\t1\t42460 2740\n",
      "96\t42497\t0.2\t1\t39903 2594\n",
      "97\t39947\t0.2\t1\t37451 2496\n",
      "98\t36683\t0.2\t1\t34398 2285\n",
      "99\t36692\t0.2\t1\t34265 2427\n",
      "100\t32868\t0.2\t1\t30853 2015\n",
      "101\t30512\t0.2\t1\t28489 2023\n",
      "102\t27906\t0.2\t1\t26115 1791\n",
      "103\t26175\t0.2\t1\t24518 1657\n",
      "104\t23983\t0.2\t1\t22398 1585\n",
      "105\t21704\t0.2\t1\t20274 1430\n",
      "106\t19667\t0.2\t1\t18358 1309\n",
      "107\t17851\t0.2\t1\t16610 1241\n",
      "108\t16494\t0.2\t1\t15292 1202\n",
      "109\t14962\t0.2\t1\t13856 1106\n",
      "110\t13424\t0.2\t1\t12418 1006\n",
      "111\t12110\t0.2\t1\t11218 892\n",
      "112\t10668\t0.2\t1\t9941 727\n",
      "113\t8879\t0.2\t1\t8200 679\n",
      "114\t7950\t0.2\t1\t7388 562\n",
      "115\t7058\t0.2\t1\t6505 553\n",
      "116\t6198\t0.2\t1\t5702 496\n",
      "117\t5071\t0.2\t1\t4638 433\n",
      "118\t4567\t0.2\t1\t4226 341\n",
      "119\t3829\t0.2\t1\t3494 335\n",
      "120\t3187\t0.2\t1\t2901 286\n",
      "121\t2891\t0.2\t1\t2612 279\n",
      "122\t2303\t0.2\t1\t2110 193\n",
      "123\t1890\t0.2\t1\t1705 185\n",
      "124\t1701\t0.2\t1\t1504 197\n",
      "125\t1363\t0.2\t1\t1208 155\n",
      "126\t1199\t0.2\t1\t1086 113\n",
      "127\t954\t0.2\t1\t859 95\n",
      "128\t812\t0.2\t1\t702 110\n",
      "129\t610\t0.2\t1\t539 71\n",
      "130\t529\t0.2\t1\t472 57\n",
      "131\t405\t0.2\t1\t354 51\n",
      "132\t324\t0.2\t1\t279 45\n",
      "133\t284\t0.2\t1\t248 36\n",
      "134\t306\t0.2\t1\t272 34\n",
      "135\t179\t0.2\t1\t161 18\n",
      "136\t123\t0.2\t1\t97 26\n",
      "137\t92\t0.2\t1\t81 11\n",
      "138\t74\t0.2\t1\t60 14\n",
      "139\t42\t0.2\t1\t38 4\n",
      "140\t25\t0.2\t1\t20 5\n",
      "141\t21\t0.2\t1\t16 5\n",
      "142\t15\t0.2\t1\t14 1\n",
      "143\t11\t0.2\t1\t8 3\n",
      "144\t7\t0.2\t1\t5 2\n",
      "145\t18\t0.2\t1\t14 4\n",
      "146\t10\t0.2\t1\t8 2\n",
      "147\t32\t0.2\t1\t32\n",
      "148\t16\t0.2\t1\t16\n",
      "149\t9\t0.2\t1\t5 4\n",
      "150\t128\t0.2\t1\t105 23\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "12939154 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_2_S6_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 505.09 s (39 us/read; 1.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,939,154\n",
      "Reads with adapters:                10,288,624 (79.5%)\n",
      "Reads written (passing filters):    12,939,154 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,940,873,100 bp\n",
      "Quality-trimmed:               9,202,420 bp (0.5%)\n",
      "Total written (filtered):  1,495,378,762 bp (77.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10288624 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.2%\n",
      "  C: 13.0%\n",
      "  G: 23.8%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1595771\t3234788.5\t0\t1595771\n",
      "2\t240375\t808697.1\t0\t240375\n",
      "3\t138443\t202174.3\t0\t138443\n",
      "4\t81428\t50543.6\t0\t81428\n",
      "5\t75038\t12635.9\t0\t75038\n",
      "6\t64775\t3159.0\t0\t64775\n",
      "7\t52085\t789.7\t0\t52085\n",
      "8\t75127\t197.4\t0\t75127\n",
      "9\t60527\t49.4\t0\t60355 172\n",
      "10\t65326\t12.3\t1\t62475 2851\n",
      "11\t69712\t3.1\t1\t65682 4030\n",
      "12\t70487\t0.8\t1\t66516 3971\n",
      "13\t78320\t0.2\t1\t74147 4173\n",
      "14\t73989\t0.2\t1\t69268 4721\n",
      "15\t60342\t0.2\t1\t57080 3262\n",
      "16\t88180\t0.2\t1\t83244 4936\n",
      "17\t69750\t0.2\t1\t65392 4358\n",
      "18\t67540\t0.2\t1\t63841 3699\n",
      "19\t85313\t0.2\t1\t80354 4959\n",
      "20\t79769\t0.2\t1\t75478 4291\n",
      "21\t70275\t0.2\t1\t66324 3951\n",
      "22\t79177\t0.2\t1\t74939 4238\n",
      "23\t82659\t0.2\t1\t78081 4578\n",
      "24\t81915\t0.2\t1\t77232 4683\n",
      "25\t93464\t0.2\t1\t88557 4907\n",
      "26\t77147\t0.2\t1\t72835 4312\n",
      "27\t85811\t0.2\t1\t80374 5437\n",
      "28\t80396\t0.2\t1\t76862 3534\n",
      "29\t85946\t0.2\t1\t81104 4842\n",
      "30\t85726\t0.2\t1\t81647 4079\n",
      "31\t92050\t0.2\t1\t87288 4762\n",
      "32\t83804\t0.2\t1\t79934 3870\n",
      "33\t92087\t0.2\t1\t87200 4887\n",
      "34\t119466\t0.2\t1\t113056 6410\n",
      "35\t67982\t0.2\t1\t64809 3173\n",
      "36\t94466\t0.2\t1\t89392 5074\n",
      "37\t100368\t0.2\t1\t95090 5278\n",
      "38\t94027\t0.2\t1\t89878 4149\n",
      "39\t91028\t0.2\t1\t87016 4012\n",
      "40\t101155\t0.2\t1\t95949 5206\n",
      "41\t98098\t0.2\t1\t93343 4755\n",
      "42\t100747\t0.2\t1\t96031 4716\n",
      "43\t92371\t0.2\t1\t88129 4242\n",
      "44\t106061\t0.2\t1\t101051 5010\n",
      "45\t109648\t0.2\t1\t104507 5141\n",
      "46\t85668\t0.2\t1\t81729 3939\n",
      "47\t104356\t0.2\t1\t99808 4548\n",
      "48\t101281\t0.2\t1\t96981 4300\n",
      "49\t106782\t0.2\t1\t101997 4785\n",
      "50\t100514\t0.2\t1\t95958 4556\n",
      "51\t125782\t0.2\t1\t120079 5703\n",
      "52\t90772\t0.2\t1\t86983 3789\n",
      "53\t109598\t0.2\t1\t105306 4292\n",
      "54\t83322\t0.2\t1\t79849 3473\n",
      "55\t107016\t0.2\t1\t102862 4154\n",
      "56\t110967\t0.2\t1\t106233 4734\n",
      "57\t104357\t0.2\t1\t100229 4128\n",
      "58\t100691\t0.2\t1\t96894 3797\n",
      "59\t102578\t0.2\t1\t98489 4089\n",
      "60\t106303\t0.2\t1\t102044 4259\n",
      "61\t108531\t0.2\t1\t104028 4503\n",
      "62\t118161\t0.2\t1\t113451 4710\n",
      "63\t158201\t0.2\t1\t152213 5988\n",
      "64\t129612\t0.2\t1\t125287 4325\n",
      "65\t83984\t0.2\t1\t80809 3175\n",
      "66\t91901\t0.2\t1\t88385 3516\n",
      "67\t89742\t0.2\t1\t86128 3614\n",
      "68\t91530\t0.2\t1\t88057 3473\n",
      "69\t90402\t0.2\t1\t86917 3485\n",
      "70\t96053\t0.2\t1\t92305 3748\n",
      "71\t99803\t0.2\t1\t96104 3699\n",
      "72\t94159\t0.2\t1\t90619 3540\n",
      "73\t98787\t0.2\t1\t95125 3662\n",
      "74\t90680\t0.2\t1\t87379 3301\n",
      "75\t86705\t0.2\t1\t83434 3271\n",
      "76\t84928\t0.2\t1\t81792 3136\n",
      "77\t84096\t0.2\t1\t80874 3222\n",
      "78\t80178\t0.2\t1\t77109 3069\n",
      "79\t77254\t0.2\t1\t74397 2857\n",
      "80\t77028\t0.2\t1\t74180 2848\n",
      "81\t75249\t0.2\t1\t72550 2699\n",
      "82\t72339\t0.2\t1\t69794 2545\n",
      "83\t70398\t0.2\t1\t67790 2608\n",
      "84\t68189\t0.2\t1\t65635 2554\n",
      "85\t66784\t0.2\t1\t64244 2540\n",
      "86\t65482\t0.2\t1\t63096 2386\n",
      "87\t67016\t0.2\t1\t64553 2463\n",
      "88\t64151\t0.2\t1\t61741 2410\n",
      "89\t57363\t0.2\t1\t55205 2158\n",
      "90\t54985\t0.2\t1\t52902 2083\n",
      "91\t52080\t0.2\t1\t50216 1864\n",
      "92\t48422\t0.2\t1\t46687 1735\n",
      "93\t47310\t0.2\t1\t45600 1710\n",
      "94\t45459\t0.2\t1\t43795 1664\n",
      "95\t43150\t0.2\t1\t41608 1542\n",
      "96\t40380\t0.2\t1\t38964 1416\n",
      "97\t36752\t0.2\t1\t35440 1312\n",
      "98\t33446\t0.2\t1\t32251 1195\n",
      "99\t33482\t0.2\t1\t32255 1227\n",
      "100\t30657\t0.2\t1\t29566 1091\n",
      "101\t28785\t0.2\t1\t27832 953\n",
      "102\t27470\t0.2\t1\t26566 904\n",
      "103\t25360\t0.2\t1\t24494 866\n",
      "104\t23640\t0.2\t1\t22887 753\n",
      "105\t21192\t0.2\t1\t20482 710\n",
      "106\t19098\t0.2\t1\t18446 652\n",
      "107\t16996\t0.2\t1\t16368 628\n",
      "108\t15172\t0.2\t1\t14616 556\n",
      "109\t14001\t0.2\t1\t13446 555\n",
      "110\t12575\t0.2\t1\t12154 421\n",
      "111\t11902\t0.2\t1\t11531 371\n",
      "112\t10341\t0.2\t1\t9986 355\n",
      "113\t8669\t0.2\t1\t8372 297\n",
      "114\t7690\t0.2\t1\t7405 285\n",
      "115\t6634\t0.2\t1\t6377 257\n",
      "116\t5717\t0.2\t1\t5480 237\n",
      "117\t4541\t0.2\t1\t4344 197\n",
      "118\t4158\t0.2\t1\t3977 181\n",
      "119\t3510\t0.2\t1\t3351 159\n",
      "120\t2896\t0.2\t1\t2771 125\n",
      "121\t2680\t0.2\t1\t2560 120\n",
      "122\t2179\t0.2\t1\t2075 104\n",
      "123\t1805\t0.2\t1\t1731 74\n",
      "124\t1586\t0.2\t1\t1495 91\n",
      "125\t1276\t0.2\t1\t1206 70\n",
      "126\t1153\t0.2\t1\t1112 41\n",
      "127\t891\t0.2\t1\t852 39\n",
      "128\t721\t0.2\t1\t684 37\n",
      "129\t559\t0.2\t1\t529 30\n",
      "130\t484\t0.2\t1\t453 31\n",
      "131\t369\t0.2\t1\t355 14\n",
      "132\t299\t0.2\t1\t280 19\n",
      "133\t268\t0.2\t1\t245 23\n",
      "134\t297\t0.2\t1\t280 17\n",
      "135\t162\t0.2\t1\t155 7\n",
      "136\t110\t0.2\t1\t102 8\n",
      "137\t81\t0.2\t1\t76 5\n",
      "138\t67\t0.2\t1\t59 8\n",
      "139\t34\t0.2\t1\t30 4\n",
      "140\t26\t0.2\t1\t23 3\n",
      "141\t15\t0.2\t1\t15\n",
      "142\t15\t0.2\t1\t15\n",
      "143\t8\t0.2\t1\t8\n",
      "144\t6\t0.2\t1\t4 2\n",
      "145\t17\t0.2\t1\t15 2\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t33\t0.2\t1\t27 6\n",
      "148\t14\t0.2\t1\t14\n",
      "149\t8\t0.2\t1\t6 2\n",
      "150\t114\t0.2\t1\t107 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "12939154 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz, file_2: 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 12939154\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 23056 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_2_S6_L001_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_2_S6_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 483.35 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,868,767\n",
      "Reads with adapters:                10,286,843 (79.9%)\n",
      "Reads written (passing filters):    12,868,767 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,930,315,050 bp\n",
      "Quality-trimmed:               3,674,826 bp (0.2%)\n",
      "Total written (filtered):  1,483,181,496 bp (76.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10286843 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.7%\n",
      "  G: 20.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1541222\t3217191.8\t0\t1541222\n",
      "2\t228030\t804297.9\t0\t228030\n",
      "3\t129528\t201074.5\t0\t129528\n",
      "4\t78688\t50268.6\t0\t78688\n",
      "5\t67124\t12567.2\t0\t67124\n",
      "6\t66330\t3141.8\t0\t66330\n",
      "7\t52508\t785.4\t0\t52508\n",
      "8\t67652\t196.4\t0\t67652\n",
      "9\t67615\t49.1\t0\t66819 796\n",
      "10\t63063\t12.3\t1\t59213 3850\n",
      "11\t71894\t3.1\t1\t65954 5940\n",
      "12\t67897\t0.8\t1\t62829 5068\n",
      "13\t71151\t0.2\t1\t65849 5302\n",
      "14\t69055\t0.2\t1\t63709 5346\n",
      "15\t75069\t0.2\t1\t69302 5767\n",
      "16\t71424\t0.2\t1\t65420 6004\n",
      "17\t86810\t0.2\t1\t79021 7789\n",
      "18\t77955\t0.2\t1\t72600 5355\n",
      "19\t63383\t0.2\t1\t58809 4574\n",
      "20\t78965\t0.2\t1\t73063 5902\n",
      "21\t85485\t0.2\t1\t79014 6471\n",
      "22\t72855\t0.2\t1\t68021 4834\n",
      "23\t85346\t0.2\t1\t78944 6402\n",
      "24\t80318\t0.2\t1\t73301 7017\n",
      "25\t85430\t0.2\t1\t78969 6461\n",
      "26\t78740\t0.2\t1\t73485 5255\n",
      "27\t88451\t0.2\t1\t81474 6977\n",
      "28\t85790\t0.2\t1\t79338 6452\n",
      "29\t91791\t0.2\t1\t83845 7946\n",
      "30\t95449\t0.2\t1\t89482 5967\n",
      "31\t75838\t0.2\t1\t70423 5415\n",
      "32\t92731\t0.2\t1\t85851 6880\n",
      "33\t86762\t0.2\t1\t80055 6707\n",
      "34\t99437\t0.2\t1\t91057 8380\n",
      "35\t85081\t0.2\t1\t79384 5697\n",
      "36\t99297\t0.2\t1\t91300 7997\n",
      "37\t101918\t0.2\t1\t94163 7755\n",
      "38\t96379\t0.2\t1\t88997 7382\n",
      "39\t103865\t0.2\t1\t95226 8639\n",
      "40\t94376\t0.2\t1\t87615 6761\n",
      "41\t104329\t0.2\t1\t95123 9206\n",
      "42\t94892\t0.2\t1\t86892 8000\n",
      "43\t148535\t0.2\t1\t139624 8911\n",
      "44\t48085\t0.2\t1\t43529 4556\n",
      "45\t105311\t0.2\t1\t97887 7424\n",
      "46\t95734\t0.2\t1\t88889 6845\n",
      "47\t101958\t0.2\t1\t94804 7154\n",
      "48\t102091\t0.2\t1\t94808 7283\n",
      "49\t112404\t0.2\t1\t104035 8369\n",
      "50\t93295\t0.2\t1\t87104 6191\n",
      "51\t104120\t0.2\t1\t96595 7525\n",
      "52\t107257\t0.2\t1\t99149 8108\n",
      "53\t113672\t0.2\t1\t105961 7711\n",
      "54\t91213\t0.2\t1\t85172 6041\n",
      "55\t107994\t0.2\t1\t100085 7909\n",
      "56\t133513\t0.2\t1\t124815 8698\n",
      "57\t81884\t0.2\t1\t75254 6630\n",
      "58\t110562\t0.2\t1\t103755 6807\n",
      "59\t98179\t0.2\t1\t91032 7147\n",
      "60\t104453\t0.2\t1\t97116 7337\n",
      "61\t104191\t0.2\t1\t97239 6952\n",
      "62\t114950\t0.2\t1\t107431 7519\n",
      "63\t89541\t0.2\t1\t82900 6641\n",
      "64\t101345\t0.2\t1\t94868 6477\n",
      "65\t113295\t0.2\t1\t105436 7859\n",
      "66\t116379\t0.2\t1\t108386 7993\n",
      "67\t124994\t0.2\t1\t114943 10051\n",
      "68\t229157\t0.2\t1\t217120 12037\n",
      "69\t41354\t0.2\t1\t37561 3793\n",
      "70\t17861\t0.2\t1\t14972 2889\n",
      "71\t66015\t0.2\t1\t61819 4196\n",
      "72\t88699\t0.2\t1\t83225 5474\n",
      "73\t92274\t0.2\t1\t86536 5738\n",
      "74\t90496\t0.2\t1\t84396 6100\n",
      "75\t91072\t0.2\t1\t84946 6126\n",
      "76\t89676\t0.2\t1\t83357 6319\n",
      "77\t89231\t0.2\t1\t83068 6163\n",
      "78\t83699\t0.2\t1\t77980 5719\n",
      "79\t83174\t0.2\t1\t77470 5704\n",
      "80\t80873\t0.2\t1\t75140 5733\n",
      "81\t79905\t0.2\t1\t74411 5494\n",
      "82\t77850\t0.2\t1\t72450 5400\n",
      "83\t75463\t0.2\t1\t70383 5080\n",
      "84\t72909\t0.2\t1\t67760 5149\n",
      "85\t70237\t0.2\t1\t65392 4845\n",
      "86\t67846\t0.2\t1\t63270 4576\n",
      "87\t65212\t0.2\t1\t60837 4375\n",
      "88\t63724\t0.2\t1\t59434 4290\n",
      "89\t60287\t0.2\t1\t56154 4133\n",
      "90\t60221\t0.2\t1\t55941 4280\n",
      "91\t56630\t0.2\t1\t52736 3894\n",
      "92\t53891\t0.2\t1\t50132 3759\n",
      "93\t50029\t0.2\t1\t46571 3458\n",
      "94\t48812\t0.2\t1\t45399 3413\n",
      "95\t45157\t0.2\t1\t41906 3251\n",
      "96\t42860\t0.2\t1\t39861 2999\n",
      "97\t40400\t0.2\t1\t37713 2687\n",
      "98\t37633\t0.2\t1\t34970 2663\n",
      "99\t36386\t0.2\t1\t33717 2669\n",
      "100\t32744\t0.2\t1\t30375 2369\n",
      "101\t30684\t0.2\t1\t28451 2233\n",
      "102\t28336\t0.2\t1\t26275 2061\n",
      "103\t26125\t0.2\t1\t24257 1868\n",
      "104\t24496\t0.2\t1\t22774 1722\n",
      "105\t21603\t0.2\t1\t20114 1489\n",
      "106\t19703\t0.2\t1\t18252 1451\n",
      "107\t18114\t0.2\t1\t16740 1374\n",
      "108\t16491\t0.2\t1\t15275 1216\n",
      "109\t15050\t0.2\t1\t13866 1184\n",
      "110\t13423\t0.2\t1\t12430 993\n",
      "111\t12182\t0.2\t1\t11225 957\n",
      "112\t10547\t0.2\t1\t9689 858\n",
      "113\t9161\t0.2\t1\t8400 761\n",
      "114\t7859\t0.2\t1\t7209 650\n",
      "115\t6948\t0.2\t1\t6344 604\n",
      "116\t6269\t0.2\t1\t5743 526\n",
      "117\t5202\t0.2\t1\t4768 434\n",
      "118\t4673\t0.2\t1\t4242 431\n",
      "119\t3797\t0.2\t1\t3484 313\n",
      "120\t3288\t0.2\t1\t2990 298\n",
      "121\t2786\t0.2\t1\t2513 273\n",
      "122\t2451\t0.2\t1\t2220 231\n",
      "123\t1988\t0.2\t1\t1766 222\n",
      "124\t1684\t0.2\t1\t1506 178\n",
      "125\t1280\t0.2\t1\t1137 143\n",
      "126\t1164\t0.2\t1\t1040 124\n",
      "127\t900\t0.2\t1\t789 111\n",
      "128\t728\t0.2\t1\t654 74\n",
      "129\t628\t0.2\t1\t547 81\n",
      "130\t442\t0.2\t1\t393 49\n",
      "131\t450\t0.2\t1\t380 70\n",
      "132\t321\t0.2\t1\t277 44\n",
      "133\t283\t0.2\t1\t243 40\n",
      "134\t293\t0.2\t1\t262 31\n",
      "135\t181\t0.2\t1\t154 27\n",
      "136\t135\t0.2\t1\t102 33\n",
      "137\t78\t0.2\t1\t61 17\n",
      "138\t63\t0.2\t1\t56 7\n",
      "139\t49\t0.2\t1\t46 3\n",
      "140\t36\t0.2\t1\t28 8\n",
      "141\t27\t0.2\t1\t20 7\n",
      "142\t11\t0.2\t1\t8 3\n",
      "143\t11\t0.2\t1\t10 1\n",
      "144\t9\t0.2\t1\t4 5\n",
      "145\t13\t0.2\t1\t10 3\n",
      "146\t15\t0.2\t1\t11 4\n",
      "147\t27\t0.2\t1\t25 2\n",
      "148\t26\t0.2\t1\t23 3\n",
      "149\t15\t0.2\t1\t15\n",
      "150\t143\t0.2\t1\t117 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "12868767 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_2_S6_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 513.25 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              12,868,767\n",
      "Reads with adapters:                10,225,148 (79.5%)\n",
      "Reads written (passing filters):    12,868,767 (100.0%)\n",
      "\n",
      "Total basepairs processed: 1,930,315,050 bp\n",
      "Quality-trimmed:               8,911,212 bp (0.5%)\n",
      "Total written (filtered):  1,486,049,208 bp (77.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10225148 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 13.1%\n",
      "  G: 23.9%\n",
      "  T: 34.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1581903\t3217191.8\t0\t1581903\n",
      "2\t233784\t804297.9\t0\t233784\n",
      "3\t139412\t201074.5\t0\t139412\n",
      "4\t79292\t50268.6\t0\t79292\n",
      "5\t72507\t12567.2\t0\t72507\n",
      "6\t64695\t3141.8\t0\t64695\n",
      "7\t50784\t785.4\t0\t50784\n",
      "8\t73943\t196.4\t0\t73943\n",
      "9\t59253\t49.1\t0\t59073 180\n",
      "10\t62906\t12.3\t1\t58824 4082\n",
      "11\t69220\t3.1\t1\t63844 5376\n",
      "12\t67633\t0.8\t1\t62443 5190\n",
      "13\t78337\t0.2\t1\t72455 5882\n",
      "14\t72684\t0.2\t1\t66542 6142\n",
      "15\t59253\t0.2\t1\t54996 4257\n",
      "16\t85878\t0.2\t1\t79407 6471\n",
      "17\t71616\t0.2\t1\t66058 5558\n",
      "18\t64381\t0.2\t1\t59775 4606\n",
      "19\t83620\t0.2\t1\t77611 6009\n",
      "20\t78383\t0.2\t1\t72906 5477\n",
      "21\t69471\t0.2\t1\t64687 4784\n",
      "22\t78479\t0.2\t1\t73016 5463\n",
      "23\t83210\t0.2\t1\t77230 5980\n",
      "24\t82011\t0.2\t1\t76117 5894\n",
      "25\t93094\t0.2\t1\t87025 6069\n",
      "26\t77459\t0.2\t1\t71634 5825\n",
      "27\t86527\t0.2\t1\t79790 6737\n",
      "28\t77466\t0.2\t1\t72862 4604\n",
      "29\t87078\t0.2\t1\t81023 6055\n",
      "30\t83340\t0.2\t1\t78135 5205\n",
      "31\t92587\t0.2\t1\t86504 6083\n",
      "32\t80491\t0.2\t1\t75656 4835\n",
      "33\t92967\t0.2\t1\t86621 6346\n",
      "34\t120697\t0.2\t1\t112803 7894\n",
      "35\t64048\t0.2\t1\t59979 4069\n",
      "36\t95138\t0.2\t1\t88610 6528\n",
      "37\t100610\t0.2\t1\t93904 6706\n",
      "38\t93157\t0.2\t1\t87717 5440\n",
      "39\t88669\t0.2\t1\t83789 4880\n",
      "40\t101334\t0.2\t1\t95182 6152\n",
      "41\t98089\t0.2\t1\t92269 5820\n",
      "42\t100106\t0.2\t1\t94212 5894\n",
      "43\t90866\t0.2\t1\t85570 5296\n",
      "44\t106583\t0.2\t1\t100029 6554\n",
      "45\t108549\t0.2\t1\t101877 6672\n",
      "46\t84237\t0.2\t1\t79394 4843\n",
      "47\t103605\t0.2\t1\t98067 5538\n",
      "48\t100222\t0.2\t1\t94797 5425\n",
      "49\t105528\t0.2\t1\t99377 6151\n",
      "50\t101252\t0.2\t1\t95306 5946\n",
      "51\t124662\t0.2\t1\t117482 7180\n",
      "52\t89980\t0.2\t1\t85108 4872\n",
      "53\t112122\t0.2\t1\t106274 5848\n",
      "54\t81922\t0.2\t1\t77760 4162\n",
      "55\t105894\t0.2\t1\t100558 5336\n",
      "56\t109567\t0.2\t1\t103471 6096\n",
      "57\t105084\t0.2\t1\t100060 5024\n",
      "58\t101375\t0.2\t1\t96453 4922\n",
      "59\t101412\t0.2\t1\t96219 5193\n",
      "60\t106019\t0.2\t1\t100584 5435\n",
      "61\t107866\t0.2\t1\t102516 5350\n",
      "62\t117046\t0.2\t1\t111012 6034\n",
      "63\t155210\t0.2\t1\t148115 7095\n",
      "64\t127345\t0.2\t1\t122328 5017\n",
      "65\t83662\t0.2\t1\t79683 3979\n",
      "66\t91095\t0.2\t1\t86676 4419\n",
      "67\t90432\t0.2\t1\t85682 4750\n",
      "68\t91566\t0.2\t1\t87021 4545\n",
      "69\t91038\t0.2\t1\t86455 4583\n",
      "70\t97071\t0.2\t1\t92328 4743\n",
      "71\t100160\t0.2\t1\t95222 4938\n",
      "72\t95370\t0.2\t1\t90793 4577\n",
      "73\t99146\t0.2\t1\t94377 4769\n",
      "74\t90186\t0.2\t1\t85699 4487\n",
      "75\t87292\t0.2\t1\t82908 4384\n",
      "76\t84846\t0.2\t1\t80550 4296\n",
      "77\t84110\t0.2\t1\t79782 4328\n",
      "78\t80083\t0.2\t1\t76044 4039\n",
      "79\t77641\t0.2\t1\t73764 3877\n",
      "80\t76649\t0.2\t1\t72933 3716\n",
      "81\t75643\t0.2\t1\t71998 3645\n",
      "82\t72511\t0.2\t1\t69001 3510\n",
      "83\t70456\t0.2\t1\t67059 3397\n",
      "84\t67826\t0.2\t1\t64473 3353\n",
      "85\t66652\t0.2\t1\t63633 3019\n",
      "86\t65922\t0.2\t1\t62797 3125\n",
      "87\t65939\t0.2\t1\t62926 3013\n",
      "88\t64115\t0.2\t1\t61226 2889\n",
      "89\t57559\t0.2\t1\t54978 2581\n",
      "90\t55692\t0.2\t1\t53148 2544\n",
      "91\t51986\t0.2\t1\t49669 2317\n",
      "92\t49131\t0.2\t1\t46914 2217\n",
      "93\t46886\t0.2\t1\t44668 2218\n",
      "94\t45747\t0.2\t1\t43801 1946\n",
      "95\t42950\t0.2\t1\t40887 2063\n",
      "96\t40716\t0.2\t1\t38860 1856\n",
      "97\t37404\t0.2\t1\t35730 1674\n",
      "98\t34443\t0.2\t1\t32843 1600\n",
      "99\t33264\t0.2\t1\t31734 1530\n",
      "100\t30705\t0.2\t1\t29262 1443\n",
      "101\t28943\t0.2\t1\t27671 1272\n",
      "102\t27964\t0.2\t1\t26691 1273\n",
      "103\t25366\t0.2\t1\t24255 1111\n",
      "104\t24064\t0.2\t1\t22990 1074\n",
      "105\t21070\t0.2\t1\t20080 990\n",
      "106\t19078\t0.2\t1\t18244 834\n",
      "107\t17343\t0.2\t1\t16582 761\n",
      "108\t15238\t0.2\t1\t14525 713\n",
      "109\t14062\t0.2\t1\t13403 659\n",
      "110\t12670\t0.2\t1\t12101 569\n",
      "111\t11892\t0.2\t1\t11362 530\n",
      "112\t10243\t0.2\t1\t9780 463\n",
      "113\t8834\t0.2\t1\t8449 385\n",
      "114\t7653\t0.2\t1\t7300 353\n",
      "115\t6468\t0.2\t1\t6167 301\n",
      "116\t5767\t0.2\t1\t5513 254\n",
      "117\t4744\t0.2\t1\t4519 225\n",
      "118\t4267\t0.2\t1\t4031 236\n",
      "119\t3534\t0.2\t1\t3331 203\n",
      "120\t2971\t0.2\t1\t2813 158\n",
      "121\t2616\t0.2\t1\t2484 132\n",
      "122\t2300\t0.2\t1\t2180 120\n",
      "123\t1889\t0.2\t1\t1769 120\n",
      "124\t1591\t0.2\t1\t1513 78\n",
      "125\t1198\t0.2\t1\t1143 55\n",
      "126\t1107\t0.2\t1\t1053 54\n",
      "127\t843\t0.2\t1\t791 52\n",
      "128\t694\t0.2\t1\t650 44\n",
      "129\t581\t0.2\t1\t539 42\n",
      "130\t407\t0.2\t1\t381 26\n",
      "131\t412\t0.2\t1\t395 17\n",
      "132\t294\t0.2\t1\t275 19\n",
      "133\t274\t0.2\t1\t263 11\n",
      "134\t295\t0.2\t1\t282 13\n",
      "135\t162\t0.2\t1\t152 10\n",
      "136\t127\t0.2\t1\t120 7\n",
      "137\t72\t0.2\t1\t67 5\n",
      "138\t53\t0.2\t1\t46 7\n",
      "139\t48\t0.2\t1\t43 5\n",
      "140\t33\t0.2\t1\t30 3\n",
      "141\t23\t0.2\t1\t20 3\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t12\t0.2\t1\t11 1\n",
      "144\t5\t0.2\t1\t3 2\n",
      "145\t11\t0.2\t1\t11\n",
      "146\t13\t0.2\t1\t12 1\n",
      "147\t30\t0.2\t1\t28 2\n",
      "148\t16\t0.2\t1\t14 2\n",
      "149\t14\t0.2\t1\t13 1\n",
      "150\t92\t0.2\t1\t87 5\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "12868767 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz, file_2: 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 12868767\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 21438 (0.17%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_2_S6_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_2_S6_L002_R1_001_trimmed.fq.gz and 8C_32psu_2_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_3_S7_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 500.42 s (35 us/read; 1.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,166,205\n",
      "Reads with adapters:                11,846,180 (83.6%)\n",
      "Reads written (passing filters):    14,166,205 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,124,930,750 bp\n",
      "Quality-trimmed:               4,298,957 bp (0.2%)\n",
      "Total written (filtered):  1,559,729,002 bp (73.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11846180 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 15.1%\n",
      "  G: 21.0%\n",
      "  T: 34.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1373757\t3541551.2\t0\t1373757\n",
      "2\t216270\t885387.8\t0\t216270\n",
      "3\t129588\t221347.0\t0\t129588\n",
      "4\t80077\t55336.7\t0\t80077\n",
      "5\t69163\t13834.2\t0\t69163\n",
      "6\t67518\t3458.5\t0\t67518\n",
      "7\t56035\t864.6\t0\t56035\n",
      "8\t70267\t216.2\t0\t70267\n",
      "9\t70629\t54.0\t0\t69702 927\n",
      "10\t66297\t13.5\t1\t62862 3435\n",
      "11\t73903\t3.4\t1\t68283 5620\n",
      "12\t71104\t0.8\t1\t66580 4524\n",
      "13\t72370\t0.2\t1\t67647 4723\n",
      "14\t73869\t0.2\t1\t68608 5261\n",
      "15\t77649\t0.2\t1\t72603 5046\n",
      "16\t76835\t0.2\t1\t71293 5542\n",
      "17\t93103\t0.2\t1\t85654 7449\n",
      "18\t80962\t0.2\t1\t76131 4831\n",
      "19\t69251\t0.2\t1\t64739 4512\n",
      "20\t83140\t0.2\t1\t77746 5394\n",
      "21\t91983\t0.2\t1\t85635 6348\n",
      "22\t78972\t0.2\t1\t74340 4632\n",
      "23\t91741\t0.2\t1\t85612 6129\n",
      "24\t86730\t0.2\t1\t79637 7093\n",
      "25\t93905\t0.2\t1\t87575 6330\n",
      "26\t84785\t0.2\t1\t79850 4935\n",
      "27\t95659\t0.2\t1\t88641 7018\n",
      "28\t93289\t0.2\t1\t86777 6512\n",
      "29\t102324\t0.2\t1\t94020 8304\n",
      "30\t106573\t0.2\t1\t100735 5838\n",
      "31\t82302\t0.2\t1\t76920 5382\n",
      "32\t102146\t0.2\t1\t95503 6643\n",
      "33\t96707\t0.2\t1\t90380 6327\n",
      "34\t106974\t0.2\t1\t99423 7551\n",
      "35\t107836\t0.2\t1\t100264 7572\n",
      "36\t113175\t0.2\t1\t105186 7989\n",
      "37\t100704\t0.2\t1\t95019 5685\n",
      "38\t121468\t0.2\t1\t112012 9456\n",
      "39\t103157\t0.2\t1\t96822 6335\n",
      "40\t115108\t0.2\t1\t108010 7098\n",
      "41\t129750\t0.2\t1\t120171 9579\n",
      "42\t102179\t0.2\t1\t94823 7356\n",
      "43\t171532\t0.2\t1\t162826 8706\n",
      "44\t55606\t0.2\t1\t50892 4714\n",
      "45\t120800\t0.2\t1\t113289 7511\n",
      "46\t113431\t0.2\t1\t106413 7018\n",
      "47\t120541\t0.2\t1\t113256 7285\n",
      "48\t119747\t0.2\t1\t111882 7865\n",
      "49\t130730\t0.2\t1\t121756 8974\n",
      "50\t111508\t0.2\t1\t104914 6594\n",
      "51\t122095\t0.2\t1\t114233 7862\n",
      "52\t127642\t0.2\t1\t118613 9029\n",
      "53\t137409\t0.2\t1\t129309 8100\n",
      "54\t108827\t0.2\t1\t102141 6686\n",
      "55\t130904\t0.2\t1\t122687 8217\n",
      "56\t157783\t0.2\t1\t148768 9015\n",
      "57\t104771\t0.2\t1\t97389 7382\n",
      "58\t132433\t0.2\t1\t125333 7100\n",
      "59\t115612\t0.2\t1\t108308 7304\n",
      "60\t130650\t0.2\t1\t122277 8373\n",
      "61\t128910\t0.2\t1\t121303 7607\n",
      "62\t139690\t0.2\t1\t131489 8201\n",
      "63\t114068\t0.2\t1\t106664 7404\n",
      "64\t125901\t0.2\t1\t118809 7092\n",
      "65\t141358\t0.2\t1\t132756 8602\n",
      "66\t146823\t0.2\t1\t137956 8867\n",
      "67\t159981\t0.2\t1\t148074 11907\n",
      "68\t301471\t0.2\t1\t287602 13869\n",
      "69\t51550\t0.2\t1\t47181 4369\n",
      "70\t21339\t0.2\t1\t17862 3477\n",
      "71\t81265\t0.2\t1\t76296 4969\n",
      "72\t113463\t0.2\t1\t106973 6490\n",
      "73\t120176\t0.2\t1\t113199 6977\n",
      "74\t119128\t0.2\t1\t112036 7092\n",
      "75\t118829\t0.2\t1\t111689 7140\n",
      "76\t117859\t0.2\t1\t110976 6883\n",
      "77\t119047\t0.2\t1\t112165 6882\n",
      "78\t111406\t0.2\t1\t104659 6747\n",
      "79\t110628\t0.2\t1\t103829 6799\n",
      "80\t108577\t0.2\t1\t102197 6380\n",
      "81\t107611\t0.2\t1\t101254 6357\n",
      "82\t104685\t0.2\t1\t98322 6363\n",
      "83\t101689\t0.2\t1\t95553 6136\n",
      "84\t99042\t0.2\t1\t93220 5822\n",
      "85\t96588\t0.2\t1\t90890 5698\n",
      "86\t93102\t0.2\t1\t87585 5517\n",
      "87\t89497\t0.2\t1\t84148 5349\n",
      "88\t88681\t0.2\t1\t83310 5371\n",
      "89\t83884\t0.2\t1\t78862 5022\n",
      "90\t83820\t0.2\t1\t78644 5176\n",
      "91\t79519\t0.2\t1\t74630 4889\n",
      "92\t75751\t0.2\t1\t71312 4439\n",
      "93\t71060\t0.2\t1\t66734 4326\n",
      "94\t68805\t0.2\t1\t64395 4410\n",
      "95\t63794\t0.2\t1\t59908 3886\n",
      "96\t60732\t0.2\t1\t57025 3707\n",
      "97\t57614\t0.2\t1\t54075 3539\n",
      "98\t53272\t0.2\t1\t49912 3360\n",
      "99\t52704\t0.2\t1\t49079 3625\n",
      "100\t47213\t0.2\t1\t44245 2968\n",
      "101\t43905\t0.2\t1\t41069 2836\n",
      "102\t40757\t0.2\t1\t38090 2667\n",
      "103\t36990\t0.2\t1\t34581 2409\n",
      "104\t34678\t0.2\t1\t32449 2229\n",
      "105\t30517\t0.2\t1\t28453 2064\n",
      "106\t28231\t0.2\t1\t26371 1860\n",
      "107\t25422\t0.2\t1\t23676 1746\n",
      "108\t24129\t0.2\t1\t22480 1649\n",
      "109\t20909\t0.2\t1\t19298 1611\n",
      "110\t19145\t0.2\t1\t17778 1367\n",
      "111\t17228\t0.2\t1\t15946 1282\n",
      "112\t14959\t0.2\t1\t13818 1141\n",
      "113\t12878\t0.2\t1\t11894 984\n",
      "114\t11383\t0.2\t1\t10467 916\n",
      "115\t9899\t0.2\t1\t9096 803\n",
      "116\t8618\t0.2\t1\t7872 746\n",
      "117\t7168\t0.2\t1\t6561 607\n",
      "118\t6256\t0.2\t1\t5711 545\n",
      "119\t5200\t0.2\t1\t4764 436\n",
      "120\t4401\t0.2\t1\t4012 389\n",
      "121\t3754\t0.2\t1\t3388 366\n",
      "122\t3023\t0.2\t1\t2711 312\n",
      "123\t2648\t0.2\t1\t2374 274\n",
      "124\t2210\t0.2\t1\t1991 219\n",
      "125\t1851\t0.2\t1\t1661 190\n",
      "126\t1538\t0.2\t1\t1379 159\n",
      "127\t1281\t0.2\t1\t1146 135\n",
      "128\t1028\t0.2\t1\t906 122\n",
      "129\t844\t0.2\t1\t750 94\n",
      "130\t578\t0.2\t1\t506 72\n",
      "131\t477\t0.2\t1\t408 69\n",
      "132\t403\t0.2\t1\t349 54\n",
      "133\t425\t0.2\t1\t364 61\n",
      "134\t500\t0.2\t1\t457 43\n",
      "135\t253\t0.2\t1\t229 24\n",
      "136\t154\t0.2\t1\t133 21\n",
      "137\t116\t0.2\t1\t99 17\n",
      "138\t91\t0.2\t1\t74 17\n",
      "139\t63\t0.2\t1\t48 15\n",
      "140\t40\t0.2\t1\t34 6\n",
      "141\t25\t0.2\t1\t20 5\n",
      "142\t20\t0.2\t1\t12 8\n",
      "143\t19\t0.2\t1\t15 4\n",
      "144\t21\t0.2\t1\t17 4\n",
      "145\t20\t0.2\t1\t16 4\n",
      "146\t14\t0.2\t1\t12 2\n",
      "147\t31\t0.2\t1\t27 4\n",
      "148\t15\t0.2\t1\t13 2\n",
      "149\t18\t0.2\t1\t14 4\n",
      "150\t240\t0.2\t1\t195 45\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14166205 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_3_S7_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 523.24 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,166,205\n",
      "Reads with adapters:                11,771,953 (83.1%)\n",
      "Reads written (passing filters):    14,166,205 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,124,930,750 bp\n",
      "Quality-trimmed:              10,797,724 bp (0.5%)\n",
      "Total written (filtered):  1,563,454,753 bp (73.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11771953 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.6%\n",
      "  C: 13.3%\n",
      "  G: 24.5%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1427757\t3541551.2\t0\t1427757\n",
      "2\t222493\t885387.8\t0\t222493\n",
      "3\t133332\t221347.0\t0\t133332\n",
      "4\t79137\t55336.7\t0\t79137\n",
      "5\t75786\t13834.2\t0\t75786\n",
      "6\t65915\t3458.5\t0\t65915\n",
      "7\t53576\t864.6\t0\t53576\n",
      "8\t76744\t216.2\t0\t76744\n",
      "9\t61976\t54.0\t0\t61779 197\n",
      "10\t67080\t13.5\t1\t64106 2974\n",
      "11\t71681\t3.4\t1\t67501 4180\n",
      "12\t72222\t0.8\t1\t68053 4169\n",
      "13\t80561\t0.2\t1\t76346 4215\n",
      "14\t76520\t0.2\t1\t71606 4914\n",
      "15\t62607\t0.2\t1\t59130 3477\n",
      "16\t91383\t0.2\t1\t86350 5033\n",
      "17\t72745\t0.2\t1\t68212 4533\n",
      "18\t70883\t0.2\t1\t67020 3863\n",
      "19\t89801\t0.2\t1\t84529 5272\n",
      "20\t83841\t0.2\t1\t79367 4474\n",
      "21\t75027\t0.2\t1\t70918 4109\n",
      "22\t84425\t0.2\t1\t79923 4502\n",
      "23\t88024\t0.2\t1\t83097 4927\n",
      "24\t88855\t0.2\t1\t83891 4964\n",
      "25\t101226\t0.2\t1\t95974 5252\n",
      "26\t83308\t0.2\t1\t78591 4717\n",
      "27\t92909\t0.2\t1\t87066 5843\n",
      "28\t87156\t0.2\t1\t83145 4011\n",
      "29\t93947\t0.2\t1\t88848 5099\n",
      "30\t93821\t0.2\t1\t89436 4385\n",
      "31\t101908\t0.2\t1\t96520 5388\n",
      "32\t92154\t0.2\t1\t87903 4251\n",
      "33\t101850\t0.2\t1\t96541 5309\n",
      "34\t132968\t0.2\t1\t125972 6996\n",
      "35\t75102\t0.2\t1\t71554 3548\n",
      "36\t106526\t0.2\t1\t100872 5654\n",
      "37\t112114\t0.2\t1\t106255 5859\n",
      "38\t106187\t0.2\t1\t101516 4671\n",
      "39\t103250\t0.2\t1\t98795 4455\n",
      "40\t116350\t0.2\t1\t110242 6108\n",
      "41\t111543\t0.2\t1\t106565 4978\n",
      "42\t115913\t0.2\t1\t110317 5596\n",
      "43\t105510\t0.2\t1\t100613 4897\n",
      "44\t121504\t0.2\t1\t115737 5767\n",
      "45\t131840\t0.2\t1\t125697 6143\n",
      "46\t94875\t0.2\t1\t90604 4271\n",
      "47\t121018\t0.2\t1\t115877 5141\n",
      "48\t118842\t0.2\t1\t113797 5045\n",
      "49\t124446\t0.2\t1\t118832 5614\n",
      "50\t118268\t0.2\t1\t112777 5491\n",
      "51\t147285\t0.2\t1\t140570 6715\n",
      "52\t109448\t0.2\t1\t105060 4388\n",
      "53\t130385\t0.2\t1\t125319 5066\n",
      "54\t99487\t0.2\t1\t95470 4017\n",
      "55\t129239\t0.2\t1\t124277 4962\n",
      "56\t133599\t0.2\t1\t127967 5632\n",
      "57\t126390\t0.2\t1\t121587 4803\n",
      "58\t121811\t0.2\t1\t117475 4336\n",
      "59\t124529\t0.2\t1\t119540 4989\n",
      "60\t130401\t0.2\t1\t125258 5143\n",
      "61\t134015\t0.2\t1\t128535 5480\n",
      "62\t145629\t0.2\t1\t139763 5866\n",
      "63\t197423\t0.2\t1\t190204 7219\n",
      "64\t168240\t0.2\t1\t162928 5312\n",
      "65\t106543\t0.2\t1\t102625 3918\n",
      "66\t114942\t0.2\t1\t110706 4236\n",
      "67\t114164\t0.2\t1\t109736 4428\n",
      "68\t116350\t0.2\t1\t112096 4254\n",
      "69\t114565\t0.2\t1\t110115 4450\n",
      "70\t124406\t0.2\t1\t119762 4644\n",
      "71\t128031\t0.2\t1\t123514 4517\n",
      "72\t120687\t0.2\t1\t116066 4621\n",
      "73\t126907\t0.2\t1\t122307 4600\n",
      "74\t117912\t0.2\t1\t113666 4246\n",
      "75\t113448\t0.2\t1\t109343 4105\n",
      "76\t111278\t0.2\t1\t107269 4009\n",
      "77\t112780\t0.2\t1\t108679 4101\n",
      "78\t106864\t0.2\t1\t102856 4008\n",
      "79\t103353\t0.2\t1\t99664 3689\n",
      "80\t103266\t0.2\t1\t99595 3671\n",
      "81\t101983\t0.2\t1\t98265 3718\n",
      "82\t97716\t0.2\t1\t94285 3431\n",
      "83\t95425\t0.2\t1\t92073 3352\n",
      "84\t92931\t0.2\t1\t89513 3418\n",
      "85\t92256\t0.2\t1\t88832 3424\n",
      "86\t91168\t0.2\t1\t87863 3305\n",
      "87\t91423\t0.2\t1\t88169 3254\n",
      "88\t89550\t0.2\t1\t86420 3130\n",
      "89\t80366\t0.2\t1\t77566 2800\n",
      "90\t77351\t0.2\t1\t74500 2851\n",
      "91\t72940\t0.2\t1\t70306 2634\n",
      "92\t68488\t0.2\t1\t66133 2355\n",
      "93\t65935\t0.2\t1\t63542 2393\n",
      "94\t63561\t0.2\t1\t61328 2233\n",
      "95\t59923\t0.2\t1\t57811 2112\n",
      "96\t58074\t0.2\t1\t56092 1982\n",
      "97\t54057\t0.2\t1\t52194 1863\n",
      "98\t50980\t0.2\t1\t49204 1776\n",
      "99\t50084\t0.2\t1\t48441 1643\n",
      "100\t43642\t0.2\t1\t42153 1489\n",
      "101\t40618\t0.2\t1\t39288 1330\n",
      "102\t38299\t0.2\t1\t37001 1298\n",
      "103\t34998\t0.2\t1\t33809 1189\n",
      "104\t34454\t0.2\t1\t33375 1079\n",
      "105\t29814\t0.2\t1\t28817 997\n",
      "106\t27473\t0.2\t1\t26595 878\n",
      "107\t24384\t0.2\t1\t23578 806\n",
      "108\t22338\t0.2\t1\t21561 777\n",
      "109\t19549\t0.2\t1\t18854 695\n",
      "110\t18085\t0.2\t1\t17505 580\n",
      "111\t16893\t0.2\t1\t16326 567\n",
      "112\t14541\t0.2\t1\t14060 481\n",
      "113\t12649\t0.2\t1\t12207 442\n",
      "114\t11108\t0.2\t1\t10661 447\n",
      "115\t9325\t0.2\t1\t9006 319\n",
      "116\t7950\t0.2\t1\t7620 330\n",
      "117\t6499\t0.2\t1\t6272 227\n",
      "118\t5695\t0.2\t1\t5463 232\n",
      "119\t4844\t0.2\t1\t4637 207\n",
      "120\t4029\t0.2\t1\t3868 161\n",
      "121\t3484\t0.2\t1\t3355 129\n",
      "122\t2831\t0.2\t1\t2700 131\n",
      "123\t2522\t0.2\t1\t2436 86\n",
      "124\t2113\t0.2\t1\t2005 108\n",
      "125\t1739\t0.2\t1\t1651 88\n",
      "126\t1465\t0.2\t1\t1407 58\n",
      "127\t1198\t0.2\t1\t1145 53\n",
      "128\t931\t0.2\t1\t888 43\n",
      "129\t754\t0.2\t1\t713 41\n",
      "130\t539\t0.2\t1\t514 25\n",
      "131\t443\t0.2\t1\t421 22\n",
      "132\t375\t0.2\t1\t351 24\n",
      "133\t401\t0.2\t1\t386 15\n",
      "134\t477\t0.2\t1\t456 21\n",
      "135\t242\t0.2\t1\t234 8\n",
      "136\t136\t0.2\t1\t132 4\n",
      "137\t103\t0.2\t1\t99 4\n",
      "138\t81\t0.2\t1\t73 8\n",
      "139\t54\t0.2\t1\t51 3\n",
      "140\t37\t0.2\t1\t32 5\n",
      "141\t20\t0.2\t1\t19 1\n",
      "142\t21\t0.2\t1\t20 1\n",
      "143\t18\t0.2\t1\t16 2\n",
      "144\t17\t0.2\t1\t16 1\n",
      "145\t18\t0.2\t1\t16 2\n",
      "146\t15\t0.2\t1\t14 1\n",
      "147\t32\t0.2\t1\t29 3\n",
      "148\t19\t0.2\t1\t17 2\n",
      "149\t11\t0.2\t1\t10 1\n",
      "150\t176\t0.2\t1\t165 11\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14166205 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz, file_2: 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14166205\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 27647 (0.20%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_3_S7_L001_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_3_S7_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 527.34 s (37 us/read; 1.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,120,008\n",
      "Reads with adapters:                11,800,625 (83.6%)\n",
      "Reads written (passing filters):    14,120,008 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,118,001,200 bp\n",
      "Quality-trimmed:               4,373,755 bp (0.2%)\n",
      "Total written (filtered):  1,553,898,462 bp (73.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11800625 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 15.1%\n",
      "  G: 21.0%\n",
      "  T: 34.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1368833\t3530002.0\t0\t1368833\n",
      "2\t213297\t882500.5\t0\t213297\n",
      "3\t129961\t220625.1\t0\t129961\n",
      "4\t79035\t55156.3\t0\t79035\n",
      "5\t69573\t13789.1\t0\t69573\n",
      "6\t67663\t3447.3\t0\t67663\n",
      "7\t54249\t861.8\t0\t54249\n",
      "8\t69306\t215.5\t0\t69306\n",
      "9\t69302\t53.9\t0\t68512 790\n",
      "10\t64667\t13.5\t1\t60704 3963\n",
      "11\t73696\t3.4\t1\t67523 6173\n",
      "12\t69675\t0.8\t1\t64547 5128\n",
      "13\t71952\t0.2\t1\t66511 5441\n",
      "14\t73593\t0.2\t1\t67472 6121\n",
      "15\t76996\t0.2\t1\t70887 6109\n",
      "16\t75526\t0.2\t1\t69104 6422\n",
      "17\t93355\t0.2\t1\t84545 8810\n",
      "18\t79593\t0.2\t1\t73933 5660\n",
      "19\t68124\t0.2\t1\t63039 5085\n",
      "20\t82749\t0.2\t1\t76505 6244\n",
      "21\t92275\t0.2\t1\t85139 7136\n",
      "22\t77618\t0.2\t1\t72438 5180\n",
      "23\t92025\t0.2\t1\t84966 7059\n",
      "24\t85340\t0.2\t1\t77863 7477\n",
      "25\t92374\t0.2\t1\t85186 7188\n",
      "26\t84405\t0.2\t1\t78632 5773\n",
      "27\t95395\t0.2\t1\t87567 7828\n",
      "28\t93111\t0.2\t1\t85941 7170\n",
      "29\t103004\t0.2\t1\t93769 9235\n",
      "30\t106080\t0.2\t1\t99144 6936\n",
      "31\t80431\t0.2\t1\t74401 6030\n",
      "32\t102254\t0.2\t1\t94262 7992\n",
      "33\t95806\t0.2\t1\t88463 7343\n",
      "34\t106075\t0.2\t1\t96928 9147\n",
      "35\t108269\t0.2\t1\t99389 8880\n",
      "36\t112253\t0.2\t1\t102783 9470\n",
      "37\t99044\t0.2\t1\t92235 6809\n",
      "38\t121329\t0.2\t1\t110516 10813\n",
      "39\t100947\t0.2\t1\t93251 7696\n",
      "40\t114342\t0.2\t1\t105610 8732\n",
      "41\t128592\t0.2\t1\t117446 11146\n",
      "42\t100668\t0.2\t1\t92131 8537\n",
      "43\t169999\t0.2\t1\t159204 10795\n",
      "44\t56249\t0.2\t1\t50770 5479\n",
      "45\t119775\t0.2\t1\t111031 8744\n",
      "46\t112063\t0.2\t1\t103816 8247\n",
      "47\t119122\t0.2\t1\t110508 8614\n",
      "48\t118788\t0.2\t1\t109597 9191\n",
      "49\t131198\t0.2\t1\t120814 10384\n",
      "50\t109980\t0.2\t1\t102619 7361\n",
      "51\t122763\t0.2\t1\t113852 8911\n",
      "52\t127644\t0.2\t1\t117521 10123\n",
      "53\t136952\t0.2\t1\t127680 9272\n",
      "54\t107784\t0.2\t1\t99974 7810\n",
      "55\t130694\t0.2\t1\t121255 9439\n",
      "56\t158527\t0.2\t1\t147787 10740\n",
      "57\t103844\t0.2\t1\t95331 8513\n",
      "58\t132382\t0.2\t1\t124247 8135\n",
      "59\t116544\t0.2\t1\t107902 8642\n",
      "60\t129622\t0.2\t1\t120349 9273\n",
      "61\t128569\t0.2\t1\t119774 8795\n",
      "62\t140522\t0.2\t1\t130790 9732\n",
      "63\t114632\t0.2\t1\t105994 8638\n",
      "64\t124792\t0.2\t1\t116366 8426\n",
      "65\t140416\t0.2\t1\t130644 9772\n",
      "66\t145932\t0.2\t1\t135671 10261\n",
      "67\t159891\t0.2\t1\t146767 13124\n",
      "68\t288595\t0.2\t1\t272910 15685\n",
      "69\t63057\t0.2\t1\t57575 5482\n",
      "70\t26024\t0.2\t1\t21992 4032\n",
      "71\t79266\t0.2\t1\t73983 5283\n",
      "72\t112824\t0.2\t1\t105728 7096\n",
      "73\t119835\t0.2\t1\t111892 7943\n",
      "74\t119611\t0.2\t1\t111533 8078\n",
      "75\t117923\t0.2\t1\t109788 8135\n",
      "76\t117731\t0.2\t1\t109296 8435\n",
      "77\t118271\t0.2\t1\t109919 8352\n",
      "78\t111566\t0.2\t1\t103610 7956\n",
      "79\t111519\t0.2\t1\t103611 7908\n",
      "80\t109197\t0.2\t1\t101453 7744\n",
      "81\t107570\t0.2\t1\t99856 7714\n",
      "82\t104569\t0.2\t1\t97118 7451\n",
      "83\t102383\t0.2\t1\t95228 7155\n",
      "84\t99102\t0.2\t1\t92021 7081\n",
      "85\t96000\t0.2\t1\t89098 6902\n",
      "86\t93524\t0.2\t1\t86838 6686\n",
      "87\t89373\t0.2\t1\t83285 6088\n",
      "88\t88917\t0.2\t1\t82644 6273\n",
      "89\t84001\t0.2\t1\t77961 6040\n",
      "90\t83693\t0.2\t1\t77869 5824\n",
      "91\t80101\t0.2\t1\t74574 5527\n",
      "92\t76032\t0.2\t1\t70730 5302\n",
      "93\t71132\t0.2\t1\t66158 4974\n",
      "94\t68649\t0.2\t1\t63740 4909\n",
      "95\t64070\t0.2\t1\t59484 4586\n",
      "96\t60750\t0.2\t1\t56444 4306\n",
      "97\t57341\t0.2\t1\t53268 4073\n",
      "98\t53260\t0.2\t1\t49347 3913\n",
      "99\t52636\t0.2\t1\t48599 4037\n",
      "100\t46791\t0.2\t1\t43393 3398\n",
      "101\t44342\t0.2\t1\t41093 3249\n",
      "102\t40590\t0.2\t1\t37587 3003\n",
      "103\t37417\t0.2\t1\t34694 2723\n",
      "104\t34470\t0.2\t1\t31943 2527\n",
      "105\t30930\t0.2\t1\t28658 2272\n",
      "106\t28279\t0.2\t1\t26119 2160\n",
      "107\t25736\t0.2\t1\t23845 1891\n",
      "108\t23849\t0.2\t1\t22042 1807\n",
      "109\t21192\t0.2\t1\t19537 1655\n",
      "110\t19116\t0.2\t1\t17675 1441\n",
      "111\t17231\t0.2\t1\t15808 1423\n",
      "112\t14798\t0.2\t1\t13558 1240\n",
      "113\t12618\t0.2\t1\t11582 1036\n",
      "114\t11317\t0.2\t1\t10327 990\n",
      "115\t9619\t0.2\t1\t8804 815\n",
      "116\t8405\t0.2\t1\t7688 717\n",
      "117\t7048\t0.2\t1\t6428 620\n",
      "118\t6316\t0.2\t1\t5766 550\n",
      "119\t5076\t0.2\t1\t4600 476\n",
      "120\t4250\t0.2\t1\t3802 448\n",
      "121\t3670\t0.2\t1\t3303 367\n",
      "122\t3096\t0.2\t1\t2757 339\n",
      "123\t2589\t0.2\t1\t2312 277\n",
      "124\t2201\t0.2\t1\t1979 222\n",
      "125\t1708\t0.2\t1\t1510 198\n",
      "126\t1570\t0.2\t1\t1383 187\n",
      "127\t1266\t0.2\t1\t1112 154\n",
      "128\t926\t0.2\t1\t821 105\n",
      "129\t844\t0.2\t1\t748 96\n",
      "130\t568\t0.2\t1\t488 80\n",
      "131\t507\t0.2\t1\t451 56\n",
      "132\t422\t0.2\t1\t368 54\n",
      "133\t416\t0.2\t1\t358 58\n",
      "134\t449\t0.2\t1\t401 48\n",
      "135\t219\t0.2\t1\t188 31\n",
      "136\t167\t0.2\t1\t147 20\n",
      "137\t91\t0.2\t1\t79 12\n",
      "138\t80\t0.2\t1\t64 16\n",
      "139\t47\t0.2\t1\t42 5\n",
      "140\t27\t0.2\t1\t21 6\n",
      "141\t27\t0.2\t1\t23 4\n",
      "142\t18\t0.2\t1\t10 8\n",
      "143\t22\t0.2\t1\t11 11\n",
      "144\t14\t0.2\t1\t10 4\n",
      "145\t18\t0.2\t1\t16 2\n",
      "146\t18\t0.2\t1\t14 4\n",
      "147\t32\t0.2\t1\t28 4\n",
      "148\t19\t0.2\t1\t17 2\n",
      "149\t14\t0.2\t1\t10 4\n",
      "150\t202\t0.2\t1\t177 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14120008 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_3_S7_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 560.35 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,120,008\n",
      "Reads with adapters:                11,722,164 (83.0%)\n",
      "Reads written (passing filters):    14,120,008 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,118,001,200 bp\n",
      "Quality-trimmed:              10,398,441 bp (0.5%)\n",
      "Total written (filtered):  1,557,917,839 bp (73.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11722164 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.6%\n",
      "  C: 13.3%\n",
      "  G: 24.5%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1423484\t3530002.0\t0\t1423484\n",
      "2\t217404\t882500.5\t0\t217404\n",
      "3\t135747\t220625.1\t0\t135747\n",
      "4\t77416\t55156.3\t0\t77416\n",
      "5\t73163\t13789.1\t0\t73163\n",
      "6\t65957\t3447.3\t0\t65957\n",
      "7\t52727\t861.8\t0\t52727\n",
      "8\t75617\t215.5\t0\t75617\n",
      "9\t60824\t53.9\t0\t60611 213\n",
      "10\t64613\t13.5\t1\t60561 4052\n",
      "11\t71071\t3.4\t1\t65663 5408\n",
      "12\t69904\t0.8\t1\t64695 5209\n",
      "13\t80991\t0.2\t1\t75018 5973\n",
      "14\t75627\t0.2\t1\t69287 6340\n",
      "15\t61630\t0.2\t1\t57275 4355\n",
      "16\t89541\t0.2\t1\t82804 6737\n",
      "17\t75362\t0.2\t1\t69553 5809\n",
      "18\t67936\t0.2\t1\t63097 4839\n",
      "19\t88766\t0.2\t1\t82324 6442\n",
      "20\t83180\t0.2\t1\t77328 5852\n",
      "21\t74299\t0.2\t1\t69124 5175\n",
      "22\t83499\t0.2\t1\t77755 5744\n",
      "23\t88403\t0.2\t1\t82296 6107\n",
      "24\t88389\t0.2\t1\t82109 6280\n",
      "25\t100119\t0.2\t1\t93881 6238\n",
      "26\t84031\t0.2\t1\t77871 6160\n",
      "27\t93205\t0.2\t1\t86091 7114\n",
      "28\t83882\t0.2\t1\t78923 4959\n",
      "29\t95544\t0.2\t1\t89031 6513\n",
      "30\t91423\t0.2\t1\t85830 5593\n",
      "31\t102885\t0.2\t1\t96159 6726\n",
      "32\t88346\t0.2\t1\t83048 5298\n",
      "33\t103020\t0.2\t1\t96084 6936\n",
      "34\t134691\t0.2\t1\t125875 8816\n",
      "35\t71511\t0.2\t1\t67043 4468\n",
      "36\t106372\t0.2\t1\t99204 7168\n",
      "37\t111667\t0.2\t1\t104414 7253\n",
      "38\t105292\t0.2\t1\t99304 5988\n",
      "39\t99654\t0.2\t1\t94273 5381\n",
      "40\t115518\t0.2\t1\t108179 7339\n",
      "41\t111214\t0.2\t1\t104737 6477\n",
      "42\t115443\t0.2\t1\t108784 6659\n",
      "43\t104707\t0.2\t1\t98628 6079\n",
      "44\t122428\t0.2\t1\t115040 7388\n",
      "45\t129896\t0.2\t1\t122003 7893\n",
      "46\t92988\t0.2\t1\t87594 5394\n",
      "47\t119950\t0.2\t1\t113634 6316\n",
      "48\t117307\t0.2\t1\t110893 6414\n",
      "49\t123614\t0.2\t1\t116211 7403\n",
      "50\t118595\t0.2\t1\t111825 6770\n",
      "51\t146157\t0.2\t1\t137800 8357\n",
      "52\t108528\t0.2\t1\t102556 5972\n",
      "53\t133051\t0.2\t1\t126144 6907\n",
      "54\t97258\t0.2\t1\t92286 4972\n",
      "55\t128012\t0.2\t1\t121714 6298\n",
      "56\t133239\t0.2\t1\t125868 7371\n",
      "57\t126482\t0.2\t1\t120493 5989\n",
      "58\t123495\t0.2\t1\t117409 6086\n",
      "59\t123171\t0.2\t1\t116902 6269\n",
      "60\t129991\t0.2\t1\t123447 6544\n",
      "61\t132791\t0.2\t1\t126344 6447\n",
      "62\t145203\t0.2\t1\t137595 7608\n",
      "63\t193193\t0.2\t1\t184570 8623\n",
      "64\t164505\t0.2\t1\t158250 6255\n",
      "65\t106154\t0.2\t1\t101191 4963\n",
      "66\t116219\t0.2\t1\t110707 5512\n",
      "67\t115499\t0.2\t1\t109857 5642\n",
      "68\t116592\t0.2\t1\t111069 5523\n",
      "69\t115665\t0.2\t1\t109783 5882\n",
      "70\t124608\t0.2\t1\t118610 5998\n",
      "71\t128211\t0.2\t1\t122084 6127\n",
      "72\t122112\t0.2\t1\t116125 5987\n",
      "73\t126921\t0.2\t1\t120760 6161\n",
      "74\t118790\t0.2\t1\t112963 5827\n",
      "75\t113173\t0.2\t1\t107712 5461\n",
      "76\t111463\t0.2\t1\t105998 5465\n",
      "77\t111662\t0.2\t1\t106140 5522\n",
      "78\t107207\t0.2\t1\t101895 5312\n",
      "79\t104292\t0.2\t1\t99234 5058\n",
      "80\t103845\t0.2\t1\t98916 4929\n",
      "81\t101938\t0.2\t1\t97104 4834\n",
      "82\t97773\t0.2\t1\t93124 4649\n",
      "83\t95971\t0.2\t1\t91487 4484\n",
      "84\t92983\t0.2\t1\t88672 4311\n",
      "85\t91731\t0.2\t1\t87597 4134\n",
      "86\t91538\t0.2\t1\t87240 4298\n",
      "87\t90919\t0.2\t1\t86818 4101\n",
      "88\t89775\t0.2\t1\t85803 3972\n",
      "89\t80478\t0.2\t1\t76988 3490\n",
      "90\t77967\t0.2\t1\t74477 3490\n",
      "91\t73781\t0.2\t1\t70569 3212\n",
      "92\t69447\t0.2\t1\t66340 3107\n",
      "93\t66394\t0.2\t1\t63362 3032\n",
      "94\t63823\t0.2\t1\t61051 2772\n",
      "95\t60418\t0.2\t1\t57617 2801\n",
      "96\t57934\t0.2\t1\t55484 2450\n",
      "97\t54242\t0.2\t1\t51870 2372\n",
      "98\t51153\t0.2\t1\t49011 2142\n",
      "99\t50193\t0.2\t1\t47988 2205\n",
      "100\t43401\t0.2\t1\t41462 1939\n",
      "101\t41188\t0.2\t1\t39459 1729\n",
      "102\t38238\t0.2\t1\t36636 1602\n",
      "103\t35697\t0.2\t1\t34094 1603\n",
      "104\t34260\t0.2\t1\t32771 1489\n",
      "105\t30330\t0.2\t1\t29031 1299\n",
      "106\t27539\t0.2\t1\t26325 1214\n",
      "107\t24680\t0.2\t1\t23635 1045\n",
      "108\t22009\t0.2\t1\t21062 947\n",
      "109\t19974\t0.2\t1\t19117 857\n",
      "110\t18015\t0.2\t1\t17209 806\n",
      "111\t16915\t0.2\t1\t16205 710\n",
      "112\t14332\t0.2\t1\t13689 643\n",
      "113\t12446\t0.2\t1\t11895 551\n",
      "114\t10968\t0.2\t1\t10478 490\n",
      "115\t9016\t0.2\t1\t8590 426\n",
      "116\t7818\t0.2\t1\t7478 340\n",
      "117\t6323\t0.2\t1\t6027 296\n",
      "118\t5766\t0.2\t1\t5518 248\n",
      "119\t4704\t0.2\t1\t4500 204\n",
      "120\t3901\t0.2\t1\t3708 193\n",
      "121\t3388\t0.2\t1\t3195 193\n",
      "122\t2862\t0.2\t1\t2731 131\n",
      "123\t2458\t0.2\t1\t2315 143\n",
      "124\t2060\t0.2\t1\t1986 74\n",
      "125\t1642\t0.2\t1\t1547 95\n",
      "126\t1477\t0.2\t1\t1395 82\n",
      "127\t1193\t0.2\t1\t1127 66\n",
      "128\t861\t0.2\t1\t803 58\n",
      "129\t803\t0.2\t1\t750 53\n",
      "130\t513\t0.2\t1\t479 34\n",
      "131\t475\t0.2\t1\t451 24\n",
      "132\t384\t0.2\t1\t349 35\n",
      "133\t402\t0.2\t1\t380 22\n",
      "134\t430\t0.2\t1\t416 14\n",
      "135\t203\t0.2\t1\t185 18\n",
      "136\t160\t0.2\t1\t156 4\n",
      "137\t84\t0.2\t1\t74 10\n",
      "138\t68\t0.2\t1\t64 4\n",
      "139\t47\t0.2\t1\t44 3\n",
      "140\t21\t0.2\t1\t20 1\n",
      "141\t24\t0.2\t1\t21 3\n",
      "142\t16\t0.2\t1\t13 3\n",
      "143\t16\t0.2\t1\t13 3\n",
      "144\t10\t0.2\t1\t10\n",
      "145\t20\t0.2\t1\t19 1\n",
      "146\t18\t0.2\t1\t18\n",
      "147\t25\t0.2\t1\t23 2\n",
      "148\t19\t0.2\t1\t15 4\n",
      "149\t13\t0.2\t1\t12 1\n",
      "150\t153\t0.2\t1\t145 8\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14120008 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz, file_2: 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14120008\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 25725 (0.18%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_3_S7_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_3_S7_L002_R1_001_trimmed.fq.gz and 8C_32psu_3_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_4_S8_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 602.49 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,686,084\n",
      "Reads with adapters:                11,112,819 (75.7%)\n",
      "Reads written (passing filters):    14,686,084 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,202,912,600 bp\n",
      "Quality-trimmed:               9,079,514 bp (0.4%)\n",
      "Total written (filtered):  1,783,990,340 bp (81.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11112819 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 11.7%\n",
      "  G: 21.0%\n",
      "  T: 36.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2196567\t3671521.0\t0\t2196567\n",
      "2\t292186\t917880.2\t0\t292186\n",
      "3\t157941\t229470.1\t0\t157941\n",
      "4\t100753\t57367.5\t0\t100753\n",
      "5\t83202\t14341.9\t0\t83202\n",
      "6\t81280\t3585.5\t0\t81280\n",
      "7\t68619\t896.4\t0\t68619\n",
      "8\t84673\t224.1\t0\t84673\n",
      "9\t83412\t56.0\t0\t82605 807\n",
      "10\t77900\t14.0\t1\t74163 3737\n",
      "11\t86076\t3.5\t1\t80161 5915\n",
      "12\t82933\t0.9\t1\t78087 4846\n",
      "13\t83856\t0.2\t1\t78977 4879\n",
      "14\t84709\t0.2\t1\t79208 5501\n",
      "15\t88097\t0.2\t1\t82938 5159\n",
      "16\t88174\t0.2\t1\t82008 6166\n",
      "17\t105389\t0.2\t1\t97630 7759\n",
      "18\t89620\t0.2\t1\t84611 5009\n",
      "19\t77413\t0.2\t1\t72875 4538\n",
      "20\t91427\t0.2\t1\t86037 5390\n",
      "21\t101267\t0.2\t1\t94618 6649\n",
      "22\t86534\t0.2\t1\t82143 4391\n",
      "23\t99603\t0.2\t1\t93638 5965\n",
      "24\t92844\t0.2\t1\t85594 7250\n",
      "25\t98567\t0.2\t1\t92532 6035\n",
      "26\t89487\t0.2\t1\t84719 4768\n",
      "27\t100312\t0.2\t1\t93524 6788\n",
      "28\t97676\t0.2\t1\t91379 6297\n",
      "29\t103869\t0.2\t1\t95980 7889\n",
      "30\t115073\t0.2\t1\t109144 5929\n",
      "31\t81679\t0.2\t1\t76971 4708\n",
      "32\t104832\t0.2\t1\t98556 6276\n",
      "33\t97135\t0.2\t1\t91030 6105\n",
      "34\t104303\t0.2\t1\t98156 6147\n",
      "35\t102079\t0.2\t1\t95269 6810\n",
      "36\t105105\t0.2\t1\t98314 6791\n",
      "37\t106489\t0.2\t1\t100050 6439\n",
      "38\t111587\t0.2\t1\t104304 7283\n",
      "39\t98240\t0.2\t1\t93162 5078\n",
      "40\t110224\t0.2\t1\t103236 6988\n",
      "41\t119138\t0.2\t1\t109010 10128\n",
      "42\t107751\t0.2\t1\t99258 8493\n",
      "43\t167048\t0.2\t1\t157990 9058\n",
      "44\t54792\t0.2\t1\t50043 4749\n",
      "45\t116148\t0.2\t1\t107107 9041\n",
      "46\t253343\t0.2\t1\t239166 14177\n",
      "47\t141220\t0.2\t1\t133245 7975\n",
      "48\t70788\t0.2\t1\t63989 6799\n",
      "49\t218187\t0.2\t1\t207304 10883\n",
      "50\t67793\t0.2\t1\t62869 4924\n",
      "51\t84957\t0.2\t1\t78961 5996\n",
      "52\t231141\t0.2\t1\t220209 10932\n",
      "53\t159552\t0.2\t1\t153244 6308\n",
      "54\t57820\t0.2\t1\t54077 3743\n",
      "55\t96525\t0.2\t1\t92498 4027\n",
      "56\t56201\t0.2\t1\t52298 3903\n",
      "57\t134736\t0.2\t1\t129198 5538\n",
      "58\t33322\t0.2\t1\t31241 2081\n",
      "59\t35361\t0.2\t1\t32301 3060\n",
      "60\t121909\t0.2\t1\t116692 5217\n",
      "61\t80368\t0.2\t1\t77075 3293\n",
      "62\t37240\t0.2\t1\t34469 2771\n",
      "63\t91660\t0.2\t1\t86780 4880\n",
      "64\t81468\t0.2\t1\t77877 3591\n",
      "65\t53211\t0.2\t1\t49893 3318\n",
      "66\t93831\t0.2\t1\t88616 5215\n",
      "67\t107034\t0.2\t1\t100335 6699\n",
      "68\t187851\t0.2\t1\t179197 8654\n",
      "69\t100255\t0.2\t1\t94892 5363\n",
      "70\t35399\t0.2\t1\t32342 3057\n",
      "71\t19897\t0.2\t1\t17997 1900\n",
      "72\t63849\t0.2\t1\t60613 3236\n",
      "73\t81267\t0.2\t1\t77051 4216\n",
      "74\t82339\t0.2\t1\t77944 4395\n",
      "75\t80643\t0.2\t1\t76099 4544\n",
      "76\t79016\t0.2\t1\t74790 4226\n",
      "77\t78401\t0.2\t1\t74108 4293\n",
      "78\t73966\t0.2\t1\t69851 4115\n",
      "79\t71627\t0.2\t1\t67584 4043\n",
      "80\t70426\t0.2\t1\t66563 3863\n",
      "81\t69818\t0.2\t1\t65954 3864\n",
      "82\t67366\t0.2\t1\t63575 3791\n",
      "83\t63649\t0.2\t1\t60193 3456\n",
      "84\t61172\t0.2\t1\t57824 3348\n",
      "85\t59276\t0.2\t1\t56040 3236\n",
      "86\t56575\t0.2\t1\t53454 3121\n",
      "87\t53078\t0.2\t1\t50236 2842\n",
      "88\t51788\t0.2\t1\t48829 2959\n",
      "89\t48314\t0.2\t1\t45638 2676\n",
      "90\t48321\t0.2\t1\t45629 2692\n",
      "91\t45543\t0.2\t1\t43073 2470\n",
      "92\t42075\t0.2\t1\t39779 2296\n",
      "93\t38814\t0.2\t1\t36658 2156\n",
      "94\t37722\t0.2\t1\t35616 2106\n",
      "95\t34423\t0.2\t1\t32423 2000\n",
      "96\t32157\t0.2\t1\t30284 1873\n",
      "97\t29965\t0.2\t1\t28206 1759\n",
      "98\t27347\t0.2\t1\t25797 1550\n",
      "99\t26166\t0.2\t1\t24583 1583\n",
      "100\t23354\t0.2\t1\t21969 1385\n",
      "101\t21467\t0.2\t1\t20157 1310\n",
      "102\t19539\t0.2\t1\t18344 1195\n",
      "103\t17855\t0.2\t1\t16735 1120\n",
      "104\t16360\t0.2\t1\t15329 1031\n",
      "105\t14296\t0.2\t1\t13369 927\n",
      "106\t12633\t0.2\t1\t11870 763\n",
      "107\t11358\t0.2\t1\t10612 746\n",
      "108\t10534\t0.2\t1\t9836 698\n",
      "109\t9078\t0.2\t1\t8461 617\n",
      "110\t8053\t0.2\t1\t7464 589\n",
      "111\t6944\t0.2\t1\t6454 490\n",
      "112\t6153\t0.2\t1\t5691 462\n",
      "113\t5306\t0.2\t1\t4906 400\n",
      "114\t4518\t0.2\t1\t4189 329\n",
      "115\t3910\t0.2\t1\t3603 307\n",
      "116\t3193\t0.2\t1\t2951 242\n",
      "117\t2651\t0.2\t1\t2439 212\n",
      "118\t2314\t0.2\t1\t2103 211\n",
      "119\t1875\t0.2\t1\t1673 202\n",
      "120\t1577\t0.2\t1\t1427 150\n",
      "121\t1320\t0.2\t1\t1198 122\n",
      "122\t1105\t0.2\t1\t995 110\n",
      "123\t920\t0.2\t1\t816 104\n",
      "124\t791\t0.2\t1\t709 82\n",
      "125\t642\t0.2\t1\t583 59\n",
      "126\t507\t0.2\t1\t450 57\n",
      "127\t422\t0.2\t1\t367 55\n",
      "128\t333\t0.2\t1\t289 44\n",
      "129\t238\t0.2\t1\t206 32\n",
      "130\t186\t0.2\t1\t162 24\n",
      "131\t174\t0.2\t1\t151 23\n",
      "132\t134\t0.2\t1\t118 16\n",
      "133\t148\t0.2\t1\t129 19\n",
      "134\t197\t0.2\t1\t182 15\n",
      "135\t113\t0.2\t1\t97 16\n",
      "136\t58\t0.2\t1\t50 8\n",
      "137\t58\t0.2\t1\t48 10\n",
      "138\t42\t0.2\t1\t35 7\n",
      "139\t30\t0.2\t1\t28 2\n",
      "140\t22\t0.2\t1\t20 2\n",
      "141\t15\t0.2\t1\t9 6\n",
      "142\t13\t0.2\t1\t11 2\n",
      "143\t11\t0.2\t1\t6 5\n",
      "144\t2\t0.2\t1\t1 1\n",
      "145\t7\t0.2\t1\t6 1\n",
      "146\t5\t0.2\t1\t4 1\n",
      "147\t21\t0.2\t1\t20 1\n",
      "148\t23\t0.2\t1\t19 4\n",
      "149\t14\t0.2\t1\t12 2\n",
      "150\t119\t0.2\t1\t95 24\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "14686084 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_4_S8_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 636.57 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,686,084\n",
      "Reads with adapters:                11,052,394 (75.3%)\n",
      "Reads written (passing filters):    14,686,084 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,202,912,600 bp\n",
      "Quality-trimmed:               8,170,892 bp (0.4%)\n",
      "Total written (filtered):  1,785,498,326 bp (81.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11052394 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 11.2%\n",
      "  G: 23.2%\n",
      "  T: 36.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2172306\t3671521.0\t0\t2172306\n",
      "2\t315945\t917880.2\t0\t315945\n",
      "3\t169586\t229470.1\t0\t169586\n",
      "4\t102198\t57367.5\t0\t102198\n",
      "5\t92121\t14341.9\t0\t92121\n",
      "6\t78915\t3585.5\t0\t78915\n",
      "7\t65916\t896.4\t0\t65916\n",
      "8\t91966\t224.1\t0\t91966\n",
      "9\t73557\t56.0\t0\t73312 245\n",
      "10\t79999\t14.0\t1\t76173 3826\n",
      "11\t83777\t3.5\t1\t78744 5033\n",
      "12\t84445\t0.9\t1\t79272 5173\n",
      "13\t92865\t0.2\t1\t87504 5361\n",
      "14\t88444\t0.2\t1\t82453 5991\n",
      "15\t72972\t0.2\t1\t68400 4572\n",
      "16\t103628\t0.2\t1\t97383 6245\n",
      "17\t83622\t0.2\t1\t78153 5469\n",
      "18\t78826\t0.2\t1\t74157 4669\n",
      "19\t100647\t0.2\t1\t94372 6275\n",
      "20\t93537\t0.2\t1\t88067 5470\n",
      "21\t82005\t0.2\t1\t77188 4817\n",
      "22\t92897\t0.2\t1\t87543 5354\n",
      "23\t95873\t0.2\t1\t90242 5631\n",
      "24\t96154\t0.2\t1\t90197 5957\n",
      "25\t107344\t0.2\t1\t101210 6134\n",
      "26\t88692\t0.2\t1\t83193 5499\n",
      "27\t96620\t0.2\t1\t90107 6513\n",
      "28\t92786\t0.2\t1\t88163 4623\n",
      "29\t97544\t0.2\t1\t91712 5832\n",
      "30\t97471\t0.2\t1\t92326 5145\n",
      "31\t103994\t0.2\t1\t98068 5926\n",
      "32\t93992\t0.2\t1\t89009 4983\n",
      "33\t103947\t0.2\t1\t97692 6255\n",
      "34\t130356\t0.2\t1\t123003 7353\n",
      "35\t75888\t0.2\t1\t71960 3928\n",
      "36\t101074\t0.2\t1\t95184 5890\n",
      "37\t109064\t0.2\t1\t102746 6318\n",
      "38\t100719\t0.2\t1\t95809 4910\n",
      "39\t99097\t0.2\t1\t94226 4871\n",
      "40\t108626\t0.2\t1\t102518 6108\n",
      "41\t105593\t0.2\t1\t100195 5398\n",
      "42\t108346\t0.2\t1\t102605 5741\n",
      "43\t99135\t0.2\t1\t93859 5276\n",
      "44\t112964\t0.2\t1\t106912 6052\n",
      "45\t120273\t0.2\t1\t114139 6134\n",
      "46\t86822\t0.2\t1\t82356 4466\n",
      "47\t107964\t0.2\t1\t102668 5296\n",
      "48\t105861\t0.2\t1\t100664 5197\n",
      "49\t110040\t0.2\t1\t104589 5451\n",
      "50\t103888\t0.2\t1\t98336 5552\n",
      "51\t130541\t0.2\t1\t123943 6598\n",
      "52\t92963\t0.2\t1\t88623 4340\n",
      "53\t111922\t0.2\t1\t106898 5024\n",
      "54\t84092\t0.2\t1\t80032 4060\n",
      "55\t108967\t0.2\t1\t104198 4769\n",
      "56\t110137\t0.2\t1\t104842 5295\n",
      "57\t104380\t0.2\t1\t99728 4652\n",
      "58\t100352\t0.2\t1\t95962 4390\n",
      "59\t101272\t0.2\t1\t96515 4757\n",
      "60\t103619\t0.2\t1\t98712 4907\n",
      "61\t107020\t0.2\t1\t101830 5190\n",
      "62\t116477\t0.2\t1\t111010 5467\n",
      "63\t153560\t0.2\t1\t147083 6477\n",
      "64\t118136\t0.2\t1\t113593 4543\n",
      "65\t80776\t0.2\t1\t77236 3540\n",
      "66\t87168\t0.2\t1\t83301 3867\n",
      "67\t85836\t0.2\t1\t81942 3894\n",
      "68\t86253\t0.2\t1\t82443 3810\n",
      "69\t84506\t0.2\t1\t80622 3884\n",
      "70\t92306\t0.2\t1\t88038 4268\n",
      "71\t90825\t0.2\t1\t86927 3898\n",
      "72\t86185\t0.2\t1\t82424 3761\n",
      "73\t89325\t0.2\t1\t85551 3774\n",
      "74\t82222\t0.2\t1\t78765 3457\n",
      "75\t77552\t0.2\t1\t74138 3414\n",
      "76\t75204\t0.2\t1\t71865 3339\n",
      "77\t74674\t0.2\t1\t71463 3211\n",
      "78\t71366\t0.2\t1\t68191 3175\n",
      "79\t67543\t0.2\t1\t64632 2911\n",
      "80\t67004\t0.2\t1\t64137 2867\n",
      "81\t66579\t0.2\t1\t63792 2787\n",
      "82\t63260\t0.2\t1\t60623 2637\n",
      "83\t59885\t0.2\t1\t57259 2626\n",
      "84\t57771\t0.2\t1\t55217 2554\n",
      "85\t56672\t0.2\t1\t54103 2569\n",
      "86\t55528\t0.2\t1\t53139 2389\n",
      "87\t54690\t0.2\t1\t52283 2407\n",
      "88\t52598\t0.2\t1\t50360 2238\n",
      "89\t46378\t0.2\t1\t44318 2060\n",
      "90\t44805\t0.2\t1\t42852 1953\n",
      "91\t41839\t0.2\t1\t39962 1877\n",
      "92\t38539\t0.2\t1\t36849 1690\n",
      "93\t36522\t0.2\t1\t35046 1476\n",
      "94\t35648\t0.2\t1\t34150 1498\n",
      "95\t33020\t0.2\t1\t31626 1394\n",
      "96\t31039\t0.2\t1\t29760 1279\n",
      "97\t28162\t0.2\t1\t26958 1204\n",
      "98\t25327\t0.2\t1\t24248 1079\n",
      "99\t23909\t0.2\t1\t22911 998\n",
      "100\t21397\t0.2\t1\t20519 878\n",
      "101\t19468\t0.2\t1\t18700 768\n",
      "102\t18321\t0.2\t1\t17542 779\n",
      "103\t16795\t0.2\t1\t16092 703\n",
      "104\t15607\t0.2\t1\t14948 659\n",
      "105\t13615\t0.2\t1\t13013 602\n",
      "106\t12134\t0.2\t1\t11629 505\n",
      "107\t10718\t0.2\t1\t10267 451\n",
      "108\t9677\t0.2\t1\t9284 393\n",
      "109\t8561\t0.2\t1\t8182 379\n",
      "110\t7592\t0.2\t1\t7242 350\n",
      "111\t6810\t0.2\t1\t6531 279\n",
      "112\t6017\t0.2\t1\t5741 276\n",
      "113\t5139\t0.2\t1\t4918 221\n",
      "114\t4338\t0.2\t1\t4134 204\n",
      "115\t3645\t0.2\t1\t3478 167\n",
      "116\t2960\t0.2\t1\t2807 153\n",
      "117\t2433\t0.2\t1\t2312 121\n",
      "118\t2180\t0.2\t1\t2058 122\n",
      "119\t1702\t0.2\t1\t1624 78\n",
      "120\t1461\t0.2\t1\t1377 84\n",
      "121\t1230\t0.2\t1\t1157 73\n",
      "122\t1031\t0.2\t1\t968 63\n",
      "123\t857\t0.2\t1\t808 49\n",
      "124\t756\t0.2\t1\t706 50\n",
      "125\t617\t0.2\t1\t580 37\n",
      "126\t477\t0.2\t1\t448 29\n",
      "127\t390\t0.2\t1\t359 31\n",
      "128\t303\t0.2\t1\t291 12\n",
      "129\t226\t0.2\t1\t207 19\n",
      "130\t179\t0.2\t1\t161 18\n",
      "131\t150\t0.2\t1\t142 8\n",
      "132\t131\t0.2\t1\t123 8\n",
      "133\t129\t0.2\t1\t118 11\n",
      "134\t199\t0.2\t1\t187 12\n",
      "135\t112\t0.2\t1\t107 5\n",
      "136\t50\t0.2\t1\t48 2\n",
      "137\t54\t0.2\t1\t49 5\n",
      "138\t38\t0.2\t1\t37 1\n",
      "139\t28\t0.2\t1\t24 4\n",
      "140\t20\t0.2\t1\t18 2\n",
      "141\t13\t0.2\t1\t12 1\n",
      "142\t11\t0.2\t1\t10 1\n",
      "143\t8\t0.2\t1\t7 1\n",
      "144\t5\t0.2\t1\t3 2\n",
      "145\t4\t0.2\t1\t3 1\n",
      "146\t7\t0.2\t1\t3 4\n",
      "147\t20\t0.2\t1\t18 2\n",
      "148\t18\t0.2\t1\t17 1\n",
      "149\t13\t0.2\t1\t11 2\n",
      "150\t93\t0.2\t1\t85 8\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "14686084 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz, file_2: 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14686084\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 15241 (0.10%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_4_S8_L001_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_4_S8_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 586.47 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,607,942\n",
      "Reads with adapters:                11,056,021 (75.7%)\n",
      "Reads written (passing filters):    14,607,942 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,191,191,300 bp\n",
      "Quality-trimmed:               9,138,103 bp (0.4%)\n",
      "Total written (filtered):  1,772,690,067 bp (80.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11056021 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 11.8%\n",
      "  G: 21.0%\n",
      "  T: 36.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2179670\t3651985.5\t0\t2179670\n",
      "2\t288357\t912996.4\t0\t288357\n",
      "3\t156435\t228249.1\t0\t156435\n",
      "4\t98004\t57062.3\t0\t98004\n",
      "5\t83153\t14265.6\t0\t83153\n",
      "6\t80849\t3566.4\t0\t80849\n",
      "7\t66215\t891.6\t0\t66215\n",
      "8\t83652\t222.9\t0\t83652\n",
      "9\t81421\t55.7\t0\t80631 790\n",
      "10\t76420\t13.9\t1\t72069 4351\n",
      "11\t85164\t3.5\t1\t78240 6924\n",
      "12\t81348\t0.9\t1\t75711 5637\n",
      "13\t83598\t0.2\t1\t77773 5825\n",
      "14\t83352\t0.2\t1\t76916 6436\n",
      "15\t87371\t0.2\t1\t80774 6597\n",
      "16\t86829\t0.2\t1\t79522 7307\n",
      "17\t105942\t0.2\t1\t96774 9168\n",
      "18\t88487\t0.2\t1\t82567 5920\n",
      "19\t76290\t0.2\t1\t70996 5294\n",
      "20\t90707\t0.2\t1\t84433 6274\n",
      "21\t101596\t0.2\t1\t93808 7788\n",
      "22\t84911\t0.2\t1\t79523 5388\n",
      "23\t99744\t0.2\t1\t92683 7061\n",
      "24\t90626\t0.2\t1\t82850 7776\n",
      "25\t98474\t0.2\t1\t91331 7143\n",
      "26\t89046\t0.2\t1\t83387 5659\n",
      "27\t100470\t0.2\t1\t92717 7753\n",
      "28\t97480\t0.2\t1\t90362 7118\n",
      "29\t103196\t0.2\t1\t94309 8887\n",
      "30\t114370\t0.2\t1\t107320 7050\n",
      "31\t79392\t0.2\t1\t73939 5453\n",
      "32\t105208\t0.2\t1\t97664 7544\n",
      "33\t95289\t0.2\t1\t88120 7169\n",
      "34\t103964\t0.2\t1\t96266 7698\n",
      "35\t100819\t0.2\t1\t93035 7784\n",
      "36\t104164\t0.2\t1\t96092 8072\n",
      "37\t105927\t0.2\t1\t98240 7687\n",
      "38\t112135\t0.2\t1\t103582 8553\n",
      "39\t96387\t0.2\t1\t90041 6346\n",
      "40\t110619\t0.2\t1\t102036 8583\n",
      "41\t118329\t0.2\t1\t106686 11643\n",
      "42\t106988\t0.2\t1\t97227 9761\n",
      "43\t165097\t0.2\t1\t154382 10715\n",
      "44\t56454\t0.2\t1\t50914 5540\n",
      "45\t117022\t0.2\t1\t106731 10291\n",
      "46\t248165\t0.2\t1\t231776 16389\n",
      "47\t141744\t0.2\t1\t132412 9332\n",
      "48\t71383\t0.2\t1\t63944 7439\n",
      "49\t214966\t0.2\t1\t202098 12868\n",
      "50\t67123\t0.2\t1\t61729 5394\n",
      "51\t84537\t0.2\t1\t77868 6669\n",
      "52\t227448\t0.2\t1\t214969 12479\n",
      "53\t153876\t0.2\t1\t146571 7305\n",
      "54\t58063\t0.2\t1\t53681 4382\n",
      "55\t96470\t0.2\t1\t91390 5080\n",
      "56\t61719\t0.2\t1\t56899 4820\n",
      "57\t135082\t0.2\t1\t128247 6835\n",
      "58\t34302\t0.2\t1\t31774 2528\n",
      "59\t37018\t0.2\t1\t33361 3657\n",
      "60\t120449\t0.2\t1\t114261 6188\n",
      "61\t79440\t0.2\t1\t75334 4106\n",
      "62\t40303\t0.2\t1\t37043 3260\n",
      "63\t92254\t0.2\t1\t86676 5578\n",
      "64\t78944\t0.2\t1\t74628 4316\n",
      "65\t54545\t0.2\t1\t50567 3978\n",
      "66\t92915\t0.2\t1\t86970 5945\n",
      "67\t106337\t0.2\t1\t98786 7551\n",
      "68\t178710\t0.2\t1\t168688 10022\n",
      "69\t108665\t0.2\t1\t102241 6424\n",
      "70\t43742\t0.2\t1\t39918 3824\n",
      "71\t19154\t0.2\t1\t17119 2035\n",
      "72\t62321\t0.2\t1\t58726 3595\n",
      "73\t80997\t0.2\t1\t76319 4678\n",
      "74\t82292\t0.2\t1\t77235 5057\n",
      "75\t80453\t0.2\t1\t75230 5223\n",
      "76\t78946\t0.2\t1\t73647 5299\n",
      "77\t78496\t0.2\t1\t73454 5042\n",
      "78\t73891\t0.2\t1\t68991 4900\n",
      "79\t72359\t0.2\t1\t67790 4569\n",
      "80\t70385\t0.2\t1\t65644 4741\n",
      "81\t70687\t0.2\t1\t66072 4615\n",
      "82\t67765\t0.2\t1\t63360 4405\n",
      "83\t64119\t0.2\t1\t59858 4261\n",
      "84\t61128\t0.2\t1\t56996 4132\n",
      "85\t58834\t0.2\t1\t54971 3863\n",
      "86\t56645\t0.2\t1\t52920 3725\n",
      "87\t53397\t0.2\t1\t50007 3390\n",
      "88\t52020\t0.2\t1\t48563 3457\n",
      "89\t48843\t0.2\t1\t45677 3166\n",
      "90\t48488\t0.2\t1\t45284 3204\n",
      "91\t45775\t0.2\t1\t42701 3074\n",
      "92\t42633\t0.2\t1\t39741 2892\n",
      "93\t39061\t0.2\t1\t36500 2561\n",
      "94\t37084\t0.2\t1\t34707 2377\n",
      "95\t34306\t0.2\t1\t31956 2350\n",
      "96\t31990\t0.2\t1\t29804 2186\n",
      "97\t29916\t0.2\t1\t28016 1900\n",
      "98\t27386\t0.2\t1\t25513 1873\n",
      "99\t26105\t0.2\t1\t24176 1929\n",
      "100\t23728\t0.2\t1\t22014 1714\n",
      "101\t21360\t0.2\t1\t19893 1467\n",
      "102\t19677\t0.2\t1\t18296 1381\n",
      "103\t17691\t0.2\t1\t16474 1217\n",
      "104\t16298\t0.2\t1\t15194 1104\n",
      "105\t14499\t0.2\t1\t13498 1001\n",
      "106\t13016\t0.2\t1\t12109 907\n",
      "107\t11299\t0.2\t1\t10529 770\n",
      "108\t10657\t0.2\t1\t9906 751\n",
      "109\t9041\t0.2\t1\t8358 683\n",
      "110\t8101\t0.2\t1\t7484 617\n",
      "111\t7203\t0.2\t1\t6631 572\n",
      "112\t6115\t0.2\t1\t5677 438\n",
      "113\t5247\t0.2\t1\t4830 417\n",
      "114\t4477\t0.2\t1\t4092 385\n",
      "115\t3837\t0.2\t1\t3499 338\n",
      "116\t3178\t0.2\t1\t2885 293\n",
      "117\t2717\t0.2\t1\t2476 241\n",
      "118\t2308\t0.2\t1\t2079 229\n",
      "119\t1913\t0.2\t1\t1734 179\n",
      "120\t1562\t0.2\t1\t1431 131\n",
      "121\t1324\t0.2\t1\t1204 120\n",
      "122\t1075\t0.2\t1\t965 110\n",
      "123\t859\t0.2\t1\t768 91\n",
      "124\t724\t0.2\t1\t640 84\n",
      "125\t591\t0.2\t1\t519 72\n",
      "126\t475\t0.2\t1\t409 66\n",
      "127\t455\t0.2\t1\t404 51\n",
      "128\t340\t0.2\t1\t294 46\n",
      "129\t243\t0.2\t1\t193 50\n",
      "130\t216\t0.2\t1\t182 34\n",
      "131\t170\t0.2\t1\t146 24\n",
      "132\t150\t0.2\t1\t133 17\n",
      "133\t129\t0.2\t1\t114 15\n",
      "134\t222\t0.2\t1\t199 23\n",
      "135\t80\t0.2\t1\t75 5\n",
      "136\t46\t0.2\t1\t42 4\n",
      "137\t44\t0.2\t1\t37 7\n",
      "138\t28\t0.2\t1\t28\n",
      "139\t21\t0.2\t1\t18 3\n",
      "140\t10\t0.2\t1\t6 4\n",
      "141\t12\t0.2\t1\t8 4\n",
      "142\t16\t0.2\t1\t13 3\n",
      "143\t7\t0.2\t1\t3 4\n",
      "144\t10\t0.2\t1\t10\n",
      "145\t6\t0.2\t1\t4 2\n",
      "146\t6\t0.2\t1\t4 2\n",
      "147\t26\t0.2\t1\t24 2\n",
      "148\t13\t0.2\t1\t11 2\n",
      "149\t11\t0.2\t1\t11\n",
      "150\t141\t0.2\t1\t126 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "14607942 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/8C_32psu_4_S8_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 628.60 s (43 us/read; 1.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              14,607,942\n",
      "Reads with adapters:                10,984,887 (75.2%)\n",
      "Reads written (passing filters):    14,607,942 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,191,191,300 bp\n",
      "Quality-trimmed:               8,040,292 bp (0.4%)\n",
      "Total written (filtered):  1,774,637,805 bp (81.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10984887 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 11.2%\n",
      "  G: 23.3%\n",
      "  T: 36.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2154515\t3651985.5\t0\t2154515\n",
      "2\t307769\t912996.4\t0\t307769\n",
      "3\t171355\t228249.1\t0\t171355\n",
      "4\t100114\t57062.3\t0\t100114\n",
      "5\t89281\t14265.6\t0\t89281\n",
      "6\t78941\t3566.4\t0\t78941\n",
      "7\t64482\t891.6\t0\t64482\n",
      "8\t90895\t222.9\t0\t90895\n",
      "9\t71519\t55.7\t0\t71259 260\n",
      "10\t77769\t13.9\t1\t72602 5167\n",
      "11\t81991\t3.5\t1\t75629 6362\n",
      "12\t81945\t0.9\t1\t75457 6488\n",
      "13\t93276\t0.2\t1\t85901 7375\n",
      "14\t86548\t0.2\t1\t79069 7479\n",
      "15\t71575\t0.2\t1\t66062 5513\n",
      "16\t101873\t0.2\t1\t93937 7936\n",
      "17\t86756\t0.2\t1\t79766 6990\n",
      "18\t75151\t0.2\t1\t69487 5664\n",
      "19\t100031\t0.2\t1\t92361 7670\n",
      "20\t92540\t0.2\t1\t85573 6967\n",
      "21\t80563\t0.2\t1\t74553 6010\n",
      "22\t91001\t0.2\t1\t84166 6835\n",
      "23\t96186\t0.2\t1\t88873 7313\n",
      "24\t95097\t0.2\t1\t87831 7266\n",
      "25\t107490\t0.2\t1\t100206 7284\n",
      "26\t89500\t0.2\t1\t82469 7031\n",
      "27\t97464\t0.2\t1\t89433 8031\n",
      "28\t89257\t0.2\t1\t83469 5788\n",
      "29\t97776\t0.2\t1\t90646 7130\n",
      "30\t94678\t0.2\t1\t88320 6358\n",
      "31\t104562\t0.2\t1\t97122 7440\n",
      "32\t90375\t0.2\t1\t84247 6128\n",
      "33\t104255\t0.2\t1\t96395 7860\n",
      "34\t131765\t0.2\t1\t122731 9034\n",
      "35\t71827\t0.2\t1\t67071 4756\n",
      "36\t101000\t0.2\t1\t93594 7406\n",
      "37\t108384\t0.2\t1\t100841 7543\n",
      "38\t100305\t0.2\t1\t93893 6412\n",
      "39\t96265\t0.2\t1\t90338 5927\n",
      "40\t108609\t0.2\t1\t101413 7196\n",
      "41\t105469\t0.2\t1\t98528 6941\n",
      "42\t108491\t0.2\t1\t101465 7026\n",
      "43\t97504\t0.2\t1\t91262 6242\n",
      "44\t114735\t0.2\t1\t107039 7696\n",
      "45\t119691\t0.2\t1\t111821 7870\n",
      "46\t85524\t0.2\t1\t79914 5610\n",
      "47\t107169\t0.2\t1\t100947 6222\n",
      "48\t104472\t0.2\t1\t98077 6395\n",
      "49\t109888\t0.2\t1\t102993 6895\n",
      "50\t104895\t0.2\t1\t98149 6746\n",
      "51\t128024\t0.2\t1\t120005 8019\n",
      "52\t92440\t0.2\t1\t86601 5839\n",
      "53\t114005\t0.2\t1\t107513 6492\n",
      "54\t82023\t0.2\t1\t77215 4808\n",
      "55\t107798\t0.2\t1\t101620 6178\n",
      "56\t110208\t0.2\t1\t103344 6864\n",
      "57\t105022\t0.2\t1\t99470 5552\n",
      "58\t100686\t0.2\t1\t95258 5428\n",
      "59\t100045\t0.2\t1\t94333 5712\n",
      "60\t103616\t0.2\t1\t97757 5859\n",
      "61\t106446\t0.2\t1\t100410 6036\n",
      "62\t115794\t0.2\t1\t108819 6975\n",
      "63\t149726\t0.2\t1\t142046 7680\n",
      "64\t116619\t0.2\t1\t111405 5214\n",
      "65\t80564\t0.2\t1\t76264 4300\n",
      "66\t87049\t0.2\t1\t82185 4864\n",
      "67\t86779\t0.2\t1\t81921 4858\n",
      "68\t87371\t0.2\t1\t82619 4752\n",
      "69\t85219\t0.2\t1\t80357 4862\n",
      "70\t93639\t0.2\t1\t88366 5273\n",
      "71\t91437\t0.2\t1\t86289 5148\n",
      "72\t86910\t0.2\t1\t81943 4967\n",
      "73\t89655\t0.2\t1\t84810 4845\n",
      "74\t82770\t0.2\t1\t78094 4676\n",
      "75\t77878\t0.2\t1\t73550 4328\n",
      "76\t75167\t0.2\t1\t70959 4208\n",
      "77\t74827\t0.2\t1\t70567 4260\n",
      "78\t71263\t0.2\t1\t67257 4006\n",
      "79\t68082\t0.2\t1\t64314 3768\n",
      "80\t66978\t0.2\t1\t63430 3548\n",
      "81\t67480\t0.2\t1\t63757 3723\n",
      "82\t63500\t0.2\t1\t60093 3407\n",
      "83\t60414\t0.2\t1\t57170 3244\n",
      "84\t57646\t0.2\t1\t54576 3070\n",
      "85\t56216\t0.2\t1\t53261 2955\n",
      "86\t55705\t0.2\t1\t52806 2899\n",
      "87\t54555\t0.2\t1\t51737 2818\n",
      "88\t52657\t0.2\t1\t49839 2818\n",
      "89\t47083\t0.2\t1\t44512 2571\n",
      "90\t45017\t0.2\t1\t42539 2478\n",
      "91\t42292\t0.2\t1\t40068 2224\n",
      "92\t39093\t0.2\t1\t37073 2020\n",
      "93\t36824\t0.2\t1\t34979 1845\n",
      "94\t35114\t0.2\t1\t33278 1836\n",
      "95\t33006\t0.2\t1\t31281 1725\n",
      "96\t30878\t0.2\t1\t29246 1632\n",
      "97\t28202\t0.2\t1\t26749 1453\n",
      "98\t25663\t0.2\t1\t24339 1324\n",
      "99\t24079\t0.2\t1\t22771 1308\n",
      "100\t21840\t0.2\t1\t20690 1150\n",
      "101\t19586\t0.2\t1\t18574 1012\n",
      "102\t18375\t0.2\t1\t17447 928\n",
      "103\t16706\t0.2\t1\t15871 835\n",
      "104\t15675\t0.2\t1\t14815 860\n",
      "105\t13756\t0.2\t1\t13047 709\n",
      "106\t12568\t0.2\t1\t11946 622\n",
      "107\t10619\t0.2\t1\t10045 574\n",
      "108\t9793\t0.2\t1\t9277 516\n",
      "109\t8520\t0.2\t1\t8082 438\n",
      "110\t7642\t0.2\t1\t7254 388\n",
      "111\t7055\t0.2\t1\t6693 362\n",
      "112\t5993\t0.2\t1\t5692 301\n",
      "113\t5089\t0.2\t1\t4789 300\n",
      "114\t4335\t0.2\t1\t4087 248\n",
      "115\t3576\t0.2\t1\t3383 193\n",
      "116\t2943\t0.2\t1\t2759 184\n",
      "117\t2497\t0.2\t1\t2342 155\n",
      "118\t2119\t0.2\t1\t1989 130\n",
      "119\t1740\t0.2\t1\t1643 97\n",
      "120\t1465\t0.2\t1\t1354 111\n",
      "121\t1235\t0.2\t1\t1152 83\n",
      "122\t1011\t0.2\t1\t954 57\n",
      "123\t812\t0.2\t1\t749 63\n",
      "124\t706\t0.2\t1\t655 51\n",
      "125\t559\t0.2\t1\t512 47\n",
      "126\t438\t0.2\t1\t415 23\n",
      "127\t439\t0.2\t1\t406 33\n",
      "128\t306\t0.2\t1\t286 20\n",
      "129\t222\t0.2\t1\t208 14\n",
      "130\t202\t0.2\t1\t190 12\n",
      "131\t161\t0.2\t1\t148 13\n",
      "132\t146\t0.2\t1\t132 14\n",
      "133\t137\t0.2\t1\t126 11\n",
      "134\t210\t0.2\t1\t203 7\n",
      "135\t81\t0.2\t1\t72 9\n",
      "136\t43\t0.2\t1\t38 5\n",
      "137\t44\t0.2\t1\t36 8\n",
      "138\t25\t0.2\t1\t22 3\n",
      "139\t23\t0.2\t1\t23\n",
      "140\t5\t0.2\t1\t5\n",
      "141\t9\t0.2\t1\t9\n",
      "142\t12\t0.2\t1\t11 1\n",
      "143\t2\t0.2\t1\t2\n",
      "144\t10\t0.2\t1\t7 3\n",
      "145\t5\t0.2\t1\t5\n",
      "146\t6\t0.2\t1\t6\n",
      "147\t23\t0.2\t1\t22 1\n",
      "148\t13\t0.2\t1\t13\n",
      "149\t11\t0.2\t1\t10 1\n",
      "150\t112\t0.2\t1\t105 7\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "14607942 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "file_1: 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz, file_2: 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 14607942\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 14573 (0.10%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 90% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for 8C_32psu_4_S8_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 8C_32psu_4_S8_L002_R1_001_trimmed.fq.gz and 8C_32psu_4_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 742.18 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,576,202\n",
      "Reads with adapters:                14,608,493 (78.6%)\n",
      "Reads written (passing filters):    18,576,202 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,786,430,300 bp\n",
      "Quality-trimmed:               5,505,335 bp (0.2%)\n",
      "Total written (filtered):  2,183,212,736 bp (78.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14608493 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 14.1%\n",
      "  G: 20.7%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2367687\t4644050.5\t0\t2367687\n",
      "2\t362390\t1161012.6\t0\t362390\n",
      "3\t199560\t290253.2\t0\t199560\n",
      "4\t122411\t72563.3\t0\t122411\n",
      "5\t101824\t18140.8\t0\t101824\n",
      "6\t98694\t4535.2\t0\t98694\n",
      "7\t85930\t1133.8\t0\t85930\n",
      "8\t108616\t283.5\t0\t108616\n",
      "9\t98214\t70.9\t0\t96968 1246\n",
      "10\t98262\t17.7\t1\t93000 5262\n",
      "11\t106716\t4.4\t1\t98716 8000\n",
      "12\t102761\t1.1\t1\t96030 6731\n",
      "13\t106394\t0.3\t1\t99361 7033\n",
      "14\t105192\t0.3\t1\t97880 7312\n",
      "15\t111384\t0.3\t1\t104056 7328\n",
      "16\t109489\t0.3\t1\t101402 8087\n",
      "17\t132410\t0.3\t1\t121978 10432\n",
      "18\t113576\t0.3\t1\t106813 6763\n",
      "19\t98216\t0.3\t1\t92113 6103\n",
      "20\t116366\t0.3\t1\t109085 7281\n",
      "21\t128770\t0.3\t1\t119833 8937\n",
      "22\t109971\t0.3\t1\t103795 6176\n",
      "23\t129883\t0.3\t1\t121543 8340\n",
      "24\t115035\t0.3\t1\t105774 9261\n",
      "25\t127548\t0.3\t1\t119043 8505\n",
      "26\t115805\t0.3\t1\t109020 6785\n",
      "27\t128062\t0.3\t1\t118816 9246\n",
      "28\t133464\t0.3\t1\t123994 9470\n",
      "29\t128355\t0.3\t1\t118623 9732\n",
      "30\t145787\t0.3\t1\t137741 8046\n",
      "31\t107347\t0.3\t1\t100584 6763\n",
      "32\t136432\t0.3\t1\t127704 8728\n",
      "33\t126911\t0.3\t1\t118459 8452\n",
      "34\t150928\t0.3\t1\t140208 10720\n",
      "35\t123580\t0.3\t1\t115931 7649\n",
      "36\t142531\t0.3\t1\t133504 9027\n",
      "37\t133143\t0.3\t1\t125509 7634\n",
      "38\t143413\t0.3\t1\t133925 9488\n",
      "39\t142038\t0.3\t1\t132863 9175\n",
      "40\t145926\t0.3\t1\t136263 9663\n",
      "41\t151709\t0.3\t1\t139050 12659\n",
      "42\t137644\t0.3\t1\t127697 9947\n",
      "43\t208996\t0.3\t1\t198310 10686\n",
      "44\t73738\t0.3\t1\t67589 6149\n",
      "45\t147037\t0.3\t1\t138219 8818\n",
      "46\t140744\t0.3\t1\t131869 8875\n",
      "47\t152223\t0.3\t1\t143116 9107\n",
      "48\t139803\t0.3\t1\t130633 9170\n",
      "49\t156282\t0.3\t1\t145534 10748\n",
      "50\t133834\t0.3\t1\t126283 7551\n",
      "51\t143388\t0.3\t1\t134512 8876\n",
      "52\t151003\t0.3\t1\t140213 10790\n",
      "53\t166892\t0.3\t1\t157230 9662\n",
      "54\t123800\t0.3\t1\t116115 7685\n",
      "55\t151573\t0.3\t1\t142303 9270\n",
      "56\t178039\t0.3\t1\t168271 9768\n",
      "57\t131467\t0.3\t1\t122895 8572\n",
      "58\t147115\t0.3\t1\t139737 7378\n",
      "59\t118925\t0.3\t1\t111394 7531\n",
      "60\t151799\t0.3\t1\t142575 9224\n",
      "61\t144120\t0.3\t1\t136068 8052\n",
      "62\t147428\t0.3\t1\t139233 8195\n",
      "63\t128075\t0.3\t1\t119968 8107\n",
      "64\t135764\t0.3\t1\t128209 7555\n",
      "65\t146614\t0.3\t1\t138085 8529\n",
      "66\t155870\t0.3\t1\t146365 9505\n",
      "67\t168571\t0.3\t1\t156591 11980\n",
      "68\t310336\t0.3\t1\t295745 14591\n",
      "69\t61543\t0.3\t1\t56640 4903\n",
      "70\t21841\t0.3\t1\t18433 3408\n",
      "71\t77515\t0.3\t1\t72834 4681\n",
      "72\t115950\t0.3\t1\t109389 6561\n",
      "73\t122667\t0.3\t1\t115604 7063\n",
      "74\t120840\t0.3\t1\t113936 6904\n",
      "75\t118343\t0.3\t1\t111514 6829\n",
      "76\t117750\t0.3\t1\t110989 6761\n",
      "77\t117217\t0.3\t1\t110486 6731\n",
      "78\t109143\t0.3\t1\t102786 6357\n",
      "79\t107883\t0.3\t1\t101668 6215\n",
      "80\t105101\t0.3\t1\t99015 6086\n",
      "81\t104048\t0.3\t1\t98069 5979\n",
      "82\t100791\t0.3\t1\t95040 5751\n",
      "83\t96002\t0.3\t1\t90436 5566\n",
      "84\t92813\t0.3\t1\t87473 5340\n",
      "85\t89718\t0.3\t1\t84646 5072\n",
      "86\t86520\t0.3\t1\t81561 4959\n",
      "87\t81906\t0.3\t1\t77243 4663\n",
      "88\t81347\t0.3\t1\t76501 4846\n",
      "89\t76428\t0.3\t1\t72015 4413\n",
      "90\t76790\t0.3\t1\t72233 4557\n",
      "91\t72129\t0.3\t1\t67780 4349\n",
      "92\t67716\t0.3\t1\t63638 4078\n",
      "93\t63437\t0.3\t1\t59587 3850\n",
      "94\t61274\t0.3\t1\t57720 3554\n",
      "95\t56931\t0.3\t1\t53504 3427\n",
      "96\t53326\t0.3\t1\t50061 3265\n",
      "97\t50390\t0.3\t1\t47429 2961\n",
      "98\t46074\t0.3\t1\t43201 2873\n",
      "99\t45190\t0.3\t1\t42231 2959\n",
      "100\t40830\t0.3\t1\t38265 2565\n",
      "101\t37913\t0.3\t1\t35541 2372\n",
      "102\t35110\t0.3\t1\t32897 2213\n",
      "103\t31900\t0.3\t1\t29985 1915\n",
      "104\t30069\t0.3\t1\t28092 1977\n",
      "105\t26590\t0.3\t1\t24861 1729\n",
      "106\t23847\t0.3\t1\t22283 1564\n",
      "107\t21922\t0.3\t1\t20517 1405\n",
      "108\t20575\t0.3\t1\t19261 1314\n",
      "109\t18220\t0.3\t1\t17014 1206\n",
      "110\t16359\t0.3\t1\t15225 1134\n",
      "111\t14698\t0.3\t1\t13709 989\n",
      "112\t12826\t0.3\t1\t11919 907\n",
      "113\t11006\t0.3\t1\t10256 750\n",
      "114\t9719\t0.3\t1\t9039 680\n",
      "115\t8407\t0.3\t1\t7839 568\n",
      "116\t7289\t0.3\t1\t6725 564\n",
      "117\t6390\t0.3\t1\t5903 487\n",
      "118\t5317\t0.3\t1\t4911 406\n",
      "119\t4366\t0.3\t1\t4006 360\n",
      "120\t3632\t0.3\t1\t3354 278\n",
      "121\t3334\t0.3\t1\t3055 279\n",
      "122\t2775\t0.3\t1\t2532 243\n",
      "123\t2262\t0.3\t1\t2067 195\n",
      "124\t1853\t0.3\t1\t1718 135\n",
      "125\t1573\t0.3\t1\t1418 155\n",
      "126\t1249\t0.3\t1\t1114 135\n",
      "127\t1068\t0.3\t1\t970 98\n",
      "128\t811\t0.3\t1\t720 91\n",
      "129\t753\t0.3\t1\t679 74\n",
      "130\t521\t0.3\t1\t476 45\n",
      "131\t504\t0.3\t1\t454 50\n",
      "132\t357\t0.3\t1\t326 31\n",
      "133\t348\t0.3\t1\t306 42\n",
      "134\t413\t0.3\t1\t388 25\n",
      "135\t242\t0.3\t1\t222 20\n",
      "136\t135\t0.3\t1\t122 13\n",
      "137\t135\t0.3\t1\t121 14\n",
      "138\t66\t0.3\t1\t62 4\n",
      "139\t41\t0.3\t1\t36 5\n",
      "140\t32\t0.3\t1\t27 5\n",
      "141\t40\t0.3\t1\t34 6\n",
      "142\t26\t0.3\t1\t24 2\n",
      "143\t30\t0.3\t1\t25 5\n",
      "144\t18\t0.3\t1\t15 3\n",
      "145\t14\t0.3\t1\t12 2\n",
      "146\t21\t0.3\t1\t19 2\n",
      "147\t69\t0.3\t1\t65 4\n",
      "148\t44\t0.3\t1\t40 4\n",
      "149\t37\t0.3\t1\t35 2\n",
      "150\t304\t0.3\t1\t264 40\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "18576202 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 781.98 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,576,202\n",
      "Reads with adapters:                14,527,088 (78.2%)\n",
      "Reads written (passing filters):    18,576,202 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,786,430,300 bp\n",
      "Quality-trimmed:              12,297,720 bp (0.4%)\n",
      "Total written (filtered):  2,186,026,388 bp (78.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14527088 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 12.4%\n",
      "  G: 24.5%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2397075\t4644050.5\t0\t2397075\n",
      "2\t377772\t1161012.6\t0\t377772\n",
      "3\t209931\t290253.2\t0\t209931\n",
      "4\t123717\t72563.3\t0\t123717\n",
      "5\t112624\t18140.8\t0\t112624\n",
      "6\t97477\t4535.2\t0\t97477\n",
      "7\t81404\t1133.8\t0\t81404\n",
      "8\t114796\t283.5\t0\t114796\n",
      "9\t89736\t70.9\t0\t89386 350\n",
      "10\t100021\t17.7\t1\t95112 4909\n",
      "11\t103726\t4.4\t1\t97357 6369\n",
      "12\t104603\t1.1\t1\t97938 6665\n",
      "13\t116572\t0.3\t1\t109738 6834\n",
      "14\t110440\t0.3\t1\t102780 7660\n",
      "15\t91038\t0.3\t1\t85192 5846\n",
      "16\t129548\t0.3\t1\t121431 8117\n",
      "17\t105093\t0.3\t1\t97969 7124\n",
      "18\t99076\t0.3\t1\t93067 6009\n",
      "19\t126890\t0.3\t1\t118635 8255\n",
      "20\t118457\t0.3\t1\t111491 6966\n",
      "21\t103736\t0.3\t1\t97365 6371\n",
      "22\t118151\t0.3\t1\t110881 7270\n",
      "23\t121946\t0.3\t1\t114464 7482\n",
      "24\t121962\t0.3\t1\t114186 7776\n",
      "25\t136854\t0.3\t1\t128708 8146\n",
      "26\t113321\t0.3\t1\t106068 7253\n",
      "27\t126381\t0.3\t1\t117798 8583\n",
      "28\t118846\t0.3\t1\t112727 6119\n",
      "29\t125505\t0.3\t1\t117793 7712\n",
      "30\t125215\t0.3\t1\t118459 6756\n",
      "31\t135187\t0.3\t1\t127291 7896\n",
      "32\t121491\t0.3\t1\t114900 6591\n",
      "33\t135254\t0.3\t1\t127041 8213\n",
      "34\t169655\t0.3\t1\t159717 9938\n",
      "35\t101342\t0.3\t1\t95776 5566\n",
      "36\t136505\t0.3\t1\t128363 8142\n",
      "37\t143293\t0.3\t1\t134875 8418\n",
      "38\t134051\t0.3\t1\t127357 6694\n",
      "39\t131563\t0.3\t1\t124738 6825\n",
      "40\t145300\t0.3\t1\t136735 8565\n",
      "41\t138947\t0.3\t1\t131652 7295\n",
      "42\t144334\t0.3\t1\t136660 7674\n",
      "43\t131871\t0.3\t1\t124782 7089\n",
      "44\t149402\t0.3\t1\t141199 8203\n",
      "45\t161283\t0.3\t1\t152880 8403\n",
      "46\t115740\t0.3\t1\t109551 6189\n",
      "47\t146650\t0.3\t1\t139277 7373\n",
      "48\t144979\t0.3\t1\t137763 7216\n",
      "49\t149370\t0.3\t1\t141741 7629\n",
      "50\t140556\t0.3\t1\t132993 7563\n",
      "51\t176763\t0.3\t1\t167628 9135\n",
      "52\t125327\t0.3\t1\t119270 6057\n",
      "53\t150468\t0.3\t1\t143751 6717\n",
      "54\t114488\t0.3\t1\t108938 5550\n",
      "55\t150264\t0.3\t1\t143436 6828\n",
      "56\t154642\t0.3\t1\t147075 7567\n",
      "57\t145698\t0.3\t1\t138895 6803\n",
      "58\t139003\t0.3\t1\t132772 6231\n",
      "59\t141639\t0.3\t1\t135020 6619\n",
      "60\t145216\t0.3\t1\t138147 7069\n",
      "61\t149173\t0.3\t1\t141978 7195\n",
      "62\t161580\t0.3\t1\t153777 7803\n",
      "63\t211777\t0.3\t1\t202721 9056\n",
      "64\t166013\t0.3\t1\t159545 6468\n",
      "65\t115072\t0.3\t1\t109927 5145\n",
      "66\t124032\t0.3\t1\t118392 5640\n",
      "67\t121997\t0.3\t1\t116256 5741\n",
      "68\t123305\t0.3\t1\t117689 5616\n",
      "69\t119979\t0.3\t1\t114424 5555\n",
      "70\t126522\t0.3\t1\t120645 5877\n",
      "71\t131428\t0.3\t1\t125503 5925\n",
      "72\t124583\t0.3\t1\t118750 5833\n",
      "73\t129753\t0.3\t1\t124066 5687\n",
      "74\t119016\t0.3\t1\t113726 5290\n",
      "75\t113191\t0.3\t1\t108419 4772\n",
      "76\t111393\t0.3\t1\t106547 4846\n",
      "77\t111470\t0.3\t1\t106513 4957\n",
      "78\t104799\t0.3\t1\t100049 4750\n",
      "79\t100883\t0.3\t1\t96421 4462\n",
      "80\t100236\t0.3\t1\t95706 4530\n",
      "81\t98928\t0.3\t1\t94651 4277\n",
      "82\t94279\t0.3\t1\t90155 4124\n",
      "83\t90115\t0.3\t1\t86173 3942\n",
      "84\t87249\t0.3\t1\t83373 3876\n",
      "85\t85689\t0.3\t1\t81793 3896\n",
      "86\t84888\t0.3\t1\t81207 3681\n",
      "87\t84171\t0.3\t1\t80558 3613\n",
      "88\t82226\t0.3\t1\t78630 3596\n",
      "89\t73279\t0.3\t1\t70084 3195\n",
      "90\t71368\t0.3\t1\t68189 3179\n",
      "91\t66468\t0.3\t1\t63502 2966\n",
      "92\t61869\t0.3\t1\t59204 2665\n",
      "93\t59677\t0.3\t1\t57086 2591\n",
      "94\t57840\t0.3\t1\t55318 2522\n",
      "95\t54875\t0.3\t1\t52595 2280\n",
      "96\t51698\t0.3\t1\t49449 2249\n",
      "97\t47519\t0.3\t1\t45519 2000\n",
      "98\t43527\t0.3\t1\t41676 1851\n",
      "99\t42094\t0.3\t1\t40263 1831\n",
      "100\t37940\t0.3\t1\t36384 1556\n",
      "101\t34646\t0.3\t1\t33175 1471\n",
      "102\t32213\t0.3\t1\t30921 1292\n",
      "103\t29202\t0.3\t1\t27960 1242\n",
      "104\t28501\t0.3\t1\t27313 1188\n",
      "105\t25308\t0.3\t1\t24224 1084\n",
      "106\t22773\t0.3\t1\t21858 915\n",
      "107\t20894\t0.3\t1\t20051 843\n",
      "108\t18912\t0.3\t1\t18081 831\n",
      "109\t17072\t0.3\t1\t16336 736\n",
      "110\t15374\t0.3\t1\t14700 674\n",
      "111\t14518\t0.3\t1\t13895 623\n",
      "112\t12456\t0.3\t1\t11891 565\n",
      "113\t10771\t0.3\t1\t10329 442\n",
      "114\t9374\t0.3\t1\t8976 398\n",
      "115\t7920\t0.3\t1\t7555 365\n",
      "116\t6787\t0.3\t1\t6458 329\n",
      "117\t5803\t0.3\t1\t5538 265\n",
      "118\t4857\t0.3\t1\t4636 221\n",
      "119\t4060\t0.3\t1\t3861 199\n",
      "120\t3400\t0.3\t1\t3228 172\n",
      "121\t3132\t0.3\t1\t2976 156\n",
      "122\t2609\t0.3\t1\t2493 116\n",
      "123\t2165\t0.3\t1\t2072 93\n",
      "124\t1770\t0.3\t1\t1676 94\n",
      "125\t1516\t0.3\t1\t1432 84\n",
      "126\t1213\t0.3\t1\t1153 60\n",
      "127\t1001\t0.3\t1\t941 60\n",
      "128\t776\t0.3\t1\t728 48\n",
      "129\t744\t0.3\t1\t699 45\n",
      "130\t486\t0.3\t1\t463 23\n",
      "131\t460\t0.3\t1\t438 22\n",
      "132\t343\t0.3\t1\t315 28\n",
      "133\t355\t0.3\t1\t338 17\n",
      "134\t428\t0.3\t1\t403 25\n",
      "135\t226\t0.3\t1\t216 10\n",
      "136\t133\t0.3\t1\t120 13\n",
      "137\t125\t0.3\t1\t117 8\n",
      "138\t70\t0.3\t1\t62 8\n",
      "139\t38\t0.3\t1\t35 3\n",
      "140\t38\t0.3\t1\t31 7\n",
      "141\t35\t0.3\t1\t33 2\n",
      "142\t27\t0.3\t1\t26 1\n",
      "143\t32\t0.3\t1\t30 2\n",
      "144\t14\t0.3\t1\t13 1\n",
      "145\t17\t0.3\t1\t16 1\n",
      "146\t23\t0.3\t1\t23\n",
      "147\t69\t0.3\t1\t65 4\n",
      "148\t40\t0.3\t1\t38 2\n",
      "149\t38\t0.3\t1\t34 4\n",
      "150\t229\t0.3\t1\t219 10\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "18576202 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz, file_2: CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18576202\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 29180 (0.16%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_16C_26psu_1_S19_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 714.53 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,490,322\n",
      "Reads with adapters:                14,542,227 (78.6%)\n",
      "Reads written (passing filters):    18,490,322 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,773,548,300 bp\n",
      "Quality-trimmed:               5,682,539 bp (0.2%)\n",
      "Total written (filtered):  2,170,540,982 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14542227 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.1%\n",
      "  G: 20.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2352565\t4622580.5\t0\t2352565\n",
      "2\t355744\t1155645.1\t0\t355744\n",
      "3\t197435\t288911.3\t0\t197435\n",
      "4\t119803\t72227.8\t0\t119803\n",
      "5\t101491\t18057.0\t0\t101491\n",
      "6\t98683\t4514.2\t0\t98683\n",
      "7\t82227\t1128.6\t0\t82227\n",
      "8\t107087\t282.1\t0\t107087\n",
      "9\t96108\t70.5\t0\t94984 1124\n",
      "10\t95916\t17.6\t1\t89656 6260\n",
      "11\t105518\t4.4\t1\t96331 9187\n",
      "12\t101326\t1.1\t1\t93731 7595\n",
      "13\t105529\t0.3\t1\t97295 8234\n",
      "14\t104207\t0.3\t1\t95707 8500\n",
      "15\t110166\t0.3\t1\t101496 8670\n",
      "16\t108369\t0.3\t1\t98900 9469\n",
      "17\t132137\t0.3\t1\t119869 12268\n",
      "18\t111785\t0.3\t1\t103776 8009\n",
      "19\t94978\t0.3\t1\t88024 6954\n",
      "20\t115852\t0.3\t1\t107171 8681\n",
      "21\t128602\t0.3\t1\t118248 10354\n",
      "22\t106928\t0.3\t1\t99662 7266\n",
      "23\t130246\t0.3\t1\t120393 9853\n",
      "24\t113598\t0.3\t1\t103348 10250\n",
      "25\t127529\t0.3\t1\t117557 9972\n",
      "26\t114329\t0.3\t1\t106365 7964\n",
      "27\t127671\t0.3\t1\t117117 10554\n",
      "28\t133545\t0.3\t1\t122886 10659\n",
      "29\t127218\t0.3\t1\t116368 10850\n",
      "30\t145807\t0.3\t1\t136218 9589\n",
      "31\t104710\t0.3\t1\t97327 7383\n",
      "32\t136712\t0.3\t1\t126029 10683\n",
      "33\t125051\t0.3\t1\t114983 10068\n",
      "34\t151172\t0.3\t1\t138255 12917\n",
      "35\t121506\t0.3\t1\t112294 9212\n",
      "36\t142450\t0.3\t1\t131732 10718\n",
      "37\t131704\t0.3\t1\t122359 9345\n",
      "38\t144055\t0.3\t1\t132700 11355\n",
      "39\t140259\t0.3\t1\t129402 10857\n",
      "40\t145119\t0.3\t1\t132998 12121\n",
      "41\t151317\t0.3\t1\t136837 14480\n",
      "42\t135755\t0.3\t1\t124185 11570\n",
      "43\t206150\t0.3\t1\t193096 13054\n",
      "44\t75981\t0.3\t1\t68776 7205\n",
      "45\t147101\t0.3\t1\t136563 10538\n",
      "46\t140010\t0.3\t1\t129569 10441\n",
      "47\t150202\t0.3\t1\t139726 10476\n",
      "48\t137985\t0.3\t1\t127406 10579\n",
      "49\t157361\t0.3\t1\t144848 12513\n",
      "50\t132126\t0.3\t1\t123324 8802\n",
      "51\t143254\t0.3\t1\t133169 10085\n",
      "52\t152534\t0.3\t1\t140225 12309\n",
      "53\t166653\t0.3\t1\t155543 11110\n",
      "54\t122896\t0.3\t1\t113877 9019\n",
      "55\t150505\t0.3\t1\t139851 10654\n",
      "56\t178712\t0.3\t1\t166946 11766\n",
      "57\t130999\t0.3\t1\t120759 10240\n",
      "58\t145047\t0.3\t1\t136344 8703\n",
      "59\t119684\t0.3\t1\t110878 8806\n",
      "60\t151206\t0.3\t1\t140682 10524\n",
      "61\t142715\t0.3\t1\t133251 9464\n",
      "62\t148943\t0.3\t1\t139114 9829\n",
      "63\t129912\t0.3\t1\t120416 9496\n",
      "64\t134038\t0.3\t1\t125474 8564\n",
      "65\t146595\t0.3\t1\t136655 9940\n",
      "66\t153978\t0.3\t1\t143441 10537\n",
      "67\t168100\t0.3\t1\t155171 12929\n",
      "68\t298170\t0.3\t1\t281868 16302\n",
      "69\t74393\t0.3\t1\t68405 5988\n",
      "70\t26909\t0.3\t1\t22880 4029\n",
      "71\t75109\t0.3\t1\t70102 5007\n",
      "72\t114931\t0.3\t1\t107946 6985\n",
      "73\t122734\t0.3\t1\t114818 7916\n",
      "74\t120820\t0.3\t1\t112962 7858\n",
      "75\t119099\t0.3\t1\t111010 8089\n",
      "76\t117029\t0.3\t1\t108855 8174\n",
      "77\t117800\t0.3\t1\t109726 8074\n",
      "78\t109388\t0.3\t1\t101815 7573\n",
      "79\t108107\t0.3\t1\t100589 7518\n",
      "80\t105715\t0.3\t1\t98229 7486\n",
      "81\t104556\t0.3\t1\t97305 7251\n",
      "82\t101011\t0.3\t1\t93996 7015\n",
      "83\t97159\t0.3\t1\t90577 6582\n",
      "84\t92283\t0.3\t1\t85890 6393\n",
      "85\t90035\t0.3\t1\t83799 6236\n",
      "86\t86319\t0.3\t1\t80378 5941\n",
      "87\t82376\t0.3\t1\t76893 5483\n",
      "88\t81375\t0.3\t1\t75753 5622\n",
      "89\t77211\t0.3\t1\t71720 5491\n",
      "90\t76508\t0.3\t1\t71173 5335\n",
      "91\t72845\t0.3\t1\t67788 5057\n",
      "92\t67776\t0.3\t1\t63180 4596\n",
      "93\t63675\t0.3\t1\t59371 4304\n",
      "94\t61289\t0.3\t1\t57047 4242\n",
      "95\t57075\t0.3\t1\t53036 4039\n",
      "96\t54087\t0.3\t1\t50327 3760\n",
      "97\t51019\t0.3\t1\t47507 3512\n",
      "98\t46423\t0.3\t1\t43108 3315\n",
      "99\t45164\t0.3\t1\t41742 3422\n",
      "100\t40910\t0.3\t1\t38024 2886\n",
      "101\t38601\t0.3\t1\t35925 2676\n",
      "102\t35185\t0.3\t1\t32582 2603\n",
      "103\t32331\t0.3\t1\t30034 2297\n",
      "104\t30306\t0.3\t1\t28179 2127\n",
      "105\t26497\t0.3\t1\t24651 1846\n",
      "106\t24125\t0.3\t1\t22458 1667\n",
      "107\t22000\t0.3\t1\t20417 1583\n",
      "108\t20719\t0.3\t1\t19189 1530\n",
      "109\t18051\t0.3\t1\t16712 1339\n",
      "110\t16439\t0.3\t1\t15194 1245\n",
      "111\t14466\t0.3\t1\t13407 1059\n",
      "112\t12793\t0.3\t1\t11768 1025\n",
      "113\t11344\t0.3\t1\t10481 863\n",
      "114\t9637\t0.3\t1\t8889 748\n",
      "115\t8596\t0.3\t1\t7840 756\n",
      "116\t7497\t0.3\t1\t6926 571\n",
      "117\t6216\t0.3\t1\t5692 524\n",
      "118\t5413\t0.3\t1\t4943 470\n",
      "119\t4488\t0.3\t1\t4121 367\n",
      "120\t3934\t0.3\t1\t3598 336\n",
      "121\t3277\t0.3\t1\t2991 286\n",
      "122\t2701\t0.3\t1\t2461 240\n",
      "123\t2240\t0.3\t1\t2028 212\n",
      "124\t1944\t0.3\t1\t1730 214\n",
      "125\t1578\t0.3\t1\t1422 156\n",
      "126\t1320\t0.3\t1\t1199 121\n",
      "127\t1172\t0.3\t1\t1056 116\n",
      "128\t861\t0.3\t1\t773 88\n",
      "129\t759\t0.3\t1\t683 76\n",
      "130\t595\t0.3\t1\t520 75\n",
      "131\t487\t0.3\t1\t426 61\n",
      "132\t384\t0.3\t1\t352 32\n",
      "133\t353\t0.3\t1\t318 35\n",
      "134\t434\t0.3\t1\t399 35\n",
      "135\t273\t0.3\t1\t245 28\n",
      "136\t156\t0.3\t1\t137 19\n",
      "137\t94\t0.3\t1\t83 11\n",
      "138\t77\t0.3\t1\t69 8\n",
      "139\t51\t0.3\t1\t44 7\n",
      "140\t30\t0.3\t1\t25 5\n",
      "141\t33\t0.3\t1\t24 9\n",
      "142\t30\t0.3\t1\t24 6\n",
      "143\t35\t0.3\t1\t27 8\n",
      "144\t18\t0.3\t1\t17 1\n",
      "145\t11\t0.3\t1\t9 2\n",
      "146\t25\t0.3\t1\t21 4\n",
      "147\t75\t0.3\t1\t73 2\n",
      "148\t42\t0.3\t1\t37 5\n",
      "149\t44\t0.3\t1\t41 3\n",
      "150\t294\t0.3\t1\t257 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "18490322 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 758.59 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              18,490,322\n",
      "Reads with adapters:                14,450,091 (78.1%)\n",
      "Reads written (passing filters):    18,490,322 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,773,548,300 bp\n",
      "Quality-trimmed:              11,878,254 bp (0.4%)\n",
      "Total written (filtered):  2,174,052,769 bp (78.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14450091 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 12.4%\n",
      "  G: 24.6%\n",
      "  T: 35.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2377683\t4622580.5\t0\t2377683\n",
      "2\t369953\t1155645.1\t0\t369953\n",
      "3\t212079\t288911.3\t0\t212079\n",
      "4\t120512\t72227.8\t0\t120512\n",
      "5\t108962\t18057.0\t0\t108962\n",
      "6\t97421\t4514.2\t0\t97421\n",
      "7\t79386\t1128.6\t0\t79386\n",
      "8\t113054\t282.1\t0\t113054\n",
      "9\t87971\t70.5\t0\t87624 347\n",
      "10\t96473\t17.6\t1\t89879 6594\n",
      "11\t101734\t4.4\t1\t93536 8198\n",
      "12\t102089\t1.1\t1\t93747 8342\n",
      "13\t117081\t0.3\t1\t107549 9532\n",
      "14\t108685\t0.3\t1\t98929 9756\n",
      "15\t89117\t0.3\t1\t82133 6984\n",
      "16\t127440\t0.3\t1\t117058 10382\n",
      "17\t108227\t0.3\t1\t99362 8865\n",
      "18\t94672\t0.3\t1\t87210 7462\n",
      "19\t124816\t0.3\t1\t114994 9822\n",
      "20\t116877\t0.3\t1\t108056 8821\n",
      "21\t101808\t0.3\t1\t94028 7780\n",
      "22\t115329\t0.3\t1\t106403 8926\n",
      "23\t122296\t0.3\t1\t112903 9393\n",
      "24\t121720\t0.3\t1\t111954 9766\n",
      "25\t136356\t0.3\t1\t126910 9446\n",
      "26\t113979\t0.3\t1\t104759 9220\n",
      "27\t127054\t0.3\t1\t116622 10432\n",
      "28\t113808\t0.3\t1\t106074 7734\n",
      "29\t126916\t0.3\t1\t117431 9485\n",
      "30\t122281\t0.3\t1\t113729 8552\n",
      "31\t135760\t0.3\t1\t126137 9623\n",
      "32\t117176\t0.3\t1\t109066 8110\n",
      "33\t136110\t0.3\t1\t125744 10366\n",
      "34\t171537\t0.3\t1\t159519 12018\n",
      "35\t95945\t0.3\t1\t89044 6901\n",
      "36\t137046\t0.3\t1\t126917 10129\n",
      "37\t143037\t0.3\t1\t132698 10339\n",
      "38\t133329\t0.3\t1\t124497 8832\n",
      "39\t127534\t0.3\t1\t119434 8100\n",
      "40\t144687\t0.3\t1\t134651 10036\n",
      "41\t140384\t0.3\t1\t131070 9314\n",
      "42\t144299\t0.3\t1\t134780 9519\n",
      "43\t129916\t0.3\t1\t121294 8622\n",
      "44\t151641\t0.3\t1\t141122 10519\n",
      "45\t160400\t0.3\t1\t149699 10701\n",
      "46\t114273\t0.3\t1\t106605 7668\n",
      "47\t144997\t0.3\t1\t136130 8867\n",
      "48\t143166\t0.3\t1\t134124 9042\n",
      "49\t148482\t0.3\t1\t138822 9660\n",
      "50\t140955\t0.3\t1\t131828 9127\n",
      "51\t174978\t0.3\t1\t163816 11162\n",
      "52\t126038\t0.3\t1\t118003 8035\n",
      "53\t153652\t0.3\t1\t144809 8843\n",
      "54\t111499\t0.3\t1\t104710 6789\n",
      "55\t148731\t0.3\t1\t140120 8611\n",
      "56\t152852\t0.3\t1\t143418 9434\n",
      "57\t145621\t0.3\t1\t137708 7913\n",
      "58\t139629\t0.3\t1\t131703 7926\n",
      "59\t139468\t0.3\t1\t131381 8087\n",
      "60\t145979\t0.3\t1\t137374 8605\n",
      "61\t147098\t0.3\t1\t138678 8420\n",
      "62\t160909\t0.3\t1\t151040 9869\n",
      "63\t208484\t0.3\t1\t197875 10609\n",
      "64\t163339\t0.3\t1\t155950 7389\n",
      "65\t115211\t0.3\t1\t108911 6300\n",
      "66\t124628\t0.3\t1\t117681 6947\n",
      "67\t122836\t0.3\t1\t115645 7191\n",
      "68\t124659\t0.3\t1\t117627 7032\n",
      "69\t121178\t0.3\t1\t114114 7064\n",
      "70\t126813\t0.3\t1\t119514 7299\n",
      "71\t132039\t0.3\t1\t124540 7499\n",
      "72\t125242\t0.3\t1\t118005 7237\n",
      "73\t130262\t0.3\t1\t123006 7256\n",
      "74\t119815\t0.3\t1\t112862 6953\n",
      "75\t114051\t0.3\t1\t107533 6518\n",
      "76\t111158\t0.3\t1\t104747 6411\n",
      "77\t111769\t0.3\t1\t105274 6495\n",
      "78\t104990\t0.3\t1\t98882 6108\n",
      "79\t101071\t0.3\t1\t95399 5672\n",
      "80\t100953\t0.3\t1\t95281 5672\n",
      "81\t99176\t0.3\t1\t93590 5586\n",
      "82\t94350\t0.3\t1\t89142 5208\n",
      "83\t91413\t0.3\t1\t86366 5047\n",
      "84\t87258\t0.3\t1\t82554 4704\n",
      "85\t85916\t0.3\t1\t81265 4651\n",
      "86\t84604\t0.3\t1\t80107 4497\n",
      "87\t84396\t0.3\t1\t79885 4511\n",
      "88\t82425\t0.3\t1\t78207 4218\n",
      "89\t73764\t0.3\t1\t69903 3861\n",
      "90\t71339\t0.3\t1\t67438 3901\n",
      "91\t67256\t0.3\t1\t63682 3574\n",
      "92\t62204\t0.3\t1\t58897 3307\n",
      "93\t60201\t0.3\t1\t57003 3198\n",
      "94\t58001\t0.3\t1\t54921 3080\n",
      "95\t54924\t0.3\t1\t52030 2894\n",
      "96\t52451\t0.3\t1\t49666 2785\n",
      "97\t48389\t0.3\t1\t45913 2476\n",
      "98\t44117\t0.3\t1\t41843 2274\n",
      "99\t42086\t0.3\t1\t39795 2291\n",
      "100\t38235\t0.3\t1\t36240 1995\n",
      "101\t35522\t0.3\t1\t33621 1901\n",
      "102\t32226\t0.3\t1\t30512 1714\n",
      "103\t29643\t0.3\t1\t27969 1674\n",
      "104\t28709\t0.3\t1\t27141 1568\n",
      "105\t25222\t0.3\t1\t23921 1301\n",
      "106\t23132\t0.3\t1\t21958 1174\n",
      "107\t20950\t0.3\t1\t19852 1098\n",
      "108\t19130\t0.3\t1\t18127 1003\n",
      "109\t16951\t0.3\t1\t16010 941\n",
      "110\t15532\t0.3\t1\t14697 835\n",
      "111\t14263\t0.3\t1\t13491 772\n",
      "112\t12448\t0.3\t1\t11737 711\n",
      "113\t11105\t0.3\t1\t10532 573\n",
      "114\t9378\t0.3\t1\t8878 500\n",
      "115\t8004\t0.3\t1\t7541 463\n",
      "116\t7028\t0.3\t1\t6643 385\n",
      "117\t5705\t0.3\t1\t5388 317\n",
      "118\t5009\t0.3\t1\t4724 285\n",
      "119\t4196\t0.3\t1\t3948 248\n",
      "120\t3592\t0.3\t1\t3360 232\n",
      "121\t3098\t0.3\t1\t2900 198\n",
      "122\t2572\t0.3\t1\t2409 163\n",
      "123\t2164\t0.3\t1\t2037 127\n",
      "124\t1827\t0.3\t1\t1705 122\n",
      "125\t1543\t0.3\t1\t1435 108\n",
      "126\t1244\t0.3\t1\t1160 84\n",
      "127\t1099\t0.3\t1\t1018 81\n",
      "128\t829\t0.3\t1\t778 51\n",
      "129\t718\t0.3\t1\t677 41\n",
      "130\t539\t0.3\t1\t503 36\n",
      "131\t460\t0.3\t1\t425 35\n",
      "132\t375\t0.3\t1\t343 32\n",
      "133\t337\t0.3\t1\t318 19\n",
      "134\t432\t0.3\t1\t410 22\n",
      "135\t267\t0.3\t1\t250 17\n",
      "136\t145\t0.3\t1\t136 9\n",
      "137\t88\t0.3\t1\t77 11\n",
      "138\t77\t0.3\t1\t69 8\n",
      "139\t43\t0.3\t1\t40 3\n",
      "140\t29\t0.3\t1\t26 3\n",
      "141\t28\t0.3\t1\t23 5\n",
      "142\t25\t0.3\t1\t23 2\n",
      "143\t30\t0.3\t1\t30\n",
      "144\t16\t0.3\t1\t15 1\n",
      "145\t18\t0.3\t1\t15 3\n",
      "146\t29\t0.3\t1\t26 3\n",
      "147\t70\t0.3\t1\t66 4\n",
      "148\t40\t0.3\t1\t37 3\n",
      "149\t46\t0.3\t1\t43 3\n",
      "150\t248\t0.3\t1\t232 16\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "18490322 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz, file_2: CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 18490322\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 27761 (0.15%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 55% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_1_S19_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_16C_26psu_1_S19_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_1_S19_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 478.70 s (36 us/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,433,110\n",
      "Reads with adapters:                11,700,096 (87.1%)\n",
      "Reads written (passing filters):    13,433,110 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,014,966,500 bp\n",
      "Quality-trimmed:               3,988,642 bp (0.2%)\n",
      "Total written (filtered):  1,426,352,975 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11700096 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 12.5%\n",
      "  G: 23.6%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t952009\t3358277.5\t0\t952009\n",
      "2\t175701\t839569.4\t0\t175701\n",
      "3\t106315\t209892.3\t0\t106315\n",
      "4\t68924\t52473.1\t0\t68924\n",
      "5\t62282\t13118.3\t0\t62282\n",
      "6\t61633\t3279.6\t0\t61633\n",
      "7\t52554\t819.9\t0\t52554\n",
      "8\t65132\t205.0\t0\t65132\n",
      "9\t65306\t51.2\t0\t64521 785\n",
      "10\t61900\t12.8\t1\t58796 3104\n",
      "11\t68989\t3.2\t1\t63880 5109\n",
      "12\t67245\t0.8\t1\t63248 3997\n",
      "13\t69213\t0.2\t1\t64859 4354\n",
      "14\t71768\t0.2\t1\t66846 4922\n",
      "15\t75062\t0.2\t1\t70553 4509\n",
      "16\t74514\t0.2\t1\t69133 5381\n",
      "17\t90491\t0.2\t1\t83695 6796\n",
      "18\t77417\t0.2\t1\t72980 4437\n",
      "19\t68949\t0.2\t1\t64670 4279\n",
      "20\t81015\t0.2\t1\t76032 4983\n",
      "21\t89626\t0.2\t1\t83646 5980\n",
      "22\t77941\t0.2\t1\t73613 4328\n",
      "23\t90688\t0.2\t1\t84959 5729\n",
      "24\t86332\t0.2\t1\t79509 6823\n",
      "25\t91755\t0.2\t1\t85888 5867\n",
      "26\t86085\t0.2\t1\t81297 4788\n",
      "27\t95329\t0.2\t1\t88851 6478\n",
      "28\t95528\t0.2\t1\t89551 5977\n",
      "29\t99349\t0.2\t1\t92034 7315\n",
      "30\t106567\t0.2\t1\t101023 5544\n",
      "31\t85587\t0.2\t1\t80666 4921\n",
      "32\t103697\t0.2\t1\t97691 6006\n",
      "33\t99669\t0.2\t1\t93324 6345\n",
      "34\t119451\t0.2\t1\t111551 7900\n",
      "35\t94360\t0.2\t1\t88955 5405\n",
      "36\t108596\t0.2\t1\t102097 6499\n",
      "37\t113594\t0.2\t1\t106631 6963\n",
      "38\t108790\t0.2\t1\t101383 7407\n",
      "39\t126706\t0.2\t1\t119375 7331\n",
      "40\t103711\t0.2\t1\t97817 5894\n",
      "41\t130432\t0.2\t1\t122153 8279\n",
      "42\t110139\t0.2\t1\t101991 8148\n",
      "43\t183261\t0.2\t1\t174857 8404\n",
      "44\t57830\t0.2\t1\t53254 4576\n",
      "45\t123796\t0.2\t1\t116975 6821\n",
      "46\t119988\t0.2\t1\t113024 6964\n",
      "47\t130141\t0.2\t1\t122976 7165\n",
      "48\t120988\t0.2\t1\t113912 7076\n",
      "49\t138463\t0.2\t1\t129394 9069\n",
      "50\t122247\t0.2\t1\t115770 6477\n",
      "51\t132587\t0.2\t1\t124814 7773\n",
      "52\t139338\t0.2\t1\t129873 9465\n",
      "53\t153954\t0.2\t1\t145822 8132\n",
      "54\t118275\t0.2\t1\t111358 6917\n",
      "55\t139368\t0.2\t1\t131390 7978\n",
      "56\t164199\t0.2\t1\t155690 8509\n",
      "57\t126552\t0.2\t1\t118427 8125\n",
      "58\t144214\t0.2\t1\t137457 6757\n",
      "59\t116278\t0.2\t1\t109533 6745\n",
      "60\t144971\t0.2\t1\t136747 8224\n",
      "61\t139800\t0.2\t1\t132274 7526\n",
      "62\t147932\t0.2\t1\t140286 7646\n",
      "63\t129438\t0.2\t1\t121520 7918\n",
      "64\t136148\t0.2\t1\t129121 7027\n",
      "65\t148136\t0.2\t1\t139981 8155\n",
      "66\t158867\t0.2\t1\t150035 8832\n",
      "67\t172296\t0.2\t1\t160373 11923\n",
      "68\t325502\t0.2\t1\t311428 14074\n",
      "69\t62909\t0.2\t1\t58208 4701\n",
      "70\t23526\t0.2\t1\t20186 3340\n",
      "71\t82740\t0.2\t1\t78158 4582\n",
      "72\t124178\t0.2\t1\t117785 6393\n",
      "73\t132921\t0.2\t1\t125940 6981\n",
      "74\t131071\t0.2\t1\t124071 7000\n",
      "75\t128974\t0.2\t1\t122028 6946\n",
      "76\t128069\t0.2\t1\t121139 6930\n",
      "77\t126522\t0.2\t1\t119822 6700\n",
      "78\t120502\t0.2\t1\t113788 6714\n",
      "79\t118821\t0.2\t1\t112166 6655\n",
      "80\t115866\t0.2\t1\t109547 6319\n",
      "81\t115264\t0.2\t1\t108988 6276\n",
      "82\t110755\t0.2\t1\t104576 6179\n",
      "83\t107566\t0.2\t1\t101669 5897\n",
      "84\t105597\t0.2\t1\t99846 5751\n",
      "85\t102751\t0.2\t1\t97220 5531\n",
      "86\t97615\t0.2\t1\t92219 5396\n",
      "87\t92989\t0.2\t1\t87786 5203\n",
      "88\t92437\t0.2\t1\t87288 5149\n",
      "89\t88099\t0.2\t1\t83258 4841\n",
      "90\t86764\t0.2\t1\t81847 4917\n",
      "91\t81829\t0.2\t1\t77279 4550\n",
      "92\t77590\t0.2\t1\t73265 4325\n",
      "93\t73175\t0.2\t1\t69095 4080\n",
      "94\t69476\t0.2\t1\t65477 3999\n",
      "95\t65431\t0.2\t1\t61622 3809\n",
      "96\t61527\t0.2\t1\t57946 3581\n",
      "97\t57358\t0.2\t1\t54065 3293\n",
      "98\t53633\t0.2\t1\t50498 3135\n",
      "99\t51351\t0.2\t1\t48285 3066\n",
      "100\t46130\t0.2\t1\t43457 2673\n",
      "101\t43563\t0.2\t1\t40904 2659\n",
      "102\t39390\t0.2\t1\t37016 2374\n",
      "103\t35867\t0.2\t1\t33682 2185\n",
      "104\t32606\t0.2\t1\t30634 1972\n",
      "105\t28646\t0.2\t1\t26808 1838\n",
      "106\t26227\t0.2\t1\t24640 1587\n",
      "107\t23172\t0.2\t1\t21743 1429\n",
      "108\t21298\t0.2\t1\t19891 1407\n",
      "109\t18913\t0.2\t1\t17580 1333\n",
      "110\t16998\t0.2\t1\t15871 1127\n",
      "111\t15038\t0.2\t1\t14044 994\n",
      "112\t12635\t0.2\t1\t11815 820\n",
      "113\t11352\t0.2\t1\t10604 748\n",
      "114\t9838\t0.2\t1\t9171 667\n",
      "115\t8151\t0.2\t1\t7529 622\n",
      "116\t7025\t0.2\t1\t6508 517\n",
      "117\t5831\t0.2\t1\t5406 425\n",
      "118\t5154\t0.2\t1\t4783 371\n",
      "119\t4064\t0.2\t1\t3743 321\n",
      "120\t3640\t0.2\t1\t3365 275\n",
      "121\t2769\t0.2\t1\t2540 229\n",
      "122\t2341\t0.2\t1\t2118 223\n",
      "123\t2056\t0.2\t1\t1883 173\n",
      "124\t1687\t0.2\t1\t1543 144\n",
      "125\t1340\t0.2\t1\t1233 107\n",
      "126\t1317\t0.2\t1\t1201 116\n",
      "127\t1059\t0.2\t1\t975 84\n",
      "128\t810\t0.2\t1\t740 70\n",
      "129\t681\t0.2\t1\t600 81\n",
      "130\t748\t0.2\t1\t679 69\n",
      "131\t837\t0.2\t1\t795 42\n",
      "132\t954\t0.2\t1\t903 51\n",
      "133\t1339\t0.2\t1\t1266 73\n",
      "134\t1750\t0.2\t1\t1667 83\n",
      "135\t759\t0.2\t1\t718 41\n",
      "136\t229\t0.2\t1\t209 20\n",
      "137\t121\t0.2\t1\t104 17\n",
      "138\t84\t0.2\t1\t73 11\n",
      "139\t75\t0.2\t1\t63 12\n",
      "140\t73\t0.2\t1\t58 15\n",
      "141\t35\t0.2\t1\t32 3\n",
      "142\t38\t0.2\t1\t35 3\n",
      "143\t19\t0.2\t1\t15 4\n",
      "144\t19\t0.2\t1\t11 8\n",
      "145\t22\t0.2\t1\t21 1\n",
      "146\t64\t0.2\t1\t59 5\n",
      "147\t278\t0.2\t1\t250 28\n",
      "148\t168\t0.2\t1\t142 26\n",
      "149\t83\t0.2\t1\t70 13\n",
      "150\t577\t0.2\t1\t506 71\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13433110 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 507.87 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,433,110\n",
      "Reads with adapters:                11,642,556 (86.7%)\n",
      "Reads written (passing filters):    13,433,110 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,014,966,500 bp\n",
      "Quality-trimmed:               8,709,221 bp (0.4%)\n",
      "Total written (filtered):  1,432,856,177 bp (71.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11642556 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 11.7%\n",
      "  G: 26.2%\n",
      "  T: 34.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1015848\t3358277.5\t0\t1015848\n",
      "2\t179924\t839569.4\t0\t179924\n",
      "3\t110740\t209892.3\t0\t110740\n",
      "4\t68288\t52473.1\t0\t68288\n",
      "5\t68265\t13118.3\t0\t68265\n",
      "6\t61219\t3279.6\t0\t61219\n",
      "7\t51190\t819.9\t0\t51190\n",
      "8\t71859\t205.0\t0\t71859\n",
      "9\t56699\t51.2\t0\t56454 245\n",
      "10\t63956\t12.8\t1\t61051 2905\n",
      "11\t67245\t3.2\t1\t63190 4055\n",
      "12\t68588\t0.8\t1\t64563 4025\n",
      "13\t77409\t0.2\t1\t73173 4236\n",
      "14\t74943\t0.2\t1\t70185 4758\n",
      "15\t61575\t0.2\t1\t58112 3463\n",
      "16\t87672\t0.2\t1\t82660 5012\n",
      "17\t73552\t0.2\t1\t68954 4598\n",
      "18\t68802\t0.2\t1\t65063 3739\n",
      "19\t88259\t0.2\t1\t82939 5320\n",
      "20\t82556\t0.2\t1\t78141 4415\n",
      "21\t73225\t0.2\t1\t69045 4180\n",
      "22\t83863\t0.2\t1\t79245 4618\n",
      "23\t87811\t0.2\t1\t82813 4998\n",
      "24\t89014\t0.2\t1\t83968 5046\n",
      "25\t100238\t0.2\t1\t94973 5265\n",
      "26\t83574\t0.2\t1\t78786 4788\n",
      "27\t93033\t0.2\t1\t87412 5621\n",
      "28\t89387\t0.2\t1\t85387 4000\n",
      "29\t95696\t0.2\t1\t90729 4967\n",
      "30\t94787\t0.2\t1\t90441 4346\n",
      "31\t102169\t0.2\t1\t97061 5108\n",
      "32\t94007\t0.2\t1\t89720 4287\n",
      "33\t105833\t0.2\t1\t100418 5415\n",
      "34\t134739\t0.2\t1\t127847 6892\n",
      "35\t80075\t0.2\t1\t76383 3692\n",
      "36\t104516\t0.2\t1\t98895 5621\n",
      "37\t113410\t0.2\t1\t107671 5739\n",
      "38\t107520\t0.2\t1\t102889 4631\n",
      "39\t105637\t0.2\t1\t101025 4612\n",
      "40\t118785\t0.2\t1\t112992 5793\n",
      "41\t115631\t0.2\t1\t110394 5237\n",
      "42\t122110\t0.2\t1\t116550 5560\n",
      "43\t111230\t0.2\t1\t106271 4959\n",
      "44\t128541\t0.2\t1\t122788 5753\n",
      "45\t137963\t0.2\t1\t131813 6150\n",
      "46\t99340\t0.2\t1\t94897 4443\n",
      "47\t125922\t0.2\t1\t120367 5555\n",
      "48\t125595\t0.2\t1\t120423 5172\n",
      "49\t132842\t0.2\t1\t126935 5907\n",
      "50\t127676\t0.2\t1\t122004 5672\n",
      "51\t163515\t0.2\t1\t156244 7271\n",
      "52\t117884\t0.2\t1\t113079 4805\n",
      "53\t142736\t0.2\t1\t137315 5421\n",
      "54\t107101\t0.2\t1\t102800 4301\n",
      "55\t139222\t0.2\t1\t133903 5319\n",
      "56\t143656\t0.2\t1\t137859 5797\n",
      "57\t140073\t0.2\t1\t134871 5202\n",
      "58\t134625\t0.2\t1\t129700 4925\n",
      "59\t134950\t0.2\t1\t129802 5148\n",
      "60\t140295\t0.2\t1\t134781 5514\n",
      "61\t144854\t0.2\t1\t139165 5689\n",
      "62\t159948\t0.2\t1\t153492 6456\n",
      "63\t215119\t0.2\t1\t207339 7780\n",
      "64\t166765\t0.2\t1\t161464 5301\n",
      "65\t114876\t0.2\t1\t110601 4275\n",
      "66\t125878\t0.2\t1\t121273 4605\n",
      "67\t126455\t0.2\t1\t121514 4941\n",
      "68\t128071\t0.2\t1\t123304 4767\n",
      "69\t126908\t0.2\t1\t122039 4869\n",
      "70\t132340\t0.2\t1\t127378 4962\n",
      "71\t137419\t0.2\t1\t132563 4856\n",
      "72\t133254\t0.2\t1\t128394 4860\n",
      "73\t140768\t0.2\t1\t135639 5129\n",
      "74\t130186\t0.2\t1\t125634 4552\n",
      "75\t123476\t0.2\t1\t119176 4300\n",
      "76\t121581\t0.2\t1\t117266 4315\n",
      "77\t120182\t0.2\t1\t115605 4577\n",
      "78\t115742\t0.2\t1\t111521 4221\n",
      "79\t111421\t0.2\t1\t107330 4091\n",
      "80\t110726\t0.2\t1\t106773 3953\n",
      "81\t109599\t0.2\t1\t105677 3922\n",
      "82\t103036\t0.2\t1\t99365 3671\n",
      "83\t100716\t0.2\t1\t97173 3543\n",
      "84\t98439\t0.2\t1\t94811 3628\n",
      "85\t96847\t0.2\t1\t93310 3537\n",
      "86\t96817\t0.2\t1\t93338 3479\n",
      "87\t95905\t0.2\t1\t92582 3323\n",
      "88\t93526\t0.2\t1\t90163 3363\n",
      "89\t83831\t0.2\t1\t80785 3046\n",
      "90\t80158\t0.2\t1\t77319 2839\n",
      "91\t74964\t0.2\t1\t72203 2761\n",
      "92\t70024\t0.2\t1\t67553 2471\n",
      "93\t68027\t0.2\t1\t65623 2404\n",
      "94\t64372\t0.2\t1\t62143 2229\n",
      "95\t61456\t0.2\t1\t59284 2172\n",
      "96\t57585\t0.2\t1\t55560 2025\n",
      "97\t53463\t0.2\t1\t51605 1858\n",
      "98\t50229\t0.2\t1\t48488 1741\n",
      "99\t48358\t0.2\t1\t46710 1648\n",
      "100\t43743\t0.2\t1\t42301 1442\n",
      "101\t39334\t0.2\t1\t37976 1358\n",
      "102\t35681\t0.2\t1\t34444 1237\n",
      "103\t32169\t0.2\t1\t31042 1127\n",
      "104\t30344\t0.2\t1\t29354 990\n",
      "105\t26673\t0.2\t1\t25747 926\n",
      "106\t24610\t0.2\t1\t23753 857\n",
      "107\t21493\t0.2\t1\t20733 760\n",
      "108\t19129\t0.2\t1\t18449 680\n",
      "109\t17216\t0.2\t1\t16604 612\n",
      "110\t15680\t0.2\t1\t15121 559\n",
      "111\t14422\t0.2\t1\t13916 506\n",
      "112\t12048\t0.2\t1\t11652 396\n",
      "113\t10682\t0.2\t1\t10293 389\n",
      "114\t9191\t0.2\t1\t8830 361\n",
      "115\t7355\t0.2\t1\t7077 278\n",
      "116\t6245\t0.2\t1\t6016 229\n",
      "117\t5167\t0.2\t1\t4936 231\n",
      "118\t4511\t0.2\t1\t4330 181\n",
      "119\t3552\t0.2\t1\t3385 167\n",
      "120\t3201\t0.2\t1\t3062 139\n",
      "121\t2450\t0.2\t1\t2343 107\n",
      "122\t2029\t0.2\t1\t1935 94\n",
      "123\t1814\t0.2\t1\t1715 99\n",
      "124\t1524\t0.2\t1\t1436 88\n",
      "125\t1206\t0.2\t1\t1139 67\n",
      "126\t1198\t0.2\t1\t1135 63\n",
      "127\t925\t0.2\t1\t875 50\n",
      "128\t706\t0.2\t1\t663 43\n",
      "129\t633\t0.2\t1\t590 43\n",
      "130\t698\t0.2\t1\t666 32\n",
      "131\t844\t0.2\t1\t815 29\n",
      "132\t910\t0.2\t1\t876 34\n",
      "133\t1412\t0.2\t1\t1368 44\n",
      "134\t1698\t0.2\t1\t1641 57\n",
      "135\t718\t0.2\t1\t696 22\n",
      "136\t190\t0.2\t1\t177 13\n",
      "137\t107\t0.2\t1\t98 9\n",
      "138\t70\t0.2\t1\t67 3\n",
      "139\t53\t0.2\t1\t50 3\n",
      "140\t53\t0.2\t1\t52 1\n",
      "141\t26\t0.2\t1\t26\n",
      "142\t26\t0.2\t1\t24 2\n",
      "143\t21\t0.2\t1\t20 1\n",
      "144\t18\t0.2\t1\t16 2\n",
      "145\t35\t0.2\t1\t32 3\n",
      "146\t39\t0.2\t1\t35 4\n",
      "147\t177\t0.2\t1\t169 8\n",
      "148\t82\t0.2\t1\t76 6\n",
      "149\t45\t0.2\t1\t43 2\n",
      "150\t267\t0.2\t1\t241 26\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13433110 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz, file_2: CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13433110\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 28131 (0.21%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_16C_26psu_2_S21_L001_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 472.50 s (35 us/read; 1.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,408,232\n",
      "Reads with adapters:                11,662,676 (87.0%)\n",
      "Reads written (passing filters):    13,408,232 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,011,234,800 bp\n",
      "Quality-trimmed:               4,075,788 bp (0.2%)\n",
      "Total written (filtered):  1,423,488,059 bp (70.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11662676 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 12.5%\n",
      "  G: 23.6%\n",
      "  T: 35.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t950454\t3352058.0\t0\t950454\n",
      "2\t175107\t838014.5\t0\t175107\n",
      "3\t106043\t209503.6\t0\t106043\n",
      "4\t68134\t52375.9\t0\t68134\n",
      "5\t61886\t13094.0\t0\t61886\n",
      "6\t61175\t3273.5\t0\t61175\n",
      "7\t51159\t818.4\t0\t51159\n",
      "8\t64511\t204.6\t0\t64511\n",
      "9\t64557\t51.1\t0\t63785 772\n",
      "10\t60792\t12.8\t1\t57082 3710\n",
      "11\t68970\t3.2\t1\t63184 5786\n",
      "12\t66553\t0.8\t1\t61883 4670\n",
      "13\t68274\t0.2\t1\t63204 5070\n",
      "14\t71364\t0.2\t1\t65643 5721\n",
      "15\t74174\t0.2\t1\t68692 5482\n",
      "16\t73515\t0.2\t1\t67346 6169\n",
      "17\t90659\t0.2\t1\t82653 8006\n",
      "18\t76471\t0.2\t1\t71237 5234\n",
      "19\t67430\t0.2\t1\t62447 4983\n",
      "20\t80487\t0.2\t1\t74742 5745\n",
      "21\t89676\t0.2\t1\t82775 6901\n",
      "22\t76355\t0.2\t1\t71205 5150\n",
      "23\t90468\t0.2\t1\t83777 6691\n",
      "24\t85067\t0.2\t1\t77644 7423\n",
      "25\t91076\t0.2\t1\t84134 6942\n",
      "26\t84890\t0.2\t1\t79317 5573\n",
      "27\t95709\t0.2\t1\t88308 7401\n",
      "28\t95026\t0.2\t1\t88099 6927\n",
      "29\t98118\t0.2\t1\t89948 8170\n",
      "30\t107848\t0.2\t1\t101119 6729\n",
      "31\t84062\t0.2\t1\t78313 5749\n",
      "32\t103994\t0.2\t1\t96660 7334\n",
      "33\t98133\t0.2\t1\t90574 7559\n",
      "34\t120505\t0.2\t1\t111057 9448\n",
      "35\t94055\t0.2\t1\t87482 6573\n",
      "36\t107881\t0.2\t1\t100206 7675\n",
      "37\t113227\t0.2\t1\t104836 8391\n",
      "38\t107162\t0.2\t1\t98615 8547\n",
      "39\t126650\t0.2\t1\t117366 9284\n",
      "40\t101784\t0.2\t1\t94417 7367\n",
      "41\t130084\t0.2\t1\t120107 9977\n",
      "42\t108652\t0.2\t1\t99173 9479\n",
      "43\t179621\t0.2\t1\t169283 10338\n",
      "44\t59500\t0.2\t1\t54067 5433\n",
      "45\t124819\t0.2\t1\t116443 8376\n",
      "46\t119134\t0.2\t1\t110810 8324\n",
      "47\t129268\t0.2\t1\t120606 8662\n",
      "48\t119694\t0.2\t1\t111395 8299\n",
      "49\t138243\t0.2\t1\t127784 10459\n",
      "50\t121057\t0.2\t1\t113597 7460\n",
      "51\t131350\t0.2\t1\t122521 8829\n",
      "52\t138351\t0.2\t1\t127802 10549\n",
      "53\t154024\t0.2\t1\t144429 9595\n",
      "54\t116951\t0.2\t1\t108870 8081\n",
      "55\t139783\t0.2\t1\t130545 9238\n",
      "56\t165267\t0.2\t1\t155186 10081\n",
      "57\t126758\t0.2\t1\t117494 9264\n",
      "58\t142781\t0.2\t1\t134977 7804\n",
      "59\t117589\t0.2\t1\t109427 8162\n",
      "60\t144831\t0.2\t1\t135423 9408\n",
      "61\t138318\t0.2\t1\t129552 8766\n",
      "62\t148945\t0.2\t1\t139636 9309\n",
      "63\t129691\t0.2\t1\t120718 8973\n",
      "64\t134879\t0.2\t1\t126688 8191\n",
      "65\t146410\t0.2\t1\t137112 9298\n",
      "66\t156960\t0.2\t1\t146744 10216\n",
      "67\t171283\t0.2\t1\t158133 13150\n",
      "68\t312723\t0.2\t1\t296738 15985\n",
      "69\t76525\t0.2\t1\t70767 5758\n",
      "70\t28798\t0.2\t1\t24712 4086\n",
      "71\t79564\t0.2\t1\t74491 5073\n",
      "72\t122325\t0.2\t1\t115172 7153\n",
      "73\t132393\t0.2\t1\t124378 8015\n",
      "74\t131213\t0.2\t1\t123087 8126\n",
      "75\t128592\t0.2\t1\t120236 8356\n",
      "76\t128119\t0.2\t1\t119430 8689\n",
      "77\t126752\t0.2\t1\t118441 8311\n",
      "78\t121397\t0.2\t1\t113322 8075\n",
      "79\t118367\t0.2\t1\t110488 7879\n",
      "80\t116333\t0.2\t1\t108557 7776\n",
      "81\t115083\t0.2\t1\t107691 7392\n",
      "82\t111285\t0.2\t1\t103893 7392\n",
      "83\t107858\t0.2\t1\t100817 7041\n",
      "84\t105350\t0.2\t1\t98390 6960\n",
      "85\t102602\t0.2\t1\t95870 6732\n",
      "86\t97359\t0.2\t1\t90922 6437\n",
      "87\t94015\t0.2\t1\t87784 6231\n",
      "88\t92275\t0.2\t1\t86366 5909\n",
      "89\t87740\t0.2\t1\t82108 5632\n",
      "90\t86642\t0.2\t1\t80943 5699\n",
      "91\t81958\t0.2\t1\t76615 5343\n",
      "92\t78763\t0.2\t1\t73589 5174\n",
      "93\t72932\t0.2\t1\t68198 4734\n",
      "94\t70403\t0.2\t1\t65851 4552\n",
      "95\t65162\t0.2\t1\t60795 4367\n",
      "96\t61147\t0.2\t1\t56990 4157\n",
      "97\t57845\t0.2\t1\t54051 3794\n",
      "98\t54088\t0.2\t1\t50353 3735\n",
      "99\t50912\t0.2\t1\t47388 3524\n",
      "100\t46451\t0.2\t1\t43306 3145\n",
      "101\t43581\t0.2\t1\t40541 3040\n",
      "102\t39156\t0.2\t1\t36546 2610\n",
      "103\t35574\t0.2\t1\t33122 2452\n",
      "104\t32789\t0.2\t1\t30678 2111\n",
      "105\t29034\t0.2\t1\t27055 1979\n",
      "106\t26298\t0.2\t1\t24343 1955\n",
      "107\t23552\t0.2\t1\t21942 1610\n",
      "108\t21227\t0.2\t1\t19666 1561\n",
      "109\t18723\t0.2\t1\t17348 1375\n",
      "110\t16933\t0.2\t1\t15728 1205\n",
      "111\t14813\t0.2\t1\t13715 1098\n",
      "112\t12788\t0.2\t1\t11763 1025\n",
      "113\t11310\t0.2\t1\t10437 873\n",
      "114\t9920\t0.2\t1\t9142 778\n",
      "115\t8131\t0.2\t1\t7451 680\n",
      "116\t6933\t0.2\t1\t6415 518\n",
      "117\t5884\t0.2\t1\t5388 496\n",
      "118\t5136\t0.2\t1\t4702 434\n",
      "119\t4097\t0.2\t1\t3751 346\n",
      "120\t3546\t0.2\t1\t3238 308\n",
      "121\t2947\t0.2\t1\t2707 240\n",
      "122\t2422\t0.2\t1\t2210 212\n",
      "123\t1924\t0.2\t1\t1761 163\n",
      "124\t1733\t0.2\t1\t1576 157\n",
      "125\t1418\t0.2\t1\t1276 142\n",
      "126\t1305\t0.2\t1\t1178 127\n",
      "127\t1104\t0.2\t1\t1004 100\n",
      "128\t823\t0.2\t1\t732 91\n",
      "129\t659\t0.2\t1\t593 66\n",
      "130\t710\t0.2\t1\t634 76\n",
      "131\t875\t0.2\t1\t812 63\n",
      "132\t926\t0.2\t1\t862 64\n",
      "133\t1378\t0.2\t1\t1297 81\n",
      "134\t1693\t0.2\t1\t1594 99\n",
      "135\t721\t0.2\t1\t691 30\n",
      "136\t229\t0.2\t1\t205 24\n",
      "137\t172\t0.2\t1\t157 15\n",
      "138\t80\t0.2\t1\t66 14\n",
      "139\t66\t0.2\t1\t55 11\n",
      "140\t62\t0.2\t1\t53 9\n",
      "141\t37\t0.2\t1\t35 2\n",
      "142\t33\t0.2\t1\t27 6\n",
      "143\t21\t0.2\t1\t19 2\n",
      "144\t18\t0.2\t1\t17 1\n",
      "145\t32\t0.2\t1\t28 4\n",
      "146\t69\t0.2\t1\t67 2\n",
      "147\t283\t0.2\t1\t250 33\n",
      "148\t183\t0.2\t1\t166 17\n",
      "149\t88\t0.2\t1\t79 9\n",
      "150\t625\t0.2\t1\t537 88\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13408232 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 493.05 s (37 us/read; 1.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,408,232\n",
      "Reads with adapters:                11,603,960 (86.5%)\n",
      "Reads written (passing filters):    13,408,232 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,011,234,800 bp\n",
      "Quality-trimmed:               8,433,724 bp (0.4%)\n",
      "Total written (filtered):  1,430,263,611 bp (71.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11603960 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 11.7%\n",
      "  G: 26.2%\n",
      "  T: 34.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1017131\t3352058.0\t0\t1017131\n",
      "2\t177058\t838014.5\t0\t177058\n",
      "3\t112787\t209503.6\t0\t112787\n",
      "4\t67003\t52375.9\t0\t67003\n",
      "5\t66504\t13094.0\t0\t66504\n",
      "6\t60966\t3273.5\t0\t60966\n",
      "7\t50684\t818.4\t0\t50684\n",
      "8\t71309\t204.6\t0\t71309\n",
      "9\t56145\t51.1\t0\t55917 228\n",
      "10\t61633\t12.8\t1\t57525 4108\n",
      "11\t66766\t3.2\t1\t61633 5133\n",
      "12\t66970\t0.8\t1\t61817 5153\n",
      "13\t77136\t0.2\t1\t71259 5877\n",
      "14\t74020\t0.2\t1\t67910 6110\n",
      "15\t60571\t0.2\t1\t56229 4342\n",
      "16\t86270\t0.2\t1\t79871 6399\n",
      "17\t75982\t0.2\t1\t70248 5734\n",
      "18\t65820\t0.2\t1\t61090 4730\n",
      "19\t87582\t0.2\t1\t81169 6413\n",
      "20\t81347\t0.2\t1\t75534 5813\n",
      "21\t71738\t0.2\t1\t66607 5131\n",
      "22\t82571\t0.2\t1\t76726 5845\n",
      "23\t87855\t0.2\t1\t81586 6269\n",
      "24\t88796\t0.2\t1\t82344 6452\n",
      "25\t99857\t0.2\t1\t93510 6347\n",
      "26\t83385\t0.2\t1\t77300 6085\n",
      "27\t93625\t0.2\t1\t86769 6856\n",
      "28\t85817\t0.2\t1\t80874 4943\n",
      "29\t96248\t0.2\t1\t90002 6246\n",
      "30\t93727\t0.2\t1\t88168 5559\n",
      "31\t103646\t0.2\t1\t97099 6547\n",
      "32\t90867\t0.2\t1\t85391 5476\n",
      "33\t105892\t0.2\t1\t98959 6933\n",
      "34\t137038\t0.2\t1\t128489 8549\n",
      "35\t77044\t0.2\t1\t72414 4630\n",
      "36\t105552\t0.2\t1\t98517 7035\n",
      "37\t112682\t0.2\t1\t105690 6992\n",
      "38\t106873\t0.2\t1\t100867 6006\n",
      "39\t102355\t0.2\t1\t96889 5466\n",
      "40\t117602\t0.2\t1\t110487 7115\n",
      "41\t116162\t0.2\t1\t109429 6733\n",
      "42\t121108\t0.2\t1\t114095 7013\n",
      "43\t109639\t0.2\t1\t103525 6114\n",
      "44\t129653\t0.2\t1\t122002 7651\n",
      "45\t138376\t0.2\t1\t130424 7952\n",
      "46\t97190\t0.2\t1\t91779 5411\n",
      "47\t124803\t0.2\t1\t118327 6476\n",
      "48\t124244\t0.2\t1\t117667 6577\n",
      "49\t131343\t0.2\t1\t123780 7563\n",
      "50\t128654\t0.2\t1\t121411 7243\n",
      "51\t160151\t0.2\t1\t151381 8770\n",
      "52\t116727\t0.2\t1\t110469 6258\n",
      "53\t144929\t0.2\t1\t137769 7160\n",
      "54\t104645\t0.2\t1\t99281 5364\n",
      "55\t138649\t0.2\t1\t131848 6801\n",
      "56\t142696\t0.2\t1\t134983 7713\n",
      "57\t141410\t0.2\t1\t134925 6485\n",
      "58\t134726\t0.2\t1\t128298 6428\n",
      "59\t134797\t0.2\t1\t128190 6607\n",
      "60\t140631\t0.2\t1\t133581 7050\n",
      "61\t142568\t0.2\t1\t135579 6989\n",
      "62\t160172\t0.2\t1\t152064 8108\n",
      "63\t209874\t0.2\t1\t200586 9288\n",
      "64\t163821\t0.2\t1\t157603 6218\n",
      "65\t114086\t0.2\t1\t108681 5405\n",
      "66\t126793\t0.2\t1\t120700 6093\n",
      "67\t126587\t0.2\t1\t120373 6214\n",
      "68\t128999\t0.2\t1\t122868 6131\n",
      "69\t127101\t0.2\t1\t121101 6000\n",
      "70\t133367\t0.2\t1\t126848 6519\n",
      "71\t137940\t0.2\t1\t131348 6592\n",
      "72\t133744\t0.2\t1\t127253 6491\n",
      "73\t140847\t0.2\t1\t134101 6746\n",
      "74\t131039\t0.2\t1\t124610 6429\n",
      "75\t123648\t0.2\t1\t117677 5971\n",
      "76\t121797\t0.2\t1\t115805 5992\n",
      "77\t120668\t0.2\t1\t114606 6062\n",
      "78\t116805\t0.2\t1\t111027 5778\n",
      "79\t111250\t0.2\t1\t105828 5422\n",
      "80\t110955\t0.2\t1\t105751 5204\n",
      "81\t109348\t0.2\t1\t104246 5102\n",
      "82\t103784\t0.2\t1\t98831 4953\n",
      "83\t101052\t0.2\t1\t96319 4733\n",
      "84\t98311\t0.2\t1\t93628 4683\n",
      "85\t96878\t0.2\t1\t92456 4422\n",
      "86\t95996\t0.2\t1\t91571 4425\n",
      "87\t96605\t0.2\t1\t92256 4349\n",
      "88\t93536\t0.2\t1\t89501 4035\n",
      "89\t83632\t0.2\t1\t79805 3827\n",
      "90\t80377\t0.2\t1\t76751 3626\n",
      "91\t75608\t0.2\t1\t72292 3316\n",
      "92\t71253\t0.2\t1\t68176 3077\n",
      "93\t68064\t0.2\t1\t65127 2937\n",
      "94\t65349\t0.2\t1\t62356 2993\n",
      "95\t61296\t0.2\t1\t58482 2814\n",
      "96\t57547\t0.2\t1\t54961 2586\n",
      "97\t53666\t0.2\t1\t51329 2337\n",
      "98\t50699\t0.2\t1\t48576 2123\n",
      "99\t48177\t0.2\t1\t46098 2079\n",
      "100\t43932\t0.2\t1\t42005 1927\n",
      "101\t39615\t0.2\t1\t37850 1765\n",
      "102\t35478\t0.2\t1\t33828 1650\n",
      "103\t32345\t0.2\t1\t30762 1583\n",
      "104\t30580\t0.2\t1\t29207 1373\n",
      "105\t27062\t0.2\t1\t25870 1192\n",
      "106\t24593\t0.2\t1\t23539 1054\n",
      "107\t21633\t0.2\t1\t20641 992\n",
      "108\t19150\t0.2\t1\t18297 853\n",
      "109\t17101\t0.2\t1\t16306 795\n",
      "110\t15550\t0.2\t1\t14882 668\n",
      "111\t14159\t0.2\t1\t13496 663\n",
      "112\t12046\t0.2\t1\t11497 549\n",
      "113\t10553\t0.2\t1\t10087 466\n",
      "114\t9330\t0.2\t1\t8879 451\n",
      "115\t7323\t0.2\t1\t7020 303\n",
      "116\t6173\t0.2\t1\t5878 295\n",
      "117\t5159\t0.2\t1\t4924 235\n",
      "118\t4546\t0.2\t1\t4310 236\n",
      "119\t3676\t0.2\t1\t3456 220\n",
      "120\t3075\t0.2\t1\t2912 163\n",
      "121\t2608\t0.2\t1\t2451 157\n",
      "122\t2115\t0.2\t1\t1990 125\n",
      "123\t1712\t0.2\t1\t1620 92\n",
      "124\t1534\t0.2\t1\t1451 83\n",
      "125\t1259\t0.2\t1\t1192 67\n",
      "126\t1123\t0.2\t1\t1061 62\n",
      "127\t965\t0.2\t1\t910 55\n",
      "128\t708\t0.2\t1\t665 43\n",
      "129\t619\t0.2\t1\t586 33\n",
      "130\t674\t0.2\t1\t630 44\n",
      "131\t895\t0.2\t1\t850 45\n",
      "132\t923\t0.2\t1\t873 50\n",
      "133\t1446\t0.2\t1\t1396 50\n",
      "134\t1648\t0.2\t1\t1601 47\n",
      "135\t705\t0.2\t1\t677 28\n",
      "136\t204\t0.2\t1\t192 12\n",
      "137\t151\t0.2\t1\t142 9\n",
      "138\t69\t0.2\t1\t59 10\n",
      "139\t62\t0.2\t1\t53 9\n",
      "140\t50\t0.2\t1\t43 7\n",
      "141\t28\t0.2\t1\t22 6\n",
      "142\t25\t0.2\t1\t22 3\n",
      "143\t20\t0.2\t1\t20\n",
      "144\t13\t0.2\t1\t12 1\n",
      "145\t39\t0.2\t1\t35 4\n",
      "146\t44\t0.2\t1\t41 3\n",
      "147\t178\t0.2\t1\t162 16\n",
      "148\t92\t0.2\t1\t79 13\n",
      "149\t44\t0.2\t1\t42 2\n",
      "150\t302\t0.2\t1\t273 29\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13408232 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz, file_2: CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13408232\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 26942 (0.20%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 60% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_16C_26psu_2_S21_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_16C_26psu_2_S21_L002_R1_001_trimmed.fq.gz and CTRL_16C_26psu_2_S21_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 530.18 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,781,290\n",
      "Reads with adapters:                10,464,679 (75.9%)\n",
      "Reads written (passing filters):    13,781,290 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,067,193,500 bp\n",
      "Quality-trimmed:               4,066,824 bp (0.2%)\n",
      "Total written (filtered):  1,647,743,968 bp (79.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10464679 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 15.6%\n",
      "  G: 18.6%\n",
      "  T: 34.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2115194\t3445322.5\t0\t2115194\n",
      "2\t272894\t861330.6\t0\t272894\n",
      "3\t153504\t215332.7\t0\t153504\n",
      "4\t91420\t53833.2\t0\t91420\n",
      "5\t70039\t13458.3\t0\t70039\n",
      "6\t67409\t3364.6\t0\t67409\n",
      "7\t55787\t841.1\t0\t55787\n",
      "8\t69144\t210.3\t0\t69144\n",
      "9\t70298\t52.6\t0\t69211 1087\n",
      "10\t65479\t13.1\t1\t61710 3769\n",
      "11\t71864\t3.3\t1\t66237 5627\n",
      "12\t69709\t0.8\t1\t65113 4596\n",
      "13\t68649\t0.2\t1\t64099 4550\n",
      "14\t71371\t0.2\t1\t66030 5341\n",
      "15\t72335\t0.2\t1\t67349 4986\n",
      "16\t73096\t0.2\t1\t67373 5723\n",
      "17\t87823\t0.2\t1\t80650 7173\n",
      "18\t73513\t0.2\t1\t68795 4718\n",
      "19\t64905\t0.2\t1\t60375 4530\n",
      "20\t76453\t0.2\t1\t71101 5352\n",
      "21\t83756\t0.2\t1\t77496 6260\n",
      "22\t71860\t0.2\t1\t67380 4480\n",
      "23\t82380\t0.2\t1\t76793 5587\n",
      "24\t76931\t0.2\t1\t70531 6400\n",
      "25\t82900\t0.2\t1\t77013 5887\n",
      "26\t75729\t0.2\t1\t70800 4929\n",
      "27\t84549\t0.2\t1\t78103 6446\n",
      "28\t82744\t0.2\t1\t76781 5963\n",
      "29\t87966\t0.2\t1\t80505 7461\n",
      "30\t92743\t0.2\t1\t87256 5487\n",
      "31\t72956\t0.2\t1\t68033 4923\n",
      "32\t86797\t0.2\t1\t81028 5769\n",
      "33\t84425\t0.2\t1\t77963 6462\n",
      "34\t100255\t0.2\t1\t92044 8211\n",
      "35\t94925\t0.2\t1\t87424 7501\n",
      "36\t151403\t0.2\t1\t144008 7395\n",
      "37\t34431\t0.2\t1\t30234 4197\n",
      "38\t81160\t0.2\t1\t76071 5089\n",
      "39\t98494\t0.2\t1\t91430 7064\n",
      "40\t94229\t0.2\t1\t88176 6053\n",
      "41\t105491\t0.2\t1\t97539 7952\n",
      "42\t89350\t0.2\t1\t81822 7528\n",
      "43\t144554\t0.2\t1\t136557 7997\n",
      "44\t52667\t0.2\t1\t48059 4608\n",
      "45\t80312\t0.2\t1\t74870 5442\n",
      "46\t96347\t0.2\t1\t89778 6569\n",
      "47\t100619\t0.2\t1\t94323 6296\n",
      "48\t87690\t0.2\t1\t81510 6180\n",
      "49\t103414\t0.2\t1\t95838 7576\n",
      "50\t86689\t0.2\t1\t81209 5480\n",
      "51\t92177\t0.2\t1\t85927 6250\n",
      "52\t98128\t0.2\t1\t90537 7591\n",
      "53\t111470\t0.2\t1\t104728 6742\n",
      "54\t79643\t0.2\t1\t74106 5537\n",
      "55\t98690\t0.2\t1\t92247 6443\n",
      "56\t114543\t0.2\t1\t107715 6828\n",
      "57\t91001\t0.2\t1\t84632 6369\n",
      "58\t94668\t0.2\t1\t89637 5031\n",
      "59\t73433\t0.2\t1\t68107 5326\n",
      "60\t100893\t0.2\t1\t94469 6424\n",
      "61\t94258\t0.2\t1\t88482 5776\n",
      "62\t93437\t0.2\t1\t87751 5686\n",
      "63\t86189\t0.2\t1\t80341 5848\n",
      "64\t87860\t0.2\t1\t82577 5283\n",
      "65\t95579\t0.2\t1\t89442 6137\n",
      "66\t102630\t0.2\t1\t95991 6639\n",
      "67\t110068\t0.2\t1\t101200 8868\n",
      "68\t211731\t0.2\t1\t201178 10553\n",
      "69\t44241\t0.2\t1\t40907 3334\n",
      "70\t13519\t0.2\t1\t11187 2332\n",
      "71\t49103\t0.2\t1\t45715 3388\n",
      "72\t77796\t0.2\t1\t73169 4627\n",
      "73\t81869\t0.2\t1\t76719 5150\n",
      "74\t82069\t0.2\t1\t76896 5173\n",
      "75\t80967\t0.2\t1\t75833 5134\n",
      "76\t79680\t0.2\t1\t74662 5018\n",
      "77\t80336\t0.2\t1\t75241 5095\n",
      "78\t75811\t0.2\t1\t70973 4838\n",
      "79\t74367\t0.2\t1\t69524 4843\n",
      "80\t73534\t0.2\t1\t69012 4522\n",
      "81\t72214\t0.2\t1\t67540 4674\n",
      "82\t70474\t0.2\t1\t66112 4362\n",
      "83\t68507\t0.2\t1\t64201 4306\n",
      "84\t66184\t0.2\t1\t61986 4198\n",
      "85\t64533\t0.2\t1\t60445 4088\n",
      "86\t62422\t0.2\t1\t58352 4070\n",
      "87\t59642\t0.2\t1\t55842 3800\n",
      "88\t59336\t0.2\t1\t55541 3795\n",
      "89\t56406\t0.2\t1\t52805 3601\n",
      "90\t56172\t0.2\t1\t52495 3677\n",
      "91\t53832\t0.2\t1\t50312 3520\n",
      "92\t51101\t0.2\t1\t47868 3233\n",
      "93\t48168\t0.2\t1\t45095 3073\n",
      "94\t47770\t0.2\t1\t44644 3126\n",
      "95\t44085\t0.2\t1\t41250 2835\n",
      "96\t41623\t0.2\t1\t38858 2765\n",
      "97\t39066\t0.2\t1\t36394 2672\n",
      "98\t35943\t0.2\t1\t33581 2362\n",
      "99\t35291\t0.2\t1\t32748 2543\n",
      "100\t32499\t0.2\t1\t30308 2191\n",
      "101\t30357\t0.2\t1\t28386 1971\n",
      "102\t28530\t0.2\t1\t26552 1978\n",
      "103\t26074\t0.2\t1\t24341 1733\n",
      "104\t24528\t0.2\t1\t22773 1755\n",
      "105\t21898\t0.2\t1\t20353 1545\n",
      "106\t20241\t0.2\t1\t18793 1448\n",
      "107\t18328\t0.2\t1\t16963 1365\n",
      "108\t17608\t0.2\t1\t16339 1269\n",
      "109\t15247\t0.2\t1\t14123 1124\n",
      "110\t14054\t0.2\t1\t12990 1064\n",
      "111\t12816\t0.2\t1\t11851 965\n",
      "112\t11319\t0.2\t1\t10448 871\n",
      "113\t9601\t0.2\t1\t8829 772\n",
      "114\t8785\t0.2\t1\t8065 720\n",
      "115\t7727\t0.2\t1\t7075 652\n",
      "116\t6667\t0.2\t1\t6137 530\n",
      "117\t5842\t0.2\t1\t5341 501\n",
      "118\t5116\t0.2\t1\t4650 466\n",
      "119\t4257\t0.2\t1\t3896 361\n",
      "120\t3744\t0.2\t1\t3400 344\n",
      "121\t3378\t0.2\t1\t3078 300\n",
      "122\t2708\t0.2\t1\t2469 239\n",
      "123\t2529\t0.2\t1\t2246 283\n",
      "124\t2015\t0.2\t1\t1814 201\n",
      "125\t1810\t0.2\t1\t1626 184\n",
      "126\t1647\t0.2\t1\t1490 157\n",
      "127\t1294\t0.2\t1\t1150 144\n",
      "128\t1010\t0.2\t1\t909 101\n",
      "129\t964\t0.2\t1\t860 104\n",
      "130\t736\t0.2\t1\t664 72\n",
      "131\t614\t0.2\t1\t553 61\n",
      "132\t536\t0.2\t1\t487 49\n",
      "133\t628\t0.2\t1\t563 65\n",
      "134\t1415\t0.2\t1\t1342 73\n",
      "135\t608\t0.2\t1\t567 41\n",
      "136\t207\t0.2\t1\t188 19\n",
      "137\t132\t0.2\t1\t118 14\n",
      "138\t84\t0.2\t1\t75 9\n",
      "139\t56\t0.2\t1\t44 12\n",
      "140\t46\t0.2\t1\t36 10\n",
      "141\t40\t0.2\t1\t35 5\n",
      "142\t28\t0.2\t1\t23 5\n",
      "143\t39\t0.2\t1\t32 7\n",
      "144\t49\t0.2\t1\t44 5\n",
      "145\t178\t0.2\t1\t162 16\n",
      "146\t250\t0.2\t1\t233 17\n",
      "147\t1272\t0.2\t1\t1190 82\n",
      "148\t812\t0.2\t1\t771 41\n",
      "149\t417\t0.2\t1\t393 24\n",
      "150\t2828\t0.2\t1\t2398 430\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "13781290 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 561.78 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,781,290\n",
      "Reads with adapters:                10,393,178 (75.4%)\n",
      "Reads written (passing filters):    13,781,290 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,067,193,500 bp\n",
      "Quality-trimmed:              10,332,398 bp (0.5%)\n",
      "Total written (filtered):  1,650,693,153 bp (79.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10393178 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 12.4%\n",
      "  G: 22.9%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2177342\t3445322.5\t0\t2177342\n",
      "2\t279599\t861330.6\t0\t279599\n",
      "3\t156926\t215332.7\t0\t156926\n",
      "4\t90665\t53833.2\t0\t90665\n",
      "5\t76306\t13458.3\t0\t76306\n",
      "6\t64993\t3364.6\t0\t64993\n",
      "7\t53352\t841.1\t0\t53352\n",
      "8\t75967\t210.3\t0\t75967\n",
      "9\t61280\t52.6\t0\t61074 206\n",
      "10\t66110\t13.1\t1\t62970 3140\n",
      "11\t69828\t3.3\t1\t65500 4328\n",
      "12\t69328\t0.8\t1\t64877 4451\n",
      "13\t77454\t0.2\t1\t72977 4477\n",
      "14\t71203\t0.2\t1\t66291 4912\n",
      "15\t59758\t0.2\t1\t55879 3879\n",
      "16\t86601\t0.2\t1\t81117 5484\n",
      "17\t65384\t0.2\t1\t60933 4451\n",
      "18\t67208\t0.2\t1\t63111 4097\n",
      "19\t81178\t0.2\t1\t76029 5149\n",
      "20\t77499\t0.2\t1\t72833 4666\n",
      "21\t68546\t0.2\t1\t64113 4433\n",
      "22\t76276\t0.2\t1\t71754 4522\n",
      "23\t78702\t0.2\t1\t73839 4863\n",
      "24\t78191\t0.2\t1\t73447 4744\n",
      "25\t88316\t0.2\t1\t83190 5126\n",
      "26\t73679\t0.2\t1\t68983 4696\n",
      "27\t82923\t0.2\t1\t77281 5642\n",
      "28\t76296\t0.2\t1\t72446 3850\n",
      "29\t82111\t0.2\t1\t77123 4988\n",
      "30\t79614\t0.2\t1\t75308 4306\n",
      "31\t89456\t0.2\t1\t84223 5233\n",
      "32\t78118\t0.2\t1\t73947 4171\n",
      "33\t86787\t0.2\t1\t81520 5267\n",
      "34\t111454\t0.2\t1\t105036 6418\n",
      "35\t64200\t0.2\t1\t60716 3484\n",
      "36\t89863\t0.2\t1\t84483 5380\n",
      "37\t97027\t0.2\t1\t91271 5756\n",
      "38\t90403\t0.2\t1\t85738 4665\n",
      "39\t89856\t0.2\t1\t85089 4767\n",
      "40\t98244\t0.2\t1\t92516 5728\n",
      "41\t92652\t0.2\t1\t87737 4915\n",
      "42\t94536\t0.2\t1\t89519 5017\n",
      "43\t85154\t0.2\t1\t80486 4668\n",
      "44\t96817\t0.2\t1\t91511 5306\n",
      "45\t102115\t0.2\t1\t96823 5292\n",
      "46\t77381\t0.2\t1\t73350 4031\n",
      "47\t94700\t0.2\t1\t89756 4944\n",
      "48\t92948\t0.2\t1\t88160 4788\n",
      "49\t96945\t0.2\t1\t91985 4960\n",
      "50\t91133\t0.2\t1\t86206 4927\n",
      "51\t111309\t0.2\t1\t105778 5531\n",
      "52\t80724\t0.2\t1\t76802 3922\n",
      "53\t97109\t0.2\t1\t92582 4527\n",
      "54\t75460\t0.2\t1\t71575 3885\n",
      "55\t96939\t0.2\t1\t92436 4503\n",
      "56\t100452\t0.2\t1\t95531 4921\n",
      "57\t93931\t0.2\t1\t89647 4284\n",
      "58\t91630\t0.2\t1\t87584 4046\n",
      "59\t91476\t0.2\t1\t87193 4283\n",
      "60\t94295\t0.2\t1\t89743 4552\n",
      "61\t96075\t0.2\t1\t91458 4617\n",
      "62\t103947\t0.2\t1\t98742 5205\n",
      "63\t139881\t0.2\t1\t133644 6237\n",
      "64\t115738\t0.2\t1\t111196 4542\n",
      "65\t76600\t0.2\t1\t73143 3457\n",
      "66\t81388\t0.2\t1\t77826 3562\n",
      "67\t79784\t0.2\t1\t76063 3721\n",
      "68\t81137\t0.2\t1\t77492 3645\n",
      "69\t80187\t0.2\t1\t76468 3719\n",
      "70\t85090\t0.2\t1\t81074 4016\n",
      "71\t89717\t0.2\t1\t85778 3939\n",
      "72\t84263\t0.2\t1\t80409 3854\n",
      "73\t87216\t0.2\t1\t83465 3751\n",
      "74\t81093\t0.2\t1\t77528 3565\n",
      "75\t77500\t0.2\t1\t74052 3448\n",
      "76\t75266\t0.2\t1\t71946 3320\n",
      "77\t75853\t0.2\t1\t72329 3524\n",
      "78\t72260\t0.2\t1\t68950 3310\n",
      "79\t68908\t0.2\t1\t65817 3091\n",
      "80\t69469\t0.2\t1\t66410 3059\n",
      "81\t68119\t0.2\t1\t65042 3077\n",
      "82\t65495\t0.2\t1\t62627 2868\n",
      "83\t64033\t0.2\t1\t61248 2785\n",
      "84\t61976\t0.2\t1\t59216 2760\n",
      "85\t61317\t0.2\t1\t58515 2802\n",
      "86\t60982\t0.2\t1\t58300 2682\n",
      "87\t61128\t0.2\t1\t58498 2630\n",
      "88\t60002\t0.2\t1\t57306 2696\n",
      "89\t53699\t0.2\t1\t51332 2367\n",
      "90\t51966\t0.2\t1\t49630 2336\n",
      "91\t49026\t0.2\t1\t46908 2118\n",
      "92\t46213\t0.2\t1\t44244 1969\n",
      "93\t44796\t0.2\t1\t42840 1956\n",
      "94\t44237\t0.2\t1\t42309 1928\n",
      "95\t41556\t0.2\t1\t39752 1804\n",
      "96\t39335\t0.2\t1\t37550 1785\n",
      "97\t36382\t0.2\t1\t34875 1507\n",
      "98\t33450\t0.2\t1\t32005 1445\n",
      "99\t32975\t0.2\t1\t31610 1365\n",
      "100\t29764\t0.2\t1\t28510 1254\n",
      "101\t27741\t0.2\t1\t26626 1115\n",
      "102\t25948\t0.2\t1\t24908 1040\n",
      "103\t23781\t0.2\t1\t22799 982\n",
      "104\t22951\t0.2\t1\t22001 950\n",
      "105\t20726\t0.2\t1\t19855 871\n",
      "106\t19269\t0.2\t1\t18460 809\n",
      "107\t17116\t0.2\t1\t16344 772\n",
      "108\t16142\t0.2\t1\t15422 720\n",
      "109\t14186\t0.2\t1\t13581 605\n",
      "110\t13235\t0.2\t1\t12682 553\n",
      "111\t12589\t0.2\t1\t12054 535\n",
      "112\t10912\t0.2\t1\t10466 446\n",
      "113\t9393\t0.2\t1\t8942 451\n",
      "114\t8457\t0.2\t1\t8091 366\n",
      "115\t7197\t0.2\t1\t6874 323\n",
      "116\t6211\t0.2\t1\t5908 303\n",
      "117\t5216\t0.2\t1\t4981 235\n",
      "118\t4612\t0.2\t1\t4406 206\n",
      "119\t3969\t0.2\t1\t3800 169\n",
      "120\t3391\t0.2\t1\t3233 158\n",
      "121\t3136\t0.2\t1\t3000 136\n",
      "122\t2549\t0.2\t1\t2419 130\n",
      "123\t2350\t0.2\t1\t2231 119\n",
      "124\t1891\t0.2\t1\t1802 89\n",
      "125\t1725\t0.2\t1\t1617 108\n",
      "126\t1554\t0.2\t1\t1484 70\n",
      "127\t1178\t0.2\t1\t1121 57\n",
      "128\t930\t0.2\t1\t883 47\n",
      "129\t904\t0.2\t1\t851 53\n",
      "130\t653\t0.2\t1\t624 29\n",
      "131\t560\t0.2\t1\t529 31\n",
      "132\t495\t0.2\t1\t468 27\n",
      "133\t660\t0.2\t1\t626 34\n",
      "134\t1423\t0.2\t1\t1385 38\n",
      "135\t575\t0.2\t1\t556 19\n",
      "136\t211\t0.2\t1\t198 13\n",
      "137\t131\t0.2\t1\t127 4\n",
      "138\t83\t0.2\t1\t76 7\n",
      "139\t59\t0.2\t1\t52 7\n",
      "140\t46\t0.2\t1\t40 6\n",
      "141\t37\t0.2\t1\t35 2\n",
      "142\t28\t0.2\t1\t27 1\n",
      "143\t41\t0.2\t1\t38 3\n",
      "144\t59\t0.2\t1\t58 1\n",
      "145\t179\t0.2\t1\t178 1\n",
      "146\t263\t0.2\t1\t249 14\n",
      "147\t1229\t0.2\t1\t1166 63\n",
      "148\t791\t0.2\t1\t756 35\n",
      "149\t435\t0.2\t1\t415 20\n",
      "150\t2389\t0.2\t1\t2255 134\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "13781290 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz, file_2: CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13781290\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 36197 (0.26%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_8C_26psu_1_S17_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 532.54 s (39 us/read; 1.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,627,098\n",
      "Reads with adapters:                10,357,003 (76.0%)\n",
      "Reads written (passing filters):    13,627,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,044,064,700 bp\n",
      "Quality-trimmed:               4,208,565 bp (0.2%)\n",
      "Total written (filtered):  1,627,255,744 bp (79.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10357003 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.3%\n",
      "  C: 15.7%\n",
      "  G: 18.6%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2086122\t3406774.5\t0\t2086122\n",
      "2\t266568\t851693.6\t0\t266568\n",
      "3\t152542\t212923.4\t0\t152542\n",
      "4\t89399\t53230.9\t0\t89399\n",
      "5\t70209\t13307.7\t0\t70209\n",
      "6\t66787\t3326.9\t0\t66787\n",
      "7\t53852\t831.7\t0\t53852\n",
      "8\t67948\t207.9\t0\t67948\n",
      "9\t69323\t52.0\t0\t68200 1123\n",
      "10\t63939\t13.0\t1\t59604 4335\n",
      "11\t71265\t3.2\t1\t64858 6407\n",
      "12\t68721\t0.8\t1\t63494 5227\n",
      "13\t67596\t0.2\t1\t62302 5294\n",
      "14\t70216\t0.2\t1\t64057 6159\n",
      "15\t71638\t0.2\t1\t65898 5740\n",
      "16\t71622\t0.2\t1\t65051 6571\n",
      "17\t87796\t0.2\t1\t79687 8109\n",
      "18\t71999\t0.2\t1\t66604 5395\n",
      "19\t63344\t0.2\t1\t58390 4954\n",
      "20\t75324\t0.2\t1\t69407 5917\n",
      "21\t84178\t0.2\t1\t77194 6984\n",
      "22\t70154\t0.2\t1\t65038 5116\n",
      "23\t81908\t0.2\t1\t75394 6514\n",
      "24\t75905\t0.2\t1\t68761 7144\n",
      "25\t81978\t0.2\t1\t75339 6639\n",
      "26\t74054\t0.2\t1\t68713 5341\n",
      "27\t83694\t0.2\t1\t76279 7415\n",
      "28\t82366\t0.2\t1\t75736 6630\n",
      "29\t86579\t0.2\t1\t78687 7892\n",
      "30\t91938\t0.2\t1\t85372 6566\n",
      "31\t70649\t0.2\t1\t65160 5489\n",
      "32\t86162\t0.2\t1\t79280 6882\n",
      "33\t82680\t0.2\t1\t75408 7272\n",
      "34\t100154\t0.2\t1\t90695 9459\n",
      "35\t94358\t0.2\t1\t85860 8498\n",
      "36\t148458\t0.2\t1\t139524 8934\n",
      "37\t35889\t0.2\t1\t31244 4645\n",
      "38\t78400\t0.2\t1\t72661 5739\n",
      "39\t96881\t0.2\t1\t88590 8291\n",
      "40\t92469\t0.2\t1\t85322 7147\n",
      "41\t105619\t0.2\t1\t96453 9166\n",
      "42\t87195\t0.2\t1\t78548 8647\n",
      "43\t141743\t0.2\t1\t132343 9400\n",
      "44\t53443\t0.2\t1\t48101 5342\n",
      "45\t80210\t0.2\t1\t73892 6318\n",
      "46\t94587\t0.2\t1\t87548 7039\n",
      "47\t98921\t0.2\t1\t91675 7246\n",
      "48\t85740\t0.2\t1\t78755 6985\n",
      "49\t103381\t0.2\t1\t94837 8544\n",
      "50\t84900\t0.2\t1\t78738 6162\n",
      "51\t91291\t0.2\t1\t84483 6808\n",
      "52\t97726\t0.2\t1\t89264 8462\n",
      "53\t111539\t0.2\t1\t103945 7594\n",
      "54\t78508\t0.2\t1\t72278 6230\n",
      "55\t97750\t0.2\t1\t90509 7241\n",
      "56\t115622\t0.2\t1\t107685 7937\n",
      "57\t89129\t0.2\t1\t81822 7307\n",
      "58\t92584\t0.2\t1\t86772 5812\n",
      "59\t73435\t0.2\t1\t67515 5920\n",
      "60\t100185\t0.2\t1\t92834 7351\n",
      "61\t93396\t0.2\t1\t86891 6505\n",
      "62\t93505\t0.2\t1\t86821 6684\n",
      "63\t85463\t0.2\t1\t78881 6582\n",
      "64\t86705\t0.2\t1\t80704 6001\n",
      "65\t94221\t0.2\t1\t87522 6699\n",
      "66\t100565\t0.2\t1\t93273 7292\n",
      "67\t109840\t0.2\t1\t100139 9701\n",
      "68\t202049\t0.2\t1\t190563 11486\n",
      "69\t52634\t0.2\t1\t48583 4051\n",
      "70\t17146\t0.2\t1\t14429 2717\n",
      "71\t47197\t0.2\t1\t43653 3544\n",
      "72\t76160\t0.2\t1\t71164 4996\n",
      "73\t81740\t0.2\t1\t76081 5659\n",
      "74\t82050\t0.2\t1\t76332 5718\n",
      "75\t80287\t0.2\t1\t74463 5824\n",
      "76\t79839\t0.2\t1\t73807 6032\n",
      "77\t80016\t0.2\t1\t74278 5738\n",
      "78\t74795\t0.2\t1\t69265 5530\n",
      "79\t74038\t0.2\t1\t68576 5462\n",
      "80\t73576\t0.2\t1\t68130 5446\n",
      "81\t72243\t0.2\t1\t66987 5256\n",
      "82\t69871\t0.2\t1\t64751 5120\n",
      "83\t67762\t0.2\t1\t62929 4833\n",
      "84\t65749\t0.2\t1\t60930 4819\n",
      "85\t64210\t0.2\t1\t59436 4774\n",
      "86\t62228\t0.2\t1\t57683 4545\n",
      "87\t59511\t0.2\t1\t55197 4314\n",
      "88\t59103\t0.2\t1\t54779 4324\n",
      "89\t57029\t0.2\t1\t52911 4118\n",
      "90\t56111\t0.2\t1\t51929 4182\n",
      "91\t53588\t0.2\t1\t49754 3834\n",
      "92\t50869\t0.2\t1\t47115 3754\n",
      "93\t47789\t0.2\t1\t44363 3426\n",
      "94\t47204\t0.2\t1\t43712 3492\n",
      "95\t44549\t0.2\t1\t41263 3286\n",
      "96\t41699\t0.2\t1\t38638 3061\n",
      "97\t39043\t0.2\t1\t36086 2957\n",
      "98\t35735\t0.2\t1\t32882 2853\n",
      "99\t35123\t0.2\t1\t32338 2785\n",
      "100\t32434\t0.2\t1\t30000 2434\n",
      "101\t30205\t0.2\t1\t27895 2310\n",
      "102\t28238\t0.2\t1\t26075 2163\n",
      "103\t26099\t0.2\t1\t24128 1971\n",
      "104\t24785\t0.2\t1\t22897 1888\n",
      "105\t21850\t0.2\t1\t20176 1674\n",
      "106\t20336\t0.2\t1\t18803 1533\n",
      "107\t18435\t0.2\t1\t16991 1444\n",
      "108\t17298\t0.2\t1\t15900 1398\n",
      "109\t15693\t0.2\t1\t14418 1275\n",
      "110\t13857\t0.2\t1\t12707 1150\n",
      "111\t12767\t0.2\t1\t11710 1057\n",
      "112\t11323\t0.2\t1\t10411 912\n",
      "113\t9639\t0.2\t1\t8850 789\n",
      "114\t8830\t0.2\t1\t8052 778\n",
      "115\t7781\t0.2\t1\t7105 676\n",
      "116\t6721\t0.2\t1\t6151 570\n",
      "117\t5849\t0.2\t1\t5270 579\n",
      "118\t5202\t0.2\t1\t4706 496\n",
      "119\t4310\t0.2\t1\t3919 391\n",
      "120\t3867\t0.2\t1\t3492 375\n",
      "121\t3291\t0.2\t1\t2960 331\n",
      "122\t2743\t0.2\t1\t2485 258\n",
      "123\t2310\t0.2\t1\t2091 219\n",
      "124\t2101\t0.2\t1\t1881 220\n",
      "125\t1771\t0.2\t1\t1559 212\n",
      "126\t1539\t0.2\t1\t1390 149\n",
      "127\t1290\t0.2\t1\t1141 149\n",
      "128\t1070\t0.2\t1\t950 120\n",
      "129\t924\t0.2\t1\t833 91\n",
      "130\t685\t0.2\t1\t617 68\n",
      "131\t612\t0.2\t1\t545 67\n",
      "132\t539\t0.2\t1\t487 52\n",
      "133\t666\t0.2\t1\t596 70\n",
      "134\t1493\t0.2\t1\t1414 79\n",
      "135\t574\t0.2\t1\t548 26\n",
      "136\t201\t0.2\t1\t183 18\n",
      "137\t159\t0.2\t1\t145 14\n",
      "138\t108\t0.2\t1\t98 10\n",
      "139\t49\t0.2\t1\t44 5\n",
      "140\t42\t0.2\t1\t38 4\n",
      "141\t43\t0.2\t1\t37 6\n",
      "142\t35\t0.2\t1\t31 4\n",
      "143\t45\t0.2\t1\t39 6\n",
      "144\t56\t0.2\t1\t49 7\n",
      "145\t155\t0.2\t1\t139 16\n",
      "146\t240\t0.2\t1\t219 21\n",
      "147\t1341\t0.2\t1\t1228 113\n",
      "148\t773\t0.2\t1\t707 66\n",
      "149\t433\t0.2\t1\t399 34\n",
      "150\t2986\t0.2\t1\t2547 439\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "13627098 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 568.20 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              13,627,098\n",
      "Reads with adapters:                10,276,263 (75.4%)\n",
      "Reads written (passing filters):    13,627,098 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,044,064,700 bp\n",
      "Quality-trimmed:               9,937,551 bp (0.5%)\n",
      "Total written (filtered):  1,630,739,955 bp (79.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 10276263 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 12.4%\n",
      "  G: 23.0%\n",
      "  T: 34.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2146357\t3406774.5\t0\t2146357\n",
      "2\t269360\t851693.6\t0\t269360\n",
      "3\t157893\t212923.4\t0\t157893\n",
      "4\t87930\t53230.9\t0\t87930\n",
      "5\t73543\t13307.7\t0\t73543\n",
      "6\t64791\t3326.9\t0\t64791\n",
      "7\t52189\t831.7\t0\t52189\n",
      "8\t74756\t207.9\t0\t74756\n",
      "9\t60599\t52.0\t0\t60366 233\n",
      "10\t63543\t13.0\t1\t59315 4228\n",
      "11\t68667\t3.2\t1\t63268 5399\n",
      "12\t67618\t0.8\t1\t62224 5394\n",
      "13\t77226\t0.2\t1\t71131 6095\n",
      "14\t69709\t0.2\t1\t63900 5809\n",
      "15\t58686\t0.2\t1\t54055 4631\n",
      "16\t84262\t0.2\t1\t77680 6582\n",
      "17\t67924\t0.2\t1\t62515 5409\n",
      "18\t64114\t0.2\t1\t59144 4970\n",
      "19\t80499\t0.2\t1\t74211 6288\n",
      "20\t76034\t0.2\t1\t70274 5760\n",
      "21\t67847\t0.2\t1\t62554 5293\n",
      "22\t74609\t0.2\t1\t68953 5656\n",
      "23\t78584\t0.2\t1\t72661 5923\n",
      "24\t77507\t0.2\t1\t71592 5915\n",
      "25\t87777\t0.2\t1\t81767 6010\n",
      "26\t72611\t0.2\t1\t66753 5858\n",
      "27\t83123\t0.2\t1\t76411 6712\n",
      "28\t73079\t0.2\t1\t68272 4807\n",
      "29\t82291\t0.2\t1\t76291 6000\n",
      "30\t76541\t0.2\t1\t71237 5304\n",
      "31\t89563\t0.2\t1\t83077 6486\n",
      "32\t74346\t0.2\t1\t69371 4975\n",
      "33\t86840\t0.2\t1\t80487 6353\n",
      "34\t112360\t0.2\t1\t104461 7899\n",
      "35\t60619\t0.2\t1\t56337 4282\n",
      "36\t90028\t0.2\t1\t83414 6614\n",
      "37\t96370\t0.2\t1\t89511 6859\n",
      "38\t89130\t0.2\t1\t83444 5686\n",
      "39\t86200\t0.2\t1\t80889 5311\n",
      "40\t97029\t0.2\t1\t90271 6758\n",
      "41\t93202\t0.2\t1\t87114 6088\n",
      "42\t93307\t0.2\t1\t87268 6039\n",
      "43\t83469\t0.2\t1\t77943 5526\n",
      "44\t97590\t0.2\t1\t91014 6576\n",
      "45\t100276\t0.2\t1\t93792 6484\n",
      "46\t75444\t0.2\t1\t70447 4997\n",
      "47\t93307\t0.2\t1\t87603 5704\n",
      "48\t90968\t0.2\t1\t85370 5598\n",
      "49\t95545\t0.2\t1\t89305 6240\n",
      "50\t91141\t0.2\t1\t85291 5850\n",
      "51\t108887\t0.2\t1\t102107 6780\n",
      "52\t80355\t0.2\t1\t75335 5020\n",
      "53\t99048\t0.2\t1\t93311 5737\n",
      "54\t72741\t0.2\t1\t68334 4407\n",
      "55\t95947\t0.2\t1\t90438 5509\n",
      "56\t99513\t0.2\t1\t93368 6145\n",
      "57\t93556\t0.2\t1\t88452 5104\n",
      "58\t90539\t0.2\t1\t85415 5124\n",
      "59\t90467\t0.2\t1\t85091 5376\n",
      "60\t93608\t0.2\t1\t88124 5484\n",
      "61\t95101\t0.2\t1\t89698 5403\n",
      "62\t102226\t0.2\t1\t96046 6180\n",
      "63\t136011\t0.2\t1\t129004 7007\n",
      "64\t112722\t0.2\t1\t107622 5100\n",
      "65\t75623\t0.2\t1\t71479 4144\n",
      "66\t81004\t0.2\t1\t76564 4440\n",
      "67\t79874\t0.2\t1\t75268 4606\n",
      "68\t81531\t0.2\t1\t76952 4579\n",
      "69\t80094\t0.2\t1\t75476 4618\n",
      "70\t85599\t0.2\t1\t80759 4840\n",
      "71\t89528\t0.2\t1\t84657 4871\n",
      "72\t84362\t0.2\t1\t79637 4725\n",
      "73\t87554\t0.2\t1\t82710 4844\n",
      "74\t81739\t0.2\t1\t77139 4600\n",
      "75\t76846\t0.2\t1\t72538 4308\n",
      "76\t75616\t0.2\t1\t71298 4318\n",
      "77\t75671\t0.2\t1\t71375 4296\n",
      "78\t71297\t0.2\t1\t67330 3967\n",
      "79\t68959\t0.2\t1\t65062 3897\n",
      "80\t69385\t0.2\t1\t65444 3941\n",
      "81\t68113\t0.2\t1\t64379 3734\n",
      "82\t65128\t0.2\t1\t61578 3550\n",
      "83\t63398\t0.2\t1\t59890 3508\n",
      "84\t61642\t0.2\t1\t58327 3315\n",
      "85\t61077\t0.2\t1\t57799 3278\n",
      "86\t60708\t0.2\t1\t57467 3241\n",
      "87\t60697\t0.2\t1\t57594 3103\n",
      "88\t59562\t0.2\t1\t56463 3099\n",
      "89\t54239\t0.2\t1\t51398 2841\n",
      "90\t51773\t0.2\t1\t48980 2793\n",
      "91\t49319\t0.2\t1\t46756 2563\n",
      "92\t46223\t0.2\t1\t43736 2487\n",
      "93\t44798\t0.2\t1\t42455 2343\n",
      "94\t43852\t0.2\t1\t41584 2268\n",
      "95\t41908\t0.2\t1\t39647 2261\n",
      "96\t39622\t0.2\t1\t37536 2086\n",
      "97\t36420\t0.2\t1\t34621 1799\n",
      "98\t33185\t0.2\t1\t31489 1696\n",
      "99\t32771\t0.2\t1\t31125 1646\n",
      "100\t29865\t0.2\t1\t28321 1544\n",
      "101\t27753\t0.2\t1\t26308 1445\n",
      "102\t25712\t0.2\t1\t24349 1363\n",
      "103\t23843\t0.2\t1\t22528 1315\n",
      "104\t23330\t0.2\t1\t22084 1246\n",
      "105\t20720\t0.2\t1\t19658 1062\n",
      "106\t19342\t0.2\t1\t18364 978\n",
      "107\t17248\t0.2\t1\t16375 873\n",
      "108\t15847\t0.2\t1\t15052 795\n",
      "109\t14536\t0.2\t1\t13807 729\n",
      "110\t12976\t0.2\t1\t12292 684\n",
      "111\t12529\t0.2\t1\t11885 644\n",
      "112\t10963\t0.2\t1\t10373 590\n",
      "113\t9349\t0.2\t1\t8908 441\n",
      "114\t8548\t0.2\t1\t8120 428\n",
      "115\t7299\t0.2\t1\t6916 383\n",
      "116\t6237\t0.2\t1\t5941 296\n",
      "117\t5243\t0.2\t1\t4920 323\n",
      "118\t4696\t0.2\t1\t4465 231\n",
      "119\t3975\t0.2\t1\t3763 212\n",
      "120\t3508\t0.2\t1\t3309 199\n",
      "121\t3040\t0.2\t1\t2875 165\n",
      "122\t2575\t0.2\t1\t2428 147\n",
      "123\t2186\t0.2\t1\t2073 113\n",
      "124\t1945\t0.2\t1\t1832 113\n",
      "125\t1652\t0.2\t1\t1551 101\n",
      "126\t1464\t0.2\t1\t1388 76\n",
      "127\t1181\t0.2\t1\t1120 61\n",
      "128\t990\t0.2\t1\t926 64\n",
      "129\t867\t0.2\t1\t802 65\n",
      "130\t639\t0.2\t1\t610 29\n",
      "131\t564\t0.2\t1\t532 32\n",
      "132\t517\t0.2\t1\t490 27\n",
      "133\t691\t0.2\t1\t653 38\n",
      "134\t1520\t0.2\t1\t1458 62\n",
      "135\t553\t0.2\t1\t527 26\n",
      "136\t201\t0.2\t1\t194 7\n",
      "137\t155\t0.2\t1\t146 9\n",
      "138\t95\t0.2\t1\t92 3\n",
      "139\t52\t0.2\t1\t49 3\n",
      "140\t44\t0.2\t1\t40 4\n",
      "141\t42\t0.2\t1\t38 4\n",
      "142\t33\t0.2\t1\t31 2\n",
      "143\t49\t0.2\t1\t44 5\n",
      "144\t63\t0.2\t1\t60 3\n",
      "145\t162\t0.2\t1\t155 7\n",
      "146\t237\t0.2\t1\t226 11\n",
      "147\t1276\t0.2\t1\t1212 64\n",
      "148\t739\t0.2\t1\t700 39\n",
      "149\t423\t0.2\t1\t396 27\n",
      "150\t2548\t0.2\t1\t2400 148\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "13627098 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz, file_2: CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 13627098\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 34739 (0.25%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_1_S17_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_8C_26psu_1_S17_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_1_S17_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 540.37 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,982,389\n",
      "Reads with adapters:                15,316,406 (90.2%)\n",
      "Reads written (passing filters):    16,982,389 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,547,358,350 bp\n",
      "Quality-trimmed:               5,122,052 bp (0.2%)\n",
      "Total written (filtered):  1,661,718,543 bp (65.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15316406 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 17.9%\n",
      "  G: 21.3%\n",
      "  T: 32.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t934198\t4245597.2\t0\t934198\n",
      "2\t161909\t1061399.3\t0\t161909\n",
      "3\t105071\t265349.8\t0\t105071\n",
      "4\t65361\t66337.5\t0\t65361\n",
      "5\t61151\t16584.4\t0\t61151\n",
      "6\t60130\t4146.1\t0\t60130\n",
      "7\t51265\t1036.5\t0\t51265\n",
      "8\t64755\t259.1\t0\t64755\n",
      "9\t64351\t64.8\t0\t63580 771\n",
      "10\t61337\t16.2\t1\t58020 3317\n",
      "11\t70221\t4.0\t1\t64705 5516\n",
      "12\t66216\t1.0\t1\t61889 4327\n",
      "13\t69137\t0.3\t1\t64630 4507\n",
      "14\t70780\t0.3\t1\t65742 5038\n",
      "15\t75204\t0.3\t1\t70097 5107\n",
      "16\t73779\t0.3\t1\t68019 5760\n",
      "17\t90043\t0.3\t1\t82540 7503\n",
      "18\t81214\t0.3\t1\t76392 4822\n",
      "19\t66550\t0.3\t1\t62390 4160\n",
      "20\t83867\t0.3\t1\t78220 5647\n",
      "21\t92423\t0.3\t1\t86031 6392\n",
      "22\t79989\t0.3\t1\t75306 4683\n",
      "23\t93784\t0.3\t1\t87424 6360\n",
      "24\t89031\t0.3\t1\t81718 7313\n",
      "25\t96197\t0.3\t1\t89537 6660\n",
      "26\t88926\t0.3\t1\t83710 5216\n",
      "27\t100077\t0.3\t1\t92668 7409\n",
      "28\t100613\t0.3\t1\t93553 7060\n",
      "29\t104938\t0.3\t1\t96491 8447\n",
      "30\t117488\t0.3\t1\t110880 6608\n",
      "31\t89018\t0.3\t1\t83492 5526\n",
      "32\t107667\t0.3\t1\t100716 6951\n",
      "33\t110564\t0.3\t1\t102547 8017\n",
      "34\t137673\t0.3\t1\t127216 10457\n",
      "35\t119092\t0.3\t1\t112273 6819\n",
      "36\t94075\t0.3\t1\t88092 5983\n",
      "37\t125638\t0.3\t1\t116775 8863\n",
      "38\t138305\t0.3\t1\t128145 10160\n",
      "39\t120373\t0.3\t1\t113314 7059\n",
      "40\t138522\t0.3\t1\t128397 10125\n",
      "41\t158012\t0.3\t1\t146728 11284\n",
      "42\t111371\t0.3\t1\t103118 8253\n",
      "43\t217278\t0.3\t1\t206218 11060\n",
      "44\t57407\t0.3\t1\t51985 5422\n",
      "45\t143497\t0.3\t1\t134258 9239\n",
      "46\t142849\t0.3\t1\t133725 9124\n",
      "47\t147972\t0.3\t1\t139076 8896\n",
      "48\t148858\t0.3\t1\t138933 9925\n",
      "49\t163896\t0.3\t1\t152740 11156\n",
      "50\t140197\t0.3\t1\t131951 8246\n",
      "51\t154425\t0.3\t1\t144692 9733\n",
      "52\t163399\t0.3\t1\t151525 11874\n",
      "53\t181558\t0.3\t1\t170911 10647\n",
      "54\t136765\t0.3\t1\t128031 8734\n",
      "55\t172905\t0.3\t1\t161877 11028\n",
      "56\t211987\t0.3\t1\t200164 11823\n",
      "57\t138899\t0.3\t1\t129027 9872\n",
      "58\t181048\t0.3\t1\t171675 9373\n",
      "59\t152882\t0.3\t1\t143208 9674\n",
      "60\t179132\t0.3\t1\t167976 11156\n",
      "61\t179957\t0.3\t1\t169292 10665\n",
      "62\t195431\t0.3\t1\t184537 10894\n",
      "63\t158140\t0.3\t1\t148011 10129\n",
      "64\t177885\t0.3\t1\t167763 10122\n",
      "65\t200563\t0.3\t1\t188662 11901\n",
      "66\t211859\t0.3\t1\t199050 12809\n",
      "67\t228942\t0.3\t1\t211658 17284\n",
      "68\t447452\t0.3\t1\t426829 20623\n",
      "69\t83220\t0.3\t1\t76361 6859\n",
      "70\t30543\t0.3\t1\t25376 5167\n",
      "71\t119720\t0.3\t1\t112578 7142\n",
      "72\t172151\t0.3\t1\t162467 9684\n",
      "73\t185670\t0.3\t1\t174903 10767\n",
      "74\t186107\t0.3\t1\t175086 11021\n",
      "75\t187142\t0.3\t1\t176040 11102\n",
      "76\t187583\t0.3\t1\t176804 10779\n",
      "77\t190166\t0.3\t1\t178927 11239\n",
      "78\t180337\t0.3\t1\t169693 10644\n",
      "79\t182061\t0.3\t1\t171352 10709\n",
      "80\t179279\t0.3\t1\t168912 10367\n",
      "81\t180871\t0.3\t1\t170399 10472\n",
      "82\t177227\t0.3\t1\t166777 10450\n",
      "83\t175671\t0.3\t1\t165379 10292\n",
      "84\t171899\t0.3\t1\t161834 10065\n",
      "85\t167562\t0.3\t1\t157917 9645\n",
      "86\t164690\t0.3\t1\t155117 9573\n",
      "87\t159808\t0.3\t1\t150364 9444\n",
      "88\t160575\t0.3\t1\t151109 9466\n",
      "89\t154675\t0.3\t1\t145660 9015\n",
      "90\t155657\t0.3\t1\t146266 9391\n",
      "91\t151129\t0.3\t1\t142165 8964\n",
      "92\t144841\t0.3\t1\t136295 8546\n",
      "93\t137285\t0.3\t1\t129298 7987\n",
      "94\t135380\t0.3\t1\t127295 8085\n",
      "95\t127081\t0.3\t1\t119343 7738\n",
      "96\t122579\t0.3\t1\t115210 7369\n",
      "97\t117441\t0.3\t1\t110330 7111\n",
      "98\t110426\t0.3\t1\t103591 6835\n",
      "99\t109189\t0.3\t1\t102159 7030\n",
      "100\t100853\t0.3\t1\t94763 6090\n",
      "101\t96345\t0.3\t1\t90423 5922\n",
      "102\t89845\t0.3\t1\t84149 5696\n",
      "103\t85390\t0.3\t1\t80150 5240\n",
      "104\t79941\t0.3\t1\t74924 5017\n",
      "105\t72924\t0.3\t1\t68266 4658\n",
      "106\t67356\t0.3\t1\t63014 4342\n",
      "107\t62697\t0.3\t1\t58614 4083\n",
      "108\t60572\t0.3\t1\t56580 3992\n",
      "109\t54023\t0.3\t1\t50378 3645\n",
      "110\t49550\t0.3\t1\t46112 3438\n",
      "111\t45365\t0.3\t1\t42247 3118\n",
      "112\t40518\t0.3\t1\t37628 2890\n",
      "113\t35484\t0.3\t1\t32846 2638\n",
      "114\t32042\t0.3\t1\t29706 2336\n",
      "115\t28111\t0.3\t1\t26004 2107\n",
      "116\t25248\t0.3\t1\t23329 1919\n",
      "117\t21991\t0.3\t1\t20192 1799\n",
      "118\t18981\t0.3\t1\t17374 1607\n",
      "119\t16315\t0.3\t1\t14943 1372\n",
      "120\t14190\t0.3\t1\t13063 1127\n",
      "121\t12626\t0.3\t1\t11644 982\n",
      "122\t10330\t0.3\t1\t9443 887\n",
      "123\t9025\t0.3\t1\t8218 807\n",
      "124\t7634\t0.3\t1\t6995 639\n",
      "125\t6434\t0.3\t1\t5834 600\n",
      "126\t5520\t0.3\t1\t5002 518\n",
      "127\t4681\t0.3\t1\t4249 432\n",
      "128\t3676\t0.3\t1\t3257 419\n",
      "129\t3146\t0.3\t1\t2839 307\n",
      "130\t2431\t0.3\t1\t2189 242\n",
      "131\t2160\t0.3\t1\t1904 256\n",
      "132\t1916\t0.3\t1\t1719 197\n",
      "133\t2331\t0.3\t1\t2120 211\n",
      "134\t4696\t0.3\t1\t4441 255\n",
      "135\t1717\t0.3\t1\t1602 115\n",
      "136\t624\t0.3\t1\t562 62\n",
      "137\t434\t0.3\t1\t390 44\n",
      "138\t279\t0.3\t1\t239 40\n",
      "139\t193\t0.3\t1\t171 22\n",
      "140\t139\t0.3\t1\t121 18\n",
      "141\t116\t0.3\t1\t101 15\n",
      "142\t99\t0.3\t1\t87 12\n",
      "143\t71\t0.3\t1\t56 15\n",
      "144\t68\t0.3\t1\t49 19\n",
      "145\t142\t0.3\t1\t121 21\n",
      "146\t149\t0.3\t1\t136 13\n",
      "147\t732\t0.3\t1\t658 74\n",
      "148\t489\t0.3\t1\t451 38\n",
      "149\t276\t0.3\t1\t251 25\n",
      "150\t3068\t0.3\t1\t2637 431\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "16982389 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 565.89 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,982,389\n",
      "Reads with adapters:                15,175,955 (89.4%)\n",
      "Reads written (passing filters):    16,982,389 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,547,358,350 bp\n",
      "Quality-trimmed:              14,637,859 bp (0.6%)\n",
      "Total written (filtered):  1,670,081,107 bp (65.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15175955 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.2%\n",
      "  C: 14.2%\n",
      "  G: 26.5%\n",
      "  T: 33.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t988355\t4245597.2\t0\t988355\n",
      "2\t174327\t1061399.3\t0\t174327\n",
      "3\t112326\t265349.8\t0\t112326\n",
      "4\t66944\t66337.5\t0\t66944\n",
      "5\t68049\t16584.4\t0\t68049\n",
      "6\t59622\t4146.1\t0\t59622\n",
      "7\t48077\t1036.5\t0\t48077\n",
      "8\t70632\t259.1\t0\t70632\n",
      "9\t56419\t64.8\t0\t56220 199\n",
      "10\t62892\t16.2\t1\t59906 2986\n",
      "11\t67246\t4.0\t1\t63200 4046\n",
      "12\t67488\t1.0\t1\t63307 4181\n",
      "13\t76798\t0.3\t1\t72553 4245\n",
      "14\t73938\t0.3\t1\t68873 5065\n",
      "15\t59370\t0.3\t1\t55860 3510\n",
      "16\t87943\t0.3\t1\t82783 5160\n",
      "17\t71376\t0.3\t1\t66695 4681\n",
      "18\t68792\t0.3\t1\t64885 3907\n",
      "19\t88616\t0.3\t1\t83090 5526\n",
      "20\t83852\t0.3\t1\t79094 4758\n",
      "21\t75082\t0.3\t1\t70636 4446\n",
      "22\t86387\t0.3\t1\t81609 4778\n",
      "23\t90257\t0.3\t1\t85055 5202\n",
      "24\t90691\t0.3\t1\t85295 5396\n",
      "25\t104055\t0.3\t1\t98264 5791\n",
      "26\t86753\t0.3\t1\t81712 5041\n",
      "27\t96821\t0.3\t1\t90516 6305\n",
      "28\t92589\t0.3\t1\t88280 4309\n",
      "29\t100006\t0.3\t1\t94283 5723\n",
      "30\t100693\t0.3\t1\t95722 4971\n",
      "31\t110238\t0.3\t1\t104335 5903\n",
      "32\t99509\t0.3\t1\t94661 4848\n",
      "33\t112333\t0.3\t1\t106128 6205\n",
      "34\t145441\t0.3\t1\t137472 7969\n",
      "35\t84427\t0.3\t1\t80249 4178\n",
      "36\t119286\t0.3\t1\t112593 6693\n",
      "37\t127293\t0.3\t1\t120329 6964\n",
      "38\t123047\t0.3\t1\t117261 5786\n",
      "39\t119775\t0.3\t1\t114281 5494\n",
      "40\t135696\t0.3\t1\t128262 7434\n",
      "41\t130689\t0.3\t1\t124366 6323\n",
      "42\t135604\t0.3\t1\t129108 6496\n",
      "43\t126835\t0.3\t1\t120741 6094\n",
      "44\t146922\t0.3\t1\t139741 7181\n",
      "45\t154339\t0.3\t1\t146909 7430\n",
      "46\t118308\t0.3\t1\t112832 5476\n",
      "47\t149295\t0.3\t1\t142485 6810\n",
      "48\t148606\t0.3\t1\t142089 6517\n",
      "49\t156890\t0.3\t1\t149693 7197\n",
      "50\t148375\t0.3\t1\t141320 7055\n",
      "51\t190171\t0.3\t1\t181254 8917\n",
      "52\t136804\t0.3\t1\t130968 5836\n",
      "53\t165906\t0.3\t1\t159264 6642\n",
      "54\t128157\t0.3\t1\t122871 5286\n",
      "55\t169343\t0.3\t1\t162704 6639\n",
      "56\t177473\t0.3\t1\t169804 7669\n",
      "57\t169211\t0.3\t1\t162646 6565\n",
      "58\t164197\t0.3\t1\t158052 6145\n",
      "59\t169249\t0.3\t1\t162429 6820\n",
      "60\t177455\t0.3\t1\t170133 7322\n",
      "61\t185762\t0.3\t1\t177933 7829\n",
      "62\t204728\t0.3\t1\t196280 8448\n",
      "63\t277535\t0.3\t1\t267196 10339\n",
      "64\t238674\t0.3\t1\t230905 7769\n",
      "65\t152399\t0.3\t1\t146631 5768\n",
      "66\t166909\t0.3\t1\t160634 6275\n",
      "67\t166804\t0.3\t1\t160075 6729\n",
      "68\t169255\t0.3\t1\t162913 6342\n",
      "69\t169940\t0.3\t1\t163142 6798\n",
      "70\t182641\t0.3\t1\t175571 7070\n",
      "71\t198941\t0.3\t1\t191661 7280\n",
      "72\t188343\t0.3\t1\t181180 7163\n",
      "73\t204850\t0.3\t1\t197436 7414\n",
      "74\t188381\t0.3\t1\t181668 6713\n",
      "75\t180633\t0.3\t1\t173941 6692\n",
      "76\t178500\t0.3\t1\t172078 6422\n",
      "77\t180330\t0.3\t1\t173546 6784\n",
      "78\t173641\t0.3\t1\t167180 6461\n",
      "79\t170287\t0.3\t1\t164090 6197\n",
      "80\t170838\t0.3\t1\t164653 6185\n",
      "81\t172175\t0.3\t1\t165934 6241\n",
      "82\t165553\t0.3\t1\t159555 5998\n",
      "83\t164742\t0.3\t1\t158792 5950\n",
      "84\t160682\t0.3\t1\t154778 5904\n",
      "85\t159281\t0.3\t1\t153367 5914\n",
      "86\t160523\t0.3\t1\t154638 5885\n",
      "87\t163301\t0.3\t1\t157413 5888\n",
      "88\t161676\t0.3\t1\t155871 5805\n",
      "89\t147453\t0.3\t1\t142105 5348\n",
      "90\t143818\t0.3\t1\t138625 5193\n",
      "91\t138225\t0.3\t1\t133103 5122\n",
      "92\t131088\t0.3\t1\t126317 4771\n",
      "93\t128053\t0.3\t1\t123615 4438\n",
      "94\t125505\t0.3\t1\t121074 4431\n",
      "95\t119294\t0.3\t1\t114964 4330\n",
      "96\t116155\t0.3\t1\t112159 3996\n",
      "97\t108812\t0.3\t1\t105094 3718\n",
      "98\t104976\t0.3\t1\t101373 3603\n",
      "99\t105895\t0.3\t1\t102281 3614\n",
      "100\t97778\t0.3\t1\t94611 3167\n",
      "101\t95386\t0.3\t1\t92351 3035\n",
      "102\t88691\t0.3\t1\t85785 2906\n",
      "103\t80552\t0.3\t1\t77853 2699\n",
      "104\t77505\t0.3\t1\t74941 2564\n",
      "105\t70382\t0.3\t1\t68000 2382\n",
      "106\t65184\t0.3\t1\t63051 2133\n",
      "107\t60057\t0.3\t1\t58146 1911\n",
      "108\t55681\t0.3\t1\t53827 1854\n",
      "109\t50629\t0.3\t1\t48896 1733\n",
      "110\t46572\t0.3\t1\t45046 1526\n",
      "111\t44577\t0.3\t1\t43131 1446\n",
      "112\t39344\t0.3\t1\t38114 1230\n",
      "113\t34530\t0.3\t1\t33382 1148\n",
      "114\t30993\t0.3\t1\t30031 962\n",
      "115\t26355\t0.3\t1\t25472 883\n",
      "116\t23180\t0.3\t1\t22328 852\n",
      "117\t19825\t0.3\t1\t19141 684\n",
      "118\t17184\t0.3\t1\t16546 638\n",
      "119\t14938\t0.3\t1\t14436 502\n",
      "120\t12856\t0.3\t1\t12387 469\n",
      "121\t11667\t0.3\t1\t11246 421\n",
      "122\t9613\t0.3\t1\t9258 355\n",
      "123\t8582\t0.3\t1\t8274 308\n",
      "124\t7156\t0.3\t1\t6897 259\n",
      "125\t6130\t0.3\t1\t5897 233\n",
      "126\t5195\t0.3\t1\t5039 156\n",
      "127\t4372\t0.3\t1\t4226 146\n",
      "128\t3397\t0.3\t1\t3245 152\n",
      "129\t2899\t0.3\t1\t2785 114\n",
      "130\t2231\t0.3\t1\t2140 91\n",
      "131\t2005\t0.3\t1\t1925 80\n",
      "132\t1815\t0.3\t1\t1740 75\n",
      "133\t2411\t0.3\t1\t2321 90\n",
      "134\t4708\t0.3\t1\t4577 131\n",
      "135\t1657\t0.3\t1\t1601 56\n",
      "136\t589\t0.3\t1\t574 15\n",
      "137\t406\t0.3\t1\t391 15\n",
      "138\t265\t0.3\t1\t252 13\n",
      "139\t182\t0.3\t1\t175 7\n",
      "140\t132\t0.3\t1\t124 8\n",
      "141\t111\t0.3\t1\t106 5\n",
      "142\t101\t0.3\t1\t96 5\n",
      "143\t63\t0.3\t1\t60 3\n",
      "144\t61\t0.3\t1\t57 4\n",
      "145\t160\t0.3\t1\t154 6\n",
      "146\t146\t0.3\t1\t139 7\n",
      "147\t687\t0.3\t1\t666 21\n",
      "148\t439\t0.3\t1\t426 13\n",
      "149\t260\t0.3\t1\t251 9\n",
      "150\t2448\t0.3\t1\t2325 123\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "16982389 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz, file_2: CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 16982389\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 67701 (0.40%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_8C_26psu_2_S18_L001_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 563.12 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,972,486\n",
      "Reads with adapters:                15,279,419 (90.0%)\n",
      "Reads written (passing filters):    16,972,486 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,545,872,900 bp\n",
      "Quality-trimmed:               5,170,443 bp (0.2%)\n",
      "Total written (filtered):  1,660,911,595 bp (65.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15279419 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 17.9%\n",
      "  G: 21.2%\n",
      "  T: 32.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t937714\t4243121.5\t0\t937714\n",
      "2\t159877\t1060780.4\t0\t159877\n",
      "3\t104645\t265195.1\t0\t104645\n",
      "4\t64394\t66298.8\t0\t64394\n",
      "5\t60206\t16574.7\t0\t60206\n",
      "6\t60143\t4143.7\t0\t60143\n",
      "7\t49779\t1035.9\t0\t49779\n",
      "8\t63607\t259.0\t0\t63607\n",
      "9\t63633\t64.7\t0\t62858 775\n",
      "10\t59492\t16.2\t1\t55609 3883\n",
      "11\t70292\t4.0\t1\t63860 6432\n",
      "12\t65718\t1.0\t1\t60580 5138\n",
      "13\t68404\t0.3\t1\t63070 5334\n",
      "14\t69496\t0.3\t1\t63637 5859\n",
      "15\t74672\t0.3\t1\t68604 6068\n",
      "16\t72665\t0.3\t1\t66207 6458\n",
      "17\t90074\t0.3\t1\t81360 8714\n",
      "18\t80053\t0.3\t1\t74306 5747\n",
      "19\t65594\t0.3\t1\t60638 4956\n",
      "20\t83351\t0.3\t1\t76880 6471\n",
      "21\t92562\t0.3\t1\t85115 7447\n",
      "22\t78449\t0.3\t1\t72991 5458\n",
      "23\t93136\t0.3\t1\t85970 7166\n",
      "24\t87973\t0.3\t1\t80003 7970\n",
      "25\t95468\t0.3\t1\t87873 7595\n",
      "26\t87079\t0.3\t1\t80742 6337\n",
      "27\t100093\t0.3\t1\t91614 8479\n",
      "28\t99879\t0.3\t1\t92166 7713\n",
      "29\t105478\t0.3\t1\t95962 9516\n",
      "30\t116678\t0.3\t1\t108856 7822\n",
      "31\t86678\t0.3\t1\t80098 6580\n",
      "32\t107759\t0.3\t1\t99613 8146\n",
      "33\t109432\t0.3\t1\t99874 9558\n",
      "34\t138284\t0.3\t1\t125790 12494\n",
      "35\t117450\t0.3\t1\t109376 8074\n",
      "36\t91614\t0.3\t1\t84735 6879\n",
      "37\t124777\t0.3\t1\t114386 10391\n",
      "38\t138573\t0.3\t1\t126348 12225\n",
      "39\t118413\t0.3\t1\t109918 8495\n",
      "40\t138324\t0.3\t1\t126104 12220\n",
      "41\t158476\t0.3\t1\t144761 13715\n",
      "42\t109596\t0.3\t1\t99995 9601\n",
      "43\t214148\t0.3\t1\t200447 13701\n",
      "44\t58836\t0.3\t1\t52468 6368\n",
      "45\t143215\t0.3\t1\t132338 10877\n",
      "46\t142842\t0.3\t1\t132071 10771\n",
      "47\t146328\t0.3\t1\t136022 10306\n",
      "48\t147735\t0.3\t1\t136182 11553\n",
      "49\t164993\t0.3\t1\t151819 13174\n",
      "50\t138531\t0.3\t1\t129039 9492\n",
      "51\t154359\t0.3\t1\t142921 11438\n",
      "52\t163104\t0.3\t1\t149477 13627\n",
      "53\t181208\t0.3\t1\t168727 12481\n",
      "54\t135823\t0.3\t1\t125879 9944\n",
      "55\t172933\t0.3\t1\t159987 12946\n",
      "56\t212770\t0.3\t1\t198564 14206\n",
      "57\t137905\t0.3\t1\t126707 11198\n",
      "58\t179533\t0.3\t1\t168232 11301\n",
      "59\t154787\t0.3\t1\t143238 11549\n",
      "60\t178048\t0.3\t1\t165291 12757\n",
      "61\t178798\t0.3\t1\t166319 12479\n",
      "62\t194586\t0.3\t1\t181612 12974\n",
      "63\t159546\t0.3\t1\t147478 12068\n",
      "64\t175838\t0.3\t1\t164219 11619\n",
      "65\t199150\t0.3\t1\t185246 13904\n",
      "66\t207681\t0.3\t1\t193071 14610\n",
      "67\t227820\t0.3\t1\t208839 18981\n",
      "68\t430170\t0.3\t1\t406697 23473\n",
      "69\t101761\t0.3\t1\t93311 8450\n",
      "70\t37741\t0.3\t1\t31764 5977\n",
      "71\t116561\t0.3\t1\t108649 7912\n",
      "72\t170878\t0.3\t1\t160245 10633\n",
      "73\t185808\t0.3\t1\t173840 11968\n",
      "74\t185713\t0.3\t1\t173236 12477\n",
      "75\t186685\t0.3\t1\t173898 12787\n",
      "76\t187774\t0.3\t1\t174524 13250\n",
      "77\t190253\t0.3\t1\t177207 13046\n",
      "78\t180228\t0.3\t1\t167656 12572\n",
      "79\t181970\t0.3\t1\t169235 12735\n",
      "80\t180269\t0.3\t1\t167504 12765\n",
      "81\t180129\t0.3\t1\t167834 12295\n",
      "82\t176778\t0.3\t1\t164262 12516\n",
      "83\t174648\t0.3\t1\t162652 11996\n",
      "84\t170683\t0.3\t1\t158618 12065\n",
      "85\t167207\t0.3\t1\t155648 11559\n",
      "86\t165775\t0.3\t1\t154402 11373\n",
      "87\t159710\t0.3\t1\t148769 10941\n",
      "88\t161141\t0.3\t1\t150001 11140\n",
      "89\t154725\t0.3\t1\t143979 10746\n",
      "90\t156314\t0.3\t1\t145746 10568\n",
      "91\t151370\t0.3\t1\t140729 10641\n",
      "92\t145834\t0.3\t1\t135746 10088\n",
      "93\t137534\t0.3\t1\t127986 9548\n",
      "94\t136265\t0.3\t1\t126825 9440\n",
      "95\t127646\t0.3\t1\t118675 8971\n",
      "96\t123166\t0.3\t1\t114494 8672\n",
      "97\t117318\t0.3\t1\t109181 8137\n",
      "98\t111017\t0.3\t1\t103130 7887\n",
      "99\t109081\t0.3\t1\t101041 8040\n",
      "100\t101164\t0.3\t1\t94080 7084\n",
      "101\t96193\t0.3\t1\t89369 6824\n",
      "102\t90471\t0.3\t1\t84087 6384\n",
      "103\t85738\t0.3\t1\t79491 6247\n",
      "104\t80436\t0.3\t1\t74816 5620\n",
      "105\t72717\t0.3\t1\t67687 5030\n",
      "106\t67684\t0.3\t1\t62759 4925\n",
      "107\t62704\t0.3\t1\t58171 4533\n",
      "108\t59951\t0.3\t1\t55617 4334\n",
      "109\t54357\t0.3\t1\t50236 4121\n",
      "110\t50115\t0.3\t1\t46382 3733\n",
      "111\t45371\t0.3\t1\t41955 3416\n",
      "112\t40800\t0.3\t1\t37574 3226\n",
      "113\t35447\t0.3\t1\t32646 2801\n",
      "114\t32325\t0.3\t1\t29802 2523\n",
      "115\t28709\t0.3\t1\t26438 2271\n",
      "116\t25389\t0.3\t1\t23337 2052\n",
      "117\t21950\t0.3\t1\t20122 1828\n",
      "118\t19200\t0.3\t1\t17593 1607\n",
      "119\t16278\t0.3\t1\t14896 1382\n",
      "120\t14121\t0.3\t1\t12882 1239\n",
      "121\t12963\t0.3\t1\t11779 1184\n",
      "122\t10628\t0.3\t1\t9652 976\n",
      "123\t9232\t0.3\t1\t8435 797\n",
      "124\t7617\t0.3\t1\t6893 724\n",
      "125\t6474\t0.3\t1\t5870 604\n",
      "126\t5569\t0.3\t1\t5024 545\n",
      "127\t4721\t0.3\t1\t4238 483\n",
      "128\t3597\t0.3\t1\t3214 383\n",
      "129\t3083\t0.3\t1\t2780 303\n",
      "130\t2334\t0.3\t1\t2063 271\n",
      "131\t2035\t0.3\t1\t1816 219\n",
      "132\t1850\t0.3\t1\t1686 164\n",
      "133\t2327\t0.3\t1\t2083 244\n",
      "134\t4554\t0.3\t1\t4279 275\n",
      "135\t1805\t0.3\t1\t1652 153\n",
      "136\t606\t0.3\t1\t541 65\n",
      "137\t448\t0.3\t1\t385 63\n",
      "138\t325\t0.3\t1\t262 63\n",
      "139\t199\t0.3\t1\t170 29\n",
      "140\t136\t0.3\t1\t115 21\n",
      "141\t92\t0.3\t1\t70 22\n",
      "142\t61\t0.3\t1\t44 17\n",
      "143\t68\t0.3\t1\t60 8\n",
      "144\t63\t0.3\t1\t59 4\n",
      "145\t123\t0.3\t1\t108 15\n",
      "146\t160\t0.3\t1\t140 20\n",
      "147\t706\t0.3\t1\t621 85\n",
      "148\t490\t0.3\t1\t450 40\n",
      "149\t296\t0.3\t1\t261 35\n",
      "150\t3088\t0.3\t1\t2616 472\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "16972486 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 574.33 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              16,972,486\n",
      "Reads with adapters:                15,133,409 (89.2%)\n",
      "Reads written (passing filters):    16,972,486 (100.0%)\n",
      "\n",
      "Total basepairs processed: 2,545,872,900 bp\n",
      "Quality-trimmed:              14,118,915 bp (0.6%)\n",
      "Total written (filtered):  1,669,911,692 bp (65.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15133409 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.2%\n",
      "  C: 14.2%\n",
      "  G: 26.5%\n",
      "  T: 33.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t991330\t4243121.5\t0\t991330\n",
      "2\t172347\t1060780.4\t0\t172347\n",
      "3\t115382\t265195.1\t0\t115382\n",
      "4\t65389\t66298.8\t0\t65389\n",
      "5\t65284\t16574.7\t0\t65284\n",
      "6\t59675\t4143.7\t0\t59675\n",
      "7\t47228\t1035.9\t0\t47228\n",
      "8\t69329\t259.0\t0\t69329\n",
      "9\t55587\t64.7\t0\t55396 191\n",
      "10\t60044\t16.2\t1\t55911 4133\n",
      "11\t66669\t4.0\t1\t61266 5403\n",
      "12\t66198\t1.0\t1\t60904 5294\n",
      "13\t76711\t0.3\t1\t70697 6014\n",
      "14\t72459\t0.3\t1\t66111 6348\n",
      "15\t58334\t0.3\t1\t54019 4315\n",
      "16\t86573\t0.3\t1\t79901 6672\n",
      "17\t73457\t0.3\t1\t67527 5930\n",
      "18\t65883\t0.3\t1\t60982 4901\n",
      "19\t88155\t0.3\t1\t81417 6738\n",
      "20\t82480\t0.3\t1\t76391 6089\n",
      "21\t74222\t0.3\t1\t68897 5325\n",
      "22\t85010\t0.3\t1\t78866 6144\n",
      "23\t89940\t0.3\t1\t83333 6607\n",
      "24\t90783\t0.3\t1\t83898 6885\n",
      "25\t103545\t0.3\t1\t96553 6992\n",
      "26\t86049\t0.3\t1\t79414 6635\n",
      "27\t97578\t0.3\t1\t89843 7735\n",
      "28\t88487\t0.3\t1\t82899 5588\n",
      "29\t101443\t0.3\t1\t94374 7069\n",
      "30\t98065\t0.3\t1\t91540 6525\n",
      "31\t110927\t0.3\t1\t103494 7433\n",
      "32\t96042\t0.3\t1\t90040 6002\n",
      "33\t113007\t0.3\t1\t105053 7954\n",
      "34\t146678\t0.3\t1\t136868 9810\n",
      "35\t80284\t0.3\t1\t75023 5261\n",
      "36\t119636\t0.3\t1\t111148 8488\n",
      "37\t126797\t0.3\t1\t118353 8444\n",
      "38\t121741\t0.3\t1\t114473 7268\n",
      "39\t116427\t0.3\t1\t109798 6629\n",
      "40\t134566\t0.3\t1\t125810 8756\n",
      "41\t132132\t0.3\t1\t124200 7932\n",
      "42\t135345\t0.3\t1\t127009 8336\n",
      "43\t125343\t0.3\t1\t118018 7325\n",
      "44\t147397\t0.3\t1\t138044 9353\n",
      "45\t153169\t0.3\t1\t143585 9584\n",
      "46\t116858\t0.3\t1\t110003 6855\n",
      "47\t148662\t0.3\t1\t140422 8240\n",
      "48\t146456\t0.3\t1\t138141 8315\n",
      "49\t155663\t0.3\t1\t146241 9422\n",
      "50\t150144\t0.3\t1\t140998 9146\n",
      "51\t186605\t0.3\t1\t175426 11179\n",
      "52\t135487\t0.3\t1\t127796 7691\n",
      "53\t169781\t0.3\t1\t161047 8734\n",
      "54\t125345\t0.3\t1\t118710 6635\n",
      "55\t168692\t0.3\t1\t160003 8689\n",
      "56\t176257\t0.3\t1\t166258 9999\n",
      "57\t169891\t0.3\t1\t161571 8320\n",
      "58\t164276\t0.3\t1\t156000 8276\n",
      "59\t168462\t0.3\t1\t159572 8890\n",
      "60\t177599\t0.3\t1\t168506 9093\n",
      "61\t183440\t0.3\t1\t174203 9237\n",
      "62\t202372\t0.3\t1\t191386 10986\n",
      "63\t270936\t0.3\t1\t258181 12755\n",
      "64\t234996\t0.3\t1\t225774 9222\n",
      "65\t151631\t0.3\t1\t144050 7581\n",
      "66\t166302\t0.3\t1\t157972 8330\n",
      "67\t167833\t0.3\t1\t159201 8632\n",
      "68\t170330\t0.3\t1\t161782 8548\n",
      "69\t171332\t0.3\t1\t162512 8820\n",
      "70\t183649\t0.3\t1\t174535 9114\n",
      "71\t199117\t0.3\t1\t189420 9697\n",
      "72\t190292\t0.3\t1\t180708 9584\n",
      "73\t205025\t0.3\t1\t195024 10001\n",
      "74\t188852\t0.3\t1\t179445 9407\n",
      "75\t181257\t0.3\t1\t172280 8977\n",
      "76\t178734\t0.3\t1\t169881 8853\n",
      "77\t180452\t0.3\t1\t171277 9175\n",
      "78\t172679\t0.3\t1\t164051 8628\n",
      "79\t170532\t0.3\t1\t162087 8445\n",
      "80\t171521\t0.3\t1\t163340 8181\n",
      "81\t171140\t0.3\t1\t162630 8510\n",
      "82\t165207\t0.3\t1\t157233 7974\n",
      "83\t163608\t0.3\t1\t155825 7783\n",
      "84\t160013\t0.3\t1\t152625 7388\n",
      "85\t159454\t0.3\t1\t151925 7529\n",
      "86\t161481\t0.3\t1\t153907 7574\n",
      "87\t162581\t0.3\t1\t155126 7455\n",
      "88\t162054\t0.3\t1\t154673 7381\n",
      "89\t147788\t0.3\t1\t141132 6656\n",
      "90\t145172\t0.3\t1\t138698 6474\n",
      "91\t139015\t0.3\t1\t132745 6270\n",
      "92\t132299\t0.3\t1\t126316 5983\n",
      "93\t128552\t0.3\t1\t122735 5817\n",
      "94\t126711\t0.3\t1\t120900 5811\n",
      "95\t119947\t0.3\t1\t114454 5493\n",
      "96\t117162\t0.3\t1\t111939 5223\n",
      "97\t108989\t0.3\t1\t104182 4807\n",
      "98\t105848\t0.3\t1\t101282 4566\n",
      "99\t105989\t0.3\t1\t101349 4640\n",
      "100\t98144\t0.3\t1\t93908 4236\n",
      "101\t94920\t0.3\t1\t90864 4056\n",
      "102\t89211\t0.3\t1\t85259 3952\n",
      "103\t81188\t0.3\t1\t77654 3534\n",
      "104\t77565\t0.3\t1\t74081 3484\n",
      "105\t70248\t0.3\t1\t67249 2999\n",
      "106\t65288\t0.3\t1\t62604 2684\n",
      "107\t60022\t0.3\t1\t57428 2594\n",
      "108\t55515\t0.3\t1\t53019 2496\n",
      "109\t50990\t0.3\t1\t48728 2262\n",
      "110\t46869\t0.3\t1\t44831 2038\n",
      "111\t44489\t0.3\t1\t42630 1859\n",
      "112\t39619\t0.3\t1\t37888 1731\n",
      "113\t34543\t0.3\t1\t33015 1528\n",
      "114\t31368\t0.3\t1\t30023 1345\n",
      "115\t26935\t0.3\t1\t25774 1161\n",
      "116\t23611\t0.3\t1\t22614 997\n",
      "117\t19912\t0.3\t1\t18993 919\n",
      "118\t17572\t0.3\t1\t16789 783\n",
      "119\t14871\t0.3\t1\t14194 677\n",
      "120\t12844\t0.3\t1\t12244 600\n",
      "121\t11858\t0.3\t1\t11313 545\n",
      "122\t9930\t0.3\t1\t9463 467\n",
      "123\t8753\t0.3\t1\t8342 411\n",
      "124\t7126\t0.3\t1\t6812 314\n",
      "125\t6158\t0.3\t1\t5890 268\n",
      "126\t5177\t0.3\t1\t4954 223\n",
      "127\t4434\t0.3\t1\t4233 201\n",
      "128\t3338\t0.3\t1\t3185 153\n",
      "129\t2850\t0.3\t1\t2742 108\n",
      "130\t2111\t0.3\t1\t2001 110\n",
      "131\t1887\t0.3\t1\t1807 80\n",
      "132\t1727\t0.3\t1\t1658 69\n",
      "133\t2467\t0.3\t1\t2378 89\n",
      "134\t4553\t0.3\t1\t4401 152\n",
      "135\t1770\t0.3\t1\t1714 56\n",
      "136\t598\t0.3\t1\t559 39\n",
      "137\t389\t0.3\t1\t374 15\n",
      "138\t283\t0.3\t1\t271 12\n",
      "139\t188\t0.3\t1\t181 7\n",
      "140\t133\t0.3\t1\t124 9\n",
      "141\t75\t0.3\t1\t73 2\n",
      "142\t64\t0.3\t1\t61 3\n",
      "143\t64\t0.3\t1\t59 5\n",
      "144\t68\t0.3\t1\t66 2\n",
      "145\t139\t0.3\t1\t132 7\n",
      "146\t159\t0.3\t1\t151 8\n",
      "147\t678\t0.3\t1\t647 31\n",
      "148\t467\t0.3\t1\t450 17\n",
      "149\t278\t0.3\t1\t263 15\n",
      "150\t2366\t0.3\t1\t2224 142\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "16972486 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "file_1: CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz, file_2: CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 16972486\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 65635 (0.39%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 50% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for CTRL_8C_26psu_2_S18_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files CTRL_8C_26psu_2_S18_L002_R1_001_trimmed.fq.gz and CTRL_8C_26psu_2_S18_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F1_S20_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 4738.46 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             113,474,961\n",
      "Reads with adapters:                78,847,369 (69.5%)\n",
      "Reads written (passing filters):   113,474,961 (100.0%)\n",
      "\n",
      "Total basepairs processed: 17,021,244,150 bp\n",
      "Quality-trimmed:              22,715,789 bp (0.1%)\n",
      "Total written (filtered):  14,426,732,983 bp (84.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78847369 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.1%\n",
      "  C: 9.7%\n",
      "  G: 19.7%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t22838934\t28368740.2\t0\t22838934\n",
      "2\t2342589\t7092185.1\t0\t2342589\n",
      "3\t1202279\t1773046.3\t0\t1202279\n",
      "4\t750593\t443261.6\t0\t750593\n",
      "5\t559528\t110815.4\t0\t559528\n",
      "6\t524826\t27703.8\t0\t524826\n",
      "7\t449332\t6926.0\t0\t449332\n",
      "8\t548899\t1731.5\t0\t548899\n",
      "9\t548856\t432.9\t0\t541308 7548\n",
      "10\t508456\t108.2\t1\t478983 29473\n",
      "11\t562956\t27.1\t1\t517668 45288\n",
      "12\t559587\t6.8\t1\t520090 39497\n",
      "13\t547304\t1.7\t1\t508519 38785\n",
      "14\t570426\t1.7\t1\t525734 44692\n",
      "15\t575858\t1.7\t1\t534867 40991\n",
      "16\t569860\t1.7\t1\t523134 46726\n",
      "17\t678914\t1.7\t1\t621846 57068\n",
      "18\t565739\t1.7\t1\t526451 39288\n",
      "19\t504906\t1.7\t1\t468246 36660\n",
      "20\t580103\t1.7\t1\t539012 41091\n",
      "21\t633303\t1.7\t1\t584154 49149\n",
      "22\t551495\t1.7\t1\t516023 35472\n",
      "23\t623244\t1.7\t1\t577696 45548\n",
      "24\t597040\t1.7\t1\t545080 51960\n",
      "25\t610942\t1.7\t1\t565949 44993\n",
      "26\t578327\t1.7\t1\t539305 39022\n",
      "27\t631060\t1.7\t1\t581266 49794\n",
      "28\t620380\t1.7\t1\t573987 46393\n",
      "29\t630417\t1.7\t1\t577901 52516\n",
      "30\t665372\t1.7\t1\t624016 41356\n",
      "31\t521479\t1.7\t1\t485824 35655\n",
      "32\t637552\t1.7\t1\t593972 43580\n",
      "33\t605503\t1.7\t1\t560522 44981\n",
      "34\t753213\t1.7\t1\t699064 54149\n",
      "35\t512573\t1.7\t1\t477265 35308\n",
      "36\t633868\t1.7\t1\t587959 45909\n",
      "37\t659372\t1.7\t1\t612861 46511\n",
      "38\t603868\t1.7\t1\t563307 40561\n",
      "39\t656560\t1.7\t1\t609454 47106\n",
      "40\t670911\t1.7\t1\t620635 50276\n",
      "41\t733444\t1.7\t1\t678908 54536\n",
      "42\t554355\t1.7\t1\t510902 43453\n",
      "43\t1004589\t1.7\t1\t949034 55555\n",
      "44\t281497\t1.7\t1\t254795 26702\n",
      "45\t650424\t1.7\t1\t607820 42604\n",
      "46\t622059\t1.7\t1\t580617 41442\n",
      "47\t641812\t1.7\t1\t599651 42161\n",
      "48\t616364\t1.7\t1\t574406 41958\n",
      "49\t673245\t1.7\t1\t623804 49441\n",
      "50\t604428\t1.7\t1\t565417 39011\n",
      "51\t645021\t1.7\t1\t600411 44610\n",
      "52\t648661\t1.7\t1\t600394 48267\n",
      "53\t693327\t1.7\t1\t647436 45891\n",
      "54\t550715\t1.7\t1\t514274 36441\n",
      "55\t631316\t1.7\t1\t588270 43046\n",
      "56\t767345\t1.7\t1\t719044 48301\n",
      "57\t493710\t1.7\t1\t456524 37186\n",
      "58\t647241\t1.7\t1\t610352 36889\n",
      "59\t560612\t1.7\t1\t522206 38406\n",
      "60\t620281\t1.7\t1\t578774 41507\n",
      "61\t603855\t1.7\t1\t565232 38623\n",
      "62\t664795\t1.7\t1\t623425 41370\n",
      "63\t508452\t1.7\t1\t471688 36764\n",
      "64\t573553\t1.7\t1\t538426 35127\n",
      "65\t625525\t1.7\t1\t584793 40732\n",
      "66\t654791\t1.7\t1\t611542 43249\n",
      "67\t693655\t1.7\t1\t637398 56257\n",
      "68\t1295847\t1.7\t1\t1228868 66979\n",
      "69\t207198\t1.7\t1\t188952 18246\n",
      "70\t83872\t1.7\t1\t69823 14049\n",
      "71\t351217\t1.7\t1\t328576 22641\n",
      "72\t486564\t1.7\t1\t456539 30025\n",
      "73\t511552\t1.7\t1\t479835 31717\n",
      "74\t500225\t1.7\t1\t468801 31424\n",
      "75\t493734\t1.7\t1\t462637 31097\n",
      "76\t485479\t1.7\t1\t455415 30064\n",
      "77\t472130\t1.7\t1\t442425 29705\n",
      "78\t457249\t1.7\t1\t427856 29393\n",
      "79\t452598\t1.7\t1\t423758 28840\n",
      "80\t436888\t1.7\t1\t409466 27422\n",
      "81\t424029\t1.7\t1\t397325 26704\n",
      "82\t413331\t1.7\t1\t387262 26069\n",
      "83\t395648\t1.7\t1\t370721 24927\n",
      "84\t386472\t1.7\t1\t362163 24309\n",
      "85\t373564\t1.7\t1\t350510 23054\n",
      "86\t350177\t1.7\t1\t327542 22635\n",
      "87\t338817\t1.7\t1\t317459 21358\n",
      "88\t329564\t1.7\t1\t308892 20672\n",
      "89\t310326\t1.7\t1\t290522 19804\n",
      "90\t302165\t1.7\t1\t282977 19188\n",
      "91\t286619\t1.7\t1\t268441 18178\n",
      "92\t275091\t1.7\t1\t257788 17303\n",
      "93\t261850\t1.7\t1\t245237 16613\n",
      "94\t246459\t1.7\t1\t230780 15679\n",
      "95\t229270\t1.7\t1\t214573 14697\n",
      "96\t213682\t1.7\t1\t200015 13667\n",
      "97\t200167\t1.7\t1\t187352 12815\n",
      "98\t186673\t1.7\t1\t174782 11891\n",
      "99\t177389\t1.7\t1\t165973 11416\n",
      "100\t164170\t1.7\t1\t153549 10621\n",
      "101\t152020\t1.7\t1\t142083 9937\n",
      "102\t138827\t1.7\t1\t129592 9235\n",
      "103\t126885\t1.7\t1\t118589 8296\n",
      "104\t114180\t1.7\t1\t106471 7709\n",
      "105\t103407\t1.7\t1\t96323 7084\n",
      "106\t93053\t1.7\t1\t86648 6405\n",
      "107\t82305\t1.7\t1\t76807 5498\n",
      "108\t73960\t1.7\t1\t68774 5186\n",
      "109\t66073\t1.7\t1\t61323 4750\n",
      "110\t58316\t1.7\t1\t54124 4192\n",
      "111\t51210\t1.7\t1\t47563 3647\n",
      "112\t46457\t1.7\t1\t43051 3406\n",
      "113\t39166\t1.7\t1\t36179 2987\n",
      "114\t35653\t1.7\t1\t32939 2714\n",
      "115\t29363\t1.7\t1\t27078 2285\n",
      "116\t25352\t1.7\t1\t23356 1996\n",
      "117\t20912\t1.7\t1\t19235 1677\n",
      "118\t17421\t1.7\t1\t15974 1447\n",
      "119\t14864\t1.7\t1\t13576 1288\n",
      "120\t12650\t1.7\t1\t11578 1072\n",
      "121\t10471\t1.7\t1\t9498 973\n",
      "122\t8802\t1.7\t1\t7972 830\n",
      "123\t7467\t1.7\t1\t6769 698\n",
      "124\t6267\t1.7\t1\t5666 601\n",
      "125\t5296\t1.7\t1\t4740 556\n",
      "126\t5081\t1.7\t1\t4533 548\n",
      "127\t4350\t1.7\t1\t3958 392\n",
      "128\t3255\t1.7\t1\t2949 306\n",
      "129\t2684\t1.7\t1\t2428 256\n",
      "130\t3108\t1.7\t1\t2815 293\n",
      "131\t4344\t1.7\t1\t4012 332\n",
      "132\t4969\t1.7\t1\t4639 330\n",
      "133\t7178\t1.7\t1\t6729 449\n",
      "134\t9066\t1.7\t1\t8514 552\n",
      "135\t3718\t1.7\t1\t3456 262\n",
      "136\t1190\t1.7\t1\t1067 123\n",
      "137\t629\t1.7\t1\t542 87\n",
      "138\t481\t1.7\t1\t425 56\n",
      "139\t351\t1.7\t1\t310 41\n",
      "140\t237\t1.7\t1\t196 41\n",
      "141\t164\t1.7\t1\t133 31\n",
      "142\t89\t1.7\t1\t63 26\n",
      "143\t91\t1.7\t1\t63 28\n",
      "144\t96\t1.7\t1\t72 24\n",
      "145\t237\t1.7\t1\t190 47\n",
      "146\t222\t1.7\t1\t196 26\n",
      "147\t852\t1.7\t1\t739 113\n",
      "148\t478\t1.7\t1\t374 104\n",
      "149\t260\t1.7\t1\t214 46\n",
      "150\t2350\t1.7\t1\t1965 385\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "113474961 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F1_S20_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 4974.32 s (44 us/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             113,474,961\n",
      "Reads with adapters:                79,235,271 (69.8%)\n",
      "Reads written (passing filters):   113,474,961 (100.0%)\n",
      "\n",
      "Total basepairs processed: 17,021,244,150 bp\n",
      "Quality-trimmed:              73,052,751 bp (0.4%)\n",
      "Total written (filtered):  14,433,737,028 bp (84.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 79235271 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 9.2%\n",
      "  G: 21.3%\n",
      "  T: 32.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t24292490\t28368740.2\t0\t24292490\n",
      "2\t2172289\t7092185.1\t0\t2172289\n",
      "3\t1212765\t1773046.3\t0\t1212765\n",
      "4\t728934\t443261.6\t0\t728934\n",
      "5\t615953\t110815.4\t0\t615953\n",
      "6\t518496\t27703.8\t0\t518496\n",
      "7\t424176\t6926.0\t0\t424176\n",
      "8\t595932\t1731.5\t0\t595932\n",
      "9\t493164\t432.9\t0\t491188 1976\n",
      "10\t515557\t108.2\t1\t490642 24915\n",
      "11\t544480\t27.1\t1\t510284 34196\n",
      "12\t557302\t6.8\t1\t522559 34743\n",
      "13\t624852\t1.7\t1\t586293 38559\n",
      "14\t566086\t1.7\t1\t527536 38550\n",
      "15\t479671\t1.7\t1\t447634 32037\n",
      "16\t677197\t1.7\t1\t634485 42712\n",
      "17\t518178\t1.7\t1\t482968 35210\n",
      "18\t513709\t1.7\t1\t481055 32654\n",
      "19\t623697\t1.7\t1\t583105 40592\n",
      "20\t590778\t1.7\t1\t553837 36941\n",
      "21\t523012\t1.7\t1\t489156 33856\n",
      "22\t582166\t1.7\t1\t546634 35532\n",
      "23\t607194\t1.7\t1\t567433 39761\n",
      "24\t596018\t1.7\t1\t558835 37183\n",
      "25\t658603\t1.7\t1\t618497 40106\n",
      "26\t563795\t1.7\t1\t525767 38028\n",
      "27\t625280\t1.7\t1\t580031 45249\n",
      "28\t565755\t1.7\t1\t537011 28744\n",
      "29\t604057\t1.7\t1\t566572 37485\n",
      "30\t575342\t1.7\t1\t545309 30033\n",
      "31\t637117\t1.7\t1\t598679 38438\n",
      "32\t567491\t1.7\t1\t538376 29115\n",
      "33\t633139\t1.7\t1\t594872 38267\n",
      "34\t807925\t1.7\t1\t759853 48072\n",
      "35\t450122\t1.7\t1\t426055 24067\n",
      "36\t611610\t1.7\t1\t574682 36928\n",
      "37\t655880\t1.7\t1\t616910 38970\n",
      "38\t600986\t1.7\t1\t571449 29537\n",
      "39\t595629\t1.7\t1\t565252 30377\n",
      "40\t651655\t1.7\t1\t614241 37414\n",
      "41\t634336\t1.7\t1\t600514 33822\n",
      "42\t641088\t1.7\t1\t607660 33428\n",
      "43\t603840\t1.7\t1\t570768 33072\n",
      "44\t681587\t1.7\t1\t644069 37518\n",
      "45\t707281\t1.7\t1\t670903 36378\n",
      "46\t517703\t1.7\t1\t489820 27883\n",
      "47\t633055\t1.7\t1\t601375 31680\n",
      "48\t613751\t1.7\t1\t584361 29390\n",
      "49\t644336\t1.7\t1\t611009 33327\n",
      "50\t626710\t1.7\t1\t592951 33759\n",
      "51\t770564\t1.7\t1\t731245 39319\n",
      "52\t552184\t1.7\t1\t525737 26447\n",
      "53\t655644\t1.7\t1\t624724 30920\n",
      "54\t497055\t1.7\t1\t473150 23905\n",
      "55\t622404\t1.7\t1\t594084 28320\n",
      "56\t630771\t1.7\t1\t599636 31135\n",
      "57\t608401\t1.7\t1\t579359 29042\n",
      "58\t589791\t1.7\t1\t563588 26203\n",
      "59\t602895\t1.7\t1\t574430 28465\n",
      "60\t610621\t1.7\t1\t581075 29546\n",
      "61\t631221\t1.7\t1\t600250 30971\n",
      "62\t678983\t1.7\t1\t645436 33547\n",
      "63\t884166\t1.7\t1\t845237 38929\n",
      "64\t697534\t1.7\t1\t670069 27465\n",
      "65\t461327\t1.7\t1\t440187 21140\n",
      "66\t505461\t1.7\t1\t482444 23017\n",
      "67\t504336\t1.7\t1\t480520 23816\n",
      "68\t506997\t1.7\t1\t484099 22898\n",
      "69\t501262\t1.7\t1\t477906 23356\n",
      "70\t506318\t1.7\t1\t482633 23685\n",
      "71\t530040\t1.7\t1\t506417 23623\n",
      "72\t512724\t1.7\t1\t488940 23784\n",
      "73\t535324\t1.7\t1\t511542 23782\n",
      "74\t491314\t1.7\t1\t470184 21130\n",
      "75\t468392\t1.7\t1\t447958 20434\n",
      "76\t453588\t1.7\t1\t433629 19959\n",
      "77\t443997\t1.7\t1\t423454 20543\n",
      "78\t435469\t1.7\t1\t416134 19335\n",
      "79\t419163\t1.7\t1\t400603 18560\n",
      "80\t414024\t1.7\t1\t395828 18196\n",
      "81\t398372\t1.7\t1\t381040 17332\n",
      "82\t382299\t1.7\t1\t365451 16848\n",
      "83\t366779\t1.7\t1\t350673 16106\n",
      "84\t355643\t1.7\t1\t339567 16076\n",
      "85\t348592\t1.7\t1\t332671 15921\n",
      "86\t340981\t1.7\t1\t325646 15335\n",
      "87\t343724\t1.7\t1\t328719 15005\n",
      "88\t328419\t1.7\t1\t314072 14347\n",
      "89\t293572\t1.7\t1\t280847 12725\n",
      "90\t277907\t1.7\t1\t265475 12432\n",
      "91\t261801\t1.7\t1\t250188 11613\n",
      "92\t247933\t1.7\t1\t237089 10844\n",
      "93\t241002\t1.7\t1\t230436 10566\n",
      "94\t225907\t1.7\t1\t216109 9798\n",
      "95\t212543\t1.7\t1\t203112 9431\n",
      "96\t199922\t1.7\t1\t191511 8411\n",
      "97\t189942\t1.7\t1\t181742 8200\n",
      "98\t181068\t1.7\t1\t173356 7712\n",
      "99\t177800\t1.7\t1\t170721 7079\n",
      "100\t161532\t1.7\t1\t154936 6596\n",
      "101\t142590\t1.7\t1\t136815 5775\n",
      "102\t127109\t1.7\t1\t121824 5285\n",
      "103\t114820\t1.7\t1\t109872 4948\n",
      "104\t106755\t1.7\t1\t102353 4402\n",
      "105\t97032\t1.7\t1\t93102 3930\n",
      "106\t87939\t1.7\t1\t84311 3628\n",
      "107\t77043\t1.7\t1\t73799 3244\n",
      "108\t66878\t1.7\t1\t63922 2956\n",
      "109\t60697\t1.7\t1\t58149 2548\n",
      "110\t53817\t1.7\t1\t51530 2287\n",
      "111\t49656\t1.7\t1\t47564 2092\n",
      "112\t44212\t1.7\t1\t42328 1884\n",
      "113\t37568\t1.7\t1\t35930 1638\n",
      "114\t33504\t1.7\t1\t32034 1470\n",
      "115\t26873\t1.7\t1\t25557 1316\n",
      "116\t23021\t1.7\t1\t21918 1103\n",
      "117\t18702\t1.7\t1\t17804 898\n",
      "118\t15710\t1.7\t1\t14954 756\n",
      "119\t13467\t1.7\t1\t12827 640\n",
      "120\t11166\t1.7\t1\t10610 556\n",
      "121\t9578\t1.7\t1\t9044 534\n",
      "122\t7970\t1.7\t1\t7515 455\n",
      "123\t6749\t1.7\t1\t6389 360\n",
      "124\t5790\t1.7\t1\t5438 352\n",
      "125\t4982\t1.7\t1\t4674 308\n",
      "126\t4691\t1.7\t1\t4415 276\n",
      "127\t3926\t1.7\t1\t3689 237\n",
      "128\t2937\t1.7\t1\t2717 220\n",
      "129\t2589\t1.7\t1\t2397 192\n",
      "130\t2951\t1.7\t1\t2762 189\n",
      "131\t4241\t1.7\t1\t4050 191\n",
      "132\t4816\t1.7\t1\t4604 212\n",
      "133\t7287\t1.7\t1\t6984 303\n",
      "134\t8892\t1.7\t1\t8589 303\n",
      "135\t3540\t1.7\t1\t3386 154\n",
      "136\t1144\t1.7\t1\t1071 73\n",
      "137\t598\t1.7\t1\t565 33\n",
      "138\t406\t1.7\t1\t369 37\n",
      "139\t302\t1.7\t1\t277 25\n",
      "140\t227\t1.7\t1\t197 30\n",
      "141\t147\t1.7\t1\t128 19\n",
      "142\t76\t1.7\t1\t62 14\n",
      "143\t79\t1.7\t1\t62 17\n",
      "144\t94\t1.7\t1\t81 13\n",
      "145\t260\t1.7\t1\t229 31\n",
      "146\t164\t1.7\t1\t145 19\n",
      "147\t597\t1.7\t1\t528 69\n",
      "148\t305\t1.7\t1\t255 50\n",
      "149\t176\t1.7\t1\t150 26\n",
      "150\t1321\t1.7\t1\t1192 129\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "113474961 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files Sealice_F1_S20_L001_R1_001_trimmed.fq.gz and Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "file_1: Sealice_F1_S20_L001_R1_001_trimmed.fq.gz, file_2: Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: Sealice_F1_S20_L001_R1_001_trimmed.fq.gz and Sealice_F1_S20_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 113474961\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 221041 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F1_S20_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F1_S20_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files Sealice_F1_S20_L001_R1_001_trimmed.fq.gz and Sealice_F1_S20_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F1_S20_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 4801.59 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             112,706,276\n",
      "Reads with adapters:                78,369,527 (69.5%)\n",
      "Reads written (passing filters):   112,706,276 (100.0%)\n",
      "\n",
      "Total basepairs processed: 16,905,941,400 bp\n",
      "Quality-trimmed:              25,158,265 bp (0.1%)\n",
      "Total written (filtered):  14,312,572,664 bp (84.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78369527 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.1%\n",
      "  C: 9.8%\n",
      "  G: 19.7%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t22613071\t28176569.0\t0\t22613071\n",
      "2\t2300394\t7044142.2\t0\t2300394\n",
      "3\t1191481\t1761035.6\t0\t1191481\n",
      "4\t736265\t440258.9\t0\t736265\n",
      "5\t556744\t110064.7\t0\t556744\n",
      "6\t521852\t27516.2\t0\t521852\n",
      "7\t432500\t6879.0\t0\t432500\n",
      "8\t542896\t1719.8\t0\t542896\n",
      "9\t541192\t429.9\t0\t533921 7271\n",
      "10\t495976\t107.5\t1\t461928 34048\n",
      "11\t558120\t26.9\t1\t506660 51460\n",
      "12\t551298\t6.7\t1\t505922 45376\n",
      "13\t542939\t1.7\t1\t497854 45085\n",
      "14\t564136\t1.7\t1\t512614 51522\n",
      "15\t569498\t1.7\t1\t521423 48075\n",
      "16\t563578\t1.7\t1\t509820 53758\n",
      "17\t682231\t1.7\t1\t615144 67087\n",
      "18\t558002\t1.7\t1\t512187 45815\n",
      "19\t496544\t1.7\t1\t454420 42124\n",
      "20\t576298\t1.7\t1\t529271 47027\n",
      "21\t637299\t1.7\t1\t580654 56645\n",
      "22\t542521\t1.7\t1\t500784 41737\n",
      "23\t624224\t1.7\t1\t570946 53278\n",
      "24\t590519\t1.7\t1\t532912 57607\n",
      "25\t607047\t1.7\t1\t556132 50915\n",
      "26\t572302\t1.7\t1\t527531 44771\n",
      "27\t630635\t1.7\t1\t573588 57047\n",
      "28\t620404\t1.7\t1\t568899 51505\n",
      "29\t625305\t1.7\t1\t567099 58206\n",
      "30\t664393\t1.7\t1\t616071 48322\n",
      "31\t510018\t1.7\t1\t469355 40663\n",
      "32\t637215\t1.7\t1\t584852 52363\n",
      "33\t597607\t1.7\t1\t545528 52079\n",
      "34\t753926\t1.7\t1\t688559 65367\n",
      "35\t503645\t1.7\t1\t462577 41068\n",
      "36\t630961\t1.7\t1\t577612 53349\n",
      "37\t659262\t1.7\t1\t604319 54943\n",
      "38\t595913\t1.7\t1\t547877 48036\n",
      "39\t653746\t1.7\t1\t597162 56584\n",
      "40\t670529\t1.7\t1\t610664 59865\n",
      "41\t732564\t1.7\t1\t667967 64597\n",
      "42\t545711\t1.7\t1\t495424 50287\n",
      "43\t991735\t1.7\t1\t924678 67057\n",
      "44\t286953\t1.7\t1\t256328 30625\n",
      "45\t652219\t1.7\t1\t601722 50497\n",
      "46\t619313\t1.7\t1\t571113 48200\n",
      "47\t635341\t1.7\t1\t585973 49368\n",
      "48\t610799\t1.7\t1\t562441 48358\n",
      "49\t674495\t1.7\t1\t617955 56540\n",
      "50\t598712\t1.7\t1\t554202 44510\n",
      "51\t649220\t1.7\t1\t598179 51041\n",
      "52\t650919\t1.7\t1\t595866 55053\n",
      "53\t695670\t1.7\t1\t642854 52816\n",
      "54\t545982\t1.7\t1\t504022 41960\n",
      "55\t632591\t1.7\t1\t583378 49213\n",
      "56\t771796\t1.7\t1\t714466 57330\n",
      "57\t493419\t1.7\t1\t450919 42500\n",
      "58\t643644\t1.7\t1\t599707 43937\n",
      "59\t562738\t1.7\t1\t517495 45243\n",
      "60\t620089\t1.7\t1\t571985 48104\n",
      "61\t599005\t1.7\t1\t554664 44341\n",
      "62\t665940\t1.7\t1\t617471 48469\n",
      "63\t511736\t1.7\t1\t469216 42520\n",
      "64\t570922\t1.7\t1\t530032 40890\n",
      "65\t624134\t1.7\t1\t576865 47269\n",
      "66\t644685\t1.7\t1\t596010 48675\n",
      "67\t692204\t1.7\t1\t630686 61518\n",
      "68\t1251198\t1.7\t1\t1176122 75076\n",
      "69\t255502\t1.7\t1\t232815 22687\n",
      "70\t102840\t1.7\t1\t86533 16307\n",
      "71\t343135\t1.7\t1\t318441 24694\n",
      "72\t484356\t1.7\t1\t451690 32666\n",
      "73\t511359\t1.7\t1\t476313 35046\n",
      "74\t501885\t1.7\t1\t465645 36240\n",
      "75\t495613\t1.7\t1\t458847 36766\n",
      "76\t482971\t1.7\t1\t446569 36402\n",
      "77\t471725\t1.7\t1\t437257 34468\n",
      "78\t457894\t1.7\t1\t423781 34113\n",
      "79\t452070\t1.7\t1\t418062 34008\n",
      "80\t438484\t1.7\t1\t405088 33396\n",
      "81\t425410\t1.7\t1\t393919 31491\n",
      "82\t414132\t1.7\t1\t383041 31091\n",
      "83\t396751\t1.7\t1\t367831 28920\n",
      "84\t386912\t1.7\t1\t357963 28949\n",
      "85\t373056\t1.7\t1\t345296 27760\n",
      "86\t351920\t1.7\t1\t325697 26223\n",
      "87\t339788\t1.7\t1\t315178 24610\n",
      "88\t331230\t1.7\t1\t306931 24299\n",
      "89\t311047\t1.7\t1\t288166 22881\n",
      "90\t302757\t1.7\t1\t280581 22176\n",
      "91\t287473\t1.7\t1\t266368 21105\n",
      "92\t274821\t1.7\t1\t254675 20146\n",
      "93\t261527\t1.7\t1\t242454 19073\n",
      "94\t247083\t1.7\t1\t228486 18597\n",
      "95\t229569\t1.7\t1\t212524 17045\n",
      "96\t215425\t1.7\t1\t199345 16080\n",
      "97\t200580\t1.7\t1\t186258 14322\n",
      "98\t186104\t1.7\t1\t172508 13596\n",
      "99\t176938\t1.7\t1\t163984 12954\n",
      "100\t163824\t1.7\t1\t151757 12067\n",
      "101\t152410\t1.7\t1\t141174 11236\n",
      "102\t139508\t1.7\t1\t128964 10544\n",
      "103\t126668\t1.7\t1\t117083 9585\n",
      "104\t114659\t1.7\t1\t106240 8419\n",
      "105\t103515\t1.7\t1\t95951 7564\n",
      "106\t93675\t1.7\t1\t86344 7331\n",
      "107\t82801\t1.7\t1\t76358 6443\n",
      "108\t74274\t1.7\t1\t68589 5685\n",
      "109\t65750\t1.7\t1\t60455 5295\n",
      "110\t58361\t1.7\t1\t53718 4643\n",
      "111\t51503\t1.7\t1\t47445 4058\n",
      "112\t46515\t1.7\t1\t42704 3811\n",
      "113\t39788\t1.7\t1\t36495 3293\n",
      "114\t35516\t1.7\t1\t32502 3014\n",
      "115\t29286\t1.7\t1\t26808 2478\n",
      "116\t25103\t1.7\t1\t22972 2131\n",
      "117\t21040\t1.7\t1\t19128 1912\n",
      "118\t17330\t1.7\t1\t15763 1567\n",
      "119\t14917\t1.7\t1\t13608 1309\n",
      "120\t12642\t1.7\t1\t11544 1098\n",
      "121\t10638\t1.7\t1\t9581 1057\n",
      "122\t8906\t1.7\t1\t8053 853\n",
      "123\t7247\t1.7\t1\t6562 685\n",
      "124\t6246\t1.7\t1\t5598 648\n",
      "125\t5323\t1.7\t1\t4772 551\n",
      "126\t5106\t1.7\t1\t4581 525\n",
      "127\t4365\t1.7\t1\t3924 441\n",
      "128\t3325\t1.7\t1\t2922 403\n",
      "129\t2686\t1.7\t1\t2381 305\n",
      "130\t3038\t1.7\t1\t2747 291\n",
      "131\t4431\t1.7\t1\t4079 352\n",
      "132\t5138\t1.7\t1\t4758 380\n",
      "133\t6888\t1.7\t1\t6396 492\n",
      "134\t9151\t1.7\t1\t8614 537\n",
      "135\t3733\t1.7\t1\t3470 263\n",
      "136\t1176\t1.7\t1\t1019 157\n",
      "137\t621\t1.7\t1\t528 93\n",
      "138\t473\t1.7\t1\t413 60\n",
      "139\t304\t1.7\t1\t257 47\n",
      "140\t204\t1.7\t1\t168 36\n",
      "141\t175\t1.7\t1\t143 32\n",
      "142\t95\t1.7\t1\t74 21\n",
      "143\t82\t1.7\t1\t66 16\n",
      "144\t113\t1.7\t1\t89 24\n",
      "145\t229\t1.7\t1\t203 26\n",
      "146\t212\t1.7\t1\t174 38\n",
      "147\t863\t1.7\t1\t726 137\n",
      "148\t473\t1.7\t1\t383 90\n",
      "149\t260\t1.7\t1\t220 40\n",
      "150\t2364\t1.7\t1\t1963 401\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "112706276 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F1_S20_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 5021.62 s (45 us/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             112,706,276\n",
      "Reads with adapters:                78,702,417 (69.8%)\n",
      "Reads written (passing filters):   112,706,276 (100.0%)\n",
      "\n",
      "Total basepairs processed: 16,905,941,400 bp\n",
      "Quality-trimmed:              71,979,281 bp (0.4%)\n",
      "Total written (filtered):  14,323,556,948 bp (84.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 78702417 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 9.3%\n",
      "  G: 21.4%\n",
      "  T: 32.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t24049259\t28176569.0\t0\t24049259\n",
      "2\t2103822\t7044142.2\t0\t2103822\n",
      "3\t1234623\t1761035.6\t0\t1234623\n",
      "4\t713926\t440258.9\t0\t713926\n",
      "5\t593548\t110064.7\t0\t593548\n",
      "6\t520489\t27516.2\t0\t520489\n",
      "7\t412592\t6879.0\t0\t412592\n",
      "8\t591207\t1719.8\t0\t591207\n",
      "9\t484607\t429.9\t0\t482791 1816\n",
      "10\t495729\t107.5\t1\t462070 33659\n",
      "11\t538210\t26.9\t1\t494011 44199\n",
      "12\t541780\t6.7\t1\t498135 43645\n",
      "13\t624482\t1.7\t1\t573035 51447\n",
      "14\t557547\t1.7\t1\t507941 49606\n",
      "15\t471226\t1.7\t1\t432359 38867\n",
      "16\t663767\t1.7\t1\t609668 54099\n",
      "17\t541007\t1.7\t1\t495593 45414\n",
      "18\t492493\t1.7\t1\t452345 40148\n",
      "19\t620390\t1.7\t1\t571312 49078\n",
      "20\t584573\t1.7\t1\t538235 46338\n",
      "21\t515948\t1.7\t1\t474451 41497\n",
      "22\t574265\t1.7\t1\t529265 45000\n",
      "23\t609172\t1.7\t1\t559895 49277\n",
      "24\t593165\t1.7\t1\t546488 46677\n",
      "25\t658890\t1.7\t1\t611466 47424\n",
      "26\t563159\t1.7\t1\t515387 47772\n",
      "27\t631617\t1.7\t1\t576879 54738\n",
      "28\t542598\t1.7\t1\t506943 35655\n",
      "29\t609830\t1.7\t1\t564437 45393\n",
      "30\t560210\t1.7\t1\t522268 37942\n",
      "31\t641241\t1.7\t1\t594510 46731\n",
      "32\t544093\t1.7\t1\t508074 36019\n",
      "33\t636374\t1.7\t1\t588179 48195\n",
      "34\t815281\t1.7\t1\t756328 58953\n",
      "35\t430454\t1.7\t1\t399829 30625\n",
      "36\t615273\t1.7\t1\t568703 46570\n",
      "37\t652049\t1.7\t1\t605036 47013\n",
      "38\t597268\t1.7\t1\t559355 37913\n",
      "39\t577372\t1.7\t1\t541971 35401\n",
      "40\t649729\t1.7\t1\t605597 44132\n",
      "41\t640639\t1.7\t1\t597788 42851\n",
      "42\t636924\t1.7\t1\t595731 41193\n",
      "43\t595156\t1.7\t1\t555850 39306\n",
      "44\t689081\t1.7\t1\t641836 47245\n",
      "45\t705414\t1.7\t1\t658767 46647\n",
      "46\t511671\t1.7\t1\t477756 33915\n",
      "47\t627959\t1.7\t1\t589895 38064\n",
      "48\t606191\t1.7\t1\t569490 36701\n",
      "49\t638780\t1.7\t1\t597375 41405\n",
      "50\t632872\t1.7\t1\t591113 41759\n",
      "51\t767529\t1.7\t1\t718431 49098\n",
      "52\t550691\t1.7\t1\t515808 34883\n",
      "53\t670844\t1.7\t1\t631272 39572\n",
      "54\t486646\t1.7\t1\t457582 29064\n",
      "55\t619044\t1.7\t1\t583828 35216\n",
      "56\t629194\t1.7\t1\t589212 39982\n",
      "57\t612138\t1.7\t1\t577341 34797\n",
      "58\t592465\t1.7\t1\t558875 33590\n",
      "59\t595468\t1.7\t1\t560221 35247\n",
      "60\t612569\t1.7\t1\t576229 36340\n",
      "61\t622280\t1.7\t1\t586602 35678\n",
      "62\t676352\t1.7\t1\t634848 41504\n",
      "63\t865976\t1.7\t1\t819740 46236\n",
      "64\t686055\t1.7\t1\t654087 31968\n",
      "65\t461983\t1.7\t1\t436125 25858\n",
      "66\t506619\t1.7\t1\t477735 28884\n",
      "67\t507018\t1.7\t1\t477302 29716\n",
      "68\t510556\t1.7\t1\t481168 29388\n",
      "69\t507332\t1.7\t1\t477239 30093\n",
      "70\t512308\t1.7\t1\t482423 29885\n",
      "71\t532127\t1.7\t1\t501527 30600\n",
      "72\t518080\t1.7\t1\t487908 30172\n",
      "73\t537157\t1.7\t1\t506806 30351\n",
      "74\t495539\t1.7\t1\t466890 28649\n",
      "75\t470503\t1.7\t1\t443438 27065\n",
      "76\t452980\t1.7\t1\t426860 26120\n",
      "77\t443451\t1.7\t1\t417656 25795\n",
      "78\t435086\t1.7\t1\t409995 25091\n",
      "79\t419597\t1.7\t1\t395575 24022\n",
      "80\t414921\t1.7\t1\t392020 22901\n",
      "81\t399796\t1.7\t1\t377201 22595\n",
      "82\t383327\t1.7\t1\t361982 21345\n",
      "83\t367682\t1.7\t1\t347411 20271\n",
      "84\t357304\t1.7\t1\t337566 19738\n",
      "85\t348720\t1.7\t1\t329480 19240\n",
      "86\t342051\t1.7\t1\t323486 18565\n",
      "87\t342842\t1.7\t1\t324860 17982\n",
      "88\t330713\t1.7\t1\t313318 17395\n",
      "89\t294825\t1.7\t1\t279096 15729\n",
      "90\t280371\t1.7\t1\t265521 14850\n",
      "91\t263612\t1.7\t1\t249496 14116\n",
      "92\t248322\t1.7\t1\t235160 13162\n",
      "93\t241085\t1.7\t1\t228414 12671\n",
      "94\t227365\t1.7\t1\t215260 12105\n",
      "95\t212715\t1.7\t1\t201138 11577\n",
      "96\t201763\t1.7\t1\t191081 10682\n",
      "97\t190347\t1.7\t1\t180592 9755\n",
      "98\t180927\t1.7\t1\t171879 9048\n",
      "99\t177126\t1.7\t1\t168081 9045\n",
      "100\t161967\t1.7\t1\t153628 8339\n",
      "101\t143338\t1.7\t1\t136043 7295\n",
      "102\t127435\t1.7\t1\t120719 6716\n",
      "103\t114642\t1.7\t1\t108603 6039\n",
      "104\t106928\t1.7\t1\t101222 5706\n",
      "105\t97130\t1.7\t1\t92087 5043\n",
      "106\t88346\t1.7\t1\t83845 4501\n",
      "107\t77435\t1.7\t1\t73389 4046\n",
      "108\t67272\t1.7\t1\t63761 3511\n",
      "109\t60831\t1.7\t1\t57633 3198\n",
      "110\t53937\t1.7\t1\t51056 2881\n",
      "111\t49603\t1.7\t1\t47000 2603\n",
      "112\t44226\t1.7\t1\t41868 2358\n",
      "113\t38028\t1.7\t1\t35980 2048\n",
      "114\t33519\t1.7\t1\t31680 1839\n",
      "115\t26723\t1.7\t1\t25283 1440\n",
      "116\t22936\t1.7\t1\t21626 1310\n",
      "117\t19039\t1.7\t1\t17966 1073\n",
      "118\t15805\t1.7\t1\t14850 955\n",
      "119\t13430\t1.7\t1\t12634 796\n",
      "120\t11390\t1.7\t1\t10687 703\n",
      "121\t9723\t1.7\t1\t9101 622\n",
      "122\t8226\t1.7\t1\t7693 533\n",
      "123\t6741\t1.7\t1\t6334 407\n",
      "124\t5699\t1.7\t1\t5276 423\n",
      "125\t5010\t1.7\t1\t4637 373\n",
      "126\t4661\t1.7\t1\t4329 332\n",
      "127\t3987\t1.7\t1\t3713 274\n",
      "128\t2901\t1.7\t1\t2665 236\n",
      "129\t2466\t1.7\t1\t2254 212\n",
      "130\t2939\t1.7\t1\t2734 205\n",
      "131\t4275\t1.7\t1\t4050 225\n",
      "132\t4899\t1.7\t1\t4690 209\n",
      "133\t7071\t1.7\t1\t6772 299\n",
      "134\t8958\t1.7\t1\t8601 357\n",
      "135\t3598\t1.7\t1\t3428 170\n",
      "136\t1085\t1.7\t1\t1002 83\n",
      "137\t586\t1.7\t1\t522 64\n",
      "138\t435\t1.7\t1\t381 54\n",
      "139\t265\t1.7\t1\t235 30\n",
      "140\t195\t1.7\t1\t157 38\n",
      "141\t157\t1.7\t1\t136 21\n",
      "142\t96\t1.7\t1\t75 21\n",
      "143\t89\t1.7\t1\t70 19\n",
      "144\t116\t1.7\t1\t91 25\n",
      "145\t262\t1.7\t1\t239 23\n",
      "146\t193\t1.7\t1\t171 22\n",
      "147\t646\t1.7\t1\t552 94\n",
      "148\t333\t1.7\t1\t271 62\n",
      "149\t178\t1.7\t1\t147 31\n",
      "150\t1314\t1.7\t1\t1144 170\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "112706276 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files Sealice_F1_S20_L002_R1_001_trimmed.fq.gz and Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "file_1: Sealice_F1_S20_L002_R1_001_trimmed.fq.gz, file_2: Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: Sealice_F1_S20_L002_R1_001_trimmed.fq.gz and Sealice_F1_S20_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 112706276\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 213274 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F1_S20_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Approx 100% complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F1_S20_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Approx 100% complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for Sealice_F1_S20_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files Sealice_F1_S20_L002_R1_001_trimmed.fq.gz and Sealice_F1_S20_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F2_S22_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3388.64 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              82,177,525\n",
      "Reads with adapters:                58,714,265 (71.4%)\n",
      "Reads written (passing filters):    82,177,525 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,326,628,750 bp\n",
      "Quality-trimmed:              17,863,332 bp (0.1%)\n",
      "Total written (filtered):  10,356,260,345 bp (84.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58714265 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.4%\n",
      "  C: 11.4%\n",
      "  G: 19.3%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15334703\t20544381.2\t0\t15334703\n",
      "2\t1608191\t5136095.3\t0\t1608191\n",
      "3\t847327\t1284023.8\t0\t847327\n",
      "4\t558993\t321006.0\t0\t558993\n",
      "5\t451836\t80251.5\t0\t451836\n",
      "6\t436528\t20062.9\t0\t436528\n",
      "7\t362332\t5015.7\t0\t362332\n",
      "8\t445624\t1253.9\t0\t445624\n",
      "9\t448853\t313.5\t0\t444071 4782\n",
      "10\t414974\t78.4\t1\t395643 19331\n",
      "11\t451608\t19.6\t1\t421964 29644\n",
      "12\t446668\t4.9\t1\t421394 25274\n",
      "13\t448758\t1.2\t1\t423173 25585\n",
      "14\t458328\t1.2\t1\t429437 28891\n",
      "15\t470816\t1.2\t1\t443807 27009\n",
      "16\t459713\t1.2\t1\t429546 30167\n",
      "17\t546940\t1.2\t1\t508999 37941\n",
      "18\t471227\t1.2\t1\t445754 25473\n",
      "19\t402058\t1.2\t1\t378920 23138\n",
      "20\t471685\t1.2\t1\t445002 26683\n",
      "21\t519729\t1.2\t1\t487230 32499\n",
      "22\t441834\t1.2\t1\t419209 22625\n",
      "23\t505202\t1.2\t1\t475325 29877\n",
      "24\t480556\t1.2\t1\t445808 34748\n",
      "25\t495463\t1.2\t1\t465784 29679\n",
      "26\t468036\t1.2\t1\t442448 25588\n",
      "27\t512154\t1.2\t1\t478482 33672\n",
      "28\t492492\t1.2\t1\t462349 30143\n",
      "29\t513596\t1.2\t1\t476912 36684\n",
      "30\t549763\t1.2\t1\t521613 28150\n",
      "31\t416541\t1.2\t1\t392371 24170\n",
      "32\t537990\t1.2\t1\t507240 30750\n",
      "33\t461062\t1.2\t1\t434172 26890\n",
      "34\t514898\t1.2\t1\t482060 32838\n",
      "35\t565682\t1.2\t1\t527137 38545\n",
      "36\t520514\t1.2\t1\t491038 29476\n",
      "37\t563159\t1.2\t1\t535989 27170\n",
      "38\t400017\t1.2\t1\t369694 30323\n",
      "39\t571304\t1.2\t1\t535130 36174\n",
      "40\t474621\t1.2\t1\t448623 25998\n",
      "41\t600561\t1.2\t1\t562019 38542\n",
      "42\t458766\t1.2\t1\t426402 32364\n",
      "43\t800955\t1.2\t1\t763696 37259\n",
      "44\t235002\t1.2\t1\t215582 19420\n",
      "45\t521706\t1.2\t1\t491650 30056\n",
      "46\t537461\t1.2\t1\t505916 31545\n",
      "47\t540258\t1.2\t1\t510434 29824\n",
      "48\t449883\t1.2\t1\t420809 29074\n",
      "49\t566867\t1.2\t1\t529971 36896\n",
      "50\t457025\t1.2\t1\t432856 24169\n",
      "51\t503689\t1.2\t1\t474572 29117\n",
      "52\t554879\t1.2\t1\t516852 38027\n",
      "53\t630277\t1.2\t1\t598162 32115\n",
      "54\t395812\t1.2\t1\t371351 24461\n",
      "55\t516387\t1.2\t1\t488413 27974\n",
      "56\t554072\t1.2\t1\t525098 28974\n",
      "57\t493826\t1.2\t1\t465005 28821\n",
      "58\t455432\t1.2\t1\t435241 20191\n",
      "59\t348004\t1.2\t1\t326224 21780\n",
      "60\t524024\t1.2\t1\t495485 28539\n",
      "61\t494677\t1.2\t1\t470013 24664\n",
      "62\t445581\t1.2\t1\t422101 23480\n",
      "63\t440868\t1.2\t1\t415824 25044\n",
      "64\t419014\t1.2\t1\t397741 21273\n",
      "65\t430484\t1.2\t1\t407046 23438\n",
      "66\t478836\t1.2\t1\t451552 27284\n",
      "67\t526917\t1.2\t1\t490973 35944\n",
      "68\t974235\t1.2\t1\t930569 43666\n",
      "69\t263384\t1.2\t1\t246743 16641\n",
      "70\t78060\t1.2\t1\t68230 9830\n",
      "71\t188944\t1.2\t1\t177000 11944\n",
      "72\t350249\t1.2\t1\t331713 18536\n",
      "73\t385004\t1.2\t1\t364669 20335\n",
      "74\t373731\t1.2\t1\t353304 20427\n",
      "75\t372636\t1.2\t1\t352683 19953\n",
      "76\t364052\t1.2\t1\t344608 19444\n",
      "77\t349658\t1.2\t1\t330711 18947\n",
      "78\t336268\t1.2\t1\t317764 18504\n",
      "79\t333907\t1.2\t1\t315525 18382\n",
      "80\t320027\t1.2\t1\t302554 17473\n",
      "81\t308343\t1.2\t1\t291381 16962\n",
      "82\t303079\t1.2\t1\t286638 16441\n",
      "83\t285198\t1.2\t1\t269542 15656\n",
      "84\t281618\t1.2\t1\t266189 15429\n",
      "85\t270078\t1.2\t1\t255167 14911\n",
      "86\t256741\t1.2\t1\t242477 14264\n",
      "87\t246300\t1.2\t1\t232699 13601\n",
      "88\t239359\t1.2\t1\t226087 13272\n",
      "89\t221157\t1.2\t1\t208813 12344\n",
      "90\t214943\t1.2\t1\t202774 12169\n",
      "91\t203469\t1.2\t1\t192053 11416\n",
      "92\t193081\t1.2\t1\t182042 11039\n",
      "93\t185749\t1.2\t1\t175077 10672\n",
      "94\t173524\t1.2\t1\t163677 9847\n",
      "95\t160182\t1.2\t1\t150706 9476\n",
      "96\t148958\t1.2\t1\t140034 8924\n",
      "97\t139705\t1.2\t1\t131505 8200\n",
      "98\t129444\t1.2\t1\t121874 7570\n",
      "99\t121244\t1.2\t1\t114040 7204\n",
      "100\t113165\t1.2\t1\t106483 6682\n",
      "101\t102845\t1.2\t1\t96512 6333\n",
      "102\t94108\t1.2\t1\t88374 5734\n",
      "103\t85449\t1.2\t1\t80229 5220\n",
      "104\t76917\t1.2\t1\t72056 4861\n",
      "105\t69454\t1.2\t1\t64912 4542\n",
      "106\t62657\t1.2\t1\t58630 4027\n",
      "107\t54683\t1.2\t1\t51109 3574\n",
      "108\t49110\t1.2\t1\t45908 3202\n",
      "109\t44137\t1.2\t1\t41058 3079\n",
      "110\t38372\t1.2\t1\t35617 2755\n",
      "111\t33619\t1.2\t1\t31136 2483\n",
      "112\t30434\t1.2\t1\t28239 2195\n",
      "113\t25366\t1.2\t1\t23490 1876\n",
      "114\t23496\t1.2\t1\t21717 1779\n",
      "115\t19033\t1.2\t1\t17590 1443\n",
      "116\t16344\t1.2\t1\t14972 1372\n",
      "117\t13454\t1.2\t1\t12326 1128\n",
      "118\t11466\t1.2\t1\t10507 959\n",
      "119\t8924\t1.2\t1\t8136 788\n",
      "120\t8008\t1.2\t1\t7309 699\n",
      "121\t6748\t1.2\t1\t6146 602\n",
      "122\t5464\t1.2\t1\t4939 525\n",
      "123\t4525\t1.2\t1\t4074 451\n",
      "124\t3864\t1.2\t1\t3437 427\n",
      "125\t3136\t1.2\t1\t2820 316\n",
      "126\t2478\t1.2\t1\t2171 307\n",
      "127\t2123\t1.2\t1\t1865 258\n",
      "128\t1659\t1.2\t1\t1463 196\n",
      "129\t1306\t1.2\t1\t1161 145\n",
      "130\t1169\t1.2\t1\t1027 142\n",
      "131\t997\t1.2\t1\t875 122\n",
      "132\t756\t1.2\t1\t652 104\n",
      "133\t674\t1.2\t1\t579 95\n",
      "134\t665\t1.2\t1\t580 85\n",
      "135\t442\t1.2\t1\t374 68\n",
      "136\t281\t1.2\t1\t246 35\n",
      "137\t223\t1.2\t1\t178 45\n",
      "138\t175\t1.2\t1\t145 30\n",
      "139\t132\t1.2\t1\t108 24\n",
      "140\t105\t1.2\t1\t88 17\n",
      "141\t84\t1.2\t1\t61 23\n",
      "142\t69\t1.2\t1\t59 10\n",
      "143\t49\t1.2\t1\t38 11\n",
      "144\t68\t1.2\t1\t56 12\n",
      "145\t42\t1.2\t1\t31 11\n",
      "146\t49\t1.2\t1\t40 9\n",
      "147\t75\t1.2\t1\t67 8\n",
      "148\t68\t1.2\t1\t58 10\n",
      "149\t58\t1.2\t1\t50 8\n",
      "150\t375\t1.2\t1\t309 66\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "82177525 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F2_S22_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3587.03 s (44 us/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              82,177,525\n",
      "Reads with adapters:                58,336,382 (71.0%)\n",
      "Reads written (passing filters):    82,177,525 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,326,628,750 bp\n",
      "Quality-trimmed:              41,419,626 bp (0.3%)\n",
      "Total written (filtered):  10,364,431,817 bp (84.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58336382 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.7%\n",
      "  C: 9.9%\n",
      "  G: 22.0%\n",
      "  T: 34.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15350057\t20544381.2\t0\t15350057\n",
      "2\t1685474\t5136095.3\t0\t1685474\n",
      "3\t903882\t1284023.8\t0\t903882\n",
      "4\t571222\t321006.0\t0\t571222\n",
      "5\t501936\t80251.5\t0\t501936\n",
      "6\t415827\t20062.9\t0\t415827\n",
      "7\t347542\t5015.7\t0\t347542\n",
      "8\t483968\t1253.9\t0\t483968\n",
      "9\t396796\t313.5\t0\t395692 1104\n",
      "10\t420274\t78.4\t1\t402160 18114\n",
      "11\t438864\t19.6\t1\t414608 24256\n",
      "12\t449536\t4.9\t1\t424165 25371\n",
      "13\t510213\t1.2\t1\t482515 27698\n",
      "14\t455618\t1.2\t1\t427602 28016\n",
      "15\t389210\t1.2\t1\t366432 22778\n",
      "16\t555171\t1.2\t1\t523342 31829\n",
      "17\t419400\t1.2\t1\t394087 25313\n",
      "18\t418646\t1.2\t1\t395171 23475\n",
      "19\t509488\t1.2\t1\t480141 29347\n",
      "20\t480908\t1.2\t1\t453798 27110\n",
      "21\t427296\t1.2\t1\t402705 24591\n",
      "22\t474401\t1.2\t1\t448452 25949\n",
      "23\t490252\t1.2\t1\t461826 28426\n",
      "24\t484904\t1.2\t1\t457572 27332\n",
      "25\t539104\t1.2\t1\t510128 28976\n",
      "26\t453073\t1.2\t1\t425411 27662\n",
      "27\t501857\t1.2\t1\t468630 33227\n",
      "28\t457877\t1.2\t1\t436189 21688\n",
      "29\t491156\t1.2\t1\t462299 28857\n",
      "30\t462417\t1.2\t1\t439850 22567\n",
      "31\t522905\t1.2\t1\t492898 30007\n",
      "32\t460424\t1.2\t1\t437863 22561\n",
      "33\t512386\t1.2\t1\t483241 29145\n",
      "34\t657426\t1.2\t1\t620019 37407\n",
      "35\t363385\t1.2\t1\t345024 18361\n",
      "36\t487371\t1.2\t1\t459211 28160\n",
      "37\t533683\t1.2\t1\t504069 29614\n",
      "38\t481104\t1.2\t1\t458652 22452\n",
      "39\t485376\t1.2\t1\t462217 23159\n",
      "40\t532302\t1.2\t1\t503234 29068\n",
      "41\t511389\t1.2\t1\t485806 25583\n",
      "42\t521546\t1.2\t1\t495601 25945\n",
      "43\t480650\t1.2\t1\t455868 24782\n",
      "44\t546125\t1.2\t1\t517755 28370\n",
      "45\t570120\t1.2\t1\t542069 28051\n",
      "46\t418884\t1.2\t1\t397786 21098\n",
      "47\t505326\t1.2\t1\t481272 24054\n",
      "48\t488484\t1.2\t1\t466397 22087\n",
      "49\t514214\t1.2\t1\t489094 25120\n",
      "50\t501739\t1.2\t1\t476010 25729\n",
      "51\t610289\t1.2\t1\t580951 29338\n",
      "52\t441047\t1.2\t1\t421068 19979\n",
      "53\t520314\t1.2\t1\t497213 23101\n",
      "54\t398195\t1.2\t1\t379916 18279\n",
      "55\t496805\t1.2\t1\t475323 21482\n",
      "56\t496057\t1.2\t1\t473000 23057\n",
      "57\t480165\t1.2\t1\t458650 21515\n",
      "58\t465002\t1.2\t1\t445044 19958\n",
      "59\t473368\t1.2\t1\t452197 21171\n",
      "60\t477456\t1.2\t1\t455816 21640\n",
      "61\t491054\t1.2\t1\t468379 22675\n",
      "62\t525084\t1.2\t1\t500148 24936\n",
      "63\t688105\t1.2\t1\t659460 28645\n",
      "64\t513363\t1.2\t1\t493656 19707\n",
      "65\t351437\t1.2\t1\t336316 15121\n",
      "66\t388237\t1.2\t1\t371824 16413\n",
      "67\t391608\t1.2\t1\t373999 17609\n",
      "68\t389520\t1.2\t1\t372639 16881\n",
      "69\t387449\t1.2\t1\t370434 17015\n",
      "70\t386662\t1.2\t1\t369736 16926\n",
      "71\t401493\t1.2\t1\t384632 16861\n",
      "72\t387884\t1.2\t1\t370945 16939\n",
      "73\t410793\t1.2\t1\t393742 17051\n",
      "74\t371310\t1.2\t1\t356033 15277\n",
      "75\t357117\t1.2\t1\t342286 14831\n",
      "76\t343694\t1.2\t1\t329251 14443\n",
      "77\t331451\t1.2\t1\t317224 14227\n",
      "78\t322073\t1.2\t1\t308399 13674\n",
      "79\t311753\t1.2\t1\t298772 12981\n",
      "80\t303996\t1.2\t1\t291594 12402\n",
      "81\t292352\t1.2\t1\t280150 12202\n",
      "82\t283173\t1.2\t1\t271407 11766\n",
      "83\t268006\t1.2\t1\t256925 11081\n",
      "84\t264798\t1.2\t1\t253456 11342\n",
      "85\t258296\t1.2\t1\t247093 11203\n",
      "86\t251541\t1.2\t1\t240826 10715\n",
      "87\t251948\t1.2\t1\t241533 10415\n",
      "88\t239999\t1.2\t1\t230066 9933\n",
      "89\t210279\t1.2\t1\t201522 8757\n",
      "90\t198432\t1.2\t1\t189963 8469\n",
      "91\t186654\t1.2\t1\t178634 8020\n",
      "92\t175644\t1.2\t1\t168171 7473\n",
      "93\t173446\t1.2\t1\t166287 7159\n",
      "94\t161637\t1.2\t1\t154843 6794\n",
      "95\t151052\t1.2\t1\t144859 6193\n",
      "96\t140237\t1.2\t1\t134508 5729\n",
      "97\t130886\t1.2\t1\t125542 5344\n",
      "98\t122501\t1.2\t1\t117472 5029\n",
      "99\t115054\t1.2\t1\t110586 4468\n",
      "100\t107801\t1.2\t1\t103547 4254\n",
      "101\t95638\t1.2\t1\t91917 3721\n",
      "102\t87834\t1.2\t1\t84365 3469\n",
      "103\t80762\t1.2\t1\t77514 3248\n",
      "104\t75976\t1.2\t1\t73089 2887\n",
      "105\t67619\t1.2\t1\t64884 2735\n",
      "106\t60836\t1.2\t1\t58481 2355\n",
      "107\t52108\t1.2\t1\t50074 2034\n",
      "108\t45224\t1.2\t1\t43372 1852\n",
      "109\t41239\t1.2\t1\t39535 1704\n",
      "110\t35967\t1.2\t1\t34472 1495\n",
      "111\t33060\t1.2\t1\t31766 1294\n",
      "112\t29404\t1.2\t1\t28175 1229\n",
      "113\t24708\t1.2\t1\t23673 1035\n",
      "114\t22438\t1.2\t1\t21537 901\n",
      "115\t17752\t1.2\t1\t17001 751\n",
      "116\t15010\t1.2\t1\t14377 633\n",
      "117\t12079\t1.2\t1\t11520 559\n",
      "118\t10437\t1.2\t1\t9967 470\n",
      "119\t8208\t1.2\t1\t7826 382\n",
      "120\t7212\t1.2\t1\t6891 321\n",
      "121\t6273\t1.2\t1\t5988 285\n",
      "122\t5055\t1.2\t1\t4818 237\n",
      "123\t4309\t1.2\t1\t4110 199\n",
      "124\t3571\t1.2\t1\t3389 182\n",
      "125\t2948\t1.2\t1\t2833 115\n",
      "126\t2296\t1.2\t1\t2182 114\n",
      "127\t1959\t1.2\t1\t1851 108\n",
      "128\t1542\t1.2\t1\t1473 69\n",
      "129\t1221\t1.2\t1\t1160 61\n",
      "130\t1047\t1.2\t1\t984 63\n",
      "131\t945\t1.2\t1\t892 53\n",
      "132\t678\t1.2\t1\t641 37\n",
      "133\t631\t1.2\t1\t593 38\n",
      "134\t648\t1.2\t1\t616 32\n",
      "135\t387\t1.2\t1\t364 23\n",
      "136\t258\t1.2\t1\t239 19\n",
      "137\t195\t1.2\t1\t184 11\n",
      "138\t163\t1.2\t1\t151 12\n",
      "139\t116\t1.2\t1\t108 8\n",
      "140\t92\t1.2\t1\t83 9\n",
      "141\t73\t1.2\t1\t63 10\n",
      "142\t63\t1.2\t1\t54 9\n",
      "143\t46\t1.2\t1\t37 9\n",
      "144\t54\t1.2\t1\t48 6\n",
      "145\t44\t1.2\t1\t35 9\n",
      "146\t46\t1.2\t1\t42 4\n",
      "147\t78\t1.2\t1\t75 3\n",
      "148\t63\t1.2\t1\t57 6\n",
      "149\t48\t1.2\t1\t46 2\n",
      "150\t297\t1.2\t1\t280 17\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "82177525 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files Sealice_F2_S22_L001_R1_001_trimmed.fq.gz and Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "file_1: Sealice_F2_S22_L001_R1_001_trimmed.fq.gz, file_2: Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: Sealice_F2_S22_L001_R1_001_trimmed.fq.gz and Sealice_F2_S22_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 82177525\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 97957 (0.12%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F2_S22_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F2_S22_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files Sealice_F2_S22_L001_R1_001_trimmed.fq.gz and Sealice_F2_S22_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F2_S22_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3435.79 s (42 us/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              81,699,034\n",
      "Reads with adapters:                58,422,606 (71.5%)\n",
      "Reads written (passing filters):    81,699,034 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,254,855,100 bp\n",
      "Quality-trimmed:              19,487,828 bp (0.2%)\n",
      "Total written (filtered):  10,283,682,296 bp (83.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 58422606 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.4%\n",
      "  C: 11.4%\n",
      "  G: 19.4%\n",
      "  T: 33.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15199129\t20424758.5\t0\t15199129\n",
      "2\t1580922\t5106189.6\t0\t1580922\n",
      "3\t842043\t1276547.4\t0\t842043\n",
      "4\t550022\t319136.9\t0\t550022\n",
      "5\t449997\t79784.2\t0\t449997\n",
      "6\t434481\t19946.1\t0\t434481\n",
      "7\t347898\t4986.5\t0\t347898\n",
      "8\t441084\t1246.6\t0\t441084\n",
      "9\t443045\t311.7\t0\t438482 4563\n",
      "10\t405413\t77.9\t1\t382098 23315\n",
      "11\t450388\t19.5\t1\t415672 34716\n",
      "12\t442770\t4.9\t1\t412586 30184\n",
      "13\t446024\t1.2\t1\t415108 30916\n",
      "14\t453376\t1.2\t1\t419388 33988\n",
      "15\t468278\t1.2\t1\t435263 33015\n",
      "16\t452666\t1.2\t1\t416929 35737\n",
      "17\t549238\t1.2\t1\t503600 45638\n",
      "18\t466886\t1.2\t1\t435945 30941\n",
      "19\t393788\t1.2\t1\t366567 27221\n",
      "20\t467902\t1.2\t1\t436166 31736\n",
      "21\t521792\t1.2\t1\t483620 38172\n",
      "22\t433680\t1.2\t1\t406255 27425\n",
      "23\t504819\t1.2\t1\t468719 36100\n",
      "24\t475517\t1.2\t1\t437053 38464\n",
      "25\t491820\t1.2\t1\t457067 34753\n",
      "26\t461178\t1.2\t1\t431235 29943\n",
      "27\t512653\t1.2\t1\t473769 38884\n",
      "28\t491651\t1.2\t1\t457681 33970\n",
      "29\t512510\t1.2\t1\t470963 41547\n",
      "30\t548141\t1.2\t1\t514010 34131\n",
      "31\t409025\t1.2\t1\t380832 28193\n",
      "32\t540631\t1.2\t1\t502491 38140\n",
      "33\t452892\t1.2\t1\t421055 31837\n",
      "34\t512380\t1.2\t1\t471655 40725\n",
      "35\t569764\t1.2\t1\t523370 46394\n",
      "36\t519856\t1.2\t1\t484148 35708\n",
      "37\t553142\t1.2\t1\t519802 33340\n",
      "38\t400544\t1.2\t1\t364810 35734\n",
      "39\t573002\t1.2\t1\t528499 44503\n",
      "40\t466836\t1.2\t1\t435154 31682\n",
      "41\t600726\t1.2\t1\t554364 46362\n",
      "42\t453939\t1.2\t1\t416218 37721\n",
      "43\t790864\t1.2\t1\t744421 46443\n",
      "44\t241028\t1.2\t1\t218439 22589\n",
      "45\t522905\t1.2\t1\t486598 36307\n",
      "46\t536313\t1.2\t1\t498381 37932\n",
      "47\t536772\t1.2\t1\t501509 35263\n",
      "48\t445960\t1.2\t1\t412336 33624\n",
      "49\t569976\t1.2\t1\t527084 42892\n",
      "50\t451713\t1.2\t1\t423569 28144\n",
      "51\t503554\t1.2\t1\t469351 34203\n",
      "52\t559241\t1.2\t1\t515373 43868\n",
      "53\t626961\t1.2\t1\t588950 38011\n",
      "54\t397202\t1.2\t1\t368180 29022\n",
      "55\t510273\t1.2\t1\t477059 33214\n",
      "56\t566018\t1.2\t1\t530313 35705\n",
      "57\t490065\t1.2\t1\t455879 34186\n",
      "58\t448852\t1.2\t1\t423488 25364\n",
      "59\t352650\t1.2\t1\t326693 25957\n",
      "60\t521343\t1.2\t1\t488341 33002\n",
      "61\t489445\t1.2\t1\t459888 29557\n",
      "62\t454351\t1.2\t1\t425498 28853\n",
      "63\t442859\t1.2\t1\t412719 30140\n",
      "64\t413454\t1.2\t1\t388538 24916\n",
      "65\t428487\t1.2\t1\t400331 28156\n",
      "66\t475641\t1.2\t1\t444595 31046\n",
      "67\t527546\t1.2\t1\t487484 40062\n",
      "68\t932793\t1.2\t1\t883259 49534\n",
      "69\t306211\t1.2\t1\t285490 20721\n",
      "70\t98330\t1.2\t1\t86452 11878\n",
      "71\t183213\t1.2\t1\t170097 13116\n",
      "72\t347828\t1.2\t1\t327310 20518\n",
      "73\t384493\t1.2\t1\t361546 22947\n",
      "74\t375434\t1.2\t1\t351755 23679\n",
      "75\t372810\t1.2\t1\t349151 23659\n",
      "76\t366345\t1.2\t1\t341732 24613\n",
      "77\t349631\t1.2\t1\t326987 22644\n",
      "78\t336085\t1.2\t1\t313727 22358\n",
      "79\t334791\t1.2\t1\t312735 22056\n",
      "80\t320766\t1.2\t1\t299322 21444\n",
      "81\t308813\t1.2\t1\t288662 20151\n",
      "82\t303376\t1.2\t1\t282871 20505\n",
      "83\t287613\t1.2\t1\t268884 18729\n",
      "84\t281524\t1.2\t1\t262646 18878\n",
      "85\t269523\t1.2\t1\t251601 17922\n",
      "86\t255982\t1.2\t1\t238963 17019\n",
      "87\t246467\t1.2\t1\t230371 16096\n",
      "88\t240096\t1.2\t1\t224172 15924\n",
      "89\t222064\t1.2\t1\t207380 14684\n",
      "90\t214626\t1.2\t1\t200391 14235\n",
      "91\t204511\t1.2\t1\t191212 13299\n",
      "92\t193574\t1.2\t1\t180679 12895\n",
      "93\t185417\t1.2\t1\t173008 12409\n",
      "94\t174914\t1.2\t1\t163234 11680\n",
      "95\t159670\t1.2\t1\t148732 10938\n",
      "96\t150695\t1.2\t1\t140365 10330\n",
      "97\t140751\t1.2\t1\t131592 9159\n",
      "98\t129553\t1.2\t1\t120450 9103\n",
      "99\t122439\t1.2\t1\t114105 8334\n",
      "100\t113910\t1.2\t1\t106001 7909\n",
      "101\t103551\t1.2\t1\t96319 7232\n",
      "102\t95288\t1.2\t1\t88666 6622\n",
      "103\t86137\t1.2\t1\t80145 5992\n",
      "104\t77327\t1.2\t1\t71858 5469\n",
      "105\t69993\t1.2\t1\t65045 4948\n",
      "106\t63171\t1.2\t1\t58489 4682\n",
      "107\t54679\t1.2\t1\t50722 3957\n",
      "108\t49406\t1.2\t1\t45775 3631\n",
      "109\t44550\t1.2\t1\t41176 3374\n",
      "110\t38825\t1.2\t1\t35771 3054\n",
      "111\t33501\t1.2\t1\t30848 2653\n",
      "112\t30928\t1.2\t1\t28428 2500\n",
      "113\t25660\t1.2\t1\t23574 2086\n",
      "114\t23733\t1.2\t1\t21729 2004\n",
      "115\t19374\t1.2\t1\t17711 1663\n",
      "116\t16667\t1.2\t1\t15185 1482\n",
      "117\t13526\t1.2\t1\t12278 1248\n",
      "118\t11411\t1.2\t1\t10398 1013\n",
      "119\t9123\t1.2\t1\t8243 880\n",
      "120\t8085\t1.2\t1\t7320 765\n",
      "121\t6821\t1.2\t1\t6153 668\n",
      "122\t5630\t1.2\t1\t5011 619\n",
      "123\t4598\t1.2\t1\t4108 490\n",
      "124\t3796\t1.2\t1\t3390 406\n",
      "125\t3140\t1.2\t1\t2802 338\n",
      "126\t2509\t1.2\t1\t2205 304\n",
      "127\t2125\t1.2\t1\t1865 260\n",
      "128\t1671\t1.2\t1\t1452 219\n",
      "129\t1329\t1.2\t1\t1142 187\n",
      "130\t1120\t1.2\t1\t991 129\n",
      "131\t940\t1.2\t1\t808 132\n",
      "132\t761\t1.2\t1\t666 95\n",
      "133\t675\t1.2\t1\t585 90\n",
      "134\t682\t1.2\t1\t604 78\n",
      "135\t447\t1.2\t1\t388 59\n",
      "136\t330\t1.2\t1\t286 44\n",
      "137\t223\t1.2\t1\t175 48\n",
      "138\t187\t1.2\t1\t160 27\n",
      "139\t149\t1.2\t1\t117 32\n",
      "140\t100\t1.2\t1\t80 20\n",
      "141\t64\t1.2\t1\t50 14\n",
      "142\t63\t1.2\t1\t51 12\n",
      "143\t43\t1.2\t1\t29 14\n",
      "144\t41\t1.2\t1\t32 9\n",
      "145\t44\t1.2\t1\t36 8\n",
      "146\t35\t1.2\t1\t24 11\n",
      "147\t75\t1.2\t1\t69 6\n",
      "148\t65\t1.2\t1\t52 13\n",
      "149\t34\t1.2\t1\t28 6\n",
      "150\t405\t1.2\t1\t345 60\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "81699034 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/Sealice_F2_S22_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 3646.69 s (45 us/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              81,699,034\n",
      "Reads with adapters:                57,979,113 (71.0%)\n",
      "Reads written (passing filters):    81,699,034 (100.0%)\n",
      "\n",
      "Total basepairs processed: 12,254,855,100 bp\n",
      "Quality-trimmed:              41,067,962 bp (0.3%)\n",
      "Total written (filtered):  10,294,963,583 bp (84.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 57979113 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.7%\n",
      "  C: 9.9%\n",
      "  G: 22.0%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15214651\t20424758.5\t0\t15214651\n",
      "2\t1629144\t5106189.6\t0\t1629144\n",
      "3\t914799\t1276547.4\t0\t914799\n",
      "4\t558975\t319136.9\t0\t558975\n",
      "5\t482109\t79784.2\t0\t482109\n",
      "6\t418136\t19946.1\t0\t418136\n",
      "7\t337570\t4986.5\t0\t337570\n",
      "8\t479586\t1246.6\t0\t479586\n",
      "9\t390650\t311.7\t0\t389615 1035\n",
      "10\t405041\t77.9\t1\t379605 25436\n",
      "11\t435020\t19.5\t1\t401922 33098\n",
      "12\t439217\t4.9\t1\t406727 32490\n",
      "13\t510754\t1.2\t1\t472376 38378\n",
      "14\t448612\t1.2\t1\t412140 36472\n",
      "15\t384027\t1.2\t1\t355388 28639\n",
      "16\t544298\t1.2\t1\t502865 41433\n",
      "17\t437427\t1.2\t1\t403719 33708\n",
      "18\t402979\t1.2\t1\t373023 29956\n",
      "19\t503067\t1.2\t1\t466488 36579\n",
      "20\t474971\t1.2\t1\t440326 34645\n",
      "21\t421874\t1.2\t1\t390910 30964\n",
      "22\t467522\t1.2\t1\t433687 33835\n",
      "23\t490952\t1.2\t1\t454530 36422\n",
      "24\t482710\t1.2\t1\t447678 35032\n",
      "25\t536922\t1.2\t1\t501311 35611\n",
      "26\t451785\t1.2\t1\t415820 35965\n",
      "27\t506974\t1.2\t1\t466123 40851\n",
      "28\t440254\t1.2\t1\t412940 27314\n",
      "29\t498335\t1.2\t1\t462544 35791\n",
      "30\t447616\t1.2\t1\t418293 29323\n",
      "31\t531574\t1.2\t1\t494290 37284\n",
      "32\t441801\t1.2\t1\t413285 28516\n",
      "33\t515665\t1.2\t1\t478090 37575\n",
      "34\t666212\t1.2\t1\t619804 46408\n",
      "35\t346551\t1.2\t1\t323116 23435\n",
      "36\t489499\t1.2\t1\t454133 35366\n",
      "37\t532182\t1.2\t1\t495525 36657\n",
      "38\t478707\t1.2\t1\t449260 29447\n",
      "39\t471359\t1.2\t1\t443454 27905\n",
      "40\t529670\t1.2\t1\t494697 34973\n",
      "41\t516177\t1.2\t1\t483272 32905\n",
      "42\t520597\t1.2\t1\t487926 32671\n",
      "43\t474470\t1.2\t1\t444318 30152\n",
      "44\t553215\t1.2\t1\t517017 36198\n",
      "45\t566191\t1.2\t1\t530084 36107\n",
      "46\t413041\t1.2\t1\t386808 26233\n",
      "47\t502147\t1.2\t1\t473043 29104\n",
      "48\t484308\t1.2\t1\t456231 28077\n",
      "49\t510392\t1.2\t1\t478462 31930\n",
      "50\t506166\t1.2\t1\t473768 32398\n",
      "51\t605207\t1.2\t1\t567914 37293\n",
      "52\t440187\t1.2\t1\t413511 26676\n",
      "53\t531819\t1.2\t1\t501544 30275\n",
      "54\t389426\t1.2\t1\t367011 22415\n",
      "55\t491938\t1.2\t1\t464562 27376\n",
      "56\t495033\t1.2\t1\t464600 30433\n",
      "57\t481452\t1.2\t1\t455340 26112\n",
      "58\t468033\t1.2\t1\t442245 25788\n",
      "59\t468551\t1.2\t1\t441634 26917\n",
      "60\t478522\t1.2\t1\t450900 27622\n",
      "61\t487423\t1.2\t1\t460617 26806\n",
      "62\t522057\t1.2\t1\t491364 30693\n",
      "63\t673446\t1.2\t1\t638844 34602\n",
      "64\t507915\t1.2\t1\t484583 23332\n",
      "65\t350472\t1.2\t1\t331804 18668\n",
      "66\t389451\t1.2\t1\t368357 21094\n",
      "67\t396849\t1.2\t1\t374262 22587\n",
      "68\t394541\t1.2\t1\t372684 21857\n",
      "69\t391045\t1.2\t1\t368794 22251\n",
      "70\t390887\t1.2\t1\t368825 22062\n",
      "71\t401886\t1.2\t1\t379624 22262\n",
      "72\t392080\t1.2\t1\t370490 21590\n",
      "73\t411065\t1.2\t1\t388688 22377\n",
      "74\t374532\t1.2\t1\t353760 20772\n",
      "75\t357961\t1.2\t1\t338145 19816\n",
      "76\t346375\t1.2\t1\t327002 19373\n",
      "77\t331945\t1.2\t1\t313179 18766\n",
      "78\t321538\t1.2\t1\t303798 17740\n",
      "79\t313177\t1.2\t1\t295979 17198\n",
      "80\t304441\t1.2\t1\t287925 16516\n",
      "81\t293724\t1.2\t1\t277950 15774\n",
      "82\t282766\t1.2\t1\t267333 15433\n",
      "83\t270058\t1.2\t1\t255507 14551\n",
      "84\t265196\t1.2\t1\t251091 14105\n",
      "85\t257861\t1.2\t1\t244467 13394\n",
      "86\t250545\t1.2\t1\t237466 13079\n",
      "87\t251526\t1.2\t1\t238890 12636\n",
      "88\t240557\t1.2\t1\t228280 12277\n",
      "89\t211890\t1.2\t1\t200619 11271\n",
      "90\t199074\t1.2\t1\t188762 10312\n",
      "91\t188118\t1.2\t1\t178454 9664\n",
      "92\t176434\t1.2\t1\t167450 8984\n",
      "93\t173462\t1.2\t1\t164594 8868\n",
      "94\t163597\t1.2\t1\t155286 8311\n",
      "95\t151070\t1.2\t1\t143169 7901\n",
      "96\t142096\t1.2\t1\t134776 7320\n",
      "97\t132469\t1.2\t1\t125885 6584\n",
      "98\t123111\t1.2\t1\t116966 6145\n",
      "99\t116382\t1.2\t1\t110385 5997\n",
      "100\t108683\t1.2\t1\t103116 5567\n",
      "101\t96750\t1.2\t1\t91809 4941\n",
      "102\t88826\t1.2\t1\t84306 4520\n",
      "103\t81399\t1.2\t1\t77208 4191\n",
      "104\t76198\t1.2\t1\t72290 3908\n",
      "105\t68158\t1.2\t1\t64724 3434\n",
      "106\t61130\t1.2\t1\t58100 3030\n",
      "107\t52011\t1.2\t1\t49328 2683\n",
      "108\t45529\t1.2\t1\t43239 2290\n",
      "109\t41683\t1.2\t1\t39576 2107\n",
      "110\t36218\t1.2\t1\t34303 1915\n",
      "111\t32883\t1.2\t1\t31247 1636\n",
      "112\t29793\t1.2\t1\t28318 1475\n",
      "113\t24999\t1.2\t1\t23719 1280\n",
      "114\t22662\t1.2\t1\t21525 1137\n",
      "115\t18096\t1.2\t1\t17176 920\n",
      "116\t15270\t1.2\t1\t14496 774\n",
      "117\t12170\t1.2\t1\t11514 656\n",
      "118\t10414\t1.2\t1\t9832 582\n",
      "119\t8403\t1.2\t1\t7959 444\n",
      "120\t7343\t1.2\t1\t6952 391\n",
      "121\t6317\t1.2\t1\t5964 353\n",
      "122\t5204\t1.2\t1\t4941 263\n",
      "123\t4340\t1.2\t1\t4122 218\n",
      "124\t3564\t1.2\t1\t3376 188\n",
      "125\t2965\t1.2\t1\t2783 182\n",
      "126\t2377\t1.2\t1\t2237 140\n",
      "127\t1922\t1.2\t1\t1812 110\n",
      "128\t1530\t1.2\t1\t1436 94\n",
      "129\t1224\t1.2\t1\t1157 67\n",
      "130\t1037\t1.2\t1\t967 70\n",
      "131\t861\t1.2\t1\t806 55\n",
      "132\t686\t1.2\t1\t633 53\n",
      "133\t661\t1.2\t1\t620 41\n",
      "134\t651\t1.2\t1\t603 48\n",
      "135\t432\t1.2\t1\t408 24\n",
      "136\t309\t1.2\t1\t286 23\n",
      "137\t192\t1.2\t1\t170 22\n",
      "138\t168\t1.2\t1\t155 13\n",
      "139\t128\t1.2\t1\t109 19\n",
      "140\t85\t1.2\t1\t78 7\n",
      "141\t53\t1.2\t1\t46 7\n",
      "142\t58\t1.2\t1\t51 7\n",
      "143\t31\t1.2\t1\t26 5\n",
      "144\t40\t1.2\t1\t32 8\n",
      "145\t40\t1.2\t1\t33 7\n",
      "146\t40\t1.2\t1\t29 11\n",
      "147\t75\t1.2\t1\t67 8\n",
      "148\t58\t1.2\t1\t52 6\n",
      "149\t37\t1.2\t1\t34 3\n",
      "150\t319\t1.2\t1\t294 25\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "81699034 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files Sealice_F2_S22_L002_R1_001_trimmed.fq.gz and Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "file_1: Sealice_F2_S22_L002_R1_001_trimmed.fq.gz, file_2: Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: Sealice_F2_S22_L002_R1_001_trimmed.fq.gz and Sealice_F2_S22_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 81699034\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 93016 (0.11%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F2_S22_L002_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data Sealice_F2_S22_L002_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for Sealice_F2_S22_L002_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files Sealice_F2_S22_L002_R1_001_trimmed.fq.gz and Sealice_F2_S22_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!/Users/strigg/anaconda3/bin/TrimGalore-0.6.0/trim_galore \\\n",
    "--output_dir /Users/strigg/Desktop/Salmo_Calig/TRIM \\\n",
    "--fastqc_args \\\n",
    "\"--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/FASTQC\" \\\n",
    "--clip_R1 5 \\\n",
    "--clip_R2 5 \\\n",
    "--paired \\\n",
    "--path_to_cutadapt /Users/strigg/anaconda3/bin/cutadapt \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_2_S14_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_3_S15_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_4_S16_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_1_S1_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_2_S2_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_3_S3_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_32psu_4_S4_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_1_S9_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_2_S10_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_3_S11_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_26psu_4_S12_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_1_S5_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_2_S6_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_3_S7_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/8C_32psu_4_S8_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_1_S19_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_16C_26psu_2_S21_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_1_S17_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/CTRL_8C_26psu_2_S18_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F1_S20_L002_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L001_R2_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/Sealice_F2_S22_L002_R2_001.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Run a test to see % reads retained when 'retain unpaired' option is used"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Multicore support not enabled. Proceeding with single-core trimming.\n",
      "Path to Cutadapt set as: '/Users/strigg/anaconda3/bin/cutadapt' (user defined)\n",
      "Cutadapt seems to be working fine (tested command '/Users/strigg/anaconda3/bin/cutadapt --version')\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /Users/strigg/Desktop/Salmo_Calig/TRIM/retained/\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t538094\tAGATCGGAAGAGC\t1000000\t53.81\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 538094). Second best hit was Nextera (count: 0)\n",
      "\n",
      "gzip: error writing to output: Broken pipe\n",
      "gzip: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz: uncompress failed\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/retained/16C_26psu_1_S13_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Length cut-off for read 1: 35 bp (default)\n",
      "Length cut-off for read 2: 35 bb (default)\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/retained/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 837.07 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,366,785 (78.4%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              20,636,466 bp (0.6%)\n",
      "Total written (filtered):  2,599,584,885 bp (78.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17366785 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 14.0%\n",
      "  G: 20.5%\n",
      "  T: 35.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2904827\t5539983.8\t0\t2904827\n",
      "2\t423079\t1384995.9\t0\t423079\n",
      "3\t231866\t346249.0\t0\t231866\n",
      "4\t143449\t86562.2\t0\t143449\n",
      "5\t117570\t21640.6\t0\t117570\n",
      "6\t115529\t5410.1\t0\t115529\n",
      "7\t96119\t1352.5\t0\t96119\n",
      "8\t119364\t338.1\t0\t119364\n",
      "9\t118406\t84.5\t0\t116886 1520\n",
      "10\t112051\t21.1\t1\t106032 6019\n",
      "11\t122987\t5.3\t1\t113734 9253\n",
      "12\t119477\t1.3\t1\t111842 7635\n",
      "13\t120023\t0.3\t1\t112246 7777\n",
      "14\t122963\t0.3\t1\t114085 8878\n",
      "15\t127528\t0.3\t1\t119209 8319\n",
      "16\t126445\t0.3\t1\t117002 9443\n",
      "17\t152761\t0.3\t1\t140488 12273\n",
      "18\t129037\t0.3\t1\t121213 7824\n",
      "19\t112889\t0.3\t1\t105773 7116\n",
      "20\t133616\t0.3\t1\t125004 8612\n",
      "21\t146516\t0.3\t1\t136291 10225\n",
      "22\t126419\t0.3\t1\t119045 7374\n",
      "23\t145391\t0.3\t1\t135716 9675\n",
      "24\t136894\t0.3\t1\t125670 11224\n",
      "25\t145117\t0.3\t1\t135313 9804\n",
      "26\t134897\t0.3\t1\t126717 8180\n",
      "27\t148757\t0.3\t1\t137734 11023\n",
      "28\t145635\t0.3\t1\t135492 10143\n",
      "29\t153188\t0.3\t1\t140948 12240\n",
      "30\t163373\t0.3\t1\t154030 9343\n",
      "31\t130689\t0.3\t1\t122129 8560\n",
      "32\t157390\t0.3\t1\t147156 10234\n",
      "33\t145226\t0.3\t1\t135269 9957\n",
      "34\t158524\t0.3\t1\t147826 10698\n",
      "35\t161582\t0.3\t1\t148734 12848\n",
      "36\t173098\t0.3\t1\t162838 10260\n",
      "37\t141921\t0.3\t1\t133053 8868\n",
      "38\t164933\t0.3\t1\t154468 10465\n",
      "39\t167185\t0.3\t1\t155913 11272\n",
      "40\t175799\t0.3\t1\t163207 12592\n",
      "41\t197239\t0.3\t1\t176427 20812\n",
      "42\t165247\t0.3\t1\t150517 14730\n",
      "43\t270149\t0.3\t1\t254538 15611\n",
      "44\t85600\t0.3\t1\t76961 8639\n",
      "45\t197628\t0.3\t1\t175697 21931\n",
      "46\t723044\t0.3\t1\t688287 34757\n",
      "47\t254889\t0.3\t1\t240372 14517\n",
      "48\t114855\t0.3\t1\t101708 13147\n",
      "49\t409546\t0.3\t1\t390013 19533\n",
      "50\t125438\t0.3\t1\t116192 9246\n",
      "51\t110069\t0.3\t1\t99667 10402\n",
      "52\t381945\t0.3\t1\t364475 17470\n",
      "53\t196688\t0.3\t1\t188239 8449\n",
      "54\t63003\t0.3\t1\t58174 4829\n",
      "55\t106109\t0.3\t1\t101117 4992\n",
      "56\t55521\t0.3\t1\t50154 5367\n",
      "57\t169184\t0.3\t1\t161794 7390\n",
      "58\t35182\t0.3\t1\t32212 2970\n",
      "59\t44244\t0.3\t1\t39185 5059\n",
      "60\t173147\t0.3\t1\t165425 7722\n",
      "61\t86440\t0.3\t1\t82402 4038\n",
      "62\t40559\t0.3\t1\t36479 4080\n",
      "63\t130914\t0.3\t1\t123404 7510\n",
      "64\t122315\t0.3\t1\t116337 5978\n",
      "65\t65805\t0.3\t1\t60428 5377\n",
      "66\t148115\t0.3\t1\t139100 9015\n",
      "67\t175930\t0.3\t1\t163369 12561\n",
      "68\t293688\t0.3\t1\t278052 15636\n",
      "69\t189726\t0.3\t1\t179114 10612\n",
      "70\t75474\t0.3\t1\t69457 6017\n",
      "71\t27345\t0.3\t1\t23952 3393\n",
      "72\t97969\t0.3\t1\t91968 6001\n",
      "73\t138145\t0.3\t1\t130111 8034\n",
      "74\t143624\t0.3\t1\t135279 8345\n",
      "75\t142674\t0.3\t1\t134066 8608\n",
      "76\t141682\t0.3\t1\t133308 8374\n",
      "77\t141205\t0.3\t1\t132947 8258\n",
      "78\t133036\t0.3\t1\t124966 8070\n",
      "79\t131630\t0.3\t1\t123633 7997\n",
      "80\t128859\t0.3\t1\t121116 7743\n",
      "81\t126119\t0.3\t1\t118667 7452\n",
      "82\t123492\t0.3\t1\t116262 7230\n",
      "83\t118847\t0.3\t1\t111615 7232\n",
      "84\t115583\t0.3\t1\t108642 6941\n",
      "85\t111409\t0.3\t1\t104817 6592\n",
      "86\t107099\t0.3\t1\t100545 6554\n",
      "87\t102731\t0.3\t1\t96477 6254\n",
      "88\t101527\t0.3\t1\t95245 6282\n",
      "89\t95383\t0.3\t1\t89600 5783\n",
      "90\t95037\t0.3\t1\t89230 5807\n",
      "91\t89836\t0.3\t1\t84395 5441\n",
      "92\t84977\t0.3\t1\t79865 5112\n",
      "93\t79479\t0.3\t1\t74762 4717\n",
      "94\t76672\t0.3\t1\t72010 4662\n",
      "95\t70676\t0.3\t1\t66313 4363\n",
      "96\t66886\t0.3\t1\t62766 4120\n",
      "97\t63105\t0.3\t1\t59150 3955\n",
      "98\t58813\t0.3\t1\t55101 3712\n",
      "99\t56801\t0.3\t1\t52921 3880\n",
      "100\t51251\t0.3\t1\t47973 3278\n",
      "101\t48131\t0.3\t1\t44969 3162\n",
      "102\t43975\t0.3\t1\t41134 2841\n",
      "103\t40749\t0.3\t1\t38132 2617\n",
      "104\t37602\t0.3\t1\t35059 2543\n",
      "105\t33601\t0.3\t1\t31318 2283\n",
      "106\t30687\t0.3\t1\t28622 2065\n",
      "107\t27550\t0.3\t1\t25698 1852\n",
      "108\t25919\t0.3\t1\t24135 1784\n",
      "109\t23271\t0.3\t1\t21584 1687\n",
      "110\t20524\t0.3\t1\t19077 1447\n",
      "111\t18563\t0.3\t1\t17263 1300\n",
      "112\t16204\t0.3\t1\t15036 1168\n",
      "113\t14110\t0.3\t1\t13017 1093\n",
      "114\t12321\t0.3\t1\t11394 927\n",
      "115\t10629\t0.3\t1\t9884 745\n",
      "116\t9483\t0.3\t1\t8756 727\n",
      "117\t8057\t0.3\t1\t7440 617\n",
      "118\t7067\t0.3\t1\t6491 576\n",
      "119\t5777\t0.3\t1\t5264 513\n",
      "120\t4917\t0.3\t1\t4511 406\n",
      "121\t4181\t0.3\t1\t3800 381\n",
      "122\t3506\t0.3\t1\t3174 332\n",
      "123\t2912\t0.3\t1\t2640 272\n",
      "124\t2546\t0.3\t1\t2297 249\n",
      "125\t2101\t0.3\t1\t1902 199\n",
      "126\t1717\t0.3\t1\t1548 169\n",
      "127\t1536\t0.3\t1\t1361 175\n",
      "128\t1201\t0.3\t1\t1095 106\n",
      "129\t1012\t0.3\t1\t912 100\n",
      "130\t739\t0.3\t1\t668 71\n",
      "131\t638\t0.3\t1\t572 66\n",
      "132\t553\t0.3\t1\t500 53\n",
      "133\t556\t0.3\t1\t500 56\n",
      "134\t665\t0.3\t1\t607 58\n",
      "135\t362\t0.3\t1\t324 38\n",
      "136\t207\t0.3\t1\t176 31\n",
      "137\t143\t0.3\t1\t128 15\n",
      "138\t99\t0.3\t1\t86 13\n",
      "139\t84\t0.3\t1\t72 12\n",
      "140\t58\t0.3\t1\t50 8\n",
      "141\t42\t0.3\t1\t38 4\n",
      "142\t37\t0.3\t1\t29 8\n",
      "143\t24\t0.3\t1\t22 2\n",
      "144\t25\t0.3\t1\t17 8\n",
      "145\t26\t0.3\t1\t22 4\n",
      "146\t29\t0.3\t1\t21 8\n",
      "147\t85\t0.3\t1\t77 8\n",
      "148\t64\t0.3\t1\t59 5\n",
      "149\t40\t0.3\t1\t35 5\n",
      "150\t355\t0.3\t1\t318 37\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/Users/strigg/Desktop/Salmo_Calig/TRIM/retained/16C_26psu_1_S13_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.6.0\n",
      "Cutadapt version: 2.4\n",
      "Number of cores used for trimming: 1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Length cut-off for read 1: 35 bp (default)\n",
      "Length cut-off for read 2: 35 bb (default)\n",
      "All Read 1 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 5 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "Running FastQC on the data once trimming has completed\n",
      "Running FastQC with the following extra arguments: '--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/retained/FASTQC'\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz <<< \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "This is cutadapt 2.4 with Python 3.6.8\n",
      "Command line parameters: -j 1 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in single-end mode ...\n",
      "Finished in 899.57 s (41 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              22,159,935\n",
      "Reads with adapters:                17,263,070 (77.9%)\n",
      "Reads written (passing filters):    22,159,935 (100.0%)\n",
      "\n",
      "Total basepairs processed: 3,323,990,250 bp\n",
      "Quality-trimmed:              16,760,179 bp (0.5%)\n",
      "Total written (filtered):  2,603,569,143 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17263070 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 12.4%\n",
      "  G: 23.7%\n",
      "  T: 35.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t2980698\t5539983.8\t0\t2980698\n",
      "2\t434290\t1384995.9\t0\t434290\n",
      "3\t244115\t346249.0\t0\t244115\n",
      "4\t143736\t86562.2\t0\t143736\n",
      "5\t130207\t21640.6\t0\t130207\n",
      "6\t112454\t5410.1\t0\t112454\n",
      "7\t91383\t1352.5\t0\t91383\n",
      "8\t130413\t338.1\t0\t130413\n",
      "9\t102897\t84.5\t0\t102495 402\n",
      "10\t113241\t21.1\t1\t107719 5522\n",
      "11\t119188\t5.3\t1\t111572 7616\n",
      "12\t120663\t1.3\t1\t112735 7928\n",
      "13\t133942\t0.3\t1\t125965 7977\n",
      "14\t126547\t0.3\t1\t117452 9095\n",
      "15\t102830\t0.3\t1\t96138 6692\n",
      "16\t149163\t0.3\t1\t139761 9402\n",
      "17\t119889\t0.3\t1\t111631 8258\n",
      "18\t113552\t0.3\t1\t106594 6958\n",
      "19\t144826\t0.3\t1\t135202 9624\n",
      "20\t135016\t0.3\t1\t126722 8294\n",
      "21\t118040\t0.3\t1\t110327 7713\n",
      "22\t135122\t0.3\t1\t126795 8327\n",
      "23\t139623\t0.3\t1\t130868 8755\n",
      "24\t139327\t0.3\t1\t130312 9015\n",
      "25\t156486\t0.3\t1\t147346 9140\n",
      "26\t131367\t0.3\t1\t122873 8494\n",
      "27\t144343\t0.3\t1\t134126 10217\n",
      "28\t135732\t0.3\t1\t128445 7287\n",
      "29\t144460\t0.3\t1\t135341 9119\n",
      "30\t143460\t0.3\t1\t135475 7985\n",
      "31\t154935\t0.3\t1\t145507 9428\n",
      "32\t140211\t0.3\t1\t132501 7710\n",
      "33\t154714\t0.3\t1\t145023 9691\n",
      "34\t197197\t0.3\t1\t185506 11691\n",
      "35\t114069\t0.3\t1\t107701 6368\n",
      "36\t154713\t0.3\t1\t145095 9618\n",
      "37\t166945\t0.3\t1\t157027 9918\n",
      "38\t155394\t0.3\t1\t147358 8036\n",
      "39\t151879\t0.3\t1\t143914 7965\n",
      "40\t168764\t0.3\t1\t158797 9967\n",
      "41\t162964\t0.3\t1\t154146 8818\n",
      "42\t166211\t0.3\t1\t157158 9053\n",
      "43\t152395\t0.3\t1\t144005 8390\n",
      "44\t174656\t0.3\t1\t164812 9844\n",
      "45\t186871\t0.3\t1\t176811 10060\n",
      "46\t135036\t0.3\t1\t127666 7370\n",
      "47\t168636\t0.3\t1\t159853 8783\n",
      "48\t166288\t0.3\t1\t157802 8486\n",
      "49\t174530\t0.3\t1\t165351 9179\n",
      "50\t164769\t0.3\t1\t155907 8862\n",
      "51\t207765\t0.3\t1\t197011 10754\n",
      "52\t147151\t0.3\t1\t139795 7356\n",
      "53\t177051\t0.3\t1\t168817 8234\n",
      "54\t133625\t0.3\t1\t126900 6725\n",
      "55\t174567\t0.3\t1\t166338 8229\n",
      "56\t178950\t0.3\t1\t169886 9064\n",
      "57\t168945\t0.3\t1\t161037 7908\n",
      "58\t161931\t0.3\t1\t154618 7313\n",
      "59\t165789\t0.3\t1\t157874 7915\n",
      "60\t171736\t0.3\t1\t163328 8408\n",
      "61\t175826\t0.3\t1\t167046 8780\n",
      "62\t192123\t0.3\t1\t182534 9589\n",
      "63\t252743\t0.3\t1\t241513 11230\n",
      "64\t207112\t0.3\t1\t199124 7988\n",
      "65\t135660\t0.3\t1\t129354 6306\n",
      "66\t145866\t0.3\t1\t139110 6756\n",
      "67\t145748\t0.3\t1\t138731 7017\n",
      "68\t146161\t0.3\t1\t139522 6639\n",
      "69\t144487\t0.3\t1\t137768 6719\n",
      "70\t154292\t0.3\t1\t147160 7132\n",
      "71\t158353\t0.3\t1\t151327 7026\n",
      "72\t150227\t0.3\t1\t143181 7046\n",
      "73\t157376\t0.3\t1\t150704 6672\n",
      "74\t144388\t0.3\t1\t138042 6346\n",
      "75\t137234\t0.3\t1\t131094 6140\n",
      "76\t133825\t0.3\t1\t127901 5924\n",
      "77\t133664\t0.3\t1\t127640 6024\n",
      "78\t127632\t0.3\t1\t121889 5743\n",
      "79\t122934\t0.3\t1\t117397 5537\n",
      "80\t122791\t0.3\t1\t117317 5474\n",
      "81\t119094\t0.3\t1\t113976 5118\n",
      "82\t115048\t0.3\t1\t109953 5095\n",
      "83\t110957\t0.3\t1\t106119 4838\n",
      "84\t107760\t0.3\t1\t102959 4801\n",
      "85\t105919\t0.3\t1\t100973 4946\n",
      "86\t104575\t0.3\t1\t99936 4639\n",
      "87\t105119\t0.3\t1\t100537 4582\n",
      "88\t102473\t0.3\t1\t97892 4581\n",
      "89\t90861\t0.3\t1\t86820 4041\n",
      "90\t87725\t0.3\t1\t83791 3934\n",
      "91\t82796\t0.3\t1\t79052 3744\n",
      "92\t76768\t0.3\t1\t73386 3382\n",
      "93\t74497\t0.3\t1\t71224 3273\n",
      "94\t71670\t0.3\t1\t68502 3168\n",
      "95\t67300\t0.3\t1\t64357 2943\n",
      "96\t63751\t0.3\t1\t61088 2663\n",
      "97\t58781\t0.3\t1\t56250 2531\n",
      "98\t54871\t0.3\t1\t52462 2409\n",
      "99\t53000\t0.3\t1\t50773 2227\n",
      "100\t48407\t0.3\t1\t46381 2026\n",
      "101\t45093\t0.3\t1\t43177 1916\n",
      "102\t40752\t0.3\t1\t38996 1756\n",
      "103\t37584\t0.3\t1\t35906 1678\n",
      "104\t35749\t0.3\t1\t34226 1523\n",
      "105\t31850\t0.3\t1\t30464 1386\n",
      "106\t29282\t0.3\t1\t28064 1218\n",
      "107\t26084\t0.3\t1\t24956 1128\n",
      "108\t23726\t0.3\t1\t22669 1057\n",
      "109\t21646\t0.3\t1\t20704 942\n",
      "110\t19323\t0.3\t1\t18515 808\n",
      "111\t18235\t0.3\t1\t17436 799\n",
      "112\t15797\t0.3\t1\t15095 702\n",
      "113\t13713\t0.3\t1\t13089 624\n",
      "114\t11968\t0.3\t1\t11416 552\n",
      "115\t10124\t0.3\t1\t9656 468\n",
      "116\t8777\t0.3\t1\t8354 423\n",
      "117\t7339\t0.3\t1\t6995 344\n",
      "118\t6452\t0.3\t1\t6137 315\n",
      "119\t5422\t0.3\t1\t5101 321\n",
      "120\t4499\t0.3\t1\t4281 218\n",
      "121\t3883\t0.3\t1\t3680 203\n",
      "122\t3313\t0.3\t1\t3156 157\n",
      "123\t2807\t0.3\t1\t2669 138\n",
      "124\t2400\t0.3\t1\t2285 115\n",
      "125\t2022\t0.3\t1\t1907 115\n",
      "126\t1650\t0.3\t1\t1555 95\n",
      "127\t1434\t0.3\t1\t1346 88\n",
      "128\t1129\t0.3\t1\t1055 74\n",
      "129\t969\t0.3\t1\t902 67\n",
      "130\t682\t0.3\t1\t644 38\n",
      "131\t587\t0.3\t1\t552 35\n",
      "132\t523\t0.3\t1\t500 23\n",
      "133\t524\t0.3\t1\t494 30\n",
      "134\t633\t0.3\t1\t604 29\n",
      "135\t337\t0.3\t1\t318 19\n",
      "136\t197\t0.3\t1\t186 11\n",
      "137\t147\t0.3\t1\t133 14\n",
      "138\t92\t0.3\t1\t88 4\n",
      "139\t79\t0.3\t1\t71 8\n",
      "140\t57\t0.3\t1\t52 5\n",
      "141\t37\t0.3\t1\t35 2\n",
      "142\t37\t0.3\t1\t35 2\n",
      "143\t25\t0.3\t1\t20 5\n",
      "144\t20\t0.3\t1\t20\n",
      "145\t26\t0.3\t1\t25 1\n",
      "146\t24\t0.3\t1\t23 1\n",
      "147\t85\t0.3\t1\t85\n",
      "148\t61\t0.3\t1\t59 2\n",
      "149\t39\t0.3\t1\t36 3\n",
      "150\t325\t0.3\t1\t310 15\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "22159935 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "file_1: 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz, file_2: 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Writing unpaired read 1 reads to 16C_26psu_1_S13_L001_R1_001_unpaired_1.fq.gz\n",
      "Writing unpaired read 2 reads to 16C_26psu_1_S13_L001_R2_001_unpaired_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 22159935\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 42248 (0.19%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R1_001_val_1.fq.gz\n",
      "\n",
      "  >>> Now running FastQC on the validated data 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz<<<\n",
      "\n",
      "Started analysis of 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 55% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for 16C_26psu_1_S13_L001_R2_001_val_2.fq.gz\n",
      "Deleting both intermediate output files 16C_26psu_1_S13_L001_R1_001_trimmed.fq.gz and 16C_26psu_1_S13_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!/Users/strigg/anaconda3/bin/TrimGalore-0.6.0/trim_galore \\\n",
    "--output_dir /Users/strigg/Desktop/Salmo_Calig/TRIM/retained \\\n",
    "--fastqc_args \\\n",
    "\"--outdir /Users/strigg/Desktop/Salmo_Calig/TRIM/retained/FASTQC\" \\\n",
    "--clip_R1 5 \\\n",
    "--clip_R2 5 \\\n",
    "--paired \\\n",
    "--retain_unpaired \\\n",
    "--path_to_cutadapt /Users/strigg/anaconda3/bin/cutadapt \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R1_001.fastq.gz \\\n",
    "/Users/strigg/Desktop/Salmo_Calig/16C_26psu_1_S13_L001_R2_001.fastq.gz "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
